
<html lang="en"     class="pb-page"  data-request-id="3439f2f6-62c8-4d81-b851-bcd15ed09767"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm401309q;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2013.56.issue-21;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Efficient Search of Chemical Space: Navigating from Fragments to Structurally Diverse Chemotypes" /></meta><meta name="dc.Creator" content="Anne Mai  Wassermann" /></meta><meta name="dc.Creator" content="Peter S.  Kutchukian" /></meta><meta name="dc.Creator" content="Eugen  Lounkine" /></meta><meta name="dc.Creator" content="Tiffany  Luethi" /></meta><meta name="dc.Creator" content="Jacques  Hamon" /></meta><meta name="dc.Creator" content="Michael T.  Bocker" /></meta><meta name="dc.Creator" content="Hasnain A.  Malik" /></meta><meta name="dc.Creator" content="Sandra W.  Cowan-Jacob" /></meta><meta name="dc.Creator" content="Meir  Glick" /></meta><meta name="dc.Description" content="We introduce a novel strategy to sample bioactive chemical space, which follows-up on hits from fragment campaigns without the need for a crystal structure. Our results strongly suggest that screen..." /></meta><meta name="Description" content="We introduce a novel strategy to sample bioactive chemical space, which follows-up on hits from fragment campaigns without the need for a crystal structure. Our results strongly suggest that screen..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 31, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm401309q" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm401309q" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm401309q" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm401309q" /></link>
        
    
    

<title>Efficient Search of Chemical Space: Navigating from Fragments to Structurally Diverse Chemotypes | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm401309q" /></meta><meta property="og:title" content="Efficient Search of Chemical Space: Navigating from Fragments to Structurally Diverse Chemotypes" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0008.jpeg" /></meta><meta property="og:description" content="We introduce a novel strategy to sample bioactive chemical space, which follows-up on hits from fragment campaigns without the need for a crystal structure. Our results strongly suggest that screening a few hundred or thousand fragments can substantially improve the selection of small-molecule screening subsets. By combining fragment-based screening with virtual fragment linking and HTS fingerprints, we have developed an effective strategy not only to expand from low-affinity hits to potent compounds but also to hop in chemical space to substantially novel chemotypes. In benchmark calculations, our approach accessed subsets of compounds that were substantially enriched in chemically diverse hit compounds for various activity classes. Overall, half of the hits in the screening collection were found by screening only 10% of the library. Furthermore, a prospective application led to the discovery of two structurally novel histone deacetylase 4 inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm401309q"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm401309q">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm401309q&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm401309q&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm401309q&amp;href=/doi/10.1021/jm401309q" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 8879-8891</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm401290y" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm401389u" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Efficient Search of Chemical Space: Navigating from Fragments to Structurally Diverse Chemotypes</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anne+Mai++Wassermann">Anne Mai Wassermann</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+S.++Kutchukian">Peter S. Kutchukian</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Eugen++Lounkine">Eugen Lounkine</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tiffany++Luethi">Tiffany Luethi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacques++Hamon">Jacques Hamon</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+T.++Bocker">Michael T. Bocker</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hasnain+A.++Malik">Hasnain A. Malik</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sandra+W.++Cowan-Jacob">Sandra W. Cowan-Jacob</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Meir++Glick">Meir Glick</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">â </span> <span class="aff-text">Novartis Institutes for Biomedical Research Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">â¡</span> <span class="aff-text">Novartis Institutes for Biomedical Research Inc., Novartis Campus, Basel 4056, Switzerland</span></div><div class="corresp-info"><strong>*</strong>Tel: +1 617 8715256; E-mail: <a href="/cdn-cgi/l/email-protection#6c0d02020933010d05421b0d1f1f091e010d02022c02031a0d1e18051f420f0301"><span class="__cf_email__" data-cfemail="e5848b8b80ba88848ccb92849696809788848b8ba58b8a938497918c96cb868a88">[emailÂ protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm401309q&amp;href=/doi/10.1021%2Fjm401309q" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 8879â8891</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 11, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 August 2013</li><li><span class="item_label"><b>Published</b> online</span>31 October 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 November 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm401309q" title="DOI URL">https://doi.org/10.1021/jm401309q</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8879%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAnne%2BMai%2BWassermann%252C%2BPeter%2BS.%2BKutchukian%252C%2BEugen%2BLounkine%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D21%26contentID%3Djm401309q%26title%3DEfficient%2BSearch%2Bof%2BChemical%2BSpace%253A%2BNavigating%2Bfrom%2BFragments%2Bto%2BStructurally%2BDiverse%2BChemotypes%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8891%26publicationDate%3DNovember%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm401309q"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1955</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm401309q" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Efficient Search of Chemical Space: Navigating from Fragments to Structurally Diverse Chemotypes&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Anne&quot;,&quot;last_name&quot;:&quot;Mai Wassermann&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;S. Kutchukian&quot;},{&quot;first_name&quot;:&quot;Eugen&quot;,&quot;last_name&quot;:&quot;Lounkine&quot;},{&quot;first_name&quot;:&quot;Tiffany&quot;,&quot;last_name&quot;:&quot;Luethi&quot;},{&quot;first_name&quot;:&quot;Jacques&quot;,&quot;last_name&quot;:&quot;Hamon&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;T. Bocker&quot;},{&quot;first_name&quot;:&quot;Hasnain&quot;,&quot;last_name&quot;:&quot;A. Malik&quot;},{&quot;first_name&quot;:&quot;Sandra&quot;,&quot;last_name&quot;:&quot;W. Cowan-Jacob&quot;},{&quot;first_name&quot;:&quot;Meir&quot;,&quot;last_name&quot;:&quot;Glick&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;8879-8891&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm401309q&quot;},&quot;abstract&quot;:&quot;We introduce a novel strategy to sample bioactive chemical space, which follows-up on hits from fragment campaigns without the need for a crystal structure. Our results strongly suggest that screening a few hundred or thousand fragments can substantially improve the selection of small-molecule screening subsets. By combining fragment-based screening with virtual fragment linking and HTS fingerprints, we have developed an effective strategy not only to expand from low-affinity hits to potent compounds but also to hop in chemical space to substantially novel chemotypes. In benchmark calculations, our approach accessed subsets of compounds that were substantially enriched in chemically diverse hit compounds for various activity classes. Overall, half of the hits in the screening collection were found by screening only 10% of the library. Furthermore, a prospective application led to the discovery of two structurally novel histone deacetylase 4 inhibitors.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm401309q&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm401309q" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm401309q&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm401309q" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm401309q&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm401309q" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm401309q&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm401309q&amp;href=/doi/10.1021/jm401309q" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm401309q" /></input><a href="/doi/pdf/10.1021/jm401309q" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm401309q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm401309q%26sid%3Dliteratum%253Aachs%26pmid%3D24117015%26genre%3Darticle%26aulast%3DWassermann%26date%3D2013%26atitle%3DEfficient%2BSearch%2Bof%2BChemical%2BSpace%253A%2BNavigating%2Bfrom%2BFragments%2Bto%2BStructurally%2BDiverse%2BChemotypes%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D21%26spage%3D8879%26epage%3D8891%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292276" title="Small molecules">Small molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jmcmar.2013.56.issue-21/production/jmcmar.2013.56.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We introduce a novel strategy to sample bioactive chemical space, which follows-up on hits from fragment campaigns without the need for a crystal structure. Our results strongly suggest that screening a few hundred or thousand fragments can substantially improve the selection of small-molecule screening subsets. By combining fragment-based screening with virtual fragment linking and HTS fingerprints, we have developed an effective strategy not only to expand from low-affinity hits to potent compounds but also to hop in chemical space to substantially novel chemotypes. In benchmark calculations, our approach accessed subsets of compounds that were substantially enriched in chemically diverse hit compounds for various activity classes. Overall, half of the hits in the screening collection were found by screening only 10% of the library. Furthermore, a prospective application led to the discovery of two structurally novel histone deacetylase 4 inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23211" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23211" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The ability to efficiently access a small molecule to perturb a biological state in a desirable manner stands as one of the long sought after goals in drug discovery. As a first step, an approach that has proven useful in modern drug discovery is to identify a protein that can be targeted in order to reverse a diseased state to a more desirable state. Once a putative protein target has been identified, the next challenge is the identification of chemical matter that binds to it and perturbs its activity. Indeed, vast amounts of resources have been invested in exploring how to best access a small molecule to perturb a target of interest. Although approaches have spanned seemingly unrelated research areas such as virtual docking<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> or de novo drug design<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> in the computational field and high-throughput screening<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> (HTS) or DNA-encoded chemical libraries<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> in the experimental arena, they all share one goal: to efficiently sample as much relevant chemical space as possible in order to access the most desirable ligand for the target at hand. Fragment-based lead discovery (FBLD) is an approach that shares this goal, and in many embodiments represents a synergistic marriage of experimental and computational techniques.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6-8)</a></div><div class="NLM_p">There are a number of advantages of FBLD that have strengthened its place in the modern toolbox of lead discovery. First, for a given sized library of compounds, fragments cover a greater fraction of possible chemical space when compared to typically larger drug-sized compounds.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Second, fragments tend to have a high ligand efficiency<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> for a target, and in many cases this high ligand efficiency can be maintained as the fragment is developed into a lead.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> However, despite these desirable aspects, there are a number of aspects of FBLD that have limited its potential impact on lead discovery. First, in many cases, a cocrystal structure is necessary for fragment hits to be progressed. When the binding mode of a fragment is unknown, the number of possible ways to grow the fragment is seemingly unlimited and can hinder its progression in a drug-discovery setting. Therefore, it is important to fix the fragment in the correct binding mode before growing it. This often relegates fragment screens to targets that have a soakable crystallography system established. Second, even with the aid of a crystal structure, it is often very difficult to follow-up a fragment hit and to grow or evolve it into a leadlike compound. For example, the fragment or analogues of the fragment may not be amenable to synthesis. Third, when a path to growing the fragment can be envisioned, in practice the evolution of a fragment into a leadlike compound is resource-intensive and time-consuming. There is frequently a need for chemical synthesis to access a compound with potency similar to a compound that might be identified through other means such as HTS. Fourth, it is hard to imagine certain types of chemotypes ever being obtained through FBLD, for example, a complex natural product with a few fused rings or a macrocyclic compound. This suggests that FBLD has bounds in terms of what subset of the chemical space it can capture.</div><div class="NLM_p">We sought to develop a method that leverages the advantages of FBLD while overcoming a number of barriers that can hinder its success. In the first step of our approach, information about desirable substructures is extracted from a fragment screen, and this information is used to select larger compounds containing these substructures (chemistry-based expansion). In the next step, the larger compounds are tested for activity, and information regarding the biological and biochemical activity enriched in the actives is used to select a subsequent set of small molecules for evaluation (biology-based expansion). At no point is crystallographic information employed in the models. We found that in the first expansion (chemistry-based) the desirable linking of the fragment substructures is revealed. Furthermore, in the second step, we were able to uncover the biological fingerprint of the active hits, and by expanding on the basis of these fingerprints, active compounds were identified that contain chemotypes not covered by the original fragment library. Indeed, some of these chemotypes are complex natural products that one would never imagine finding as a result of an effort initiated by FBLD.</div><div class="NLM_p last">Overall, a retrospective evaluation of our approach led to the conclusion that, by screening only 10% of the compound collection, half of the hits in the collection can be captured. Furthermore, we successfully employed our approach in the prospective identification of histone deacetylase (HDAC) 4 inhibitors, as reported herein.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04224" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04224" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">With this work, we wanted to learn if and how FBS hits can be expanded to cover a broad range of bioactive compounds. We tested our novel FBS follow-up strategy using a combination of chemistry- and biology-based expansion on 13 different benchmark sets covering targets from multiple protein families (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). For all but one target, a fragment campaign testing the Novartis fragment core library of â¼1400 fragments was run in-house. For one target, a significantly smaller library of 466 fragments was tested. Furthermore, for all FBS targets, a corresponding HTS campaign was carried out, which allowed us to retrospectively evaluate the ability of our approach to enrich small-molecule subsets with hits and to extrapolate to a chemical space greater than 3 orders of magnitude. To assess better the observed performance of our method, we compared it to two control strategies that used only chemical information for fragment-hit expansion. Special emphasis was given to the ability of the different approaches to sample chemically diverse bioactive molecules. Furthermore, we investigated our method in molecular detail by investigating the cocrystal structures of compounds identified by the method at various steps of the chemical space navigation for a kinase campaign. Finally, we prospectively tested our method against HDAC4.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Data Sets<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="5" align="center">Fragments</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">data set</th><th class="colsep0 rowsep0" align="center">abbr.</th><th class="colsep0 rowsep0" align="center">technology</th><th class="colsep0 rowsep0" align="center">no. fragments</th><th class="colsep0 rowsep0" align="center">HRÂ (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dehydrogenase</td><td class="colsep0 rowsep0" align="left">DH</td><td class="colsep0 rowsep0" align="left">fluorescence, DSF, NMR, SPR, LCâMS</td><td class="colsep0 rowsep0" align="left">1371</td><td class="colsep0 rowsep0" align="left">19.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">histone deacetylase</td><td class="colsep0 rowsep0" align="left">HDAC</td><td class="colsep0 rowsep0" align="left">fluorescence intensity</td><td class="colsep0 rowsep0" align="left">1370</td><td class="colsep0 rowsep0" align="left">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">histone demethylase</td><td class="colsep0 rowsep0" align="left">HDME</td><td class="colsep0 rowsep0" align="left">DSF, NMR, SPR, LCâMS, TR-FRET</td><td class="colsep0 rowsep0" align="left">1371</td><td class="colsep0 rowsep0" align="left">21.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">adenylyltransferase</td><td class="colsep0 rowsep0" align="left">ADT</td><td class="colsep0 rowsep0" align="left">DSF, NMR, fluorescence, Xtal</td><td class="colsep0 rowsep0" align="left">1380</td><td class="colsep0 rowsep0" align="left">15.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">histone methyltransferase</td><td class="colsep0 rowsep0" align="left">HMET</td><td class="colsep0 rowsep0" align="left">NMR, SPR, LCâMS, Xtal</td><td class="colsep0 rowsep0" align="left">1356</td><td class="colsep0 rowsep0" align="left">6.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">immunoglobulin</td><td class="colsep0 rowsep0" align="left">IG</td><td class="colsep0 rowsep0" align="left">NMR, SPR</td><td class="colsep0 rowsep0" align="left">1372</td><td class="colsep0 rowsep0" align="left">7.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine protease</td><td class="colsep0 rowsep0" align="left">SER PROT</td><td class="colsep0 rowsep0" align="left">NMR, SPR, fluorescence, MSA</td><td class="colsep0 rowsep0" align="left">1371</td><td class="colsep0 rowsep0" align="left">17.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine/threonine kinase 1</td><td class="colsep0 rowsep0" align="left">SER/THR 1</td><td class="colsep0 rowsep0" align="left">NMR,SPR, Xtal</td><td class="colsep0 rowsep0" align="left">1371</td><td class="colsep0 rowsep0" align="left">46.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine/threonine kinase 2</td><td class="colsep0 rowsep0" align="left">SER/THR 2</td><td class="colsep0 rowsep0" align="left">DSF, NMR, MSA</td><td class="colsep0 rowsep0" align="left">1371</td><td class="colsep0 rowsep0" align="left">9.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine/threonine kinase 3</td><td class="colsep0 rowsep0" align="left">SER/THR 3</td><td class="colsep0 rowsep0" align="left">NMR, SPR, AlphaScreen</td><td class="colsep0 rowsep0" align="left">1380</td><td class="colsep0 rowsep0" align="left">36.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine/threonine kinase 4</td><td class="colsep0 rowsep0" align="left">SER/THR 4</td><td class="colsep0 rowsep0" align="left">NMR, Xtal</td><td class="colsep0 rowsep0" align="left">466</td><td class="colsep0 rowsep0" align="left">14.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tyrosine kinase 1</td><td class="colsep0 rowsep0" align="left">TYR 1</td><td class="colsep0 rowsep0" align="left">NMR, fluorescence polarization, MSA</td><td class="colsep0 rowsep0" align="left">1371</td><td class="colsep0 rowsep0" align="left">12.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tyrosine kinase 2</td><td class="colsep0 rowsep0" align="left">TYR 2</td><td class="colsep0 rowsep0" align="left">SPR, TR-FRET, MSA</td><td class="colsep0 rowsep0" align="left">1381</td><td class="colsep0 rowsep0" align="left">23.0</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="7" align="center">Small Molecules</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="2" align="center">with frequent hitters</th><th class="rowsep1 colsep0" colspan="2" align="center">without frequent hitters</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">data set</th><th class="colsep0 rowsep0" align="center">abbr.</th><th class="colsep0 rowsep0" align="center">technology</th><th class="colsep0 rowsep0" align="center">no. mol.</th><th class="colsep0 rowsep0" align="center">HRÂ (%)</th><th class="colsep0 rowsep0" align="center">no. mol.</th><th class="colsep0 rowsep0" align="center">HRÂ (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dehydrogenase</td><td class="colsep0 rowsep0" align="left">DH</td><td class="colsep0 rowsep0" align="left">fluorescence</td><td class="colsep0 rowsep0" align="left">1â260â224</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left">1â228â468</td><td class="colsep0 rowsep0" align="left">0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">histone deacetylase</td><td class="colsep0 rowsep0" align="left">HDAC</td><td class="colsep0 rowsep0" align="left">fluorescence</td><td class="colsep0 rowsep0" align="left">1â359â660</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">1â325â744</td><td class="colsep0 rowsep0" align="left">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">histone demethylase</td><td class="colsep0 rowsep0" align="left">HDME</td><td class="colsep0 rowsep0" align="left">TR-FRET</td><td class="colsep0 rowsep0" align="left">1â258â967</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">1â227â278</td><td class="colsep0 rowsep0" align="left">0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">adenylyltransferase</td><td class="colsep0 rowsep0" align="left">ADT</td><td class="colsep0 rowsep0" align="left">fluorescence</td><td class="colsep0 rowsep0" align="left">1â290â512</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">1â256â944</td><td class="colsep0 rowsep0" align="left">1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">histone methyltransferase</td><td class="colsep0 rowsep0" align="left">HMET</td><td class="colsep0 rowsep0" align="left">AlphaScreen</td><td class="colsep0 rowsep0" align="left">1â265â344</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">1â233â470</td><td class="colsep0 rowsep0" align="left">0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">immunoglobulin</td><td class="colsep0 rowsep0" align="left">IG</td><td class="colsep0 rowsep0" align="left">TR-FRET</td><td class="colsep0 rowsep0" align="left">1â320â15</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">1â287â058</td><td class="colsep0 rowsep0" align="left">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine protease</td><td class="colsep0 rowsep0" align="left">SER PROT</td><td class="colsep0 rowsep0" align="left">fluorescence</td><td class="colsep0 rowsep0" align="left">896â755</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">881â338</td><td class="colsep0 rowsep0" align="left">1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine/threonine kinase 1</td><td class="colsep0 rowsep0" align="left">SER/THR 1</td><td class="colsep0 rowsep0" align="left">luminescence</td><td class="colsep0 rowsep0" align="left">1â279â524</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">1â246â943</td><td class="colsep0 rowsep0" align="left">1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine/threonine kinase 2</td><td class="colsep0 rowsep0" align="left">SER/THR 2</td><td class="colsep0 rowsep0" align="left">luminescence</td><td class="colsep0 rowsep0" align="left">1â175â181</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">1â144â281</td><td class="colsep0 rowsep0" align="left">0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine/threonine kinase 3</td><td class="colsep0 rowsep0" align="left">SER/THR 3</td><td class="colsep0 rowsep0" align="left">luminescence</td><td class="colsep0 rowsep0" align="left">1â258â684</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">1â226â089</td><td class="colsep0 rowsep0" align="left">1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">serine/threonine kinase 4</td><td class="colsep0 rowsep0" align="left">SER/THR 4</td><td class="colsep0 rowsep0" align="left">TR-FRET</td><td class="colsep0 rowsep0" align="left">929â479</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">910â828</td><td class="colsep0 rowsep0" align="left">0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tyrosine kinase 1</td><td class="colsep0 rowsep0" align="left">TYR 1</td><td class="colsep0 rowsep0" align="left">TR-FRET</td><td class="colsep0 rowsep0" align="left">1â119â051</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">1â097â437</td><td class="colsep0 rowsep0" align="left">0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tyrosine kinase 2</td><td class="colsep0 rowsep0" align="left">TYR 2</td><td class="colsep0 rowsep0" align="left">TR-FRET</td><td class="colsep0 rowsep0" align="left">1â335â683</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">1â301â341</td><td class="colsep0 rowsep0" align="left">0.8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">For the 13 benchmark data sets used in our study, abbreviations (abbr.), numbers of screened fragments (no. fragments) and small molecules (no. mol), and their hit rates (HR (%)) are reported. Moreover, the assay technologies used in fragment-based screening and small-molecule HTS are given. DSF, differential scanning fluorimetry; LCâMS, liquid chromatographyâmass spectrometry; MSA, mobility-shift assay; NMR, nuclear magnetic resonance; SPR, surface plasmon resonance; TR-FRET, time-resolved fluorescence resonance energy transfer; and xtal, crystal.</p></div></div></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Global Performance</h3><div class="NLM_p">We use an iterative expansion process to select a small-molecule subset for screening (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A). In the first step, we apply virtual fragment linking<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (VFL), a method that was introduced to learn bioactivity-relevant substructures from fragments for the subsequent selection of a focused set of small molecules. More precisely, substructures (ECFP4<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> features; see the <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>) are systematically extracted from active and inactive fragments to build a naÄ±Ìve Bayesian (NB) classifier<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> that is applied to rank the small-molecule screening collection. The top 5000 molecules, which typically contain multiple active fragment substructures, are then chosen for testing. In a second expansion step, bioactivity profiles termed HTS fingerprints<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> (HTS-FPs) of active and inactive compounds are used to train a second NB classifier that is ignorant of the chemical structure of the molecules. Rather, the classifier learns activity patterns that distinguish active from inactive compounds based on historical HTS campaigns run at Novartis. The remaining screening collection is subsequently ranked using this classifier. Small molecules that show characteristic bioactivity patterns of active compounds from the tested 5000 compound set are picked for the next screening round. As controls, we tested two methods that employed only chemical expansion (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B,C).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fragment-hit expansion strategies. Shown are different strategies to explore bioactive small-molecule space starting from FBS. (a) Strategy VFL + HTS-FP uses a combination of chemistry- and biology-based expansion to select small molecules for testing. In the first step, chemical features are extracted from active and inactive fragments and used to train a model that is applied to the small-molecule screening collection. The top-ranked molecules are screened and then divided into an active and inactive training set to build a second classifier using bioactivity profiles as molecular descriptors. On the basis of this classifier, a second set of molecules is chosen for testing. (b) Strategy VFL selects the entire screening subset based on the chemistry-based expansion of fragments. (c) Strategy VFL + ECFP4 uses two expansion steps and is similar to VFL + HTS-FP but uses chemical features instead of bioactivity profiles to train the second model (i.e., both expansion steps are chemistry-based).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To obtain a first performance assessment for each benchmark set and the VFL + HTS-FP method, we selected the top 1, 5, or 10% of the screened collection by two-step expansion and calculated the percentage of all active compounds that was found in the selected subset (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Because the total number of screened molecules differed for the 13 benchmark sets, selected compound subsets were also variable in size. For example, 1% of the screened compounds ranged from 8813 molecules for the serine protease set (SER PROT) to 13â257 molecules for the HDAC set. On average, 14.7, 35.3, and 45.6% of all hits were recovered by screening only 1, 5, or 10% of the full sets. For the kinase set SER/THR 4, our method showed a surprisingly low performance compared to other kinase sets. This was most likely due to the combination of a limited fragment library screened in this single case (only 466 fragments) and, for kinases, a comparably low hit rate (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). The low percent of hits found for the HMET set may be a result of different domains being screened in FBS and HTS. The average performance of the 11 remaining sets was 16.4, 38.8, and 49.5%, respectively. Taken together, screening only 10% of the full library led to the discovery of half of the hits.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Recovery rates at different selection set sizes. Fractions of identified active molecules are reported for subsets comprising 1, 5, or 10% of the full screening sets.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Kinase Case Study</h3><div class="NLM_p">We wanted to explore on a structural level how substructures identified as active from FBS were ultimately incorporated into subsequent compounds identified during expansion. Fragments tend to bind to a hot spot of a protein, and we wondered whether the active fragment substructures contained within small molecules were also located in this region or whether there was no relationship. To do this, we analyzed fragment and small-molecule hits that would be identified by our method (FBS, VFL, HTS-FP) for the SER/THR 1 campaign; the hits had been cocrystallized with either the target of interest or other kinases. Because we cannot disclose the identity of the SER/THR 1 target, a tyrosine kinase-like kinase (TKL), we chose to study hits from this analysis in complex with either other TKL members (interleukin-1 receptor-associated kinase 4 (IRAK-4)) or related tyrosine kinase (TK) members (lymphocyte-specific protein tyrosine kinase (Lck), epidermal growth factor receptor (EGFR)).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Analysis of fragment cocrystal structures with SER/THR 1 revealed that fragments bound mostly to the hinge region of the kinase, forming hydrogen bonds with backbone amides. Interestingly, enriched fragment substructures in small molecules obtained from VFL also resided in this region, or hot spot, whereas other moieties on the compounds explored other regions of the binding pocket. For example, representative VFL hit <b>4</b> is depicted in complex with Lck, where it is apparent that the enriched fragment substructures (colored green in the 2D structure) maintain hydrogen bonds with the Lck backbone in the hot spot (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A). When biology-based expansion was subsequently used, novel chemotypes such as that demonstrated by staurosporine <b>5</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B) also explore this hot spot as well as other regions of the binding pocket. Furthermore, we realized that compounds that were identified from biological expansion often contained substructures that had been enriched in fragment hits in the hot-spot region while exploring other parts of the binding pocket with substructures that FBS identified as unfavorable for the hotspot. For example, Gefitinib <b>6</b>, which is depicted in complex with EGFR (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C), contains favorable fragment substructures that interact intimately with the hinge region as well as unfavorable fragment substructures that reside away from the hot spot. Although it might be due solely to chance that these unfavorable substructures were present in the biological expansion hits, it is notable because the presence of the unfavorable substructures would preclude them from being identified earlier during the chemical expansion. Thus, the biological expansion can âsaveâ compounds containing favorable substructures that might have been missed in the chemical expansion as a result of the concomitant presence of unfavorable substructures because the presence of the latter substructures (whether they reside in the hinge region or not) will lower the overall score of the compound during chemical expansion.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. From fragments to diverse chemotypes. For the data set SER/THR 1, exemplary structures of active ligands discovered in the different steps of the VFL + HTS-FP approach are shown. In the skeletal formulas, substructures identified as enriched in the FBS hits (favorable) are rendered green, whereas those identified as enriched in nonhits (unfavorable) are rendered red. Cocrystal structures of some of the identified ligands bound to the conserved ATP site of similar kinases (although not identical) are used to illustrate the likely binding pose of the compounds to the serine/threonine kinase that was screened. For compound <b>4</b>, identified by VFL, the favorable substructures intimately interact with the hinge region of Lck (A; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3F">4C3F</a>). When HTS fingerprint expansion was employed, the new chemotype contained in compound <b>5</b> was identified, and likewise portions of <b>5</b> make contacts with the hinge region of IRAK-4 (B; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2NRY">2NRY</a>). For Compound <b>6</b>, substructures identified as favorable in FBS hits interact with the hinge region, whereas an unfavorable substructure is also present that interacts elsewhere in the ATP-binding pocket of EGFR (C; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ITY">2ITY</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In summary, this case study illustrates that substructures enriched in FBS hits have been observed to reside in a similar region when contained in compounds identified by VFL. Furthermore, it demonstrates that the subsequent biological expansion identifies novel chemotypes that explore this region, and other regions of the binding pocket, as well as identifying compounds that were missed by VFL because they paired both favorable (exploring hot spot) and unfavorable (exploring other binding-pocket regions) fragment substructures.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Performance Contributions</h3><div class="NLM_p">We then wanted to investigate, more generally, how the individual parts of our approach contributed to overall enrichments. For this purpose, numbers of active molecules found by the different steps were analyzed in more detail, and single components of the method were replaced in control calculations.</div><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7">(i) VFL</h4><div class="NLM_p">First, we analyzed the number of active molecules found in the individual expansion steps when a fixed-sized subset of 20â000 molecules was selected per benchmark set (5000 molecules were chosen by chemistry-based expansion and 15â000 molecules were chosen by biology-based expansion; Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). In the first step, the minimum number of discovered active compounds was 16 for the set DH. Hence, in all cases, sufficient numbers of active compounds were found to build an active training class for a bioactivity-profile-based small-molecule model. On average, active compounds were 4.4-fold enriched in the sets of 5000 (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), in accordance with the previously demonstrated ability of VFL to extrapolate from fragments to small molecules and enrich subsets with bioactive compounds.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In only one instance (IG set) did VFL retrieve less active compounds than expected by random compound selection (i.e., the fraction of active molecules in the selected set was lower than in the entire screening collection). To understand better the failure of VFL in this case, we analyzed the NB classifier built on the active and inactive fragments for the target IG. We observed that only very few substructures obtained positive weights (i.e., were identified as favorable) in the training of the classifier. In other words, there was not much structural consensus among the FBS hits, and not many chemical features could be detected that were enriched among the active fragments. Furthermore, enrichment factors were very modest for the sets HDAC and HMET (1.16 and 1.26, respectively). For HMET, the inability of VFL to lead to a substantial increase in hits might be attributed to different protein domains used to identify ligands in FBS and small-molecule HTS. For SER/THR 4, using only 466 fragments to train the classifier led to a 2-fold enrichment of active compounds.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Hit Statistics<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="3" align="center">VFL + HTS-FP</th><th class="rowsep1 colsep0" colspan="3" align="center">VFL + ECFP4</th><th class="rowsep1 colsep0" align="center">VFL</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">data set</th><th class="colsep0 rowsep0" align="center">step 1</th><th class="colsep0 rowsep0" align="center">step 2</th><th class="colsep0 rowsep0" align="center">sum</th><th class="colsep0 rowsep0" align="center">step 1</th><th class="colsep0 rowsep0" align="center">step 2</th><th class="colsep0 rowsep0" align="center">sum</th><th class="colsep0 rowsep0" align="center">sum</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" colspan="7" align="center">Absolute Hit Numbers</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DH</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">222</td><td class="colsep0 rowsep0" align="left">238</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="left">105</td><td class="colsep0 rowsep0" align="left">65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">387</td><td class="colsep0 rowsep0" align="left">408</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">99</td><td class="colsep0 rowsep0" align="left">120</td><td class="colsep0 rowsep0" align="left">92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDME</td><td class="colsep0 rowsep0" align="left">147</td><td class="colsep0 rowsep0" align="left">3341</td><td class="colsep0 rowsep0" align="left">3488</td><td class="colsep0 rowsep0" align="left">147</td><td class="colsep0 rowsep0" align="left">2984</td><td class="colsep0 rowsep0" align="left">3131</td><td class="colsep0 rowsep0" align="left">417</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ADT</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">1928</td><td class="colsep0 rowsep0" align="left">2028</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">709</td><td class="colsep0 rowsep0" align="left">809</td><td class="colsep0 rowsep0" align="left">399</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HMET</td><td class="colsep0 rowsep0" align="left">53</td><td class="colsep0 rowsep0" align="left">635</td><td class="colsep0 rowsep0" align="left">688</td><td class="colsep0 rowsep0" align="left">53</td><td class="colsep0 rowsep0" align="left">284</td><td class="colsep0 rowsep0" align="left">337</td><td class="colsep0 rowsep0" align="left">210</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IG</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">1259</td><td class="colsep0 rowsep0" align="left">1278</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">144</td><td class="colsep0 rowsep0" align="left">163</td><td class="colsep0 rowsep0" align="left">53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER PROT</td><td class="colsep0 rowsep0" align="left">410</td><td class="colsep0 rowsep0" align="left">2809</td><td class="colsep0 rowsep0" align="left">3219</td><td class="colsep0 rowsep0" align="left">410</td><td class="colsep0 rowsep0" align="left">1137</td><td class="colsep0 rowsep0" align="left">1547</td><td class="colsep0 rowsep0" align="left">1090</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 1</td><td class="colsep0 rowsep0" align="left">1003</td><td class="colsep0 rowsep0" align="left">4414</td><td class="colsep0 rowsep0" align="left">5417</td><td class="colsep0 rowsep0" align="left">1003</td><td class="colsep0 rowsep0" align="left">3459</td><td class="colsep0 rowsep0" align="left">4462</td><td class="colsep0 rowsep0" align="left">2480</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 2</td><td class="colsep0 rowsep0" align="left">273</td><td class="colsep0 rowsep0" align="left">2430</td><td class="colsep0 rowsep0" align="left">2703</td><td class="colsep0 rowsep0" align="left">273</td><td class="colsep0 rowsep0" align="left">1178</td><td class="colsep0 rowsep0" align="left">1451</td><td class="colsep0 rowsep0" align="left">834</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 3</td><td class="colsep0 rowsep0" align="left">308</td><td class="colsep0 rowsep0" align="left">2332</td><td class="colsep0 rowsep0" align="left">2640</td><td class="colsep0 rowsep0" align="left">308</td><td class="colsep0 rowsep0" align="left">902</td><td class="colsep0 rowsep0" align="left">1210</td><td class="colsep0 rowsep0" align="left">901</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 4</td><td class="colsep0 rowsep0" align="left">91</td><td class="colsep0 rowsep0" align="left">821</td><td class="colsep0 rowsep0" align="left">912</td><td class="colsep0 rowsep0" align="left">91</td><td class="colsep0 rowsep0" align="left">310</td><td class="colsep0 rowsep0" align="left">401</td><td class="colsep0 rowsep0" align="left">321</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TYRÂ 1</td><td class="colsep0 rowsep0" align="left">172</td><td class="colsep0 rowsep0" align="left">1016</td><td class="colsep0 rowsep0" align="left">1188</td><td class="colsep0 rowsep0" align="left">172</td><td class="colsep0 rowsep0" align="left">612</td><td class="colsep0 rowsep0" align="left">784</td><td class="colsep0 rowsep0" align="left">414</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TYRÂ 2</td><td class="colsep0 rowsep0" align="left">115</td><td class="colsep0 rowsep0" align="left">2167</td><td class="colsep0 rowsep0" align="left">2282</td><td class="colsep0 rowsep0" align="left">115</td><td class="colsep0 rowsep0" align="left">1563</td><td class="colsep0 rowsep0" align="left">1678</td><td class="colsep0 rowsep0" align="left">484</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" colspan="7" align="center">Hit Rates (%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DH</td><td class="colsep0 rowsep0" align="left">0.3</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">0.3</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDME</td><td class="colsep0 rowsep0" align="left">2.9</td><td class="colsep0 rowsep0" align="left">22.3</td><td class="colsep0 rowsep0" align="left">17.4</td><td class="colsep0 rowsep0" align="left">2.9</td><td class="colsep0 rowsep0" align="left">19.9</td><td class="colsep0 rowsep0" align="left">15.7</td><td class="colsep0 rowsep0" align="left">2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ADT</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">12.9</td><td class="colsep0 rowsep0" align="left">10.1</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">4.7</td><td class="colsep0 rowsep0" align="left">4.0</td><td class="colsep0 rowsep0" align="left">2.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HMET</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">4.2</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IG</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">8.4</td><td class="colsep0 rowsep0" align="left">6.4</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER PROT</td><td class="colsep0 rowsep0" align="left">8.2</td><td class="colsep0 rowsep0" align="left">18.7</td><td class="colsep0 rowsep0" align="left">16.1</td><td class="colsep0 rowsep0" align="left">8.2</td><td class="colsep0 rowsep0" align="left">7.6</td><td class="colsep0 rowsep0" align="left">7.7</td><td class="colsep0 rowsep0" align="left">5.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 1</td><td class="colsep0 rowsep0" align="left">20.1</td><td class="colsep0 rowsep0" align="left">29.4</td><td class="colsep0 rowsep0" align="left">27.1</td><td class="colsep0 rowsep0" align="left">20.1</td><td class="colsep0 rowsep0" align="left">23.1</td><td class="colsep0 rowsep0" align="left">22.3</td><td class="colsep0 rowsep0" align="left">12.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 2</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">16.2</td><td class="colsep0 rowsep0" align="left">13.5</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">7.9</td><td class="colsep0 rowsep0" align="left">7.3</td><td class="colsep0 rowsep0" align="left">4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 3</td><td class="colsep0 rowsep0" align="left">6.2</td><td class="colsep0 rowsep0" align="left">15.5</td><td class="colsep0 rowsep0" align="left">13.2</td><td class="colsep0 rowsep0" align="left">6.2</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">6.1</td><td class="colsep0 rowsep0" align="left">4.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 4</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TYRÂ 1</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">6.8</td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">3.9</td><td class="colsep0 rowsep0" align="left">2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TYRÂ 2</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">14.4</td><td class="colsep0 rowsep0" align="left">11.4</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">10.4</td><td class="colsep0 rowsep0" align="left">8.4</td><td class="colsep0 rowsep0" align="left">2.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" colspan="7" align="center">Enrichment Factors</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DH</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">7.4</td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">2.9</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDAC</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">7.1</td><td class="colsep0 rowsep0" align="left">5.6</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HDME</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left">38.9</td><td class="colsep0 rowsep0" align="left">30.4</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left">34.7</td><td class="colsep0 rowsep0" align="left">27.3</td><td class="colsep0 rowsep0" align="left">3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ADT</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">12.2</td><td class="colsep0 rowsep0" align="left">9.7</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">4.5</td><td class="colsep0 rowsep0" align="left">3.9</td><td class="colsep0 rowsep0" align="left">1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HMET</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IG</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">18.1</td><td class="colsep0 rowsep0" align="left">13.8</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER PROT</td><td class="colsep0 rowsep0" align="left">7.4</td><td class="colsep0 rowsep0" align="left">16.9</td><td class="colsep0 rowsep0" align="left">14.6</td><td class="colsep0 rowsep0" align="left">7.4</td><td class="colsep0 rowsep0" align="left">6.9</td><td class="colsep0 rowsep0" align="left">7.0</td><td class="colsep0 rowsep0" align="left">4.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 1</td><td class="colsep0 rowsep0" align="left">14.7</td><td class="colsep0 rowsep0" align="left">21.6</td><td class="colsep0 rowsep0" align="left">19.9</td><td class="colsep0 rowsep0" align="left">14.7</td><td class="colsep0 rowsep0" align="left">17.0</td><td class="colsep0 rowsep0" align="left">16.4</td><td class="colsep0 rowsep0" align="left">9.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 2</td><td class="colsep0 rowsep0" align="left">6.9</td><td class="colsep0 rowsep0" align="left">20.6</td><td class="colsep0 rowsep0" align="left">17.2</td><td class="colsep0 rowsep0" align="left">6.9</td><td class="colsep0 rowsep0" align="left">10.0</td><td class="colsep0 rowsep0" align="left">9.2</td><td class="colsep0 rowsep0" align="left">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 3</td><td class="colsep0 rowsep0" align="left">6.2</td><td class="colsep0 rowsep0" align="left">15.5</td><td class="colsep0 rowsep0" align="left">13.2</td><td class="colsep0 rowsep0" align="left">6.2</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">4.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SER/THRÂ 4</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">6.2</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TYRÂ 1</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="left">10.3</td><td class="colsep0 rowsep0" align="left">9.1</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="left">6.2</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">3.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TYRÂ 2</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">17.5</td><td class="colsep0 rowsep0" align="left">13.9</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">12.7</td><td class="colsep0 rowsep0" align="left">10.2</td><td class="colsep0 rowsep0" align="left">2.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">For the three investigated fragment-hit expansion strategies, absolute numbers of recovered active molecules, hit rates, and enrichment factors over a random compound selection are reported for a set of 20â000 tested molecules (column âsumâ). The strategies VFL + HTS-FP and VFL + ECFP4 consist of two consecutive expansion steps. The column âstep 1â reports the number of active molecules, hit rates, and enrichment factors found among the top 5000 compounds that were selected using VFL. The column âstep 2â gives the number of active molecules, hit rates, and enrichment factors among the subsequently tested 15â000 compounds chosen on the basis of a small-molecule model. Data set abbreviations are used according to Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>.</p></div></div></div><div class="NLM_p last">Hence, overall, VFL discovered more active molecules than expected by a random compound collection and provided a larger active training set for learning in the second model-building step. Effects of this extended active training set on the second expansion step will be discussed later in this article.</div></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">(ii) Biology-Based Expansion</h4><div class="NLM_p">Compound sets were then expanded by training classifiers on the bioactivity profiles of the 5000 tested molecules. Compound subsets selected on the basis of VFL + HTS-FP outperformed sets assembled by selecting the 20â000 compounds best-ranked by VFL (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B) (i.e., an expansion strategy using the chemical information of the fragments only) by up to 24-fold (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). Although, from a practical point of view, testing all molecules in one step might be preferred, the results clearly demonstrate that an iterative expansion process increased hit rates substantially. For VFL + HTS-FP, when considering the 20â000 tested compounds as a whole, top enrichment factors of 30.4, 19.9, and 17.2 were obtained for the sets HDME and SER/THR 1 and 2, respectively. To see whether the combination of two expansion steps that use orthogonal views on molecular similarity was crucial to the success of VFL + HTS-FP, we replaced the biology-based expansion step by another chemistry-based expansion step (i.e., we replaced HTS-FPs by structural features (ECFP4 fingerprint) to select 15,000 molecules in the second step (VFL + ECFP4; Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C)). We observed that bioactivity profiles as small-molecule descriptors consistently led to the discovery of more active compounds than structural fingerprints (i.e., VFL + HTS-FP exceeded the performance of VFL + ECFP4) (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). Importantly, the higher number of retrieved hits also translated to a higher number of discovered chemotypes, here measured as the number of unique Bemis and Murcko scaffolds<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> found among the active molecules. For five data sets, VFL + HTS-FP discovered more than 30% of the active scaffolds in the screened collection by testing only 20â000 molecules (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). In absolute numbers, a maximum of 2857 different active scaffolds were identified for the set SER/THR 1.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Active compound and scaffold retrieval for different selection strategies. For each data set, enrichment factors of active compounds in sets selected by VFL, VFL + HTS-FP, or VFL + ECFP4 over randomly chosen screening sets of the same size are displayed (left plot for each target). For all active compounds in a data set, scaffolds are calculated, and the percentages of active scaffolds retrieved at different screening set sizes are shown for the three different strategies (right plot for each target). Data set designations are used according to Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To visualize how VFL + HTS-FP samples bioactive chemical space, we projected active compounds in three dimensions using multidimensional scaling (MDS; see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>)<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> on the basis of pairwise compound structure similarities (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). Active molecules (all points in the plot) were colored according to whether they had been discovered by fragment-hit expansion or not. Compounds identified by chemistry-based expansion (VFL) were colored in red, compounds found by biology-based expansion (HTS-FP) were colored in green, and active compound that remained unidentified were colored in blue. For the data set IG, active compounds populated two distinct regions of chemical space (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). Chemistry-based expansion of fragment hits was able to identify only active compounds concentrated in a very narrow area of chemical space. However, as nonstructural descriptors, HTS-FPs in biology-based expansion were able to reach out to active compounds further away in chemical space (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A, left), and chemically very diverse hits populating both active islands in chemical space were found.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Sampling bioactive chemical space. For the data sets (a) IG and (b) HDAC, active molecules are projected into 3D chemical space using multidimensional scaling (MDS; three coordinates of MDS: CMDS1, CMDS2, and CMDS3). For strategies VFL + HTSP-FP and VFL + ECFP4, compounds are colored according to the set to which they belong: found by VFL (red), found by second biology- or chemistry-based expansion step (green), or not tested (blue). For both data sets, the second expansion step was very critical for the overall performance of our approach because the first step (VFL) failed or showed only limited enrichment of active compounds. The figure demonstrates the limited capability of the chemistry-based expansion step to move into new chemical space which is, in most cases, resolved by HTS-FP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10">(iii) Differences in Sampling of Chemical Space by Chemistry- and Biology-Based Expansions</h4><div class="NLM_p last">By contrast, when structural descriptors were used for the second expansion step (strategy VFL + ECFP4; here molecules shown in green were discovered by the second chemistry-based expansion step), the discovered active compounds mainly surrounded the hits found by the first expansion step, and only a very small and focused region of the bioactive chemical landscape was explored (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A, right). Accordingly, VFL + HTS-FP sampled a much broader region of chemical space than VFL + ECFP4 that used chemical information only. Similar trends were found for HDAC (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B). The hits found by VFL clustered together in a very narrow region of chemical space. Whereas biology-based expansion was able to retrieve hits from very different regions in chemical space (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B, left), a second chemistry-based expansion step mainly led to the discovery of active compounds that were closely located to each other (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B, right) These observations show that it is the combination of orthogonal views on molecular similarity (chemical and biological) in the two hit-expansion steps that is crucial for a successful follow-up on fragment hits.</div></div><div id="sec2_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11">(iv) Screening Set Sizes</h4><div class="NLM_p">For SER/THR 4, screening of only 466 fragments still led to a 2-fold enrichment of hits by VFL. To analyze required fragment library sizes for our approach more systematically, we recalculated results for VFL + HTS-FP for a test set of 20â000 molecules when only half of the Novartis core library was used in FBS. (For each of the 12 data sets that had used the full fragment library, three trials with a random selection of only half of the fragments were carried out, and results were averaged over the three trials; <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Table S1">Supporting Information Table S1</a>.) Perhaps surprisingly, numbers of active compounds found by VFL were very similar for the differently sized input libraries. Screening a few hundred fragments generally led to the identification of enough active fragments that patterns of chemically favorable substructures emerged, and meaningful classifiers could be built to prioritize active small molecules for testing.</div><div class="NLM_p last">Moreover, to explore whether the number of small molecules selected by chemistry-based expansion and used as input for biology-based expansion is a decisive factor in VFL + HTS-FP, we increased the number of molecules selected by VFL to 10â000 (i.e., the overall tested 20â000 molecules were now equally split into two sets of 10â000 compounds). Hit numbers were comparable to those achieved by the 5000/15â000 compound subset split, although a slight preference for the original split could be observed (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S1 and Table S2">Supporting Information Figure S1 and Table S2</a>). By contrast, decreasing the number of compounds selected by VFL to 500 or 1000 generally led to a substantial loss in the number of active molecules found after biological expansion to 20â000 molecules (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S1">Supporting Information Figure S1</a>). Still, active compounds were enriched in comparison to a random compound selection, especially if only a few hundred or thousand compounds were selected in the biological expansion. Reducing the numbers of molecules screened in the individual steps might be desirable for academia or start-up companies that do not have the same screening capacities as large pharmaceutical companies. Therefore, to provide guidelines and set expectations for how many hits can be obtained with a reduced number of screened molecules, we report enrichment factors and absolute numbers of active molecules for different combinations of smaller-sized screening sets in step 1 and step 2 in <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Table S3">Supporting Information Table S3</a>.</div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Comparison to Current Practice</h3><div class="NLM_p">The current practice in lead-discovery projects in industry is to screen a diverse set of small molecules to find initial hits and then to expand using chemical similarity. To assess whether screening a focused set designed by chemical expansion of fragment hits had a significant benefit over a rather random selection of small molecules to begin the lead-discovery workflow, we repeated our experiments, but this time we chose 5000 random compounds for the first round of testing that were then expanded by HTS-FP or structural fingerprints (strategies random + HTS-FP and random + ECFP4). Consistent with the observation that hits were generally enriched within the VFL-based selected subsets of 5000 molecules, VFL + HTS-FP performed mostly better than random + HTS-FP (higher hit rates in 9 of 13 data sets; <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S2A and Table S4">Supporting Information Figure S2A and Table S4</a>). The most notable exception that strongly favored a random selection of initially tested compounds was the data set IG for which VFL had failed. Strikingly, the picture was completely different when chemical similarity was used for small-molecule expansion. Here, a random selection of compounds led to much higher enrichments of active molecules (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S2B and Table S4">Supporting Information Figure S2B and Table S4</a>). Because random compound selection in the first step led to the identification of rather structurally diverse active molecules, the subsequent chemistry-based expansion was not biased toward a particular region but was able to retrieve active molecules scattered over large parts of chemical space (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S3">Supporting Information Figure S3</a>).</div><div class="NLM_p">Accordingly, to alleviate the strong bias toward structurally similar molecules introduced by VFL-based compound selection, we decided to change our selection strategy in a control experiment: instead of selecting the top 5000 molecules, we picked the 5000 most diverse among the top-ranking 50â000 compounds. Indeed, in line with our hypothesis, improved hit rates were obtained for chemistry-based expansion (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S2B and Table S4">Supporting Information Figure S2B and Table S4</a>) and almost reached hit rates observed with our VFL + HTS-FP method. By contrast, biology-based expansion did not further benefit from a more diverse hit set from the first step because the method was already able to hop in chemical space and was less affected by the chemical location of the hit molecules that were expanded (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S2A and Table S4">Supporting Information Figure S2A and Table S4</a>).</div><div class="NLM_p last">Taken together, considering all possible strategies to sample bioactive chemical space, best enrichments of active compounds and structural diversity were obtained when FBS was carried out and followed by VFL and HTS-FP-based expansion. VFL led to an enrichment of active compounds in the chemistry-based expansion, and this additional information about active molecules in the training set was then exploited by the biology-based classifier in the second expansion step. Because this step is structure-independent, it reached out to different regions of chemical space. Thus, our method retained knowledge gained from the FBS screen (using VFL as the first expansion step) and at the same time retrieved diverse bioactive compounds (using HTS-FP as the second expansion step). The ability of VFL + HTS-FP to hop in chemical space also has direct implications for the design of fragment libraries, which is currently often focused on diversity, to have as many fragment chemotypes to follow up on. With our findings in mind, it may be more desirable to focus on diversifying a subset of chemotypes in terms of their substitution patterns and substituent types. Once active fragments have been identified, more chemotypes will be discovered during the biological expansion (using HTS-FP), so it is not necessary to include all of these chemotypes in the initial library. Of course, one caveat that one has to keep in mind is that the same fragment library is usually screened against a range of diverse targets. Hence, including less chemotypes in the initial library might come at the risk of identifying very few or no hits for some projects.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Prospective Testing: Identification of Novel HDAC4 Inhibitors</h3><div class="NLM_p">For histone deacetylase 4 (HDAC set in the benchmark system), we prospectively applied the biology-based expansion step in VFL + HTS-FP. Out of â¼150â000 compounds not originally screened, we predicted 47 small molecules to be active, which we subsequently screened in an HDAC4 fluorescence-intensity assay. Two compounds showed substantial activity: in doseâresponse experiments, IC<sub>50</sub> values of 6.6 and 9.0 Î¼M were determined for 3-benzoyl-6-chloro-7-(2,4-dioxopentan-3-yl)-1-benzofuran-4,5-dione (<b>7</b>) and 2-chloro-3-[(4-methylphenyl)amino]-1,4-naphthoquinone (<b>8</b>), respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. HDAC4 inhibitors. Structures of the two HDAC4 inhibitors (<b>7</b> and <b>8</b>) identified in the prospective application of the VFL + HTS-FP approach are shown and annotated with their IC<sub>50</sub> values and PubChem<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> compound identifiers (CIDs). Furthermore, we also report IC<sub>50</sub> values for three structural analogs (<b>9</b>, <b>10</b>, and <b>11</b>) that were tested in the HDAC4 assay to explore structureâactivity relationships.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Remarkably, the two identified hits <b>7</b> and <b>8</b> had very low structural similarity to all active fragments and small molecules found in the chemistry-based expansion of the VFL + HTS-FP approach; the highest ECFP4 Tanimoto coefficient<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (Tc; see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>) between one of the hits and an active reference structure was 0.239. Similarly, the hit structures were very different from all publicly known HDAC4 inhibitors in the ChEMBL<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and GVK<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> databases. <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S4">Supporting Information Figure S4</a> displays the closest structural neighbor for both hit compounds determined on the basis of ECFP4 fingerprints and Tanimoto similarity. Both compounds lack acidic groups, which raises the hypothesis that they possess a novel binding mode. For <b>7</b>, the Michael acceptor moiety and/or the 4,5-dione motif, which overlap, appear to be important for activity because we also tested the aromatic hydroquinone analog 3-(3-benzoyl-6-chloro-4,5-dihydroxy-1-benzofuran-7-yl)pentane-2,4-dione (<b>9</b>), which did not show any detectable activity (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). We also tested two analogs of <b>8</b>, 2-amino-3-(4-chloroanilino)naphthalene-1,4-dione (<b>10</b>) and 2-chloro-3-[(4-nitrophenyl)amino]naphthalene-1,4-dione (<b>11</b>). We found that replacing the methyl by a nitro group at the para position of the phenyl ring led to a slight increase in activity (IC<sub>50</sub> value of 4.8 Î¼M for <b>10</b>). By contrast, replacing this methyl group by a chlorine atom while simultaneously changing the chlorine substituent at the naphthoquinone moiety to an amino group led to activity loss (no detectable activity for <b>11</b>; Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). With a molecular weight as low as 297.7 Da, <b>8</b> has a good binding-efficiency index<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (BEI) of 16.95, which is close to the reported median BEI of 18.4 calculated for leads in successful drug-discovery programs (BEI is calculated as the ratio of the pIC<sub>50</sub> to the molecular weight in kDa).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p last">Although the size of the tested compound set was very small and cannot provide statistical significance, it should be noted that the performance of the HTS-FP expansion matched the hit rates observed in benchmark calculations. For the HDAC set, a hit rate of 3.8% was observed for the biology-based expansion step in retrospective simulated virtual-screening experiments; in the prospective evaluation, the discovery of two active ligands among 47 tested compounds corresponds to a hit rate of 4.3%. Furthermore, it should be noted that the two identified inhibitors were confirmed as hits in multiple doseâresponse measurements, whereas the hit definition in the benchmarking system was based on single-dose experiments (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15052" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15052" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">FBS has become increasingly utilized in the pharmaceutical industry. Different strategies have been introduced to grow binders from FBS into larger, more potent molecules. However, in many cases, the sampled chemical space is still limited by the size and diversity of the fragment library. We have therefore developed an in silico workflow that, on the basis of VFL and HTS-FP, departs from fragments and arrives at diverse chemotypes not necessarily assembled from active fragment substructures. X-ray structures demonstrate our methodâs ability to find bioactive compounds that were informed by, but not limited to, fragment-binding modes. A prospective application of our approach revealed two novel histone deacetylase 4 inhibitors that were structurally very distinct from all active fragments and small molecules found by chemistry-based expansion.</div><div class="NLM_p">HTS-FPs that are used in the second step of our approach are derived from Novartis screening campaigns and are proprietary. However, with increasing amounts of screening data being released to the public domain, similar bioactivity profiles can be generated from public sources, such as the NCI60 data set<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and HTS data in the PubChem bioassay database.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Even more data is anticipated to be available in the future because many open-access screening efforts are ongoing (e.g., Lillyâs PDD initiative<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> or the LINCS project at Harvard Medical School<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>). Hence, the methodology presented in this article is applicable in both academic and industrial institutions.</div><div class="NLM_p last">Our method offers an alternate path that may adapt FBS to drug-discovery settings where other lead-discovery methods might get more attention otherwise. We could demonstrate that by screening about 10% of the screening collection we can expect to find half of the primary hits contained in the screening deck. That being said, we currently consider the introduced VFL + HTS-FP approach best practice for the integration of FBS and small-molecule screening into a common workflow.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62768" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62768" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Virtual Fragment Linking and HTS Fingerprints</h3><div class="NLM_p">Virtual fragment linking was introduced by Crisman et al. to bridge the gap between low-affinity fragment hits and larger, generally more potent, small molecules.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The key idea is to learn structural features that distinguish active from inactive fragments and then prioritize small-molecule structures for screening in which these features are enriched. For doing so, circular atom environments (ECFP4)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> are calculated for active and inactive fragments and used as molecular descriptors to train a naÄ±Ìve Bayes classifier. Subsequently, the Bayes classifier is applied to rank small molecules according to their likelihood of being active.</div><div class="NLM_p last">HTS-FPs are primary-assay-based compound-bioactivity profiles that have been introduced by Petrone et al.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> They capture the activity of a compound across 234 biochemical and cellular assays that have historically been run at Novartis and can be used to compare molecules based on their previously profiled interaction with the proteome. To have a consistent reference frame across all assays, activity outcomes for a compound are reported as Z scores. Because the screening library is constantly changing with new compounds being added and others being depleted, most compounds have not been tested in all assays and hence their bioactivity profile vectors are incomplete and contain missing values.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Small-Molecule Subset-Selection Strategies</h3><div class="NLM_p">To explore the utility of fragment-based screening as starting point to design focused small-molecule subsets, we evaluated the following strategies to follow-up on FBS and cherry-pick <i>N</i> compounds from the full screening deck for experimental testing.<table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">(i)</td><td colspan="5" valign="top" class="paragraph"><p class="last">VFL + HTS-FP: The <i>N</i> compounds are picked in an iterative fashion by screening two subsets. First, the <i>M</i> best compounds from VFL are selected and tested. HTS-FPs of assayed active and inactive molecules are then used as descriptors to train a second Bayes classifier. The remaining screening database is ranked according to this classifier, and the best <i>NâM</i> compounds are picked for experimental testing.</p></td></tr><tr class="li1"><td valign="top">(ii)</td><td colspan="5" valign="top" class="paragraph"><p class="last">VFL: The <i>N</i> top-ranked compounds from VFL are selected.</p></td></tr><tr class="li1"><td valign="top">(iii)</td><td colspan="5" valign="top" class="paragraph"><p class="last">VFL + ECFP4: Same strategy as VFL + HTS-FP, but this time ECFP4 fingerprints are used instead of HTS-FPs to train the second Bayes classifier.</p></td></tr></table></div><div class="NLM_p last">In this study, presented results were mostly obtained with <i>M</i> set to 5000. For this setting, explored values for <i>N</i> ranged approximately from 9000 to 130â000.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Data Sets</h3><div class="NLM_p last">To assess the ability of our different strategies to efficiently sample bioactive chemical space, we assembled 13 benchmark data sets. For 12 sets, the Novartis fragment core library consisting of about 1400 structures was screened against a specific protein target, whereas for one data set a smaller collection of 466 fragments overlapping only partly with the core library was tested. For a fragment, the average number of heavy atoms and the average molecular weight were 14.8 and 211.7 Da, respectively. Furthermore, for each target, a high-throughput campaign had been run at Novartis (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Hit numbers varied significantly (i.e., between 23 and 642 active fragments were found for different proteins). All assays considered in this study were cell-free and designed to detect direct target-ligand interactions. For the set HMET, a single protein domain was used in the FBS campaign, whereas the corresponding HTS campaign was run on a protein complex containing this domain as a subunit. A fragment was considered a hit if project teams considered it to be a hit in at least one technology. For example, if a biochemical and NMR screen were employed for a given campaign, all hits from either technology were considered hits for model building. The cutoffs for calling a fragment a hit for a given technology were chosen by each of the individual project teams. For small molecules, a Z score equal to or smaller than â3 was used as the hit-calling criterion. Furthermore, two different small-molecule sets were created for each protein target. Because we used primary assay data, we were careful not to bias our results by false positives so that, in addition to the full set of molecules tested against a target in an HTS campaign, we built a subset that excluded all compounds with a hit rate of at least 10% across the 234 assays in the HTS-FP. However, results for sets with or without these frequent hitters were very similar. Hence, for brevity, only data for sets excluding frequently hitting molecules were reported herein, with an exception being made for the kinase case study. Because a lot of kinase assays are included in the 234 HTS-FP assays, compounds that inhibit multiple kinases can easily be discarded by a rather conservative 10% hit rate cutoff for the identification of frequent hitters, although they are true kinase binders. Therefore, the original nonreduced data set was used for this case study, for which reported results were obtained by selecting 5000 molecules by chemistry-based expansion and 15â000 molecules by biology-based expansion.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Calculations and Implementation</h3><div class="NLM_p">For each data set, the three described strategies (VFL + HTS-FP, VFL, and VFL + ECFP4) were applied in simulated virtual-screening trials to assemble small-molecule subsets of 20â000 molecules. Numbers of active molecules in selected subsets were determined retrospectively, and enrichment factors over the number of hits expected by a random selection of 20â000 molecules were calculated. Furthermore, control calculations that replaced the chemistry-based expansion of fragment hits by random compound subset selection were carried out. For each data set, three different random subsets of 5000 molecules were assembled, and the results were averaged over the three independent trials. NB classifiers were built with PipelinePilot v8.0<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> using standard settings and allowing missing descriptor values for HTS-FPs. Perl scripts were used to organize the results and facilitate the analysis.</div><div class="NLM_p last">To visualize bioactive chemical space for individual data sets, we represented molecules by their ECFP4 fingerprints and calculated coordinates in 3D space using MDS, a dimension-reduction technique that tries to transform a set of data points to fewer dimensions while retaining the distances between objects as accurately as possible. Coordinates in 3D space were determined with the R Classical MDS component in Pipeline Pilot and default settings that minimize the sum of the squared differences between the distances in the original and the new coordinate systems.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Identification of HDAC4 Inhibitors</h3><div class="NLM_p">We prospectively evaluated the biology-based expansion step of our VFL + HTS-FP approach on the HDAC4 data set. We expanded hits from an HDAC4 fragment campaign by VFL and selected the 10â000 highest-scoring small molecules that were part of the small-molecule set reported in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and that had been tested in an HDAC4 HTS campaign in the past. Forty eight active compounds (Z score â¤ â3) were among the selected subset and then served as a positive training class for an HTS-FP Bayes classifier, whereas the remaining 9952 inactive compounds served as negative training examples. The classifier was used to rank 157â005 compounds that had not been tested previously in the HDAC4 campaign. We selected the top 160 compounds and calculated Bemis and Murcko scaffolds.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> For each scaffold, we just retained the highest-scoring compound, thereby reducing the list to 95 molecules. We then discarded structures that could possibly interfere with a fluorescent assay readout using the PAINS<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> substructure and internal filters. Of the remaining 78 compounds, 47 were in stock with sufficient quantity for experimental testing. Each compound was tested at eight different concentrations up to 10 Î¼M in an HDAC4 fluorescence-intensity assay (see <a class="ref internalNav" href="#sec4_6" aria-label="Experimental Section">Experimental Section</a>). Two of the 47 compounds showed moderate activity against HDAC4 and were both retested at higher concentrations together with three structural analogs. On the basis of doseâresponse experiments with compound concentrations up to 30 Î¼M (see <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S5">Supporting Information Figure S5</a> for doseâresponse curves), IC<sub>50</sub> values of 6.6, 9.0, and 4.8 Î¼M were determined for <b>7</b>, <b>8</b>, and <b>10</b>, respectively.</div><div class="NLM_p last">For structural comparisons to known HDAC4 inhibitors, compounds were represented by ECFP4 fingerprints, and pairwise Tc values were calculated. The Tc value for a compound pair is the ratio of the number of structural features shared by two molecules to the union of structural features present in the two molecules.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Experimental Section</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Compound Data</h4><div class="NLM_p last">The tested compounds <b>7</b>, <b>9</b>, <b>10,</b> and <b>11</b> were available in the Novartis corporate compound collection, whereas <b>8</b> was purchased from Sigma-Aldrich (S703729; CAS number: 62101-46-6) and used without further purification. Purity was confirmed by <sup>1</sup>H NMR analysis of this known compound.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The four compounds from the Novartis library were >95% pure as analyzed via LCâMS and <sup>1</sup>H NMR (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure S6">Supporting Information Figure S6</a>).</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> HDAC Assay</h4><div class="NLM_p">HDAC4 activity was detected by a two-step enzymatic assay based on successive cleavages of a substrate peptide. An Îµ-trifluoroacetylated (TFA) peptide (Ac-Gly-Ala-Lys(TFA)-AMC) was used as an HDAC class IIA-specific substrate.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> First, the trifluoroacetyl group on the Îµ-amino groups of lysine was cleaved by HDAC4, which unmasked the trypsin cleavage site. In a second step, the deacetylated substrate was cleaved by trypsin, resulting in the release of fluorogenic AMC (7-amino-4-methylcoumarin). HDAC4 inhibition by an added compound generally results in a decreasing amount of fluorogenic AMC.</div><div class="NLM_p">All enzyme reactions were performed in reaction buffer consisting of the following components (final concentrations): 25 mM Tris, pH 8.0, 137 mM NaCl, 2.7 mM KCl, 0.05% (w/v) BSA, 0.005% (v/v) Triton-X-100, and 1 mM MgCl2.</div><div class="NLM_p">Compounds were diluted in 384-well compound plates to obtain eight-point serial dilutions for 40 test compounds in 90% DMSO as well as 4 reference compounds plus 16 high controls and 16 low controls. Assay plates were prepared by dispensing 50 nL of compound solutions into 384-well plates (Greiner, reference 784900) using a Hummingbird nanoliter dispenser (Cartesian Technologies Inc.). All following pipetting steps were carried out with the help of a nanodrop (Innovadyne Technologies Inc.), and the fluorescence detection step was performed on a Pherastar fluorescence reader (BMG).</div><div class="NLM_p">Recombinant HDAC4 was expressed in full-length form (a.a. 2â1084) in Sf9 insect cells (obtained from ATCC) using baculovirus generated with the Bac-to-Bac system (Invitrogen). A 0.15 mg/mL stock solution was produced and stored at â80 Â°C. Trypsin was purchased from Worthington Biochemical Corporation (no. 3740). A 20 mg/mL stock solution was prepared in 1 mM HCl, which was stored at â20 Â°C or used as a fresh solution prepared in assay buffer. The substrate (Ac-Gly-Ala-Lys(TFA)-AMC) was purchased from Biosyntan. A 10 mM stock solution prepared in 90% DMSO was stored at â20 Â°C.</div><div class="NLM_p last">Enzyme reactions were prepared as follows: 4.5 Î¼L of a 10 Î¼M peptide substrate solution and 4.5 Î¼L of a 0.1 nM enzyme solution were added to the assay plates containing 50 nL of compound solutions, and the plates were incubated for 120 min 30 Â°C. Enzyme reactions were terminated by the addition of 4.5 Î¼L of assay buffer containing a pan-HDAC inhibitor at high concentration and 100 nM trypsin for 15 min at 30 Â°C. Fluorescence-intensity readout was at 450 nm (10 nm bandpass) following excitation at 350 nm.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24"><a href="/doi/suppl/10.1021/jm401309q">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46247" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46247" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Enrichment factors for different subset splits; comparison of chemistry-based fragment hit expansion to random small molecule selection; coverage of bioactive chemical space with chemistry-based hit expansion; two publicly known HDAC4 inhibitors that are structurally most similar to the identified hits; fitted doseâresponse curves for the three newly identified active compounds; and <sup>1</sup>H NMR and LCâMS results for tested compounds. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm401309q/suppl_file/jm401309q_si_001.pdf">jm401309q_si_001.pdf (1.04 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The crystal structure of compound <b>4</b> with Lck can be accessed using the PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3F">4C3F</a>.</p><div class="testing" data-doi="10.1021/jm401309q" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne Mai Wassermann</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#cbaaa5a5ae94a6aaa2e5bcaab8b8aeb9a6aaa5a58ba5a4bdaab9bfa2b8e5a8a4a6"><span class="__cf_email__" data-cfemail="ccada2a2a993a1ada5e2bbadbfbfa9bea1ada2a28ca2a3baadbeb8a5bfe2afa3a1">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter S. Kutchukian</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eugen Lounkine</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tiffany Luethi</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., Novartis Campus, Basel 4056, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacques Hamon</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., Novartis Campus, Basel 4056, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael T. Bocker</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hasnain A. Malik</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sandra W. Cowan-Jacob</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., Novartis Campus, Basel 4056, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meir Glick</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d3143e1949-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the Education Office of the Novartis Institutes for Biomedical Research for funding and Nina Gommermann, Jerome Giovannoni, John Peltier, Melissa Grippo, and Mika Lindvall for helpful discussions and insights.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i29" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i29"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i30" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i30"> Abbreviations Used</h2><tr><td class="NLM_term">AA</td><td class="NLM_def"><p class="first last">amino acid</p></td></tr><tr><td class="NLM_term">BEI</td><td class="NLM_def"><p class="first last">binding-efficiency index</p></td></tr><tr><td class="NLM_term">CID</td><td class="NLM_def"><p class="first last">compound identifier</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">FBLD</td><td class="NLM_def"><p class="first last">fragment-based lead discovery</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fragment-based screening</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">HTS-FP</td><td class="NLM_def"><p class="first last">high-throughput screening fingerprint</p></td></tr><tr><td class="NLM_term">IRAK-4</td><td class="NLM_def"><p class="first last">interleukin-1 receptor-associated kinase 4</p></td></tr><tr><td class="NLM_term">Lck</td><td class="NLM_def"><p class="first last">lymphocyte-specific protein tyrosine kinase</p></td></tr><tr><td class="NLM_term">MDS</td><td class="NLM_def"><p class="first last">multidimensional scaling</p></td></tr><tr><td class="NLM_term">NB</td><td class="NLM_def"><p class="first last">naÄ±Ìve Bayesian</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compounds</p></td></tr><tr><td class="NLM_term">Tc</td><td class="NLM_def"><p class="first last">Tanimoto coefficient</p></td></tr><tr><td class="NLM_term">TKL</td><td class="NLM_def"><p class="first last">tyrosine kinase-like kinase</p></td></tr><tr><td class="NLM_term">TK</td><td class="NLM_def"><p class="first last">tyrosine kinase</p></td></tr><tr><td class="NLM_term">VFL</td><td class="NLM_def"><p class="first last">virtual fragment linking</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 30 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kitchen, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decornez, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furr, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Docking and scoring in virtual screening for drug discovery: Methods and applications</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">â</span> <span class="NLM_lpage">949</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2Fnrd1549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=15520816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=935-949&author=D.+B.+Kitchenauthor=H.+Decornezauthor=J.+R.+Furrauthor=J.+Bajorath&title=Docking+and+scoring+in+virtual+screening+for+drug+discovery%3A+Methods+and+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and scoring in virtual screening for drug discovery: methods and applications</span></div><div class="casAuthors">Kitchen, Douglas B.; Decornez, Helene; Furr, John R.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">935-949</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Computational approaches that 'dock' small mols. into the structures of macromol. targets and 'score' their potential complementarity to binding sites are widely used in hit identification and lead optimization.  Indeed, there are now a no. of drugs whose development was heavily influenced by or based on structure-based design and screening strategies, such as HIV protease inhibitors.  Nevertheless, there remain significant challenges in the application of these approaches, in particular in relation to current scoring schemes.  Here, we review key concepts and specific features of small-mol.-protein docking methods, highlight selected applications and discuss recent advances that aim to address the acknowledged limitations of established approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMLibgFe3mfLVg90H21EOLACvtfcHk0liMDi6vyEvtNA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D&md5=875a2b37a4299181509a1922b11dbd2f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1549%26sid%3Dliteratum%253Aachs%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DDecornez%26aufirst%3DH.%26aulast%3DFurr%26aufirst%3DJ.%2BR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDocking%2520and%2520scoring%2520in%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520Methods%2520and%2520applications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D935%26epage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Kutchukian, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakhnovich, E. I.</span><span> </span><span class="NLM_article-title">De novo design: balancing novelty and confined chemical space</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">789</span><span class="NLM_x">â</span> <span class="NLM_lpage">812</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1517%2F17460441.2010.497534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=22827800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCgtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=789-812&author=P.+S.+Kutchukianauthor=E.+I.+Shakhnovich&title=De+novo+design%3A+balancing+novelty+and+confined+chemical+space"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">De novo design: balancing novelty and confined chemical space</span></div><div class="casAuthors">Kutchukian, Peter S.; Shakhnovich, Eugene I.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">789-812</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: De novo drug design serves as a tool for the discovery of new ligands for macromol. targets as well as optimization of known ligands.  Recently developed tools aim to address the multi-objective nature of drug design in an unprecedented manner.  Areas covered in this review: This article discusses recent advances in de novo drug design programs and accessory programs used to evaluate compds. post-generation.  What the reader will gain: The reader is introduced to the challenges inherent in de novo drug design and will become familiar with current trends in de novo design.  Furthermore, the reader will be better prepd. to assess the value of a tool, and be equipped to design more elegant tools in the future.  Take home message: De novo drug design can assist in the efficient discovery of new compds. with a high affinity for a given target.  The inclusion of existing chemoinformatic methods with current structure-based de novo design tools provides a means of enhancing the therapeutic value of these generated compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPazAD3Wic-rVg90H21EOLACvtfcHk0lhVe9xJ5aYMRw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCgtLk%253D&md5=404b8e0c9ca878454d236417c6a82af8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1517%2F17460441.2010.497534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2010.497534%26sid%3Dliteratum%253Aachs%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26atitle%3DDe%2520novo%2520design%253A%2520balancing%2520novelty%2520and%2520confined%2520chemical%2520space%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D789%26epage%3D812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Hartenfeller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span> </span><span class="NLM_article-title">De novo design</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">672</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">â</span> <span class="NLM_lpage">323</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1007%2F978-1-60761-839-3_12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=20838974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlejsr7L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=672&publication_year=2011&pages=299-323&author=M.+Hartenfellerauthor=G.+Schneider&title=De+novo+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">De novo drug design</span></div><div class="casAuthors">Hartenfeller, Markus; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">672</span>
        (<span class="NLM_cas:issue">Chemoinformatics and Computational Chemical Biology</span>),
    <span class="NLM_cas:pages">299-323</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Computer-assisted mol. design supports drug discovery by suggesting novel chemotypes and compd. modifications for lead structure optimization.  While the aspect of synthetic feasibility of the automatically designed compds. has been neglected for a long time, we are currently witnessing an increased interest in this topic.  Here, we review state-of-the-art software for de novo drug design with a special emphasis on fragment-based techniques that generate druglike, synthetically accessible compds.  The importance of scoring functions that can be used to predict compd. reactivity and potency is highlighted, and several promising solns. are discussed.  Recent practical validation studies are presented that have already demonstrated that rule-based fragment assembly can result in novel synthesizable compds. with druglike properties and a desired biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6_YdqEvu-hbVg90H21EOLACvtfcHk0lhVe9xJ5aYMRw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlejsr7L&md5=3a30147da9187fa4895ba807cefed82a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2F978-1-60761-839-3_12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60761-839-3_12%26sid%3Dliteratum%253Aachs%26aulast%3DHartenfeller%26aufirst%3DM.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DDe%2520novo%2520design%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D672%26spage%3D299%26epage%3D323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Broach, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorner, J.</span><span> </span><span class="NLM_article-title">High-throughput screening for drug discovery</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">â</span> <span class="NLM_lpage">16</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2F384014a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=8895594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADyaK28Xmsl2rsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=14-16&author=J.+R.+Broachauthor=J.+Thorner&title=High-throughput+screening+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput screening for drug discovery</span></div><div class="casAuthors">Broach, James R.; Thorner, Jeremy</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">6604, Suppl.</span>),
    <span class="NLM_cas:pages">14-16</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">A review with 23 refs.  There are now a large no. of known proteins available as potential targets for drug treatment.  Concurrently, new approaches have increased the no. of compds. that can be tested for activity against these targets.  Together, these trends have stimulated the adoption of high-throughput screening as a primary tool for early-stage drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHliuK3qN3urVg90H21EOLACvtfcHk0lhzN8ZPba6Ixw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmsl2rsr4%253D&md5=356ca2edc709ece1a0c80f560ddd893d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2F384014a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F384014a0%26sid%3Dliteratum%253Aachs%26aulast%3DBroach%26aufirst%3DJ.%2BR.%26aulast%3DThorner%26aufirst%3DJ.%26atitle%3DHigh-throughput%2520screening%2520for%2520drug%2520discovery%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D14%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Clark, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arico-Muendel, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belyanskaya, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benjamin, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Centrella, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creaser, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuozzo, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davie, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franzen, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gefter, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hale, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, N. J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Israel, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavarana, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelley, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollmann, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lind, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mataruse, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medeiros, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Messer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâKeefe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliff, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rise, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satz, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skinner, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svendsen, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vloten, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, B. A.</span><span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">â</span> <span class="NLM_lpage">654</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+V.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kelleyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lhzN8ZPba6Ixw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelley%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3Dvan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Loving, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alberts, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Computational approaches for fragment-based and de novo design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">â</span> <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.2174%2F156802610790232305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=19929832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=14-32&author=K.+Lovingauthor=I.+Albertsauthor=W.+Sherman&title=Computational+approaches+for+fragment-based+and+de+novo+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Computational approaches for fragment-based and de novo design</span></div><div class="casAuthors">Loving, Kathryn; Alberts, Ian; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-32</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based and de novo design strategies have been used in drug discovery for years.  The methodologies for these strategies are typically discussed sep., yet the applications of these techniques overlap substantially.  We present a review of various fragment-based discovery and de novo design protocols with an emphasis on successful applications in real-world drug discovery projects.  Furthermore, we illustrate the strengths and weaknesses of the various approaches and discuss how one method can be used to complement another.  We also discuss how the incorporation of exptl. data as constraints in computational models can produce novel compds. that occupy unique areas in intellectual property (IP) space yet are biased toward the desired chem. property space.  Finally, we present recent research results suggesting that computational tools applied to fragment-based discovery and de novo design can have a greater impact on the discovery process when coupled with the right expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ4xFpqSbTRbVg90H21EOLACvtfcHk0lh1JNA2__ULuQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu70%253D&md5=c68f8cf413e1006c478f9cff5b170386</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F156802610790232305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232305%26sid%3Dliteratum%253Aachs%26aulast%3DLoving%26aufirst%3DK.%26aulast%3DAlberts%26aufirst%3DI.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DComputational%2520approaches%2520for%2520fragment-based%2520and%2520de%2520novo%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D14%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span> <span class="citation_source-book">Library Design, Search Methods, and Applications of Fragment-Based Drug Design</span>; <span class="NLM_contrib-group">Bienstock, R. J.</span>, Ed.; <span class="NLM_publisher-name">American Chemical Society</span>: <span class="NLM_publisher-loc">Washington, DC</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks">[<a href="/doi/10.1021/bk-2011-1076" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=R.+J.+Bienstock&title=Library+Design%2C+Search+Methods%2C+and+Applications+of+Fragment-Based+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fbk-2011-1076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1021%252Fbk-2011-1076%26sid%3Dliteratum%253Aachs%26btitle%3DLibrary%2520Design%252C%2520Search%2520Methods%252C%2520and%2520Applications%2520of%2520Fragment-Based%2520Drug%2520Design%26aulast%3DBienstock%26aufirst%3DR.%2BJ.%26pub%3DAmerican%2520Chemical%2520Society%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Baker, M.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery grows up</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">â</span> <span class="NLM_lpage">7</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2Fnrd3926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=23274457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFOi" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=5-7&author=M.+Baker&title=Fragment-based+lead+discovery+grows+up"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery grows up</span></div><div class="casAuthors">Baker, Monya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With multiple drug candidates in the clinic that originated from fragment-based lead discovery, the approach of starting small has become big.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKfzq4txN2bVg90H21EOLACvtfcHk0lh1JNA2__ULuQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFOi&md5=854c670bd7388d04f76867d5818a5191</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DM.%26atitle%3DFragment-based%2520lead%2520discovery%2520grows%2520up%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D5%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Abad-Zapatero, C.</span><span> </span><span class="NLM_article-title">Ligand efficiency indices for effective drug discovery</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">â</span> <span class="NLM_lpage">488</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1517%2F17460441.2.4.469" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=23484756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVCqsL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=469-488&author=C.+Abad-Zapatero&title=Ligand+efficiency+indices+for+effective+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency indices for effective drug discovery</span></div><div class="casAuthors">Abad-Zapatero, Cele</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-488</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Successful drug discovery requires the optimization of a large no. of variables ranging from strictly physicochem. parameters such as mol. wt. to more complex parameters related to toxicity and bioavailability.  Presently, structure-based methodologies influence many aspects of the drug discovery process from lead discovery to the final preclin. characterization.  However, crit. biol. issues along the path to the market have diminished the impact and power of this methodol.  The physicochem. properties of the novel chem. entities designed and guided by structural methods have become the subject of intense scrutiny from lead discovery to drug candidate.  The idea of ligand efficiency (binding energy/non-hydrogen atoms) has recently emerged as a useful guide to optimize fragment and lead selection in the discovery process.  More generalized concepts of ligand efficiency, related to efficiency per dalton and per unit of polar surface area, have also been introduced and will be discussed in the broader context.  Preliminary results and trends obtained using ligand efficiencies as guides are reviewed and their future application to guide drug discovery will be discussed, as well as their integration into the structure-based drug design methods to make them more effective and numerically robust.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmKduo41JcS7Vg90H21EOLACvtfcHk0ljBxvIX4naBFA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVCqsL4%253D&md5=a1682fbc4545d8eab3a44417c0e88869</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1517%2F17460441.2.4.469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2.4.469%26sid%3Dliteratum%253Aachs%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26atitle%3DLigand%2520efficiency%2520indices%2520for%2520effective%2520drug%2520discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2007%26volume%3D2%26spage%3D469%26epage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Ferenczy, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">How fragments are optimized? A retrospective analysis of 145 fragment optimizations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2478</span><span class="NLM_x">â</span> <span class="NLM_lpage">2486</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm301851v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2478-2486&author=G.+G.+Ferenczyauthor=G.+M.+Keseru&title=How+fragments+are+optimized%3F+A+retrospective+analysis+of+145+fragment+optimizations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm301851v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301851v%26sid%3Dliteratum%253Aachs%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DHow%2520fragments%2520are%2520optimized%253F%2520A%2520retrospective%2520analysis%2520of%2520145%2520fragment%2520optimizations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2478%26epage%3D2486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Crisman, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheiber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sukuru, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fejzo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hommel, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">Virtual fragment linking: An approach to identify potent binders from low affinity fragment hits</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2481</span><span class="NLM_x">â</span> <span class="NLM_lpage">2491</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701314u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2481-2491&author=T.+J.+Crismanauthor=A.+Benderauthor=M.+Milikauthor=J.+L.+Jenkinsauthor=J.+Scheiberauthor=S.+C.+Sukuruauthor=J.+Fejzoauthor=U.+Hommelauthor=J.+W.+Daviesauthor=M.+Glick&title=Virtual+fragment+linking%3A+An+approach+to+identify+potent+binders+from+low+affinity+fragment+hits"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm701314u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701314u%26sid%3Dliteratum%253Aachs%26aulast%3DCrisman%26aufirst%3DT.%2BJ.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMilik%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DSukuru%26aufirst%3DS.%2BC.%26aulast%3DFejzo%26aufirst%3DJ.%26aulast%3DHommel%26aufirst%3DU.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DVirtual%2520fragment%2520linking%253A%2520An%2520approach%2520to%2520identify%2520potent%2520binders%2520from%2520low%2520affinity%2520fragment%2520hits%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2481%26epage%3D2491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Rogers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, M.</span><span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">â</span> <span class="NLM_lpage">754</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0ljBxvIX4naBFA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Duda, R. O.; Hart, P. E.; Stork, D. G.</span> <span class="citation_source-book">Pattern Classification</span>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Wiley-Interscience</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2001</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">20</span><span class="NLM_x">â</span> <span class="NLM_lpage">83</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=20-83&author=R.+O.+Duda&author=P.+E.+Hart&author=D.+G.+Stork&title=Pattern+Classification"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDuda%26aufirst%3DR.%2BO.%26btitle%3DPattern%2520Classification%26pub%3DWiley-Interscience%26date%3D2001%26spage%3D20%26epage%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Petrone, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simms, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigsch, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutchukian, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">Rethinking molecular similarity: Comparing compounds on the basis of biological activity</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1399</span><span class="NLM_x">â</span> <span class="NLM_lpage">1409</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=1399-1409&author=P.+Petroneauthor=B.+Simmsauthor=F.+Nigschauthor=E.+Lounkineauthor=P.+Kutchukianauthor=Z.+Dengauthor=J.+W.+Daviesauthor=J.+L.+Jenkinsauthor=M.+Glick&title=Rethinking+molecular+similarity%3A+Comparing+compounds+on+the+basis+of+biological+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPetrone%26aufirst%3DP.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKutchukian%26aufirst%3DP.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DRethinking%2520molecular%2520similarity%253A%2520Comparing%2520compounds%2520on%2520the%2520basis%2520of%2520biological%2520activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D17%26spage%3D1399%26epage%3D1409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">Bioturbo similarity searching: combining chemical and biological similarity to discover structurally diverse bioactive molecules</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">692</span><span class="NLM_x">â</span> <span class="NLM_lpage">703</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci300607r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsVylsro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=692-703&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=M.+Glick&title=Bioturbo+similarity+searching%3A+combining+chemical+and+biological+similarity+to+discover+structurally+diverse+bioactive+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Bioturbo Similarity Searching: Combining Chemical and Biological Similarity To Discover Structurally Diverse Bioactive Molecules</span></div><div class="casAuthors">Wassermann, Anne Mai; Lounkine, Eugen; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">692-703</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening using bioactivity profiles has become an integral part of currently applied hit finding methods in pharmaceutical industry.  However, a significant drawback of this approach is that it is only applicable to compds. that have been biol. tested in the past and have sufficient activity annotations for meaningful profile comparisons.  Although bioactivity data generated in pharmaceutical institutions are growing on an unprecedented scale, the no. of biol. annotated compds. still covers only a minuscule fraction of chem. space.  For a newly synthesized compd. or an isolated natural product to be biol. characterized across multiple assays, it may take a considerable amt. of time.  Consequently, this chem. matter will not be included in virtual screening campaigns based on bioactivity profiles.  To overcome this problem, we herein introduce bioturbo similarity searching that uses chem. similarity to map mols. without biol. annotations into bioactivity space and then searches for biol. similar compds. in this ref. system.  In benchmark calcns. on primary screening data, we demonstrate that our approach generally achieves higher hit rates and identifies structurally more diverse compds. than approaches using chem. information only.  Furthermore, our method is able to discover hits with novel modes of inhibition that traditional 2D and 3D similarity approaches are unlikely to discover.  Test calcns. on a set of natural products reveal the practical utility of the approach for identifying novel and synthetically more accessible chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5lay0o3odq7Vg90H21EOLACvtfcHk0ljBxvIX4naBFA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsVylsro%253D&md5=0df2d6699f97e95b0f04e40ac28c17fe</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci300607r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300607r%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DBioturbo%2520similarity%2520searching%253A%2520combining%2520chemical%2520and%2520biological%2520similarity%2520to%2520discover%2520structurally%2520diverse%2520bioactive%2520molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D692%26epage%3D703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">â</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lgvxMtor5pMTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">â</span> <span class="NLM_lpage">2893</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lgvxMtor5pMTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Borg, I.; Groenen, P.</span> <span class="citation_source-book">Modern Multidimensional Scaling: Theory and Applications</span>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=I.+Borg&author=P.+Groenen&title=Modern+Multidimensional+Scaling%3A+Theory+and+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBorg%26aufirst%3DI.%26btitle%3DModern%2520Multidimensional%2520Scaling%253A%2520Theory%2520and%2520Applications%26pub%3DSpringer%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Chemical similarity searching</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">â</span> <span class="NLM_lpage">996</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9800211" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADyaK1cXks1ynu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=983-996&author=P.+Willett&title=Chemical+similarity+searching"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical similarity searching</span></div><div class="casAuthors">Willett, Peter; Barnard, John M.; Downs, Geoffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">983-996</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper reviews the use of similarity searching in chem. databases.  It begins by introducing the concept of similarity searching, differentiating it from the more common substructure searching, and then discusses the current generation of fragment-based measures that are used for searching chem. structure databases.  The next sections focus upon two of the principal characteristics of a similarity measure: the coeff. that is used to quantify the degree of structural resemblance between pairs of mols. and the structural representations that are used to characterize mols. that are being compared in a similarity calcn.  New types of similarity measure are then compared with current approaches, and examples are given of several applications that are related to similarity searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxjjSk407VCLVg90H21EOLACvtfcHk0lgvxMtor5pMTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXks1ynu74%253D&md5=4233b0ea4edc052f0186cccbd5eae427</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci9800211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9800211%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DChemical%2520similarity%2520searching%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D983%26epage%3D996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: A large-scale bioactivity database for drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">â</span> <span class="NLM_lpage">D1107</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+large-scale+bioactivity+database+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0liUC2l3kiSU-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span>GOSTAR Online StructureâActivity Relationship Database. <a href="http://gostardb.com/gostar/" class="extLink">http://gostardb.com/gostar/</a> (accessed October 10,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GOSTAR+Online+Structure%E2%80%93Activity+Relationship+Database.+http%3A%2F%2Fgostardb.com%2Fgostar%2F+%28accessed+October+10%2C+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DGOSTAR%2520Online%2520Structure%25E2%2580%2593Activity%2520Relationship%2520Database%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Perola, E.</span><span> </span><span class="NLM_article-title">An analysis of the binding efficiencies of drugs and their leads in successful drug discovery programs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2986</span><span class="NLM_x">â</span> <span class="NLM_lpage">2997</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100118x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlWqu7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2986-2997&author=E.+Perola&title=An+analysis+of+the+binding+efficiencies+of+drugs+and+their+leads+in+successful+drug+discovery+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An Analysis of the Binding Efficiencies of Drugs and their Leads in Successful Drug Discovery Programs</span></div><div class="casAuthors">Perola, Emanuele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2986-2997</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In order to investigate the evolution of binding efficiency in successful drug discovery programs, a data set of 60 lead/drug pairs with known binding affinities has been compiled and analyzed.  Low-end thresholds for the binding efficiencies of viable leads and drugs have been derived.  On av., the drugs in the set are significantly larger and more potent but have similar lipophilicity relative to their originating leads, suggesting that the ability to maintain low levels of lipophilicity while increasing mol. wt. is one of the keys to a successful drug discovery program.  A no. of examples demonstrate that large increases in binding efficiency from leads to more elaborate drugs sharing the same scaffold can be achieved.  The importance of dissecting a lead structure to identify the most efficient fragments and the option of sacrificing binding efficiency to optimize other properties are discussed, and relevant examples are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRqzYFBuX2Q7Vg90H21EOLACvtfcHk0liUC2l3kiSU-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlWqu7o%253D&md5=a7eaad792f1518d093588951df64d22f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm100118x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100118x%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26atitle%3DAn%2520analysis%2520of%2520the%2520binding%2520efficiencies%2520of%2520drugs%2520and%2520their%2520leads%2520in%2520successful%2520drug%2520discovery%2520programs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2986%26epage%3D2997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span> </span><span class="NLM_article-title">The NCI60 human tumour cell line anticancer drug screen</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">â</span> <span class="NLM_lpage">823</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2Fnrc1951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=16990858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVCmu7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=813-823&author=R.+H.+Shoemaker&title=The+NCI60+human+tumour+cell+line+anticancer+drug+screen"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The NCI60 human tumour cell line anticancer drug screen</span></div><div class="casAuthors">Shoemaker, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">813-823</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The US National Cancer Institute (NCI) 60 human tumor cell line anticancer drug screen (NCI60) was developed in the late 1980s as an in vitro drug-discovery tool intended to supplant the use of transplantable animal tumors in anticancer drug screening.  This screening model was rapidly recognized as a rich source of information about the mechanisms of growth inhibition and tumor-cell kill.  Recently, its role has changed to that of a service screen supporting the cancer research community.  The author reviews the development, use and productivity of the screen, highlighting several outcomes that have contributed to advances in cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ysyTSc1-p7Vg90H21EOLACvtfcHk0liUC2l3kiSU-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVCmu7c%253D&md5=c6c1a074b9dc7e138f45734499769284</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrc1951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1951%26sid%3Dliteratum%253Aachs%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26atitle%3DThe%2520NCI60%2520human%2520tumour%2520cell%2520line%2520anticancer%2520drug%2520screen%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2006%26volume%3D6%26spage%3D813%26epage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">PubChem as a public resource for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1052</span><span class="NLM_x">â</span> <span class="NLM_lpage">1057</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=1052-1057&author=Q.+Liauthor=T.+Chengauthor=Y.+Wangauthor=S.+H.+Bryant&title=PubChem+as+a+public+resource+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubChem%2520as%2520a%2520public%2520resource%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26spage%3D1052%26epage%3D1057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span>Lilly Open Innovation Drug Discovery Home Page. <a href="https://openinnovation.lilly.com/dd/" class="extLink">https://openinnovation.lilly.com/dd/</a> (accessed September 24,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Lilly+Open+Innovation+Drug+Discovery+Home+Page.+https%3A%2F%2Fopeninnovation.lilly.com%2Fdd%2F+%28accessed+September+24%2C+2013%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span>Harvard Medical School Library of Integrated Network-Based Cellular Signatures Home Page. <a href="http://lincs.hms.harvard.edu/" class="extLink">http://lincs.hms.harvard.edu/</a> (accessed September 24,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Harvard+Medical+School+Library+of+Integrated+Network-Based+Cellular+Signatures+Home+Page.+http%3A%2F%2Flincs.hms.harvard.edu%2F+%28accessed+September+24%2C+2013%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span>Pipeline Pilot, version 8.0; <span class="NLM_publisher-name">Accelrys Software Inc.</span>: <span class="NLM_publisher-loc">San Diego, CA</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pipeline+Pilot%2C+version+8.0%3B+Accelrys+Software+Inc.%3A+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPipeline%2520Pilot%26pub%3DAccelrys%2520Software%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">â</span> <span class="NLM_lpage">2740</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljQnc44vAvX2g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Lisboa, C. d. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaz, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Lucas, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlin, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garden, S. J.</span><span> </span><span class="NLM_article-title">C-H functionalization of 1,4-napthoquinone by oxidative coupling with anilines in the presence of a catalytic quantity of copper(III) acetate</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">5264</span><span class="NLM_x">â</span> <span class="NLM_lpage">5273</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo200354u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=5264-5273&author=C.+d.+S.+Lisboaauthor=V.+G.+Santosauthor=B.+G.+Vazauthor=N.+C.+de+Lucasauthor=M.+N.+Eberlinauthor=S.+J.+Garden&title=C-H+functionalization+of+1%2C4-napthoquinone+by+oxidative+coupling+with+anilines+in+the+presence+of+a+catalytic+quantity+of+copper%28III%29+acetate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjo200354u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo200354u%26sid%3Dliteratum%253Aachs%26aulast%3DLisboa%26aufirst%3DC.%2Bd.%2BS.%26aulast%3DSantos%26aufirst%3DV.%2BG.%26aulast%3DVaz%26aufirst%3DB.%2BG.%26aulast%3Dde%2BLucas%26aufirst%3DN.%2BC.%26aulast%3DEberlin%26aufirst%3DM.%2BN.%26aulast%3DGarden%26aufirst%3DS.%2BJ.%26atitle%3DC-H%2520functionalization%2520of%25201%252C4-napthoquinone%2520by%2520oxidative%2520coupling%2520with%2520anilines%2520in%2520the%2520presence%2520of%2520a%2520catalytic%2520quantity%2520of%2520copper%2528III%2529%2520acetate%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2011%26volume%3D76%26spage%3D5264%26epage%3D5273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Lahm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paolini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallaoro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nardi, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neddermann, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sambucini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottomley, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo Surdo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carfi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Francesco, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SteinkÃ¼hler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallinari, P.</span><span> </span><span class="NLM_article-title">Unraveling the hidden catalytic activity of vertebrate class IIa histone deacetylases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">17335</span><span class="NLM_x">â</span> <span class="NLM_lpage">17340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1073%2Fpnas.0706487104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=17956988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1ymtrjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=17335-17340&author=A.+Lahmauthor=C.+Paoliniauthor=M.+Pallaoroauthor=M.+C.+Nardiauthor=P.+Jonesauthor=P.+Neddermannauthor=S.+Sambuciniauthor=M.+J.+Bottomleyauthor=P.+Lo+Surdoauthor=A.+Carfiauthor=U.+Kochauthor=R.+De+Francescoauthor=C.+Steink%C3%BChlerauthor=P.+Gallinari&title=Unraveling+the+hidden+catalytic+activity+of+vertebrate+class+IIa+histone+deacetylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling the hidden catalytic activity of vertebrate class IIa histone deacetylases</span></div><div class="casAuthors">Lahm, A.; Paolini, C.; Pallaoro, M.; Nardi, M. C.; Jones, P.; Neddermann, P.; Sambucini, S.; Bottomley, M. J.; Lo Surdo, P.; Carfi, A.; Koch, U.; De Francesco, R.; Steinkuehler, C.; Gallinari, P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17335-17340</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Previous findings have suggested that class IIa histone deacetylases (HDACs) (HDAC4, -5, -7, and -9) are inactive on acetylated substrates, thus differing from class I and IIb enzymes.  Here, we present evidence supporting this view and demonstrate that class IIa HDACs are very inefficient enzymes on std. substrates.  We identified HDAC inhibitors unable to bind recombinant human HDAC4 while showing inhibition in a typical HDAC4 enzymic assay, suggesting that the obsd. activity rather reflects the involvement of endogenous copurified class I HDACs.  Moreover, an HDAC4 catalytic domain purified from bacteria was 1,000-fold less active than class I HDACs on std. substrates.  A catalytic Tyr is conserved in all HDACs except for vertebrate class IIa enzymes where it is replaced by His.  Given the high structural conservation of HDAC active sites, we predicted the class IIa His-NÎµ2 to be too far away to functionally substitute the class I Tyr-OH in catalysis.  Consistently, a Tyr-to-His mutation in class I HDACs severely reduced their activity.  More importantly, a His-976-Tyr mutation in HDAC4 produced an enzyme with a catalytic efficiency 1,000-fold higher than WT, and this "gain of function phenotype" could be extended to HDAC5 and -7.  We also identified trifluoroacetyl-lysine as a class IIa-specific substrate in vitro.  Hence, vertebrate class IIa HDACs may have evolved to maintain low basal activities on acetyl-lysines and to efficiently process restricted sets of specific, still undiscovered natural substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjZBigKr0Lb7Vg90H21EOLACvtfcHk0ljQnc44vAvX2g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1ymtrjO&md5=a8345ac68fbddc344e9e0737472e2863</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0706487104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0706487104%26sid%3Dliteratum%253Aachs%26aulast%3DLahm%26aufirst%3DA.%26aulast%3DPaolini%26aufirst%3DC.%26aulast%3DPallaoro%26aufirst%3DM.%26aulast%3DNardi%26aufirst%3DM.%2BC.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DNeddermann%26aufirst%3DP.%26aulast%3DSambucini%26aufirst%3DS.%26aulast%3DBottomley%26aufirst%3DM.%2BJ.%26aulast%3DLo%2BSurdo%26aufirst%3DP.%26aulast%3DCarfi%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DU.%26aulast%3DDe%2BFrancesco%26aufirst%3DR.%26aulast%3DSteink%25C3%25BChler%26aufirst%3DC.%26aulast%3DGallinari%26aufirst%3DP.%26atitle%3DUnraveling%2520the%2520hidden%2520catalytic%2520activity%2520of%2520vertebrate%2520class%2520IIa%2520histone%2520deacetylases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D17335%26epage%3D17340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Tao Zeng, Zhihong Liu, Huawei Liu, Wengan He, Xiaowen Tang, Liwei Xie, <span class="NLM_string-name hlFld-ContribAuthor">Ruibo Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Chemical and Biological Space of Terpenoids. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (9)
                                     , 3667-3678. <a href="https://doi.org/10.1021/acs.jcim.9b00443" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00443</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00443%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExploring%252BChemical%252Band%252BBiological%252BSpace%252Bof%252BTerpenoids%26aulast%3DZeng%26aufirst%3DTao%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D30052019%26date%3D21082019%26date%3D12082019%26volume%3D59%26issue%3D9%26spage%3D3667%26epage%3D3678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">NoÃ© Sturm, Jiangming Sun, Yves Vandriessche, Andreas Mayr, GÃ¼nter Klambauer, Lars Carlsson, Ola Engkvist, <span class="NLM_string-name hlFld-ContribAuthor">Hongming Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Application of Bioactivity Profile-Based Fingerprints for Building Machine Learning Models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 962-972. <a href="https://doi.org/10.1021/acs.jcim.8b00550" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00550</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00550%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DApplication%252Bof%252BBioactivity%252BProfile-Based%252BFingerprints%252Bfor%252BBuilding%252BMachine%252BLearning%252BModels%26aulast%3DSturm%26aufirst%3DNo%25C3%25A9%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D15082018%26date%3D29112018%26date%3D09112018%26volume%3D59%26issue%3D3%26spage%3D962%26epage%3D972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter S.  Kutchukian</span>, <span class="hlFld-ContribAuthor ">Lee  Warren</span>, <span class="hlFld-ContribAuthor ">Brian C.  Magliaro</span>, <span class="hlFld-ContribAuthor ">Adam  Amoss</span>, <span class="hlFld-ContribAuthor ">Jason A.  Cassaday</span>, <span class="hlFld-ContribAuthor ">Gregory  OâDonnell</span>, <span class="hlFld-ContribAuthor ">Brian  Squadroni</span>, <span class="hlFld-ContribAuthor ">Paul  Zuck</span>, <span class="hlFld-ContribAuthor ">Danette  Pascarella</span>, <span class="hlFld-ContribAuthor ">J. Chris  Culberson</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Cooke</span>, <span class="hlFld-ContribAuthor ">Danielle  Hurzy</span>, <span class="hlFld-ContribAuthor ">Kelly-Ann Sondra  Schlegel</span>, <span class="hlFld-ContribAuthor ">Fiona  Thomson</span>, <span class="hlFld-ContribAuthor ">Eric N.  Johnson</span>, <span class="hlFld-ContribAuthor ">Victor N.  Uebele</span>, <span class="hlFld-ContribAuthor ">Jeffrey D.  Hermes</span>, <span class="hlFld-ContribAuthor ">Sophie  Parmentier-Batteur</span>, and <span class="hlFld-ContribAuthor ">Michael  Finley</span>  . </span><span class="cited-content_cbyCitation_article-title">Iterative Focused Screening with Biological Fingerprints Identifies Selective Asc-1 Inhibitors Distinct from Traditional High Throughput Screening. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2017,</strong> <em>12 </em>
                                    (2)
                                     , 519-527. <a href="https://doi.org/10.1021/acschembio.6b00913" title="DOI URL">https://doi.org/10.1021/acschembio.6b00913</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.6b00913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.6b00913%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DIterative%252BFocused%252BScreening%252Bwith%252BBiological%252BFingerprints%252BIdentifies%252BSelective%252BAsc-1%252BInhibitors%252BDistinct%252Bfrom%252BTraditional%252BHigh%252BThroughput%252BScreening%26aulast%3DKutchukian%26aufirst%3DPeter%2BS.%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D2016%26date%3D17102016%26date%3D29122016%26date%3D06012017%26date%3D17022017%26date%3D29122016%26volume%3D12%26issue%3D2%26spage%3D519%26epage%3D527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kazi Yasin  Helal</span>, <span class="hlFld-ContribAuthor ">Mateusz  Maciejewski</span>, <span class="hlFld-ContribAuthor ">Elisabet  Gregori-PuigjanÃ©</span>, <span class="hlFld-ContribAuthor ">Meir  Glick</span>, and <span class="hlFld-ContribAuthor ">Anne Mai  Wassermann</span>  . </span><span class="cited-content_cbyCitation_article-title">Public Domain HTS Fingerprints: Design and Evaluation of Compound Bioactivity Profiles from PubChemâs Bioassay Repository. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (2)
                                     , 390-398. <a href="https://doi.org/10.1021/acs.jcim.5b00498" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00498</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00498%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPublic%252BDomain%252BHTS%252BFingerprints%25253A%252BDesign%252Band%252BEvaluation%252Bof%252BCompound%252BBioactivity%252BProfiles%252Bfrom%252BPubChem%2525E2%252580%252599s%252BBioassay%252BRepository%26aulast%3DHelal%26aufirst%3DKazi%2BYasin%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D10082015%26date%3D14012016%26date%3D22022016%26volume%3D56%26issue%3D2%26spage%3D390%26epage%3D398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jenny  Balfer</span> and <span class="hlFld-ContribAuthor ">JÃ¼rgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Introduction of a Methodology for Visualization and Graphical Interpretation of Bayesian Classification Models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (9)
                                     , 2451-2468. <a href="https://doi.org/10.1021/ci500410g" title="DOI URL">https://doi.org/10.1021/ci500410g</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500410g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500410g%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIntroduction%252Bof%252Ba%252BMethodology%252Bfor%252BVisualization%252Band%252BGraphical%252BInterpretation%252Bof%252BBayesian%252BClassification%252BModels%26aulast%3DBalfer%26aufirst%3DJenny%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D09072014%26date%3D27082014%26date%3D22092014%26date%3D19082014%26volume%3D54%26issue%3D9%26spage%3D2451%26epage%3D2468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Atsushi  Yoshimori</span>, <span class="hlFld-ContribAuthor ">Enzo  Kawasaki</span>, <span class="hlFld-ContribAuthor ">Ryuta  Murakami</span>, <span class="hlFld-ContribAuthor ">Chisato  Kanai</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel eEF2K Inhibitors Using HTS Fingerprint Generated from Predicted Profiling of Compound-Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">Medicines</span><span> <strong>2021,</strong> <em>8 </em>
                                    (5)
                                     , 23. <a href="https://doi.org/10.3390/medicines8050023" title="DOI URL">https://doi.org/10.3390/medicines8050023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/medicines8050023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmedicines8050023%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicines%26atitle%3DDiscovery%252Bof%252BNovel%252BeEF2K%252BInhibitors%252BUsing%252BHTS%252BFingerprint%252BGenerated%252Bfrom%252BPredicted%252BProfiling%252Bof%252BCompound-Protein%252BInteractions%26aulast%3DYoshimori%26aufirst%3DAtsushi%26date%3D2021%26date%3D2021%26volume%3D8%26issue%3D5%26spage%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandrine  Baltzer</span>, <span class="hlFld-ContribAuthor ">Enno  Klussmann</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecules for modulating the localisation of the water channel aquaporin-2âdisease relevance and perspectives for targeting local cAMP signalling. </span><span class="cited-content_cbyCitation_journal-name">Naunyn-Schmiedeberg's Archives of Pharmacology</span><span> <strong>2019,</strong> <em>392 </em>
                                    (9)
                                     , 1049-1064. <a href="https://doi.org/10.1007/s00210-019-01686-3" title="DOI URL">https://doi.org/10.1007/s00210-019-01686-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00210-019-01686-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00210-019-01686-3%26sid%3Dliteratum%253Aachs%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Archives%2520of%2520Pharmacology%26atitle%3DSmall%252Bmolecules%252Bfor%252Bmodulating%252Bthe%252Blocalisation%252Bof%252Bthe%252Bwater%252Bchannel%252Baquaporin-2%2525E2%252580%252594disease%252Brelevance%252Band%252Bperspectives%252Bfor%252Btargeting%252Blocal%252BcAMP%252Bsignalling%26aulast%3DBaltzer%26aufirst%3DSandrine%26date%3D2019%26date%3D2019%26volume%3D392%26issue%3D9%26spage%3D1049%26epage%3D1064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel Gedder  Silva</span>, <span class="hlFld-ContribAuthor ">Flavio da Silva  Emery</span>. </span><span class="cited-content_cbyCitation_article-title">Strategies towards expansion of chemical space of natural product-based compounds to enable drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Brazilian Journal of Pharmaceutical Sciences</span><span> <strong>2018,</strong> <em>54 </em>
                                    (spe)
                                     <a href="https://doi.org/10.1590/s2175-97902018000001004" title="DOI URL">https://doi.org/10.1590/s2175-97902018000001004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1590/s2175-97902018000001004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1590%2Fs2175-97902018000001004%26sid%3Dliteratum%253Aachs%26jtitle%3DBrazilian%2520Journal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DStrategies%252Btowards%252Bexpansion%252Bof%252Bchemical%252Bspace%252Bof%252Bnatural%252Bproduct-based%252Bcompounds%252Bto%252Benable%252Bdrug%252Bdiscovery%26aulast%3DSilva%26aufirst%3DDaniel%2BGedder%26date%3D2018%26date%3D2018%26volume%3D54%26issue%3Dspe" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ben  Hu</span>, <span class="hlFld-ContribAuthor ">Zheng-Kun  Kuang</span>, <span class="hlFld-ContribAuthor ">Shi-Yu  Feng</span>, <span class="hlFld-ContribAuthor ">Dong  Wang</span>, <span class="hlFld-ContribAuthor ">Song-Bing  He</span>, <span class="hlFld-ContribAuthor ">De-Xin  Kong</span>. </span><span class="cited-content_cbyCitation_article-title">Three-Dimensional Biologically Relevant Spectrum (BRS-3D): Shape Similarity Profile Based on PDB Ligands as Molecular Descriptors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2016,</strong> <em>21 </em>
                                    (11)
                                     , 1554. <a href="https://doi.org/10.3390/molecules21111554" title="DOI URL">https://doi.org/10.3390/molecules21111554</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules21111554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules21111554%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DThree-Dimensional%252BBiologically%252BRelevant%252BSpectrum%252B%252528BRS-3D%252529%25253A%252BShape%252BSimilarity%252BProfile%252BBased%252Bon%252BPDB%252BLigands%252Bas%252BMolecular%252BDescriptors%26aulast%3DHu%26aufirst%3DBen%26date%3D2016%26date%3D2016%26volume%3D21%26issue%3D11%26spage%3D1554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanmin  Zhang</span>, <span class="hlFld-ContribAuthor ">Yu  Jiao</span>, <span class="hlFld-ContribAuthor ">Xiao  Xiong</span>, <span class="hlFld-ContribAuthor ">Haichun  Liu</span>, <span class="hlFld-ContribAuthor ">Ting  Ran</span>, <span class="hlFld-ContribAuthor ">Jinxing  Xu</span>, <span class="hlFld-ContribAuthor ">Shuai  Lu</span>, <span class="hlFld-ContribAuthor ">Anyang  Xu</span>, <span class="hlFld-ContribAuthor ">Jing  Pan</span>, <span class="hlFld-ContribAuthor ">Xin  Qiao</span>, <span class="hlFld-ContribAuthor ">Zhihao  Shi</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Yadong  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment virtual screening based on Bayesian categorization for discovering novel VEGFR-2 scaffolds. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2015,</strong> <em>19 </em>
                                    (4)
                                     , 895-913. <a href="https://doi.org/10.1007/s11030-015-9592-4" title="DOI URL">https://doi.org/10.1007/s11030-015-9592-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-015-9592-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-015-9592-4%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DFragment%252Bvirtual%252Bscreening%252Bbased%252Bon%252BBayesian%252Bcategorization%252Bfor%252Bdiscovering%252Bnovel%252BVEGFR-2%252Bscaffolds%26aulast%3DZhang%26aufirst%3DYanmin%26date%3D2015%26date%3D2015%26volume%3D19%26issue%3D4%26spage%3D895%26epage%3D913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark J  Wigglesworth</span>, <span class="hlFld-ContribAuthor ">David C  Murray</span>, <span class="hlFld-ContribAuthor ">Carolyn J  Blackett</span>, <span class="hlFld-ContribAuthor ">Michael  Kossenjans</span>, <span class="hlFld-ContribAuthor ">J Willem M  Nissink</span>. </span><span class="cited-content_cbyCitation_article-title">Increasing the delivery of next generation therapeutics from high throughput screening libraries. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2015,</strong> <em>26 </em>, 104-110. <a href="https://doi.org/10.1016/j.cbpa.2015.04.006" title="DOI URL">https://doi.org/10.1016/j.cbpa.2015.04.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2015.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2015.04.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DIncreasing%252Bthe%252Bdelivery%252Bof%252Bnext%252Bgeneration%252Btherapeutics%252Bfrom%252Bhigh%252Bthroughput%252Bscreening%252Blibraries%26aulast%3DWigglesworth%26aufirst%3DMark%2BJ%26date%3D2015%26volume%3D26%26spage%3D104%26epage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anne Mai  Wassermann</span>, <span class="hlFld-ContribAuthor ">Eugen  Lounkine</span>, <span class="hlFld-ContribAuthor ">John W.  Davies</span>, <span class="hlFld-ContribAuthor ">Meir  Glick</span>, <span class="hlFld-ContribAuthor ">L. Miguel  Camargo</span>. </span><span class="cited-content_cbyCitation_article-title">The opportunities of mining historical and collective data in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2015,</strong> <em>20 </em>
                                    (4)
                                     , 422-434. <a href="https://doi.org/10.1016/j.drudis.2014.11.004" title="DOI URL">https://doi.org/10.1016/j.drudis.2014.11.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2014.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2014.11.004%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DThe%252Bopportunities%252Bof%252Bmining%252Bhistorical%252Band%252Bcollective%252Bdata%252Bin%252Bdrug%252Bdiscovery%26aulast%3DWassermann%26aufirst%3DAnne%2BMai%26date%3D2015%26volume%3D20%26issue%3D4%26spage%3D422%26epage%3D434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter S.  Kutchukian</span>, <span class="hlFld-ContribAuthor ">Sung-Sau  So</span>, <span class="hlFld-ContribAuthor ">Christian  Fischer</span>, <span class="hlFld-ContribAuthor ">Chris L.  Waller</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Library Design: Using Cheminformatics and Expert Chemists to Fill Gaps in Existing Fragment Libraries. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 43-53. <a href="https://doi.org/10.1007/978-1-4939-2486-8_5" title="DOI URL">https://doi.org/10.1007/978-1-4939-2486-8_5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-2486-8_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-2486-8_5%26sid%3Dliteratum%253Aachs%26atitle%3DFragment%252BLibrary%252BDesign%25253A%252BUsing%252BCheminformatics%252Band%252BExpert%252BChemists%252Bto%252BFill%252BGaps%252Bin%252BExisting%252BFragment%252BLibraries%26aulast%3DKutchukian%26aufirst%3DPeter%2BS.%26date%3D2015%26date%3D2015%26spage%3D43%26epage%3D53%26pub%3DSpringer%2520New%2520York%26atitle%3DFragment-Based%252BMethods%252Bin%252BDrug%252BDiscovery%26aulast%3DKlon%26aufirst%3DAnthony%2BE.%26date%3D2015%26volume%3D1289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fragment-hit expansion strategies. Shown are different strategies to explore bioactive small-molecule space starting from FBS. (a) Strategy VFL + HTS-FP uses a combination of chemistry- and biology-based expansion to select small molecules for testing. In the first step, chemical features are extracted from active and inactive fragments and used to train a model that is applied to the small-molecule screening collection. The top-ranked molecules are screened and then divided into an active and inactive training set to build a second classifier using bioactivity profiles as molecular descriptors. On the basis of this classifier, a second set of molecules is chosen for testing. (b) Strategy VFL selects the entire screening subset based on the chemistry-based expansion of fragments. (c) Strategy VFL + ECFP4 uses two expansion steps and is similar to VFL + HTS-FP but uses chemical features instead of bioactivity profiles to train the second model (i.e., both expansion steps are chemistry-based).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Recovery rates at different selection set sizes. Fractions of identified active molecules are reported for subsets comprising 1, 5, or 10% of the full screening sets.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. From fragments to diverse chemotypes. For the data set SER/THR 1, exemplary structures of active ligands discovered in the different steps of the VFL + HTS-FP approach are shown. In the skeletal formulas, substructures identified as enriched in the FBS hits (favorable) are rendered green, whereas those identified as enriched in nonhits (unfavorable) are rendered red. Cocrystal structures of some of the identified ligands bound to the conserved ATP site of similar kinases (although not identical) are used to illustrate the likely binding pose of the compounds to the serine/threonine kinase that was screened. For compound <b>4</b>, identified by VFL, the favorable substructures intimately interact with the hinge region of Lck (A; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3F">4C3F</a>). When HTS fingerprint expansion was employed, the new chemotype contained in compound <b>5</b> was identified, and likewise portions of <b>5</b> make contacts with the hinge region of IRAK-4 (B; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2NRY">2NRY</a>). For Compound <b>6</b>, substructures identified as favorable in FBS hits interact with the hinge region, whereas an unfavorable substructure is also present that interacts elsewhere in the ATP-binding pocket of EGFR (C; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ITY">2ITY</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Active compound and scaffold retrieval for different selection strategies. For each data set, enrichment factors of active compounds in sets selected by VFL, VFL + HTS-FP, or VFL + ECFP4 over randomly chosen screening sets of the same size are displayed (left plot for each target). For all active compounds in a data set, scaffolds are calculated, and the percentages of active scaffolds retrieved at different screening set sizes are shown for the three different strategies (right plot for each target). Data set designations are used according to Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Sampling bioactive chemical space. For the data sets (a) IG and (b) HDAC, active molecules are projected into 3D chemical space using multidimensional scaling (MDS; three coordinates of MDS: CMDS1, CMDS2, and CMDS3). For strategies VFL + HTSP-FP and VFL + ECFP4, compounds are colored according to the set to which they belong: found by VFL (red), found by second biology- or chemistry-based expansion step (green), or not tested (blue). For both data sets, the second expansion step was very critical for the overall performance of our approach because the first step (VFL) failed or showed only limited enrichment of active compounds. The figure demonstrates the limited capability of the chemistry-based expansion step to move into new chemical space which is, in most cases, resolved by HTS-FP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/medium/jm-2013-01309q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. HDAC4 inhibitors. Structures of the two HDAC4 inhibitors (<b>7</b> and <b>8</b>) identified in the prospective application of the VFL + HTS-FP approach are shown and annotated with their IC<sub>50</sub> values and PubChem<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> compound identifiers (CIDs). Furthermore, we also report IC<sub>50</sub> values for three structural analogs (<b>9</b>, <b>10</b>, and <b>11</b>) that were tested in the HDAC4 assay to explore structureâactivity relationships.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-21/jm401309q/production/images/large/jm-2013-01309q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401309q&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14670" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14670" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 30 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kitchen, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decornez, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furr, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Docking and scoring in virtual screening for drug discovery: Methods and applications</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">â</span> <span class="NLM_lpage">949</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2Fnrd1549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=15520816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=935-949&author=D.+B.+Kitchenauthor=H.+Decornezauthor=J.+R.+Furrauthor=J.+Bajorath&title=Docking+and+scoring+in+virtual+screening+for+drug+discovery%3A+Methods+and+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and scoring in virtual screening for drug discovery: methods and applications</span></div><div class="casAuthors">Kitchen, Douglas B.; Decornez, Helene; Furr, John R.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">935-949</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Computational approaches that 'dock' small mols. into the structures of macromol. targets and 'score' their potential complementarity to binding sites are widely used in hit identification and lead optimization.  Indeed, there are now a no. of drugs whose development was heavily influenced by or based on structure-based design and screening strategies, such as HIV protease inhibitors.  Nevertheless, there remain significant challenges in the application of these approaches, in particular in relation to current scoring schemes.  Here, we review key concepts and specific features of small-mol.-protein docking methods, highlight selected applications and discuss recent advances that aim to address the acknowledged limitations of established approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMLibgFe3mfLVg90H21EOLACvtfcHk0li_TknIFalL7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D&md5=875a2b37a4299181509a1922b11dbd2f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1549%26sid%3Dliteratum%253Aachs%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DDecornez%26aufirst%3DH.%26aulast%3DFurr%26aufirst%3DJ.%2BR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDocking%2520and%2520scoring%2520in%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520Methods%2520and%2520applications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D935%26epage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Kutchukian, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakhnovich, E. I.</span><span> </span><span class="NLM_article-title">De novo design: balancing novelty and confined chemical space</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">789</span><span class="NLM_x">â</span> <span class="NLM_lpage">812</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1517%2F17460441.2010.497534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=22827800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCgtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=789-812&author=P.+S.+Kutchukianauthor=E.+I.+Shakhnovich&title=De+novo+design%3A+balancing+novelty+and+confined+chemical+space"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">De novo design: balancing novelty and confined chemical space</span></div><div class="casAuthors">Kutchukian, Peter S.; Shakhnovich, Eugene I.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">789-812</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: De novo drug design serves as a tool for the discovery of new ligands for macromol. targets as well as optimization of known ligands.  Recently developed tools aim to address the multi-objective nature of drug design in an unprecedented manner.  Areas covered in this review: This article discusses recent advances in de novo drug design programs and accessory programs used to evaluate compds. post-generation.  What the reader will gain: The reader is introduced to the challenges inherent in de novo drug design and will become familiar with current trends in de novo design.  Furthermore, the reader will be better prepd. to assess the value of a tool, and be equipped to design more elegant tools in the future.  Take home message: De novo drug design can assist in the efficient discovery of new compds. with a high affinity for a given target.  The inclusion of existing chemoinformatic methods with current structure-based de novo design tools provides a means of enhancing the therapeutic value of these generated compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPazAD3Wic-rVg90H21EOLACvtfcHk0li_TknIFalL7g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCgtLk%253D&md5=404b8e0c9ca878454d236417c6a82af8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1517%2F17460441.2010.497534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2010.497534%26sid%3Dliteratum%253Aachs%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26atitle%3DDe%2520novo%2520design%253A%2520balancing%2520novelty%2520and%2520confined%2520chemical%2520space%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D789%26epage%3D812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Hartenfeller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span> </span><span class="NLM_article-title">De novo design</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">672</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">â</span> <span class="NLM_lpage">323</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1007%2F978-1-60761-839-3_12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=20838974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlejsr7L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=672&publication_year=2011&pages=299-323&author=M.+Hartenfellerauthor=G.+Schneider&title=De+novo+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">De novo drug design</span></div><div class="casAuthors">Hartenfeller, Markus; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">672</span>
        (<span class="NLM_cas:issue">Chemoinformatics and Computational Chemical Biology</span>),
    <span class="NLM_cas:pages">299-323</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Computer-assisted mol. design supports drug discovery by suggesting novel chemotypes and compd. modifications for lead structure optimization.  While the aspect of synthetic feasibility of the automatically designed compds. has been neglected for a long time, we are currently witnessing an increased interest in this topic.  Here, we review state-of-the-art software for de novo drug design with a special emphasis on fragment-based techniques that generate druglike, synthetically accessible compds.  The importance of scoring functions that can be used to predict compd. reactivity and potency is highlighted, and several promising solns. are discussed.  Recent practical validation studies are presented that have already demonstrated that rule-based fragment assembly can result in novel synthesizable compds. with druglike properties and a desired biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6_YdqEvu-hbVg90H21EOLACvtfcHk0lhNqwuldm4Irg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlejsr7L&md5=3a30147da9187fa4895ba807cefed82a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2F978-1-60761-839-3_12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60761-839-3_12%26sid%3Dliteratum%253Aachs%26aulast%3DHartenfeller%26aufirst%3DM.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DDe%2520novo%2520design%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D672%26spage%3D299%26epage%3D323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Broach, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorner, J.</span><span> </span><span class="NLM_article-title">High-throughput screening for drug discovery</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">â</span> <span class="NLM_lpage">16</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2F384014a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=8895594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADyaK28Xmsl2rsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=14-16&author=J.+R.+Broachauthor=J.+Thorner&title=High-throughput+screening+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput screening for drug discovery</span></div><div class="casAuthors">Broach, James R.; Thorner, Jeremy</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">6604, Suppl.</span>),
    <span class="NLM_cas:pages">14-16</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">A review with 23 refs.  There are now a large no. of known proteins available as potential targets for drug treatment.  Concurrently, new approaches have increased the no. of compds. that can be tested for activity against these targets.  Together, these trends have stimulated the adoption of high-throughput screening as a primary tool for early-stage drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHliuK3qN3urVg90H21EOLACvtfcHk0lhNqwuldm4Irg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmsl2rsr4%253D&md5=356ca2edc709ece1a0c80f560ddd893d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2F384014a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F384014a0%26sid%3Dliteratum%253Aachs%26aulast%3DBroach%26aufirst%3DJ.%2BR.%26aulast%3DThorner%26aufirst%3DJ.%26atitle%3DHigh-throughput%2520screening%2520for%2520drug%2520discovery%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D14%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Clark, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arico-Muendel, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belyanskaya, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benjamin, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Centrella, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creaser, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuozzo, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davie, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franzen, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gefter, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hale, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, N. J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Israel, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavarana, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelley, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollmann, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lind, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mataruse, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medeiros, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Messer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâKeefe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliff, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rise, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satz, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skinner, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svendsen, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vloten, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, B. A.</span><span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">â</span> <span class="NLM_lpage">654</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+V.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kelleyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lhNqwuldm4Irg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelley%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3Dvan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Loving, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alberts, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Computational approaches for fragment-based and de novo design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">â</span> <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.2174%2F156802610790232305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=19929832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=14-32&author=K.+Lovingauthor=I.+Albertsauthor=W.+Sherman&title=Computational+approaches+for+fragment-based+and+de+novo+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Computational approaches for fragment-based and de novo design</span></div><div class="casAuthors">Loving, Kathryn; Alberts, Ian; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-32</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based and de novo design strategies have been used in drug discovery for years.  The methodologies for these strategies are typically discussed sep., yet the applications of these techniques overlap substantially.  We present a review of various fragment-based discovery and de novo design protocols with an emphasis on successful applications in real-world drug discovery projects.  Furthermore, we illustrate the strengths and weaknesses of the various approaches and discuss how one method can be used to complement another.  We also discuss how the incorporation of exptl. data as constraints in computational models can produce novel compds. that occupy unique areas in intellectual property (IP) space yet are biased toward the desired chem. property space.  Finally, we present recent research results suggesting that computational tools applied to fragment-based discovery and de novo design can have a greater impact on the discovery process when coupled with the right expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ4xFpqSbTRbVg90H21EOLACvtfcHk0lj9OK7oBGJHbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu70%253D&md5=c68f8cf413e1006c478f9cff5b170386</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F156802610790232305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232305%26sid%3Dliteratum%253Aachs%26aulast%3DLoving%26aufirst%3DK.%26aulast%3DAlberts%26aufirst%3DI.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DComputational%2520approaches%2520for%2520fragment-based%2520and%2520de%2520novo%2520design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D14%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span> <span class="citation_source-book">Library Design, Search Methods, and Applications of Fragment-Based Drug Design</span>; <span class="NLM_contrib-group">Bienstock, R. J.</span>, Ed.; <span class="NLM_publisher-name">American Chemical Society</span>: <span class="NLM_publisher-loc">Washington, DC</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks">[<a href="/doi/10.1021/bk-2011-1076" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=R.+J.+Bienstock&title=Library+Design%2C+Search+Methods%2C+and+Applications+of+Fragment-Based+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fbk-2011-1076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1021%252Fbk-2011-1076%26sid%3Dliteratum%253Aachs%26btitle%3DLibrary%2520Design%252C%2520Search%2520Methods%252C%2520and%2520Applications%2520of%2520Fragment-Based%2520Drug%2520Design%26aulast%3DBienstock%26aufirst%3DR.%2BJ.%26pub%3DAmerican%2520Chemical%2520Society%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Baker, M.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery grows up</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">â</span> <span class="NLM_lpage">7</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2Fnrd3926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=23274457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFOi" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=5-7&author=M.+Baker&title=Fragment-based+lead+discovery+grows+up"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery grows up</span></div><div class="casAuthors">Baker, Monya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With multiple drug candidates in the clinic that originated from fragment-based lead discovery, the approach of starting small has become big.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKfzq4txN2bVg90H21EOLACvtfcHk0lj9OK7oBGJHbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFOi&md5=854c670bd7388d04f76867d5818a5191</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DM.%26atitle%3DFragment-based%2520lead%2520discovery%2520grows%2520up%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D5%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Abad-Zapatero, C.</span><span> </span><span class="NLM_article-title">Ligand efficiency indices for effective drug discovery</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">â</span> <span class="NLM_lpage">488</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1517%2F17460441.2.4.469" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=23484756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVCqsL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=469-488&author=C.+Abad-Zapatero&title=Ligand+efficiency+indices+for+effective+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency indices for effective drug discovery</span></div><div class="casAuthors">Abad-Zapatero, Cele</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-488</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Successful drug discovery requires the optimization of a large no. of variables ranging from strictly physicochem. parameters such as mol. wt. to more complex parameters related to toxicity and bioavailability.  Presently, structure-based methodologies influence many aspects of the drug discovery process from lead discovery to the final preclin. characterization.  However, crit. biol. issues along the path to the market have diminished the impact and power of this methodol.  The physicochem. properties of the novel chem. entities designed and guided by structural methods have become the subject of intense scrutiny from lead discovery to drug candidate.  The idea of ligand efficiency (binding energy/non-hydrogen atoms) has recently emerged as a useful guide to optimize fragment and lead selection in the discovery process.  More generalized concepts of ligand efficiency, related to efficiency per dalton and per unit of polar surface area, have also been introduced and will be discussed in the broader context.  Preliminary results and trends obtained using ligand efficiencies as guides are reviewed and their future application to guide drug discovery will be discussed, as well as their integration into the structure-based drug design methods to make them more effective and numerically robust.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmKduo41JcS7Vg90H21EOLACvtfcHk0lj9OK7oBGJHbw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVCqsL4%253D&md5=a1682fbc4545d8eab3a44417c0e88869</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1517%2F17460441.2.4.469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2.4.469%26sid%3Dliteratum%253Aachs%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26atitle%3DLigand%2520efficiency%2520indices%2520for%2520effective%2520drug%2520discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2007%26volume%3D2%26spage%3D469%26epage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Ferenczy, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">How fragments are optimized? A retrospective analysis of 145 fragment optimizations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2478</span><span class="NLM_x">â</span> <span class="NLM_lpage">2486</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm301851v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2478-2486&author=G.+G.+Ferenczyauthor=G.+M.+Keseru&title=How+fragments+are+optimized%3F+A+retrospective+analysis+of+145+fragment+optimizations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm301851v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301851v%26sid%3Dliteratum%253Aachs%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DHow%2520fragments%2520are%2520optimized%253F%2520A%2520retrospective%2520analysis%2520of%2520145%2520fragment%2520optimizations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2478%26epage%3D2486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Crisman, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheiber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sukuru, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fejzo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hommel, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">Virtual fragment linking: An approach to identify potent binders from low affinity fragment hits</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2481</span><span class="NLM_x">â</span> <span class="NLM_lpage">2491</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701314u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2481-2491&author=T.+J.+Crismanauthor=A.+Benderauthor=M.+Milikauthor=J.+L.+Jenkinsauthor=J.+Scheiberauthor=S.+C.+Sukuruauthor=J.+Fejzoauthor=U.+Hommelauthor=J.+W.+Daviesauthor=M.+Glick&title=Virtual+fragment+linking%3A+An+approach+to+identify+potent+binders+from+low+affinity+fragment+hits"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm701314u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701314u%26sid%3Dliteratum%253Aachs%26aulast%3DCrisman%26aufirst%3DT.%2BJ.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMilik%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DSukuru%26aufirst%3DS.%2BC.%26aulast%3DFejzo%26aufirst%3DJ.%26aulast%3DHommel%26aufirst%3DU.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DVirtual%2520fragment%2520linking%253A%2520An%2520approach%2520to%2520identify%2520potent%2520binders%2520from%2520low%2520affinity%2520fragment%2520hits%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2481%26epage%3D2491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Rogers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, M.</span><span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">â</span> <span class="NLM_lpage">754</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0lhbsvx_Qc7IbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Duda, R. O.; Hart, P. E.; Stork, D. G.</span> <span class="citation_source-book">Pattern Classification</span>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Wiley-Interscience</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2001</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">20</span><span class="NLM_x">â</span> <span class="NLM_lpage">83</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=20-83&author=R.+O.+Duda&author=P.+E.+Hart&author=D.+G.+Stork&title=Pattern+Classification"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDuda%26aufirst%3DR.%2BO.%26btitle%3DPattern%2520Classification%26pub%3DWiley-Interscience%26date%3D2001%26spage%3D20%26epage%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Petrone, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simms, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigsch, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutchukian, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">Rethinking molecular similarity: Comparing compounds on the basis of biological activity</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1399</span><span class="NLM_x">â</span> <span class="NLM_lpage">1409</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=1399-1409&author=P.+Petroneauthor=B.+Simmsauthor=F.+Nigschauthor=E.+Lounkineauthor=P.+Kutchukianauthor=Z.+Dengauthor=J.+W.+Daviesauthor=J.+L.+Jenkinsauthor=M.+Glick&title=Rethinking+molecular+similarity%3A+Comparing+compounds+on+the+basis+of+biological+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPetrone%26aufirst%3DP.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKutchukian%26aufirst%3DP.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DRethinking%2520molecular%2520similarity%253A%2520Comparing%2520compounds%2520on%2520the%2520basis%2520of%2520biological%2520activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D17%26spage%3D1399%26epage%3D1409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">Bioturbo similarity searching: combining chemical and biological similarity to discover structurally diverse bioactive molecules</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">692</span><span class="NLM_x">â</span> <span class="NLM_lpage">703</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci300607r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsVylsro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=692-703&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=M.+Glick&title=Bioturbo+similarity+searching%3A+combining+chemical+and+biological+similarity+to+discover+structurally+diverse+bioactive+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Bioturbo Similarity Searching: Combining Chemical and Biological Similarity To Discover Structurally Diverse Bioactive Molecules</span></div><div class="casAuthors">Wassermann, Anne Mai; Lounkine, Eugen; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">692-703</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening using bioactivity profiles has become an integral part of currently applied hit finding methods in pharmaceutical industry.  However, a significant drawback of this approach is that it is only applicable to compds. that have been biol. tested in the past and have sufficient activity annotations for meaningful profile comparisons.  Although bioactivity data generated in pharmaceutical institutions are growing on an unprecedented scale, the no. of biol. annotated compds. still covers only a minuscule fraction of chem. space.  For a newly synthesized compd. or an isolated natural product to be biol. characterized across multiple assays, it may take a considerable amt. of time.  Consequently, this chem. matter will not be included in virtual screening campaigns based on bioactivity profiles.  To overcome this problem, we herein introduce bioturbo similarity searching that uses chem. similarity to map mols. without biol. annotations into bioactivity space and then searches for biol. similar compds. in this ref. system.  In benchmark calcns. on primary screening data, we demonstrate that our approach generally achieves higher hit rates and identifies structurally more diverse compds. than approaches using chem. information only.  Furthermore, our method is able to discover hits with novel modes of inhibition that traditional 2D and 3D similarity approaches are unlikely to discover.  Test calcns. on a set of natural products reveal the practical utility of the approach for identifying novel and synthetically more accessible chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5lay0o3odq7Vg90H21EOLACvtfcHk0lhbsvx_Qc7IbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsVylsro%253D&md5=0df2d6699f97e95b0f04e40ac28c17fe</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci300607r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300607r%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DBioturbo%2520similarity%2520searching%253A%2520combining%2520chemical%2520and%2520biological%2520similarity%2520to%2520discover%2520structurally%2520diverse%2520bioactive%2520molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D692%26epage%3D703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">â</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lhbsvx_Qc7IbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">â</span> <span class="NLM_lpage">2893</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lj6Tr--R4qXqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Borg, I.; Groenen, P.</span> <span class="citation_source-book">Modern Multidimensional Scaling: Theory and Applications</span>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=I.+Borg&author=P.+Groenen&title=Modern+Multidimensional+Scaling%3A+Theory+and+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBorg%26aufirst%3DI.%26btitle%3DModern%2520Multidimensional%2520Scaling%253A%2520Theory%2520and%2520Applications%26pub%3DSpringer%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Chemical similarity searching</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">â</span> <span class="NLM_lpage">996</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9800211" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADyaK1cXks1ynu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=983-996&author=P.+Willett&title=Chemical+similarity+searching"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical similarity searching</span></div><div class="casAuthors">Willett, Peter; Barnard, John M.; Downs, Geoffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">983-996</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper reviews the use of similarity searching in chem. databases.  It begins by introducing the concept of similarity searching, differentiating it from the more common substructure searching, and then discusses the current generation of fragment-based measures that are used for searching chem. structure databases.  The next sections focus upon two of the principal characteristics of a similarity measure: the coeff. that is used to quantify the degree of structural resemblance between pairs of mols. and the structural representations that are used to characterize mols. that are being compared in a similarity calcn.  New types of similarity measure are then compared with current approaches, and examples are given of several applications that are related to similarity searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxjjSk407VCLVg90H21EOLACvtfcHk0lj6Tr--R4qXqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXks1ynu74%253D&md5=4233b0ea4edc052f0186cccbd5eae427</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci9800211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9800211%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DChemical%2520similarity%2520searching%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D983%26epage%3D996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: A large-scale bioactivity database for drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">â</span> <span class="NLM_lpage">D1107</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+large-scale+bioactivity+database+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lhtCSfLegBMvA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span>GOSTAR Online StructureâActivity Relationship Database. <a href="http://gostardb.com/gostar/" class="extLink">http://gostardb.com/gostar/</a> (accessed October 10,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GOSTAR+Online+Structure%E2%80%93Activity+Relationship+Database.+http%3A%2F%2Fgostardb.com%2Fgostar%2F+%28accessed+October+10%2C+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DGOSTAR%2520Online%2520Structure%25E2%2580%2593Activity%2520Relationship%2520Database%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Perola, E.</span><span> </span><span class="NLM_article-title">An analysis of the binding efficiencies of drugs and their leads in successful drug discovery programs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2986</span><span class="NLM_x">â</span> <span class="NLM_lpage">2997</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100118x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlWqu7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2986-2997&author=E.+Perola&title=An+analysis+of+the+binding+efficiencies+of+drugs+and+their+leads+in+successful+drug+discovery+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An Analysis of the Binding Efficiencies of Drugs and their Leads in Successful Drug Discovery Programs</span></div><div class="casAuthors">Perola, Emanuele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2986-2997</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In order to investigate the evolution of binding efficiency in successful drug discovery programs, a data set of 60 lead/drug pairs with known binding affinities has been compiled and analyzed.  Low-end thresholds for the binding efficiencies of viable leads and drugs have been derived.  On av., the drugs in the set are significantly larger and more potent but have similar lipophilicity relative to their originating leads, suggesting that the ability to maintain low levels of lipophilicity while increasing mol. wt. is one of the keys to a successful drug discovery program.  A no. of examples demonstrate that large increases in binding efficiency from leads to more elaborate drugs sharing the same scaffold can be achieved.  The importance of dissecting a lead structure to identify the most efficient fragments and the option of sacrificing binding efficiency to optimize other properties are discussed, and relevant examples are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRqzYFBuX2Q7Vg90H21EOLACvtfcHk0lhtCSfLegBMvA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlWqu7o%253D&md5=a7eaad792f1518d093588951df64d22f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm100118x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100118x%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26atitle%3DAn%2520analysis%2520of%2520the%2520binding%2520efficiencies%2520of%2520drugs%2520and%2520their%2520leads%2520in%2520successful%2520drug%2520discovery%2520programs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2986%26epage%3D2997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span> </span><span class="NLM_article-title">The NCI60 human tumour cell line anticancer drug screen</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">â</span> <span class="NLM_lpage">823</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1038%2Fnrc1951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=16990858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVCmu7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=813-823&author=R.+H.+Shoemaker&title=The+NCI60+human+tumour+cell+line+anticancer+drug+screen"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The NCI60 human tumour cell line anticancer drug screen</span></div><div class="casAuthors">Shoemaker, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">813-823</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The US National Cancer Institute (NCI) 60 human tumor cell line anticancer drug screen (NCI60) was developed in the late 1980s as an in vitro drug-discovery tool intended to supplant the use of transplantable animal tumors in anticancer drug screening.  This screening model was rapidly recognized as a rich source of information about the mechanisms of growth inhibition and tumor-cell kill.  Recently, its role has changed to that of a service screen supporting the cancer research community.  The author reviews the development, use and productivity of the screen, highlighting several outcomes that have contributed to advances in cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ysyTSc1-p7Vg90H21EOLACvtfcHk0lh4IZ-I9xSWEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVCmu7c%253D&md5=c6c1a074b9dc7e138f45734499769284</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrc1951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1951%26sid%3Dliteratum%253Aachs%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26atitle%3DThe%2520NCI60%2520human%2520tumour%2520cell%2520line%2520anticancer%2520drug%2520screen%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2006%26volume%3D6%26spage%3D813%26epage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">PubChem as a public resource for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1052</span><span class="NLM_x">â</span> <span class="NLM_lpage">1057</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=1052-1057&author=Q.+Liauthor=T.+Chengauthor=Y.+Wangauthor=S.+H.+Bryant&title=PubChem+as+a+public+resource+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubChem%2520as%2520a%2520public%2520resource%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26spage%3D1052%26epage%3D1057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span>Lilly Open Innovation Drug Discovery Home Page. <a href="https://openinnovation.lilly.com/dd/" class="extLink">https://openinnovation.lilly.com/dd/</a> (accessed September 24,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Lilly+Open+Innovation+Drug+Discovery+Home+Page.+https%3A%2F%2Fopeninnovation.lilly.com%2Fdd%2F+%28accessed+September+24%2C+2013%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span>Harvard Medical School Library of Integrated Network-Based Cellular Signatures Home Page. <a href="http://lincs.hms.harvard.edu/" class="extLink">http://lincs.hms.harvard.edu/</a> (accessed September 24,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Harvard+Medical+School+Library+of+Integrated+Network-Based+Cellular+Signatures+Home+Page.+http%3A%2F%2Flincs.hms.harvard.edu%2F+%28accessed+September+24%2C+2013%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span>Pipeline Pilot, version 8.0; <span class="NLM_publisher-name">Accelrys Software Inc.</span>: <span class="NLM_publisher-loc">San Diego, CA</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pipeline+Pilot%2C+version+8.0%3B+Accelrys+Software+Inc.%3A+San+Diego%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPipeline%2520Pilot%26pub%3DAccelrys%2520Software%2520Inc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">â</span> <span class="NLM_lpage">2740</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljRDAhZhEyDlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Lisboa, C. d. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaz, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Lucas, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlin, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garden, S. J.</span><span> </span><span class="NLM_article-title">C-H functionalization of 1,4-napthoquinone by oxidative coupling with anilines in the presence of a catalytic quantity of copper(III) acetate</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">5264</span><span class="NLM_x">â</span> <span class="NLM_lpage">5273</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo200354u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=5264-5273&author=C.+d.+S.+Lisboaauthor=V.+G.+Santosauthor=B.+G.+Vazauthor=N.+C.+de+Lucasauthor=M.+N.+Eberlinauthor=S.+J.+Garden&title=C-H+functionalization+of+1%2C4-napthoquinone+by+oxidative+coupling+with+anilines+in+the+presence+of+a+catalytic+quantity+of+copper%28III%29+acetate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjo200354u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo200354u%26sid%3Dliteratum%253Aachs%26aulast%3DLisboa%26aufirst%3DC.%2Bd.%2BS.%26aulast%3DSantos%26aufirst%3DV.%2BG.%26aulast%3DVaz%26aufirst%3DB.%2BG.%26aulast%3Dde%2BLucas%26aufirst%3DN.%2BC.%26aulast%3DEberlin%26aufirst%3DM.%2BN.%26aulast%3DGarden%26aufirst%3DS.%2BJ.%26atitle%3DC-H%2520functionalization%2520of%25201%252C4-napthoquinone%2520by%2520oxidative%2520coupling%2520with%2520anilines%2520in%2520the%2520presence%2520of%2520a%2520catalytic%2520quantity%2520of%2520copper%2528III%2529%2520acetate%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2011%26volume%3D76%26spage%3D5264%26epage%3D5273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Lahm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paolini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallaoro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nardi, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neddermann, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sambucini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottomley, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo Surdo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carfi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Francesco, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SteinkÃ¼hler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallinari, P.</span><span> </span><span class="NLM_article-title">Unraveling the hidden catalytic activity of vertebrate class IIa histone deacetylases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">17335</span><span class="NLM_x">â</span> <span class="NLM_lpage">17340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm401309q&amp;key=10.1073%2Fpnas.0706487104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm401309q&amp;key=17956988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm401309q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1ymtrjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=17335-17340&author=A.+Lahmauthor=C.+Paoliniauthor=M.+Pallaoroauthor=M.+C.+Nardiauthor=P.+Jonesauthor=P.+Neddermannauthor=S.+Sambuciniauthor=M.+J.+Bottomleyauthor=P.+Lo+Surdoauthor=A.+Carfiauthor=U.+Kochauthor=R.+De+Francescoauthor=C.+Steink%C3%BChlerauthor=P.+Gallinari&title=Unraveling+the+hidden+catalytic+activity+of+vertebrate+class+IIa+histone+deacetylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling the hidden catalytic activity of vertebrate class IIa histone deacetylases</span></div><div class="casAuthors">Lahm, A.; Paolini, C.; Pallaoro, M.; Nardi, M. C.; Jones, P.; Neddermann, P.; Sambucini, S.; Bottomley, M. J.; Lo Surdo, P.; Carfi, A.; Koch, U.; De Francesco, R.; Steinkuehler, C.; Gallinari, P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17335-17340</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Previous findings have suggested that class IIa histone deacetylases (HDACs) (HDAC4, -5, -7, and -9) are inactive on acetylated substrates, thus differing from class I and IIb enzymes.  Here, we present evidence supporting this view and demonstrate that class IIa HDACs are very inefficient enzymes on std. substrates.  We identified HDAC inhibitors unable to bind recombinant human HDAC4 while showing inhibition in a typical HDAC4 enzymic assay, suggesting that the obsd. activity rather reflects the involvement of endogenous copurified class I HDACs.  Moreover, an HDAC4 catalytic domain purified from bacteria was 1,000-fold less active than class I HDACs on std. substrates.  A catalytic Tyr is conserved in all HDACs except for vertebrate class IIa enzymes where it is replaced by His.  Given the high structural conservation of HDAC active sites, we predicted the class IIa His-NÎµ2 to be too far away to functionally substitute the class I Tyr-OH in catalysis.  Consistently, a Tyr-to-His mutation in class I HDACs severely reduced their activity.  More importantly, a His-976-Tyr mutation in HDAC4 produced an enzyme with a catalytic efficiency 1,000-fold higher than WT, and this "gain of function phenotype" could be extended to HDAC5 and -7.  We also identified trifluoroacetyl-lysine as a class IIa-specific substrate in vitro.  Hence, vertebrate class IIa HDACs may have evolved to maintain low basal activities on acetyl-lysines and to efficiently process restricted sets of specific, still undiscovered natural substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjZBigKr0Lb7Vg90H21EOLACvtfcHk0ljRDAhZhEyDlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1ymtrjO&md5=a8345ac68fbddc344e9e0737472e2863</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0706487104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0706487104%26sid%3Dliteratum%253Aachs%26aulast%3DLahm%26aufirst%3DA.%26aulast%3DPaolini%26aufirst%3DC.%26aulast%3DPallaoro%26aufirst%3DM.%26aulast%3DNardi%26aufirst%3DM.%2BC.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DNeddermann%26aufirst%3DP.%26aulast%3DSambucini%26aufirst%3DS.%26aulast%3DBottomley%26aufirst%3DM.%2BJ.%26aulast%3DLo%2BSurdo%26aufirst%3DP.%26aulast%3DCarfi%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DU.%26aulast%3DDe%2BFrancesco%26aufirst%3DR.%26aulast%3DSteink%25C3%25BChler%26aufirst%3DC.%26aulast%3DGallinari%26aufirst%3DP.%26atitle%3DUnraveling%2520the%2520hidden%2520catalytic%2520activity%2520of%2520vertebrate%2520class%2520IIa%2520histone%2520deacetylases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D17335%26epage%3D17340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3F','PDB','4C3F'); return false;">PDB: 4C3F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2NRY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2NRY','PDB','2NRY'); return false;">PDB: 2NRY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ITY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ITY','PDB','2ITY'); return false;">PDB: 2ITY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24"><a href="/doi/suppl/10.1021/jm401309q">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_38173"></div></div></div></div></div><hr /></hr><p class="last">Enrichment factors for different subset splits; comparison of chemistry-based fragment hit expansion to random small molecule selection; coverage of bioactive chemical space with chemistry-based hit expansion; two publicly known HDAC4 inhibitors that are structurally most similar to the identified hits; fitted doseâresponse curves for the three newly identified active compounds; and <sup>1</sup>H NMR and LCâMS results for tested compounds. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm401309q/suppl_file/jm401309q_si_001.pdf">jm401309q_si_001.pdf (1.04 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The crystal structure of compound <b>4</b> with Lck can be accessed using the PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3F">4C3F</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm401309q&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm401309q%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-21%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm401309q" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677aa163789f3d7c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
