
<html lang="en"     class="pb-page"  data-request-id="bf3ad9c4-52b6-4c8c-9fb6-b2ed9f77ae00"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2015.58.issue-8;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/jm501759m;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Selective Inhibitors of Cyclin G Associated Kinase (GAK) as Anti-Hepatitis C Agents" /></meta><meta name="dc.Creator" content="Sona  Kovackova" /></meta><meta name="dc.Creator" content="Lei  Chang" /></meta><meta name="dc.Creator" content="Elena  Bekerman" /></meta><meta name="dc.Creator" content="Gregory  Neveu" /></meta><meta name="dc.Creator" content="Rina  Barouch-Bentov" /></meta><meta name="dc.Creator" content="Apirat  Chaikuad" /></meta><meta name="dc.Creator" content="Christina  Heroven" /></meta><meta name="dc.Creator" content="Michal  Šála" /></meta><meta name="dc.Creator" content="Steven  De Jonghe" /></meta><meta name="dc.Creator" content="Stefan  Knapp" /></meta><meta name="dc.Creator" content="Shirit  Einav" /></meta><meta name="dc.Creator" content="Piet  Herdewijn" /></meta><meta name="dc.Description" content="Cyclin G associated kinase (GAK) emerged as a promising drug target for the treatment of viral infections. However, no potent and selective GAK inhibitors have been reported in the literature to da..." /></meta><meta name="Description" content="Cyclin G associated kinase (GAK) emerged as a promising drug target for the treatment of viral infections. However, no potent and selective GAK inhibitors have been reported in the literature to da..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 9, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm501759m" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm501759m" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm501759m" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm501759m" /></link>
        
    
    

<title>Selective Inhibitors of Cyclin G Associated Kinase (GAK) as Anti-Hepatitis C Agents | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm501759m" /></meta><meta property="og:title" content="Selective Inhibitors of Cyclin G Associated Kinase (GAK) as Anti-Hepatitis C Agents" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0011.jpeg" /></meta><meta property="og:description" content="Cyclin G associated kinase (GAK) emerged as a promising drug target for the treatment of viral infections. However, no potent and selective GAK inhibitors have been reported in the literature to date. This paper describes the discovery of isothiazolo[5,4-b]pyridines as selective GAK inhibitors, with the most potent congeners displaying low nanomolar binding affinity for GAK. Cocrystallization experiments revealed that these compounds behaved as classic type I ATP-competitive kinase inhibitors. In addition, we have demonstrated that these compounds exhibit a potent activity against hepatitis C virus (HCV) by inhibiting two temporally distinct steps in the HCV life cycle (i.e., viral entry and assembly). Hence, these GAK inhibitors represent chemical probes to study GAK function in different disease areas where GAK has been implicated (including viral infection, cancer, and Parkinson’s disease)." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm501759m"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm501759m">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm501759m&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm501759m&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm501759m&amp;href=/doi/10.1021/jm501759m" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 3393-3410</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm501740a" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm501809x" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Selective Inhibitors of Cyclin G Associated Kinase (GAK) as Anti-Hepatitis C Agents</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sona++Kovackova">Sona Kovackova</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lei++Chang">Lei Chang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elena++Bekerman">Elena Bekerman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gregory++Neveu">Gregory Neveu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rina++Barouch-Bentov">Rina Barouch-Bentov</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Apirat++Chaikuad">Apirat Chaikuad</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christina++Heroven">Christina Heroven</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michal++%C5%A0%C3%A1la">Michal Šála</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven++De+Jonghe">Steven De Jonghe</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Knapp">Stefan Knapp</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shirit++Einav">Shirit Einav</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Piet++Herdewijn">Piet Herdewijn</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Laboratory of Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Minderbroedersstraat 10, 3000 Leuven, Belgium</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Interface Valorisation Platform, KU Leuven, Kapucijnenvoer 33, 3000 Leuven, Belgium</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Medicine, Division of Infectious Diseases and Geographic Medicine, and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Target Discovery Institute (TDI), and Structural Genomics Consortium (SGC), University of Oxford, Old Road Campus Research Building, Oxford OX3 7DQ, United Kingdom</span></div><div class="corresp-info"><strong>*</strong>Phone: +32 16 337387. Fax: +32 16 337340. E-mail: <a href="/cdn-cgi/l/email-protection#8adae3effea4c2eff8eeeffde3e0e4caf8efedeba4e1ffe6effffcefe4a4e8ef"><span class="__cf_email__" data-cfemail="2878414d5c06604d5a4c4d5f414246685a4d4f4906435d444d5d5e4d46064a4d">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm501759m&amp;href=/doi/10.1021%2Fjm501759m" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 3393–3410</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 30, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 November 2014</li><li><span class="item_label"><b>Published</b> online</span>9 April 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 April 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm501759m" title="DOI URL">https://doi.org/10.1021/jm501759m</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3393%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSona%2BKovackova%252C%2BLei%2BChang%252C%2BElena%2BBekerman%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D58%26issueNum%3D8%26contentID%3Djm501759m%26title%3DSelective%2BInhibitors%2Bof%2BCyclin%2BG%2BAssociated%2BKinase%2B%2528GAK%2529%2Bas%2BAnti-Hepatitis%2BC%2BAgents%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3410%26publicationDate%3DApril%2B2015">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm501759m"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2118</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">38</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm501759m" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Selective Inhibitors of Cyclin G Associated Kinase (GAK) as Anti-Hepatitis C Agents&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sona&quot;,&quot;last_name&quot;:&quot;Kovackova&quot;},{&quot;first_name&quot;:&quot;Lei&quot;,&quot;last_name&quot;:&quot;Chang&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Bekerman&quot;},{&quot;first_name&quot;:&quot;Gregory&quot;,&quot;last_name&quot;:&quot;Neveu&quot;},{&quot;first_name&quot;:&quot;Rina&quot;,&quot;last_name&quot;:&quot;Barouch-Bentov&quot;},{&quot;first_name&quot;:&quot;Apirat&quot;,&quot;last_name&quot;:&quot;Chaikuad&quot;},{&quot;first_name&quot;:&quot;Christina&quot;,&quot;last_name&quot;:&quot;Heroven&quot;},{&quot;first_name&quot;:&quot;Michal&quot;,&quot;last_name&quot;:&quot;Šála&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;De Jonghe&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Knapp&quot;},{&quot;first_name&quot;:&quot;Shirit&quot;,&quot;last_name&quot;:&quot;Einav&quot;},{&quot;first_name&quot;:&quot;Piet&quot;,&quot;last_name&quot;:&quot;Herdewijn&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;3393-3410&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm501759m&quot;},&quot;abstract&quot;:&quot;Cyclin G associated kinase (GAK) emerged as a promising drug target for the treatment of viral infections. However, no potent and selective GAK inhibitors have been reported in the literature to date. This paper describes the discovery of isothiazolo[5,4-b]pyridines as selective GAK inhibitors, with the most potent congeners displaying low nanomolar binding affinity for GAK. Cocrystallization experiments revealed that these compounds behaved as classic type I ATP-competitive kinase inhibitors. In addition, we have demonstrated that these compounds exhibit a potent activity against hepatitis C virus (HCV) by inhibiting two temporally distinct steps in the HCV life cycle (i.e., viral entry and assembly). Hence, these GAK inhibitors represent chemical probes to study GAK function in different disease areas where GAK has been implicated (including viral infection, cancer, and Parkinson’s disease).&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501759m&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501759m" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501759m&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501759m" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501759m&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501759m" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm501759m&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501759m&amp;href=/doi/10.1021/jm501759m" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm501759m" /></input><a href="/doi/pdf/10.1021/jm501759m" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm501759m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm501759m%26sid%3Dliteratum%253Aachs%26pmid%3D25822739%26genre%3Darticle%26aulast%3DKovackova%26date%3D2015%26atitle%3DSelective%2BInhibitors%2Bof%2BCyclin%2BG%2BAssociated%2BKinase%2B%2528GAK%2529%2Bas%2BAnti-Hepatitis%2BC%2BAgents%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D8%26spage%3D3393%26epage%3D3410%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291430" title="Infectious diseases">Infectious diseases</a>,</li><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jmcmar.2015.58.issue-8/20150423/jmcmar.2015.58.issue-8.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Cyclin G associated kinase (GAK) emerged as a promising drug target for the treatment of viral infections. However, no potent and selective GAK inhibitors have been reported in the literature to date. This paper describes the discovery of isothiazolo[5,4-<i>b</i>]pyridines as selective GAK inhibitors, with the most potent congeners displaying low nanomolar binding affinity for GAK. Cocrystallization experiments revealed that these compounds behaved as classic type I ATP-competitive kinase inhibitors. In addition, we have demonstrated that these compounds exhibit a potent activity against hepatitis C virus (HCV) by inhibiting two temporally distinct steps in the HCV life cycle (i.e., viral entry and assembly). Hence, these GAK inhibitors represent chemical probes to study GAK function in different disease areas where GAK has been implicated (including viral infection, cancer, and Parkinson’s disease).</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88818" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88818" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cyclin G associated kinase (GAK) was first identified in experiments investigating proteins associated with cyclin G, a protein involved in cell cycle regulation.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> GAK (also known as auxillin 2) is a 160 kDa serine/threonine protein kinase that belongs to the numb-associated kinase (NAK) family, which also includes STK16/MPSK1 (serine/threonine kinase 16/myristoylated and palmitoylated serine/threonine kinase 1), AAK1 (adaptor-associated kinase), and BIKE (BMP-2 inducible kinase).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">GAK is expressed ubiquitously and bears a strong homology (43%) to the neuronal-specific protein auxilin, a heat shock cognate 70 (Hsc70) cochaperone with a role in uncoating clathrin vesicles. GAK is a key regulator of clathrin-mediated trafficking in both the endocytic and secretory pathways. It recruits clathrin and clathrin adaptor protein complex 2 (AP-2) to the plasma membrane<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and phosphorylates a T156 residue within AP2M1, the μ subunit of AP-2, thereby stimulating its binding to cargo proteins and enhancing cargo recruitment, vesicle assembly, and efficient internalization.<a onclick="showRef(event, 'ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6">(3-6)</a> Moreover, GAK regulates endocytosis of receptors mediated by alternate clathrin adaptors<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and is implicated in later steps of endocytosis, including regulation of clathrin-coated vesicles (CCVs) uncoating, which enables recycling of clathrin back to the cell surface.<a onclick="showRef(event, 'ref3 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref5">(3, 5)</a> GAK is an important regulator of epidermal growth factor receptor (EGFR); it is known to promote EGF uptake<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and may also function in receptor signaling.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Last, GAK also plays an important role in regulating clathrin-mediated sorting events in the trans-Golgi network.<a onclick="showRef(event, 'ref3 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref5">(3, 5)</a></div><div class="NLM_p">Interestingly, GAK-dependent phosphorylation of clathrin adaptor proteins has been implicated in the regulation of viruses. AP2M1 was shown to be recruited to the surface of lipid droplets by the HCV capsid protein, core.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The interaction between HCV core and AP2M1 was shown to be critical for HCV assembly.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Notably, either overexpression of an AP2M1 phosphorylation-site mutant or suppression of GAK expression disrupted core-AP2M1 binding and HCV assembly.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> More recently, GAK was shown to regulate HCV entry independently of its effect on HCV assembly, in part by activating AP2M1.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Hence, GAK represents a cellular host factor essential for regulation of HCV entry and assembly and a potential target for antiviral strategies. Indeed, erlotinib, an approved anticancer drug that potently inhibits GAK (in addition to its known cancer target, EGFR<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a>), inhibits HCV entry as well as core-AP2M1 binding, thereby also disrupting HCV assembly but not HCV RNA replication.<a onclick="showRef(event, 'ref8 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref11">(8, 11)</a></div><div class="NLM_p">To the best of our knowledge, no potent and selective GAK inhibitors have been reported in the literature to date. Like erlotinib, other approved kinase inhibitors, such as dasatinib, gefitinib, and pelitinib, display a high affinity for GAK with <i>K</i><sub>d</sub> values in the low nanomolar range (Chart <a class="ref internalNav" href="#cht1" aria-label="1">1</a>).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Similarly, pyridinylimidazoles, such as SB203580 and SB201290 that have been developed as p38 inhibitors, potently inhibit GAK.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Nevertheless, since all these compounds were designed to target other kinases, their inhibitory effect on GAK represents an off-target effect, and their use is limited by significant toxicities resulting from lack of selectivity. Moreover, while several compounds that bind GAK with an excellent ligand efficiency (LE) of 0.51 kcal/mol (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) were discovered by a fragment-based screening using weak affinity chromatography, their binding affinity was low (<i>K</i><sub>d</sub> value of 2 μM).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0013.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Known GAK Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Because of the potential for GAK to serve as an antiviral drug target and the lack of selective small-molecule GAK inhibitors, we embarked on the synthesis and biological evaluation of a novel series of GAK inhibitors. In addition to their potential as lead molecules for the development of a novel antiviral strategy, these compounds represent useful chemical probes to further investigate the function of GAK in aspects of general cell biology and other disease conditions, such as cancer<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and Parkinson’s disease,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> where GAK plays an important role.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Screening: Hit Discovery</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03607" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03607" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In order to discover novel GAK inhibitors, a druglike compound library of 150 analogues was screened to identify potential ligands of GAK. This compound collection was made as part of a program to synthesize novel compound libraries based on original and patentable chemistry, which has not been explored before in drug discovery and hence represent unexplored chemical space. This library consists mainly of compounds based on a bicyclic, heteroaromatic flat scaffold. The different scaffolds are unrelated to each other, and in addition, they differ in their substitution pattern (the nature, as well as the spatial orientation of the substituents). These flat core structures are typical for kinase inhibitors, since they function as a central scaffold that binds to the ATP-binding site of the enzyme and is therefore predicted to yield a high hit rate in kinase assays. We opted for the KINOMEscan screening platform that employs an active site-directed competition binding assay to quantitatively measure interactions between a test compound and a given kinase.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Compounds that bind the kinase active site and either directly (sterically) or indirectly (allosterically) prevent kinase binding to the immobilized ligand reduce the amount of kinase captured on the solid support. Conversely, test molecules that do not bind the kinase have no effect on the amount of kinase captured on the solid support. Hits are identified by measuring the amount of kinase captured in test versus control samples using a quantitative, precise, and sensitive qPCR method that detects the associated DNA label. In the first round of screening, the compound library was tested at a single concentration of 10 μM. The results are reported as the percentage of kinase/phage remaining bound to the ligands/beads, relative to a control. High affinity compounds have % of control values close to zero, while weaker binders have higher % control values. For the most promising compounds, dose–response curves were generated to determine binding constant (<i>K</i><sub>d</sub>) values. This initial screening led to the discovery of a promising hit (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), with a good potency (%Ctrl = 0.3 ; <i>K</i><sub>d</sub> = 0.3 μM), moderate lipophilicity (cLogP = 2.44), and a favorable ligand efficiency (LE) of 0.494 kcal/mol. Overall, these parameters qualified this hit compound as an optimal starting point for the discovery of novel GAK inhibitors. Notably, this hit compound is based on an isothiazolo[4,3-<i>b</i>]pyridine scaffold, a skeleton that is unexplored in organic chemistry. This is in contrast to the bicyclic heteroaromatics (such as purines, quinazolines, benzimidazoles, and indoles), which are considered “privileged structures” in kinase drug discovery and are widely described in scientific and patent literature, making novelty difficult to achieve. Indeed, SciFinder searches revealed only 19 known isothiazolo[4,3-<i>b</i>]pyridine derivatives,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and hence, further elaboration of this scaffold gives access to a largely unexplored chemical space.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0014.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Hit compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66284" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66284" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The general approach for the synthesis of the isothiazolo[4,3-<i>b</i>]pyridine derivatives is shown in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>. The synthesis started from 3-nitro-5-bromopyridine-2-carbonitrile <b>1</b>. Reduction of the nitro group to the corresponding amino group was achieved by treatment with iron under acidic conditions.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The major compound was the desired compound <b>2</b>; however, it was always accompanied by formation of the corresponding carboxamide, resulting from acidic hydrolysis of the cyano group. This mixture was used as such in the subsequent reaction. The aromatic thioamide <b>3</b> was obtained by treatment of the mixture with phosphorus pentasulfide as thionation reagent.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> An oxidative ring closure using hydrogen peroxide yielded the 3-amino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine <b>4</b>.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Subsequent palladium-catalyzed Suzuki coupling<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> with appropriate arylboronic acids afforded a series of 3-amino-6-aryl-isothiazolo[4,3-<i>b</i>]pyridines <b>5a</b>–<b>e</b>. Alternatively, diazotation of the exocyclic amino group with sodium nitrite, hydrogen bromide, and CuBr furnished the 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine <b>6</b>.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Treatment of <b>6</b> with sodium methoxide or nitrogen-containing nucleophiles yielded a series of 3-substituted-6-bromoisothiazolo[4,3-<i>b</i>]pyridine analogues <b>7</b>. Finally, reaction of <b>7</b> with a number of arylboronic acids then yielded target compounds <b>8</b>–<b>13</b>. These Suzuki reactions were conducted in a mixture of dioxane/water, using sodium carbonate as base and Pd(dppf)Cl<sub>2</sub> as a catalyst, or alternatively in a mixture of dimethoxyethane/water with potassium carbonate as a base and Pd(PPh<sub>3</sub>)<sub>4</sub> as a catalyst.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0005.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Isothiazolo[4,3-<i>b</i>]pyridine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Fe, CH<sub>3</sub>COOH, 0 °C to rt; (b) P<sub>2</sub>S<sub>5</sub>, EtOH, 75 °C; (c) 30% aq H<sub>2</sub>O<sub>2</sub>, CH<sub>3</sub>OH, rt; (d) R<sub>6</sub>B(OH)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, Pd(dppf)Cl<sub>2</sub>, dioxane/water, 100 °C; (e) CuBr, HBr, NaNO<sub>2</sub>,H<sub>2</sub>O, 0 °C to rt; (f) R<sub>3</sub>H, EtOH, 75 °C; (g) R<sub>6</sub>B(OH)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, Pd(dppf)Cl<sub>2</sub>, dioxane/water, 100 °C or R<sub>6</sub>B(OH)<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, H<sub>2</sub>O, DME.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Structure–Activity Relationship Studies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08351" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08351" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The isothiazolo[4,3-<i>b</i>]pyridine scaffold offers two sites for structural variation. Therefore, the structure–activity relationship (SAR) study started with the synthesis of a small library matrix of isothiazolo[4,3-<i>b</i>]pyridines with structural variation at positions 3 (amino, methoxy, ethanolamine, morpholine, and <i>N</i>-Me-piperazine) and 6 (substituted aryl and heteroaryl groups). Similar to our primary screening strategy, all compounds were tested at a 10 μM concentration and the most promising ones were then subjected to determination of <i>K</i><sub>d</sub> values. The matrix depicted in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> demonstrates that compounds <b>9c</b> and <b>9d</b>, resulting from the combination of a 3,4-dimethoxyphenyl or a 3-thienyl substituent at position 6 with an ethanolamino or a morpholino moiety at position 3, have a strong affinity for GAK (<i>K</i><sub>d</sub> values of 52 and 42 nM, respectively).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. GAK Affinity Data of an Isothiazolo[4,3-<i>b</i>]pyridine Library<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0006.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0007.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Values represent the average of two independent experiments.</p></div></div><div></div></div><div class="NLM_p">A dual strategy was then used for further optimization. To probe the optimal substitution pattern at position 6, the morpholino substituent at position 3 was fixed, and a wide range of substituted aryl or heteroaryl groups was evaluated. As it became clear that a 3,4-dimethoxyphenyl or 3-thienyl substituents were optimal for GAK binding, the SAR focused on closely related derivatives of both of these aryl groups (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).</div><div class="NLM_p">Replacing the 3-thienyl group of compound <b>9d</b> by a 2-thienyl group (compound <b>12a</b>) led to equipotent compounds. On the other hand, the presence of a 2-furanyl moiety led to a 5-fold decrease in GAK affinity, yielding compound <b>12b</b> with a <i>K</i><sub>d</sub> value of 0.2 μM. The SAR of the dimethoxyphenyl ring was investigated by the synthesis of a number of regioisomeric dimethoxyphenyl derivatives. Compared to the original 3,4-dimethoxyphenyl analogue <b>9c</b> (<i>K</i><sub>d</sub> = 0.052 μM), the 2,4-dimethoxyphenyl analogue (compound <b>12c</b>) lacks any affinity for the GAK enzyme, the 2,5-dimethoxyphenyl derivative <b>12d</b> shows intermediate affinity (<i>K</i><sub>d</sub> = 0.13 μM), whereas the 3,5-dimethoxyphenyl derivative <b>12e</b> demonstrates a similar GAK affinity (<i>K</i><sub>d</sub> = 0.072 μM). The dioxolane analogue <b>12f</b> (a ring-closed analogue of the 3,4-dimethoxyphenyl moiety) is completely inactive. The insertion of an additional methoxy group yielded the 3,4,5-trimethoxyphenyl analogue <b>12g</b>, displaying very strong binding affinity for the GAK enzyme (<i>K</i><sub>d</sub> = 0.0083 μM). In another round of SAR, the 3-methoxy group was kept intact and the 4-methoxy moiety was replaced by an ester moiety (compound <b>12h</b>), an amino group (compound <b>12i</b>), and a hydroxyl function (compound <b>12j</b>), generating compounds with a strong affinity for the GAK enzyme, displaying <i>K</i><sub>d</sub> values of 0.018, 0.0089, and 0.018 μM, respectively.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the Aryl Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0008.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">Values represent the average of two independent experiments. ND = not determined.</p></div></div><div></div></div><div class="NLM_p">Since the first library screening demonstrated that a 3,4-dimethoxyphenyl moiety at position 6 was favorable for GAK binding, this substituent was kept intact, and a variety of amines were introduced at position 3 (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Although a free amino group at position 3 of the isothiazolo[5,4-<i>b</i>]pyridine scaffold is not tolerated for GAK binding (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, compound <b>5c</b>), the presence of small aliphatic amines (primary as well as secondary), such as dimethylamino (compound <b>13a</b>), methoxyethylamino (compound <b>13b</b>), diethanolamino (<b>13c</b>), gives rise to compounds displaying <i>K</i><sub>d</sub> values in the range of 0.14–0.3 μM. Cycloaliphatic amines, such as a pyrrolidino (compound <b>13d</b>) and a cyclopropylmethylamino (compound <b>13e</b>), yielded compounds with a higher potency, with <i>K</i><sub>d</sub> values of 100 and 27 nM for compounds <b>13d</b> and <b>13e</b>, respectively.</div><div class="NLM_p">The isosteric replacement of the oxygen of the morpholine moiety by a carbon (resulting in compound <b>13f</b>) or a sulfur (resulting in compound <b>13g</b>) leads to a 5-fold drop in GAK affinity when compared to the corresponding morpholine analogue <b>9c</b>. To better mimic the morpholine substituent, a 4-hydroxypiperidino and a 4-aminopyrane moiety were introduced, giving rise to isothiazolo[4,3-<i>b</i>]pyridines <b>13i</b> and <b>13j</b>, both exhibiting a strong affinity for the GAK enzyme (<i>K</i><sub>d</sub> values of 63 and 88 nM, respectively). Overall, the data in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> indicate quite high tolerance for structural variation at position 3, with only an aromatic phenethylamino group (compound <b>13k</b>) completely lacking GAK affinity.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the Amine Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0009.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0010.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">Values represent the average of two independent experiments. ND = not determined.</p></div></div><div></div></div><div class="NLM_p">Although improving binding affinity is important in an optimization campaign, an overemphasis on potency has often resulted in molecules with unsuitable physicochemical properties for further development for in vivo use. We therefore complemented our lead optimization by monitoring a number of in silico parameters (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). The optimized compounds had a low molecular weight, which in combination with their potent binding affinity yielded very “efficient” LE values exceeding 0.4 kcal/mol. Moreover, the optimization campaign did not result in any gain in lipophilicity, as the most potent GAK ligands, <b>12g</b> and <b>12i</b>, displayed cLogP values of 2.49 and 2.28, respectively.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. In Silico Parameters of Hit and Optimized Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">MW</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>d</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">LE (kcal/mol)</th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b> (hit)</td><td class="colsep0 rowsep0" align="char" char=".">277.37</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">0.494</td><td class="colsep0 rowsep0" align="char" char=".">2.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12g</b></td><td class="colsep0 rowsep0" align="char" char=".">387.45</td><td class="colsep0 rowsep0" align="char" char=".">0.0083</td><td class="colsep0 rowsep0" align="char" char=".">0.408</td><td class="colsep0 rowsep0" align="char" char=".">2.49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12i</b></td><td class="colsep0 rowsep0" align="char" char=".">342.42</td><td class="colsep0 rowsep0" align="char" char=".">0.0089</td><td class="colsep0 rowsep0" align="char" char=".">0.458</td><td class="colsep0 rowsep0" align="char" char=".">2.28</td></tr></tbody></table></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Selectivity Profiling</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26133" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26133" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Most kinase inhibitors are ATP-competitive and bind to the ATP domain, which is highly conserved across the kinome. Selectivity screens usually involve close analogues of the target kinase harboring similar ATP binding sites. However, we opted for a kinome-wide selectivity screen, using a panel of 456 kinases available at DiscoverX. As a representative example, <b>12g</b>, the most potent congener, was selected for the selectivity profiling at a single concentration of 10 μM. Remarkably, no affinity of <b>12g</b> was observed for any of the other members of the NAK family (AAK1, 90%; STK16, 92%; BIKE, 92% compared to the DMSO control) (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Moreover, only seven additional kinases appeared to interact with <b>12g</b>, with a binding activity of less than 10%, when compared to DMSO control. The exact <i>K</i><sub>d</sub> values for these kinases were determined (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). This large scale kinase profiling confirms that the identified lead compound is selective. Compound <b>12g</b> displays the strongest affinity for GAK (<i>K</i><sub>d</sub> = 8.3 nM). A 3-fold level of selectivity is observed when compared with the affinity for KIT (<i>K</i><sub>d</sub> = 29 nM), whereas for the other kinases (CLK2, CSF1R, FLT3, MEK5, PDGFRA, and PDGFRB), at least a 8-fold level of selectivity is achieved.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. <i>K</i><sub>d</sub> Values for Off-Target Kinases Affected by <b>12g</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> (nM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GAK</td><td class="colsep0 rowsep0" align="left">8.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLK2</td><td class="colsep0 rowsep0" align="left">710</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CSF1R</td><td class="colsep0 rowsep0" align="left">320</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FLT3</td><td class="colsep0 rowsep0" align="left">110</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KIT</td><td class="colsep0 rowsep0" align="left">29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MEK5</td><td class="colsep0 rowsep0" align="left">150</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDGFRA</td><td class="colsep0 rowsep0" align="left">220</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDGFRB</td><td class="colsep0 rowsep0" align="left">70</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Values represent the average of two independent experiments.</p></div></div></div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Specificity profile of compound <b>12g</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">X-ray Crystallography</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55184" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55184" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To shed a light on the binding mode of the compounds to GAK, cocrystallization experiments were initiated. We have recently solved the structure of GAK in complex with a single chain antibody (nanobody), which resulted in reproducible crystallization conditions for the GAK kinase domain.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The nanobody bound distal to the ATP binding site interacting with the lower kinase lobe (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). The lead compound <b>12i</b> was cocrystallized and bound as expected to the ATP binding site of GAK. The binding mode of the inhibitor was well-defined by electron density, and the structure was refined to 2.1 Å resolution. Data collection and refinement statistics are provided in supplemental Table 2 (see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). Two kinase domain:nanobody complexes were present in the asymmetric unit that interacted via the extended activation loops of GAK as described previously.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The upper lobe of the kinase domain was flexible as indicated by high <i>B</i>-factors in particular in the N-terminal region (<a class="ref internalNav" href="#notes-2" aria-label="Supporting Information Figure 1">Supporting Information Figure 1</a>). The helix αC and the DFG motive were in an active conformation compatible with the type I binding mode of the inhibitor (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). The nitrogen of the isothiazolo moiety of the inhibitor formed a hydrogen bond with the backbone amide of Cys126. Interactions with small gatekeepers, which are found in only a small subset of kinases, have been explored for the development of selective kinase inhibitors.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Therefore, our observation of this contact in the case of <b>12i</b> may explain its excellent selectivity for GAK. The 4-amino-3-methoxyphenyl substituent of <b>12i</b> was oriented toward the solvent exposed area forming a number of hydrophobic contacts, in particular with Leu46 and a long-range polar interaction with Arg44. Overall, <b>12i</b> exhibits a good shape complementarity with the GAK ATP binding site.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of GAK in complex with <b>12i</b>. (A) Overview of GAK:NbGAK_4 complex with inhibitor <b>12i</b> bound. The inset shows |<i>F</i><sub>O</sub>| – |<i>F</i><sub>C</sub>| omitted map contoured at 3σ for the bound inhibitor. (B) Detailed interactions of <b>12i</b> within the GAK ATP binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Antiviral Activity</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72244" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72244" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have recently shown that GAK is a regulator of HCV entry and assembly and therefore represents a potential target for anti-HCV treatment. Moreover, erlotinib, an approved anticancer drug known to target GAK significantly inhibited binding of HCV core to AP2M1, HCV entry, and assembly.<a onclick="showRef(event, 'ref8 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref11">(8, 11)</a> Nevertheless, while erlotinib binds GAK with a high affinity (<i>K</i><sub>d</sub> = 3.4 nM), it binds EGFR, its primary anticancer target, with a comparable affinity (<i>K</i><sub>d</sub> = 1 nM) and several other kinases (albeit at a lower affinity).<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> The use of erlotinib as a chemical tool to probe the role of GAK in HCV infection is therefore somewhat limited, particularly since EGFR has also been recognized as an essential host factor for HCV infection.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Moreover, the limited selectivity contributes to erlotinib’s side effects, which may reduce its potential as an antiviral agent, particularly in long duration regimens as those required for the treatment of HCV. We therefore hypothesized that since the isothiazolo[4,3-<i>b</i>]pyridines <b>12g</b> and <b>12i</b> are structurally unrelated to erlotinib and lack anti-EGFR activity, they represent attractive chemical tools to study the role of GAK in HCV infection and that their strong potency, promising selectivity profile, and favorable drug-like properties make them a potential novel class of antiviral agents.</div><div class="NLM_p">To further validate GAK as an antiviral target and determine the antiviral effect of these compounds on HCV infection, Huh-7.5 (human hepatoma) cells were infected with cell culture grown J6/JFH(p7-Rluc2A) HCV (HCVcc), a <i>Renilla</i> luciferase-containing reporter virus that replicates and produces high viral titers in Huh-7.5 cells.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Infected cells were treated with various concentrations of <b>12g</b>, <b>12i</b>, or DMSO. Drug-containing medium was replenished every 24 h. Antiviral activity and cellular viability were measured by luciferase and alamarBlue-based assays, respectively, 72 h postinfection. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A, treatment with either <b>12g</b> or <b>12i</b> resulted in a dose-dependent inhibition of viral replication. Half maximal effective concentration (EC<sub>50</sub>) values were 2.55 ± 0.43 μM (<i>p</i> = 0.0002) and 2.81 ± 0.8 μM (<i>p</i> = 0.009) for <b>12g</b> and <b>12i</b>, respectively. The half maximal cytotoxic concentration (CC<sub>50</sub>) values were 23.27 ± 3.4 μM (<i>p</i> = 0.000 012) for <b>12g</b> and 8.92 ± 1.43 μM (<i>p</i> = 0.000 044) for <b>12i</b>.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Selective GAK inhibitors inhibit HCV infection and AP2M1 phosphorylation: Dose–response curves of <b>12g</b> and <b>12i</b> effects on infection of Huh-7.5 cells with cell culture grown HCV (HCVcc) (A), entry of pseudoparticles of HCV (HCVpp) (B), HCV RNA replication 72 h postelectroporation with J6/JFH(p7-Rluc2A) RNA (C), and intra- (D) and extracellular (E) infectivity in naive cells inoculated with cell lysates or supernatants derived from the electroporated cells, respectively. Plotted in black (left <i>y</i>-axes) are relative luciferase values normalized to DMSO treated controls. Corresponding cellular viability, as measured by alamarBlue-based assays, are plotted in gray (right <i>y</i>-axes). Data reflect mean values and sd (error bars). (F) Effect of the inhibitors on AP2M1 phosphorylation by Western analysis in lysates derived from Huh-7.5 cells. A representative membrane blotted with antiphospho-AP2M1 (p-AP2M1) and anti-AP2M1 antibodies is shown. Numbers represent relative p-AP2M1/total AP2M1 protein ratio normalized to DMSO controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we sought to pinpoint the step of the viral life cycle that is disrupted by these GAK inhibitors. To study the effect of the GAK inhibitors on HCV entry, we measured the entry of pseudoparticles of HCV (HCVpp) (lentiviral vectors that incorporate the HCV glycoproteins on the viral envelope<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>) upon a 4 h treatment with various concentrations of <b>12g</b> and <b>12i</b> using luciferase reporter-based assays. Huh-7.5 cells were infected with HCVpp for 1 h on ice followed by a temperature shift to 37 °C, 4 h treatment with various concentrations of the compounds or DMSO, and medium replacement for removal of residual drugs and unbound virus. These compounds inhibited HCVpp entry in a dose-dependent manner, with half maximal effective concentrations (EC<sub>50</sub> values) of 3.6 ± 1.2 μM (<i>p</i> = 0.03) and 2.05 ± 0.36 μM (<i>p</i> = 0.0025), respectively, and a minimal effect on cellular viability (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B).</div><div class="NLM_p">To determine the effect of the compounds on later steps of the HCV life cycle and distinguish between a defect in viral RNA replication, assembly and/or release, Huh-7.5 cells were electroporated with in vitro transcribed luciferase reporter J6/JFH(p7-Rluc2A) HCV RNA<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and treated daily for 72 h with <b>12g</b>, <b>12i</b>, or DMSO. HCV RNA replication was measured by luciferase assays 72 h postelectroporation and intra- and extracellular infectivity was measured by luciferase assays in naive cells inoculated with either clarified cell lysates or supernatants derived from the electroporated cells, respectively. Treatment with <b>12g</b> and <b>12i</b> did not affect HCV RNA replication (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C). Nevertheless, <b>12g</b> and <b>12i</b> demonstrated a dose-dependent effect on intracellular infectivity (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>D), with EC<sub>50</sub> values of 1.64 ± 0.272 μM (<i>p</i> = 0.009) and 2.43 ± 0.72 μM (<i>p</i> = 0.0281), respectively. Moreover, <b>12g</b> and <b>12i</b> significantly inhibited extracellular infectivity (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>E), with EC<sub>50</sub> values of 2.13 ± 0.72 μM (<i>p</i> = 0.04) and 2.47 ± 0.9 μM (<i>p</i> = 0.04), respectively. These data indicate that the two selective GAK inhibitors tested disrupt HCV assembly and infectious virus production without affecting HCV RNA replication.</div><div class="NLM_p">Similar to small interfering RNAs (siRNAs) that target GAK as well as erlotinib,<a onclick="showRef(event, 'ref8 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref11">(8, 11)</a> these compounds thus inhibit two temporally distinct steps in the HCV life cycle: entry and assembly. The EC<sub>50</sub> values for the antiviral effect of these compounds range from ∼1.5–3 μM, depending on the assay used. These EC<sub>50</sub> values are slightly higher or comparable to those of erlotinib (0.5–1.5 μM),<a onclick="showRef(event, 'ref8 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref11">(8, 11)</a> likely reflecting the absence of EGFR inhibition exhibited by erlotinib.</div><div class="NLM_p">Taken together, these results provide a pharmacological validation of the requirement for GAK in the regulation of HCV entry and assembly. Moreover, the selective GAK inhibitors represent candidate compounds to target two steps of the HCV life cycle: viral entry and assembly (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><div class="NLM_p">To validate the mechanism of action of these compounds, we studied their effect on AP2M1 phosphorylation. Huh-7.5 cells were treated with various concentrations of the compounds or DMSO. Since AP2M1 phosphorylation is transient (because of the activity of the phosphatase PP2A),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> to allow capturing of the phosphorylated AP2M1 state, these cells were incubated for 1 h in the presence of the PP2A inhibitor, calyculin A, prior to lysis. The ratio of phosphorylated AP2M1 (p-AP2M1) to total AP2M1 was measured by quantitating Western blot band intensity. p-AP2M1 to AP2M1 ratios were reduced by either <b>12g</b> or <b>12i</b> in a dose-dependent manner and were significantly lower than the ratios measured in the DMSO control (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>F). These results indicate that <b>12g</b> and <b>12i</b> modulate AP2M1 phosphorylation, as predicted for compounds that target GAK.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proposed model for the mechanism by which the selective GAK inhibitors, <b>12g</b> and <b>12i</b>, inhibit HCV infection. By directly inhibiting GAK kinase activity, <b>12g</b> and <b>12i</b> interfere with clathrin/AP-2-mediated endocytosis of HCV (step 1) as well as AP-2 binding to core in HCV assembly (step 5).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25721" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25721" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have previously reported that GAK is an essential host factor for HCV entry and assembly and that erlotinib, an approved anticancer drug with inhibitory activity against GAK, effectively disrupts HCV entry and assembly.<a onclick="showRef(event, 'ref8 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref11">(8, 11)</a> Here, for the first time, we report on a specific optimization campaign toward the identification of novel GAK inhibitors. This endeavor resulted in the identification of novel and highly selective GAK inhibitors, based on an isothiazolo[4,3-<i>b</i>]pyridine scaffold, an unexplored chemical space. Their druglike properties, as well as their selectivity profile, make these compounds an attractive tool to study regulatory mechanisms of intracellular traffic in cell biology and disease conditions, where GAK is known to be implicated. We used these compounds to better understand the role of GAK in HCV infection and further validate this host factor as an antiviral target. We have demonstrated that compounds <b>12g</b> and <b>12i</b> exhibit a potent in vitro anti-HCV activity and that their antiviral effect correlates with a decrease in AP2M1 phosphorylation. Although to the best of our knowledge, KIT has not been reported to have a role in HCV infection, it is bound by these compounds, albeit at a lower affinity (∼30 nM). We therefore cannot exclude the possibility that inhibition of KIT or other cellular kinases, in addition to GAK, contributes to the antiviral effect of these compounds.</div><div class="NLM_p last">Unlike approved direct acting antivirals (DAA), which target HCV RNA replication, the developed compounds inhibit two distinct steps of the HCV life cycle: entry and assembly. By displaying limited off-target binding to other kinases, these more selective GAK inhibitors have the potential to reduce drug toxicity. Moreover, inhibiting a host rather than a viral target is more likely to provide a higher genetic barrier for drug resistance than most DAAs and a comprehensive protection against all HCV genotypes. Hence, selective GAK inhibitors may potentially represent a novel class of antivirals for inclusion in future combination drug regimens for treating HCV. Last, since multiple other viruses hijack clathrin adaptor proteins-mediated pathways for either viral entry or late steps of their viral life cycle,<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28-30)</a> we predict that the requirement for GAK is broadly conserved among RNA viruses and therefore that selective GAK inhibitors may display a broad-spectrum antiviral activity.</div></div><div id="sec9" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84586" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84586" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec9_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General</h3><div class="NLM_p last">For all reactions, analytical grade solvents were used. All moisture-sensitive reactions were carried out in oven-dried glassware (135 °C). <sup>1</sup>H and <sup>13</sup>C NMR spectra: Bruker Avance 300 MHz instrument (<sup>1</sup>H NMR, 300 MHz; <sup>13</sup>C NMR, 75 MHz), 500 MHz instrument (<sup>1</sup>H NMR, 500 MHz; <sup>13</sup>C NMR, 125 MHz), and 600 MHz instrument (<sup>1</sup>H NMR, 600 MHz; <sup>13</sup>C NMR, 150 MHz), using tetramethylsilane as internal standard for <sup>1</sup>H NMR spectra and DMSO-<i>d</i><sub>6</sub> (39.5 ppm) or CDCl<sub>3</sub> (77.2 ppm) and CD<sub>3</sub>OD (49.0 ppm) for <sup>13</sup>C NMR spectra. Abbreviations used are s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, br = broad. Coupling constants are expressed in Hz. Mass spectra are obtained with a Finnigan LCQ Advantage Max (ion trap) mass spectrometer from Thermo Finnigan, San Jose, CA, USA. High resolution mass spectra were acquired on a quadrupole orthogonal acceleration time-of-flight mass spectrometer (Synapt G2 HDMS, Waters, Milford, MA). Samples were infused at 3 μL/min, and spectra were obtained in positive (or negative) ionization mode with a resolution of 15 000 (fwhm) using leucine enkephalin as lock mass. Precoated aluminum sheets (Fluka silica gel/TLC-cards, 254 nm) were used for TLC. Column chromatography (CC) was performed on ICN silica gel 63–200, 60 Å. Purity of final compounds was determined by analytical RP-HPLC analysis on a XBridge column (C-18, 5 μm, 4.6 mm × 150 mm) in combination with a Waters 600 HPLC system and a Waters 2996 photodiode array detector from Waters, Milford, MA, USA. Elution was done using a gradient mixture of H<sub>2</sub>O containing 0.2% (vol) of TFA (A) and acetonitrile (B) (<a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). All compounds were at least 95% pure, with the exception of compound <b>11b</b> (90.32% purity).</div></div><div id="sec9_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 3-Amino-5-bromopyridine-2-carbonitrile (<b>2</b>)</h3><div class="NLM_p last">A solution of iron powder (3.36 g, 60 mmol) in acetic acid (15 mL) was stirred at 0 °C. To this solution was added dropwise a solution of 3-nitro-5-bromopyridine-2-carbonitrile (2.51 g, 11 mmol) in acetic acid (15 mL). The reaction mixture was stirred at room temperature for 2 h. Then, ethyl acetate (300 mL) was added and the mixture was filtered (paper filter). The filter cake was washed with ethyl acetate. The filtrate was evaporated and partitioned between ethyl acetate (500 mL) and water (250 mL). The organic phase was washed with a 1 N NaOH solution (∼200 mL). The combined organic phases were dried and evaporated in vacuo, yielding a mixture of two compounds, i.e., 3-amino-5-bromopyridine-2-carbonitrile (major compound) and 3-amino-5-bromopyridine-2-carboxamide (minor compound). This mixture was used as such in the next reaction.</div></div><div id="sec9_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 3-Amino-5-bromo-2-pyridinecarbothioamide (<b>3</b>)</h3><div class="NLM_p last">To a solution of a mixture of 3-amino-5-bromopyridine-2-carbonitrile and 3-amino-5-bromopyridine-2-carboxamide (crude) in ethanol (25 mL) was added phosphorus pentasulfide (2 equiv; 4.84 g). The mixture was heated overnight at 75 °C. The solvents were evaporated and the crude residue was purified by flash chromatography on silica, using a mixture of cyclohexane/ethyl acetate (in a ratio of 7:1) as mobile phase, yielding the title compound (3.36 g crude). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 6.95 (bs, 2H, NH<sub>2</sub>), 7.27 (d, <i>J</i> = 1.95 Hz, 1H, arom H), 7.89 (d, <i>J</i> = 1.95 Hz, 1H, arom H), 9.37 (bs, 2H, NH<sub>2</sub>) ppm.</div></div><div id="sec9_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 3-Amino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (<b>4</b>)</h3><div class="NLM_p last">To a solution of 3-amino-5-bromo-2-pyridinecarbothioamide in methanol (50 mL) was added dropwise a 30% H<sub>2</sub>O<sub>2</sub> solution in water (3.5 mL) at 0 °C. The reaction mixture was stirred overnight at room temperature and then cooled again to 0 °C. The crystals were filtered off and washed with cold methanol, yielding the title compound (1.6 g, 63%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 8.03 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.08 (bs, 2H, NH<sub>2</sub>), 8.29 (d, <i>J</i> = 2.01 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 120.56 (C<sub>q</sub>), 129.39 (CH), 133.23 (C<sub>q</sub>), 143.69 (C<sub>q</sub>), 153.58 (CH), 173.34 (C<sub>q</sub>) ppm.</div></div><div id="sec9_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Synthesis of 3-Amino-6-arylisothiazolo[4,3-<i>b</i>]pyridines (<b>5a</b>–<b>e</b>)</h3><div id="sec9_1_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> General Procedure</h4><div class="NLM_p">To a solution of 3-amino-6-bromo-isothiazolo[4,3-<i>b</i>]pyridine (166 mg, 0.72 mmol) in a mixture of dioxane (10 mL) and water (1.5 mL) were added an appropriate boronic acid (2 equiv), sodium carbonate (2 equiv, 153 mg), and Pd(dppf)Cl<sub>2</sub> (0.1 equiv, 59 mg) The reaction mixture was stirred overnight at 100 °C. The reaction mixture was cooled to room temperature and the reaction was partitioned between ethyl acetate (60 mL) and brine (30 mL). The aqueous phase was then extracted with ethyl acetate (40 mL). The combined organic phases were dried over MgSO<sub>4</sub> and evaporated in vacuo. The residue was purified by flash chromatography on silica gel, yielding the pure title compounds.</div><div class="NLM_p last">The following compounds were made according to this procedure.</div></div></div><div id="sec9_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 3-Amino-6-phenylisothiazolo[4,3-<i>b</i>]pyridine (<b>5a</b>)</h3><div class="NLM_p last">This compound was obtained using phenylboronic acid and the crude residue was purified by flash chromatography using a mixture of cyclohexane and ethyl acetate (in a ratio of 4:1) as mobile phase, affording the title compound in 77% yield (125 mg, 0.55 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 5.84 (s, 2H, NH<sub>2</sub>), 7.52 (m, 3H, arom H), 7.66 (m, 2H, arom H), 7.95 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.68 (d, <i>J</i> = 1.95 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 124.37 (CH), 127.38 (CH), 128.40 (CH), 129.18 (CH), 134.06 (C<sub>q</sub>), 135.37 (C<sub>q</sub>), 137.25 (C<sub>q</sub>), 143.27 (CH), 153.67 (C<sub>q</sub>), 172.38 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 228.0594, calculated 228.0551.</div></div><div id="sec9_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 3-Amino-6-(4-fluorophenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>5b</b>)</h3><div class="NLM_p last">This compound was obtained using 4-fluorophenylboronic acid and the crude residue was purified by flash chromatography using a mixture of cyclohexane and ethyl acetate (in a ratio of 4:1) as mobile phase, affording the title compound in 76% yield (134 mg, 0.54 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 6.36 (br s, 2H, NH<sub>2</sub>), 7.34 (m, 2H, arom H), 7.38 (d, <i>J</i> = 3.1 Hz, 1H, arom H), 7.71 (m, 2H, arom H), 8.17 (d, <i>J</i> = 1.92 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 115.94 (CH), 116.23 (CH), 124.43 (CH), 129.53 (CH), 129.65 (CH), 133.76 (C<sub>q</sub>), 134.43 (C<sub>q</sub>), 143.19 (CH), 153.65 (C<sub>q</sub>), 160.88 (C<sub>q</sub>), 164.24 (C<sub>q</sub>), 172.49 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 246.0497, calculated 243.0457.</div></div><div id="sec9_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 3-Amino-6-(3,4-dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>5c</b>)</h3><div class="NLM_p last">This compound was obtained using 3,4-dimethoxyphenylboronic acid and the crude residue was purified by flash chromatography using a mixture of methanol and dichloromethane (in a ratio of 1:40) as mobile phase, affording the title compound in 57% yield (117 mg, 0.41 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.81 (s, 3H, OCH<sub>3</sub>), 3.88 (s, 3H, OCH<sub>3</sub>), 7.07 (d, <i>J</i> = 8.89 Hz, 1H, arom H), 7.35 (m, 2H, arom H), 7.84 (br s, 2H, NH<sub>2</sub>), 7.89 (d, <i>J</i> = 1.92 Hz, 1H, arom H), 8.63 (d, <i>J</i> = 1.95 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 55.76 (CH<sub>3</sub>), 55.82 (CH<sub>3</sub>), 111.00 (CH), 112.34 (CH), 119.76 (CH), 123.56 (CH), 129.82 (C<sub>q</sub>), 133.77 (C<sub>q</sub>), 135.31 (C<sub>q</sub>), 143.53 (CH), 149.38 (2 × C<sub>q</sub>), 153.96 (C<sub>q</sub>), 172.20 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 288.0800, calculated 288.0762.</div></div><div id="sec9_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 3-Amino-6-(3-thienyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>5d</b>)</h3><div class="NLM_p last">This compound was obtained using 3-thienylboronic acid and the crude residue was purified by flash chromatography using a mixture of cyclohexane and ethyl acetate (in a ratio of 3:2) as mobile phase, affording the title compound in 69% yield (115 mg, 0.49 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 7.70 (m, 2H, arom H), 7.84 (br s, 2H, NH<sub>2</sub>), 7.98 (d, <i>J</i> = 1.89 Hz, 1H, arom H), 8.17 (q, <i>J</i> = 1.32 Hz, 1H, arom H), 8.74 (d, <i>J</i> = 1.95 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 122.99 (CH), 123.20 (CH), 126.60 (CH), 127.65 (CH), 130.35 (C<sub>q</sub>), 133.71 (C<sub>q</sub>), 138.38 (C<sub>q</sub>), 143.17 (CH), 153.91 (C<sub>q</sub>), 172.22 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 243.0157, calculated 243.0115.</div></div><div id="sec9_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 3-Amino-6-(3-pyridyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>5e</b>)</h3><div class="NLM_p last">This compound was obtained using 3-pyridylboronic acid and the crude residue was purified by flash chromatography using a mixture of methanol and dichloromethane (in a ratio of 1:20) as mobile phase, affording the title compound in 75% yield (123 mg, 0.54 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 7.52 (m, 1H, arom H), 7.95 (br s, 2H, NH<sub>2</sub>), 8.02 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.24 (m, 1H, arom H), 8.66 (m, 2H, arom H), 9.03 (d, <i>J</i> = 1.98 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 124.06 (CH), 125.16 (CH), 132.54 (C<sub>q</sub>), 133.01 (C<sub>q</sub>), 134.92 (CH), 142.85 (CH), 148.26 (CH), 149.37 (CH), 153.46 (C<sub>q</sub>), 173.63 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 229.0537, calculated 229.0503.</div></div><div id="sec9_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 3,6-Dibromoisothiazolo[4,3-<i>b</i>]pyridine (<b>6</b>)</h3><div class="NLM_p last">A solution of 3-amino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (1.1 g, 4.78 mmol) in HBr (50 mL) was stirred for 10 min at room temperature, and then CuBr was added in one portion (1.37 g, 9.56 mmol, 2.0 equiv). The resulting mixture was cooled to 0 °C. A solution of sodium nitrite (0.99 g, 14.34 mmol, 3.0 equiv) in H<sub>2</sub>O (15 mL) was added dropwise to the mixture over a period of 30 min. The reaction mixture was stirred for 2 h at 0 °C and then overnight at room temperature. Then, the mixture was cooled to 0 °C and carefully neutralized with solid potassium carbonate and extracted with ethyl acetate (2 × 50 mL). The combined organic phases were dried over MgSO<sub>4</sub> and evaporated in vacuo. The crude residue was purified by silica gel flash chromatography, using the mixture of cyclohexane and ethyl acetate (in a ratio of 95:5) as mobile phase. The title compound was obtained in 78% yield (1.1 g, 3.75 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 8.70 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.92 (d, <i>J</i> = 2.04 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 121.12 (C<sub>q</sub>), 131.29 (CH), 137.79 (C<sub>q</sub>), 144.54 (C<sub>q</sub>), 153.31 (CH), 154.58 (C<sub>q</sub>) ppm.</div></div><div id="sec9_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 6-Bromo-3-substituted-isothiazolo[4,3-<i>b</i>]pyridines (<b>7a</b>–<b>c</b>)</h3><div id="sec9_1_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> General Procedure</h4><div class="NLM_p">To a solution of 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (150 mg, 0.51 mmol) in ethanol (10 mL) was added an appropriate nitrogen nucleophile (3 equiv). The reaction was stirred overnight at 75 °C. The solvent was evaporated in vacuo and the crude residue was purified by silica gel flash chromatography, the mobile phase being a mixture of cyclohexane and ethyl acetate (in a ratio of 4:1), yielding the pure title compounds.</div><div class="NLM_p last">The following compounds were made according to this procedure.</div></div></div><div id="sec9_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 6-Bromo-3-morpholinoisothiazolo[4,3-<i>b</i>]pyridine (<b>7a</b>)</h3><div class="NLM_p last">This compound was made using morpholine as nucleophile and the crude residue was purified using a mixture of cyclohexane and ethyl acetate (in a ratio of 4:1) as mobile phase, affording the title compound in 92% yield (140 mg, 0.46 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.84 (br s, 4H, 2 × NCH<sub>2</sub>), 3.87 (br s, 4 H, 2 × OCH<sub>2</sub>), 8.15 (s, 1H, arom H), 8.37 (s, 1H, arom H) ppm.</div></div><div id="sec9_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 6-Bromo-6-ethanolaminoisothiazolo[4,3-<i>b</i>]pyridine (<b>7b</b>)</h3><div class="NLM_p last">This compound was made using ethanolamine as nucleophile and the crude residue was purified using a mixture of cyclohexane and ethyl acetate (in a ratio gradually ranging from 1:1 to 1:4) as mobile phase, affording the title compound in 94% yield (131 mg, 0.47 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.39 (q, <i>J</i> = 5.75 Hz, 2H, NCH<sub>2</sub>), 3.68 (q, <i>J</i> = 5.68 Hz, 2H, OCH<sub>2</sub>), 4.91 (t, <i>J</i> = 5.57 Hz, 1H), 8.04 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.29 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.66 (t, <i>J</i> = 5.78 Hz, 1H) ppm.</div></div><div id="sec9_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 6-Bromo-3-(<i>N</i>-methylpiperazino)isothiazolo[4,3-<i>b</i>]pyridine (<b>7c</b>)</h3><div class="NLM_p last">This compound was made using <i>N</i>-methylpiperazine as nucleophile and the crude residue was purified using a mixture of methanol and dichloromethane (in a ratio of 3:100) as mobile phase, affording the title compound in 92% yield (146 mg, 0.46 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 2.24 (s, 3H, NCH<sub>3</sub>), 2.52 (m, 4H, 2 × NCH<sub>2</sub>), 3.88 (t, <i>J</i> = Hz, 4 H, 2 × NCH<sub>2</sub>), 8.12 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.36 (d, <i>J</i> = 1.23 Hz, 1H, arom H) ppm.</div></div><div id="sec9_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Synthesis of 3-Ethanolamino-6-arylisothiazolo[4,3-<i>b</i>]pyridines (<b>8a</b>–<b>e</b>)</h3><div id="sec9_1_15_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> General Procedure</h4><div class="NLM_p">To a solution of 3-ethanolamino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (0.37 mmol, 101 mg) in dioxane (5 mL) and water (1 mL) were added an appropriate boronic acid (2 equiv), sodium carbonate (2 equiv, 78 mg) and Pd(dppf)Cl<sub>2</sub> (0.01 equiv, 30 mg). The reaction was stirred overnight at 75 °C. The reaction mixture was diluted with ethyl acetate (50 mL) and brine (30 mL). The aqueous phase was extracted with ethyl acetate (50 mL). The organic phases were dried and evaporated. The crude residue was purified by silica gel flash chromatography yielding the pure title compounds.</div><div class="NLM_p last">The following compounds were made according to this procedure.</div></div></div><div id="sec9_1_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 3-Ethanolamino-6-phenylisothiazolo[4,3-<i>b</i>]pyridine (<b>8a</b>)</h3><div class="NLM_p last">This compound was prepared using phenylboronic acid and was purified using a mixture of dichloromethane and methanol (in a ratio 20:1), affording the title compound in 99% yield (99 mg, 0.36 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.41 (q, <i>J</i> = 5.80 Hz, 2H, NCH<sub>2</sub>), 3.71 (q, <i>J</i> = 5.61 Hz, 2H, OCH<sub>2</sub>), 4.95 (t, <i>J</i> = 5.52 Hz, 1H), 7.49 (m, 5H, arom H), 7.80 (br d, <i>J</i> = 7.23 Hz, 1H, arom H), 7.91 (s, 1H, arom H), 8.50 (br t, 1H), 8.63 (d, <i>J</i> = 1.59 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.09 (CH<sub>2</sub>), 59.84 (CH<sub>2</sub>), 125.97 (CH), 127.42 (CH), 128.47 (CH), 129.14 (CH), 132.86 (C<sub>q</sub>), 136.81 (C<sub>q</sub>), 137.46 (C<sub>q</sub>), 144.10 (CH), 154.71 (C<sub>q</sub>), 172.35 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 272.0852, calculated 272.0552.</div></div><div id="sec9_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 3-Ethanolamino-6-(3-(4-fluorophenyl))isothiazolo[4,3-<i>b</i>]pyridine (<b>8b</b>)</h3><div class="NLM_p last">This compound was prepared using 4-fluorophenylboronic acid and was purified using a mixture of dichloromethane and methanol (in a ratio 20:1), affording the title compound in 81% yield (86 mg, 0.29 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.30 (2H, NCH<sub>2</sub>, hidden under H<sub>2</sub>O peak), 3.71 (q, <i>J</i> = 5.61 Hz, 2H, OCH<sub>2</sub>), 4.96 (t, <i>J</i> = 5.45 Hz, 1H), 7.35 (t, <i>J</i> = 8.64 Hz, 1H, arom H), 7.88 (m, 3H, arom H), 8.47 (br t, 1H), 8.62 (br s, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.06 (CH<sub>2</sub>), 60.28 (CH<sub>2</sub>), 116.10 (CH), 116.25 (CH), 125.57 (CH), 129.14 (CH), 129.19 (CH), 133.43 (C<sub>q</sub>), 133.74 (C<sub>q</sub>), 135.94 (C<sub>q</sub>), 144.19 (CH), 153.53 (C<sub>q</sub>), 162.26 (C<sub>q</sub>), 163.90 (C<sub>q</sub>), 172.30 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 290.0765, calculated 290.0719.</div></div><div id="sec9_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 3-Ethanolamino-6-(3,4-dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>8c</b>)</h3><div class="NLM_p last">This compound was prepared using 3,4-dimethoxyphenylboronic acid and was purified using a mixture of dichloromethane and methanol (in a ratio 30:1), affording the title compound in 95% yield (116 mg, 0.35 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.42 (q, <i>J</i> = 5.62 Hz, 2H, NCH<sub>2</sub>), 3.70 (q, <i>J</i> = 5.42 Hz, 2H, OCH<sub>2</sub>), 3.81 (s, 3H, OCH<sub>3</sub>), 3.88 (s, 3H, OCH<sub>3</sub>), 4.93 (t, <i>J</i> = 5.67 Hz, 1H), 7.07 (d, <i>J</i> = 8.34 Hz, 1H, arom H), 7.36 (m, 2H, arom H), 7.92 (s, 1H, arom H), 8.40 (t, <i>J</i> = 5.55 Hz, 1 arom H), 8.64 (br s, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.06 (CH<sub>2</sub>), 56.01 (CH<sub>3</sub>), 56.05 (CH<sub>3</sub>), 59.75 (CH<sub>2</sub>), 110.40 (CH), 111.63 (CH), 119.93 (CH), 125.18 (CH), 130.13 (C<sub>q</sub>), 132.53 (C<sub>q</sub>), 136.55 (C<sub>q</sub>), 144.01 (CH), 149.50 (C<sub>q</sub>), 149.49 (C<sub>q</sub>), 154.82 (C<sub>q</sub>), 172.29 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 332.1067, calculated 332.1063.</div></div><div id="sec9_1_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 3-Ethanolamino-6-(3-thienyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>8d</b>)</h3><div class="NLM_p last">This compound was prepared using 3-thienylboronic acid and was purified using a mixture of dichloromethane and methanol (in a ratio 30:1), affording the title compound in 90% yield (92 mg, 0.33 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.40 (q, <i>J</i> = 5.58 Hz, 2H, NCH<sub>2</sub>), 3.70 (q, <i>J</i> = 5.64 Hz, 2H, OCH<sub>2</sub>), 4.93 (t, <i>J</i> = 5.49 Hz, 1H), 7.74 (m, 2H, arom H), 8.01 (br s, 1H, arom H), 8.19 (br s, 1 arom H), 8.42 (br t, 1H), 8.75 (br s, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.08 (CH<sub>2</sub>), 60.18 (CH<sub>2</sub>), 122.30 (CH), 124.47 (CH), 126.15 (CH), 127.13 (CH), 131.51 (C<sub>q</sub>), 134.20 (C<sub>q</sub>), 138.54 (C<sub>q</sub>), 143.86 (CH), 154.73 (C<sub>q</sub>), 172.20 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 278.1415, calculated 278.0416.</div></div><div id="sec9_1_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 3-Ethanolamino-6-(3-pyridyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>8e</b>)</h3><div class="NLM_p last">This compound was prepared using 3-pyridylboronic acid and was purified using a mixture of dichloromethane and methanol (in a ratio gradually ranging from 20:1 to 10:1), affording the title compound in 90% yield (90 mg, 0.33 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.42 (q, <i>J</i> = 5.67 Hz, 2H, NCH<sub>2</sub>), 3.71 (t, <i>J</i> = 5.64 Hz, 2H, OCH<sub>2</sub>), 4.95 (br s, 1H), 7.54 (m, 1H, arom H), 8.05 (br s, 1H, arom H), 8.25 (br d, <i>J</i> = 7.95 Hz, 1H, arom H), 8.52 (br t, 1H), 8.61 (m, 2H, arom H), 9.04 (d, <i>J</i> = 2.13 Hz 1H, arom H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 50.70 (CH<sub>2</sub>), 59.16 (CH<sub>2</sub>), 124.08 (CH), 125.05 (CH), 132.76 (C<sub>q</sub>), 132.97 (C<sub>q</sub>), 133.90 (C<sub>q</sub>), 134.98 (CH), 142.52 (CH), 148.22 (CH), 149.34 (CH), 153.81 (C<sub>q</sub>), 173.35 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 273.0807, calculated 273.0804.</div></div><div id="sec9_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Synthesis of 3-Morpholino-6-arylisothiazolo[4,3-<i>b</i>]pyridines (<b>9a</b>–<b>e</b>)</h3><div id="sec9_1_21_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> General Procedure</h4><div class="NLM_p">To a solution of 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (0.37 mmol, 111 mg) in DME (5 mL) and water (1 mL) were added an appropriate boronic acid (2 equiv), sodium carbonate (2 equiv, 78 mg), and Pd(dppf)Cl<sub>2</sub> (0.01 equiv, 30 mg). The reaction was heated overnight at 75 °C. The reaction mixture was diluted with ethyl acetate (50 mL) and brine (30 mL). The aqueous phase was extracted with ethyl acetate (50 mL). The organic phases were dried and evaporated. The crude residue was purified by silica gel flash chromatography, yielding the pure title compounds.</div><div class="NLM_p last">The following compounds were made according to this procedure.</div></div></div><div id="sec9_1_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 3-Morpholino-6-phenylisothiazolo[4,3-<i>b</i>]pyridine (<b>9a</b>)</h3><div class="NLM_p last">This compound was prepared using phenylboronic acid and was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 78% yield (85 mg. 0.28 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.86 (br s, 4 H, 2 × NCH<sub>2</sub>), 3.90 (br s, 4H, 2 × OCH<sub>2</sub>), 7.50 (m, 3H, arom H), 7.82 (m, 2H, arom H), 8.01 (d, <i>J</i> = 2.1 Hz, 1H, arom H), 8.72 (d, <i>J</i> = 2.1 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.40 (CH<sub>2</sub>), 66.20 (CH<sub>2</sub>), 125.69 (CH), 127.38 (CH), 128.39 (CH), 129.13 (CH), 134.35 (C<sub>q</sub>), 135.81 (C<sub>q</sub>), 137.56 (C<sub>q</sub>), 144.54 (CH), 156.23 (C<sub>q</sub>), 173.32 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 298.1010, calculated 298.1008.</div></div><div id="sec9_1_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 3-Morpholino-6-(4-fluorophenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>9b</b>)</h3><div class="NLM_p last">This compound was prepared using 4-fluorophenylboronic acid and was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 3:1, affording the title compound in 95% yield (110 mg, 0.35 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.85 (br s, 4H, 2 × NCH<sub>2</sub>), 3.89 (br s, 4H, 2 × OCH<sub>2</sub>), 7.34 (t, <i>J</i> = 8.82 Hz, 2H, arom H), 7.87 (m, 2H, arom H), 8.00 (d, J = 2.04 Hz, 1H, arom H), 8.69 (d, <i>J</i> = 2.04 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.38 (CH<sub>2</sub>), 66.18 (CH<sub>2</sub>), 116.08 (CH), 116.23 (CH), 125.52 (CH), 129.03 (CH), 129.09 (CH), 133.65 (C<sub>q</sub>), 134.31 (C<sub>q</sub>), 144.17 (CH), 156.05 (C<sub>q</sub>), 162.22 (C<sub>q</sub>), 163.86 (C<sub>q</sub>), 173.38 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 316.0912, calculated 316.0914.</div></div><div id="sec9_1_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 3-Morpholino-6-(3,4-dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>9c</b>)</h3><div class="NLM_p last">This compound was prepared using 3,4-dimethoxyphenylboronic acid and was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 72% yield (95 mg, 0.26 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.82 (s, 3H, OCH<sub>3</sub>), 3.89 (br s, 11H, 2 × NCH<sub>2</sub>, 2 × OCH<sub>2</sub> and OCH<sub>3</sub>), 7.09 (d, <i>J</i> = 8.04 Hz, 1H, arom H), 7.39 (m, 2H, arom H), 8.02 (d, <i>J</i> = 2.01 Hz, 1 H, arom H), 8.75 (d, <i>J</i> = 1.95 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.37 (CH<sub>2</sub>), 55.97 (CH<sub>3</sub>), 56.00 (CH<sub>3</sub>), 66.18 (CH<sub>2</sub>), 110.33 (CH), 111.64 (CH), 119.84 (CH), 124.80 (CH), 130.23 (C<sub>q</sub>), 134.06 (C<sub>q</sub>), 135.55 (C<sub>q</sub>), 144.51 (CH), 149.47 (C<sub>q</sub>), 149.53 (C<sub>q</sub>), 156.31 (C<sub>q</sub>), 173.21 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 358.1223, calculated 358.1219.</div></div><div id="sec9_1_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 3-Morpholino-6-(3-thienyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>9d</b>)</h3><div class="NLM_p last">This compound was prepared using 3-thienylboronic acid and was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 71% yield (79 mg, 0.26 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.86 (br s, 4 H, 2 × NCH<sub>2</sub>), 3.88 (br s, 4H, 2 × OCH<sub>2</sub>), 7.73 (m, 1H, arom H), 7.77 (m, 1H, arom H), 8.09 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.21 (m, 1H, arom H), 8.84 (d, <i>J</i> = 2.04 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.36 (CH<sub>2</sub>), 66.19 (CH<sub>2</sub>), 122.22 (CH), 124.29 (CH), 126.08 (CH), 127.07 (CH), 130.46 (C<sub>q</sub>), 134.06 (C<sub>q</sub>), 138.54 (C<sub>q</sub>), 143.99 (CH), 156.24 (C<sub>q</sub>), 173.23 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 304.0575, calculated 304.0572.</div></div><div id="sec9_1_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 3-Morpholino-6-(3-pyridyl))isothiazolo[4,3-<i>b</i>]pyridine (<b>9e</b>)</h3><div class="NLM_p last">This compound was prepared using 3-pyridinylboronic acid and was purified using a mixture of methanol/dichloromethane in a ratio of 1:25, affording the title compound in 80% yield (88 mg, 0.29 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.86 (br s, 4H, 2 × NCH<sub>2</sub>), 3.91 (br s, 4H, 2 × OCH<sub>2</sub>), 7.55 (m, 1H, arom H), 8.15 (d, <i>J</i> = 1.95 Hz, 2H, arom H), 8.26 (br d, 1 H, arom H), 8.65 (br d, 1H, arom H), 8.76 (d, <i>J</i> = 1.92 Hz, 1H, arom H), 9.05 (br s, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.37 (CH<sub>2</sub>), 66.15 (CH<sub>2</sub>), 123.82 (CH), 126.22 (CH), 132.59 (C<sub>q</sub>), 133.33 (C<sub>q</sub>), 134.60 (CH), 134.71 (C<sub>q</sub>), 143.39 (CH), 148.36 (CH), 149.50 (CH), 155.70 (C<sub>q</sub>), 173.56 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 299.0959, calculated 299.0961.</div></div><div id="sec9_1_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Synthesis of 3-(<i>N</i>-Me-piperazinyl)-6-arylisothiazolo[4,3-<i>b</i>]pyridines <b>10a</b>–<b>e</b></h3><div id="sec9_1_27_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> General Procedure</h4><div class="NLM_p">To a solution of 3-(<i>N</i>-methylpiperazino)-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (100 mg, 0.32 mmol) in dioxane (6 mL) and water (2.5 mL) were added an appropriate boronic acid (2 equiv), sodium carbonate (2 equiv, 68 mg), and Pd(dppf)Cl<sub>2</sub> (0.01 equiv, 26 mg). The reaction mixture was heated at 95 °C overnight. The reaction mixture was diluted with ethyl acetate (50 mL) and brine (30 mL). The aqueous phase was extracted with ethyl acetate (50 mL). The organic phases were dried and evaporated. The crude residue was purified by silica gel flash chromatography, yielding the pure title compounds</div><div class="NLM_p last">The following compounds were made according to this procedure:</div></div></div><div id="sec9_1_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 3-(<i>N</i>-Me-piperazinyl)-6-phenylisothiazolo[4,3-<i>b</i>]pyridine (<b>10a</b>)</h3><div class="NLM_p last">This compound was prepared using phenylboronic acid and the crude residue was purified using a mixture of methanol and dichloromethane (in a ratio of 3:100), affording the title compound in 91% yield (90 mg, 0.29 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 2.39 (s, 3H, NCH<sub>3</sub>), 2.67 (t, <i>J</i> = 5.01 Hz, 4H, 2 × NCH<sub>2</sub>), 4.00 (t, <i>J</i> = 4.99 Hz, 4H, 2 × NCH<sub>2</sub>), 7.47 (m, 3H, arom H), 7.65 (m, 2H, arom H), 7.89 (d, <i>J</i> = 1.83 Hz, 1H, arom H), 8.63 (d, <i>J</i> = 1.83 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 46.19 (CH<sub>3</sub>), 50.25 (CH<sub>2</sub>), 54.21 (CH<sub>2</sub>), 125.61 (CH), 127.35 (CH), 128.32 (CH), 129.09 (CH), 134.19 (C<sub>q</sub>), 135.64 (C<sub>q</sub>), 137.64 (C<sub>q</sub>), 144.13 (CH), 156.20 (C<sub>q</sub>), 173.21 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 311.1330, calculated 311.1324.</div></div><div id="sec9_1_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 3-(<i>N</i>-Me-piperazinyl)-6-(4-fluorophenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>10b</b>)</h3><div class="NLM_p last">This compound was prepared using 4-fluorophenylboronic acid and the crude residue was purified using a mixture of methanol and dichloromethane (in a ratio of 3:100), affording the title compound in 96% yield (100 mg, 0.30 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 2.40 (s, 3H, NCH<sub>3</sub>), 2.68 (br s, 4H, 2 × NCH<sub>2</sub>), 4.00 (br s, 4H, 2 × NCH<sub>2</sub>), 7.19 (t, <i>J</i> = 8.61 Hz, 1H, arom H), 7.59 (m, 2H, arom H), 7.84 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.57 (d, <i>J</i> = 2.01 Hz, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 46.18 (CH<sub>3</sub>), 50.24 (CH<sub>2</sub>), 54.20 (CH<sub>2</sub>), 116.05 (CH), 116.19 (CH), 125.46 (CH), 129.01 (CH), 129.06 (CH), 133.76 (C<sub>q</sub>), 134.68 (C<sub>q</sub>), 143.78 (CH), 156.03 (C<sub>q</sub>), 162.18 (C<sub>q</sub>), 163.83 (C<sub>q</sub>), 173.26 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 329.1231, calculated 329.1230.</div></div><div id="sec9_1_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 3-(<i>N</i>-Me-piperazinyl)-6-(3,4-dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>10c</b>)</h3><div class="NLM_p last">This compound was prepared using 3,4-dimethoxyphenylboronic acid and the crude residue was purified using a mixture of methanol and dichloromethane (in a ratio of 1:30), affording the title compound in 66% yield (78 mg, 0.21 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 2.27 (s, 3H, NCH<sub>3</sub>), 2.56 (t, <i>J</i> = 4.91 Hz, 4H, 2 × NCH<sub>2</sub>), 3.82 (s, 3H, OCH<sub>3</sub>), 3.88 (s, 3H, OCH<sub>3</sub>), 3.92 (t, <i>J</i> = 4.88 Hz, 4H, 2 × NCH<sub>2</sub>), 7.09 (d, <i>J</i> = 8.43 Hz, 1H, arom H), 7.38 (m, 2H, arom H), 7.99 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.75 (d, <i>J</i> = 2.01 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 46.20 (CH<sub>3</sub>), 50.24 (CH<sub>2</sub>), 54.22 (CH<sub>2</sub>), 55.96 (CH<sub>3</sub>), 55.99 (CH<sub>3</sub>), 110.33 (CH), 111.64 (CH), 119.81 (CH), 124.76 (CH), 130.34 (C<sub>q</sub>), 133.93 (C<sub>q</sub>), 135.39 (C<sub>q</sub>), 144.11 (CH), 149.45 (C<sub>q</sub>), 149.47 (C<sub>q</sub>), 156.30 (C<sub>q</sub>), 173.12 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 371.1532, calculated 371.1536.</div></div><div id="sec9_1_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 3-(<i>N</i>-Me-piperazinyl)-6-(3-thienyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>10d</b>)</h3><div class="NLM_p last">This compound was prepared using 3-thienylboronic acid and the crude residue was purified using a mixture of methanol and dichloromethane (in a ratio of 3:100), affording the title compound in 84% yield (84 mg, 0.26 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 2.26 (s, 3H, NCH<sub>3</sub>), 2.55 (t, <i>J</i> = 4.91 Hz, 4H, 2 × NCH<sub>2</sub>), 3.91 (t, <i>J</i> = 4.88 Hz, 4H, 2 × NCH<sub>2</sub>), 7.72 (m, 2H, arom H), 8.07 (br s, 1H, arom H), 8.07 (br s, 1H, arom H), 8.19 (br s, 1H, arom H), 8.83 (d, <i>J</i> = 1.98 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 46.18 (CH<sub>3</sub>), 50.20 (CH<sub>2</sub>), 54.20 (CH<sub>2</sub>), 122.12 (CH), 124.24 (CH), 126.09 (CH), 127.00 (CH), 130.31 (C<sub>q</sub>), 133.91 (C<sub>q</sub>), 138.62 (C<sub>q</sub>), 143.59 (CH), 156.21 (C<sub>q</sub>), 173.10 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 317.0893, calculated 317.0850.</div></div><div id="sec9_1_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 3-(<i>N</i>-Me-piperazinyl)-6-(3-pyridyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>10e</b>)</h3><div class="NLM_p last">This compound was prepared using 3-pyridylboronic acid and the crude residue was purified using a mixture of methanol and dichloromethane (in a ratio of 1:20), affording the title compound in 83% yield (82 mg, 0.26 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 2.26 (s, 3H, NCH<sub>3</sub>), 2.55 (t, <i>J</i> = 4.85 Hz, 4H, 2 × NCH<sub>2</sub>), 3.92 (t, <i>J</i> = 4.80 Hz, 4H, 2 × NCH<sub>2</sub>), 7.54 (m, 1H, arom H), 8.12 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.24 (d, <i>J</i> = 7.86 Hz, 1H, arom H), 8.65 (d, <i>J</i> = 4.74 Hz, 1H, arom H), 8.75 (d, <i>J</i> = 1.92 Hz, 1H, arom H), 9.04 (d, <i>J</i> = 1.95 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 46.17 (CH<sub>3</sub>), 50.24 (CH<sub>2</sub>), 54.17 (CH<sub>2</sub>), 123.79 (CH), 126.15 (CH), 132.44 (C<sub>q</sub>), 133.41 (C<sub>q</sub>), 134.58 (CH, C<sub>q</sub>), 142.99 (CH), 148.36 (CH), 149.45 (CH), 155.68 (C<sub>q</sub>), 173.43 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 312.1277, calculated 312.1277.</div></div><div id="sec9_1_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 3-Methoxy-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (<b>7d</b>)</h3><div class="NLM_p last">To a solution of 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (700 mg, 2.38 mmol) in absolute methanol (50 mL) was added carefully at 0 °C sodium methoxide (2.5 equiv, 322 mg) in small portions. The resulting reaction mixture was stirred overnight at room temperature and then heated at 55 °C for 8 h. The reaction was cooled to room temperature, neutralized with a 5% HCl solution, and evaporated in vacuo. The residue was divided between ethyl acetate (250 mL) and water (150 mL). The organic phase was dried and evaporated. The crude residue was purified by silica gel flash chromatography, the mobile phase being a mixture of cyclohexane and ethyl acetate (in a ratio gradually ranging from 5:1 to 4:1), yielding the pure title compound in 96% yield (558 mg, 2.27 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 4.35 (s, 3H, OCH<sub>3</sub>), 8.37 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.60 (d, <i>J</i> = 1.98 Hz, 1H, arom H) ppm.</div></div><div id="sec9_1_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 3-Methoxy-6-arylisothiazolo[4,3-<i>b</i>]pyridines (<b>11a</b>–<b>e</b>)</h3><div id="sec9_1_34_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> General Procedure</h4><div class="NLM_p">To a solution of 3-methoxy-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (100 mg, 0.41 mmol) in dioxane (5 mL) and water (1 mL) were added an appropriate boronic acid (2 equiv), sodium carbonate (2 equiv, 87 mg) and Pd(dppf)Cl<sub>2</sub> (0.01 equiv, 33 mg). The reaction mixture was heated at 95 °C overnight. The reaction mixture was diluted with ethyl acetate (50 mL) and brine (30 mL). The aqueous phase was extracted with ethyl acetate (50 mL). The organic phases were dried and evaporated. The crude residue was purified by silica gel flash chromatography, yielding the pure title compounds.</div><div class="NLM_p last">The following compounds were made according to this procedure.</div></div></div><div id="sec9_1_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 3-Methoxy-6-phenylisothiazolo[4,3-<i>b</i>]pyridine (<b>11a</b>)</h3><div class="NLM_p last">This compound was prepared using phenylboronic acid and the crude residue was purified using a mixture of cyclohexane and ethyl acetate (in a ratio of 2:1), affording the title compound in 87% yield (86 mg, 0.35 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 4.37 (s, 3H, OCH3), 7.50 (m, 3H, arom H), 7.68 (m, 2H, arom H), 8.00 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.86 (d, <i>J</i> = 1.98 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 62.37 (CH<sub>3</sub>), 125.55 (CH), 127.15 (CH), 128.28 (CH), 128.84 (CH), 133.08 (C<sub>q</sub>), 136.47 (C<sub>q</sub>), 136.85 (C<sub>q</sub>), 148.37 (CH), 154.54 (C<sub>q</sub>), 182.58 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 243.0584, calculated 243.0547.</div></div><div id="sec9_1_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 3-Methoxy-6-(4-fluorophenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>11b</b>)</h3><div class="NLM_p last">This compound was prepared using 4-fluorophenylboronic acid and the crude residue was purified using a mixture of cyclohexane and ethyl acetate (in a ratio of 2:1), affording the title compound in 76% yield (81 mg, 0.31 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 4.36 (s, 3H, OCH<sub>3</sub>), 7.37 (t, <i>J</i> = 8.79, 3H, arom H), 7.93 (m, 2H, arom H), 8.18 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.93 (d, <i>J</i> = 1.92 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 62.75 (CH<sub>3</sub>), 116.18 (CH), 116.33 (CH), 125.77 (CH), 129.21 (CH), 129.27 (CH), 133.31 (C<sub>q</sub>), 135.83 (C<sub>q</sub>), 148.39 (CH), 154.72 (C<sub>q</sub>), 162.35 (C<sub>q</sub>), 164.00 (C<sub>q</sub>), 183.04 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 261.0503, calculated 261.0453.</div></div><div id="sec9_1_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 3-Methoxy-6-(3,4-dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>11c</b>)</h3><div class="NLM_p last">This compound was prepared using 3,4-dimethoxyphenylboronic acid and the crude residue was purified using a mixture of cyclohexane and ethyl acetate (in a ratio of 1:1), affording the title compound in 74% yield (91 mg, 0.30 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.82 (s, 3H, OCH<sub>3</sub>), 3.89 (s, 3H, OCH<sub>3</sub>), 4.35 (s, 3H, OCH<sub>3</sub>), 7.09 (d, <i>J</i> = 9 Hz, 1H, arom H), 7.42 (m, 2H, arom H), 8.16 (d, <i>J</i> = 1.92 Hz, 1H, arom H), 8.97 (d, <i>J</i> = 1.92 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 56.02 (CH<sub>3</sub>), 56.04 (CH<sub>3</sub>), 62.69 (CH<sub>2</sub>), 110.42 (CH), 111.67 (CH), 120.11 (CH), 125.02 (CH), 129.90 (C<sub>q</sub>), 133.18 (C<sub>q</sub>), 136.58 (C<sub>q</sub>), 148.71 (CH), 149.55 (C<sub>q</sub>), 149.74 (C<sub>q</sub>), 155.01 (C<sub>q</sub>), 182.83 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 303.0796, calculated 303.0797.</div></div><div id="sec9_1_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 3-Methoxy-6-(3-thienyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>11d</b>)</h3><div class="NLM_p last">This compound was prepared using 3-thienyllboronic acid and the crude residue was purified using a mixture of cyclohexane and ethylacetae (in a ratio of 2:1), affording the title compound in 75% yield (74 mg, 0.30 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 4.35 (s, 3H, OCH<sub>3</sub>), 7.74 (m, 1H, arom H), 7.82 (m, 1H, arom H), 8.25 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.29 (m, 1H, arom H), 9.06 (d, <i>J</i> = 1.95 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 62.70 (CH<sub>3</sub>), 122.73 (CH), 124.44 (CH), 126.08 (CH), 127.26 (CH), 131.43 (C<sub>q</sub>), 133.17 (C<sub>q</sub>), 138.19 (C<sub>q</sub>), 148.18 (CH), 154.92 (C<sub>q</sub>), 182.84 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 249.0156, calculated 249.0150.</div></div><div id="sec9_1_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 3-Methoxy-6-(3-pyridyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>11e</b>)</h3><div class="NLM_p last">This compound was prepared using 3-pyridylboronic acid and the crude residue was purified using a mixture of methanol and dichloromethane (in a ratio of 1:30), affording the title compound in 55% yield (54 mg, 0.22 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 4.40 (s, 3H, OCH<sub>3</sub>), 7.57 (m, 1H, arom H), 8.31 (m, 2H, arom H), 8.67 (d, <i>J</i> = 4.23 Hz, 1H, arom H), 8.99 (d, <i>J</i> = 1.71 Hz, 1H, arom H), 9.09 (d, <i>J</i> = 1.89 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 62.87 (CH<sub>2</sub>), 123.86 (CH), 126.60 (CH), 133.05 (C<sub>q</sub>), 133.64 (C<sub>q</sub>), 133.87 (C<sub>q</sub>), 134.77 (CH), 147.75 (CH), 148.49 (CH), 149.78 (CH), 154.39 (C<sub>q</sub>), 183.38 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 244.0542, calculated 244.0500.</div></div><div id="sec9_1_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Synthesis of 3-Morpholino-6-arylisothiazolo[4,3-<i>b</i>]pyridines (<b>12a</b>–<b>j</b>)</h3><div id="sec9_1_40_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> General Procedure</h4><div class="NLM_p">To a solution of 3-morpholino-6-bromo-isothiazolo[4,3-<i>b</i>]pyridine in DME (2 mL) were added an appropriate boronic acid (2 equiv or 1.5 equiv) and potassium carbonate (2 equiv, 1 M solution in H<sub>2</sub>O). Mixture was degassed, and Pd(PPh<sub>3</sub>)<sub>4</sub> (10 mol %) was added. The reaction was heated at 80 °C. After the completion of reaction, solvents were evaporated. The crude residue was purified by silica gel flash chromatography, yielding the pure title compounds.</div><div class="NLM_p last">The following compounds were made according to this procedure.</div></div></div><div id="sec9_1_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 4-(6-(Thiophen-2-yl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)morpholine (<b>12a</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (90 mg, 0.3 mmol) using thiophene-2-boronic acid (0.6 mmol, 76 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.6 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.03 mmol, 34 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 1:1, affording the title compound in 78% yield (71 mg, 0.234 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.96 (m, 8H, 4 × CH<sub>2</sub>), 7.16 (dd, <i>J</i> = 3.66 Hz, <i>J</i> = 5.10 Hz, 1H, arom H), 7.42 (dd, <i>J</i> = 1.08 Hz, <i>J</i> = 5.10 Hz, 1H, arom H), 7.49 (dd, <i>J</i> = 3.66 Hz, <i>J</i> = 1.08 Hz, 1H, arom H), 7.93 (d, <i>J</i> = 2.07 Hz, 1H, arom H), 8.69 (d, <i>J</i> = 2.10 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.35 (CH<sub>2</sub>), 66.17 (CH<sub>2</sub>), 123.49 (CH), 124.99 (CH), 126.56 (CH), 128.39 (CH), 129.36 (C<sub>q</sub>), 134.08 (C<sub>q</sub>), 140.42 (C<sub>q</sub>), 143.12 (CH), 155.91 (C<sub>q</sub>), 173.24 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 304.0577, calculated 304.0572.</div></div><div id="sec9_1_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 4-(6-(Furan-3-yl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)morpholine (<b>12b</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (40 mg, 0.133 mmol) using furan-3-boronic acid (0.2 mmol, 23 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.26 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.013 mmol, 36 mg). The product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 51% yield (19 mg, 0.067 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.96 (m, 8H, 4 × CH<sub>2</sub>), 6.79 (s, 1H, arom H), 7.56 (m, 1H, arom H), 7.81 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 7.90 (s, 1H, arom H), 8.56 (d, <i>J</i> = 2.01 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.37 (CH<sub>2</sub>), 66.19 (CH<sub>2</sub>), 108.55 (CH), 123.25 (C<sub>q</sub>), 123.59 (CH), 127.58 (C<sub>q</sub>), 133.94 (C<sub>q</sub>), 139.58 (CH), 143.40 (CH), 144.26 (CH), 156.19 (C<sub>q</sub>), 173.23 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 288.0803, calculated 288.0801.</div></div><div id="sec9_1_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 4-(6-(2,4-Dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)morpholine (<b>12c</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (90 mg, 0.3 mmol) using 2,4-dimethoxyphenylboronic acid (0.6 mmol, 109 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.6 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.03 mmol, 34 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 73% yield (79 mg, 0.221 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.84 (s, 3H, OCH<sub>3</sub>), 3.88 (s, 3H, OCH<sub>3</sub>), 3.96 (m, 8H, 4 × CH<sub>2</sub>), 6.60–6.64 (m, 2H, arom H), 7.33 (d, <i>J</i> = 8.19 Hz, 1H, arom H), 7.84 (d, <i>J</i> = 1.95 Hz, 1H, arom H), 8.57 (d, <i>J</i> = 1.92 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.14 (CH<sub>2</sub>), 55.17 (CH<sub>3</sub>), 55.22 (CH<sub>3</sub>), 65.92 (CH<sub>2</sub>), 98.76 (CH), 104.77 (CH), 119.29 (C<sub>q</sub>), 127.22 (CH), 131.03 (CH), 133.16 (C<sub>q</sub>), 133.52 (C<sub>q</sub>), 146.66 (CH), 156.22 (C<sub>q</sub>), 157.56 (C<sub>q</sub>), 161.00 (C<sub>q</sub>), 172.70 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 358.1224, calculated 358.1219.</div></div><div id="sec9_1_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 4-(6-(2,5-Dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)morpholine (<b>12d</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (90 mg, 0.3 mmol) using 2,5-dimethoxyphenylboronic acid (0.6 mmol, 109 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.6 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.03 mmol, 34 mg). The product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 69% yield (74 mg, 0.207 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.80 (s, 3H, OCH<sub>3</sub>), 3.84 (s, 3H, OCH<sub>3</sub>), 3.97 (m, 8H, 4 × CH<sub>2</sub>), 6.95–6.97 (m, 3H, arom H), 7.89 (d, <i>J</i> = 1.95 Hz, 1H, arom H), 8.58 (d, <i>J</i> = 1.92 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.13 (CH<sub>2</sub>), 55.51 (CH<sub>3</sub>), 55.89 (CH<sub>3</sub>), 65.92 (CH<sub>2</sub>), 112.35 (CH), 114.15 (CH), 116.19 (CH), 123.33 (C<sub>q</sub>), 127.36 (C<sub>q</sub>), 127.77 (CH), 133.55 (C<sub>q</sub>), 146.23 (CH), 150.71 (C<sub>q</sub>), 153.60 (C<sub>q</sub>), 155.98 (C<sub>q</sub>), 172.84 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 358.1221, calculated 358.1219.</div></div><div id="sec9_1_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> 4-(6-(3,5-Dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)morpholine (<b>12e</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (60 mg, 0.2 mmol) using 3,5-dimethoxyphenylboronic acid (0.4 mmol, 72 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.4 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 70% yield (50 mg, 0.140 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.88 (s, 6H, 2 × OCH<sub>3</sub>), 3.98 (m, 8H, 4 × CH<sub>2</sub>), 6.55 (m, 1H, arom H), 6.80 (d, <i>J</i> = 2.19 Hz, 2H, arom H), 7.92 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.63 (d, <i>J</i> = 2.01 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.09 (CH<sub>2</sub>), 55.16 (CH<sub>3</sub>), 65.88 (CH<sub>2</sub>), 99.97 (CH), 105.32 (CH), 125.46 (CH), 134.18 (C<sub>q</sub>), 135.48 (C<sub>q</sub>), 139.32 (C<sub>q</sub>), 144.14 (CH), 155.83 (C<sub>q</sub>), 160.99 (C<sub>q</sub>), 173.01 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 358.1218, calculated 358.1219.</div></div><div id="sec9_1_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 4-(6-(Benzo[<i>d</i>][1,3]dioxol-5-yl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)morpholine (<b>12f</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (60 mg, 0.2 mmol) using 3,4-methylenedioxyphenylboronic acid (0.4 mmol, 66 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.4 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). Product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 64% yield (43 mg, 0.128 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.96 (m, 8H, 4 × CH<sub>2</sub>), 6.05 (s, 2H, CH<sub>2</sub>), 6.94 (d, <i>J</i> = 7.86 Hz, 1H, arom H), 7.15 (m, 2H, arom H), 7.83 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.58 (d, <i>J</i> = 2.04 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 50.38 (CH<sub>2</sub>), 66.19 (CH<sub>2</sub>), 101.41 (CH<sub>2</sub>), 107.66 (CH), 108.94 (CH), 121.23 (CH), 124.98 (CH), 131.65 (C<sub>q</sub>), 134.11 (C<sub>q</sub>), 135.55 (C<sub>q</sub>), 144.47 (CH), 148.05 (C<sub>q</sub>), 148.45 (C<sub>q</sub>), 156.22 (C<sub>q</sub>), 173.24 (C<sub>q</sub>). HR-MS [M + H]<sup>+</sup> found 342.0905, calculated 342.0906.</div></div><div id="sec9_1_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 4-(6-(3,4,5-Trimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)morpholine (<b>12g</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (76 mg, 0.25 mmol) using 3,4,5-trimethoxyphenylboronic acid (0.5 mmol, 107 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.5 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.025 mmol, 29 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 56% yield (54 mg, 0.141 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.72 (s, 3H, OCH<sub>3</sub>), 3.86 (s, 6H, 2 × OCH<sub>3</sub>), 3.90 (m, 8H, 4 × CH<sub>2</sub>), 7.10 (s, 2H, arom H), 8.08 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.78 (d, <i>J</i> = 1.98 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 50.15 (CH<sub>2</sub>), 56.24 (OCH<sub>3</sub>), 60.21 (OCH<sub>3</sub>), 65.52 (CH<sub>2</sub>), 104.97 (CH), 124.83 (CH), 132.39 (C), 133.59 (C), 135.00 (C), 138.11 (C), 144.37 (CH), 153.53 (C), 156.68 (C), 172.18 (C) ppm. HR-MS [M + H]<sup>+</sup> found 388.1323, calculated 388.1325.</div></div><div id="sec9_1_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> 2-Methoxy-4-(3-morpholinoisothiazolo[4,3-<i>b</i>]pyridin-6-yl)phenyl Acetate (<b>12h</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (60 mg, 0.2 mmol) using 4-acetoxy-3-methoxyphenylboronic acid pinacol ester (0.4 mmol, 116 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.4 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). Product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 58% yield (45 mg, 0.116 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 2.37 (s, 3H, CH<sub>3</sub>(acetyl)), 3.93 (s, 3H, OCH<sub>3</sub>), 3.98 (m, 8H, 4 × CH<sub>2</sub>), 7.16–7.28 (m, 3H, arom H), 7.91 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.53 (d, <i>J</i> = 2.01 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ = 20.67 (CH<sub>3</sub>), 50.38 (CH<sub>2</sub>), 55.97 (CH<sub>3</sub>), 66.18 (CH<sub>2</sub>), 111.51 (CH), 119.80 (CH), 123.44 (CH), 125.71 (CH), 134.40 (C<sub>q</sub>), 135.32 (C<sub>q</sub>), 136.58 (C<sub>q</sub>), 140.06 (C<sub>q</sub>), 144.27 (CH), 151.53 (C<sub>q</sub>), 156.06 (C<sub>q</sub>), 169.00 (CO), 173.36 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 386.1165, calculated 386.1168.</div></div><div id="sec9_1_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 2-Methoxy-4-(3-morpholinoisothiazolo[4,3-<i>b</i>]pyridin-6-yl)aniline (<b>12i</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (60 mg, 0.2 mmol) using 4-amino-3-methoxyphenylboronic acid pinacol ester (0.4 mmol, 99 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.4 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 2:1, affording the title compound in 58% yield (40 mg, 0.116 mmol). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 3.89 (m, 11H, 4 × CH<sub>2</sub> and OCH<sub>3</sub>), 5.09 (br s, 2H, NH<sub>2</sub>), 6.75 (d, <i>J</i> = 8.1 Hz, 1H, arom H), 7.22 (m, 2H, arom H), 7.90 (d, <i>J</i> = 1.86 Hz, 1H, arom H), 8.74 (d, <i>J</i> = 1.86 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 50.12 (CH<sub>2</sub>), 55.63 (CH<sub>3</sub>), 65.54 (CH<sub>2</sub>), 109.43 (CH), 113.91 (CH), 120.21 (CH), 122.21 (CH), 124.09 (CH), 132.86 (C<sub>q</sub>), 135.49 (C<sub>q</sub>), 138.89 (C<sub>q</sub>), 144.44 (CH), 146.85 (C<sub>q</sub>), 156.18 (C<sub>q</sub>), 172.53 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 343.1222, calculated 343.1223.</div></div><div id="sec9_1_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 2-Methoxy-4-(3-morpholinoisothiazolo[4,3-<i>b</i>]pyridin-6-yl)phenol (<b>12j</b>)</h3><div class="NLM_p last">This compound was prepared from 3-morpholino-6-bromoisothiazolo[4,3-<i>b</i>]pyridine (60 mg, 0.2 mmol) using methoxy-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenol (0.3 mmol, 75 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.4 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 3:2, affording the title compound in 44% yield (30 mg, 0.087 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.97 (m, 11H, 4 × CH<sub>2</sub>, CH<sub>3</sub>), 5.93 (s, 1H, OH), 7.04 (d, <i>J</i> = 8.16 Hz, 1H, arom H), 7.14 (d, <i>J</i> = 1.95 Hz, 1H, arom H), 7.19 (dd, <i>J</i> = 2.04 Hz, <i>J</i> = 8.16 Hz, 1H, arom H), 7.87 (d, <i>J</i> = 2.07 Hz, 1H, arom H), 8.64 (d, <i>J</i> = 2.07 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 50.43 (CH<sub>2</sub>), 56.05 (OCH3), 66.23 (CH<sub>2</sub>), 109.80 (CH), 115.14 (CH), 120.61 (CH), 124.74 (CH), 129.75 (C), 132.12 (C), 135.78 (C), 144.62 (CH), 146.37 (C), 147.15 (C), 156.37 (C), 173.25 (C) ppm. HR-MS [M + H]<sup>+</sup> found 344.1067, calculated 344.1063.</div></div><div id="sec9_1_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Synthesis of 3-Substituted-6-(3,4-dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridine (<b>13a</b>–<b>k</b>)</h3><div id="sec9_1_51_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> General Procedure A</h4><div class="NLM_p">To a solution of 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine in ethanol was added an appropriate nitrogen nucleophile (3 equiv). The reaction was stirred at 75 °C. After reaction finished, solvent was evaporated in vacuo and the crude residue was purified by silica gel flash chromatography yielding the 3-substituted-6-bromoisothiazolo[4,3-<i>b</i>]pyridine derivatives. To a solution of 3-substituted-6-bromoisothiazolo[4,3-<i>b</i>]pyridine in DME (2 mL) were added 3,4-dimethoxyphenylboronic acid (1.5 equiv) and potassium carbonate (2 equiv, 1 M solution in H<sub>2</sub>O). The mixture was degassed and Pd(PPh<sub>3</sub>)<sub>4</sub> (10 mol %) was added. The reaction was heated at 80 °C. After the completion of reaction, solvents were evaporated. The crude residue was purified by silica gel flash chromatography, yielding the pure title compounds.</div><div class="NLM_p last">The following compounds were made according to this procedure: <b>13b</b>, <b>13c</b>, <b>13d</b>, <b>13e</b>, <b>13i</b>, <b>13j</b>, <b>13k</b>.</div></div><div id="sec9_1_51_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> General Procedure B</h4><div class="NLM_p">To a solution of 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine in ethanol was added an appropriate nitrogen nucleophile (3 equiv). The reaction was stirred at 70 °C. After reaction finished, solvent was evaporated in vacuo and the crude residue was purified by silica gel flash chromatography, yielding the 3-substituted-6-bromoisothiazolo[4,3-<i>b</i>]pyridine derivatives. To a solution of 3-substituted-6-bromoisothiazolo[4,3-<i>b</i>]pyridine in dioxane (2 mL) were added 3,4-dimethoxyphenylboronic acid (1.5 equiv) and potassium carbonate (3 equiv). The mixture was degassed, and Pd(dppf)Cl<sub>2</sub> (5 mol %) was added. The reaction was heated at 100 °C. After the completion of reaction, solvents were evaporated. The crude residue was purified by silica gel flash chromatography, yielding the pure title compounds.</div><div class="NLM_p last">The following compounds were made according to this procedure: <b>13a</b>, <b>13f</b>, <b>13g</b>, and <b>13h</b>.</div></div></div><div id="sec9_1_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> 6-(3,4-Dimethoxyphenyl)-<i>N</i>,<i>N</i>-dimethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine (<b>13a</b>)</h3><div class="NLM_p last">6-Bromo-<i>N</i>,<i>N</i>-dimethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine was prepared from 3,6-di-bromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and dimethylamine (44 μL, 1.5 mmol) in EtOH (10 mL). The product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 4:1, affording the 6-bromo-<i>N</i>,<i>N</i>-dimethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine in 98 % yield (379 mg, 0.49 mmol). The title compound was prepared from 6-bromo-<i>N</i>,<i>N</i>-dimethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine (52 mg, 0.2 mmol) using 3,4-dimethoxyphenylboronic acid (0.3 mmol, 54.5 mg), K<sub>2</sub>CO<sub>3</sub> (83 mg, 0.6 mmol), and Pd(dppf)Cl<sub>2</sub> (0.01 mmol, 7.4 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 1:1, affording the title compound in 72% yield (46.6 mg, 1.5 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 3.49 (s, 6H, 2 × CH<sub>3</sub>), 3.92 (s, 3H, OCH<sub>3</sub>), 3.94 (s, 3H, OCH<sub>3</sub>), 6.97 (d, <i>J</i> = 8.3 Hz, 1H, arom H), 7.15 (s, 1H, arom H), 7.21 (dd, <i>J</i> = 2.1 Hz, <i>J</i> = 8.3 Hz, 1H arom H), 7.80 (d, <i>J</i> = 2.0 Hz, 1H, arom H), 8.56 (d, <i>J</i> = 2.0 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (300 MHz, CDCl<sub>3</sub>), δ = 43.0 (CH<sub>3</sub>), 56.1 (OCH<sub>3</sub>), 56.1 (OCH<sub>3</sub>), 110.5 (CH), 111.8 (CH), 119.9 (CH), 124.6 (CH), 130.6 (C), 133.5 (C), 135.3 (C), 143.4 (CH), 149.5 (C), 156.2 (C), 173.2 (C) ppm. HR-MS: [M + H]<sup>+</sup> found 316.1113, calculated 316.1114.</div></div><div id="sec9_1_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> 6-(3,4-Dimethoxyphenyl)-<i>N</i>-(2-methoxyethyl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine (<b>13b</b>)</h3><div class="NLM_p last">6-Bromo-<i>N</i>-(2-methoxyethyl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine was prepared from 3,6-di-bromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and 2-methoxyethylamine (0.13 mL, 1.5 mmol) in EtOH (10 mL). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 4:1, affording the 6-bromo-<i>N</i>-(2-methoxyethyl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine in 83% yield (120 mg, 0.416 mmol). The title compound was prepared from 6-bromo-<i>N</i>-(2-methoxyethyl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine (57 mg, 0.2 mmol) using 3,4-dimethoxyphenylboronic acid (0.3 mmol, 54.5 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.40 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). The product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 1:1, affording the title compound in 22% yield (15 mg, 0.043 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.40 (s, 3H, OCH<sub>3</sub>), 3.55 (m, 2H, CH<sub>2</sub>), 3.58 (m, 2H, CH<sub>2</sub>), 3.95 (s, 3H, OCH<sub>3</sub>), 3.97 (s, 3H, OCH<sub>3</sub>), 6.53 (m, 1H, NH), 6.99 (d, <i>J</i> = 8.31 Hz, 1H, arom H), 7.15 (d, 1H, <i>J</i> = 1.98 Hz, arom H), 7.21 (dd, <i>J</i> = 1.98 Hz, <i>J</i> = 8.22 Hz, 1H, arom H), 7.87 (d, <i>J</i> = 1.80 Hz, 1H, arom H), 8.60 (d, <i>J</i> = 1.80 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 47.33 (CH<sub>2</sub>), 55.70 (OCH<sub>3</sub>), 58.67 (OCH<sub>3</sub>), 69.76 (CH2), 110.24 (CH), 111.38 (CH), 119.64 (CH), 124.25 (CH), 130.24 (C), 133.24 (C), 136.37 (C), 144.33 (CH), 149.20 (C), 149.23 (C), 154.40 (C), 171.72 (C) ppm. HR-MS [M + H]<sup>+</sup> found 346.1223, calculated 346.1219.</div></div><div id="sec9_1_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> 2,2′-(6-(3,4-Dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridin-3-ylazanediyl)diethanol (<b>13c</b>)</h3><div class="NLM_p last">2,2′-(6-Bromoisothiazolo[4,3-<i>b</i>]pyridin-3-ylazanediyl)diethanol was prepared from 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and diethanolamine (0.145 mL, 1.5 mmol) in EtOH (10 mL). The crude product was purified using a mixture of DCM/MeOH in a ratio of 95:5, affording 2,2′-(6-bromoisothiazolo[4,3-<i>b</i>]pyridin-3-ylazanediyl)diethanol in 81% yield (129 mg, 0.405 mmol). The title compound was prepared from 2,2′-(6-bromoisothiazolo[4,3-<i>b</i>]pyridin-3-ylazanediyl)diethanol (63.6 mg, 0.2 mmol) using 3,4-dimethoxyphenylboronic acid (0.3 mmol, 54.5 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.4 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). The crude product was purified using a mixture of DCM/MeOH in a ratio of 100:1, affording the title compound in 60% yield (45 mg, 0.12 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.96 (s, 3H, OCH<sub>3</sub>), 3.98 (s, 3H, OCH<sub>3</sub>), 4.05 (m, 8H, 4 × CH<sub>2</sub>), 4.20 (m, 2H, 2 × OH), 7.02 (d, <i>J</i> = 8.31 Hz, 1H, arom H), 7.15 (d, <i>J</i> = 2.04 Hz, 1H, arom H), 7.24 (dd, <i>J</i> = 8.28, <i>J</i> = 2.13 Hz, 1H, arom H), 7.86 (d, <i>J</i> = 2.04 Hz, 1H, arom H), 8.56 (d, <i>J</i> = 2.04 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ = 55.70 (OCH<sub>3</sub>), 57.80 (CH<sub>2</sub>), 59.36 (CH2), 109.98 (CH), 111.38 (CH), 119.52 (CH), 124.54 (CH), 129.60 (C), 132.80 (C), 135.39 (C), 143.58 (CH), 149.21 (C), 149.33 (C), 155.62 (C), 172.05 (C) ppm. HR-MS [M + H]<sup>+</sup> found 376.1322, calculated 376.1252.</div></div><div id="sec9_1_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> 6-(3,4-Dimethoxyphenyl)-3-(pyrrolidin-1-yl)isothiazolo[4,3-<i>b</i>]pyridine (<b>13d</b>)</h3><div class="NLM_p last">6-Bromo-3-(pyrrolidin-1-yl)isothiazolo[4,3-<i>b</i>]pyridine was prepared from 3,6-di-bromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and pyrrolidine (0.04 mL, 1.5 mmol) in EtOH (10 mL). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 4:1, affording the 6-bromo-3-(pyrrolidin-1-yl)isothiazolo[4,3-<i>b</i>]pyridine in 90% yield (128 mg, 0.450 mmol). The title compound was prepared from 6-bromo-3-(pyrrolidin-1-yl)isothiazolo[4,3-<i>b</i>]pyridine (71 mg, 0.25 mmol) using 3,4-dimethoxyphenylboronic acid (0.375 mmol, 68 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.50 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.025 mmol, 29 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 3:2, affording the title compound in 63% yield (54 mg, 0.158 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 2.17 (m, 4H, 2 × CH<sub>2</sub>), 3.87 (m, 4H, 2 × CH<sub>2</sub>), 3.96 (s, 3H, OCH<sub>3</sub>), 3.98 (s, 3H, OCH<sub>3</sub>), 7.00 (d, <i>J</i> = 8.28 Hz, 1H, arom H), 7.18 (d, 1H, <i>J</i> = 2.01 Hz, arom H), 7.26 (dd, <i>J</i> = 8.52, <i>J</i> = 2.28 Hz, 1H, arom H), 7.82 (d, <i>J</i> = 1.95 Hz, 1H, arom H), 8.58 (d, <i>J</i> = 1.95 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ = 25.37 (CH<sub>2</sub>), 51.68 (CH<sub>2</sub>), 55.63 (OCH3), 55.67 (OCH<sub>3</sub>), 110.12 (CH), 111.36 (CH), 119.45 (CH), 123.73 (CH), 130.89 (C), 133.44 (C), 135.20 (C), 142.98 (CH), 149.06 (C), 149.11 (C), 155.50 (C), 169.25 (C) ppm. HR-MS [M + H]<sup>+</sup> found 342.1273, calculated 342.1270.</div></div><div id="sec9_1_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> <i>N</i>-(Cyclopropylmethyl)-6-(3,4-dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine (<b>13e</b>)</h3><div class="NLM_p last">6-Bromo-<i>N</i>-(cyclopropylmethyl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine was prepared from 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and aminomethylcyclopropane (0.086 mL, 3.0 mmol) in EtOH (10 mL). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 9:1, affording the 6-bromo-<i>N</i>-(cyclopropylmethyl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine in 84 % yield (120 mg, 0.422 mmol). The title compound was prepared from 6-bromo-<i>N</i>-(cyclopropylmethyl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine (85 mg, 0.3 mmol) using 3,4-dimethoxyphenylboronic acid (0.45 mmol, 82 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.60 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.03 mmol, 34 mg). The crude residue was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 4:1, affording the title compound in 73% yield (75 mg, 0.210 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 0.67 (q, 2H, CH<sub>2</sub>), 0.69 (q, 2H, CH<sub>2</sub>), 1.25 (m, 1H, CH), 3.24 (q, 2H, NCH<sub>2</sub>), 3.96 (s, 3H, OCH<sub>3</sub>), 3.98 (s, 3H, OCH<sub>3</sub>), 6.36 (t, 1H, NH), 7.02 (d, <i>J</i> = 8.34 Hz, 1H, arom H), 7.17 (d, <i>J</i> = 2.07 Hz, 1H, arom H), 7.25 (dd, <i>J</i> = 8.28, <i>J</i> = 2.1 Hz, 1H, arom H), 7.88 (d, <i>J</i> = 1.92 Hz, 1H, arom H), 8.59 (d, <i>J</i> = 1.95 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ = 3.93 (CH<sub>2</sub>), 10.42 (CH), 53.11 (CH<sub>2</sub>), 56.07 (OCH<sub>3</sub>), 110.58 (CH), 111.43 (CH), 120.00 (CH), 124.71 (CH), 130.58 (C), 132.14 (C), 136.74 (C), 144.39 (CH), 149.59 (C), 149.65 (C), 154.71 (C), 171.77 (C) ppm. HR-MS [M + H]<sup>+</sup> found 342.1269, calculated 342.1270.</div></div><div id="sec9_1_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> 6-(3,4-Dimethoxyphenyl)-3-(piperidin-1-yl)isothiazolo[4,3-<i>b</i>]pyridine (<b>13f</b>)</h3><div class="NLM_p last">6-Bromo-3-(piperidin-1-yl)isothiazolo[4,3-<i>b</i>]pyridine was prepared from 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and piperidine (148 μL, 1.5 mmol) in EtOH (10 mL). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 4:1, affording 6-bromo-<i>N</i>,<i>N</i>-dimethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine in 84 % yield (125 mg, 0.42 mmol). The title compound was prepared from 6-bromo-3-(piperidin-1-yl)isothiazolo[4,3-<i>b</i>]pyridine (60 mg, 0.2 mmol) using 3,4-dimethoxyphenylboronic acid (0.3 mmol, 54.5 mg), K<sub>2</sub>CO<sub>3</sub> (83 mg, 0.6 mmol), and Pd(dppf)Cl<sub>2</sub> (0.01 mmol, 7.4 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 1:1, affording the title compound in 62% yield (43 mg, 0.12 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 1.74 (m, 6H, 3 × CH<sub>2</sub>), 3.88 (br s, 4H, 2 × NCH<sub>2</sub>), 3.91 (s, 3H, OCH<sub>3</sub>), 3.90 (s, 3H, OCH<sub>3</sub>), 6.95 (d, <i>J</i> = 8.31 Hz, 1H, arom H), 7.13 (s, 1H, arom H), 7.19 (dd, <i>J</i> = 1.93 Hz, <i>J</i> = 8.26 Hz, 1H arom H), 7.80 (d, <i>J</i> = 1.98 Hz, 1H, arom H), 8.57 (d, <i>J</i> = 1.98 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (300 MHz, CDCl<sub>3</sub>), δ = 24.0 (CH<sub>2</sub>), 25.3 (CH<sub>2</sub>), 51.8 (CH<sub>2</sub>), 56.0 (CH<sub>3</sub>), 56.0 (CH<sub>3</sub>), 110.4 (CH), 111.7 (CH), 119.8 (CH), 124.6 (CH), 130.5 (C), 133.7 (C), 135.3 (C), 143.5 (CH), 149.5 (C), 156.3 (C), 173.5 (C) ppm. HR-MS [M + H]<sup>+</sup> found 356.1426, calculated 356.1427.</div></div><div id="sec9_1_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> 6-(3,4-Dimethoxyphenyl)-3-thiomorpholinoisothiazolo[4,3-<i>b</i>]pyridine (<b>13g</b>)</h3><div class="NLM_p last">6-Bromo-3-thiomorpholinoisothiazolo[4,3-<i>b</i>]pyridine was prepared from 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and thiomorpholine (151 μL, 1.5 mmol) in EtOH (10 mL). The product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 4:1, affording the 6-bromo-3-thiomorpholinoisothiazolo[4,3-<i>b</i>]pyridine in 85 % yield (134.5 mg, 0.425 mmol). The title compound was prepared from 6-bromo-3-thiomorpholinoisothiazolo[4,3-<i>b</i>]pyridine (63 mg, 0.2 mmol) using 3,4-dimethoxyphenylboronic acid (0.3 mmol, 54.5 mg), K<sub>2</sub>CO<sub>3</sub> (83 mg, 0.6 mmol), and Pd(dppf)Cl<sub>2</sub> (0.01 mmol, 7.4 mg). The crude residue was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 1:1, affording the title compound in 51% yield (38 mg, 0.102 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 2.85 (t, <i>J</i> = 4.97 Hz 4H, 2 × SCH<sub>2</sub>), 3.92 (s, 3H, OCH<sub>3</sub>), 3.94 (s, 3H, OCH<sub>3</sub>), 4.29 (t, <i>J</i> = 4.99 Hz 4H, 2 × NCH<sub>2</sub>), 6.96 (d, <i>J</i> = 8.25 Hz, 1H, arom H), 7.13 (s, 1H, arom H), 7.20 (dd, <i>J</i> = 1.45, <i>J</i> = 8.26 Hz, 1H, arom H), 7.81 (d, <i>J</i> = 1.95 Hz, 1H, arom H), 8.58 (d, <i>J</i> = 1.41 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (300 MHz, CDCl<sub>3</sub>), δ = 26.5 (CH<sub>2</sub>), 53.2 (CH<sub>2</sub>), 56.0 (CH<sub>3</sub>), 56.0 (CH<sub>3</sub>), 110.4 (CH), 111.7 (CH), 119.9 (CH), 124.8 (CH), 130.3 (C), 133.7 (C), 135.6 (C), 144.2 (CH), 149.5 (C), 149.6 (C), 156.6 (C), 172.3 (C) ppm. HR-MS [M + H]<sup>+</sup> found 374.0987, calculated 374.0991.</div></div><div id="sec9_1_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> (2<i>R</i>,6<i>S</i>)-4-(6-(3,4-Dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)-2,6-dimethylmorpholine (<b>13h</b>)</h3><div class="NLM_p last">(2<i>R</i>,6<i>S</i>)-4-(6-Bromoisothiazolo[4,3-<i>b</i>]pyridin-3-yl)-2,6-dimethylmorpholine was prepared from 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and <i>cis</i>-2,6-dimethylmopholine (185 μL, 1.5 mmol) in EtOH (10 mL). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 4:1, affording the (2<i>R</i>,6<i>S</i>)-4-(6-bromoisothiazolo[4,3-<i>b</i>]pyridin-3-yl)-2,6-dimethylmorpholine in 74 % yield (121.4 mg, 0.37 mmol). The title compound was prepared from (2<i>R</i>,6<i>S</i>)-4-(6-bromoisothiazolo[4,3-<i>b</i>]pyridin-3-yl)-2,6-dimethylmorpholine (65 mg, 0.2 mmol) using 3,4-dimethoxyphenylboronic acid (0.3 mmol, 54.5 mg), K<sub>2</sub>CO<sub>3</sub> (83 mg, 0.6 mmol), and Pd(dppf)Cl<sub>2</sub> (0.01 mmol, 7.4 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 1:1, affording the title compound in 40% yield (31 mg, 0.08 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 1.31 (d, <i>J</i> = 6.3 Hz, 6H, 2 × CH<sub>3</sub>), 2.90 (t, <i>J</i> = 11.7 Hz, 2H, 2 × NCH), 3.94 (m, 8H, 2 × OCH<sub>3</sub>, 2 × OCH), 4.50, (d, <i>J</i> = 12.6 Hz, 2 × NCH), 6.98 (d, <i>J</i> = 8.4 Hz, 1H, arom H), 7.15 (d, <i>J</i> = 2.1 Hz, 1H, arom H), 7.23 (dd, <i>J</i> = 2.2 Hz, <i>J</i> = 8.3 Hz, 1H arom H), 7.84 (d, <i>J</i> = 2.1 Hz, 1H, arom H), 8.62 (d, <i>J</i> = 2.1 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (300 MHz, CDCl<sub>3</sub>), δ = 18.9 (CH<sub>3</sub>), 55.5 (CH<sub>2</sub>), 56.1 (CH<sub>3</sub>), 56.1 (CH<sub>3</sub>), 71.3 (CH), 110.5 (CH), 111.8 (CH), 119.9 (CH), 124.8 (CH), 130.4 (C), 134.0 (C), 135.6 (C), 144.4 (CH), 149.6 (C), 156.5 (C), 172.9 (C) ppm. HR-MS [M + H]<sup>+</sup> found 386.1523, calculated 386.1533.</div></div><div id="sec9_1_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> 1-(6-(3,4-Dimethoxyphenyl)isothiazolo[4,3-<i>b</i>]pyridin-3-yl)piperidin-4-ol (<b>13i</b>)</h3><div class="NLM_p last">1-(6-Bromoisothiazolo[4,3-<i>b</i>]pyridin-3-yl)piperidin-4-ol was prepared from 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and 4-hydroxypiperidine (1.5 mmol, 151 mg) in EtOH (10 mL). The product was purified using a mixture of DCM/MeOH in a ratio of 95:5, affording 1-(6-bromoisothiazolo[4,3-<i>b</i>]pyridin-3-yl)piperidin-4-ol in 82 % yield (130 mg, 0.414 mmol). The title compound was prepared from 1-(6-bromoisothiazolo[4,3-<i>b</i>]pyridin-3-yl)piperidin-4-ol (63 mg, 0.2 mmol) using 3,4-dimethoxyphenylboronic acid (0.3 mmol, 54.5 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.4 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). The product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 1:1, affording the title compound in 57% yield (42.8 mg, 0.115 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 1.83 (m, 2H, CH<sub>2</sub>), 2.09 (m, 2H, CH<sub>2</sub>), 3.70 (m, 2H, CH<sub>2</sub>), 3.94 (s, 3H, OCH<sub>3</sub>), 3.96 (s, 3H, OCH<sub>3</sub>), 4.06 (m, 1H, CH), 4.35 (m, 2H, CH<sub>2</sub>), 6.97 (d, <i>J</i> = 8.34 Hz, 1H, arom H), 7.17 (d, <i>J</i> = 2.07 Hz, 1H, arom H), 7.24 (dd, <i>J</i> = 8.28 Hz, <i>J</i> = 2.07 Hz, 1H, arom H), 7.86 (d, <i>J</i> = 2.01 Hz, 1H, arom H), 8.61 (d, <i>J</i> = 2.04 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ = 33.06 (CH<sub>2</sub>), 47.76 (CH<sub>2</sub>), 55.69 (OCH<sub>3</sub>), 66.07 (CH) 110.11 (CH), 111.40 (CH), 119.56 (CH), 124.22 (CH), 129.97 (C), 133.34 (C), 135.22 (C), 143.58 (CH), 149.18 (C), 149.24 (C), 155.84 (C), 172.69 (C) ppm. HR-MS [M + H]<sup>+</sup> found 372.1378, calculated 372.1376.</div></div><div id="sec9_1_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> 6-(3,4-Dimethoxyphenyl)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine (<b>13j</b>)</h3><div class="NLM_p last">6-Bromo-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine was prepared from 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and 4-aminopyrane (0.151 mL, 1.5 mmol) in EtOH (10 mL). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 3:1, affording 6-bromo-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine in 40% yield (64 mg, 0.203 mmol). The title compound was prepared from 6-bromo-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)isothiazolo[4,3-<i>b</i>]pyridin-3-amine (64 mg, 0.203 mmol) using 3,4-dimethoxyphenylboronic acid (0.304 mmol, 55.3 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.406 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.02 mmol, 23 mg). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 1:1, affording the title compound in 16% yield (5.5 mg, 0.015 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 1.72 (m, 2H, 2 × CH<sub>2</sub>), 2.25 (m, 2H, CH<sub>2</sub>), 3.54 (m, 3H, CH, CH<sub>2</sub>) 3.95 (s, 3H, OCH<sub>3</sub>), 3.97 (s, 3H, OCH<sub>3</sub>), 4.12 (m, 2H, CH<sub>2</sub>), 6.26 (d, <i>J</i> = 7.59, 1H, NH) 6.99 (d, <i>J</i> = 8.34 Hz, 1H, arom H), 7.16 (d, <i>J</i> = 2.04 Hz, 1H, arom H), 7.24 (dd, <i>J</i> = 8.28 Hz, <i>J</i> = 2.04 Hz, 1H, arom H), 7.89 (d, <i>J</i> = 1.89 Hz, 1H, arom H), 8.59 (d, <i>J</i> = 1.89 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ = 31.90 (CH<sub>2</sub>), 54.18 (CH), 55.73 (CH<sub>3</sub>), 66.06 (CH<sub>2</sub>), 110.20 (CH), 111.38 (CH), 119.66 (CH), 124.33 (CH), 130.02 (C<sub>q</sub>), 133.15 (C<sub>q</sub>), 136.63 (C<sub>q</sub>), 144.33 (CH), 149.23 (C<sub>q</sub>), 149.32 (C<sub>q</sub>), 154.19 (C<sub>q</sub>), 169.57 (C<sub>q</sub>) ppm. HR-MS [M + H]<sup>+</sup> found 372.1377, calculated 372.1376.</div></div><div id="sec9_1_62" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> 6-(3,4-Dimethoxyphenyl)-<i>N</i>-phenethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine (<b>13k</b>)</h3><div class="NLM_p last">6-Bromo-<i>N</i>-phenethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine was prepared from 3,6-dibromoisothiazolo[4,3-<i>b</i>]pyridine (0.5 mmol, 146 mg) and phenethylamine (0.188 mL, 1.5 mmol) in EtOH (10 mL). The crude product was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 4:1, affording the 6-bromo-<i>N</i>-phenethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine in 77 % yield (130 mg, 0.389 mmol). The title compound was prepared from 6-bromo-<i>N</i>-phenethylisothiazolo[4,3-<i>b</i>]pyridin-3-amine (130 mg, 0.389 mmol) using 3,4-dimethoxyphenylboronic acid (0.583 mmol, 106 mg), 1 M K<sub>2</sub>CO<sub>3</sub> (0.78 mL), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.038 mmol, 45 mg). The crude residue was purified using a mixture of cyclohexane/ethyl acetate in a ratio of 9:1, affording the title compound in 25% yield (39 mg, 0.099 mmol). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 3.11 (t, <i>J</i> = 7.05 Hz, <i>J</i> = 6.99 Hz, 1H, CH<sub>2</sub>), 3.66 (m, 2H, CH<sub>2</sub>), 3.95 (s, 3H, OCH<sub>3</sub>), 3.97 (s, 3H, OCH<sub>3</sub>), 6.36 (m, 1H, NH) 7.01 (d, <i>J</i> = 8.31 Hz, 1H, arom H), 7.15 (d, 1H, <i>J</i> = 1.89 Hz, arom H), 7.21–7.37 (m, 6H, arom H), 7.87 (d, <i>J</i> = 1.86 Hz, 1H, arom H), 8.66 (d, <i>J</i> = 1.83 Hz, 1H, arom H) ppm. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ = 34.60 (CH<sub>2</sub>), 49.04 (CH<sub>2</sub>), 55.70 (OCH<sub>3</sub>), 110.23 (CH), 111.38 (CH), 119.64 (CH), 124.22 (CH), 126.64 (CH), 128.38 (CH), 128.56 (CH), 130.21 (C), 133.15 (C), 136.42 (C), 137.43 (C), 144.30 (CH), 149.20 (C), 149.24 (C), 154.33 (C), 171.39 (C) ppm. HR-MS [M + H]<sup>+</sup> found 392.1422, calculated 392.1427.</div></div><div id="sec9_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Structure Determination</h3><div class="NLM_p last">Recombinant GAK (aa 14–351) was mixed with the GAK-specific NbGAK_4 nanobody (a gift from Serge Muyldermans, Vrije Universiteit Brussel), and the complex was purified as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The GAK:NbGAK_4 complex was preincubated with 1 mM compound <b>12i</b> and crystallized using the sitting drop vapor diffusion method at 20 °C and the reservoir solution containing 9% broad-molecular-weight PEG smears (mixture of PEGs with their molecular weight ranging from 400 to 10 000 Da), 0.1 M MES, pH 6.2, and 0.15 M CaCl<sub>2</sub>. Viable crystals were cryoprotected with mother liquor supplemented with 22% ethylene glycol and flash-frozen in liquid nitrogen. Diffraction data collected at Diamond Light Source, beamline I04-1, were processed and scaled with XDS<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and SCALA from the CCP4 suite,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> respectively. Initial structure solution was solved by molecular replacement using PHASER<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and the coordinate of the GAK:NbGAK_4 complex.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Manual model building in COOT<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> alternated with refinement in REFMAC<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> was performed, and the geometric correctness of the complete model was verified with MolProbity.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Data collection and refinement statistics are summarized in Table 2 of the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec10" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Plasmids</h3><div class="NLM_p last">pFL-J6/JFH(p7-Rluc2A) was a gift from Dr. C. M. Rice.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Plasmids used in the HCVpp entry assays (pNL4-3.Luc.R-E, pcDM8, and pcDM8-E1E2) were a gift from Dr. Shoshana Levy.</div></div><div id="sec11" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Cells</h3><div class="NLM_p last">Huh-7.5 cells and 293T cells were grown in Dulbecco’s modified Eagle medium (DMEM; Mediatech) supplemented with 10% fetal bovine serum (Omega Scientific), nonessential amino acids (Gibco), 1% <span class="smallcaps smallerCapital">l</span>-glutamine (Gibco), and 1% penicillin–streptomycin (Gibco) and maintained in 5% CO<sub>2</sub> at 37 °C.</div></div><div id="sec12" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> In Vitro Transcription of Viral RNA, Transfection, and HCVcc Generation</h3><div class="NLM_p last">HCV RNA was generated and delivered into Huh-7.5 cells, as previously described.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Briefly, RNA was reverse transcribed from XbaI-linearized J6/JFH(p7-Rluc2A) template using the T7 MEGAscript kit according to the manufacturer’s instructions (Ambion). Viral RNA was purified using the RNeasy kit (Qiagen). 6 ×10<sup>6</sup> Huh7.5 cells were washed three times with ice-cold RNase-free PBS (BioWhittaker) and electroporated (0.82 kV, five 99 μs pulses) with 2 μg of viral RNA in a 2 mm gap cuvette (BTX) using a BTX-830 electroporator. After a 15 min recovery at room temperature, cells were resuspended in prewarmed growth medium and plated into 96 or 6-well culture plates. For HCVcc generation, cells were diluted in 30 mL of prewarmed growth medium and plated. Viral supernatants were collected daily from up to five passages of the electroporated cells, filtered through a 0.22 μm cellulose nitrate filter, and kept at −80 °C. Viral titers were determined by limiting dilution and immunohistochemical staining using an antibody directed to core. 50% tissue culture infectious dose (TCID<sub>50</sub>) was calculated, as described.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Results are expressed as TCID<sub>50</sub>/mL.</div></div><div id="sec13" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> HCVcc Infection</h3><div class="NLM_p last">6 ×10<sup>3</sup> Huh-7.5 cells seeded in 96-well plates were infected in triplicate with HCVcc J6/JFH(p7-Rluc2A) at MOI (multiplicity of infection) of 0.1 in the presence of serial dilutions of the compounds. Culture medium was replaced daily with medium containing serial dilutions of the inhibitors. HCVcc infection was measured by standard luciferase assays at 72 h postinfection, using a <i>Renilla</i> luciferase substrate and a Tecan luminometer (Tecan) according to the manufacturers’ protocols. Alternatively, 4 h after infection, cells were washed and medium was replaced, followed by standard luciferase assays at 24 h after infection.</div></div><div id="sec14" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> HCV RNA Replication by Luciferase Assays</h3><div class="NLM_p last">HCV RNA replication was measured at 72 h postelectroporation, as described.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Electroporated cells plated in quadruplicates in 96-well plates were washed twice with PBS and lysed with 30 μL of <i>Renilla</i> lysis buffer (Promega). Following 15 min shaking at room temperature, luciferase activity was quantified by standard luciferase assays.</div></div><div id="sec15" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Extracellular and Intracellular Infectivity</h3><div class="NLM_p last">Huh-7.5 cells electroporated with J6/JFH(p7-Rluc2A) RNA and plated in six-well dishes were treated every 24 h with GAK inhibitors for a total of 72 h. For measurements of extracellular infectivity, supernatants were harvested, filtered using a 0.22 μm pore size filter and used to infect naïve Huh-7.5 cells in triplicate. For intracellular infectivity measurements, electroporated cells were trypsinized, collected by centrifugation, resuspended in 500 μL medium, lysed by four freeze–thaw cycles, and pelleted at 3650<i>g</i>. Clarified supernatants were diluted in complete medium and used to inoculate naive Huh-7.5 cells in triplicate. At 72 h postinfection cells were lysed and luciferase activity quantified as above.</div></div><div id="sec16" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> HCVpp Production and Entry Assays</h3><div class="NLM_p last">HCVpp (H77c strain, genotype 1a) was generated as described previously.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Briefly, 293T cells were transfected with a 1:1 ratio of plasmids encoding HIV provirus expressing luciferase and HCV E1E2 envelope glycoproteins. Supernatants were harvested 48 h posttransfection and filtered. Huh-7.5 cells were infected with HCVpp and 8 mg/mL Polybrene (Sigma) for 4 h. Cell lysates were collected at 48 h after HCVpp infection, and firefly luciferase (Promega) activity was measured using a Tecan luminometer (Tecan).</div></div><div id="sec17" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> Viability Assay</h3><div class="NLM_p last">Following treatment with GAK inhibitors, Huh-7.5 cells either infected with HCVcc and HCVpp or electroporated with HCV RNA were incubated for 2–4 h with medium supplemented with 10% alamarBlue reagent (TREK Diagnostic Systems) at 37 °C. Fluorescence at 560 nm was measured via FLEXstation II 384 (Molecular Devices, Inc.) as readout of cellular metabolic activity.</div></div><div id="sec18" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> Effect of the Compounds on AP2M1 Phosphorylation</h3><div class="NLM_p last">Huh-7.5 cells were treated with various concentrations of the compounds or DMSO in serum free medium. To allow capturing of the phosphorylated AP2M1 state, 100 nM PP2A inhibitor calyculin A was added to all wells 30 min after drug administration. One hour later cells were lysed and samples subjected to SDS–PAGE and blotting with antibodies targeting AP2M1 (Santa Cruz Biotechnology) and p-AP2M1 (Cell Signaling).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i99"><a href="/doi/suppl/10.1021/jm501759m">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">NMR spectra of representative compounds, HPLC purity data of final compounds, and data collection and refinement statistics for the X-ray crystallography. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm501759m/suppl_file/jm501759m_si_001.pdf">jm501759m_si_001.pdf (524.89 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates have been deposited in the PDB with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y8D">4Y8D</a>.</p><div class="testing" data-doi="10.1021/jm501759m" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13414" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13414" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Piet Herdewijn</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Minderbroedersstraat 10, 3000 Leuven, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Interface
Valorisation Platform, KU Leuven, Kapucijnenvoer 33, 3000 Leuven, Belgium</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4a1a232f3e64022f382e2f3d2320240a382f2d2b64213f262f3f3c2f2464282f"><span class="__cf_email__" data-cfemail="a1f1c8c4d58fe9c4d3c5c4d6c8cbcfe1d3c4c6c08fcad4cdc4d4d7c4cf8fc3c4">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sona Kovackova</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Minderbroedersstraat 10, 3000 Leuven, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Interface
Valorisation Platform, KU Leuven, Kapucijnenvoer 33, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei Chang</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Minderbroedersstraat 10, 3000 Leuven, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Interface
Valorisation Platform, KU Leuven, Kapucijnenvoer 33, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Bekerman</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Infectious Diseases and Geographic Medicine,
and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gregory Neveu</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Infectious Diseases and Geographic Medicine,
and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rina Barouch-Bentov</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Infectious Diseases and Geographic Medicine,
and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Apirat Chaikuad</span> - <span class="hlFld-Affiliation affiliation">Target Discovery
Institute (TDI), and Structural Genomics Consortium (SGC), University
of Oxford, Old Road Campus Research
Building, Oxford OX3 7DQ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christina Heroven</span> - <span class="hlFld-Affiliation affiliation">Target Discovery
Institute (TDI), and Structural Genomics Consortium (SGC), University
of Oxford, Old Road Campus Research
Building, Oxford OX3 7DQ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michal Šála</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Minderbroedersstraat 10, 3000 Leuven, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Interface
Valorisation Platform, KU Leuven, Kapucijnenvoer 33, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven De Jonghe</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Medicinal Chemistry, Rega Institute for Medical Research, KU Leuven, Minderbroedersstraat 10, 3000 Leuven, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Interface
Valorisation Platform, KU Leuven, Kapucijnenvoer 33, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Knapp</span> - <span class="hlFld-Affiliation affiliation">Target Discovery
Institute (TDI), and Structural Genomics Consortium (SGC), University
of Oxford, Old Road Campus Research
Building, Oxford OX3 7DQ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shirit Einav</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Infectious Diseases and Geographic Medicine,
and Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, California 94305, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.K. and L.C. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d495e5979-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i102">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70418" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70418" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported by a grant from the IWT (Agentschap voor Innovatie door Wetenschap en Technologie-Vlaanderen, Grant IWT-SBO 100014, to P.H). Mass spectrometry was made possible by the support of the Hercules Foundation of the Flemish Government (Grant 20100225-7). This work was supported in part by grants from Stanford Clinical and Translational Science Award (CTSA) to Spectrum (Grant UL1 TR001085), Stanford SPARK program, Stanford Bio-X, Stanford Translational Research and Applied Medicine Program, Grant 2013100 from the Doris Duke Charitable Foundation, and Grant RSG-14-11 0-0 1-MPC from the American Cancer Society to S.E. G.N. is supported by the Child Health Research Institute, Luclie Packard Foundation for Children’s Health, as well as the Stanford CSTA (Grant UL1 TR000093). We thank Dr. Charles M. Rice for kindly providing us the pFL-J6/JFH, pFL-J6/JFH(p7-Rluc2A) plasmid, Apath LLC (St. Louis, MO) for providing Huh-7.5 cells, and Dr. Shoshana Levy for providing the plasmids used in the HCVpp entry assays (pNL4-3.Luc.R-E, pcDM8, and pcDM8-E1E2). S.K. is supported by the SGC, a registered charity (No. 1097737) that receives funds from AbbVie, Bayer, Boehringer Ingelheim, the Canada Foundation for Innovation, the Canadian Institutes for Health Research, Genome Canada, GlaxoSmithKline, Janssen, Lilly Canada, the Novartis Research Foundation, the Ontario Ministry of Economic Development and Innovation, Pfizer, Takeda, and the Wellcome Trust [Grant 092809/Z/10/Z]. A.C. is supported by the European Union FP7 Grant 278568 “PRIMES” (Protein Interaction Machines in Oncogenic EGF Receptor Signaling). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i103" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i103"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i104" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i104"> Abbreviations Used</h2><tr><td class="NLM_term">AAK1</td><td class="NLM_def"><p class="first last">adaptor-associated kinase</p></td></tr><tr><td class="NLM_term">AcOH</td><td class="NLM_def"><p class="first last">acetic acid</p></td></tr><tr><td class="NLM_term">AP-2</td><td class="NLM_def"><p class="first last">adaptor protein complex 2</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">BIKE</td><td class="NLM_def"><p class="first last">BMP-2 inducible kinase</p></td></tr><tr><td class="NLM_term">CCV</td><td class="NLM_def"><p class="first last">clathrin-coated vesicle</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">GAK</td><td class="NLM_def"><p class="first last">cyclin G associated kinase</p></td></tr><tr><td class="NLM_term">HCV</td><td class="NLM_def"><p class="first last">hepatitis C virus</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>d</sub></td><td class="NLM_def"><p class="first last">binding affinity constant</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">NAK</td><td class="NLM_def"><p class="first last">numb-associated kinase</p></td></tr><tr><td class="NLM_term">ND</td><td class="NLM_def"><p class="first last">not determined</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">STK16/MPSK1</td><td class="NLM_def"><p class="first last">serine/threonine kinase 16/myristoylated and palmitoylated serine/threonine kinase 1</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i105">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 37 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kanaoka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimura, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okazaki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nojima, H.</span><span> </span><span class="NLM_article-title">GAK: a cyclin G associated kinase contains a tensin/auxilin-like domain</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">402</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=402&publication_year=1997&pages=73-80&author=Y.+Kanaokaauthor=S.+H.+Kimuraauthor=I.+Okazakiauthor=M.+Ikedaauthor=H.+Nojima&title=GAK%3A+a+cyclin+G+associated+kinase+contains+a+tensin%2Fauxilin-like+domain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKanaoka%26aufirst%3DY.%26aulast%3DKimura%26aufirst%3DS.%2BH.%26aulast%3DOkazaki%26aufirst%3DI.%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DNojima%26aufirst%3DH.%26atitle%3DGAK%253A%2520a%2520cyclin%2520G%2520associated%2520kinase%2520contains%2520a%2520tensin%252Fauxilin-like%2520domain%26jtitle%3DFEBS%2520Lett.%26date%3D1997%26volume%3D402%26spage%3D73%26epage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">–</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lhKHVp2TnXPJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Lee, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, L. E.</span><span> </span><span class="NLM_article-title">Depletion of GAK/auxilin 2 inhibits receptor-mediated endocytosis and recruitment of both clathrin and clathrin adaptors</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">4311</span><span class="NLM_x">–</span> <span class="NLM_lpage">4321</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1242%2Fjcs.02548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=16155256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFCgt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2005&pages=4311-4321&author=D.+W.+Leeauthor=X.+Zhaoauthor=F.+Zhangauthor=E.+Eisenbergauthor=L.+E.+Greene&title=Depletion+of+GAK%2Fauxilin+2+inhibits+receptor-mediated+endocytosis+and+recruitment+of+both+clathrin+and+clathrin+adaptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of GAK/auxilin 2 inhibits receptor-mediated endocytosis and recruitment of both clathrin and clathrin adaptors</span></div><div class="casAuthors">Lee, Dong-won; Zhao, Xiaohong; Zhang, Fang; Eisenberg, Evan; Greene, Lois E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4311-4321</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Cyclin G-assocd. kinase (GAK/auxilin 2), the ubiquitous form of the neuronal-specific protein auxilin 1, is an essential cofactor for the Hsc70-dependent uncoating of clathrin-coated vesicles.  We have now investigated the effect of knocking down GAK in HeLa cells by vector-based small hairpin RNA.  Functionally, depletion of GAK caused a marked decrease in internalization of both transferrin and epidermal growth factor and altered mannose 6-phosphate receptor trafficking, but had little effect on the recycling of transferrin receptor back to the plasma membrane.  Structurally, depletion of GAK caused a marked redn. in perinuclear clathrin assocd. with the trans-Golgi network and in the no. of clathrin-coated pits on the plasma membrane, and reduced clathrin exchange on the few clathrin-coated pits that remained.  Surprisingly, while clathrin depletion does not prevent adaptors from assembling on the membrane, depletion of GAK caused a dramatic redn. in AP2 and epsin on the plasma membrane and AP1 and GGA at the trans-Golgi network.  A similar effect was caused by expression of a dominant neg. Hsp70 mutant.  These results suggest that GAK, in conjunction with Hsc70, not only uncoats clathrin-coated vesicles and induces clathrin exchange on clathrin-coated pits, but also mediates binding of clathrin and adaptors to the plasma membrane and the trans-Golgi network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8IvRfVSaZT7Vg90H21EOLACvtfcHk0li7FNXJ_BwbKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFCgt7fK&md5=0172190491cf9b908e705e434b763d60</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1242%2Fjcs.02548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.02548%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%2BW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DEisenberg%26aufirst%3DE.%26aulast%3DGreene%26aufirst%3DL.%2BE.%26atitle%3DDepletion%2520of%2520GAK%252Fauxilin%25202%2520inhibits%2520receptor-mediated%2520endocytosis%2520and%2520recruitment%2520of%2520both%2520clathrin%2520and%2520clathrin%2520adaptors%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2005%26volume%3D118%26spage%3D4311%26epage%3D4321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Korolchuk, V. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banting, G.</span><span> </span><span class="NLM_article-title">CK2 and GAK/auxilin2 are major protein kinases in clathrin-coated vesicles</span> <span class="citation_source-journal">Traffic</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">428</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=428-439&author=V.+I.+Korolchukauthor=G.+Banting&title=CK2+and+GAK%2Fauxilin2+are+major+protein+kinases+in+clathrin-coated+vesicles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKorolchuk%26aufirst%3DV.%2BI.%26aulast%3DBanting%26aufirst%3DG.%26atitle%3DCK2%2520and%2520GAK%252Fauxilin2%2520are%2520major%2520protein%2520kinases%2520in%2520clathrin-coated%2520vesicles%26jtitle%3DTraffic%26date%3D2002%26volume%3D3%26spage%3D428%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Zhang, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engqvist-Goldstein, Å.E.Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carreno, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smythe, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drubin, D. G.</span><span> </span><span class="NLM_article-title">Multiple roles for cyclin G-associated kinase in clathrin-mediated sorting events</span> <span class="citation_source-journal">Traffic</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1103</span><span class="NLM_x">–</span> <span class="NLM_lpage">1113</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=1103-1113&author=C.+X.+Zhangauthor=%C3%85.E.Y.+Engqvist-Goldsteinauthor=S.+Carrenoauthor=D.+J.+Owenauthor=E.+Smytheauthor=D.+G.+Drubin&title=Multiple+roles+for+cyclin+G-associated+kinase+in+clathrin-mediated+sorting+events"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%2BX.%26aulast%3DEngqvist-Goldstein%26aufirst%3D%25C3%2585.E.Y.%26aulast%3DCarreno%26aufirst%3DS.%26aulast%3DOwen%26aufirst%3DD.%2BJ.%26aulast%3DSmythe%26aufirst%3DE.%26aulast%3DDrubin%26aufirst%3DD.%2BG.%26atitle%3DMultiple%2520roles%2520for%2520cyclin%2520G-associated%2520kinase%2520in%2520clathrin-mediated%2520sorting%2520events%26jtitle%3DTraffic%26date%3D2005%26volume%3D6%26spage%3D1103%26epage%3D1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greener, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Hasani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cushman, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, L. E.</span><span> </span><span class="NLM_article-title">Expression of auxilin or AP180 inhibits endocytosis by mislocalizing clathrin: evidence for formation of nascent pits containing AP1 or AP2 but not clathrin</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">365</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=11148137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD3MXht1Kgur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2001&pages=353-365&author=X.+Zhaoauthor=T.+Greenerauthor=H.+Al-Hasaniauthor=S.+W.+Cushmanauthor=E.+Eisenbergauthor=L.+E.+Greene&title=Expression+of+auxilin+or+AP180+inhibits+endocytosis+by+mislocalizing+clathrin%3A+evidence+for+formation+of+nascent+pits+containing+AP1+or+AP2+but+not+clathrin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of auxilin or AP180 inhibits endocytosis by mislocalizing clathrin: evidence for formation of nascent pits containing AP1 or AP2 but not clathrin</span></div><div class="casAuthors">Zhao, Xiaohong; Greener, Tsvika; Al-Hasani, Hadi; Cushman, Samuel W.; Eisenberg, Evan; Greene, Lois E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">353-365</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Although uncoating of clathrin-coated vesicles is a key event in clathrin-mediated endocytosis it is unclear what prevents uncoating of clathrin-coated pits before they pinch off to become clathrin-coated vesicles.  We have shown that the J-domain proteins auxilin and GAK are required for uncoating by Hsc70 in vitro.  In the present study, we expressed auxilin in cultured cells to det. if this would block endocytosis by causing premature uncoating of clathrin-coated pits.  We found that expression of auxilin indeed inhibited endocytosis.  However, expression of auxilin with its J-domain mutated so that it no longer interacted with Hsc70 also inhibited endocytosis as did expression of the clathrin-assembly protein, AP180, or its clathrin-binding domain.  Accompanying this inhibition, we obsd. a marked decrease in clathrin assocd. with the plasma membrane and the trans-Golgi network, which provided us with an opportunity to det. whether the absence of clathrin from clathrin-coated pits affected the distribution of the clathrin assembly proteins AP1 and AP2.  Surprisingly we found almost no change in the assocn. of AP2 and AP1 with the plasma membrane and the trans-Golgi network, resp.  This was particularly obvious when auxilin or GAK was expressed with functional J-domains since, in these cases, almost all of the clathrin was sequestered in granules that also contained Hsc70 and auxilin or GAK.  We conclude that expression of clathrin-binding proteins inhibits clathrin-mediated endocytosis by sequestering clathrin so that it is no longer available to bind to nascent pits but that assembly proteins bind to these pits independently of clathrin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcyfXFVf8SQ7Vg90H21EOLACvtfcHk0li7FNXJ_BwbKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXht1Kgur0%253D&md5=1b52211f83e64142ec4a4d470f3abdb0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DGreener%26aufirst%3DT.%26aulast%3DAl-Hasani%26aufirst%3DH.%26aulast%3DCushman%26aufirst%3DS.%2BW.%26aulast%3DEisenberg%26aufirst%3DE.%26aulast%3DGreene%26aufirst%3DL.%2BE.%26atitle%3DExpression%2520of%2520auxilin%2520or%2520AP180%2520inhibits%2520endocytosis%2520by%2520mislocalizing%2520clathrin%253A%2520evidence%2520for%2520formation%2520of%2520nascent%2520pits%2520containing%2520AP1%2520or%2520AP2%2520but%2520not%2520clathrin%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2001%26volume%3D114%26spage%3D353%26epage%3D365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gjoerup, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, T. M.</span><span> </span><span class="NLM_article-title">The serine threonine kinase cyclin G-associated kinase regulates epidermal growth factor receptor signaling</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">10296</span><span class="NLM_x">–</span> <span class="NLM_lpage">10301</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=10296-10301&author=L.+Zhangauthor=O.+Gjoerupauthor=T.+M.+Roberts&title=The+serine+threonine+kinase+cyclin+G-associated+kinase+regulates+epidermal+growth+factor+receptor+signaling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DGjoerup%26aufirst%3DO.%26aulast%3DRoberts%26aufirst%3DT.%2BM.%26atitle%3DThe%2520serine%2520threonine%2520kinase%2520cyclin%2520G-associated%2520kinase%2520regulates%2520epidermal%2520growth%2520factor%2520receptor%2520signaling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D10296%26epage%3D10301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Neveu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barouch-Bentov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziv-Av, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerber, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einav, S.</span><span> </span><span class="NLM_article-title">Identification and targeting of an interaction between a tyrosine motif within hepatitis C virus core protein and AP2M1 essential for viral assembly</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002845</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=e1002845&issue=8&author=G.+Neveuauthor=R.+Barouch-Bentovauthor=A.+Ziv-Avauthor=D.+Gerberauthor=Y.+Jacobauthor=S.+Einav&title=Identification+and+targeting+of+an+interaction+between+a+tyrosine+motif+within+hepatitis+C+virus+core+protein+and+AP2M1+essential+for+viral+assembly"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DZiv-Av%26aufirst%3DA.%26aulast%3DGerber%26aufirst%3DD.%26aulast%3DJacob%26aufirst%3DY.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DIdentification%2520and%2520targeting%2520of%2520an%2520interaction%2520between%2520a%2520tyrosine%2520motif%2520within%2520hepatitis%2520C%2520virus%2520core%2520protein%2520and%2520AP2M1%2520essential%2520for%2520viral%2520assembly%26jtitle%3DPLoS%2520Pathog.%26date%3D2012%26volume%3D8%26issue%3D8%26spage%3De1002845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lg1SZRFRpHtVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span> SuperNova Life Science. Human Kinome Heat Map. <a href="http://www.supernovalifescience.com/HM/HM%2041.pdf" class="extLink">http://www.supernovalifescience.com/HM/HM%2041.pdf</a><span class="NLM_x"> (</span><span class="NLM_year">2008</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SuperNova+Life+Science.+Human+Kinome+Heat+Map.+http%3A%2F%2Fwww.supernovalifescience.com%2FHM%2FHM%252041.pdf+%282008%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Neveu, G.; Ziv-Av, A.; Barouch-Bentov, R.; Bekerman, E.; Mulholland, J.; Einav, S.</span><span> </span><span class="NLM_article-title">AAK1 and GAK regulate hepatitis C virus entry and are potential drug targets</span>.  <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1128/JVI.02705-14</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1128%2FJVI.02705-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=25653444" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=G.+Neveu&author=A.+Ziv-Av&author=R.+Barouch-Bentov&author=E.+Bekerman&author=J.+Mulholland&author=S.+Einav&title=AAK1+and+GAK+regulate+hepatitis+C+virus+entry+and+are+potential+drug+targets&doi=10.1128%2FJVI.02705-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FJVI.02705-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02705-14%26sid%3Dliteratum%253Aachs%26aulast%3DNeveu%26aufirst%3DG.%26atitle%3DAAK1%2520and%2520GAK%2520regulate%2520hepatitis%2520C%2520virus%2520entry%2520and%2520are%2520potential%2520drug%2520targets%26jtitle%3DJ.%2520Virol.%26date%3D2015%26doi%3D10.1128%2FJVI.02705-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W.H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lélias, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+P.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lg1SZRFRpHtVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.H.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BP.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Bellei, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitisci, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Migliano, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardinali, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picardo, M.</span><span> </span><span class="NLM_article-title">Pyridinyl imidazole compounds interfere with melanosomes sorting through the inhibition of cyclin G-associated kinase, a regulator of cathepsins maturation</span> <span class="citation_source-journal">Cell. Signalling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">716</span><span class="NLM_x">–</span> <span class="NLM_lpage">723</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1016%2Fj.cellsig.2013.12.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=24412755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=716-723&author=B.+Belleiauthor=A.+Pitisciauthor=E.+Miglianoauthor=G.+Cardinaliauthor=M.+Picardo&title=Pyridinyl+imidazole+compounds+interfere+with+melanosomes+sorting+through+the+inhibition+of+cyclin+G-associated+kinase%2C+a+regulator+of+cathepsins+maturation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridinyl imidazole compounds interfere with melanosomes sorting through the inhibition of Cyclin G-associated Kinase, a regulator of cathepsins maturation</span></div><div class="casAuthors">Bellei, Barbara; Pitisci, Angela; Migliano, Emilia; Cardinali, Giorgia; Picardo, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">716-723</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transfer of melanin-contg. melanosomes from melanocytes to neighboring keratinocytes results in skin pigmentation.  Pharmacol. modulation of melanosomal transfer has recently gained much attention as a strategy for modifying normal or abnormal pigmentation.  In this study, while investigating the impact of pyridinyl imidazole (PI) compds., a class of p38 MAPK inhibitors, on melanocyte differentiation we obsd. that some, but not all PIs interfere with the physiol. melanosome sorting producing a strong retention of melanin in the intracellular compartment assocd. with a general redn. of melanin synthesis.  Electron microscopy studies illustrated an accumulation of melanosomes inside melanocytes with enrichment in immature melanosome at stages II and III at the end of dendrites.  We identified cyclin G-assocd. kinase GAK, a protein expressed ubiquitously in various tissues, as the off-target responsible of intracellular melanin accumulation and we report evidence that reduced GAK-dependent cathepsin maturation is implicated in melanosome sorting deficiency.  The co-regulation of GAK and cathepsin B and L expression with the melanogenic biosynthetic pathway in normal human melanocytes as well as in B16-F0 melanoma cells strengthen the idea that these proteins represent new possible targets for prevention and treatment of irregular pigmentation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLyREgMwFFirVg90H21EOLACvtfcHk0lhFIFMb43i3AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGls7s%253D&md5=b373f2cbfde96773364849b878219e2d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2013.12.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2013.12.023%26sid%3Dliteratum%253Aachs%26aulast%3DBellei%26aufirst%3DB.%26aulast%3DPitisci%26aufirst%3DA.%26aulast%3DMigliano%26aufirst%3DE.%26aulast%3DCardinali%26aufirst%3DG.%26aulast%3DPicardo%26aufirst%3DM.%26atitle%3DPyridinyl%2520imidazole%2520compounds%2520interfere%2520with%2520melanosomes%2520sorting%2520through%2520the%2520inhibition%2520of%2520cyclin%2520G-associated%2520kinase%252C%2520a%2520regulator%2520of%2520cathepsins%2520maturation%26jtitle%3DCell.%2520Signalling%26date%3D2014%26volume%3D26%26spage%3D716%26epage%3D723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Meiby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elkins, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohlson, S.</span><span> </span><span class="NLM_article-title">Fragment screening of cyclin G-associated kinase by weak affinity chromatography</span> <span class="citation_source-journal">Anal. Bioanal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">404</span><span class="NLM_x">, </span> <span class="NLM_fpage">2417</span><span class="NLM_x">–</span> <span class="NLM_lpage">2425</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=22918538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Cgtb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=404&publication_year=2012&pages=2417-2425&author=E.+Meibyauthor=S.+Knappauthor=J.+M.+Elkinsauthor=S.+Ohlson&title=Fragment+screening+of+cyclin+G-associated+kinase+by+weak+affinity+chromatography"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment screening of cyclin G-associated kinase by weak affinity chromatography</span></div><div class="casAuthors">Meiby, Elinor; Knapp, Stefan; Elkins, Jonathan M.; Ohlson, Sten</div><div class="citationInfo"><span class="NLM_cas:title">Analytical and Bioanalytical Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">404</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2417-2425</span>CODEN:
                <span class="NLM_cas:coden">ABCNBP</span>;
        ISSN:<span class="NLM_cas:issn">1618-2642</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) has become a new strategy for drug discovery where lead compds. are evolved from small mols.  These fragments form low affinity interactions (dissocn. const. (KD) = mM - μM) with protein targets, which require fragment screening methods of sufficient sensitivity.  Weak affinity chromatog. (WAC) is a promising new technol. for fragment screening based on selective retention of fragments by a drug target.  Kinases are a major pharmaceutical target, and FBDD has been successfully applied to several of these targets.  In this work, we have demonstrated the potential to use WAC in combination with mass spectrometry (MS) detection for fragment screening of a kinase target-cyclin G-assocd. kinase (GAK).  One hundred seventy fragments were selected for WAC screening by virtual screening of a com. fragment library against the ATP-binding site of five different proteins.  GAK protein was immobilized on a capillary HPLC column, and compd. binding was characterized by frontal affinity chromatog.  Compds. were screened in sets of 13 or 14, in combination with MS detection for enhanced throughput.  Seventy-eight fragments (46 %) with KD < 200 μM were detected, including a few highly efficient GAK binders (KD of 2 μM; ligand efficiency = 0.51).  Of special interest is that chiral screening by WAC may be possible, as two stereoisomeric fragments, which both contained one chiral center, demonstrated twin peaks.  This ability, in combination with the robustness, sensitivity, and simplicity of WAC makes it a new method for fragment screening of considerable potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeCQXELkNLdLVg90H21EOLACvtfcHk0lhFIFMb43i3AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Cgtb7N&md5=b990b75b1d9cc5c4149ef1a81f199ee4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMeiby%26aufirst%3DE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DOhlson%26aufirst%3DS.%26atitle%3DFragment%2520screening%2520of%2520cyclin%2520G-associated%2520kinase%2520by%2520weak%2520affinity%2520chromatography%26jtitle%3DAnal.%2520Bioanal.%2520Chem.%26date%3D2012%26volume%3D404%26spage%3D2417%26epage%3D2425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Sakurai, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuzaki, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasakura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tabara, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabuta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nojima, H.</span><span> </span><span class="NLM_article-title">Gefitinib and luteolin cause growth arrest of human prostate cancer PC-3 cells via inhibition of cyclin G-associated kinase and induction of miR-630</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">e100124</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e100124&issue=6&author=M.+A.+Sakuraiauthor=Y.+Ozakiauthor=D.+Okuzakiauthor=Y.+Naitoauthor=T.+Sasakuraauthor=A.+Okamotoauthor=H.+Tabaraauthor=T.+Inoueauthor=M.+Hagiyamaauthor=A.+Itoauthor=N.+Yabutaauthor=H.+Nojima&title=Gefitinib+and+luteolin+cause+growth+arrest+of+human+prostate+cancer+PC-3+cells+via+inhibition+of+cyclin+G-associated+kinase+and+induction+of+miR-630"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSakurai%26aufirst%3DM.%2BA.%26aulast%3DOzaki%26aufirst%3DY.%26aulast%3DOkuzaki%26aufirst%3DD.%26aulast%3DNaito%26aufirst%3DY.%26aulast%3DSasakura%26aufirst%3DT.%26aulast%3DOkamoto%26aufirst%3DA.%26aulast%3DTabara%26aufirst%3DH.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DHagiyama%26aufirst%3DM.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DYabuta%26aufirst%3DN.%26aulast%3DNojima%26aufirst%3DH.%26atitle%3DGefitinib%2520and%2520luteolin%2520cause%2520growth%2520arrest%2520of%2520human%2520prostate%2520cancer%2520PC-3%2520cells%2520via%2520inhibition%2520of%2520cyclin%2520G-associated%2520kinase%2520and%2520induction%2520of%2520miR-630%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D6%26spage%3De100124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Susa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwab, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornicek, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mankin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, Z.</span><span> </span><span class="NLM_article-title">Cyclin G-associated kinase is necessary for osteosarcoma cell proliferation and receptor trafficking</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3342</span><span class="NLM_x">–</span> <span class="NLM_lpage">3350</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1158%2F1535-7163.MCT-10-0637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=20881269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=3342-3350&author=M.+Susaauthor=E.+Choyauthor=X.+Liuauthor=J.+Schwabauthor=F.+J.+Hornicekauthor=H.+Mankinauthor=Z.+Duan&title=Cyclin+G-associated+kinase+is+necessary+for+osteosarcoma+cell+proliferation+and+receptor+trafficking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin G-Associated Kinase Is Necessary for Osteosarcoma Cell Proliferation and Receptor Trafficking</span></div><div class="casAuthors">Susa, Michiro; Choy, Edwin; Liu, Xianzhe; Schwab, Joseph; Hornicek, Francis J.; Mankin, Henry; Duan, Zhenfeng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3342-3350</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Osteosarcoma is the most frequent primary malignant bone tumor among the children.  The advent of neoadjuvant chemotherapy significantly improved the prognosis of patients with osteosarcoma in the 1980s, but it has since plateaued in the past decades.  Recently, one of the most researched areas in sarcoma treatment is tyrosine kinases.  Here, we describe research on a serine/threonine kinase, cyclin G-assocd. kinase (GAK), which has not been reported in osteosarcoma previously.  In this study, a lentiviral based human shRNA library was utilized to screen for kinases in KHOS and U-2OS osteosarcoma cells.  The expression of GAK was examd. in osteosarcoma and the effect on cell proliferation was analyzed by GAK siRNA knockdown.  The level of GAK expression and its correlation to prognosis was analyzed in osteosarcoma tissue microarray.  The effect of GAK depletion on insulin-like growth factor and epidermal growth factor receptor-mediated signal transduction was analyzed by Western blot.  We obsd. that GAK was overexpressed in both osteosarcoma cell lines and tissue samples when compared with human osteoblasts.  GAK knockdown by siRNA decreased cell proliferation in both drug-sensitive and multidrug-resistant osteosarcoma cell lines.  Immunohistochem. of osteosarcoma tissue microarray revealed that overexpression of GAK was assocd. with poor prognosis.  Finally, knockdown of GAK resulted in alterations of receptor trafficking and several downstream proteins.  In conclusion, our results suggest that osteosarcoma cell proliferation and survival are dependent on GAK.  These findings may lead to the development of new therapeutic options for osteosarcoma.  Mol Cancer Ther; 9(12); 3342-50.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaz8xXEuEtOLVg90H21EOLACvtfcHk0lhFIFMb43i3AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsLrL&md5=6a087e9b0ca52770617d59967b190df2</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-10-0637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-10-0637%26sid%3Dliteratum%253Aachs%26aulast%3DSusa%26aufirst%3DM.%26aulast%3DChoy%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSchwab%26aufirst%3DJ.%26aulast%3DHornicek%26aufirst%3DF.%2BJ.%26aulast%3DMankin%26aufirst%3DH.%26aulast%3DDuan%26aufirst%3DZ.%26atitle%3DCyclin%2520G-associated%2520kinase%2520is%2520necessary%2520for%2520osteosarcoma%2520cell%2520proliferation%2520and%2520receptor%2520trafficking%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D3342%26epage%3D3350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Dzamko, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halliday, G. M.</span><span> </span><span class="NLM_article-title">Parkinson’s disease-implicated kinases in the brain; insights into disease pathogenesis</span> <span class="citation_source-journal">Front. Mol. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">–</span> <span class="NLM_lpage">57</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=7-57&author=N.+Dzamkoauthor=J.+Zhouauthor=Y.+Huangauthor=G.+M.+Halliday&title=Parkinson%E2%80%99s+disease-implicated+kinases+in+the+brain%3B+insights+into+disease+pathogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDzamko%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DHalliday%26aufirst%3DG.%2BM.%26atitle%3DParkinson%25E2%2580%2599s%2520disease-implicated%2520kinases%2520in%2520the%2520brain%253B%2520insights%2520into%2520disease%2520pathogenesis%26jtitle%3DFront.%2520Mol.%2520Neurosci.%26date%3D2014%26volume%3D24%26spage%3D7%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Kuduk, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Marco, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ray, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mab, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seager, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeplinger, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartman, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilodeau, M. T.</span><span> </span><span class="NLM_article-title">Heterocyclic fused pyridone carboxylic acid M1 positive allosteric modulators</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2533</span><span class="NLM_x">–</span> <span class="NLM_lpage">2537</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=2533-2537&author=S.+D.+Kudukauthor=C.+N.+Di+Marcoauthor=R.+K.+Changauthor=W.+J.+Rayauthor=L.+Mabauthor=M.+Wittmannauthor=M.+A.+Seagerauthor=K.+A.+Koeplingerauthor=C.+D.+Thompsonauthor=G.+D.+Hartmanauthor=M.+T.+Bilodeau&title=Heterocyclic+fused+pyridone+carboxylic+acid+M1+positive+allosteric+modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKuduk%26aufirst%3DS.%2BD.%26aulast%3DDi%2BMarco%26aufirst%3DC.%2BN.%26aulast%3DChang%26aufirst%3DR.%2BK.%26aulast%3DRay%26aufirst%3DW.%2BJ.%26aulast%3DMab%26aufirst%3DL.%26aulast%3DWittmann%26aufirst%3DM.%26aulast%3DSeager%26aufirst%3DM.%2BA.%26aulast%3DKoeplinger%26aufirst%3DK.%2BA.%26aulast%3DThompson%26aufirst%3DC.%2BD.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26aulast%3DBilodeau%26aufirst%3DM.%2BT.%26atitle%3DHeterocyclic%2520fused%2520pyridone%2520carboxylic%2520acid%2520M1%2520positive%2520allosteric%2520modulators%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D2533%26epage%3D2537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Wohlfahrt, G.; Tormakangas, O.; Salo, H.; Hoglund, L.; Karjalainen, A.; Knuutila, P.; Holm, P.; Rasku, S.; Vesalainen, A.</span><span> </span><span class="NLM_article-title">Androgen receptor modulating compounds</span>. WO2011/051540,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=G.+Wohlfahrt&author=O.+Tormakangas&author=H.+Salo&author=L.+Hoglund&author=A.+Karjalainen&author=P.+Knuutila&author=P.+Holm&author=S.+Rasku&author=A.+Vesalainen&title=Androgen+receptor+modulating+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWohlfahrt%26aufirst%3DG.%26atitle%3DAndrogen%2520receptor%2520modulating%2520compounds%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Zhang, N.; Ayral-Kaloustian, S.; Mansour, T.; Nguyen, T.; Niu, C.; Rosfjord, E.; Suayan, R.; Tsou, H.</span><span> </span><span class="NLM_article-title">2-Aryl and 2-heteroarylthiazolyl compounds, methods for their preparation and use thereof</span>. WO2009/120826,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=N.+Zhang&author=S.+Ayral-Kaloustian&author=T.+Mansour&author=T.+Nguyen&author=C.+Niu&author=E.+Rosfjord&author=R.+Suayan&author=H.+Tsou&title=2-Aryl+and+2-heteroarylthiazolyl+compounds%2C+methods+for+their+preparation+and+use+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DN.%26atitle%3D2-Aryl%2520and%25202-heteroarylthiazolyl%2520compounds%252C%2520methods%2520for%2520their%2520preparation%2520and%2520use%2520thereof%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Taurins, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan Khouw, V.</span><span> </span><span class="NLM_article-title">Isothiazolopyridines. I. Synthesis and spectra of isothiazolo[3,4-b]-, 3-amino-isothiazolo[4,3-b]-, isothiazolo[5,4-b]-, and 3-methylisothiazolo[5,4-c]pyridines. Preparation and spectra of some 2,3- and 3,4-disubstituted pyridines</span> <span class="citation_source-journal">Can. J. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1741</span><span class="NLM_x">–</span> <span class="NLM_lpage">1748</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1973&pages=1741-1748&author=A.+Taurinsauthor=V.+Tan+Khouw&title=Isothiazolopyridines.+I.+Synthesis+and+spectra+of+isothiazolo%5B3%2C4-b%5D-%2C+3-amino-isothiazolo%5B4%2C3-b%5D-%2C+isothiazolo%5B5%2C4-b%5D-%2C+and+3-methylisothiazolo%5B5%2C4-c%5Dpyridines.+Preparation+and+spectra+of+some+2%2C3-+and+3%2C4-disubstituted+pyridines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTaurins%26aufirst%3DA.%26aulast%3DTan%2BKhouw%26aufirst%3DV.%26atitle%3DIsothiazolopyridines.%2520I.%2520Synthesis%2520and%2520spectra%2520of%2520isothiazolo%255B3%252C4-b%255D-%252C%25203-amino-isothiazolo%255B4%252C3-b%255D-%252C%2520isothiazolo%255B5%252C4-b%255D-%252C%2520and%25203-methylisothiazolo%255B5%252C4-c%255Dpyridines.%2520Preparation%2520and%2520spectra%2520of%2520some%25202%252C3-%2520and%25203%252C4-disubstituted%2520pyridines%26jtitle%3DCan.%2520J.%2520Chem.%26date%3D1973%26volume%3D51%26spage%3D1741%26epage%3D1748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Miyaura, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, A.</span><span> </span><span class="NLM_article-title">Palladium-catalyzed cross-coupling reactions of organoboron compounds</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">2457</span><span class="NLM_x">–</span> <span class="NLM_lpage">2483</span></span><div class="citationLinks">[<a href="/doi/10.1021/cr00039a007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADyaK2MXoslGiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1995&pages=2457-2483&author=N.+Miyauraauthor=A.+Suzuki&title=Palladium-catalyzed+cross-coupling+reactions+of+organoboron+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-Catalyzed Cross-Coupling Reactions of Organoboron Compounds</span></div><div class="casAuthors">Miyaura, Norio; Suzuki, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2457-83</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with >250 refs. including title reactions and their mechanisms, prepn. of organoboron reagents, alkoxycarbonylation and dimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHwupBJT1UCbVg90H21EOLACvtfcHk0lhCBU2gMIYMWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXoslGiurg%253D&md5=d127b414a75161652876eebc3ed0c486</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fcr00039a007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00039a007%26sid%3Dliteratum%253Aachs%26aulast%3DMiyaura%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DA.%26atitle%3DPalladium-catalyzed%2520cross-coupling%2520reactions%2520of%2520organoboron%2520compounds%26jtitle%3DChem.%2520Rev.%26date%3D1995%26volume%3D95%26spage%3D2457%26epage%3D2483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincke, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufholz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zenn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmermann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutiérrez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzos-Skintges, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joachimiak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muyldermans, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herberg, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span> </span><span class="NLM_article-title">Structure of cyclin G-associated kinase (GAK) trapped in different conformations using nanobodies</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">459</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2014&pages=59-69&author=A.+Chaikuadauthor=T.+Keatesauthor=C.+Vinckeauthor=M.+Kaufholzauthor=M.+Zennauthor=B.+Zimmermannauthor=C.+Guti%C3%A9rrezauthor=R.+G.+Zhangauthor=C.+Hatzos-Skintgesauthor=A.+Joachimiakauthor=S.+Muyldermansauthor=F.+W.+Herbergauthor=S.+Knappauthor=S.+M%C3%BCller&title=Structure+of+cyclin+G-associated+kinase+%28GAK%29+trapped+in+different+conformations+using+nanobodies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DVincke%26aufirst%3DC.%26aulast%3DKaufholz%26aufirst%3DM.%26aulast%3DZenn%26aufirst%3DM.%26aulast%3DZimmermann%26aufirst%3DB.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DR.%2BG.%26aulast%3DHatzos-Skintges%26aufirst%3DC.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DMuyldermans%26aufirst%3DS.%26aulast%3DHerberg%26aufirst%3DF.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DStructure%2520of%2520cyclin%2520G-associated%2520kinase%2520%2528GAK%2529%2520trapped%2520in%2520different%2520conformations%2520using%2520nanobodies%26jtitle%3DBiochem.%2520J.%26date%3D2014%26volume%3D459%26spage%3D59%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Exploration of type II binding mode: A privileged approach for kinase inhibitor focused drug discovery?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1230</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+A+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520A%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Lupberger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeisel, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thumann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fofana, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zona, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mee, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Royer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahid, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavillette, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fresquet, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosset, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothenberg, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietschmann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pessaux, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doffoël, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raffelsberger, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poch, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mckeating, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brino, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumert, T. F.</span><span> </span><span class="NLM_article-title">EGFR and EphA2 are host factors for hepatitis C virus entry and possible targets for antiviral therapy</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">589</span><span class="NLM_x">–</span> <span class="NLM_lpage">595</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1038%2Fnm.2341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=21516087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFyqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=589-595&author=J.+Lupbergerauthor=M.+B.+Zeiselauthor=F.+Xiaoauthor=C.+Thumannauthor=I.+Fofanaauthor=L.+Zonaauthor=C.+Davisauthor=C.+J.+Meeauthor=M.+Turekauthor=S.+Gorkeauthor=C.+Royerauthor=B.+Fischerauthor=M.+N.+Zahidauthor=D.+Lavilletteauthor=J.+Fresquetauthor=F.+Cossetauthor=S.+M.+Rothenbergauthor=T.+Pietschmannauthor=A.+H.+Patelauthor=P.+Pessauxauthor=M.+Doffo%C3%ABlauthor=W.+Raffelsbergerauthor=O.+Pochauthor=J.+A.+Mckeatingauthor=L.+A.+Brinoauthor=T.+F.+Baumert&title=EGFR+and+EphA2+are+host+factors+for+hepatitis+C+virus+entry+and+possible+targets+for+antiviral+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR and EphA2 are host factors for hepatitis C virus entry and possible targets for antiviral therapy</span></div><div class="casAuthors">Lupberger, Joachim; Zeisel, Mirjam B.; Xiao, Fei; Thumann, Christine; Fofana, Isabel; Zona, Laetitia; Davis, Christopher; Mee, Christopher J.; Turek, Marine; Gorke, Sebastian; Royer, Cathy; Fischer, Benoit; Zahid, Muhammad N.; Lavillette, Dimitri; Fresquet, Judith; Cosset, Francois-Loic; Rothenberg, S. Michael; Pietschmann, Thomas; Patel, Arvind H.; Pessaux, Patrick; Doffoel, Michel; Raffelsberger, Wolfgang; Poch, Olivier; McKeating, Jane A.; Brino, Laurent; Baumert, Thomas F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">589-595</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) is a major cause of liver disease, but therapeutic options are limited and there are no prevention strategies.  Viral entry is the first step of infection and requires the cooperative interaction of several host cell factors.  Using a functional RNAi kinase screen, we identified epidermal growth factor receptor and ephrin receptor A2 as host cofactors for HCV entry.  Blocking receptor kinase activity by approved inhibitors broadly impaired infection by all major HCV genotypes and viral escape variants in cell culture and in a human liver chimeric mouse model in vivo.  The identified receptor tyrosine kinases (RTKs) mediate HCV entry by regulating CD81-claudin-1 co-receptor assocns. and viral glycoprotein-dependent membrane fusion.  These results identify RTKs as previously unknown HCV entry cofactors and show that tyrosine kinase inhibitors have substantial antiviral activity.  Inhibition of RTK function may constitute a new approach for prevention and treatment of HCV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPNZcMOXBl57Vg90H21EOLACvtfcHk0liM1W3MtDD8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFyqtL4%253D&md5=9a772f9cb03077335c7e7f5918a6e1ef</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnm.2341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2341%26sid%3Dliteratum%253Aachs%26aulast%3DLupberger%26aufirst%3DJ.%26aulast%3DZeisel%26aufirst%3DM.%2BB.%26aulast%3DXiao%26aufirst%3DF.%26aulast%3DThumann%26aufirst%3DC.%26aulast%3DFofana%26aufirst%3DI.%26aulast%3DZona%26aufirst%3DL.%26aulast%3DDavis%26aufirst%3DC.%26aulast%3DMee%26aufirst%3DC.%2BJ.%26aulast%3DTurek%26aufirst%3DM.%26aulast%3DGorke%26aufirst%3DS.%26aulast%3DRoyer%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DB.%26aulast%3DZahid%26aufirst%3DM.%2BN.%26aulast%3DLavillette%26aufirst%3DD.%26aulast%3DFresquet%26aufirst%3DJ.%26aulast%3DCosset%26aufirst%3DF.%26aulast%3DRothenberg%26aufirst%3DS.%2BM.%26aulast%3DPietschmann%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DA.%2BH.%26aulast%3DPessaux%26aufirst%3DP.%26aulast%3DDoffo%25C3%25ABl%26aufirst%3DM.%26aulast%3DRaffelsberger%26aufirst%3DW.%26aulast%3DPoch%26aufirst%3DO.%26aulast%3DMckeating%26aufirst%3DJ.%2BA.%26aulast%3DBrino%26aufirst%3DL.%2BA.%26aulast%3DBaumert%26aufirst%3DT.%2BF.%26atitle%3DEGFR%2520and%2520EphA2%2520are%2520host%2520factors%2520for%2520hepatitis%2520C%2520virus%2520entry%2520and%2520possible%2520targets%2520for%2520antiviral%2520therapy%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D589%26epage%3D595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Murray, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tassello, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span> </span><span class="NLM_article-title">Alanine scanning of the hepatitis C virus core protein reveals numerous residues essential for production of infectious virus</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">10220</span><span class="NLM_x">–</span> <span class="NLM_lpage">10231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1128%2FJVI.00793-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=17634240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtV2gtb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=10220-10231&author=C.+L.+Murrayauthor=C.+T.+Jonesauthor=J.+Tasselloauthor=C.+M.+Rice&title=Alanine+scanning+of+the+hepatitis+C+virus+core+protein+reveals+numerous+residues+essential+for+production+of+infectious+virus"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Alanine scanning of the hepatitis C virus core protein reveals numerous residues essential for production of infectious virus</span></div><div class="casAuthors">Murray, Catherine L.; Jones, Christopher T.; Tassello, Jodie; Rice, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10220-10231</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) is an important human pathogen affecting an estd. 3% of the world's population.  Recent advances have enabled in vitro propagation of the virus and allow assembly and egress to be investigated for the first time.  As a component of the virion, the HCV core protein likely functions primarily in infectious virus prodn., although little is known about the determinants of this activity.  To investigate the roles of core in the viral life cycle, we performed a comprehensive deletion and alanine scanning mutagenesis study of this protein in the context of a genotype 2a reporter virus.  We have confirmed that core protein is essential for infectious virion prodn. and have identified numerous residues required for this role.  The infectivity of several assembly-defective core mutants could be rescued by compensatory mutations identified in p7 and NS2, suggesting genetic interactions with core and highlighting the importance of these nonstructural proteins in infectious virion morphogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptehuI2myC37Vg90H21EOLACvtfcHk0limGbIDNX-9UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtV2gtb3E&md5=a0e817ce366fdbc576e513c4a080df46</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FJVI.00793-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00793-07%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BL.%26aulast%3DJones%26aufirst%3DC.%2BT.%26aulast%3DTassello%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26atitle%3DAlanine%2520scanning%2520of%2520the%2520hepatitis%2520C%2520virus%2520core%2520protein%2520reveals%2520numerous%2520residues%2520essential%2520for%2520production%2520of%2520infectious%2520virus%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D10220%26epage%3D10231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Bartosch, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubuisson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosset, F. L.</span><span> </span><span class="NLM_article-title">Infectious hepatitis C virus pseudo-particles containing functional E1-E2 envelope protein complexes</span> <span class="citation_source-journal">J. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">197</span><span class="NLM_x">, </span> <span class="NLM_fpage">633</span><span class="NLM_x">–</span> <span class="NLM_lpage">642</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=2003&pages=633-642&author=B.+Bartoschauthor=J.+Dubuissonauthor=F.+L.+Cosset&title=Infectious+hepatitis+C+virus+pseudo-particles+containing+functional+E1-E2+envelope+protein+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBartosch%26aufirst%3DB.%26aulast%3DDubuisson%26aufirst%3DJ.%26aulast%3DCosset%26aufirst%3DF.%2BL.%26atitle%3DInfectious%2520hepatitis%2520C%2520virus%2520pseudo-particles%2520containing%2520functional%2520E1-E2%2520envelope%2520protein%2520complexes%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2003%26volume%3D197%26spage%3D633%26epage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Ricotta, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preiss, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teichert, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Höning, S.</span><span> </span><span class="NLM_article-title">Characterization of a protein phosphatase 2A holoenzyme that dephosphorylates the clathrin adaptors AP-1 and AP-2</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">5510</span><span class="NLM_x">–</span> <span class="NLM_lpage">5517</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=5510-5517&author=D.+Ricottaauthor=J.+Hansenauthor=C.+Preissauthor=D.+Teichertauthor=S.+H%C3%B6ning&title=Characterization+of+a+protein+phosphatase+2A+holoenzyme+that+dephosphorylates+the+clathrin+adaptors+AP-1+and+AP-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRicotta%26aufirst%3DD.%26aulast%3DHansen%26aufirst%3DJ.%26aulast%3DPreiss%26aufirst%3DC.%26aulast%3DTeichert%26aufirst%3DD.%26aulast%3DH%25C3%25B6ning%26aufirst%3DS.%26atitle%3DCharacterization%2520of%2520a%2520protein%2520phosphatase%25202A%2520holoenzyme%2520that%2520dephosphorylates%2520the%2520clathrin%2520adaptors%2520AP-1%2520and%2520AP-2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D5510%26epage%3D5517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Bhattacharyya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warfield, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruthel, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bavari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hope, T. J.</span><span> </span><span class="NLM_article-title">Ebola virus uses clathrin-mediated endocytosis as an entry pathway</span> <span class="citation_source-journal">Virology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">401</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=401&publication_year=2010&pages=18-28&author=S.+Bhattacharyyaauthor=K.+L.+Warfieldauthor=G.+Ruthelauthor=S.+Bavariauthor=M.+J.+Amanauthor=T.+J.+Hope&title=Ebola+virus+uses+clathrin-mediated+endocytosis+as+an+entry+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharyya%26aufirst%3DS.%26aulast%3DWarfield%26aufirst%3DK.%2BL.%26aulast%3DRuthel%26aufirst%3DG.%26aulast%3DBavari%26aufirst%3DS.%26aulast%3DAman%26aufirst%3DM.%2BJ.%26aulast%3DHope%26aufirst%3DT.%2BJ.%26atitle%3DEbola%2520virus%2520uses%2520clathrin-mediated%2520endocytosis%2520as%2520an%2520entry%2520pathway%26jtitle%3DVirology%26date%3D2010%26volume%3D401%26spage%3D18%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Jin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, K.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, K.</span><span> </span><span class="NLM_article-title">Hantaan virus enters cells by clathrin-dependent receptor-mediated endocytosis</span> <span class="citation_source-journal">Virology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">294</span><span class="NLM_x">, </span> <span class="NLM_fpage">60</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=2002&pages=60-69&author=M.+Jinauthor=J.+Parkauthor=S.+Leeauthor=B.+Parkauthor=J.+Shinauthor=K.-J.+Songauthor=T.+I.+Ahnauthor=S.+Y.+Hwangauthor=B.+Y.+Ahnauthor=K.+Ahn&title=Hantaan+virus+enters+cells+by+clathrin-dependent+receptor-mediated+endocytosis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DPark%26aufirst%3DB.%26aulast%3DShin%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DK.-J.%26aulast%3DAhn%26aufirst%3DT.%2BI.%26aulast%3DHwang%26aufirst%3DS.%2BY.%26aulast%3DAhn%26aufirst%3DB.%2BY.%26aulast%3DAhn%26aufirst%3DK.%26atitle%3DHantaan%2520virus%2520enters%2520cells%2520by%2520clathrin-dependent%2520receptor-mediated%2520endocytosis%26jtitle%3DVirology%26date%3D2002%26volume%3D294%26spage%3D60%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Boge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyss, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonifacino, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thali, M.</span><span> </span><span class="NLM_article-title">A membrane-proximal tyrosine-based signal mediates internalization of the HIV-1 envelope glycoprotein via interaction with the AP-2 clathrin adaptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">15773</span><span class="NLM_x">–</span> <span class="NLM_lpage">15778</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=15773-15778&author=M.+Bogeauthor=S.+P.+Wyssauthor=J.+S.+Bonifacinoauthor=M.+Thali&title=A+membrane-proximal+tyrosine-based+signal+mediates+internalization+of+the+HIV-1+envelope+glycoprotein+via+interaction+with+the+AP-2+clathrin+adaptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoge%26aufirst%3DM.%26aulast%3DWyss%26aufirst%3DS.%2BP.%26aulast%3DBonifacino%26aufirst%3DJ.%2BS.%26aulast%3DThali%26aufirst%3DM.%26atitle%3DA%2520membrane-proximal%2520tyrosine-based%2520signal%2520mediates%2520internalization%2520of%2520the%2520HIV-1%2520envelope%2520glycoprotein%2520via%2520interaction%2520with%2520the%2520AP-2%2520clathrin%2520adaptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D15773%26epage%3D15778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lj-rViMZYYh6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">CCP4</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=CCP4&title=The+CCP4+suite%3A+programs+for+protein+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lj-rViMZYYh6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lj-rViMZYYh6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lj-rViMZYYh6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leaver-Fay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snoeyink, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">W375</span><span class="NLM_x">–</span> <span class="NLM_lpage">W383</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1093%2Fnar%2Fgkm216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=17452350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W375-W383&author=I.+W.+Davisauthor=A.+Leaver-Fayauthor=V.+B.+Chenauthor=J.+N.+Blockauthor=G.+J.+Kapralauthor=X.+Wangauthor=L.+W.+Murrayauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+contacts+and+structure+validation+for+proteins+and+nucleic+acids"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span></div><div class="casAuthors">Davis Ian W; Leaver-Fay Andrew; Chen Vincent B; Block Jeremy N; Kapral Gary J; Wang Xueyi; Murray Laura W; Arendall W Bryan 3rd; Snoeyink Jack; Richardson Jane S; Richardson David C</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W375-83</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MolProbity is a general-purpose web server offering quality validation for 3D structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact analysis of any steric problems within the molecules as well as updated dihedral-angle diagnostics, and it can calculate and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addition and full optimization of all hydrogen atoms, both polar and nonpolar.  New analysis functions have been added for RNA, for interfaces, and for NMR ensembles.  Additionally, both the web site and major component programs have been rewritten to improve speed, convenience, clarity and integration with other resources.  MolProbity results are reported in multiple forms: as overall numeric scores, as lists or charts of local problems, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KiNG viewer.  This service is available free to all users at http://molprobity.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuyhuetoNS59bHMBt0mMIXfW6udTcc2eawdQ4Fu9t1Irntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D&md5=726ca1e7e162ab2a4a05048f493731fb</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm216%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DLeaver-Fay%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DBlock%26aufirst%3DJ.%2BN.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520contacts%2520and%2520structure%2520validation%2520for%2520proteins%2520and%2520nucleic%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW375%26epage%3DW383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Lindenbach, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Syder, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wölk, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tellinghuisen, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maruyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hynes, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeating, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span> </span><span class="NLM_article-title">Complete replication of hepatitis C virus in cell culture</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">309</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">626</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1126%2Fscience.1114016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=15947137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsFChtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=309&publication_year=2005&pages=623-626&author=B.+D.+Lindenbachauthor=M.+J.+Evansauthor=A.+J.+Syderauthor=B.+W%C3%B6lkauthor=T.+L.+Tellinghuisenauthor=C.+C.+Liuauthor=T.+Maruyamaauthor=R.+O.+Hynesauthor=D.+R.+Burtonauthor=J.+A.+McKeatingauthor=C.+M.+Rice&title=Complete+replication+of+hepatitis+C+virus+in+cell+culture"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Complete Replication of Hepatitis C Virus in Cell Culture</span></div><div class="casAuthors">Lindenbach, Brett D.; Evans, Matthew J.; Syder, Andrew J.; Woelk, Benno; Tellinghuisen, Timothy L.; Liu, Christopher C.; Maruyama, Toshiaki; Hynes, Richard O.; Burton, Dennis R.; McKeating, Jane A.; Rice, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">309</span>
        (<span class="NLM_cas:issue">5734</span>),
    <span class="NLM_cas:pages">623-626</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Many aspects of the hepatitis C virus (HCV) life cycle have not been reproduced in cell culture, which has slowed research progress on this important human pathogen.  Here, we describe a full-length HCV genome that replicates and produces virus particles that are infectious in cell culture (HCVcc).  Replication of HCVcc was robust, producing nearly 105 infectious units per mL within 48 h.  Virus particles were filterable and neutralized with a monoclonal antibody against the viral glycoprotein E2.  Viral entry was dependent on cellular expression of a putative HCV receptor, CD81.  HCVcc replication was inhibited by interferon-α and by several HCV-specific antiviral compds., suggesting that this in vitro system will aid in the search for improved antivirals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroGMVonlJq-rVg90H21EOLACvtfcHk0liYAU7LhxgBJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsFChtr4%253D&md5=05a4723cb8bf6b4e33cecab9be5bcdea</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1126%2Fscience.1114016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1114016%26sid%3Dliteratum%253Aachs%26aulast%3DLindenbach%26aufirst%3DB.%2BD.%26aulast%3DEvans%26aufirst%3DM.%2BJ.%26aulast%3DSyder%26aufirst%3DA.%2BJ.%26aulast%3DW%25C3%25B6lk%26aufirst%3DB.%26aulast%3DTellinghuisen%26aufirst%3DT.%2BL.%26aulast%3DLiu%26aufirst%3DC.%2BC.%26aulast%3DMaruyama%26aufirst%3DT.%26aulast%3DHynes%26aufirst%3DR.%2BO.%26aulast%3DBurton%26aufirst%3DD.%2BR.%26aulast%3DMcKeating%26aufirst%3DJ.%2BA.%26aulast%3DRice%26aufirst%3DC.%2BM.%26atitle%3DComplete%2520replication%2520of%2520hepatitis%2520C%2520virus%2520in%2520cell%2520culture%26jtitle%3DScience%26date%3D2005%26volume%3D309%26spage%3D623%26epage%3D626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15a','cit15b'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 38 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Javier García-Cárceles, Elena Caballero, Carmen Gil, <span class="NLM_string-name hlFld-ContribAuthor">Ana Martínez</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase Inhibitors as Underexplored Antiviral Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00302%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DKinase%252BInhibitors%252Bas%252BUnderexplored%252BAntiviral%252BAgents%26aulast%3DGarc%25C3%25ADa-C%25C3%25A1rceles%26aufirst%3DJavier%26date%3D2021%26date%3D2021%26date%3D16022021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhouling Xie, Xiaoxiao Yang, Yajun Duan, Jihong Han, <span class="NLM_string-name hlFld-ContribAuthor">Chenzhong Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Kinase Inhibitors for the Treatment of Nononcologic Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1283-1345. <a href="https://doi.org/10.1021/acs.jmedchem.0c01511" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01511</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01511%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BKinase%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BNononcologic%252BDiseases%26aulast%3DXie%26aufirst%3DZhouling%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29082020%26date%3D22012021%26volume%3D64%26issue%3D3%26spage%3D1283%26epage%3D1345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rinky Raghuvanshi, <span class="NLM_string-name hlFld-ContribAuthor">Sandip B. Bharate</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Developments in the Use of Kinase Inhibitors for Management of Viral Infections. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01467%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BDevelopments%252Bin%252Bthe%252BUse%252Bof%252BKinase%252BInhibitors%252Bfor%252BManagement%252Bof%252BViral%252BInfections%26aulast%3DRaghuvanshi%26aufirst%3DRinky%26date%3D2021%26date%3D2020%26date%3D23082020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher
R. M. Asquith, Benedict-Tilman Berger, Jing Wan, James M. Bennett, Stephen J. Capuzzi, Daniel J. Crona, David H. Drewry, Michael P. East, Jonathan M. Elkins, Oleg Fedorov, Paulo H. Godoi, Debra M. Hunter, Stefan Knapp, Susanne Müller, Chad D. Torrice, Carrow I. Wells, H. Shelton Earp, Timothy M. Willson, <span class="NLM_string-name hlFld-ContribAuthor">William J. Zuercher</span>. </span><span class="cited-content_cbyCitation_article-title">SGC-GAK-1: A Chemical Probe for Cyclin G Associated Kinase (GAK). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (5)
                                     , 2830-2836. <a href="https://doi.org/10.1021/acs.jmedchem.8b01213" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01213</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01213%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSGC-GAK-1%25253A%252BA%252BChemical%252BProbe%252Bfor%252BCyclin%252BG%252BAssociated%252BKinase%252B%252528GAK%252529%26aulast%3DAsquith%26aufirst%3DChristopher%2BR.%2BM.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D01082018%26date%3D26022019%26date%3D15022019%26volume%3D62%26issue%3D5%26spage%3D2830%26epage%3D2836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Szu-Yuan Pu, Randy Wouters, Stanford Schor, Jef Rozenski, Rina Barouch-Bentov, Laura I. Prugar, Cecilia M. O’Brien, Jennifer M. Brannan, John M. Dye, Piet Herdewijn, Steven De Jonghe, <span class="NLM_string-name hlFld-ContribAuthor">Shirit Einav</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of Isothiazolo[4,3-b]pyridine-Based Inhibitors of Cyclin G Associated Kinase (GAK) with Broad-Spectrum Antiviral Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (14)
                                     , 6178-6192. <a href="https://doi.org/10.1021/acs.jmedchem.8b00613" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00613</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00613%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252BIsothiazolo%25255B4%25252C3-b%25255Dpyridine-Based%252BInhibitors%252Bof%252BCyclin%252BG%252BAssociated%252BKinase%252B%252528GAK%252529%252Bwith%252BBroad-Spectrum%252BAntiviral%252BActivity%26aulast%3DPu%26aufirst%3DSzu-Yuan%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D17042018%26date%3D16072018%26date%3D28062018%26volume%3D61%26issue%3D14%26spage%3D6178%26epage%3D6192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bouchra  Kitab</span>, <span class="hlFld-ContribAuthor ">Michinori  Kohara</span>, <span class="hlFld-ContribAuthor ">Kyoko  Tsukiyama-Kohara</span>. </span><span class="cited-content_cbyCitation_article-title">Host-Targeting Antivirals for Treatment of Hepatitis C. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.5772/intechopen.95373" title="DOI URL">https://doi.org/10.5772/intechopen.95373</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5772/intechopen.95373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5772%2Fintechopen.95373%26sid%3Dliteratum%253Aachs%26atitle%3DHost-Targeting%252BAntivirals%252Bfor%252BTreatment%252Bof%252BHepatitis%252BC%26aulast%3DKitab%26aufirst%3DBouchra%26date%3D2021%26date%3D2021%26pub%3DIntechOpen%26atitle%3DAdvances%252Bin%252BHepatology%26aulast%3DRodrigo%26aufirst%3DLuis%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Belén  Martinez-Gualda</span>, <span class="hlFld-ContribAuthor ">Sirle  Saul</span>, <span class="hlFld-ContribAuthor ">Mathy  Froeyen</span>, <span class="hlFld-ContribAuthor ">Dominique  Schols</span>, <span class="hlFld-ContribAuthor ">Piet  Herdewijn</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>, <span class="hlFld-ContribAuthor ">Steven  De Jonghe</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 3-phenyl- and 3-N-piperidinyl-isothiazolo[4,3-b]pyridines as highly potent inhibitors of cyclin G-associated kinase. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>213 </em>, 113158. <a href="https://doi.org/10.1016/j.ejmech.2021.113158" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113158</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113158%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B3-phenyl-%252Band%252B3-N-piperidinyl-isothiazolo%25255B4%25252C3-b%25255Dpyridines%252Bas%252Bhighly%252Bpotent%252Binhibitors%252Bof%252Bcyclin%252BG-associated%252Bkinase%26aulast%3DMartinez-Gualda%26aufirst%3DBel%25C3%25A9n%26date%3D2021%26volume%3D213%26spage%3D113158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ana-Belén  Blázquez</span>, <span class="hlFld-ContribAuthor ">Juan-Carlos  Saiz</span>. </span><span class="cited-content_cbyCitation_article-title">Potential for Protein Kinase Pharmacological Regulation in Flaviviridae Infections. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (24)
                                     , 9524. <a href="https://doi.org/10.3390/ijms21249524" title="DOI URL">https://doi.org/10.3390/ijms21249524</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21249524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21249524%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DPotential%252Bfor%252BProtein%252BKinase%252BPharmacological%252BRegulation%252Bin%252BFlaviviridae%252BInfections%26aulast%3DBl%25C3%25A1zquez%26aufirst%3DAna-Bel%25C3%25A9n%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D24%26spage%3D9524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">María Jesús  Cabrera-Afonso</span>, <span class="hlFld-ContribAuthor ">Sara  Cembellín</span>, <span class="hlFld-ContribAuthor ">Adnane  Halima-Salem</span>, <span class="hlFld-ContribAuthor ">Mateo  Berton</span>, <span class="hlFld-ContribAuthor ">Leyre  Marzo</span>, <span class="hlFld-ContribAuthor ">Abdellah  Miloudi</span>, <span class="hlFld-ContribAuthor ">M. Carmen  Maestro</span>, <span class="hlFld-ContribAuthor ">José  Alemán</span>. </span><span class="cited-content_cbyCitation_article-title">Metal-free visible light-promoted synthesis of isothiazoles: a catalytic approach for N–S bond formation from iminyl radicals under batch and flow conditions. </span><span class="cited-content_cbyCitation_journal-name">Green Chemistry</span><span> <strong>2020,</strong> <em>22 </em>
                                    (20)
                                     , 6792-6797. <a href="https://doi.org/10.1039/D0GC02618B" title="DOI URL">https://doi.org/10.1039/D0GC02618B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0GC02618B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0GC02618B%26sid%3Dliteratum%253Aachs%26jtitle%3DGreen%2520Chemistry%26atitle%3DMetal-free%252Bvisible%252Blight-promoted%252Bsynthesis%252Bof%252Bisothiazoles%25253A%252Ba%252Bcatalytic%252Bapproach%252Bfor%252BN%2525E2%252580%252593S%252Bbond%252Bformation%252Bfrom%252Biminyl%252Bradicals%252Bunder%252Bbatch%252Band%252Bflow%252Bconditions%26aulast%3DCabrera-Afonso%26aufirst%3DMar%25C3%25ADa%2BJes%25C3%25BAs%26date%3D2020%26date%3D2020%26volume%3D22%26issue%3D20%26spage%3D6792%26epage%3D6797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xingyue  Ji</span>, <span class="hlFld-ContribAuthor ">Zhuorong  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Medicinal chemistry strategies toward host targeting antiviral agents. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2020,</strong> <em>40 </em>
                                    (5)
                                     , 1519-1557. <a href="https://doi.org/10.1002/med.21664" title="DOI URL">https://doi.org/10.1002/med.21664</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21664%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DMedicinal%252Bchemistry%252Bstrategies%252Btoward%252Bhost%252Btargeting%252Bantiviral%252Bagents%26aulast%3DJi%26aufirst%3DXingyue%26date%3D2020%26date%3D2020%26volume%3D40%26issue%3D5%26spage%3D1519%26epage%3D1557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher R. M.  Asquith</span>, <span class="hlFld-ContribAuthor ">Graham J.  Tizzard</span>, <span class="hlFld-ContribAuthor ">James M.  Bennett</span>, <span class="hlFld-ContribAuthor ">Carrow I.  Wells</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Elkins</span>, <span class="hlFld-ContribAuthor ">Timothy M.  Willson</span>, <span class="hlFld-ContribAuthor ">Antti  Poso</span>, <span class="hlFld-ContribAuthor ">Tuomo  Laitinen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the Water Network in Cyclin G‐Associated Kinase (GAK) with 4‐Anilino‐quin(az)oline Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (13)
                                     , 1200-1215. <a href="https://doi.org/10.1002/cmdc.202000150" title="DOI URL">https://doi.org/10.1002/cmdc.202000150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000150%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DTargeting%252Bthe%252BWater%252BNetwork%252Bin%252BCyclin%252BG%2525E2%252580%252590Associated%252BKinase%252B%252528GAK%252529%252Bwith%252B4%2525E2%252580%252590Anilino%2525E2%252580%252590quin%252528az%252529oline%252BInhibitors%26aulast%3DAsquith%26aufirst%3DChristopher%2BR.%2BM.%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D13%26spage%3D1200%26epage%3D1215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher R. M.  Asquith</span>, <span class="hlFld-ContribAuthor ">Tuomo  Laitinen</span>, <span class="hlFld-ContribAuthor ">James M.  Bennett</span>, <span class="hlFld-ContribAuthor ">Carrow I.  Wells</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Elkins</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>, <span class="hlFld-ContribAuthor ">Graham J.  Tizzard</span>, <span class="hlFld-ContribAuthor ">Antti  Poso</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Analysis of the 4‐Anilinoquin(az)oline Kinase Inhibition Profiles of GAK/SLK/STK10 Using Quantitative Structure‐Activity Relationships. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (1)
                                     , 26-49. <a href="https://doi.org/10.1002/cmdc.201900521" title="DOI URL">https://doi.org/10.1002/cmdc.201900521</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201900521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201900521%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDesign%252Band%252BAnalysis%252Bof%252Bthe%252B4%2525E2%252580%252590Anilinoquin%252528az%252529oline%252BKinase%252BInhibition%252BProfiles%252Bof%252BGAK%25252FSLK%25252FSTK10%252BUsing%252BQuantitative%252BStructure%2525E2%252580%252590Activity%252BRelationships%26aulast%3DAsquith%26aufirst%3DChristopher%2BR.%2BM.%26date%3D2020%26date%3D2019%26volume%3D15%26issue%3D1%26spage%3D26%26epage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Janos  Sapi</span>, <span class="hlFld-ContribAuthor ">Stéphane  Gérard</span>. </span><span class="cited-content_cbyCitation_article-title">Bicyclic 5-6 Systems: Three Heteroatoms 2:1. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.1016/B978-0-12-818655-8.00021-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-818655-8.00021-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-818655-8.00021-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-818655-8.00021-4%26sid%3Dliteratum%253Aachs%26atitle%3DBicyclic%252B5-6%252BSystems%25253A%252BThree%252BHeteroatoms%252B2%25253A1%26aulast%3DSapi%26aufirst%3DJanos%26date%3D2020%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2020%26volume%3Dvol.%252010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Belén  Martinez-Gualda</span>, <span class="hlFld-ContribAuthor ">Szu-Yuan  Pu</span>, <span class="hlFld-ContribAuthor ">Mathy  Froeyen</span>, <span class="hlFld-ContribAuthor ">Piet  Herdewijn</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>, <span class="hlFld-ContribAuthor ">Steven  De Jonghe</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity relationship study of the pyridine moiety of isothiazolo[4,3-b]pyridines as antiviral agents targeting cyclin G-associated kinase. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (1)
                                     , 115188. <a href="https://doi.org/10.1016/j.bmc.2019.115188" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.115188</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.115188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.115188%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructure-activity%252Brelationship%252Bstudy%252Bof%252Bthe%252Bpyridine%252Bmoiety%252Bof%252Bisothiazolo%25255B4%25252C3-b%25255Dpyridines%252Bas%252Bantiviral%252Bagents%252Btargeting%252Bcyclin%252BG-associated%252Bkinase%26aulast%3DMartinez-Gualda%26aufirst%3DBel%25C3%25A9n%26date%3D2020%26volume%3D28%26issue%3D1%26spage%3D115188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J. Humberto  Pérez-Olais</span>, <span class="hlFld-ContribAuthor ">Fernando  Ruiz-Jiménez</span>, <span class="hlFld-ContribAuthor ">Esther J.  Calderón-Garcia</span>, <span class="hlFld-ContribAuthor ">L. Adrián  De Jesús-González</span>, <span class="hlFld-ContribAuthor ">Rosaura  Hernández-Rivas</span>, <span class="hlFld-ContribAuthor ">Rosa M.  del Angel</span>. </span><span class="cited-content_cbyCitation_article-title">The activity of Aurora kinase B is required for dengue virus release. </span><span class="cited-content_cbyCitation_journal-name">Virus Research</span><span> <strong>2019,</strong> <em>274 </em>, 197777. <a href="https://doi.org/10.1016/j.virusres.2019.197777" title="DOI URL">https://doi.org/10.1016/j.virusres.2019.197777</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.virusres.2019.197777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.virusres.2019.197777%26sid%3Dliteratum%253Aachs%26jtitle%3DVirus%2520Research%26atitle%3DThe%252Bactivity%252Bof%252BAurora%252Bkinase%252BB%252Bis%252Brequired%252Bfor%252Bdengue%252Bvirus%252Brelease%26aulast%3DP%25C3%25A9rez-Olais%26aufirst%3DJ.%2BHumberto%26date%3D2019%26volume%3D274%26spage%3D197777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher R. M.  Asquith</span>, <span class="hlFld-ContribAuthor ">James M.  Bennett</span>, <span class="hlFld-ContribAuthor ">Lianyong  Su</span>, <span class="hlFld-ContribAuthor ">Tuomo  Laitinen</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Elkins</span>, <span class="hlFld-ContribAuthor ">Julie E.  Pickett</span>, <span class="hlFld-ContribAuthor ">Carrow I.  Wells</span>, <span class="hlFld-ContribAuthor ">Zengbiao  Li</span>, <span class="hlFld-ContribAuthor ">Timothy M.  Willson</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>. </span><span class="cited-content_cbyCitation_article-title">Towards the Development of an In vivo Chemical Probe for Cyclin G Associated Kinase (GAK). </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (22)
                                     , 4016. <a href="https://doi.org/10.3390/molecules24224016" title="DOI URL">https://doi.org/10.3390/molecules24224016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24224016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24224016%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DTowards%252Bthe%252BDevelopment%252Bof%252Ban%252BIn%252Bvivo%252BChemical%252BProbe%252Bfor%252BCyclin%252BG%252BAssociated%252BKinase%252B%252528GAK%252529%26aulast%3DAsquith%26aufirst%3DChristopher%2BR.%2BM.%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D22%26spage%3D4016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Foysal  Ahammad</span>, <span class="hlFld-ContribAuthor ">Tengku Rogayah  Tengku Abd Rashid</span>, <span class="hlFld-ContribAuthor ">Maizan  Mohamed</span>, <span class="hlFld-ContribAuthor ">Suriyea  Tanbin</span>, <span class="hlFld-ContribAuthor ">Fazia Adyani  Ahmad Fuad</span>. </span><span class="cited-content_cbyCitation_article-title">Contemporary Strategies and Current Trends in Designing Antiviral Drugs against Dengue Fever via Targeting Host-Based Approaches. </span><span class="cited-content_cbyCitation_journal-name">Microorganisms</span><span> <strong>2019,</strong> <em>7 </em>
                                    (9)
                                     , 296. <a href="https://doi.org/10.3390/microorganisms7090296" title="DOI URL">https://doi.org/10.3390/microorganisms7090296</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/microorganisms7090296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmicroorganisms7090296%26sid%3Dliteratum%253Aachs%26jtitle%3DMicroorganisms%26atitle%3DContemporary%252BStrategies%252Band%252BCurrent%252BTrends%252Bin%252BDesigning%252BAntiviral%252BDrugs%252Bagainst%252BDengue%252BFever%252Bvia%252BTargeting%252BHost-Based%252BApproaches%26aulast%3DAhammad%26aufirst%3DFoysal%26date%3D2019%26date%3D2019%26volume%3D7%26issue%3D9%26spage%3D296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yusuke  Yabuno</span>, <span class="hlFld-ContribAuthor ">Toshihiro  Uchihashi</span>, <span class="hlFld-ContribAuthor ">Towa  Sasakura</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Shimizu</span>, <span class="hlFld-ContribAuthor ">Yoko  Naito</span>, <span class="hlFld-ContribAuthor ">Kohshiro  Fukushima</span>, <span class="hlFld-ContribAuthor ">Kaori  Ota</span>, <span class="hlFld-ContribAuthor ">Mikihiko  Kogo</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Nojima</span>, <span class="hlFld-ContribAuthor ">Norikazu  Yabuta</span>. </span><span class="cited-content_cbyCitation_article-title">Clathrin heavy chain phosphorylated at T606 plays a role in proper cell division. </span><span class="cited-content_cbyCitation_journal-name">Cell Cycle</span><span> <strong>2019,</strong> <em>18 </em>
                                    (16)
                                     , 1976-1994. <a href="https://doi.org/10.1080/15384101.2019.1637201" title="DOI URL">https://doi.org/10.1080/15384101.2019.1637201</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15384101.2019.1637201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15384101.2019.1637201%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Cycle%26atitle%3DClathrin%252Bheavy%252Bchain%252Bphosphorylated%252Bat%252BT606%252Bplays%252Ba%252Brole%252Bin%252Bproper%252Bcell%252Bdivision%26aulast%3DYabuno%26aufirst%3DYusuke%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D16%26spage%3D1976%26epage%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Satish N.  Dighe</span>, <span class="hlFld-ContribAuthor ">O'mezie  Ekwudu</span>, <span class="hlFld-ContribAuthor ">Kamal  Dua</span>, <span class="hlFld-ContribAuthor ">Dinesh Kumar  Chellappan</span>, <span class="hlFld-ContribAuthor ">Peter L.  Katavic</span>, <span class="hlFld-ContribAuthor ">Trudi A.  Collet</span>. </span><span class="cited-content_cbyCitation_article-title">Recent update on anti-dengue drug discovery. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>176 </em>, 431-455. <a href="https://doi.org/10.1016/j.ejmech.2019.05.010" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.05.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.05.010%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bupdate%252Bon%252Banti-dengue%252Bdrug%252Bdiscovery%26aulast%3DDighe%26aufirst%3DSatish%2BN.%26date%3D2019%26volume%3D176%26spage%3D431%26epage%3D455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher R.M.  Asquith</span>, <span class="hlFld-ContribAuthor ">Daniel K.  Treiber</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>. </span><span class="cited-content_cbyCitation_article-title">Utilizing comprehensive and mini-kinome panels to optimize the selectivity of quinoline inhibitors for cyclin G associated kinase (GAK). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (14)
                                     , 1727-1731. <a href="https://doi.org/10.1016/j.bmcl.2019.05.025" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.05.025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.05.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.05.025%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DUtilizing%252Bcomprehensive%252Band%252Bmini-kinome%252Bpanels%252Bto%252Boptimize%252Bthe%252Bselectivity%252Bof%252Bquinoline%252Binhibitors%252Bfor%252Bcyclin%252BG%252Bassociated%252Bkinase%252B%252528GAK%252529%26aulast%3DAsquith%26aufirst%3DChristopher%2BR.M.%26date%3D2019%26volume%3D29%26issue%3D14%26spage%3D1727%26epage%3D1731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moisés  García-Serradilla</span>, <span class="hlFld-ContribAuthor ">Cristina  Risco</span>, <span class="hlFld-ContribAuthor ">Beatriz  Pacheco</span>. </span><span class="cited-content_cbyCitation_article-title">Drug repurposing for new, efficient, broad spectrum antivirals. </span><span class="cited-content_cbyCitation_journal-name">Virus Research</span><span> <strong>2019,</strong> <em>264 </em>, 22-31. <a href="https://doi.org/10.1016/j.virusres.2019.02.011" title="DOI URL">https://doi.org/10.1016/j.virusres.2019.02.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.virusres.2019.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.virusres.2019.02.011%26sid%3Dliteratum%253Aachs%26jtitle%3DVirus%2520Research%26atitle%3DDrug%252Brepurposing%252Bfor%252Bnew%25252C%252Befficient%25252C%252Bbroad%252Bspectrum%252Bantivirals%26aulast%3DGarc%25C3%25ADa-Serradilla%26aufirst%3DMois%25C3%25A9s%26date%3D2019%26volume%3D264%26spage%3D22%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Junjun  Wang</span>, <span class="hlFld-ContribAuthor ">Xinmiao  Ji</span>, <span class="hlFld-ContribAuthor ">Juanjuan  Liu</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Serine/Threonine Protein Kinase STK16. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (7)
                                     , 1760. <a href="https://doi.org/10.3390/ijms20071760" title="DOI URL">https://doi.org/10.3390/ijms20071760</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20071760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20071760%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DSerine%25252FThreonine%252BProtein%252BKinase%252BSTK16%26aulast%3DWang%26aufirst%3DJunjun%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D7%26spage%3D1760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siew Pheng  Lim</span>. </span><span class="cited-content_cbyCitation_article-title">Dengue drug discovery: Progress, challenges and outlook. </span><span class="cited-content_cbyCitation_journal-name">Antiviral Research</span><span> <strong>2019,</strong> <em>163 </em>, 156-178. <a href="https://doi.org/10.1016/j.antiviral.2018.12.016" title="DOI URL">https://doi.org/10.1016/j.antiviral.2018.12.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.antiviral.2018.12.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.antiviral.2018.12.016%26sid%3Dliteratum%253Aachs%26jtitle%3DAntiviral%2520Research%26atitle%3DDengue%252Bdrug%252Bdiscovery%25253A%252BProgress%25252C%252Bchallenges%252Band%252Boutlook%26aulast%3DLim%26aufirst%3DSiew%2BPheng%26date%3D2019%26volume%3D163%26spage%3D156%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Randy  Wouters</span>, <span class="hlFld-ContribAuthor ">Szu-Yuan  Pu</span>, <span class="hlFld-ContribAuthor ">Mathy  Froeyen</span>, <span class="hlFld-ContribAuthor ">Eveline  Lescrinier</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>, <span class="hlFld-ContribAuthor ">Piet  Herdewijn</span>, <span class="hlFld-ContribAuthor ">Steven  De Jonghe</span>. </span><span class="cited-content_cbyCitation_article-title">Cyclin G-associated kinase (GAK) affinity and antiviral activity studies of a series of 3-C-substituted isothiazolo[4,3-b]pyridines. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>163 </em>, 256-265. <a href="https://doi.org/10.1016/j.ejmech.2018.11.065" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.11.065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.11.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.11.065%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCyclin%252BG-associated%252Bkinase%252B%252528GAK%252529%252Baffinity%252Band%252Bantiviral%252Bactivity%252Bstudies%252Bof%252Ba%252Bseries%252Bof%252B3-C-substituted%252Bisothiazolo%25255B4%25252C3-b%25255Dpyridines%26aulast%3DWouters%26aufirst%3DRandy%26date%3D2019%26volume%3D163%26spage%3D256%26epage%3D265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Randy  Wouters</span>, <span class="hlFld-ContribAuthor ">Junjun  Tian</span>, <span class="hlFld-ContribAuthor ">Piet  Herdewijn</span>, <span class="hlFld-ContribAuthor ">Steven  De Jonghe</span>. </span><span class="cited-content_cbyCitation_article-title">A Scaffold‐Hopping Strategy toward the Identification of Inhibitors of Cyclin G Associated Kinase. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (2)
                                     , 237-254. <a href="https://doi.org/10.1002/cmdc.201800690" title="DOI URL">https://doi.org/10.1002/cmdc.201800690</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800690%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DA%252BScaffold%2525E2%252580%252590Hopping%252BStrategy%252Btoward%252Bthe%252BIdentification%252Bof%252BInhibitors%252Bof%252BCyclin%2525E2%252580%252585G%252BAssociated%252BKinase%26aulast%3DWouters%26aufirst%3DRandy%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D2%26spage%3D237%26epage%3D254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amy W  Lin</span>, <span class="hlFld-ContribAuthor ">Kalbinder K  Gill</span>, <span class="hlFld-ContribAuthor ">Marisol Sampedro  Castañeda</span>, <span class="hlFld-ContribAuthor ">Irene  Matucci</span>, <span class="hlFld-ContribAuthor ">Noreen  Eder</span>, <span class="hlFld-ContribAuthor ">Suzanne  Claxton</span>, <span class="hlFld-ContribAuthor ">Helen  Flynn</span>, <span class="hlFld-ContribAuthor ">Ambrosius P  Snijders</span>, <span class="hlFld-ContribAuthor ">Roger  George</span>, <span class="hlFld-ContribAuthor ">Sila K  Ultanir</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical genetic identification of GAK substrates reveals its role in regulating Na
              +
              /K
              +
              -ATPase. </span><span class="cited-content_cbyCitation_journal-name">Life Science Alliance</span><span> <strong>2018,</strong> <em>1 </em>
                                    (6)
                                     , e201800118. <a href="https://doi.org/10.26508/lsa.201800118" title="DOI URL">https://doi.org/10.26508/lsa.201800118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.26508/lsa.201800118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.26508%2Flsa.201800118%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Science%2520Alliance%26atitle%3DChemical%252Bgenetic%252Bidentification%252Bof%252BGAK%252Bsubstrates%252Breveals%252Bits%252Brole%252Bin%252Bregulating%252BNa%252B%25252B%252B%25252FK%252B%25252B%252B-ATPase%26aulast%3DLin%26aufirst%3DAmy%2BW%26date%3D2018%26date%3D2018%26volume%3D1%26issue%3D6%26spage%3De201800118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naomi  Ohbayashi</span>, <span class="hlFld-ContribAuthor ">Kazutaka  Murayama</span>, <span class="hlFld-ContribAuthor ">Miyuki  Kato‐Murayama</span>, <span class="hlFld-ContribAuthor ">Mutsuko  Kukimoto‐Niino</span>, <span class="hlFld-ContribAuthor ">Tamami  Uejima</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Matsuda</span>, <span class="hlFld-ContribAuthor ">Noboru  Ohsawa</span>, <span class="hlFld-ContribAuthor ">Shigeyuki  Yokoyama</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Nojima</span>, <span class="hlFld-ContribAuthor ">Mikako  Shirouzu</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Basis for the Inhibition of Cyclin G‐Associated Kinase by Gefitinib. </span><span class="cited-content_cbyCitation_journal-name">ChemistryOpen</span><span> <strong>2018,</strong> <em>7 </em>
                                    (9)
                                     , 713-719. <a href="https://doi.org/10.1002/open.201800177" title="DOI URL">https://doi.org/10.1002/open.201800177</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/open.201800177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fopen.201800177%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistryOpen%26atitle%3DStructural%252BBasis%252Bfor%252Bthe%252BInhibition%252Bof%252BCyclin%252BG%2525E2%252580%252590Associated%252BKinase%252Bby%252BGefitinib%26aulast%3DOhbayashi%26aufirst%3DNaomi%26date%3D2018%26date%3D2018%26volume%3D7%26issue%3D9%26spage%3D713%26epage%3D719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Szu-Yuan  Pu</span>, <span class="hlFld-ContribAuthor ">Fei  Xiao</span>, <span class="hlFld-ContribAuthor ">Stanford  Schor</span>, <span class="hlFld-ContribAuthor ">Elena  Bekerman</span>, <span class="hlFld-ContribAuthor ">Fabio  Zanini</span>, <span class="hlFld-ContribAuthor ">Rina  Barouch-Bentov</span>, <span class="hlFld-ContribAuthor ">Claude M.  Nagamine</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>. </span><span class="cited-content_cbyCitation_article-title">Feasibility and biological rationale of repurposing sunitinib and erlotinib for dengue treatment. </span><span class="cited-content_cbyCitation_journal-name">Antiviral Research</span><span> <strong>2018,</strong> <em>155 </em>, 67-75. <a href="https://doi.org/10.1016/j.antiviral.2018.05.001" title="DOI URL">https://doi.org/10.1016/j.antiviral.2018.05.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.antiviral.2018.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.antiviral.2018.05.001%26sid%3Dliteratum%253Aachs%26jtitle%3DAntiviral%2520Research%26atitle%3DFeasibility%252Band%252Bbiological%252Brationale%252Bof%252Brepurposing%252Bsunitinib%252Band%252Berlotinib%252Bfor%252Bdengue%252Btreatment%26aulast%3DPu%26aufirst%3DSzu-Yuan%26date%3D2018%26volume%3D155%26spage%3D67%26epage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Xiao</span>, <span class="hlFld-ContribAuthor ">Stanley  Wang</span>, <span class="hlFld-ContribAuthor ">Rina  Barouch-Bentov</span>, <span class="hlFld-ContribAuthor ">Gregory  Neveu</span>, <span class="hlFld-ContribAuthor ">Szuyuan  Pu</span>, <span class="hlFld-ContribAuthor ">Melanie  Beer</span>, <span class="hlFld-ContribAuthor ">Stanford  Schor</span>, <span class="hlFld-ContribAuthor ">Sathish  Kumar</span>, <span class="hlFld-ContribAuthor ">Vlad  Nicolaescu</span>, <span class="hlFld-ContribAuthor ">Brett D.  Lindenbach</span>, <span class="hlFld-ContribAuthor ">Glenn  Randall</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>, , . </span><span class="cited-content_cbyCitation_article-title">Interactions between the Hepatitis C Virus Nonstructural 2 Protein and Host Adaptor Proteins 1 and 4 Orchestrate Virus Release. </span><span class="cited-content_cbyCitation_journal-name">mBio</span><span> <strong>2018,</strong> <em>9 </em>
                                    (2)
                                     <a href="https://doi.org/10.1128/mBio.02233-17" title="DOI URL">https://doi.org/10.1128/mBio.02233-17</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/mBio.02233-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FmBio.02233-17%26sid%3Dliteratum%253Aachs%26jtitle%3DmBio%26atitle%3DInteractions%252Bbetween%252Bthe%252BHepatitis%252BC%252BVirus%252BNonstructural%252B2%252BProtein%252Band%252BHost%252BAdaptor%252BProteins%252B1%252Band%252B4%252BOrchestrate%252BVirus%252BRelease%26aulast%3DXiao%26aufirst%3DFei%26date%3D2018%26volume%3D9%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher R. M.  Asquith</span>, <span class="hlFld-ContribAuthor ">Tuomo  Laitinen</span>, <span class="hlFld-ContribAuthor ">James M.  Bennett</span>, <span class="hlFld-ContribAuthor ">Paulo H.  Godoi</span>, <span class="hlFld-ContribAuthor ">Michael P.  East</span>, <span class="hlFld-ContribAuthor ">Graham J.  Tizzard</span>, <span class="hlFld-ContribAuthor ">Lee M.  Graves</span>, <span class="hlFld-ContribAuthor ">Gary L.  Johnson</span>, <span class="hlFld-ContribAuthor ">Ronna E.  Dornsife</span>, <span class="hlFld-ContribAuthor ">Carrow I.  Wells</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Elkins</span>, <span class="hlFld-ContribAuthor ">Timothy M.  Willson</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Optimization of 4-Anilinoquinolines as Inhibitors of Cyclin G Associated Kinase. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (1)
                                     , 48-66. <a href="https://doi.org/10.1002/cmdc.201700663" title="DOI URL">https://doi.org/10.1002/cmdc.201700663</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700663%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DIdentification%252Band%252BOptimization%252Bof%252B4-Anilinoquinolines%252Bas%252BInhibitors%252Bof%252BCyclin%2525E2%252580%252585G%252BAssociated%252BKinase%26aulast%3DAsquith%26aufirst%3DChristopher%2BR.%2BM.%26date%3D2018%26date%3D2017%26volume%3D13%26issue%3D1%26spage%3D48%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emilie  Crouchet</span>, <span class="hlFld-ContribAuthor ">Florian  Wrensch</span>, <span class="hlFld-ContribAuthor ">Catherine  Schuster</span>, <span class="hlFld-ContribAuthor ">Mirjam B.  Zeisel</span>, <span class="hlFld-ContribAuthor ">Thomas F.  Baumert</span>. </span><span class="cited-content_cbyCitation_article-title">Host-targeting therapies for hepatitis C virus infection: current developments and future applications. </span><span class="cited-content_cbyCitation_journal-name">Therapeutic Advances in Gastroenterology</span><span> <strong>2018,</strong> <em>11 </em>, 175628481875948. <a href="https://doi.org/10.1177/1756284818759483" title="DOI URL">https://doi.org/10.1177/1756284818759483</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1756284818759483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1756284818759483%26sid%3Dliteratum%253Aachs%26jtitle%3DTherapeutic%2520Advances%2520in%2520Gastroenterology%26atitle%3DHost-targeting%252Btherapies%252Bfor%252Bhepatitis%252BC%252Bvirus%252Binfection%25253A%252Bcurrent%252Bdevelopments%252Band%252Bfuture%252Bapplications%26aulast%3DCrouchet%26aufirst%3DEmilie%26date%3D2018%26date%3D2018%26volume%3D11%26spage%3D175628481875948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miguel O.  Mitchell</span>. </span><span class="cited-content_cbyCitation_article-title">Discovering protein−ligand chalcogen bonding in the protein data bank using endocyclic sulfur-containing heterocycles as ligand search subsets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2017,</strong> <em>23 </em>
                                    (10)
                                     <a href="https://doi.org/10.1007/s00894-017-3452-3" title="DOI URL">https://doi.org/10.1007/s00894-017-3452-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-017-3452-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-017-3452-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DDiscovering%252Bprotein%2525E2%252588%252592ligand%252Bchalcogen%252Bbonding%252Bin%252Bthe%252Bprotein%252Bdata%252Bbank%252Busing%252Bendocyclic%252Bsulfur-containing%252Bheterocycles%252Bas%252Bligand%252Bsearch%252Bsubsets%26aulast%3DMitchell%26aufirst%3DMiguel%2BO.%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saoirse O  Dolly</span>, <span class="hlFld-ContribAuthor ">Mark D  Gurden</span>, <span class="hlFld-ContribAuthor ">Konstantinos  Drosopoulos</span>, <span class="hlFld-ContribAuthor ">Paul  Clarke</span>, <span class="hlFld-ContribAuthor ">Johann  de Bono</span>, <span class="hlFld-ContribAuthor ">Stan  Kaye</span>, <span class="hlFld-ContribAuthor ">Paul  Workman</span>, <span class="hlFld-ContribAuthor ">Spiros  Linardopoulos</span>. </span><span class="cited-content_cbyCitation_article-title">RNAi screen reveals synthetic lethality between cyclin G-associated kinase and FBXW7 by inducing aberrant mitoses. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Cancer</span><span> <strong>2017,</strong> <em>117 </em>
                                    (7)
                                     , 954-964. <a href="https://doi.org/10.1038/bjc.2017.277" title="DOI URL">https://doi.org/10.1038/bjc.2017.277</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/bjc.2017.277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fbjc.2017.277%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Cancer%26atitle%3DRNAi%252Bscreen%252Breveals%252Bsynthetic%252Blethality%252Bbetween%252Bcyclin%252BG-associated%252Bkinase%252Band%252BFBXW7%252Bby%252Binducing%252Baberrant%252Bmitoses%26aulast%3DDolly%26aufirst%3DSaoirse%2BO%26date%3D2017%26date%3D2017%26volume%3D117%26issue%3D7%26spage%3D954%26epage%3D964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangdi  Li</span>, <span class="hlFld-ContribAuthor ">Erik  De Clercq</span>. </span><span class="cited-content_cbyCitation_article-title">Current therapy for chronic hepatitis C: The role of direct-acting antivirals. </span><span class="cited-content_cbyCitation_journal-name">Antiviral Research</span><span> <strong>2017,</strong> <em>142 </em>, 83-122. <a href="https://doi.org/10.1016/j.antiviral.2017.02.014" title="DOI URL">https://doi.org/10.1016/j.antiviral.2017.02.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.antiviral.2017.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.antiviral.2017.02.014%26sid%3Dliteratum%253Aachs%26jtitle%3DAntiviral%2520Research%26atitle%3DCurrent%252Btherapy%252Bfor%252Bchronic%252Bhepatitis%252BC%25253A%252BThe%252Brole%252Bof%252Bdirect-acting%252Bantivirals%26aulast%3DLi%26aufirst%3DGuangdi%26date%3D2017%26volume%3D142%26spage%3D83%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena  Bekerman</span>, <span class="hlFld-ContribAuthor ">Gregory  Neveu</span>, <span class="hlFld-ContribAuthor ">Ana  Shulla</span>, <span class="hlFld-ContribAuthor ">Jennifer  Brannan</span>, <span class="hlFld-ContribAuthor ">Szu-Yuan  Pu</span>, <span class="hlFld-ContribAuthor ">Stanley  Wang</span>, <span class="hlFld-ContribAuthor ">Fei  Xiao</span>, <span class="hlFld-ContribAuthor ">Rina  Barouch-Bentov</span>, <span class="hlFld-ContribAuthor ">Russell R.  Bakken</span>, <span class="hlFld-ContribAuthor ">Roberto  Mateo</span>, <span class="hlFld-ContribAuthor ">Jennifer  Govero</span>, <span class="hlFld-ContribAuthor ">Claude M.  Nagamine</span>, <span class="hlFld-ContribAuthor ">Michael S.  Diamond</span>, <span class="hlFld-ContribAuthor ">Steven  De Jonghe</span>, <span class="hlFld-ContribAuthor ">Piet  Herdewijn</span>, <span class="hlFld-ContribAuthor ">John M.  Dye</span>, <span class="hlFld-ContribAuthor ">Glenn  Randall</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer kinase inhibitors impair intracellular viral trafficking and exert broad-spectrum antiviral effects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Clinical Investigation</span><span> <strong>2017,</strong> <em>127 </em>
                                    (4)
                                     , 1338-1352. <a href="https://doi.org/10.1172/JCI89857" title="DOI URL">https://doi.org/10.1172/JCI89857</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1172/JCI89857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1172%2FJCI89857%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Clinical%2520Investigation%26atitle%3DAnticancer%252Bkinase%252Binhibitors%252Bimpair%252Bintracellular%252Bviral%252Btrafficking%252Band%252Bexert%252Bbroad-spectrum%252Bantiviral%252Beffects%26aulast%3DBekerman%26aufirst%3DElena%26date%3D2017%26volume%3D127%26issue%3D4%26spage%3D1338%26epage%3D1352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hermann AM  Mucke</span>. </span><span class="cited-content_cbyCitation_article-title">Patent highlights: December 2015–January 2016. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Patent Analyst</span><span> <strong>2016,</strong> <em>5 </em>
                                    (3)
                                     , 147-153. <a href="https://doi.org/10.4155/ppa-2016-0009" title="DOI URL">https://doi.org/10.4155/ppa-2016-0009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/ppa-2016-0009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Fppa-2016-0009%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Patent%2520Analyst%26atitle%3DPatent%252Bhighlights%25253A%252BDecember%252B2015%2525E2%252580%252593January%252B2016%26aulast%3DMucke%26aufirst%3DHermann%2BAM%26date%3D2016%26volume%3D5%26issue%3D3%26spage%3D147%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mirjam  Zeisel</span>, <span class="hlFld-ContribAuthor ">Emilie  Crouchet</span>, <span class="hlFld-ContribAuthor ">Thomas  Baumert</span>, <span class="hlFld-ContribAuthor ">Catherine  Schuster</span>. </span><span class="cited-content_cbyCitation_article-title">Host-Targeting Agents to Prevent and Cure Hepatitis C Virus Infection. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2015,</strong> <em>7 </em>
                                    (11)
                                     , 5659-5685. <a href="https://doi.org/10.3390/v7112898" title="DOI URL">https://doi.org/10.3390/v7112898</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v7112898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv7112898%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DHost-Targeting%252BAgents%252Bto%252BPrevent%252Band%252BCure%252BHepatitis%252BC%252BVirus%252BInfection%26aulast%3DZeisel%26aufirst%3DMirjam%26date%3D2015%26date%3D2015%26volume%3D7%26issue%3D11%26spage%3D5659%26epage%3D5685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiahong  Li</span>, <span class="hlFld-ContribAuthor ">Sona  Kovackova</span>, <span class="hlFld-ContribAuthor ">Szuyuan  Pu</span>, <span class="hlFld-ContribAuthor ">Jef  Rozenski</span>, <span class="hlFld-ContribAuthor ">Steven  De Jonghe</span>, <span class="hlFld-ContribAuthor ">Shirit  Einav</span>, <span class="hlFld-ContribAuthor ">Piet  Herdewijn</span>. </span><span class="cited-content_cbyCitation_article-title">Isothiazolo[4,3-b]pyridines as inhibitors of cyclin G associated kinase: synthesis, structure–activity relationship studies and antiviral activity. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (9)
                                     , 1666-1672. <a href="https://doi.org/10.1039/C5MD00229J" title="DOI URL">https://doi.org/10.1039/C5MD00229J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5MD00229J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5MD00229J%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DIsothiazolo%25255B4%25252C3-b%25255Dpyridines%252Bas%252Binhibitors%252Bof%252Bcyclin%252BG%252Bassociated%252Bkinase%25253A%252Bsynthesis%25252C%252Bstructure%2525E2%252580%252593activity%252Brelationship%252Bstudies%252Band%252Bantiviral%252Bactivity%26aulast%3DLi%26aufirst%3DJiahong%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D9%26spage%3D1666%26epage%3D1672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0013.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Known GAK Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0014.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Hit compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0005.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Isothiazolo[4,3-<i>b</i>]pyridine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Fe, CH<sub>3</sub>COOH, 0 °C to rt; (b) P<sub>2</sub>S<sub>5</sub>, EtOH, 75 °C; (c) 30% aq H<sub>2</sub>O<sub>2</sub>, CH<sub>3</sub>OH, rt; (d) R<sub>6</sub>B(OH)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, Pd(dppf)Cl<sub>2</sub>, dioxane/water, 100 °C; (e) CuBr, HBr, NaNO<sub>2</sub>,H<sub>2</sub>O, 0 °C to rt; (f) R<sub>3</sub>H, EtOH, 75 °C; (g) R<sub>6</sub>B(OH)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, Pd(dppf)Cl<sub>2</sub>, dioxane/water, 100 °C or R<sub>6</sub>B(OH)<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, H<sub>2</sub>O, DME.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Specificity profile of compound <b>12g</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of GAK in complex with <b>12i</b>. (A) Overview of GAK:NbGAK_4 complex with inhibitor <b>12i</b> bound. The inset shows |<i>F</i><sub>O</sub>| – |<i>F</i><sub>C</sub>| omitted map contoured at 3σ for the bound inhibitor. (B) Detailed interactions of <b>12i</b> within the GAK ATP binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Selective GAK inhibitors inhibit HCV infection and AP2M1 phosphorylation: Dose–response curves of <b>12g</b> and <b>12i</b> effects on infection of Huh-7.5 cells with cell culture grown HCV (HCVcc) (A), entry of pseudoparticles of HCV (HCVpp) (B), HCV RNA replication 72 h postelectroporation with J6/JFH(p7-Rluc2A) RNA (C), and intra- (D) and extracellular (E) infectivity in naive cells inoculated with cell lysates or supernatants derived from the electroporated cells, respectively. Plotted in black (left <i>y</i>-axes) are relative luciferase values normalized to DMSO treated controls. Corresponding cellular viability, as measured by alamarBlue-based assays, are plotted in gray (right <i>y</i>-axes). Data reflect mean values and sd (error bars). (F) Effect of the inhibitors on AP2M1 phosphorylation by Western analysis in lysates derived from Huh-7.5 cells. A representative membrane blotted with antiphospho-AP2M1 (p-AP2M1) and anti-AP2M1 antibodies is shown. Numbers represent relative p-AP2M1/total AP2M1 protein ratio normalized to DMSO controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/medium/jm-2014-01759m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proposed model for the mechanism by which the selective GAK inhibitors, <b>12g</b> and <b>12i</b>, inhibit HCV infection. By directly inhibiting GAK kinase activity, <b>12g</b> and <b>12i</b> interfere with clathrin/AP-2-mediated endocytosis of HCV (step 1) as well as AP-2 binding to core in HCV assembly (step 5).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-8/jm501759m/20150417/images/large/jm-2014-01759m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501759m&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i105">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47103" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47103" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 37 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kanaoka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimura, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okazaki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nojima, H.</span><span> </span><span class="NLM_article-title">GAK: a cyclin G associated kinase contains a tensin/auxilin-like domain</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">402</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=402&publication_year=1997&pages=73-80&author=Y.+Kanaokaauthor=S.+H.+Kimuraauthor=I.+Okazakiauthor=M.+Ikedaauthor=H.+Nojima&title=GAK%3A+a+cyclin+G+associated+kinase+contains+a+tensin%2Fauxilin-like+domain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKanaoka%26aufirst%3DY.%26aulast%3DKimura%26aufirst%3DS.%2BH.%26aulast%3DOkazaki%26aufirst%3DI.%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DNojima%26aufirst%3DH.%26atitle%3DGAK%253A%2520a%2520cyclin%2520G%2520associated%2520kinase%2520contains%2520a%2520tensin%252Fauxilin-like%2520domain%26jtitle%3DFEBS%2520Lett.%26date%3D1997%26volume%3D402%26spage%3D73%26epage%3D80" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">–</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lhi7nkqNUA5nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Lee, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, L. E.</span><span> </span><span class="NLM_article-title">Depletion of GAK/auxilin 2 inhibits receptor-mediated endocytosis and recruitment of both clathrin and clathrin adaptors</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">4311</span><span class="NLM_x">–</span> <span class="NLM_lpage">4321</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1242%2Fjcs.02548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=16155256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFCgt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2005&pages=4311-4321&author=D.+W.+Leeauthor=X.+Zhaoauthor=F.+Zhangauthor=E.+Eisenbergauthor=L.+E.+Greene&title=Depletion+of+GAK%2Fauxilin+2+inhibits+receptor-mediated+endocytosis+and+recruitment+of+both+clathrin+and+clathrin+adaptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of GAK/auxilin 2 inhibits receptor-mediated endocytosis and recruitment of both clathrin and clathrin adaptors</span></div><div class="casAuthors">Lee, Dong-won; Zhao, Xiaohong; Zhang, Fang; Eisenberg, Evan; Greene, Lois E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4311-4321</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Cyclin G-assocd. kinase (GAK/auxilin 2), the ubiquitous form of the neuronal-specific protein auxilin 1, is an essential cofactor for the Hsc70-dependent uncoating of clathrin-coated vesicles.  We have now investigated the effect of knocking down GAK in HeLa cells by vector-based small hairpin RNA.  Functionally, depletion of GAK caused a marked decrease in internalization of both transferrin and epidermal growth factor and altered mannose 6-phosphate receptor trafficking, but had little effect on the recycling of transferrin receptor back to the plasma membrane.  Structurally, depletion of GAK caused a marked redn. in perinuclear clathrin assocd. with the trans-Golgi network and in the no. of clathrin-coated pits on the plasma membrane, and reduced clathrin exchange on the few clathrin-coated pits that remained.  Surprisingly, while clathrin depletion does not prevent adaptors from assembling on the membrane, depletion of GAK caused a dramatic redn. in AP2 and epsin on the plasma membrane and AP1 and GGA at the trans-Golgi network.  A similar effect was caused by expression of a dominant neg. Hsp70 mutant.  These results suggest that GAK, in conjunction with Hsc70, not only uncoats clathrin-coated vesicles and induces clathrin exchange on clathrin-coated pits, but also mediates binding of clathrin and adaptors to the plasma membrane and the trans-Golgi network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8IvRfVSaZT7Vg90H21EOLACvtfcHk0liRRWpZqb93Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFCgt7fK&md5=0172190491cf9b908e705e434b763d60</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1242%2Fjcs.02548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.02548%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%2BW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DEisenberg%26aufirst%3DE.%26aulast%3DGreene%26aufirst%3DL.%2BE.%26atitle%3DDepletion%2520of%2520GAK%252Fauxilin%25202%2520inhibits%2520receptor-mediated%2520endocytosis%2520and%2520recruitment%2520of%2520both%2520clathrin%2520and%2520clathrin%2520adaptors%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2005%26volume%3D118%26spage%3D4311%26epage%3D4321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Korolchuk, V. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banting, G.</span><span> </span><span class="NLM_article-title">CK2 and GAK/auxilin2 are major protein kinases in clathrin-coated vesicles</span> <span class="citation_source-journal">Traffic</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">428</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=428-439&author=V.+I.+Korolchukauthor=G.+Banting&title=CK2+and+GAK%2Fauxilin2+are+major+protein+kinases+in+clathrin-coated+vesicles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKorolchuk%26aufirst%3DV.%2BI.%26aulast%3DBanting%26aufirst%3DG.%26atitle%3DCK2%2520and%2520GAK%252Fauxilin2%2520are%2520major%2520protein%2520kinases%2520in%2520clathrin-coated%2520vesicles%26jtitle%3DTraffic%26date%3D2002%26volume%3D3%26spage%3D428%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Zhang, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engqvist-Goldstein, Å.E.Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carreno, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smythe, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drubin, D. G.</span><span> </span><span class="NLM_article-title">Multiple roles for cyclin G-associated kinase in clathrin-mediated sorting events</span> <span class="citation_source-journal">Traffic</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1103</span><span class="NLM_x">–</span> <span class="NLM_lpage">1113</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=1103-1113&author=C.+X.+Zhangauthor=%C3%85.E.Y.+Engqvist-Goldsteinauthor=S.+Carrenoauthor=D.+J.+Owenauthor=E.+Smytheauthor=D.+G.+Drubin&title=Multiple+roles+for+cyclin+G-associated+kinase+in+clathrin-mediated+sorting+events"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%2BX.%26aulast%3DEngqvist-Goldstein%26aufirst%3D%25C3%2585.E.Y.%26aulast%3DCarreno%26aufirst%3DS.%26aulast%3DOwen%26aufirst%3DD.%2BJ.%26aulast%3DSmythe%26aufirst%3DE.%26aulast%3DDrubin%26aufirst%3DD.%2BG.%26atitle%3DMultiple%2520roles%2520for%2520cyclin%2520G-associated%2520kinase%2520in%2520clathrin-mediated%2520sorting%2520events%26jtitle%3DTraffic%26date%3D2005%26volume%3D6%26spage%3D1103%26epage%3D1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greener, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Hasani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cushman, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, L. E.</span><span> </span><span class="NLM_article-title">Expression of auxilin or AP180 inhibits endocytosis by mislocalizing clathrin: evidence for formation of nascent pits containing AP1 or AP2 but not clathrin</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">365</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=11148137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD3MXht1Kgur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2001&pages=353-365&author=X.+Zhaoauthor=T.+Greenerauthor=H.+Al-Hasaniauthor=S.+W.+Cushmanauthor=E.+Eisenbergauthor=L.+E.+Greene&title=Expression+of+auxilin+or+AP180+inhibits+endocytosis+by+mislocalizing+clathrin%3A+evidence+for+formation+of+nascent+pits+containing+AP1+or+AP2+but+not+clathrin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of auxilin or AP180 inhibits endocytosis by mislocalizing clathrin: evidence for formation of nascent pits containing AP1 or AP2 but not clathrin</span></div><div class="casAuthors">Zhao, Xiaohong; Greener, Tsvika; Al-Hasani, Hadi; Cushman, Samuel W.; Eisenberg, Evan; Greene, Lois E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">353-365</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Although uncoating of clathrin-coated vesicles is a key event in clathrin-mediated endocytosis it is unclear what prevents uncoating of clathrin-coated pits before they pinch off to become clathrin-coated vesicles.  We have shown that the J-domain proteins auxilin and GAK are required for uncoating by Hsc70 in vitro.  In the present study, we expressed auxilin in cultured cells to det. if this would block endocytosis by causing premature uncoating of clathrin-coated pits.  We found that expression of auxilin indeed inhibited endocytosis.  However, expression of auxilin with its J-domain mutated so that it no longer interacted with Hsc70 also inhibited endocytosis as did expression of the clathrin-assembly protein, AP180, or its clathrin-binding domain.  Accompanying this inhibition, we obsd. a marked decrease in clathrin assocd. with the plasma membrane and the trans-Golgi network, which provided us with an opportunity to det. whether the absence of clathrin from clathrin-coated pits affected the distribution of the clathrin assembly proteins AP1 and AP2.  Surprisingly we found almost no change in the assocn. of AP2 and AP1 with the plasma membrane and the trans-Golgi network, resp.  This was particularly obvious when auxilin or GAK was expressed with functional J-domains since, in these cases, almost all of the clathrin was sequestered in granules that also contained Hsc70 and auxilin or GAK.  We conclude that expression of clathrin-binding proteins inhibits clathrin-mediated endocytosis by sequestering clathrin so that it is no longer available to bind to nascent pits but that assembly proteins bind to these pits independently of clathrin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcyfXFVf8SQ7Vg90H21EOLACvtfcHk0liRRWpZqb93Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXht1Kgur0%253D&md5=1b52211f83e64142ec4a4d470f3abdb0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DGreener%26aufirst%3DT.%26aulast%3DAl-Hasani%26aufirst%3DH.%26aulast%3DCushman%26aufirst%3DS.%2BW.%26aulast%3DEisenberg%26aufirst%3DE.%26aulast%3DGreene%26aufirst%3DL.%2BE.%26atitle%3DExpression%2520of%2520auxilin%2520or%2520AP180%2520inhibits%2520endocytosis%2520by%2520mislocalizing%2520clathrin%253A%2520evidence%2520for%2520formation%2520of%2520nascent%2520pits%2520containing%2520AP1%2520or%2520AP2%2520but%2520not%2520clathrin%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2001%26volume%3D114%26spage%3D353%26epage%3D365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gjoerup, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, T. M.</span><span> </span><span class="NLM_article-title">The serine threonine kinase cyclin G-associated kinase regulates epidermal growth factor receptor signaling</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">10296</span><span class="NLM_x">–</span> <span class="NLM_lpage">10301</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=10296-10301&author=L.+Zhangauthor=O.+Gjoerupauthor=T.+M.+Roberts&title=The+serine+threonine+kinase+cyclin+G-associated+kinase+regulates+epidermal+growth+factor+receptor+signaling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DGjoerup%26aufirst%3DO.%26aulast%3DRoberts%26aufirst%3DT.%2BM.%26atitle%3DThe%2520serine%2520threonine%2520kinase%2520cyclin%2520G-associated%2520kinase%2520regulates%2520epidermal%2520growth%2520factor%2520receptor%2520signaling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D10296%26epage%3D10301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Neveu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barouch-Bentov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziv-Av, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerber, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einav, S.</span><span> </span><span class="NLM_article-title">Identification and targeting of an interaction between a tyrosine motif within hepatitis C virus core protein and AP2M1 essential for viral assembly</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002845</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=e1002845&issue=8&author=G.+Neveuauthor=R.+Barouch-Bentovauthor=A.+Ziv-Avauthor=D.+Gerberauthor=Y.+Jacobauthor=S.+Einav&title=Identification+and+targeting+of+an+interaction+between+a+tyrosine+motif+within+hepatitis+C+virus+core+protein+and+AP2M1+essential+for+viral+assembly"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNeveu%26aufirst%3DG.%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DZiv-Av%26aufirst%3DA.%26aulast%3DGerber%26aufirst%3DD.%26aulast%3DJacob%26aufirst%3DY.%26aulast%3DEinav%26aufirst%3DS.%26atitle%3DIdentification%2520and%2520targeting%2520of%2520an%2520interaction%2520between%2520a%2520tyrosine%2520motif%2520within%2520hepatitis%2520C%2520virus%2520core%2520protein%2520and%2520AP2M1%2520essential%2520for%2520viral%2520assembly%26jtitle%3DPLoS%2520Pathog.%26date%3D2012%26volume%3D8%26issue%3D8%26spage%3De1002845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0livayKEzXp81A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span> SuperNova Life Science. Human Kinome Heat Map. <a href="http://www.supernovalifescience.com/HM/HM%2041.pdf" class="extLink">http://www.supernovalifescience.com/HM/HM%2041.pdf</a><span class="NLM_x"> (</span><span class="NLM_year">2008</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SuperNova+Life+Science.+Human+Kinome+Heat+Map.+http%3A%2F%2Fwww.supernovalifescience.com%2FHM%2FHM%252041.pdf+%282008%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Neveu, G.; Ziv-Av, A.; Barouch-Bentov, R.; Bekerman, E.; Mulholland, J.; Einav, S.</span><span> </span><span class="NLM_article-title">AAK1 and GAK regulate hepatitis C virus entry and are potential drug targets</span>.  <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1128/JVI.02705-14</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1128%2FJVI.02705-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=25653444" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=G.+Neveu&author=A.+Ziv-Av&author=R.+Barouch-Bentov&author=E.+Bekerman&author=J.+Mulholland&author=S.+Einav&title=AAK1+and+GAK+regulate+hepatitis+C+virus+entry+and+are+potential+drug+targets&doi=10.1128%2FJVI.02705-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FJVI.02705-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02705-14%26sid%3Dliteratum%253Aachs%26aulast%3DNeveu%26aufirst%3DG.%26atitle%3DAAK1%2520and%2520GAK%2520regulate%2520hepatitis%2520C%2520virus%2520entry%2520and%2520are%2520potential%2520drug%2520targets%26jtitle%3DJ.%2520Virol.%26date%3D2015%26doi%3D10.1128%2FJVI.02705-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W.H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lélias, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+P.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0livayKEzXp81A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.H.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BP.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Bellei, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitisci, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Migliano, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardinali, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picardo, M.</span><span> </span><span class="NLM_article-title">Pyridinyl imidazole compounds interfere with melanosomes sorting through the inhibition of cyclin G-associated kinase, a regulator of cathepsins maturation</span> <span class="citation_source-journal">Cell. Signalling</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">716</span><span class="NLM_x">–</span> <span class="NLM_lpage">723</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1016%2Fj.cellsig.2013.12.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=24412755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=716-723&author=B.+Belleiauthor=A.+Pitisciauthor=E.+Miglianoauthor=G.+Cardinaliauthor=M.+Picardo&title=Pyridinyl+imidazole+compounds+interfere+with+melanosomes+sorting+through+the+inhibition+of+cyclin+G-associated+kinase%2C+a+regulator+of+cathepsins+maturation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridinyl imidazole compounds interfere with melanosomes sorting through the inhibition of Cyclin G-associated Kinase, a regulator of cathepsins maturation</span></div><div class="casAuthors">Bellei, Barbara; Pitisci, Angela; Migliano, Emilia; Cardinali, Giorgia; Picardo, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">716-723</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transfer of melanin-contg. melanosomes from melanocytes to neighboring keratinocytes results in skin pigmentation.  Pharmacol. modulation of melanosomal transfer has recently gained much attention as a strategy for modifying normal or abnormal pigmentation.  In this study, while investigating the impact of pyridinyl imidazole (PI) compds., a class of p38 MAPK inhibitors, on melanocyte differentiation we obsd. that some, but not all PIs interfere with the physiol. melanosome sorting producing a strong retention of melanin in the intracellular compartment assocd. with a general redn. of melanin synthesis.  Electron microscopy studies illustrated an accumulation of melanosomes inside melanocytes with enrichment in immature melanosome at stages II and III at the end of dendrites.  We identified cyclin G-assocd. kinase GAK, a protein expressed ubiquitously in various tissues, as the off-target responsible of intracellular melanin accumulation and we report evidence that reduced GAK-dependent cathepsin maturation is implicated in melanosome sorting deficiency.  The co-regulation of GAK and cathepsin B and L expression with the melanogenic biosynthetic pathway in normal human melanocytes as well as in B16-F0 melanoma cells strengthen the idea that these proteins represent new possible targets for prevention and treatment of irregular pigmentation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLyREgMwFFirVg90H21EOLACvtfcHk0lhS5r-GtVNYPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGls7s%253D&md5=b373f2cbfde96773364849b878219e2d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2013.12.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2013.12.023%26sid%3Dliteratum%253Aachs%26aulast%3DBellei%26aufirst%3DB.%26aulast%3DPitisci%26aufirst%3DA.%26aulast%3DMigliano%26aufirst%3DE.%26aulast%3DCardinali%26aufirst%3DG.%26aulast%3DPicardo%26aufirst%3DM.%26atitle%3DPyridinyl%2520imidazole%2520compounds%2520interfere%2520with%2520melanosomes%2520sorting%2520through%2520the%2520inhibition%2520of%2520cyclin%2520G-associated%2520kinase%252C%2520a%2520regulator%2520of%2520cathepsins%2520maturation%26jtitle%3DCell.%2520Signalling%26date%3D2014%26volume%3D26%26spage%3D716%26epage%3D723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Meiby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elkins, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohlson, S.</span><span> </span><span class="NLM_article-title">Fragment screening of cyclin G-associated kinase by weak affinity chromatography</span> <span class="citation_source-journal">Anal. Bioanal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">404</span><span class="NLM_x">, </span> <span class="NLM_fpage">2417</span><span class="NLM_x">–</span> <span class="NLM_lpage">2425</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=22918538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Cgtb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=404&publication_year=2012&pages=2417-2425&author=E.+Meibyauthor=S.+Knappauthor=J.+M.+Elkinsauthor=S.+Ohlson&title=Fragment+screening+of+cyclin+G-associated+kinase+by+weak+affinity+chromatography"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment screening of cyclin G-associated kinase by weak affinity chromatography</span></div><div class="casAuthors">Meiby, Elinor; Knapp, Stefan; Elkins, Jonathan M.; Ohlson, Sten</div><div class="citationInfo"><span class="NLM_cas:title">Analytical and Bioanalytical Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">404</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2417-2425</span>CODEN:
                <span class="NLM_cas:coden">ABCNBP</span>;
        ISSN:<span class="NLM_cas:issn">1618-2642</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) has become a new strategy for drug discovery where lead compds. are evolved from small mols.  These fragments form low affinity interactions (dissocn. const. (KD) = mM - μM) with protein targets, which require fragment screening methods of sufficient sensitivity.  Weak affinity chromatog. (WAC) is a promising new technol. for fragment screening based on selective retention of fragments by a drug target.  Kinases are a major pharmaceutical target, and FBDD has been successfully applied to several of these targets.  In this work, we have demonstrated the potential to use WAC in combination with mass spectrometry (MS) detection for fragment screening of a kinase target-cyclin G-assocd. kinase (GAK).  One hundred seventy fragments were selected for WAC screening by virtual screening of a com. fragment library against the ATP-binding site of five different proteins.  GAK protein was immobilized on a capillary HPLC column, and compd. binding was characterized by frontal affinity chromatog.  Compds. were screened in sets of 13 or 14, in combination with MS detection for enhanced throughput.  Seventy-eight fragments (46 %) with KD < 200 μM were detected, including a few highly efficient GAK binders (KD of 2 μM; ligand efficiency = 0.51).  Of special interest is that chiral screening by WAC may be possible, as two stereoisomeric fragments, which both contained one chiral center, demonstrated twin peaks.  This ability, in combination with the robustness, sensitivity, and simplicity of WAC makes it a new method for fragment screening of considerable potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeCQXELkNLdLVg90H21EOLACvtfcHk0lhS5r-GtVNYPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Cgtb7N&md5=b990b75b1d9cc5c4149ef1a81f199ee4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMeiby%26aufirst%3DE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DOhlson%26aufirst%3DS.%26atitle%3DFragment%2520screening%2520of%2520cyclin%2520G-associated%2520kinase%2520by%2520weak%2520affinity%2520chromatography%26jtitle%3DAnal.%2520Bioanal.%2520Chem.%26date%3D2012%26volume%3D404%26spage%3D2417%26epage%3D2425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Sakurai, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuzaki, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naito, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasakura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tabara, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabuta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nojima, H.</span><span> </span><span class="NLM_article-title">Gefitinib and luteolin cause growth arrest of human prostate cancer PC-3 cells via inhibition of cyclin G-associated kinase and induction of miR-630</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">e100124</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e100124&issue=6&author=M.+A.+Sakuraiauthor=Y.+Ozakiauthor=D.+Okuzakiauthor=Y.+Naitoauthor=T.+Sasakuraauthor=A.+Okamotoauthor=H.+Tabaraauthor=T.+Inoueauthor=M.+Hagiyamaauthor=A.+Itoauthor=N.+Yabutaauthor=H.+Nojima&title=Gefitinib+and+luteolin+cause+growth+arrest+of+human+prostate+cancer+PC-3+cells+via+inhibition+of+cyclin+G-associated+kinase+and+induction+of+miR-630"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSakurai%26aufirst%3DM.%2BA.%26aulast%3DOzaki%26aufirst%3DY.%26aulast%3DOkuzaki%26aufirst%3DD.%26aulast%3DNaito%26aufirst%3DY.%26aulast%3DSasakura%26aufirst%3DT.%26aulast%3DOkamoto%26aufirst%3DA.%26aulast%3DTabara%26aufirst%3DH.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DHagiyama%26aufirst%3DM.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DYabuta%26aufirst%3DN.%26aulast%3DNojima%26aufirst%3DH.%26atitle%3DGefitinib%2520and%2520luteolin%2520cause%2520growth%2520arrest%2520of%2520human%2520prostate%2520cancer%2520PC-3%2520cells%2520via%2520inhibition%2520of%2520cyclin%2520G-associated%2520kinase%2520and%2520induction%2520of%2520miR-630%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D6%26spage%3De100124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Susa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwab, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornicek, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mankin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, Z.</span><span> </span><span class="NLM_article-title">Cyclin G-associated kinase is necessary for osteosarcoma cell proliferation and receptor trafficking</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3342</span><span class="NLM_x">–</span> <span class="NLM_lpage">3350</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1158%2F1535-7163.MCT-10-0637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=20881269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=3342-3350&author=M.+Susaauthor=E.+Choyauthor=X.+Liuauthor=J.+Schwabauthor=F.+J.+Hornicekauthor=H.+Mankinauthor=Z.+Duan&title=Cyclin+G-associated+kinase+is+necessary+for+osteosarcoma+cell+proliferation+and+receptor+trafficking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin G-Associated Kinase Is Necessary for Osteosarcoma Cell Proliferation and Receptor Trafficking</span></div><div class="casAuthors">Susa, Michiro; Choy, Edwin; Liu, Xianzhe; Schwab, Joseph; Hornicek, Francis J.; Mankin, Henry; Duan, Zhenfeng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3342-3350</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Osteosarcoma is the most frequent primary malignant bone tumor among the children.  The advent of neoadjuvant chemotherapy significantly improved the prognosis of patients with osteosarcoma in the 1980s, but it has since plateaued in the past decades.  Recently, one of the most researched areas in sarcoma treatment is tyrosine kinases.  Here, we describe research on a serine/threonine kinase, cyclin G-assocd. kinase (GAK), which has not been reported in osteosarcoma previously.  In this study, a lentiviral based human shRNA library was utilized to screen for kinases in KHOS and U-2OS osteosarcoma cells.  The expression of GAK was examd. in osteosarcoma and the effect on cell proliferation was analyzed by GAK siRNA knockdown.  The level of GAK expression and its correlation to prognosis was analyzed in osteosarcoma tissue microarray.  The effect of GAK depletion on insulin-like growth factor and epidermal growth factor receptor-mediated signal transduction was analyzed by Western blot.  We obsd. that GAK was overexpressed in both osteosarcoma cell lines and tissue samples when compared with human osteoblasts.  GAK knockdown by siRNA decreased cell proliferation in both drug-sensitive and multidrug-resistant osteosarcoma cell lines.  Immunohistochem. of osteosarcoma tissue microarray revealed that overexpression of GAK was assocd. with poor prognosis.  Finally, knockdown of GAK resulted in alterations of receptor trafficking and several downstream proteins.  In conclusion, our results suggest that osteosarcoma cell proliferation and survival are dependent on GAK.  These findings may lead to the development of new therapeutic options for osteosarcoma.  Mol Cancer Ther; 9(12); 3342-50.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaz8xXEuEtOLVg90H21EOLACvtfcHk0lhf9sLzGaYLrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsLrL&md5=6a087e9b0ca52770617d59967b190df2</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-10-0637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-10-0637%26sid%3Dliteratum%253Aachs%26aulast%3DSusa%26aufirst%3DM.%26aulast%3DChoy%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSchwab%26aufirst%3DJ.%26aulast%3DHornicek%26aufirst%3DF.%2BJ.%26aulast%3DMankin%26aufirst%3DH.%26aulast%3DDuan%26aufirst%3DZ.%26atitle%3DCyclin%2520G-associated%2520kinase%2520is%2520necessary%2520for%2520osteosarcoma%2520cell%2520proliferation%2520and%2520receptor%2520trafficking%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D3342%26epage%3D3350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Dzamko, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halliday, G. M.</span><span> </span><span class="NLM_article-title">Parkinson’s disease-implicated kinases in the brain; insights into disease pathogenesis</span> <span class="citation_source-journal">Front. Mol. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">–</span> <span class="NLM_lpage">57</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=7-57&author=N.+Dzamkoauthor=J.+Zhouauthor=Y.+Huangauthor=G.+M.+Halliday&title=Parkinson%E2%80%99s+disease-implicated+kinases+in+the+brain%3B+insights+into+disease+pathogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDzamko%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DHalliday%26aufirst%3DG.%2BM.%26atitle%3DParkinson%25E2%2580%2599s%2520disease-implicated%2520kinases%2520in%2520the%2520brain%253B%2520insights%2520into%2520disease%2520pathogenesis%26jtitle%3DFront.%2520Mol.%2520Neurosci.%26date%3D2014%26volume%3D24%26spage%3D7%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Kuduk, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Marco, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ray, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mab, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seager, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeplinger, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartman, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilodeau, M. T.</span><span> </span><span class="NLM_article-title">Heterocyclic fused pyridone carboxylic acid M1 positive allosteric modulators</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2533</span><span class="NLM_x">–</span> <span class="NLM_lpage">2537</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=2533-2537&author=S.+D.+Kudukauthor=C.+N.+Di+Marcoauthor=R.+K.+Changauthor=W.+J.+Rayauthor=L.+Mabauthor=M.+Wittmannauthor=M.+A.+Seagerauthor=K.+A.+Koeplingerauthor=C.+D.+Thompsonauthor=G.+D.+Hartmanauthor=M.+T.+Bilodeau&title=Heterocyclic+fused+pyridone+carboxylic+acid+M1+positive+allosteric+modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKuduk%26aufirst%3DS.%2BD.%26aulast%3DDi%2BMarco%26aufirst%3DC.%2BN.%26aulast%3DChang%26aufirst%3DR.%2BK.%26aulast%3DRay%26aufirst%3DW.%2BJ.%26aulast%3DMab%26aufirst%3DL.%26aulast%3DWittmann%26aufirst%3DM.%26aulast%3DSeager%26aufirst%3DM.%2BA.%26aulast%3DKoeplinger%26aufirst%3DK.%2BA.%26aulast%3DThompson%26aufirst%3DC.%2BD.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26aulast%3DBilodeau%26aufirst%3DM.%2BT.%26atitle%3DHeterocyclic%2520fused%2520pyridone%2520carboxylic%2520acid%2520M1%2520positive%2520allosteric%2520modulators%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D2533%26epage%3D2537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Wohlfahrt, G.; Tormakangas, O.; Salo, H.; Hoglund, L.; Karjalainen, A.; Knuutila, P.; Holm, P.; Rasku, S.; Vesalainen, A.</span><span> </span><span class="NLM_article-title">Androgen receptor modulating compounds</span>. WO2011/051540,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=G.+Wohlfahrt&author=O.+Tormakangas&author=H.+Salo&author=L.+Hoglund&author=A.+Karjalainen&author=P.+Knuutila&author=P.+Holm&author=S.+Rasku&author=A.+Vesalainen&title=Androgen+receptor+modulating+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWohlfahrt%26aufirst%3DG.%26atitle%3DAndrogen%2520receptor%2520modulating%2520compounds%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Zhang, N.; Ayral-Kaloustian, S.; Mansour, T.; Nguyen, T.; Niu, C.; Rosfjord, E.; Suayan, R.; Tsou, H.</span><span> </span><span class="NLM_article-title">2-Aryl and 2-heteroarylthiazolyl compounds, methods for their preparation and use thereof</span>. WO2009/120826,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=N.+Zhang&author=S.+Ayral-Kaloustian&author=T.+Mansour&author=T.+Nguyen&author=C.+Niu&author=E.+Rosfjord&author=R.+Suayan&author=H.+Tsou&title=2-Aryl+and+2-heteroarylthiazolyl+compounds%2C+methods+for+their+preparation+and+use+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DN.%26atitle%3D2-Aryl%2520and%25202-heteroarylthiazolyl%2520compounds%252C%2520methods%2520for%2520their%2520preparation%2520and%2520use%2520thereof%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Taurins, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan Khouw, V.</span><span> </span><span class="NLM_article-title">Isothiazolopyridines. I. Synthesis and spectra of isothiazolo[3,4-b]-, 3-amino-isothiazolo[4,3-b]-, isothiazolo[5,4-b]-, and 3-methylisothiazolo[5,4-c]pyridines. Preparation and spectra of some 2,3- and 3,4-disubstituted pyridines</span> <span class="citation_source-journal">Can. J. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1741</span><span class="NLM_x">–</span> <span class="NLM_lpage">1748</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1973&pages=1741-1748&author=A.+Taurinsauthor=V.+Tan+Khouw&title=Isothiazolopyridines.+I.+Synthesis+and+spectra+of+isothiazolo%5B3%2C4-b%5D-%2C+3-amino-isothiazolo%5B4%2C3-b%5D-%2C+isothiazolo%5B5%2C4-b%5D-%2C+and+3-methylisothiazolo%5B5%2C4-c%5Dpyridines.+Preparation+and+spectra+of+some+2%2C3-+and+3%2C4-disubstituted+pyridines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTaurins%26aufirst%3DA.%26aulast%3DTan%2BKhouw%26aufirst%3DV.%26atitle%3DIsothiazolopyridines.%2520I.%2520Synthesis%2520and%2520spectra%2520of%2520isothiazolo%255B3%252C4-b%255D-%252C%25203-amino-isothiazolo%255B4%252C3-b%255D-%252C%2520isothiazolo%255B5%252C4-b%255D-%252C%2520and%25203-methylisothiazolo%255B5%252C4-c%255Dpyridines.%2520Preparation%2520and%2520spectra%2520of%2520some%25202%252C3-%2520and%25203%252C4-disubstituted%2520pyridines%26jtitle%3DCan.%2520J.%2520Chem.%26date%3D1973%26volume%3D51%26spage%3D1741%26epage%3D1748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Miyaura, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, A.</span><span> </span><span class="NLM_article-title">Palladium-catalyzed cross-coupling reactions of organoboron compounds</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">2457</span><span class="NLM_x">–</span> <span class="NLM_lpage">2483</span></span><div class="citationLinks">[<a href="/doi/10.1021/cr00039a007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADyaK2MXoslGiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1995&pages=2457-2483&author=N.+Miyauraauthor=A.+Suzuki&title=Palladium-catalyzed+cross-coupling+reactions+of+organoboron+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-Catalyzed Cross-Coupling Reactions of Organoboron Compounds</span></div><div class="casAuthors">Miyaura, Norio; Suzuki, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2457-83</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with >250 refs. including title reactions and their mechanisms, prepn. of organoboron reagents, alkoxycarbonylation and dimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHwupBJT1UCbVg90H21EOLACvtfcHk0lhf9sLzGaYLrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXoslGiurg%253D&md5=d127b414a75161652876eebc3ed0c486</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fcr00039a007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00039a007%26sid%3Dliteratum%253Aachs%26aulast%3DMiyaura%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DA.%26atitle%3DPalladium-catalyzed%2520cross-coupling%2520reactions%2520of%2520organoboron%2520compounds%26jtitle%3DChem.%2520Rev.%26date%3D1995%26volume%3D95%26spage%3D2457%26epage%3D2483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincke, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufholz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zenn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmermann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutiérrez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzos-Skintges, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joachimiak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muyldermans, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herberg, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span> </span><span class="NLM_article-title">Structure of cyclin G-associated kinase (GAK) trapped in different conformations using nanobodies</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">459</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2014&pages=59-69&author=A.+Chaikuadauthor=T.+Keatesauthor=C.+Vinckeauthor=M.+Kaufholzauthor=M.+Zennauthor=B.+Zimmermannauthor=C.+Guti%C3%A9rrezauthor=R.+G.+Zhangauthor=C.+Hatzos-Skintgesauthor=A.+Joachimiakauthor=S.+Muyldermansauthor=F.+W.+Herbergauthor=S.+Knappauthor=S.+M%C3%BCller&title=Structure+of+cyclin+G-associated+kinase+%28GAK%29+trapped+in+different+conformations+using+nanobodies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DVincke%26aufirst%3DC.%26aulast%3DKaufholz%26aufirst%3DM.%26aulast%3DZenn%26aufirst%3DM.%26aulast%3DZimmermann%26aufirst%3DB.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DR.%2BG.%26aulast%3DHatzos-Skintges%26aufirst%3DC.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DMuyldermans%26aufirst%3DS.%26aulast%3DHerberg%26aufirst%3DF.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DStructure%2520of%2520cyclin%2520G-associated%2520kinase%2520%2528GAK%2529%2520trapped%2520in%2520different%2520conformations%2520using%2520nanobodies%26jtitle%3DBiochem.%2520J.%26date%3D2014%26volume%3D459%26spage%3D59%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Exploration of type II binding mode: A privileged approach for kinase inhibitor focused drug discovery?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1230</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+A+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520A%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Lupberger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeisel, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thumann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fofana, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zona, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mee, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Royer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahid, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavillette, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fresquet, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosset, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothenberg, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietschmann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pessaux, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doffoël, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raffelsberger, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poch, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mckeating, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brino, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumert, T. F.</span><span> </span><span class="NLM_article-title">EGFR and EphA2 are host factors for hepatitis C virus entry and possible targets for antiviral therapy</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">589</span><span class="NLM_x">–</span> <span class="NLM_lpage">595</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1038%2Fnm.2341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=21516087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFyqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=589-595&author=J.+Lupbergerauthor=M.+B.+Zeiselauthor=F.+Xiaoauthor=C.+Thumannauthor=I.+Fofanaauthor=L.+Zonaauthor=C.+Davisauthor=C.+J.+Meeauthor=M.+Turekauthor=S.+Gorkeauthor=C.+Royerauthor=B.+Fischerauthor=M.+N.+Zahidauthor=D.+Lavilletteauthor=J.+Fresquetauthor=F.+Cossetauthor=S.+M.+Rothenbergauthor=T.+Pietschmannauthor=A.+H.+Patelauthor=P.+Pessauxauthor=M.+Doffo%C3%ABlauthor=W.+Raffelsbergerauthor=O.+Pochauthor=J.+A.+Mckeatingauthor=L.+A.+Brinoauthor=T.+F.+Baumert&title=EGFR+and+EphA2+are+host+factors+for+hepatitis+C+virus+entry+and+possible+targets+for+antiviral+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR and EphA2 are host factors for hepatitis C virus entry and possible targets for antiviral therapy</span></div><div class="casAuthors">Lupberger, Joachim; Zeisel, Mirjam B.; Xiao, Fei; Thumann, Christine; Fofana, Isabel; Zona, Laetitia; Davis, Christopher; Mee, Christopher J.; Turek, Marine; Gorke, Sebastian; Royer, Cathy; Fischer, Benoit; Zahid, Muhammad N.; Lavillette, Dimitri; Fresquet, Judith; Cosset, Francois-Loic; Rothenberg, S. Michael; Pietschmann, Thomas; Patel, Arvind H.; Pessaux, Patrick; Doffoel, Michel; Raffelsberger, Wolfgang; Poch, Olivier; McKeating, Jane A.; Brino, Laurent; Baumert, Thomas F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">589-595</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) is a major cause of liver disease, but therapeutic options are limited and there are no prevention strategies.  Viral entry is the first step of infection and requires the cooperative interaction of several host cell factors.  Using a functional RNAi kinase screen, we identified epidermal growth factor receptor and ephrin receptor A2 as host cofactors for HCV entry.  Blocking receptor kinase activity by approved inhibitors broadly impaired infection by all major HCV genotypes and viral escape variants in cell culture and in a human liver chimeric mouse model in vivo.  The identified receptor tyrosine kinases (RTKs) mediate HCV entry by regulating CD81-claudin-1 co-receptor assocns. and viral glycoprotein-dependent membrane fusion.  These results identify RTKs as previously unknown HCV entry cofactors and show that tyrosine kinase inhibitors have substantial antiviral activity.  Inhibition of RTK function may constitute a new approach for prevention and treatment of HCV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPNZcMOXBl57Vg90H21EOLACvtfcHk0lg9VI82Okk0Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFyqtL4%253D&md5=9a772f9cb03077335c7e7f5918a6e1ef</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnm.2341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2341%26sid%3Dliteratum%253Aachs%26aulast%3DLupberger%26aufirst%3DJ.%26aulast%3DZeisel%26aufirst%3DM.%2BB.%26aulast%3DXiao%26aufirst%3DF.%26aulast%3DThumann%26aufirst%3DC.%26aulast%3DFofana%26aufirst%3DI.%26aulast%3DZona%26aufirst%3DL.%26aulast%3DDavis%26aufirst%3DC.%26aulast%3DMee%26aufirst%3DC.%2BJ.%26aulast%3DTurek%26aufirst%3DM.%26aulast%3DGorke%26aufirst%3DS.%26aulast%3DRoyer%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DB.%26aulast%3DZahid%26aufirst%3DM.%2BN.%26aulast%3DLavillette%26aufirst%3DD.%26aulast%3DFresquet%26aufirst%3DJ.%26aulast%3DCosset%26aufirst%3DF.%26aulast%3DRothenberg%26aufirst%3DS.%2BM.%26aulast%3DPietschmann%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DA.%2BH.%26aulast%3DPessaux%26aufirst%3DP.%26aulast%3DDoffo%25C3%25ABl%26aufirst%3DM.%26aulast%3DRaffelsberger%26aufirst%3DW.%26aulast%3DPoch%26aufirst%3DO.%26aulast%3DMckeating%26aufirst%3DJ.%2BA.%26aulast%3DBrino%26aufirst%3DL.%2BA.%26aulast%3DBaumert%26aufirst%3DT.%2BF.%26atitle%3DEGFR%2520and%2520EphA2%2520are%2520host%2520factors%2520for%2520hepatitis%2520C%2520virus%2520entry%2520and%2520possible%2520targets%2520for%2520antiviral%2520therapy%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D589%26epage%3D595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Murray, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tassello, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span> </span><span class="NLM_article-title">Alanine scanning of the hepatitis C virus core protein reveals numerous residues essential for production of infectious virus</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">10220</span><span class="NLM_x">–</span> <span class="NLM_lpage">10231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1128%2FJVI.00793-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=17634240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtV2gtb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=10220-10231&author=C.+L.+Murrayauthor=C.+T.+Jonesauthor=J.+Tasselloauthor=C.+M.+Rice&title=Alanine+scanning+of+the+hepatitis+C+virus+core+protein+reveals+numerous+residues+essential+for+production+of+infectious+virus"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Alanine scanning of the hepatitis C virus core protein reveals numerous residues essential for production of infectious virus</span></div><div class="casAuthors">Murray, Catherine L.; Jones, Christopher T.; Tassello, Jodie; Rice, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10220-10231</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) is an important human pathogen affecting an estd. 3% of the world's population.  Recent advances have enabled in vitro propagation of the virus and allow assembly and egress to be investigated for the first time.  As a component of the virion, the HCV core protein likely functions primarily in infectious virus prodn., although little is known about the determinants of this activity.  To investigate the roles of core in the viral life cycle, we performed a comprehensive deletion and alanine scanning mutagenesis study of this protein in the context of a genotype 2a reporter virus.  We have confirmed that core protein is essential for infectious virion prodn. and have identified numerous residues required for this role.  The infectivity of several assembly-defective core mutants could be rescued by compensatory mutations identified in p7 and NS2, suggesting genetic interactions with core and highlighting the importance of these nonstructural proteins in infectious virion morphogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptehuI2myC37Vg90H21EOLACvtfcHk0lg9VI82Okk0Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtV2gtb3E&md5=a0e817ce366fdbc576e513c4a080df46</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FJVI.00793-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00793-07%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BL.%26aulast%3DJones%26aufirst%3DC.%2BT.%26aulast%3DTassello%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26atitle%3DAlanine%2520scanning%2520of%2520the%2520hepatitis%2520C%2520virus%2520core%2520protein%2520reveals%2520numerous%2520residues%2520essential%2520for%2520production%2520of%2520infectious%2520virus%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D10220%26epage%3D10231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Bartosch, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubuisson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosset, F. L.</span><span> </span><span class="NLM_article-title">Infectious hepatitis C virus pseudo-particles containing functional E1-E2 envelope protein complexes</span> <span class="citation_source-journal">J. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">197</span><span class="NLM_x">, </span> <span class="NLM_fpage">633</span><span class="NLM_x">–</span> <span class="NLM_lpage">642</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=2003&pages=633-642&author=B.+Bartoschauthor=J.+Dubuissonauthor=F.+L.+Cosset&title=Infectious+hepatitis+C+virus+pseudo-particles+containing+functional+E1-E2+envelope+protein+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBartosch%26aufirst%3DB.%26aulast%3DDubuisson%26aufirst%3DJ.%26aulast%3DCosset%26aufirst%3DF.%2BL.%26atitle%3DInfectious%2520hepatitis%2520C%2520virus%2520pseudo-particles%2520containing%2520functional%2520E1-E2%2520envelope%2520protein%2520complexes%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2003%26volume%3D197%26spage%3D633%26epage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Ricotta, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preiss, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teichert, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Höning, S.</span><span> </span><span class="NLM_article-title">Characterization of a protein phosphatase 2A holoenzyme that dephosphorylates the clathrin adaptors AP-1 and AP-2</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">5510</span><span class="NLM_x">–</span> <span class="NLM_lpage">5517</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=5510-5517&author=D.+Ricottaauthor=J.+Hansenauthor=C.+Preissauthor=D.+Teichertauthor=S.+H%C3%B6ning&title=Characterization+of+a+protein+phosphatase+2A+holoenzyme+that+dephosphorylates+the+clathrin+adaptors+AP-1+and+AP-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRicotta%26aufirst%3DD.%26aulast%3DHansen%26aufirst%3DJ.%26aulast%3DPreiss%26aufirst%3DC.%26aulast%3DTeichert%26aufirst%3DD.%26aulast%3DH%25C3%25B6ning%26aufirst%3DS.%26atitle%3DCharacterization%2520of%2520a%2520protein%2520phosphatase%25202A%2520holoenzyme%2520that%2520dephosphorylates%2520the%2520clathrin%2520adaptors%2520AP-1%2520and%2520AP-2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D5510%26epage%3D5517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Bhattacharyya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warfield, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruthel, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bavari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hope, T. J.</span><span> </span><span class="NLM_article-title">Ebola virus uses clathrin-mediated endocytosis as an entry pathway</span> <span class="citation_source-journal">Virology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">401</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=401&publication_year=2010&pages=18-28&author=S.+Bhattacharyyaauthor=K.+L.+Warfieldauthor=G.+Ruthelauthor=S.+Bavariauthor=M.+J.+Amanauthor=T.+J.+Hope&title=Ebola+virus+uses+clathrin-mediated+endocytosis+as+an+entry+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharyya%26aufirst%3DS.%26aulast%3DWarfield%26aufirst%3DK.%2BL.%26aulast%3DRuthel%26aufirst%3DG.%26aulast%3DBavari%26aufirst%3DS.%26aulast%3DAman%26aufirst%3DM.%2BJ.%26aulast%3DHope%26aufirst%3DT.%2BJ.%26atitle%3DEbola%2520virus%2520uses%2520clathrin-mediated%2520endocytosis%2520as%2520an%2520entry%2520pathway%26jtitle%3DVirology%26date%3D2010%26volume%3D401%26spage%3D18%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Jin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, K.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, K.</span><span> </span><span class="NLM_article-title">Hantaan virus enters cells by clathrin-dependent receptor-mediated endocytosis</span> <span class="citation_source-journal">Virology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">294</span><span class="NLM_x">, </span> <span class="NLM_fpage">60</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=2002&pages=60-69&author=M.+Jinauthor=J.+Parkauthor=S.+Leeauthor=B.+Parkauthor=J.+Shinauthor=K.-J.+Songauthor=T.+I.+Ahnauthor=S.+Y.+Hwangauthor=B.+Y.+Ahnauthor=K.+Ahn&title=Hantaan+virus+enters+cells+by+clathrin-dependent+receptor-mediated+endocytosis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DPark%26aufirst%3DB.%26aulast%3DShin%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DK.-J.%26aulast%3DAhn%26aufirst%3DT.%2BI.%26aulast%3DHwang%26aufirst%3DS.%2BY.%26aulast%3DAhn%26aufirst%3DB.%2BY.%26aulast%3DAhn%26aufirst%3DK.%26atitle%3DHantaan%2520virus%2520enters%2520cells%2520by%2520clathrin-dependent%2520receptor-mediated%2520endocytosis%26jtitle%3DVirology%26date%3D2002%26volume%3D294%26spage%3D60%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Boge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyss, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonifacino, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thali, M.</span><span> </span><span class="NLM_article-title">A membrane-proximal tyrosine-based signal mediates internalization of the HIV-1 envelope glycoprotein via interaction with the AP-2 clathrin adaptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">15773</span><span class="NLM_x">–</span> <span class="NLM_lpage">15778</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=15773-15778&author=M.+Bogeauthor=S.+P.+Wyssauthor=J.+S.+Bonifacinoauthor=M.+Thali&title=A+membrane-proximal+tyrosine-based+signal+mediates+internalization+of+the+HIV-1+envelope+glycoprotein+via+interaction+with+the+AP-2+clathrin+adaptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoge%26aufirst%3DM.%26aulast%3DWyss%26aufirst%3DS.%2BP.%26aulast%3DBonifacino%26aufirst%3DJ.%2BS.%26aulast%3DThali%26aufirst%3DM.%26atitle%3DA%2520membrane-proximal%2520tyrosine-based%2520signal%2520mediates%2520internalization%2520of%2520the%2520HIV-1%2520envelope%2520glycoprotein%2520via%2520interaction%2520with%2520the%2520AP-2%2520clathrin%2520adaptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D15773%26epage%3D15778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhMRBs5pEKcAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">CCP4</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=CCP4&title=The+CCP4+suite%3A+programs+for+protein+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liK_45xd1pxWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liK_45xd1pxWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0liK_45xd1pxWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leaver-Fay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snoeyink, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">W375</span><span class="NLM_x">–</span> <span class="NLM_lpage">W383</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1093%2Fnar%2Fgkm216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=17452350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W375-W383&author=I.+W.+Davisauthor=A.+Leaver-Fayauthor=V.+B.+Chenauthor=J.+N.+Blockauthor=G.+J.+Kapralauthor=X.+Wangauthor=L.+W.+Murrayauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+contacts+and+structure+validation+for+proteins+and+nucleic+acids"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span></div><div class="casAuthors">Davis Ian W; Leaver-Fay Andrew; Chen Vincent B; Block Jeremy N; Kapral Gary J; Wang Xueyi; Murray Laura W; Arendall W Bryan 3rd; Snoeyink Jack; Richardson Jane S; Richardson David C</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W375-83</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MolProbity is a general-purpose web server offering quality validation for 3D structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact analysis of any steric problems within the molecules as well as updated dihedral-angle diagnostics, and it can calculate and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addition and full optimization of all hydrogen atoms, both polar and nonpolar.  New analysis functions have been added for RNA, for interfaces, and for NMR ensembles.  Additionally, both the web site and major component programs have been rewritten to improve speed, convenience, clarity and integration with other resources.  MolProbity results are reported in multiple forms: as overall numeric scores, as lists or charts of local problems, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KiNG viewer.  This service is available free to all users at http://molprobity.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuyhuetoNS59bHMBt0mMIXfW6udTcc2eYDtZcQwcAbNrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D&md5=726ca1e7e162ab2a4a05048f493731fb</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm216%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DLeaver-Fay%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DBlock%26aufirst%3DJ.%2BN.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520contacts%2520and%2520structure%2520validation%2520for%2520proteins%2520and%2520nucleic%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW375%26epage%3DW383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Lindenbach, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Syder, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wölk, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tellinghuisen, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maruyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hynes, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeating, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span> </span><span class="NLM_article-title">Complete replication of hepatitis C virus in cell culture</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">309</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">626</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm501759m&amp;key=10.1126%2Fscience.1114016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm501759m&amp;key=15947137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm501759m&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsFChtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=309&publication_year=2005&pages=623-626&author=B.+D.+Lindenbachauthor=M.+J.+Evansauthor=A.+J.+Syderauthor=B.+W%C3%B6lkauthor=T.+L.+Tellinghuisenauthor=C.+C.+Liuauthor=T.+Maruyamaauthor=R.+O.+Hynesauthor=D.+R.+Burtonauthor=J.+A.+McKeatingauthor=C.+M.+Rice&title=Complete+replication+of+hepatitis+C+virus+in+cell+culture"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Complete Replication of Hepatitis C Virus in Cell Culture</span></div><div class="casAuthors">Lindenbach, Brett D.; Evans, Matthew J.; Syder, Andrew J.; Woelk, Benno; Tellinghuisen, Timothy L.; Liu, Christopher C.; Maruyama, Toshiaki; Hynes, Richard O.; Burton, Dennis R.; McKeating, Jane A.; Rice, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">309</span>
        (<span class="NLM_cas:issue">5734</span>),
    <span class="NLM_cas:pages">623-626</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Many aspects of the hepatitis C virus (HCV) life cycle have not been reproduced in cell culture, which has slowed research progress on this important human pathogen.  Here, we describe a full-length HCV genome that replicates and produces virus particles that are infectious in cell culture (HCVcc).  Replication of HCVcc was robust, producing nearly 105 infectious units per mL within 48 h.  Virus particles were filterable and neutralized with a monoclonal antibody against the viral glycoprotein E2.  Viral entry was dependent on cellular expression of a putative HCV receptor, CD81.  HCVcc replication was inhibited by interferon-α and by several HCV-specific antiviral compds., suggesting that this in vitro system will aid in the search for improved antivirals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroGMVonlJq-rVg90H21EOLACvtfcHk0lj_otZmy7yqNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsFChtr4%253D&md5=05a4723cb8bf6b4e33cecab9be5bcdea</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1126%2Fscience.1114016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1114016%26sid%3Dliteratum%253Aachs%26aulast%3DLindenbach%26aufirst%3DB.%2BD.%26aulast%3DEvans%26aufirst%3DM.%2BJ.%26aulast%3DSyder%26aufirst%3DA.%2BJ.%26aulast%3DW%25C3%25B6lk%26aufirst%3DB.%26aulast%3DTellinghuisen%26aufirst%3DT.%2BL.%26aulast%3DLiu%26aufirst%3DC.%2BC.%26aulast%3DMaruyama%26aufirst%3DT.%26aulast%3DHynes%26aufirst%3DR.%2BO.%26aulast%3DBurton%26aufirst%3DD.%2BR.%26aulast%3DMcKeating%26aufirst%3DJ.%2BA.%26aulast%3DRice%26aufirst%3DC.%2BM.%26atitle%3DComplete%2520replication%2520of%2520hepatitis%2520C%2520virus%2520in%2520cell%2520culture%26jtitle%3DScience%26date%3D2005%26volume%3D309%26spage%3D623%26epage%3D626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y8D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y8D','PDB','4Y8D'); return false;">PDB: 4Y8D</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i99"><a href="/doi/suppl/10.1021/jm501759m">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65758"></div></div></div></div></div><hr /></hr><p class="last">NMR spectra of representative compounds, HPLC purity data of final compounds, and data collection and refinement statistics for the X-ray crystallography. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm501759m/suppl_file/jm501759m_si_001.pdf">jm501759m_si_001.pdf (524.89 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates have been deposited in the PDB with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y8D">4Y8D</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm501759m&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-8%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Fjm501759m%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm501759m" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677acbfa4a5525a5","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
