
<html lang="en"     class="pb-page"  data-request-id="c8b39614-aa26-46a5-ad7a-de46cf252dba"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm901147e;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2009.52.issue-20;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Kinase Inhibitor Data Modeling and de Novo Inhibitor Design with Fragment Approaches" /></meta><meta name="dc.Creator" content="Michal  Vieth" /></meta><meta name="dc.Creator" content="Jon  Erickson" /></meta><meta name="dc.Creator" content="Jibo  Wang" /></meta><meta name="dc.Creator" content="Yue  Webster" /></meta><meta name="dc.Creator" content="Mary  Mader" /></meta><meta name="dc.Creator" content="Richard  Higgs" /></meta><meta name="dc.Creator" content="Ian  Watson" /></meta><meta name="dc.Description" content="A reconstructive approach based on computational fragmentation of existing inhibitors and validated kinase potency models to recombine and create “de novo” kinase inhibitor small molecule libraries..." /></meta><meta name="Description" content="A reconstructive approach based on computational fragmentation of existing inhibitors and validated kinase potency models to recombine and create “de novo” kinase inhibitor small molecule libraries..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 30, 2009" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm901147e" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2009 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm901147e" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm901147e" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm901147e" /></link>
        
    
    

<title>Kinase Inhibitor Data Modeling and de Novo Inhibitor Design with Fragment Approaches | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm901147e" /></meta><meta property="og:title" content="Kinase Inhibitor Data Modeling and de Novo Inhibitor Design with Fragment Approaches" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0001.jpeg" /></meta><meta property="og:description" content="A reconstructive approach based on computational fragmentation of existing inhibitors and validated kinase potency models to recombine and create “de novo” kinase inhibitor small molecule libraries is described. The screening results from model selected molecules from the corporate database and seven computationally derived small molecule libraries were used to evaluate this approach. Specifically, 1895 model selected database molecules were screened at 20 μM in six kinase assays and yielded an overall hit rate of 84%. These models were then used in the de novo design of seven chemical libraries consisting of 20−50 compounds each. Then 179 compounds from synthesized libraries were tested against these six kinases with an overall hit rate of 92%. Comparing predicted and observed selectivity profiles serves to highlight the strengths and limitations of the methodology, while analysis of functional group contributions from the libraries suggest general principles governing binding of ATP competitive compounds." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm901147e"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm901147e">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm901147e&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm901147e&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm901147e&amp;href=/doi/10.1021/jm901147e" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2009</span><span class="cit-fg-volume">, 52</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 6456-6466</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/52/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm9009873" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm901167t" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Kinase Inhibitor Data Modeling and de Novo Inhibitor Design with Fragment Approaches</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michal++Vieth">Michal Vieth</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jon++Erickson">Jon Erickson</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jibo++Wang">Jibo Wang</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yue++Webster">Yue Webster</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mary++Mader">Mary Mader</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Higgs">Richard Higgs</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ian++Watson">Ian Watson</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Eli Lilly and Company Lilly Research Laboratories, Lilly Corporate Center, DC 1931, Indianapolis, Indiana 46285</span></div><div class="corresp-info"><strong>*</strong>To whom correspondence should be addressed. Phone: (317) 277-3959. Fax: (317) 276-6545. E-mail: <a href="/cdn-cgi/l/email-protection#305d1e4659554458705c595c5c491e535f5d"><span class="__cf_email__" data-cfemail="7c11520a151908143c1015101005521f1311">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm901147e&amp;href=/doi/10.1021%2Fjm901147e" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2009</span></span><span class="cit-volume">, 52</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 6456–6466</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 30, 2009</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 July 2009</li><li><span class="item_label"><b>Published</b> online</span>30 September 2009</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 October 2009</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm901147e" title="DOI URL">https://doi.org/10.1021/jm901147e</a></div><div class="article_header-article-copyright"><strong>Copyright © 2009 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6456%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMichal%2BVieth%252C%2BJon%2BErickson%252C%2BJibo%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D52%26issueNum%3D20%26contentID%3Djm901147e%26title%3DKinase%2BInhibitor%2BData%2BModeling%2Band%2Bde%2BNovo%2BInhibitor%2BDesign%2Bwith%2BFragment%2BApproaches%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6466%26publicationDate%3DOctober%2B2009">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm901147e"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3575</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">36</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm901147e" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Kinase Inhibitor Data Modeling and de Novo Inhibitor Design with Fragment Approaches&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Michal&quot;,&quot;last_name&quot;:&quot;Vieth&quot;},{&quot;first_name&quot;:&quot;Jon&quot;,&quot;last_name&quot;:&quot;Erickson&quot;},{&quot;first_name&quot;:&quot;Jibo&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yue&quot;,&quot;last_name&quot;:&quot;Webster&quot;},{&quot;first_name&quot;:&quot;Mary&quot;,&quot;last_name&quot;:&quot;Mader&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Higgs&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;Watson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2009&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;52&quot;,&quot;pages&quot;:&quot;6456-6466&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm901147e&quot;},&quot;abstract&quot;:&quot;A reconstructive approach based on computational fragmentation of existing inhibitors and validated kinase potency models to recombine and create “de novo” kinase inhibitor small molecule libraries is described. The screening results from model selected molecules from the corporate database and seven computationally derived small molecule libraries were used to evaluate this approach. Specifically, 1895 model selected database molecules were screened at 20 μM in six kinase assays and yielded an overall hit rate of 84%. These models were then used in the de novo design of seven chemical libraries consisting of 20−50 compounds each. Then 179 compounds from synthesized libraries were tested against these six kinases with an overall hit rate of 92%. Comparing predicted and observed selectivity profiles serves to highlight the strengths and limitations of the methodology, while analysis of functional group contributions from the libraries suggest general principles governing binding of ATP competitive compounds.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm901147e&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm901147e" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm901147e&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm901147e" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm901147e&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm901147e" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm901147e&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm901147e&amp;href=/doi/10.1021/jm901147e" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm901147e" /></input><a href="/doi/pdf/10.1021/jm901147e" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm901147e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm901147e%26sid%3Dliteratum%253Aachs%26pmid%3D19791746%26genre%3Darticle%26aulast%3DVieth%26date%3D2009%26atitle%3DKinase%2BInhibitor%2BData%2BModeling%2Band%2Bde%2BNovo%2BInhibitor%2BDesign%2Bwith%2BFragment%2BApproaches%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D52%26issue%3D20%26spage%3D6456%26epage%3D6466%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291120" title="Hydrophobicity">Hydrophobicity</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291133" title="Solubilization">Solubilization</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/52/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A reconstructive approach based on computational fragmentation of existing inhibitors and validated kinase potency models to recombine and create “de novo” kinase inhibitor small molecule libraries is described. The screening results from model selected molecules from the corporate database and seven computationally derived small molecule libraries were used to evaluate this approach. Specifically, 1895 model selected database molecules were screened at 20 μM in six kinase assays and yielded an overall hit rate of 84%. These models were then used in the de novo design of seven chemical libraries consisting of 20−50 compounds each. Then 179 compounds from synthesized libraries were tested against these six kinases with an overall hit rate of 92%. Comparing predicted and observed selectivity profiles serves to highlight the strengths and limitations of the methodology, while analysis of functional group contributions from the libraries suggest general principles governing binding of ATP competitive compounds.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76316" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76316" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The kinase families’ function in signal transduction pathways has given rise to multiple attractive targets for therapeutic intervention in many disease states such as cancer, diabetes, inflammation, and arthritis.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Recently, multiple drug launches and a variety of clinical studies with kinase inhibition as the primary mechanism of action has probed therapeutic space with inhibitors of this gene family. The success of drugs with kinase inhibition as the mode of action demonstrates the ability to deliver kinase inhibitor drugs with varying selectivity, potency, and pharmacokinetic properties.</div><div class="NLM_p">Kinase-targeted compounds carry various degrees of interfamily selectivity as some marketed drugs inhibit several kinases (e.g., sunitinib), and others are more selective (e.g., gefitinib).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Thus, the understanding and practical application of design principles which lead to kinase inhibitors with appropriate selectivity are of high interest. Many clinical and marketed ATP competitive kinase inhibitors exhibit a similar pharmacophore.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> This pharmacophore, exemplified by gefitinib among others (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>a,b), contains a hinge binding element that is connected to both hydrophobic and solubilizing groups. A method that takes advantage of new technology for the efficient design of kinase inhibitors carrying this generalized pharmacophore has been developed. Here we describe an application of this method to design, synthesize, and assay kinase inhibitors to test the ability of our pharmacophore hypothesis to probe general kinase inhibition.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Kinase pharmacophore exemplified in marketed and clinical compounds. Solubilizing groups are highlighted with green, hydrophobic with yellow, and hinge binders with cyan. (b) Three-dimensional picture of pharmacophore drawn over X-ray structures of gefitinib (yellow carbons, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ity">2ity</a>.pdb) in EGFR and danusertib in Aurora A (magenta carbons, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2j50">2j50</a>.pdb). EGFR kinase from <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ity">2ity</a>.pdb is shown. Transparent ovals in yellow, cyan, and green are drawn over hydrophobic, hinge binding, and solubilizing groups, respectively. Note that the positions of both solubilizing and hydrophobic groups occupy slightly different region in both structures in contrast to the hinge binding elements, which occupy similar space. Thr790 gate keeper in EGFR is shown in yellow stick representation. (c) Schematic representation of de novo design process starting from actives, fragmentation, and reassembly. Reproduced with ACS permission from ref <a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">7</a>. (d) Hypothetical application to three kinase drugs from Figure 1. Solubilizing groups are highlighted in green, hydrophobic in yellow, and hinge binders in cyan. Fragment connection atoms contain isotopic labels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to understand kinase inhibitor potency and selectivity, we have developed informatics and computational approaches to characterize kinase target space from profiling data.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> We have demonstrated that correlations exist between fragments or “privileged structures” within kinase inhibitors and their binding affinity to particular kinases.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In addition, it was shown that similarity between kinase targets could be derived from the similarity of the fragments making up inhibitors of those particular kinases in a complementary fashion to relating targets by whole ligand similarity,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> three-dimensional protein−ligand structure,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and full profiling approaches.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Also described was the use of small molecule inhibitor fragment frequencies for understanding kinase potency and selectivity.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In those efforts, Naïve Bayes models employing fragment frequencies provide highly interpretable and reliable means for predicting potency in individual kinases. Specifically, the models were utilized with moderate success (40% hit rate) in prospective screening for new kinase inhibitors.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p last">In this report, these concepts are taken a step further, beyond retrospective analysis to prospective generation of kinase inhibitor libraries. We introduce and prospectively validate improved computational models utilizing a support vector machine based method built with fingerprint descriptors (SVMFP<a class="ref internalNav" href="#fn9" aria-label="a">a</a><div class="NLM_fn" id="fn9">a<p class="last">Abbreviations: FBDD, fragment-based drug design; SBDD, structure-based drug design; QSAR, quantitative structure−activity relationship; SVM, support vector machine; SVMFP, support vector machine fingerprints; PPV, positive predicted value; PDB, Protein Databank; ATP, adenosine triphosphate; ABL1, c-abl oncogene 1, receptor tyrosine kinase; CHK2, CHK2 checkpoint kinase; FLT3, fms-related tyrosine kinase 3; MET, met proto-oncogene (hepatocyte growth factor receptor); P70S6K, RPS6KB1 ribosomal protein S6 kinase; ROCK2, Rho-associated, coiled-coil containing protein kinase 2; SP, single point; rms, root-mean-square; NN, nearest neighbor.</p></div>),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> in the process of computer model driven selection of compounds for biological testing (also referred to as virtual screening<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a>). The models are used as the driving force to select among new combinations of hinge-binding, hydrophobic and solubilizing fragments to design kinase inhibitors. Building blocks for these experiments were generated from fragmentation of existing inhibitors.<a onclick="showRef(event, 'ref7 ref14'); return false;" href="javascript:void(0);" class="ref ref7 ref14">(7, 14)</a> The resulting fragments were then recombined to create “de novo” chemical libraries and scored using qualified models trained on kinase potency. Analysis of the results from this experiment and data for a variety of universal hydrophobic and solubilizing groups for ABL1, CHK2, FLT3, MET, P70S6K, and ROCK2 kinases are presented.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42810" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42810" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Overview</h3><table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">1</td><td colspan="5" valign="top" class="paragraph"><p class="last">SVMFP models used to drive library selection were validated in chronological testing (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure 1">Supporting Information Figure 1</a>) and prospective screening of 1895 compounds against six kinases (ABL1, CHK2, FLT3, MET, P70S6K, and ROCK2).</p></td></tr><tr class="li1"><td valign="top">2</td><td colspan="5" valign="top" class="paragraph"><p class="last">A general kinase pharmacophore, exemplified in the EGFR marketed drug gefitinib (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>a), was used as a framework for libraries. SVMFP models were used to predict combinations of hypothetical hinge elements, hydrophobic and solubilizing groups from fragmented kinase inhbitors to design seven “de novo” kinase libraries with predicted activity in six kinases (ABL1, CHK2, FLT3, MET, P70S6K, and ROCK2) as outlined in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>c,d</p></td></tr><tr class="li1"><td valign="top">3</td><td colspan="5" valign="top" class="paragraph"><p class="last">Fourteen candidate libraries with all compounds predicted active (predicted >50% inhibition at 20 μM, 60% of compounds had predicted activity >70%) in at least one of six SVMFP kinase inhibition models, were initially selected and subsequently filtered to seven based on ease of synthesis. Compounds were then synthesized and assayed for single point (SP) % inhibition at 20 μM in these six kinases. The results showed a 92% confirmation rate of model predictions.</p></td></tr><tr class="li1"><td valign="top">4</td><td colspan="5" valign="top" class="paragraph"><p class="last">Model performance was analyzed for each library focusing on group contributions of hydrophobic and solubilizing moieties for utility of different kinase inhibition.</p></td></tr></table></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> SVMFP Model Development and Validation</h3><div class="NLM_p">Previously described models used for kinase inhibition prediction<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> utilized fragment frequency based naïve Bayes models. In this work, support vector machine (SVM)<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> models using Merck atom pair<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> derived fingerprints were employed. These models provided superior results with respect to prediction accuracy as compared to the naïve Bayes models. Support vector machines (SVMs) are supervised learning methods used for classification and regression.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> They function by simultaneously minimizing the empirical classification error and maximizing the geometric margin. Others have also reported on the utility of SVM models in QSAR-based virtual screening.<a onclick="showRef(event, 'ref11 ref17'); return false;" href="javascript:void(0);" class="ref ref11 ref17">(11, 17)</a> As with most models, the quality and relevance of model predictions and metrics of model performance depend on the number and structural diversity of compounds in the training set as well as the structural diversity of compounds to be predicted, i.e., things close to the training set generally get more reliable predictions than those further away but specific results might vary. Best practices indicate that one must split data into a test and training set to get reliable unbiased performance estimates.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19, 20)</a> Validation studies of the SVMFP models used here are based on multiple test/train splits and statistical measures, including positive predicted value (PPV) and active enrichments representing the average of the results as reported in earlier work.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> To further validate the models, chronological test/train splits were used to validate these activity models using earlier collected training data to predict test data collected at a later date. This was done to provide a more realistic estimate of model performance in a prospective study with existing compounds than the standard random selection of train/test splits, which can be overly optimistic given the sequential structural explorations inherent to an ongoing SAR studies. The less biased error estimates from chronological validation originate from the observation that new chemical diversity tends to be tested over time, better revealing a model’s “extrapolation” prediction error on new chemical diversity. This chronological model validation scheme provides additional assurance that the models are suitable for use in the design of new compounds. In this work a variety of parameters were considered to qualify the models including, number of observations in the training and test sets, the number of actives, PPV, negative predictive value (NPV), sensitivity, and specificity.</div><div class="NLM_p">A set of six previously described kinase assays was assembled from our internal research efforts for this study;<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> we substituted MKK6 with MET. To increase statistical significance, at least 1400 actives (≥70% inhibition at 20 μM test substance concentration) and 16000 compounds assayed (17K average) for each of the six kinase assays was required. Details of the data used in modeling are given in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 1">Supporting Information Table 1</a>. Data for each target was sorted in a chronological fashion, and the first 80% of compounds was used to build models, while the next 10% of compounds was used to test the model performance. The models were also built with the first 90% of the data, for training, and the last 10% to test model performance. This allowed for two chronological evaluations of model performance. In addition, three standard random 50−50 splits were performed using the first 80% of the data and the first 90% of the data, providing an additional six random test sets. The final model for prospective testing was then built using all data for each target. A graphical representation of the validation protocol and performance metrics is presented in <a class="ref internalNav" href="#notes-1" aria-label="Figure 1 of Supporting Information">Figure 1 of Supporting Information</a>.</div><div class="NLM_p last">The final models were used to select a set of 1895 previously untested compounds from our compound collection. Any compound predicted to be more than 70% active in any of six models was considered for screening. As much as possible we tried to vary predictive selectivity of compounds so that we could test performance of each model in this experiment.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> De Novo Design with Kinase Pharmacophore</h3><div class="NLM_p last">To demonstrate proof of concept for the utility of the SVMFP models in driving the design of kinase inhibitors, the pharmacophore shared by many ATP competitive kinase clinical and marketed compounds, exemplified in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>, were used. The elements of the pharmacophore were supplied by fragmentation of existing kinase actives (defined as <1 μM IC<sub>50</sub> in any kinase) with our previously described dicer approach,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> followed by binding mode predictions of hypothethical hinge binding elements.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> Inputs for the virtual library enumeration arose from 300 hinge binders, hydrophobic (less than or equal to two heteroatoms) and solubilizing group (presence of primary or secondary amine) coming from diced fragments to more than 300 hinge binders. SVMFP models were used to narrow down selection of hinge binders. A positional scanning<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>like strategy was used to avoid impractical enumeration of all possible structures. In this strategy, substituents for each candidate group (i.e., hydrophobic and solubilizing groups) at each of two positions on the hinge-binding group were scored relative to a random selection of five substituents for the other groups. Independence of the two positions/groups was assumed for this strategy. This assumption is likely to be working better for groups, which are spatially separated and not as well for groups which are in close proximity. It was hoped that the use of 5 substituents at the other position instead of just one would allow for some correction to this assumption. The enumeration resulted in 7500 compounds (5 × 5 × 300), which were then selected by ranking the top 50 groups in each position by the highest predicted activities for each target. A final 50 × 50 × 50 matrix was then enumerated for each target, and the final best scoring 20 hinge binders and their best scoring compounds were considered for synthetic feasibility. Once the chemistry schemes were prepared, the libraries were re-enumerated using reasonably priced reagents and scored with the model for each target to select 20−60 compounds predicted to have >50% activity in any of the six models. Initially, we were hoping to use 70% threshold, but for many libraries that proved rather difficult as models were too conservative in venturing into new chemical space. This difficulty was particularly evident for library 6, used as a positive control in this experiment. The compounds were described in the literature as MET inhibitors,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> however, only 3 of 41 were predicted to have more than 70% inhibition in one of the kinases. A total of 265 compounds were targeted for the seven libraries ranging from 19 to 60 compounds each. The final 179 synthesized compounds were evaluated in SP assays in six kinases at 20 μM, with SP hits defined as >70% inhibition.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">It should be noted that the hypotheses discussed here are based on interpretation of actives as defined by single point (SP) percent inhibition data measured at a 20 μM concentration. Although this measure is a good indication of a binding event, any selectivity or potency comparison is limited relative to analysis of dose response data. The percent inhibition data was used for model comparison and analysis of group activities for our libraries because this response was consistent with values used to train the models. IC<sub>50</sub> data for one library is presented to understand the correlation to dose response data. Overall, the translation is good, but the final dose response data tends to vary in magnitude for compounds showing similar, high percent inhibition.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> SVMFP Model Performance and Validation</h3><div class="NLM_p">An element critical to these retrospective studies was a sufficient number of training set compounds and actives to properly estimate the model performance in blind prospective validation tests. In addition, model qualification for “de novo” library design employed a stringent chronological validation studies to give a more realistic estimate of model performance metrics. Parts a−d of Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> show a comparison of performance metrics means derived from the prospective study along with two chronological validation studies and six traditional, randomized validation studies. The positive predictive value (PPV), which is the most critical performance metric for utilization of models in “de novo” design studies, is above 0.6 for all studied targets in the prospective study results and appears to be reasonably estimated by chronological and random split validation studies. For MET, P70S6K, and ROCK2, prospective study PPV appears to be very well estimated by chronological validation, while random split validation appears to provide a slight overestimate. For the ABL1, FLT3, the prospective study PPV is overestimated by both chronological and random split validation with the former, giving more realistic estimates. For CHK2, chronological validation gives a lower estimate, with random split validation giving a slight overestimate of prospective PPV.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Comparison of Chronological, random validation with prospective metrics. (a) PPVs. (b) Sensitivities. (c) Root mean square prediction error. (d) <i>R</i><sup>2</sup> values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Prospective sensitivity is important in understanding the false negative ratio and high sensitivity values are essential for prediction of selectivity. Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b shows that prediction sensitivity varies from 0.3, for the historically lowest hit rate target MET, to 0.9 for FLT3. In almost all cases, prospective sensitivity is underestimated by both random and chronological validations, with random validation estimates giving significantly better estimates for P70S6K and CHK2 and chronological validation being closer for ROCK2 and FLT3.</div><div class="NLM_p">The rms error (see Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>c) is another metric important for selectivity predictions. The prospective study rms results are significantly higher in chronological and random split validations for two targets (ABL1, FLT3) and only slightly higher than chronological validation error for CHK2 and P70S6K. For the two other targets, medium range differences from chronological results (MET, ROCK2) are observed. In all cases, random split validation is underestimating the prospective errors more than the chronological method.</div><div class="NLM_p"><i>R</i><sup>2</sup> values, i.e., the variance in real data explained by the models, may also be important for ranking the compounds and determining model performance (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>d). For all targets, <i>R</i><sup>2</sup> prospective validation values are significantly overestimated by both chronological and random split validations, with the chronological validation method giving lower overestimates for all targets except FLT3—note that all overestimates are large, especially for ABL1 and FLT3.</div><div class="NLM_p">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> (and <a class="ref internalNav" href="#notes-1" aria-label="Table 1 in Supporting Information">Table 1 in Supporting Information</a>) shows the general statistics for SVMFP model prospective validation studies of 1895 compounds in six kinase assays. Per target hit rate, described as % of compounds >70% at 20 μM, are 2−4 fold higher than observed historically. Overall kinase hit rate for the prospective validation compounds increased 2.5 fold to 84%. The hit rate increased most (4 fold) for MET, which historically was the lowest hit rate target at 9%, while the increase was the lowest at 2.2 fold (to 54%) for FLT3—historically the most promiscuous of the six kinase targets with a hit rate of 24%. Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>a shows the distribution of nearest neighbors (NN) using the fingerprint similarity (a Tanimoto similarity index using molecular properties, MACCS keys, and extended connectivity fingerprints<a onclick="showRef(event, 'ref7 ref25'); return false;" href="javascript:void(0);" class="ref ref7 ref25">(7, 25)</a>) to the actives used to derive the models of the compounds selected for the prospective experiment. The test set was designed to eliminate as many closely related compounds to the training set as possible, but a small number of closely related structures were also included (with similarity >0.9, where a value of 1 is identical) to test the similarity dependence of active predictions. Very few compounds from our collection with similarity <0.8 to known actives were predicted active in the models. The mean similarity of the selected set to existing actives was 0.82. We also examined the dependence of PPV as a function of similarity (NN similarity bins of >0.9, <0.8, and >0.8) and unexpectedly found that the PPVs for all targets were mostly independent of similarity. We note, however, that the test set contained very few compounds that are very similar to training set by design. Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>c shows a bioprint visual comparison (i.e., a heat map depiction of actives/inactives<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a>) of predicted activities with experimentally determined values for all prospective compounds. The comparison looks favorable for all targets and qualitatively confirms the values shown for the performance metrics.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Results from prospective testing. Distribution of nearest neighbor similarity of 1895 compounds with respect to 5870 historical actives. The fingerprints used for this study are a combination of molecular properties (atom count, size of largest ring, number of rings, number of ring atoms, number of aromatic atoms, number of fused ring atoms, number of heteroatoms), MACCS keys,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and extended connectivity fingerprints with size 3 shell. Tanimoto similarity in this fingerprint space between kinase drugs from Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> was the following: gefinitib−cedinarib 0.728, danusertib−cedinarib 0.592, danusertib−gefinitib 0.585. (a) Distribution of nearest neighbor similarity. (b) Comparison of historical hit rates with prospective validation. (c) Bioprint of predicted (top) and observed (bottom) activities at 20uM for 1895 compounds in 6 kinases. MET had 1392 measurements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Library Design Results</h3><div class="NLM_p">Armed with prospectively validated models for six targets (overall 84% hit rate, good PPV estimates, and sensitivities) high hit rates, although probably lower than obtained for the validation, was anticipated for the “de novo” libraries. This was the first time the models were applied to previously unexplored chemical space where a majority of compounds are less than 0.8, similar to the training set actives of the six kinase targets. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>a, the NN similarity distribution of library compounds averaged 0.74, much lower than the prospective validation set. On the other hand, all library compounds shared a common pharmacophore and hinge binding elements that were present in actives, although with different connections and R groups. Results for all libraries significantly exceeded expectations as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>b. In fact the overall hit rate of 92% exceeded even that of the prospective test set. Hit and enrichment rates for each target were also higher, with values ranging from 50% (CHK2) to 90% (FLT3). The largest increase was observed for ABL1, with >80% hit rate compared to 10% for historical compounds and 30% for prospective test set. Not surprisingly, a qualitative correspondence shown in the bioprint format is low, as the activity of many compounds were significantly under predicted in cases where compounds showed activity across all targets (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>c). While it is rewarding that the models led to active compounds, it appeared that the strategy and pharmacophore led to compounds with multiple kinase activities, i.e., promiscuous inhibitors. To learn from this experiment, the model performance metrics were contrasted for the library compounds and those from the prospective validation studies. The results are graphically displayed in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>a−d (and in <a class="ref internalNav" href="#notes-1" aria-label="Table 2 in Supporting Information">Table 2 in Supporting Information</a>). First, the mean prediction values are significantly higher for library compounds in four targets, with FLT3 and ROCK2 minimally lower, than predictions for the validation set. Standard deviations of these values are significantly lower for the libraries than the validation set, suggesting that the predictions for library compounds were higher and within a narrower range, i.e., most library compounds were predicted to be more active across the board. Model PPV values were significantly higher for ABL1, FLT3, and ROCK2, comparable for MET and P70S6K and lower than those found in the prospective set for CHK2. Sensitivities were notably lower for library compounds, with the largest differences for ROCK2 (0.63 vs 0.01 for libraries) and ABL1 (0.55 vs 0.11). The only target with comparable sensitivity between the library and validation sets was P70S6K (0.48 vs 0.46). Low sensitivities point to significant underpredictions of activity for all targets, representing a significant challenge in model utilization for selectivity design. As a consequence of narrower, higher predictions, the <i>R</i><sup>2</sup> values are no better than random indicators for model quality for all targets except ROCK2 (target with marginal sensitivity).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Results from library testing. (a) Distribution of nearest neighbor similarity of 179 compounds with respect to 6422 historical actives. (b) Comparison of historical hit rates with prospective validation and library hit rate. (c) Bioprint of predicted (top) and observed (bottom) activities at 20 μM for 179 compounds in six kinases. MET had 116 measurements..</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of model performance metrics for prospective validation and library compounds. (a) Mean model predictions and standard deviations for six targets. All differences in means are statistically significant except for FLT3. (b) PPVs and sensitivities. (c) Prediction rms. (d) <i>R</i><sup>2</sup> values: the low values for library compounds might be influenced by higher means for predicted values and lower standard deviations..</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Another way to look into the results for the prospective set and library compounds is to compute cross-target correlations for both experimental and predicted values. The results are presented in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> for all target pairs. Correlations between pairs of targets for the experimental values for 1895 prospective compounds is low, except for P70S6K and CHK2 (<i>R</i><sup>2</sup> = 0.35, <i>R</i> = 0.6). On the other hand, the 179 library compounds give higher correlations for P70S6K and MET (<i>R</i><sup>2</sup> = 0.6), FLT3 and ABL1 (<i>R</i><sup>2</sup> = 0.5), ROCK2 and MET (<i>R</i><sup>2</sup> = 0.44), ROCK2 and P70S6K (<i>R</i><sup>2</sup> = 0.37), and similar for P70S6K and CHK2 (<i>R</i><sup>2</sup> = 0.36). Thus, it appears that the pharmacophore used in the designed libraries increases the likelihood of getting similar potency for multiple pairs of targets. Interestingly, the models predict the highest correlations for FLT3 and CHK2 for library compounds—this is not observed in the experimental data. For library compounds, the models also predict high correlations for P70S6K against four other targets, with three of these confirmed by the experimental data. Most notably, the models underestimate the highest experimental correlations between P70S6K and MET. These results (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) show the difficulty of accurately estimating target relationships (and thus selectivity) for new data sets either from previous relationships or predictive models. In summary of the model performance, we noted that utilization of model combination strategy (selecting molecules predicted active in any of the models) worked well in designing active compounds, while we still remain significantly challenged in utilization of models to design selective compounds. It is likely that a different, more target specific pharmacophore, more similar chemotypes to existing actives, or a different set of targets could result in a more specific design, but this experiment highlights successes and challenges of good model driven de novo design.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Target Cross-Correlation <i>R</i><sup>2</sup> Tables for Experimental and Predicted % Inhibition Values<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col /></col></colgroup><tbody><tr valign="top"><td><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0007.gif" alt="" id="_i10" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last"><i>R</i><sup>2</sup> values are reported. Upper diagonal: experimental values, prospective <i>N</i> = 1895, except for MET <i>N</i> = 1392, library in parentheses <i>N</i> = 179, (MET <i>N</i> = 116). Lower diagonal: predicted values, <i>N</i> = 179, except for MET <i>N</i> = 116, library compound correlations in parentheses. Kinase group and gate keeper residues (in parentheses) are included in table headings. Sequence identities between six kinases are available in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. *<i>P</i> > 0.05.</p></div></div></div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selected Data for One Standard Library</h3><div class="NLM_p">As a part of our proof of concept, one library (library 6) was used as positive control. For this library, we used model driven substituent selection to make up compounds previously described as MET kinase inhibitors.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Interestingly, most compounds from this library showed rather low predicted activity as only 3 of 44 synthesized compounds were predicted to be higher than 70% percent in any of six kinases (all compounds were predicted >50%). This compares to 76% predicted actives in the other six libraries. As anticipated from the literature<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> all compounds from this library showed activity (>70% at 20 μM) in at least one kinase. A sample of data, including dose response, is shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>. To summarize, the data for these compounds highlights good translation of % inhibition to dose response for CHK2, underestimation of activity for this library, and little differentiation between % inhibition and the IC<sub>50</sub> values (necessitating determination of IC<sub>50</sub>s for compounds of specific interest). Compounds from this library exemplify the multikinase inhibition nature of other libraries designed in this study.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structures and Data for Example of Compounds from Library 6<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col /></col></colgroup><tbody><tr valign="top"><td><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0008.gif" alt="" id="fx1" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">Hydrophobic groups are drawn in red, solubilizing in green. Predicted values are shown in bottom row for each structure. Heat map color coding is used to denote the degree of predicted and observed inhibition. Dose response data for CHK2 and MET shown.</p></div></div></div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Analysis of Hydrophobic and Solubilizing Groups</h3><div class="NLM_p">To gain structural insights into the results of solubilizing and hydrophobic groups in our libraries, the data for compounds in which common solubilizing and hydrophobic groups were present were analyzed. Groups appearing in more than one library were identified to ensure the results were not dependent on a particular choice of a hinge binder. Parts a and b of Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> show mean % inhibition values for all compounds containing a particular group for the 12 solubilizing groups (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>a) and nine hydrophobic groups (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>b) examined. In addition, we examined the mean best target (the best activity for all targets) % inhibitions, mean number of atoms in compounds with that group, together with mean polar surface area (PSA)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and CLOGD<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> values. The groups are sorted based on their mean best target inhibition. Some solubilizing groups, like the phenyl sulfonamide piperazine, are highly potent across all targets. Others like meta- and ortho-substituted benzyl piperidines are more active in Abl1, FLT3 (above in 90%) but have low activity in MET. Meta-substituted phenyl piperazines appear to give the lowest activity likely due to steric hindrance. The smallest solubilizing group used, <i>para</i>-benzyl-<i>N</i>-Me-amine, shows mean activity >80.5% in MET and FLT3, with lowest activity in ROCK2. All solubilizing groups appear on average most potent in FLT3 as compared to the other five kinases.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structures and Experimentally Observed, Averaged % Inhibitions for Library Compounds: (a) Solubilizing Groups, (b) Hydrophobic Groups<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col /></col></colgroup><tbody><tr valign="top"><td><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0009.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0010.gif" alt="" id="tbl3_2" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup>a</sup><p class="last">Each depicted fragment had to be present in minimum of two libraries. Mean values are given for all targets and properties. Red to blue heat-map color coding with distinct colors for >90%, >70%, >50%, and <50%. Mean values for atom counts, CLOGD, and PSA are shown to give connection to physical properties. Connection atoms contain isotopic labels.</p></div></div></div><div class="NLM_p">In slight contrast to the solubilizing groups, the nine common hydrophobic groups show, on average, less promiscuity and more target differentiation. This may be understandable, as they target the gate keeper region which is often hypothesized to be responsible for selectivity.<a onclick="showRef(event, 'ref4 ref28'); return false;" href="javascript:void(0);" class="ref ref4 ref28">(4, 28)</a> However gate keeper identity is not likely to be the only determinant of hydrophobic group differentiation. While similarity between activity of hydrophobic groups exemplified in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>b for two Leu gate keeper targets (P70 and CHK2) is high, there is also high correlation of averaged activity of these groups between Abl (Thr gate keeper) and Flt3 (Phe gate keeper) as well as between ROCK2 (Met gate keeper) and P70S6K (Leu gate keeper).</div><div class="NLM_p">Hydrophobic groups in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>b  are sorted with respect to their mean maximum % inhibition. 4-Substituted-indole carries high (>90%) mean activity in Abl1 and FLT3, with lower values in MET, while 5-substituted indole is more ubiquitous across the board, with >90% mean inhibition in FLT3 and MET. Phenyl is quite similar for all targets with <70% mean inhibition only in ROCK2, but benzyl gives the highest mean % inhibition in MET across all targets and groups. CHK2 appears to have preference for 5-subsituted indole from all examined groups. Thus the CHK2 activity of compounds from library 6 exemplified in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> likely comes from the presence of the 5-substituted indole. The presence of halogens in the multiple hydrophobic groups exemplified here may play a role in potential halogen bonding<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> phenomenon in addition to providing steric and hydrophobic effect.</div><div class="NLM_p last">Mean values for atom counts, CLOGD, and polar surface area (PSA) are shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> to give connection to physical properties. Most structures are in favorable physical property space,<a onclick="showRef(event, 'ref5 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref5 ref30 ref31">(5, 30, 31)</a> with computed CLOGD values less than 3, a reasonable size with less than 35 heavy atoms and less than 120 PSA values.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have shown how model driven kinase inhibitor design can be used to probe kinase space with new inhibitors. It was demonstrated that highly validated models can be used in virtual screening and to design active, multikinase inhibitors. Our prospective results show more balanced predictability for all targets and better potential to probe selectivity for compounds closely related to training set compounds. The library results, which make use of a generic framework, showed that while this pharmacophore approach is good at uncovering kinase actives, it is challenged in terms of designing selective compounds. For instance, the inability to achieve good model sensitivity suggests that more work is needed to utilize this methodology to create selective tools for kinase target validation. Nevertheless, the group analysis described here sheds some light into the future design of more targeted inhibitors. In general, the ability to validate and use predictive modeling in blind/prospective experiments and de novo design of new compounds from existing information was demonstrated. To our knowledge, the overall virtual screening results show unprecedented success as compared to previous prospective literature reports.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Our de novo driven library results are even more encouraging when compared to reported computer driven designs, which utilized minor modifications of existing inhibitors with 30−40% success rate.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The implications of this research span fragment based drug discovery,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> selectivity considerations, and predictive modeling applications/expectations. Understanding the proper utilization of predictive models demonstrated by our work should raise the standards of new compound design not only for protein kinases but for other gene families for which sufficient high quality data is available.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17"><a href="/doi/suppl/10.1021/jm901147e">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65850" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65850" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">General statistics for models, historical hit rates, and prospective validation result; model performance metrics for prospective validation (1895 compounds) and libraries (179 compounds); target correlation table with sequence identity included; graphical demonstration of chronological validation and PPV concepts; biological assay description. Biological profiling data and compound data in fingerprint format are available from authors upon request. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm901147e/suppl_file/jm901147e_si_001.pdf">jm901147e_si_001.pdf (105.12 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm901147e" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48350" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48350" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michal Vieth</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly and Company Lilly Research Laboratories, Lilly Corporate Center, DC 1931, Indianapolis, Indiana 46285</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#64094a120d01100c24080d08081d4a070b09"><span class="__cf_email__" data-cfemail="3d50134b545849557d5154515144135e5250">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jon Erickson</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly and Company Lilly Research Laboratories, Lilly Corporate Center, DC 1931, Indianapolis, Indiana 46285</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jibo Wang</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly and Company Lilly Research Laboratories, Lilly Corporate Center, DC 1931, Indianapolis, Indiana 46285</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yue Webster</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly and Company Lilly Research Laboratories, Lilly Corporate Center, DC 1931, Indianapolis, Indiana 46285</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mary Mader</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly and Company Lilly Research Laboratories, Lilly Corporate Center, DC 1931, Indianapolis, Indiana 46285</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Higgs</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly and Company Lilly Research Laboratories, Lilly Corporate Center, DC 1931, Indianapolis, Indiana 46285</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian Watson</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly and Company Lilly Research Laboratories, Lilly Corporate Center, DC 1931, Indianapolis, Indiana 46285</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53600" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53600" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Drs. Sutherland, Robertson, Hemmerle, Toth, and Bloem, and the Lilly Library Sciences Group for supporting this research.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62505" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62505" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Grant, S. K.</span><span> </span><span class="NLM_article-title">Therapeutic Protein Kinase Inhibitors</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">1163</span><span class="NLM_x">–</span> <span class="NLM_lpage">1177</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1007%2Fs00018-008-8539-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=19011754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslWrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2009&pages=1163-1177&author=S.+K.+Grant&title=Therapeutic+Protein+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Protein Kinase Inhibitors</span></div><div class="casAuthors">Grant, S. K.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1163-1177</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Protein kinase inhibitors represent an important and still emerging class of targeted therapeutic agents.  Drug discovery and development strategies have explored numerous approaches to target the inhibition of protein kinase signaling.  This review will highlight some of the strategies that have led to the successful clin. development of therapeutic protein kinase inhibitors, particularly as anticancer drugs.  Some notable advances were made in the development of novel protein and oligonucleotide-based biologics that target growth factor or receptor tyrosine kinases.  Also, advances were made in the rational design of small-mol. inhibitors that target unique kinase conformational forms and binding sites, and have specific kinase selectivity profiles.  A review will also be given of some of the potential clin. toxicities and adverse side-effects assocd. with these kinase-targeted drugs.  Therapeutic protein kinase inhibitors were highly beneficial to cancer patients and offer the promise of future therapies for other diseases as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooJMLRyntX7bVg90H21EOLACvtfcHk0lj0s74mFNrQEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslWrsbk%253D&md5=1fe445348168f5632b4c47b4ce27fd6e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fs00018-008-8539-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-008-8539-7%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DS.%2BK.%26atitle%3DTherapeutic%2520Protein%2520Kinase%2520Inhibitors%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2009%26volume%3D66%26spage%3D1163%26epage%3D1177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Echeverria, C. G.</span><span> </span><span class="NLM_article-title">Targeting protein kinases in cancer therapy</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=12630290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnsFekuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2002&pages=701-712&author=D.+Fabbroauthor=C.+G.+Garcia-Echeverria&title=Targeting+protein+kinases+in+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein kinases in cancer therapy</span></div><div class="casAuthors">Fabbro, Doriano; Garcia-Echeverria, Carlos Garcia</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">701-712</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">PharmaPress Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the years, many different tyrosine and serine/threonine protein kinases have been selected as candidates for drug discovery activities in oncol. research, based either on their overexpression and/or dysfunction in a particular organ or tissue, or through their assocn. in deregulated signal transduction/cell cycle pathways.  This review summarizes current preclin. and clin. knowledge of ATP-competitive, small mol. kinase inhibitors, and receptor or ligand-competitive antibodies of selected protein kinases in which drug discovery and development activities have advanced with some success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNWx-SB9X157Vg90H21EOLACvtfcHk0lgfIXokJv-v6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnsFekuw%253D%253D&md5=e0470671e56759bfbeefc0f84ff939be</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGarcia-Echeverria%26aufirst%3DC.%2BG.%26atitle%3DTargeting%2520protein%2520kinases%2520in%2520cancer%2520therapy%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2002%26volume%3D5%26spage%3D701%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule−kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule%E2%88%92kinase+interaction+map+for+clinical+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule%25E2%2588%2592kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Gill, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. A</span><span> </span><span class="NLM_article-title">Comparison of Physicochemical Property Profiles of Marketed Oral Drugs and Orally Bioavailable Anti-Cancer Protein Kinase Inhibitors in Clinical Development</span> <span class="citation_source-journal">Curr Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1408</span><span class="NLM_x">–</span> <span class="NLM_lpage">1422</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.2174%2F156802607781696819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=17692029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1SmtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1408-1422&author=A.+L.+Gillauthor=M.+Verdonkauthor=R.+G.+Boyleauthor=R.+A+Taylor&title=Comparison+of+Physicochemical+Property+Profiles+of+Marketed+Oral+Drugs+and+Orally+Bioavailable+Anti-Cancer+Protein+Kinase+Inhibitors+in+Clinical+Development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A comparison of physicochemical property profiles of marketed oral drugs and orally bioavailable anti-cancer protein kinase inhibitors in clinical development</span></div><div class="casAuthors">Gill, Adrian L.; Verdonk, Marcel; Boyle, Robert G.; Taylor, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1408-1422</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  This manuscript describes a comparison of the physicochem. properties of marketed oral drugs with those of 45 structurally confirmed orally bioavailable anti-cancer protein kinase inhibitors currently in different phases of clin. development.  It is evident from the data presented that these kinase inhibitors are on av. larger (over 110Da), more lipophilic (over 1.5 log units) and more complex (approx. two more rotatable bonds) than those of marketed oral drugs.  In contrast, hydrogen bond donor (HBD) and hydrogen bond acceptor (HBA) counts are not significantly different.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY8vw6yZzFirVg90H21EOLACvtfcHk0lgfIXokJv-v6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1SmtbvM&md5=03b0a1943f8d1a08ed436da75e9a296f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F156802607781696819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607781696819%26sid%3Dliteratum%253Aachs%26aulast%3DGill%26aufirst%3DA.%2BL.%26aulast%3DVerdonk%26aufirst%3DM.%26aulast%3DBoyle%26aufirst%3DR.%2BG.%26aulast%3DTaylor%26aufirst%3DR.%2BA%26atitle%3DComparison%2520of%2520Physicochemical%2520Property%2520Profiles%2520of%2520Marketed%2520Oral%2520Drugs%2520and%2520Orally%2520Bioavailable%2520Anti-Cancer%2520Protein%2520Kinase%2520Inhibitors%2520in%2520Clinical%2520Development%26jtitle%3DCurr%2520Top.%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D1408%26epage%3D1422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegel, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savin, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durst, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hipskind, P. A.</span><span> </span><span class="NLM_article-title">Characteristic physical properties and structural fragments of marketed oral drugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">232</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030267j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=224-232&author=M.+Viethauthor=M.+G.+Siegelauthor=R.+E.+Higgsauthor=I.+A.+Watsonauthor=D.+H.+Robertsonauthor=K.+A.+Savinauthor=G.+L.+Durstauthor=P.+A.+Hipskind&title=Characteristic+physical+properties+and+structural+fragments+of+marketed+oral+drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm030267j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030267j%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DSiegel%26aufirst%3DM.%2BG.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DSavin%26aufirst%3DK.%2BA.%26aulast%3DDurst%26aufirst%3DG.%2BL.%26aulast%3DHipskind%26aufirst%3DP.%2BA.%26atitle%3DCharacteristic%2520physical%2520properties%2520and%2520structural%2520fragments%2520of%2520marketed%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D224%26epage%3D232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span> </span><span class="NLM_article-title">Kinomics: characterizing the therapeutically validated kinase space</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">839</span><span class="NLM_x">–</span> <span class="NLM_lpage">46</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=839-46&author=M.+Viethauthor=J.+J.+Sutherlandauthor=D.+H.+Robertsonauthor=R.+M.+Campbell&title=Kinomics%3A+characterizing+the+therapeutically+validated+kinase+space"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DKinomics%253A%2520characterizing%2520the%2520therapeutically%2520validated%2520kinase%2520space%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D839%26epage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Chemical fragments as foundations for understanding target space and activity prediction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2689</span><span class="NLM_x">–</span> <span class="NLM_lpage">2700</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701399f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2689-2700&author=J.+J.+Sutherlandauthor=R.+E.+Higgsauthor=I.+Watsonauthor=M.+Vieth&title=Chemical+fragments+as+foundations+for+understanding+target+space+and+activity+prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm701399f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701399f%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DWatson%26aufirst%3DI.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DChemical%2520fragments%2520as%2520foundations%2520for%2520understanding%2520target%2520space%2520and%2520activity%2520prediction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2689%26epage%3D2700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0li5yD-scrD2yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Fernandez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddipati, S.</span><span> </span><span class="NLM_article-title">A priori inference of cross reactivity for drug-targeted kinases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">3092</span><span class="NLM_x">–</span> <span class="NLM_lpage">3100</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060163j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3092-3100&author=A.+Fernandezauthor=S.+Maddipati&title=A+priori+inference+of+cross+reactivity+for+drug-targeted+kinases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm060163j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060163j%26sid%3Dliteratum%253Aachs%26aulast%3DFernandez%26aufirst%3DA.%26aulast%3DMaddipati%26aufirst%3DS.%26atitle%3DA%2520priori%2520inference%2520of%2520cross%2520reactivity%2520for%2520drug-targeted%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3092%26epage%3D3100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapland, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoorn, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Global mapping of pharmacological space</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">805</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&author=G.+V.+Paoliniauthor=R.+H.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+mapping+of+pharmacological+space"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0li5yD-scrD2yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520mapping%2520of%2520pharmacological%2520space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Liew, C. Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, C. W.</span><span> </span><span class="NLM_article-title">SVM Model for Virtual Screening of Lck Inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">877</span><span class="NLM_x">–</span> <span class="NLM_lpage">885</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800387z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXislOiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=877-885&author=C.+Y.+M.+Liewauthor=H.+Xiaoauthor=X.+Liuauthor=C.+W.+Yap&title=SVM+Model+for+Virtual+Screening+of+Lck+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">SVM Model for Virtual Screening of Lck Inhibitors</span></div><div class="casAuthors">Liew, Chin Y.; Ma, Xiao H.; Liu, Xianghui; Yap, Chun W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">877-885</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lymphocyte-specific protein tyrosine kinase (Lck) inhibitors have treatment potential for autoimmune diseases and transplant rejection.  A support vector machine (SVM) model trained with 820 pos. compds. (Lck inhibitors) and 70 neg. compds. (Lck noninhibitors) combined with 65 142 generated putative negatives was developed for predicting compds. with a Lck inhibitory activity of IC50 ≤ 10 μM.  The SVM model, with an estd. sensitivity of greater than 83% and specificity of greater than 99%, was used to screen 168 014 compds. in the MDDR and was found to have a yield of 45.8% and a false pos. rate of 0.52%.  The model was also able to identify novel Lck inhibitors and distinguish inhibitors from structurally similar noninhibitors at a false pos. rate of 0.27%.  To the best of our knowledge, the SVM model developed in this work is the first model with a broad applicability domain and low false pos. rate, which makes it very suitable for the virtual screening of chem. libraries for Lck inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRGGWnqtxkzrVg90H21EOLACvtfcHk0ljMEU_MpMdkCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXislOiurw%253D&md5=1fbd326f057f173f26b4a2231b5f3cfb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fci800387z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800387z%26sid%3Dliteratum%253Aachs%26aulast%3DLiew%26aufirst%3DC.%2BY.%2BM.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYap%26aufirst%3DC.%2BW.%26atitle%3DSVM%2520Model%2520for%2520Virtual%2520Screening%2520of%2520Lck%2520Inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D877%26epage%3D885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Kitchen, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decornez, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furr, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Docking and Scoring in Virtual Screening for Drug Discovery: Methods and Applications</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">949</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnrd1549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=15520816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=935-949&author=D.+B.+Kitchenauthor=H.+Decornezauthor=J.+R.+Furrauthor=J.+Bajorath&title=Docking+and+Scoring+in+Virtual+Screening+for+Drug+Discovery%3A+Methods+and+Applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and scoring in virtual screening for drug discovery: methods and applications</span></div><div class="casAuthors">Kitchen, Douglas B.; Decornez, Helene; Furr, John R.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">935-949</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Computational approaches that 'dock' small mols. into the structures of macromol. targets and 'score' their potential complementarity to binding sites are widely used in hit identification and lead optimization.  Indeed, there are now a no. of drugs whose development was heavily influenced by or based on structure-based design and screening strategies, such as HIV protease inhibitors.  Nevertheless, there remain significant challenges in the application of these approaches, in particular in relation to current scoring schemes.  Here, we review key concepts and specific features of small-mol.-protein docking methods, highlight selected applications and discuss recent advances that aim to address the acknowledged limitations of established approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMLibgFe3mfLVg90H21EOLACvtfcHk0ljMEU_MpMdkCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D&md5=875a2b37a4299181509a1922b11dbd2f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd1549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1549%26sid%3Dliteratum%253Aachs%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DDecornez%26aufirst%3DH.%26aulast%3DFurr%26aufirst%3DJ.%2BR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDocking%2520and%2520Scoring%2520in%2520Virtual%2520Screening%2520for%2520Drug%2520Discovery%253A%2520Methods%2520and%2520Applications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D935%26epage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Virtual screening of chemical libraries</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">862</span><span class="NLM_x">–</span> <span class="NLM_lpage">865</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnature03197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=15602552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=862-865&author=B.+K.+Shoichet&title=Virtual+screening+of+chemical+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of chemical libraries</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">862-865</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Virtual screening uses computer-based methods to discover new ligands on the basis of biol. structures.  Although widely heralded in the 1970s and 1980s, the technique has since struggled to meet its initial promise, and drug discovery remains dominated by empirical screening.  Recent successes in predicting new ligands and their receptor-bound structures, and better rates of ligand discovery compared to empirical screening, have reignited interest in virtual screening, which is now widely used in drug discovery, albeit on a more limited scale than empirical screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY9IbbFldkrVg90H21EOLACvtfcHk0ljMEU_MpMdkCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK&md5=e351e226e8985d7c304a05fb67dfdc60</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature03197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03197%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DVirtual%2520screening%2520of%2520chemical%2520libraries%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D862%26epage%3D865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Vieth, M.; Siegel, M.</span><span> </span><span class="NLM_article-title">Structural fragments in marketed oral drugs</span>. In  <span class="citation_source-book">Fragment Based Drug Discovery</span>; <span class="NLM_contrib-group">Erlanson, J.</span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">123</span>− <span class="NLM_lpage">135</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=123-135&author=M.+Vieth&author=M.+Siegelauthor=J.+Erlanson&title=Fragment+Based+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DStructural%2520fragments%2520in%2520marketed%2520oral%2520drugs%26btitle%3DFragment%2520Based%2520Drug%2520Discovery%26aulast%3DErlanson%26aufirst%3DJ.%26pub%3DWiley-VCH%26date%3D2006%26spage%3D123%26epage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Joachims, T.</span><span> </span><span class="NLM_article-title">Making Large-Scale SVM Learning Practical</span>. In  <span class="citation_source-book">Advances in Kernel Methods—Support Vector Learning</span>; <span class="NLM_contrib-group">Schölkopf, B.; Burges, C.; Smola, A.</span>, Eds.; <span class="NLM_publisher-name">MIT Press</span>: <span class="NLM_publisher-loc">Cambridge</span>,<span class="NLM_x"> </span><span class="NLM_year">1999</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&author=T.+Joachimsauthor=B.+Sch%C3%B6lkopf&author=C.+Burges&author=A.+Smola&title=Advances+in+Kernel+Methods%E2%80%94Support+Vector+Learning"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJoachims%26aufirst%3DT.%26atitle%3DMaking%2520Large-Scale%2520SVM%2520Learning%2520Practical%26btitle%3DAdvances%2520in%2520Kernel%2520Methods%25E2%2580%2594Support%2520Vector%2520Learning%26aulast%3DSch%25C3%25B6lkopf%26aufirst%3DB.%26pub%3DMIT%2520Press%26date%3D1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Carhart, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkataraghavan, R.</span><span> </span><span class="NLM_article-title">Atom pairs as molecular features in structure−activity studies: definition and applications</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">64</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci00046a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADyaL2MXitVehu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1985&pages=64-73&author=R.+E.+Carhartauthor=D.+H.+Smithauthor=R.+Venkataraghavan&title=Atom+pairs+as+molecular+features+in+structure%E2%88%92activity+studies%3A+definition+and+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Atom pairs as molecular features in structure-activity studies:  definition and applications</span></div><div class="casAuthors">Carhart, Raymond E.; Smith, Dennis H.; Venkataraghavan, R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">64-73</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">A simple type of substructure called an atom pair is defined in terms of the at. environments of, and shortest path sepns. between, all pairs of atoms in the topol. representation of a chem. structure.  An algorithm is presented for computing atom pairs from such a representation.  Two applications of atom pairs to structure-activity problems are described.  In the 1st, a measure of similarity between compds. is defined, and the use of this measure in probing large data bases of structures is discussed.  In the 2nd, an heuristic technique called trend vector anal. is described.  The trend vector summarizes the correlation, within a set of structures, of the occurrence of atom pairs of different types with measured biol. activity.  These correlations can be used to est. the biol. activity of new compds.  A comparison of trend vector anal. with discriminant plane anal. is presented for 1 series of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV9lxLct8p0rVg90H21EOLACvtfcHk0lhQw2LgwXXb9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXitVehu7o%253D&md5=9e2cc9e9fda27db87328b738cd6878e2</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci00046a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00046a002%26sid%3Dliteratum%253Aachs%26aulast%3DCarhart%26aufirst%3DR.%2BE.%26aulast%3DSmith%26aufirst%3DD.%2BH.%26aulast%3DVenkataraghavan%26aufirst%3DR.%26atitle%3DAtom%2520pairs%2520as%2520molecular%2520features%2520in%2520structure%25E2%2588%2592activity%2520studies%253A%2520definition%2520and%2520applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1985%26volume%3D25%26spage%3D64%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Doucet, J.-P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panaye, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nonlinear, B.</span><span> </span><span class="NLM_article-title">SVM Approaches to QSPR/QSAR Studies and Drug Design</span> <span class="citation_source-journal">Curr. Comput.-Aided Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">289</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.2174%2F157340907782799372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVSqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=263-289&author=J.-P.+B.+Doucetauthor=F.+Xiaauthor=H.+Panayeauthor=A.+Fanauthor=B.+Nonlinear&title=SVM+Approaches+to+QSPR%2FQSAR+Studies+and+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nonlinear SVM approaches to QSPR/QSAR studies and drug design</span></div><div class="casAuthors">Doucet, Jean-Pierre; Barbault, Florent; Xia, Hairong; Panaye, Annick; Fan, Botao</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-289</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Recently, a new promising nonlinear method, the support vector machine (SVM), was proposed by Vapnik.  It rapidly found numerous applications in chem., biochem. and pharmacochem.  Several attempts using SVM in drug design have been reported.  It became an attractive nonlinear approach in this field.  In this review, the theor. basis of SVM in classification and regression is briefly described.  Its applications in QSPR/QSAR studies, and particularly in drug design are discussed.  Comparative studies with some linear and other nonlinear methods show SVM's high performance both in classification and correlation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlUWphlsyC4rVg90H21EOLACvtfcHk0lhQw2LgwXXb9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVSqtLk%253D&md5=5552439113da40b712c3467d7277bf95</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F157340907782799372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340907782799372%26sid%3Dliteratum%253Aachs%26aulast%3DDoucet%26aufirst%3DJ.-P.%2BB.%26aulast%3DXia%26aufirst%3DF.%26aulast%3DPanaye%26aufirst%3DH.%26aulast%3DFan%26aufirst%3DA.%26aulast%3DNonlinear%26aufirst%3DB.%26atitle%3DSVM%2520Approaches%2520to%2520QSPR%252FQSAR%2520Studies%2520and%2520Drug%2520Design%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2007%26volume%3D3%26spage%3D263%26epage%3D289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Han, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xuec, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Z. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. Z.</span><span> </span><span class="NLM_article-title">A support vector machines approach for virtual screening of active compounds of single and multiple mechanisms from large libraries at an improved hit-rate and enrichment factor</span> <span class="citation_source-journal">J. Mol. Graph. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1276</span><span class="NLM_x">–</span> <span class="NLM_lpage">1286</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1016%2Fj.jmgm.2007.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=18218332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvVWntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=1276-1286&author=L.+Y.+Hanauthor=X.+H.+Maauthor=H.+H.+Linauthor=J.+Jiaauthor=F.+Zhuauthor=Y.+Xuecauthor=Z.+R.+Liauthor=Z.+W.+Caoauthor=Z.+L.+Jiauthor=Y.+Z.+Chen&title=A+support+vector+machines+approach+for+virtual+screening+of+active+compounds+of+single+and+multiple+mechanisms+from+large+libraries+at+an+improved+hit-rate+and+enrichment+factor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A support vector machines approach for virtual screening of active compounds of single and multiple mechanisms from large libraries at an improved hit-rate and enrichment factor</span></div><div class="casAuthors">Han, L. Y.; Ma, X. H.; Lin, H. H.; Jia, J.; Zhu, F.; Xue, Y.; Li, Z. R.; Cao, Z. W.; Ji, Z. L.; Chen, Y. Z.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1276-1286</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Support vector machines (SVM) and other machine-learning (ML) methods have been explored as ligand-based virtual screening (VS) tools for facilitating lead discovery.  While exhibiting good hit selection performance, in screening large compd. libraries, these methods tend to produce lower hit-rate than those of the best performing VS tools, partly because their training-sets contain limited spectrum of inactive compds.  We tested whether the performance of SVM can be improved by using training-sets of diverse inactive compds.  In retrospective database screening of active compds. of single mechanism (HIV protease inhibitors, DHFR inhibitors, dopamine antagonists) and multiple mechanisms (CNS active agents) from large libraries of 2.986 million compds., the yields, hit-rates, and enrichment factors of our SVM models are 52.4-78.0%, 4.7-73.8%, and 214-10,543, resp., compared to those of 62-95%, 0.65-35%, and 20-1200 by structure-based VS and 55-81%, 0.2-0.7%, and 110-795 by other ligand-based VS tools in screening libraries of ≥1 million compds.  The hit-rates are comparable and the enrichment factors are substantially better than the best results of other VS tools.  Of the predicted hits, 24.3-87.6% are outside the known hit families.  SVM appears to be potentially useful for facilitating lead discovery in VS of large compd. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxffgspRhPqrVg90H21EOLACvtfcHk0lhQw2LgwXXb9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvVWntrk%253D&md5=ea010461c34a589f68e1f5447dbc13df</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2007.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2007.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DL.%2BY.%26aulast%3DMa%26aufirst%3DX.%2BH.%26aulast%3DLin%26aufirst%3DH.%2BH.%26aulast%3DJia%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DXuec%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DCao%26aufirst%3DZ.%2BW.%26aulast%3DJi%26aufirst%3DZ.%2BL.%26aulast%3DChen%26aufirst%3DY.%2BZ.%26atitle%3DA%2520support%2520vector%2520machines%2520approach%2520for%2520virtual%2520screening%2520of%2520active%2520compounds%2520of%2520single%2520and%2520multiple%2520mechanisms%2520from%2520large%2520libraries%2520at%2520an%2520improved%2520hit-rate%2520and%2520enrichment%2520factor%26jtitle%3DJ.%2520Mol.%2520Graph.%2520Model.%26date%3D2008%26volume%3D26%26spage%3D1276%26epage%3D1286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Gedeck, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R. A.</span><span> </span><span class="NLM_article-title">Exploiting QSAR models in lead optimization</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">575</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=18600573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosFWmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=569-575&author=P.+Gedeckauthor=R.+A.+Lewis&title=Exploiting+QSAR+models+in+lead+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting QSAR models in lead optimization</span></div><div class="casAuthors">Gedeck, Peter; Lewis, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">569-575</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  QSAR models can play a vital role in both the opening phase and the endgame of lead optimization.  In the opening phase, there is often a large quantity of data from high-throughput screening (HTS), and potential leads need to be selected from several distinct chemotypes.  In the endgame, the throughput of the final, crit. ADMET and pharmacokinetic assays is often not sufficient to allow full exptl. characterization of all the structures in the available time.  A considerable amt. of the current research toward new QSAR models is based on the modeling of the general ADMET phenomena, with the aim of constructing globally applicable models.  The process to construct QSAR models is relatively straightforward; however, it is also simple to build misleading, or even incorrect, models.  This review considers the key developments in the field of QSAR modeling: how QSAR models are constructed, how they can be validated, their reliability and their applicability.  If applied carefully and appropriately, the QSAR technique has a valuable role to play during lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGgUjjqE55JbVg90H21EOLACvtfcHk0lgysQ4xMTtFmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosFWmurc%253D&md5=bf4d2d141bda32bf4fc1b01d1bd46b57</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGedeck%26aufirst%3DP.%26aulast%3DLewis%26aufirst%3DR.%2BA.%26atitle%3DExploiting%2520QSAR%2520models%2520in%2520lead%2520optimization%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2008%26volume%3D11%26spage%3D569%26epage%3D575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Variable selection QSAR modeling, model validation, and virtual screening</span> <span class="citation_source-journal">Annu Rep. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1016%2FS1574-1400%2806%2902007-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjslWru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=113-126&author=A.+Tropsha&title=Variable+selection+QSAR+modeling%2C+model+validation%2C+and+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Variable selection QSAR modeling, model validation, and virtual screening</span></div><div class="casAuthors">Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Computational Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113-126</span>CODEN:
                <span class="NLM_cas:coden">ARCCC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-1400</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review on the quant. structure-activity relationship model validation as an integral component of model development.  It illustrates that rigorously validated QSAR models with statistically significant external predictive power ultimately afford their application as reliable virtual screening tools for database mining or chem. library design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquHgZ4ShzkybVg90H21EOLACvtfcHk0lgysQ4xMTtFmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjslWru7g%253D&md5=ebc18b6f2f01205d32ee69ed7f2e1ea2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS1574-1400%2806%2902007-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1574-1400%252806%252902007-X%26sid%3Dliteratum%253Aachs%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DVariable%2520selection%2520QSAR%2520modeling%252C%2520model%2520validation%252C%2520and%2520virtual%2520screening%26jtitle%3DAnnu%2520Rep.%2520Comput.%2520Chem.%26date%3D2006%26volume%3D2%26spage%3D113%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nandigam, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erickson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Lessons in Molecular Recognition. 2. Assessing and Improving Cross-Docking Accuracy</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2293</span><span class="NLM_x">–</span> <span class="NLM_lpage">2302</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700253h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1SgtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2293-2302&author=J.+J.+Sutherlandauthor=R.+K.+Nandigamauthor=J.+A.+Ericksonauthor=M.+Vieth&title=Lessons+in+Molecular+Recognition.+2.+Assessing+and+Improving+Cross-Docking+Accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in Molecular Recognition. 2. Assessing and Improving Cross-Docking Accuracy</span></div><div class="casAuthors">Sutherland, Jeffrey J.; Nandigam, Ravi K.; Erickson, Jon A.; Vieth, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2293-2302</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking methods are used to predict the manner in which a ligand binds to a protein receptor.  Many studies have assessed the success rate of programs in self-docking tests, whereby a ligand is docked into the protein structure from which it was extd.  Cross-docking, or using a protein structure from a complex contg. a different ligand, provides a more realistic assessment of a docking program's ability to reproduce X-ray results.  Cross-docking was performed with CDocker, Fred, and Rocs using multiple X-ray structures for eight proteins (two kinases, one nuclear hormone receptor, one serine protease, two metalloproteases, and two phosphodiesterases).  While av. cross-docking accuracy is not encouraging, it is shown that using the protein structure from the complex that contains the bound ligand most similar to the docked ligand increases docking accuracy for all methods ("similarity selection").  Identifying the most successful protein conformer ("best selection") and similarity selection substantially reduce the difference between self-docking and av. cross-docking accuracy.  The authors identify universal predictors of docking accuracy (i.e., showing consistent behavior across most protein-method combinations), and show that models for predicting docking accuracy built using these parameters can be used to select the most appropriate docking method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5Nh5Ky1zOc7Vg90H21EOLACvtfcHk0lgysQ4xMTtFmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1SgtbbP&md5=c3792ab8645ab459941f5d3a223f9ce4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci700253h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700253h%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DNandigam%26aufirst%3DR.%2BK.%26aulast%3DErickson%26aufirst%3DJ.%2BA.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DLessons%2520in%2520Molecular%2520Recognition.%25202.%2520Assessing%2520and%2520Improving%2520Cross-Docking%2520Accuracy%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2293%26epage%3D2302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Nandigam, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erickson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, J. J.</span><span> </span><span class="NLM_article-title">Predicting the accuracy of ligand overlay methods with Random Forest models</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2386</span><span class="NLM_x">–</span> <span class="NLM_lpage">2394</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800216f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSntbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=2386-2394&author=R.+K.+Nandigamauthor=D.+A.+Evansauthor=J.+A.+Ericksonauthor=S.+Kimauthor=J.+J.+Sutherland&title=Predicting+the+accuracy+of+ligand+overlay+methods+with+Random+Forest+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting the accuracy of ligand overlay methods with Random Forest models</span></div><div class="casAuthors">Nandigam, Ravi K.; Evans, David A.; Erickson, Jon A.; Kim, Sangtae; Sutherland, Jeffrey J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2386-2394</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accuracy of binding mode prediction using std. mol. overlay methods (ROCS, FlexS, Phase, and FieldCompare) is studied.  Previous work has shown that simple decision tree modeling can be used to improve accuracy by selection of the best overlay template.  This concept is extended to the use of Random Forest (RF) modeling for template and algorithm selection.  An extensive data set of 815 ligand-bound X-ray structures representing 5 gene families was used for generating ca. 70,000 overlays using four programs.  RF models, trained using std. measures of ligand and protein similarity and Lipinski-related descriptors, are used for automatically selecting the ref. ligand and overlay method maximizing the probability of reproducing the overlay deduced from X-ray structures (i.e., using rmsd ≤ 2 Å as the criteria for success).  RF model scores are highly predictive of overlay accuracy, and their use in template and method selection produces correct overlays in 57% of cases for 349 overlay ligands not used for training RF models.  The inclusion in the models of protein sequence similarity enables the use of templates bound to related protein structures, yielding useful results even for proteins having no available X-ray structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbbC86gHhVarVg90H21EOLACvtfcHk0lgysQ4xMTtFmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSntbrE&md5=c1e6661350b640f98073b753017b9c23</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci800216f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800216f%26sid%3Dliteratum%253Aachs%26aulast%3DNandigam%26aufirst%3DR.%2BK.%26aulast%3DEvans%26aufirst%3DD.%2BA.%26aulast%3DErickson%26aufirst%3DJ.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26atitle%3DPredicting%2520the%2520accuracy%2520of%2520ligand%2520overlay%2520methods%2520with%2520Random%2520Forest%2520models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D2386%26epage%3D2394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Dooley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houghten, R.</span><span> </span><span class="NLM_article-title">The use of positional scanning synthetic peptide combinatorial libraries for the rapid determination of opioid receptor ligands</span> <span class="citation_source-journal">Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1509</span><span class="NLM_x">–</span> <span class="NLM_lpage">1517</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1993&pages=1509-1517&author=C.+Dooleyauthor=R.+Houghten&title=The+use+of+positional+scanning+synthetic+peptide+combinatorial+libraries+for+the+rapid+determination+of+opioid+receptor+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDooley%26aufirst%3DC.%26aulast%3DHoughten%26aufirst%3DR.%26atitle%3DThe%2520use%2520of%2520positional%2520scanning%2520synthetic%2520peptide%2520combinatorial%2520libraries%2520for%2520the%2520rapid%2520determination%2520of%2520opioid%2520receptor%2520ligands%26jtitle%3DLife%2520Sci.%26date%3D1993%26volume%3D52%26spage%3D1509%26epage%3D1517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Cui, J.; Funk, L. A.; Jia, L.; Kung, P.-P.; Meng, J. J.; Nambu, M. D.; Pairish, M. A.; Sheng, H.; Tran-Dube, M. B.</span><span> </span><span class="NLM_article-title">Aminoheteroaryl compounds as protein tyrosine kinase inhibitors</span>. PCT Gazette, WO 2006/021886 Al,<span class="NLM_x"> </span><span class="NLM_year">2006</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=J.+Cui&author=L.+A.+Funk&author=L.+Jia&author=P.-P.+Kung&author=J.+J.+Meng&author=M.+D.+Nambu&author=M.+A.+Pairish&author=H.+Sheng&author=M.+B.+Tran-Dube&title=Aminoheteroaryl+compounds+as+protein+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%26atitle%3DAminoheteroaryl%2520compounds%2520as%2520protein%2520tyrosine%2520kinase%2520inhibitors%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Givehchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span> </span><span class="NLM_article-title">Analysis of activity space by fragment fingerprints, 2D descriptors, and multitarget dependent transformation of 2D descriptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1078</span><span class="NLM_x">–</span> <span class="NLM_lpage">1083</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0500233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvF2is7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1078-1083&author=A.+Givehchiauthor=A.+Benderauthor=R.+C.+Glen&title=Analysis+of+activity+space+by+fragment+fingerprints%2C+2D+descriptors%2C+and+multitarget+dependent+transformation+of+2D+descriptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Activity Space by Fragment Fingerprints, 2D Descriptors, and Multitarget Dependent Transformation of 2D Descriptors</span></div><div class="casAuthors">Givehchi, Alireza; Bender, Andreas; Glen, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1078-1083</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effect of multitarget dependent descriptor transformation on classification performance is explored in this work.  To this end decision trees as well as neural net QSAR in combination with PLS were applied to predict the activity class of 5HT3 ligands, angiotensin converting enzyme inhibitors, 3-hydroxyl-3-Me glutaryl CoA reductase inhibitors, platelet activating factor antagonists, and thromboxane A2 antagonists.  Physicochem. descriptors calcd. by MOE and fragment-based descriptors (MOLPRINT 2D) were employed to generate descriptor vectors.  In a subsequent step the physicochem. descriptor vectors were transformed to a lower dimensional space using multitarget dependent descriptor transformation.  Cross-validation of the original physicochem. descriptors in combination with decision trees and neural net QSAR as well as cross-validation of PLS multitarget transformed descriptors with neural net QSAR were performed.  For comparison this was repeated using fragment-based descriptors in combination with decision trees.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoalnrrxP0_5bVg90H21EOLACvtfcHk0lj8H2vsVfCUAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvF2is7g%253D&md5=8f8c84590d6d36471687162e652f9d43</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci0500233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0500233%26sid%3Dliteratum%253Aachs%26aulast%3DGivehchi%26aufirst%3DA.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26atitle%3DAnalysis%2520of%2520activity%2520space%2520by%2520fragment%2520fingerprints%252C%25202D%2520descriptors%252C%2520and%2520multitarget%2520dependent%2520transformation%2520of%25202D%2520descriptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D1078%26epage%3D1083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohde, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selzer, P.</span><span> </span><span class="NLM_article-title">Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport properties</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3727</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000942e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3714-3727&author=P.+Ertlauthor=B.+Rohdeauthor=P.+Selzer&title=Fast+calculation+of+molecular+polar+surface+area+as+a+sum+of+fragment-based+contributions+and+its+application+to+the+prediction+of+drug+transport+properties"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of Molecular Polar Surface Area as a Sum of Fragment-Based Contributions and Its Application to the Prediction of Drug Transport Properties</span></div><div class="casAuthors">Ertl, Peter; Rohde, Bernhard; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3714-3717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. polar surface area (PSA), i.e., surface belonging to polar atoms, is a descriptor that was shown to correlate well with passive mol. transport through membranes and, therefore, allows prediction of transport properties of drugs.  The calcn. of PSA, however, is rather time-consuming because of the necessity to generate a reasonable 3D mol. geometry and the calcn. of the surface itself.  A new approach for the calcn. of the PSA is presented here, based on the summation of tabulated surface contributions of polar fragments.  The method, termed topol. PSA (TPSA), provides results which are practically identical with the 3D PSA (the correlation coeff. between 3D PSA and fragment-based TPSA for 34 810 mols. from the World Drug Index is 0.99), while the computation speed is 2-3 orders of magnitude faster.  The new methodol. may, therefore, be used for fast bioavailability screening of virtual libraries having millions of mols.  This article describes the new methodol. and shows the results of validation studies based on sets of published absorption data, including intestinal absorption, Caco-2 monolayer penetration, and blood-brain barrier penetration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhP6Qncx_h6LVg90H21EOLACvtfcHk0lj8H2vsVfCUAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D&md5=23ae4e7708bd8ddeb4a24ad160a29bd6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm000942e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000942e%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRohde%26aufirst%3DB.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DFast%2520calculation%2520of%2520molecular%2520polar%2520surface%2520area%2520as%2520a%2520sum%2520of%2520fragment-based%2520contributions%2520and%2520its%2520application%2520to%2520the%2520prediction%2520of%2520drug%2520transport%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3714%26epage%3D3727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span> <span class="citation_source-book">Chemaxon LogD</span>; <span class="NLM_publisher-name">ChemAxon Kft</span>: <span class="NLM_publisher-loc">Budapest, 1037 Hungary</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span><span class="NLM_x">; </span><a href="http://www.chemaxon.com" class="extLink">www.chemaxon.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Chemaxon+LogD%3B+ChemAxon+Kft%3A+Budapest%2C+1037+Hungary%2C+2008%3B+www.chemaxon.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DChemaxon%2520LogD%26pub%3DChemAxon%2520Kft%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gragg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmerle, H.</span><span> </span><span class="NLM_article-title">Kinomics−structural biology and chemogenomics of kinase inhibitors and targets</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1697</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">257</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1697&publication_year=2004&pages=243-257&author=M.+Viethauthor=R.+E.+Higgsauthor=D.+H.+Robertsonauthor=M.+Shapiroauthor=E.+A.+Graggauthor=H.+Hemmerle&title=Kinomics%E2%88%92structural+biology+and+chemogenomics+of+kinase+inhibitors+and+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DShapiro%26aufirst%3DM.%26aulast%3DGragg%26aufirst%3DE.%2BA.%26aulast%3DHemmerle%26aufirst%3DH.%26atitle%3DKinomics%25E2%2588%2592structural%2520biology%2520and%2520chemogenomics%2520of%2520kinase%2520inhibitors%2520and%2520targets%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2004%26volume%3D1697%26spage%3D243%26epage%3D257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Voth, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, S. P.</span><span> </span><span class="NLM_article-title">The Role of Halogen Bonding in Inhibitor Recognition and Binding by Protein Kinases</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1336</span><span class="NLM_x">–</span> <span class="NLM_lpage">1348</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.2174%2F156802607781696846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=17692024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1SmtbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1336-1348&author=R.+A.+Vothauthor=S.+P.+Ho&title=The+Role+of+Halogen+Bonding+in+Inhibitor+Recognition+and+Binding+by+Protein+Kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The role of halogen bonding in inhibitor recognition and binding by protein kinases</span></div><div class="casAuthors">Voth, Andrea Regier; Ho, P. Shing</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1336-1348</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Halogen bonds are short-range mol. interactions that are analogous to classical hydrogen bonds, except that a polarized halogen replaces the hydrogen as the acid in the Lewis acid/base pair.  Such interactions occur regularly in the structures of many ligand-protein complexes, but have only recently been recognized in biol. systems as a distinct class with well-defined phys. characteristics.  In this review, we survey twelve single crystal structures of protein kinase complexes with halogenated ligands in order to characterize the role of halogen bonds in conferring specificity and affinity for halogenated inhibitors in this important class of enzymes.  From this survey, we attempt to identify the properties of halogen bonds that can be generally applied to bottom-up strategies for designing inhibitors for this and other enzyme targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0lKs3fgdLV7Vg90H21EOLACvtfcHk0ljvQraju1lRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1SmtbrJ&md5=da1c1d603db9c2d93682f428198f0515</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F156802607781696846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607781696846%26sid%3Dliteratum%253Aachs%26aulast%3DVoth%26aufirst%3DR.%2BA.%26aulast%3DHo%26aufirst%3DS.%2BP.%26atitle%3DThe%2520Role%2520of%2520Halogen%2520Bonding%2520in%2520Inhibitor%2520Recognition%2520and%2520Binding%2520by%2520Protein%2520Kinases%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D1336%26epage%3D1348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Muegge, I.</span><span> </span><span class="NLM_article-title">Selection criteria for drug-like compounds</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">302</span><span class="NLM_x">–</span> <span class="NLM_lpage">321</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1002%2Fmed.10041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=12647312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjsVWitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=302-321&author=I.+Muegge&title=Selection+criteria+for+drug-like+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Selection criteria for drug-like compounds</span></div><div class="casAuthors">Muegge, Ingo</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">302-321</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The fast identification of quality lead compds. in the pharmaceutical industry through a combination of high throughput synthesis and screening has become more challenging in recent years.  Although the no. of available compds. for high throughput screening (HTS) has dramatically increased, large-scale random combinatorial libraries have contributed proportionally less to identify novel leads for drug discovery projects.  Therefore, the concept of 'drug-likeness' of compd. selections has become a focus in recent years.  In parallel, the low success rate of converting lead compds. into drugs often due to unfavorable pharmacokinetic parameters has sparked a renewed interest in understanding more clearly what makes a compd. drug-like.  Various approaches have been devised to address the drug-likeness of mols. employing retrospective analyses of known drug collections as well as attempting to capture 'chem. wisdom' in algorithms.  For example, simple property counting schemes, machine learning methods, regression models, and clustering methods have been employed to distinguish between drugs and non-drugs.  Here we review computational techniques to address the drug-likeness of compd. selections and offer an outlook for the further development of the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEqC8ZM-CqZrVg90H21EOLACvtfcHk0ljvQraju1lRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjsVWitLY%253D&md5=a855034b700ff48c64262ca90f43a928</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fmed.10041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.10041%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26atitle%3DSelection%2520criteria%2520for%2520drug-like%2520compounds%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2003%26volume%3D23%26spage%3D302%26epage%3D321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Lipinski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A.</span><span> </span><span class="NLM_article-title">Navigating chemical space for biology and medicine</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">855</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnature03193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=15602551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=855-861&author=C.+Lipinskiauthor=A.+Hopkins&title=Navigating+chemical+space+for+biology+and+medicine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating chemical space for biology and medicine</span></div><div class="casAuthors">Lipinski, Christopher; Hopkins, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite over a century of applying org. synthesis to the search for drugs, the authors are still far from even a cursory examn. of the vast no. of possible small mols. that could be created.  Indeed, a thorough examn. of all 'chem. space' is practically impossible.  Given this, what are the best strategies for identifying small mols. that modulate biol. targets And how might such strategies differ, depending on whether the primary goal is to understand biol. systems or to develop potential drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hQm7xaX5MLVg90H21EOLACvtfcHk0ljvQraju1lRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO&md5=077237d55fa1225fd0aaee481e376172</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature03193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03193%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%26aulast%3DHopkins%26aufirst%3DA.%26atitle%3DNavigating%2520chemical%2520space%2520for%2520biology%2520and%2520medicine%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D855%26epage%3D861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Deanda, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reno, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span> </span><span class="NLM_article-title">Kinase-Targeted Library Design through the Application of the PharmPrint Methodology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2395</span><span class="NLM_x">–</span> <span class="NLM_lpage">2403</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800276t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWgt7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=2395-2403&author=F.+Deandaauthor=E.+L.+Stewartauthor=M.+J.+Renoauthor=D.+H.+Drewry&title=Kinase-Targeted+Library+Design+through+the+Application+of+the+PharmPrint+Methodology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase-Targeted Library Design through the Application of the PharmPrint Methodology</span></div><div class="casAuthors">Deanda, Felix; Stewart, Eugene L.; Reno, Michael J.; Drewry, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2395-2403</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The PharmPrint methodol., as modified and implemented by Deanda and Stewart, was prospectively evaluated for use as a virtual high-throughput screening tool by applying it to the design of target-focused arrays.  To this end, PharmPrint quant. structure-activity relationship (QSAR) models for the prediction of AKT1, Aurora-A, and ROCK1 inhibition were constructed and used to virtually screen two large combinatorial libraries.  Based on predicted activities, an Aurora-A targeted array and a ROCK1 targeted array were designed and synthesized.  One control group per designed array was also synthesized to assess the enrichment levels achieved by the QSAR models.  For the Aurora-A targeted array, the hit rate, against the intended target, was 42.9%, whereas that of the control group was 0%.  Thus, the enrichment level achieved by the Aurora-A QSAR model was incalculable.  For the ROCK1 targeted array, the hit rate against the intended target was 30.6%, whereas that of the control group was 5.10%, making the enrichment level achieved by the ROCK1 QSAR model 6-fold above control.  Clearly, these results support the use of the PharmPrint methodol. as a virtual screening tool for the design of kinase-targeted arrays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXVi-syw96s7Vg90H21EOLACvtfcHk0ljvQraju1lRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWgt7%252FF&md5=0a7c65ed90b10dc8de5016af052304f6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci800276t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800276t%26sid%3Dliteratum%253Aachs%26aulast%3DDeanda%26aufirst%3DF.%26aulast%3DStewart%26aufirst%3DE.%2BL.%26aulast%3DReno%26aufirst%3DM.%2BJ.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DKinase-Targeted%2520Library%2520Design%2520through%2520the%2520Application%2520of%2520the%2520PharmPrint%2520Methodology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D2395%26epage%3D2403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Pellecchia, M.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery takes a virtual turn</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">274</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnchembio0509-274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=19377449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXks12kur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=274-275&author=M.+Pellecchia&title=Fragment-based+drug+discovery+takes+a+virtual+turn"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery takes a virtual turn</span></div><div class="casAuthors">Pellecchia, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">274-275</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the realm of modern drug discovery technologies, fragment-based approaches and virtual ligand screening are emerging alternative approaches to high-throughput screening (HTS).  For as simple as it sounds, a hybrid approach in which fragments are discovered first in silico may be an unbeatable route to hit identification for some drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Xqf4HqouS7Vg90H21EOLACvtfcHk0liIh19erQRJTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXks12kur4%253D&md5=03aa16de49eb18000db8376ecfb7d268</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0509-274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0509-274%26sid%3Dliteratum%253Aachs%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DFragment-based%2520drug%2520discovery%2520takes%2520a%2520virtual%2520turn%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D274%26epage%3D275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 36 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Lewis R. Vidler, <span class="NLM_string-name hlFld-ContribAuthor">Matthew P. Baumgartner</span>. </span><span class="cited-content_cbyCitation_article-title">Creating a Virtual Assistant for Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (7)
                                     , 1051-1055. <a href="https://doi.org/10.1021/acsmedchemlett.9b00151" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00151</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00151%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCreating%252Ba%252BVirtual%252BAssistant%252Bfor%252BMedicinal%252BChemistry%26aulast%3DVidler%26aufirst%3DLewis%2BR.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D03042019%26date%3D06062019%26date%3D17062019%26date%3D07062019%26volume%3D10%26issue%3D7%26spage%3D1051%26epage%3D1055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christos A. Nicolaou, Christine Humblet, Hong Hu, Eva M. Martin, Frank C. Dorsey, Thomas M. Castle, Keith Ian Burton, Haitao Hu, Jorg Hendle, Michael J. Hickey, Joel Duerksen, Jibo Wang, <span class="NLM_string-name hlFld-ContribAuthor">Jon A. Erickson</span>. </span><span class="cited-content_cbyCitation_article-title">Idea2Data: Toward a New Paradigm for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (3)
                                     , 278-286. <a href="https://doi.org/10.1021/acsmedchemlett.8b00488" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00488</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00488%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdea2Data%25253A%252BToward%252Ba%252BNew%252BParadigm%252Bfor%252BDrug%252BDiscovery%26aulast%3DNicolaou%26aufirst%3DChristos%2BA.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D18102018%26date%3D04022019%26date%3D18022019%26date%3D04022019%26volume%3D10%26issue%3D3%26spage%3D278%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena  Dolgikh</span>, <span class="hlFld-ContribAuthor ">Ian A.  Watson</span>, <span class="hlFld-ContribAuthor ">Prashant V.  Desai</span>, <span class="hlFld-ContribAuthor ">Geri A.  Sawada</span>, <span class="hlFld-ContribAuthor ">Stuart  Morton</span>, <span class="hlFld-ContribAuthor ">Timothy M.  Jones</span>, and <span class="hlFld-ContribAuthor ">Thomas J.  Raub</span>  . </span><span class="cited-content_cbyCitation_article-title">QSAR Model of Unbound Brain-to-Plasma Partition Coefficient, Kp,uu,brain: Incorporating P-glycoprotein Efflux as a Variable. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (11)
                                     , 2225-2233. <a href="https://doi.org/10.1021/acs.jcim.6b00229" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00229</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00229%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DQSAR%252BModel%252Bof%252BUnbound%252BBrain-to-Plasma%252BPartition%252BCoefficient%25252C%252BKp%25252Cuu%25252Cbrain%25253A%252BIncorporating%252BP-glycoprotein%252BEfflux%252Bas%252Ba%252BVariable%26aulast%3DDolgikh%26aufirst%3DElena%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D24042016%26date%3D13102016%26date%3D28112016%26date%3D29092016%26volume%3D56%26issue%3D11%26spage%3D2225%26epage%3D2233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cen  Gao</span>, <span class="hlFld-ContribAuthor ">Suntara  Cahya</span>, <span class="hlFld-ContribAuthor ">Christos A.  Nicolaou</span>, <span class="hlFld-ContribAuthor ">Jibo  Wang</span>, <span class="hlFld-ContribAuthor ">Ian A.  Watson</span>, <span class="hlFld-ContribAuthor ">David J.  Cummins</span>, <span class="hlFld-ContribAuthor ">Philip W.  Iversen</span>, and <span class="hlFld-ContribAuthor ">Michal  Vieth</span>  . </span><span class="cited-content_cbyCitation_article-title">Selectivity Data: Assessment, Predictions, Concordance, and Implications. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (17)
                                     , 6991-7002. <a href="https://doi.org/10.1021/jm400798j" title="DOI URL">https://doi.org/10.1021/jm400798j</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm400798j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm400798j%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelectivity%252BData%25253A%252BAssessment%25252C%252BPredictions%25252C%252BConcordance%25252C%252Band%252BImplications%26aulast%3DGao%26aufirst%3DCen%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D29052013%26date%3D30082013%26date%3D12092013%26date%3D12082013%26volume%3D56%26issue%3D17%26spage%3D6991%26epage%3D7002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert  Urich</span>, <span class="hlFld-ContribAuthor ">Grant  Wishart</span>, <span class="hlFld-ContribAuthor ">Michael  Kiczun</span>, <span class="hlFld-ContribAuthor ">André  Richters</span>, <span class="hlFld-ContribAuthor ">Naomi  Tidten-Luksch</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>, <span class="hlFld-ContribAuthor ">Brad  Sherborne</span>, <span class="hlFld-ContribAuthor ">Paul G.  Wyatt</span>, and <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>  . </span><span class="cited-content_cbyCitation_article-title">De Novo Design of Protein Kinase Inhibitors by in Silico Identification of Hinge Region-Binding Fragments. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2013,</strong> <em>8 </em>
                                    (5)
                                     , 1044-1052. <a href="https://doi.org/10.1021/cb300729y" title="DOI URL">https://doi.org/10.1021/cb300729y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb300729y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb300729y%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDe%252BNovo%252BDesign%252Bof%252BProtein%252BKinase%252BInhibitors%252Bby%252Bin%252BSilico%252BIdentification%252Bof%252BHinge%252BRegion-Binding%252BFragments%26aulast%3DUrich%26aufirst%3DRobert%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D28122012%26date%3D12032013%26date%3D27032013%26date%3D17052013%26volume%3D8%26issue%3D5%26spage%3D1044%26epage%3D1052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dušica  Vidović</span>, <span class="hlFld-ContribAuthor ">Steven M.  Muskal</span>, and <span class="hlFld-ContribAuthor ">Stephan C.  Schürer</span>  . </span><span class="cited-content_cbyCitation_article-title">Novel Kinase Inhibitors by Reshuffling Ligand Functionalities Across the Human Kinome. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (12)
                                     , 3107-3115. <a href="https://doi.org/10.1021/ci3003842" title="DOI URL">https://doi.org/10.1021/ci3003842</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci3003842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci3003842%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNovel%252BKinase%252BInhibitors%252Bby%252BReshuffling%252BLigand%252BFunctionalities%252BAcross%252Bthe%252BHuman%252BKinome%26aulast%3DVidovi%25C4%2587%26aufirst%3DDu%25C5%25A1ica%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D16082012%26date%3D20112012%26date%3D21122012%26date%3D05112012%26volume%3D52%26issue%3D12%26spage%3D3107%26epage%3D3115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Obdulia  Rabal</span> and <span class="hlFld-ContribAuthor ">Julen  Oyarzabal</span>  . </span><span class="cited-content_cbyCitation_article-title">Using Novel Descriptor Accounting for Ligand–Receptor Interactions To Define and Visually Explore Biologically Relevant Chemical Space. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (5)
                                     , 1086-1102. <a href="https://doi.org/10.1021/ci200627v" title="DOI URL">https://doi.org/10.1021/ci200627v</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200627v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200627v%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DUsing%252BNovel%252BDescriptor%252BAccounting%252Bfor%252BLigand%2525E2%252580%252593Receptor%252BInteractions%252BTo%252BDefine%252Band%252BVisually%252BExplore%252BBiologically%252BRelevant%252BChemical%252BSpace%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D29122011%26date%3D19042012%26date%3D25052012%26date%3D08042012%26volume%3D52%26issue%3D5%26spage%3D1086%26epage%3D1102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Satoshi  Niijima</span>, <span class="hlFld-ContribAuthor ">Akira  Shiraishi</span>, and <span class="hlFld-ContribAuthor ">Yasushi  Okuno</span>  . </span><span class="cited-content_cbyCitation_article-title">Dissecting Kinase Profiling Data to Predict Activity and Understand Cross-Reactivity of Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (4)
                                     , 901-912. <a href="https://doi.org/10.1021/ci200607f" title="DOI URL">https://doi.org/10.1021/ci200607f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200607f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200607f%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDissecting%252BKinase%252BProfiling%252BData%252Bto%252BPredict%252BActivity%252Band%252BUnderstand%252BCross-Reactivity%252Bof%252BKinase%252BInhibitors%26aulast%3DNiijima%26aufirst%3DSatoshi%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D17122011%26date%3D26032012%26date%3D23042012%26date%3D13032012%26volume%3D52%26issue%3D4%26spage%3D901%26epage%3D912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Martin</span> and <span class="hlFld-ContribAuthor ">Prasenjit  Mukherjee</span>  . </span><span class="cited-content_cbyCitation_article-title">Kinase-Kernel Models: Accurate In silico Screening of 4 Million Compounds Across the Entire Human Kinome. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (1)
                                     , 156-170. <a href="https://doi.org/10.1021/ci200314j" title="DOI URL">https://doi.org/10.1021/ci200314j</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200314j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200314j%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DKinase-Kernel%252BModels%25253A%252BAccurate%252BIn%252Bsilico%252BScreening%252Bof%252B4%252BMillion%252BCompounds%252BAcross%252Bthe%252BEntire%252BHuman%252BKinome%26aulast%3DMartin%26aufirst%3DEric%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D08072011%26date%3D06012012%26date%3D23012012%26date%3D01122011%26volume%3D52%26issue%3D1%26spage%3D156%26epage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julio  Caballero</span>, <span class="hlFld-ContribAuthor ">Jans H.  Alzate-Morales</span>, and <span class="hlFld-ContribAuthor ">Ariela  Vergara-Jaque</span>  . </span><span class="cited-content_cbyCitation_article-title">Investigation of the Differences in Activity between Hydroxycycloalkyl N1 Substituted Pyrazole Derivatives As Inhibitors of B-Raf Kinase by Using Docking, Molecular Dynamics, QM/MM, and Fragment-Based De Novo Design: Study of Binding Mode of Diastereomer Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (11)
                                     , 2920-2931. <a href="https://doi.org/10.1021/ci200306w" title="DOI URL">https://doi.org/10.1021/ci200306w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200306w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200306w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DInvestigation%252Bof%252Bthe%252BDifferences%252Bin%252BActivity%252Bbetween%252BHydroxycycloalkyl%252BN1%252BSubstituted%252BPyrazole%252BDerivatives%252BAs%252BInhibitors%252Bof%252BB-Raf%252BKinase%252Bby%252BUsing%252BDocking%25252C%252BMolecular%252BDynamics%25252C%252BQM%25252FMM%25252C%252Band%252BFragment-Based%252BDe%252BNovo%252BDesign%25253A%252BStudy%252Bof%252BBinding%252BMode%252Bof%252BDiastereomer%252BCompounds%26aulast%3DCaballero%26aufirst%3DJulio%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D04072011%26date%3D31102011%26date%3D28112011%26date%3D20102011%26volume%3D51%26issue%3D11%26spage%3D2920%26epage%3D2931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Martin</span>, <span class="hlFld-ContribAuthor ">Prasenjit  Mukherjee</span>, <span class="hlFld-ContribAuthor ">David  Sullivan</span>, and <span class="hlFld-ContribAuthor ">Johanna  Jansen</span>  . </span><span class="cited-content_cbyCitation_article-title">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (8)
                                     , 1942-1956. <a href="https://doi.org/10.1021/ci1005004" title="DOI URL">https://doi.org/10.1021/ci1005004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci1005004%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProfile-QSAR%25253A%252BA%252BNovel%252Bmeta-QSAR%252BMethod%252Bthat%252BCombines%252BActivities%252Bacross%252Bthe%252BKinase%252BFamily%252BTo%252BAccurately%252BPredict%252BAffinity%25252C%252BSelectivity%25252C%252Band%252BCellular%252BActivity%26aulast%3DMartin%26aufirst%3DEric%26date%3D2011%26date%3D2011%26date%3D2010%26date%3D20122010%26date%3D19072011%26date%3D22082011%26date%3D13062011%26volume%3D51%26issue%3D8%26spage%3D1942%26epage%3D1956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shana L.  Posy</span>, <span class="hlFld-ContribAuthor ">Mark A.  Hermsmeier</span>, <span class="hlFld-ContribAuthor ">Wayne  Vaccaro</span>, <span class="hlFld-ContribAuthor ">Karl-Heinz  Ott</span>, <span class="hlFld-ContribAuthor ">Gordon  Todderud</span>, <span class="hlFld-ContribAuthor ">Jonathan S.  Lippy</span>, <span class="hlFld-ContribAuthor ">George L.  Trainor</span>, <span class="hlFld-ContribAuthor ">Deborah A.  Loughney</span>, and <span class="hlFld-ContribAuthor ">Stephen R.  Johnson</span>  . </span><span class="cited-content_cbyCitation_article-title">Trends in Kinase Selectivity: Insights for Target Class-Focused Library Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (1)
                                     , 54-66. <a href="https://doi.org/10.1021/jm101195a" title="DOI URL">https://doi.org/10.1021/jm101195a</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm101195a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm101195a%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTrends%252Bin%252BKinase%252BSelectivity%25253A%252BInsights%252Bfor%252BTarget%252BClass-Focused%252BLibrary%252BScreening%26aulast%3DPosy%26aufirst%3DShana%2BL.%26date%3D2011%26date%3D2010%26date%3D2010%26date%3D26032010%26date%3D03122010%26date%3D13012011%26volume%3D54%26issue%3D1%26spage%3D54%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Govindan  Subramanian</span> and <span class="hlFld-ContribAuthor ">Manish  Sud</span>  . </span><span class="cited-content_cbyCitation_article-title">Computational Modeling of Kinase Inhibitor Selectivity. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2010,</strong> <em>1 </em>
                                    (8)
                                     , 395-399. <a href="https://doi.org/10.1021/ml1001097" title="DOI URL">https://doi.org/10.1021/ml1001097</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml1001097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml1001097%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DComputational%252BModeling%252Bof%252BKinase%252BInhibitor%252BSelectivity%26aulast%3DSubramanian%26aufirst%3DGovindan%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D11052010%26date%3D26072010%26date%3D28072010%26date%3D11112010%26volume%3D1%26issue%3D8%26spage%3D395%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jans H.  Alzate-Morales</span>, <span class="hlFld-ContribAuthor ">Ariela  Vergara-Jaque</span> and <span class="hlFld-ContribAuthor ">Julio  Caballero</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Study on the Interaction of N1 Substituted Pyrazole Derivatives with B-Raf Kinase: An Unusual Water Wire Hydrogen-Bond Network and Novel Interactions at the Entrance of the Active Site. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (6)
                                     , 1101-1112. <a href="https://doi.org/10.1021/ci100049h" title="DOI URL">https://doi.org/10.1021/ci100049h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci100049h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci100049h%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComputational%252BStudy%252Bon%252Bthe%252BInteraction%252Bof%252BN1%252BSubstituted%252BPyrazole%252BDerivatives%252Bwith%252BB-Raf%252BKinase%25253A%252BAn%252BUnusual%252BWater%252BWire%252BHydrogen-Bond%252BNetwork%252Band%252BNovel%252BInteractions%252Bat%252Bthe%252BEntrance%252Bof%252Bthe%252BActive%252BSite%26aulast%3DAlzate-Morales%26aufirst%3DJans%2BH.%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D3%252002%25202010%26date%3D04062010%26date%3D28062010%26volume%3D50%26issue%3D6%26spage%3D1101%26epage%3D1112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amanda J.  DeGraw</span>, <span class="hlFld-ContribAuthor ">Michael J.  Keiser</span>, <span class="hlFld-ContribAuthor ">Joshua D.  Ochocki</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span> and <span class="hlFld-ContribAuthor ">Mark D.  Distefano</span>  . </span><span class="cited-content_cbyCitation_article-title">Prediction and Evaluation of Protein Farnesyltransferase Inhibition by Commercial Drugs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2010,</strong> <em>53 </em>
                                    (6)
                                     , 2464-2471. <a href="https://doi.org/10.1021/jm901613f" title="DOI URL">https://doi.org/10.1021/jm901613f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm901613f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm901613f%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPrediction%252Band%252BEvaluation%252Bof%252BProtein%252BFarnesyltransferase%252BInhibition%252Bby%252BCommercial%252BDrugs%26aulast%3DDeGraw%26aufirst%3DAmanda%2BJ.%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D03112009%26date%3D24022010%26date%3D25032010%26volume%3D53%26issue%3D6%26spage%3D2464%26epage%3D2471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vasudha  Abbhi</span>, <span class="hlFld-ContribAuthor ">Poonam  Piplani</span>. </span><span class="cited-content_cbyCitation_article-title">Rho-kinase (ROCK) Inhibitors - A Neuroprotective Therapeutic Paradigm with a Focus on Ocular Utility. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (14)
                                     , 2222-2256. <a href="https://doi.org/10.2174/0929867325666181031102829" title="DOI URL">https://doi.org/10.2174/0929867325666181031102829</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666181031102829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666181031102829%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DRho-kinase%252B%252528ROCK%252529%252BInhibitors%252B-%252BA%252BNeuroprotective%252BTherapeutic%252BParadigm%252Bwith%252Ba%252BFocus%252Bon%252BOcular%252BUtility%26aulast%3DAbbhi%26aufirst%3DVasudha%26date%3D2020%26volume%3D27%26issue%3D14%26spage%3D2222%26epage%3D2256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vasudha  Abbhi</span>, <span class="hlFld-ContribAuthor ">Lovneet  Saini</span>, <span class="hlFld-ContribAuthor ">Srishti  Mishra</span>, <span class="hlFld-ContribAuthor ">Gautam  Sethi</span>, <span class="hlFld-ContribAuthor ">Alan Prem  Kumar</span>, <span class="hlFld-ContribAuthor ">Poonam  Piplani</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of benzimidazole-based Rho kinase inhibitors for the treatment of glaucoma. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (21)
                                     , 6071-6085. <a href="https://doi.org/10.1016/j.bmc.2017.09.045" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.09.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.09.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.09.045%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Bbenzimidazole-based%252BRho%252Bkinase%252Binhibitors%252Bfor%252Bthe%252Btreatment%252Bof%252Bglaucoma%26aulast%3DAbbhi%26aufirst%3DVasudha%26date%3D2017%26volume%3D25%26issue%3D21%26spage%3D6071%26epage%3D6085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ingo  Muegge</span>, <span class="hlFld-ContribAuthor ">Prasenjit  Mukherjee</span>. </span><span class="cited-content_cbyCitation_article-title">An overview of molecular fingerprint similarity search in virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2016,</strong> <em>11 </em>
                                    (2)
                                     , 137-148. <a href="https://doi.org/10.1517/17460441.2016.1117070" title="DOI URL">https://doi.org/10.1517/17460441.2016.1117070</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2016.1117070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2016.1117070%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DAn%252Boverview%252Bof%252Bmolecular%252Bfingerprint%252Bsimilarity%252Bsearch%252Bin%252Bvirtual%252Bscreening%26aulast%3DMuegge%26aufirst%3DIngo%26date%3D2016%26date%3D2015%26volume%3D11%26issue%3D2%26spage%3D137%26epage%3D148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jon A.  Erickson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Design of Kinase Inhibitors: A Practical Guide. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 157-183. <a href="https://doi.org/10.1007/978-1-4939-2486-8_13" title="DOI URL">https://doi.org/10.1007/978-1-4939-2486-8_13</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-2486-8_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-2486-8_13%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BDesign%252Bof%252BKinase%252BInhibitors%25253A%252BA%252BPractical%252BGuide%26aulast%3DErickson%26aufirst%3DJon%2BA.%26date%3D2015%26date%3D2015%26spage%3D157%26epage%3D183%26pub%3DSpringer%2520New%2520York%26atitle%3DFragment-Based%252BMethods%252Bin%252BDrug%252BDiscovery%26aulast%3DKlon%26aufirst%3DAnthony%2BE.%26date%3D2015%26volume%3D1289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anuradha  Kumar</span>, <span class="hlFld-ContribAuthor ">Allen  Casey</span>, <span class="hlFld-ContribAuthor ">Joshua  Odingo</span>, <span class="hlFld-ContribAuthor ">Edward A.  Kesicki</span>, <span class="hlFld-ContribAuthor ">Garth  Abrahams</span>, <span class="hlFld-ContribAuthor ">Michal  Vieth</span>, <span class="hlFld-ContribAuthor ">Thierry  Masquelin</span>, <span class="hlFld-ContribAuthor ">Valerie  Mizrahi</span>, <span class="hlFld-ContribAuthor ">Philip A.  Hipskind</span>, <span class="hlFld-ContribAuthor ">David R.  Sherman</span>, <span class="hlFld-ContribAuthor ">Tanya  Parish</span>, . </span><span class="cited-content_cbyCitation_article-title">A High-Throughput Screen against Pantothenate Synthetase (PanC) Identifies 3-Biphenyl-4-Cyanopyrrole-2-Carboxylic Acids as a New Class of Inhibitor with Activity against Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2013,</strong> <em>8 </em>
                                    (11)
                                     , e72786. <a href="https://doi.org/10.1371/journal.pone.0072786" title="DOI URL">https://doi.org/10.1371/journal.pone.0072786</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0072786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0072786%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DA%252BHigh-Throughput%252BScreen%252Bagainst%252BPantothenate%252BSynthetase%252B%252528PanC%252529%252BIdentifies%252B3-Biphenyl-4-Cyanopyrrole-2-Carboxylic%252BAcids%252Bas%252Ba%252BNew%252BClass%252Bof%252BInhibitor%252Bwith%252BActivity%252Bagainst%252BMycobacterium%252Btuberculosis%26aulast%3DKumar%26aufirst%3DAnuradha%26date%3D2013%26date%3D2013%26volume%3D8%26issue%3D11%26spage%3De72786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Mazanetz</span>, <span class="hlFld-ContribAuthor ">Richard  Law</span>, <span class="hlFld-ContribAuthor ">Mark  Whittaker</span>. </span><span class="cited-content_cbyCitation_article-title">Hit and Lead Identification from Fragments. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,, 143-200. <a href="https://doi.org/10.1002/9783527677016.ch6" title="DOI URL">https://doi.org/10.1002/9783527677016.ch6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527677016.ch6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527677016.ch6%26sid%3Dliteratum%253Aachs%26atitle%3DHit%252Band%252BLead%252BIdentification%252Bfrom%252BFragments%26aulast%3DMazanetz%26aufirst%3DMichael%26date%3D2013%26date%3D2013%26spage%3D143%26epage%3D200%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DDe%252Bnovo%252BMolecular%252BDesign%26aulast%3DSchneider%26aufirst%3DGisbert%26date%3D2013%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunquan  Sheng</span>, <span class="hlFld-ContribAuthor ">Wannian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Informatics and Computational Fragment-Based Drug Design: An Overview and Update. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2013,</strong> <em>33 </em>
                                    (3)
                                     , 554-598. <a href="https://doi.org/10.1002/med.21255" title="DOI URL">https://doi.org/10.1002/med.21255</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21255%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DFragment%252BInformatics%252Band%252BComputational%252BFragment-Based%252BDrug%252BDesign%25253A%252BAn%252BOverview%252Band%252BUpdate%26aulast%3DSheng%26aufirst%3DChunquan%26date%3D2013%26date%3D2012%26volume%3D33%26issue%3D3%26spage%3D554%26epage%3D598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eman M.  Dokla</span>, <span class="hlFld-ContribAuthor ">Amr H.  Mahmoud</span>, <span class="hlFld-ContribAuthor ">Mohamed S. A.  Elsayed</span>, <span class="hlFld-ContribAuthor ">Ahmed H.  El-Khatib</span>, <span class="hlFld-ContribAuthor ">Michael W.  Linscheid</span>, <span class="hlFld-ContribAuthor ">Khaled A.  Abouzid</span>, . </span><span class="cited-content_cbyCitation_article-title">Applying Ligands Profiling Using Multiple Extended Electron Distribution Based Field Templates and Feature Trees Similarity Searching in the Discovery of New Generation of Urea-Based Antineoplastic Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2012,</strong> <em>7 </em>
                                    (11)
                                     , e49284. <a href="https://doi.org/10.1371/journal.pone.0049284" title="DOI URL">https://doi.org/10.1371/journal.pone.0049284</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0049284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0049284%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DApplying%252BLigands%252BProfiling%252BUsing%252BMultiple%252BExtended%252BElectron%252BDistribution%252BBased%252BField%252BTemplates%252Band%252BFeature%252BTrees%252BSimilarity%252BSearching%252Bin%252Bthe%252BDiscovery%252Bof%252BNew%252BGeneration%252Bof%252BUrea-Based%252BAntineoplastic%252BKinase%252BInhibitors%26aulast%3DDokla%26aufirst%3DEman%2BM.%26date%3D2012%26date%3D2012%26volume%3D7%26issue%3D11%26spage%3De49284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Camila  Muñoz</span>, <span class="hlFld-ContribAuthor ">Francisco  Adasme</span>, <span class="hlFld-ContribAuthor ">Jans H.  Alzate-Morales</span>, <span class="hlFld-ContribAuthor ">Ariela  Vergara-Jaque</span>, <span class="hlFld-ContribAuthor ">Torsten  Kniess</span>, <span class="hlFld-ContribAuthor ">Julio  Caballero</span>. </span><span class="cited-content_cbyCitation_article-title">Study of differences in the VEGFR2 inhibitory activities between semaxanib and SU5205 using 3D-QSAR, docking, and molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2012,</strong> <em>32 </em>, 39-48. <a href="https://doi.org/10.1016/j.jmgm.2011.10.005" title="DOI URL">https://doi.org/10.1016/j.jmgm.2011.10.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2011.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2011.10.005%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DStudy%252Bof%252Bdifferences%252Bin%252Bthe%252BVEGFR2%252Binhibitory%252Bactivities%252Bbetween%252Bsemaxanib%252Band%252BSU5205%252Busing%252B3D-QSAR%25252C%252Bdocking%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DMu%25C3%25B1oz%26aufirst%3DCamila%26date%3D2012%26volume%3D32%26spage%3D39%26epage%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao-Feng  Le</span>, <span class="hlFld-ContribAuthor ">Weiqun  Mao</span>, <span class="hlFld-ContribAuthor ">Guangan  He</span>, <span class="hlFld-ContribAuthor ">Francois-Xavier  Claret</span>, <span class="hlFld-ContribAuthor ">Weiya  Xia</span>, <span class="hlFld-ContribAuthor ">Ahmed Ashour  Ahmed</span>, <span class="hlFld-ContribAuthor ">Mien-Chie  Hung</span>, <span class="hlFld-ContribAuthor ">Zahid H.  Siddik</span>, <span class="hlFld-ContribAuthor ">Robert C.  Bast</span>. </span><span class="cited-content_cbyCitation_article-title">The Role of p27 Kip1 in Dasatinib-Enhanced Paclitaxel Cytotoxicity in Human Ovarian Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">JNCI: Journal of the National Cancer Institute</span><span> <strong>2011,</strong> <em>103 </em>
                                    (18)
                                     , 1403-1422. <a href="https://doi.org/10.1093/jnci/djr280" title="DOI URL">https://doi.org/10.1093/jnci/djr280</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/jnci/djr280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fjnci%2Fdjr280%26sid%3Dliteratum%253Aachs%26jtitle%3DJNCI%253A%2520Journal%2520of%2520the%2520National%2520Cancer%2520Institute%26atitle%3DThe%252BRole%252Bof%252Bp27%252BKip1%252Bin%252BDasatinib-Enhanced%252BPaclitaxel%252BCytotoxicity%252Bin%252BHuman%252BOvarian%252BCancer%252BCells%26aulast%3DLe%26aufirst%3DXiao-Feng%26date%3D2011%26date%3D2011%26volume%3D103%26issue%3D18%26spage%3D1403%26epage%3D1422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Markus  Boehm</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening of Chemical Space: From Generic Compound Collections to Tailored Screening Libraries. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 1-33. <a href="https://doi.org/10.1002/9783527633326.ch1" title="DOI URL">https://doi.org/10.1002/9783527633326.ch1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527633326.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527633326.ch1%26sid%3Dliteratum%253Aachs%26atitle%3DVirtual%252BScreening%252Bof%252BChemical%252BSpace%25253A%252BFrom%252BGeneric%252BCompound%252BCollections%252Bto%252BTailored%252BScreening%252BLibraries%26aulast%3DBoehm%26aufirst%3DMarkus%26date%3D2011%26date%3D2011%26spage%3D1%26epage%3D33%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DVirtual%252BScreening%26aulast%3DSotriffer%26aufirst%3DChristoph%26date%3D2011%26volume%3D432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julio  Caballero</span>, <span class="hlFld-ContribAuthor ">Miguel  Quiliano</span>, <span class="hlFld-ContribAuthor ">Jans H.  Alzate-Morales</span>, <span class="hlFld-ContribAuthor ">Mirko  Zimic</span>, <span class="hlFld-ContribAuthor ">Eric  Deharo</span>. </span><span class="cited-content_cbyCitation_article-title">Docking and quantitative structure–activity relationship studies for 3-fluoro-4-(pyrrolo[2,1-f][1,2,4]triazin-4-yloxy)aniline, 3-fluoro-4-(1H-pyrrolo[2,3-b]pyridin-4-yloxy)aniline, and 4-(4-amino-2-fluorophenoxy)-2-pyridinylamine derivatives as c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2011,</strong> <em>25 </em>
                                    (4)
                                     , 349-369. <a href="https://doi.org/10.1007/s10822-011-9425-1" title="DOI URL">https://doi.org/10.1007/s10822-011-9425-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-011-9425-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-011-9425-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DDocking%252Band%252Bquantitative%252Bstructure%2525E2%252580%252593activity%252Brelationship%252Bstudies%252Bfor%252B3-fluoro-4-%252528pyrrolo%25255B2%25252C1-f%25255D%25255B1%25252C2%25252C4%25255Dtriazin-4-yloxy%252529aniline%25252C%252B3-fluoro-4-%2525281H-pyrrolo%25255B2%25252C3-b%25255Dpyridin-4-yloxy%252529aniline%25252C%252Band%252B4-%2525284-amino-2-fluorophenoxy%252529-2-pyridinylamine%252Bderivatives%252Bas%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DCaballero%26aufirst%3DJulio%26date%3D2011%26date%3D2011%26volume%3D25%26issue%3D4%26spage%3D349%26epage%3D369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Approaches and Computer-Aided Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 201-222. <a href="https://doi.org/10.1007/128_2011_182" title="DOI URL">https://doi.org/10.1007/128_2011_182</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/128_2011_182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F128_2011_182%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BApproaches%252Band%252BComputer-Aided%252BDrug%252BDiscovery%26aulast%3DRognan%26aufirst%3DDidier%26date%3D2011%26date%3D2011%26spage%3D201%26epage%3D222%26pub%3DSpringer%2520Berlin%2520Heidelberg%26atitle%3DFragment-Based%252BDrug%252BDiscovery%252Band%252BX-Ray%252BCrystallography%26aulast%3DDavies%26aufirst%3DThomas%2BG.%26date%3D2012%26volume%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Obdulia  Rabal</span>, <span class="hlFld-ContribAuthor ">Manuel  Urbano-Cuadrado</span>, <span class="hlFld-ContribAuthor ">Julen  Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Computational medicinal chemistry in fragment-based drug discovery: what, how and when. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2011,</strong> <em>3 </em>
                                    (1)
                                     , 95-134. <a href="https://doi.org/10.4155/fmc.10.277" title="DOI URL">https://doi.org/10.4155/fmc.10.277</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.10.277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.10.277%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DComputational%252Bmedicinal%252Bchemistry%252Bin%252Bfragment-based%252Bdrug%252Bdiscovery%25253A%252Bwhat%25252C%252Bhow%252Band%252Bwhen%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2011%26volume%3D3%26issue%3D1%26spage%3D95%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter S  Kutchukian</span>, <span class="hlFld-ContribAuthor ">Eugene I  Shakhnovich</span>. </span><span class="cited-content_cbyCitation_article-title">De novo
              design: balancing novelty and confined chemical space. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2010,</strong> <em>5 </em>
                                    (8)
                                     , 789-812. <a href="https://doi.org/10.1517/17460441.2010.497534" title="DOI URL">https://doi.org/10.1517/17460441.2010.497534</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2010.497534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2010.497534%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DDe%252Bnovo%252Bdesign%25253A%252Bbalancing%252Bnovelty%252Band%252Bconfined%252Bchemical%252Bspace%26aulast%3DKutchukian%26aufirst%3DPeter%2BS%26date%3D2010%26date%3D2010%26volume%3D5%26issue%3D8%26spage%3D789%26epage%3D812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rong  Huang</span>, <span class="hlFld-ContribAuthor ">Isabel  Martinez-Ferrando</span>, <span class="hlFld-ContribAuthor ">Philip A  Cole</span>. </span><span class="cited-content_cbyCitation_article-title">Enhanced interrogation: emerging strategies for cell signaling inhibition. </span><span class="cited-content_cbyCitation_journal-name">Nature Structural & Molecular Biology</span><span> <strong>2010,</strong> <em>17 </em>
                                    (6)
                                     , 646-649. <a href="https://doi.org/10.1038/nsmb0610-646" title="DOI URL">https://doi.org/10.1038/nsmb0610-646</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nsmb0610-646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnsmb0610-646%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Structural%2520%2526%2520Molecular%2520Biology%26atitle%3DEnhanced%252Binterrogation%25253A%252Bemerging%252Bstrategies%252Bfor%252Bcell%252Bsignaling%252Binhibition%26aulast%3DHuang%26aufirst%3DRong%26date%3D2010%26volume%3D17%26issue%3D6%26spage%3D646%26epage%3D649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John  Drewe</span>, <span class="hlFld-ContribAuthor ">Sui Xiong  Cai</span>. </span><span class="cited-content_cbyCitation_article-title">Cell-based apoptosis assays in oncology drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2010,</strong> <em>5 </em>
                                    (6)
                                     , 583-596. <a href="https://doi.org/10.1517/17460441.2010.486829" title="DOI URL">https://doi.org/10.1517/17460441.2010.486829</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2010.486829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2010.486829%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DCell-based%252Bapoptosis%252Bassays%252Bin%252Boncology%252Bdrug%252Bdiscovery%26aulast%3DDrewe%26aufirst%3DJohn%26date%3D2010%26date%3D2010%26volume%3D5%26issue%3D6%26spage%3D583%26epage%3D596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongyu  Zhao</span>, <span class="hlFld-ContribAuthor ">Irini  Akritopoulou-Zanze</span>. </span><span class="cited-content_cbyCitation_article-title">When analoging is not enough: scaffold discovery in medicinal chemistry. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2010,</strong> <em>5 </em>
                                    (2)
                                     , 123-134. <a href="https://doi.org/10.1517/17460440903584874" title="DOI URL">https://doi.org/10.1517/17460440903584874</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460440903584874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460440903584874%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DWhen%252Banaloging%252Bis%252Bnot%252Benough%25253A%252Bscaffold%252Bdiscovery%252Bin%252Bmedicinal%252Bchemistry%26aulast%3DZhao%26aufirst%3DHongyu%26date%3D2010%26date%3D2010%26volume%3D5%26issue%3D2%26spage%3D123%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas  Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Bayesian Methods in Virtual Screening and Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2010,</strong>,, 175-196. <a href="https://doi.org/10.1007/978-1-60761-839-3_7" title="DOI URL">https://doi.org/10.1007/978-1-60761-839-3_7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-60761-839-3_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-60761-839-3_7%26sid%3Dliteratum%253Aachs%26atitle%3DBayesian%252BMethods%252Bin%252BVirtual%252BScreening%252Band%252BChemical%252BBiology%26aulast%3DBender%26aufirst%3DAndreas%26date%3D2010%26date%3D2010%26spage%3D175%26epage%3D196%26pub%3DHumana%2520Press%26atitle%3DChemoinformatics%252Band%252BComputational%252BChemical%252BBiology%26aulast%3DBajorath%26aufirst%3DJ%25C3%25BCrgen%26date%3D2011%26volume%3D672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stevan W.  Djuric</span>, <span class="hlFld-ContribAuthor ">Irini  Akritopoulou-Zanze</span>, <span class="hlFld-ContribAuthor ">Philip B.  Cox</span>, <span class="hlFld-ContribAuthor ">Scott  Galasinski</span>. </span><span class="cited-content_cbyCitation_article-title">Compound Collection Enhancement and Paradigms for High-Throughput Screening — an Update. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2010,</strong>,, 409-428. <a href="https://doi.org/10.1016/S0065-7743(10)45025-3" title="DOI URL">https://doi.org/10.1016/S0065-7743(10)45025-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S0065-7743(10)45025-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS0065-7743%2810%2945025-3%26sid%3Dliteratum%253Aachs%26atitle%3DCompound%252BCollection%252BEnhancement%252Band%252BParadigms%252Bfor%252BHigh-Throughput%252BScreening%2525E2%252580%252589%2525E2%252580%252594%2525E2%252580%252589an%252BUpdate%26aulast%3DDjuric%26aufirst%3DStevan%2BW.%26date%3D2010%26spage%3D409%26epage%3D428%26pub%3DElsevier%26date%3D2010%26volume%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N.K.  Terrett</span>. </span><span class="cited-content_cbyCitation_article-title">Combinatorial Chemistry Online. </span><span class="cited-content_cbyCitation_journal-name">Combinatorial Chemistry - an Online Journal</span><span> <strong>2010,</strong> <em>12 </em>
                                    (1)
                                     , 1-3. <a href="https://doi.org/10.1016/j.comche.2009.12.001" title="DOI URL">https://doi.org/10.1016/j.comche.2009.12.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.comche.2009.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.comche.2009.12.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCombinatorial%2520Chemistry%2520-%2520an%2520Online%2520Journal%26atitle%3DCombinatorial%252BChemistry%252BOnline%26aulast%3DTerrett%26aufirst%3DN.K.%26date%3D2010%26volume%3D12%26issue%3D1%26spage%3D1%26epage%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Kinase pharmacophore exemplified in marketed and clinical compounds. Solubilizing groups are highlighted with green, hydrophobic with yellow, and hinge binders with cyan. (b) Three-dimensional picture of pharmacophore drawn over X-ray structures of gefitinib (yellow carbons, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ity">2ity</a>.pdb) in EGFR and danusertib in Aurora A (magenta carbons, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2j50">2j50</a>.pdb). EGFR kinase from <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ity">2ity</a>.pdb is shown. Transparent ovals in yellow, cyan, and green are drawn over hydrophobic, hinge binding, and solubilizing groups, respectively. Note that the positions of both solubilizing and hydrophobic groups occupy slightly different region in both structures in contrast to the hinge binding elements, which occupy similar space. Thr790 gate keeper in EGFR is shown in yellow stick representation. (c) Schematic representation of de novo design process starting from actives, fragmentation, and reassembly. Reproduced with ACS permission from ref <a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">7</a>. (d) Hypothetical application to three kinase drugs from Figure 1. Solubilizing groups are highlighted in green, hydrophobic in yellow, and hinge binders in cyan. Fragment connection atoms contain isotopic labels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Comparison of Chronological, random validation with prospective metrics. (a) PPVs. (b) Sensitivities. (c) Root mean square prediction error. (d) <i>R</i><sup>2</sup> values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Results from prospective testing. Distribution of nearest neighbor similarity of 1895 compounds with respect to 5870 historical actives. The fingerprints used for this study are a combination of molecular properties (atom count, size of largest ring, number of rings, number of ring atoms, number of aromatic atoms, number of fused ring atoms, number of heteroatoms), MACCS keys,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and extended connectivity fingerprints with size 3 shell. Tanimoto similarity in this fingerprint space between kinase drugs from Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> was the following: gefinitib−cedinarib 0.728, danusertib−cedinarib 0.592, danusertib−gefinitib 0.585. (a) Distribution of nearest neighbor similarity. (b) Comparison of historical hit rates with prospective validation. (c) Bioprint of predicted (top) and observed (bottom) activities at 20uM for 1895 compounds in 6 kinases. MET had 1392 measurements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Results from library testing. (a) Distribution of nearest neighbor similarity of 179 compounds with respect to 6422 historical actives. (b) Comparison of historical hit rates with prospective validation and library hit rate. (c) Bioprint of predicted (top) and observed (bottom) activities at 20 μM for 179 compounds in six kinases. MET had 116 measurements..</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/medium/jm-2009-01147e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of model performance metrics for prospective validation and library compounds. (a) Mean model predictions and standard deviations for six targets. All differences in means are statistically significant except for FLT3. (b) PPVs and sensitivities. (c) Prediction rms. (d) <i>R</i><sup>2</sup> values: the low values for library compounds might be influenced by higher means for predicted values and lower standard deviations..</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-20/jm901147e/production/images/large/jm-2009-01147e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm901147e&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i18">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Grant, S. K.</span><span> </span><span class="NLM_article-title">Therapeutic Protein Kinase Inhibitors</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">1163</span><span class="NLM_x">–</span> <span class="NLM_lpage">1177</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1007%2Fs00018-008-8539-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=19011754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslWrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2009&pages=1163-1177&author=S.+K.+Grant&title=Therapeutic+Protein+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Protein Kinase Inhibitors</span></div><div class="casAuthors">Grant, S. K.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1163-1177</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Protein kinase inhibitors represent an important and still emerging class of targeted therapeutic agents.  Drug discovery and development strategies have explored numerous approaches to target the inhibition of protein kinase signaling.  This review will highlight some of the strategies that have led to the successful clin. development of therapeutic protein kinase inhibitors, particularly as anticancer drugs.  Some notable advances were made in the development of novel protein and oligonucleotide-based biologics that target growth factor or receptor tyrosine kinases.  Also, advances were made in the rational design of small-mol. inhibitors that target unique kinase conformational forms and binding sites, and have specific kinase selectivity profiles.  A review will also be given of some of the potential clin. toxicities and adverse side-effects assocd. with these kinase-targeted drugs.  Therapeutic protein kinase inhibitors were highly beneficial to cancer patients and offer the promise of future therapies for other diseases as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooJMLRyntX7bVg90H21EOLACvtfcHk0lhimksrTkdthQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslWrsbk%253D&md5=1fe445348168f5632b4c47b4ce27fd6e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fs00018-008-8539-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-008-8539-7%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DS.%2BK.%26atitle%3DTherapeutic%2520Protein%2520Kinase%2520Inhibitors%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2009%26volume%3D66%26spage%3D1163%26epage%3D1177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Echeverria, C. G.</span><span> </span><span class="NLM_article-title">Targeting protein kinases in cancer therapy</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=12630290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnsFekuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2002&pages=701-712&author=D.+Fabbroauthor=C.+G.+Garcia-Echeverria&title=Targeting+protein+kinases+in+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein kinases in cancer therapy</span></div><div class="casAuthors">Fabbro, Doriano; Garcia-Echeverria, Carlos Garcia</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">701-712</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">PharmaPress Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the years, many different tyrosine and serine/threonine protein kinases have been selected as candidates for drug discovery activities in oncol. research, based either on their overexpression and/or dysfunction in a particular organ or tissue, or through their assocn. in deregulated signal transduction/cell cycle pathways.  This review summarizes current preclin. and clin. knowledge of ATP-competitive, small mol. kinase inhibitors, and receptor or ligand-competitive antibodies of selected protein kinases in which drug discovery and development activities have advanced with some success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNWx-SB9X157Vg90H21EOLACvtfcHk0lg4vIb_GYqMvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnsFekuw%253D%253D&md5=e0470671e56759bfbeefc0f84ff939be</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGarcia-Echeverria%26aufirst%3DC.%2BG.%26atitle%3DTargeting%2520protein%2520kinases%2520in%2520cancer%2520therapy%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2002%26volume%3D5%26spage%3D701%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule−kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule%E2%88%92kinase+interaction+map+for+clinical+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule%25E2%2588%2592kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Gill, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. A</span><span> </span><span class="NLM_article-title">Comparison of Physicochemical Property Profiles of Marketed Oral Drugs and Orally Bioavailable Anti-Cancer Protein Kinase Inhibitors in Clinical Development</span> <span class="citation_source-journal">Curr Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1408</span><span class="NLM_x">–</span> <span class="NLM_lpage">1422</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.2174%2F156802607781696819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=17692029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1SmtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1408-1422&author=A.+L.+Gillauthor=M.+Verdonkauthor=R.+G.+Boyleauthor=R.+A+Taylor&title=Comparison+of+Physicochemical+Property+Profiles+of+Marketed+Oral+Drugs+and+Orally+Bioavailable+Anti-Cancer+Protein+Kinase+Inhibitors+in+Clinical+Development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A comparison of physicochemical property profiles of marketed oral drugs and orally bioavailable anti-cancer protein kinase inhibitors in clinical development</span></div><div class="casAuthors">Gill, Adrian L.; Verdonk, Marcel; Boyle, Robert G.; Taylor, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1408-1422</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  This manuscript describes a comparison of the physicochem. properties of marketed oral drugs with those of 45 structurally confirmed orally bioavailable anti-cancer protein kinase inhibitors currently in different phases of clin. development.  It is evident from the data presented that these kinase inhibitors are on av. larger (over 110Da), more lipophilic (over 1.5 log units) and more complex (approx. two more rotatable bonds) than those of marketed oral drugs.  In contrast, hydrogen bond donor (HBD) and hydrogen bond acceptor (HBA) counts are not significantly different.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY8vw6yZzFirVg90H21EOLACvtfcHk0lgBqT3Vl14Oqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1SmtbvM&md5=03b0a1943f8d1a08ed436da75e9a296f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F156802607781696819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607781696819%26sid%3Dliteratum%253Aachs%26aulast%3DGill%26aufirst%3DA.%2BL.%26aulast%3DVerdonk%26aufirst%3DM.%26aulast%3DBoyle%26aufirst%3DR.%2BG.%26aulast%3DTaylor%26aufirst%3DR.%2BA%26atitle%3DComparison%2520of%2520Physicochemical%2520Property%2520Profiles%2520of%2520Marketed%2520Oral%2520Drugs%2520and%2520Orally%2520Bioavailable%2520Anti-Cancer%2520Protein%2520Kinase%2520Inhibitors%2520in%2520Clinical%2520Development%26jtitle%3DCurr%2520Top.%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D1408%26epage%3D1422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegel, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savin, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durst, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hipskind, P. A.</span><span> </span><span class="NLM_article-title">Characteristic physical properties and structural fragments of marketed oral drugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">232</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030267j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=224-232&author=M.+Viethauthor=M.+G.+Siegelauthor=R.+E.+Higgsauthor=I.+A.+Watsonauthor=D.+H.+Robertsonauthor=K.+A.+Savinauthor=G.+L.+Durstauthor=P.+A.+Hipskind&title=Characteristic+physical+properties+and+structural+fragments+of+marketed+oral+drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm030267j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030267j%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DSiegel%26aufirst%3DM.%2BG.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DSavin%26aufirst%3DK.%2BA.%26aulast%3DDurst%26aufirst%3DG.%2BL.%26aulast%3DHipskind%26aufirst%3DP.%2BA.%26atitle%3DCharacteristic%2520physical%2520properties%2520and%2520structural%2520fragments%2520of%2520marketed%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D224%26epage%3D232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span> </span><span class="NLM_article-title">Kinomics: characterizing the therapeutically validated kinase space</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">839</span><span class="NLM_x">–</span> <span class="NLM_lpage">46</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=839-46&author=M.+Viethauthor=J.+J.+Sutherlandauthor=D.+H.+Robertsonauthor=R.+M.+Campbell&title=Kinomics%3A+characterizing+the+therapeutically+validated+kinase+space"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DKinomics%253A%2520characterizing%2520the%2520therapeutically%2520validated%2520kinase%2520space%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D839%26epage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Chemical fragments as foundations for understanding target space and activity prediction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2689</span><span class="NLM_x">–</span> <span class="NLM_lpage">2700</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701399f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2689-2700&author=J.+J.+Sutherlandauthor=R.+E.+Higgsauthor=I.+Watsonauthor=M.+Vieth&title=Chemical+fragments+as+foundations+for+understanding+target+space+and+activity+prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm701399f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701399f%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DWatson%26aufirst%3DI.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DChemical%2520fragments%2520as%2520foundations%2520for%2520understanding%2520target%2520space%2520and%2520activity%2520prediction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2689%26epage%3D2700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0ljVeWoI2Vd3-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Fernandez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddipati, S.</span><span> </span><span class="NLM_article-title">A priori inference of cross reactivity for drug-targeted kinases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">3092</span><span class="NLM_x">–</span> <span class="NLM_lpage">3100</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060163j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3092-3100&author=A.+Fernandezauthor=S.+Maddipati&title=A+priori+inference+of+cross+reactivity+for+drug-targeted+kinases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm060163j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060163j%26sid%3Dliteratum%253Aachs%26aulast%3DFernandez%26aufirst%3DA.%26aulast%3DMaddipati%26aufirst%3DS.%26atitle%3DA%2520priori%2520inference%2520of%2520cross%2520reactivity%2520for%2520drug-targeted%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3092%26epage%3D3100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapland, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoorn, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Global mapping of pharmacological space</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">805</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&author=G.+V.+Paoliniauthor=R.+H.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+mapping+of+pharmacological+space"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0ljVeWoI2Vd3-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520mapping%2520of%2520pharmacological%2520space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Liew, C. Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, C. W.</span><span> </span><span class="NLM_article-title">SVM Model for Virtual Screening of Lck Inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">877</span><span class="NLM_x">–</span> <span class="NLM_lpage">885</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800387z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXislOiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=877-885&author=C.+Y.+M.+Liewauthor=H.+Xiaoauthor=X.+Liuauthor=C.+W.+Yap&title=SVM+Model+for+Virtual+Screening+of+Lck+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">SVM Model for Virtual Screening of Lck Inhibitors</span></div><div class="casAuthors">Liew, Chin Y.; Ma, Xiao H.; Liu, Xianghui; Yap, Chun W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">877-885</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lymphocyte-specific protein tyrosine kinase (Lck) inhibitors have treatment potential for autoimmune diseases and transplant rejection.  A support vector machine (SVM) model trained with 820 pos. compds. (Lck inhibitors) and 70 neg. compds. (Lck noninhibitors) combined with 65 142 generated putative negatives was developed for predicting compds. with a Lck inhibitory activity of IC50 ≤ 10 μM.  The SVM model, with an estd. sensitivity of greater than 83% and specificity of greater than 99%, was used to screen 168 014 compds. in the MDDR and was found to have a yield of 45.8% and a false pos. rate of 0.52%.  The model was also able to identify novel Lck inhibitors and distinguish inhibitors from structurally similar noninhibitors at a false pos. rate of 0.27%.  To the best of our knowledge, the SVM model developed in this work is the first model with a broad applicability domain and low false pos. rate, which makes it very suitable for the virtual screening of chem. libraries for Lck inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRGGWnqtxkzrVg90H21EOLACvtfcHk0ljVeWoI2Vd3-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXislOiurw%253D&md5=1fbd326f057f173f26b4a2231b5f3cfb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fci800387z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800387z%26sid%3Dliteratum%253Aachs%26aulast%3DLiew%26aufirst%3DC.%2BY.%2BM.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYap%26aufirst%3DC.%2BW.%26atitle%3DSVM%2520Model%2520for%2520Virtual%2520Screening%2520of%2520Lck%2520Inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D877%26epage%3D885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Kitchen, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decornez, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furr, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Docking and Scoring in Virtual Screening for Drug Discovery: Methods and Applications</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">949</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnrd1549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=15520816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=935-949&author=D.+B.+Kitchenauthor=H.+Decornezauthor=J.+R.+Furrauthor=J.+Bajorath&title=Docking+and+Scoring+in+Virtual+Screening+for+Drug+Discovery%3A+Methods+and+Applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and scoring in virtual screening for drug discovery: methods and applications</span></div><div class="casAuthors">Kitchen, Douglas B.; Decornez, Helene; Furr, John R.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">935-949</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Computational approaches that 'dock' small mols. into the structures of macromol. targets and 'score' their potential complementarity to binding sites are widely used in hit identification and lead optimization.  Indeed, there are now a no. of drugs whose development was heavily influenced by or based on structure-based design and screening strategies, such as HIV protease inhibitors.  Nevertheless, there remain significant challenges in the application of these approaches, in particular in relation to current scoring schemes.  Here, we review key concepts and specific features of small-mol.-protein docking methods, highlight selected applications and discuss recent advances that aim to address the acknowledged limitations of established approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMLibgFe3mfLVg90H21EOLACvtfcHk0lgOYafzjNIw_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFemtrg%253D&md5=875a2b37a4299181509a1922b11dbd2f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd1549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1549%26sid%3Dliteratum%253Aachs%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DDecornez%26aufirst%3DH.%26aulast%3DFurr%26aufirst%3DJ.%2BR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDocking%2520and%2520Scoring%2520in%2520Virtual%2520Screening%2520for%2520Drug%2520Discovery%253A%2520Methods%2520and%2520Applications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D935%26epage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Virtual screening of chemical libraries</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">862</span><span class="NLM_x">–</span> <span class="NLM_lpage">865</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnature03197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=15602552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=862-865&author=B.+K.+Shoichet&title=Virtual+screening+of+chemical+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of chemical libraries</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">862-865</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Virtual screening uses computer-based methods to discover new ligands on the basis of biol. structures.  Although widely heralded in the 1970s and 1980s, the technique has since struggled to meet its initial promise, and drug discovery remains dominated by empirical screening.  Recent successes in predicting new ligands and their receptor-bound structures, and better rates of ligand discovery compared to empirical screening, have reignited interest in virtual screening, which is now widely used in drug discovery, albeit on a more limited scale than empirical screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY9IbbFldkrVg90H21EOLACvtfcHk0lgOYafzjNIw_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK&md5=e351e226e8985d7c304a05fb67dfdc60</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature03197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03197%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DVirtual%2520screening%2520of%2520chemical%2520libraries%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D862%26epage%3D865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Vieth, M.; Siegel, M.</span><span> </span><span class="NLM_article-title">Structural fragments in marketed oral drugs</span>. In  <span class="citation_source-book">Fragment Based Drug Discovery</span>; <span class="NLM_contrib-group">Erlanson, J.</span>, Ed.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">123</span>− <span class="NLM_lpage">135</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=123-135&author=M.+Vieth&author=M.+Siegelauthor=J.+Erlanson&title=Fragment+Based+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DStructural%2520fragments%2520in%2520marketed%2520oral%2520drugs%26btitle%3DFragment%2520Based%2520Drug%2520Discovery%26aulast%3DErlanson%26aufirst%3DJ.%26pub%3DWiley-VCH%26date%3D2006%26spage%3D123%26epage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Joachims, T.</span><span> </span><span class="NLM_article-title">Making Large-Scale SVM Learning Practical</span>. In  <span class="citation_source-book">Advances in Kernel Methods—Support Vector Learning</span>; <span class="NLM_contrib-group">Schölkopf, B.; Burges, C.; Smola, A.</span>, Eds.; <span class="NLM_publisher-name">MIT Press</span>: <span class="NLM_publisher-loc">Cambridge</span>,<span class="NLM_x"> </span><span class="NLM_year">1999</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&author=T.+Joachimsauthor=B.+Sch%C3%B6lkopf&author=C.+Burges&author=A.+Smola&title=Advances+in+Kernel+Methods%E2%80%94Support+Vector+Learning"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJoachims%26aufirst%3DT.%26atitle%3DMaking%2520Large-Scale%2520SVM%2520Learning%2520Practical%26btitle%3DAdvances%2520in%2520Kernel%2520Methods%25E2%2580%2594Support%2520Vector%2520Learning%26aulast%3DSch%25C3%25B6lkopf%26aufirst%3DB.%26pub%3DMIT%2520Press%26date%3D1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Carhart, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkataraghavan, R.</span><span> </span><span class="NLM_article-title">Atom pairs as molecular features in structure−activity studies: definition and applications</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">64</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci00046a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADyaL2MXitVehu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1985&pages=64-73&author=R.+E.+Carhartauthor=D.+H.+Smithauthor=R.+Venkataraghavan&title=Atom+pairs+as+molecular+features+in+structure%E2%88%92activity+studies%3A+definition+and+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Atom pairs as molecular features in structure-activity studies:  definition and applications</span></div><div class="casAuthors">Carhart, Raymond E.; Smith, Dennis H.; Venkataraghavan, R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">64-73</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">A simple type of substructure called an atom pair is defined in terms of the at. environments of, and shortest path sepns. between, all pairs of atoms in the topol. representation of a chem. structure.  An algorithm is presented for computing atom pairs from such a representation.  Two applications of atom pairs to structure-activity problems are described.  In the 1st, a measure of similarity between compds. is defined, and the use of this measure in probing large data bases of structures is discussed.  In the 2nd, an heuristic technique called trend vector anal. is described.  The trend vector summarizes the correlation, within a set of structures, of the occurrence of atom pairs of different types with measured biol. activity.  These correlations can be used to est. the biol. activity of new compds.  A comparison of trend vector anal. with discriminant plane anal. is presented for 1 series of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV9lxLct8p0rVg90H21EOLACvtfcHk0lgOYafzjNIw_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXitVehu7o%253D&md5=9e2cc9e9fda27db87328b738cd6878e2</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci00046a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00046a002%26sid%3Dliteratum%253Aachs%26aulast%3DCarhart%26aufirst%3DR.%2BE.%26aulast%3DSmith%26aufirst%3DD.%2BH.%26aulast%3DVenkataraghavan%26aufirst%3DR.%26atitle%3DAtom%2520pairs%2520as%2520molecular%2520features%2520in%2520structure%25E2%2588%2592activity%2520studies%253A%2520definition%2520and%2520applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1985%26volume%3D25%26spage%3D64%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Doucet, J.-P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panaye, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nonlinear, B.</span><span> </span><span class="NLM_article-title">SVM Approaches to QSPR/QSAR Studies and Drug Design</span> <span class="citation_source-journal">Curr. Comput.-Aided Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">289</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.2174%2F157340907782799372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVSqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=263-289&author=J.-P.+B.+Doucetauthor=F.+Xiaauthor=H.+Panayeauthor=A.+Fanauthor=B.+Nonlinear&title=SVM+Approaches+to+QSPR%2FQSAR+Studies+and+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nonlinear SVM approaches to QSPR/QSAR studies and drug design</span></div><div class="casAuthors">Doucet, Jean-Pierre; Barbault, Florent; Xia, Hairong; Panaye, Annick; Fan, Botao</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-289</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Recently, a new promising nonlinear method, the support vector machine (SVM), was proposed by Vapnik.  It rapidly found numerous applications in chem., biochem. and pharmacochem.  Several attempts using SVM in drug design have been reported.  It became an attractive nonlinear approach in this field.  In this review, the theor. basis of SVM in classification and regression is briefly described.  Its applications in QSPR/QSAR studies, and particularly in drug design are discussed.  Comparative studies with some linear and other nonlinear methods show SVM's high performance both in classification and correlation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlUWphlsyC4rVg90H21EOLACvtfcHk0ljf-z-Hwf3BXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVSqtLk%253D&md5=5552439113da40b712c3467d7277bf95</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F157340907782799372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340907782799372%26sid%3Dliteratum%253Aachs%26aulast%3DDoucet%26aufirst%3DJ.-P.%2BB.%26aulast%3DXia%26aufirst%3DF.%26aulast%3DPanaye%26aufirst%3DH.%26aulast%3DFan%26aufirst%3DA.%26aulast%3DNonlinear%26aufirst%3DB.%26atitle%3DSVM%2520Approaches%2520to%2520QSPR%252FQSAR%2520Studies%2520and%2520Drug%2520Design%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2007%26volume%3D3%26spage%3D263%26epage%3D289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Han, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xuec, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Z. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. Z.</span><span> </span><span class="NLM_article-title">A support vector machines approach for virtual screening of active compounds of single and multiple mechanisms from large libraries at an improved hit-rate and enrichment factor</span> <span class="citation_source-journal">J. Mol. Graph. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1276</span><span class="NLM_x">–</span> <span class="NLM_lpage">1286</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1016%2Fj.jmgm.2007.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=18218332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvVWntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=1276-1286&author=L.+Y.+Hanauthor=X.+H.+Maauthor=H.+H.+Linauthor=J.+Jiaauthor=F.+Zhuauthor=Y.+Xuecauthor=Z.+R.+Liauthor=Z.+W.+Caoauthor=Z.+L.+Jiauthor=Y.+Z.+Chen&title=A+support+vector+machines+approach+for+virtual+screening+of+active+compounds+of+single+and+multiple+mechanisms+from+large+libraries+at+an+improved+hit-rate+and+enrichment+factor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A support vector machines approach for virtual screening of active compounds of single and multiple mechanisms from large libraries at an improved hit-rate and enrichment factor</span></div><div class="casAuthors">Han, L. Y.; Ma, X. H.; Lin, H. H.; Jia, J.; Zhu, F.; Xue, Y.; Li, Z. R.; Cao, Z. W.; Ji, Z. L.; Chen, Y. Z.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1276-1286</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Support vector machines (SVM) and other machine-learning (ML) methods have been explored as ligand-based virtual screening (VS) tools for facilitating lead discovery.  While exhibiting good hit selection performance, in screening large compd. libraries, these methods tend to produce lower hit-rate than those of the best performing VS tools, partly because their training-sets contain limited spectrum of inactive compds.  We tested whether the performance of SVM can be improved by using training-sets of diverse inactive compds.  In retrospective database screening of active compds. of single mechanism (HIV protease inhibitors, DHFR inhibitors, dopamine antagonists) and multiple mechanisms (CNS active agents) from large libraries of 2.986 million compds., the yields, hit-rates, and enrichment factors of our SVM models are 52.4-78.0%, 4.7-73.8%, and 214-10,543, resp., compared to those of 62-95%, 0.65-35%, and 20-1200 by structure-based VS and 55-81%, 0.2-0.7%, and 110-795 by other ligand-based VS tools in screening libraries of ≥1 million compds.  The hit-rates are comparable and the enrichment factors are substantially better than the best results of other VS tools.  Of the predicted hits, 24.3-87.6% are outside the known hit families.  SVM appears to be potentially useful for facilitating lead discovery in VS of large compd. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxffgspRhPqrVg90H21EOLACvtfcHk0ljf-z-Hwf3BXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvVWntrk%253D&md5=ea010461c34a589f68e1f5447dbc13df</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2007.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2007.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DL.%2BY.%26aulast%3DMa%26aufirst%3DX.%2BH.%26aulast%3DLin%26aufirst%3DH.%2BH.%26aulast%3DJia%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DXuec%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DCao%26aufirst%3DZ.%2BW.%26aulast%3DJi%26aufirst%3DZ.%2BL.%26aulast%3DChen%26aufirst%3DY.%2BZ.%26atitle%3DA%2520support%2520vector%2520machines%2520approach%2520for%2520virtual%2520screening%2520of%2520active%2520compounds%2520of%2520single%2520and%2520multiple%2520mechanisms%2520from%2520large%2520libraries%2520at%2520an%2520improved%2520hit-rate%2520and%2520enrichment%2520factor%26jtitle%3DJ.%2520Mol.%2520Graph.%2520Model.%26date%3D2008%26volume%3D26%26spage%3D1276%26epage%3D1286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Gedeck, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R. A.</span><span> </span><span class="NLM_article-title">Exploiting QSAR models in lead optimization</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">575</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=18600573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosFWmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=569-575&author=P.+Gedeckauthor=R.+A.+Lewis&title=Exploiting+QSAR+models+in+lead+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting QSAR models in lead optimization</span></div><div class="casAuthors">Gedeck, Peter; Lewis, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">569-575</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  QSAR models can play a vital role in both the opening phase and the endgame of lead optimization.  In the opening phase, there is often a large quantity of data from high-throughput screening (HTS), and potential leads need to be selected from several distinct chemotypes.  In the endgame, the throughput of the final, crit. ADMET and pharmacokinetic assays is often not sufficient to allow full exptl. characterization of all the structures in the available time.  A considerable amt. of the current research toward new QSAR models is based on the modeling of the general ADMET phenomena, with the aim of constructing globally applicable models.  The process to construct QSAR models is relatively straightforward; however, it is also simple to build misleading, or even incorrect, models.  This review considers the key developments in the field of QSAR modeling: how QSAR models are constructed, how they can be validated, their reliability and their applicability.  If applied carefully and appropriately, the QSAR technique has a valuable role to play during lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGgUjjqE55JbVg90H21EOLACvtfcHk0ljf-z-Hwf3BXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosFWmurc%253D&md5=bf4d2d141bda32bf4fc1b01d1bd46b57</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGedeck%26aufirst%3DP.%26aulast%3DLewis%26aufirst%3DR.%2BA.%26atitle%3DExploiting%2520QSAR%2520models%2520in%2520lead%2520optimization%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2008%26volume%3D11%26spage%3D569%26epage%3D575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Variable selection QSAR modeling, model validation, and virtual screening</span> <span class="citation_source-journal">Annu Rep. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1016%2FS1574-1400%2806%2902007-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjslWru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=113-126&author=A.+Tropsha&title=Variable+selection+QSAR+modeling%2C+model+validation%2C+and+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Variable selection QSAR modeling, model validation, and virtual screening</span></div><div class="casAuthors">Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Computational Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">113-126</span>CODEN:
                <span class="NLM_cas:coden">ARCCC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-1400</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review on the quant. structure-activity relationship model validation as an integral component of model development.  It illustrates that rigorously validated QSAR models with statistically significant external predictive power ultimately afford their application as reliable virtual screening tools for database mining or chem. library design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquHgZ4ShzkybVg90H21EOLACvtfcHk0ljf-z-Hwf3BXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjslWru7g%253D&md5=ebc18b6f2f01205d32ee69ed7f2e1ea2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS1574-1400%2806%2902007-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1574-1400%252806%252902007-X%26sid%3Dliteratum%253Aachs%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DVariable%2520selection%2520QSAR%2520modeling%252C%2520model%2520validation%252C%2520and%2520virtual%2520screening%26jtitle%3DAnnu%2520Rep.%2520Comput.%2520Chem.%26date%3D2006%26volume%3D2%26spage%3D113%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Sutherland, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nandigam, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erickson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Lessons in Molecular Recognition. 2. Assessing and Improving Cross-Docking Accuracy</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2293</span><span class="NLM_x">–</span> <span class="NLM_lpage">2302</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700253h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1SgtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2293-2302&author=J.+J.+Sutherlandauthor=R.+K.+Nandigamauthor=J.+A.+Ericksonauthor=M.+Vieth&title=Lessons+in+Molecular+Recognition.+2.+Assessing+and+Improving+Cross-Docking+Accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in Molecular Recognition. 2. Assessing and Improving Cross-Docking Accuracy</span></div><div class="casAuthors">Sutherland, Jeffrey J.; Nandigam, Ravi K.; Erickson, Jon A.; Vieth, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2293-2302</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking methods are used to predict the manner in which a ligand binds to a protein receptor.  Many studies have assessed the success rate of programs in self-docking tests, whereby a ligand is docked into the protein structure from which it was extd.  Cross-docking, or using a protein structure from a complex contg. a different ligand, provides a more realistic assessment of a docking program's ability to reproduce X-ray results.  Cross-docking was performed with CDocker, Fred, and Rocs using multiple X-ray structures for eight proteins (two kinases, one nuclear hormone receptor, one serine protease, two metalloproteases, and two phosphodiesterases).  While av. cross-docking accuracy is not encouraging, it is shown that using the protein structure from the complex that contains the bound ligand most similar to the docked ligand increases docking accuracy for all methods ("similarity selection").  Identifying the most successful protein conformer ("best selection") and similarity selection substantially reduce the difference between self-docking and av. cross-docking accuracy.  The authors identify universal predictors of docking accuracy (i.e., showing consistent behavior across most protein-method combinations), and show that models for predicting docking accuracy built using these parameters can be used to select the most appropriate docking method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5Nh5Ky1zOc7Vg90H21EOLACvtfcHk0liB8HwEqjuceg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1SgtbbP&md5=c3792ab8645ab459941f5d3a223f9ce4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci700253h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700253h%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DNandigam%26aufirst%3DR.%2BK.%26aulast%3DErickson%26aufirst%3DJ.%2BA.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DLessons%2520in%2520Molecular%2520Recognition.%25202.%2520Assessing%2520and%2520Improving%2520Cross-Docking%2520Accuracy%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2293%26epage%3D2302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Nandigam, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erickson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, J. J.</span><span> </span><span class="NLM_article-title">Predicting the accuracy of ligand overlay methods with Random Forest models</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2386</span><span class="NLM_x">–</span> <span class="NLM_lpage">2394</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800216f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSntbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=2386-2394&author=R.+K.+Nandigamauthor=D.+A.+Evansauthor=J.+A.+Ericksonauthor=S.+Kimauthor=J.+J.+Sutherland&title=Predicting+the+accuracy+of+ligand+overlay+methods+with+Random+Forest+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting the accuracy of ligand overlay methods with Random Forest models</span></div><div class="casAuthors">Nandigam, Ravi K.; Evans, David A.; Erickson, Jon A.; Kim, Sangtae; Sutherland, Jeffrey J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2386-2394</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accuracy of binding mode prediction using std. mol. overlay methods (ROCS, FlexS, Phase, and FieldCompare) is studied.  Previous work has shown that simple decision tree modeling can be used to improve accuracy by selection of the best overlay template.  This concept is extended to the use of Random Forest (RF) modeling for template and algorithm selection.  An extensive data set of 815 ligand-bound X-ray structures representing 5 gene families was used for generating ca. 70,000 overlays using four programs.  RF models, trained using std. measures of ligand and protein similarity and Lipinski-related descriptors, are used for automatically selecting the ref. ligand and overlay method maximizing the probability of reproducing the overlay deduced from X-ray structures (i.e., using rmsd ≤ 2 Å as the criteria for success).  RF model scores are highly predictive of overlay accuracy, and their use in template and method selection produces correct overlays in 57% of cases for 349 overlay ligands not used for training RF models.  The inclusion in the models of protein sequence similarity enables the use of templates bound to related protein structures, yielding useful results even for proteins having no available X-ray structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbbC86gHhVarVg90H21EOLACvtfcHk0liB8HwEqjuceg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSntbrE&md5=c1e6661350b640f98073b753017b9c23</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci800216f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800216f%26sid%3Dliteratum%253Aachs%26aulast%3DNandigam%26aufirst%3DR.%2BK.%26aulast%3DEvans%26aufirst%3DD.%2BA.%26aulast%3DErickson%26aufirst%3DJ.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26atitle%3DPredicting%2520the%2520accuracy%2520of%2520ligand%2520overlay%2520methods%2520with%2520Random%2520Forest%2520models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D2386%26epage%3D2394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Dooley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houghten, R.</span><span> </span><span class="NLM_article-title">The use of positional scanning synthetic peptide combinatorial libraries for the rapid determination of opioid receptor ligands</span> <span class="citation_source-journal">Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1509</span><span class="NLM_x">–</span> <span class="NLM_lpage">1517</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1993&pages=1509-1517&author=C.+Dooleyauthor=R.+Houghten&title=The+use+of+positional+scanning+synthetic+peptide+combinatorial+libraries+for+the+rapid+determination+of+opioid+receptor+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDooley%26aufirst%3DC.%26aulast%3DHoughten%26aufirst%3DR.%26atitle%3DThe%2520use%2520of%2520positional%2520scanning%2520synthetic%2520peptide%2520combinatorial%2520libraries%2520for%2520the%2520rapid%2520determination%2520of%2520opioid%2520receptor%2520ligands%26jtitle%3DLife%2520Sci.%26date%3D1993%26volume%3D52%26spage%3D1509%26epage%3D1517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Cui, J.; Funk, L. A.; Jia, L.; Kung, P.-P.; Meng, J. J.; Nambu, M. D.; Pairish, M. A.; Sheng, H.; Tran-Dube, M. B.</span><span> </span><span class="NLM_article-title">Aminoheteroaryl compounds as protein tyrosine kinase inhibitors</span>. PCT Gazette, WO 2006/021886 Al,<span class="NLM_x"> </span><span class="NLM_year">2006</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=J.+Cui&author=L.+A.+Funk&author=L.+Jia&author=P.-P.+Kung&author=J.+J.+Meng&author=M.+D.+Nambu&author=M.+A.+Pairish&author=H.+Sheng&author=M.+B.+Tran-Dube&title=Aminoheteroaryl+compounds+as+protein+tyrosine+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%26atitle%3DAminoheteroaryl%2520compounds%2520as%2520protein%2520tyrosine%2520kinase%2520inhibitors%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Givehchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span> </span><span class="NLM_article-title">Analysis of activity space by fragment fingerprints, 2D descriptors, and multitarget dependent transformation of 2D descriptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1078</span><span class="NLM_x">–</span> <span class="NLM_lpage">1083</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0500233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvF2is7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1078-1083&author=A.+Givehchiauthor=A.+Benderauthor=R.+C.+Glen&title=Analysis+of+activity+space+by+fragment+fingerprints%2C+2D+descriptors%2C+and+multitarget+dependent+transformation+of+2D+descriptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Activity Space by Fragment Fingerprints, 2D Descriptors, and Multitarget Dependent Transformation of 2D Descriptors</span></div><div class="casAuthors">Givehchi, Alireza; Bender, Andreas; Glen, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1078-1083</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effect of multitarget dependent descriptor transformation on classification performance is explored in this work.  To this end decision trees as well as neural net QSAR in combination with PLS were applied to predict the activity class of 5HT3 ligands, angiotensin converting enzyme inhibitors, 3-hydroxyl-3-Me glutaryl CoA reductase inhibitors, platelet activating factor antagonists, and thromboxane A2 antagonists.  Physicochem. descriptors calcd. by MOE and fragment-based descriptors (MOLPRINT 2D) were employed to generate descriptor vectors.  In a subsequent step the physicochem. descriptor vectors were transformed to a lower dimensional space using multitarget dependent descriptor transformation.  Cross-validation of the original physicochem. descriptors in combination with decision trees and neural net QSAR as well as cross-validation of PLS multitarget transformed descriptors with neural net QSAR were performed.  For comparison this was repeated using fragment-based descriptors in combination with decision trees.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoalnrrxP0_5bVg90H21EOLACvtfcHk0liB8HwEqjuceg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvF2is7g%253D&md5=8f8c84590d6d36471687162e652f9d43</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci0500233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0500233%26sid%3Dliteratum%253Aachs%26aulast%3DGivehchi%26aufirst%3DA.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26atitle%3DAnalysis%2520of%2520activity%2520space%2520by%2520fragment%2520fingerprints%252C%25202D%2520descriptors%252C%2520and%2520multitarget%2520dependent%2520transformation%2520of%25202D%2520descriptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D1078%26epage%3D1083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohde, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selzer, P.</span><span> </span><span class="NLM_article-title">Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport properties</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3727</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000942e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3714-3727&author=P.+Ertlauthor=B.+Rohdeauthor=P.+Selzer&title=Fast+calculation+of+molecular+polar+surface+area+as+a+sum+of+fragment-based+contributions+and+its+application+to+the+prediction+of+drug+transport+properties"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of Molecular Polar Surface Area as a Sum of Fragment-Based Contributions and Its Application to the Prediction of Drug Transport Properties</span></div><div class="casAuthors">Ertl, Peter; Rohde, Bernhard; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3714-3717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. polar surface area (PSA), i.e., surface belonging to polar atoms, is a descriptor that was shown to correlate well with passive mol. transport through membranes and, therefore, allows prediction of transport properties of drugs.  The calcn. of PSA, however, is rather time-consuming because of the necessity to generate a reasonable 3D mol. geometry and the calcn. of the surface itself.  A new approach for the calcn. of the PSA is presented here, based on the summation of tabulated surface contributions of polar fragments.  The method, termed topol. PSA (TPSA), provides results which are practically identical with the 3D PSA (the correlation coeff. between 3D PSA and fragment-based TPSA for 34 810 mols. from the World Drug Index is 0.99), while the computation speed is 2-3 orders of magnitude faster.  The new methodol. may, therefore, be used for fast bioavailability screening of virtual libraries having millions of mols.  This article describes the new methodol. and shows the results of validation studies based on sets of published absorption data, including intestinal absorption, Caco-2 monolayer penetration, and blood-brain barrier penetration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhP6Qncx_h6LVg90H21EOLACvtfcHk0lirU452p4ZzJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D&md5=23ae4e7708bd8ddeb4a24ad160a29bd6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm000942e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000942e%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRohde%26aufirst%3DB.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DFast%2520calculation%2520of%2520molecular%2520polar%2520surface%2520area%2520as%2520a%2520sum%2520of%2520fragment-based%2520contributions%2520and%2520its%2520application%2520to%2520the%2520prediction%2520of%2520drug%2520transport%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3714%26epage%3D3727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span> <span class="citation_source-book">Chemaxon LogD</span>; <span class="NLM_publisher-name">ChemAxon Kft</span>: <span class="NLM_publisher-loc">Budapest, 1037 Hungary</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span><span class="NLM_x">; </span><a href="http://www.chemaxon.com" class="extLink">www.chemaxon.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Chemaxon+LogD%3B+ChemAxon+Kft%3A+Budapest%2C+1037+Hungary%2C+2008%3B+www.chemaxon.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DChemaxon%2520LogD%26pub%3DChemAxon%2520Kft%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Vieth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gragg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmerle, H.</span><span> </span><span class="NLM_article-title">Kinomics−structural biology and chemogenomics of kinase inhibitors and targets</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1697</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">257</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1697&publication_year=2004&pages=243-257&author=M.+Viethauthor=R.+E.+Higgsauthor=D.+H.+Robertsonauthor=M.+Shapiroauthor=E.+A.+Graggauthor=H.+Hemmerle&title=Kinomics%E2%88%92structural+biology+and+chemogenomics+of+kinase+inhibitors+and+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DHiggs%26aufirst%3DR.%2BE.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DShapiro%26aufirst%3DM.%26aulast%3DGragg%26aufirst%3DE.%2BA.%26aulast%3DHemmerle%26aufirst%3DH.%26atitle%3DKinomics%25E2%2588%2592structural%2520biology%2520and%2520chemogenomics%2520of%2520kinase%2520inhibitors%2520and%2520targets%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2004%26volume%3D1697%26spage%3D243%26epage%3D257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Voth, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, S. P.</span><span> </span><span class="NLM_article-title">The Role of Halogen Bonding in Inhibitor Recognition and Binding by Protein Kinases</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1336</span><span class="NLM_x">–</span> <span class="NLM_lpage">1348</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.2174%2F156802607781696846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=17692024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1SmtbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1336-1348&author=R.+A.+Vothauthor=S.+P.+Ho&title=The+Role+of+Halogen+Bonding+in+Inhibitor+Recognition+and+Binding+by+Protein+Kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The role of halogen bonding in inhibitor recognition and binding by protein kinases</span></div><div class="casAuthors">Voth, Andrea Regier; Ho, P. Shing</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1336-1348</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Halogen bonds are short-range mol. interactions that are analogous to classical hydrogen bonds, except that a polarized halogen replaces the hydrogen as the acid in the Lewis acid/base pair.  Such interactions occur regularly in the structures of many ligand-protein complexes, but have only recently been recognized in biol. systems as a distinct class with well-defined phys. characteristics.  In this review, we survey twelve single crystal structures of protein kinase complexes with halogenated ligands in order to characterize the role of halogen bonds in conferring specificity and affinity for halogenated inhibitors in this important class of enzymes.  From this survey, we attempt to identify the properties of halogen bonds that can be generally applied to bottom-up strategies for designing inhibitors for this and other enzyme targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0lKs3fgdLV7Vg90H21EOLACvtfcHk0lirU452p4ZzJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1SmtbrJ&md5=da1c1d603db9c2d93682f428198f0515</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F156802607781696846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607781696846%26sid%3Dliteratum%253Aachs%26aulast%3DVoth%26aufirst%3DR.%2BA.%26aulast%3DHo%26aufirst%3DS.%2BP.%26atitle%3DThe%2520Role%2520of%2520Halogen%2520Bonding%2520in%2520Inhibitor%2520Recognition%2520and%2520Binding%2520by%2520Protein%2520Kinases%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D1336%26epage%3D1348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Muegge, I.</span><span> </span><span class="NLM_article-title">Selection criteria for drug-like compounds</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">302</span><span class="NLM_x">–</span> <span class="NLM_lpage">321</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1002%2Fmed.10041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=12647312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjsVWitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=302-321&author=I.+Muegge&title=Selection+criteria+for+drug-like+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Selection criteria for drug-like compounds</span></div><div class="casAuthors">Muegge, Ingo</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">302-321</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The fast identification of quality lead compds. in the pharmaceutical industry through a combination of high throughput synthesis and screening has become more challenging in recent years.  Although the no. of available compds. for high throughput screening (HTS) has dramatically increased, large-scale random combinatorial libraries have contributed proportionally less to identify novel leads for drug discovery projects.  Therefore, the concept of 'drug-likeness' of compd. selections has become a focus in recent years.  In parallel, the low success rate of converting lead compds. into drugs often due to unfavorable pharmacokinetic parameters has sparked a renewed interest in understanding more clearly what makes a compd. drug-like.  Various approaches have been devised to address the drug-likeness of mols. employing retrospective analyses of known drug collections as well as attempting to capture 'chem. wisdom' in algorithms.  For example, simple property counting schemes, machine learning methods, regression models, and clustering methods have been employed to distinguish between drugs and non-drugs.  Here we review computational techniques to address the drug-likeness of compd. selections and offer an outlook for the further development of the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEqC8ZM-CqZrVg90H21EOLACvtfcHk0lgm7EX9jRW7EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjsVWitLY%253D&md5=a855034b700ff48c64262ca90f43a928</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fmed.10041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.10041%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26atitle%3DSelection%2520criteria%2520for%2520drug-like%2520compounds%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2003%26volume%3D23%26spage%3D302%26epage%3D321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Lipinski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A.</span><span> </span><span class="NLM_article-title">Navigating chemical space for biology and medicine</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">855</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnature03193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=15602551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=855-861&author=C.+Lipinskiauthor=A.+Hopkins&title=Navigating+chemical+space+for+biology+and+medicine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating chemical space for biology and medicine</span></div><div class="casAuthors">Lipinski, Christopher; Hopkins, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite over a century of applying org. synthesis to the search for drugs, the authors are still far from even a cursory examn. of the vast no. of possible small mols. that could be created.  Indeed, a thorough examn. of all 'chem. space' is practically impossible.  Given this, what are the best strategies for identifying small mols. that modulate biol. targets And how might such strategies differ, depending on whether the primary goal is to understand biol. systems or to develop potential drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hQm7xaX5MLVg90H21EOLACvtfcHk0lgm7EX9jRW7EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO&md5=077237d55fa1225fd0aaee481e376172</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature03193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03193%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%26aulast%3DHopkins%26aufirst%3DA.%26atitle%3DNavigating%2520chemical%2520space%2520for%2520biology%2520and%2520medicine%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D855%26epage%3D861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Deanda, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reno, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span> </span><span class="NLM_article-title">Kinase-Targeted Library Design through the Application of the PharmPrint Methodology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2395</span><span class="NLM_x">–</span> <span class="NLM_lpage">2403</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800276t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWgt7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=2395-2403&author=F.+Deandaauthor=E.+L.+Stewartauthor=M.+J.+Renoauthor=D.+H.+Drewry&title=Kinase-Targeted+Library+Design+through+the+Application+of+the+PharmPrint+Methodology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase-Targeted Library Design through the Application of the PharmPrint Methodology</span></div><div class="casAuthors">Deanda, Felix; Stewart, Eugene L.; Reno, Michael J.; Drewry, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2395-2403</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The PharmPrint methodol., as modified and implemented by Deanda and Stewart, was prospectively evaluated for use as a virtual high-throughput screening tool by applying it to the design of target-focused arrays.  To this end, PharmPrint quant. structure-activity relationship (QSAR) models for the prediction of AKT1, Aurora-A, and ROCK1 inhibition were constructed and used to virtually screen two large combinatorial libraries.  Based on predicted activities, an Aurora-A targeted array and a ROCK1 targeted array were designed and synthesized.  One control group per designed array was also synthesized to assess the enrichment levels achieved by the QSAR models.  For the Aurora-A targeted array, the hit rate, against the intended target, was 42.9%, whereas that of the control group was 0%.  Thus, the enrichment level achieved by the Aurora-A QSAR model was incalculable.  For the ROCK1 targeted array, the hit rate against the intended target was 30.6%, whereas that of the control group was 5.10%, making the enrichment level achieved by the ROCK1 QSAR model 6-fold above control.  Clearly, these results support the use of the PharmPrint methodol. as a virtual screening tool for the design of kinase-targeted arrays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXVi-syw96s7Vg90H21EOLACvtfcHk0lgm7EX9jRW7EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWgt7%252FF&md5=0a7c65ed90b10dc8de5016af052304f6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci800276t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800276t%26sid%3Dliteratum%253Aachs%26aulast%3DDeanda%26aufirst%3DF.%26aulast%3DStewart%26aufirst%3DE.%2BL.%26aulast%3DReno%26aufirst%3DM.%2BJ.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DKinase-Targeted%2520Library%2520Design%2520through%2520the%2520Application%2520of%2520the%2520PharmPrint%2520Methodology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D2395%26epage%3D2403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Pellecchia, M.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery takes a virtual turn</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">274</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm901147e&amp;key=10.1038%2Fnchembio0509-274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm901147e&amp;key=19377449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm901147e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXks12kur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=274-275&author=M.+Pellecchia&title=Fragment-based+drug+discovery+takes+a+virtual+turn"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery takes a virtual turn</span></div><div class="casAuthors">Pellecchia, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">274-275</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the realm of modern drug discovery technologies, fragment-based approaches and virtual ligand screening are emerging alternative approaches to high-throughput screening (HTS).  For as simple as it sounds, a hybrid approach in which fragments are discovered first in silico may be an unbeatable route to hit identification for some drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Xqf4HqouS7Vg90H21EOLACvtfcHk0lgm7EX9jRW7EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXks12kur4%253D&md5=03aa16de49eb18000db8376ecfb7d268</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0509-274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0509-274%26sid%3Dliteratum%253Aachs%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DFragment-based%2520drug%2520discovery%2520takes%2520a%2520virtual%2520turn%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D274%26epage%3D275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ity" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ity','PDB','2ity'); return false;">PDB: 2ity</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2j50" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2j50','PDB','2j50'); return false;">PDB: 2j50</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i17"><a href="/doi/suppl/10.1021/jm901147e">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_56607"></div></div></div></div></div><hr /></hr><p class="last">General statistics for models, historical hit rates, and prospective validation result; model performance metrics for prospective validation (1895 compounds) and libraries (179 compounds); target correlation table with sequence identity included; graphical demonstration of chronological validation and PPV concepts; biological assay description. Biological profiling data and compound data in fingerprint format are available from authors upon request. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm901147e/suppl_file/jm901147e_si_001.pdf">jm901147e_si_001.pdf (105.12 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm901147e&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm901147e%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2009.52.issue-20%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm901147e" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a824e3a8c3c52","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
