
<html lang="en"     class="pb-page"  data-request-id="0cf12c8a-c655-4c2a-a8c4-7b761e304172"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2017.60.issue-1;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b01611;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay" /></meta><meta name="dc.Creator" content="Benjamin  Merget" /></meta><meta name="dc.Creator" content="Samo  Turk" /></meta><meta name="dc.Creator" content="Sameh  Eid" /></meta><meta name="dc.Creator" content="Friedrich  Rippmann" /></meta><meta name="dc.Creator" content="Simone  Fulle" /></meta><meta name="dc.Description" content="Kinome-wide screening would have the advantage of providing structure–activity relationships against hundreds of targets simultaneously. Here, we report the generation of ligand-based activity pred..." /></meta><meta name="Description" content="Kinome-wide screening would have the advantage of providing structure–activity relationships against hundreds of targets simultaneously. Here, we report the generation of ligand-based activity pred..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 14, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01611" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01611" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01611" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01611" /></link>
        
    
    

<title>Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01611" /></meta><meta property="og:title" content="Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0011.jpeg" /></meta><meta property="og:description" content="Kinome-wide screening would have the advantage of providing structure–activity relationships against hundreds of targets simultaneously. Here, we report the generation of ligand-based activity prediction models for over 280 kinases by employing Machine Learning methods on an extensive data set of proprietary bioactivity data combined with open data. High quality (AUC &gt; 0.7) was achieved for ∼200 kinases by (1) combining open with proprietary data, (2) choosing Random Forest over alternative tested Machine Learning methods, and (3) balancing the training data sets. Tests on left-out and external data indicate a high value for virtual screening projects. Importantly, the derived models are evenly distributed across the kinome tree, allowing reliable profiling prediction for all kinase branches. The prediction quality was further improved by employing experimental bioactivity fingerprints of a small kinase subset. Overall, the generated models can support various hit identification tasks, including virtual screening, compound repurposing, and the detection of potential off-targets." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01611"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01611">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01611&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01611&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01611&amp;href=/doi/10.1021/acs.jmedchem.6b01611" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 474-485</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01539" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01666" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin++Merget">Benjamin Merget</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Samo++Turk">Samo Turk</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sameh++Eid">Sameh Eid</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Friedrich++Rippmann">Friedrich Rippmann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Simone++Fulle">Simone Fulle</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-7646-5889" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Global Computational Chemistry, Merck KGaA, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div><div class="corresp-info"><strong>*</strong>Phone: +49 6221 4261113. E-mail: <a href="/cdn-cgi/l/email-protection#c4a2b1a8a8a184a6adabeaa9bc"><span class="__cf_email__" data-cfemail="a0c6d5ccccc5e0c2c9cf8ecdd8">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01611&amp;href=/doi/10.1021%2Facs.jmedchem.6b01611" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 474–485</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 18, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 October 2016</li><li><span class="item_label"><b>Published</b> online</span>14 December 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 January 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01611" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01611</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D474%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBenjamin%2BMerget%252C%2BSamo%2BTurk%252C%2BSameh%2BEid%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D1%26contentID%3Dacs.jmedchem.6b01611%26title%3DProfiling%2BPrediction%2Bof%2BKinase%2BInhibitors%253A%2BToward%2Bthe%2BVirtual%2BAssay%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D485%26publicationDate%3DJanuary%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01611"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3375</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">51</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01611" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;Merget&quot;},{&quot;first_name&quot;:&quot;Samo&quot;,&quot;last_name&quot;:&quot;Turk&quot;},{&quot;first_name&quot;:&quot;Sameh&quot;,&quot;last_name&quot;:&quot;Eid&quot;},{&quot;first_name&quot;:&quot;Friedrich&quot;,&quot;last_name&quot;:&quot;Rippmann&quot;},{&quot;first_name&quot;:&quot;Simone&quot;,&quot;last_name&quot;:&quot;Fulle&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;474-485&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01611&quot;},&quot;abstract&quot;:&quot;Kinome-wide screening would have the advantage of providing structure–activity relationships against hundreds of targets simultaneously. Here, we report the generation of ligand-based activity prediction models for over 280 kinases by employing Machine Learning methods on an extensive data set of proprietary bioactivity data combined with open data. High quality (AUC &gt; 0.7) was achieved for ∼200 kinases by (1) combining open with proprietary data, (2) choosing Random Forest over alternative tested Machine Learning methods, and (3) balancing the training data sets. Tests on left-out and external data indicate a high value for virtual screening projects. Importantly, the derived models are evenly distributed across the kinome tree, allowing reliable profiling prediction for all kinase branches. The prediction quality was further improved by employing experimental bioactivity fingerprints of a small kinase subset. Overall, the generated models can support various hit identification tasks, including virtual scree&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01611&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01611" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01611&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01611" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01611&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01611" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01611&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01611&amp;href=/doi/10.1021/acs.jmedchem.6b01611" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01611" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01611" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01611%26sid%3Dliteratum%253Aachs%26pmid%3D27966949%26genre%3Darticle%26aulast%3DMerget%26date%3D2017%26atitle%3DProfiling%2BPrediction%2Bof%2BKinase%2BInhibitors%253A%2BToward%2Bthe%2BVirtual%2BAssay%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D1%26spage%3D474%26epage%3D485%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291831" title="Chemical specificity">Chemical specificity</a>,</li><li><a href="/action/doSearch?ConceptID=292412" title="Power">Power</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291321" title="Bioactivity">Bioactivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/jmcmar.2017.60.issue-1/20170112/jmcmar.2017.60.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Kinome-wide screening would have the advantage of providing structure–activity relationships against hundreds of targets simultaneously. Here, we report the generation of ligand-based activity prediction models for over 280 kinases by employing Machine Learning methods on an extensive data set of proprietary bioactivity data combined with open data. High quality (AUC > 0.7) was achieved for ∼200 kinases by (1) combining open with proprietary data, (2) choosing Random Forest over alternative tested Machine Learning methods, and (3) balancing the training data sets. Tests on left-out and external data indicate a high value for virtual screening projects. Importantly, the derived models are evenly distributed across the kinome tree, allowing reliable profiling prediction for all kinase branches. The prediction quality was further improved by employing experimental bioactivity fingerprints of a small kinase subset. Overall, the generated models can support various hit identification tasks, including virtual screening, compound repurposing, and the detection of potential off-targets.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20318" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20318" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein kinases have been the focus of drug discovery efforts for many years due to their central roles in signaling pathways involved in the formation and progression of human cancer, inflammation, and Alzheimer’s disease.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Until February 2016, 30 small molecules targeting kinases were approved by the FDA with many potential compounds still in clinical trials.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> The majority of kinase inhibitors bind to the highly conserved ATP-binding pocket, leading to low selectivity, which can easily translate into unwanted side effects.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Thus, having an understanding about the binding profile of kinase inhibitors is a prerequisite for drug discovery efforts. Sequence-based phylogenetic relationships of kinases do not always allow extrapolation to the bioactivity space,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> underlining the need for kinome-wide profiling data for cross-reactivity estimation. Although experimental profiling of compounds against a large fraction of the kinome is experimentally feasible, it is too expensive to be done on a regular basis for hundreds of compounds even for big pharma companies. Extensive panels of compounds tested against many different kinases exist, containing both positive and, importantly, also negative results.<a onclick="showRef(event, 'ref5 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref8 ref9 ref10">(5, 7-10)</a> Together with other open resources, like ChEMBL,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> a wealth of bioactivity measurements is freely accessible. These sources provide valuable training data for computational activity prediction models or <i>virtual assays</i>, which are of great importance for hit identification, compound repurposing and off-target detection.</div><div class="NLM_p">Several attempts to predict binding profiles or free-energy differences exist (including classical QSAR models and free energy calculation tools) and can be feasible for drug design projects (e.g., with respect to accuracy and project time lines).<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12-14)</a> The similarity ensemble approach (SEA) is based on the chemical similarity of query ligands to known inhibitors and resulted in successful predictions of unanticipated cross-reactivity.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> Aside from predictors based on chemical similarity, Machine Learning (ML) algorithms such as Support Vector Machines (SVM), Naive Bayes (NB), and Neural Networks comprise a popular toolbox for ligand-centric activity and selectivity prediction.<a onclick="showRef(event, 'ref7 ref14 ref17 ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref7 ref14 ref17 ref18 ref19 ref20 ref21 ref22">(7, 14, 17-22)</a> For instance, Yabuuchi and colleagues successfully employed SVM to detect experimentally confirmed inhibitors of GPCR and kinase targets.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Furthermore, NB classification and regression yielded kinase activity models with high predictive power.<a onclick="showRef(event, 'ref14 ref22'); return false;" href="javascript:void(0);" class="ref ref14 ref22">(14, 22)</a> Neural Networks employed by Manallack and colleagues achieved 79% correct classifications on an external test set of 120 kinase inhibitors.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Deep Learning networks showed an average AUC of 0.83 on ChEMBL data, outperforming competing methods, such as SVM, K-Nearest Neighbor (KNN), NB, and also SEA.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Beside these ML algorithms, Random Forest (RF) is a popular method to solve classification and regression problems using an ensemble of decision trees and is able to yield very good performance in QSAR modeling even without careful feature selection and extensive parameter tuning.<a onclick="showRef(event, 'ref21 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref23 ref24">(21, 23, 24)</a></div><div class="NLM_p last">Extending a traditional target-centric screen to multiple targets (e.g., kinome-wide) has the advantage of providing multidimensional structure–activity relationships against hundreds of targets simultaneously.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Several studies address this issue using a proteochemometric (PCM) approach and, thus, combine chemical information with biological data about the target.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Although PCM holds promises, this survey will primarily focus on assessing the predictive power of activity prediction models derived solely from compound fingerprints (FP) and experimental activities. Accordingly, we will address the following questions: (1) which ML algorithm is best suited for generating high-quality (HQ) activity prediction models for a large kinase panel, (2) how does the composition of the used data set and the inherent chemical diversity influence the predictive quality in internal cross-validations and external testing, (3) what is the most suited strategy to balance the data in preprocessing and thus address the class imbalance problem of ML,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and (4) can ML be employed for accurate selectivity and off-target prediction? In doing so, different ML methodologies and data balancing schemes are evaluated to yield kinase-specific models with high accuracy. A combination of encoding the chemical (<i>Morgan fingerprints</i>) and biological space (<i>bioactivity fingerprints</i>) resulted in the best activity prediction and allowed accurate inference of compound selectivity.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Material and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38514" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38514" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Data Sets</h3><div class="NLM_p">Three data sets were employed for the generation of activity prediction models, hereinafter termed <i>Proprietary</i>, <i>Open</i>, and <i>Combined</i>. (1) The data set <i>Proprietary</i> is composed from an in-house profiling panel from Merck KGaA with 4,712 compounds, 220 kinases, and 1,035,549 data points in the form of pIC<sub>50</sub> values (∼100% coverage).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> (2) The second data set (<i>Open</i>) contains the Tang set<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (which is a collection of a the kinase profiling data sets of Metz,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Davis,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and Anastassiadis<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a>), PKIS,<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29-31)</a> and a curated ChEMBL kinase inhibitor panel.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The <i>Open</i> set provides a rich source of kinase inhibitor data accessible to the public domain and can be obtained from <a href="https://github.com/Team-SKI/Publications" class="extLink">https://github.com/Team-SKI/Publications</a>. After initial filtering, the Tang data set comprises 1,356 compounds, 188 kinases, and a total of 120,194 data points (∼50% coverage). The KIBA scores introduced by Tang were converted to the negative log<sub>10</sub> of the molar concentration to make the values comparable to the remaining data sets. The <i>Open</i> set was then extended by the <i>Published Kinase Inhibitor Set</i> (PKIS) of GlaxoSmithKline, containing 366 compounds, 195 kinases, and a total of 71,369 data points in the form of pIC<sub>50</sub> values (100% coverage).<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> In the last step, the <i>Open</i> set was extended by a curated in-house databank of kinase inhibitor data from ChEMBL 21.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> This sparse panel added 38,988 compounds, 314 kinases, and 64,157 measurements. (3) The third data set is a <i>Combined</i> master table of the data sets <i>Proprietary</i> and <i>Open</i>. Overlaps in compounds and kinases from the different sources are shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. Only one representative of duplicate compounds was kept according to the following priority: <i>Proprietary</i>, Tang, PKIS, in-house ChEMBL. Furthermore, we only considered data sets with at least 50 bioactivity values per kinase (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Although it is known that bioactivity values derived from different experimental designs do not always agree and the ChEMBL database can be error prone, it is still feasible to combine these data sets for large-scale ML predictions.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The chemical space of the used data sets was subsequently analyzed by means of a Principal Component Analysis (PCA) using the statistical framework R and associated plug-ins.<a onclick="showRef(event, 'ref34 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36 ref37">(34-37)</a> Two different types of FPs were calculated for all compounds using the RDKit<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (version 2015.09.2) implemented connectivity- and feature-based Morgan FPs<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (ECFP-like and FCFP-like, respectively) with an array length of 4,096 bits and a radius of 4 each. The two FPs were concatenated for each compound. In contrast to classical binary FPs, all calculated FPs were count-based, where each feature is an integer number indicating how often a substructure appears in a molecule. The pIC<sub>50</sub> cutoff to compose the active (positive) and inactive (negative) classes for each kinase was set to 6.3, which corresponds to a concentration of 500 nM. This cutoff was also used for the generation of binary bioactivity fingerprints, where a compound is described by the experimental bioactivity against a selected subset of kinases. For evaluation, an activity threshold of pIC<sub>50</sub> = 6 (corresponding to 1 μM) was also tested.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Venn diagram of compounds (left) and kinases (right) from different sources. Whereas only few kinases are unique for a single data source, many compounds only appear in one source.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Sizes of Final Data Sets Used for Creating Activity Prediction Models</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">compounds</th><th class="colsep0 rowsep0" align="center" char=".">kinases</th><th class="colsep0 rowsep0" align="center" char=".">data points</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Proprietary</i></td><td class="colsep0 rowsep0" align="char" char=".">4,712</td><td class="colsep0 rowsep0" align="char" char=".">220</td><td class="colsep0 rowsep0" align="char" char=".">1,035,549</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Open</i></td><td class="colsep0 rowsep0" align="char" char=".">39,970</td><td class="colsep0 rowsep0" align="char" char=".">263</td><td class="colsep0 rowsep0" align="char" char=".">247,739</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Combined</i></td><td class="colsep0 rowsep0" align="char" char=".">44,603</td><td class="colsep0 rowsep0" align="char" char=".">291</td><td class="colsep0 rowsep0" align="char" char=".">1,280,016</td></tr></tbody></table></div></div><div class="NLM_p last">The RF classifier of the Python library Scikit-learn (version 0.17.1) was mainly used for the generation of activity classification models.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The number of estimators (decision trees) was set to 2500 and the maximum number of features to the log<sub>2</sub> of the total number of features. To assess the performance of the RF models over other classification techniques, Scikit-learn’s NB classifiers (Gaussian NB and Bernoulli NB<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> using default parameters) and a Tensorflow Deep Neural Network (DNN) classifier were used (version 0.7.1 using skflow version 0.1.0).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> For DNN calculations, a Deep Learning network with two hidden layers of 2048 neurons each, rectified linear units,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and stochastic gradient descent for weight optimization were used. The input layer was preprocessed using normalization and application of a tanh function. Furthermore, neuron Dropout at a threshold of 0.5 was introduced to avoid overfitting.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> DNN parameters were selected by starting from values reported in the literature<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and optimizing on ABL1 as an exemplary kinase. A simple K-Nearest Neighbor classification serves as a baseline model. All ML calculations were performed in Python, making further use of the packages NumPy (version 1.10.4) and Pandas (version 0.17.1). An exemplary jupyter notebook is provided at <a href="https://github.com/Team-SKI/Publications" class="extLink">https://github.com/Team-SKI/Publications</a>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Balancing Methods</h3><div class="NLM_p last">In kinase profiling data, active compounds are heavily outnumbered by inactive compounds. This is referred to as the class imbalance problem, which can lead to poor performance of ML algorithms.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Besides <i>random</i> under- and oversampling of the majority/minority class, undersampling was performed based on centroid clustering, Nearest Neighbor (NearMiss<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a>) search and <i>PCA-Centroids</i>, while oversampling of the minority class was performed by the SMOTE-algorithm (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf" class="ext-link">Supporting Information</a>).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Cross-Validation and Model Evaluation</h3><div class="NLM_p last">The balanced data sets were split into five equal parts for a 5-fold cross-validation (CV). Thus, in each fold, a model was trained on 80% and tested on the remaining 20% of the data. Additionally, 10 <i>external</i> (left-out) data sets were composed for every single fold, each containing the active compounds from the respective test set combined with randomly drawn inactives from the compounds dismissed in the preceding undersampling step. For model quality assessment, the average Areas Under the ROC-Curve (AUC), the average sensitivities (recall of positive class) and the average specificities (recall of negative class) of the 5-fold CVs were evaluated. Furthermore, the average AUCs and specificities of the <i>external</i> test sets were calculated. There is no need to calculate the sensitivity for these data sets because the positive compounds are identical to those of the CV test sets. Paired Mann–Whitney U tests were performed to assess statistical significance.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86537" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86537" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This section will be structured as follows: first, the effect of different data sources (<i>Proprietary</i> and/or <i>Open</i>) on the predictive power of RF models will be assessed. An analysis of the chemical space of the data sets will serve as an explanation for the dependency of the model applicability on the chosen data source. Moreover, the distribution of high-quality (HQ) models across the kinome phylogeny will be analyzed. Second, alternative ML classifiers will be trained and the predictive power evaluated. In an analogous fashion, various data balancing techniques will be used and evaluated in comparison to the results obtained by random undersampling. Third, the training data will be extended by bioactivity fingerprints and the corresponding RF classifiers evaluated. In the last section, the applicability of the various presented methods for predicting selectivity scores will be analyzed.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Kinase Activity Classification Models</h3><div id="sec3_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Activity Prediction with Random Forest Classifier Yields High-Quality Models</h4><div class="NLM_p">The Random Forest (RF) methodology was used on the Morgan FPs to create a dedicated activity prediction model for each kinase after balancing the data sets using random undersampling. Random undersampling is the easiest, most intuitive method for data balancing but nevertheless yields very good performance.<a onclick="showRef(event, 'ref27 ref47'); return false;" href="javascript:void(0);" class="ref ref27 ref47">(27, 47)</a> The RF models yield reasonable prediction results on the <i>Proprietary</i> data set with an average AUC of 0.67 ± 0.09, a sensitivity of 0.56 ± 0.06, and a specificity of 0.69 ± 0.11 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a). Surprisingly, models trained on <i>Open</i> data exhibit a much higher average predictive power with an average AUC of 0.88 ± 0.09. However, the much higher standard deviations of the 5-fold CVs indicate that these results are largely affected by the random splits of the underlying training and test data (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). Therefore, the models trained with <i>Open</i> data are much less robust and can be expected to generally have less prediction power. The <i>Combined</i> data set shows the best results regarding average model quality metrics (AUC of 0.76 ± 0.12, sensitivity of 0.63 ± 0.12, and specificity of 0.78 ± 0.13) and simultaneously very high robustness (i.e., low standard deviations in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). It is notable that only 0.7% of all models show a drop in AUC > 0.05 on left-out data, making the models trained on <i>Combined</i> data well applicable on new data sets. Overall, 118 models were generated with an average AUC > 0.8 based on the <i>Combined</i> data set. As a baseline, the results were compared to a simple Nearest Neighbor (KNN with K = 1) classification on raw, unbalanced data. The resulting models have significantly lower predictive power with an average AUC of 0.66 ± 0.10 (<i>p</i> ≪ 0.001). Employing a threshold of pIC<sub>50</sub> = 6 did not change the model performance (0.76 ± 0.11), although the average number of actives increased from 485 to 651.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Boxplots of (a) average Random Forest model quality metrics and (b) standard deviations of 5-fold cross-validations (CV) on various data sets. Black lines depict the median, and boxes illustrate the interquartile range (IQR) of the distribution. Whiskers extent to 1.5·<i>IQR</i> from the median. Although models based on <i>Open</i> data show higher average AUC, sensitivity, and specificity, the standard deviations of the 5-fold CV are significantly higher compared to the results obtained based on the <i>Proprietary</i> and <i>Combined</i> data sets. Hence, models derived from the latter two data sets are more robust.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Effect of Training Data on Model Performance</h4><div class="NLM_p">Next, we investigated how the content of the training set, such as the number of actives and the diversity of covered chemical space, affects the prediction power. Interestingly, the obtained AUCs do not correlate with the number of actives (<a class="ref internalNav" href="#fig3" aria-label="Figures 3">Figures 3</a>a,b). Please note that most models of the <i>Open</i> set were trained on a low number of actives (33% have <50 actives), indicating that these models might be overfitted. By including the <i>Proprietary</i> kinase panel, the average number of actives increased from 219 to 485 compounds. Although the average AUC values exhibit a slight drop, the models are more robust, as evidenced by the much lower standard deviation in the CV (cf. <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). As expected, the higher number of active compounds (and associated increased size of the balanced training set) has a positive effect on the model quality. To further test whether the models derived from the <i>Open</i> data are overfitted, the <i>Proprietary</i> data set was used for external validation. The performance of the <i>Open</i> models drops significantly (average AUC of 0.56 ± 0.06, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf" class="ext-link">Figure S1</a>). On the other hand, models based on <i>Proprietary</i> data show reasonable prediction results when externally tested on the <i>Open</i> data (average AUC of 0.65 ± 0.13; <i>p</i> ≪ 0.001 compared to <i>Open</i> models; Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf" class="ext-link">Figure S1</a>). Although there is no direct correlation between the number of active compounds and the AUC, models with a large number of actives (>1000) usually result into prediction models with very good AUC values above 0.8 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). The large error bar of the first bin in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c indicates that, regardless of a possibly high AUC, reliable predictions can hardly be derived from models with very small training sets.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. AUC plotted against the number of active compounds per kinase for the (a) <i>Open</i> and (b) <i>Combined</i> data sets. In both cases, no correlation between the achieved AUC and number of actives exists. (c) Binned average AUC against number of active compounds. Bins have a range of 50 actives. Models with a high number of actives generally result in prediction models with high AUC values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Analysis of Chemical Space of Data Sets Explains Importance of <i>Proprietary</i> Data Set</h4><div class="NLM_p">To explain the positive effect of the <i>Proprietary</i> data set on model robustness, a PCA was performed on the connectivity-based part of the Morgan FPs of compounds measured against the kinases KDR, EGFR, and MET (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>), respectively. These kinases show drastic differences in AUC when trained on <i>Proprietary</i> and evaluated on <i>Open</i> data and vice versa (<i>p</i> ≪ 0.001); furthermore, these three kinases provide very large data sets with overall 10,072, 9,307, and 7,636 measurements, respectively. Distinct clusters are observable in the chemical space of the PC1–PC2 planes. Whereas the majority of the compounds in the <i>Proprietary</i>, Tang, and PKIS data accumulate to one big cluster, the <i>Proprietary</i> panel additionally occupies several further distinct clusters. Thus, the <i>Proprietary</i> data set has a larger chemical diversity in the training and test sets. Resulting models accordingly have a higher chance of being applicable in diverse drug discovery efforts. This indicates that aside from model quality metrics, such as AUC, also the chemical space of the training sets and real test sets should always be assessed. Interestingly, the compounds from the public domain (Tang, PKIS, and ChEMBL) tested against EGFR also show a notable diversity from the main cluster, which might explain the higher AUC of the EGFR model based on <i>Open</i> data compared to the KDR and MET models (0.65 vs 0.56 and 0.58, respectively). Analyzing the principal component space to higher dimensions up to PC20 further underlines the chemical diversity in the <i>Proprietary</i> panel (data not shown). Worth mentioning is that on average 97% of the compounds are DFG-in binders according to a classification scheme described by Zhao et al.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Considering only “DFG-in” binders did not change the prediction performance (both AUC: 0.76 ± 0.12), while only 92 models with AUC ≥ 0.7 could be obtained for “DFG-out” binders. This indicates that DFG-out binders can safely be included when training DFG-in models and that not enough data is in general available to derive dedicated DFG-out models.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. First two principal components of PCA of compounds from the <i>Combined</i> data set with measurements against kinases KDR, EGFR, and MET, respectively. Connectivity-based Morgan FPs were used as input. Besides the majority cluster, the <i>Proprietary</i> data (from Merck KGaA) occupies several further distinct clusters in the chemical space.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> High-Quality Models Are Evenly Distributed Across the Kinome</h4><div class="NLM_p">Although a strong traditional research bias shifts kinase drug discovery toward already validated drug targets, such as the tyrosine kinases (TK group),<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49, 50)</a> driver mutations in kinases in all kinase groups are present in a variety of cancer types.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Many of these yet untargeted kinases show high predicted druggability scores and might, thus, provide opportunities for novel, competition-free drug discovery projects.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Encouragingly, HQ models, i.e., models with an AUC of ≥0.8, are evenly distributed across the entire kinome (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Kinome map of the performance of activity prediction models by Random Forest. Kinases are colored based on their AUC value. High-quality models (AUC ≥ 0.8) are scattered well across the kinome tree and cover almost all kinase families. FDA approved kinase inhibitor targets are depicted as red triangles. Figure was created with KinMap (<a href="http://kinhub.org/kinmap" class="extLink">http://kinhub.org/kinmap</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> summarizes the 35 best models (AUC ≥ 0.9). Beside AUC, sensitivity and specificity are generally also very high in these models, making them excellent tools for the identification of truly active and inactive compounds. As indicated by the high AUC and specificity on external data sets, the models are well applicable on external data sets for VS for new inhibitors of these kinases. Exceptions might be the models obtained for kinases DCAMKL1, DYRK4, MNK1, NEK9, and TGFbR2, as they are derived from a very small number of active compounds (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Quality Measures of Activity Prediction Models<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> with an AUC ≥ 0.9 of the 5-Fold CV</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">group</th><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center" char="±">av AUC<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±">av sens<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±">av spec<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±">av AUC ext<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±">av spec ext<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">no. actives</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AGC</td><td class="colsep0 rowsep0" align="left">PKCa</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.85 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.86 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.84 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">293</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PKCb</td><td class="colsep0 rowsep0" align="char" char="±">0.97 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.87 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.97 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.00</td><td class="colsep0 rowsep0" align="char" char=".">195</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PKCg</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.79 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ROCK2</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.85 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">1169</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="8" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CAMK</td><td class="colsep0 rowsep0" align="left">CHK1</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">1482</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DCAMKL1</td><td class="colsep0 rowsep0" align="char" char="±">1.00 ± 0.00</td><td class="colsep0 rowsep0" align="char" char="±">0.40 ± 0.49</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.40</td><td class="colsep0 rowsep0" align="char" char="±">0.88 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.06</td><td class="colsep0 rowsep0" align="char" char=".">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MNK1</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.78 ± 0.22</td><td class="colsep0 rowsep0" align="char" char="±">0.85 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PIM3</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.74 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">372</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="8" align="left">    </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CMGC</td><td class="colsep0 rowsep0" align="left">DYRK1A</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.76 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.86 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">360</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DYRK1B</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.69 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.88 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.86 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">183</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DYRK4</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">0.85 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.27</td><td class="colsep0 rowsep0" align="char" char="±">0.88 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">0.59 ± 0.22</td><td class="colsep0 rowsep0" align="char" char=".">18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">GSK3B</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.82 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">1266</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">JNK1</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.81 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.00</td><td class="colsep0 rowsep0" align="char" char=".">648</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">p38a</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.84 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.00</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">2580</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="8" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="left">DDR2</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.70 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">273</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EGFR</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.00</td><td class="colsep0 rowsep0" align="char" char=".">1905</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ErbB2</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">765</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">JAK1</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">308</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">KIT</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.83 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">724</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TNK1</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.24</td><td class="colsep0 rowsep0" align="char" char="±">0.97 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">0.88 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">0.84 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TYK2</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">190</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="8" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TKL</td><td class="colsep0 rowsep0" align="left">BRAF</td><td class="colsep0 rowsep0" align="char" char="±">0.97 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.98 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">378</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">LRRK2</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.84 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.87 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">297</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TGFbR2</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">1.00 ± 0.00</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.16</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.72 ± 0.10</td><td class="colsep0 rowsep0" align="char" char=".">15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="8" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STE</td><td class="colsep0 rowsep0" align="left">COT</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">0.78 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.78 ± 0.18</td><td class="colsep0 rowsep0" align="char" char=".">113</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PAK1</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.64 ± 0.23</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="8" align="left">    </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">other</td><td class="colsep0 rowsep0" align="left">AurA</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.83 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.88 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.88 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">1062</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">AurB</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.82 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">973</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">AurC</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.73 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.89 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">680</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CDC7</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.87 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">353</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NEK9</td><td class="colsep0 rowsep0" align="char" char="±">1.00 ± 0.00</td><td class="colsep0 rowsep0" align="char" char="±">0.70 ± 0.40</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PLK1</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.81 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.97 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.97 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">358</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PLK3</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.75 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">125</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TTK</td><td class="colsep0 rowsep0" align="char" char="±">0.99 ± 0.00</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.98 ± 0.00</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">213</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Wee1</td><td class="colsep0 rowsep0" align="char" char="±">0.98 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.98 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">201</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Obtained via Random Forest.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">Abbreviations: av, average; sens, sensitivity; spec, specificity; ext, external.</p></div></div></div></div><div id="sec3_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Random Forest Generally Outperforms Alternative Machine Learning Methods</h4><div class="NLM_p">Model generation was repeated using two NB classifiers (Gaussian and Bernoulli NB), a K-Nearest Neighbor (KNN; with default settings, K = 5) and a Deep Neural Network (DNN) predictor, again employing randomly undersampled data sets. Each method was tested on the <i>Combined</i> data set (using the same data preparation as before). Comparison of AUC and specificity, extracted from the 5-fold CV and external data sets, indicates that the RF models are generally superior to the alternative ML methods in our analyses (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). Only DNN yields similarly good results with an average AUC of 0.76 ± 0.12. Interestingly, the DNN and KNN classifiers achieve higher sensitivity than the RF models, which, however, comes with a decline in specificity, particularly in the case of KNN.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Boxplots of model quality measurements for Random Forest and alternative Machine Learning approaches. The plotted results are based on the <i>Combined</i> data set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The AUC values of the external test sets of each classifier are highly correlated (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf" class="ext-link">Figure S2</a>). However, this does not necessarily mean that an alternative method cannot perform better than RF on a single kinase (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>). Notably, the alternative ML methods only contribute one HQ models with an AUC ≥ 0.9, which is not also obtained with RF, although the majority of these models are commonly found by at least two of the four approaches. The unique HQ model was obtained by the DNN approach. On the other hand, the RF method is able to create seven unique HQ activity prediction models. Lowering the HQ cutoff to an AUC of 0.8 further confirms the superiority of RF and DNN over the other methods. While RF generates eight unique HQ models, seven additional kinases get HQ models using DNN (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> and <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). In many cases, however, the ΔAUC between DNN and RF is very small or the models were derived from only a small number of active compounds (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Accordingly, in this study, RF was still preferred over DNN, mainly due to lower computational cost of training and the access to feature importance information, which increases model interpretability.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlap among various Machine Learning methods of high-quality models with an AUC of ≥0.9 and ≥0.8, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. High-Quality Models Generated by Deep Neural Networks But Not by Random Forest</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center" char="±">AUC</th><th class="rowsep1 colsep0" align="center" char="±">AUC</th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center" char="±">Deep Neural Networks</th><th class="colsep0 rowsep0" align="center" char="±">Random Forest</th><th class="colsep0 rowsep0" align="center" char=".">ΔAUC</th><th class="colsep0 rowsep0" align="center" char=".">no. actives<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC ≥ 0.8</td><td class="colsep0 rowsep0" align="left">BRD2</td><td class="colsep0 rowsep0" align="char" char="±">0.85 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">0.15 ± 0.30</td><td class="colsep0 rowsep0" align="char" char=".">0.70</td><td class="colsep0 rowsep0" align="char" char=".">7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CDK8</td><td class="colsep0 rowsep0" align="char" char="±">0.83 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">0.79 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td><td class="colsep0 rowsep0" align="char" char=".">101</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MLK2</td><td class="colsep0 rowsep0" align="char" char="±">0.81 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">0.76 ± 0.13</td><td class="colsep0 rowsep0" align="char" char=".">0.05</td><td class="colsep0 rowsep0" align="char" char=".">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PDK1</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.79 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">0.01</td><td class="colsep0 rowsep0" align="char" char=".">501</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PHKg1</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.75 ± 0.39</td><td class="colsep0 rowsep0" align="char" char=".">0.20</td><td class="colsep0 rowsep0" align="char" char=".">7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PKACb</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.40</td><td class="colsep0 rowsep0" align="char" char="±">0.60 ± 0.49</td><td class="colsep0 rowsep0" align="char" char=".">0.20</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PKR</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.25</td><td class="colsep0 rowsep0" align="char" char="±">0.70 ± 0.29</td><td class="colsep0 rowsep0" align="char" char=".">0.10</td><td class="colsep0 rowsep0" align="char" char=".">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC ≥ 0.9</td><td class="colsep0 rowsep0" align="left">PHKg1</td><td class="colsep0 rowsep0" align="char" char="±">0.95 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.75 ± 0.39</td><td class="colsep0 rowsep0" align="char" char=".">0.20</td><td class="colsep0 rowsep0" align="char" char=".">7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Number of active compounds in data set.</p></div></div></div><div class="NLM_p last">Although RF generally outperforms the other classifiers, it can still be useful to assess and compare the performance of alternative methods. For instance, HQ models can be generated, which might not be accessible by solely using RF. Considering multiple independent classifiers might therefore boost the probability of success in a ligand-based VS endeavor.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></div></div><div id="sec3_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> The Impact of the Balancing Technique</h4><div class="NLM_p last">Besides <i>random</i> undersampling, various other techniques for data balancing have been explored (cf. <a href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf" class="ext-link">Supporting Information</a> for detailed information). In summary, two fundamentally different approaches for data balancing, undersampling and oversampling, might serve different purposes. Whereas undersampled models might be useful to capture a large number of potentially active molecules (high sensitivity), oversampling seems to be more applicable for detecting true negative compounds (high specificity). The balancing methods which yielded the best results were random undersampling, PCA-Centroids undersampling, and random oversampling (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf" class="ext-link">Figures S3 and S4</a>).</div></div><div id="sec3_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Bioactivity Fingerprints</h4><div class="NLM_p">Compound profiling data contain valuable information on the tested compounds on a variety of targets. Comparing compounds on the basis of their biological profiles instead of chemical similarity can provide complementary information and be a valuable source for hit expansion,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> repurposing projects,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> designing screening subsets<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> as well as to identify targets of phenotypic screenings.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Here, bioactivity FPs were generated based our <i>Proprietary</i> panel, where each bit was set to one if the pIC<sub>50</sub> was ≥6.3 and to zero otherwise. To determine the 20 kinases with the largest information content for such predictions, we trained an RF regressor to predict the selectivity score <i>S</i> based on the entire experimental bioactivity profile of the <i>Proprietary</i> data set. <i>S</i> is defined as the number of kinases inhibited by the compound divided by the number of all tested kinases:<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a><span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/jm-2016-01611h_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span>Thus, the lower the value of <i>S</i>, the higher the selectivity of a given compound <i>C</i>. Importantly, the RF algorithm does not only predict <i>S</i> but also provides valuable information about the feature importance, which is a list of kinases in the present case. The identified kinases are spread across the entire kinome and include KDR, RSK2, AMPKa1, MARK2, SIK, IRAK1, CHK2, FGFR2, MELK, HIPK2, JAK2, FGFR3, MARK1, MSK1, CDK7, FGR, AMPKa2, FLT1, CDK5, and HIPK1. Next, bioactivity FPs of these 20 kinases were used for bioactivity prediction, resulting in good model performances with an average AUC of 0.79 ± 0.07 (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>); thus, these models outperform those obtained based on chemical information. Increasing the number of kinases in the bioactivity fingerprint (FP length) further improves prediction results (average AUC for 30 kinases, 0.81 ± 0.07; for 40 kinases, 0.82 ± 0.07). Also, concatenating bioactivity and chemical FPs to one combined input results in excellent activity prediction models for a larger number of kinases (154 models with AUC ≥ 0.8) (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). The main improvement over using plain Morgan FPs is an increased average sensitivity. Thus, by determining the activity of a small panel of only 20 kinases in combination with chemical FPs, a bioactivity panel can be meaningfully extended to a large number of kinases using an RF classifier. It should be noted that the bioactivity part of the combined FP was processed by a modified implementation of Google’s Winner-Takes-All hashing algorithm<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> to adjust the FP length of 20 bioactivity values to the length of the Morgan FP. This step is required to avoid highly imbalanced FP lengths, which lead to biased selection of features in the individual decision trees.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Boxplots of model quality measurements for Random Forest using chemical and/or bioactivity fingerprints based on <i>Proprietary</i> data. Adding bioactivity FPs mostly results in improved sensitivity. A concatenation of both FPs yields excellent performance in activity prediction. Plotted are results obtained from 5-fold CV.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Selectivity Score and Identification of Off-Targets</h3><div class="NLM_p">So far we have investigated the performance quality when it comes to predicting the activity of compounds against a particular kinase. It is equally important to have a reliable estimation of the selectivity profile. The latter can be quantified by the selectivity score <i>S</i> (cf. above). Employing the 118 HQ-RF models (based on Morgan FPs of the <i>Combined</i> set) with AUC ≥ 0.8, we reconstructed a complete panel of predicted activity probabilities for the compounds of the <i>Proprietary</i> data (<i>N</i> = 4,712) from the respective test sets of a 10-fold cross validation. On the basis of the experimental pIC<sub>50</sub> values (binarized at 6.3), <i>S</i> was calculated for each compound. Compounds with an <i>S</i>-value of 0 were removed due to inactivity across the entire kinase panel. From the 118 HQ models, 68 kinases were in the <i>Proprietary</i> panel, which consists of an almost complete matrix of bioactivity values. Hence, the calculated selectivity score <i>S</i><sub>calc</sub> is the average of the binary classification results (default probability cutoff: 0.5) obtained by these 68 kinase models. The calculated <i>S</i><sub>calc</sub> shows no predictive power of <i>S</i> with an <i>R</i><sub class="stack">ext</sub><sup class="stack">2</sup> < 0. Similarly, an RF regressor trained on the predicted panel of 68 kinases resulted only in an average <i>R</i><sub class="stack">ext</sub><sup class="stack">2</sup> of 0.17. However, an RF regressor trained on a bioactivity FP of 20 selected kinases (cf. above) resulted in very accurate selectivity regression model with an average <i>R</i><sub class="stack">ext</sub><sup class="stack">2</sup> of 0.85 ± 0.01 (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>a). Notably, this is not trivial because the plain average of the 20 experimental activity values does not yield an accurate estimate of the overall selectivity (<i>R</i><sub class="stack">ext</sub><sup class="stack">2</sup> < 0, <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>b).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (a) Selectivity prediction based on bioactivity fingerprints obtained by Random Forest regression achieves an <i>R</i><sub class="stack">ext</sub><sup class="stack">2</sup> of 0.85 ± 0.01. (b) Plain averaging of experimentally obtained bioactivity fingerprints (<i>S</i><sub>calc</sub>) does not correlate with the experimental <i>S</i>. The calculation both in (a) and (b) are done based on 20 kinases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Finally, the predicted panel was evaluated for correct kinase classifications per compound to assess the predictive power of our models for off-target identification and compound repurposing. Although the 68 HQ models do not allow a reliable estimation of selectivity scores (cf. above), they can in fact be used to reliably identify off-targets (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). On average, 40% of all compound targets are correctly detected while only 10% false positive targets are among the predictions. Using bioactivity FPs for off-target prediction further increases the sensitivity of the predicted off-targets while the specificity drops slightly. Again, the best results are obtained with a combination of Morgan and bioactivity FPs. These findings strongly emphasize the applicability of the presented approach for unprecedented virtual kinase profiling at a large scale.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Sensitivity and Specificity of (Off-)Target Prediction<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">average</th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">median</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">fingerprint</th><th class="colsep0 rowsep0" align="center">no. kinases</th><th class="colsep0 rowsep0" align="center" char="±">sensitivity</th><th class="colsep0 rowsep0" align="center" char="±">specificity</th><th class="colsep0 rowsep0" align="center" char="±">sensitivity</th><th class="colsep0 rowsep0" align="center" char="±">specificity</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Morgan FP</td><td class="colsep0 rowsep0" align="center">68</td><td class="colsep0 rowsep0" align="char" char="±">0.40 ± 0.39</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.33 ± 0.33</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bioactivity FP</td><td class="colsep0 rowsep0" align="center">97</td><td class="colsep0 rowsep0" align="char" char="±">0.48 ± 0.41</td><td class="colsep0 rowsep0" align="char" char="±">0.72 ± 0.33</td><td class="colsep0 rowsep0" align="char" char="±">0.50 ± 0.50</td><td class="colsep0 rowsep0" align="char" char="±">0.86 ± 0.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">combined FP</td><td class="colsep0 rowsep0" align="center">154</td><td class="colsep0 rowsep0" align="char" char="±">0.54 ± 0.41</td><td class="colsep0 rowsep0" align="char" char="±">0.69 ± 0.36</td><td class="colsep0 rowsep0" align="char" char="±">0.60 ± 0.40</td><td class="colsep0 rowsep0" align="char" char="±">0.85 ± 0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bioactivity FP</td><td class="colsep0 rowsep0" align="center">68<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">0.47 ± 0.42</td><td class="colsep0 rowsep0" align="char" char="±">0.72 ± 0.29</td><td class="colsep0 rowsep0" align="char" char="±">0.50 ± 0.50</td><td class="colsep0 rowsep0" align="char" char="±">0.83 ± 0.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">combined FP</td><td class="colsep0 rowsep0" align="center">68<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">0.61 ± 0.41</td><td class="colsep0 rowsep0" align="char" char="±">0.70 ± 0.31</td><td class="colsep0 rowsep0" align="char" char="±">0.75 ± 0.25</td><td class="colsep0 rowsep0" align="char" char="±">0.85 ± 0.12</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Considered were only prediction models with an AUC ≥ 0.8 and which were in the <i>Proprietary</i> panel.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">For comparison of the fingerprints, the bioactivity and combined FPs were also evaluated on the 68 HQ models based on Morgan FPs.</p></div></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41471" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41471" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Various methods for data set preparation and Machine Learning (ML) were employed to generate activity prediction models for over 280 kinases. The most important findings which improved the model quality were (1) the combination of open data with the proprietary data set from Merck KGaA, (2) choosing the RF classifier over alternative ML methods, and (3) balancing the data sets using random undersampling. An average ROC AUC ≥ 0.7 was achieved for ∼200 kinases, of which 118 prediction models had an AUC ≥ 0.8. Tests on left-out data suggest a reliable applicability in virtual screening projects. Moreover, the models also enable reliable virtual kinase profiling and, thus, the detection of potential off-targets for compounds of interest.</div><div class="NLM_p">The results are well in line with findings reported in recent literature, especially regarding the bias of the used training data. For instance, Bora et al. trained RF classifiers for 107 kinases, of which 100 achieved AUC values >0.9.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> However, external validation of these models on the Metz and Anastassiadis data sets<a onclick="showRef(event, 'ref5 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref9">(5, 9)</a> strongly indicated an overfitting, as also experienced in our models trained on the <i>Open</i> data set only. Namely, although in our case 113 models with average AUC values >0.9 were derived based on <i>Open</i> data, external testing on the <i>Proprietary</i> panel resulted in rather low predictive power. On the other hand, as seen in model evaluation on left-out data (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a) and the low standard deviations in cross-validations (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b), our kinase activity models based on the <i>Combined</i> data do not show a strong dependency on the respective random training/test set split and are, thus, applicable on a larger chemical space.</div><div class="NLM_p">For further external validation, we tested our models on 43 compounds, which were previously also used by Schürer and Muskal.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> These compounds were not present in our training set. Because the underlying assay for these compounds (KINOMEScan<a onclick="showRef(event, 'ref56 ref59'); return false;" href="javascript:void(0);" class="ref ref56 ref59">(56, 59)</a>) was measured at a high concentration of 10 μM, a high cutoff of 95% inhibition was used for binary classification of these external test cases (this roughly corresponds to a pIC<sub>50</sub> of 6.3, the cutoff used for model training). Without the necessity for adjusting the RF classifier probability cutoff for binary classification (default: 0.5), only three compounds were wrongly classified to be active and one compound wrongly classified as inactive (marked in gray and red, respectively; <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>a). Although the latter shows inhibition well above 95%, the predicted activity probability is slightly below the 0.5 cutoff. Encouragingly, a Cohen’s κ value of 0.65 indicates a high predictive power of the tested models.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Because it is very difficult to derive an accurate estimation of the pIC<sub>50</sub> from a single point measurement at activities above 80%, a second cutoff was set to 80% inhibition. In this case, only two false negatives appear, increasing Cohen’s κ value to 0.79. To further minimize the error that might result from inhibition data conversion, the binary classification was evaluated using a varying threshold for experimental activity in a ROC analysis (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>b). A very high AUC of 0.85 and a good early enrichment of true positives suggest that the classifiers can be applied successfully on the external data set regardless of the chosen cutoff for experimental activity.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (a) Classification results of external data using the RF classifier. With a default probability cutoff of 0.5, only four wrong classifications at a 95% inhibition cutoff and two wrong classifications at an 80% inhibition cutoff appear for the RF classifier. Cohen’s κ values of 0.65 and 0.79, respectively, suggest a very high predictive power. (b) ROC analysis of binary classification of the external data with varying experimental inhibition cutoff. An AUC of 0.85 confirms the successful application of the models on the external test set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Regarding the data balancing, random undersampling produced the models with the best performance, followed by the PCA-Centroids undersampling, which showed a higher sensitivity at the cost of a decline in specificity. A great improvement in the classification performance could be achieved by adding experimental activity values of a small kinase subset (20 kinases) to the chemical FP, yielding HQ models with an AUC ≥ 0.8 for 154 kinases of the 220 kinases in the <i>Proprietary</i> panel. This strongly suggests that bioactivity FPs from only a small number of kinases (in combination with Morgan FPs) contain enough information to accurately propagate bioactivity values to a kinase panel almost 10 times the size of the experimental subset. Furthermore, the combination of RF classification and bioactivity FPs of only 20 kinases allowed a reliable prediction of global compound selectivity <i>S</i>. Importantly, this cannot necessarily be derived directly from the raw experimental activity values.</div><div class="NLM_p">In line with other studies, the RF classifier outperformed the Naive Bayes methods,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> while the Deep Learning approach also generated models with excellent predictive power. It might be possible that a more extensive parameter study and the expansion of the Neural Network layers would further improve the performance of the Deep Learning activity prediction.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Notably, the Deep Learning approach also benefited from balanced training sets (data not shown).</div><div class="NLM_p last">Overall, the evaluation results of our prediction models strongly indicate a positive impact on future screening projects and off-target identification tasks. The former can be used to assess for a particular kinase of interest which compounds should be ordered for experimental verification (virtual screening) or to prioritize already investigated compounds from previous projects on new kinase targets (repurposing). On the other hand, the identification of off-targets is a prerequisite for the rational design of selective kinase inhibitors.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> In ongoing research, it will be assessed how the addition of proteogenomic information and the combined usage of RF classifiers with Deep Learning networks can further improve the predictive power of the activity prediction models.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01611">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03077" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03077" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01611" class="ext-link">10.1021/acs.jmedchem.6b01611</a>. The curated <i>Open</i> data set and source code for deriving prediction models can be downloaded from <a href="https://github.com/Team-SKI/Publications" class="extLink">https://github.com/Team-SKI/Publications</a>. <ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional external model validation, correlations between different ML methods, and the impact of various balancing techniques on the prediction performance. (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf">jm6b01611_si_001.pdf (968.92 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b01611" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11281" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11281" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simone Fulle</span> - <span class="hlFld-Affiliation affiliation">BioMed
X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7646-5889" title="Orcid link">http://orcid.org/0000-0002-7646-5889</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#caacbfa6a6af8aa8a3a5e4a7b2"><span class="__cf_email__" data-cfemail="4721322b2b2207252e28692a3f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin Merget</span> - <span class="hlFld-Affiliation affiliation">BioMed
X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samo Turk</span> - <span class="hlFld-Affiliation affiliation">BioMed
X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sameh Eid</span> - <span class="hlFld-Affiliation affiliation">BioMed
X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Friedrich Rippmann</span> - <span class="hlFld-Affiliation affiliation">Global
Computational Chemistry, Merck KGaA, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2640-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11840" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11840" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Andrea Volkamer and Guillaume Roellinger (BioMed X Innovation Center, Heidelberg), Paul Czodrowski and Mireille Krier (Merck KGaA, Darmstadt), and Rebecca Wade (Heidelberg Institute for Theoretical Studies) for numerous valuable discussions. Furthermore, we are grateful to Günter Klambauer and Sepp Hochreiter (Johannes Kepler University Linz) for very helpful conversations regarding Deep Neural Networks.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i23" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i23"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i24" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i24"> Abbreviations Used</h2><tr><td class="NLM_term">CV</td><td class="NLM_def"><p class="first last">cross-validation</p></td></tr><tr><td class="NLM_term">DNN</td><td class="NLM_def"><p class="first last">Deep Neural Network</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fingerprint</p></td></tr><tr><td class="NLM_term">HQ</td><td class="NLM_def"><p class="first last">high-quality</p></td></tr><tr><td class="NLM_term">IQR</td><td class="NLM_def"><p class="first last">inter-quartile range</p></td></tr><tr><td class="NLM_term">KNN</td><td class="NLM_def"><p class="first last">K-Nearest Neighbor</p></td></tr><tr><td class="NLM_term">ML</td><td class="NLM_def"><p class="first last">Machine Learning</p></td></tr><tr><td class="NLM_term">NB</td><td class="NLM_def"><p class="first last">Naïve Bayes</p></td></tr><tr><td class="NLM_term">PCM</td><td class="NLM_def"><p class="first last">proteochemometrics</p></td></tr><tr><td class="NLM_term">RF</td><td class="NLM_def"><p class="first last">Random Forest</p></td></tr><tr><td class="NLM_term">ROC</td><td class="NLM_def"><p class="first last">receiver operating characteristic</p></td></tr><tr><td class="NLM_term">SVM</td><td class="NLM_def"><p class="first last">Support Vector Machine</p></td></tr><tr><td class="NLM_term">VS</td><td class="NLM_def"><p class="first last">virtual screening</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82542" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82542" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 62 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Cohen, P.</span><span> </span><span class="NLM_article-title">Protein kinases-the major drug targets of the twenty-first century?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">309</span><span class="NLM_x">–</span> <span class="NLM_lpage">315</span><span class="refDoi"> DOI: 10.1038/nrd773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnrd773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=12120282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD38XivVGmtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=309-315&author=P.+Cohen&title=Protein+kinases-the+major+drug+targets+of+the+twenty-first+century%3F&doi=10.1038%2Fnrd773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: Protein kinases - the major drug targets of the twenty-first century?</span></div><div class="casAuthors">Cohen, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">309-315</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein phosphorylation regulates most aspects of cell life, whereas abnormal phosphorylation is a cause or consequence of disease.  A growing interest in developing orally active protein-kinase inhibitors has recently culminated in the approval of the first of these drugs for clin. use.  Protein kinases have now become the second most important group of drug targets, after G-protein-coupled receptors.  Here, I give a personal view of some of the most important advances that have shaped this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRd_GqiGAWaLVg90H21EOLACvtfcHk0lh68vAK5dGWSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivVGmtr8%253D&md5=67d2fbc11a55b5980ee0692f7bc13cc9</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd773%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DProtein%2520kinases-the%2520major%2520drug%2520targets%2520of%2520the%2520twenty-first%2520century%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D309%26epage%3D315%26doi%3D10.1038%2Fnrd773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alessi, D. R.</span><span> </span><span class="NLM_article-title">Kinase drug discovery—what’s next in the field?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">–</span> <span class="NLM_lpage">104</span><span class="refDoi"> DOI: 10.1021/cb300610s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300610s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=96-104&author=P.+Cohenauthor=D.+R.+Alessi&title=Kinase+drug+discovery%E2%80%94what%E2%80%99s+next+in+the+field%3F&doi=10.1021%2Fcb300610s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fcb300610s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300610s%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DKinase%2520drug%2520discovery%25E2%2580%2594what%25E2%2580%2599s%2520next%2520in%2520the%2520field%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D96%26epage%3D104%26doi%3D10.1021%2Fcb300610s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, M. H.</span><span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+an+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0lh68vAK5dGWSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520an%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiöth, H. B.</span><span> </span><span class="NLM_article-title">Advances in kinase targeting: current clinical use and clinical trials</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">604</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1016/j.tips.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1016%2Fj.tips.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=25312588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KgurjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=604-620&author=M.+Rask-Andersenauthor=J.+Zhangauthor=D.+Fabbroauthor=H.+B.+Schi%C3%B6th&title=Advances+in+kinase+targeting%3A+current+clinical+use+and+clinical+trials&doi=10.1016%2Fj.tips.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in kinase targeting: current clinical use and clinical trials</span></div><div class="casAuthors">Rask-Andersen, Mathias; Zhang, Jin; Fabbro, Doriano; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">604-620</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Phosphotransferases, also known as kinases, are the most intensively studied protein drug target category in current pharmacol. research, as evidenced by the vast no. of kinase-targeting agents enrolled in active clin. trials.  This development has emerged following the great success of small-mol., orally available protein kinase inhibitors for the treatment of cancer, starting with the introduction of imatinib (Gleevec) in 2003.  The pharmacol. utility of kinase-targeting has expanded to include treatment of inflammatory diseases, and rapid development is ongoing for kinase-targeted therapies in a broad array of indications in ophthalmol., analgesia, central nervous system (CNS) disorders, and the complications of diabetes, osteoporosis, and otol.  In this review we highlight specifically the kinase drug targets and kinase-targeting agents being explored in current clin. trials.  This anal. is based on a recent est. of all established and clin. trial drug mechanisms of action, utilizing private and public databases to create an extensive dataset detailing aspects of more than 3000 approved and exptl. drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLhFMSrq0fXbVg90H21EOLACvtfcHk0ljcOMxhsvaXlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KgurjL&md5=bedc3a53a9cdab170e146f89bc495ff7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DSchi%25C3%25B6th%26aufirst%3DH.%2BB.%26atitle%3DAdvances%2520in%2520kinase%2520targeting%253A%2520current%2520clinical%2520use%2520and%2520clinical%2520trials%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26spage%3D604%26epage%3D620%26doi%3D10.1016%2Fj.tips.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Metz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merta, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kifle, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Navigating the kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.1038/nchembio.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+kinome&doi=10.1038%2Fnchembio.530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0ljcOMxhsvaXlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D200%26epage%3D202%26doi%3D10.1038%2Fnchembio.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Paricharak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klenka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, U. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span> </span><span class="NLM_article-title">Are phylogenetic trees suitable for chemogenomics analyses of bioactivity data sets: the importance of shared active compounds and choosing a suitable data embedding method, as exemplified on kinases</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="refDoi"> DOI: 10.1186/1758-2946-5-49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1186%2F1758-2946-5-49" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtlyju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=49&author=S.+Paricharakauthor=T.+Klenkaauthor=M.+Augustinauthor=U.+A.+Patelauthor=A.+Bender&title=Are+phylogenetic+trees+suitable+for+chemogenomics+analyses+of+bioactivity+data+sets%3A+the+importance+of+shared+active+compounds+and+choosing+a+suitable+data+embedding+method%2C+as+exemplified+on+kinases&doi=10.1186%2F1758-2946-5-49"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Are phylogenetic trees suitable for chemogenomics analyses of bioactivity data sets: the importance of shared active compounds and choosing a suitable data embedding method, as exemplified on Kinases</span></div><div class="casAuthors">Paricharak, Shardul; Klenka, Tom; Augustin, Martin; Patel, Umesh A.; Bender, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49/1-49/20, 20 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Phylogenetic trees are commonly used for the anal. of chemogenomics datasets and to relate protein targets to each other, based on the (shared) bioactivities of their ligands.  However, no real assessment as to the suitability of this representation has been performed yet in this area.  The authors aimed to address this shortcoming in the current work, as exemplified by a kinase data set, given the importance of kinases in many diseases as well as the availability of large-scale datasets for anal.  The authors analyzed a dataset comprising 157 compds., which have been tested at concns. of 1 μM and 10 μM against a panel of 225 human protein kinases in full-matrix expts., aiming to explain kinase promiscuity and selectivity against inhibitors.  Compds. were described by chem. features, which were used to represent kinases (i.e. each kinase had an active set of features and an inactive set).  Using this representation, a bioactivity-based classification was made of the kinome, which partially resembles previous sequence-based classifications, where particularly kinases from the TK, CDK, CLK and AGC branches cluster together.  However, the authors were also able to show that in ∼57% of cases, on av. 6 kinase inhibitors exhibit activity against kinases which are located at a large distance in the sequence-based classification (at a relative distance of 0.6-0.8 on a scale from 0 to 1), but are correctly located closer to each other in the authors' bioactivity-based tree (distance 0-0.4).  Despite this improvement on sequence-based classification, also the bioactivity-based classification needed further attention: for ∼80% of all analyzed kinases, kinases classified as neighbors according to the bioactivity-based classification also show high SAR similarity (i.e. a high fraction of shared active compds. and therefore, interaction with similar inhibitors).  However, in the remaining ∼20% of cases a clear relation between kinase bioactivity profile similarity and shared active compds. could not be established, which is in agreement with previously published atypical SAR (such as for LCK, FGFR1, AKT2, DAPK1, TGFR1, MK12 and AKT1).  The authors were hence able to show that (1) targets (here kinases) with few shared activities are difficult to establish neighborhood relations for, and (2) phylogenetic tree representations make implicit assumptions (i.e. that neighboring kinases exhibit similar interaction profiles with inhibitors) that are not always suitable for analyses of bioactivity space.  While both points have been implicitly alluded to before, this is to the information of the authors first study that explores both points on a comprehensive basis.  Excluding kinases with few shared activities improved the situation greatly (the percentage of kinases for which no neighborhood relation could be established dropped from 20% to only 4%).  All of the above findings need to be taken into account when performing chemogenomics analyses, also for other target classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7J0H5BE4gCrVg90H21EOLACvtfcHk0li91tvNWp2uoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtlyju7w%253D&md5=58b32b31d9c18ae6f196824b9be97f40</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-5-49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-5-49%26sid%3Dliteratum%253Aachs%26aulast%3DParicharak%26aufirst%3DS.%26aulast%3DKlenka%26aufirst%3DT.%26aulast%3DAugustin%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DU.%2BA.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DAre%2520phylogenetic%2520trees%2520suitable%2520for%2520chemogenomics%2520analyses%2520of%2520bioactivity%2520data%2520sets%253A%2520the%2520importance%2520of%2520shared%2520active%2520compounds%2520and%2520choosing%2520a%2520suitable%2520data%2520embedding%2520method%252C%2520as%2520exemplified%2520on%2520kinases%26jtitle%3DJ.%2520Cheminf.%26date%3D2013%26volume%3D5%26spage%3D49%26doi%3D10.1186%2F1758-2946-5-49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Ferrè, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmeri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmer-Citterich, M.</span><span> </span><span class="NLM_article-title">Computational methods for analysis and inference of kinase/inhibitor relationships</span> <span class="citation_source-journal">Front. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">196</span><span class="refDoi"> DOI: 10.3389/fgene.2014.00196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.3389%2Ffgene.2014.00196" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=25071826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A280%3ADC%252BC2cbntVShsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=196&author=F.+Ferr%C3%A8author=A.+Palmeriauthor=M.+Helmer-Citterich&title=Computational+methods+for+analysis+and+inference+of+kinase%2Finhibitor+relationships&doi=10.3389%2Ffgene.2014.00196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Computational methods for analysis and inference of kinase/inhibitor relationships</span></div><div class="casAuthors">Ferre Fabrizio; Palmeri Antonio; Helmer-Citterich Manuela</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">196</span>
        ISSN:<span class="NLM_cas:issn">1664-8021</span>.
    </div><div class="casAbstract">The central role of kinases in virtually all signal transduction networks is the driving motivation for the development of compounds modulating their activity.  ATP-mimetic inhibitors are essential tools for elucidating signaling pathways and are emerging as promising therapeutic agents.  However, off-target ligand binding and complex and sometimes unexpected kinase/inhibitor relationships can occur for seemingly unrelated kinases, stressing that computational approaches are needed for learning the interaction determinants and for the inference of the effect of small compounds on a given kinase.  Recently published high-throughput profiling studies assessed the effects of thousands of small compound inhibitors, covering a substantial portion of the kinome.  This wealth of data paved the road for computational resources and methods that can offer a major contribution in understanding the reasons of the inhibition, helping in the rational design of more specific molecules, in the in silico prediction of inhibition for those neglected kinases for which no systematic analysis has been carried yet, in the selection of novel inhibitors with desired selectivity, and offering novel avenues of personalized therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnT8GombVJ2uQ4Pl9hzjQhfW6udTcc2eaG8jtKt_545bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbntVShsg%253D%253D&md5=48bc2fcaaccfb2ec3b2070cf0565b070</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2014.00196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2014.00196%26sid%3Dliteratum%253Aachs%26aulast%3DFerr%25C3%25A8%26aufirst%3DF.%26aulast%3DPalmeri%26aufirst%3DA.%26aulast%3DHelmer-Citterich%26aufirst%3DM.%26atitle%3DComputational%2520methods%2520for%2520analysis%2520and%2520inference%2520of%2520kinase%252Finhibitor%2520relationships%26jtitle%3DFront.%2520Genet.%26date%3D2014%26volume%3D5%26spage%3D196%26doi%3D10.3389%2Ffgene.2014.00196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span><span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0li91tvNWp2uoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Anastassiadis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devarajan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J. R.</span><span> </span><span class="NLM_article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1039</span><span class="NLM_x">–</span> <span class="NLM_lpage">1045</span><span class="refDoi"> DOI: 10.1038/nbt.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&author=T.+Anastassiadisauthor=S.+W.+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R.+Peterson&title=Comprehensive+assay+of+kinase+catalytic+activity+reveals+features+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0li91tvNWp2uoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DComprehensive%2520assay%2520of%2520kinase%2520catalytic%2520activity%2520reveals%2520features%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1039%26epage%3D1045%26doi%3D10.1038%2Fnbt.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szwajda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakyawar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hintsanen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wennerberg, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aittokallio, T.</span><span> </span><span class="NLM_article-title">Making sense of large-scale kinase inhibitor bioactivity data sets: a comparative and integrative analysis</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">735</span><span class="NLM_x">–</span> <span class="NLM_lpage">743</span><span class="refDoi"> DOI: 10.1021/ci400709d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400709d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXit1yls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=735-743&author=J.+Tangauthor=A.+Szwajdaauthor=S.+Shakyawarauthor=T.+Xuauthor=P.+Hintsanenauthor=K.+Wennerbergauthor=T.+Aittokallio&title=Making+sense+of+large-scale+kinase+inhibitor+bioactivity+data+sets%3A+a+comparative+and+integrative+analysis&doi=10.1021%2Fci400709d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Making Sense of Large-Scale Kinase Inhibitor Bioactivity Data Sets: A Comparative and Integrative Analysis</span></div><div class="casAuthors">Tang, Jing; Szwajda, Agnieszka; Shakyawar, Sushil; Xu, Tao; Hintsanen, Petteri; Wennerberg, Krister; Aittokallio, Tero</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">735-743</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We carried out a systematic evaluation of target selectivity profiles across three recent large-scale biochem. assays of kinase inhibitors and further compared these standardized bioactivity assays with data reported in the widely used databases ChEMBL and STITCH.  Our comparative evaluation revealed relative benefits and potential limitations among the bioactivity types, as well as pinpointed biases in the database curation processes.  Ignoring such issues in data heterogeneity and representation may lead to biased modeling of drugs' polypharmacol. effects as well as to unrealistic evaluation of computational strategies for the prediction of drug-target interaction networks.  Toward making use of the complementary information captured by the various bioactivity types, including IC50, Ki, and Kd, we also introduce a model-based integration approach, termed KIBA, and demonstrate here how it can be used to classify kinase inhibitor targets and to pinpoint potential errors in database-reported drug-target interactions.  An integrated drug-target bioactivity matrix across 52 498 chem. compds. and 467 kinase targets, including a total of 246 088 KIBA scores, has been made freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI6vIo2K_W17Vg90H21EOLACvtfcHk0liVERSRPJNPIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXit1yls7w%253D&md5=fd3bcbd39cb523f84d105a96cc4d17f2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci400709d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400709d%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DSzwajda%26aufirst%3DA.%26aulast%3DShakyawar%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DHintsanen%26aufirst%3DP.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DAittokallio%26aufirst%3DT.%26atitle%3DMaking%2520sense%2520of%2520large-scale%2520kinase%2520inhibitor%2520bioactivity%2520data%2520sets%253A%2520a%2520comparative%2520and%2520integrative%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D735%26epage%3D743%26doi%3D10.1021%2Fci400709d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krüger, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowotka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papadatos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">The ChEMBL bioactivity database: an update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1083</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1090</span><span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-D1090&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+bioactivity+database%3A+an+update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0liVERSRPJNPIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520bioactivity%2520database%253A%2520an%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1083%26epage%3DD1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urban, L.</span><span> </span><span class="NLM_article-title">Large-scale prediction and testing of drug activity on side-effect targets</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">486</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1038/nature11159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnature11159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=22722194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=361-367&author=E.+Lounkineauthor=M.+J.+Keiserauthor=S.+Whitebreadauthor=D.+Mikhailovauthor=J.+Hamonauthor=J.+L.+Jenkinsauthor=P.+Lavanauthor=E.+Weberauthor=A.+K.+Doakauthor=S.+C%C3%B4t%C3%A9author=B.+K.+Shoichetauthor=L.+Urban&title=Large-scale+prediction+and+testing+of+drug+activity+on+side-effect+targets&doi=10.1038%2Fnature11159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale prediction and testing of drug activity on side-effect targets</span></div><div class="casAuthors">Lounkine, Eugen; Keiser, Michael J.; Whitebread, Steven; Mikhailov, Dmitri; Hamon, Jacques; Jenkins, Jeremy L.; Lavan, Paul; Weber, Eckhard; Doak, Allison K.; Cote, Serge; Shoichet, Brian K.; Urban, Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">486</span>
        (<span class="NLM_cas:issue">7403</span>),
    <span class="NLM_cas:pages">361-367</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Discovering the unintended off-targets' that predict adverse drug reactions is daunting by empirical methods alone.  Drugs can act on several protein targets, some of which can be unrelated by conventional mol. metrics, and hundreds of proteins have been implicated in side effects.  Here we use a computational strategy to predict the activity of 656 marketed drugs on 73 unintended side-effect' targets.  Approx. half of the predictions were confirmed, either from proprietary databases unknown to the method or by new exptl. assays.  Affinities for these new off-targets ranged from 1 nM to 30 μM.  To explore relevance, we developed an assocn. metric to prioritize those new off-targets that explained side effects better than any known target of a given drug, creating a drug-target-adverse drug reaction network.  Among these new assocns. was the prediction that the abdominal pain side effect of the synthetic estrogen chlorotrianisene was mediated through its newly discovered inhibition of the enzyme cyclooxygenase-1.  The clin. relevance of this inhibition was borne out in whole human blood platelet aggregation assays.  This approach may have wide application to de-risking toxicol. liabilities in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5sVjfn4mgLLVg90H21EOLACvtfcHk0liVERSRPJNPIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D&md5=9c133b839d9527818a27a5834e4dfe11</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature11159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11159%26sid%3Dliteratum%253Aachs%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DLavan%26aufirst%3DP.%26aulast%3DWeber%26aufirst%3DE.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DUrban%26aufirst%3DL.%26atitle%3DLarge-scale%2520prediction%2520and%2520testing%2520of%2520drug%2520activity%2520on%2520side-effect%2520targets%26jtitle%3DNature%26date%3D2012%26volume%3D486%26spage%3D361%26epage%3D367%26doi%3D10.1038%2Fnature11159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Hillisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinrich, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wild, H.</span><span> </span><span class="NLM_article-title">Computational chemistry in the pharmaceutical industry: from childhood to adolescence</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1958</span><span class="NLM_x">–</span> <span class="NLM_lpage">1962</span><span class="refDoi"> DOI: 10.1002/cmdc.201500346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1002%2Fcmdc.201500346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=26358802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyju73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1958-1962&author=A.+Hillischauthor=N.+Heinrichauthor=H.+Wild&title=Computational+chemistry+in+the+pharmaceutical+industry%3A+from+childhood+to+adolescence&doi=10.1002%2Fcmdc.201500346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Chemistry in the Pharmaceutical Industry: From Childhood to Adolescence</span></div><div class="casAuthors">Hillisch, Alexander; Heinrich, Nikolaus; Wild, Hanno</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1958-1962</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Computational chem. within the pharmaceutical industry has grown into a field that proactively contributes to many aspects of drug design, including target selection and lead identification and optimization.  While methodol. advancements have been key to this development, organizational developments have been crucial to our success as well.  In particular, the interaction between computational and medicinal chem. and the integration of computational chem. into the entire drug discovery process have been invaluable.  Over the past ten years we have shaped and developed a highly efficient computational chem. group for small-mol. drug discovery at Bayer HealthCare that has significantly impacted the clin. development pipeline.  In this article we describe the setup and tasks of the computational group and discuss external collaborations.  We explain what we have found to be the most valuable and productive methods and discuss future directions for computational chem. method development.  We share this information with the hope of igniting interesting discussions around this topic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAFXZ0bBJeBrVg90H21EOLACvtfcHk0lhjX1-kpRVvtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyju73L&md5=1d5d7d908463585ac45b048a395b0dac</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500346%26sid%3Dliteratum%253Aachs%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DHeinrich%26aufirst%3DN.%26aulast%3DWild%26aufirst%3DH.%26atitle%3DComputational%2520chemistry%2520in%2520the%2520pharmaceutical%2520industry%253A%2520from%2520childhood%2520to%2520adolescence%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1958%26epage%3D1962%26doi%3D10.1002%2Fcmdc.201500346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Martin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansen, J.</span><span> </span><span class="NLM_article-title">Profile-QSAR: a novel meta-QSAR method that combines activities across the kinase family to accurately predict affinity, selectivity, and cellular activity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1942</span><span class="NLM_x">–</span> <span class="NLM_lpage">1956</span><span class="refDoi"> DOI: 10.1021/ci1005004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1005004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1942-1956&author=E.+Martinauthor=P.+Mukherjeeauthor=D.+Sullivanauthor=J.+Jansen&title=Profile-QSAR%3A+a+novel+meta-QSAR+method+that+combines+activities+across+the+kinase+family+to+accurately+predict+affinity%2C+selectivity%2C+and+cellular+activity&doi=10.1021%2Fci1005004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity</span></div><div class="casAuthors">Martin, Eric; Mukherjee, Prasenjit; Sullivan, David; Jansen, Johanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1942-1956</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-QSAR is a novel 2D predictive model building method for kinases.  This "meta-QSAR" method models the activity of each compd. against a new kinase target as a linear combination of its predicted activities against a large panel of 92 previously studied kinases comprised from 115 assays.  Profile-QSAR starts with a sparse incomplete kinase by compd. (KxC) activity matrix, used to generate Bayesian QSAR models for the 92 "basis-set" kinases.  These Bayesian QSARs generate a complete "synthetic" KxC activity matrix of predictions.  These synthetic activities are used as "chem. descriptors" to train partial-least squares (PLS) models, from modest amts. of medium-throughput screening data, for predicting activity against new kinases.  The Profile-QSAR predictions for the 92 kinases (115 assays) gave a median external R2ext = 0.59 on 25% held-out test sets.  The method has proven accurate enough to predict pairwise kinase selectivities with a median correlation of R2ext = 0.61 for 958 kinase pairs with at least 600 common compds.  It has been further expanded by adding a "CkXC" cellular activity matrix to the KxC matrix to predict cellular activity for 42 kinase driven cellular assays with median R2ext = 0.58 for 24 target modulation assays and R2ext = 0.41 for 18 cell proliferation assays.  The 2D Profile-QSAR, along with the 3D Surrogate AutoShim, are the foundations of an internally developed iterative medium-throughput screening (IMTS) methodol. for virtual screening (VS) of compd. archives as an alternative to exptl. high-throughput screening (HTS).  The method has been applied to 20 actual prospective kinase projects.  Biol. results have so far been obtained in eight of them.  Q2 values ranged from 0.3 to 0.7.  Hit-rates at 10 uM for exptl. tested compds. varied from 25% to 80%, except in K5, which was a special case aimed specifically at finding "type II" binders, where none of the compds. were predicted to be active at 10 μM.  These overall results are particularly striking as chem. novelty was an important criterion in selecting compds. for testing.  The method is completely automated.  Predicted activities for nearly 4 million internal and com. compds. across 115 kinase assays and 42 cellular assays are stored in the corporate database.  Like computed phys. properties, this predicted kinase activity profile can be computed and stored as each compd. is registered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTg6KqlgEyLVg90H21EOLACvtfcHk0lhjX1-kpRVvtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D&md5=49422d01fd686a4befc9db7c123cfcce</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1005004%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DSullivan%26aufirst%3DD.%26aulast%3DJansen%26aufirst%3DJ.%26atitle%3DProfile-QSAR%253A%2520a%2520novel%2520meta-QSAR%2520method%2520that%2520combines%2520activities%2520across%2520the%2520kinase%2520family%2520to%2520accurately%2520predict%2520affinity%252C%2520selectivity%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1942%26epage%3D1956%26doi%3D10.1021%2Fci1005004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span><span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lhjX1-kpRVvtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laggner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matos, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glennon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, K. L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Predicting new molecular targets for known drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">181</span><span class="refDoi"> DOI: 10.1038/nature08506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnature08506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=19881490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=175-181&author=M.+J.+Keiserauthor=V.+Setolaauthor=J.+J.+Irwinauthor=C.+Laggnerauthor=A.+I.+Abbasauthor=S.+J.+Hufeisenauthor=N.+H.+Jensenauthor=M.+B.+Kuijerauthor=R.+C.+Matosauthor=T.+B.+Tranauthor=R.+Whaleyauthor=R.+A.+Glennonauthor=J.+Hertauthor=K.+L.+H.+Thomasauthor=D.+D.+Edwardsauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Predicting+new+molecular+targets+for+known+drugs&doi=10.1038%2Fnature08506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting new molecular targets for known drugs</span></div><div class="casAuthors">Keiser, Michael J.; Setola, Vincent; Irwin, John J.; Laggner, Christian; Abbas, Atheir I.; Hufeisen, Sandra J.; Jensen, Niels H.; Kuijer, Michael B.; Matos, Roberto C.; Tran, Thuy B.; Whaley, Ryan; Glennon, Richard A.; Hert, Jerome; Thomas, Kelan L. H.; Edwards, Douglas D.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although drugs are intended to be selective, at least some bind to several physiol. targets, explaining side effects and efficacy.  Because many drug-target combinations exist, it would be useful to explore possible interactions computationally.  Here we compared 3665 US Food and Drug Administration (FDA)-approved and investigational drugs against hundreds of targets, defining each target by its ligands.  Chem. similarities between drugs and ligand sets predicted thousands of unanticipated assocns.  Thirty were tested exptl., including the antagonism of the β1 receptor by the transporter inhibitor Prozac, the inhibition of the 5-hydroxytryptamine (5-HT) transporter by the ion channel drug Vadilex, and antagonism of the histamine H4 receptor by the enzyme inhibitor Rescriptor.  Overall, 23 new drug-target assocns. were confirmed, five of which were potent (<100 nM).  The physiol. relevance of one, the drug N,N-dimethyltryptamine (DMT) on serotonergic receptors, was confirmed in a knockout mouse.  The chem. similarity approach is systematic and comprehensive, and may suggest side-effects and new indications for many drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozd-8cCXUskrVg90H21EOLACvtfcHk0ljagdt4P5eh1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO&md5=974e65f5d5b4dabde1a463c429215546</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature08506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08506%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DLaggner%26aufirst%3DC.%26aulast%3DAbbas%26aufirst%3DA.%2BI.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DJensen%26aufirst%3DN.%2BH.%26aulast%3DKuijer%26aufirst%3DM.%2BB.%26aulast%3DMatos%26aufirst%3DR.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BB.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DGlennon%26aufirst%3DR.%2BA.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DK.%2BL.%2BH.%26aulast%3DEdwards%26aufirst%3DD.%2BD.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPredicting%2520new%2520molecular%2520targets%2520for%2520known%2520drugs%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D175%26epage%3D181%26doi%3D10.1038%2Fnature08506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Manallack, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gancia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montana, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingstone, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitley, D. C.</span><span> </span><span class="NLM_article-title">Selecting screening candidates for kinase and G protein-coupled receptor targets using neural networks</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1256</span><span class="NLM_x">–</span> <span class="NLM_lpage">1262</span><span class="refDoi"> DOI: 10.1021/ci020267c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci020267c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1256-1262&author=D.+T.+Manallackauthor=W.+R.+Pittauthor=E.+Ganciaauthor=J.+G.+Montanaauthor=D.+J.+Livingstoneauthor=M.+G.+Fordauthor=D.+C.+Whitley&title=Selecting+screening+candidates+for+kinase+and+G+protein-coupled+receptor+targets+using+neural+networks&doi=10.1021%2Fci020267c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci020267c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci020267c%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGancia%26aufirst%3DE.%26aulast%3DMontana%26aufirst%3DJ.%2BG.%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26atitle%3DSelecting%2520screening%2520candidates%2520for%2520kinase%2520and%2520G%2520protein-coupled%2520receptor%2520targets%2520using%2520neural%2520networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2002%26volume%3D42%26spage%3D1256%26epage%3D1262%26doi%3D10.1021%2Fci020267c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Ning, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karypisxy, G.</span><span> </span><span class="NLM_article-title">Improved machine learning models for predicting selective compounds</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">38</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1021/ci200346b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200346b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2kt77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=38-50&author=X.+Ningauthor=M.+Waltersauthor=G.+Karypisxy&title=Improved+machine+learning+models+for+predicting+selective+compounds&doi=10.1021%2Fci200346b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Machine Learning Models for Predicting Selective Compounds</span></div><div class="casAuthors">Ning, Xia; Walters, Michael; Karypis, George</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-50</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of small potent compds. that selectively bind to the target under consideration with high affinities is a crit. step toward successful drug discovery.  However, there is still a lack of efficient and accurate computational methods to predict compd. selectivity properties.  In this paper, we propose a set of machine learning methods to do compd. selectivity prediction.  In particular, we propose a novel cascaded learning method and a multitask learning method.  The cascaded method decomps. the selectivity prediction into two steps, one model for each step, so as to effectively filter out nonselective compds.  The multitask method incorporates both activity and selectivity models into one multitask model so as to better differentiate compd. selectivity properties.  We conducted a comprehensive set of expts. and compared the results with those of other conventional selectivity prediction methods, and our results demonstrated that the cascaded and multitask methods significantly improve the selectivity prediction performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdkf0ZGDNBm7Vg90H21EOLACvtfcHk0ljagdt4P5eh1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2kt77I&md5=2038119451a4a486b77457f966276e5e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci200346b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200346b%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DX.%26aulast%3DWalters%26aufirst%3DM.%26aulast%3DKarypisxy%26aufirst%3DG.%26atitle%3DImproved%2520machine%2520learning%2520models%2520for%2520predicting%2520selective%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D38%26epage%3D50%26doi%3D10.1021%2Fci200346b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Unterthiner, T.; Mayr, A.; Klambauer, G.; Steijaert, M.; Wegner, J. K.; Ceulemans, H.; Hochreiter, S.</span><span> </span><span class="NLM_article-title">Deep learning as an opportunity in virtual screening</span>. Proceedings of the Deep Learning Workshop at NIPS, Montreal (QC), December 12, 2014;<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=T.+Unterthiner&author=A.+Mayr&author=G.+Klambauer&author=M.+Steijaert&author=J.+K.+Wegner&author=H.+Ceulemans&author=S.+Hochreiter&title=Deep+learning+as+an+opportunity+in+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DUnterthiner%26aufirst%3DT.%26atitle%3DDeep%2520learning%2520as%2520an%2520opportunity%2520in%2520virtual%2520screening%26jtitle%3DProceedings%2520of%2520the%2520Deep%2520Learning%2520Workshop%2520at%2520NIPS%252C%2520Montreal%2520%2528QC%2529%252C%2520December%252012%252C%25202014%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Yabuuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niijima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takematsu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirokawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minowa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujimoto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuno, Y.</span><span> </span><span class="NLM_article-title">Analysis of multiple compound-protein interactions reveals novel bioactive molecules</span> <span class="citation_source-journal">Mol. Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">472</span><span class="refDoi"> DOI: 10.1038/msb.2011.5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fmsb.2011.5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=472&author=H.+Yabuuchiauthor=S.+Niijimaauthor=H.+Takematsuauthor=T.+Idaauthor=T.+Hirokawaauthor=T.+Haraauthor=T.+Ogawaauthor=Y.+Minowaauthor=G.+Tsujimotoauthor=Y.+Okuno&title=Analysis+of+multiple+compound-protein+interactions+reveals+novel+bioactive+molecules&doi=10.1038%2Fmsb.2011.5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2011.5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2011.5%26sid%3Dliteratum%253Aachs%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DTakematsu%26aufirst%3DH.%26aulast%3DIda%26aufirst%3DT.%26aulast%3DHirokawa%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DOgawa%26aufirst%3DT.%26aulast%3DMinowa%26aufirst%3DY.%26aulast%3DTsujimoto%26aufirst%3DG.%26aulast%3DOkuno%26aufirst%3DY.%26atitle%3DAnalysis%2520of%2520multiple%2520compound-protein%2520interactions%2520reveals%2520novel%2520bioactive%2520molecules%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D472%26doi%3D10.1038%2Fmsb.2011.5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J. H.</span><span> </span><span class="NLM_article-title">Comparison of random forest and Pipeline Pilot naive bayes in prospective QSAR predictions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">792</span><span class="NLM_x">–</span> <span class="NLM_lpage">803</span><span class="refDoi"> DOI: 10.1021/ci200615h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200615h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38XislajtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=792-803&author=B.+Chenauthor=R.+P.+Sheridanauthor=V.+Hornakauthor=J.+H.+Voigt&title=Comparison+of+random+forest+and+Pipeline+Pilot+naive+bayes+in+prospective+QSAR+predictions&doi=10.1021%2Fci200615h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Random Forest and Pipeline Pilot Naive Bayes in Prospective QSAR Predictions</span></div><div class="casAuthors">Chen, Bin; Sheridan, Robert P.; Hornak, Viktor; Voigt, Johannes H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">792-803</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Random forest is currently considered one of the best QSAR methods available in terms of accuracy of prediction.  However, it is computationally intensive.  Naive Bayes is a simple, robust classification method.  The Laplacian-modified Naive Bayes implementation is the preferred QSAR method in the widely used com. chemoinformatics platform Pipeline Pilot.  We made a comparison of the ability of Pipeline Pilot Naive Bayes (PLPNB) and random forest to make accurate predictions on 18 large, diverse inhouse QSAR data sets.  These include on-target and ADME-related activities.  These data sets were set up as classification problems with either binary or multicategory activities.  We used a time-split method of dividing training and test sets, as we feel this is a realistic way of simulating prospective prediction.  PLPNB is computationally efficient.  However, random forest predictions are at least as good and in many cases significantly better than those of PLPNB on our data sets.  PLPNB performs better with ECFP4 and ECFP6 descriptors, which are native to Pipeline Pilot, and more poorly with other descriptors we tried.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopM-mDzNRvcLVg90H21EOLACvtfcHk0liFFhQ1LdeoMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislajtbY%253D&md5=63488794c7eebebed1f34c2e6989e104</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci200615h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200615h%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DVoigt%26aufirst%3DJ.%2BH.%26atitle%3DComparison%2520of%2520random%2520forest%2520and%2520Pipeline%2520Pilot%2520naive%2520bayes%2520in%2520prospective%2520QSAR%2520predictions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D792%26epage%3D803%26doi%3D10.1021%2Fci200615h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Schürer, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muskal, S. M.</span><span> </span><span class="NLM_article-title">Kinome-wide activity modeling from diverse public high-quality data sets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1021/ci300403k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300403k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVygu77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=27-38&author=S.+C.+Sch%C3%BCrerauthor=S.+M.+Muskal&title=Kinome-wide+activity+modeling+from+diverse+public+high-quality+data+sets&doi=10.1021%2Fci300403k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Kinome-wide Activity Modeling from Diverse Public High-Quality Data Sets</span></div><div class="casAuthors">Schurer, Stephan C.; Muskal, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Large corpora of kinase small mol. inhibitor data are accessible to public sector research from thousands of journal article and patent publications.  These data have been generated employing a wide variety of assay methodologies and exptl. procedures by numerous labs.  Here we ask the question how applicable these heterogeneous data sets are to predict kinase activities and which characteristics of the data sets contribute to their utility.  We accessed almost 500 000 mols. from the Kinase Knowledge Base (KKB) and after rigorous aggregation and standardization generated over 180 distinct data sets covering all major groups of the human kinome.  To assess the value of the data sets, we generated hundreds of classification and regression models.  Their rigorous cross-validation and characterization demonstrated highly predictive classification and quant. models for the majority of kinase targets if a min. required no. of active compds. or structure-activity data points were available.  We then applied the best classifiers to compds. most recently profiled in the NIH Library of Integrated Network-based Cellular Signatures (LINCS) program and found good agreement of profiling results with predicted activities.  Our results indicate that, although heterogeneous in nature, the publically accessible data sets are exceedingly valuable and well suited to develop highly accurate predictors for practical Kinome-wide virtual screening applications and to complement exptl. kinase profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1rhwNqRlmD7Vg90H21EOLACvtfcHk0liFFhQ1LdeoMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVygu77L&md5=cea0a7b0c22b43e7660b430e8039f4e1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci300403k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300403k%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCrer%26aufirst%3DS.%2BC.%26aulast%3DMuskal%26aufirst%3DS.%2BM.%26atitle%3DKinome-wide%2520activity%2520modeling%2520from%2520diverse%2520public%2520high-quality%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D27%26epage%3D38%26doi%3D10.1021%2Fci300403k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Breiman, L.</span><span> </span><span class="NLM_article-title">Random forests</span> <span class="citation_source-journal">Machine Learning</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1023/A:1010933404324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1023%2FA%3A1010933404324" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2001&pages=5-32&author=L.+Breiman&title=Random+forests&doi=10.1023%2FA%3A1010933404324"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1023%2FA%3A1010933404324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1010933404324%26sid%3Dliteratum%253Aachs%26aulast%3DBreiman%26aufirst%3DL.%26atitle%3DRandom%2520forests%26jtitle%3DMachine%2520Learning%26date%3D2001%26volume%3D45%26spage%3D5%26epage%3D32%26doi%3D10.1023%2FA%3A1010933404324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Svetnik, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liaw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culberson, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feuston, B. P.</span><span> </span><span class="NLM_article-title">Random forest: a classification and regression tool for compound classification and QSAR modeling</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1947</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span><span class="refDoi"> DOI: 10.1021/ci034160g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci034160g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos1Wiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=1947-1958&author=V.+Svetnikauthor=A.+Liawauthor=C.+Tongauthor=J.+C.+Culbersonauthor=R.+P.+Sheridanauthor=B.+P.+Feuston&title=Random+forest%3A+a+classification+and+regression+tool+for+compound+classification+and+QSAR+modeling&doi=10.1021%2Fci034160g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Random Forest: A Classification and Regression Tool for Compound Classification and QSAR Modeling</span></div><div class="casAuthors">Svetnik, Vladimir; Liaw, Andy; Tong, Christopher; Culberson, J. Christopher; Sheridan, Robert P.; Feuston, Bradley P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1947-1958</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new classification and regression tool, Random Forest, is introduced and investigated for predicting a compd.'s quant. or categorical biol. activity based on a quant. description of the compd.'s mol. structure.  Random Forest is an ensemble of unpruned classification or regression trees created by using bootstrap samples of the training data and random feature selection in tree induction.  Prediction is made by aggregating (majority vote or averaging) the predictions of the ensemble.  The authors built predictive models for six cheminformatics data sets.  The authors anal. demonstrates that Random Forest is a powerful tool capable of delivering performance that is among the most accurate methods to date.  The authors also present three addnl. features of Random Forest: built-in performance assessment, a measure of relative importance of descriptors, and a measure of compd. similarity that is weighted by the relative importance of descriptors.  It is the combination of relatively high prediction accuracy and its collection of desired features that makes Random Forest uniquely suited for modeling in cheminformatics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjoRt5v7rYwLVg90H21EOLACvtfcHk0ljksQl2DHfebA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos1Wiu7s%253D&md5=dea7867551ec30260b0091b90593a660</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci034160g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034160g%26sid%3Dliteratum%253Aachs%26aulast%3DSvetnik%26aufirst%3DV.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DC.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DFeuston%26aufirst%3DB.%2BP.%26atitle%3DRandom%2520forest%253A%2520a%2520classification%2520and%2520regression%2520tool%2520for%2520compound%2520classification%2520and%2520QSAR%2520modeling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2003%26volume%3D43%26spage%3D1947%26epage%3D1958%26doi%3D10.1021%2Fci034160g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Goldstein, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">High-throughput kinase profiling as a platform for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">391</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span><span class="refDoi"> DOI: 10.1038/nrd2541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnrd2541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=18404149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=391-397&author=D.+M.+Goldsteinauthor=N.+S.+Grayauthor=P.+P.+Zarrinkar&title=High-throughput+kinase+profiling+as+a+platform+for+drug+discovery&doi=10.1038%2Fnrd2541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput kinase profiling as a platform for drug discovery</span></div><div class="casAuthors">Goldstein, David M.; Gray, Nathanael S.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">391-397</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput profiling of compd. libraries against large panels of kinases is becoming tech. feasible.  In contrast to the traditional linear, target-centric approach to discovery, this approach may provide a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone, and could thereby significantly improve the productivity of kinase inhibitor discovery.  To fully exploit the potential of kinases as drug targets, novel strategies for the efficient discovery of inhibitors are required.  In contrast to the traditional, linear process of inhibitor discovery, high-throughput kinase profiling enables a parallel approach by interrogating compds. against hundreds of targets in a single screen.  Compd. potency and selectivity are detd. simultaneously, providing a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwnJ59cVxbJbVg90H21EOLACvtfcHk0ljksQl2DHfebA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D&md5=eaa61fd2958ff04fecf86a993f91833d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd2541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2541%26sid%3Dliteratum%253Aachs%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DHigh-throughput%2520kinase%2520profiling%2520as%2520a%2520platform%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D391%26epage%3D397%26doi%3D10.1038%2Fnrd2541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Cortés-Ciriano, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ain, Q. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanian, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenselink, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Méndez-Lucio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">IJzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlfahrt, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malliavin, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Westen, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span> </span><span class="NLM_article-title">Polypharmacology modelling using proteochemometrics (PCM): recent methodological developments, applications to target families, and future prospects</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">24</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1039/C4MD00216D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1039%2FC4MD00216D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslagsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=24-50&author=I.+Cort%C3%A9s-Cirianoauthor=Q.+U.+Ainauthor=V.+Subramanianauthor=E.+B.+Lenselinkauthor=O.+M%C3%A9ndez-Lucioauthor=A.+P.+IJzermanauthor=G.+Wohlfahrtauthor=P.+Prusisauthor=T.+E.+Malliavinauthor=G.+J.+van+Westenauthor=A.+Bender&title=Polypharmacology+modelling+using+proteochemometrics+%28PCM%29%3A+recent+methodological+developments%2C+applications+to+target+families%2C+and+future+prospects&doi=10.1039%2FC4MD00216D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology modelling using proteochemometrics (PCM): recent methodological developments, applications to target families, and future prospects</span></div><div class="casAuthors">Cortes-Ciriano, Isidro; Ain, Qurrat Ul; Subramanian, Vigneshwari; Lenselink, Eelke B.; Mendez-Lucio, Oscar; IJzerman, Adriaan P.; Wohlfahrt, Gerd; Prusis, Peteris; Malliavin, Therese E.; van Westen, Gerard J. P.; Bender, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-50</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Proteochemometric (PCM) modeling is a computational method to model the bioactivity of multiple ligands against multiple related protein targets simultaneously.  Hence it has been found to be particularly useful when exploring the selectivity and promiscuity of ligands on different proteins.  In this review, we will firstly provide a brief introduction to the main concepts of PCM for readers new to the field.  The next part focuses on recent tech. advances, including the application of support vector machines (SVMs) using different kernel functions, random forests, Gaussian processes and collaborative filtering.  The subsequent section will then describe some novel practical applications of PCM in the medicinal chem. field, including studies on GPCRs, kinases, viral proteins (e.g. from HIV) and epigenetic targets such as histone deacetylases.  Finally, we will conclude by summarizing novel developments in PCM, which we expect to gain further importance in the future.  These developments include adding three-dimensional protein target information, application of PCM to the prediction of binding energies, and application of the concept in the fields of pharmacogenomics and toxicogenomics.  This review is an update to a related publication in 2011 and it mainly focuses on developments in the field since then.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEtwphLWc6brVg90H21EOLACvtfcHk0ljksQl2DHfebA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslagsLrE&md5=b3e82734e224ae4990218b6473ef7859</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2FC4MD00216D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00216D%26sid%3Dliteratum%253Aachs%26aulast%3DCort%25C3%25A9s-Ciriano%26aufirst%3DI.%26aulast%3DAin%26aufirst%3DQ.%2BU.%26aulast%3DSubramanian%26aufirst%3DV.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DM%25C3%25A9ndez-Lucio%26aufirst%3DO.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DWohlfahrt%26aufirst%3DG.%26aulast%3DPrusis%26aufirst%3DP.%26aulast%3DMalliavin%26aufirst%3DT.%2BE.%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DPolypharmacology%2520modelling%2520using%2520proteochemometrics%2520%2528PCM%2529%253A%2520recent%2520methodological%2520developments%252C%2520applications%2520to%2520target%2520families%252C%2520and%2520future%2520prospects%26jtitle%3DMedChemComm%26date%3D2015%26volume%3D6%26spage%3D24%26epage%3D50%26doi%3D10.1039%2FC4MD00216D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Zakharov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peach, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitzmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicklaus, M. C.</span><span> </span><span class="NLM_article-title">QSAR modeling of imbalanced high-throughput screening data in PubChem</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">705</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span><span class="refDoi"> DOI: 10.1021/ci400737s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400737s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlartb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=705-712&author=A.+V.+Zakharovauthor=M.+L.+Peachauthor=M.+Sitzmannauthor=M.+C.+Nicklaus&title=QSAR+modeling+of+imbalanced+high-throughput+screening+data+in+PubChem&doi=10.1021%2Fci400737s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Modeling of Imbalanced High-Throughput Screening Data in PubChem</span></div><div class="casAuthors">Zakharov, Alexey V.; Peach, Megan L.; Sitzmann, Markus; Nicklaus, Marc C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">705-712</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many of the structures in PubChem are annotated with activities detd. in high-throughput screening (HTS) assays.  Because of the nature of these assays, the activity data are typically strongly imbalanced, with a small no. of active compds. contrasting with a very large no. of inactive compds.  We have used several such imbalanced PubChem HTS assays to test and develop strategies to efficiently build robust QSAR models from imbalanced data sets.  Different descriptor types [Quant. Neighborhoods of Atoms (QNA) and "biol." descriptors] were used to generate a variety of QSAR models in the program GUSAR.  The models obtained were compared using external test and validation sets.  We also report on our efforts to incorporate the most predictive of our models in the publicly available NCI/CADD Group Web services (http://cactus.nci.nih.gov/chem./apps/cap).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq51slAln8D2LVg90H21EOLACvtfcHk0lj0RQtbuPCp-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlartb0%253D&md5=f2c9ba5e2843c4ef3713ac6fb3aafbed</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci400737s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400737s%26sid%3Dliteratum%253Aachs%26aulast%3DZakharov%26aufirst%3DA.%2BV.%26aulast%3DPeach%26aufirst%3DM.%2BL.%26aulast%3DSitzmann%26aufirst%3DM.%26aulast%3DNicklaus%26aufirst%3DM.%2BC.%26atitle%3DQSAR%2520modeling%2520of%2520imbalanced%2520high-throughput%2520screening%2520data%2520in%2520PubChem%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D705%26epage%3D712%26doi%3D10.1021%2Fci400737s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Gao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, U. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovelman, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, K. J.</span><span> </span><span class="NLM_article-title">A broad activity screen in support of a chemogenomic map for kinase signalling research and drug discovery</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">451</span><span class="NLM_x">, </span> <span class="NLM_fpage">313</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.1042/BJ20121418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1042%2FBJ20121418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=23398362" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvFOgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2013&pages=313-328&author=Y.+Gaoauthor=S.+P.+Daviesauthor=M.+Augustinauthor=A.+Woodwardauthor=U.+A.+Patelauthor=R.+Kovelmanauthor=K.+J.+Harvey&title=A+broad+activity+screen+in+support+of+a+chemogenomic+map+for+kinase+signalling+research+and+drug+discovery&doi=10.1042%2FBJ20121418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A broad activity screen in support of a chemogenomic map for kinase signalling research and drug discovery</span></div><div class="casAuthors">Gao, Yinghong; Davies, Stephen P.; Augustin, Martin; Woodward, Anna; Patel, Umesh A.; Kovelman, Robert; Harvey, Kevin J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-328</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Despite the development of a no. of efficacious kinase inhibitors, the strategies for rational design of these compds. have been limited by target promiscuity.  In an effort to better understand the nature of kinase inhibition across the kinome, esp. as it relates to off-target effects, we screened a well-defined collection of kinase inhibitors using biochem. assays for inhibitory activity against 234 active human kinases and kinase complexes, representing all branches of the kinome tree.  For our study we employed 158 small mols. initially identified in the literature as potent and specific inhibitors of kinases important as therapeutic targets and/or signal transduction regulators.  Hierarchical clustering of these benchmark kinase inhibitors on the basis of their kinome activity profiles illustrates how they relate to chem. structure similarities and provides new insights into inhibitor specificity and potential applications for probing new targets.  Using this broad dataset, we provide a framework for assessing polypharmacol.  We not only discover likely off-target inhibitor activities and recommend specific inhibitors for existing targets, but also identify potential new uses for known small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNLU0ONM4SALVg90H21EOLACvtfcHk0lj0RQtbuPCp-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvFOgs7g%253D&md5=9e0f17ec5cac10198fe28b27e70b4966</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FBJ20121418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121418%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DY.%26aulast%3DDavies%26aufirst%3DS.%2BP.%26aulast%3DAugustin%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DU.%2BA.%26aulast%3DKovelman%26aufirst%3DR.%26aulast%3DHarvey%26aufirst%3DK.%2BJ.%26atitle%3DA%2520broad%2520activity%2520screen%2520in%2520support%2520of%2520a%2520chemogenomic%2520map%2520for%2520kinase%2520signalling%2520research%2520and%2520drug%2520discovery%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D451%26spage%3D313%26epage%3D328%26doi%3D10.1042%2FBJ20121418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Dranchak, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacArthur, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Profile of the GSK published protein kinase inhibitor set across ATP-dependent and-independent luciferases: implications for reporter-gene assays</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e57888</span><span class="refDoi"> DOI: 10.1371/journal.pone.0057888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1371%2Fjournal.pone.0057888" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e57888&author=P.+Dranchakauthor=R.+MacArthurauthor=R.+Guhaauthor=W.+J.+Zuercherauthor=D.+H.+Drewryauthor=D.+S.+Auldauthor=J.+Inglese&title=Profile+of+the+GSK+published+protein+kinase+inhibitor+set+across+ATP-dependent+and-independent+luciferases%3A+implications+for+reporter-gene+assays&doi=10.1371%2Fjournal.pone.0057888"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0057888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0057888%26sid%3Dliteratum%253Aachs%26aulast%3DDranchak%26aufirst%3DP.%26aulast%3DMacArthur%26aufirst%3DR.%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DProfile%2520of%2520the%2520GSK%2520published%2520protein%2520kinase%2520inhibitor%2520set%2520across%2520ATP-dependent%2520and-independent%2520luciferases%253A%2520implications%2520for%2520reporter-gene%2520assays%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De57888%26doi%3D10.1371%2Fjournal.pone.0057888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arruda, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillespie, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackey, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzafera, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomkinson, N. C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">A public-private partnership to unlock the untargeted kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1038/nchembio.1113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnchembio.1113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=23238671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVekur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3-6&author=S.+Knappauthor=P.+Arrudaauthor=J.+Blaggauthor=S.+Burleyauthor=D.+H.+Drewryauthor=A.+Edwardsauthor=D.+Fabbroauthor=P.+Gillespieauthor=N.+S.+Grayauthor=B.+Kusterauthor=K.+E.+Lackeyauthor=P.+Mazzaferaauthor=N.+C.+O.+Tomkinsonauthor=T.+M.+Willsonauthor=P.+Workmanauthor=W.+J.+Zuercher&title=A+public-private+partnership+to+unlock+the+untargeted+kinome&doi=10.1038%2Fnchembio.1113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A public-private partnership to unlock the untargeted kinome</span></div><div class="casAuthors">Knapp, Stefan; Arruda, Paulo; Blagg, Julian; Burley, Stephen; Drewry, David H.; Edwards, Aled; Fabbro, Doriano; Gillespie, Paul; Gray, Nathanael S.; Kuster, Bernhard; Lackey, Karen E.; Mazzafera, Paulo; Tomkinson, Nicholas C. O.; Willson, Timothy M.; Workman, Paul; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-6</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are urgently needed to functionally annotate hitherto-untargeted kinases and stimulate new drug discovery efforts to address unmet medical needs.  The size of the human kinome combined with the high cost assocd. with probe generation severely limits access to new probes.  We propose a large-scale public-private partnership as a new approach that offers economies of scale, minimized redundancy and sharing of risk and cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8PARtv7U7v7Vg90H21EOLACvtfcHk0lj0RQtbuPCp-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVekur%252FF&md5=1f6c13ec461f96201838934cb95012c0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1113%26sid%3Dliteratum%253Aachs%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DArruda%26aufirst%3DP.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBurley%26aufirst%3DS.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGillespie%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DLackey%26aufirst%3DK.%2BE.%26aulast%3DMazzafera%26aufirst%3DP.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DA%2520public-private%2520partnership%2520to%2520unlock%2520the%2520untargeted%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D3%26epage%3D6%26doi%3D10.1038%2Fnchembio.1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Elkins, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedele, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szklarz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdul Azeez, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salah, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikolajczyk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sepetov, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al Haj Zen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fourches, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muratov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teicher, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunkel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polley, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackey, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkinson, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">Comprehensive characterization of the Published Kinase Inhibitor Set</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">95</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1038/nbt.3374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt.3374" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=95-103&author=J.+M.+Elkinsauthor=V.+Fedeleauthor=M.+Szklarzauthor=K.+R.+Abdul+Azeezauthor=E.+Salahauthor=J.+Mikolajczykauthor=S.+Romanovauthor=N.+Sepetovauthor=X.-P.+Huangauthor=B.+L.+Rothauthor=A.+Al+Haj+Zenauthor=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropshaauthor=J.+Morrisauthor=B.+A.+Teicherauthor=M.+Kunkelauthor=E.+Polleyauthor=K.+E.+Lackeyauthor=F.+L.+Atkinsonauthor=J.+P.+Overingtonauthor=P.+Bamboroughauthor=S.+M%C3%BCllerauthor=D.+J.+Priceauthor=T.+M.+Willsonauthor=D.+H.+Drewryauthor=S.+Knappauthor=W.+J.+Zuercher&title=Comprehensive+characterization+of+the+Published+Kinase+Inhibitor+Set&doi=10.1038%2Fnbt.3374"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3374%26sid%3Dliteratum%253Aachs%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR.%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DMikolajczyk%26aufirst%3DJ.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DSepetov%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DAl%2BHaj%2BZen%26aufirst%3DA.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DTeicher%26aufirst%3DB.%2BA.%26aulast%3DKunkel%26aufirst%3DM.%26aulast%3DPolley%26aufirst%3DE.%26aulast%3DLackey%26aufirst%3DK.%2BE.%26aulast%3DAtkinson%26aufirst%3DF.%2BL.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DComprehensive%2520characterization%2520of%2520the%2520Published%2520Kinase%2520Inhibitor%2520Set%26jtitle%3DNat.%2520Biotechnol.%26date%3D2015%26volume%3D34%26spage%3D95%26epage%3D103%26doi%3D10.1038%2Fnbt.3374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eid, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaeger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fulle, S.</span><span> </span><span class="NLM_article-title">Pocketome of human kinases: prioritizing the ATP binding sites of (yet) untapped protein kinases for drug discovery</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">549</span><span class="refDoi"> DOI: 10.1021/ci500624s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500624s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A280%3ADC%252BC2MvjsVSksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=538-549&author=A.+Volkamerauthor=S.+Eidauthor=S.+Turkauthor=S.+Jaegerauthor=F.+Rippmannauthor=S.+Fulle&title=Pocketome+of+human+kinases%3A+prioritizing+the+ATP+binding+sites+of+%28yet%29+untapped+protein+kinases+for+drug+discovery&doi=10.1021%2Fci500624s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pocketome of human kinases: prioritizing the ATP binding sites of (yet) untapped protein kinases for drug discovery</span></div><div class="casAuthors">Volkamer Andrea; Eid Sameh; Turk Samo; Jaeger Sabrina; Fulle Simone; Rippmann Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">538-49</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein kinases are involved in a variety of diseases including cancer, inflammation, and autoimmune disorders.  Although the development of new kinase inhibitors is a major focus in pharmaceutical research, a large number of kinases remained so far unexplored in drug discovery projects.  The selection and assessment of targets is an essential but challenging area.  Today, a few thousands of experimentally determined kinase structures are available, covering about half of the human kinome.  This large structural source allows guiding the target selection via structure-based druggability prediction approaches such as DoGSiteScorer.  Here, a thorough analysis of the ATP pockets of the entire human kinome in the DFG-in state is presented in order to prioritize novel kinase structures for drug discovery projects.  For this, all human kinase X-ray structures available in the PDB were collected, and homology models were generated for the missing part of the kinome.  DoGSiteScorer was used to calculate geometrical and physicochemical properties of the ATP pockets and to predict the potential of each kinase to be druggable.  The results indicate that about 75% of the kinome are in principle druggable.  Top ranking structures comprise kinases that are primary targets of known approved drugs but additionally point to so far less explored kinases.  The presented analysis provides new insights into the druggability of ATP binding pockets of the entire kinome.  We anticipate this comprehensive druggability assessment of protein kinases to be helpful for the community to prioritize so far untapped kinases for drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXYhJ2uLp2lX6HPRus0rAafW6udTcc2eb5q2YRwGaHJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvjsVSksw%253D%253D&md5=85c8e48d59f1acea82d1019da51b47e1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci500624s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500624s%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DEid%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DJaeger%26aufirst%3DS.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DPocketome%2520of%2520human%2520kinases%253A%2520prioritizing%2520the%2520ATP%2520binding%2520sites%2520of%2520%2528yet%2529%2520untapped%2520protein%2520kinases%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D538%26epage%3D549%26doi%3D10.1021%2Fci500624s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Kalliokoski, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vulpetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedeck, P.</span><span> </span><span class="NLM_article-title">Comparability of mixed IC<sub>50</sub> data-a statistical analysis</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e61007</span><span class="refDoi"> DOI: 10.1371/journal.pone.0061007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1371%2Fjournal.pone.0061007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=23613770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslSitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e61007&author=T.+Kalliokoskiauthor=C.+Kramerauthor=A.+Vulpettiauthor=P.+Gedeck&title=Comparability+of+mixed+IC50+data-a+statistical+analysis&doi=10.1371%2Fjournal.pone.0061007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Comparability of mixed IC50 data - a statistical analysis</span></div><div class="casAuthors">Kalliokoski, Tuomo; Kramer, Christian; Vulpetti, Anna; Gedeck, Peter</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e61007</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The biochem. half maximal inhibitory concn. (IC50) is the most commonly used metric for on-target activity in lead optimization.  It is used to guide lead optimization, build large-scale chemogenomics anal., off-target activity and toxicity models based on public data.  However, the use of public biochem. IC50 data is problematic, because they are assay specific and comparable only under certain conditions.  For large scale anal. it is not feasible to check each data entry manually and it is very tempting to mix all available IC50 values from public database even if assay information is not reported.  As previously reported for Ki database anal., we first analyzed the types of errors, the redundancy and the variability that can be found in ChEMBL IC50 database.  For assessing the variability of IC50 data independently measured in two different labs at least ten IC50 data for identical protein-ligand systems against the same target were searched in ChEMBL.  As a not sufficient no. of cases of this type are available, the variability of IC50 data was assessed by comparing all pairs of independent IC50 measurements on identical protein-ligand systems.  The std. deviation of IC50 data is only 25% larger than the std. deviation of Ki data, suggesting that mixing IC50 data from different assays, even not knowing assay conditions details, only adds a moderate amt. of noise to the overall data.  The std. deviation of public ChEMBL IC50 data, as expected, resulted greater than the std. deviation of inhouse intra-lab./inter-day IC50 data.  Augmenting mixed public IC50 data by public Ki data does not deteriorate the quality of the mixed IC50 data, if the Ki is cor. by an offset.  For a broad dataset such as ChEMBL database a Ki-IC50 conversion factor of 2 was found to be the most reasonable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq07UjgtQJyZ7Vg90H21EOLACvtfcHk0ljsoXMT4EDAfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslSitrc%253D&md5=591144d98c123366e344688c74156e9e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0061007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0061007%26sid%3Dliteratum%253Aachs%26aulast%3DKalliokoski%26aufirst%3DT.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DGedeck%26aufirst%3DP.%26atitle%3DComparability%2520of%2520mixed%2520IC50%2520data-a%2520statistical%2520analysis%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De61007%26doi%3D10.1371%2Fjournal.pone.0061007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span>R: a Language and Environment for Statistical Computing; <span class="NLM_publisher-name">R Core Team</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span><a href="https://www.R-project.org/" class="extLink">https://www.R-project.org/</a> (accessed February 18, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=R%3A+a+Language+and+Environment+for+Statistical+Computing%3B+R+Core+Team%2C+2015%3B+https%3A%2F%2Fwww.R-project.org%2F+%28accessed+February+18%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DR%253A%2520a%2520Language%2520and%2520Environment%2520for%2520Statistical%2520Computing%26pub%3DR%2520Core%2520Team%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Oksanen, J.; Blanchet, F. G.; Kindt, R.; Legendre, P.; Minchin, P. R.; O’Hara, R. B.; Simpson, G. L.; Solymos, P.; Stevens, M. H. H.; Wagner, H.</span>Vegan: Community Ecology Package;<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><div class="note"><p class="first last">R package version 2.3–4</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+Oksanen&author=F.+G.+Blanchet&author=R.+Kindt&author=P.+Legendre&author=P.+R.+Minchin&author=R.+B.+O%E2%80%99Hara&author=G.+L.+Simpson&author=P.+Solymos&author=M.+H.+H.+Stevens&author=H.+Wagner&title=Vegan%3A+Community+Ecology+Package"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DOksanen%26aufirst%3DJ.%26jtitle%3DVegan%253A%2520Community%2520Ecology%2520Package%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Neuwirth, E.</span>RColorBrewer: Colorbrewer Palettes;<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span><div class="note"><p class="first last">R package version 1.1–2</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=E.+Neuwirth&title=RColorBrewer%3A+Colorbrewer+Palettes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DNeuwirth%26aufirst%3DE.%26jtitle%3DRColorBrewer%253A%2520Colorbrewer%2520Palettes%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Wal, J. V. D.; Falconi, L.; Januchowski, S.; Shoo, L.; Storlie, C.</span>SDMTools: Species Distribution Modelling Tools; Tools for Processing Data Associated with Species Distribution Modelling Exercises,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span><div class="note"><p class="first last">R package version 1.1–221</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=J.+V.+D.+Wal&author=L.+Falconi&author=S.+Januchowski&author=L.+Shoo&author=C.+Storlie&title=SDMTools%3A+Species+Distribution+Modelling+Tools%3B+Tools+for+Processing+Data+Associated+with+Species+Distribution+Modelling+Exercises"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DWal%26aufirst%3DJ.%2BV.%2BD.%26jtitle%3DSDMTools%253A%2520Species%2520Distribution%2520Modelling%2520Tools%253B%2520Tools%2520for%2520Processing%2520Data%2520Associated%2520with%2520Species%2520Distribution%2520Modelling%2520Exercises%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span>RDKit: Open-Source Cheminformatics Software,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a>, (accessed February 18, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%3A+Open-Source+Cheminformatics+Software%2C+2016%3B+http%3A%2F%2Fwww.rdkit.org%2C+%28accessed+February+18%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DRDKit%253A%2520Open-Source%2520Cheminformatics%2520Software%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Rogers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, M.</span><span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">754</span><span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0ljsoXMT4EDAfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Pedregosa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varoquaux, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gramfort, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michel, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thirion, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grisel, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blondel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prettenhofer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubourg, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanderplas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Passos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cournapeau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brucher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duchesnay, E.</span><span> </span><span class="NLM_article-title"><i>Scikit-learn</i>: machine learning in Python</span> <span class="citation_source-journal">J. Mach. Learn. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2825</span><span class="NLM_x">–</span> <span class="NLM_lpage">2830</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=2825-2830&author=F.+Pedregosaauthor=G.+Varoquauxauthor=A.+Gramfortauthor=V.+Michelauthor=B.+Thirionauthor=O.+Griselauthor=M.+Blondelauthor=P.+Prettenhoferauthor=R.+Weissauthor=V.+Dubourgauthor=J.+Vanderplasauthor=A.+Passosauthor=D.+Cournapeauauthor=M.+Brucherauthor=M.+Perrotauthor=E.+Duchesnay&title=Scikit-learn%3A+machine+learning+in+Python"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPedregosa%26aufirst%3DF.%26aulast%3DVaroquaux%26aufirst%3DG.%26aulast%3DGramfort%26aufirst%3DA.%26aulast%3DMichel%26aufirst%3DV.%26aulast%3DThirion%26aufirst%3DB.%26aulast%3DGrisel%26aufirst%3DO.%26aulast%3DBlondel%26aufirst%3DM.%26aulast%3DPrettenhofer%26aufirst%3DP.%26aulast%3DWeiss%26aufirst%3DR.%26aulast%3DDubourg%26aufirst%3DV.%26aulast%3DVanderplas%26aufirst%3DJ.%26aulast%3DPassos%26aufirst%3DA.%26aulast%3DCournapeau%26aufirst%3DD.%26aulast%3DBrucher%26aufirst%3DM.%26aulast%3DPerrot%26aufirst%3DM.%26aulast%3DDuchesnay%26aufirst%3DE.%26atitle%3DScikit-learn%253A%2520machine%2520learning%2520in%2520Python%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2011%26volume%3D12%26spage%3D2825%26epage%3D2830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Xia, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maliski, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, D.</span><span> </span><span class="NLM_article-title">Classification of kinase inhibitors using a bayesian model</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4463</span><span class="NLM_x">–</span> <span class="NLM_lpage">4470</span><span class="refDoi"> DOI: 10.1021/jm0303195</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0303195" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1Sgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4463-4470&author=X.+Xiaauthor=E.+G.+Maliskiauthor=P.+Gallantauthor=D.+Rogers&title=Classification+of+kinase+inhibitors+using+a+bayesian+model&doi=10.1021%2Fjm0303195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Kinase Inhibitors Using a Bayesian Model</span></div><div class="casAuthors">Xia, Xiaoyang; Maliski, Edward G.; Gallant, Paul; Rogers, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4463-4470</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of Bayesian statistics to model both general (multifamily) and specific (single-target) kinase inhibitors is investigated.  The approach demonstrates an alternative to current computational methods applied to heterogeneous structure/activity data sets.  This approach operates rapidly and is readily modifiable as required.  A generalized model generated using inhibitor data from multiple kinase classes shows meaningful enrichment for several specific kinase targets.  Such an approach can be used to prioritize compds. for screening or to optimally select compds. from third-party data collections.  The obsd. benefit of the approach is finding compds. that are not structurally related to known actives, or novel targets for which there is not enough information to build a specific kinase model.  The general kinase model described was built from a basis of mostly tyrosine kinase inhibitors, with some serine/threonine inhibitors; all the test cases used in prediction were also on tyrosine kinase targets.  Confirming the applicability of this technique to other kinase families will be detd. once those biol. assays become available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrySh6p6L1C8rVg90H21EOLACvtfcHk0likhlqivUGbwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmt1Sgu78%253D&md5=6a7a80edee72fc9d33709a7dfdb7f0b5</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm0303195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303195%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DX.%26aulast%3DMaliski%26aufirst%3DE.%2BG.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DRogers%26aufirst%3DD.%26atitle%3DClassification%2520of%2520kinase%2520inhibitors%2520using%2520a%2520bayesian%2520model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4463%26epage%3D4470%26doi%3D10.1021%2Fjm0303195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Abadi, M.; Agarwal, A.; Barham, P.; Brevdo, E.; Chen, Z.; Citro, C.; Corrado, G. S.; Davis, A.; Dean, J.; Devin, M.; Ghemawat, S.; Goodfellow, I.; Harp, A.; Irving, G.; Isard, M.; Jia, Y.; Jozefowicz, R.; Kaiser, L.; Kudlur, M.; Levenberg, J.; Mané, D.; Monga, R.; Moore, S.; Murray, D.; Olah, C.; Schuster, M.; Shlens, J.; Steiner, B.; Sutskever, I.; Talwar, K.; Tucker, P.; Vanhoucke, V.; Vasudevan, V.; Viégas, F.; Vinyals, O.; Warden, P.; Wattenberg, M.; Wicke, M.; Yu, Y.; Zheng, X.</span><i>TensorFlow</i>: large-scale machine learning on heterogeneous systems.<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span><a href="http://tensorflow.org/" class="extLink">http://tensorflow.org/</a>, accessed February 18, 2016.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Abadi%2C+M.%3B+Agarwal%2C+A.%3B+Barham%2C+P.%3B+Brevdo%2C+E.%3B+Chen%2C+Z.%3B+Citro%2C+C.%3B+Corrado%2C+G.+S.%3B+Davis%2C+A.%3B+Dean%2C+J.%3B+Devin%2C+M.%3B+Ghemawat%2C+S.%3B+Goodfellow%2C+I.%3B+Harp%2C+A.%3B+Irving%2C+G.%3B+Isard%2C+M.%3B+Jia%2C+Y.%3B+Jozefowicz%2C+R.%3B+Kaiser%2C+L.%3B+Kudlur%2C+M.%3B+Levenberg%2C+J.%3B+Man%C3%A9%2C+D.%3B+Monga%2C+R.%3B+Moore%2C+S.%3B+Murray%2C+D.%3B+Olah%2C+C.%3B+Schuster%2C+M.%3B+Shlens%2C+J.%3B+Steiner%2C+B.%3B+Sutskever%2C+I.%3B+Talwar%2C+K.%3B+Tucker%2C+P.%3B+Vanhoucke%2C+V.%3B+Vasudevan%2C+V.%3B+Vi%C3%A9gas%2C+F.%3B+Vinyals%2C+O.%3B+Warden%2C+P.%3B+Wattenberg%2C+M.%3B+Wicke%2C+M.%3B+Yu%2C+Y.%3B+Zheng%2C+X.TensorFlow%3A+large-scale+machine+learning+on+heterogeneous+systems.+2015%3B+http%3A%2F%2Ftensorflow.org%2F%2C+accessed+February+18%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26aulast%3DAbadi%26aufirst%3DM.%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Nair, V.; Hinton, G. E.</span><span> </span><span class="NLM_article-title">Rectified linear units improve restricted boltzmann machines</span>. <i>Proceedings of the 27th International Conference on Machine Learning (ICML-10), Haifa, Isreal, June 21–24, 2010</i>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">807</span><span class="NLM_x">–</span> <span class="NLM_lpage">814</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=807-814&author=V.+Nair&author=G.+E.+Hinton&title=Rectified+linear+units+improve+restricted+boltzmann+machines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DV.%26atitle%3DRectified%2520linear%2520units%2520improve%2520restricted%2520boltzmann%2520machines%26date%3D2010%26spage%3D807%26epage%3D814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Srivastava, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hinton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krizhevsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutskever, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salakhutdinov, R.</span><span> </span><span class="NLM_article-title">Dropout: a simple way to prevent neural networks from overfitting</span> <span class="citation_source-journal">J. Mach. Learn. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1929</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=1929-1958&author=N.+Srivastavaauthor=G.+Hintonauthor=A.+Krizhevskyauthor=I.+Sutskeverauthor=R.+Salakhutdinov&title=Dropout%3A+a+simple+way+to+prevent+neural+networks+from+overfitting"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DN.%26aulast%3DHinton%26aufirst%3DG.%26aulast%3DKrizhevsky%26aufirst%3DA.%26aulast%3DSutskever%26aufirst%3DI.%26aulast%3DSalakhutdinov%26aufirst%3DR.%26atitle%3DDropout%253A%2520a%2520simple%2520way%2520to%2520prevent%2520neural%2520networks%2520from%2520overfitting%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2014%26volume%3D15%26spage%3D1929%26epage%3D1958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Mani, I.; Zhang, I.</span><span> </span><span class="NLM_article-title">kNN approach to unbalanced data distributions: a case study involving information extraction</span>. Proceedings of Workshop on Learning from Imbalanced Datasets, ICML, Washington DC, August 21, 2003.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mani%2C+I.%3B+Zhang%2C+I.+kNN+approach+to+unbalanced+data+distributions%3A+a+case+study+involving+information+extraction.+Proceedings+of+Workshop+on+Learning+from+Imbalanced+Datasets%2C+ICML%2C+Washington+DC%2C+August+21%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DMani%26aufirst%3DI.%26atitle%3DkNN%2520approach%2520to%2520unbalanced%2520data%2520distributions%253A%2520a%2520case%2520study%2520involving%2520information%2520extraction%26jtitle%3DProceedings%2520of%2520Workshop%2520on%2520Learning%2520from%2520Imbalanced%2520Datasets%252C%2520ICML%252C%2520Washington%2520DC%252C%2520August%252021%252C%25202003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Chawla, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowyer, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kegelmeyer, W. P.</span><span> </span><span class="NLM_article-title">SMOTE: synthetic minority over-sampling technique</span> <span class="citation_source-journal">J. Artif. Intell. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_fpage">321</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1613/jair.953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1613%2Fjair.953" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&pages=321-357&author=N.+V.+Chawlaauthor=K.+W.+Bowyerauthor=L.+O.+Hallauthor=W.+P.+Kegelmeyer&title=SMOTE%3A+synthetic+minority+over-sampling+technique&doi=10.1613%2Fjair.953"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1613%2Fjair.953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1613%252Fjair.953%26sid%3Dliteratum%253Aachs%26aulast%3DChawla%26aufirst%3DN.%2BV.%26aulast%3DBowyer%26aufirst%3DK.%2BW.%26aulast%3DHall%26aufirst%3DL.%2BO.%26aulast%3DKegelmeyer%26aufirst%3DW.%2BP.%26atitle%3DSMOTE%253A%2520synthetic%2520minority%2520over-sampling%2520technique%26jtitle%3DJ.%2520Artif.%2520Intell.%2520Res.%26date%3D2002%26spage%3D321%26epage%3D357%26doi%3D10.1613%2Fjair.953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Japkowicz, N.</span><span> </span><span class="NLM_article-title">The class imbalance problem: significance and strategies</span>. Proceedings of the 2000 International Conference on Artificial Intelligence, Las Vegas, Nevada, June 26–29, 2000.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Japkowicz%2C+N.+The+class+imbalance+problem%3A+significance+and+strategies.+Proceedings+of+the+2000+International+Conference+on+Artificial+Intelligence%2C+Las+Vegas%2C+Nevada%2C+June+26%E2%80%9329%2C+2000."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DJapkowicz%26aufirst%3DN.%26atitle%3DThe%2520class%2520imbalance%2520problem%253A%2520significance%2520and%2520strategies%26jtitle%3DProceedings%2520of%2520the%25202000%2520International%2520Conference%2520on%2520Artificial%2520Intelligence%252C%2520Las%2520Vegas%252C%2520Nevada%252C%2520June%252026%25E2%2580%259329%252C%25202000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Exploration of type II binding mode: a privileged approach for kinase inhibitor focused drug discovery?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1230</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span><span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+a+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0lg1199bTkH12g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520a%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The (un)targeted cancer kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">166</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1038/nchembio.297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnchembio.297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=20154661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=166-169&author=O.+Fedorovauthor=S.+M%C3%BCllerauthor=S.+Knapp&title=The+%28un%29targeted+cancer+kinome&doi=10.1038%2Fnchembio.297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The (un)targeted cancer kinome</span></div><div class="casAuthors">Fedorov, Oleg; Mueller, Susanne; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">166-169</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review and discussion.  The complexity of cancer signaling and the resulting difficulties in target selection have strongly biased kinase drug discovery towards clin. validated targets.  Recently, novel kinase targets that are uncharacterized have emerged from genome sequencing and RNAi studies.  Chem. probes are urgently needed to functionally annotate these kinases and to stimulate new drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwcnq0-qjPirVg90H21EOLACvtfcHk0lg1199bTkH12g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D&md5=198ae8d39d40247e17aef5e9de0f2a99</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.297%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520%2528un%2529targeted%2520cancer%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D166%26epage%3D169%26doi%3D10.1038%2Fnchembio.297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, R. J.</span><span> </span><span class="NLM_article-title">Targeting the human kinome for cancer therapy: current perspectives</span> <span class="citation_source-journal">Crit. Rev. Oncog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1615/CritRevOncog.v17.i2.70</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1615%2FCritRevOncog.v17.i2.70" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=22471710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A280%3ADC%252BC38rhvVylsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=233-246&author=L.+Zhangauthor=R.+J.+Daly&title=Targeting+the+human+kinome+for+cancer+therapy%3A+current+perspectives&doi=10.1615%2FCritRevOncog.v17.i2.70"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the human kinome for cancer therapy: current perspectives</span></div><div class="casAuthors">Zhang Luxi; Daly Roger J</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncogenesis</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">233-46</span>
        ISSN:<span class="NLM_cas:issn">0893-9675</span>.
    </div><div class="casAbstract">As highlighted by other articles in this review issue, great progress has been made in the development of targeted treatment modalities directed against particular oncogenic kinases, and the translation of these therapies into the clinic.  However, recent data from cancer genome sequencing projects indicate that the spectrum of kinases that contribute to cancer progression is wider than previously imagined, and as international projects in this area gather momentum, it appears likely that further kinase oncogenes will be identified, and the roles of previously characterized ones will be extended to subsets of additional cancers.  In addition, complementary approaches such as functional genomics and mass spectrometry (MS)-based proteomics are providing important insights into the functional roles played by specific kinases in particular cancers, the dependency of these roles on genetic background, how altered kinase regulation perturbs intracellular signaling networks, and how the latter respond to targeted agents that target the kinase in question.  While other articles in this issue focus on individual cancer-associated kinases and their therapeutic targeting, the aim of this review is to take a broader perspective regarding our current knowledge of the cancer kinome and how this can be expanded and exploited for clinical utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR-aUq9kTcyKn6oPH0mYWL1fW6udTcc2ebHeN5Kk-_kwrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38rhvVylsQ%253D%253D&md5=ad039a3a5d03ea32ee6a4d64c279d82b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1615%2FCritRevOncog.v17.i2.70&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1615%252FCritRevOncog.v17.i2.70%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DDaly%26aufirst%3DR.%2BJ.%26atitle%3DTargeting%2520the%2520human%2520kinome%2520for%2520cancer%2520therapy%253A%2520current%2520perspectives%26jtitle%3DCrit.%2520Rev.%2520Oncog.%26date%3D2012%26volume%3D17%26spage%3D233%26epage%3D246%26doi%3D10.1615%2FCritRevOncog.v17.i2.70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Riniker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fechner, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landrum, G. A.</span><span> </span><span class="NLM_article-title">Heterogeneous classifier fusion for ligand-based virtual screening: or, how decision making by committee can be a good thing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2829</span><span class="NLM_x">–</span> <span class="NLM_lpage">2836</span><span class="refDoi"> DOI: 10.1021/ci400466r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400466r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1yku7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=2829-2836&author=S.+Rinikerauthor=N.+Fechnerauthor=G.+A.+Landrum&title=Heterogeneous+classifier+fusion+for+ligand-based+virtual+screening%3A+or%2C+how+decision+making+by+committee+can+be+a+good+thing&doi=10.1021%2Fci400466r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Heterogeneous Classifier Fusion for Ligand-Based Virtual Screening: Or, How Decision Making by Committee Can Be a Good Thing</span></div><div class="casAuthors">Riniker, Sereina; Fechner, Nikolas; Landrum, Gregory A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2829-2836</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The concept of data fusion - the combination of information from different sources describing the same object with the expectation to generate a more accurate representation - has found application in a very broad range of disciplines.  In the context of ligand-based virtual screening (VS), data fusion has been applied to combine knowledge from either different active mols. or different fingerprints to improve similarity search performance.  Machine-learning (ML) methods based on fusion of multiple homogeneous classifiers, in particular random forests, have also been widely applied in the ML literature.  The heterogeneous version of classifier fusion - fusing the predictions from different model types - has been less explored.  Here, we investigate heterogeneous classifier fusion for ligand-based VS using three different ML methods, RF, naive Bayes (NB), and logistic regression (LR), with four 2D fingerprints, atom pairs, topol. torsions, RDKit fingerprint, and circular fingerprint.  The methods are compared using a previously developed benchmarking platform for 2D fingerprints which is extended to ML methods in this article.  The original data sets are filtered for difficulty, and a new set of challenging data sets from ChEMBL is added.  Data sets were also generated for a second use case: starting from a small set of related actives instead of diverse actives.  The final fused model consistently outperforms the other approaches across the broad variety of targets studied, indicating that heterogeneous classifier fusion is a very promising approach for ligand-based VS.  The new data sets together with the adapted source code for ML methods are provided in the Supporting Information.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1hnwYfGw2FbVg90H21EOLACvtfcHk0lgwHJZ1vLjQew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yku7fJ&md5=99f0bd62b569cd73a2e1539afca7fddb</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci400466r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400466r%26sid%3Dliteratum%253Aachs%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DFechner%26aufirst%3DN.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26atitle%3DHeterogeneous%2520classifier%2520fusion%2520for%2520ligand-based%2520virtual%2520screening%253A%2520or%252C%2520how%2520decision%2520making%2520by%2520committee%2520can%2520be%2520a%2520good%2520thing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D2829%26epage%3D2836%26doi%3D10.1021%2Fci400466r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Riniker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landrum, G. A.</span><span> </span><span class="NLM_article-title">Using information from historical high-throughput screens to predict active compounds</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1880</span><span class="NLM_x">–</span> <span class="NLM_lpage">1891</span><span class="refDoi"> DOI: 10.1021/ci500190p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500190p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslegtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1880-1891&author=S.+Rinikerauthor=Y.+Wangauthor=J.+L.+Jenkinsauthor=G.+A.+Landrum&title=Using+information+from+historical+high-throughput+screens+to+predict+active+compounds&doi=10.1021%2Fci500190p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Using Information from Historical High-Throughput Screens to Predict Active Compounds</span></div><div class="casAuthors">Riniker, Sereina; Wang, Yuan; Jenkins, Jeremy L.; Landrum, Gregory A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1880-1891</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern high-throughput screening (HTS) is a well-established approach for hit finding in drug discovery that is routinely employed in the pharmaceutical industry to screen more than a million compds. within a few weeks.  However, as the industry shifts to more disease-relevant but more complex phenotypic screens, the focus has moved to piloting smaller but smarter chem./biol. diverse subsets followed by an expansion around hit compds.  One std. method for doing this is to train a machine-learning (ML) model with the chem. fingerprints of the tested subset of mols. and then select the next compds. based on the predictions of this model.  An alternative approach would be to take advantage of the wealth of bioactivity information contained in older (full-deck) screens using so-called HTS fingerprints, where each element of the fingerprint corresponds to the outcome of a particular assay, as input to machine-learning algorithms.  We constructed HTS fingerprints using two collections of data: 93 inhouse assays and 95 publicly available assays from PubChem.  For each source, an addnl. set of 51 and 46 assays, resp., was collected for testing.  Three different ML methods, random forest (RF), logistic regression (LR), and naive Bayes (NB), were investigated for both the HTS fingerprint and a chem. fingerprint, Morgan2.  RF was best suited for learning from HTS fingerprints yielding area under the receiver operating characteristic curve (AUC) values >0.8 for 78% of the internal assays and enrichment factors at 5% (EF(5%)) >10 for 55% of the assays.  The RF(HTS-fp) generally outperformed the LR trained with Morgan2, which was the best ML method for the chem. fingerprint, for the majority of assays.  In addn., HTS fingerprints were found to retrieve more diverse chemotypes.  Combining the two models through heterogeneous classifier fusion led to a similar or better performance than the best individual model for all assays.  Further validation using a pair of inhouse assays and data from a confirmatory screen-including a prospective set of around 2000 compds. selected based on our approach-confirmed the good performance.  Thus, the combination of machine-learning with HTS fingerprints and chem. fingerprints utilizes information from both domains and presents a very promising approach for hit expansion, leading to more hits.  The source code used with the public data is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAosVSBEbE57Vg90H21EOLACvtfcHk0lgwHJZ1vLjQew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslegtb4%253D&md5=a2fe1906d259aa91e07918ff6dbd1b3b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fci500190p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500190p%26sid%3Dliteratum%253Aachs%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26atitle%3DUsing%2520information%2520from%2520historical%2520high-throughput%2520screens%2520to%2520predict%2520active%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1880%26epage%3D1891%26doi%3D10.1021%2Fci500190p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urban, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutlina, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fekete, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klumpp, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">A screening pattern recognition method finds new and divergent targets for drugs and natural products</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1622</span><span class="NLM_x">–</span> <span class="NLM_lpage">1631</span><span class="refDoi"> DOI: 10.1021/cb5001839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5001839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1622-1631&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=L.+Urbanauthor=S.+Whitebreadauthor=S.+Chenauthor=K.+Hughesauthor=H.+Guoauthor=E.+Kutlinaauthor=A.+Feketeauthor=M.+Klumppauthor=M.+Glick&title=A+screening+pattern+recognition+method+finds+new+and+divergent+targets+for+drugs+and+natural+products&doi=10.1021%2Fcb5001839"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fcb5001839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5001839%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DUrban%26aufirst%3DL.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DK.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DKutlina%26aufirst%3DE.%26aulast%3DFekete%26aufirst%3DA.%26aulast%3DKlumpp%26aufirst%3DM.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DA%2520screening%2520pattern%2520recognition%2520method%2520finds%2520new%2520and%2520divergent%2520targets%2520for%2520drugs%2520and%2520natural%2520products%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1622%26epage%3D1631%26doi%3D10.1021%2Fcb5001839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Petrone, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simms, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigsch, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutchukian, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">Rethinking molecular similarity: comparing compounds on the basis of biological activity</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1399</span><span class="NLM_x">–</span> <span class="NLM_lpage">1409</span><span class="refDoi"> DOI: 10.1021/cb3001028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3001028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1399-1409&author=P.+M.+Petroneauthor=B.+Simmsauthor=F.+Nigschauthor=E.+Lounkineauthor=P.+Kutchukianauthor=A.+Cornettauthor=Z.+Dengauthor=J.+W.+Daviesauthor=J.+L.+Jenkinsauthor=M.+Glick&title=Rethinking+molecular+similarity%3A+comparing+compounds+on+the+basis+of+biological+activity&doi=10.1021%2Fcb3001028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking Molecular Similarity: Comparing Compounds on the Basis of Biological Activity</span></div><div class="casAuthors">Petrone, Paula M.; Simms, Benjamin; Nigsch, Florian; Lounkine, Eugen; Kutchukian, Peter; Cornett, Allen; Deng, Zhan; Davies, John W.; Jenkins, Jeremy L.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1399-1409</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since the advent of high-throughput screening (HTS), there has been an urgent need for methods that facilitate the interrogation of large-scale chem. biol. data to build a mode of action (MoA) hypothesis.  This can be done either prior to the HTS by subset design of compds. with known MoA or post HTS by data annotation and mining.  To enable this process, we developed a tool that compares compds. solely on the basis of their bioactivity: the chem. biol. descriptor high-throughput screening fingerprint (HTS-FP).  In the current embodiment, data are aggregated from 195 biochem. and cell-based assays developed at Novartis and can be used to identify bioactivity relationships among the inhouse collection comprising ∼1.5 million compds.  We demonstrate the value of the HTS-FP for virtual screening and in particular scaffold hopping.  HTS-FP outperforms state of the art methods in several aspects, retrieving bioactive compds. with remarkable chem. dissimilarity to a probe structure.  We also apply HTS-FP for the design of screening subsets in HTS.  Using retrospective data, we show that a biodiverse selection of plates performs significantly better than a chem. diverse selection of plates, both in terms of no. of hits and diversity of chemotypes retrieved.  This is also true in the case of hit expansion predictions using HTS-FP similarity.  Sets of compds. clustered with HTS-FP are biol. meaningful, in the sense that these clusters enrich for genes and gene ontol. (GO) terms, showing that compds. that are bioactively similar also tend to target proteins that operate together in the cell.  HTS-FP are valuable not only because of their predictive power but mainly because they relate compds. solely on the basis of bioactivity, harnessing the accumulated knowledge of a high-throughput screening facility toward the understanding of how compds. interact with the proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFjchAxW_TSLVg90H21EOLACvtfcHk0lhMDMNJVwmalw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D&md5=66ecd494f9200a0fdc1c142402ff255f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fcb3001028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3001028%26sid%3Dliteratum%253Aachs%26aulast%3DPetrone%26aufirst%3DP.%2BM.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKutchukian%26aufirst%3DP.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DRethinking%2520molecular%2520similarity%253A%2520comparing%2520compounds%2520on%2520the%2520basis%2520of%2520biological%2520activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1399%26epage%3D1409%26doi%3D10.1021%2Fcb3001028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Cortes Cabrera, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucena-Agell, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redondo-Horcajo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barasoain, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fasching, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrone, P.</span><span> </span><span class="NLM_article-title">Compound biological signatures facilitate phenotypic screening and target elucidation</span> <span class="citation_source-journal">bioRxiv</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_fpage">041947</span><span class="refDoi"> DOI: 10.1101/041947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1101%2F041947" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=041947&author=A.+Cortes+Cabreraauthor=D.+Lucena-Agellauthor=M.+Redondo-Horcajoauthor=I.+Barasoainauthor=F.+Diazauthor=B.+Faschingauthor=P.+Petrone&title=Compound+biological+signatures+facilitate+phenotypic+screening+and+target+elucidation&doi=10.1101%2F041947"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1101%2F041947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F041947%26sid%3Dliteratum%253Aachs%26aulast%3DCortes%2BCabrera%26aufirst%3DA.%26aulast%3DLucena-Agell%26aufirst%3DD.%26aulast%3DRedondo-Horcajo%26aufirst%3DM.%26aulast%3DBarasoain%26aufirst%3DI.%26aulast%3DDiaz%26aufirst%3DF.%26aulast%3DFasching%26aufirst%3DB.%26aulast%3DPetrone%26aufirst%3DP.%26atitle%3DCompound%2520biological%2520signatures%2520facilitate%2520phenotypic%2520screening%2520and%2520target%2520elucidation%26jtitle%3DbioRxiv%26date%3D2016%26spage%3D041947%26doi%3D10.1101%2F041947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lhMDMNJVwmalw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Yagnik, J.; Strelow, D.; Ross, D. A.; Lin, R.-s.</span><span> </span><span class="NLM_article-title">The power of comparative reasoning</span>. International Conference on Computer Vision (IEEE), Barcelona, Spain, November 6–13, 2011; pp  <span class="NLM_fpage">2431</span><span class="NLM_x">–</span> <span class="NLM_lpage">2438</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Yagnik%2C+J.%3B+Strelow%2C+D.%3B+Ross%2C+D.+A.%3B+Lin%2C+R.-s.+The+power+of+comparative+reasoning.+International+Conference+on+Computer+Vision+%28IEEE%29%2C+Barcelona%2C+Spain%2C+November+6%E2%80%9313%2C+2011%3B+pp+2431%E2%80%932438."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DYagnik%26aufirst%3DJ.%26atitle%3DThe%2520power%2520of%2520comparative%2520reasoning%26jtitle%3DInternational%2520Conference%2520on%2520Computer%2520Vision%2520%2528IEEE%2529%252C%2520Barcelona%252C%2520Spain%252C%2520November%25206%25E2%2580%259313%252C%25202011%26spage%3D2431%26epage%3D2438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Bora, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciucanu, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raica, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avram, S.</span><span> </span><span class="NLM_article-title">Predictive models for fast and effective profiling of kinase inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">895</span><span class="NLM_x">–</span> <span class="NLM_lpage">905</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00646</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00646" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlslyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=895-905&author=A.+Boraauthor=S.+Avramauthor=I.+Ciucanuauthor=M.+Raicaauthor=S.+Avram&title=Predictive+models+for+fast+and+effective+profiling+of+kinase+inhibitors&doi=10.1021%2Facs.jcim.5b00646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive Models for Fast and Effective Profiling of Kinase Inhibitors</span></div><div class="casAuthors">Bora, Alina; Avram, Sorin; Ciucanu, Ionel; Raica, Marius; Avram, Stefana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">895-905</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study we developed two-dimensional pharmacophore-based random forest models for the effective profiling of kinase inhibitors.  One hundred seven prediction models were developed to address distinct kinases spanning over all kinase groups.  Rigorous external validation demonstrates excellent virtual screening and classification potential of the predictors and, more importantly, the capacity to prioritize novel chem. scaffolds in large chem. libraries.  The models built upon more diverse and more potent compds. tend to exert the highest predictive power.  The anal. of ColBioS-FlavRC (Collection of Bioselective Flavonoids and Related Compds.) highlighted several potentially promiscuous derivs. with undesirable selectivity against kinases.  The prediction models can be downloaded from www.chembioinf.ro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTZSs1GEkYHLVg90H21EOLACvtfcHk0lhMDMNJVwmalw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlslyqtr4%253D&md5=cd8d17ea9616165b115dd9d75352e0ac</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00646%26sid%3Dliteratum%253Aachs%26aulast%3DBora%26aufirst%3DA.%26aulast%3DAvram%26aufirst%3DS.%26aulast%3DCiucanu%26aufirst%3DI.%26aulast%3DRaica%26aufirst%3DM.%26aulast%3DAvram%26aufirst%3DS.%26atitle%3DPredictive%2520models%2520for%2520fast%2520and%2520effective%2520profiling%2520of%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D895%26epage%3D905%26doi%3D10.1021%2Facs.jcim.5b00646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lélias, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0ljwwADBa_VT5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">McHugh, M. L.</span><span> </span><span class="NLM_article-title">Interrater reliability: the kappa statistic</span> <span class="citation_source-journal">Biochem. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">276</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span><span class="refDoi"> DOI: 10.11613/BM.2012.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.11613%2FBM.2012.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A280%3ADC%252BC3s%252FnsFCqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=276-282&author=M.+L.+McHugh&title=Interrater+reliability%3A+the+kappa+statistic&doi=10.11613%2FBM.2012.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Interrater reliability: the kappa statistic</span></div><div class="casAuthors">McHugh Mary L</div><div class="citationInfo"><span class="NLM_cas:title">Biochemia medica</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">276-82</span>
        ISSN:<span class="NLM_cas:issn">1330-0962</span>.
    </div><div class="casAbstract">The kappa statistic is frequently used to test interrater reliability.  The importance of rater reliability lies in the fact that it represents the extent to which the data collected in the study are correct representations of the variables measured.  Measurement of the extent to which data collectors (raters) assign the same score to the same variable is called interrater reliability.  While there have been a variety of methods to measure interrater reliability, traditionally it was measured as percent agreement, calculated as the number of agreement scores divided by the total number of scores.  In 1960, Jacob Cohen critiqued use of percent agreement due to its inability to account for chance agreement.  He introduced the Cohen's kappa, developed to account for the possibility that raters actually guess on at least some variables due to uncertainty.  Like most correlation statistics, the kappa can range from -1 to +1.  While the kappa is one of the most commonly used statistics to test interrater reliability, it has limitations.  Judgments about what level of kappa should be acceptable for health research are questioned.  Cohen's suggested interpretation may be too lenient for health related studies because it implies that a score as low as 0.41 might be acceptable.  Kappa and percent agreement are compared, and levels for both kappa and percent agreement that should be demanded in healthcare studies are suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS8oUvZdI4KVzdblLE_E9PWfW6udTcc2eZTsNBogQ_C8Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s%252FnsFCqtQ%253D%253D&md5=ba7a87fe0088906722bf15282bfa1d2a</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.11613%2FBM.2012.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.11613%252FBM.2012.031%26sid%3Dliteratum%253Aachs%26aulast%3DMcHugh%26aufirst%3DM.%2BL.%26atitle%3DInterrater%2520reliability%253A%2520the%2520kappa%2520statistic%26jtitle%3DBiochem.%2520Med.%26date%3D2012%26volume%3D22%26spage%3D276%26epage%3D282%26doi%3D10.11613%2FBM.2012.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liaw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahl, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svetnik, V.</span><span> </span><span class="NLM_article-title">Deep neural nets as a method for quantitative structure-activity relationships</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">274</span><span class="refDoi"> DOI: 10.1021/ci500747n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500747n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=263-274&author=J.+Maauthor=R.+P.+Sheridanauthor=A.+Liawauthor=G.+E.+Dahlauthor=V.+Svetnik&title=Deep+neural+nets+as+a+method+for+quantitative+structure-activity+relationships&doi=10.1021%2Fci500747n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Deep Neural Nets as a Method for Quantitative Structure-Activity Relationships</span></div><div class="casAuthors">Ma, Junshui; Sheridan, Robert P.; Liaw, Andy; Dahl, George E.; Svetnik, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-274</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Neural networks were widely used for quant. structure-activity relationships (QSAR) in the 1990s.  Because of various practical issues (e.g., slow on large problems, difficult to train, prone to overfitting, etc.), they were superseded by more robust methods like support vector machine (SVM) and random forest (RF), which arose in the early 2000s.  The last 10 years has witnessed a revival of neural networks in the machine learning community thanks to new methods for preventing overfitting, more efficient training algorithms, and advancements in computer hardware.  In particular, deep neural nets (DNNs), i.e. neural nets with more than one hidden layer, have found great successes in many applications, such as computer vision and natural language processing.  Here we show that DNNs can routinely make better prospective predictions than RF on a set of large diverse QSAR data sets that are taken from Merck's drug discovery effort.  The no. of adjustable parameters needed for DNNs is fairly large, but our results show that it is not necessary to optimize them for individual data sets, and a single set of recommended parameters can achieve better performance than RF for most of the data sets we studied.  The usefulness of the parameters is demonstrated on addnl. data sets not used in the calibration.  Although training DNNs is still computationally intensive, using graphical processing units (GPUs) can make this issue manageable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwVMQ-gQwZd7Vg90H21EOLACvtfcHk0lgnxcfFpmy8gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGns70%253D&md5=2c6a3f33f85a4abc32ebc33108426667</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci500747n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500747n%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DDahl%26aufirst%3DG.%2BE.%26aulast%3DSvetnik%26aufirst%3DV.%26atitle%3DDeep%2520neural%2520nets%2520as%2520a%2520method%2520for%2520quantitative%2520structure-activity%2520relationships%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D263%26epage%3D274%26doi%3D10.1021%2Fci500747n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eid, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fulle, S.</span><span> </span><span class="NLM_article-title">Identification and visualization of kinase-specific subpockets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">335</span><span class="NLM_x">–</span> <span class="NLM_lpage">346</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00627</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00627" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC28Xis1OgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=335-346&author=A.+Volkamerauthor=S.+Eidauthor=S.+Turkauthor=F.+Rippmannauthor=S.+Fulle&title=Identification+and+visualization+of+kinase-specific+subpockets&doi=10.1021%2Facs.jcim.5b00627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Visualization of Kinase-Specific Subpockets</span></div><div class="casAuthors">Volkamer, Andrea; Eid, Sameh; Turk, Samo; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">335-346</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification and design of selective compds. is important for the redn. of unwanted side effects as well as for the development of tool compds. for target validation studies.  This is, in particular, true for therapeutically important protein families that possess conserved folds and have numerous members such as kinases.  To support the design of selective kinase inhibitors, we developed a novel approach that allows identification of specificity detg. subpockets between closely related kinases solely based on their three-dimensional structures.  To account for the intrinsic flexibility of the proteins, multiple X-ray structures of the target protein of interest as well as of unwanted off-target(s) are taken into account.  The binding pockets of these protein structures are calcd. and fused to a combined target and off-target pocket, resp.  Subsequently, shape differences between these two combined pockets are identified via fusion rules.  The approach provides a user-friendly visualization of target-specific areas in a binding pocket which should be explored when designing selective compds.  Furthermore, the approach can be easily combined with in silico alanine mutation studies to identify selectivity detg. residues.  The potential impact of the approach is demonstrated in four retrospective expts. on closely related kinases, i.e., p38α vs Erk2, PAK1 vs PAK4, ITK vs AurA, and BRAF vs VEGFR2.  Overall, the presented approach does not require any profiling data for training purposes, provides an intuitive visualization of a large no. of protein structures at once, and could also be applied to other target classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlaQhBsAUeNbVg90H21EOLACvtfcHk0lgnxcfFpmy8gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xis1OgtQ%253D%253D&md5=c9eec4cca15085669921cc16f3c121fb</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00627%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DEid%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DIdentification%2520and%2520visualization%2520of%2520kinase-specific%2520subpockets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D335%26epage%3D346%26doi%3D10.1021%2Facs.jcim.5b00627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 51 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Lewis H. Mervin, Avid M. Afzal, Ola Engkvist, <span class="NLM_string-name hlFld-ContribAuthor">Andreas Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of Scaling Methods to Obtain Calibrated Probabilities of Activity for Protein–Ligand Predictions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 4546-4559. <a href="https://doi.org/10.1021/acs.jcim.0c00476" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00476</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00476%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComparison%252Bof%252BScaling%252BMethods%252Bto%252BObtain%252BCalibrated%252BProbabilities%252Bof%252BActivity%252Bfor%252BProtein%2525E2%252580%252593Ligand%252BPredictions%26aulast%3DMervin%26aufirst%3DLewis%2BH.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05052020%26date%3D21092020%26date%3D31082020%26volume%3D60%26issue%3D10%26spage%3D4546%26epage%3D4559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xutong Li, Zhaojun Li, Xiaolong Wu, Zhaoping Xiong, Tianbiao Yang, Zunyun Fu, Xiaohong Liu, Xiaoqin Tan, Feisheng Zhong, Xiaozhe Wan, Dingyan Wang, Xiaoyu Ding, Ruirui Yang, Hui Hou, Chunpu Li, Hong Liu, Kaixian Chen, Hualiang Jiang, <span class="NLM_string-name hlFld-ContribAuthor">Mingyue Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Deep Learning Enhancing Kinome-Wide Polypharmacology Profiling: Model Construction and Experiment Validation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8723-8737. <a href="https://doi.org/10.1021/acs.jmedchem.9b00855" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00855</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00855%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDeep%252BLearning%252BEnhancing%252BKinome-Wide%252BPolypharmacology%252BProfiling%25253A%252BModel%252BConstruction%252Band%252BExperiment%252BValidation%26aulast%3DLi%26aufirst%3DXutong%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D26052019%26date%3D15082019%26date%3D31072019%26volume%3D63%26issue%3D16%26spage%3D8723%26epage%3D8737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Filip Miljković, Raquel Rodríguez-Pérez, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Machine Learning Models for Accurate Prediction of Kinase Inhibitors with Different Binding Modes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8738-8748. <a href="https://doi.org/10.1021/acs.jmedchem.9b00867" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00867</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00867%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMachine%252BLearning%252BModels%252Bfor%252BAccurate%252BPrediction%252Bof%252BKinase%252BInhibitors%252Bwith%252BDifferent%252BBinding%252BModes%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DFilip%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D29052019%26date%3D30082019%26volume%3D63%26issue%3D16%26spage%3D8738%26epage%3D8748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Minjian Yang, Bingzhong Tao, Chengjuan Chen, Wenqiang Jia, Shaolei Sun, Tiantai Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Xiaojian Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Machine Learning Models Based on Molecular Fingerprints and an Extreme Gradient Boosting Method Lead to the Discovery of JAK2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (12)
                                     , 5002-5012. <a href="https://doi.org/10.1021/acs.jcim.9b00798" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00798</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00798%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMachine%252BLearning%252BModels%252BBased%252Bon%252BMolecular%252BFingerprints%252Band%252Ban%252BExtreme%252BGradient%252BBoosting%252BMethod%252BLead%252Bto%252Bthe%252BDiscovery%252Bof%252BJAK2%252BInhibitors%26aulast%3DYang%26aufirst%3DMinjian%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15092019%26date%3D04122019%26date%3D20112019%26volume%3D59%26issue%3D12%26spage%3D5002%26epage%3D5012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ruimin Ma, Zeyu Liu, Quanwei Zhang, Zhiyu Liu, <span class="NLM_string-name hlFld-ContribAuthor">Tengfei Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluating Polymer Representations via Quantifying Structure–Property Relationships. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (7)
                                     , 3110-3119. <a href="https://doi.org/10.1021/acs.jcim.9b00358" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00358</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00358%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DEvaluating%252BPolymer%252BRepresentations%252Bvia%252BQuantifying%252BStructure%2525E2%252580%252593Property%252BRelationships%26aulast%3DMa%26aufirst%3DRuimin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D29042019%26date%3D03072019%26date%3D25062019%26volume%3D59%26issue%3D7%26spage%3D3110%26epage%3D3119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Antonius
P. A. Janssen, Sebastian H. Grimm, Ruud H. M. Wijdeven, Eelke B. Lenselink, Jacques Neefjes, Constant A. A. van Boeckel, Gerard J. P. van Westen, <span class="NLM_string-name hlFld-ContribAuthor">Mario van der Stelt</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Discovery Maps, a Machine Learning Model That Visualizes and Predicts Kinome–Inhibitor Interaction Landscapes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1221-1229. <a href="https://doi.org/10.1021/acs.jcim.8b00640" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00640</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00640%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDrug%252BDiscovery%252BMaps%25252C%252Ba%252BMachine%252BLearning%252BModel%252BThat%252BVisualizes%252Band%252BPredicts%252BKinome%2525E2%252580%252593Inhibitor%252BInteraction%252BLandscapes%26aulast%3DJanssen%26aufirst%3DAntonius%2BP.%2BA.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D18092018%26date%3D08112018%26date%3D29102018%26volume%3D59%26issue%3D3%26spage%3D1221%26epage%3D1229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">T. M. Whitehead, B. W. J. Irwin, P. Hunt, M. D. Segall, <span class="NLM_string-name hlFld-ContribAuthor">G. J. Conduit</span>. </span><span class="cited-content_cbyCitation_article-title">Imputation of Assay Bioactivity Data Using Deep Learning. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1197-1204. <a href="https://doi.org/10.1021/acs.jcim.8b00768" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00768</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00768%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImputation%252Bof%252BAssay%252BBioactivity%252BData%252BUsing%252BDeep%252BLearning%26aulast%3DWhitehead%26aufirst%3DT.%2BM.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D31102018%26date%3D21022019%26date%3D12022019%26volume%3D59%26issue%3D3%26spage%3D1197%26epage%3D1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Scott D. Bembenek, Gavin Hirst, <span class="NLM_string-name hlFld-ContribAuthor">Taraneh Mirzadegan</span>. </span><span class="cited-content_cbyCitation_article-title">Determination of a Focused Mini Kinase Panel for Early Identification of Selective Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (7)
                                     , 1434-1440. <a href="https://doi.org/10.1021/acs.jcim.8b00222" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00222</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00222%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDetermination%252Bof%252Ba%252BFocused%252BMini%252BKinase%252BPanel%252Bfor%252BEarly%252BIdentification%252Bof%252BSelective%252BKinase%252BInhibitors%26aulast%3DBembenek%26aufirst%3DScott%2BD.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D12042018%26date%3D22062018%26date%3D24052018%26volume%3D58%26issue%3D7%26spage%3D1434%26epage%3D1440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Samo Turk, Benjamin Merget, Sameh Eid, <span class="NLM_string-name hlFld-ContribAuthor">Simone Fulle</span>. </span><span class="cited-content_cbyCitation_article-title">From Cancer to Pain Target by Automated Selectivity Inversion of a Clinical Candidate. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (11)
                                     , 4851-4859. <a href="https://doi.org/10.1021/acs.jmedchem.8b00140" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00140%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFrom%252BCancer%252Bto%252BPain%252BTarget%252Bby%252BAutomated%252BSelectivity%252BInversion%252Bof%252Ba%252BClinical%252BCandidate%26aulast%3DTurk%26aufirst%3DSamo%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D27012018%26date%3D24052018%26date%3D10052018%26volume%3D61%26issue%3D11%26spage%3D4851%26epage%3D4859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sorin Avram, Alina Bora, Liliana Halip, <span class="NLM_string-name hlFld-ContribAuthor">Ramona Curpăn</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling Kinase Inhibition Using Highly Confident Data Sets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 957-967. <a href="https://doi.org/10.1021/acs.jcim.7b00729" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00729</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00729%26sid%3Dliteratum%253Aachs%26aulast%3DBora%26aufirst%3DA.%26atitle%3DPredictive%2520Models%2520for%2520Fast%2520and%2520Effective%2520Profiling%2520of%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D895%26epage%3D905%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DModeling%252BKinase%252BInhibition%252BUsing%252BHighly%252BConfident%252BData%252BSets%26aulast%3DAvram%26aufirst%3DSorin%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D20122017%26date%3D09052018%26date%3D30042018%26volume%3D58%26issue%3D5%26spage%3D957%26epage%3D967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sabrina  Jaeger</span>, <span class="hlFld-ContribAuthor ">Simone  Fulle</span>, and <span class="hlFld-ContribAuthor ">Samo  Turk</span>  . </span><span class="cited-content_cbyCitation_article-title">Mol2vec: Unsupervised Machine Learning Approach with Chemical Intuition. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (1)
                                     , 27-35. <a href="https://doi.org/10.1021/acs.jcim.7b00616" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00616</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00616&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00616%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMol2vec%25253A%252BUnsupervised%252BMachine%252BLearning%252BApproach%252Bwith%252BChemical%252BIntuition%26aulast%3DJaeger%26aufirst%3DSabrina%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D20102017%26date%3D10012018%26date%3D22012018%26date%3D22122017%26volume%3D58%26issue%3D1%26spage%3D27%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samo  Turk</span>, <span class="hlFld-ContribAuthor ">Benjamin  Merget</span>, <span class="hlFld-ContribAuthor ">Friedrich  Rippmann</span>, and <span class="hlFld-ContribAuthor ">Simone  Fulle</span>  . </span><span class="cited-content_cbyCitation_article-title">Coupling Matched Molecular Pairs with Machine Learning for Virtual Compound Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (12)
                                     , 3079-3085. <a href="https://doi.org/10.1021/acs.jcim.7b00298" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00298%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCoupling%252BMatched%252BMolecular%252BPairs%252Bwith%252BMachine%252BLearning%252Bfor%252BVirtual%252BCompound%252BOptimization%26aulast%3DTurk%26aufirst%3DSamo%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D23052017%26date%3D27112017%26date%3D26122017%26date%3D13112017%26volume%3D57%26issue%3D12%26spage%3D3079%26epage%3D3085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric J.  Martin</span>, <span class="hlFld-ContribAuthor ">Valery R.  Polyakov</span>, <span class="hlFld-ContribAuthor ">Li  Tian</span>, and <span class="hlFld-ContribAuthor ">Rolando C.  Perez</span>  . </span><span class="cited-content_cbyCitation_article-title">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (8)
                                     , 2077-2088. <a href="https://doi.org/10.1021/acs.jcim.7b00166" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00166</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00166%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProfile-QSAR%252B2.0%25253A%252BKinase%252BVirtual%252BScreening%252BAccuracy%252BComparable%252Bto%252BFour-Concentration%252BIC50s%252Bfor%252BRealistically%252BNovel%252BCompounds%26aulast%3DMartin%26aufirst%3DEric%2BJ.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D22032017%26date%3D26072017%26date%3D28082017%26date%3D26062017%26volume%3D57%26issue%3D8%26spage%3D2077%26epage%3D2088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tongxing  Wang</span>, <span class="hlFld-ContribAuthor ">Lu  Liang</span>, <span class="hlFld-ContribAuthor ">Chunlai  Zhao</span>, <span class="hlFld-ContribAuthor ">Jia  Sun</span>, <span class="hlFld-ContribAuthor ">Hairong  Wang</span>, <span class="hlFld-ContribAuthor ">Wenjia  Wang</span>, <span class="hlFld-ContribAuthor ">Jianping  Lin</span>, <span class="hlFld-ContribAuthor ">Yunhui  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Elucidating direct kinase targets of compound Danshen dropping pills employing archived data and prediction models. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-021-89035-4" title="DOI URL">https://doi.org/10.1038/s41598-021-89035-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-021-89035-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-021-89035-4%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DElucidating%252Bdirect%252Bkinase%252Btargets%252Bof%252Bcompound%252BDanshen%252Bdropping%252Bpills%252Bemploying%252Barchived%252Bdata%252Band%252Bprediction%252Bmodels%26aulast%3DWang%26aufirst%3DTongxing%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabio  Urbina</span>, <span class="hlFld-ContribAuthor ">Ana C.  Puhl</span>, <span class="hlFld-ContribAuthor ">Sean  Ekins</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in drug repurposing using machine learning. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>65 </em>, 74-84. <a href="https://doi.org/10.1016/j.cbpa.2021.06.001" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.06.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.06.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DRecent%252Badvances%252Bin%252Bdrug%252Brepurposing%252Busing%252Bmachine%252Blearning%26aulast%3DUrbina%26aufirst%3DFabio%26date%3D2021%26volume%3D65%26spage%3D74%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhaoping  Xiong</span>, <span class="hlFld-ContribAuthor ">Ziqiang  Cheng</span>, <span class="hlFld-ContribAuthor ">Xinyuan  Lin</span>, <span class="hlFld-ContribAuthor ">Chi  Xu</span>, <span class="hlFld-ContribAuthor ">Xiaohong  Liu</span>, <span class="hlFld-ContribAuthor ">Dingyan  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Yong  Zhang</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Nan  Qiao</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Facing small and biased data dilemma in drug discovery with enhanced federated learning approaches. </span><span class="cited-content_cbyCitation_journal-name">Science China Life Sciences</span><span> <strong>2021,</strong> <em>46 </em><a href="https://doi.org/10.1007/s11427-021-1946-0" title="DOI URL">https://doi.org/10.1007/s11427-021-1946-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11427-021-1946-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11427-021-1946-0%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520China%2520Life%2520Sciences%26atitle%3DFacing%252Bsmall%252Band%252Bbiased%252Bdata%252Bdilemma%252Bin%252Bdrug%252Bdiscovery%252Bwith%252Benhanced%252Bfederated%252Blearning%252Bapproaches%26aulast%3DXiong%26aufirst%3DZhaoping%26date%3D2021%26date%3D2021%26volume%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Minimal screening requirements for identifying highly promiscuous kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (13)
                                     , 1083-1085. <a href="https://doi.org/10.4155/fmc-2021-0077" title="DOI URL">https://doi.org/10.4155/fmc-2021-0077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0077%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DMinimal%252Bscreening%252Brequirements%252Bfor%252Bidentifying%252Bhighly%252Bpromiscuous%252Bkinase%252Binhibitors%26aulast%3DBajorath%26aufirst%3DJ%25C3%25BCrgen%26date%3D2021%26volume%3D13%26issue%3D13%26spage%3D1083%26epage%3D1085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Riya  Datta</span>, <span class="hlFld-ContribAuthor ">Dibyendu  Das</span>, <span class="hlFld-ContribAuthor ">Srinjoy  Das</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient lipophilicity prediction of molecules employing deep-learning models. </span><span class="cited-content_cbyCitation_journal-name">Chemometrics and Intelligent Laboratory Systems</span><span> <strong>2021,</strong> <em>213 </em>, 104309. <a href="https://doi.org/10.1016/j.chemolab.2021.104309" title="DOI URL">https://doi.org/10.1016/j.chemolab.2021.104309</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chemolab.2021.104309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chemolab.2021.104309%26sid%3Dliteratum%253Aachs%26jtitle%3DChemometrics%2520and%2520Intelligent%2520Laboratory%2520Systems%26atitle%3DEfficient%252Blipophilicity%252Bprediction%252Bof%252Bmolecules%252Bemploying%252Bdeep-learning%252Bmodels%26aulast%3DDatta%26aufirst%3DRiya%26date%3D2021%26volume%3D213%26spage%3D104309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Albert A.  Antolin</span>, <span class="hlFld-ContribAuthor ">Paul A.  Clarke</span>, <span class="hlFld-ContribAuthor ">Ian  Collins</span>, <span class="hlFld-ContribAuthor ">Paul  Workman</span>, <span class="hlFld-ContribAuthor ">Bissan  Al-Lazikani</span>. </span><span class="cited-content_cbyCitation_article-title">Evolution of kinase polypharmacology across HSP90 drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>5 </em><a href="https://doi.org/10.1016/j.chembiol.2021.05.004" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.05.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.05.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DEvolution%252Bof%252Bkinase%252Bpolypharmacology%252Bacross%252BHSP90%252Bdrug%252Bdiscovery%26aulast%3DAntolin%26aufirst%3DAlbert%2BA.%26date%3D2021%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rohan  Gupta</span>, <span class="hlFld-ContribAuthor ">Devesh  Srivastava</span>, <span class="hlFld-ContribAuthor ">Mehar  Sahu</span>, <span class="hlFld-ContribAuthor ">Swati  Tiwari</span>, <span class="hlFld-ContribAuthor ">Rashmi K.  Ambasta</span>, <span class="hlFld-ContribAuthor ">Pravir  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Artificial intelligence to deep learning: machine intelligence approach for drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2021,</strong> <em>5 </em><a href="https://doi.org/10.1007/s11030-021-10217-3" title="DOI URL">https://doi.org/10.1007/s11030-021-10217-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-021-10217-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-021-10217-3%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DArtificial%252Bintelligence%252Bto%252Bdeep%252Blearning%25253A%252Bmachine%252Bintelligence%252Bapproach%252Bfor%252Bdrug%252Bdiscovery%26aulast%3DGupta%26aufirst%3DRohan%26date%3D2021%26date%3D2021%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José  Jiménez-Luna</span>, <span class="hlFld-ContribAuthor ">Francesca  Grisoni</span>, <span class="hlFld-ContribAuthor ">Nils  Weskamp</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Artificial intelligence in drug discovery: recent advances and future perspectives. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>3 </em>, 1-11. <a href="https://doi.org/10.1080/17460441.2021.1909567" title="DOI URL">https://doi.org/10.1080/17460441.2021.1909567</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1909567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1909567%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DArtificial%252Bintelligence%252Bin%252Bdrug%252Bdiscovery%25253A%252Brecent%252Badvances%252Band%252Bfuture%252Bperspectives%26aulast%3DJim%25C3%25A9nez-Luna%26aufirst%3DJos%25C3%25A9%26date%3D2021%26date%3D2021%26volume%3D3%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jamal  Shamsara</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of the performance of various machine learning methods on the discrimination of the active compounds. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2021,</strong> <em>97 </em>
                                    (4)
                                     , 930-943. <a href="https://doi.org/10.1111/cbdd.13819" title="DOI URL">https://doi.org/10.1111/cbdd.13819</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13819%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DEvaluation%252Bof%252Bthe%252Bperformance%252Bof%252Bvarious%252Bmachine%252Blearning%252Bmethods%252Bon%252Bthe%252Bdiscrimination%252Bof%252Bthe%252Bactive%252Bcompounds%26aulast%3DShamsara%26aufirst%3DJamal%26date%3D2021%26date%3D2021%26volume%3D97%26issue%3D4%26spage%3D930%26epage%3D943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cong  Shen</span>, <span class="hlFld-ContribAuthor ">Jiawei  Luo</span>, <span class="hlFld-ContribAuthor ">Wenjue  Ouyang</span>, <span class="hlFld-ContribAuthor ">Pingjian  Ding</span>, <span class="hlFld-ContribAuthor ">Xiangtao  Chen</span>, . </span><span class="cited-content_cbyCitation_article-title">IDDkin: network-based influence deep diffusion model for enhancing prediction of kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2021,</strong> <em>36 </em>
                                    (22-23)
                                     , 5481-5491. <a href="https://doi.org/10.1093/bioinformatics/btaa1058" title="DOI URL">https://doi.org/10.1093/bioinformatics/btaa1058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btaa1058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtaa1058%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DIDDkin%25253A%252Bnetwork-based%252Binfluence%252Bdeep%252Bdiffusion%252Bmodel%252Bfor%252Benhancing%252Bprediction%252Bof%252Bkinase%252Binhibitors%26aulast%3DShen%26aufirst%3DCong%26date%3D2021%26date%3D2020%26volume%3D36%26issue%3D22-23%26spage%3D5481%26epage%3D5491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hammad  Naveed</span>, <span class="hlFld-ContribAuthor ">Corinna  Reglin</span>, <span class="hlFld-ContribAuthor ">Thomas  Schubert</span>, <span class="hlFld-ContribAuthor ">Xin  Gao</span>, <span class="hlFld-ContribAuthor ">Stefan T.  Arold</span>, <span class="hlFld-ContribAuthor ">Michael L.  Maitland</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying Novel Drug Targets by iDTPnd: A Case Study of Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Genomics, Proteomics & Bioinformatics</span><span> <strong>2021,</strong> <em>101 </em><a href="https://doi.org/10.1016/j.gpb.2020.05.006" title="DOI URL">https://doi.org/10.1016/j.gpb.2020.05.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.gpb.2020.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.gpb.2020.05.006%26sid%3Dliteratum%253Aachs%26jtitle%3DGenomics%252C%2520Proteomics%2520%2526%2520Bioinformatics%26atitle%3DIdentifying%252BNovel%252BDrug%252BTargets%252Bby%252BiDTPnd%25253A%252BA%252BCase%252BStudy%252Bof%252BKinase%252BInhibitors%26aulast%3DNaveed%26aufirst%3DHammad%26date%3D2021%26volume%3D101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Srinivas Niranj  Chandrasekaran</span>, <span class="hlFld-ContribAuthor ">Hugo  Ceulemans</span>, <span class="hlFld-ContribAuthor ">Justin D.  Boyd</span>, <span class="hlFld-ContribAuthor ">Anne E.  Carpenter</span>. </span><span class="cited-content_cbyCitation_article-title">Image-based profiling for drug discovery: due for a machine-learning upgrade?. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (2)
                                     , 145-159. <a href="https://doi.org/10.1038/s41573-020-00117-w" title="DOI URL">https://doi.org/10.1038/s41573-020-00117-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-020-00117-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-020-00117-w%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DImage-based%252Bprofiling%252Bfor%252Bdrug%252Bdiscovery%25253A%252Bdue%252Bfor%252Ba%252Bmachine-learning%252Bupgrade%25253F%26aulast%3DChandrasekaran%26aufirst%3DSrinivas%2BNiranj%26date%3D2021%26date%3D2020%26volume%3D20%26issue%3D2%26spage%3D145%26epage%3D159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bryce K.  Allen</span>, <span class="hlFld-ContribAuthor ">Nagi G.  Ayad</span>, <span class="hlFld-ContribAuthor ">Stephan  Schürer</span>. </span><span class="cited-content_cbyCitation_article-title">Kinome-Wide Activity Classification of Small Molecules by Deep Learning. </span><span class="cited-content_cbyCitation_journal-name">SSRN Electronic Journal</span><span> <strong>2021,</strong> <em>17 </em><a href="https://doi.org/10.2139/ssrn.3879088" title="DOI URL">https://doi.org/10.2139/ssrn.3879088</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2139/ssrn.3879088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2139%2Fssrn.3879088%26sid%3Dliteratum%253Aachs%26jtitle%3DSSRN%2520Electronic%2520Journal%26atitle%3DKinome-Wide%252BActivity%252BClassification%252Bof%252BSmall%252BMolecules%252Bby%252BDeep%252BLearning%26aulast%3DAllen%26aufirst%3DBryce%2BK.%26date%3D2021%26volume%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">C.  Škuta</span>, <span class="hlFld-ContribAuthor ">I.  Cortés-Ciriano</span>, <span class="hlFld-ContribAuthor ">W.  Dehaen</span>, <span class="hlFld-ContribAuthor ">P.  Kříž</span>, <span class="hlFld-ContribAuthor ">G. J. P.  van Westen</span>, <span class="hlFld-ContribAuthor ">I. V.  Tetko</span>, <span class="hlFld-ContribAuthor ">A.  Bender</span>, <span class="hlFld-ContribAuthor ">D.  Svozil</span>. </span><span class="cited-content_cbyCitation_article-title">QSAR-derived affinity fingerprints (part 1): fingerprint construction and modeling performance for similarity searching, bioactivity classification and scaffold hopping. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-020-00443-6" title="DOI URL">https://doi.org/10.1186/s13321-020-00443-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-020-00443-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-020-00443-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DQSAR-derived%252Baffinity%252Bfingerprints%252B%252528part%252B1%252529%25253A%252Bfingerprint%252Bconstruction%252Band%252Bmodeling%252Bperformance%252Bfor%252Bsimilarity%252Bsearching%25252C%252Bbioactivity%252Bclassification%252Band%252Bscaffold%252Bhopping%26aulast%3D%25C5%25A0kuta%26aufirst%3DC.%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabella  Mendolia</span>, <span class="hlFld-ContribAuthor ">Salvatore  Contino</span>, <span class="hlFld-ContribAuthor ">Ugo  Perricone</span>, <span class="hlFld-ContribAuthor ">Edoardo  Ardizzone</span>, <span class="hlFld-ContribAuthor ">Roberto  Pirrone</span>. </span><span class="cited-content_cbyCitation_article-title">Convolutional architectures for virtual screening. </span><span class="cited-content_cbyCitation_journal-name">BMC Bioinformatics</span><span> <strong>2020,</strong> <em>21 </em>
                                    (S8)
                                     <a href="https://doi.org/10.1186/s12859-020-03645-9" title="DOI URL">https://doi.org/10.1186/s12859-020-03645-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12859-020-03645-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12859-020-03645-9%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Bioinformatics%26atitle%3DConvolutional%252Barchitectures%252Bfor%252Bvirtual%252Bscreening%26aulast%3DMendolia%26aufirst%3DIsabella%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3DS8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaja Bergant  Loboda</span>, <span class="hlFld-ContribAuthor ">Katja  Valjavec</span>, <span class="hlFld-ContribAuthor ">Martina  Štampar</span>, <span class="hlFld-ContribAuthor ">Gerhard  Wolber</span>, <span class="hlFld-ContribAuthor ">Bojana  Žegura</span>, <span class="hlFld-ContribAuthor ">Metka  Filipič</span>, <span class="hlFld-ContribAuthor ">Marija Sollner  Dolenc</span>, <span class="hlFld-ContribAuthor ">Andrej  Perdih</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of 3,5-substituted 1,2,4-oxadiazoles as catalytic inhibitors of human DNA topoisomerase IIα. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>99 </em>, 103828. <a href="https://doi.org/10.1016/j.bioorg.2020.103828" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103828</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103828%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252B3%25252C5-substituted%252B1%25252C2%25252C4-oxadiazoles%252Bas%252Bcatalytic%252Binhibitors%252Bof%252Bhuman%252BDNA%252Btopoisomerase%252BII%2525CE%2525B1%26aulast%3DLoboda%26aufirst%3DKaja%2BBergant%26date%3D2020%26volume%3D99%26spage%3D103828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zarko  Gagic</span>, <span class="hlFld-ContribAuthor ">Dusan  Ruzic</span>, <span class="hlFld-ContribAuthor ">Nemanja  Djokovic</span>, <span class="hlFld-ContribAuthor ">Teodora  Djikic</span>, <span class="hlFld-ContribAuthor ">Katarina  Nikolic</span>. </span><span class="cited-content_cbyCitation_article-title">In silico Methods for Design of Kinase Inhibitors as Anticancer Drugs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00873" title="DOI URL">https://doi.org/10.3389/fchem.2019.00873</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00873%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DIn%252Bsilico%252BMethods%252Bfor%252BDesign%252Bof%252BKinase%252BInhibitors%252Bas%252BAnticancer%252BDrugs%26aulast%3DGagic%26aufirst%3DZarko%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">João  Correia</span>, <span class="hlFld-ContribAuthor ">Tiago  Resende</span>, <span class="hlFld-ContribAuthor ">Delora  Baptista</span>, <span class="hlFld-ContribAuthor ">Miguel  Rocha</span>. </span><span class="cited-content_cbyCitation_article-title">Artificial Intelligence in Biological Activity Prediction. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 164-172. <a href="https://doi.org/10.1007/978-3-030-23873-5_20" title="DOI URL">https://doi.org/10.1007/978-3-030-23873-5_20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-23873-5_20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-23873-5_20%26sid%3Dliteratum%253Aachs%26atitle%3DArtificial%252BIntelligence%252Bin%252BBiological%252BActivity%252BPrediction%26aulast%3DCorreia%26aufirst%3DJo%25C3%25A3o%26date%3D2020%26date%3D2019%26spage%3D164%26epage%3D172%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DPractical%252BApplications%252Bof%252BComputational%252BBiology%252Band%252BBioinformatics%25252C%252B13th%252BInternational%252BConference%26aulast%3DFdez-Riverola%26aufirst%3DFlorentino%26date%3D2020%26volume%3D1005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhaojun  Li</span>, <span class="hlFld-ContribAuthor ">Xutong  Li</span>, <span class="hlFld-ContribAuthor ">Xiaohong  Liu</span>, <span class="hlFld-ContribAuthor ">Zunyun  Fu</span>, <span class="hlFld-ContribAuthor ">Zhaoping  Xiong</span>, <span class="hlFld-ContribAuthor ">Xiaolong  Wu</span>, <span class="hlFld-ContribAuthor ">Xiaoqin  Tan</span>, <span class="hlFld-ContribAuthor ">Jihui  Zhao</span>, <span class="hlFld-ContribAuthor ">Feisheng  Zhong</span>, <span class="hlFld-ContribAuthor ">Xiaozhe  Wan</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, . </span><span class="cited-content_cbyCitation_article-title">KinomeX: a web application for predicting kinome-wide polypharmacology effect of small molecules. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2019,</strong> <em>35 </em>
                                    (24)
                                     , 5354-5356. <a href="https://doi.org/10.1093/bioinformatics/btz519" title="DOI URL">https://doi.org/10.1093/bioinformatics/btz519</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btz519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtz519%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DKinomeX%25253A%252Ba%252Bweb%252Bapplication%252Bfor%252Bpredicting%252Bkinome-wide%252Bpolypharmacology%252Beffect%252Bof%252Bsmall%252Bmolecules%26aulast%3DLi%26aufirst%3DZhaojun%26date%3D2019%26date%3D2019%26volume%3D35%26issue%3D24%26spage%3D5354%26epage%3D5356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thierry  Hanser</span>, <span class="hlFld-ContribAuthor ">Fabian P.  Steinmetz</span>, <span class="hlFld-ContribAuthor ">Jeffrey  Plante</span>, <span class="hlFld-ContribAuthor ">Friedrich  Rippmann</span>, <span class="hlFld-ContribAuthor ">Mireille  Krier</span>. </span><span class="cited-content_cbyCitation_article-title">Avoiding hERG-liability in drug design via synergetic combinations of different (Q)SAR methodologies and data sources: a case study in an industrial setting. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-019-0334-y" title="DOI URL">https://doi.org/10.1186/s13321-019-0334-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-019-0334-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-019-0334-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DAvoiding%252BhERG-liability%252Bin%252Bdrug%252Bdesign%252Bvia%252Bsynergetic%252Bcombinations%252Bof%252Bdifferent%252B%252528Q%252529SAR%252Bmethodologies%252Band%252Bdata%252Bsources%25253A%252Ba%252Bcase%252Bstudy%252Bin%252Ban%252Bindustrial%252Bsetting%26aulast%3DHanser%26aufirst%3DThierry%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Balaguru  Ravikumar</span>, <span class="hlFld-ContribAuthor ">Sanna  Timonen</span>, <span class="hlFld-ContribAuthor ">Zaid  Alam</span>, <span class="hlFld-ContribAuthor ">Elina  Parri</span>, <span class="hlFld-ContribAuthor ">Krister  Wennerberg</span>, <span class="hlFld-ContribAuthor ">Tero  Aittokallio</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomic Analysis of the Druggable Kinome and Its Application to Repositioning and Lead Identification Studies. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (11)
                                     , 1608-1622.e6. <a href="https://doi.org/10.1016/j.chembiol.2019.08.007" title="DOI URL">https://doi.org/10.1016/j.chembiol.2019.08.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2019.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2019.08.007%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DChemogenomic%252BAnalysis%252Bof%252Bthe%252BDruggable%252BKinome%252Band%252BIts%252BApplication%252Bto%252BRepositioning%252Band%252BLead%252BIdentification%252BStudies%26aulast%3DRavikumar%26aufirst%3DBalaguru%26date%3D2019%26volume%3D26%26issue%3D11%26spage%3D1608%26epage%3D1622.e6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gerhard  Hessler</span>, <span class="hlFld-ContribAuthor ">Christoph  Grebner</span>, <span class="hlFld-ContribAuthor ">Hans  Matter</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Approaches for Target Inference. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 277-322. <a href="https://doi.org/10.1002/9783527818242.ch10" title="DOI URL">https://doi.org/10.1002/9783527818242.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527818242.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527818242.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BApproaches%252Bfor%252BTarget%252BInference%26aulast%3DHessler%26aufirst%3DGerhard%26date%3D2019%26date%3D2019%26spage%3D277%26epage%3D322%26pub%3DWiley%26atitle%3DTarget%252BDiscovery%252Band%252BValidation%26aulast%3DPlowright%26aufirst%3DAlleyn%2BT%26date%3D2019%26date%3D2019%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weixing  Dai</span>, <span class="hlFld-ContribAuthor ">Dianjing  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">A Ligand-Based Virtual Screening Method Using Direct Quantification of Generalization Ability. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (13)
                                     , 2414. <a href="https://doi.org/10.3390/molecules24132414" title="DOI URL">https://doi.org/10.3390/molecules24132414</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24132414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24132414%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DA%252BLigand-Based%252BVirtual%252BScreening%252BMethod%252BUsing%252BDirect%252BQuantification%252Bof%252BGeneralization%252BAbility%26aulast%3DDai%26aufirst%3DWeixing%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D13%26spage%3D2414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hansaim  Lim</span>, <span class="hlFld-ContribAuthor ">Di  He</span>, <span class="hlFld-ContribAuthor ">Yue  Qiu</span>, <span class="hlFld-ContribAuthor ">Patrycja  Krawczuk</span>, <span class="hlFld-ContribAuthor ">Xiaoru  Sun</span>, <span class="hlFld-ContribAuthor ">Lei  Xie</span>, . </span><span class="cited-content_cbyCitation_article-title">Rational discovery of dual-indication multi-target PDE/Kinase inhibitor for precision anti-cancer therapy using structural systems pharmacology. </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2019,</strong> <em>15 </em>
                                    (6)
                                     , e1006619. <a href="https://doi.org/10.1371/journal.pcbi.1006619" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1006619</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1006619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1006619%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DRational%252Bdiscovery%252Bof%252Bdual-indication%252Bmulti-target%252BPDE%25252FKinase%252Binhibitor%252Bfor%252Bprecision%252Banti-cancer%252Btherapy%252Busing%252Bstructural%252Bsystems%252Bpharmacology%26aulast%3DLim%26aufirst%3DHansaim%26date%3D2019%26date%3D2019%26volume%3D15%26issue%3D6%26spage%3De1006619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rayees  Rahman</span>, <span class="hlFld-ContribAuthor ">Peter Man-Un  Ung</span>, <span class="hlFld-ContribAuthor ">Avner  Schlessinger</span>. </span><span class="cited-content_cbyCitation_article-title">KinaMetrix: a web resource to investigate kinase conformations and inhibitor space. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>47 </em>
                                    (D1)
                                     , D361-D366. <a href="https://doi.org/10.1093/nar/gky916" title="DOI URL">https://doi.org/10.1093/nar/gky916</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gky916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgky916%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DKinaMetrix%25253A%252Ba%252Bweb%252Bresource%252Bto%252Binvestigate%252Bkinase%252Bconformations%252Band%252Binhibitor%252Bspace%26aulast%3DRahman%26aufirst%3DRayees%26date%3D2019%26date%3D2018%26volume%3D47%26issue%3DD1%26spage%3DD361%26epage%3DD366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabella  Mendolia</span>, <span class="hlFld-ContribAuthor ">Salvatore  Contino</span>, <span class="hlFld-ContribAuthor ">Ugo  Perricone</span>, <span class="hlFld-ContribAuthor ">Roberto  Pirrone</span>, <span class="hlFld-ContribAuthor ">Edoardo  Ardizzone</span>. </span><span class="cited-content_cbyCitation_article-title">A Convolutional Neural Network for Virtual Screening of Molecular Fingerprints. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 399-409. <a href="https://doi.org/10.1007/978-3-030-30642-7_36" title="DOI URL">https://doi.org/10.1007/978-3-030-30642-7_36</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-30642-7_36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-30642-7_36%26sid%3Dliteratum%253Aachs%26atitle%3DA%252BConvolutional%252BNeural%252BNetwork%252Bfor%252BVirtual%252BScreening%252Bof%252BMolecular%252BFingerprints%26aulast%3DMendolia%26aufirst%3DIsabella%26date%3D2019%26date%3D2019%26spage%3D399%26epage%3D409%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DImage%252BAnalysis%252Band%252BProcessing%252B%2525E2%252580%252593%252BICIAP%252B2019%26aulast%3DRicci%26aufirst%3DElisa%26date%3D2019%26volume%3D11751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yibo  Li</span>, <span class="hlFld-ContribAuthor ">Liangren  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhenming  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-objective de novo drug design with conditional graph generative model. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2018,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-018-0287-6" title="DOI URL">https://doi.org/10.1186/s13321-018-0287-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-018-0287-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-018-0287-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DMulti-objective%252Bde%252Bnovo%252Bdrug%252Bdesign%252Bwith%252Bconditional%252Bgraph%252Bgenerative%252Bmodel%26aulast%3DLi%26aufirst%3DYibo%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raghuram  Srinivas</span>, <span class="hlFld-ContribAuthor ">Pavel V.  Klimovich</span>, <span class="hlFld-ContribAuthor ">Eric C.  Larson</span>. </span><span class="cited-content_cbyCitation_article-title">Implicit-descriptor ligand-based virtual screening by means of collaborative filtering. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2018,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-018-0310-y" title="DOI URL">https://doi.org/10.1186/s13321-018-0310-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-018-0310-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-018-0310-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DImplicit-descriptor%252Bligand-based%252Bvirtual%252Bscreening%252Bby%252Bmeans%252Bof%252Bcollaborative%252Bfiltering%26aulast%3DSrinivas%26aufirst%3DRaghuram%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gerhard  Hessler</span>, <span class="hlFld-ContribAuthor ">Karl-Heinz  Baringhaus</span>. </span><span class="cited-content_cbyCitation_article-title">Artificial Intelligence in Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (10)
                                     , 2520. <a href="https://doi.org/10.3390/molecules23102520" title="DOI URL">https://doi.org/10.3390/molecules23102520</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23102520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23102520%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DArtificial%252BIntelligence%252Bin%252BDrug%252BDesign%26aulast%3DHessler%26aufirst%3DGerhard%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D10%26spage%3D2520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Cichonska</span>, <span class="hlFld-ContribAuthor ">Tapio  Pahikkala</span>, <span class="hlFld-ContribAuthor ">Sandor  Szedmak</span>, <span class="hlFld-ContribAuthor ">Heli  Julkunen</span>, <span class="hlFld-ContribAuthor ">Antti  Airola</span>, <span class="hlFld-ContribAuthor ">Markus  Heinonen</span>, <span class="hlFld-ContribAuthor ">Tero  Aittokallio</span>, <span class="hlFld-ContribAuthor ">Juho  Rousu</span>. </span><span class="cited-content_cbyCitation_article-title">Learning with multiple pairwise kernels for drug bioactivity prediction. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2018,</strong> <em>34 </em>
                                    (13)
                                     , i509-i518. <a href="https://doi.org/10.1093/bioinformatics/bty277" title="DOI URL">https://doi.org/10.1093/bioinformatics/bty277</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/bty277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbty277%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DLearning%252Bwith%252Bmultiple%252Bpairwise%252Bkernels%252Bfor%252Bdrug%252Bbioactivity%252Bprediction%26aulast%3DCichonska%26aufirst%3DAnna%26date%3D2018%26date%3D2018%26volume%3D34%26issue%3D13%26spage%3Di509%26epage%3Di518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edgar  Jacoby</span>, <span class="hlFld-ContribAuthor ">Berthold  Wroblowski</span>, <span class="hlFld-ContribAuthor ">Christophe  Buyck</span>, <span class="hlFld-ContribAuthor ">Jean-Marc  Neefs</span>, <span class="hlFld-ContribAuthor ">Christophe  Meyer</span>, <span class="hlFld-ContribAuthor ">Maxwell D.  Cummings</span>, <span class="hlFld-ContribAuthor ">Herman  van Vlijmen</span>. </span><span class="cited-content_cbyCitation_article-title">Protocols for the Design of Kinase-focused Compound Libraries. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2018,</strong> <em>37 </em>
                                    (5)
                                     , 1700119. <a href="https://doi.org/10.1002/minf.201700119" title="DOI URL">https://doi.org/10.1002/minf.201700119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201700119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201700119%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DProtocols%252Bfor%252Bthe%252BDesign%252Bof%252BKinase-focused%252BCompound%252BLibraries%26aulast%3DJacoby%26aufirst%3DEdgar%26date%3D2018%26date%3D2017%26volume%3D37%26issue%3D5%26spage%3D1700119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fleur M.  Ferguson</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase inhibitors: the road ahead. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2018,</strong> <em>17 </em>
                                    (5)
                                     , 353-377. <a href="https://doi.org/10.1038/nrd.2018.21" title="DOI URL">https://doi.org/10.1038/nrd.2018.21</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DKinase%252Binhibitors%25253A%252Bthe%252Broad%252Bahead%26aulast%3DFerguson%26aufirst%3DFleur%2BM.%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D5%26spage%3D353%26epage%3D377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pavel V.  Pogodin</span>, <span class="hlFld-ContribAuthor ">Alexey A.  Lagunin</span>, <span class="hlFld-ContribAuthor ">Anastasia V.  Rudik</span>, <span class="hlFld-ContribAuthor ">Dmitry A.  Filimonov</span>, <span class="hlFld-ContribAuthor ">Dmitry S.  Druzhilovskiy</span>, <span class="hlFld-ContribAuthor ">Mark C.  Nicklaus</span>, <span class="hlFld-ContribAuthor ">Vladimir V.  Poroikov</span>. </span><span class="cited-content_cbyCitation_article-title">How to Achieve Better Results Using PASS-Based Virtual Screening: Case Study for Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00133" title="DOI URL">https://doi.org/10.3389/fchem.2018.00133</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00133%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DHow%252Bto%252BAchieve%252BBetter%252BResults%252BUsing%252BPASS-Based%252BVirtual%252BScreening%25253A%252BCase%252BStudy%252Bfor%252BKinase%252BInhibitors%26aulast%3DPogodin%26aufirst%3DPavel%2BV.%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Frieda A.  Sorgenfrei</span>, <span class="hlFld-ContribAuthor ">Simone  Fulle</span>, <span class="hlFld-ContribAuthor ">Benjamin  Merget</span>. </span><span class="cited-content_cbyCitation_article-title">Kinome-Wide Profiling Prediction of Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (6)
                                     , 495-499. <a href="https://doi.org/10.1002/cmdc.201700180" title="DOI URL">https://doi.org/10.1002/cmdc.201700180</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700180%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DKinome-Wide%252BProfiling%252BPrediction%252Bof%252BSmall%252BMolecules%26aulast%3DSorgenfrei%26aufirst%3DFrieda%2BA.%26date%3D2018%26date%3D2017%26volume%3D13%26issue%3D6%26spage%3D495%26epage%3D499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christin  Rakers</span>, <span class="hlFld-ContribAuthor ">Rifat Ara  Najnin</span>, <span class="hlFld-ContribAuthor ">Ahsan Habib  Polash</span>, <span class="hlFld-ContribAuthor ">Shunichi  Takeda</span>, <span class="hlFld-ContribAuthor ">J.B.  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomic Active Learning's Domain of Applicability on Small, Sparse qHTS Matrices: A Study Using Cytochrome P450 and Nuclear Hormone Receptor Families. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (6)
                                     , 511-521. <a href="https://doi.org/10.1002/cmdc.201700677" title="DOI URL">https://doi.org/10.1002/cmdc.201700677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700677%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DChemogenomic%252BActive%252BLearning%252527s%252BDomain%252Bof%252BApplicability%252Bon%252BSmall%25252C%252BSparse%252BqHTS%252BMatrices%25253A%252BA%252BStudy%252BUsing%252BCytochrome%252BP450%252Band%252BNuclear%252BHormone%252BReceptor%252BFamilies%26aulast%3DRakers%26aufirst%3DChristin%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D6%26spage%3D511%26epage%3D521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tong  He</span>, <span class="hlFld-ContribAuthor ">Marten  Heidemeyer</span>, <span class="hlFld-ContribAuthor ">Fuqiang  Ban</span>, <span class="hlFld-ContribAuthor ">Artem  Cherkasov</span>, <span class="hlFld-ContribAuthor ">Martin  Ester</span>. </span><span class="cited-content_cbyCitation_article-title">SimBoost: a read-across approach for predicting drug–target binding affinities using gradient boosting machines. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2017,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-017-0209-z" title="DOI URL">https://doi.org/10.1186/s13321-017-0209-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-017-0209-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-017-0209-z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DSimBoost%25253A%252Ba%252Bread-across%252Bapproach%252Bfor%252Bpredicting%252Bdrug%2525E2%252580%252593target%252Bbinding%252Baffinities%252Busing%252Bgradient%252Bboosting%252Bmachines%26aulast%3DHe%26aufirst%3DTong%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Cichonska</span>, <span class="hlFld-ContribAuthor ">Balaguru  Ravikumar</span>, <span class="hlFld-ContribAuthor ">Elina  Parri</span>, <span class="hlFld-ContribAuthor ">Sanna  Timonen</span>, <span class="hlFld-ContribAuthor ">Tapio  Pahikkala</span>, <span class="hlFld-ContribAuthor ">Antti  Airola</span>, <span class="hlFld-ContribAuthor ">Krister  Wennerberg</span>, <span class="hlFld-ContribAuthor ">Juho  Rousu</span>, <span class="hlFld-ContribAuthor ">Tero  Aittokallio</span>, . </span><span class="cited-content_cbyCitation_article-title">Computational-experimental approach to drug-target interaction mapping: A case study on kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2017,</strong> <em>13 </em>
                                    (8)
                                     , e1005678. <a href="https://doi.org/10.1371/journal.pcbi.1005678" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1005678</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1005678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1005678%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DComputational-experimental%252Bapproach%252Bto%252Bdrug-target%252Binteraction%252Bmapping%25253A%252BA%252Bcase%252Bstudy%252Bon%252Bkinase%252Binhibitors%26aulast%3DCichonska%26aufirst%3DAnna%26date%3D2017%26date%3D2017%26volume%3D13%26issue%3D8%26spage%3De1005678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Albert J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">Andrea  Volkamer</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase-Centric Computational Drug Development. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 197-236. <a href="https://doi.org/10.1016/bs.armc.2017.08.001" title="DOI URL">https://doi.org/10.1016/bs.armc.2017.08.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2017.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2017.08.001%26sid%3Dliteratum%253Aachs%26atitle%3DKinase-Centric%252BComputational%252BDrug%252BDevelopment%26aulast%3DKooistra%26aufirst%3DAlbert%2BJ.%26date%3D2017%26spage%3D197%26epage%3D236%26pub%3DElsevier%26atitle%3DPlatform%252BTechnologies%252Bin%252BDrug%252BDiscovery%252Band%252BValidation%26date%3D2017%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Venn diagram of compounds (left) and kinases (right) from different sources. Whereas only few kinases are unique for a single data source, many compounds only appear in one source.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Boxplots of (a) average Random Forest model quality metrics and (b) standard deviations of 5-fold cross-validations (CV) on various data sets. Black lines depict the median, and boxes illustrate the interquartile range (IQR) of the distribution. Whiskers extent to 1.5·<i>IQR</i> from the median. Although models based on <i>Open</i> data show higher average AUC, sensitivity, and specificity, the standard deviations of the 5-fold CV are significantly higher compared to the results obtained based on the <i>Proprietary</i> and <i>Combined</i> data sets. Hence, models derived from the latter two data sets are more robust.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. AUC plotted against the number of active compounds per kinase for the (a) <i>Open</i> and (b) <i>Combined</i> data sets. In both cases, no correlation between the achieved AUC and number of actives exists. (c) Binned average AUC against number of active compounds. Bins have a range of 50 actives. Models with a high number of actives generally result in prediction models with high AUC values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. First two principal components of PCA of compounds from the <i>Combined</i> data set with measurements against kinases KDR, EGFR, and MET, respectively. Connectivity-based Morgan FPs were used as input. Besides the majority cluster, the <i>Proprietary</i> data (from Merck KGaA) occupies several further distinct clusters in the chemical space.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Kinome map of the performance of activity prediction models by Random Forest. Kinases are colored based on their AUC value. High-quality models (AUC ≥ 0.8) are scattered well across the kinome tree and cover almost all kinase families. FDA approved kinase inhibitor targets are depicted as red triangles. Figure was created with KinMap (<a href="http://kinhub.org/kinmap" class="extLink">http://kinhub.org/kinmap</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Boxplots of model quality measurements for Random Forest and alternative Machine Learning approaches. The plotted results are based on the <i>Combined</i> data set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlap among various Machine Learning methods of high-quality models with an AUC of ≥0.9 and ≥0.8, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Boxplots of model quality measurements for Random Forest using chemical and/or bioactivity fingerprints based on <i>Proprietary</i> data. Adding bioactivity FPs mostly results in improved sensitivity. A concatenation of both FPs yields excellent performance in activity prediction. Plotted are results obtained from 5-fold CV.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (a) Selectivity prediction based on bioactivity fingerprints obtained by Random Forest regression achieves an <i>R</i><sub class="stack">ext</sub><sup class="stack">2</sup> of 0.85 ± 0.01. (b) Plain averaging of experimentally obtained bioactivity fingerprints (<i>S</i><sub>calc</sub>) does not correlate with the experimental <i>S</i>. The calculation both in (a) and (b) are done based on 20 kinases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/medium/jm-2016-01611h_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (a) Classification results of external data using the RF classifier. With a default probability cutoff of 0.5, only four wrong classifications at a 95% inhibition cutoff and two wrong classifications at an 80% inhibition cutoff appear for the RF classifier. Cohen’s κ values of 0.65 and 0.79, respectively, suggest a very high predictive power. (b) ROC analysis of binary classification of the external data with varying experimental inhibition cutoff. An AUC of 0.85 confirms the successful application of the models on the external test set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01611/20170106/images/large/jm-2016-01611h_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01611&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66443" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66443" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 62 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Cohen, P.</span><span> </span><span class="NLM_article-title">Protein kinases-the major drug targets of the twenty-first century?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">309</span><span class="NLM_x">–</span> <span class="NLM_lpage">315</span><span class="refDoi"> DOI: 10.1038/nrd773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnrd773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=12120282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD38XivVGmtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=309-315&author=P.+Cohen&title=Protein+kinases-the+major+drug+targets+of+the+twenty-first+century%3F&doi=10.1038%2Fnrd773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: Protein kinases - the major drug targets of the twenty-first century?</span></div><div class="casAuthors">Cohen, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">309-315</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein phosphorylation regulates most aspects of cell life, whereas abnormal phosphorylation is a cause or consequence of disease.  A growing interest in developing orally active protein-kinase inhibitors has recently culminated in the approval of the first of these drugs for clin. use.  Protein kinases have now become the second most important group of drug targets, after G-protein-coupled receptors.  Here, I give a personal view of some of the most important advances that have shaped this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRd_GqiGAWaLVg90H21EOLACvtfcHk0liLFqWWOe8THg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivVGmtr8%253D&md5=67d2fbc11a55b5980ee0692f7bc13cc9</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd773%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DProtein%2520kinases-the%2520major%2520drug%2520targets%2520of%2520the%2520twenty-first%2520century%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D309%26epage%3D315%26doi%3D10.1038%2Fnrd773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alessi, D. R.</span><span> </span><span class="NLM_article-title">Kinase drug discovery—what’s next in the field?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">–</span> <span class="NLM_lpage">104</span><span class="refDoi"> DOI: 10.1021/cb300610s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300610s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=96-104&author=P.+Cohenauthor=D.+R.+Alessi&title=Kinase+drug+discovery%E2%80%94what%E2%80%99s+next+in+the+field%3F&doi=10.1021%2Fcb300610s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fcb300610s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300610s%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DKinase%2520drug%2520discovery%25E2%2580%2594what%25E2%2580%2599s%2520next%2520in%2520the%2520field%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D96%26epage%3D104%26doi%3D10.1021%2Fcb300610s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausen, M. H.</span><span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+an+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0liLFqWWOe8THg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520an%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiöth, H. B.</span><span> </span><span class="NLM_article-title">Advances in kinase targeting: current clinical use and clinical trials</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">604</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1016/j.tips.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1016%2Fj.tips.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=25312588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KgurjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=604-620&author=M.+Rask-Andersenauthor=J.+Zhangauthor=D.+Fabbroauthor=H.+B.+Schi%C3%B6th&title=Advances+in+kinase+targeting%3A+current+clinical+use+and+clinical+trials&doi=10.1016%2Fj.tips.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in kinase targeting: current clinical use and clinical trials</span></div><div class="casAuthors">Rask-Andersen, Mathias; Zhang, Jin; Fabbro, Doriano; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">604-620</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Phosphotransferases, also known as kinases, are the most intensively studied protein drug target category in current pharmacol. research, as evidenced by the vast no. of kinase-targeting agents enrolled in active clin. trials.  This development has emerged following the great success of small-mol., orally available protein kinase inhibitors for the treatment of cancer, starting with the introduction of imatinib (Gleevec) in 2003.  The pharmacol. utility of kinase-targeting has expanded to include treatment of inflammatory diseases, and rapid development is ongoing for kinase-targeted therapies in a broad array of indications in ophthalmol., analgesia, central nervous system (CNS) disorders, and the complications of diabetes, osteoporosis, and otol.  In this review we highlight specifically the kinase drug targets and kinase-targeting agents being explored in current clin. trials.  This anal. is based on a recent est. of all established and clin. trial drug mechanisms of action, utilizing private and public databases to create an extensive dataset detailing aspects of more than 3000 approved and exptl. drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLhFMSrq0fXbVg90H21EOLACvtfcHk0ljZ4p7L_Alp3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KgurjL&md5=bedc3a53a9cdab170e146f89bc495ff7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DSchi%25C3%25B6th%26aufirst%3DH.%2BB.%26atitle%3DAdvances%2520in%2520kinase%2520targeting%253A%2520current%2520clinical%2520use%2520and%2520clinical%2520trials%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26spage%3D604%26epage%3D620%26doi%3D10.1016%2Fj.tips.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Metz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merta, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kifle, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Navigating the kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.1038/nchembio.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+kinome&doi=10.1038%2Fnchembio.530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0ljZ4p7L_Alp3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D200%26epage%3D202%26doi%3D10.1038%2Fnchembio.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Paricharak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klenka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, U. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span> </span><span class="NLM_article-title">Are phylogenetic trees suitable for chemogenomics analyses of bioactivity data sets: the importance of shared active compounds and choosing a suitable data embedding method, as exemplified on kinases</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="refDoi"> DOI: 10.1186/1758-2946-5-49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1186%2F1758-2946-5-49" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtlyju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=49&author=S.+Paricharakauthor=T.+Klenkaauthor=M.+Augustinauthor=U.+A.+Patelauthor=A.+Bender&title=Are+phylogenetic+trees+suitable+for+chemogenomics+analyses+of+bioactivity+data+sets%3A+the+importance+of+shared+active+compounds+and+choosing+a+suitable+data+embedding+method%2C+as+exemplified+on+kinases&doi=10.1186%2F1758-2946-5-49"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Are phylogenetic trees suitable for chemogenomics analyses of bioactivity data sets: the importance of shared active compounds and choosing a suitable data embedding method, as exemplified on Kinases</span></div><div class="casAuthors">Paricharak, Shardul; Klenka, Tom; Augustin, Martin; Patel, Umesh A.; Bender, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49/1-49/20, 20 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Phylogenetic trees are commonly used for the anal. of chemogenomics datasets and to relate protein targets to each other, based on the (shared) bioactivities of their ligands.  However, no real assessment as to the suitability of this representation has been performed yet in this area.  The authors aimed to address this shortcoming in the current work, as exemplified by a kinase data set, given the importance of kinases in many diseases as well as the availability of large-scale datasets for anal.  The authors analyzed a dataset comprising 157 compds., which have been tested at concns. of 1 μM and 10 μM against a panel of 225 human protein kinases in full-matrix expts., aiming to explain kinase promiscuity and selectivity against inhibitors.  Compds. were described by chem. features, which were used to represent kinases (i.e. each kinase had an active set of features and an inactive set).  Using this representation, a bioactivity-based classification was made of the kinome, which partially resembles previous sequence-based classifications, where particularly kinases from the TK, CDK, CLK and AGC branches cluster together.  However, the authors were also able to show that in ∼57% of cases, on av. 6 kinase inhibitors exhibit activity against kinases which are located at a large distance in the sequence-based classification (at a relative distance of 0.6-0.8 on a scale from 0 to 1), but are correctly located closer to each other in the authors' bioactivity-based tree (distance 0-0.4).  Despite this improvement on sequence-based classification, also the bioactivity-based classification needed further attention: for ∼80% of all analyzed kinases, kinases classified as neighbors according to the bioactivity-based classification also show high SAR similarity (i.e. a high fraction of shared active compds. and therefore, interaction with similar inhibitors).  However, in the remaining ∼20% of cases a clear relation between kinase bioactivity profile similarity and shared active compds. could not be established, which is in agreement with previously published atypical SAR (such as for LCK, FGFR1, AKT2, DAPK1, TGFR1, MK12 and AKT1).  The authors were hence able to show that (1) targets (here kinases) with few shared activities are difficult to establish neighborhood relations for, and (2) phylogenetic tree representations make implicit assumptions (i.e. that neighboring kinases exhibit similar interaction profiles with inhibitors) that are not always suitable for analyses of bioactivity space.  While both points have been implicitly alluded to before, this is to the information of the authors first study that explores both points on a comprehensive basis.  Excluding kinases with few shared activities improved the situation greatly (the percentage of kinases for which no neighborhood relation could be established dropped from 20% to only 4%).  All of the above findings need to be taken into account when performing chemogenomics analyses, also for other target classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7J0H5BE4gCrVg90H21EOLACvtfcHk0lg0yCg-_FI0Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtlyju7w%253D&md5=58b32b31d9c18ae6f196824b9be97f40</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-5-49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-5-49%26sid%3Dliteratum%253Aachs%26aulast%3DParicharak%26aufirst%3DS.%26aulast%3DKlenka%26aufirst%3DT.%26aulast%3DAugustin%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DU.%2BA.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DAre%2520phylogenetic%2520trees%2520suitable%2520for%2520chemogenomics%2520analyses%2520of%2520bioactivity%2520data%2520sets%253A%2520the%2520importance%2520of%2520shared%2520active%2520compounds%2520and%2520choosing%2520a%2520suitable%2520data%2520embedding%2520method%252C%2520as%2520exemplified%2520on%2520kinases%26jtitle%3DJ.%2520Cheminf.%26date%3D2013%26volume%3D5%26spage%3D49%26doi%3D10.1186%2F1758-2946-5-49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Ferrè, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmeri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmer-Citterich, M.</span><span> </span><span class="NLM_article-title">Computational methods for analysis and inference of kinase/inhibitor relationships</span> <span class="citation_source-journal">Front. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">196</span><span class="refDoi"> DOI: 10.3389/fgene.2014.00196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.3389%2Ffgene.2014.00196" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=25071826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A280%3ADC%252BC2cbntVShsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=196&author=F.+Ferr%C3%A8author=A.+Palmeriauthor=M.+Helmer-Citterich&title=Computational+methods+for+analysis+and+inference+of+kinase%2Finhibitor+relationships&doi=10.3389%2Ffgene.2014.00196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Computational methods for analysis and inference of kinase/inhibitor relationships</span></div><div class="casAuthors">Ferre Fabrizio; Palmeri Antonio; Helmer-Citterich Manuela</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">196</span>
        ISSN:<span class="NLM_cas:issn">1664-8021</span>.
    </div><div class="casAbstract">The central role of kinases in virtually all signal transduction networks is the driving motivation for the development of compounds modulating their activity.  ATP-mimetic inhibitors are essential tools for elucidating signaling pathways and are emerging as promising therapeutic agents.  However, off-target ligand binding and complex and sometimes unexpected kinase/inhibitor relationships can occur for seemingly unrelated kinases, stressing that computational approaches are needed for learning the interaction determinants and for the inference of the effect of small compounds on a given kinase.  Recently published high-throughput profiling studies assessed the effects of thousands of small compound inhibitors, covering a substantial portion of the kinome.  This wealth of data paved the road for computational resources and methods that can offer a major contribution in understanding the reasons of the inhibition, helping in the rational design of more specific molecules, in the in silico prediction of inhibition for those neglected kinases for which no systematic analysis has been carried yet, in the selection of novel inhibitors with desired selectivity, and offering novel avenues of personalized therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnT8GombVJ2uQ4Pl9hzjQhfW6udTcc2ebYh93N5nQ_JLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbntVShsg%253D%253D&md5=48bc2fcaaccfb2ec3b2070cf0565b070</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2014.00196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2014.00196%26sid%3Dliteratum%253Aachs%26aulast%3DFerr%25C3%25A8%26aufirst%3DF.%26aulast%3DPalmeri%26aufirst%3DA.%26aulast%3DHelmer-Citterich%26aufirst%3DM.%26atitle%3DComputational%2520methods%2520for%2520analysis%2520and%2520inference%2520of%2520kinase%252Finhibitor%2520relationships%26jtitle%3DFront.%2520Genet.%26date%3D2014%26volume%3D5%26spage%3D196%26doi%3D10.3389%2Ffgene.2014.00196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hocker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1051</span><span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lh0Wxggv9b4Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Anastassiadis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devarajan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J. R.</span><span> </span><span class="NLM_article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1039</span><span class="NLM_x">–</span> <span class="NLM_lpage">1045</span><span class="refDoi"> DOI: 10.1038/nbt.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&author=T.+Anastassiadisauthor=S.+W.+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R.+Peterson&title=Comprehensive+assay+of+kinase+catalytic+activity+reveals+features+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0lh0Wxggv9b4Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DComprehensive%2520assay%2520of%2520kinase%2520catalytic%2520activity%2520reveals%2520features%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1039%26epage%3D1045%26doi%3D10.1038%2Fnbt.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szwajda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakyawar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hintsanen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wennerberg, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aittokallio, T.</span><span> </span><span class="NLM_article-title">Making sense of large-scale kinase inhibitor bioactivity data sets: a comparative and integrative analysis</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">735</span><span class="NLM_x">–</span> <span class="NLM_lpage">743</span><span class="refDoi"> DOI: 10.1021/ci400709d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400709d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXit1yls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=735-743&author=J.+Tangauthor=A.+Szwajdaauthor=S.+Shakyawarauthor=T.+Xuauthor=P.+Hintsanenauthor=K.+Wennerbergauthor=T.+Aittokallio&title=Making+sense+of+large-scale+kinase+inhibitor+bioactivity+data+sets%3A+a+comparative+and+integrative+analysis&doi=10.1021%2Fci400709d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Making Sense of Large-Scale Kinase Inhibitor Bioactivity Data Sets: A Comparative and Integrative Analysis</span></div><div class="casAuthors">Tang, Jing; Szwajda, Agnieszka; Shakyawar, Sushil; Xu, Tao; Hintsanen, Petteri; Wennerberg, Krister; Aittokallio, Tero</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">735-743</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We carried out a systematic evaluation of target selectivity profiles across three recent large-scale biochem. assays of kinase inhibitors and further compared these standardized bioactivity assays with data reported in the widely used databases ChEMBL and STITCH.  Our comparative evaluation revealed relative benefits and potential limitations among the bioactivity types, as well as pinpointed biases in the database curation processes.  Ignoring such issues in data heterogeneity and representation may lead to biased modeling of drugs' polypharmacol. effects as well as to unrealistic evaluation of computational strategies for the prediction of drug-target interaction networks.  Toward making use of the complementary information captured by the various bioactivity types, including IC50, Ki, and Kd, we also introduce a model-based integration approach, termed KIBA, and demonstrate here how it can be used to classify kinase inhibitor targets and to pinpoint potential errors in database-reported drug-target interactions.  An integrated drug-target bioactivity matrix across 52 498 chem. compds. and 467 kinase targets, including a total of 246 088 KIBA scores, has been made freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI6vIo2K_W17Vg90H21EOLACvtfcHk0lga4Aq3qHn1Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXit1yls7w%253D&md5=fd3bcbd39cb523f84d105a96cc4d17f2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci400709d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400709d%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DSzwajda%26aufirst%3DA.%26aulast%3DShakyawar%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DHintsanen%26aufirst%3DP.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DAittokallio%26aufirst%3DT.%26atitle%3DMaking%2520sense%2520of%2520large-scale%2520kinase%2520inhibitor%2520bioactivity%2520data%2520sets%253A%2520a%2520comparative%2520and%2520integrative%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D735%26epage%3D743%26doi%3D10.1021%2Fci400709d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krüger, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowotka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papadatos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">The ChEMBL bioactivity database: an update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1083</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1090</span><span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-D1090&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+bioactivity+database%3A+an+update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0lga4Aq3qHn1Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520bioactivity%2520database%253A%2520an%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1083%26epage%3DD1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urban, L.</span><span> </span><span class="NLM_article-title">Large-scale prediction and testing of drug activity on side-effect targets</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">486</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1038/nature11159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnature11159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=22722194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=361-367&author=E.+Lounkineauthor=M.+J.+Keiserauthor=S.+Whitebreadauthor=D.+Mikhailovauthor=J.+Hamonauthor=J.+L.+Jenkinsauthor=P.+Lavanauthor=E.+Weberauthor=A.+K.+Doakauthor=S.+C%C3%B4t%C3%A9author=B.+K.+Shoichetauthor=L.+Urban&title=Large-scale+prediction+and+testing+of+drug+activity+on+side-effect+targets&doi=10.1038%2Fnature11159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale prediction and testing of drug activity on side-effect targets</span></div><div class="casAuthors">Lounkine, Eugen; Keiser, Michael J.; Whitebread, Steven; Mikhailov, Dmitri; Hamon, Jacques; Jenkins, Jeremy L.; Lavan, Paul; Weber, Eckhard; Doak, Allison K.; Cote, Serge; Shoichet, Brian K.; Urban, Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">486</span>
        (<span class="NLM_cas:issue">7403</span>),
    <span class="NLM_cas:pages">361-367</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Discovering the unintended off-targets' that predict adverse drug reactions is daunting by empirical methods alone.  Drugs can act on several protein targets, some of which can be unrelated by conventional mol. metrics, and hundreds of proteins have been implicated in side effects.  Here we use a computational strategy to predict the activity of 656 marketed drugs on 73 unintended side-effect' targets.  Approx. half of the predictions were confirmed, either from proprietary databases unknown to the method or by new exptl. assays.  Affinities for these new off-targets ranged from 1 nM to 30 μM.  To explore relevance, we developed an assocn. metric to prioritize those new off-targets that explained side effects better than any known target of a given drug, creating a drug-target-adverse drug reaction network.  Among these new assocns. was the prediction that the abdominal pain side effect of the synthetic estrogen chlorotrianisene was mediated through its newly discovered inhibition of the enzyme cyclooxygenase-1.  The clin. relevance of this inhibition was borne out in whole human blood platelet aggregation assays.  This approach may have wide application to de-risking toxicol. liabilities in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5sVjfn4mgLLVg90H21EOLACvtfcHk0lgbrTuWfZVm4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D&md5=9c133b839d9527818a27a5834e4dfe11</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature11159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11159%26sid%3Dliteratum%253Aachs%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DLavan%26aufirst%3DP.%26aulast%3DWeber%26aufirst%3DE.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DUrban%26aufirst%3DL.%26atitle%3DLarge-scale%2520prediction%2520and%2520testing%2520of%2520drug%2520activity%2520on%2520side-effect%2520targets%26jtitle%3DNature%26date%3D2012%26volume%3D486%26spage%3D361%26epage%3D367%26doi%3D10.1038%2Fnature11159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Hillisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinrich, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wild, H.</span><span> </span><span class="NLM_article-title">Computational chemistry in the pharmaceutical industry: from childhood to adolescence</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1958</span><span class="NLM_x">–</span> <span class="NLM_lpage">1962</span><span class="refDoi"> DOI: 10.1002/cmdc.201500346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1002%2Fcmdc.201500346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=26358802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyju73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1958-1962&author=A.+Hillischauthor=N.+Heinrichauthor=H.+Wild&title=Computational+chemistry+in+the+pharmaceutical+industry%3A+from+childhood+to+adolescence&doi=10.1002%2Fcmdc.201500346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Chemistry in the Pharmaceutical Industry: From Childhood to Adolescence</span></div><div class="casAuthors">Hillisch, Alexander; Heinrich, Nikolaus; Wild, Hanno</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1958-1962</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Computational chem. within the pharmaceutical industry has grown into a field that proactively contributes to many aspects of drug design, including target selection and lead identification and optimization.  While methodol. advancements have been key to this development, organizational developments have been crucial to our success as well.  In particular, the interaction between computational and medicinal chem. and the integration of computational chem. into the entire drug discovery process have been invaluable.  Over the past ten years we have shaped and developed a highly efficient computational chem. group for small-mol. drug discovery at Bayer HealthCare that has significantly impacted the clin. development pipeline.  In this article we describe the setup and tasks of the computational group and discuss external collaborations.  We explain what we have found to be the most valuable and productive methods and discuss future directions for computational chem. method development.  We share this information with the hope of igniting interesting discussions around this topic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAFXZ0bBJeBrVg90H21EOLACvtfcHk0lgbrTuWfZVm4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyju73L&md5=1d5d7d908463585ac45b048a395b0dac</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500346%26sid%3Dliteratum%253Aachs%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DHeinrich%26aufirst%3DN.%26aulast%3DWild%26aufirst%3DH.%26atitle%3DComputational%2520chemistry%2520in%2520the%2520pharmaceutical%2520industry%253A%2520from%2520childhood%2520to%2520adolescence%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1958%26epage%3D1962%26doi%3D10.1002%2Fcmdc.201500346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Martin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansen, J.</span><span> </span><span class="NLM_article-title">Profile-QSAR: a novel meta-QSAR method that combines activities across the kinase family to accurately predict affinity, selectivity, and cellular activity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1942</span><span class="NLM_x">–</span> <span class="NLM_lpage">1956</span><span class="refDoi"> DOI: 10.1021/ci1005004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1005004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1942-1956&author=E.+Martinauthor=P.+Mukherjeeauthor=D.+Sullivanauthor=J.+Jansen&title=Profile-QSAR%3A+a+novel+meta-QSAR+method+that+combines+activities+across+the+kinase+family+to+accurately+predict+affinity%2C+selectivity%2C+and+cellular+activity&doi=10.1021%2Fci1005004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity</span></div><div class="casAuthors">Martin, Eric; Mukherjee, Prasenjit; Sullivan, David; Jansen, Johanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1942-1956</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-QSAR is a novel 2D predictive model building method for kinases.  This "meta-QSAR" method models the activity of each compd. against a new kinase target as a linear combination of its predicted activities against a large panel of 92 previously studied kinases comprised from 115 assays.  Profile-QSAR starts with a sparse incomplete kinase by compd. (KxC) activity matrix, used to generate Bayesian QSAR models for the 92 "basis-set" kinases.  These Bayesian QSARs generate a complete "synthetic" KxC activity matrix of predictions.  These synthetic activities are used as "chem. descriptors" to train partial-least squares (PLS) models, from modest amts. of medium-throughput screening data, for predicting activity against new kinases.  The Profile-QSAR predictions for the 92 kinases (115 assays) gave a median external R2ext = 0.59 on 25% held-out test sets.  The method has proven accurate enough to predict pairwise kinase selectivities with a median correlation of R2ext = 0.61 for 958 kinase pairs with at least 600 common compds.  It has been further expanded by adding a "CkXC" cellular activity matrix to the KxC matrix to predict cellular activity for 42 kinase driven cellular assays with median R2ext = 0.58 for 24 target modulation assays and R2ext = 0.41 for 18 cell proliferation assays.  The 2D Profile-QSAR, along with the 3D Surrogate AutoShim, are the foundations of an internally developed iterative medium-throughput screening (IMTS) methodol. for virtual screening (VS) of compd. archives as an alternative to exptl. high-throughput screening (HTS).  The method has been applied to 20 actual prospective kinase projects.  Biol. results have so far been obtained in eight of them.  Q2 values ranged from 0.3 to 0.7.  Hit-rates at 10 uM for exptl. tested compds. varied from 25% to 80%, except in K5, which was a special case aimed specifically at finding "type II" binders, where none of the compds. were predicted to be active at 10 μM.  These overall results are particularly striking as chem. novelty was an important criterion in selecting compds. for testing.  The method is completely automated.  Predicted activities for nearly 4 million internal and com. compds. across 115 kinase assays and 42 cellular assays are stored in the corporate database.  Like computed phys. properties, this predicted kinase activity profile can be computed and stored as each compd. is registered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTg6KqlgEyLVg90H21EOLACvtfcHk0li8l9duRCibnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D&md5=49422d01fd686a4befc9db7c123cfcce</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1005004%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DSullivan%26aufirst%3DD.%26aulast%3DJansen%26aufirst%3DJ.%26atitle%3DProfile-QSAR%253A%2520a%2520novel%2520meta-QSAR%2520method%2520that%2520combines%2520activities%2520across%2520the%2520kinase%2520family%2520to%2520accurately%2520predict%2520affinity%252C%2520selectivity%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1942%26epage%3D1956%26doi%3D10.1021%2Fci1005004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span><span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0li8l9duRCibnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laggner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matos, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glennon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, K. L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Predicting new molecular targets for known drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">181</span><span class="refDoi"> DOI: 10.1038/nature08506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnature08506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=19881490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=175-181&author=M.+J.+Keiserauthor=V.+Setolaauthor=J.+J.+Irwinauthor=C.+Laggnerauthor=A.+I.+Abbasauthor=S.+J.+Hufeisenauthor=N.+H.+Jensenauthor=M.+B.+Kuijerauthor=R.+C.+Matosauthor=T.+B.+Tranauthor=R.+Whaleyauthor=R.+A.+Glennonauthor=J.+Hertauthor=K.+L.+H.+Thomasauthor=D.+D.+Edwardsauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Predicting+new+molecular+targets+for+known+drugs&doi=10.1038%2Fnature08506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting new molecular targets for known drugs</span></div><div class="casAuthors">Keiser, Michael J.; Setola, Vincent; Irwin, John J.; Laggner, Christian; Abbas, Atheir I.; Hufeisen, Sandra J.; Jensen, Niels H.; Kuijer, Michael B.; Matos, Roberto C.; Tran, Thuy B.; Whaley, Ryan; Glennon, Richard A.; Hert, Jerome; Thomas, Kelan L. H.; Edwards, Douglas D.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although drugs are intended to be selective, at least some bind to several physiol. targets, explaining side effects and efficacy.  Because many drug-target combinations exist, it would be useful to explore possible interactions computationally.  Here we compared 3665 US Food and Drug Administration (FDA)-approved and investigational drugs against hundreds of targets, defining each target by its ligands.  Chem. similarities between drugs and ligand sets predicted thousands of unanticipated assocns.  Thirty were tested exptl., including the antagonism of the β1 receptor by the transporter inhibitor Prozac, the inhibition of the 5-hydroxytryptamine (5-HT) transporter by the ion channel drug Vadilex, and antagonism of the histamine H4 receptor by the enzyme inhibitor Rescriptor.  Overall, 23 new drug-target assocns. were confirmed, five of which were potent (<100 nM).  The physiol. relevance of one, the drug N,N-dimethyltryptamine (DMT) on serotonergic receptors, was confirmed in a knockout mouse.  The chem. similarity approach is systematic and comprehensive, and may suggest side-effects and new indications for many drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozd-8cCXUskrVg90H21EOLACvtfcHk0li8l9duRCibnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO&md5=974e65f5d5b4dabde1a463c429215546</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature08506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08506%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DLaggner%26aufirst%3DC.%26aulast%3DAbbas%26aufirst%3DA.%2BI.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DJensen%26aufirst%3DN.%2BH.%26aulast%3DKuijer%26aufirst%3DM.%2BB.%26aulast%3DMatos%26aufirst%3DR.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BB.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DGlennon%26aufirst%3DR.%2BA.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DK.%2BL.%2BH.%26aulast%3DEdwards%26aufirst%3DD.%2BD.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPredicting%2520new%2520molecular%2520targets%2520for%2520known%2520drugs%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D175%26epage%3D181%26doi%3D10.1038%2Fnature08506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Manallack, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gancia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montana, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingstone, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitley, D. C.</span><span> </span><span class="NLM_article-title">Selecting screening candidates for kinase and G protein-coupled receptor targets using neural networks</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1256</span><span class="NLM_x">–</span> <span class="NLM_lpage">1262</span><span class="refDoi"> DOI: 10.1021/ci020267c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci020267c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1256-1262&author=D.+T.+Manallackauthor=W.+R.+Pittauthor=E.+Ganciaauthor=J.+G.+Montanaauthor=D.+J.+Livingstoneauthor=M.+G.+Fordauthor=D.+C.+Whitley&title=Selecting+screening+candidates+for+kinase+and+G+protein-coupled+receptor+targets+using+neural+networks&doi=10.1021%2Fci020267c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci020267c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci020267c%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGancia%26aufirst%3DE.%26aulast%3DMontana%26aufirst%3DJ.%2BG.%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26atitle%3DSelecting%2520screening%2520candidates%2520for%2520kinase%2520and%2520G%2520protein-coupled%2520receptor%2520targets%2520using%2520neural%2520networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2002%26volume%3D42%26spage%3D1256%26epage%3D1262%26doi%3D10.1021%2Fci020267c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Ning, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karypisxy, G.</span><span> </span><span class="NLM_article-title">Improved machine learning models for predicting selective compounds</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">38</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1021/ci200346b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200346b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2kt77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=38-50&author=X.+Ningauthor=M.+Waltersauthor=G.+Karypisxy&title=Improved+machine+learning+models+for+predicting+selective+compounds&doi=10.1021%2Fci200346b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Machine Learning Models for Predicting Selective Compounds</span></div><div class="casAuthors">Ning, Xia; Walters, Michael; Karypis, George</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-50</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of small potent compds. that selectively bind to the target under consideration with high affinities is a crit. step toward successful drug discovery.  However, there is still a lack of efficient and accurate computational methods to predict compd. selectivity properties.  In this paper, we propose a set of machine learning methods to do compd. selectivity prediction.  In particular, we propose a novel cascaded learning method and a multitask learning method.  The cascaded method decomps. the selectivity prediction into two steps, one model for each step, so as to effectively filter out nonselective compds.  The multitask method incorporates both activity and selectivity models into one multitask model so as to better differentiate compd. selectivity properties.  We conducted a comprehensive set of expts. and compared the results with those of other conventional selectivity prediction methods, and our results demonstrated that the cascaded and multitask methods significantly improve the selectivity prediction performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdkf0ZGDNBm7Vg90H21EOLACvtfcHk0licmJ7kWAWrqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2kt77I&md5=2038119451a4a486b77457f966276e5e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci200346b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200346b%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DX.%26aulast%3DWalters%26aufirst%3DM.%26aulast%3DKarypisxy%26aufirst%3DG.%26atitle%3DImproved%2520machine%2520learning%2520models%2520for%2520predicting%2520selective%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D38%26epage%3D50%26doi%3D10.1021%2Fci200346b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Unterthiner, T.; Mayr, A.; Klambauer, G.; Steijaert, M.; Wegner, J. K.; Ceulemans, H.; Hochreiter, S.</span><span> </span><span class="NLM_article-title">Deep learning as an opportunity in virtual screening</span>. Proceedings of the Deep Learning Workshop at NIPS, Montreal (QC), December 12, 2014;<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=T.+Unterthiner&author=A.+Mayr&author=G.+Klambauer&author=M.+Steijaert&author=J.+K.+Wegner&author=H.+Ceulemans&author=S.+Hochreiter&title=Deep+learning+as+an+opportunity+in+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DUnterthiner%26aufirst%3DT.%26atitle%3DDeep%2520learning%2520as%2520an%2520opportunity%2520in%2520virtual%2520screening%26jtitle%3DProceedings%2520of%2520the%2520Deep%2520Learning%2520Workshop%2520at%2520NIPS%252C%2520Montreal%2520%2528QC%2529%252C%2520December%252012%252C%25202014%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Yabuuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niijima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takematsu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirokawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minowa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujimoto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuno, Y.</span><span> </span><span class="NLM_article-title">Analysis of multiple compound-protein interactions reveals novel bioactive molecules</span> <span class="citation_source-journal">Mol. Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">472</span><span class="refDoi"> DOI: 10.1038/msb.2011.5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fmsb.2011.5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=472&author=H.+Yabuuchiauthor=S.+Niijimaauthor=H.+Takematsuauthor=T.+Idaauthor=T.+Hirokawaauthor=T.+Haraauthor=T.+Ogawaauthor=Y.+Minowaauthor=G.+Tsujimotoauthor=Y.+Okuno&title=Analysis+of+multiple+compound-protein+interactions+reveals+novel+bioactive+molecules&doi=10.1038%2Fmsb.2011.5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2011.5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2011.5%26sid%3Dliteratum%253Aachs%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DTakematsu%26aufirst%3DH.%26aulast%3DIda%26aufirst%3DT.%26aulast%3DHirokawa%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DOgawa%26aufirst%3DT.%26aulast%3DMinowa%26aufirst%3DY.%26aulast%3DTsujimoto%26aufirst%3DG.%26aulast%3DOkuno%26aufirst%3DY.%26atitle%3DAnalysis%2520of%2520multiple%2520compound-protein%2520interactions%2520reveals%2520novel%2520bioactive%2520molecules%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D472%26doi%3D10.1038%2Fmsb.2011.5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J. H.</span><span> </span><span class="NLM_article-title">Comparison of random forest and Pipeline Pilot naive bayes in prospective QSAR predictions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">792</span><span class="NLM_x">–</span> <span class="NLM_lpage">803</span><span class="refDoi"> DOI: 10.1021/ci200615h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200615h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38XislajtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=792-803&author=B.+Chenauthor=R.+P.+Sheridanauthor=V.+Hornakauthor=J.+H.+Voigt&title=Comparison+of+random+forest+and+Pipeline+Pilot+naive+bayes+in+prospective+QSAR+predictions&doi=10.1021%2Fci200615h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Random Forest and Pipeline Pilot Naive Bayes in Prospective QSAR Predictions</span></div><div class="casAuthors">Chen, Bin; Sheridan, Robert P.; Hornak, Viktor; Voigt, Johannes H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">792-803</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Random forest is currently considered one of the best QSAR methods available in terms of accuracy of prediction.  However, it is computationally intensive.  Naive Bayes is a simple, robust classification method.  The Laplacian-modified Naive Bayes implementation is the preferred QSAR method in the widely used com. chemoinformatics platform Pipeline Pilot.  We made a comparison of the ability of Pipeline Pilot Naive Bayes (PLPNB) and random forest to make accurate predictions on 18 large, diverse inhouse QSAR data sets.  These include on-target and ADME-related activities.  These data sets were set up as classification problems with either binary or multicategory activities.  We used a time-split method of dividing training and test sets, as we feel this is a realistic way of simulating prospective prediction.  PLPNB is computationally efficient.  However, random forest predictions are at least as good and in many cases significantly better than those of PLPNB on our data sets.  PLPNB performs better with ECFP4 and ECFP6 descriptors, which are native to Pipeline Pilot, and more poorly with other descriptors we tried.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopM-mDzNRvcLVg90H21EOLACvtfcHk0licmJ7kWAWrqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislajtbY%253D&md5=63488794c7eebebed1f34c2e6989e104</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci200615h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200615h%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DVoigt%26aufirst%3DJ.%2BH.%26atitle%3DComparison%2520of%2520random%2520forest%2520and%2520Pipeline%2520Pilot%2520naive%2520bayes%2520in%2520prospective%2520QSAR%2520predictions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D792%26epage%3D803%26doi%3D10.1021%2Fci200615h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Schürer, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muskal, S. M.</span><span> </span><span class="NLM_article-title">Kinome-wide activity modeling from diverse public high-quality data sets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1021/ci300403k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300403k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVygu77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=27-38&author=S.+C.+Sch%C3%BCrerauthor=S.+M.+Muskal&title=Kinome-wide+activity+modeling+from+diverse+public+high-quality+data+sets&doi=10.1021%2Fci300403k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Kinome-wide Activity Modeling from Diverse Public High-Quality Data Sets</span></div><div class="casAuthors">Schurer, Stephan C.; Muskal, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Large corpora of kinase small mol. inhibitor data are accessible to public sector research from thousands of journal article and patent publications.  These data have been generated employing a wide variety of assay methodologies and exptl. procedures by numerous labs.  Here we ask the question how applicable these heterogeneous data sets are to predict kinase activities and which characteristics of the data sets contribute to their utility.  We accessed almost 500 000 mols. from the Kinase Knowledge Base (KKB) and after rigorous aggregation and standardization generated over 180 distinct data sets covering all major groups of the human kinome.  To assess the value of the data sets, we generated hundreds of classification and regression models.  Their rigorous cross-validation and characterization demonstrated highly predictive classification and quant. models for the majority of kinase targets if a min. required no. of active compds. or structure-activity data points were available.  We then applied the best classifiers to compds. most recently profiled in the NIH Library of Integrated Network-based Cellular Signatures (LINCS) program and found good agreement of profiling results with predicted activities.  Our results indicate that, although heterogeneous in nature, the publically accessible data sets are exceedingly valuable and well suited to develop highly accurate predictors for practical Kinome-wide virtual screening applications and to complement exptl. kinase profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1rhwNqRlmD7Vg90H21EOLACvtfcHk0limviTlnUzySg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVygu77L&md5=cea0a7b0c22b43e7660b430e8039f4e1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci300403k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300403k%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCrer%26aufirst%3DS.%2BC.%26aulast%3DMuskal%26aufirst%3DS.%2BM.%26atitle%3DKinome-wide%2520activity%2520modeling%2520from%2520diverse%2520public%2520high-quality%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D27%26epage%3D38%26doi%3D10.1021%2Fci300403k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Breiman, L.</span><span> </span><span class="NLM_article-title">Random forests</span> <span class="citation_source-journal">Machine Learning</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1023/A:1010933404324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1023%2FA%3A1010933404324" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2001&pages=5-32&author=L.+Breiman&title=Random+forests&doi=10.1023%2FA%3A1010933404324"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1023%2FA%3A1010933404324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1010933404324%26sid%3Dliteratum%253Aachs%26aulast%3DBreiman%26aufirst%3DL.%26atitle%3DRandom%2520forests%26jtitle%3DMachine%2520Learning%26date%3D2001%26volume%3D45%26spage%3D5%26epage%3D32%26doi%3D10.1023%2FA%3A1010933404324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Svetnik, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liaw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culberson, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feuston, B. P.</span><span> </span><span class="NLM_article-title">Random forest: a classification and regression tool for compound classification and QSAR modeling</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1947</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span><span class="refDoi"> DOI: 10.1021/ci034160g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci034160g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos1Wiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=1947-1958&author=V.+Svetnikauthor=A.+Liawauthor=C.+Tongauthor=J.+C.+Culbersonauthor=R.+P.+Sheridanauthor=B.+P.+Feuston&title=Random+forest%3A+a+classification+and+regression+tool+for+compound+classification+and+QSAR+modeling&doi=10.1021%2Fci034160g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Random Forest: A Classification and Regression Tool for Compound Classification and QSAR Modeling</span></div><div class="casAuthors">Svetnik, Vladimir; Liaw, Andy; Tong, Christopher; Culberson, J. Christopher; Sheridan, Robert P.; Feuston, Bradley P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1947-1958</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new classification and regression tool, Random Forest, is introduced and investigated for predicting a compd.'s quant. or categorical biol. activity based on a quant. description of the compd.'s mol. structure.  Random Forest is an ensemble of unpruned classification or regression trees created by using bootstrap samples of the training data and random feature selection in tree induction.  Prediction is made by aggregating (majority vote or averaging) the predictions of the ensemble.  The authors built predictive models for six cheminformatics data sets.  The authors anal. demonstrates that Random Forest is a powerful tool capable of delivering performance that is among the most accurate methods to date.  The authors also present three addnl. features of Random Forest: built-in performance assessment, a measure of relative importance of descriptors, and a measure of compd. similarity that is weighted by the relative importance of descriptors.  It is the combination of relatively high prediction accuracy and its collection of desired features that makes Random Forest uniquely suited for modeling in cheminformatics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjoRt5v7rYwLVg90H21EOLACvtfcHk0limviTlnUzySg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos1Wiu7s%253D&md5=dea7867551ec30260b0091b90593a660</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci034160g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034160g%26sid%3Dliteratum%253Aachs%26aulast%3DSvetnik%26aufirst%3DV.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DC.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DFeuston%26aufirst%3DB.%2BP.%26atitle%3DRandom%2520forest%253A%2520a%2520classification%2520and%2520regression%2520tool%2520for%2520compound%2520classification%2520and%2520QSAR%2520modeling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2003%26volume%3D43%26spage%3D1947%26epage%3D1958%26doi%3D10.1021%2Fci034160g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Goldstein, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">High-throughput kinase profiling as a platform for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">391</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span><span class="refDoi"> DOI: 10.1038/nrd2541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnrd2541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=18404149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=391-397&author=D.+M.+Goldsteinauthor=N.+S.+Grayauthor=P.+P.+Zarrinkar&title=High-throughput+kinase+profiling+as+a+platform+for+drug+discovery&doi=10.1038%2Fnrd2541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput kinase profiling as a platform for drug discovery</span></div><div class="casAuthors">Goldstein, David M.; Gray, Nathanael S.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">391-397</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput profiling of compd. libraries against large panels of kinases is becoming tech. feasible.  In contrast to the traditional linear, target-centric approach to discovery, this approach may provide a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone, and could thereby significantly improve the productivity of kinase inhibitor discovery.  To fully exploit the potential of kinases as drug targets, novel strategies for the efficient discovery of inhibitors are required.  In contrast to the traditional, linear process of inhibitor discovery, high-throughput kinase profiling enables a parallel approach by interrogating compds. against hundreds of targets in a single screen.  Compd. potency and selectivity are detd. simultaneously, providing a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwnJ59cVxbJbVg90H21EOLACvtfcHk0limviTlnUzySg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D&md5=eaa61fd2958ff04fecf86a993f91833d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd2541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2541%26sid%3Dliteratum%253Aachs%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DHigh-throughput%2520kinase%2520profiling%2520as%2520a%2520platform%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D391%26epage%3D397%26doi%3D10.1038%2Fnrd2541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Cortés-Ciriano, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ain, Q. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanian, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenselink, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Méndez-Lucio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">IJzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlfahrt, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malliavin, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Westen, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span> </span><span class="NLM_article-title">Polypharmacology modelling using proteochemometrics (PCM): recent methodological developments, applications to target families, and future prospects</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">24</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1039/C4MD00216D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1039%2FC4MD00216D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslagsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=24-50&author=I.+Cort%C3%A9s-Cirianoauthor=Q.+U.+Ainauthor=V.+Subramanianauthor=E.+B.+Lenselinkauthor=O.+M%C3%A9ndez-Lucioauthor=A.+P.+IJzermanauthor=G.+Wohlfahrtauthor=P.+Prusisauthor=T.+E.+Malliavinauthor=G.+J.+van+Westenauthor=A.+Bender&title=Polypharmacology+modelling+using+proteochemometrics+%28PCM%29%3A+recent+methodological+developments%2C+applications+to+target+families%2C+and+future+prospects&doi=10.1039%2FC4MD00216D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology modelling using proteochemometrics (PCM): recent methodological developments, applications to target families, and future prospects</span></div><div class="casAuthors">Cortes-Ciriano, Isidro; Ain, Qurrat Ul; Subramanian, Vigneshwari; Lenselink, Eelke B.; Mendez-Lucio, Oscar; IJzerman, Adriaan P.; Wohlfahrt, Gerd; Prusis, Peteris; Malliavin, Therese E.; van Westen, Gerard J. P.; Bender, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-50</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Proteochemometric (PCM) modeling is a computational method to model the bioactivity of multiple ligands against multiple related protein targets simultaneously.  Hence it has been found to be particularly useful when exploring the selectivity and promiscuity of ligands on different proteins.  In this review, we will firstly provide a brief introduction to the main concepts of PCM for readers new to the field.  The next part focuses on recent tech. advances, including the application of support vector machines (SVMs) using different kernel functions, random forests, Gaussian processes and collaborative filtering.  The subsequent section will then describe some novel practical applications of PCM in the medicinal chem. field, including studies on GPCRs, kinases, viral proteins (e.g. from HIV) and epigenetic targets such as histone deacetylases.  Finally, we will conclude by summarizing novel developments in PCM, which we expect to gain further importance in the future.  These developments include adding three-dimensional protein target information, application of PCM to the prediction of binding energies, and application of the concept in the fields of pharmacogenomics and toxicogenomics.  This review is an update to a related publication in 2011 and it mainly focuses on developments in the field since then.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEtwphLWc6brVg90H21EOLACvtfcHk0ljfrF71rVxgVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslagsLrE&md5=b3e82734e224ae4990218b6473ef7859</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2FC4MD00216D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00216D%26sid%3Dliteratum%253Aachs%26aulast%3DCort%25C3%25A9s-Ciriano%26aufirst%3DI.%26aulast%3DAin%26aufirst%3DQ.%2BU.%26aulast%3DSubramanian%26aufirst%3DV.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DM%25C3%25A9ndez-Lucio%26aufirst%3DO.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DWohlfahrt%26aufirst%3DG.%26aulast%3DPrusis%26aufirst%3DP.%26aulast%3DMalliavin%26aufirst%3DT.%2BE.%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DPolypharmacology%2520modelling%2520using%2520proteochemometrics%2520%2528PCM%2529%253A%2520recent%2520methodological%2520developments%252C%2520applications%2520to%2520target%2520families%252C%2520and%2520future%2520prospects%26jtitle%3DMedChemComm%26date%3D2015%26volume%3D6%26spage%3D24%26epage%3D50%26doi%3D10.1039%2FC4MD00216D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Zakharov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peach, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitzmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicklaus, M. C.</span><span> </span><span class="NLM_article-title">QSAR modeling of imbalanced high-throughput screening data in PubChem</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">705</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span><span class="refDoi"> DOI: 10.1021/ci400737s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400737s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlartb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=705-712&author=A.+V.+Zakharovauthor=M.+L.+Peachauthor=M.+Sitzmannauthor=M.+C.+Nicklaus&title=QSAR+modeling+of+imbalanced+high-throughput+screening+data+in+PubChem&doi=10.1021%2Fci400737s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Modeling of Imbalanced High-Throughput Screening Data in PubChem</span></div><div class="casAuthors">Zakharov, Alexey V.; Peach, Megan L.; Sitzmann, Markus; Nicklaus, Marc C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">705-712</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many of the structures in PubChem are annotated with activities detd. in high-throughput screening (HTS) assays.  Because of the nature of these assays, the activity data are typically strongly imbalanced, with a small no. of active compds. contrasting with a very large no. of inactive compds.  We have used several such imbalanced PubChem HTS assays to test and develop strategies to efficiently build robust QSAR models from imbalanced data sets.  Different descriptor types [Quant. Neighborhoods of Atoms (QNA) and "biol." descriptors] were used to generate a variety of QSAR models in the program GUSAR.  The models obtained were compared using external test and validation sets.  We also report on our efforts to incorporate the most predictive of our models in the publicly available NCI/CADD Group Web services (http://cactus.nci.nih.gov/chem./apps/cap).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq51slAln8D2LVg90H21EOLACvtfcHk0ljfrF71rVxgVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlartb0%253D&md5=f2c9ba5e2843c4ef3713ac6fb3aafbed</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci400737s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400737s%26sid%3Dliteratum%253Aachs%26aulast%3DZakharov%26aufirst%3DA.%2BV.%26aulast%3DPeach%26aufirst%3DM.%2BL.%26aulast%3DSitzmann%26aufirst%3DM.%26aulast%3DNicklaus%26aufirst%3DM.%2BC.%26atitle%3DQSAR%2520modeling%2520of%2520imbalanced%2520high-throughput%2520screening%2520data%2520in%2520PubChem%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D705%26epage%3D712%26doi%3D10.1021%2Fci400737s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Gao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, U. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovelman, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, K. J.</span><span> </span><span class="NLM_article-title">A broad activity screen in support of a chemogenomic map for kinase signalling research and drug discovery</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">451</span><span class="NLM_x">, </span> <span class="NLM_fpage">313</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.1042/BJ20121418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1042%2FBJ20121418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=23398362" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvFOgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2013&pages=313-328&author=Y.+Gaoauthor=S.+P.+Daviesauthor=M.+Augustinauthor=A.+Woodwardauthor=U.+A.+Patelauthor=R.+Kovelmanauthor=K.+J.+Harvey&title=A+broad+activity+screen+in+support+of+a+chemogenomic+map+for+kinase+signalling+research+and+drug+discovery&doi=10.1042%2FBJ20121418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A broad activity screen in support of a chemogenomic map for kinase signalling research and drug discovery</span></div><div class="casAuthors">Gao, Yinghong; Davies, Stephen P.; Augustin, Martin; Woodward, Anna; Patel, Umesh A.; Kovelman, Robert; Harvey, Kevin J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-328</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Despite the development of a no. of efficacious kinase inhibitors, the strategies for rational design of these compds. have been limited by target promiscuity.  In an effort to better understand the nature of kinase inhibition across the kinome, esp. as it relates to off-target effects, we screened a well-defined collection of kinase inhibitors using biochem. assays for inhibitory activity against 234 active human kinases and kinase complexes, representing all branches of the kinome tree.  For our study we employed 158 small mols. initially identified in the literature as potent and specific inhibitors of kinases important as therapeutic targets and/or signal transduction regulators.  Hierarchical clustering of these benchmark kinase inhibitors on the basis of their kinome activity profiles illustrates how they relate to chem. structure similarities and provides new insights into inhibitor specificity and potential applications for probing new targets.  Using this broad dataset, we provide a framework for assessing polypharmacol.  We not only discover likely off-target inhibitor activities and recommend specific inhibitors for existing targets, but also identify potential new uses for known small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNLU0ONM4SALVg90H21EOLACvtfcHk0ljfrF71rVxgVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvFOgs7g%253D&md5=9e0f17ec5cac10198fe28b27e70b4966</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FBJ20121418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121418%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DY.%26aulast%3DDavies%26aufirst%3DS.%2BP.%26aulast%3DAugustin%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DU.%2BA.%26aulast%3DKovelman%26aufirst%3DR.%26aulast%3DHarvey%26aufirst%3DK.%2BJ.%26atitle%3DA%2520broad%2520activity%2520screen%2520in%2520support%2520of%2520a%2520chemogenomic%2520map%2520for%2520kinase%2520signalling%2520research%2520and%2520drug%2520discovery%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D451%26spage%3D313%26epage%3D328%26doi%3D10.1042%2FBJ20121418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Dranchak, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacArthur, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Profile of the GSK published protein kinase inhibitor set across ATP-dependent and-independent luciferases: implications for reporter-gene assays</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e57888</span><span class="refDoi"> DOI: 10.1371/journal.pone.0057888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1371%2Fjournal.pone.0057888" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e57888&author=P.+Dranchakauthor=R.+MacArthurauthor=R.+Guhaauthor=W.+J.+Zuercherauthor=D.+H.+Drewryauthor=D.+S.+Auldauthor=J.+Inglese&title=Profile+of+the+GSK+published+protein+kinase+inhibitor+set+across+ATP-dependent+and-independent+luciferases%3A+implications+for+reporter-gene+assays&doi=10.1371%2Fjournal.pone.0057888"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0057888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0057888%26sid%3Dliteratum%253Aachs%26aulast%3DDranchak%26aufirst%3DP.%26aulast%3DMacArthur%26aufirst%3DR.%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DProfile%2520of%2520the%2520GSK%2520published%2520protein%2520kinase%2520inhibitor%2520set%2520across%2520ATP-dependent%2520and-independent%2520luciferases%253A%2520implications%2520for%2520reporter-gene%2520assays%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De57888%26doi%3D10.1371%2Fjournal.pone.0057888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arruda, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillespie, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackey, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzafera, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomkinson, N. C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">A public-private partnership to unlock the untargeted kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1038/nchembio.1113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnchembio.1113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=23238671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVekur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3-6&author=S.+Knappauthor=P.+Arrudaauthor=J.+Blaggauthor=S.+Burleyauthor=D.+H.+Drewryauthor=A.+Edwardsauthor=D.+Fabbroauthor=P.+Gillespieauthor=N.+S.+Grayauthor=B.+Kusterauthor=K.+E.+Lackeyauthor=P.+Mazzaferaauthor=N.+C.+O.+Tomkinsonauthor=T.+M.+Willsonauthor=P.+Workmanauthor=W.+J.+Zuercher&title=A+public-private+partnership+to+unlock+the+untargeted+kinome&doi=10.1038%2Fnchembio.1113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A public-private partnership to unlock the untargeted kinome</span></div><div class="casAuthors">Knapp, Stefan; Arruda, Paulo; Blagg, Julian; Burley, Stephen; Drewry, David H.; Edwards, Aled; Fabbro, Doriano; Gillespie, Paul; Gray, Nathanael S.; Kuster, Bernhard; Lackey, Karen E.; Mazzafera, Paulo; Tomkinson, Nicholas C. O.; Willson, Timothy M.; Workman, Paul; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-6</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are urgently needed to functionally annotate hitherto-untargeted kinases and stimulate new drug discovery efforts to address unmet medical needs.  The size of the human kinome combined with the high cost assocd. with probe generation severely limits access to new probes.  We propose a large-scale public-private partnership as a new approach that offers economies of scale, minimized redundancy and sharing of risk and cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8PARtv7U7v7Vg90H21EOLACvtfcHk0lg5cSh2JYMnHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVekur%252FF&md5=1f6c13ec461f96201838934cb95012c0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1113%26sid%3Dliteratum%253Aachs%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DArruda%26aufirst%3DP.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBurley%26aufirst%3DS.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGillespie%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DLackey%26aufirst%3DK.%2BE.%26aulast%3DMazzafera%26aufirst%3DP.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DA%2520public-private%2520partnership%2520to%2520unlock%2520the%2520untargeted%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D3%26epage%3D6%26doi%3D10.1038%2Fnchembio.1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Elkins, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedele, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szklarz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdul Azeez, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salah, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikolajczyk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sepetov, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al Haj Zen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fourches, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muratov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teicher, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunkel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polley, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackey, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkinson, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">Comprehensive characterization of the Published Kinase Inhibitor Set</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">95</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1038/nbt.3374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt.3374" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=95-103&author=J.+M.+Elkinsauthor=V.+Fedeleauthor=M.+Szklarzauthor=K.+R.+Abdul+Azeezauthor=E.+Salahauthor=J.+Mikolajczykauthor=S.+Romanovauthor=N.+Sepetovauthor=X.-P.+Huangauthor=B.+L.+Rothauthor=A.+Al+Haj+Zenauthor=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropshaauthor=J.+Morrisauthor=B.+A.+Teicherauthor=M.+Kunkelauthor=E.+Polleyauthor=K.+E.+Lackeyauthor=F.+L.+Atkinsonauthor=J.+P.+Overingtonauthor=P.+Bamboroughauthor=S.+M%C3%BCllerauthor=D.+J.+Priceauthor=T.+M.+Willsonauthor=D.+H.+Drewryauthor=S.+Knappauthor=W.+J.+Zuercher&title=Comprehensive+characterization+of+the+Published+Kinase+Inhibitor+Set&doi=10.1038%2Fnbt.3374"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3374%26sid%3Dliteratum%253Aachs%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR.%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DMikolajczyk%26aufirst%3DJ.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DSepetov%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DAl%2BHaj%2BZen%26aufirst%3DA.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DTeicher%26aufirst%3DB.%2BA.%26aulast%3DKunkel%26aufirst%3DM.%26aulast%3DPolley%26aufirst%3DE.%26aulast%3DLackey%26aufirst%3DK.%2BE.%26aulast%3DAtkinson%26aufirst%3DF.%2BL.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DComprehensive%2520characterization%2520of%2520the%2520Published%2520Kinase%2520Inhibitor%2520Set%26jtitle%3DNat.%2520Biotechnol.%26date%3D2015%26volume%3D34%26spage%3D95%26epage%3D103%26doi%3D10.1038%2Fnbt.3374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eid, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaeger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fulle, S.</span><span> </span><span class="NLM_article-title">Pocketome of human kinases: prioritizing the ATP binding sites of (yet) untapped protein kinases for drug discovery</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">549</span><span class="refDoi"> DOI: 10.1021/ci500624s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500624s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A280%3ADC%252BC2MvjsVSksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=538-549&author=A.+Volkamerauthor=S.+Eidauthor=S.+Turkauthor=S.+Jaegerauthor=F.+Rippmannauthor=S.+Fulle&title=Pocketome+of+human+kinases%3A+prioritizing+the+ATP+binding+sites+of+%28yet%29+untapped+protein+kinases+for+drug+discovery&doi=10.1021%2Fci500624s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pocketome of human kinases: prioritizing the ATP binding sites of (yet) untapped protein kinases for drug discovery</span></div><div class="casAuthors">Volkamer Andrea; Eid Sameh; Turk Samo; Jaeger Sabrina; Fulle Simone; Rippmann Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">538-49</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein kinases are involved in a variety of diseases including cancer, inflammation, and autoimmune disorders.  Although the development of new kinase inhibitors is a major focus in pharmaceutical research, a large number of kinases remained so far unexplored in drug discovery projects.  The selection and assessment of targets is an essential but challenging area.  Today, a few thousands of experimentally determined kinase structures are available, covering about half of the human kinome.  This large structural source allows guiding the target selection via structure-based druggability prediction approaches such as DoGSiteScorer.  Here, a thorough analysis of the ATP pockets of the entire human kinome in the DFG-in state is presented in order to prioritize novel kinase structures for drug discovery projects.  For this, all human kinase X-ray structures available in the PDB were collected, and homology models were generated for the missing part of the kinome.  DoGSiteScorer was used to calculate geometrical and physicochemical properties of the ATP pockets and to predict the potential of each kinase to be druggable.  The results indicate that about 75% of the kinome are in principle druggable.  Top ranking structures comprise kinases that are primary targets of known approved drugs but additionally point to so far less explored kinases.  The presented analysis provides new insights into the druggability of ATP binding pockets of the entire kinome.  We anticipate this comprehensive druggability assessment of protein kinases to be helpful for the community to prioritize so far untapped kinases for drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXYhJ2uLp2lX6HPRus0rAafW6udTcc2eY1l1BWaczeGrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvjsVSksw%253D%253D&md5=85c8e48d59f1acea82d1019da51b47e1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci500624s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500624s%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DEid%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DJaeger%26aufirst%3DS.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DPocketome%2520of%2520human%2520kinases%253A%2520prioritizing%2520the%2520ATP%2520binding%2520sites%2520of%2520%2528yet%2529%2520untapped%2520protein%2520kinases%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D538%26epage%3D549%26doi%3D10.1021%2Fci500624s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Kalliokoski, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vulpetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gedeck, P.</span><span> </span><span class="NLM_article-title">Comparability of mixed IC<sub>50</sub> data-a statistical analysis</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e61007</span><span class="refDoi"> DOI: 10.1371/journal.pone.0061007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1371%2Fjournal.pone.0061007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=23613770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslSitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e61007&author=T.+Kalliokoskiauthor=C.+Kramerauthor=A.+Vulpettiauthor=P.+Gedeck&title=Comparability+of+mixed+IC50+data-a+statistical+analysis&doi=10.1371%2Fjournal.pone.0061007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Comparability of mixed IC50 data - a statistical analysis</span></div><div class="casAuthors">Kalliokoski, Tuomo; Kramer, Christian; Vulpetti, Anna; Gedeck, Peter</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e61007</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The biochem. half maximal inhibitory concn. (IC50) is the most commonly used metric for on-target activity in lead optimization.  It is used to guide lead optimization, build large-scale chemogenomics anal., off-target activity and toxicity models based on public data.  However, the use of public biochem. IC50 data is problematic, because they are assay specific and comparable only under certain conditions.  For large scale anal. it is not feasible to check each data entry manually and it is very tempting to mix all available IC50 values from public database even if assay information is not reported.  As previously reported for Ki database anal., we first analyzed the types of errors, the redundancy and the variability that can be found in ChEMBL IC50 database.  For assessing the variability of IC50 data independently measured in two different labs at least ten IC50 data for identical protein-ligand systems against the same target were searched in ChEMBL.  As a not sufficient no. of cases of this type are available, the variability of IC50 data was assessed by comparing all pairs of independent IC50 measurements on identical protein-ligand systems.  The std. deviation of IC50 data is only 25% larger than the std. deviation of Ki data, suggesting that mixing IC50 data from different assays, even not knowing assay conditions details, only adds a moderate amt. of noise to the overall data.  The std. deviation of public ChEMBL IC50 data, as expected, resulted greater than the std. deviation of inhouse intra-lab./inter-day IC50 data.  Augmenting mixed public IC50 data by public Ki data does not deteriorate the quality of the mixed IC50 data, if the Ki is cor. by an offset.  For a broad dataset such as ChEMBL database a Ki-IC50 conversion factor of 2 was found to be the most reasonable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq07UjgtQJyZ7Vg90H21EOLACvtfcHk0lgSt0iGSGTN-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslSitrc%253D&md5=591144d98c123366e344688c74156e9e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0061007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0061007%26sid%3Dliteratum%253Aachs%26aulast%3DKalliokoski%26aufirst%3DT.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DGedeck%26aufirst%3DP.%26atitle%3DComparability%2520of%2520mixed%2520IC50%2520data-a%2520statistical%2520analysis%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De61007%26doi%3D10.1371%2Fjournal.pone.0061007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span>R: a Language and Environment for Statistical Computing; <span class="NLM_publisher-name">R Core Team</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span><a href="https://www.R-project.org/" class="extLink">https://www.R-project.org/</a> (accessed February 18, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=R%3A+a+Language+and+Environment+for+Statistical+Computing%3B+R+Core+Team%2C+2015%3B+https%3A%2F%2Fwww.R-project.org%2F+%28accessed+February+18%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DR%253A%2520a%2520Language%2520and%2520Environment%2520for%2520Statistical%2520Computing%26pub%3DR%2520Core%2520Team%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Oksanen, J.; Blanchet, F. G.; Kindt, R.; Legendre, P.; Minchin, P. R.; O’Hara, R. B.; Simpson, G. L.; Solymos, P.; Stevens, M. H. H.; Wagner, H.</span>Vegan: Community Ecology Package;<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><div class="note"><p class="first last">R package version 2.3–4</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+Oksanen&author=F.+G.+Blanchet&author=R.+Kindt&author=P.+Legendre&author=P.+R.+Minchin&author=R.+B.+O%E2%80%99Hara&author=G.+L.+Simpson&author=P.+Solymos&author=M.+H.+H.+Stevens&author=H.+Wagner&title=Vegan%3A+Community+Ecology+Package"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DOksanen%26aufirst%3DJ.%26jtitle%3DVegan%253A%2520Community%2520Ecology%2520Package%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Neuwirth, E.</span>RColorBrewer: Colorbrewer Palettes;<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span><div class="note"><p class="first last">R package version 1.1–2</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=E.+Neuwirth&title=RColorBrewer%3A+Colorbrewer+Palettes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DNeuwirth%26aufirst%3DE.%26jtitle%3DRColorBrewer%253A%2520Colorbrewer%2520Palettes%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Wal, J. V. D.; Falconi, L.; Januchowski, S.; Shoo, L.; Storlie, C.</span>SDMTools: Species Distribution Modelling Tools; Tools for Processing Data Associated with Species Distribution Modelling Exercises,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span><div class="note"><p class="first last">R package version 1.1–221</p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=J.+V.+D.+Wal&author=L.+Falconi&author=S.+Januchowski&author=L.+Shoo&author=C.+Storlie&title=SDMTools%3A+Species+Distribution+Modelling+Tools%3B+Tools+for+Processing+Data+Associated+with+Species+Distribution+Modelling+Exercises"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DWal%26aufirst%3DJ.%2BV.%2BD.%26jtitle%3DSDMTools%253A%2520Species%2520Distribution%2520Modelling%2520Tools%253B%2520Tools%2520for%2520Processing%2520Data%2520Associated%2520with%2520Species%2520Distribution%2520Modelling%2520Exercises%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span>RDKit: Open-Source Cheminformatics Software,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a>, (accessed February 18, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%3A+Open-Source+Cheminformatics+Software%2C+2016%3B+http%3A%2F%2Fwww.rdkit.org%2C+%28accessed+February+18%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DRDKit%253A%2520Open-Source%2520Cheminformatics%2520Software%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Rogers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, M.</span><span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">754</span><span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0lgSt0iGSGTN-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Pedregosa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varoquaux, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gramfort, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michel, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thirion, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grisel, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blondel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prettenhofer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubourg, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanderplas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Passos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cournapeau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brucher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duchesnay, E.</span><span> </span><span class="NLM_article-title"><i>Scikit-learn</i>: machine learning in Python</span> <span class="citation_source-journal">J. Mach. Learn. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2825</span><span class="NLM_x">–</span> <span class="NLM_lpage">2830</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=2825-2830&author=F.+Pedregosaauthor=G.+Varoquauxauthor=A.+Gramfortauthor=V.+Michelauthor=B.+Thirionauthor=O.+Griselauthor=M.+Blondelauthor=P.+Prettenhoferauthor=R.+Weissauthor=V.+Dubourgauthor=J.+Vanderplasauthor=A.+Passosauthor=D.+Cournapeauauthor=M.+Brucherauthor=M.+Perrotauthor=E.+Duchesnay&title=Scikit-learn%3A+machine+learning+in+Python"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPedregosa%26aufirst%3DF.%26aulast%3DVaroquaux%26aufirst%3DG.%26aulast%3DGramfort%26aufirst%3DA.%26aulast%3DMichel%26aufirst%3DV.%26aulast%3DThirion%26aufirst%3DB.%26aulast%3DGrisel%26aufirst%3DO.%26aulast%3DBlondel%26aufirst%3DM.%26aulast%3DPrettenhofer%26aufirst%3DP.%26aulast%3DWeiss%26aufirst%3DR.%26aulast%3DDubourg%26aufirst%3DV.%26aulast%3DVanderplas%26aufirst%3DJ.%26aulast%3DPassos%26aufirst%3DA.%26aulast%3DCournapeau%26aufirst%3DD.%26aulast%3DBrucher%26aufirst%3DM.%26aulast%3DPerrot%26aufirst%3DM.%26aulast%3DDuchesnay%26aufirst%3DE.%26atitle%3DScikit-learn%253A%2520machine%2520learning%2520in%2520Python%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2011%26volume%3D12%26spage%3D2825%26epage%3D2830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Xia, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maliski, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, D.</span><span> </span><span class="NLM_article-title">Classification of kinase inhibitors using a bayesian model</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4463</span><span class="NLM_x">–</span> <span class="NLM_lpage">4470</span><span class="refDoi"> DOI: 10.1021/jm0303195</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0303195" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1Sgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4463-4470&author=X.+Xiaauthor=E.+G.+Maliskiauthor=P.+Gallantauthor=D.+Rogers&title=Classification+of+kinase+inhibitors+using+a+bayesian+model&doi=10.1021%2Fjm0303195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Kinase Inhibitors Using a Bayesian Model</span></div><div class="casAuthors">Xia, Xiaoyang; Maliski, Edward G.; Gallant, Paul; Rogers, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4463-4470</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of Bayesian statistics to model both general (multifamily) and specific (single-target) kinase inhibitors is investigated.  The approach demonstrates an alternative to current computational methods applied to heterogeneous structure/activity data sets.  This approach operates rapidly and is readily modifiable as required.  A generalized model generated using inhibitor data from multiple kinase classes shows meaningful enrichment for several specific kinase targets.  Such an approach can be used to prioritize compds. for screening or to optimally select compds. from third-party data collections.  The obsd. benefit of the approach is finding compds. that are not structurally related to known actives, or novel targets for which there is not enough information to build a specific kinase model.  The general kinase model described was built from a basis of mostly tyrosine kinase inhibitors, with some serine/threonine inhibitors; all the test cases used in prediction were also on tyrosine kinase targets.  Confirming the applicability of this technique to other kinase families will be detd. once those biol. assays become available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrySh6p6L1C8rVg90H21EOLACvtfcHk0lhQPPG6yBM3Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmt1Sgu78%253D&md5=6a7a80edee72fc9d33709a7dfdb7f0b5</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm0303195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303195%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DX.%26aulast%3DMaliski%26aufirst%3DE.%2BG.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DRogers%26aufirst%3DD.%26atitle%3DClassification%2520of%2520kinase%2520inhibitors%2520using%2520a%2520bayesian%2520model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4463%26epage%3D4470%26doi%3D10.1021%2Fjm0303195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Abadi, M.; Agarwal, A.; Barham, P.; Brevdo, E.; Chen, Z.; Citro, C.; Corrado, G. S.; Davis, A.; Dean, J.; Devin, M.; Ghemawat, S.; Goodfellow, I.; Harp, A.; Irving, G.; Isard, M.; Jia, Y.; Jozefowicz, R.; Kaiser, L.; Kudlur, M.; Levenberg, J.; Mané, D.; Monga, R.; Moore, S.; Murray, D.; Olah, C.; Schuster, M.; Shlens, J.; Steiner, B.; Sutskever, I.; Talwar, K.; Tucker, P.; Vanhoucke, V.; Vasudevan, V.; Viégas, F.; Vinyals, O.; Warden, P.; Wattenberg, M.; Wicke, M.; Yu, Y.; Zheng, X.</span><i>TensorFlow</i>: large-scale machine learning on heterogeneous systems.<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span><a href="http://tensorflow.org/" class="extLink">http://tensorflow.org/</a>, accessed February 18, 2016.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Abadi%2C+M.%3B+Agarwal%2C+A.%3B+Barham%2C+P.%3B+Brevdo%2C+E.%3B+Chen%2C+Z.%3B+Citro%2C+C.%3B+Corrado%2C+G.+S.%3B+Davis%2C+A.%3B+Dean%2C+J.%3B+Devin%2C+M.%3B+Ghemawat%2C+S.%3B+Goodfellow%2C+I.%3B+Harp%2C+A.%3B+Irving%2C+G.%3B+Isard%2C+M.%3B+Jia%2C+Y.%3B+Jozefowicz%2C+R.%3B+Kaiser%2C+L.%3B+Kudlur%2C+M.%3B+Levenberg%2C+J.%3B+Man%C3%A9%2C+D.%3B+Monga%2C+R.%3B+Moore%2C+S.%3B+Murray%2C+D.%3B+Olah%2C+C.%3B+Schuster%2C+M.%3B+Shlens%2C+J.%3B+Steiner%2C+B.%3B+Sutskever%2C+I.%3B+Talwar%2C+K.%3B+Tucker%2C+P.%3B+Vanhoucke%2C+V.%3B+Vasudevan%2C+V.%3B+Vi%C3%A9gas%2C+F.%3B+Vinyals%2C+O.%3B+Warden%2C+P.%3B+Wattenberg%2C+M.%3B+Wicke%2C+M.%3B+Yu%2C+Y.%3B+Zheng%2C+X.TensorFlow%3A+large-scale+machine+learning+on+heterogeneous+systems.+2015%3B+http%3A%2F%2Ftensorflow.org%2F%2C+accessed+February+18%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26aulast%3DAbadi%26aufirst%3DM.%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Nair, V.; Hinton, G. E.</span><span> </span><span class="NLM_article-title">Rectified linear units improve restricted boltzmann machines</span>. <i>Proceedings of the 27th International Conference on Machine Learning (ICML-10), Haifa, Isreal, June 21–24, 2010</i>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">807</span><span class="NLM_x">–</span> <span class="NLM_lpage">814</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=807-814&author=V.+Nair&author=G.+E.+Hinton&title=Rectified+linear+units+improve+restricted+boltzmann+machines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DV.%26atitle%3DRectified%2520linear%2520units%2520improve%2520restricted%2520boltzmann%2520machines%26date%3D2010%26spage%3D807%26epage%3D814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Srivastava, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hinton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krizhevsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutskever, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salakhutdinov, R.</span><span> </span><span class="NLM_article-title">Dropout: a simple way to prevent neural networks from overfitting</span> <span class="citation_source-journal">J. Mach. Learn. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1929</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=1929-1958&author=N.+Srivastavaauthor=G.+Hintonauthor=A.+Krizhevskyauthor=I.+Sutskeverauthor=R.+Salakhutdinov&title=Dropout%3A+a+simple+way+to+prevent+neural+networks+from+overfitting"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DN.%26aulast%3DHinton%26aufirst%3DG.%26aulast%3DKrizhevsky%26aufirst%3DA.%26aulast%3DSutskever%26aufirst%3DI.%26aulast%3DSalakhutdinov%26aufirst%3DR.%26atitle%3DDropout%253A%2520a%2520simple%2520way%2520to%2520prevent%2520neural%2520networks%2520from%2520overfitting%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2014%26volume%3D15%26spage%3D1929%26epage%3D1958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Mani, I.; Zhang, I.</span><span> </span><span class="NLM_article-title">kNN approach to unbalanced data distributions: a case study involving information extraction</span>. Proceedings of Workshop on Learning from Imbalanced Datasets, ICML, Washington DC, August 21, 2003.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mani%2C+I.%3B+Zhang%2C+I.+kNN+approach+to+unbalanced+data+distributions%3A+a+case+study+involving+information+extraction.+Proceedings+of+Workshop+on+Learning+from+Imbalanced+Datasets%2C+ICML%2C+Washington+DC%2C+August+21%2C+2003."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DMani%26aufirst%3DI.%26atitle%3DkNN%2520approach%2520to%2520unbalanced%2520data%2520distributions%253A%2520a%2520case%2520study%2520involving%2520information%2520extraction%26jtitle%3DProceedings%2520of%2520Workshop%2520on%2520Learning%2520from%2520Imbalanced%2520Datasets%252C%2520ICML%252C%2520Washington%2520DC%252C%2520August%252021%252C%25202003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Chawla, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowyer, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kegelmeyer, W. P.</span><span> </span><span class="NLM_article-title">SMOTE: synthetic minority over-sampling technique</span> <span class="citation_source-journal">J. Artif. Intell. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_fpage">321</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1613/jair.953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1613%2Fjair.953" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&pages=321-357&author=N.+V.+Chawlaauthor=K.+W.+Bowyerauthor=L.+O.+Hallauthor=W.+P.+Kegelmeyer&title=SMOTE%3A+synthetic+minority+over-sampling+technique&doi=10.1613%2Fjair.953"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1613%2Fjair.953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1613%252Fjair.953%26sid%3Dliteratum%253Aachs%26aulast%3DChawla%26aufirst%3DN.%2BV.%26aulast%3DBowyer%26aufirst%3DK.%2BW.%26aulast%3DHall%26aufirst%3DL.%2BO.%26aulast%3DKegelmeyer%26aufirst%3DW.%2BP.%26atitle%3DSMOTE%253A%2520synthetic%2520minority%2520over-sampling%2520technique%26jtitle%3DJ.%2520Artif.%2520Intell.%2520Res.%26date%3D2002%26spage%3D321%26epage%3D357%26doi%3D10.1613%2Fjair.953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Japkowicz, N.</span><span> </span><span class="NLM_article-title">The class imbalance problem: significance and strategies</span>. Proceedings of the 2000 International Conference on Artificial Intelligence, Las Vegas, Nevada, June 26–29, 2000.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Japkowicz%2C+N.+The+class+imbalance+problem%3A+significance+and+strategies.+Proceedings+of+the+2000+International+Conference+on+Artificial+Intelligence%2C+Las+Vegas%2C+Nevada%2C+June+26%E2%80%9329%2C+2000."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DJapkowicz%26aufirst%3DN.%26atitle%3DThe%2520class%2520imbalance%2520problem%253A%2520significance%2520and%2520strategies%26jtitle%3DProceedings%2520of%2520the%25202000%2520International%2520Conference%2520on%2520Artificial%2520Intelligence%252C%2520Las%2520Vegas%252C%2520Nevada%252C%2520June%252026%25E2%2580%259329%252C%25202000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Exploration of type II binding mode: a privileged approach for kinase inhibitor focused drug discovery?</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1230</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span><span class="refDoi"> DOI: 10.1021/cb500129t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500129t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1230-1241&author=Z.+Zhaoauthor=H.+Wuauthor=L.+Wangauthor=Y.+Liuauthor=S.+Knappauthor=Q.+Liuauthor=N.+S.+Gray&title=Exploration+of+type+II+binding+mode%3A+a+privileged+approach+for+kinase+inhibitor+focused+drug+discovery%3F&doi=10.1021%2Fcb500129t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Type II Binding Mode: A Privileged Approach for Kinase Inhibitor Focused Drug Discovery?</span></div><div class="casAuthors">Zhao, Zheng; Wu, Hong; Wang, Li; Liu, Yi; Knapp, Stefan; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1230-1241</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ATP site of kinases displays remarkable conformational flexibility when accommodating chem. diverse small mol. inhibitors.  The so-called activation segment, whose conformation controls catalytic activity and access to the substrate binding pocket, can undergo a large conformational change with the active state assuming a 'DFG-in' and an inactive state assuming a 'DFG-out' conformation.  Compds. that preferentially bind to the DFG-out conformation are typically called 'type II' inhibitors in contrast to 'type I' inhibitors that bind to the DFG-in conformation.  This review surveys the large no. of type II inhibitors that have been developed and provides an anal. of their crystallog. detd. binding modes.  Using a small library of type II inhibitors, we demonstrate that more than 200 kinases can be targeted, suggesting that type II inhibitors may not be intrinsically more selective than type I inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4jYt6Db3Ap7Vg90H21EOLACvtfcHk0lg3p8-WJEPxcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2nt7s%253D&md5=95bf185ff7f973eda5a65f4096bfaa82</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fcb500129t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500129t%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DExploration%2520of%2520type%2520II%2520binding%2520mode%253A%2520a%2520privileged%2520approach%2520for%2520kinase%2520inhibitor%2520focused%2520drug%2520discovery%253F%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1230%26epage%3D1241%26doi%3D10.1021%2Fcb500129t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The (un)targeted cancer kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">166</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1038/nchembio.297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnchembio.297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=20154661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=166-169&author=O.+Fedorovauthor=S.+M%C3%BCllerauthor=S.+Knapp&title=The+%28un%29targeted+cancer+kinome&doi=10.1038%2Fnchembio.297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The (un)targeted cancer kinome</span></div><div class="casAuthors">Fedorov, Oleg; Mueller, Susanne; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">166-169</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review and discussion.  The complexity of cancer signaling and the resulting difficulties in target selection have strongly biased kinase drug discovery towards clin. validated targets.  Recently, novel kinase targets that are uncharacterized have emerged from genome sequencing and RNAi studies.  Chem. probes are urgently needed to functionally annotate these kinases and to stimulate new drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwcnq0-qjPirVg90H21EOLACvtfcHk0lg3p8-WJEPxcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D&md5=198ae8d39d40247e17aef5e9de0f2a99</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.297%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520%2528un%2529targeted%2520cancer%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D166%26epage%3D169%26doi%3D10.1038%2Fnchembio.297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, R. J.</span><span> </span><span class="NLM_article-title">Targeting the human kinome for cancer therapy: current perspectives</span> <span class="citation_source-journal">Crit. Rev. Oncog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1615/CritRevOncog.v17.i2.70</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1615%2FCritRevOncog.v17.i2.70" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=22471710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A280%3ADC%252BC38rhvVylsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=233-246&author=L.+Zhangauthor=R.+J.+Daly&title=Targeting+the+human+kinome+for+cancer+therapy%3A+current+perspectives&doi=10.1615%2FCritRevOncog.v17.i2.70"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the human kinome for cancer therapy: current perspectives</span></div><div class="casAuthors">Zhang Luxi; Daly Roger J</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncogenesis</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">233-46</span>
        ISSN:<span class="NLM_cas:issn">0893-9675</span>.
    </div><div class="casAbstract">As highlighted by other articles in this review issue, great progress has been made in the development of targeted treatment modalities directed against particular oncogenic kinases, and the translation of these therapies into the clinic.  However, recent data from cancer genome sequencing projects indicate that the spectrum of kinases that contribute to cancer progression is wider than previously imagined, and as international projects in this area gather momentum, it appears likely that further kinase oncogenes will be identified, and the roles of previously characterized ones will be extended to subsets of additional cancers.  In addition, complementary approaches such as functional genomics and mass spectrometry (MS)-based proteomics are providing important insights into the functional roles played by specific kinases in particular cancers, the dependency of these roles on genetic background, how altered kinase regulation perturbs intracellular signaling networks, and how the latter respond to targeted agents that target the kinase in question.  While other articles in this issue focus on individual cancer-associated kinases and their therapeutic targeting, the aim of this review is to take a broader perspective regarding our current knowledge of the cancer kinome and how this can be expanded and exploited for clinical utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR-aUq9kTcyKn6oPH0mYWL1fW6udTcc2eZuufBq3TZTdrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38rhvVylsQ%253D%253D&md5=ad039a3a5d03ea32ee6a4d64c279d82b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1615%2FCritRevOncog.v17.i2.70&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1615%252FCritRevOncog.v17.i2.70%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DDaly%26aufirst%3DR.%2BJ.%26atitle%3DTargeting%2520the%2520human%2520kinome%2520for%2520cancer%2520therapy%253A%2520current%2520perspectives%26jtitle%3DCrit.%2520Rev.%2520Oncog.%26date%3D2012%26volume%3D17%26spage%3D233%26epage%3D246%26doi%3D10.1615%2FCritRevOncog.v17.i2.70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Riniker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fechner, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landrum, G. A.</span><span> </span><span class="NLM_article-title">Heterogeneous classifier fusion for ligand-based virtual screening: or, how decision making by committee can be a good thing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2829</span><span class="NLM_x">–</span> <span class="NLM_lpage">2836</span><span class="refDoi"> DOI: 10.1021/ci400466r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400466r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1yku7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=2829-2836&author=S.+Rinikerauthor=N.+Fechnerauthor=G.+A.+Landrum&title=Heterogeneous+classifier+fusion+for+ligand-based+virtual+screening%3A+or%2C+how+decision+making+by+committee+can+be+a+good+thing&doi=10.1021%2Fci400466r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Heterogeneous Classifier Fusion for Ligand-Based Virtual Screening: Or, How Decision Making by Committee Can Be a Good Thing</span></div><div class="casAuthors">Riniker, Sereina; Fechner, Nikolas; Landrum, Gregory A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2829-2836</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The concept of data fusion - the combination of information from different sources describing the same object with the expectation to generate a more accurate representation - has found application in a very broad range of disciplines.  In the context of ligand-based virtual screening (VS), data fusion has been applied to combine knowledge from either different active mols. or different fingerprints to improve similarity search performance.  Machine-learning (ML) methods based on fusion of multiple homogeneous classifiers, in particular random forests, have also been widely applied in the ML literature.  The heterogeneous version of classifier fusion - fusing the predictions from different model types - has been less explored.  Here, we investigate heterogeneous classifier fusion for ligand-based VS using three different ML methods, RF, naive Bayes (NB), and logistic regression (LR), with four 2D fingerprints, atom pairs, topol. torsions, RDKit fingerprint, and circular fingerprint.  The methods are compared using a previously developed benchmarking platform for 2D fingerprints which is extended to ML methods in this article.  The original data sets are filtered for difficulty, and a new set of challenging data sets from ChEMBL is added.  Data sets were also generated for a second use case: starting from a small set of related actives instead of diverse actives.  The final fused model consistently outperforms the other approaches across the broad variety of targets studied, indicating that heterogeneous classifier fusion is a very promising approach for ligand-based VS.  The new data sets together with the adapted source code for ML methods are provided in the Supporting Information.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1hnwYfGw2FbVg90H21EOLACvtfcHk0lgBkIglFX0cHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yku7fJ&md5=99f0bd62b569cd73a2e1539afca7fddb</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci400466r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400466r%26sid%3Dliteratum%253Aachs%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DFechner%26aufirst%3DN.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26atitle%3DHeterogeneous%2520classifier%2520fusion%2520for%2520ligand-based%2520virtual%2520screening%253A%2520or%252C%2520how%2520decision%2520making%2520by%2520committee%2520can%2520be%2520a%2520good%2520thing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D2829%26epage%3D2836%26doi%3D10.1021%2Fci400466r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Riniker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landrum, G. A.</span><span> </span><span class="NLM_article-title">Using information from historical high-throughput screens to predict active compounds</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1880</span><span class="NLM_x">–</span> <span class="NLM_lpage">1891</span><span class="refDoi"> DOI: 10.1021/ci500190p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500190p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslegtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1880-1891&author=S.+Rinikerauthor=Y.+Wangauthor=J.+L.+Jenkinsauthor=G.+A.+Landrum&title=Using+information+from+historical+high-throughput+screens+to+predict+active+compounds&doi=10.1021%2Fci500190p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Using Information from Historical High-Throughput Screens to Predict Active Compounds</span></div><div class="casAuthors">Riniker, Sereina; Wang, Yuan; Jenkins, Jeremy L.; Landrum, Gregory A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1880-1891</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern high-throughput screening (HTS) is a well-established approach for hit finding in drug discovery that is routinely employed in the pharmaceutical industry to screen more than a million compds. within a few weeks.  However, as the industry shifts to more disease-relevant but more complex phenotypic screens, the focus has moved to piloting smaller but smarter chem./biol. diverse subsets followed by an expansion around hit compds.  One std. method for doing this is to train a machine-learning (ML) model with the chem. fingerprints of the tested subset of mols. and then select the next compds. based on the predictions of this model.  An alternative approach would be to take advantage of the wealth of bioactivity information contained in older (full-deck) screens using so-called HTS fingerprints, where each element of the fingerprint corresponds to the outcome of a particular assay, as input to machine-learning algorithms.  We constructed HTS fingerprints using two collections of data: 93 inhouse assays and 95 publicly available assays from PubChem.  For each source, an addnl. set of 51 and 46 assays, resp., was collected for testing.  Three different ML methods, random forest (RF), logistic regression (LR), and naive Bayes (NB), were investigated for both the HTS fingerprint and a chem. fingerprint, Morgan2.  RF was best suited for learning from HTS fingerprints yielding area under the receiver operating characteristic curve (AUC) values >0.8 for 78% of the internal assays and enrichment factors at 5% (EF(5%)) >10 for 55% of the assays.  The RF(HTS-fp) generally outperformed the LR trained with Morgan2, which was the best ML method for the chem. fingerprint, for the majority of assays.  In addn., HTS fingerprints were found to retrieve more diverse chemotypes.  Combining the two models through heterogeneous classifier fusion led to a similar or better performance than the best individual model for all assays.  Further validation using a pair of inhouse assays and data from a confirmatory screen-including a prospective set of around 2000 compds. selected based on our approach-confirmed the good performance.  Thus, the combination of machine-learning with HTS fingerprints and chem. fingerprints utilizes information from both domains and presents a very promising approach for hit expansion, leading to more hits.  The source code used with the public data is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAosVSBEbE57Vg90H21EOLACvtfcHk0lgBkIglFX0cHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslegtb4%253D&md5=a2fe1906d259aa91e07918ff6dbd1b3b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fci500190p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500190p%26sid%3Dliteratum%253Aachs%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26atitle%3DUsing%2520information%2520from%2520historical%2520high-throughput%2520screens%2520to%2520predict%2520active%2520compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1880%26epage%3D1891%26doi%3D10.1021%2Fci500190p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urban, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutlina, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fekete, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klumpp, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">A screening pattern recognition method finds new and divergent targets for drugs and natural products</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1622</span><span class="NLM_x">–</span> <span class="NLM_lpage">1631</span><span class="refDoi"> DOI: 10.1021/cb5001839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5001839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1622-1631&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=L.+Urbanauthor=S.+Whitebreadauthor=S.+Chenauthor=K.+Hughesauthor=H.+Guoauthor=E.+Kutlinaauthor=A.+Feketeauthor=M.+Klumppauthor=M.+Glick&title=A+screening+pattern+recognition+method+finds+new+and+divergent+targets+for+drugs+and+natural+products&doi=10.1021%2Fcb5001839"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fcb5001839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5001839%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DUrban%26aufirst%3DL.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DK.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DKutlina%26aufirst%3DE.%26aulast%3DFekete%26aufirst%3DA.%26aulast%3DKlumpp%26aufirst%3DM.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DA%2520screening%2520pattern%2520recognition%2520method%2520finds%2520new%2520and%2520divergent%2520targets%2520for%2520drugs%2520and%2520natural%2520products%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1622%26epage%3D1631%26doi%3D10.1021%2Fcb5001839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Petrone, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simms, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigsch, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lounkine, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutchukian, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span> </span><span class="NLM_article-title">Rethinking molecular similarity: comparing compounds on the basis of biological activity</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1399</span><span class="NLM_x">–</span> <span class="NLM_lpage">1409</span><span class="refDoi"> DOI: 10.1021/cb3001028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3001028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1399-1409&author=P.+M.+Petroneauthor=B.+Simmsauthor=F.+Nigschauthor=E.+Lounkineauthor=P.+Kutchukianauthor=A.+Cornettauthor=Z.+Dengauthor=J.+W.+Daviesauthor=J.+L.+Jenkinsauthor=M.+Glick&title=Rethinking+molecular+similarity%3A+comparing+compounds+on+the+basis+of+biological+activity&doi=10.1021%2Fcb3001028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking Molecular Similarity: Comparing Compounds on the Basis of Biological Activity</span></div><div class="casAuthors">Petrone, Paula M.; Simms, Benjamin; Nigsch, Florian; Lounkine, Eugen; Kutchukian, Peter; Cornett, Allen; Deng, Zhan; Davies, John W.; Jenkins, Jeremy L.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1399-1409</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since the advent of high-throughput screening (HTS), there has been an urgent need for methods that facilitate the interrogation of large-scale chem. biol. data to build a mode of action (MoA) hypothesis.  This can be done either prior to the HTS by subset design of compds. with known MoA or post HTS by data annotation and mining.  To enable this process, we developed a tool that compares compds. solely on the basis of their bioactivity: the chem. biol. descriptor high-throughput screening fingerprint (HTS-FP).  In the current embodiment, data are aggregated from 195 biochem. and cell-based assays developed at Novartis and can be used to identify bioactivity relationships among the inhouse collection comprising ∼1.5 million compds.  We demonstrate the value of the HTS-FP for virtual screening and in particular scaffold hopping.  HTS-FP outperforms state of the art methods in several aspects, retrieving bioactive compds. with remarkable chem. dissimilarity to a probe structure.  We also apply HTS-FP for the design of screening subsets in HTS.  Using retrospective data, we show that a biodiverse selection of plates performs significantly better than a chem. diverse selection of plates, both in terms of no. of hits and diversity of chemotypes retrieved.  This is also true in the case of hit expansion predictions using HTS-FP similarity.  Sets of compds. clustered with HTS-FP are biol. meaningful, in the sense that these clusters enrich for genes and gene ontol. (GO) terms, showing that compds. that are bioactively similar also tend to target proteins that operate together in the cell.  HTS-FP are valuable not only because of their predictive power but mainly because they relate compds. solely on the basis of bioactivity, harnessing the accumulated knowledge of a high-throughput screening facility toward the understanding of how compds. interact with the proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFjchAxW_TSLVg90H21EOLACvtfcHk0liKMODJHtYxJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D&md5=66ecd494f9200a0fdc1c142402ff255f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fcb3001028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3001028%26sid%3Dliteratum%253Aachs%26aulast%3DPetrone%26aufirst%3DP.%2BM.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKutchukian%26aufirst%3DP.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DRethinking%2520molecular%2520similarity%253A%2520comparing%2520compounds%2520on%2520the%2520basis%2520of%2520biological%2520activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1399%26epage%3D1409%26doi%3D10.1021%2Fcb3001028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Cortes Cabrera, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucena-Agell, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redondo-Horcajo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barasoain, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fasching, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrone, P.</span><span> </span><span class="NLM_article-title">Compound biological signatures facilitate phenotypic screening and target elucidation</span> <span class="citation_source-journal">bioRxiv</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_fpage">041947</span><span class="refDoi"> DOI: 10.1101/041947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1101%2F041947" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=041947&author=A.+Cortes+Cabreraauthor=D.+Lucena-Agellauthor=M.+Redondo-Horcajoauthor=I.+Barasoainauthor=F.+Diazauthor=B.+Faschingauthor=P.+Petrone&title=Compound+biological+signatures+facilitate+phenotypic+screening+and+target+elucidation&doi=10.1101%2F041947"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1101%2F041947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F041947%26sid%3Dliteratum%253Aachs%26aulast%3DCortes%2BCabrera%26aufirst%3DA.%26aulast%3DLucena-Agell%26aufirst%3DD.%26aulast%3DRedondo-Horcajo%26aufirst%3DM.%26aulast%3DBarasoain%26aufirst%3DI.%26aulast%3DDiaz%26aufirst%3DF.%26aulast%3DFasching%26aufirst%3DB.%26aulast%3DPetrone%26aufirst%3DP.%26atitle%3DCompound%2520biological%2520signatures%2520facilitate%2520phenotypic%2520screening%2520and%2520target%2520elucidation%26jtitle%3DbioRxiv%26date%3D2016%26spage%3D041947%26doi%3D10.1101%2F041947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0liKMODJHtYxJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Yagnik, J.; Strelow, D.; Ross, D. A.; Lin, R.-s.</span><span> </span><span class="NLM_article-title">The power of comparative reasoning</span>. International Conference on Computer Vision (IEEE), Barcelona, Spain, November 6–13, 2011; pp  <span class="NLM_fpage">2431</span><span class="NLM_x">–</span> <span class="NLM_lpage">2438</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Yagnik%2C+J.%3B+Strelow%2C+D.%3B+Ross%2C+D.+A.%3B+Lin%2C+R.-s.+The+power+of+comparative+reasoning.+International+Conference+on+Computer+Vision+%28IEEE%29%2C+Barcelona%2C+Spain%2C+November+6%E2%80%9313%2C+2011%3B+pp+2431%E2%80%932438."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DYagnik%26aufirst%3DJ.%26atitle%3DThe%2520power%2520of%2520comparative%2520reasoning%26jtitle%3DInternational%2520Conference%2520on%2520Computer%2520Vision%2520%2528IEEE%2529%252C%2520Barcelona%252C%2520Spain%252C%2520November%25206%25E2%2580%259313%252C%25202011%26spage%3D2431%26epage%3D2438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Bora, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciucanu, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raica, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avram, S.</span><span> </span><span class="NLM_article-title">Predictive models for fast and effective profiling of kinase inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">895</span><span class="NLM_x">–</span> <span class="NLM_lpage">905</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00646</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00646" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlslyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=895-905&author=A.+Boraauthor=S.+Avramauthor=I.+Ciucanuauthor=M.+Raicaauthor=S.+Avram&title=Predictive+models+for+fast+and+effective+profiling+of+kinase+inhibitors&doi=10.1021%2Facs.jcim.5b00646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive Models for Fast and Effective Profiling of Kinase Inhibitors</span></div><div class="casAuthors">Bora, Alina; Avram, Sorin; Ciucanu, Ionel; Raica, Marius; Avram, Stefana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">895-905</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study we developed two-dimensional pharmacophore-based random forest models for the effective profiling of kinase inhibitors.  One hundred seven prediction models were developed to address distinct kinases spanning over all kinase groups.  Rigorous external validation demonstrates excellent virtual screening and classification potential of the predictors and, more importantly, the capacity to prioritize novel chem. scaffolds in large chem. libraries.  The models built upon more diverse and more potent compds. tend to exert the highest predictive power.  The anal. of ColBioS-FlavRC (Collection of Bioselective Flavonoids and Related Compds.) highlighted several potentially promiscuous derivs. with undesirable selectivity against kinases.  The prediction models can be downloaded from www.chembioinf.ro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTZSs1GEkYHLVg90H21EOLACvtfcHk0liKMODJHtYxJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlslyqtr4%253D&md5=cd8d17ea9616165b115dd9d75352e0ac</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00646%26sid%3Dliteratum%253Aachs%26aulast%3DBora%26aufirst%3DA.%26aulast%3DAvram%26aufirst%3DS.%26aulast%3DCiucanu%26aufirst%3DI.%26aulast%3DRaica%26aufirst%3DM.%26aulast%3DAvram%26aufirst%3DS.%26atitle%3DPredictive%2520models%2520for%2520fast%2520and%2520effective%2520profiling%2520of%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D895%26epage%3D905%26doi%3D10.1021%2Facs.jcim.5b00646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lélias, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0liOCFNHyOgmgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">McHugh, M. L.</span><span> </span><span class="NLM_article-title">Interrater reliability: the kappa statistic</span> <span class="citation_source-journal">Biochem. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">276</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span><span class="refDoi"> DOI: 10.11613/BM.2012.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=10.11613%2FBM.2012.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A280%3ADC%252BC3s%252FnsFCqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=276-282&author=M.+L.+McHugh&title=Interrater+reliability%3A+the+kappa+statistic&doi=10.11613%2FBM.2012.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Interrater reliability: the kappa statistic</span></div><div class="casAuthors">McHugh Mary L</div><div class="citationInfo"><span class="NLM_cas:title">Biochemia medica</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">276-82</span>
        ISSN:<span class="NLM_cas:issn">1330-0962</span>.
    </div><div class="casAbstract">The kappa statistic is frequently used to test interrater reliability.  The importance of rater reliability lies in the fact that it represents the extent to which the data collected in the study are correct representations of the variables measured.  Measurement of the extent to which data collectors (raters) assign the same score to the same variable is called interrater reliability.  While there have been a variety of methods to measure interrater reliability, traditionally it was measured as percent agreement, calculated as the number of agreement scores divided by the total number of scores.  In 1960, Jacob Cohen critiqued use of percent agreement due to its inability to account for chance agreement.  He introduced the Cohen's kappa, developed to account for the possibility that raters actually guess on at least some variables due to uncertainty.  Like most correlation statistics, the kappa can range from -1 to +1.  While the kappa is one of the most commonly used statistics to test interrater reliability, it has limitations.  Judgments about what level of kappa should be acceptable for health research are questioned.  Cohen's suggested interpretation may be too lenient for health related studies because it implies that a score as low as 0.41 might be acceptable.  Kappa and percent agreement are compared, and levels for both kappa and percent agreement that should be demanded in healthcare studies are suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS8oUvZdI4KVzdblLE_E9PWfW6udTcc2eYRb0Q-UENXhbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s%252FnsFCqtQ%253D%253D&md5=ba7a87fe0088906722bf15282bfa1d2a</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.11613%2FBM.2012.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.11613%252FBM.2012.031%26sid%3Dliteratum%253Aachs%26aulast%3DMcHugh%26aufirst%3DM.%2BL.%26atitle%3DInterrater%2520reliability%253A%2520the%2520kappa%2520statistic%26jtitle%3DBiochem.%2520Med.%26date%3D2012%26volume%3D22%26spage%3D276%26epage%3D282%26doi%3D10.11613%2FBM.2012.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liaw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahl, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svetnik, V.</span><span> </span><span class="NLM_article-title">Deep neural nets as a method for quantitative structure-activity relationships</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">274</span><span class="refDoi"> DOI: 10.1021/ci500747n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500747n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=263-274&author=J.+Maauthor=R.+P.+Sheridanauthor=A.+Liawauthor=G.+E.+Dahlauthor=V.+Svetnik&title=Deep+neural+nets+as+a+method+for+quantitative+structure-activity+relationships&doi=10.1021%2Fci500747n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Deep Neural Nets as a Method for Quantitative Structure-Activity Relationships</span></div><div class="casAuthors">Ma, Junshui; Sheridan, Robert P.; Liaw, Andy; Dahl, George E.; Svetnik, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-274</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Neural networks were widely used for quant. structure-activity relationships (QSAR) in the 1990s.  Because of various practical issues (e.g., slow on large problems, difficult to train, prone to overfitting, etc.), they were superseded by more robust methods like support vector machine (SVM) and random forest (RF), which arose in the early 2000s.  The last 10 years has witnessed a revival of neural networks in the machine learning community thanks to new methods for preventing overfitting, more efficient training algorithms, and advancements in computer hardware.  In particular, deep neural nets (DNNs), i.e. neural nets with more than one hidden layer, have found great successes in many applications, such as computer vision and natural language processing.  Here we show that DNNs can routinely make better prospective predictions than RF on a set of large diverse QSAR data sets that are taken from Merck's drug discovery effort.  The no. of adjustable parameters needed for DNNs is fairly large, but our results show that it is not necessary to optimize them for individual data sets, and a single set of recommended parameters can achieve better performance than RF for most of the data sets we studied.  The usefulness of the parameters is demonstrated on addnl. data sets not used in the calibration.  Although training DNNs is still computationally intensive, using graphical processing units (GPUs) can make this issue manageable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwVMQ-gQwZd7Vg90H21EOLACvtfcHk0ljSGZPJ7I89OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGns70%253D&md5=2c6a3f33f85a4abc32ebc33108426667</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci500747n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500747n%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DDahl%26aufirst%3DG.%2BE.%26aulast%3DSvetnik%26aufirst%3DV.%26atitle%3DDeep%2520neural%2520nets%2520as%2520a%2520method%2520for%2520quantitative%2520structure-activity%2520relationships%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D263%26epage%3D274%26doi%3D10.1021%2Fci500747n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eid, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fulle, S.</span><span> </span><span class="NLM_article-title">Identification and visualization of kinase-specific subpockets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">335</span><span class="NLM_x">–</span> <span class="NLM_lpage">346</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00627</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00627" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;key=1%3ACAS%3A528%3ADC%252BC28Xis1OgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=335-346&author=A.+Volkamerauthor=S.+Eidauthor=S.+Turkauthor=F.+Rippmannauthor=S.+Fulle&title=Identification+and+visualization+of+kinase-specific+subpockets&doi=10.1021%2Facs.jcim.5b00627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Visualization of Kinase-Specific Subpockets</span></div><div class="casAuthors">Volkamer, Andrea; Eid, Sameh; Turk, Samo; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">335-346</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification and design of selective compds. is important for the redn. of unwanted side effects as well as for the development of tool compds. for target validation studies.  This is, in particular, true for therapeutically important protein families that possess conserved folds and have numerous members such as kinases.  To support the design of selective kinase inhibitors, we developed a novel approach that allows identification of specificity detg. subpockets between closely related kinases solely based on their three-dimensional structures.  To account for the intrinsic flexibility of the proteins, multiple X-ray structures of the target protein of interest as well as of unwanted off-target(s) are taken into account.  The binding pockets of these protein structures are calcd. and fused to a combined target and off-target pocket, resp.  Subsequently, shape differences between these two combined pockets are identified via fusion rules.  The approach provides a user-friendly visualization of target-specific areas in a binding pocket which should be explored when designing selective compds.  Furthermore, the approach can be easily combined with in silico alanine mutation studies to identify selectivity detg. residues.  The potential impact of the approach is demonstrated in four retrospective expts. on closely related kinases, i.e., p38α vs Erk2, PAK1 vs PAK4, ITK vs AurA, and BRAF vs VEGFR2.  Overall, the presented approach does not require any profiling data for training purposes, provides an intuitive visualization of a large no. of protein structures at once, and could also be applied to other target classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlaQhBsAUeNbVg90H21EOLACvtfcHk0ljSGZPJ7I89OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xis1OgtQ%253D%253D&md5=c9eec4cca15085669921cc16f3c121fb</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00627%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DEid%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DIdentification%2520and%2520visualization%2520of%2520kinase-specific%2520subpockets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D335%26epage%3D346%26doi%3D10.1021%2Facs.jcim.5b00627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i21"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01611">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_37241"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01611">10.1021/acs.jmedchem.6b01611</a>. The curated <i>Open</i> data set and source code for deriving prediction models can be downloaded from <a class="extLink" href="https://github.com/Team-SKI/Publications">https://github.com/Team-SKI/Publications</a>. <ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional external model validation, correlations between different ML methods, and the impact of various balancing techniques on the prediction performance. (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01611/suppl_file/jm6b01611_si_001.pdf">jm6b01611_si_001.pdf (968.92 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-1%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01611%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01611" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a99cffa4d2495","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
