
<html lang="en"     class="pb-page"  data-request-id="fb6c5b9f-af3a-4043-8beb-48d59d363d66"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.6b00078;issue:issue:10.1021/jmcmar.2016.59.issue-9;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Impact of Binding Site Comparisons on Medicinal Chemistry and Rational Molecular Design" /></meta><meta name="dc.Creator" content="Christiane  Ehrt" /></meta><meta name="dc.Creator" content="Tobias  Brinkjost" /></meta><meta name="dc.Creator" content="Oliver  Koch" /></meta><meta name="dc.Description" content="Modern rational drug design not only deals with the search for ligands binding to interesting and promising validated targets but also aims to identify the function and ligands of yet uncharacteriz..." /></meta><meta name="Description" content="Modern rational drug design not only deals with the search for ligands binding to interesting and promising validated targets but also aims to identify the function and ligands of yet uncharacteriz..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 21, 2016" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00078" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00078" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00078" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00078" /></link>
        
    
    

<title>Impact of Binding Site Comparisons on Medicinal Chemistry and Rational Molecular Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00078" /></meta><meta property="og:title" content="Impact of Binding Site Comparisons on Medicinal Chemistry and Rational Molecular Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0014.jpeg" /></meta><meta property="og:description" content="Modern rational drug design not only deals with the search for ligands binding to interesting and promising validated targets but also aims to identify the function and ligands of yet uncharacterized proteins having impact on different diseases. Additionally, it contributes to the design of inhibitors with distinct selectivity patterns and the prediction of possible off-target effects. The identification of similarities between binding sites of various proteins is a useful approach to cope with those challenges. The main scope of this perspective is to describe applications of different protein binding site comparison approaches to outline their applicability and impact on molecular design. The article deals with various substantial application domains and provides some outstanding examples to show how various binding site comparison methods can be applied to promote in silico drug design workflows. In addition, we will also briefly introduce the fundamental principles of different protein binding site comparison methods." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00078"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00078">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00078&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00078&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00078&amp;href=/doi/10.1021/acs.jmedchem.6b00078" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4121-4151</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b02008" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.5b00150" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Impact of Binding Site Comparisons on Medicinal Chemistry and Rational Molecular Design</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christiane++Ehrt">Christiane Ehrt</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tobias++Brinkjost">Tobias Brinkjost</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oliver++Koch">Oliver Koch</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Straße 6, 44227 Dortmund, Germany</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Computer Science, TU Dortmund University, Otto-Hahn-Straße 14, 44224 Dortmund, Germany</span></div><div class="corresp-info"><strong>*</strong>O.K.: phone, +49 231 755 6104; e-mail, <a href="/cdn-cgi/l/email-protection#286744415e4d5a0663474b40685c5d054c475a5c455d464c064c4d"><span class="__cf_email__" data-cfemail="8cc3e0e5fae9fea2c7e3efe4ccf8f9a1e8e3fef8e1f9e2e8a2e8e9">[email protected]</span></a>, <a href="/cdn-cgi/l/email-protection#155a797c6370673b5e7a767d5574727e7a767d3b7170"><span class="__cf_email__" data-cfemail="3c7350554a594e1277535f547c5d5b57535f54125859">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00078&amp;href=/doi/10.1021%2Facs.jmedchem.6b00078" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4121–4151</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 5, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 January 2016</li><li><span class="item_label"><b>Published</b> online</span>21 April 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 May 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00078" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00078</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4121%26pageCount%3D31%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChristiane%2BEhrt%252C%2BTobias%2BBrinkjost%252C%2BOliver%2BKoch%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D9%26contentID%3Dacs.jmedchem.6b00078%26title%3DImpact%2Bof%2BBinding%2BSite%2BComparisons%2Bon%2BMedicinal%2BChemistry%2Band%2BRational%2BMolecular%2BDesign%26numPages%3D31%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4151%26publicationDate%3DMay%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00078"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4355</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">58</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00078" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Impact of Binding Site Comparisons on Medicinal Chemistry and Rational Molecular Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Christiane&quot;,&quot;last_name&quot;:&quot;Ehrt&quot;},{&quot;first_name&quot;:&quot;Tobias&quot;,&quot;last_name&quot;:&quot;Brinkjost&quot;},{&quot;first_name&quot;:&quot;Oliver&quot;,&quot;last_name&quot;:&quot;Koch&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;4121-4151&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00078&quot;},&quot;abstract&quot;:&quot;Modern rational drug design not only deals with the search for ligands binding to interesting and promising validated targets but also aims to identify the function and ligands of yet uncharacterized proteins having impact on different diseases. Additionally, it contributes to the design of inhibitors with distinct selectivity patterns and the prediction of possible off-target effects. The identification of similarities between binding sites of various proteins is a useful approach to cope with those challenges. The main scope of this perspective is to describe applications of different protein binding site comparison approaches to outline their applicability and impact on molecular design. The article deals with various substantial application domains and provides some outstanding examples to show how various binding site comparison methods can be applied to promote in silico drug design workflows. In addition, we will also briefly introduce the fundamental principles of different protein binding site compar&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00078&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00078" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00078&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00078" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00078&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00078" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00078&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00078&amp;href=/doi/10.1021/acs.jmedchem.6b00078" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00078" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00078" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00078%26sid%3Dliteratum%253Aachs%26pmid%3D27046190%26genre%3Darticle%26aulast%3DEhrt%26date%3D2016%26atitle%3DImpact%2Bof%2BBinding%2BSite%2BComparisons%2Bon%2BMedicinal%2BChemistry%2Band%2BRational%2BMolecular%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D9%26spage%3D4121%26epage%3D4151%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290901" title="Protein structure">Protein structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/jmcmar.2016.59.issue-9/20160512/jmcmar.2016.59.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Modern rational drug design not only deals with the search for ligands binding to interesting and promising validated targets but also aims to identify the function and ligands of yet uncharacterized proteins having impact on different diseases. Additionally, it contributes to the design of inhibitors with distinct selectivity patterns and the prediction of possible off-target effects. The identification of similarities between binding sites of various proteins is a useful approach to cope with those challenges. The main scope of this perspective is to describe applications of different protein binding site comparison approaches to outline their applicability and impact on molecular design. The article deals with various substantial application domains and provides some outstanding examples to show how various binding site comparison methods can be applied to promote in silico drug design workflows. In addition, we will also briefly introduce the fundamental principles of different protein binding site comparison methods.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/9"><issue-title>Computational Methods for Medicinal Chemistry</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The knowledge of 3D structures of proteins and protein–ligand complexes is one prerequisite for rational structure-based drug design that deals with the utilization of these structural data to generate ideas for new promising compounds displaying a certain bioactivity. Furthermore, it supports the development from hit to lead to drug in various other ways. The exploitation of known protein structures publically available in the Protein Data Bank (<a href="http://www.rcsb.org" class="extLink">www.rcsb.org</a>)<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> becomes more and more complex due to the exponentially growing number of new structures being published daily. Although a plateau can now be recognized in the emergence and discovery of novel folds (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), the number of identified novel druggable small molecule binding sites is still increasing, especially owing to the development of useful computational methods for the detection of new binding sites.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. PDB statistics on the number of released PDB entries (blue bars) and the number of new CATH<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> folds (red bars) and SCOP<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> folds (green bars) per year.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Apart from that, a recent study suggests a finite nature of possible ligand binding pocket properties and shows the impact of binding site comparison to shed light on the question of ligand promiscuity.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The fact that the binding of one ligand to different proteins is by no means an exceptional case is illustrated by the statistics of the protein–ligand complex database sc-PDB<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). Sturm et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> also showed that ligand promiscuity is a direct consequence of the presence of similar binding cavities by performing a large scale analysis of 247 promiscuous ligands extracted from 8166 sc-PDB<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> entries. The methods SiteAlign4.0,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Volsite/Shaper,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and FuzCav<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> were used for binding site comparison. Nevertheless, the authors also found certain similar chemotypes that adapt to different protein environments (especially compounds of very low or very high selectivity), examples where none of the binding site comparison methods presented here were able to succeed in predicting promiscuity.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. sc-PDB<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> statistics on the number of unique targets per ligand illustrating the impact of ligand promiscuity.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Cofactors and ions are not considered as ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Today, the comparison of whole proteins is quite easily achievable on different levels of protein structure (primary structure, sequence alignments; tertiary structure and quaternary structure, structural alignments). Nevertheless, the comparison of protein binding sites, that is, parts of a protein that are crucial for structure-based drug design approaches, remains challenging. The elucidation of similarities and differences between protein pockets enables researchers to cope with selectivity issues, off-target effects, the prediction of binding sites and supports the elucidation of yet uncharacterized protein functions. It even facilitates the analysis of protein–protein interactions (PPIs). Over the past few decades, it became obvious that similar binding sites can be found in proteins with low or even without any overall similarity. The focus was drawn toward the elucidation of local similarities within proteins on the structural level. Therefore, it becomes increasingly interesting to exploit this “pocket space” as kind of a dictionary to accelerate modern in silico drug design processes. This leads to a need for highly efficient protein binding site comparison methods.</div><div class="NLM_p">Several methods exist to elucidate similarities between protein binding sites on the basis of structural fold similarities or sequence motifs common to proteins with similar functions which were successfully applied in several cases. However, similar ligands can bind to proteins with little or no overall structural similarity and without common sequence patterns. Other methods are required to explain such scenarios. An article providing an exhaustive analysis using CATH (class, architecture, topology, homologous superfamily)<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> data showed that on the one hand, fold is closely related to ligand type, but on the other hand, the EC (enzyme commission) classification is not related to this kind of similarity.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Additionally, many case studies show that the binding of similar ligands cannot be deduced from fold but rather from local similarities between two proteins. Examples for such binding site pairs of unrelated proteins in the PDB are provided in an article by Barelier et al.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">It might be useful to apply a ligand-based approach to elucidate similarities between protein binding sites by looking for chemically similar ligands binding to different proteins.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> An article of Li et al.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> is only one example for a successful application of this approach. In addition to structural databases such as the PDB, the huge knowledge stored in chemogenomics databases such as ChEMBL,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> DrugBank,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> PubChem BioAssay,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> or MATADOR<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> can be exploited. In the latter cases, the basic problem is that often the ligand’s binding site is unknown. Therefore, it becomes difficult and tedious to compare protein structures on the basis of ligand-derived data. A drawback of searching the PDB for similar ligands which bind to different enzymes is the restricted number of such examples as compared to the protein ligand binding space.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Nevertheless, the analysis of the available data led to the development of useful publically available binding site databases such as sc-PDB<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> or Pocketome.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Unfortunately, those databases only include binding sites of already solved protein–ligand complexes. A myriad of publically available binding site detection and prediction methods such as LIGSITE,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Q-SiteFinder,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Molsite,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Rate4Site,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> or MetaPocket<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> enable researchers not only to rely on known binding sites but also to identify and compare yet uncharacterized ligand binding pockets of apo structures or homology models. The reader is referred to a comprehensive article for more details on surface-based, energy-based, or conservation-based binding site detection algorithms.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p last">For the reasons outlined above, the comparison of proteins on the binding site level using their solved 3D structures is a promising tool to reveal similarities between proteins which do not seem to have any evolutionary relationship or even do not share a similar fold or function. Moreover, some of them are suitable for the elucidation of crucial differences between binding sites of highly similar proteins. Here, we will not focus on methods using 2D descriptors to represent and compare binding sites<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> but will introduce methods to compare binding sites in terms of the spatial arrangement of binding site features. The scope of this review is to give a short overview of the basic approaches developed to compare protein binding sites in 3D space but especially to provide examples of some useful applications of those methods. Therefore, the text is divided into two main parts that deal with binding site comparison methods and their applications. The first part starts with an introduction into the different basic approaches for binding site comparison and a classification scheme. Then, it gives a brief summary of the discussed methods. Readers who are more interested in the applications and application domains described in the second part can skip this brief method overview and can refer back later for more details on a certain comparison method.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Binding Site Comparison Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02888" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02888" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The comparison of ligand binding sites in proteins is motivated by the basic assumption that similar proteins or protein binding sites bind similar ligands. This statement can be interpreted in various ways depending on the description of ligand and target space which is nicely outlined in an article by Rognan.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Typically, it is true that similar physicochemical properties of binding sites lead to the binding of ligands with high similarity in terms of overall structural similarity<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> as determined with such metrics as the Tanimoto coefficient and various other measures.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Binding site similarity regarding physicochemical properties represents the basis for many in silico drug design methods. A review on distinct ligand binding pockets focuses on this interpretation of similarity.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Nevertheless, there are known examples where the local similarity of proteins in terms of structural similarity of the protein backbone leads to a binding of similar scaffolds.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> Therefore, the question arises: How do we define binding site similarity? Unfortunately, a unique definition of binding site similarity is not possible and always depends on the way binding sites are compared with each other. On the basis of a study of the data stored in the PDB, Barelier et al.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> identified at least three classes of recognition of an identical ligand by different proteins: proteins of different families with related binding site residues, proteins of different families that bind common ligand functional groups via dissimilar residues, and different (or similar) proteins recognizing different functional groups of a similar ligand. Furthermore, they state that their detailed analysis did not include all binding events of similar ligands to different targets. So, how should we compare proteins and especially protein binding sites? To shed light on this question, the next two paragraphs deal with basic approaches toward the description of ligand binding sites and different available comparison methods.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction into Different Basic Approaches</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53345" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53345" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Binding sites are compared by considering the proteins’ surfaces, physicochemical properties, interaction profiles, or backbone structure. In general, the published methods can be differentiated on the basis of binding site modeling, the comparison algorithm, and the way the developers score or rank the identified similarities. While some of the methods use whole structures and are able to find local similarities, others rely on predefined binding sites, making it difficult to use apo structures of proteins. The automated identification of protein binding sites on its own is a challenging task with several different approaches. Therefore, we refer to a comprehensive review focusing on the different approaches to characterize protein binding sites.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Here, the following questions should be answered: What is compared? How is it compared? What does similarity mean?<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div class="NLM_p">Due to the different facets of binding site comparison, the presented methods are classified according to the model used to represent the ligand binding pocket features. A multitude of methods use graph models. Objects such as atoms, functional groups, or surface points are represented as nodes connected by edges that can be labeled with distances or other characteristics of the relation between two nodes. Other comparison algorithms use purely geometric representations of binding site atoms, for example, as points, feature clouds, or even volumes. The use of pharmacophore-based features to describe binding sites often results in fingerprints that are compared with each other, saving a huge amount of computational time. Furthermore, vectors or grids can be used to represent binding sites leading to more complex comparisons. The computationally most demanding methods compare binding sites on the basis of distributions leading to high accuracy but also to high sensitivity in terms of protein flexibility. <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> summarizes all methods in a clear manner. It has to be pointed out that the classification scheme is a very simplified one in view of the high diversity of binding site comparison algorithms. Some methods compare binding sites in multiple consecutive steps using different comparison types. Others model binding sites in various ways to reduce runtimes or to introduce implicit fuzziness, for example, to account for the flexibility of proteins and uncertainties in the protein structure models. Nevertheless, the methods are grouped accordingly to provide a crude measure of runtime and computational costs.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Different levels of binding site representation for subsequent comparisons. The figure is based on a crystal structure of matriptase in complex with an inhibitor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r0i">4r0i</a>) and was generated using MOE 2013.08.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> summarizes the presented methods and provides information about their availability as well as their respective applications described below. An overview of those and alternative approaches, their availability, and the respective Web sites can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Site Comparison Methods, Their Availability, and Their Fields of Successful Application<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">method</th><th class="colsep0 rowsep0" align="center">availability</th><th class="colsep0 rowsep0" align="center">model</th><th class="colsep0 rowsep0" align="center">application</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">APF<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></td><td class="colsep0 rowsep0" align="left">software (commercial)</td><td class="colsep0 rowsep0" align="left">grid</td><td class="colsep0 rowsep0" align="left">Off-target prediction<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Homolog analysis<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></td><td class="colsep0 rowsep0" align="left">software (commercial)</td><td class="colsep0 rowsep0" align="left">graph</td><td class="colsep0 rowsep0" align="left">Protein–ligand interactions<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Virtual screening<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Evolutionary relationships<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Drug repurposing<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CMASA<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></td><td class="colsep0 rowsep0" align="left">software (free), mail server (free)</td><td class="colsep0 rowsep0" align="left">geometric (points)</td><td class="colsep0 rowsep0" align="left">Homolog analysis<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">COFACTOR<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></td><td class="colsep0 rowsep0" align="left">Web server (free)</td><td class="colsep0 rowsep0" align="left">geometric (points)</td><td class="colsep0 rowsep0" align="left">Function prediction<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPASS<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></td><td class="colsep0 rowsep0" align="left">Web server (free for academic users)</td><td class="colsep0 rowsep0" align="left">geometric (points)</td><td class="colsep0 rowsep0" align="left">PPIs<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">eF-site<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></td><td class="colsep0 rowsep0" align="left">Web server (free)</td><td class="colsep0 rowsep0" align="left">graph</td><td class="colsep0 rowsep0" align="left">Function prediction<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FLAP<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></td><td class="colsep0 rowsep0" align="left">software (commercial)</td><td class="colsep0 rowsep0" align="left">fingerprint</td><td class="colsep0 rowsep0" align="left">Prediction of compound selectivity profiles and affinities<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">fPOP<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></td><td class="colsep0 rowsep0" align="left">Web server (free)</td><td class="colsep0 rowsep0" align="left">geometric (points)</td><td class="colsep0 rowsep0" align="left">Evolutionary relationships<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FuzCav<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></td><td class="colsep0 rowsep0" align="left">software (free)</td><td class="colsep0 rowsep0" align="left">fingerprint</td><td class="colsep0 rowsep0" align="left">Protein–ligand interactions<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IFP<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></td><td class="colsep0 rowsep0" align="left">software (free)</td><td class="colsep0 rowsep0" align="left">fingerprint</td><td class="colsep0 rowsep0" align="left">Virtual screening<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IsoCleft Finder<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></td><td class="colsep0 rowsep0" align="left">Web server (free)</td><td class="colsep0 rowsep0" align="left">graph</td><td class="colsep0 rowsep0" align="left">Drug repurposing<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KRIPO<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></td><td class="colsep0 rowsep0" align="left">not publically available</td><td class="colsep0 rowsep0" align="left">fingerprint</td><td class="colsep0 rowsep0" align="left">Off-target prediction<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Method of Brakoulias and Jackson<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></td><td class="colsep0 rowsep0" align="left">not publically available</td><td class="colsep0 rowsep0" align="left">geometric (points)</td><td class="colsep0 rowsep0" align="left">Off-target prediction<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Method of Wu et al.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a></td><td class="colsep0 rowsep0" align="left">not publically available</td><td class="colsep0 rowsep0" align="left">fingerprint</td><td class="colsep0 rowsep0" align="left">Identification of novel targets<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PESD<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a></td><td class="colsep0 rowsep0" align="left">Web server (open)</td><td class="colsep0 rowsep0" align="left">distribution</td><td class="colsep0 rowsep0" align="left">Prediction of compound selectivity profiles and affinities<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PocketFEATURE<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></td><td class="colsep0 rowsep0" align="left">Web server (free for academic users), software (free for academic users)</td><td class="colsep0 rowsep0" align="left">fingerprint</td><td class="colsep0 rowsep0" align="left">Protein–ligand interactions<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PocketMatch<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a></td><td class="colsep0 rowsep0" align="left">Web server (free), software (free)</td><td class="colsep0 rowsep0" align="left">fingerprint</td><td class="colsep0 rowsep0" align="left">Function prediction<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Polypharmacology prediction<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Homolog analysis<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PoLiMorph<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a></td><td class="colsep0 rowsep0" align="left">not publically available</td><td class="colsep0 rowsep0" align="left">graph</td><td class="colsep0 rowsep0" align="left">Binding site prediction<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ProBiS<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a></td><td class="colsep0 rowsep0" align="left">Web server (free)</td><td class="colsep0 rowsep0" align="left">graph</td><td class="colsep0 rowsep0" align="left">Function prediction<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Off-target prediction<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PSIM<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></td><td class="colsep0 rowsep0" align="left">software (commercial)</td><td class="colsep0 rowsep0" align="left">grid</td><td class="colsep0 rowsep0" align="left">Off-target prediction<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Query3d<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a></td><td class="colsep0 rowsep0" align="left">Web server (free)</td><td class="colsep0 rowsep0" align="left">geometric (points)</td><td class="colsep0 rowsep0" align="left">Evolutionary relationships<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Shaper<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></td><td class="colsep0 rowsep0" align="left">software (free)</td><td class="colsep0 rowsep0" align="left">grid</td><td class="colsep0 rowsep0" align="left">Protein–ligand interactions<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SiteAlign<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></td><td class="colsep0 rowsep0" align="left">software (free)</td><td class="colsep0 rowsep0" align="left">fingerprint</td><td class="colsep0 rowsep0" align="left">Protein–ligand interactions<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Protein–ligand interactions<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SiteEngine<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a></td><td class="colsep0 rowsep0" align="left">Web server (free for academic users), software (free for academic users)</td><td class="colsep0 rowsep0" align="left">geometric (points)</td><td class="colsep0 rowsep0" align="left">PPIs<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMAP<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a></td><td class="colsep0 rowsep0" align="left">software (free for academic users)</td><td class="colsep0 rowsep0" align="left">graph</td><td class="colsep0 rowsep0" align="left">Polypharmacology prediction<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Drug repurposing<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Drug repurposing<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SOIPPA<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a></td><td class="colsep0 rowsep0" align="left">not publically available</td><td class="colsep0 rowsep0" align="left">graph</td><td class="colsep0 rowsep0" align="left">Drug repurposing<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Off-target prediction<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SurfaceScreen<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a></td><td class="colsep0 rowsep0" align="left">not publically available</td><td class="colsep0 rowsep0" align="left">distribution</td><td class="colsep0 rowsep0" align="left">Binding site prediction<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TM-align<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a></td><td class="colsep0 rowsep0" align="left">Web server (free), software (free)</td><td class="colsep0 rowsep0" align="left">geometric (points)</td><td class="colsep0 rowsep0" align="left">Drug repurposing<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Links to the respective Web servers, mail servers, and Web sites with the downloadable software are included in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf" class="ext-link">Supporting Information</a>. Because PSSC<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> cannot be regarded as an independent algorithm, it is not included in this table.</p></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Different Levels of Binding Site Comparison</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31757" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31757" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We do not intend to explain the underlying algorithms in detail but rather aim to provide some basic information on the different methods and to place emphasis on their unique properties. Due to the huge amount of available tools, we focus on a small set of algorithms which were successfully applied in the field of medicinal chemistry. The <a href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf" class="ext-link">Supporting Information</a> contains a more comprehensive list of developed methods and their availability.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Binding Site Comparisons Based on Fingerprints</h3><div class="NLM_p">FLAP (Fingerprints for Ligands And Proteins)<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> is based on the binding site analysis using GRID<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> molecular interaction fields. A limited series of distinct chemical probes is used to scan the protein cavity. This analysis results in the identification of favorable and unfavorable interactions. The application of a weighted energy-based and space-coverage function allows the condensation into fewer pharmacophore property-based points. For each protein or binding site of interest all possible four-point pharmacophores are calculated. Target pair comparisons are performed as many times as there are binding sites under investigation. Two proteins are assumed to be similar if the four-point pharmacophores of proteins A and B are equivalent and their cavity shapes are similar. The similarity is calculated by multiplying the total number of shape coincidences <i>T</i>(<i>i</i>,<i>j</i>) with the number of shape coincidences of the inverted match <i>T</i>(<i>j</i>,<i>i</i>) and dividing this term by <i>T</i>(<i>i</i>,<i>i</i>)·<i>T</i>(<i>j</i>,<i>j</i>) to normalize the corresponding similarity values.</div><div class="NLM_p">Another method called IFP (Interaction FingerPrints)<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> translates the 3D information about protein–ligand interactions into a one-dimensional bit vector representation. Interactions (hydrogen bonds, weak hydrogen bonds and ionic, π–cation, metal, hydrophobic interactions as well as face-to-face and edge-to-face aromatic interactions) are determined using different geometric rules. A modified form of the Tanimoto coefficient is used as a measure for similarity.</div><div class="NLM_p">KRIPO (Key Representation of Interaction in Pockets)<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> analyzes ligand fragments and local binding sites that were derived using a set of transformation rules for bond cleavage. Altogether, 299 591 fragments were extracted and included in the KRIPO database with their respective local protein binding sites which are represented by residues with at least one atom within the radius of 6 Å of any ligand atom. The features to derive pharmacophore-based fingerprints are based on interactions at certain positions relative to the residues (hydrogen bond donor or acceptor, aromatic T- or π-stacking interaction, hydrophobic contacts, interaction of positively or negatively charged moieties). After feature assignment to the pocket residues, only pharmacophore features within 2.5 Å of any of the ligand fragment atoms are used for binding site representation. The distances between pharmacophore features are binned into discrete distance ranges according to a set of binning schemes. Three fingerprint schemes (two-point, three-point, and four-point pharmacophores) were compared to optimize the method. As a result of the optimization procedure, fuzzified 3-point fingerprints are applied to represent local binding sites. The feature opposite the shortest distance is prioritized to calculate pairwise distances. As a similarity measure, the developers use a modified Tanimoto similarity fingerprint based on the mean bit density.</div><div class="NLM_p">SiteAlign<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> makes use of a projection of eight topological and physicochemical properties from cavity-lining protein residues to an 80 triangle-discretized sphere placed at the center of the binding site of interest. A sphere with a radius of 1 Å is placed at the active site center and three topological (distance between Cβ and the sphere center, side chain orientation, size) as well as five chemical descriptors (hydrogen bond donor or acceptor, aromatic, aliphatic, charged) are used to represent the cavity residues. The cavity descriptors are projected onto the sphere according to the respective amino acids. This is achieved by deriving a geometrical vector from the Cα atom of the active site residues to the sphere center and a respective descriptor assignment to the 80 triangles. The resulting fingerprint of 640 integers (8 descriptors, 80 triangles) is subsequently used to compare binding sites with each other. The similarity score is calculated according to the sum of normalized differences for each triangle descriptor and the number of triangles with non-null values.</div><div class="NLM_p">FuzCav<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> uses generic 4833-integer vectors describing druggable protein–ligand binding sites. In the first step, the Cα atom coordinates of residues lining the binding site are extracted and annotated by six pharmacophoric properties depending on the corresponding residues (hydrogen bond donor or acceptor, positive or negative ionizable, aromatic, aliphatic). The vector’s integers register the count of unique pharmacophore feature triplets (three properties and three respective distances between Cα atoms). The vector size is decreased by distance binning and removal of redundant triplets. In comparison to most other methods, this approach uses exclusively Cα atoms for the structural representation of the binding site. Another publication<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> underlines the applicability of this type of pocket representation. The developers calculate the binding site similarity via division of the number of common non-null counts in both fingerprints by the minimum number of non-null counts for the fingerprints of protein A and B.</div><div class="NLM_p">The following two methods do not use fingerprints but make use of 2D representations of geometric binding site properties in the form of lists and strings. Those approaches lead to comparable or even lower computational costs as compared to fingerprint-based methods.</div><div class="NLM_p">PocketMatch<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> is based on a one-dimensional representation of the binding site by a set of all distance pairs between points derived from the chemical nature of the cavity residues. By default, 20 natural amino acids are divided into five distinct groups. A pair of sorted distance sequences is aligned, and similarities are scored according to the average of the number of matching distance elements.</div><div class="NLM_p last">The method of Wu et al.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> compares local structures in proteins on the basis of their one-dimensional structural sequence according to a so-called structural alphabet.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> The backbone of a set of nonredundant protein structures is represented by consecutive five residue segments. They are described by a vector of the respective eight backbone dihedral angles <i>V</i>(ψ<sub><i>n</i>–2</sub>,ϕ<sub><i>n</i>–1</sub>,ψ<sub><i>n</i>–1</sub>,ϕ<sub><i>n</i></sub>,ψ<sub><i>n</i></sub>,ϕ<sub><i>n</i>+1</sub>,ψ<sub><i>n</i>+1</sub>,ϕ<sub><i>n</i>+2</sub>). The dissimilarity between two of those vectors is determined by calculating the rmsd of the particular dihedral angles. Altogether, 16 distinct protein blocks were identified with the help of an unsupervised cluster analyzer. They were defined as the structural alphabet. This enables the translation of 3D protein structures into strings of structural letters by the use of a five residue sliding window and the assignment of the closest structure out of the structural alphabet. Recurring structural patterns (more than three times in a set of nonredundant proteins) are then classified as potential 3D motifs, avoiding the necessity of a query motif or the knowledge of essential residues. Furthermore, this method does not consider any local physicochemical similarities.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Binding Site Comparisons Based on Grids</h3><div class="NLM_p">The APF (Atomic Property Fields) method uses continuous pharmacophore-based atomic property fields as previously proposed for ligands.<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a>Seven atomic properties (hydrogen bond donor or acceptor, lipophilicity, size, electronegativity, charge, hybridization) of the binding sites are represented as continuous potentials projected in 3D space from atom centers using Gaussian functions.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Grids are generated with seven APF potential components. One binding site is defined as a collection of receptor atoms carved by a sphere around the ligand of the protein structure and is used to generate seven-component APF potentials on a grid. The second site (regarded as rigid) is placed in these grid potentials and subjected to a Monte Carlo minimization procedure to elucidate the optimal superposition. The similarity of atomic property distributions is represented by the APF pseudoenergy for the optimal alignment. Distance measures are derived for the calculation of distance matrices and subsequent clustering.</div><div class="NLM_p">PSIM (Protein SIMilarity)<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> is a binding site comparison method based on the morphological similarity between protein pockets. It is derived from a comparison method for small molecules.<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> Observation points are used to measure the surface of the binding site as seen from the ligand’s perspective. Originally, the presence of a bound ligand in the structure was a prerequisite for comparison. This necessity could be circumvented by the introduction of an additional binding site detection method.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> Each binding site is represented by a tessellation with so-called observation points as determined from bound molecules. The distance to the nearest steric, positively charged, and negatively charged surface is measured. The similarity function is based on the comparison of distances to the molecular surface from these points. Successively, an alignment is searched that maximizes the similarity function. For this scope, the bound ligand is placed into a uniform grid. Molecule grid points within a chosen distance are weighted on the basis of their minimum distance to the molecular surface. Because the so-called observation points will perceive the same things in the optimal alignment when the molecular surfaces are highly similar in shape and polarity, the alignment is optimized. A transformation is applied to move each observation point set of the opposing protein onto the reference pocket, and the differences in the corresponding surface measurements are minimized. The similarity is calculated as the normalized sum of weighted Gaussian-like functions of differences in distances from observation points to molecules.</div><div class="NLM_p last">A tool called Shaper<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> compares protein binding sites with respect to their physicochemical properties. Regularly spaced pharmacophore-based grid points represent the pocket, thereby creating an inverse image of the binding site (as perceived by the ligand). The annotation of cavity grid points is realized by a projection of a protein–ligand complex on a grid. A property is assigned to each cell according to its location with respect to the protein (hydrophobic, aromatic, hydrogen bond donor or acceptor, positively or negatively ionizable, null). A shape-based alignment method is used to align cavities by an optimization of the annotated shapes’ volume overlap. The similarity is calculated by a Tversky index. As compared to Tanimoto, this similarity measure allows for weight assignment toward the reference and fit pocket.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Binding Site Comparisons Based on Graph Models</h3><div class="NLM_p">In 2002, Schmitt et al. published a binding site comparison method that relies on a database called Cavbase.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> This assembly of cavities, identified using the cavity detection method LIGSITE,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> was created for the data stored in the Relibase<a onclick="showRef(event, 'ref107 ref108'); return false;" href="javascript:void(0);" class="ref ref107 ref108">(107, 108)</a> database. The physicochemical properties of the cavity-flanking residues are mapped onto so-called pseudocenters corresponding to five properties (hydrogen bond donor or acceptor, mixed donor and acceptor, hydrophobic aliphatic, aromatic), leading to the generation of a graph model. The Bron–Kerbosch clique detection algorithm is applied to find a maximum common subgraph. Similarity scoring is achieved by a calculation of the overlap in surface points which are assigned to identical physicochemical properties.</div><div class="NLM_p">eF-site (electrostatic surface of Functional site)<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> is a database of protein surfaces for the comparison of functional sites of known protein structures. The electrostatic potential of the protein’s binding site and hydrophobic scales were used to label the vertices of the graph models used for comparison. Functional sites were selected according to bound ligands or active site annotation. The molecular Connolly surfaces were labeled with electrostatic potentials as well as hydrophobic scales and represented as graphs. Finally, the binding site comparison was performed using the Bron–Kerbosch clique search algorithm. The similarity score was derived from the nodes and edges of the largest clique (based on electrostatic potentials, hydropathy values, distances, angles, and dihedral angles).</div><div class="NLM_p">The method IsoCleft<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> derives pocket atoms using the SURFNET algorithm<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> or the conservation scores based on the ConSurf-HSSP database.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> The cleft is represented as a graph model. The largest common subgraph isomorphism is searched by the construction of an association graph (nodes are atom pairs with chemical similarity, and edges are drawn if atom pairs are geometrically similar). Eight atom types are used to define chemical similarity (hydrophilic, hydrogen bond donor or acceptor, hydrophobic, aromatic, neutral, neutral donor or acceptor). The largest subset of atoms with identical types is searched in two consecutive steps to reduce computational costs. First, an initial superposition is achieved by detecting the largest clique in an association graph by the Bron–Kerbosch algorithm using only Cα atoms of equivalent residues. Atoms are then superimposed on the basis of the transformation matrix and translation vectors of the largest clique by a least-squares fitting procedure. In a second step, all non-hydrogen atoms are matched with two restrictions: atoms have to be of the same type, and the spatial distances after the first superimposition have to be within a certain threshold. Cα atoms are not used in this step. Similarity is determined by three measures: modified Tanimoto scores, clique size, and an additional measure which reflects the binding sites size.</div><div class="NLM_p">Xie and Bourne<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> developed a sequence-order independent profile–profile alignment (SOIPPA) that can be used to detect local similarities between unknown binding sites by comparing whole proteins. The approach is based on a protein structure representation using Delaunay tessellation of Cα atoms characterized by geometric potentials. A mesh surface is generated around the Cα atoms so that a normal vector that is perpendicular to the mesh surface can be assigned to each Cα atom. It also encodes a graph representation in 3D space where the Cα atoms represent nodes which are connected by edges derived from the connections in the regular tessellation. This graph representation is used to find the maximum weight common subgraph between two proteins. The weight is represented by chemical similarity using the McLachlan chemical similarity matrix<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> or evolutionary correlation represented by the BLOSUM45 substitution matrix.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> Later, a unified statistical model was introduced to assess the statistical significance of found similarities. SOIPPA and the established extreme value distribution model were combined in the software SMAP.<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a></div><div class="NLM_p">ProBiS (Protein Binding Sites)<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> was developed to compare patterns of physicochemical properties on proteins’ surfaces. Using graph models and a maximum clique detection method,<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> the software was designed to find local structural similarities between a query protein and other known protein structures. This algorithm compares complete proteins without previous definition of binding sites. Therefore, the method can also be applied to detect yet unknown binding sites. Finally, four scoring measures are provided: surface vectors angles (outer-pointing surface vector originating in the geometric center and perpendicular to the protein’s surface for each superimposed set of vertices), surface patch rmsd, surface patch size, <i>E</i>-value.</div><div class="NLM_p last">Another graph-based comparison method is PoLiMorph (Pocket-Ligand Morphing).<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> Proteins are represented by self-organizing graphs that fill the cavity volume. Instead of clique detection, a fast heuristic algorithm was developed. The vertices of these 3D models are labeled with information about the local ligand–receptor interaction potential coded by fuzzy property labels to replace a crisp property assignment to the vertices by smooth functions. The properties (buriedness, solvatability, electrostatics, and hydrogen bond terms) are directly derived from the structure and projected onto the vertices. On the basis of the original GRID<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> approach, the vertex labels are integrated from five grid potentials of their surrounding grid points. After these steps are finished, a 3D fuzzy pocket graph is fit into the pocket. The grid point coordinates are used as input vectors. The final graph comparison is based on an error-tolerant heuristic for 3D graph matching. The scoring function includes similarities with respect to property distributions and protein–ligand interaction patterns.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Binding Site Comparisons Based on Geometry (Points)</h3><div class="NLM_p">The method of Binkowski et al.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> identifies all surface pockets and interior voids of proteins stored in the PDB. A set of protein surface patterns called pvSOAR patterns (pocket and void surface patterns of amino acid residues) was built for each protein pocket and void in the CASTp database.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> The similarity of the patterns of pocket-forming residues is assessed in terms of sequence, spatial arrangement, and directional arrangement. Protein pockets and interior voids are calculated using weighted Delauney triangulation and the α shape method. A set of protein surface patterns is derived. Their sequence patterns are used to assess the similarity relationship among protein surfaces using the Smith–Waterman algorithm as implemented in SSEARCH<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> and the BLOSUM50 substitution matrix.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> Shape comparison is performed to obtain an optimal alignment in terms of rmsd. One point is used to represent a residue (more than one surface atom: geometric center of atoms). Additionally, a different scoring measure for the dissimilarity in pocket shape called oriented rmsd (ormsd) was introduced to overcome the drawbacks of coordinate rmsd (increases as the number of aligned residues increases, will be dominated by a few outliers). The method combines two levels of binding site comparison: structural binding site comparison and binding site sequence comparison.</div><div class="NLM_p">SiteEngine<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> matches binding sites or proteins on the basis of a geometric representation of their physicochemical features. In comparison to computationally demanding clique detection algorithms, this method relies on efficient hashing and matching of triangles of centers with different properties (e.g., hydrogen bond donor or acceptor, mixed donor and acceptor, hydrophobic aliphatic, aromatic). The modeling of the binding sites’ Connolly surfaces is similar to that of the Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> approach. All possible transformations for the superposition of two pockets are calculated and scored in a hierarchical manner. The final scoring function for the optimized match includes goodness-of-fit between matched pseudocenters as well as the size and shape of the common overlap region. In the following year, two Web servers were provided for the comparison of binding sites and protein–protein interfaces with SiteEngine.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a></div><div class="NLM_p">A fast protein comparison at atomic level is also possible with a method developed by Brakoulias and Jackson.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> A series of triplets is generated by building “seed” matches consisting of three atoms forming a triangle. Atom to atom distance cutoffs are introduced to restrict the number of triangles and their size (upper and lower boundary). Atom types (carbon, nitrogen, oxygen, sulfur, phosphor atoms) and distances are stored. For the reference binding site, a discretized 3D image is generated. It constitutes the reference frame where atom types are stored at indices in a Cartesian grid (grid size controls resolution). The comparison is achieved by systematically matching all possible triplets between two objects. For each match within a predefined maximum length difference a transformation is performed using a least-squares fitting procedure. The resulting transformation (translational and rotational matrix) is applied to the second object by converting the <i>x</i>, <i>y</i>, <i>z</i> coordinates. Then, a score of unity is given to each atom type match. Those steps are repeated to find the match with the highest score in a way that the rmsd of the superposed atoms is below 1 Å. Finally, the transformation matrix with the highest score is stored. As compared to an additionally implemented Bron–Kerbosch algorithm for clique detection, this matching procedure was shown to be 2 orders of magnitude faster. Nevertheless, it is stated that the definition of the binding site is crucial for the outcome of an all-against-all comparison as minor shifts introduce shifted alignments of identical environments.</div><div class="NLM_p">The method TM-align<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> was originally developed to identify the best structural alignment between protein pairs. By definition, it is not an explicit binding site comparison approach. Nevertheless, it is often used to find local similarities between proteins and was successfully applied in various scenarios. Furthermore, it is nicely suited to explain an algorithm for the spatial alignment of proteins. In the first step, an initial alignment of secondary structures of two proteins is generated using dynamic programming where the element of the score matrix is assigned to be 1 or 0 depending on the identity of the aligned secondary structure elements using the DSSP<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a> assignment. Additionally, a gapless threading against the larger structure is performed as in another method called SAL.<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> The methods differ in the assignment of scores. Whereas the rmsd is used as measure in SAL,<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> TM-align uses a different score (TM-score) to select the optimal alignment. The third alignment is again obtained by dynamic programming using a gap-opening penalty of −1. The scoring matrix is a combination of secondary structure and distance score matrix from the second alignment. A heuristic iterative algorithm is used to refine these initial alignments. The structures are rotated by the TM-score rotation matrix on the basis of the aligned residues. Again, similarity scores are calculated. A new alignment can be obtained by implementing dynamic programming on the similarity matrix. This process is iterated until a stable alignment is reached. The alignment with the highest TM-score is returned.</div><div class="NLM_p">Query3d<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> compares binding sites with respect to three similarity criteria: all residues in a set of matching amino acids have to be neighbors of at least one of the other residues in the set; the rmsd of the Cα atoms and the coordinates of the geometric average of all residue atoms has to be below a certain threshold; the similarity between two residues has to be above 0.3 according to the Dayhoff substitution matrix<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> (average substitution value of the complete structural match has to be below 1.2). For two protein structures an exhaustive depth-first search is used to find the largest set of matching residues. All matches of a length of one residue are created. Each single amino acid of the reference structure is matched to each amino acid of the target structure. For a reference protein consisting of <i>m</i> amino acids and a target protein with <i>n</i> residues, this step results in <i>n</i> × <i>m</i> matches. All matches violating one of the three above-mentioned criteria are discarded leading to a reduction of the number of possible matches. Then, match extension is performed using length two matches. All neighboring amino acids of the previously matched residues are taken into account. Assuming that the residue of the reference protein has <i>i</i> neighbors, while the matching residue has <i>j</i> neighbors, this leads to <i>i</i> × <i>j</i> new matches for the initial length one match. For length two matches which were considered valid according to the predefined exclusion criteria, length three matches are generated. The procedure is repeated as long as valid matches are obtained. The algorithm automatically stops when 10 residues are matched to save computation time. This is a hint for globally similar proteins which are of no interest for this binding site comparison method.</div><div class="NLM_p">The algorithm CPASS (Comparison of Protein Active Site Structures)<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> aims to compare ligand-defined protein active sites by means of optimal sequence and structural alignment without maintaining sequence connectivity. A maximization of the rmsd-weighted BLOSUM62 substitution matrix<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> is performed during the alignment. The rmsd between two aligned residues is additionally corrected by 1 Å to account for experimental inaccuracies of protein structures. Furthermore, the score is scaled by the shortest distance from a residue at the active site to the ligand to account for small structural changes at the boundaries of the active sites.</div><div class="NLM_p">The fPOP (footprinting Pockets Of Proteins)<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> database was generated by a binding site comparison based on 38 900 split pockets as generated by the SplitPocket<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> approach. The superimposition was performed via geometrical matching of spatial patterns of the templates with putative pockets in the unbound form. Site-specific spatial patterns were compared by the application of the Smith–Waterman algorithm (generation of local pairwise alignments) and shape matching to compare an unbound pocket against canonical functional surfaces of all split pockets. A surface conservation index is determined to evaluate the results. The assessment of the statistical significance for a surface alignment was achieved using an approach similar to that of Binkowski et al.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a></div><div class="NLM_p">CMASA (Contact MAtrix local Structural Alignment)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> is an algorithm which performs a fast and sensitive binding site comparison via various steps. Cα atoms and the residue atom furthest away from these atoms for each residue in the protein structure represent the binding site. All local structures that match with the query structure in terms of BLOSUM62 similarity<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> are aligned. Most local matches are removed with respect to the contact matrix average deviation, a measure of the distances between each two atoms belonging to the template and those two of the matched local structure. Only those matches above a certain cutoff are used. Finally, the rmsd, <i>p</i>-values, and ranks are calculated.</div><div class="NLM_p">The PocketFEATURE algorithm<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> first assigns microenvironments to proteins. Local spherical regions with a radius of 7.5 Å are extracted from the structure. Altogether, 80 physicochemical properties are collected over six concentric spherical shells centered on a predefined functional center. The FEATURE microenvironments<a onclick="showRef(event, 'ref122 ref123'); return false;" href="javascript:void(0);" class="ref ref122 ref123">(122, 123)</a> of two binding sites are compared by an exhaustive calculation of raw Tanimoto scores of all available microenvironment pairs. A search for the best scoring microenvironment pairs (with a score cutoff) is performed. Finally, a microenvironment similarity score based on a normalized Tanimoto score is calculated as a measure for binding site similarity. A rigid geometric matching criterion is avoided enabling a flexible matching between two microenvironments.</div><div class="NLM_p">COFACTOR<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> allows for the comparison of proteins to other proteins with known protein–ligand binding interactions, EC, or GO (gene ontology) classifications. The protein of interest is compared to a database of structures using TM-align<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> to reveal global structural similarities which might help to identify family or fold relationships. Nonrandom structural similarities are investigated in more detail in a subsequent local functional site identification step. By means of a heuristic algorithm, query and template structures are compared to identify the best local match. Evolutionary conserved residues according to a multiple sequence alignment are identified and excised from the query structure. The resulting set of local structural motifs is superimposed onto known functional site residues of the template structures. A subsequent iterative superposition refinement similar to TM-align<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> is applied. Finally, the match with the highest score in terms of Cα distances and BLOSUM62 substitution scores<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> is retained.</div><div class="NLM_p last">A different approach compares proteins based on secondary structure elements, which is called PSSC (Protein Structure Similarity Clustering).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Compared to all other methods presented here, it does not represent an independent algorithm. A workflow is applied to find similarities for available structural data. The alignment of the binding site secondary structure elements has to be done manually according to the interesting matches. The workflow consists of a structural alignment that involves a combinatorial extension of an alignment path defined by aligned fragment pairs,<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a> the calculation of the rmsd of all Cα atoms, and the visual inspection of the alignment to ensure its relevance. The applicability of this approach is based on the assumption that a similar spatial arrangement of secondary structure elements that are part of the ligand binding site enables the binding of similar scaffolds. Or vice versa, the molecular framework as definition of the scaffold<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> is able to recognize targets whose binding sites are similar in terms of secondary structure architecture. A review discusses the impact of secondary structure element information on the binding of similar ligands and its impact on drug design.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Binding Site Comparisons Based on Shape Distributions</h3><div class="NLM_p">A further development of the pvSOAR methodology introduced earlier was achieved by the introduction of the SurfaceScreen algorithm<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> to optimize the matching of global shape and local physicochemical features to evaluate the similarities between surface pairs. A 3D object recognition algorithm is used to find surface shape similarities, while the similarity of physicochemical features is assessed using a spatial alignment of conserved residues between surfaces. Surface shape signatures were introduced to represent the signature of an object as a probability distribution sampled from a shape function that measures global geometric properties. As compared to shape matching, this approach leads to a complexity reduction, since two probability distributions are analyzed. SurfaceAlign was developed to compare the spatial arrangement of localized surface patterns and to examine side chain orientations. The 3D representation of a surface residue is achieved by a single point at the center of mass of all residue atoms. Alignment of two surfaces is achieved by combinatorically identifying the best superposition of the maximum subset of conserved residues. A library of annotated surfaces, is used to reveal undiscovered similarities between proteins. A surface volume overlap Tanimoto was introduced to score found similarities.</div><div class="NLM_p last">The PESD (Property-Encoded Shape Distributions)<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> method compares binding sites with respect to shape and property distributions on their surface. Cavities are represented by property-mapped triangulated Gauss–Connolly surfaces. Property-encoded shape distributions are calculated for all binding site surfaces to provide a measure of the probability that a particular property will be located at a specific distance to another one. For the comparisons, D2 shape distributions are used. Each vertex of the triangulated surface is represented by its Cartesian coordinates and a property-based color code (RGB) that encodes the magnitude of its surface properties. The Euclidean distance between two randomly chosen points is calculated, and the respective color codes are recorded. Afterward, distances are binned in 19 bins of 1 Å where each bin is further divided into 4096 sub-bins for the different possible color combinations. Binding site surfaces were created using MOE active LP color coding and electrostatics color coding. The weighted sum of χ<sup>2</sup> statistic for electrostatic and active LP properties is used as dissimilarity measure. The developers also provide a Web server for the comparison of user-defined binding sites against a surface database of 106 796 sites.<a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Applications of Binding Site Comparison Algorithms</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28683" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28683" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The following section is dedicated to numerous successful applications to outline the impact of binding site comparison methods on medicinal chemistry and rational drug design in general. Certainly, one could argue that most of the previously cited publications concerned with the development of binding site algorithms also provide various interesting applications. Nevertheless, the more interesting question remains whether those methods proved to be useful in independent projects. Most of the algorithms presented above are used to explain polypharmacology, to annotate proteins, and to predict functions. However, their applicability domains clearly exceed those three examples. The section is roughly divided into two parts to address the different potential readerships: The first part deals with applications with impact on a medicinal chemistry program (hit identification, hit-to-lead, or lead optimization), whereas the second part deals with more general applications within a broad drug design context.</div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Applications with Impact on a Medicinal Chemistry Program</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03313" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03313" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Similarity-Based Virtual Screening Approaches</h3><div class="NLM_p">Protein binding site similarity can be exploited as an idea generator for new potential ligands or the improvement of existing ligands. On the basis of the assumption that similar ligands bind to similar binding sites, different virtual screening approaches were developed, most of them relying on known ligands (e.g., bioisosteric replacements, scaffold hopping, focused libraries, virtual screening of ligands with high similarity toward a known ligand). The major disadvantage of all ligand-based methods is the fact that they rely on the knowledge of at least one bioactive compound of the target of interest. If nothing is known about small molecules binding to the target, a ligand-based approach becomes useless. For such challenges, binding site comparison comes into play. Once a similarity between the binding site to be addressed and a binding site of a well-characterized target is found, the scientist can look for compounds displaying bioactivity on the similar target and test them on the protein of interest.</div><div class="NLM_p">A promising example is the application of Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> to design novel peptide aldehyde-based reversible inhibitors of the SARS coronavirus main protease, SARS-CoV M<sup>pro</sup>.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Peptide-bound crystal structures of this protease were analyzed. Subsequently, a modeling study was performed using Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> searches and molecular docking. The binding pocket of SARS CoV M<sup>pro</sup> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uk4">1uk4</a>) was divided into the different amino acid recognition subpockets which were then subjected to single Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> searches for similar subpockets. A detailed analysis of the obtained results is explained in another article.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The information about bound ligand fragments in the related subpockets was then used as a guideline for the design of suitable side chains of a potent protease peptide inhibitor. In sum, 1230 natural and non-natural amino acids were used for docking studies covering the S1, S2, and S4 binding pockets while the localization of the peptide backbone as observed in known protein–peptide complexes was retained. The analysis suggests that the S1 pocket can be addressed by a glutamine, the S2 pocket prefers aromatic side chains superior to the native leucine, and the S4 pocket is able to complex various anionic and uncharged polar amino acids. Indeed, four of the rationally designed peptides exhibit the highest potential of all peptide aldehydes tested. They all comprise glutamine residues in the P1 position (AcAsn-Ser-Thr-Asp-Gln-H, AcAsn-Ser-Thr-Ser-Gln-H, AcGlu-Ser-Thr-Leu-Gln-H, AcAsp-Ser-Thr-Leu-Gln-H) and show IC<sub>50</sub> values of 20, 7.5, 7.5, and 10 μM, respectively. Glutamate seems to be preferred over aspartate in the P5 position, while introduction of a histidine proved to decrease activity (IC<sub>50</sub> for AcHis-Ser-Thr-Leu-Gln-H of >50 μM). Introduction of hydrophobic and aromatic side chains at position P3 addressing the S2 pocket led to slight improvements. Crystal structures of four enzyme-bound peptide aldehydes allow for a more detailed analysis.<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> Additionally, <i>K</i><sub>i</sub> values of 40, 72, 8, and 41 μM were reported for the four peptide aldehydes AcAsn-Ser-Thr-Ser-Gln-H, AcAsn-Ser-Phe-Ser-Gln-H, AcGlu-Ser-Thr-Leu-Gln-H, and AcAsp-Ser-Phe-Asp-Gln-H, respectively. Those results show that hydrophilic serine or charged aspartate residues in P2 position instead of the hydrophobic residues in P3 position occupy the hydrophobic S2 pocket and represent an example for unexpected methionine–carboxylate interactions. Additionally, the preference for a glutamate in P5 position over an aspartate residue could be verified.</div><div class="NLM_p">The PSSC approach<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> was used to find potential inhibitors of lysine-specific demethylase 1 (LSD1), an enzyme removing mono- and dimethyl marks from lysine 4 or 9 of histone H3. The protein in association with the androgen receptor controls androgen-dependent gene expression and prostate tumor cell proliferation.<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> The authors extracted the LSD1 binding site from the reported crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ejr">2ejr</a>) and performed a Dali<a onclick="showRef(event, 'ref129'); return false;" href="javascript:void(0);" class="ref ref129">(129)</a> similarity search against all proteins stored in the PDB. The resulting alignments were visually inspected to verify the presence of a similar ligand-sensing core. Besides the identification of structurally related proteins, similarities between LSD1 and the members of the monoamine oxidase (MAO) family MAO-A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2bxr">2bxr</a>) and MAO-B (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1gos">1gos</a>) were found. Inspired by the finding that γ-pyrones are a novel class of reversible MAO-A and MAO-B inhibitors,<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a> a library of 705 compounds was screened for LSD1 inhibition. Consequently, namoline (<b>1</b>)<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> was found to inhibit LSD1 demethylase activity with an IC<sub>50</sub> of 51 μM. Finally, the compound was tested on MAO enzymes which are inhibited by the related molecules <i>N</i>-(3-(2,4-dichlorophenoxy)propyl)-<i>N</i>-methylprop-2-yn-1-amine (clorgyline)<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> and <i>N</i>-benzyl-<i>N</i>-methylprop-2-yn-1-amine (pargyline).<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> Compound <b>1</b> did not inhibit MAO-A or MAO-B because the sequence similarity between the ligand-sensing cores of LSD1 and both enzymes is only 19%. <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> gives a detailed picture of the superimposed binding sites of LSD1 and MAO-B after an alignment according to the common secondary structure elements. This example shows how the presence of common ligand-sensing cores can facilitate the search for new ligands without bearing the risk of not being able to design compounds with a promising selectivity profile. To further validate the applicability of <b>1</b> to treat prostate cancer, evidence for the compound’s antiproliferative effect on LNCaP cells (androgen-sensitive human prostate adenocarcinoma cells) was provided. In vivo studies with xenograft tumors (subcutaneous implantation of LNCaP cells into nude mice) were performed to analyze the effect of <b>1</b> on tumor cell proliferation. Despite some adverse effects (slight weight loss, minor liver toxicity), treatment with <b>1</b> was shown to be beneficial in terms of proliferation inhibition.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Common ligand-sensing core of LSD1 and MAO-B: (a) structure of compound <b>1</b>; (b) common ligand-sensing core of LSD1 in green (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ejr">2ejr</a>) and MAO-A with bound inhibitor in orange in ball-and-sticks representation (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2bxr">2bxr</a>); (c) active sites of both enzymes with cavity-lining residues represented as sticks and names according to the crystal structures. Parts b and c were generated using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The last example<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> discusses a successful combination of an IFP comparison<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> and a molecular docking approach for the selective virtual screening for ligands with a specific functional effect. The signaling activity of GPCRs can be stimulated (full or partial agonist), blocked (antagonist), or reduced (inverse agonist) by various small molecules. To date, it is quite demanding to find a new potential agonist or antagonist. Known complexes of different small molecules with various effects in 31 β1 and β2 adrenoceptor crystal structures were used to derive unique protein–ligand IFPs. Altogether, the IFPs of 33 β1 adrenoceptor and 15 β2 adrenoceptor binding sites were calculated. Additionally, physicochemically similar decoys were included for subsequent docking studies. After postprocessing of the complexes of 1920 unique effector molecules, function classification on the basis of the protein–ligand interaction profiles was performed. The enrichment of specific ligand types was strongly influenced by differences between receptor conformations and reference IFPs for docking pose postprocessing. Optimum structure IFP combinations could be established to successfully discriminate between antagonists, inverse agonists, and partial or full agonists. The predicted IFP for the full agonist norepinephrine gave the highest enrichment of agonists over antagonists for all structures. Additionally, the so-called biased ligands in the data set were analyzed. GPCRs transfer signals via G-proteins and β-arrestins.<a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a> Therefore, the data set was screened for compounds that induce signaling via the β-arrestin pathway and those that do not bias signaling. The IFPs were grouped according to biased and unbiased ligands. The difference in the abundance of each interaction with each pocket residue was calculated. Consequently, four interactions, which are abundant (≥50%) and unique for biased ligands, were identified. In summary, this study nicely shows how the comparison of binding sites facilitates the search for GPCR ligands that show a desired effect. IFP comparison might become a suitable tool for the elucidation of the biased signaling mechanism in GPCRs.</div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Drug Repurposing</h3><div class="NLM_p">The challenging and laborious process of discovering a small molecule being able to exhibit the desired pharmacological effect without leading to severe side effects can be facilitated by exploiting the knowledge about approved drugs. The choice of compounds with known toxicity profiles, efficacies, and pharmacokinetic and pharmacodynamic properties leads to reduced attrition rates. Drug repurposing (also called repositioning, redirecting, reprofiling) approaches benefit from this knowledge. They make use of a plethora of computational methods.<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> Generally speaking, most strategies for repositioning are based on high-throughput in vitro and in vivo screens using a panel of approved drugs. Obviously, binding site comparison methods can help to find new indications for old drugs by comparing the new target to well-characterized targets with known approved drugs.<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> The predictivity of this approach is shown by the huge number of articles published about the repositioning of prominent drugs. A review<a onclick="showRef(event, 'ref135'); return false;" href="javascript:void(0);" class="ref ref135">(135)</a> gives information about successful drug repositioning strategies. Here, we show a few examples underlining how the comparison of binding sites can facilitate this process.</div><div class="NLM_p">An exhaustive study<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> published in 2013 focused on the reasons for drug promiscuity as prerequisite for drug repositioning. The analysis investigated the subject from two perspectives: ligand features and binding site features leading to drug promiscuity. A workflow comprising three steps was conducted to address the second question whether promiscuous drugs have targets with similar binding sites: alignment of all pairs of binding sites for all promiscuous drugs using SMAP<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> without considering bound ligands, the removal of redundant targets, and the exclusion of sites with inconsistent ligand binding modes. For the third step the authors implemented LigandRMSD to find out matches where the ligands are in similar positions. For two conformers, the maximum common subgraph is determined using OpenBabel<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> or the small molecule subgraph detector<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> if no isomorphism was found by OpenBabel.<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> The rmsd was calculated with respect to the SMAP<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> binding site alignment and the optimal alignment of both ligand conformers. This method allowed for result analysis without tedious visual inspection of all aligned binding sites which becomes inappropriate for large data sets (after removal of redundant targets the data set consisted of 3948 alignments). Matches with a ligand rmsd above 3 Å were refused. The application of this analysis led to a final set of 1628 binding site matches that revealed a correlation between binding site or structural similarity and promiscuity. To become more detailed from a ligand-based perspective, it could be shown that 71% of the drugs have at least one target pair with a similar binding site and for 18% of the drugs all of their targets are similar. The top four promiscuous drugs in terms of the number of targets are those with the most similar binding site among their targets: benzimidamide (benzamidine),<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> staurosporine (<b>2</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>),<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (2<i>R</i>,3<i>R</i>,4<i>S</i>)-3-acetamido-4-hydroxy-2-((1<i>R</i>,2<i>R</i>)-1,2,3-trihydroxypropyl)-3,4-dihydro-2<i>H</i>-pyran-6-carboxylic acid (DANA),<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and (2<i>S</i>,5<i>S</i>)-2,5-diamino-6-((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)hexanoic acid (sinefungin).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Additionally, the influence of global structural similarity was assessed using TM-align.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> The investigations revealed that 55% of the protein pairs are structurally dissimilar and 15% of the pairs with similar binding sites are significantly dissimilar (TM-score below 0.5). Although local and global protein similarity correlated with ligand promiscuity, it was concluded that some off-target effects cannot be shown by conventional global sequence and structure alignment tools. This kind of similarity can exclusively be identified on the basis of efficient binding site comparison methods underlining their impact on repurposing approaches. The authors plead that their data set built from data as stored in the databases PDB and BindingDB<a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a> is limited in terms of chemical space coverage of drugs and protein structure space. A comparison of their results to those of previous promiscuity studies shows that the number of promiscuous drugs in their analysis is much smaller. Nevertheless, the inclusion of structural data in the analysis led to new observations. In conclusion, the degree of drug promiscuity (number of off-targets) weakly correlates with ligand flexibility, whereas structural similarity and, moreover, binding site similarity account for the promiscuity of known drugs. Altogether, 71% of the analyzed drugs possess at least one pair of similar target binding sites. Those results shed light on the impact of binding site comparison for the repurposing of known drugs.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of compounds analyzed in repurposing and polypharmacology studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The knowledge that the NSAIDs (nonsteroidal anti-inflammatory drugs) celecoxib (<b>3</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and valdecoxib (<b>4</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> possess an unsubstituted arylsulfonamide moiety provided the basis for the first approach presented here. As it is widely known that this group is present in a wide range of known carbonic anhydrase (CA) inhibitors, similarities between the target of both inhibitors COX-2 (cyclooxygenase) and CA were investigated.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> A nanomolar affinity of the COX-2 specific inhibitors <b>3</b> and <b>4</b> for CA family members I, II, IV, and IX could be shown with the help of enzyme kinetics and crystallographic studies. Intriguingly, 4-(4-(methylsulfonyl)phenyl)-3-phenylfuran-2(5<i>H</i>)-one (rofecoxib),<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> a COX-2 inhibitor harboring a methyl sulfone moiety, has no effect on those enzyme isoforms. Furthermore, compounds <b>3</b> and <b>4</b> show a much higher selectivity toward hCA II and hCA IX than toward hCA I and bovine CA IV as compared to known CA inhibitors. This finding was confirmed by results suggesting that the oral administration of <b>3</b> and <b>4</b> to glaucomatous rabbits leads to a lowered intraocular pressure. A crystal structure of CA II was solved in complex with <b>3</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1oq5">1oq5</a>) showing a coordinative binding to the catalytic zinc ion in the enzyme’s active site. A COX-2 complex structure with a related <i>p</i>-bromo derivative of <b>3</b> (4-(4-(4-bromophenyl)-3-(trifluoromethyl)-1<i>H</i>-pyrazol-1-yl)benzenesulfonamide (SC-558)<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a>) was also known (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6cox">6cox</a>) that allowed for a detailed comparison of the binding site geometries. The Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> approach was applied to represent the COX-2 sulfonamide anchor binding cavity by 25 pseudocenters according to the properties of the amino acids flanking the binding site. On the basis of this representation, a set of 9433 ligand-containing cavities was screened with Cavbase.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> After ranking of the similarities and removal of COX cavities, the first CA cavity was retrieved on rank 38 with more following on subsequent ranks. The subcavity bound to the trifluoromethyl substituent was also subjected to a Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> search. Again, 41 CAs were found within the 200 best ranked hits. The last screen was performed for the bromophenyl group accommodating subcavity of COX-2 and did not lead to an enrichment of CA pockets in the best ranked results. This was explained by different physicochemical properties of those cavities in both enzymes. However, the study allowed a matching of equivalent residues between COX-2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6cox">6cox</a>) and CA II (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bn4">1bn4</a>) with respect to the binding of compound <b>3</b>. It provided insights into the residues crucial for binding of this inhibitor type, thus facilitating the creation of structure-based pharmacophores. Finally, the analysis presents new indications for COX-2 selective NSAIDs with respect to glaucoma and anticancer therapy.</div><div class="NLM_p">The second example<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> represents an approach that uses repositioning to treat multidrug resistant (MDR) and extensively drug resistant (XDR) tuberculosis. A chemical systems biology approach was applied to reveal the off-targets of pharmaceuticals on a proteome-wide scale. It was predicted that the drugs entacapone (<b>5</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> and tolcapone (<b>6</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>),<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> originally prescribed for the treatment of Parkinson’s disease, might be useful for the treatment of MDR and XDR tuberculosis. The investigations of this effect led to the assumption that the drugs inhibit the enzyme InhA by directly binding to its substrate binding site. The known drugs isonicotinohydrazide (isoniazid)<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> and 2-ethylpyridine-4-carbothioamide (ethionamide)<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> are prone to resistance mechanisms, while others that require no enzymatic activation avoid the resistance mechanism of MDR <i>Mycobacterium tuberculosis</i> (<i>Mtb</i>). To validate the finding that <i>S</i>-adenosylmethionine (SAM)-dependent methyltransferases and NAD-binding enzymes share similar binding site properties, the binding site of catechol-<i>O</i>-methyltransferase (COMT) was compared to NAD-binding enzymes. This protein is involved in the breakdown of catecholamine-based neurotransmitters such as dopamine with impact on Parkinson’s disease. Known drugs are the COMT substrate (<i>S</i>)-2-amino-3-(3,4-dihydroxyphenyl)propanoic acid (levodopa),<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a><b>5</b>, and <b>6</b>. A novel polypharmacology prediction process was developed to assess the possibility that the last two drugs might bind to other interesting targets. The applied workflow includes the extraction of the binding site of a commercially available drug or its prediction based on a 3D structure or homology model of the target, the prediction of off-targets with similar ligand binding sites across the proteome, the evaluation of the interactions between the drugs and putative off-targets using protein ligand docking, and a final optimization step to increase the compound’s potency, selectivity, and ADME properties. Using the SOIPPA<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> algorithm, it was shown that the NAD binding site of the Rossmann fold and the SAM binding site of SAM methyltransferases share significant similarities. Compounds <b>5</b> and <b>6</b> were docked into 215 nonredundant NAD-binding proteins from the PDB to identify similar ligand binding sites in the cofactor binding site’s vicinity. The results of those studies suggested that enoyl–acyl carrier protein reductases (InhA) from several organisms show favorable binding affinities for those drugs. Especially, a protein identified from <i>Mtb</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4tzk">4tzk</a>) attracted the researchers’ attention. The enzyme is essential for type II fatty acid biosynthesis and the synthesis of the bacterial cell wall. Therefore, it represents a promising drug target. The next step was the search for known InhA inhibitors. The 2D molecule similarities between the 22 found known inhibitors and compounds <b>5</b> and <b>6</b> were calculated. This analysis revealed that the 2D similarities between <b>5</b> and known InhA inhibitors were statistically insignificant. As a crystal structure of human COMT, the original target of those drugs, was missing, a structure of brown rat COMT (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cl5">2cl5</a>) sharing 81% sequence similarity with the human protein was used for docking of existing and potential InhA inhibitors together with nine InhA structures. On the basis of the resulting binding poses, the authors discussed several similarities between the binding poses of <b>5</b> and <b>6</b> and those of known InhA inhibitors despite the low overall 2D similarity. Additionally, a binding pose analysis of putative InhA inhibitors and a calculation of log<i>P</i> and log<i>D</i> values of existing and putative antitubercular drugs were performed. Although both <b>5</b> and <b>6</b> had higher log<i>P</i> and log<i>D</i> values than existing antitubercular drugs, it was argued that their higher hydrophobicity allows for a more easy permeation of the <i>Mtb</i> cell envelope. A validation of this prediction by in vitro and InhA kinetic assays showed that <b>5</b> has a MIC<sub>99</sub> of about 260 μM when tested on <i>Mtb</i>. It inhibits InhA activity by 47% at a concentration of 80 μM. With respect to the results, the authors suggest the inclusion of their pipeline in drug discovery to find new drug leads with desired safety profiles.</div><div class="NLM_p">In another study<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> it was shown that the tyrosine kinase inhibitor pazopanib (<b>7</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> inhibits AChE activities at submicromolar concentrations with impact on the reversal of memory and cognitive deficits in rat model neurodegeneration. A combination of 2D virtual screening, ligand screening for 3D shape, and binding site similarity comparison led to the finding that <b>7</b> is able to restore memory loss and cognitive dysfunction to a similar extent as the marketed AChE inhibitor donepezil (<b>8</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> This not only is an impressive example of how computational tools can help to facilitate the process of repositioning but also shows the impact of previous studies using binding site comparison methods on drug design. The authors performed a docking study of 1385 FDA-approved small molecule drugs for the binding to AChE and used the resulting scores for compound ranking. The top hits resulting from this approach were three tyrosine kinase inhibitors (<b>7</b>, sorafenib (<b>9</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>),<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> and sunitinib (<b>10</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a>) aside from the known AChE inhibitor <b>8</b>. Owing to the availability of known compounds inhibiting AChE activity, a ligand-based approach could be applied. A 3D shape comparison between <b>8</b> and the 1385 FDA-approved drugs was performed using Tanimoto scores and electrostatic similarity. The final hits were reranked using a combination of shape, Tanimoto, and electrostatic similarity. Nearly one-third of the hits belong to the class of antipsychotic and antidepressant drugs. Compounds <b>7</b>, <b>9</b>, and <b>10</b> were also among the best scored hits. Additionally, pairwise binding site comparison of AChE (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1eve">1eve</a>) and 1105 PDB X-ray structures of 377 FDA-approved drug targets using TM-align<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> revealed that 14 out of the 20 top ranked identified targets belong to the protein kinase family (mainly tyrosine kinases). In conclusion, it can be stated that all three methods (docking, ligand-based, and structure-based screening) led to consistent results. Furthermore, the results were confirmed by an enzymatic assay. IC<sub>50</sub> values of 0.93 and 5.87 μM were determined for <b>7</b> and <b>10</b>, respectively. Altogether, 13 tyrosine kinase inhibitors were tested. Compound <b>9</b> and some others proved to be weak AChE binders. An MD simulation revealed that compounds <b>7</b> and <b>8</b> do not directly interact with the catalytic triad but bind via hydrophobic contacts with adjacent residues. Compound <b>7</b> was tested in vivo on cognition impairment in rat models, on spatial memory impairment, and in a test to measure the short-term working memory for model rats treated with different compounds to prove its in vitro efficacy. The results suggest that the compound restored memory loss and cognition impairment in rat models. It reduced learning and memory deficits in a dose-dependent manner. This study provides an outstanding example of the benefits of drug repositioning in terms of pharmacodynamics and pharmacokinetic properties.</div><div class="NLM_p">Another analysis<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> based on the SMAP<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> algorithm shows that repurposing of known kinase inhibitors is a promising tool to cope with alternative disease targets. Encouraged by a previous finding,<a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a> the binding site of β-secretase (BACE-1) as a potential drug target for the treatment of Alzheimer’s disease (AD) was compared against 24 well-characterized receptor tyrosine kinases using TM-align.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> EGFR was predicted to have the highest global structural similarity to BACE-1. SMAP<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> was used to compare both binding sites with each other, highlighting highly similar functional sites. Afterward, 13 FDA-approved drugs were screened against BACE-1. It could be shown that the EGFR inhibitor gefitinib (<b>11</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> had the strongest effect on the BACE-1 activity (IC<sub>50</sub> = 20 μM) while the remaining inhibitors showed IC<sub>50</sub> values above 100 μM. Cell viability assays with <b>11</b> had good outcomes. In vitro studies suggest that compound <b>11</b> has an impact on the metabolic amyloid precursor protein products, some of which are components of amyloid plaques in brains of AD patients. The binding of <b>11</b> was further analyzed via docking studies and MD simulations, providing a thorough study of this inhibitor and sufficient evidence for further testing.</div><div class="NLM_p last">The last example<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> shows the use of metabolic network construction and binding site comparison for the identification of promising targets and drug repurposing. An accurate metabolic reconstruction of the Gram-positive anaerobic bacterium <i>Clostridium difficile</i> led to the identification of enzymes essential for survival. The identified essential genes were compared to those of <i>Bacillus subtilis</i> homologues (experimental data existing). Comparative models were generated for 123 essential genes. The largest binding cavities were identified. Those binding sites were compared against a set of 7339 nonredundant small molecule binding sites using IsoCleft Finder.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> For each protein, the closest match was extracted and analyzed in more detail. For 29 predicted essential <i>C. difficile</i> targets 41 homologue proteins were found in the database DrugBank.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Altogether, 125 molecules were identified (22 of them approved) that might bind to one of the analyzed potential targets. Given that those molecules could also bind to the <i>C. difficile</i> homologue, the analysis provides starting points for further evaluation.</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Polypharmacology Prediction</h3><div class="NLM_p">The principle of polypharmacology relies on the idea that one drug is able to modulate multiple targets leading to desired effects. Therefore, this drug is more effective than drugs designed according to the one-drug–one-target paradigm. A single drug could fulfill tasks that were previously achieved by combination therapies that bear a higher risk of undesired adverse effects. But the question is the following: Can promiscuous drugs be rationally designed? The multitude of computational approaches useful to predict the activity profile of ligands to a set of targets is discussed in an article of Rastelli and Pinzi.<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(140)</a> The impact of binding site comparison approaches and protein–ligand interaction patterns on the design of molecules acting on more than one target is examined in a review by Salentin et al.<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> Protein binding site comparison can facilitate the design of new small molecule modulators of bioactivity as indicated by the following examples.</div><div class="NLM_p">The first example<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> deals with the HIV protease inhibitor nelfinavir (<b>12</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> which exhibits pleiotropic effects in cancer cells. By a SMAP<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> binding site comparison of the compound’s binding pocket in the HIV protease dimer (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ohr">1ohr</a>) against 5985 PDB structures of human proteins or their homologues, 126 similar structures were found. Starting from the superimposed binding sites, compound <b>12</b> was docked into all identified similar binding sites. After removal of proteins for which the docked ligand clashes, 92 potential off-targets were left to be investigated. The seven top ranked proteins belong to the aspartyl protease family, whereas 51 are protein kinases and 17 are nucleotide binding proteins. Of all identified protein kinases, mainly belonging to tyrosine kinase, cAMP-, cGMP-dependent kinase, or protein kinase C family, the 12 best ranked were subjected to exhaustive docking studies. Further investigations using MD simulations and MM/GBSA free energy calculations led to the conclusion that <b>12</b> directly interacts with EGFR, while it is unlikely to bind to FGFR (fibroblast growth factor receptor), EphB4, and Abl. For six other kinases (IGF-1R (insulin like growth factor 1 receptor), FAK (focal adhesion kinase), Akt2, CDK2, ARK (β adrenergic receptor kinase), PDK1 (phosphoinositide-dependent kinase 1)) it was concluded that a binding might be possible. A high-throughput screening with a compound concentration of 20 μM was performed for EGFR, ErbB2, ErbB4, and all Akt isoforms. A low inhibition was detected for ErbB2, whereas EGFR inhibition was more pronounced. The authors argue that the anticancer activity is not due to aggregation of <b>12</b>. Compared to most cellular studies that required higher inhibitor concentrations, the compound exhibited specific anticancer activity without nonspecific binding. Furthermore, they refer to an article by Gills et al.<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> who showed that 20 μM <b>12</b> reduced the activation of EGFR, IGF-1R, and Akt signaling pathways. It was concluded that the compound might inhibit other protein kinases through weaker interactions. Finally, the computational findings were discussed in the context of biological network and signal transduction pathway analysis. All these facts underline the applicability of compound <b>12</b> as potent anticancer agent in terms of polypharmacology. Today, data that were provided by the National Toxology Program substantiate this finding and can be found in the ChEMBL database.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> An IC<sub>50</sub> of approximately 56 μM was measured for EGFR inhibition by compound <b>12</b> (kinase substrate: Poly(Glu-Tyr)).</div><div class="NLM_p last">The method PocketMatch<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> proved to be useful to explain the reasons for polypharmacology observed for the competitive antagonist of serotonin type 2B/2C metabotropic receptors (5-HT<sub>2B/2C</sub>R) SB-206553 (<b>13</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a><a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> This drug also acts as a positive allosteric modulator of the ionotropic α7 nAcChR with an EC<sub>50</sub> of 1.5 μM for potentiation of calcium responses to EC<sub>20</sub> nicotine.<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a> Homology models of the extracellular and transmembrane domain of α7 nAcChR and 5-HT<sub>2C</sub>R were generated. Explicit MD simulations of the protein structures in hydrated palmitoyloleylphosphatidylcholine bilayer membranes were performed for all final models to evaluate the model quality. Blind docking of compound <b>13</b> into α7 nAcChR, 5-HT<sub>2C</sub>R, and 5-HT<sub>2B</sub>R without the definition of a binding site and further MD simulations of the obtained complexes led to the identification of putative binding sites in all three receptor proteins. This enabled the authors to perform an exhaustive binding site comparison using PocketMatch with minor modifications. The binding site was represented as a sorted list of distances between amino acids at a distance range of 3.5–8 Å from the docked ligand. Each residue was classified according to its physicochemical properties. The binding sites were aligned using a threshold of 0.5 Å to account for the dynamics of macromolecular systems. The binding sites of 5-HT<sub>2B</sub>R and 5-HT<sub>2C</sub>R were highly similar, as expected given their homology. This explains the similar affinities of <b>13</b> for both proteins.<a onclick="showRef(event, 'ref144'); return false;" href="javascript:void(0);" class="ref ref144">(144)</a> The comparison of the α7 nAcChR extracellular domain binding site with those of both serotonin receptors revealed that the binding sites are similar with respect to size and chemical nature of the pocket-lining residues. No significant similarity to the serotonin receptor binding sites was found for the putative binding site of compound <b>13</b> in the transmembrane domain of nAcChR. Altogether, the study revealed the reasons for the polypharmacological properties of the investigated drug and facilitates the structure-based design of novel drugs acting at both receptor types.</div></div><div id="sec6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Analysis of Protein–Ligand Interactions</h3><div class="NLM_p">In the past years, the generation and prediction of protein–ligand interaction networks became a promising tool for drug design. Various approaches deal with the construction and mapping of various drug-oriented interaction networks. A comprehensive review<a onclick="showRef(event, 'ref145'); return false;" href="javascript:void(0);" class="ref ref145">(145)</a> introduces the structural bioinformatics of the interactome and their application in drug development. In this section, the impact of binding site comparison on this field is discussed.</div><div class="NLM_p">Yang et al.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> constructed an in silico drug–target network for 10 AD drugs, 47 randomly chosen molecules from DrugBank,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and a set of 401 human protein pockets. Their docking results suggested that drugs targeting AChE (the only validated AD target) might also act on HDACs and members of the estrogen receptor family. The pocket comparison algorithm developed by Schmitt et al.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> revealed some promising similarities between HDAC7 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3c0y">3c0y</a>) and AChE (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f8u">1f8u</a>) active sites, indicating a similar nature of contacts to their respective ligands. Inverse docking and additional docking studies were performed to support those findings. Intriguingly, the authors state that the relationship between both enzymes is not determined by pocket shape or sequence similarities but by the interactions with AD drugs and HDAC inhibitors.</div><div class="NLM_p">Cross-reactivity with various ATP-binding targets could also be shown using SiteAlign<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> for the similarity measurement between ligand-annotated binding sites.<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> The authors extracted all druggable protein–ligand binding sites from the database sc-PDB<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> and screened them for their similarity to the ATP-binding site of Pim-1 kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yhs">1yhs</a>). Protein kinases were removed from the data set to exclude obvious similarities. Proteins which were present in a single copy were also eliminated. SiteAlign<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> was applied to the remaining complexes. The synapsin I (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aux">1aux</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1px2">1px2</a>) binding site showed the highest similarity to Pim-1 kinase. A detailed analysis revealed a relationship between binding site-lining residues (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). The docked pan-kinase inhibitor <b>2</b> showed promising binding modes for synapsin I. In fact, compound <b>2</b> binds to synapsin I, inhibits ATP-binding, and even affects the synapsin I-dependent F-actin bundling. Further analysis indicated similarities of synapsin I to the kinases CDK2 and CKII (casein kinase II), whereas other ATP-binding proteins were classified as being dissimilar (e.g., PKA, HSP90α, CHK1, DNA topoisomerase II, diacylglycerol kinase). To confirm those in silico results, eight high affinity inhibitors for the eight representative targets were purchased and tested for in vitro inhibition of synapsin I. The Pim-1 kinase inhibitor 2-(3,4-dihydroxyphenyl)-3,5,6,7-tetrahydroxy-4<i>H</i>-chromen-4-one (quercetagetin)<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> proved to be the most potent effector (IC<sub>50</sub> = 0.15 μM). Roscovitine (<b>14</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>)<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> and 2-(4,5,6,7-tetraiodo-1,3-dioxoisoindolin-2-yl)acetic acid (70159800251),<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> both inhibitors of CDK2 and CKII, also acted ATP-competitive but with higher IC<sub>50</sub> values for synapsin I (1.0 and 0.5 μM, respectively). Other protein kinase inhibitor activities were negligible. Altogether, this binding site comparison approach significantly contributed to the extension of the protein kinase inhibitor interactome.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Alignment of synapsin I to the protein kinase Pim-1 with respect to the ATP-binding site. The structure of Pim-1 is shown in forest green (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a99">3a99</a>) with bound ATP analog represented as sticks. Synapsin I (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aux">1aux</a>) is colored orange. (a) Alignment of both proteins which differ in fold. (b) Detailed view of the aligned ATP-binding sites and the cavity-lining residues. Parts a and b were generated using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The algorithm PocketFEATURE<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> was applied to explain the mechanism of action of the antimicrobial allosteric inhibitor PC190723 (<b>15</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).<a onclick="showRef(event, 'ref74 ref146'); return false;" href="javascript:void(0);" class="ref ref74 ref146">(74, 146)</a> The compound inhibits <i>Staphylococcus aureus</i> GTPase FtsZ activity in vitro and showed in vivo antimicrobial activities for Gram-positive bacteria.<a onclick="showRef(event, 'ref146'); return false;" href="javascript:void(0);" class="ref ref146">(146)</a> Its mechanism on the different bacterial species is still unknown. The use of PocketFEATURE<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> and MD simulations provided insights into the inhibitor’s properties. Initially, proteins from different species were aligned and pocket residues identified. In a second step, microenvironments around the functional centers of a residue were defined and physicochemical properties calculated. Afterward, a modified Tanimoto coefficient for each pocket pair was calculated. Obtained scores were evaluated against a set of equivalent microenvironment pairs from 1116 nonredundant PDB structures. An optimum binding pocket of <b>15</b> was defined as the microenvironment of 20 residues within a 6 Å radius of the bound compound in a cocrystal structure of the <i>S. aureus</i> enzyme (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dxd">4dxd</a>). After MD simulations were performed with the FtsZ structures of <i>S. aureus</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vo8">3vo8</a>), <i>Staphylococcus epidermidis</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4m8i">4m8i</a>), and <i>B. subtilis</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2rhl">2rhl</a>), amino acid pocket scores were calculated for the equilibrated structures. It could be shown that the conformation of the FtsZ binding pocket is strongly influenced by the species, allosteric binding, genetic perturbations, and polymerization state. Especially, oligomerization and allosteric binding of GTP seem to stabilize the pocket. Although the proteins from different organisms show a high overall structural similarity, the binding site differences lead to different affinities and experimental outcomes. Varying responses of <i>Sa</i>FtsZ and <i>Bs</i>FtsZ could be explained by different effects of bound nucleotides on binding site architecture. One study suggested that the enzymatic activity of <i>Sa</i>FtsZ is increased by <b>15</b>, while no such behavior could be observed for <i>Bs</i>FtsZ.<a onclick="showRef(event, 'ref147'); return false;" href="javascript:void(0);" class="ref ref147">(147)</a> In contrast, the compound’s antimicrobial properties in <i>S. aureus</i> and <i>B. subtilis</i> agreed with the in vitro results of another study (drug acts through excess polymer stabilization).<a onclick="showRef(event, 'ref148'); return false;" href="javascript:void(0);" class="ref ref148">(148)</a> The computational approach revealed that <b>15</b> might bind to another site of the <i>B. subtilis</i> enzyme or acts through another mechanism on <i>Bs</i>FtsZ as its binding site architecture is different from known binding pockets of <b>15</b> (see also <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>). Resistance variants were analyzed likewise. The similarity score to the “optimal pocket” considerably decreased after introduction of mutations. This study underlines the impact of binding site comparison and similarity measurements binding affinity evaluation. MIC values for the compound from the ChEMBL database<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> for different organisms support these findings.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Comparison of FtsZ from <i>S. aureus</i>, <i>S. epidermidis</i>, and <i>B. subtilis</i> with respect to the <i>Sa</i>FtsZ binding site of compound <b>15</b>: (a) structure of <b>15</b>; (b) overall alignment of <i>Sa</i>FtsZ with bound <b>15</b> in ball-and-sticks representation in green (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dxd">4dxd</a>), <i>Se</i>FtsZ in purple (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4m8i">4m8i</a>), and <i>Bs</i>FtsZ in orange (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2rhl">2rhl</a>); (c) binding site alignment of all three proteins. Their structures share highly similar binding site residues (represented as sticks, names according to crystal structures). Nevertheless, they differ with respect to their binding site geometry. Parts b and c were created using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Analysis of Protein–Protein Interactions</h3><div class="NLM_p">Host–pathogen interactions, signal cascades, and regulation mechanisms of all organisms highly rely on PPIs. This led to a huge interest in understanding the nature of those interactions and the emergence of various methods for their computational analysis. Two reviews<a onclick="showRef(event, 'ref149 ref150'); return false;" href="javascript:void(0);" class="ref ref149 ref150">(149, 150)</a> give an overview of structural bioinformatics of PPIs as well as methods currently applied to predict protein–protein interfaces.</div><div class="NLM_p">In the following example<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> the binding site comparison algorithm SiteEngine<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> was used to scan molecular surfaces of all PDB protein chains with a template ubiquitin-binding domain (UBD) to screen for new potential UBDs. Ubiquitylation signals play a significant role in trafficking of endogenous and retroviral transmembrane proteins. It was shown that the blocking of distinct UBDs in vivo can influence retroviral budding. The workflow leading to those results included (i) choice of a template UBD (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k9p">3k9p</a>), (ii) binding site comparison, (iii) superimposition of the template UBD on the candidate-binding patch and search for energetically favorable binding patches in the immediate vicinity. The authors identified the domain ALIX-V (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ojq">2ojq</a>) as potential new UBD and confirmed the in silico findings by biophysical affinity measurements. Later, another study<a onclick="showRef(event, 'ref151'); return false;" href="javascript:void(0);" class="ref ref151">(151)</a> proved this hypothesis. Additionally, it could be shown that the yeast Alix homologue Bro1 functions as a ubiquitin receptor for protein sorting into multivesicular endosomes.<a onclick="showRef(event, 'ref152'); return false;" href="javascript:void(0);" class="ref ref152">(152)</a></div><div class="NLM_p last">The binding site comparison method CPASS<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> enabled the elucidation of a structural and functional similarity between the type III secretion system needle protein PrgI from <i>Salmonella typhimurium</i> and the eukaryotic apoptosis Bcl-2 proteins.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The authors combined NMR ligand affinity screening using a fragment-based functional library and bioinformatics methods to reveal relationships between otherwise unrelated proteins. Chemical shift perturbations in the 2D <sup>15</sup>N–<sup>1</sup>H HSQC experiments between free PrgI and the complex of PrgI and its previously identified ligand <i>N</i>-decyl-<i>N</i>,<i>N</i>-dimethyldecan-1-aminium bromide (DDAB)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> helped to identify residues constituting the binding site. Docking and active site comparison revealed that PrgI is similar to the antiapoptosis regulating protein Bcl-xL (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ysn">1ysn</a>) complexed to (<i>R</i>)-4-(4-((4′-chloro-[1,1′-biphenyl]-2-yl)methyl)piperazin-1-yl)-<i>N</i>-((4-((4-(dimethylamino)-1-(phenylthio)butan-2-yl)amino)-3-nitrophenyl)sulfonyl)benzamide (ABT-737),<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> an acylsulfonamide-based inhibitor. Although both ligands are quite dissimilar from a purely chemical point of view, they show similarities in their target interactions. This finding led to the prediction of 1,2-dimethoxy-12-methyl[1,3]dioxolo[4′,5′:4,5]benzo[1,2-<i>c</i>]phenanthridin-12-ium (chelerythrine)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> as potential PrgI binder. Docking of this compound into a binding site identified by NMR studies of PrgI suggested that both proteins bind the ligand at similar binding sites and share a locally limited structural similarity. Interestingly, both binding sites are essential for PPIs, pointing to their impact on protein oligomerization.</div></div><div id="sec6_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Off-Target Prediction</h3><div class="NLM_p">Given the huge space of protein binding sites, the prediction of potential off-targets is a crucial step for target validation as well as the choice of a suitable lead. Usually, demanding experimental studies are required during lead optimization. Although prominent off-targets are already known for many compounds, the identification of distant drug off-targets<a onclick="showRef(event, 'ref153'); return false;" href="javascript:void(0);" class="ref ref153">(153)</a> is a valuable tool for compound optimization.</div><div class="NLM_p">Paolini et al.<a onclick="showRef(event, 'ref154'); return false;" href="javascript:void(0);" class="ref ref154">(154)</a> analyzed the pharmacological relationships of an assembly of annotated pharmacological data. The data set contained 4.8 million nonredundant chemical structures. About 275 000 biologically active compounds, more than 600 000 SARs of molecular binding from Pfizer’s internal screening files, commercial screening data, competitive intelligence on approved and investigational drugs, and key components of 25 years of published medicinal chemistry data were analyzed. The binding affinity threshold was set to 10 μM, and compounds violating at least one of Lipinski’s rule-of-five criteria were excluded from further analysis. At least one binding molecule is known for 727 targets, whereas 529 of them have at least one compound with a binding affinity below 100 nM. After the exclusion of nonspecific aggregation inhibitors, 276 122 compounds were analyzed with regard to known targets. This study revealed that 65% of them have recorded activity for a single target, whereas 35% hit more than one target. The results show that the activity of a compound on various targets not only can be exploited for designing more efficient drugs but also leads to severe side effects. Importantly, the various possible combinations between 276 122 compounds and at least 727 human targets leave room for various computational big data approaches to address this challenge. An analysis of PubChem<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> data revealed that 57.7% of the compounds with confirmatory bioassays showed single target activity and that the number of compounds drops exponentially when the number of targets increases.<a onclick="showRef(event, 'ref155'); return false;" href="javascript:void(0);" class="ref ref155">(155)</a> Nevertheless, we have to admit that this kind of analysis is speculative as long as the drug–target matrix is not completely filled. Another study attempted to fill this matrix with the help of docking on high-performance computing machines supporting the idea that the available drug on-target interaction data are not sufficient to statistically assess the off-target space.<a onclick="showRef(event, 'ref156'); return false;" href="javascript:void(0);" class="ref ref156">(156)</a> The following paragraph is dedicated to examples where binding site comparison methods assisted in off-target prediction.</div><div class="NLM_p">Coping with cross-reactivity of protein kinase inhibitors is a well-known challenge in medicinal chemistry. Kinnings and Jackson<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> tried to shed light on the question of how binding site comparison can help to identify potential cross-reactivity of certain well-known kinase inhibitors. They compared the ATP-binding sites of 354 structures using a geometric hashing algorithm.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Protein kinases were extracted from the PDB on the basis of their PFAM<a onclick="showRef(event, 'ref157'); return false;" href="javascript:void(0);" class="ref ref157">(157)</a> families Pkinase (PF00069) and protein tyrosine kinase (PF07714). All protein chains were superimposed onto a cAMP-dependent protein kinase structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1atp">1atp</a>) as a typical protein kinase in terms of backbone architecture. Although the algorithm takes no ligand atoms into consideration, all ATP-binding sites and their respective ligands were successfully aligned. Afterward, a “compound ligand” composed of all the ligand atoms of the aligned structures was created. Atoms were removed if less than 50 atoms of other ligands were within 1.5 Å to exclude less populated regions. The binding site was defined by choosing all atoms in a 5 Å radius of the “compound ligand”. The resulting binding sites were subsequently compared to each other using geometric hashing. Similarity scores were calculated for atom–atom correspondences (same element and relative spatial orientation). Additionally, the sizes of the two binding sites were taken into account. Binding sites were clustered according to the final similarity score. This structure-based comparison was evaluated against a sequence-based similarity analysis. It could be shown that various structural similarities exist between protein kinases sharing low sequence identity. Additionally, the authors provide evidence that several differences in the binding site conformations exist, although the respective kinases show high sequence similarity. Therefore, one can conclude that binding site comparison is strongly dependent on the state of the respective kinases. For three of nine kinase inhibitors (p38α inhibitors (4-(4-(4-fluorophenyl)-2-(4-(methylsulfinyl)phenyl)-1<i>H</i>-imidazol-5-yl)pyridine (SB-203580)<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> and 1-(3-(<i>tert</i>-butyl)-1-(<i>p</i>-tolyl)-1<i>H</i>-pyrazol-5-yl)-3-(4-(2-morpholinoethoxy)naphthalen-1-yl)urea (doramapimod),<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> CDK5 inhibitor <b>14</b><a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a>) the in silico findings in terms of enrichment factors were in line with an in vitro screening while high enrichment factors were also observed for the remaining compounds. The same comparison for another data set led to similar results. Nevertheless, the achieved and highest achievable enrichment factors for imatinib (<b>16</b>, <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>),<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a><i>N</i>-(3-chloro-4-((3-fluorobenzyl)oxy)phenyl)-6-(5-(((2-(methylsulfonyl)ethyl)amino)methyl)furan-2-yl)quinazolin-4-amine (lapatinib),<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a><b>9</b>, (<i>R</i>)-4-(1-aminoethyl)-<i>N</i>-(pyridin-4-yl)cyclohexane-1-carboxamide (Y27632)<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> deviate to a huge extent. Additionally, the binding of EGFR inhibitor <i>N</i>-(3-ethynylphenyl)-6,7-bis(2-methoxyethoxy)quinazolin-4-amine (erlotinib)<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> to c-Src, Lck, and c-Abl could be rationalized. A consensus scoring procedure was performed for c-Abl inhibitor <b>16</b> to explain its inhibitory profile (calculation of the mean similarity score of the c-Abl binding site to all kinase binding sites). The results were reranked accordingly. This procedure is described in more detail elsewhere.<a onclick="showRef(event, 'ref158'); return false;" href="javascript:void(0);" class="ref ref158">(158)</a> The most similar binding site was that of c-Src, although this protein kinase is not inhibited by compound <b>16</b> because it does not adopt the DFG-out conformation (inactive state) required for binding. Lck, c-Kit, and Hck show also significant similarities to c-Abl. Inhibition constants of <b>16</b> are in the nanomolar range for those three proteins. The similarity between Lck and c-Abl is a hint to a possible immunosuppressant effect of compound <b>16</b>. Altogether, it can be concluded that this case study of binding site comparison for kinases led to reasonable results. Application of the method of Brakoulias and Jackson<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> could facilitate the search for potential protein kinase inhibitor cross-reactivity.</div><div class="NLM_p">The next example deals with the off-target prediction for cannabinoid receptor 1 (CB1R) antagonists used for the treatment of obesity.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> The first antagonist 5-(4-chlorophenyl)-1-(2,4-dichlorophenyl)-4-methyl-<i>N</i>-(piperidin-1-yl)-1<i>H</i>-pyrazole-3-carboxamide (rimonabant)<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> was withdrawn due to psychiatric side effects. This led to the development of the 3,4-diarylpyrazoline derivative ibipinabant (<b>17</b>, <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Muscle toxicity of this drug was observed in a preclinical dog study. Furthermore, it caused mitochondrial dysfunction as measured by cellular generation of reactive oxygen species and mitochondrial ATP production. It was analyzed whether compound <b>17</b> inhibits one of the respiratory chain enzymes (complexes I–V) by measuring the oxygen consumption in C2C12 myoblasts after preincubation with the drug. In comparison to control cells, no differences were observed. It was concluded that drug exposure leads to decreased ADP availability in the mitochondrial matrix as suggested by measurement of the mitochondrial membrane potential. Supported by those results, an in silico study followed using the pharmacophore-based binding site comparison method KRIPO.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> A homology model of CB1R was built. From the list of similar targets with potential impact on mitochondrial ADP/ATP exchange, ANT1 (adenine nucleotide translocase 1) and VDAC (voltage-dependent anion channel) were manually selected. A successive docking approach with ANT1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2c3e">2c3e</a>) suggested that binding of <b>17</b> to ANT1 is likely to occur. A comparable observation for a binding to VDAC was not possible due to its large pore which allows the transport of various small molecules. Result verification was performed by measuring the ADP uptake into isolated bovine heart mitochondria. The application of compound <b>17</b> led to a decrease of mitochondrial VDAC-dependent and ANT-dependent ADP import. C2C12 mitoplasts were isolated to elucidate the inhibitor’s target. Maximal complex I- and complex II-specific respiratory rates depending on the presence of the drug were measured. The results suggest a reduced respiration by the inhibition of the ANT-mediated ADP/ATP exchange. Additionally, a negligible effect of a closely related 3,4-diarylpyrazoline derivative was observed. Together, those experimental results support the computationally obtained results.</div><div class="NLM_p">Another example of successful off-target prediction was the finding that peroxisome proliferator-activated receptor α (PPARα) and COX enzymes share similarities leading to the binding of similar ligands.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> A combination of different computational methods was applied to a known PPARα ligand. Data from chemical structures of ligands of the identified new target, the textual patient package insert for the query ligand, drugs that modulate the new target, and crystallographic structures were analyzed. The applied methods produced a set of scores (ligand structural similarity, package insert similarity) or a single score (docking and protein pocket similarity) for known drugs and the final score is calculated from all scores including <i>p</i>-values. First, 602 drugs and 91 diverse protein targets were chosen to investigate drug–target relationships by deriving a drug–target matrix. The uncharacterized potential drug–target interactions were scored using a combination of 3D ligand comparison and package insert similarity. The top ranked results had HIV reverse transcriptase as predicted target and viral or human polymerases as the intended targets. Among the analyzed compounds are examples with known HIV reverse transcriptase activities: 2-amino-9-((1<i>S</i>,3<i>R</i>,4<i>S</i>)-4-hydroxy-3-(hydroxymethyl)-2-methylenecyclopentyl)-3,9-dihydro-6<i>H</i>-purin-6-one (entecavir),<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> 1-((2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-3,4-dihydroxy-5-(hydroxymethyl)tetrahydrofuran-2-yl)-1<i>H</i>-1,2,4-triazole-3-carboxamide (ribavirin),<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> 4-amino-1-((2<i>R</i>,4<i>R</i>,5<i>R</i>)-3,3-difluoro-4-hydroxy-5-(hydroxymethyl)tetrahydrofuran-2-yl)pyrimidin-2(1<i>H</i>)-one (gemcitabine),<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> 2-((2-amino-6-oxo-3,6-dihydro-9<i>H</i>-purin-9-yl)methoxy)ethyl <span class="smallcaps smallerCapital">l</span>-valinate (valacyclovir).<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> The best scored examples suggested a relationship between PPARα, COX-1, and COX-2 with regard to the known drug gemfibrozil (<b>18</b>, <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>).<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> Additionally, the free acids of two other fibrates (clofibric acid (<b>19</b>, <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>),<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> fenofibric acid (<b>20</b>, <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>)<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a>) were included in successive calculations. An alignment of the PPARα agonist 2-(1-(4-chlorobenzoyl)-5-methoxy-2-methyl-1<i>H</i>-indol-3-yl)acetic acid (indomethacin)<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> and compound <b>20</b> showed a good overlay in terms of chemical features such as surface or electrostatic similarity. The PPARα agonists <b>18</b>, <b>19</b>, and <b>20</b> were predicted as potential COX ligands. COX protein structures (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2oyu">2oyu</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kk6">3kk6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3n8x">3n8x</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3n8z">3n8z</a> for COX-1, PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pxx">1pxx</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ln1">3ln1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nt1">3nt1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3rr3">3rr3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4cox">4cox</a> for COX-2) were used for a docking of the three ligands. After binding site alignment, docking into multiple structures was performed. The docking results were similar for COX-1 and COX-2. Compound <b>18</b> yielded the most significant scores. The created alignment between the new potential ligands and the known NSAIDs supported the hypothesis that PPARα agonists are COX ligands. PSIM<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> was applied to compare the binding sites of both proteins which share no significant sequence similarity and belong to different CATH<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> and SCOP<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> families. The overall structural comparison using TM-align<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> yielded a TM-score below 0.4. All human PPARα structures (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i7g">1i7g</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1k7l">1k7l</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kkq">1kkq</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2npa">2npa</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2p54">2p54</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2rew">2rew</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2znn">2znn</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3et1">3et1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3fei">3fei</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3g8i">3g8i</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kdt">3kdt</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kdu">3kdu</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3sp6">3sp6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vi8">3vi8</a>) were used for the comparison to nine COX structures. It could be shown that some hydrogen bond and hydrophilic features of the binding sites nicely overlap. COX enzyme assays were performed to validate the in silico results for compounds <b>18</b>, <b>19</b>, and <b>20</b>. The presence of 250 μM <b>18</b>, <b>19</b>, and <b>20</b> led to 18%, 14%, and 48% inhibition of COX-1, respectively. Compound <b>20</b> showed dose-dependent inhibition when applied in different concentrations. The analysis of the effect of <b>18</b> and <b>19</b> on COX-1 and COX-2 led to inconclusive results. Therefore, only IC<sub>50</sub> values for COX-1 and COX-2 inhibition by <b>20</b> were determined. The IC<sub>50</sub> value for COX-1 was 950 μM being at least 2-fold weaker for COX-2. The known NSAIDs <i>N</i>-(4-hydroxyphenyl)acetamide (acetaminophen)<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> and 2-hydroxybenzoic acid (salicylic acid)<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> have IC<sub>50</sub> values of about 200 and 500 μM, respectively, in a microsomal assay for COX-1. In this example, the exploitation of existing medicinal chemistry data led to the prediction of already known and unexpected cross-reactivity.</div><div class="NLM_p">In another study<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> the validity of an interesting target with regard to potential off-targets was assessed using ProBiS.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> The antiviral compound 2-([1,1′-biphenyl]-4-yl)quinoline-4-carboxylic acid (RK424)<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> was identified by a library screening in a cell-based assay. It shows inhibition for many subtypes of influenza A virus in vitro and was responsible for a partial protection of mice against a lethal dose of A/WSN/1933 (H1N1) virus in vivo. The compound inhibits the viral ribonucleoprotein (NP) complex by binding to a small pocket of the viral NP leading to its accumulation in the cell nucleus. Additionally, the molecule disrupts NP–RNA and NP–NP interactions and inhibits oligomerization. Binding site similarity analyses were applied to two ligand binding sites: one of the identified novel target NP (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2iqh">2iqh</a>) and one of a potential novel antiviral target, the polymerase protein PA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e5e">4e5e</a>). The pockets of both proteins were excised and used to search for similar binding sites in the ProBiS<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> protein structure library. As a result, the NP pocket was shown to have a unique surface structure whereas for PA 20 similar pocket structures were identified, some of them involved in host cell function. Therefore, the authors reasoned that it is more beneficial to target the NP pocket to avoid potential off-target effects. This analysis shows how binding site comparison might help to find a suitable target with low risk of designing compounds with potential side effects.</div><div class="NLM_p">The same was achieved in another study<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> dealing with the methylmalonyl CoA mutase-associated GTPase MeaB, an enzyme essential for the growth of many pathogenic bacteria such as <i>Mtb</i>. The <i>Mtb</i> protein Rv1496 and its homologues in <i>Mycobacterium smegmatis</i> and <i>Mycobacterium thermoresistibile</i> (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3md0">3md0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nxs">3nxs</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3tk1">3tk1</a>) were shown to be MeaB enzymes. The crystal structures were solved and compared to similar enzymes from other organisms using the method of Totrov et al.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The comparison of their nucleotide binding sites revealed that Rv1496 is more similar to its mycobacterial orthologous proteins and more distinct from the human homologous protein MMAA (methylmalonic aciduria associated protein A, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2www">2www</a>). <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a> illustrates the main differences between the four binding sites. Mutations in the MeaB homologue MMAA protein cause fatal methylmalonic aciduria. Therefore, research toward the design of potential MeaB inhibitors should always consider their potential activity on the human homologue of MeaB.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Comparison between three mycobacterial MeaB enzymes and human MMAA: (a) overall alignment of the different enzymes’ structures (forest green, <i>Mtb</i> MeaB with bound GDP molecule in sticks representation, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3md0">3md0</a>; orange, <i>M. smegmatis</i> MeaB, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nxs">3nxs</a>; purple, <i>M. thermoresistibile</i> MeaB, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3tk1">3tk1</a>; magenta, human MMAA protein, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2www">2www</a>); (b) detailed view of the major differences between the mycobacterial proteins and the human MMAA protein in the nucleotide binding site. Respective residues are represented as sticks and named according to the crystal structures. This figure was created using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The last example does not provide a successful prediction of side effects by binding site comparison methods but points toward their impact on the design of molecules with reduced side effects. Xie et al.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> used SOIPPA<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> to analyze the cross-reactivity of selective estrogen receptor modulators (SERMs), for example, 1-(4-(2-(azepan-1-yl)ethoxy)benzyl)-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-5-ol (bazedoxifene),<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> (5<i>R</i>,6<i>S</i>)-6-phenyl-5-(4-(2-(pyrrolidin-1-yl)ethoxy)phenyl)-5,6,7,8-tetrahydronaphthalen-2-ol (lasofoxifene),<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> 1-(2-(4-((3<i>S</i>,4<i>S</i>)-7-methoxy-2,2-dimethyl-3-phenylchroman-4-yl)phenoxy)ethyl)pyrrolidine (ormeloxifene),<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> (6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophen-3-yl)(4-(2-(piperidin-1-yl)ethoxy)phenyl)methanone (raloxifene),<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> (<i>Z</i>)-4-(1-(4-(2-(dimethylamino)ethoxy)phenyl)-2-phenylbut-1-en-1-yl)phenol (4-hydroxytamoxifen),<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> tamoxifen (<b>21</b>, <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>).<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> Compound <b>21</b> is a commercial drug targeting estrogen receptor α (ERα) to treat breast cancer. It has known side effects like cardiac abnormalities, thromboembolic disorders, and ocular toxicity. The SERM binding site in ERα (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1xpc">1xpc</a>) was used to search for similar ligand binding sites in a representative set of 825 structures using SOIPPA.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> The most significant similarity was found for a sarcoplasmic reticulum Ca<sup>2+</sup> ion channel ATPase (SERCA, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2zbd">2zbd</a>) from <i>Oryctolagus cuniculus</i> sharing 96% sequence similarity with its human homologue. SERCA regulates cytosolic calcium levels by accumulating calcium in the lumen. A complex structure of the enzyme bound to two known inhibitors (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2agv">2agv</a>) gives information about its binding site. The authors conclude that SERMs bind to a site similar to the predicted site as 30% of the residues of the known and predicted binding site overlap. Known binding sites of SERCA were scanned against a database of the druggable proteome to validate this assumption. Indeed, ERα structures were found within the top ranked hits. Additionally, the comparison of electrostatic potentials of both binding sites revealed that they share similar negative potentials. For the reasons outlined above, it was suggested that SERMs bind in a similar fashion as known inhibitors preventing the binding of two calcium ions to the ATPase. Subsequent docking studies led to the prediction of reasonable binding poses. Finally, experimental evidence supporting the in silico findings is provided: pretreatment with <b>21</b> inhibits the effect of the known SERCA inhibitor (3<i>S</i>,3a<i>R</i>,4<i>S</i>,6<i>S</i>,6a<i>R</i>,7<i>S</i>,8<i>S</i>,9b<i>S</i>)-6-acetoxy-4-(butyryloxy)-3,3a-dihydroxy-3,6,9-trimethyl-8-(((<i>Z</i>)-2-methylbut-2-enoyl)oxy)-2-oxo-2,3,3a,4,5,6,6a,7,8,9b-decahydroazuleno[4,5-<i>b</i>]furan-7-yl octanoate (thapsigargin)<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> of increasing intracellular calcium ion concentrations. Additionally, compound <b>21</b> significantly reduces intracellular calcium ion concentrations and the release of platelets which is correlated with platelet adhesion and aggregation. The authors suggest that an inhibition of SERCA could lead to cardiac abnormalities due to impaired muscle contraction. Altogether, the predicted binding of <b>21</b> to SERCA might represent one reason for the severe side effects. This prediction was supported by Liu et al.<a onclick="showRef(event, 'ref159'); return false;" href="javascript:void(0);" class="ref ref159">(159)</a> Additionally, Beca et al.<a onclick="showRef(event, 'ref160'); return false;" href="javascript:void(0);" class="ref ref160">(160)</a> could observe an effect of <b>21</b> on SERCA2a ATPase activity; they provide additional evidence that the compound inhibits a cardiac sarcoplasmic reticulum chloride channel rather than SERCA.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of compounds analyzed in off-target prediction studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Prediction of Compound Selectivity Profiles and Affinities</h3><div class="NLM_p">The ranking of ligands according to their affinity toward a certain target or even affinity prediction using different computational methods plays a significant role in different phases of rational drug design. One important application of such methods is the scoring of generated docking poses by means of different methodologies discussed in a review by Wang and Lin.<a onclick="showRef(event, 'ref161'); return false;" href="javascript:void(0);" class="ref ref161">(161)</a> In the field of proteochemometrics,<a onclick="showRef(event, 'ref162'); return false;" href="javascript:void(0);" class="ref ref162">(162)</a> this task is extended toward the prediction of selectivity profiles for certain small molecule binders. The performance of those approaches benefits significantly from calculated descriptors derived from structural binding site comparisons. The following two examples demonstrate the applicability of binding site comparison methods for proteochemometric approaches.</div><div class="NLM_p">The method FLAP<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> was used to cluster kinase families.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Furthermore, it was tested whether the method is able to identify binding site features that correlate with the reported inhibition profile of the highly promiscuous kinase inhibitor <b>2</b>. Fourteen protein kinases of potential interest as pharmaceutical targets with available X-ray structures and pocket information were selected. The authors used the generated binding site similarity matrix to compute a partial least-squares model with pIC<sub>50</sub> values of <b>2</b> for the different kinases as dependent variable. This analysis resulted in the choice of the best combination of chemical probes to generate GRID molecular interaction fields for further analyses. All common four-point pharmacophores among the kinases were computed. A variable selection routine (fractional factorial design) was used to remove irrelevant variables from the obtained similarity matrix. Finally, a partial least-squares model was created that was able to cluster binding sites according to the level of inhibition by compound <b>2</b>. This example underlines the applicability of binding site comparison methods to create selectivity profiles for promiscuous inhibitors.</div><div class="NLM_p last">Das et al.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> used the PESD<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> method for a proteochemometric approach, trying to predict p<i>K</i><sub>d</sub>/p<i>K</i><sub>i</sub> values on the basis of PESD signatures from protein and ligand interaction surfaces. Support vector machine (SVM) models were trained with the PESD signatures of 278 nonredundant protein–ligand complexes out of a refined PDBbind<a onclick="showRef(event, 'ref163'); return false;" href="javascript:void(0);" class="ref ref163">(163)</a> set of complexes with known p<i>K</i><sub>d</sub>/p<i>K</i><sub>i</sub> values. From this set 977 different protein–ligand complexes were used as test set for model validation. Altogether, the authors created five models with Pearson’s correlation coefficients ranging from 0.517 to 0.638, a leave-one-out cross-validated <i>r</i> between 0.482 and 0.633, and a maximum standard deviation of 1.86. Trained SVM models were used to predict the binding affinities for eight different targets. Furthermore, the model performance with respect to docking pose scoring was assessed and compared to another scoring function. It could be shown that both methods achieved comparable accuracy. Nevertheless, by examining data of enthalpy and entropy analyses obtained from the SCORPIO database,<a onclick="showRef(event, 'ref164'); return false;" href="javascript:void(0);" class="ref ref164">(164)</a> the authors conclude that the method works best for complexes with a dominant enthalpic contribution, as most scoring and ranking methods do.</div></div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Applications Covering the Whole Drug Design Workflow</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76108" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76108" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec7_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Identification of Novel Promising Targets</h3><div class="NLM_p">To avoid off-target effects, it can be useful to identify interesting targets that do not have anything in common with any human protein or are at least quite dissimilar to their human counterpart. Often, the life of pathogenic microorganisms depends on unique metabolic pathways. The next example shows how binding site comparison methods can help to identify promising targets on the basis of their singularity for the organisms of interest.</div><div class="NLM_p last">NAD and NADP are essential cofactors of various enzymes. A comparison of distinct locally conserved pyrophosphate-binding structures (3D motifs) of NAD(P)-bound protein structures with the help of one-dimensional motifs was used to annotate NAD(P)-binding proteins.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> The basic idea is to identify structural motifs by using an automated method for the discovery of motifs of various sizes across protein families with the help of a 16-letter structural alphabet.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Five out of 12 previously defined<a onclick="showRef(event, 'ref165'); return false;" href="javascript:void(0);" class="ref ref165">(165)</a> classes of pyrophosphate binding βα structures (β-strand followed by a turn and a phosphate binding α-helix) with distinct architectures were used to derive sequence motifs. With the help of the derived one-dimensional motifs it was not only possible to differentiate between proteins binding NAD(P) and those not binding these cofactors, but also between proteins binding only NAD or NADP. Intriguingly, one enzyme was identified that owns a unique pyrophosphate-binding 3D motif. This enoyl–acyl carrier protein reductase (EC 1.3.1.9, 1.3.1.10) is essential for bacterial fatty acid biosynthesis. The authors conclude that its uniqueness as compared to human proteins and the fact that the enzyme is conserved across many bacterial species point toward its suitability as a novel drug target.</div></div><div id="sec7_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Binding Site Identification</h3><div class="NLM_p">The PDB contains about 114 000 biological macromolecular structures, whereby approximately 28 000 apo structures have unknown binding sites. Therefore, it is of great interest to identify putative druggable binding sites by different methods. Most of them are based on geometric or energetic properties of the protein’s surface or sequence similarities. Two articles<a onclick="showRef(event, 'ref29 ref166'); return false;" href="javascript:void(0);" class="ref ref29 ref166">(29, 166)</a> give an overview over different approaches to find binding sites in proteins. Given the redundancy of PDB entries, it is also possible to identify binding sites derived from ligand-defined binding sites of similar proteins using appropriate comparison methods. This approach led to reliable results in two examples which are presented below.</div><div class="NLM_p">The first example<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> deals with the detection of a putative binding site in Cas1 of <i>Escherichia coli</i> (YgbT). The protein takes part in a system of adaptive immunity against viruses and plasmids in prokaryotes. It is a CRISPR (clustered regularly interspaced short palindromic repeats) associated protein whose function could not be elucidated. It was known that the enzyme exhibits nuclease activity against single-stranded and branched DNAs including Holliday junctions, replication forks, and 5′-flaps. By means of a Dali search<a onclick="showRef(event, 'ref167'); return false;" href="javascript:void(0);" class="ref ref167">(167)</a> three other Cas1 proteins belonging to different CRISPR subtypes with low overall sequence similarity were found. A surface charge analysis was used to detect several large patches of positively charged residues as potential DNA binding sites. The DNA binding site of the enzyme could not be identified by both methods because YgbT contains two clusters of positively charged amino acids. The application of SurfaceScreen<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> analysis revealed similarities between the enzyme of interest (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nkd">3nkd</a>) and <i>E. coli</i> topoisomerase III (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i7d">1i7d</a>). After superposition of a YgbT main basic patch and the single-stranded DNA binding site of <i>E. coli</i> topoisomerase III, a possible binding mode of single-stranded DNA to YgbT was modeled. Site-directed mutagenesis studies substantiated the results. The authors could show that YgbT belongs to a novel, structurally distinct family of nucleases acting on branched DNAs. They suggest that some components of the CRISPR-Cas system additionally have a function in DNA repair. The identified binding site was later validated by crystal structures of the DNA-bound enzyme (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5dlj">5dlj</a>).</div><div class="NLM_p last">Binding site comparison approaches are also valuable tools for the deorphanization and functional prediction of “orphan” proteins. One example<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> shows the applicability of purely computational approaches to investigate nuclear receptors (NR) without obvious ligand-binding pockets in their X-ray structures. NR4A1 is part of the steroid receptor superfamily. Its expression is induced by stress stimuli and cellular growth-factor signaling. Its impact on tumor-cell apoptosis in multiple tissue types and on apoptotic signaling of thymocytes as well as within the hypothalamic–pituitary axis was shown before, motivating the search for small molecule modulators of NR4A1. The apo NR4A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2qw4">2qw4</a>) was subjected to a 4.1 μs MD simulation. It was shown that an irreversible loop movement leads to the presence of an “open” and “closed” receptor state. A pocket analysis was performed for 15 clusters obtained from a clustering of 4100 MD snapshots. For each structure large pockets, which were absent in the original X-ray structure, were extracted. This analysis revealed the presence of a previously unidentified pocket. In comparison to the general NR ligand-binding site, it is differently located. Its presence can be observed for about 500–800 ns before a flexible loop region closes the binding site in an irreversible manner. The stability of the identified pocket was assessed by performing an independent MD simulation starting with the empty pocket. It remained stable for 1 μs. Encouraged by those results, a PoLiMorph<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> pocket graph description was used to search for structurally related protein binding sites in the sc-PDB.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> The ligands of the most similar binding sites were extracted. Apart from nucleotides (adenosine-5′-β,γ-methylene triphosphate, CoA, FAD, guanosine 5′-monophosphate), a bis-aza-indole compound was retrieved. The fact that one of the known NR4A1 activators is bis-indole substantiates the impact of the discovered binding site. The already known ligands of NR4A1 were docked into the identified binding site. MD simulations of the complexes showed stable protein conformations for the first ligand for at least 1 μs. A simulation with the second ligand shows a higher flexibility in terms of rmsd due to the ligand’s flexible alkyl chains. The presence of the second ligand led to a closure of the binding pocket forming loop and a trapping of the bound ligand. As this remote loop has the capacity to communicate with a Nurr1-binding motif NBRE within an RXR-α/NR4A1 heterodimer, the finding suggests a possible mode of action for the investigated small molecule modulators. The presented results show how previously identified pockets can be additionally verified using suitable binding site comparison tools.</div></div><div id="sec7_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Comparison of Proteins from Different Organisms</h3><div class="NLM_p">In silico drug design methods highly rely on structures of similar proteins and homologs of the protein of interest (e.g., for homology modeling or binding pose prediction). Additionally, many experimental analyses depend on the findings for enzymes from other organisms rather than from the organism of interest due to expression or crystallization problems. Two applications of binding site comparison methods presented here show that pockets of related proteins are not always similar. The assumption that the results obtained for a protein of a different organism can be transferred toward the protein of interest has to be taken with caution.</div><div class="NLM_p">MAOs (isoforms MAO-A, MAO-B) are catabolic enzymes of monoamine neurotransmitters. Therefore, they represent interesting targets of antidepressant and antiparkinsonian drugs. The study presented here<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> aimed to study the biochemical validity of zebrafish MAO as disease model for neurobehavioral studies, and virtual screening. PocketMatch<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> was used to analyze similarities and dissimilarities between the MAO isoforms from human, rat, and zebrafish. Intriguingly, the comparison between human and rat MAO-A, which share a high sequence identity with respect to the active sites, revealed that both enzymes differ in the conformations of several active site residues. Nevertheless, the use of various distance thresholds from 3 to 10 Å could show that the binding sites are more similar when considering a larger ligand environment. Homology models of zebrafish MAO based on human MAO-A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2bxs">2bxs</a>) and MAO-B (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2byb">2byb</a>) were compared to the human enzymes. While they share a high global structural similarity, their binding sites are dissimilar in terms of shape and physicochemical properties. Kinetic measurements with seven known mammalian MAO-A and MAO-B inhibitors clearly show that inhibition constants largely differ among different species. No effect on zebrafish MAO could be observed for the amphetamine derivative 1-(4-(methylthio)phenyl)propan-2-amine (MTA).<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> The same holds true for the highly selective MAO-B inhibitors 6-(4-butoxyphenyl)thiomorpholin-3-one (BTO),<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> 6-(4-(benzyloxy)phenyl)thiomorpholin-3-one (ZTO),<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> and 2-(4-(benzyloxy)phenyl)thiomorpholine (ZTI).<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> The inhibitory mechanism of the naphthylisopropylamine derivative 1-(6-methoxynapthalen-2-yl)propan-2-amine (MeONIPA)<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> was analyzed using molecular docking and MD simulations. The in silico studies suggest that the inhibitors show different modes of action for distant MAOs despite similar inhibition constants. In conclusion, the authors state that although zebrafish and human MAOs share some overlapping functional and structural properties, they might not be comparable with respect to their effector molecule responses. Therefore, studies using zebrafish as model system should be analyzed with care.</div><div class="NLM_p">A similar analysis<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> was performed for structures targeted by cardiovascular drugs. CMASA<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> was applied to compare the binding sites of different drug binding pockets which were previously defined using docking and MD simulations. All binding sites were used as queries to search for structural similarities in a nonredundant SCOP<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> database. The binding patterns of withdrawn cardiovascular drugs between human and mouse proteins were shown to be highly different due to different active site compositions. In contrast, the binding patterns and affinities seem to be similar for FDA-approved cardiovascular drugs (3<i>R</i>,5<i>R</i>)-7-(2-(4-fluorophenyl)-5-isopropyl-3-phenyl-4-(phenylcarbamoyl)-1<i>H</i>-pyrrol-1-yl)-3,5-dihydroxyheptanoic acid (atorvastatin)<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and methyl (<i>S</i>)-2-(2-chlorophenyl)-2-(6,7-dihydrothieno[3,2-<i>c</i>]pyridin-5(4<i>H</i>)-yl)acetate (clopidogrel).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The withdrawn drugs 5-(4-((6-hydroxy-2,5,7,8-tetramethylchroman-2-yl)methoxy)benzyl)thiazolidine-2,4-dione (troglitazone),<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> (3<i>R</i>,5<i>S</i>,<i>E</i>)-7-(4-(4-fluorophenyl)-2,6-diisopropyl-5-(methoxymethyl)pyridin-3-yl)-3,5-dihydroxyhept-6-enoic acid (cerivastatin),<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> 4-methoxy-<i>N</i>-(2-(2-(1-methylpiperidin-2-yl)ethyl)phenyl)benzamide (encainide),<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> (1<i>S</i>,2<i>S</i>)-2-(2-((3-(1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)propyl)(methyl)amino)ethyl)-6-fluoro-1-isopropyl-1,2,3,4-tetrahydronaphthalen-2-yl 2-methoxyacetate (mibefradil),<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and ethyl ((<i>R</i>)-1-cyclohexyl-2-((<i>S</i>)-2-((4-((<i>Z</i>)-<i>N</i>′-hydroxycarbamimidoyl)phenyl)(methyl)carbamoyl)azetidin-1-yl)-2-oxoethyl)glycinate (ximelagatran)<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> were analyzed. Altogether, the differences between humans and mouse models for the testing of compounds to treat cardiovascular diseases might arise due to differences in the binding sites of their respective target proteins. Additionally, off-target effects observed in humans might not necessarily show up in an animal model as a result of sequence divergence and, vice versa, wanted effects observed in the mouse model might not occur in humans.</div><div class="NLM_p last">The last example<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> is dedicated to a rather rarely used application of binding site comparison methods. Given that only protein structures of 16 out of 179 potential drug targets for the treatment of tuberculosis are solved, a so-called “homolog-rescue strategy” was applied. On the basis of 1675 homologs from nine mycobacterial species different from <i>Mtb</i>, it could be shown that structurally solved homologs of 52 otherwise intractable <i>Mtb</i> targets exist. Altogether, 614 proteins from mycobacterial species were extracted from the PDB to assess the applicability of those targets in antimycobacterial drug design. Pairs of non-<i>Mtb</i>-mycobacterial proteins and <i>Mtb</i> proteins with a sequence similarity above 25% and sequence coverage above 70% over the <i>Mtb</i> sequences were retained for further analyses. The active sites of the resulting 106 pairs of <i>Mtb</i> and non-<i>Mtb</i>-mycobacterial enzyme homologs were compared with experimentally determined structures to elucidate whether those 52 targets represent useful surrogates for tuberculosis drug design. For some of those pairs, binding site comparison using a method developed by Totrov<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> revealed conserved active site shape and geometry in terms of active site Cα rmsd, side chain identity, and similarity with respect to pharmacophoric properties. The results for 52 non-<i>Mtb</i> mycobacterial enzyme homologs indicate that it is possible to increase the effective structural coverage of all potential <i>Mtb</i> targets more than 3-fold. This strategy was applied to cytidylate kinase. The enzyme is crucial for the synthesis and salvage pathways of DNA and RNA precursors. Protein structures were solved for homologs from <i>M. smegmatis</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r20">3r20</a>) and <i>Mycobacterium abscessus</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4die">4die</a>) that share 68% and 74% sequence identity with the enzyme from <i>Mtb</i>, respectively. Although experimental verification of this “homolog rescue” approach is still missing, various examples in medicinal chemistry show that the presence of a homolog enzyme structure led to successful drug design strategies. This work should encourage researchers to solve crystal structures of homolog proteins or exploit the structures of known homologs if the target structure is intractable. Altogether, binding site comparison proved to be an indispensable tool to assess the suitability of homology models for rational drug design.</div></div><div id="sec7_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Function Prediction</h3><div class="NLM_p">Although the number of available protein structures increases exponentially and function annotation by means of sequence or fold comparison is possible within minutes or even seconds, a study<a onclick="showRef(event, 'ref168'); return false;" href="javascript:void(0);" class="ref ref168">(168)</a> suggested that more than 1000 PDB entries are annotated as proteins of unknown function. Binding site comparison methods facilitate the elucidation of their function and biological importance. This is also discussed in an article by Petrey et al.<a onclick="showRef(event, 'ref169'); return false;" href="javascript:void(0);" class="ref ref169">(169)</a></div><div class="NLM_p">A clique detection method was used to compare the binding site surface of a pyrophosphatase encoded by the gene MJ0226 of <i>Methanocaldococcus jannaschii</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b78">1b78</a>) to other functional sites stored in the eF-site database.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The enzyme hydrolyzes noncanonical purine nucleotides to their respective monophosphate derivatives.<a onclick="showRef(event, 'ref170'); return false;" href="javascript:void(0);" class="ref ref170">(170)</a> The method of Kinoshita et al.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> successfully revealed a high similarity of this pyrophosphatase to two other structurally unrelated nucleotide-hydrolyzing enzymes: folylpolyglutamate synthetase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jbv">1jbv</a>) and pyruvate kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1a49">1a49</a>).<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> On the basis of those similarities, a possible mononucleotide binding site was identified, underpinning the applicability of surface comparison methods for function prediction. Interestingly, the applied method helped to detect the protein’s functional binding site due to the method’s ability to find an optimal substructure match for a query surface of the complete protein. <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> shows a nucleotide binding site alignment of pyrophosphatase and pyruvate kinase and provides a comparison of both surfaces. Intriguingly, the enzymes show dissimilar folds. They exclusively share similar binding site surface properties.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Comparison of the nucleotide binding surfaces of the pyrophosphatase and pyruvate kinase after an alignment according to bound nucleotides: (a) overall structure of the <i>Methanocaldococcus jannaschii</i> pyrophosphatase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2mjp">2mjp</a>); (b) overall structure of pyruvate kinase from <i>Oryctolagus cuniculus</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1a49">1a49</a>); (c) binding site surface of pyrophosphatase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2mjp">2mjp</a>) colored according to the electrostatic potential with bound ATP analog (sticks); (d) binding site surface of pyruvate kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1a49">1a49</a>) colored according to the electrostatic potential with bound ATP (sticks). Parts a–d were created using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Konc et al. were able to partly unravel the function of a protein encoded by the TM1631 gene from <i>Thermotoga maritima</i>.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> The predicted binding site of a known crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1vpq">1vpq</a>) deposited by the Joint Center for Structural Genomics was compared to binding sites in the PDB with the help of ProBiS.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> The most similar binding sites are involved in DNA replication, phosphate transfer, or DNA repair. They are part of proteins with quite different folding patterns. A relationship between TM1631 and the DNA-binding protein endonuclease IV (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2nqj">2nqj</a>) with regard to a similar electrostatic potential of their binding sites was proposed. A model of the uncharacterized DNA-bound protein was built. The complex was validated using MD simulations. Finally, the authors suggested a possible DNA repair function of the protein. The experimental validation of this hypothesis is missing. Nevertheless, this example underlines the impact of binding site comparison if function cannot be inferred from sequence or fold similarity to well-characterized proteins.</div><div class="NLM_p">Binding site comparison methods were also successfully applied to annotate proteomes as shown for the <i>Mtb</i> proteome.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> The authors developed a workflow using the <i>Mtb</i> genome, extracted all available protein structures from the databases PDB and ModBase,<a onclick="showRef(event, 'ref171'); return false;" href="javascript:void(0);" class="ref ref171">(171)</a> and modeled 54 additional protein structures. Putative binding sites were identified using two binding site detection algorithms and compared to known sites in the PDB using PocketMatch.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> A PLP binding site of a methionine-γ-lyase of <i>Trichomonas vaginalis</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e5f">1e5f</a>) was within the best scored hits. Consequently, a possible function of the PLP dependent enzyme Rv3340 as sulfhydrylase was proposed. In the same year, a crystallographic study<a onclick="showRef(event, 'ref172'); return false;" href="javascript:void(0);" class="ref ref172">(172)</a> supported this hypothesis. Unfortunately, the crystal structure of the <i>Mtb</i> enzyme was never published. A crystal structure of the <i>Mycobacterium marinum</i> enzyme (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4kam">4kam</a>, 91% sequence identity to the <i>Mtb</i> enzyme) allows a structural comparison to methionine-γ-lyase from <i>Mtb</i>. Both enzymes share a common overall fold and highly similar active sites.</div><div class="NLM_p last">A comparable analysis<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> was performed to analyze 616 human protein-coding genes that have no or insufficient evidence of protein existence. Protein sequences with the lowest level of existence evidence (PE5 proteins) were used as most dubious set of missing proteins. Structures were modeled by folding simulations using I-TASSER<a onclick="showRef(event, 'ref173'); return false;" href="javascript:void(0);" class="ref ref173">(173)</a> which is insignificantly dependent on structures of homologous proteins as compared to other structure prediction algorithms. Functional annotation and GO assignment were achieved using COFACTOR.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> It could be shown that PE5 proteins are overrepresented in transporter and receptor activity GO terms as compared to PE1 proteins (highly credible evidence of protein existence). This observation is consistent with the fold family annotation finding that PE5 contains many membrane proteins and could explain the preclusion of their detection in mass spectrometry. Six high-scoring PE5 proteins out of 66 proteins could be found in the PeptideAtlas 2014-08<a onclick="showRef(event, 'ref174'); return false;" href="javascript:void(0);" class="ref ref174">(174)</a> (based on mass spectrometry reanalyses) data set. Therefore, they might be worth further examination. Altogether, the authors present a useful pipeline to annotate protein-coding genes for proteome analysis.</div></div><div id="sec7_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Analysis of Evolutionary Relationships</h3><div class="NLM_p">The analysis of the evolution of the protein world is commonly performed with respect to protein folds.<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a> Nevertheless, cases of convergent evolution and similarities of proteins with distant functionalities or different folds are usually not accessible from that perspective. Binding site comparison methods, especially those that are sequence and fold independent, can reveal interesting aspects of evolution not analyzed so far. The following two examples will underpin this statement.</div><div class="NLM_p">Cluster analysis for functional binding pocket classification was implemented in the Cavbase approach.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Its applicability for protein family classification was investigated.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The method was optimized and validated for a data set of 105 cavities from functionally diverse enzyme families. Results are shown for two families with pharmaceutical relevance: α-CAs and protein kinases. The CA family contains 14 members, whereas crystal structures were only available for six members showing quite different levels of sequence similarity. Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> succeeded in classifying the different structures at subfamily level. Furthermore, the approach enabled the discrimination between two conformational states of the enzyme as part of the enzymes’ catalytic mechanism. Some unique features could be identified for CA IV demarcating this family from other CA isozymes. Altogether, binding site comparisons can help to elucidate links between proteins which cannot be found using sequence comparisons. The clustering of protein kinases structures according to the physicochemical properties of their binding sites led to similarly promising results. The overall classification was in good agreement with CATH,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> SCOP,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and the landscape analysis of Naumann and Matter<a onclick="showRef(event, 'ref176'); return false;" href="javascript:void(0);" class="ref ref176">(176)</a> using a GRID/cPCA approach. Distinct activation states of protein kinases, which play a crucial role in different disease models, could also be captured by the Cavbase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> approach. As an example, the authors show how the method could distinguish between CDKs bound to cognate cyclin and phosphorylated at a threonine residue of the activation loop. The comparison also resulted in the establishment of relationships between different protein kinase family members. A superposition of Erk2 and p38α structures revealed a high level of similarity between those MAP kinases in distinct binding site regions. Such investigations could contribute to the localization of selectivity-discriminating regions within kinase families. The method’s ability to reveal relationships between remote homologs (low sequence similarity, highly similar in structure) was shown for cAMP-dependent kinases from different organisms.</div><div class="NLM_p">The term convergent evolution is used whenever different organisms evolve in a convergent manner owing to similar selection pressures. Such evolutionary scenarios can be studied on the basis of amino acid changes in protein evolution. An exhaustive search for local structural similarities between nonredundant protein functional sites was performed to find new examples for convergent evolution.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> A nonredundant set of 1924 protein chains was generated using only X-ray structures and a sequence similarity cutoff corresponding to a minimum BLAST<a onclick="showRef(event, 'ref177'); return false;" href="javascript:void(0);" class="ref ref177">(177)</a><i>p</i>-value of 10<sup>–7</sup>. 10 175 surface clefts from this data set were identified by the SURFNET algorithm.<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> The cavities were processed using PROSITE<a onclick="showRef(event, 'ref178'); return false;" href="javascript:void(0);" class="ref ref178">(178)</a> patterns to find functionally important residues and ligand binding sites. Afterward, they were subjected to an all-against-all comparison using Query3d.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> Noncollinear and significant matches in terms of the corresponding <i>Z</i>-score were analyzed. Altogether, 28 out of 32 noncollinear structural matches were identified as common incidents of noncollinearity deriving from circular permutation events in protein sequences. Of the remainder, three sequence inversion events were already published. Additionally, members of the ABC transporter (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b0u">1b0u</a>) and Hpr kinase/phosphorylase families (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kkl">1kkl</a>) represent a case of local analogy in the ATP binding sites. The cocrystallized ligands superpose well in the corresponding alignment, while the three sequence regions involved in the structural match are located 1–2–3 versus 3–1–2 (<a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a>). The authors conclude that the identification of such analogies could be relevant for suggesting experimental strategies to devise new classes of inhibitors for members of the ABC transporter family.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Illustrative example of convergent evolution as observed for a member of the ABC transporter family and a protein of the Hpr kinase/phosphorylase family: (a) alignment between the unique phosphate binding motif of the ABC transport family member (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b0u">1b0u</a>, green) with bound ATP in green sticks representation and Hpr kinase/phosphorylase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kkl">1kkl</a>, orange); (b) detailed view of the common phosphate binding motif of both proteins. Two differently oriented β-strands contain essential binding site residues which are represented as sticks. Parts a and b were created using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In the last example<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> a library of about 2000 binding surface types derived from bound forms of protein structures were generated to reflect functional and evolutionary relationships among highly divergent proteins. Pairwise similarities between the binding surfaces of 28 986 ligand-bound PDB structures and the corresponding rmsd values were calculated using fPOP<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> to build a database. The proteins’ functional surfaces as extracted on the basis of a purely geometric approach were obtained from the SplitPocket database.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> Afterward, a finer classification scheme was applied by using surface characteristics of the functional pockets (geometrical, physicochemical, evolutionary features). As a test case, 41 EC annotated oxidoreductase structures with the CATH ID 3.20.20.70 (aldolase class I) were divided into three distinct subtypes that exactly matched the EC annotations. The method was also validated in terms of function inference on an oxidoreductase test set containing 50 ligand-bound structures with aldolase class I fold. Nine of these complexes are not EC annotated. The PSC (protein surface classification) approach was used to reconstruct a tree using the three previously determined surface subtypes as references. It could be shown that the PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gka">3gka</a> belongs to the surface subtype with the EC number 1.3.1.42, whereas the proteins with PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1gwj">1gwj</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2r14">2r14</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kru">3kru</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3krz">3krz</a> show a similar surface to NADPH dehydrogenase (EC 1.6.99.1). Unfortunately, none of those proteins has got an EC classification up to now. Nevertheless, the authors conclude that a comparable approach is also suitable to identify the surface type of an unbound structure, using the classification of bound structures. PSC could thereby reveal the common ancestor of two identified subtypes by inferring from ancestral binding surfaces. This could facilitate the experimental validation of a hypothetical pathway and the investigation of functional interchangeability. The method was tested for the classification of highly divergent proteins to substantiate this assumption. As an example, 143 members of the surface type glycosidase with low sequence identity but highly conserved surfaces were clustered using a fine classification scheme. A dendrogram was obtained via hierarchical clustering analysis. It shows a division into four subtypes. The first three belong to distinct EC classes, while the last group includes enzymes with EC numbers differing in the last digit. All surfaces show similar physicochemical properties, suggesting that functional dissimilarity is related to a variation in residue composition. One class associated with EC 2.4.1.19 (other classes belonging to EC 3.2.1.<i>x</i>) is distinct from the others with respect to surface area, global skewness, and kurtosis. Altogether, this application shows not only the impact of binding site comparison on the analysis of evolutionary pathways but also the influence of different protein surface features on classification schemes.</div></div></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Summary and Outline</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13654" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13654" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Taken together, nearly all binding site comparison approaches enable protein classification and functional inference for uncharacterized proteins. Intriguingly, matches are found not only within one family but also between proteins that share no obvious relationship when comparing one binding site against a large database of small molecule binding sites. This indicates that binding site comparison should be an essential part of the rational drug design workflow to identify potential off-targets, desirable polypharmacology, or novel potential effector molecules. It could support the first steps of new projects by identifying binding sites, predicting protein function, and analyzing novel and promising targets. However, the exploitation of protein binding pocket space is by no means complete. Especially, the linkage between protein similarities and ligand similarities can be explored exhaustively given the huge amount of data available in the public domain and the possible amount of missing links between several proteins.</div><div class="NLM_p">Although this Perspective gives an overview of successful applications of different binding site comparison algorithms, the number of examples underlining the predictivity and general applicability of those methods in rational drug design is still small. We have to keep in mind that those methods were not designed to predict experimentally validated results but to promote experimental studies and to bring forward new ideas. Unfortunately, the number of such applications is limited and anything but proportional to the vast amounts of excellent binding site comparison methods. This was also the limiting factor for the number of approaches presented in this article, although we do not claim completeness.</div><div class="NLM_p">With regard to the level of binding site comparison, it has to be stated that various tools exist implementing very different binding site representations and comparison methods leading to different similarity measures. A quite recent trend tends toward the transformation of complex approaches for the comparison of ligands to the comparison of binding pockets for small molecules (e.g., APF,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> PSIM<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a>). Some approaches also include the information on bound ligands into the binding site comparison, for example, by comparing protein–ligand interactions as realized in KRIPO,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> SILIRID (simple ligand–receptor interaction descriptor),<a onclick="showRef(event, 'ref179'); return false;" href="javascript:void(0);" class="ref ref179">(179)</a> or a recently developed method<a onclick="showRef(event, 'ref180'); return false;" href="javascript:void(0);" class="ref ref180">(180)</a> using graph comparisons.</div><div class="NLM_p">On the basis of the assessment of available literature providing information on successfully applied binding site comparison methods, we conclude that there is no state-of-the-art best-performing algorithm to compare the binding sites of more or less unrelated proteins. Furthermore, it becomes clear that benchmarking of the different methods against each other in terms of quality can never be satisfying. Their performance always depends on the user’s demands, such as finding similarities between otherwise unrelated proteins, which usually equals the search for a needle in the haystack. Additionally, the variability of the approaches and parameters that can be defined and even optimized according to the desired applicability domain hampers a comparison of available methods. Some examples also show that binding site comparison methods do not lead to promising results when used separately but are encouragingly successful when applied in a workflow including different in silico or experimental methods. Nevertheless, it is quite useful to compare all methods with respect to runtime, consideration of protein flexibility, and their applicability for protein classification. Having a look at the methods presented here, it becomes also clear that appropriate data sets to assess the performance pocket comparison algorithms are missing. All benchmark studies are performed using more or less different data sets. The diverse ways of evaluation impede the choice of a suitable method for the respective scientific problem. A method comparison with respect to the test set generated by Barelier et al.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> could settle the question: What do we have to improve: the characterization of the binding site, the comparison algorithm, or the similarity measure?</div><div class="NLM_p">Another question not answered yet is the influence of secondary structure elements on ligand binding. Some first examples show that conserved motifs and spatial arrangements of secondary structure elements can recognize specific functional groups or privileged scaffolds.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> One of the presented studies<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> serves as a hint toward this possibility, but still a detailed analysis of all available binding sites is missing. GIRAF (geometric indexing with refined alignment finder)<a onclick="showRef(event, 'ref181'); return false;" href="javascript:void(0);" class="ref ref181">(181)</a> is one approach that analyzed ligand binding site similarities by clustering structural motifs of ligand binding sites. The structural diversity was assessed by an all-against-all comparison.<a onclick="showRef(event, 'ref182'); return false;" href="javascript:void(0);" class="ref ref182">(182)</a> It could be demonstrated that ligand binding sites are shared across different folds. Additionally, some links concerning bound ligands result from highly regular secondary structure elements. This observation is supported by an earlier article<a onclick="showRef(event, 'ref183'); return false;" href="javascript:void(0);" class="ref ref183">(183)</a> stating that specific combinations of fragments encode specific functions.</div><div class="NLM_p">Considering the examples for kinase binding site comparison, it becomes clear that we cannot deny the huge impact of the protein’s conformational space on the results. Therefore, further development of binding site comparison methods should focus on the inclusion of protein flexibility which is highly dependent on bound ligands and functional states of the proteins. Protein flexibility is taken into account at least to some extent by most methods. Structures retrieved from MD simulation studies are often used to take protein flexibility into account. NMR structures are also a good starting point to introduce flexibility, but they are restricted to small proteins. Additionally, new methods<a onclick="showRef(event, 'ref184'); return false;" href="javascript:void(0);" class="ref ref184">(184)</a> are developed to overcome restrictions in structure comparison which are attributable to conformational flexibility.</div><div class="NLM_p">In conclusion, we have to ask ourselves the question: Do we need another binding site comparison algorithm? Of course, many improved, faster, and more efficient binding site comparison algorithms were developed recently (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf" class="ext-link">Supporting Information</a>). The majority of methods could not be introduced here as a consequence of missing successful applications. Therefore, this question cannot be fully answered as long as binding site comparison is not applied to promote drug design strategies. A deeper look into the literature reveals that the number of methods published to compare small molecule binding sites clearly exceeds the number of publications underlining the basic, independent applicability for structure-based rational drug design (aside from retrospective analyses). Consequently, it is crucial to examine the different application domains and to test binding site comparison methods in predictive studies, including experimental validation of the results. A single basic question remains: Why is there such a low amount of successful published applications of binding site comparison algorithms? Maybe it can be explained by the fact that only a few of the methods are publically available. Nevertheless, there are many software tools that can be downloaded as well as servers that allow users a fast comparison of their protein of interest to known binding sites. Furthermore, the presented applications show that the success of the binding site comparison methods strongly depends on the combination with other in silico tools, for example, MD simulation to account for the protein’s conformational flexibility or binding site detection methods to derive binding sites for apo structures of proteins.</div><div class="NLM_p">Finally, the various applications of binding site comparison methods presented here underline their importance for rational drug design. This assembly of useful applications should encourage its reader to use the presented methods or other methods provided as ready-to-use Web servers and software packages (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf" class="ext-link">Supporting Information</a>) to cope with challenges of his or her own project. Depending on the comparison methods used, the time required to compare binding site ranges from milliseconds to minutes.<a onclick="showRef(event, 'ref185'); return false;" href="javascript:void(0);" class="ref ref185">(185)</a> Therefore, we highly recommend the application of binding site comparison tools to support or even guide experimental investigations. <a class="ref internalNav" href="#fig12" aria-label="Figure 12">Figure 12</a> gives an overview of possible applications of binding site comparison algorithms including successfully employed tools in various hit identification steps to facilitate the choice of a suitable method. Given this huge amount of applicability domains, we have to admit that all algorithms clearly exceed the limits of sequence and structure comparisons. As an emerging technology, binding site comparison algorithms proved to be suitable as orthogonal methods to support design decisions or to speed up the identification of interesting compounds. Therefore, they should become an essential part of rational drug design approaches.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Impact of binding site comparison methods on rational drug design and medicinal chemistry with focus on the importance of its combination with other structure-based drug design methods. Helpful computational tools are shown for different scenarios and are presented in green boxes, while orange boxes highlight steps in a screening workflow which have to be done in the lab. BSC marks steps in those workflows where binding site comparison can help to accelerate the search for promising hits. The superscripts refer to a summary of methods that were already successfully used in this stage of hit identification as discussed in the text. The reader has to bear in mind that the application domains partially overlap and cannot be clearly distinguished.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00078">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81481" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81481" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00078" class="ext-link">10.1021/acs.jmedchem.6b00078</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Summary of different binding site comparison methods and their availability (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf">jm6b00078_si_001.pdf (235.45 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b00078" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13683" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13683" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oliver Koch</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, 44227 Dortmund, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bef1d2d7c8dbcc90f5d1ddd6fecacb93dad1cccad3cbd0da90dadb9ef1d2d7c8dbcc90f5d1ddd6fedfd9d5d1ddd690dadb"><span class="__cf_email__" data-cfemail="8fc0e3e6f9eafda1c4e0ece7cffbfaa2ebe0fdfbe2fae1eba1ebea">[email protected]</span> <span class="__cf_email__" data-cfemail="e6a98a8f908394c8ad89858ea687818d89858ec88283">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christiane Ehrt</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobias Brinkjost</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Computer Science, TU Dortmund University, Otto-Hahn-Straße 14, 44224 Dortmund, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78563" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78563" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Christiane Ehrt</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=BIO-d7e3410-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Christiane Ehrt</b> received her M.Sc. in Biochemistry at the Martin-Luther-University Halle-Wittenberg. Currently, she works as a Ph.D. student at the Faculty of Chemistry and Chemical Biology at the TU Dortmund University in the research group of Dr. Oliver Koch. Her interests and current focus include the identification and comparison of protein ligand binding sites, virtual screening studies for novel targets, and in this context, comparative modeling, MD simulations, and biochemical assay development.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Tobias Brinkjost</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=BIO-d7e3415-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Tobias Brinkjost</b> received his Diploma in Computer Science from the TU Dortmund University in 2012. Afterward he joined the Medicinal Chemistry workgroup of Dr. Oliver Koch in collaboration with the Chair for Algorithm Engineering of Prof. Dr. Petra Mutzel at the TU Dortmund for his Ph.D. studies. His current research mainly focusses on the development of models and algorithms in the field of rational drug design.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Oliver Koch</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=BIO-d7e3420-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Oliver Koch</b> studied pharmacy and computer science at the Philipps-University Marburg, Germany, where he also obtained his Ph.D. in Pharmaceutical Chemistry with Prof. G. Klebe. After postdoctoral research at the Cambridge Crystallographic Data Centre in 2008 and working in drug discovery at MSD Animal Health Innovation, he started his independent academic career in 2012 as a junior group leader for medicinal chemistry at the TU Dortmund University, Germany. His research interests involve the development and application of computational methods in computational molecular design and medicinal chemistry.</p></figure></div><div class="ack" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74725" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74725" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">C.E. is funded by the Kekulé Mobility Fellowship of the Chemical Industry Fund (FCI). T.B. is partially supported by the German Research Foundation (DFG, Priority Program 1736, Algorithms for Big Data). O.K. is funded by the German Federal Ministry for Education and Research (BMBF, Medizinische Chemie in Dortmund, TU Dortmund University, Grant BMBF 1316053).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i33" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i33"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i34" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i34"> Abbreviations Used</h2><tr><td class="NLM_term">5-HT<sub>2B/2C</sub>R</td><td class="NLM_def"><p class="first last">serotonin type 2B/2C metabotropic receptor</p></td></tr><tr><td class="NLM_term">AD</td><td class="NLM_def"><p class="first last">Alzheimer’s disease</p></td></tr><tr><td class="NLM_term">ANT1</td><td class="NLM_def"><p class="first last">adenine nucleotide translocase 1</p></td></tr><tr><td class="NLM_term">APF</td><td class="NLM_def"><p class="first last">atomic property field</p></td></tr><tr><td class="NLM_term">ARK</td><td class="NLM_def"><p class="first last">β adrenergic receptor kinase</p></td></tr><tr><td class="NLM_term">BACE-1</td><td class="NLM_def"><p class="first last">β-secretase</p></td></tr><tr><td class="NLM_term">CA</td><td class="NLM_def"><p class="first last">carbonic anhydrase</p></td></tr><tr><td class="NLM_term">CB1R</td><td class="NLM_def"><p class="first last">cannabinoid receptor 1</p></td></tr><tr><td class="NLM_term">CKII</td><td class="NLM_def"><p class="first last">casein kinase II</p></td></tr><tr><td class="NLM_term">COMT</td><td class="NLM_def"><p class="first last">catechol-<i>O</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">COX</td><td class="NLM_def"><p class="first last">cyclooxygenase</p></td></tr><tr><td class="NLM_term">CRISPR</td><td class="NLM_def"><p class="first last">clustered regularly interspaced short palindromic repeats</p></td></tr><tr><td class="NLM_term">EC</td><td class="NLM_def"><p class="first last">enzyme commission</p></td></tr><tr><td class="NLM_term">ERα</td><td class="NLM_def"><p class="first last">estrogen receptor α</p></td></tr><tr><td class="NLM_term">FAK</td><td class="NLM_def"><p class="first last">focal adhesion kinase</p></td></tr><tr><td class="NLM_term">FGFR</td><td class="NLM_def"><p class="first last">fibroblast growth factor 1 receptor</p></td></tr><tr><td class="NLM_term">GO</td><td class="NLM_def"><p class="first last">gene ontology</p></td></tr><tr><td class="NLM_term">IFP</td><td class="NLM_def"><p class="first last">interaction fingerprint</p></td></tr><tr><td class="NLM_term">IGF-1R</td><td class="NLM_def"><p class="first last">insulin like growth factor 1 receptor</p></td></tr><tr><td class="NLM_term">LSD1</td><td class="NLM_def"><p class="first last">lysine-specific demethylase 1</p></td></tr><tr><td class="NLM_term">MAO</td><td class="NLM_def"><p class="first last">monoamine oxidase</p></td></tr><tr><td class="NLM_term">MMAA</td><td class="NLM_def"><p class="first last">methylmalonyl aciduria associated protein A</p></td></tr><tr><td class="NLM_term"><i>Mtb</i></td><td class="NLM_def"><p class="first last"><i>Mycobacterium tuberculosis</i></p></td></tr><tr><td class="NLM_term">NP</td><td class="NLM_def"><p class="first last">nucleoprotein</p></td></tr><tr><td class="NLM_term">NR</td><td class="NLM_def"><p class="first last">nuclear receptor</p></td></tr><tr><td class="NLM_term">NSAID</td><td class="NLM_def"><p class="first last">nonsteroidal anti-inflammatory drug</p></td></tr><tr><td class="NLM_term">PDK1</td><td class="NLM_def"><p class="first last">phosphoinositide-dependent kinase 1</p></td></tr><tr><td class="NLM_term">PPARα</td><td class="NLM_def"><p class="first last">peroxisome proliferator-activated receptor α</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">PSC</td><td class="NLM_def"><p class="first last">protein surface classification</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">SARS-CoV M<sup>pro</sup></td><td class="NLM_def"><p class="first last">severe acute respiratory syndrome coronavirus main protease</p></td></tr><tr><td class="NLM_term">SERCA</td><td class="NLM_def"><p class="first last">sarcoplasmic reticulum Ca<sup>2+</sup> ion channel ATPase</p></td></tr><tr><td class="NLM_term">SERM</td><td class="NLM_def"><p class="first last">selective estrogen receptor modulators</p></td></tr><tr><td class="NLM_term">SVM</td><td class="NLM_def"><p class="first last">support vector machine</p></td></tr><tr><td class="NLM_term">UBD</td><td class="NLM_def"><p class="first last">ubiquitin-binding domain</p></td></tr><tr><td class="NLM_term">VDAC</td><td class="NLM_def"><p class="first last">voltage-dependent anion channel</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52727" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52727" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 185 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&issue=1&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lg0xnPvComyWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26issue%3D1%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Johnson, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karanicolas, J.</span><span> </span><span class="NLM_article-title">Druggable protein interaction sites are more predisposed to surface pocket formation than the rest of the protein surface</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002951</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1002951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1002951" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1002951&issue=3&author=D.+K.+Johnsonauthor=J.+Karanicolas&title=Druggable+protein+interaction+sites+are+more+predisposed+to+surface+pocket+formation+than+the+rest+of+the+protein+surface&doi=10.1371%2Fjournal.pcbi.1002951"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002951%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DD.%2BK.%26aulast%3DKaranicolas%26aufirst%3DJ.%26atitle%3DDruggable%2520protein%2520interaction%2520sites%2520are%2520more%2520predisposed%2520to%2520surface%2520pocket%2520formation%2520than%2520the%2520rest%2520of%2520the%2520protein%2520surface%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D3%26spage%3De1002951%26doi%3D10.1371%2Fjournal.pcbi.1002951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Loving, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, A. C.</span><span> </span><span class="NLM_article-title">Structure-based druggability assessment of the mammalian structural proteome with inclusion of light protein flexibility</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1003741</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1003741" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003741&issue=7&author=K.+A.+Lovingauthor=A.+Linauthor=A.+C.+Cheng&title=Structure-based+druggability+assessment+of+the+mammalian+structural+proteome+with+inclusion+of+light+protein+flexibility&doi=10.1371%2Fjournal.pcbi.1003741"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003741%26sid%3Dliteratum%253Aachs%26aulast%3DLoving%26aufirst%3DK.%2BA.%26aulast%3DLin%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DA.%2BC.%26atitle%3DStructure-based%2520druggability%2520assessment%2520of%2520the%2520mammalian%2520structural%2520proteome%2520with%2520inclusion%2520of%2520light%2520protein%2520flexibility%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26issue%3D7%26spage%3De1003741%26doi%3D10.1371%2Fjournal.pcbi.1003741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Orengo, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michie, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swindells, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">CATH—a hierarchic classification of protein domain structures</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1016/S0969-2126(97)00260-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0969-2126%2897%2900260-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=1093-1108&issue=8&author=C.+A.+Orengoauthor=A.+D.+Michieauthor=S.+Jonesauthor=D.+T.+Jonesauthor=M.+B.+Swindellsauthor=J.+M.+Thornton&title=CATH%E2%80%94a+hierarchic+classification+of+protein+domain+structures&doi=10.1016%2FS0969-2126%2897%2900260-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2897%2900260-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252897%252900260-8%26sid%3Dliteratum%253Aachs%26aulast%3DOrengo%26aufirst%3DC.%2BA.%26aulast%3DMichie%26aufirst%3DA.%2BD.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DD.%2BT.%26aulast%3DSwindells%26aufirst%3DM.%2BB.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DCATH%25E2%2580%2594a%2520hierarchic%2520classification%2520of%2520protein%2520domain%2520structures%26jtitle%3DStructure%26date%3D1997%26volume%3D5%26issue%3D8%26spage%3D1093%26epage%3D1108%26doi%3D10.1016%2FS0969-2126%2897%2900260-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Murzin, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenner, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chothia, C.</span><span> </span><span class="NLM_article-title">SCOP: a structural classification of proteins database for the investigation of sequences and structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">247</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span><span class="refDoi"> DOI: 10.1016/S0022-2836(05)80134-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2805%2980134-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=7723011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2MXltVGgsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=1995&pages=536-540&issue=4&author=A.+G.+Murzinauthor=S.+E.+Brennerauthor=T.+Hubbardauthor=C.+Chothia&title=SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures&doi=10.1016%2FS0022-2836%2805%2980134-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">SCOP: a structural classification of proteins database for the investigation of sequences and structures</span></div><div class="casAuthors">Murzin, Alexey G.; Brenner, Steven E.; Hubbard, Tim; Chothia, Cyrus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">247</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">536-40</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">To facilitate understanding of, and access to, the information available for protein structures, we have constructed the Structural Classification of Proteins (SCOP) database.  This database provides a detailed and comprehensive description of the structural and evolutionary relationships of the proteins of known structure.  It also provides for each entry links to coordinates, images of the structure, interactive viewers, sequence data and literature refs.  Two search facilitates are available.  The homol. search permits users to enter a sequence and obtain a list of any structures to which it has significant levels of sequence similarity.  The key word search finds, for a word entered by the user, matches from both the text of the SCOP database and the headers of Brookhaven Protein Databank structure files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-9_Oh4CusCbVg90H21EOLACvtfcHk0lj-KpBAa6WdGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXltVGgsr4%253D&md5=65de3f2464aa75ba7efabbd183507913</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980134-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980134-2%26sid%3Dliteratum%253Aachs%26aulast%3DMurzin%26aufirst%3DA.%2BG.%26aulast%3DBrenner%26aufirst%3DS.%2BE.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DChothia%26aufirst%3DC.%26atitle%3DSCOP%253A%2520a%2520structural%2520classification%2520of%2520proteins%2520database%2520for%2520the%2520investigation%2520of%2520sequences%2520and%2520structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1995%26volume%3D247%26issue%3D4%26spage%3D536%26epage%3D540%26doi%3D10.1016%2FS0022-2836%2805%2980134-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span>RCSB PDB. <a href="http://www.rcsb.org/pdb/static.do?p=general_information/pdb_statistics/index.html" class="extLink">http://www.rcsb.org/pdb/static.do?p=general_information/pdb_statistics/index.html</a> (accessed March 21,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RCSB+PDB.+http%3A%2F%2Fwww.rcsb.org%2Fpdb%2Fstatic.do%3Fp%3Dgeneral_information%2Fpdb_statistics%2Findex.html+%28accessed+March+21%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Skolnick, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srinivasan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, H.</span><span> </span><span class="NLM_article-title">Implications of the small number of distinct ligand binding pockets in proteins for drug discovery, evolution and biochemical function</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1163</span><span class="NLM_x">–</span> <span class="NLM_lpage">1170</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.01.059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.bmcl.2015.01.059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1163-1170&issue=6&author=J.+Skolnickauthor=M.+Gaoauthor=A.+Royauthor=B.+Srinivasanauthor=H.+Zhou&title=Implications+of+the+small+number+of+distinct+ligand+binding+pockets+in+proteins+for+drug+discovery%2C+evolution+and+biochemical+function&doi=10.1016%2Fj.bmcl.2015.01.059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.01.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.01.059%26sid%3Dliteratum%253Aachs%26aulast%3DSkolnick%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DSrinivasan%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DH.%26atitle%3DImplications%2520of%2520the%2520small%2520number%2520of%2520distinct%2520ligand%2520binding%2520pockets%2520in%2520proteins%2520for%2520drug%2520discovery%252C%2520evolution%2520and%2520biochemical%2520function%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26issue%3D6%26spage%3D1163%26epage%3D1170%26doi%3D10.1016%2Fj.bmcl.2015.01.059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Paul, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Recovering the true targets of specific ligands by virtual screening of the protein data bank</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">680</span><span class="refDoi"> DOI: 10.1002/prot.10625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.10625" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2004&pages=671-680&issue=4&author=N.+Paulauthor=E.+Kellenbergerauthor=G.+Bretauthor=P.+M%C3%BCllerauthor=D.+Rognan&title=Recovering+the+true+targets+of+specific+ligands+by+virtual+screening+of+the+protein+data+bank&doi=10.1002%2Fprot.10625"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fprot.10625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10625%26sid%3Dliteratum%253Aachs%26aulast%3DPaul%26aufirst%3DN.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DM%25C3%25BCller%26aufirst%3DP.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DRecovering%2520the%2520true%2520targets%2520of%2520specific%2520ligands%2520by%2520virtual%2520screening%2520of%2520the%2520protein%2520data%2520bank%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D54%26issue%3D4%26spage%3D671%26epage%3D680%26doi%3D10.1002%2Fprot.10625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">sc-PDB: an annotated database of druggable binding sites from the Protein Data Bank</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">717</span><span class="NLM_x">–</span> <span class="NLM_lpage">727</span><span class="refDoi"> DOI: 10.1021/ci050372x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050372x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFCgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=717-727&issue=2&author=E.+Kellenbergerauthor=P.+Mullerauthor=C.+Schalonauthor=G.+Bretauthor=N.+Foataauthor=D.+Rognan&title=sc-PDB%3A+an+annotated+database+of+druggable+binding+sites+from+the+Protein+Data+Bank&doi=10.1021%2Fci050372x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">sc-PDB: An Annotated Database of Druggable Binding Sites from the Protein Data Bank</span></div><div class="casAuthors">Kellenberger, Esther; Muller, Pascal; Schalon, Claire; Bret, Guillaume; Foata, Nicolas; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">717-727</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The s.c.-PDB is a collection of 6,415 three-dimensional structures of binding sites found in the Protein Data Bank (PDB).  Binding sites were extd. from all high-resoln. crystal structures in which a complex between a protein cavity and a small-mol.-wt. ligand could be identified.  Importantly, ligands are considered from a pharmacol. and not a structural point of view.  Therefore, solvents, detergents, and most metal ions are not stored in the s.c.-PDB.  Ligands are classified into four main categories: nucleotides (< 4-mer), peptides (< 9-mer), cofactors, and org. compds.  The corresponding binding site is formed by all protein residues (including amino acids, cofactors, and important metal ions) with at least one atom within 6.5 Å of any ligand atom.  The database was carefully annotated by browsing several protein databases (PDB, UniProt, and GO) and storing, for every s.c.-PDB entry, the following features: protein name, function, source, domain and mutations, ligand name, and structure.  The repository of ligands has also been archived by diversity anal. of mol. scaffolds, and several chemoinformatics descriptors were computed to better understand the chem. space covered by stored ligands.  The s.c.-PDB may be used for several purposes: (i) screening a collection of binding sites for predicting the most likely target(s) of any ligand, (ii) analyzing the mol. similarity between different cavities, and (iii) deriving rules that describe the relationship between ligand pharmacophoric points and active-site properties.  The database is periodically updated and accessible on the web at http://bioinfo-pharma.u-strasbg.fr/scPDB/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNlZ_WkpBUQrVg90H21EOLACvtfcHk0liG2ruS8sR0oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFCgtQ%253D%253D&md5=d035f998192ff55026c619b38f4b82ab</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fci050372x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050372x%26sid%3Dliteratum%253Aachs%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DMuller%26aufirst%3DP.%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DFoata%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3Dsc-PDB%253A%2520an%2520annotated%2520database%2520of%2520druggable%2520binding%2520sites%2520from%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26issue%3D2%26spage%3D717%26epage%3D727%26doi%3D10.1021%2Fci050372x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Sturm, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span> </span><span class="NLM_article-title">Structural insights into the molecular basis of the ligand promiscuity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">2410</span><span class="NLM_x">–</span> <span class="NLM_lpage">2421</span><span class="refDoi"> DOI: 10.1021/ci300196g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300196g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1GhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2410-2421&issue=9&author=N.+Sturmauthor=J.+Desaphyauthor=R.+J.+Quinnauthor=D.+Rognanauthor=E.+Kellenberger&title=Structural+insights+into+the+molecular+basis+of+the+ligand+promiscuity&doi=10.1021%2Fci300196g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Insights into the Molecular Basis of the Ligand Promiscuity</span></div><div class="casAuthors">Sturm, Noe; Desaphy, Jeremy; Quinn, Ronald J.; Rognan, Didier; Kellenberger, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2410-2421</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selectivity is a key factor in drug development.  In this paper, we questioned the Protein Data Bank to better understand the reasons for the promiscuity of bioactive compds.  We assembled a data set of >1000 pairs of three-dimensional structures of complexes between a "drug-like" ligand (as its physicochem. properties overlap that of approved drugs) and two distinct "druggable" protein targets (as their binding sites are likely to accommodate "drug-like" ligands).  Studying the similarity between the ligand-binding sites in the different targets revealed that the lack of selectivity of a ligand can be due (i) to the fact that Nature has created the same binding pocket in different proteins, which do not necessarily have otherwise sequence or fold similarity, or (ii) to specific characteristics of the ligand itself.  In particular, we demonstrated that many ligands can adapt to different protein environments by changing their conformation, by using different chem. moieties to anchor to different targets, or by adopting unusual extreme binding modes (e.g., only apolar contact between the ligand and the protein, even though polar groups are present on the ligand or at the protein surface).  Lastly, we provided new elements in support to the recent studies which suggest that the promiscuity of a ligand might be inferred from its mol. complexity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaq6HYzXDUl7Vg90H21EOLACvtfcHk0liG2ruS8sR0oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1GhurbF&md5=808b3192a4110ec1bb26ddd41165d60d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci300196g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300196g%26sid%3Dliteratum%253Aachs%26aulast%3DSturm%26aufirst%3DN.%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DQuinn%26aufirst%3DR.%2BJ.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DStructural%2520insights%2520into%2520the%2520molecular%2520basis%2520of%2520the%2520ligand%2520promiscuity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D9%26spage%3D2410%26epage%3D2421%26doi%3D10.1021%2Fci300196g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surgand, J.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">A simple and fuzzy method to align and compare druggable ligand-binding sites</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1755</span><span class="NLM_x">–</span> <span class="NLM_lpage">1778</span><span class="refDoi"> DOI: 10.1002/prot.21858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.21858" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=1755-1778&issue=4&author=C.+Schalonauthor=J.-S.+Surgandauthor=E.+Kellenbergerauthor=D.+Rognan&title=A+simple+and+fuzzy+method+to+align+and+compare+druggable+ligand-binding+sites&doi=10.1002%2Fprot.21858"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fprot.21858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21858%26sid%3Dliteratum%253Aachs%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DSurgand%26aufirst%3DJ.-S.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DA%2520simple%2520and%2520fuzzy%2520method%2520to%2520align%2520and%2520compare%2520druggable%2520ligand-binding%2520sites%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2008%26volume%3D71%26issue%3D4%26spage%3D1755%26epage%3D1778%26doi%3D10.1002%2Fprot.21858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azdimousa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Comparison and druggability prediction of protein-ligand binding sites from pharmacophore-annotated cavity shapes</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">2287</span><span class="NLM_x">–</span> <span class="NLM_lpage">2299</span><span class="refDoi"> DOI: 10.1021/ci300184x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300184x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2isb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2287-2299&issue=8&author=J.+Desaphyauthor=K.+Azdimousaauthor=E.+Kellenbergerauthor=D.+Rognan&title=Comparison+and+druggability+prediction+of+protein-ligand+binding+sites+from+pharmacophore-annotated+cavity+shapes&doi=10.1021%2Fci300184x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison and Druggability Prediction of Protein-Ligand Binding Sites from Pharmacophore-Annotated Cavity Shapes</span></div><div class="casAuthors">Desaphy, Jeremy; Azdimousa, Karima; Kellenberger, Esther; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2287-2299</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Estg. the pairwise similarity of protein-ligand binding sites is a fast and efficient way of predicting cross-reactivity and putative side effects of drug candidates.  Among the many tools available, three-dimensional (3D) alignment-dependent methods are usually slow and based on simplified representations of binding site atoms or surfaces.  On the other hand, fast and efficient alignment-free methods have recently been described but suffer from a lack of interpretability.  We herewith present a novel binding site description (VolSite), coupled to an alignment and comparison tool (Shaper) combining the speed of alignment-free methods with the interpretability of alignment-dependent approaches.  It is based on the comparison of neg. images of binding cavities encoding both shape and pharmacophoric properties at regularly spaced grid points.  Shaper approximates the resulting mol. shape with a smooth Gaussian function and aligns protein binding sites by optimizing their vol. overlap.  Volsite and Shaper were successfully applied to compare protein-ligand binding sites and to predict their structural druggability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjUMu2riLDLbVg90H21EOLACvtfcHk0lhOkurgILsxHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2isb7I&md5=d019f05891c51085323078e5a880e30b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci300184x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300184x%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DAzdimousa%26aufirst%3DK.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DComparison%2520and%2520druggability%2520prediction%2520of%2520protein-ligand%2520binding%2520sites%2520from%2520pharmacophore-annotated%2520cavity%2520shapes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D8%26spage%3D2287%26epage%3D2299%26doi%3D10.1021%2Fci300184x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Weill, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Alignment-free ultra-high-throughput comparison of druggable protein-ligand binding sites</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1021/ci900349y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900349y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1yjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=123-135&issue=1&author=N.+Weillauthor=D.+Rognan&title=Alignment-free+ultra-high-throughput+comparison+of+druggable+protein-ligand+binding+sites&doi=10.1021%2Fci900349y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Alignment-Free Ultra-High-Throughput Comparison of Druggable Protein-Ligand Binding Sites</span></div><div class="casAuthors">Weill, Nathanael; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">123-135</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inferring the biol. function of a protein from its three-dimensional structure as well as explaining why a drug may bind to various targets is of crucial importance to modern drug discovery.  Here we present a generic 4833-integer vector describing druggable protein-ligand binding sites that can be applied to any protein and any binding cavity.  The fingerprint registers counts of pharmacophoric triplets from the Cα at. coordinates of binding-site-lining residues.  Starting from a customized data set of diverse protein-ligand binding site pairs, the most appropriate metric and a similarity threshold could be defined for similar binding sites.  The method (FuzCav) has been used in various scenarios: (i) screening a collection of 6000 binding sites for similarity to different queries; (ii) classifying protein families (serine endopeptidases, protein kinases) by binding site diversity; (iii) discriminating adenine-binding cavities from decoys.  The fingerprint generation and comparison supports ultra-high throughput (ca. 1000 measures/s), does not require prior alignment of protein binding sites, and is able to detect local similarity among subpockets.  It is thus particularly well suited to the functional annotation of novel genomic structures with low sequence identity to known X-ray templates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7OYklk3JFxLVg90H21EOLACvtfcHk0lhOkurgILsxHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1yjsg%253D%253D&md5=541a9800a3aa0891f179316412f1df24</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fci900349y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900349y%26sid%3Dliteratum%253Aachs%26aulast%3DWeill%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DAlignment-free%2520ultra-high-throughput%2520comparison%2520of%2520druggable%2520protein-ligand%2520binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D1%26spage%3D123%26epage%3D135%26doi%3D10.1021%2Fci900349y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span>scPDB: An Annotated Database of Druggable Binding Sites from the Protein DataBank. <a href="http://cheminfo.u-strasbg.fr/scPDB/ABOUT" class="extLink">http://cheminfo.u-strasbg.fr/scPDB/ABOUT</a> (accessed March 21,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=scPDB%3A+An+Annotated+Database+of+Druggable+Binding+Sites+from+the+Protein+DataBank.+http%3A%2F%2Fcheminfo.u-strasbg.fr%2FscPDB%2FABOUT+%28accessed+March+21%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Martin, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orengo, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchinson, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karmirantzou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taroni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">Protein folds and functions</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">875</span><span class="NLM_x">–</span> <span class="NLM_lpage">884</span><span class="refDoi"> DOI: 10.1016/S0969-2126(98)00089-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0969-2126%2898%2900089-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=875-884&issue=7&author=A.+C.+Martinauthor=C.+A.+Orengoauthor=E.+G.+Hutchinsonauthor=S.+Jonesauthor=M.+Karmirantzouauthor=R.+A.+Laskowskiauthor=J.+B.+Mitchellauthor=C.+Taroniauthor=J.+M.+Thornton&title=Protein+folds+and+functions&doi=10.1016%2FS0969-2126%2898%2900089-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2898%2900089-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252898%252900089-6%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DA.%2BC.%26aulast%3DOrengo%26aufirst%3DC.%2BA.%26aulast%3DHutchinson%26aufirst%3DE.%2BG.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DKarmirantzou%26aufirst%3DM.%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%26aulast%3DTaroni%26aufirst%3DC.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DProtein%2520folds%2520and%2520functions%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26issue%3D7%26spage%3D875%26epage%3D884%26doi%3D10.1016%2FS0969-2126%2898%2900089-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Meara, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">The recognition of identical ligands by unrelated proteins</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2772</span><span class="NLM_x">–</span> <span class="NLM_lpage">2784</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2772-2784&issue=12&author=S.+Barelierauthor=T.+Sterlingauthor=M.+J.+O%E2%80%99Mearaauthor=B.+K.+Shoichet&title=The+recognition+of+identical+ligands+by+unrelated+proteins&doi=10.1021%2Facschembio.5b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The recognition of identical ligands by unrelated proteins</span></div><div class="casAuthors">Barelier, Sarah; Sterling, Teague; O'Meara, Matthew J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2772-2784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of drugs and reagents to off-targets is well-known.  Whereas many off-targets are related to the primary target by sequence and fold, many ligands bind to unrelated pairs of proteins, and these are harder to anticipate.  If the binding site in the off-target can be related to that of the primary target, this challenge resolves into aligning the 2 pockets.  However, other cases are possible: the ligand might interact with entirely different residues and environments in the off-target, or wholly different ligand atoms may be implicated in the 2 complexes.  To investigate these scenarios at at. resoln., the structures of 59 ligands in 116 complexes (62 pairs in total), where the protein pairs were unrelated by fold but bound an identical ligand, were examd.  In almost half of the pairs, the ligand interacted with unrelated residues in the 2 proteins (29 pairs), and in 14 of the pairs wholly different ligand moieties were implicated in each complex.  Even in those 19 pairs of complexes that presented similar environments to the ligand, ligand superposition rarely resulted in the overlap of related residues.  There appears to be no single pattern-matching "code" for identifying binding sites in unrelated proteins that bind identical ligands, although modeling suggests that there might be a limited no. of different patterns that suffice to recognize different ligand functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1_lQJDNMCrVg90H21EOLACvtfcHk0lhOkurgILsxHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J&md5=9d6ac31c2a58bc85f77e7eb34649c195</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00683%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DThe%2520recognition%2520of%2520identical%2520ligands%2520by%2520unrelated%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D12%26spage%3D2772%26epage%3D2784%26doi%3D10.1021%2Facschembio.5b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span><span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&issue=2&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lg1ieZeFXUKjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26issue%3D2%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Li, G.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.-X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, F.-Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lou, M.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.-H.</span><span> </span><span class="NLM_article-title">In silico analysis and experimental validation of active compounds from Cichorium intybus L. ameliorating liver injury</span> <span class="citation_source-journal">Int. J. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">22190</span><span class="NLM_x">–</span> <span class="NLM_lpage">22204</span><span class="refDoi"> DOI: 10.3390/ijms160922190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.3390%2Fijms160922190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=26389883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1yht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=22190-22204&issue=9&author=G.-Y.+Liauthor=Y.-X.+Zhengauthor=F.-Z.+Sunauthor=J.+Huangauthor=M.-M.+Louauthor=J.-K.+Guauthor=J.-H.+Wang&title=In+silico+analysis+and+experimental+validation+of+active+compounds+from+Cichorium+intybus+L.+ameliorating+liver+injury&doi=10.3390%2Fijms160922190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">In silico analysis and experimental validation of active compounds from Cichorium intybus L. ameliorating liver injury</span></div><div class="casAuthors">Li, Guo-Yu; Zheng, Ya-Xin; Sun, Fu-Zhou; Huang, Jian; Lou, Meng-Meng; Gu, Jing-Kai; Wang, Jin-Hui</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">22190-22204</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">This study aimed at investigating the possible mechanisms of hepatic protective activity of Cichorium intybus L. (chicory) in acute liver injury.  Pathol. observation, reactive oxygen species (ROS) detection and measurements of biochem. indexes on mouse models proved hepatic protective effect of Cichorium intybus L.  Identification of active compds. in Cichorium intybus L. was executed through several methods including ultra performance liq. chromatog./time of flight mass spectrometry (UPLC-TOF-MS).  Similarity ensemble approach (SEA) docking, mol. modeling, mol. docking, and mol. dynamics (MD) simulation were applied in this study to explore possible mechanisms of the hepato-protective potential of Cichorium intybus L.  We then analyzed the chem. compn. of Cichorium intybus L., and found their key targets.  Furthermore, in vitro cytol. examn. and western blot were used for validating the efficacy of the selected compds.  In silico anal. and western blot together demonstrated that selected compd. 10 in Cichorium intybus L. targeted Akt-1 in hepatocytes.  Besides, compd. 13 targeted both caspase-1 and Akt-1.  These small compds. may ameliorate liver injury by acting on their targets, which are related to apoptosis or autophagy.  The conclusions above may shed light on the complex mol. mechanisms of Cichorium intybus L. acting on hepatocytes and ameliorating liver injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQb1mIlt--7bVg90H21EOLACvtfcHk0lg1ieZeFXUKjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1yht7c%253D&md5=8954a2e5f7ad0c125b3ab6a1d6631b80</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3390%2Fijms160922190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms160922190%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-Y.%26aulast%3DZheng%26aufirst%3DY.-X.%26aulast%3DSun%26aufirst%3DF.-Z.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLou%26aufirst%3DM.-M.%26aulast%3DGu%26aufirst%3DJ.-K.%26aulast%3DWang%26aufirst%3DJ.-H.%26atitle%3DIn%2520silico%2520analysis%2520and%2520experimental%2520validation%2520of%2520active%2520compounds%2520from%2520Cichorium%2520intybus%2520L.%2520ameliorating%2520liver%2520injury%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2015%26volume%3D16%26issue%3D9%26spage%3D22190%26epage%3D22204%26doi%3D10.3390%2Fijms160922190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krüger, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowotka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papadatos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">The ChEMBL bioactivity database: an update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D1083</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1090</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-D1090&issue=D1&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+bioactivity+database%3A+an+update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0lg1ieZeFXUKjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520bioactivity%2520database%253A%2520an%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD1083%26epage%3DD1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Wishart, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shrivastava, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassanali, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stothard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolsey, J.</span><span> </span><span class="NLM_article-title">DrugBank: a comprehensive resource for in silico drug discovery and exploration</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D668</span><span class="NLM_x">–</span> <span class="NLM_lpage">D672</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkj067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkj067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=D668-D672&issue=Suppl.+1&author=D.+S.+Wishartauthor=C.+Knoxauthor=A.+C.+Guoauthor=S.+Shrivastavaauthor=M.+Hassanaliauthor=P.+Stothardauthor=Z.+Changauthor=J.+Woolsey&title=DrugBank%3A+a+comprehensive+resource+for+in+silico+drug+discovery+and+exploration&doi=10.1093%2Fnar%2Fgkj067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkj067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkj067%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DShrivastava%26aufirst%3DS.%26aulast%3DHassanali%26aufirst%3DM.%26aulast%3DStothard%26aufirst%3DP.%26aulast%3DChang%26aufirst%3DZ.%26aulast%3DWoolsey%26aufirst%3DJ.%26atitle%3DDrugBank%253A%2520a%2520comprehensive%2520resource%2520for%2520in%2520silico%2520drug%2520discovery%2520and%2520exploration%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26issue%3DSuppl.%25201%26spage%3DD668%26epage%3DD672%26doi%3D10.1093%2Fnar%2Fgkj067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzek, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gindulyte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">PubChem BioAssay: 2014 update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D1075</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1082</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkt978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkt978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24198245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslaq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1075-D1082&issue=D1&author=Y.+Wangauthor=T.+Suzekauthor=J.+Zhangauthor=J.+Wangauthor=S.+Heauthor=T.+Chengauthor=B.+A.+Shoemakerauthor=A.+Gindulyteauthor=S.+H.+Bryant&title=PubChem+BioAssay%3A+2014+update&doi=10.1093%2Fnar%2Fgkt978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem BioAssay: 2014 update</span></div><div class="casAuthors">Wang, Yanli; Suzek, Tugba; Zhang, Jian; Wang, Jiyao; He, Siqian; Cheng, Tiejun; Shoemaker, Benjamin A.; Gindulyte, Asta; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1075-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem's BioAssay database (http://pubchem.ncbi.nlm.nih.gov) is a public repository for archiving biol. tests of small mols. generated through high-throughput screening expts., medicinal chem. studies, chem. biol. research and drug discovery programs.  In addn., the BioAssay database contains data from high-throughput RNA interference screening aimed at identifying crit. genes responsible for a biol. process or disease condition.  The mission of PubChem is to serve the community by providing free and easy access to all deposited data.  To this end, PubChem BioAssay is integrated into the National Center for Biotechnol. Information retrieval system, making them searchable by Entrez queries and cross-linked to other biomedical information archived at National Center for Biotechnol. Information.  Moreover, PubChem BioAssay provides web-based and programmatic tools allowing users to search, access and analyze bioassay test results and metadata.  In this work, we provide an update for the PubChem BioAssay resource, such as information content growth, new developments supporting data integration and search, and the recently deployed PubChem Upload to streamline chem. structure and bioassay submissions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1tnO28g86-rVg90H21EOLACvtfcHk0ljcUXDi5dDulg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslaq&md5=66caa053197bcaf40b49535b0a3e0c27</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt978%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSuzek%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubChem%2520BioAssay%253A%25202014%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD1075%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkt978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Günther, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunkel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campillos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsalaki, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urdiales, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gewiess, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skoblo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preissner, R.</span><span> </span><span class="NLM_article-title">SuperTarget and Matador: resources for exploring drug-target relationships</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D919</span><span class="NLM_x">–</span> <span class="NLM_lpage">D922</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkm862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkm862" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D919-D922&issue=Suppl.+1&author=S.+G%C3%BCntherauthor=M.+Kuhnauthor=M.+Dunkelauthor=M.+Campillosauthor=C.+Sengerauthor=E.+Petsalakiauthor=J.+Ahmedauthor=E.+G.+Urdialesauthor=A.+Gewiessauthor=L.+J.+Jensenauthor=R.+Schneiderauthor=R.+Skobloauthor=R.+B.+Russellauthor=P.+E.+Bourneauthor=P.+Borkauthor=R.+Preissner&title=SuperTarget+and+Matador%3A+resources+for+exploring+drug-target+relationships&doi=10.1093%2Fnar%2Fgkm862"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm862%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DDunkel%26aufirst%3DM.%26aulast%3DCampillos%26aufirst%3DM.%26aulast%3DSenger%26aufirst%3DC.%26aulast%3DPetsalaki%26aufirst%3DE.%26aulast%3DAhmed%26aufirst%3DJ.%26aulast%3DUrdiales%26aufirst%3DE.%2BG.%26aulast%3DGewiess%26aufirst%3DA.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DSkoblo%26aufirst%3DR.%26aulast%3DRussell%26aufirst%3DR.%2BB.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DBork%26aufirst%3DP.%26aulast%3DPreissner%26aufirst%3DR.%26atitle%3DSuperTarget%2520and%2520Matador%253A%2520resources%2520for%2520exploring%2520drug-target%2520relationships%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26issue%3DSuppl.%25201%26spage%3DD919%26epage%3DD922%26doi%3D10.1093%2Fnar%2Fgkm862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Kufareva, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilatovskiy, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Pocketome: an encyclopedia of small-molecule binding sites in 4D</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D535</span><span class="NLM_x">–</span> <span class="NLM_lpage">D540</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkr825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkr825" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D535-D540&issue=D1&author=I.+Kufarevaauthor=A.+V.+Ilatovskiyauthor=R.+Abagyan&title=Pocketome%3A+an+encyclopedia+of+small-molecule+binding+sites+in+4D&doi=10.1093%2Fnar%2Fgkr825"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr825%26sid%3Dliteratum%253Aachs%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DIlatovskiy%26aufirst%3DA.%2BV.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DPocketome%253A%2520an%2520encyclopedia%2520of%2520small-molecule%2520binding%2520sites%2520in%25204D%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DD1%26spage%3DD535%26epage%3DD540%26doi%3D10.1093%2Fnar%2Fgkr825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnickel, G.</span><span> </span><span class="NLM_article-title">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">63, 389</span><span class="refDoi"> DOI: 10.1016/S1093-3263(98)00002-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS1093-3263%2898%2900002-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=9704298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADyaK1czmvF2hsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=359-63%2C+389&issue=6&author=M.+Hendlichauthor=F.+Rippmannauthor=G.+Barnickel&title=LIGSITE%3A+automatic+and+efficient+detection+of+potential+small+molecule-binding+sites+in+proteins&doi=10.1016%2FS1093-3263%2898%2900002-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span></div><div class="casAuthors">Hendlich M; Rippmann F; Barnickel G</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular graphics & modelling</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">359-63, 389</span>
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    </div><div class="casAbstract">LIGSITE is a new program for the automatic and time-efficient detection of pockets on the surface of proteins that may act as binding sites for small molecule ligands.  Pockets are identified with a series of simple operations on a cubic grid.  Using a set of receptor-ligand complexes we show that LIGSITE is able to identify the binding sites of small molecule ligands with high precision.  The main advantage of LIGSITE is its speed.  Typical search times are in the range of 5 to 20 s for medium-sized proteins.  LIGSITE is therefore well suited for identification of pockets in large sets of proteins (e.g., protein families) for comparative studies.  For graphical display LIGSITE produces VRML representations of the protein-ligand complex and the binding site for display with a VRML viewer such as WebSpace from SGI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRvgmm5GX6HIgaK-_ZudimnfW6udTcc2ebuTy55nczTUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czmvF2hsw%253D%253D&md5=b7174632bdeaebcfbedec66824ec5cb6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2898%2900002-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252898%252900002-3%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DBarnickel%26aufirst%3DG.%26atitle%3DLIGSITE%253A%2520automatic%2520and%2520efficient%2520detection%2520of%2520potential%2520small%2520molecule-binding%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D1997%26volume%3D15%26issue%3D6%26spage%3D359%26epage%3D63%252C%2520389%26doi%3D10.1016%2FS1093-3263%2898%2900002-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Laurie, A. T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Q-SiteFinder: an energy-based method for the prediction of protein-ligand binding sites</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">1908</span><span class="NLM_x">–</span> <span class="NLM_lpage">1916</span><span class="refDoi"> DOI: 10.1093/bioinformatics/bti315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbti315" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2005&pages=1908-1916&issue=9&author=A.+T.+R.+Laurieauthor=R.+M.+Jackson&title=Q-SiteFinder%3A+an+energy-based+method+for+the+prediction+of+protein-ligand+binding+sites&doi=10.1093%2Fbioinformatics%2Fbti315"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbti315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbti315%26sid%3Dliteratum%253Aachs%26aulast%3DLaurie%26aufirst%3DA.%2BT.%2BR.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DQ-SiteFinder%253A%2520an%2520energy-based%2520method%2520for%2520the%2520prediction%2520of%2520protein-ligand%2520binding%2520sites%26jtitle%3DBioinformatics%26date%3D2005%26volume%3D21%26issue%3D9%26spage%3D1908%26epage%3D1916%26doi%3D10.1093%2Fbioinformatics%2Fbti315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Fukunishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Prediction of ligand-binding sites of proteins by molecular docking calculation for a random ligand library</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">95</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1002/pro.540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fpro.540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21064162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVKjsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=95-106&issue=1&author=Y.+Fukunishiauthor=H.+Nakamura&title=Prediction+of+ligand-binding+sites+of+proteins+by+molecular+docking+calculation+for+a+random+ligand+library&doi=10.1002%2Fpro.540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of ligand-binding sites of proteins by molecular docking calculation for a random ligand library</span></div><div class="casAuthors">Fukunishi, Yoshifumi; Nakamura, Haruki</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-106</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A new approach to predicting the ligand-binding sites of proteins was developed, using protein-ligand docking computation.  In this method, many compds. in a random library are docked onto the whole protein surface.  We assumed that the true ligand-binding site would exhibit stronger affinity to the compds. in the random library than the other sites, even if the random library did not include the ligand corresponding to the true binding site.  We also assumed that the affinity of the true ligand-binding site would be correlated to the docking scores of the compds. in the random library, if the ligand-binding site was correctly predicted.  We call this method the mol.-docking binding-site finding (MolSite) method.  The MolSite method was applied to 89 known protein-ligand complex structures extd. from the Protein Data Bank, and it predicted the correct binding sites with about 80-99% accuracy, when only the single top-ranked site was adopted.  In addn., the av. docking score was weakly correlated to the exptl. protein-ligand binding free energy, with a correlation coeff. of 0.44.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoOFDTHjqTZ7Vg90H21EOLACvtfcHk0lgBIOsdBgIYzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVKjsbw%253D&md5=77b4a7942208fa4b50b382332664ba6f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fpro.540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.540%26sid%3Dliteratum%253Aachs%26aulast%3DFukunishi%26aufirst%3DY.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DPrediction%2520of%2520ligand-binding%2520sites%2520of%2520proteins%2520by%2520molecular%2520docking%2520calculation%2520for%2520a%2520random%2520ligand%2520library%26jtitle%3DProtein%2520Sci.%26date%3D2011%26volume%3D20%26issue%3D1%26spage%3D95%26epage%3D106%26doi%3D10.1002%2Fpro.540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Pupko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayrose, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glaser, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ben-Tal, N.</span><span> </span><span class="NLM_article-title">Rate4Site: an algorithmic tool for the identification of functional regions in proteins by surface mapping of evolutionary determinants within their homologues</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S71</span><span class="NLM_x">–</span> <span class="NLM_lpage">S77</span><span class="refDoi"> DOI: 10.1093/bioinformatics/18.suppl_1.S71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2F18.suppl_1.S71" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2002&pages=S71-S77&issue=Suppl.+1&author=T.+Pupkoauthor=R.+E.+Bellauthor=I.+Mayroseauthor=F.+Glaserauthor=N.+Ben-Tal&title=Rate4Site%3A+an+algorithmic+tool+for+the+identification+of+functional+regions+in+proteins+by+surface+mapping+of+evolutionary+determinants+within+their+homologues&doi=10.1093%2Fbioinformatics%2F18.suppl_1.S71"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2F18.suppl_1.S71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252F18.suppl_1.S71%26sid%3Dliteratum%253Aachs%26aulast%3DPupko%26aufirst%3DT.%26aulast%3DBell%26aufirst%3DR.%2BE.%26aulast%3DMayrose%26aufirst%3DI.%26aulast%3DGlaser%26aufirst%3DF.%26aulast%3DBen-Tal%26aufirst%3DN.%26atitle%3DRate4Site%253A%2520an%2520algorithmic%2520tool%2520for%2520the%2520identification%2520of%2520functional%2520regions%2520in%2520proteins%2520by%2520surface%2520mapping%2520of%2520evolutionary%2520determinants%2520within%2520their%2520homologues%26jtitle%3DBioinformatics%26date%3D2002%26volume%3D18%26issue%3DSuppl.%25201%26spage%3DS71%26epage%3DS77%26doi%3D10.1093%2Fbioinformatics%2F18.suppl_1.S71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Huang, B.</span><span> </span><span class="NLM_article-title">MetaPocket: a meta approach to improve protein ligand binding site prediction</span> <span class="citation_source-journal">OMICS</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">330</span><span class="refDoi"> DOI: 10.1089/omi.2009.0045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1089%2Fomi.2009.0045" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=325-330&issue=4&author=B.+Huang&title=MetaPocket%3A+a+meta+approach+to+improve+protein+ligand+binding+site+prediction&doi=10.1089%2Fomi.2009.0045"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1089%2Fomi.2009.0045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fomi.2009.0045%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26atitle%3DMetaPocket%253A%2520a%2520meta%2520approach%2520to%2520improve%2520protein%2520ligand%2520binding%2520site%2520prediction%26jtitle%3DOMICS%26date%3D2009%26volume%3D13%26issue%3D4%26spage%3D325%26epage%3D330%26doi%3D10.1089%2Fomi.2009.0045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Xie, Z.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, M.-J.</span><span> </span><span class="NLM_article-title">Methods for predicting protein-ligand binding sites</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1215</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">398</span><span class="refDoi"> DOI: 10.1007/978-1-4939-1465-4_17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2F978-1-4939-1465-4_17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25330972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVOku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1215&publication_year=2015&pages=383-398&author=Z.-R.+Xieauthor=M.-J.+Hwang&title=Methods+for+predicting+protein-ligand+binding+sites&doi=10.1007%2F978-1-4939-1465-4_17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for Predicting Protein-Ligand Binding Sites</span></div><div class="casAuthors">Xie, Zhong-Ru; Hwang, Ming-Jing</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1215</span>
        (<span class="NLM_cas:issue">Molecular Modeling of Proteins</span>),
    <span class="NLM_cas:pages">383-398</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Ligand binding is required for many proteins to function properly.  A large no. of bioinformatics tools have been developed to predict ligand binding sites as a first step in understanding a protein's function or to facilitate docking computations in virtual screening based drug design.  The prediction usually requires only the three-dimensional structure (exptl. detd. or computationally modeled) of the target protein to be searched for ligand binding site(s), and Web servers have been built, allowing the free and simple use of prediction tools.  In this chapter, we review the underlying concepts of the methods used by various tools, and discuss their different features and the related issues of ligand binding site prediction.  Some cautionary notes about the use of these tools are also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjGl8HOBerlbVg90H21EOLACvtfcHk0lgBIOsdBgIYzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVOku7k%253D&md5=5214d7bbc66d968d823d50f34503ffd4</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-1465-4_17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-1465-4_17%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DZ.-R.%26aulast%3DHwang%26aufirst%3DM.-J.%26atitle%3DMethods%2520for%2520predicting%2520protein-ligand%2520binding%2520sites%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1215%26spage%3D383%26epage%3D398%26doi%3D10.1007%2F978-1-4939-1465-4_17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Andersson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linusson, A.</span><span> </span><span class="NLM_article-title">Mapping of ligand-binding cavities in proteins</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1408</span><span class="NLM_x">–</span> <span class="NLM_lpage">1422</span><span class="refDoi"> DOI: 10.1002/prot.22655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.22655" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=1408-1422&issue=6&author=C.+D.+Anderssonauthor=B.+Y.+Chenauthor=A.+Linusson&title=Mapping+of+ligand-binding+cavities+in+proteins&doi=10.1002%2Fprot.22655"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fprot.22655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22655%26sid%3Dliteratum%253Aachs%26aulast%3DAndersson%26aufirst%3DC.%2BD.%26aulast%3DChen%26aufirst%3DB.%2BY.%26aulast%3DLinusson%26aufirst%3DA.%26atitle%3DMapping%2520of%2520ligand-binding%2520cavities%2520in%2520proteins%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2010%26volume%3D78%26issue%3D6%26spage%3D1408%26epage%3D1422%26doi%3D10.1002%2Fprot.22655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Boareto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamagishi, M. E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caticha, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leite, V. B. P.</span><span> </span><span class="NLM_article-title">Relationship between global structural parameters and Enzyme Commission hierarchy: implications for function prediction</span> <span class="citation_source-journal">Comput. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">19</span><span class="refDoi"> DOI: 10.1016/j.compbiolchem.2012.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.compbiolchem.2012.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22926016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWisbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=15-19&author=M.+Boaretoauthor=M.+E.+B.+Yamagishiauthor=N.+Catichaauthor=V.+B.+P.+Leite&title=Relationship+between+global+structural+parameters+and+Enzyme+Commission+hierarchy%3A+implications+for+function+prediction&doi=10.1016%2Fj.compbiolchem.2012.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Relationship between global structural parameters and Enzyme Commission hierarchy: Implications for function prediction</span></div><div class="casAuthors">Boareto, Marcelo; Yamagishi, Michel E. B.; Caticha, Nestor; Leite, Vitor B. P.</div><div class="citationInfo"><span class="NLM_cas:title">Computational Biology and Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-19</span>CODEN:
                <span class="NLM_cas:coden">CBCOCH</span>;
        ISSN:<span class="NLM_cas:issn">1476-9271</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In protein databases there is a substantial no. of proteins structurally detd. but without function annotation.  Understanding the relationship between function and structure can be useful to predict function on a large scale.  We have analyzed the similarities in global physicochem. parameters for a set of enzymes which were classified according to the four Enzyme Commission (EC) hierarchical levels.  Using relevance theory we introduced a distance between proteins in the space of physicochem. characteristics.  This was done by minimizing a cost function of the metric tensor built to reflect the EC classification system.  Using an unsupervised clustering method on a set of 1025 enzymes, we obtained no relevant clustering formation compatible with EC classification.  The distance distributions between enzymes from the same EC group and from different EC groups were compared by histograms.  Such anal. was also performed using sequence alignment similarity as a distance.  Our results suggest that global structure parameters are not sufficient to segregate enzymes according to EC hierarchy.  This indicates that features essential for function are rather local than global.  Consequently, methods for predicting function based on global attributes should not obtain high accuracy in main EC classes prediction without relying on similarities between enzymes from training and validation datasets.  Furthermore, these results are consistent with a substantial no. of studies suggesting that function evolves fundamentally by recruitment, i.e., a same protein motif or fold can be used to perform different enzymic functions and a few specific amino acids (AAs) are actually responsible for enzyme activity.  These essential amino acids should belong to active sites and an effective method for predicting function should be able to recognize them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSJ7tit54xL7Vg90H21EOLACvtfcHk0lgR6BwZWfYVEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFWisbnF&md5=8f88e06f1c98c42ee778d08a8e46e0d0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.compbiolchem.2012.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.compbiolchem.2012.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DBoareto%26aufirst%3DM.%26aulast%3DYamagishi%26aufirst%3DM.%2BE.%2BB.%26aulast%3DCaticha%26aufirst%3DN.%26aulast%3DLeite%26aufirst%3DV.%2BB.%2BP.%26atitle%3DRelationship%2520between%2520global%2520structural%2520parameters%2520and%2520Enzyme%2520Commission%2520hierarchy%253A%2520implications%2520for%2520function%2520prediction%26jtitle%3DComput.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D40%26spage%3D15%26epage%3D19%26doi%3D10.1016%2Fj.compbiolchem.2012.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to rational drug design</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">38</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0707307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fsj.bjp.0707307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=17533416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=38-52&issue=1&author=D.+Rognan&title=Chemogenomic+approaches+to+rational+drug+design&doi=10.1038%2Fsj.bjp.0707307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to rational drug design</span></div><div class="casAuthors">Rognan, D.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-52</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Paradigms in drug design and discovery are changing at a significant pace.  Concomitant to the sequencing of over 180 several genomes, the high-throughput miniaturization of chem. synthesis and biol. evaluation of a multiple compds. on gene/protein expression and function opens the way to global drug-discovery approaches, no more focused on a single target but on an entire family of related proteins or on a full metabolic pathway.  Chemogenomics is this emerging research field aimed at systematically studying the biol. effect of a wide array of small mol.-wt. ligands on a wide array of macromol. targets.  Since the quantity of existing data (compds., targets and assays) and of produced information (gene/protein expression levels and binding consts.) are too large for manual manipulation, information technologies play a crucial role in planning, analyzing and predicting chemogenomic data.  The present review will focus on predictive in silico chemogenomic approaches to foster rational drug design and derive information from the simultaneous biol. evaluation of multiple compds. on multiple targets.  State-of-the-art methods for navigating in either ligand or target space will be presented and concrete drug design applications will be mentioned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPERITBubs0LVg90H21EOLACvtfcHk0lgR6BwZWfYVEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjurw%253D&md5=47b3b2a5155a079b486e56d31e0ca59c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707307%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DChemogenomic%2520approaches%2520to%2520rational%2520drug%2520design%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26issue%3D1%26spage%3D38%26epage%3D52%26doi%3D10.1038%2Fsj.bjp.0707307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Haupt, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daminelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M.</span><span> </span><span class="NLM_article-title">Drug promiscuity in PDB: protein binding site similarity is key</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">e65894</span><span class="refDoi"> DOI: 10.1371/journal.pone.0065894</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0065894" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=23805191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVOhsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e65894&issue=6&author=V.+J.+Hauptauthor=S.+Daminelliauthor=M.+Schroeder&title=Drug+promiscuity+in+PDB%3A+protein+binding+site+similarity+is+key&doi=10.1371%2Fjournal.pone.0065894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Drug promiscuity in PDB: protein binding site similarity is key</span></div><div class="casAuthors">Haupt, V. Joachim; Daminelli, Simone; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e65894</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Drug repositioning applies established drugs to new disease indications with increasing success.  A pre-requisite for drug repurposing is drug promiscuity (polypharmacol.) - a drug's ability to bind to several targets.  There is a long standing debate on the reasons for drug promiscuity.  Based on large compd. screens, hydrophobicity and mol. wt. have been suggested as key reasons.  However, the results are sometimes contradictory and leave space for further anal.  Protein structures offer a structural dimension to explain promiscuity: Can a drug bind multiple targets because the drug is flexible or because the targets are structurally similar or even share similar binding sites.  We present a systematic study of drug promiscuity based on structural data of PDB target proteins with a set of 164 promiscuous drugs.  We show that there is no correlation between the degree of promiscuity and ligand properties such as hydrophobicity or mol. wt. but a weak correlation to conformational flexibility.  However, we do find a correlation between promiscuity and structural similarity as well as binding site similarity of protein targets.  In particular, 71% of the drugs have at least two targets with similar binding sites.  In order to overcome issues in detection of remotely similar binding sites, we employed a score for binding site similarity: Ligand RMSD measures the similarity of the aligned ligands and uncovers remote local similarities in proteins.  It can be applied to arbitrary structural binding site alignments.  Three representative examples, namely the anti-cancer drug methotrexate, the natural product quercetin and the anti-diabetic drug acarbose are discussed in detail.  Our findings suggest that global structural and binding site similarity play a more important role to explain the obsd. drug promiscuity in the PDB than physicochem. drug properties like hydrophobicity or mol. wt.  Addnl., we find ligand flexibility to have a minor influence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSXd_1yYo5lLVg90H21EOLACvtfcHk0lgR6BwZWfYVEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVOhsLnP&md5=99574519028e547ebec5a4c2ed27f6a5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0065894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0065894%26sid%3Dliteratum%253Aachs%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DDaminelli%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DDrug%2520promiscuity%2520in%2520PDB%253A%2520protein%2520binding%2520site%2520similarity%2520is%2520key%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26issue%3D6%26spage%3De65894%26doi%3D10.1371%2Fjournal.pone.0065894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Bajusz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rácz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Héberger, K.</span><span> </span><span class="NLM_article-title">Why is Tanimoto index an appropriate choice for fingerprint-based similarity calculations?</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="refDoi"> DOI: 10.1186/s13321-015-0069-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2Fs13321-015-0069-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=20&author=D.+Bajuszauthor=A.+R%C3%A1czauthor=K.+H%C3%A9berger&title=Why+is+Tanimoto+index+an+appropriate+choice+for+fingerprint-based+similarity+calculations%3F&doi=10.1186%2Fs13321-015-0069-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0069-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0069-3%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DR%25C3%25A1cz%26aufirst%3DA.%26aulast%3DH%25C3%25A9berger%26aufirst%3DK.%26atitle%3DWhy%2520is%2520Tanimoto%2520index%2520an%2520appropriate%2520choice%2520for%2520fingerprint-based%2520similarity%2520calculations%253F%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D20%26doi%3D10.1186%2Fs13321-015-0069-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Dekker, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">Protein structure similarity clustering (PSSC) and natural product structure as inspiration sources for drug development and chemical genomics</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">232</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2005.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.cbpa.2005.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=15939324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltVaqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=232-239&issue=3&author=F.+J.+Dekkerauthor=M.+A.+Kochauthor=H.+Waldmann&title=Protein+structure+similarity+clustering+%28PSSC%29+and+natural+product+structure+as+inspiration+sources+for+drug+development+and+chemical+genomics&doi=10.1016%2Fj.cbpa.2005.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Protein structure similarity clustering (PSSC) and natural product structure as inspiration sources for drug development and chemical genomics</span></div><div class="casAuthors">Dekker, Frank J.; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">232-239</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding small mols. that modulate protein function is of primary importance in drug development and in the emerging field of chem. genomics.  To facilitate the identification of such mols., we developed a novel strategy making use of structural conservatism found in protein domain architecture and natural product inspired compd. library design.  Domains and proteins identified as being structurally similar in their ligand-sensing cores are grouped in a protein structure similarity cluster (PSSC).  Natural products can be considered as evolutionary pre-validated ligands for multiple proteins and therefore natural products that are known to interact with one of the PSSC member proteins are selected as guiding structures for compd. library synthesis.  Application of this novel strategy for compd. library design provided enhanced hit rates in small compd. libraries for structurally similar proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk6ALeUcX2kLVg90H21EOLACvtfcHk0ljXIYSsboGLTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltVaqtrk%253D&md5=87b55a2206ba7d0e9170e3d909142743</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DDekker%26aufirst%3DF.%2BJ.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DProtein%2520structure%2520similarity%2520clustering%2520%2528PSSC%2529%2520and%2520natural%2520product%2520structure%2520as%2520inspiration%2520sources%2520for%2520drug%2520development%2520and%2520chemical%2520genomics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26issue%3D3%26spage%3D232%26epage%3D239%26doi%3D10.1016%2Fj.cbpa.2005.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Koch, O.</span><span> </span><span class="NLM_article-title">Use of secondary structure element information in drug design: polypharmacology and conserved motifs in protein-ligand binding and protein-protein interfaces</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">699</span><span class="NLM_x">–</span> <span class="NLM_lpage">708</span><span class="refDoi"> DOI: 10.4155/fmc.11.26</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.4155%2Ffmc.11.26" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21554076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlvFGisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=699-708&issue=6&author=O.+Koch&title=Use+of+secondary+structure+element+information+in+drug+design%3A+polypharmacology+and+conserved+motifs+in+protein-ligand+binding+and+protein-protein+interfaces&doi=10.4155%2Ffmc.11.26"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Use of secondary structure element information in drug design: polypharmacology and conserved motifs in protein--ligand binding and protein--protein interfaces</span></div><div class="casAuthors">Koch, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">699-708</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The structure-based design of small-mol. inhibitors of protein--ligand and protein--protein interfaces is a key component of drug discovery.  The underlying protein interactions can be regarded based on structural similarity of the secondary structure elements: similarities around the binding site ( ligand-sensing cores'') or in the protein interface ( interface-sensing surfaces'') in otherwise unrelated proteins can be useful in predicting polypharmacol. and identifying new lead structures.  Even small conserved motifs can provide similar interaction patterns in proteins with a completely different fold and function.  The identification of these structural similarities can help in the design of new drugs by guiding further optimization.  Here, the concepts and ideas based on secondary structure element similarities and their successful applications in drug design are reviewed and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXOVETHrI5CLVg90H21EOLACvtfcHk0ljXIYSsboGLTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlvFGisLo%253D&md5=64425bac5c12f615b8302b0f3b4ad7f4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.26%26sid%3Dliteratum%253Aachs%26aulast%3DKoch%26aufirst%3DO.%26atitle%3DUse%2520of%2520secondary%2520structure%2520element%2520information%2520in%2520drug%2520design%253A%2520polypharmacology%2520and%2520conserved%2520motifs%2520in%2520protein-ligand%2520binding%2520and%2520protein-protein%2520interfaces%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26issue%3D6%26spage%3D699%26epage%3D708%26doi%3D10.4155%2Ffmc.11.26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Henrich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salo-Ahen, O. M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wade, R. C.</span><span> </span><span class="NLM_article-title">Computational approaches to identifying and characterizing protein binding sites for ligand design</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1002/jmr.984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fjmr.984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=19746440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2010&pages=209-219&issue=2&author=S.+Henrichauthor=O.+M.+H.+Salo-Ahenauthor=B.+Huangauthor=F.+F.+Rippmannauthor=G.+Crucianiauthor=R.+C.+Wade&title=Computational+approaches+to+identifying+and+characterizing+protein+binding+sites+for+ligand+design&doi=10.1002%2Fjmr.984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Computational approaches to identifying and characterizing protein binding sites for ligand design</span></div><div class="casAuthors">Henrich, Stefan; Salo-Ahen, Outi M. H.; Huang, Bingding; Rippmann, Friedrich F.; Cruciani, Gabriele; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">209-219</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  Given the 3-dimensional structure of a protein, how can 1 find the sites where other mols. might bind to it.  Do these sites have the properties necessary for high affinity binding.  Is this protein a suitable target for drug design.  Here, the authors discuss recent developments in computational methods to address these and related questions.  Geometric methods to identify pockets on protein surfaces have been developed over many years but, with new algorithms, their performance is still improving.  Simulation methods show promise in accounting for protein conformational variability to identify transient pockets but lack the ease of use of many of the (rigid) shape-based tools.  Sequence and structure comparison approaches are benefiting from the constantly increasing size of sequence and structure databases.  Energetic methods can aid identification and characterization of binding pockets, and have undergone recent improvements in the treatment of solvation and hydrophobicity.  The "druggability" of a binding site is still difficult to predict with an automated procedure.  The methodologies available for this purpose range from simple shape and hydrophobicity scores to computationally demanding free energy simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCcgqE6phTYbVg90H21EOLACvtfcHk0ljXIYSsboGLTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSht70%253D&md5=a114c8ffea54495e020c8908a6b092b6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjmr.984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.984%26sid%3Dliteratum%253Aachs%26aulast%3DHenrich%26aufirst%3DS.%26aulast%3DSalo-Ahen%26aufirst%3DO.%2BM.%2BH.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DRippmann%26aufirst%3DF.%2BF.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DComputational%2520approaches%2520to%2520identifying%2520and%2520characterizing%2520protein%2520binding%2520sites%2520for%2520ligand%2520design%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2010%26volume%3D23%26issue%3D2%26spage%3D209%26epage%3D219%26doi%3D10.1002%2Fjmr.984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">How to measure the similarity between protein ligand-binding sites?</span> <span class="citation_source-journal">Curr. Comput.-Aided Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.2174/157340908785747401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2174%2F157340908785747401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=209-220&issue=3&author=E.+Kellenbergerauthor=C.+Schalonauthor=D.+Rognan&title=How+to+measure+the+similarity+between+protein+ligand-binding+sites%3F&doi=10.2174%2F157340908785747401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">How to measure the similarity between protein ligand-binding sites?</span></div><div class="casAuthors">Kellenberger, Esther; Schalon, Claire; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">209-220</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Quantification of local similarity between protein 3D structures is a promising tool in computer-aided drug design and prediction of biol. function.  Over the last ten years, several computational methods were proposed, mostly based on geometrical comparisons.  This review summarizes the recent literature and gives an overview of available programs.  A particular interest is given to the underlying methodologies.  Our anal. points out strengths and weaknesses of the various approaches.  If all described methods work relatively well when two binding sites obviously resemble each other, scoring potential solns. remains a difficult issue, esp. if the similarity is low.  The other challenging question is the protein flexibility, which is indeed difficult to evaluate from a static representation.  Last, most of recently developed techniques are fast and can be applied to large amts. of data.  Examples were carefully chosen to illustrate the wide applicability domain of the most popular methods: detection of common structural motifs, identification of secondary targets for a drug-like compd., comparison of binding sites across a functional family, comparison of homol. models, database screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmk10IU2NMu7Vg90H21EOLACvtfcHk0ljlLtzYd3rXhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKqt7g%253D&md5=d10c65eb50f41ea52c826ebba3693c95</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2174%2F157340908785747401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340908785747401%26sid%3Dliteratum%253Aachs%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DHow%2520to%2520measure%2520the%2520similarity%2520between%2520protein%2520ligand-binding%2520sites%253F%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2008%26volume%3D4%26issue%3D3%26spage%3D209%26epage%3D220%26doi%3D10.2174%2F157340908785747401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span>MOE (Molecular Operating Environment); <span class="NLM_publisher-name">Chemical Computing Group Inc.</span> (<span class="NLM_publisher-loc">1010 Sherbooke St. West, Suite 910, Montreal, QC, H3A 2R7, Canada</span>),<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MOE+%28Molecular+Operating+Environment%29%3B+Chemical+Computing+Group+Inc.+%281010+Sherbooke+St.+West%2C+Suite+910%2C+Montreal%2C+QC%2C+H3A+2R7%2C+Canada%29%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMOE%2520%2528Molecular%2520Operating%2520Environment%2529%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittenberg, L.-O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeyaraj, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gourzoulidou, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinecke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Odermatt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">Compound library development guided by protein structure similarity clustering and natural product structure</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x"> (</span><span class="NLM_issue">48</span><span class="NLM_x">) </span> <span class="NLM_fpage">16721</span><span class="NLM_x">–</span> <span class="NLM_lpage">16726</span><span class="refDoi"> DOI: 10.1073/pnas.0404719101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1073%2Fpnas.0404719101" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=16721-16726&issue=48&author=M.+A.+Kochauthor=L.-O.+Wittenbergauthor=S.+Basuauthor=D.+A.+Jeyarajauthor=E.+Gourzoulidouauthor=K.+Reineckeauthor=A.+Odermattauthor=H.+Waldmann&title=Compound+library+development+guided+by+protein+structure+similarity+clustering+and+natural+product+structure&doi=10.1073%2Fpnas.0404719101"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0404719101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0404719101%26sid%3Dliteratum%253Aachs%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWittenberg%26aufirst%3DL.-O.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DJeyaraj%26aufirst%3DD.%2BA.%26aulast%3DGourzoulidou%26aufirst%3DE.%26aulast%3DReinecke%26aufirst%3DK.%26aulast%3DOdermatt%26aufirst%3DA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DCompound%2520library%2520development%2520guided%2520by%2520protein%2520structure%2520similarity%2520clustering%2520and%2520natural%2520product%2520structure%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26issue%3D48%26spage%3D16721%26epage%3D16726%26doi%3D10.1073%2Fpnas.0404719101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">Ligand binding site superposition and comparison based on Atomic Property Fields: identification of distant homologues, convergent evolution and PDB-wide clustering of binding sites</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S35</span><span class="refDoi"> DOI: 10.1186/1471-2105-12-S1-S35</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-12-S1-S35" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=S35&issue=Suppl.+1&author=M.+Totrov&title=Ligand+binding+site+superposition+and+comparison+based+on+Atomic+Property+Fields%3A+identification+of+distant+homologues%2C+convergent+evolution+and+PDB-wide+clustering+of+binding+sites&doi=10.1186%2F1471-2105-12-S1-S35"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-S1-S35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-S1-S35%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DLigand%2520binding%2520site%2520superposition%2520and%2520comparison%2520based%2520on%2520Atomic%2520Property%2520Fields%253A%2520identification%2520of%2520distant%2520homologues%252C%2520convergent%2520evolution%2520and%2520PDB-wide%2520clustering%2520of%2520binding%2520sites%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26issue%3DSuppl.%25201%26spage%3DS35%26doi%3D10.1186%2F1471-2105-12-S1-S35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Edwards, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baugh, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baydo, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witte, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompkins, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, I. Q. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abendroth, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sankaran, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Voorhis, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myler, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staker, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorimer, D. D.</span><span> </span><span class="NLM_article-title">Crystal structures of Mycobacterial MeaB and MMAA-like GTPases</span> <span class="citation_source-journal">J. Struct. Funct. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span><span class="refDoi"> DOI: 10.1007/s10969-015-9197-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2Fs10969-015-9197-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25832174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlslGjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=91-99&issue=2&author=T.+E.+Edwardsauthor=L.+Baughauthor=J.+Bullenauthor=R.+O.+Baydoauthor=P.+Witteauthor=K.+Thompkinsauthor=I.+Q.+H.+Phanauthor=J.+Abendrothauthor=M.+C.+Cliftonauthor=B.+Sankaranauthor=W.+C.+van+Voorhisauthor=P.+J.+Mylerauthor=B.+L.+Stakerauthor=C.+Grundnerauthor=D.+D.+Lorimer&title=Crystal+structures+of+Mycobacterial+MeaB+and+MMAA-like+GTPases&doi=10.1007%2Fs10969-015-9197-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of Mycobacterial MeaB and MMAA-like GTPases</span></div><div class="casAuthors">Edwards, Thomas E.; Baugh, Loren; Bullen, Jameson; Baydo, Ruth O.; Witte, Pam; Thompkins, Kaitlin; Phan, Isabelle Q. H.; Abendroth, Jan; Clifton, Matthew C.; Sankaran, Banumathi; Van Voorhis, Wesley C.; Myler, Peter J.; Staker, Bart L.; Grundner, Christoph; Lorimer, Donald D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural and Functional Genomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">91-99</span>CODEN:
                <span class="NLM_cas:coden">JSFGAW</span>;
        ISSN:<span class="NLM_cas:issn">1345-711X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The methylmalonyl Co-A mutase-assocd. GTPase MeaB from Methylobacterium extorquens is involved in glyoxylate regulation and required for growth.  In humans, mutations in the homolog methylmalonic aciduria assocd. protein (MMAA) cause methylmalonic aciduria, which is often fatal.  The central role of MeaB from bacteria to humans suggests that MeaB is also important in other, pathogenic bacteria such as Mycobacterium tuberculosis.  However, the identity of the mycobacterial MeaB homolog is presently unclear.  Here, we identify the M. tuberculosis protein Rv1496 and its homologs in M. smegmatis and M. thermoresistibile as MeaB.  The crystal structures of all three homologs are highly similar to MeaB and MMAA structures and reveal a characteristic three-domain homodimer with GDP bound in the G domain active site.  A structure of Rv1496 obtained from a crystal grown in the presence of GTP exhibited electron d. for GDP, suggesting GTPase activity.  These structures identify the mycobacterial MeaB and provide a structural framework for therapeutic targeting of M. tuberculosis MeaB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHyvqCau3g77Vg90H21EOLACvtfcHk0ljlLtzYd3rXhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlslGjs7k%253D&md5=834f0ee305e48e493f1dd22dff711dd3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2Fs10969-015-9197-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10969-015-9197-2%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DT.%2BE.%26aulast%3DBaugh%26aufirst%3DL.%26aulast%3DBullen%26aufirst%3DJ.%26aulast%3DBaydo%26aufirst%3DR.%2BO.%26aulast%3DWitte%26aufirst%3DP.%26aulast%3DThompkins%26aufirst%3DK.%26aulast%3DPhan%26aufirst%3DI.%2BQ.%2BH.%26aulast%3DAbendroth%26aufirst%3DJ.%26aulast%3DClifton%26aufirst%3DM.%2BC.%26aulast%3DSankaran%26aufirst%3DB.%26aulast%3Dvan%2BVoorhis%26aufirst%3DW.%2BC.%26aulast%3DMyler%26aufirst%3DP.%2BJ.%26aulast%3DStaker%26aufirst%3DB.%2BL.%26aulast%3DGrundner%26aufirst%3DC.%26aulast%3DLorimer%26aufirst%3DD.%2BD.%26atitle%3DCrystal%2520structures%2520of%2520Mycobacterial%2520MeaB%2520and%2520MMAA-like%2520GTPases%26jtitle%3DJ.%2520Struct.%2520Funct.%2520Genomics%26date%3D2015%26volume%3D16%26issue%3D2%26spage%3D91%26epage%3D99%26doi%3D10.1007%2Fs10969-015-9197-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Baugh, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Begley, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armour, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dranow, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muruthi, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abendroth, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairman, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dieterich, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staker, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardberg, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napuli, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mundt, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompkins, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyons-Abbott, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abramov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serbzhinskiy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorimer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchko, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stacy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Voorhis, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myler, P. J.</span><span> </span><span class="NLM_article-title">Increasing the structural coverage of tuberculosis drug targets</span> <span class="citation_source-journal">Tuberculosis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">142</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span><span class="refDoi"> DOI: 10.1016/j.tube.2014.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.tube.2014.12.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2015&pages=142-148&issue=2&author=L.+Baughauthor=I.+Phanauthor=D.+W.+Begleyauthor=M.+C.+Cliftonauthor=B.+Armourauthor=D.+M.+Dranowauthor=B.+M.+Taylorauthor=M.+M.+Muruthiauthor=J.+Abendrothauthor=J.+W.+Fairmanauthor=D.+Foxauthor=S.+H.+Dieterichauthor=B.+L.+Stakerauthor=A.+S.+Gardbergauthor=R.+Choiauthor=S.+N.+Hewittauthor=A.+J.+Napuliauthor=J.+Myersauthor=L.+K.+Barrettauthor=Y.+Zhangauthor=M.+Ferrellauthor=E.+Mundtauthor=K.+Thompkinsauthor=N.+Tranauthor=S.+Lyons-Abbottauthor=A.+Abramovauthor=A.+Sekarauthor=D.+Serbzhinskiyauthor=D.+Lorimerauthor=G.+W.+Buchkoauthor=R.+Stacyauthor=L.+J.+Stewartauthor=T.+E.+Edwardsauthor=W.+C.+van+Voorhisauthor=P.+J.+Myler&title=Increasing+the+structural+coverage+of+tuberculosis+drug+targets&doi=10.1016%2Fj.tube.2014.12.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.tube.2014.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tube.2014.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DBaugh%26aufirst%3DL.%26aulast%3DPhan%26aufirst%3DI.%26aulast%3DBegley%26aufirst%3DD.%2BW.%26aulast%3DClifton%26aufirst%3DM.%2BC.%26aulast%3DArmour%26aufirst%3DB.%26aulast%3DDranow%26aufirst%3DD.%2BM.%26aulast%3DTaylor%26aufirst%3DB.%2BM.%26aulast%3DMuruthi%26aufirst%3DM.%2BM.%26aulast%3DAbendroth%26aufirst%3DJ.%26aulast%3DFairman%26aufirst%3DJ.%2BW.%26aulast%3DFox%26aufirst%3DD.%26aulast%3DDieterich%26aufirst%3DS.%2BH.%26aulast%3DStaker%26aufirst%3DB.%2BL.%26aulast%3DGardberg%26aufirst%3DA.%2BS.%26aulast%3DChoi%26aufirst%3DR.%26aulast%3DHewitt%26aufirst%3DS.%2BN.%26aulast%3DNapuli%26aufirst%3DA.%2BJ.%26aulast%3DMyers%26aufirst%3DJ.%26aulast%3DBarrett%26aufirst%3DL.%2BK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DFerrell%26aufirst%3DM.%26aulast%3DMundt%26aufirst%3DE.%26aulast%3DThompkins%26aufirst%3DK.%26aulast%3DTran%26aufirst%3DN.%26aulast%3DLyons-Abbott%26aufirst%3DS.%26aulast%3DAbramov%26aufirst%3DA.%26aulast%3DSekar%26aufirst%3DA.%26aulast%3DSerbzhinskiy%26aufirst%3DD.%26aulast%3DLorimer%26aufirst%3DD.%26aulast%3DBuchko%26aufirst%3DG.%2BW.%26aulast%3DStacy%26aufirst%3DR.%26aulast%3DStewart%26aufirst%3DL.%2BJ.%26aulast%3DEdwards%26aufirst%3DT.%2BE.%26aulast%3Dvan%2BVoorhis%26aufirst%3DW.%2BC.%26aulast%3DMyler%26aufirst%3DP.%2BJ.%26atitle%3DIncreasing%2520the%2520structural%2520coverage%2520of%2520tuberculosis%2520drug%2520targets%26jtitle%3DTuberculosis%26date%3D2015%26volume%3D95%26issue%3D2%26spage%3D142%26epage%3D148%26doi%3D10.1016%2Fj.tube.2014.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A new method to detect related function among proteins independent of sequence and fold homology</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">323</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">387</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)00811-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2802%2900811-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2002&pages=387-406&issue=2&author=S.+Schmittauthor=D.+Kuhnauthor=G.+Klebe&title=A+new+method+to+detect+related+function+among+proteins+independent+of+sequence+and+fold+homology&doi=10.1016%2FS0022-2836%2802%2900811-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900811-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900811-2%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520new%2520method%2520to%2520detect%2520related%2520function%2520among%2520proteins%2520independent%2520of%2520sequence%2520and%2520fold%2520homology%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D323%26issue%3D2%26spage%3D387%26epage%3D406%26doi%3D10.1016%2FS0022-2836%2802%2900811-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Yang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudock, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, L.</span><span> </span><span class="NLM_article-title">Identifying unexpected therapeutic targets via chemical-protein interactome</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">e9568</span><span class="refDoi"> DOI: 10.1371/journal.pone.0009568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0009568" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e9568&issue=3&author=L.+Yangauthor=J.+Chenauthor=L.+Shiauthor=M.+P.+Hudockauthor=K.+Wangauthor=L.+He&title=Identifying+unexpected+therapeutic+targets+via+chemical-protein+interactome&doi=10.1371%2Fjournal.pone.0009568"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0009568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0009568%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DHudock%26aufirst%3DM.%2BP.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DL.%26atitle%3DIdentifying%2520unexpected%2520therapeutic%2520targets%2520via%2520chemical-protein%2520interactome%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26issue%3D3%26spage%3De9568%26doi%3D10.1371%2Fjournal.pone.0009568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Al-Gharabli, S. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S. T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesters, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilgenfeld, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rademann, J.</span><span> </span><span class="NLM_article-title">An efficient method for the synthesis of peptide aldehyde libraries employed in the discovery of reversible SARS coronavirus main protease (SARS-CoV Mpro) inhibitors</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">1048</span><span class="NLM_x">–</span> <span class="NLM_lpage">1055</span><span class="refDoi"> DOI: 10.1002/cbic.200500533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fcbic.200500533" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=1048-1055&issue=7&author=S.+I.+Al-Gharabliauthor=S.+T.+A.+Shahauthor=S.+Weikauthor=M.+F.+Schmidtauthor=J.+R.+Mestersauthor=D.+Kuhnauthor=G.+Klebeauthor=R.+Hilgenfeldauthor=J.+Rademann&title=An+efficient+method+for+the+synthesis+of+peptide+aldehyde+libraries+employed+in+the+discovery+of+reversible+SARS+coronavirus+main+protease+%28SARS-CoV+Mpro%29+inhibitors&doi=10.1002%2Fcbic.200500533"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200500533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200500533%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Gharabli%26aufirst%3DS.%2BI.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DWeik%26aufirst%3DS.%26aulast%3DSchmidt%26aufirst%3DM.%2BF.%26aulast%3DMesters%26aufirst%3DJ.%2BR.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DHilgenfeld%26aufirst%3DR.%26aulast%3DRademann%26aufirst%3DJ.%26atitle%3DAn%2520efficient%2520method%2520for%2520the%2520synthesis%2520of%2520peptide%2520aldehyde%2520libraries%2520employed%2520in%2520the%2520discovery%2520of%2520reversible%2520SARS%2520coronavirus%2520main%2520protease%2520%2528SARS-CoV%2520Mpro%2529%2520inhibitors%26jtitle%3DChemBioChem%26date%3D2006%26volume%3D7%26issue%3D7%26spage%3D1048%26epage%3D1055%26doi%3D10.1002%2Fcbic.200500533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weskamp, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hüllermeier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">From the similarity analysis of protein cavities to the functional classification of protein families using cavbase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1023</span><span class="NLM_x">–</span> <span class="NLM_lpage">1044</span><span class="refDoi"> DOI: 10.1016/j.jmb.2006.04.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.jmb.2006.04.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2006&pages=1023-1044&issue=4&author=D.+Kuhnauthor=N.+Weskampauthor=S.+Schmittauthor=E.+H%C3%BCllermeierauthor=G.+Klebe&title=From+the+similarity+analysis+of+protein+cavities+to+the+functional+classification+of+protein+families+using+cavbase&doi=10.1016%2Fj.jmb.2006.04.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.04.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.04.024%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DWeskamp%26aufirst%3DN.%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DH%25C3%25BCllermeier%26aufirst%3DE.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFrom%2520the%2520similarity%2520analysis%2520of%2520protein%2520cavities%2520to%2520the%2520functional%2520classification%2520of%2520protein%2520families%2520using%2520cavbase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D359%26issue%3D4%26spage%3D1023%26epage%3D1044%26doi%3D10.1016%2Fj.jmb.2006.04.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Weber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Supuran, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scozzafava, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Unexpected nanomolar inhibition of carbonic anhydrase by COX-2-selective celecoxib: new pharmacological opportunities due to related binding site recognition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">557</span><span class="refDoi"> DOI: 10.1021/jm030912m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030912m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFGltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=550-557&issue=3&author=A.+Weberauthor=A.+Casiniauthor=A.+Heineauthor=D.+Kuhnauthor=C.+T.+Supuranauthor=A.+Scozzafavaauthor=G.+Klebe&title=Unexpected+nanomolar+inhibition+of+carbonic+anhydrase+by+COX-2-selective+celecoxib%3A+new+pharmacological+opportunities+due+to+related+binding+site+recognition&doi=10.1021%2Fjm030912m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Unexpected Nanomolar Inhibition of Carbonic Anhydrase by COX-2-Selective Celecoxib: New Pharmacological Opportunities Due to Related Binding Site Recognition</span></div><div class="casAuthors">Weber, Alexander; Casini, Angela; Heine, Andreas; Kuhn, Daniel; Supuran, Claudiu T.; Scozzafava, Andrea; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">550-557</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By optimizing binding to a selected target protein, modern drug research strives to develop safe and efficacious agents for the treatment of disease.  Selective drug action is intended to minimize undesirable side effects from scatter pharmacol.  Celecoxib (Celebrex), valdecoxib (Bextra), and rofecoxib (Vioxx) are nonsteroidal antiinflammatory drugs (NSAIDs) due to selective inhibition of inducible cyclooxygenase COX-2 while sparing inhibition of constitutive COX-1.  While rofecoxib contains a Me sulfone constituent, celecoxib and valdecoxib possess an unsubstituted arylsulfonamide moiety.  The latter group is common to many carbonic anhydrase (CA) inhibitors.  Using enzyme kinetics and X-ray crystallog., we demonstrate an unexpected nanomolar affinity of the COX-2 specific arylsulfonamide-type celecoxib and valdecoxib for isoenzymes of the totally unrelated carbonic anhydrase (CA) family, such as CA I, II, IV, and IX, whereas the rofecoxib Me sulfone-type has no effect.  When administered orally to glaucomatous rabbits, celecoxib and valdecoxib lowered intraocular pressure, suggesting that these agents may have utility in the treatment of this disorder.  The crystal structure of celecoxib in complex with CA II reveals part of this inhibition to be mediated via binding of the sulfonamide group to the catalytic zinc of CA II.  To investigate the structural basis for cross-reactivity of these compds. between COX-2 and CA II, we compared the mol. recognition properties of both protein binding pockets in terms of local physicochem. similarities among binding site-exposed amino acids accommodating different portions of the drug mols.  Our approach Cavbase, implemented into Relibase, detects similarities between the sites, suggesting some potential to predict unexpected cross-reactivity of drugs among functionally unrelated target proteins.  The obsd. cross-reactivity with CAs may also contribute to differences in the pharmacol. profiles, in particular with respect to glaucoma and anticancer therapy and may suggest new opportunities of these COX-2 selective NSAIDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSHRdlhB-jn7Vg90H21EOLACvtfcHk0lgecSNDV9QvPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFGltLc%253D&md5=608b04daff89563144029daea8bd91b7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm030912m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030912m%26sid%3Dliteratum%253Aachs%26aulast%3DWeber%26aufirst%3DA.%26aulast%3DCasini%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DSupuran%26aufirst%3DC.%2BT.%26aulast%3DScozzafava%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DUnexpected%2520nanomolar%2520inhibition%2520of%2520carbonic%2520anhydrase%2520by%2520COX-2-selective%2520celecoxib%253A%2520new%2520pharmacological%2520opportunities%2520due%2520to%2520related%2520binding%2520site%2520recognition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D3%26spage%3D550%26epage%3D557%26doi%3D10.1021%2Fjm030912m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Li, G.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.-F.</span><span> </span><span class="NLM_article-title">CMASA: an accurate algorithm for detecting local protein structural similarity and its application to enzyme catalytic site annotation</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="refDoi"> DOI: 10.1186/1471-2105-11-439</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-11-439" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=20796320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BC3cfht1ylsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=439&author=G.-H.+Liauthor=J.-F.+Huang&title=CMASA%3A+an+accurate+algorithm+for+detecting+local+protein+structural+similarity+and+its+application+to+enzyme+catalytic+site+annotation&doi=10.1186%2F1471-2105-11-439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">CMASA: an accurate algorithm for detecting local protein structural similarity and its application to enzyme catalytic site annotation</span></div><div class="casAuthors">Li Gong-Hua; Huang Jing-Fei</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">439</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The rapid development of structural genomics has resulted in many "unknown function" proteins being deposited in Protein Data Bank (PDB), thus, the functional prediction of these proteins has become a challenge for structural bioinformatics.  Several sequence-based and structure-based methods have been developed to predict protein function, but these methods need to be improved further, such as, enhancing the accuracy, sensitivity, and the computational speed.  Here, an accurate algorithm, the CMASA (Contact MAtrix based local Structural Alignment algorithm), has been developed to predict unknown functions of proteins based on the local protein structural similarity.  This algorithm has been evaluated by building a test set including 164 enzyme families, and also been compared to other methods.  RESULTS:  The evaluation of CMASA shows that the CMASA is highly accurate (0.96), sensitive (0.86), and fast enough to be used in the large-scale functional annotation.  Comparing to both sequence-based and global structure-based methods, not only the CMASA can find remote homologous proteins, but also can find the active site convergence.  Comparing to other local structure comparison-based methods, the CMASA can obtain the better performance than both FFF (a method using geometry to predict protein function) and SPASM (a local structure alignment method); and the CMASA is more sensitive than PINTS and is more accurate than JESS (both are local structure alignment methods).  The CMASA was applied to annotate the enzyme catalytic sites of the non-redundant PDB, and at least 166 putative catalytic sites have been suggested, these sites can not be observed by the Catalytic Site Atlas (CSA).  CONCLUSIONS:  The CMASA is an accurate algorithm for detecting local protein structural similarity, and it holds several advantages in predicting enzyme active sites.  The CMASA can be used in large-scale enzyme active site annotation.  The CMASA can be available by the mail-based server (http://159.226.149.45/other1/CMASA/CMASA.htm).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRHQqnT9KEcsgvupmecNw_lfW6udTcc2ebO1KoajP3VXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfht1ylsA%253D%253D&md5=08ddacbde64685046c8a2c7a71ada06b</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-439%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-H.%26aulast%3DHuang%26aufirst%3DJ.-F.%26atitle%3DCMASA%253A%2520an%2520accurate%2520algorithm%2520for%2520detecting%2520local%2520protein%2520structural%2520similarity%2520and%2520its%2520application%2520to%2520enzyme%2520catalytic%2520site%2520annotation%26jtitle%3DBMC%2520Bioinf.%26date%3D2010%26volume%3D11%26spage%3D439%26doi%3D10.1186%2F1471-2105-11-439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span> </span><span class="NLM_article-title">A comparative analysis of protein targets of withdrawn cardiovascular drugs in human and mouse</span> <span class="citation_source-journal">J. Clin. Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">10</span><span class="refDoi"> DOI: 10.1186/2043-9113-2-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F2043-9113-2-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22548699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOiurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=10&issue=1&author=Y.+Zhaoauthor=J.+Wangauthor=Y.+Wangauthor=J.+Huang&title=A+comparative+analysis+of+protein+targets+of+withdrawn+cardiovascular+drugs+in+human+and+mouse&doi=10.1186%2F2043-9113-2-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A comparative analysis of protein targets of withdrawn cardiovascular drugs in human and mouse</span></div><div class="casAuthors">Zhao, Yuqi; Wang, Jingwen; Wang, Yanjie; Huang, Jingfei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>CODEN:
                <span class="NLM_cas:coden">JCBOBR</span>;
        ISSN:<span class="NLM_cas:issn">2043-9113</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Mouse is widely used in animal testing of cardiovascular disease.  However, a large no. of cardiovascular drugs that have been exptl. proved to work well on mouse were withdrawn because they caused adverse side effects in human.  In this study, we investigate whether binding patterns of withdrawn cardiovascular drugs are conserved between mouse and human through computational dockings and mol. dynamic simulations.  In addn., we also measured the level of conservation of gene expression patterns of the drug targets and their interacting partners by analyzing the microarray data.  The results show that target proteins of withdrawn cardiovascular drugs are functionally conserved between human and mouse.  However, all the binding patterns of withdrawn drugs we retrieved show striking difference due to sequence divergence in drug-binding pocket, mainly through loss or gain of hydrogen bond donors and distinct drug-binding pockets.  The binding affinities of withdrawn drugs to their receptors tend to be reduced from mouse to human.  In contrast, the FDA-approved and best-selling drugs are little affected.  Our anal. suggests that sequence divergence in drug-binding pocket may be a reasonable explanation for the discrepancy of drug effects between animal models and human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKl0p5QoPUGLVg90H21EOLACvtfcHk0lg8e8cBaC9jUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOiurvI&md5=683ea56930f7baaea685d327b2efa92d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1186%2F2043-9113-2-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2043-9113-2-10%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DJ.%26atitle%3DA%2520comparative%2520analysis%2520of%2520protein%2520targets%2520of%2520withdrawn%2520cardiovascular%2520drugs%2520in%2520human%2520and%2520mouse%26jtitle%3DJ.%2520Clin.%2520Bioinf.%26date%3D2012%26volume%3D2%26issue%3D1%26spage%3D10%26doi%3D10.1186%2F2043-9113-2-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Roy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">COFACTOR: an accurate comparative algorithm for structure-based protein function annotation</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x"> (</span><span class="NLM_issue">W1</span><span class="NLM_x">) </span> <span class="NLM_fpage">W471</span><span class="NLM_x">–</span> <span class="NLM_lpage">W477</span><div class="note"><p class="first last">(Web Server issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gks372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgks372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22570420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCrtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=W471-W477&issue=W1&author=A.+Royauthor=J.+Yangauthor=Y.+Zhang&title=COFACTOR%3A+an+accurate+comparative+algorithm+for+structure-based+protein+function+annotation&doi=10.1093%2Fnar%2Fgks372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">COFACTOR: an accurate comparative algorithm for structure-based protein function annotation</span></div><div class="casAuthors">Roy, Ambrish; Yang, Jianyi; Zhang, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W471-W477</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have developed a new COFACTOR webserver for automated structure-based protein function annotation.  Starting from a structural model, given by either exptl. detn. or computational modeling, COFACTOR first identifies template proteins of similar folds and functional sites by threading the target structure through three representative template libraries that have known protein-ligand binding interactions, Enzyme Commission no. or Gene Ontol. terms.  The biol. function insights in these three aspects are then deduced from the functional templates, the confidence of which is evaluated by a scoring function that combines both global and local structural similarities.  The algorithm has been extensively benchmarked by large-scale benchmarking tests and demonstrated significant advantages compared to traditional sequence-based methods.  In the recent community-wide CASP9 expt., COFACTOR was ranked as the best method for protein-ligand binding site predictions.  The COFACTOR sever and the template libraries are freely available at http://zhanglab.ccmb.med.umich.edu/COFACTOR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmrygdjGKk-LVg90H21EOLACvtfcHk0lg8e8cBaC9jUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCrtL0%253D&md5=174b2a49ad059bf00b318d2a96b7a8aa</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks372%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DCOFACTOR%253A%2520an%2520accurate%2520comparative%2520algorithm%2520for%2520structure-based%2520protein%2520function%2520annotation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DW1%26spage%3DW471%26epage%3DW477%26doi%3D10.1093%2Fnar%2Fgks372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Dong, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omenn, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Structural bioinformatics inspection of neXtProt PE5 proteins in the human proteome</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">3750</span><span class="NLM_x">–</span> <span class="NLM_lpage">3761</span><span class="refDoi"> DOI: 10.1021/acs.jproteome.5b00516</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.5b00516" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2msbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=3750-3761&issue=9&author=Q.+Dongauthor=R.+Menonauthor=G.+S.+Omennauthor=Y.+Zhang&title=Structural+bioinformatics+inspection+of+neXtProt+PE5+proteins+in+the+human+proteome&doi=10.1021%2Facs.jproteome.5b00516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Bioinformatics Inspection of neXtProt PE5 Proteins in the Human Proteome</span></div><div class="casAuthors">Dong, Qiwen; Menon, Rajasree; Omenn, Gilbert S.; Zhang, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3750-3761</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One goal of the Human Proteome Project is to identify at least one protein product for each of the ∼20,000 human protein-coding genes.  As of Oct. 2014, however, there are 3564 genes (18%) that have no or insufficient evidence of protein existence (PE), as curated by neXtProt; these comprise 2647 PE2-4 missing proteins and 616 PE5 dubious protein entries.  The authors conducted a systematic examn. of the 616 PE5 protein entries using cutting-edge protein structure and function modeling methods.  Compared to a random sample of high-confidence PE1 proteins, the putative PE5 proteins are over-represented in the membrane and cell surface proteins and peptides fold families.  Detailed functional analyses show that most PE5 proteins, if expressed, would belong to transporters and receptors localized in the plasma membrane compartment.  The results suggest that exptl. difficulty in identifying membrane-bound proteins and peptides could have precluded their detection in mass spectrometry and that special enrichment techniques with improved sensitivity for membrane proteins could be important for the characterization of the PE5 "dark matter" of the human proteome.  Finally, the authors identify 66 high scoring PE5 protein entries and find that six of them are reported in recent mass spectrometry databases; an illustrative annotation of these six is provided.  This work illustrates a new approach to examine the potential folding and function of the dubious proteins comprising PE5, which the authors will next apply to the far larger group of missing proteins comprising PE2-4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgK1Q6ycr73bVg90H21EOLACvtfcHk0lgUIclMMGETZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2msbzK&md5=0dd45b807344b901fdec7c1b26b0af1a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.5b00516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.5b00516%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DQ.%26aulast%3DMenon%26aufirst%3DR.%26aulast%3DOmenn%26aufirst%3DG.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DStructural%2520bioinformatics%2520inspection%2520of%2520neXtProt%2520PE5%2520proteins%2520in%2520the%2520human%2520proteome%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2015%26volume%3D14%26issue%3D9%26spage%3D3750%26epage%3D3761%26doi%3D10.1021%2Facs.jproteome.5b00516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Powers, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercier, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramanathan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revesz, P.</span><span> </span><span class="NLM_article-title">Comparison of protein active site structures for functional annotation of proteins and drug design</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1002/prot.21092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.21092" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=124-135&issue=1&author=R.+Powersauthor=J.+C.+Copelandauthor=K.+Germerauthor=K.+A.+Mercierauthor=V.+Ramanathanauthor=P.+Revesz&title=Comparison+of+protein+active+site+structures+for+functional+annotation+of+proteins+and+drug+design&doi=10.1002%2Fprot.21092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fprot.21092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21092%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DJ.%2BC.%26aulast%3DGermer%26aufirst%3DK.%26aulast%3DMercier%26aufirst%3DK.%2BA.%26aulast%3DRamanathan%26aufirst%3DV.%26aulast%3DRevesz%26aufirst%3DP.%26atitle%3DComparison%2520of%2520protein%2520active%2520site%2520structures%2520for%2520functional%2520annotation%2520of%2520proteins%2520and%2520drug%2520design%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D65%26issue%3D1%26spage%3D124%26epage%3D135%26doi%3D10.1002%2Fprot.21092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Shortridge, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, R.</span><span> </span><span class="NLM_article-title">Structural and functional similarity between the bacterial type III secretion system needle protein PrgI and the eukaryotic apoptosis Bcl-2 proteins</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">e7442</span><span class="refDoi"> DOI: 10.1371/journal.pone.0007442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0007442" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e7442&issue=10&author=M.+D.+Shortridgeauthor=R.+Powers&title=Structural+and+functional+similarity+between+the+bacterial+type+III+secretion+system+needle+protein+PrgI+and+the+eukaryotic+apoptosis+Bcl-2+proteins&doi=10.1371%2Fjournal.pone.0007442"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0007442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0007442%26sid%3Dliteratum%253Aachs%26aulast%3DShortridge%26aufirst%3DM.%2BD.%26aulast%3DPowers%26aufirst%3DR.%26atitle%3DStructural%2520and%2520functional%2520similarity%2520between%2520the%2520bacterial%2520type%2520III%2520secretion%2520system%2520needle%2520protein%2520PrgI%2520and%2520the%2520eukaryotic%2520apoptosis%2520Bcl-2%2520proteins%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26issue%3D10%26spage%3De7442%26doi%3D10.1371%2Fjournal.pone.0007442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Kinoshita, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furui, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Identification of protein functions from a molecular surface database, eF-site</span> <span class="citation_source-journal">J. Struct. Funct. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">9</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1023/A:1011318527094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1023%2FA%3A1011318527094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=12836670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtFaqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=9-22&issue=1&author=K.+Kinoshitaauthor=J.+Furuiauthor=H.+Nakamura&title=Identification+of+protein+functions+from+a+molecular+surface+database%2C+eF-site&doi=10.1023%2FA%3A1011318527094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of protein functions from a molecular surface database, eF-site</span></div><div class="casAuthors">Kinoshita, Kengo; Furui, Jun'ichi; Nakamura, Haruki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural and Functional Genomics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-22</span>CODEN:
                <span class="NLM_cas:coden">JSFGAW</span>;
        ISSN:<span class="NLM_cas:issn">1345-711X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A bioinformatics method was developed to identify the protein surface around the functional site and to est. the biochem. function, using a newly constructed mol. surface database named the eF-site (electrostatic surface of Functional site.).  Mol. surfaces of protein mols. were computed based on the atom coordinates, and the eF-site database was prepd. by adding the phys. properties on the constructed mol. surfaces.  The electrostatic potential on each mol. surface was individually calcd. solving the Poisson-Boltzmann equation numerically for the precise continuum model, and the hydrophobicity information of each residue was also included.  The eF-site database is accessed by the internet.  We have prepd. four different databases, eF-site/antibody, eF-site/prosite, eF-site/P-site, and eF-site/ActiveSite, corresponding to the antigen binding sites of antibodies with the same orientations, the mol. surfaces for the individual motifs in PROSITE database, the phosphate binding sites, and the active site surfaces for the representatives of the individual protein family, resp.  An algorithm using the clique detection method as an applied graph theory was developed to search of the eF-site database, so as to recognize and discriminate the characteristic mol. surfaces of the proteins.  The method identifies the active site having the similar function to those of the known proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7LC-mYJpiR7Vg90H21EOLACvtfcHk0lgUIclMMGETZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtFaqsw%253D%253D&md5=8d7cb21e099b8f48ff35fb77225119ec</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1023%2FA%3A1011318527094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1011318527094%26sid%3Dliteratum%253Aachs%26aulast%3DKinoshita%26aufirst%3DK.%26aulast%3DFurui%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520protein%2520functions%2520from%2520a%2520molecular%2520surface%2520database%252C%2520eF-site%26jtitle%3DJ.%2520Struct.%2520Funct.%2520Genomics%26date%3D2002%26volume%3D2%26issue%3D1%26spage%3D9%26epage%3D22%26doi%3D10.1023%2FA%3A1011318527094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Kinoshita, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Identification of protein biochemical functions by similarity search using the molecular surface database eF-site</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1589</span><span class="NLM_x">–</span> <span class="NLM_lpage">1595</span><span class="refDoi"> DOI: 10.1110/ps.0368703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1110%2Fps.0368703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=12876308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvFSjtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=1589-1595&issue=8&author=K.+Kinoshitaauthor=H.+Nakamura&title=Identification+of+protein+biochemical+functions+by+similarity+search+using+the+molecular+surface+database+eF-site&doi=10.1110%2Fps.0368703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of protein biochemical functions by similarity search using the molecular surface database eF-site</span></div><div class="casAuthors">Kinoshita, Kengo; Nakamura, Haruki</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1589-1595</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The identification of protein biochem. functions based on their three-dimensional structures is strongly required in the post-genome-sequencing era.  We have developed a new method to identify and predict protein biochem. functions using the similarity information of mol. surface geometries and electrostatic potentials on the surfaces.  Our prediction system consists of a similarity search method based on a clique search algorithm and the mol. surface database eF-site (electrostatic surface of functional-site in proteins).  Using this system, functional sites similar to those of phosphoenoylpyruvate carboxy kinase were detected in several mononucleotide-binding proteins, which have different folds.  We also applied our method to a hypothetical protein, MJ0226 from Methanococcus jannaschii, and detected the mononucleotide binding site from the similarity to other proteins having different folds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojqDyJFOxO4bVg90H21EOLACvtfcHk0lhAQwows29jtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvFSjtb4%253D&md5=4bc8bd3eb49558b44273487d08680215</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1110%2Fps.0368703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.0368703%26sid%3Dliteratum%253Aachs%26aulast%3DKinoshita%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520protein%2520biochemical%2520functions%2520by%2520similarity%2520search%2520using%2520the%2520molecular%2520surface%2520database%2520eF-site%26jtitle%3DProtein%2520Sci.%26date%3D2003%26volume%3D12%26issue%3D8%26spage%3D1589%26epage%3D1595%26doi%3D10.1110%2Fps.0368703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Baroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciabola, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perruccio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span> </span><span class="NLM_article-title">A common reference framework for analyzing/comparing proteins and ligands. Fingerprints for Ligands and Proteins (FLAP): theory and application</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">279</span><span class="NLM_x">–</span> <span class="NLM_lpage">294</span><span class="refDoi"> DOI: 10.1021/ci600253e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600253e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtVagsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=279-294&issue=2&author=M.+Baroniauthor=G.+Crucianiauthor=S.+Sciabolaauthor=F.+Perruccioauthor=J.+S.+Mason&title=A+common+reference+framework+for+analyzing%2Fcomparing+proteins+and+ligands.+Fingerprints+for+Ligands+and+Proteins+%28FLAP%29%3A+theory+and+application&doi=10.1021%2Fci600253e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A common reference framework for analyzing/comparing proteins and ligands. Fingerprints for ligands and proteins (FLAP): theory and application</span></div><div class="casAuthors">Baroni, Massimo; Cruciani, Gabriele; Sciabola, Simone; Perruccio, Francesca; Mason, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-294</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast new algorithm (fingerprints for ligands and proteins or FLAP) able to describe small mols. and protein structures using a common ref. framework of four-point pharmacophore fingerprints and a mol.-cavity shape is described in detail.  The procedure starts by using the GRID force field to calc. mol. interaction fields, which are then used to identify particular target locations where an energetic interaction with small mol. features would be very favorable.  The target points thus calcd. are then used by FLAP to build all possible four-point pharmacophores present in the given target site.  A related approach can be applied to small mols., using directly the GRID atom types to identify pharmacophoric features, and this complementary description of the target and ligand then leads to several novel applications.  FLAP can be used for selectivity studies or similarity analyses in order to compare macromols. without superposing them.  Protein families can be compared and clustered into target classes, without bias from previous knowledge and without requiring protein superposition, alignment, or knowledge-based comparison.  FLAP can be used effectively for ligand-based virtual screening and structure-based virtual screening, with the pharmacophore mol. recognition.  Finally, the new method can calc. descriptors for chemometric anal. and can initiate a docking procedure.  This paper presents the background to the new procedure and includes case studies illustrating several relevant applications of the new approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppMc0fYDNn7LVg90H21EOLACvtfcHk0lhAQwows29jtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtVagsA%253D%253D&md5=9facb99c5741c7158afab7a3ac387ec3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci600253e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600253e%26sid%3Dliteratum%253Aachs%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DSciabola%26aufirst%3DS.%26aulast%3DPerruccio%26aufirst%3DF.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DA%2520common%2520reference%2520framework%2520for%2520analyzing%252Fcomparing%2520proteins%2520and%2520ligands.%2520Fingerprints%2520for%2520Ligands%2520and%2520Proteins%2520%2528FLAP%2529%253A%2520theory%2520and%2520application%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D2%26spage%3D279%26epage%3D294%26doi%3D10.1021%2Fci600253e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Sciabola, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flocco, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perruccio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span> </span><span class="NLM_article-title">High-throughput virtual screening of proteins using GRID molecular interaction fields</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">155</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1021/ci9003317</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9003317" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsValurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=155-169&issue=1&author=S.+Sciabolaauthor=R.+V.+Stantonauthor=J.+E.+Millsauthor=M.+M.+Floccoauthor=M.+Baroniauthor=G.+Crucianiauthor=F.+Perruccioauthor=J.+S.+Mason&title=High-throughput+virtual+screening+of+proteins+using+GRID+molecular+interaction+fields&doi=10.1021%2Fci9003317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening of Proteins Using GRID Molecular Interaction Fields</span></div><div class="casAuthors">Sciabola, Simone; Stanton, Robert V.; Mills, James E.; Flocco, Maria M.; Baroni, Massimo; Cruciani, Gabriele; Perruccio, Francesca; Mason, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">155-169</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new computational algorithm for protein binding sites characterization and comparison has been developed, which uses a common ref. framework of the projected ligand-space four-point pharmacophore fingerprints, includes cavity shape, and can be used with diverse proteins as no structural alignment is required.  Protein binding sites are first described using GRID mol. interaction fields (GRID-MIFs), and the FLAP (fingerprints for ligands and proteins) method is then used to encode and compare this information.  The discriminating power of the algorithm and its applicability for large-scale protein anal. was validated by analyzing various scenarios: clustering of kinase protein families in a relevant manner, predicting ligand activity across related targets, and protein-protein virtual screening.  In all cases the results showed the effectiveness of the GRID-FLAP method and its potential use in applications such as identifying selectivity targets and tools/hits for new targets via the identification of other proteins with pharmacophorically similar binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6BvC5pevsZ7Vg90H21EOLACvtfcHk0lhAQwows29jtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsValurvN&md5=f7b9f08b95e584728625eea3f308d305</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fci9003317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003317%26sid%3Dliteratum%253Aachs%26aulast%3DSciabola%26aufirst%3DS.%26aulast%3DStanton%26aufirst%3DR.%2BV.%26aulast%3DMills%26aufirst%3DJ.%2BE.%26aulast%3DFlocco%26aufirst%3DM.%2BM.%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DPerruccio%26aufirst%3DF.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520of%2520proteins%2520using%2520GRID%2520molecular%2520interaction%2520fields%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D1%26spage%3D155%26epage%3D169%26doi%3D10.1021%2Fci9003317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Tseng, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.-H.</span><span> </span><span class="NLM_article-title">fPOP: footprinting functional pockets of proteins by comparative spatial patterns</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D288</span><span class="NLM_x">–</span> <span class="NLM_lpage">D295</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkp900</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkp900" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D288-D295&issue=Suppl.+1&author=Y.+Y.+Tsengauthor=Z.+J.+Chenauthor=W.-H.+Li&title=fPOP%3A+footprinting+functional+pockets+of+proteins+by+comparative+spatial+patterns&doi=10.1093%2Fnar%2Fgkp900"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp900%26sid%3Dliteratum%253Aachs%26aulast%3DTseng%26aufirst%3DY.%2BY.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26aulast%3DLi%26aufirst%3DW.-H.%26atitle%3DfPOP%253A%2520footprinting%2520functional%2520pockets%2520of%2520proteins%2520by%2520comparative%2520spatial%2520patterns%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26issue%3DSuppl.%25201%26spage%3DD288%26epage%3DD295%26doi%3D10.1093%2Fnar%2Fgkp900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Tseng, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.-H.</span><span> </span><span class="NLM_article-title">Classification of protein functional surfaces using structural characteristics</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1170</span><span class="NLM_x">–</span> <span class="NLM_lpage">1175</span><span class="refDoi"> DOI: 10.1073/pnas.1119684109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1073%2Fpnas.1119684109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=1170-1175&issue=4&author=Y.+Y.+Tsengauthor=W.-H.+Li&title=Classification+of+protein+functional+surfaces+using+structural+characteristics&doi=10.1073%2Fpnas.1119684109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1119684109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1119684109%26sid%3Dliteratum%253Aachs%26aulast%3DTseng%26aufirst%3DY.%2BY.%26aulast%3DLi%26aufirst%3DW.-H.%26atitle%3DClassification%2520of%2520protein%2520functional%2520surfaces%2520using%2520structural%2520characteristics%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D4%26spage%3D1170%26epage%3D1175%26doi%3D10.1073%2Fpnas.1119684109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">207</span><span class="refDoi"> DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&issue=1&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0lj5YubNa-by6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">Structure-based prediction of G-protein-coupled receptor ligand function: a β-adrenoceptor case study</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1045</span><span class="NLM_x">–</span> <span class="NLM_lpage">1061</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFenuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1045-1061&issue=5&author=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+P.+de+Eschauthor=C.+de+Graaf&title=Structure-based+prediction+of+G-protein-coupled+receptor+ligand+function%3A+a+%CE%B2-adrenoceptor+case+study&doi=10.1021%2Facs.jcim.5b00066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based prediction of G-protein-coupled receptor ligand function: A β-adrenoceptor case study</span></div><div class="casAuthors">Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1045-1061</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The spectacular advances in G-protein-coupled receptor (GPCR) structure detn. have opened up new possibilities for structure-based GPCR ligand discovery.  The structure-based prediction of whether a ligand stimulates (full/partial agonist), blocks (antagonist), or reduces (inverse agonist) GPCR signaling activity is, however, still challenging.  A total of 31 β1 (β1R) and β2 (β2R) adrenoceptor crystal structures, including antagonist, inverse agonist, and partial/full agonist-bound structures, allowed the authors to explore the possibilities and limitations of structure-based prediction of GPCR ligand function.  The authors used all unique protein-ligand interaction fingerprints (IFPs) derived from all ligand-bound β-adrenergic crystal structure monomers to post-process the docking poses of known β1R/β2R partial/full agonists, antagonists/inverse agonists, and physicochem. similar decoys in each of the β1R/β2R structures.  The systematic anal. of these 1920 unique IFP-structure combinations offered new insights into the relative impact of protein conformation and IFP scoring on selective virtual screening (VS) for ligands with a specific functional effect.  The authors' studies show that ligands with the same function can be efficiently classified on the basis of their protein-ligand interaction profile.  Small differences between the receptor conformation (used for docking) and ref. IFP (used for scoring of the docking poses) det., however, the enrichment of specific ligand types in VS hit lists.  Interestingly, the selective enrichment of partial/full agonists can be achieved by using agonist IFPs to post-process docking poses in agonist-bound as well as antagonist-bound structures.  The authors have identified optimal structure-IFP combinations for the identification and discrimination of antagonists/inverse agonist and partial/full agonists, and defined a predicted IFP for the small full agonist norepinephrine that gave the highest retrieval rate of agonists over antagonists for all structures (with an enrichment factor of 46 for agonists and 8 for antagonists on av. at a 1% false-pos. rate).  This β-adrenoceptor case study provides new insights into the opportunities for selective structure-based discovery of GPCR ligands with a desired function and emphasizes the importance of IFPs in scoring docking poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV5NA6pChaObVg90H21EOLACvtfcHk0lj5YubNa-by6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFenuro%253D&md5=cc51b89775d0e5443d193b2527741bca</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00066%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DStructure-based%2520prediction%2520of%2520G-protein-coupled%2520receptor%2520ligand%2520function%253A%2520a%2520%25CE%25B2-adrenoceptor%2520case%2520study%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D5%26spage%3D1045%26epage%3D1061%26doi%3D10.1021%2Facs.jcim.5b00066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Najmanovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurbatova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J.</span><span> </span><span class="NLM_article-title">Detection of 3D atomic similarities and their use in the discrimination of small molecule protein-binding sites</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x"> (</span><span class="NLM_issue">16</span><span class="NLM_x">) </span> <span class="NLM_fpage">i105</span><span class="NLM_x">–</span> <span class="NLM_lpage">i111</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btn263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtn263" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=i105-i111&issue=16&author=R.+Najmanovichauthor=N.+Kurbatovaauthor=J.+Thornton&title=Detection+of+3D+atomic+similarities+and+their+use+in+the+discrimination+of+small+molecule+protein-binding+sites&doi=10.1093%2Fbioinformatics%2Fbtn263"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtn263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtn263%26sid%3Dliteratum%253Aachs%26aulast%3DNajmanovich%26aufirst%3DR.%26aulast%3DKurbatova%26aufirst%3DN.%26aulast%3DThornton%26aufirst%3DJ.%26atitle%3DDetection%2520of%25203D%2520atomic%2520similarities%2520and%2520their%2520use%2520in%2520the%2520discrimination%2520of%2520small%2520molecule%2520protein-binding%2520sites%26jtitle%3DBioinformatics%26date%3D2008%26volume%3D24%26issue%3D16%26spage%3Di105%26epage%3Di111%26doi%3D10.1093%2Fbioinformatics%2Fbtn263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Larocque, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chénard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R. A.</span><span> </span><span class="NLM_article-title">A curated C. difficile strain 630 metabolic network: prediction of essential targets and inhibitors</span> <span class="citation_source-journal">BMC Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">117</span><span class="refDoi"> DOI: 10.1186/s12918-014-0117-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2Fs12918-014-0117-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25315994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFWhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=117&author=M.+Larocqueauthor=T.+Ch%C3%A9nardauthor=R.+A.+Najmanovich&title=A+curated+C.+difficile+strain+630+metabolic+network%3A+prediction+of+essential+targets+and+inhibitors&doi=10.1186%2Fs12918-014-0117-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">A curated C. difficile strain 630 metabolic network: prediction of essential targets and inhibitors</span></div><div class="casAuthors">Larocque, Mathieu; Chenard, Thierry; Najmanovich, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">BMC Systems Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">117/1-117/17</span>CODEN:
                <span class="NLM_cas:coden">BSBMCC</span>;
        ISSN:<span class="NLM_cas:issn">1752-0509</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background:Clostridium difficile is the leading cause of hospital-borne infections occurring when the natural intestinal flora is depleted following antibiotic treatment.  Current treatments for Clostridium difficile infections present high relapse rates and new hyper-virulent and multi-resistant strains are emerging, making the study of this nosocomial pathogen necessary to find novel therapeutic targets.  Results: We present iMLTC806cdf, an extensively curated reconstructed metabolic network for the C. difficile pathogenic strain 630. iMLTC806cdf contains 806 genes, 703 metabolites and 769 metabolic, 117 exchange and 145 transport reactions. iMLTC806cdf is the most complete and accurate metabolic reconstruction of a gram-pos. anaerobic bacteria to date.  We validate the model with simulated growth assays in different media and carbon sources and use it to predict essential genes.  We obtain 89.2% accuracy in the prediction of gene essentiality when compared to exptl. data for B. subtilis homologs (the closest organism for which such data exists).  We predict the existence of 76 essential genes and 39 essential gene pairs, a no. of which are unique to C. difficile and have non-existing or predicted non-essential human homologs.  For 29 of these potential therapeutic targets, we find 125 inhibitors of homologous proteins including approved drugs with the potential for drug repositioning, that when validated exptl. could serve as starting points in the development of new antibiotics.  Conclusions: We created a highly curated metabolic network model of C. difficile strain 630 and used it to predict essential genes as potential new therapeutic targets in the fight against Clostridium difficile infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1JjHlRdFNaLVg90H21EOLACvtfcHk0lh56m6vRc6XOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFWhsA%253D%253D&md5=83a0d4ccc76523a40ecbcf2399792541</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1186%2Fs12918-014-0117-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12918-014-0117-z%26sid%3Dliteratum%253Aachs%26aulast%3DLarocque%26aufirst%3DM.%26aulast%3DCh%25C3%25A9nard%26aufirst%3DT.%26aulast%3DNajmanovich%26aufirst%3DR.%2BA.%26atitle%3DA%2520curated%2520C.%2520difficile%2520strain%2520630%2520metabolic%2520network%253A%2520prediction%2520of%2520essential%2520targets%2520and%2520inhibitors%26jtitle%3DBMC%2520Syst.%2520Biol.%26date%3D2014%26volume%3D8%26spage%3D117%26doi%3D10.1186%2Fs12918-014-0117-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Wood, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vlieg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagener, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritschel, T.</span><span> </span><span class="NLM_article-title">Pharmacophore fingerprint-based approach to binding site subpocket similarity and its application to bioisostere replacement</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">2031</span><span class="NLM_x">–</span> <span class="NLM_lpage">2043</span><span class="refDoi"> DOI: 10.1021/ci3000776</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3000776" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVylur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2031-2043&issue=8&author=D.+J.+Woodauthor=J.+de+Vliegauthor=M.+Wagenerauthor=T.+Ritschel&title=Pharmacophore+fingerprint-based+approach+to+binding+site+subpocket+similarity+and+its+application+to+bioisostere+replacement&doi=10.1021%2Fci3000776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore Fingerprint-Based Approach to Binding Site Subpocket Similarity and Its Application to Bioisostere Replacement</span></div><div class="casAuthors">Wood, David J.; Vlieg, Jacob de; Wagener, Markus; Ritschel, Tina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2031-2043</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bioisosteres have been defined as structurally different mols. or substructures that can form comparable intermol. interactions, and therefore, fragments that bind to similar protein structures exhibit a degree of bioisosterism.  We present KRIPO (Key Representation of Interaction in POckets): a new method for quantifying the similarities of binding site subpockets based on pharmacophore fingerprints.  The binding site fingerprints have been optimized to improve their performance for both intra- and interprotein family comparisons.  A range of attributes of the fingerprints was considered in the optimization, including the placement of pharmacophore features, whether or not the fingerprints are fuzzified, and the resoln. and complexity of the pharmacophore fingerprints (2-, 3-, and 4-point fingerprints).  Fuzzy 3-point pharmacophore fingerprints were found to represent the optimal balance between computational resource requirements and the identification of potential replacements.  The complete PDB was converted into a database comprising almost 300 000 optimized fingerprints of local binding sites together with their assocd. ligand fragments.  The value of the approach is demonstrated by application to two crystal structures from the Protein Data Bank: (1) a MAP kinase P38 structure in complex with a pyridinylimidazole inhibitor (1A9U) and (2) a complex of thrombin with melagatran (1K22).  Potentially valuable bioisosteric replacements for all subpockets of the two studied protein are identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRQTi8-dp_rVg90H21EOLACvtfcHk0lhuZq3ruowQYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVylur%252FF&md5=f344aa0423546f08ea347327f56ef7cb</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fci3000776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3000776%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DD.%2BJ.%26aulast%3Dde%2BVlieg%26aufirst%3DJ.%26aulast%3DWagener%26aufirst%3DM.%26aulast%3DRitschel%26aufirst%3DT.%26atitle%3DPharmacophore%2520fingerprint-based%2520approach%2520to%2520binding%2520site%2520subpocket%2520similarity%2520and%2520its%2520application%2520to%2520bioisostere%2520replacement%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D8%26spage%3D2031%26epage%3D2043%26doi%3D10.1021%2Fci3000776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Schirris, T. J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritschel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herma Renkema, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willems</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peter, H. G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smeitink, J. A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russel, F. G. M.</span><span> </span><span class="NLM_article-title">Mitochondrial ADP/ATP exchange inhibition: a novel off-target mechanism underlying ibipinabant-induced myotoxicity</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">14533</span><span class="refDoi"> DOI: 10.1038/srep14533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fsrep14533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=26416158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFOrsrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=14533&author=T.+J.+J.+Schirrisauthor=T.+Ritschelauthor=G.+Herma+Renkemaauthor=+Willemsauthor=H.+G.+M.+Peterauthor=J.+A.+M.+Smeitinkauthor=F.+G.+M.+Russel&title=Mitochondrial+ADP%2FATP+exchange+inhibition%3A+a+novel+off-target+mechanism+underlying+ibipinabant-induced+myotoxicity&doi=10.1038%2Fsrep14533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Mitochondrial ADP/ATP exchange inhibition: a novel off-target mechanism underlying ibipinabant-induced myotoxicity</span></div><div class="casAuthors">Schirris, Tom J. J.; Ritschel, Tina; Renkema, G. Herma; Willems, Peter H. G. M.; Smeitink, Jan A. M.; Russel, Frans G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14533</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cannabinoid receptor 1 (CB1R) antagonists appear to be promising drugs for the treatment of obesity, however, serious side effects have hampered their clin. application.  Rimonabant, the first in class CB1R antagonist, was withdrawn from the market because of psychiatric side effects.  This has led to the search for more peripherally restricted CB1R antagonists, one of which is ibipinabant.  However, this 3,4-diarylpyrazoline deriv. showed muscle toxicity in a pre-clin. dog study with mitochondrial dysfunction.  Here, we studied the mol. mechanism by which ibipinabant induces mitochondrial toxicity.  We obsd. a strong cytotoxic potency of ibipinabant in C2C12 myoblasts.  Functional characterization of mitochondria revealed increased cellular reactive oxygen species generation and a decreased ATP prodn. capacity, without effects on the catalytic activities of mitochondrial enzyme complexes I-V or the complex specific-driven oxygen consumption.  Using in silico off-target prediction modeling, combined with in vitro validation in isolated mitochondria and mitoplasts, we identified adenine nucleotide translocase (ANT)-dependent mitochondrial ADP/ATP exchange as a novel mol. mechanism underlying ibipinabant-induced toxicity.  Minor structural modification of ibipinabant could abolish ANT inhibition leading to a decreased cytotoxic potency, as obsd. with the ibipinabant deriv. CB23.  Our results will be instrumental in the development of new types of safer CB1R antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF2M9vQEdYyrVg90H21EOLACvtfcHk0lhuZq3ruowQYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFOrsrrM&md5=ac01a690f1c65777e537dc597ad1e46f</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fsrep14533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep14533%26sid%3Dliteratum%253Aachs%26aulast%3DSchirris%26aufirst%3DT.%2BJ.%2BJ.%26aulast%3DRitschel%26aufirst%3DT.%26aulast%3DHerma%2BRenkema%26aufirst%3DG.%26aulast%3DWillems%26aulast%3DPeter%26aufirst%3DH.%2BG.%2BM.%26aulast%3DSmeitink%26aufirst%3DJ.%2BA.%2BM.%26aulast%3DRussel%26aufirst%3DF.%2BG.%2BM.%26atitle%3DMitochondrial%2520ADP%252FATP%2520exchange%2520inhibition%253A%2520a%2520novel%2520off-target%2520mechanism%2520underlying%2520ibipinabant-induced%2520myotoxicity%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D14533%26doi%3D10.1038%2Fsrep14533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Brakoulias, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Towards a structural classification of phosphate binding sites in protein-nucleotide complexes: an automated all-against-all structural comparison using geometric matching</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">250</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1002/prot.20123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.20123" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2004&pages=250-260&issue=2&author=A.+Brakouliasauthor=R.+M.+Jackson&title=Towards+a+structural+classification+of+phosphate+binding+sites+in+protein-nucleotide+complexes%3A+an+automated+all-against-all+structural+comparison+using+geometric+matching&doi=10.1002%2Fprot.20123"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1002%2Fprot.20123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20123%26sid%3Dliteratum%253Aachs%26aulast%3DBrakoulias%26aufirst%3DA.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DTowards%2520a%2520structural%2520classification%2520of%2520phosphate%2520binding%2520sites%2520in%2520protein-nucleotide%2520complexes%253A%2520an%2520automated%2520all-against-all%2520structural%2520comparison%2520using%2520geometric%2520matching%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D56%26issue%3D2%26spage%3D250%26epage%3D260%26doi%3D10.1002%2Fprot.20123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Binding site similarity analysis for the functional classification of the protein kinase family</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">318</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1021/ci800289y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800289y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=318-329&issue=2&author=S.+L.+Kinningsauthor=R.+M.+Jackson&title=Binding+site+similarity+analysis+for+the+functional+classification+of+the+protein+kinase+family&doi=10.1021%2Fci800289y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Similarity Analysis for the Functional Classification of the Protein Kinase Family</span></div><div class="casAuthors">Kinnings, Sarah L.; Jackson, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">318-329</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methods for analyzing complete gene families are becoming of increasing importance to the drug discovery process, because similarities and differences within a family are often the key to understanding functional differences that can be exploited in drug design.  We undertake a large-scale structural comparison of protein kinase ATP-binding sites using a geometric hashing method.  Subsequently, we propose a relevant classification of the protein kinase family based on the structural similarity of its binding sites.  Our classification is not only able to reveal the great diversity of different protein kinases and therefore their different potential for inhibitor selectivity, but it is also able to distinguish subtle differences within binding site conformation reflecting the protein activation state.  Furthermore, using exptl. inhibition profiling, we demonstrate that our classification can be used to identify protein kinase binding sites that are known exptl. to bind the same drug, demonstrating that it has potential as an inverse (protein) virtual screening tool, by identifying which other sites have the potential to bind a given drug.  In this way the cross-reactivities of the anticancer drugs Tarceva and Gleevec are rationalized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6kpf80r8s5LVg90H21EOLACvtfcHk0lhuZq3ruowQYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltF2qsg%253D%253D&md5=742343601ca1e3c0549fa4f383c2566b</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fci800289y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800289y%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DBinding%2520site%2520similarity%2520analysis%2520for%2520the%2520functional%2520classification%2520of%2520the%2520protein%2520kinase%2520family%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26issue%3D2%26spage%3D318%26epage%3D329%26doi%3D10.1021%2Fci800289y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Wu, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C.</span><span> </span><span class="NLM_article-title">A structural-alphabet-based strategy for finding structural motifs across protein families</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">e150</span><span class="refDoi"> DOI: 10.1093/nar/gkq478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkq478" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=e150&issue=14&author=C.+Y.+Wuauthor=Y.+C.+Chenauthor=C.+Lim&title=A+structural-alphabet-based+strategy+for+finding+structural+motifs+across+protein+families&doi=10.1093%2Fnar%2Fgkq478"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq478%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DC.%2BY.%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DLim%26aufirst%3DC.%26atitle%3DA%2520structural-alphabet-based%2520strategy%2520for%2520finding%2520structural%2520motifs%2520across%2520protein%2520families%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26issue%3D14%26spage%3De150%26doi%3D10.1093%2Fnar%2Fgkq478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Hua, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sargsyan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C.</span><span> </span><span class="NLM_article-title">Sequence-motif detection of NAD(P)-binding proteins: discovery of a unique antibacterial drug target</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">6471</span><span class="refDoi"> DOI: 10.1038/srep06471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fsrep06471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25253464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVygu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=6471&author=Y.+H.+Huaauthor=C.+Y.+Wuauthor=K.+Sargsyanauthor=C.+Lim&title=Sequence-motif+detection+of+NAD%28P%29-binding+proteins%3A+discovery+of+a+unique+antibacterial+drug+target&doi=10.1038%2Fsrep06471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-motif Detection of NAD(P)-binding Proteins: Discovery of a Unique Antibacterial Drug Target</span></div><div class="casAuthors">Hua, Yun Hao; Wu, Chih Yuan; Sargsyan, Karen; Lim, Carmay</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6471</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many enzymes use NAD or NADP (NAD(P)) as essential coenzymes.  These enzymes often do not share significant sequence identity and cannot be easily detected by sequence homol.  Previously, we detd. all distinct locally conserved pyrophosphate-binding structures (3d motifs) from NAD(P)-bound protein structures, from which 1d sequence motifs were derived.  Here, we aim to establish the precision of these 3d and 1d motifs to annotate NAD(P)-binding proteins.  We show that the pyrophosphate-binding 3d motifs are characteristic of NAD(P)-binding proteins, as they are rarely found in nonNAD(P)-binding proteins.  Furthermore, several 1d motifs could distinguish between proteins that bind only NAD and those that bind only NADP.  They could also distinguish between NAD(P)-binding proteins from nonNAD(P)-binding ones.  Interestingly, one of the pyrophosphate-binding 3d and corresponding 1d motifs was found only in enoyl-acyl carrier protein reductases, which are enzymes essential for bacterial fatty acid biosynthesis.  This unique 3d motif serves as an attractive novel drug target, as it is conserved across many bacterial species and is not found in human proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwcv4XrSagTbVg90H21EOLACvtfcHk0lghjBEBwjviwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVygu7g%253D&md5=088f74d56a30ab954eb4636482198031</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fsrep06471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep06471%26sid%3Dliteratum%253Aachs%26aulast%3DHua%26aufirst%3DY.%2BH.%26aulast%3DWu%26aufirst%3DC.%2BY.%26aulast%3DSargsyan%26aufirst%3DK.%26aulast%3DLim%26aufirst%3DC.%26atitle%3DSequence-motif%2520detection%2520of%2520NAD%2528P%2529-binding%2520proteins%253A%2520discovery%2520of%2520a%2520unique%2520antibacterial%2520drug%2520target%26jtitle%3DSci.%2520Rep.%26date%3D2014%26volume%3D4%26spage%3D6471%26doi%3D10.1038%2Fsrep06471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Das, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kokardekar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breneman, C. M.</span><span> </span><span class="NLM_article-title">Rapid comparison of protein binding site surfaces with property encoded shape distributions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2863</span><span class="NLM_x">–</span> <span class="NLM_lpage">2872</span><span class="refDoi"> DOI: 10.1021/ci900317x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900317x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsValtLnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2863-2872&issue=12&author=S.+Dasauthor=A.+Kokardekarauthor=C.+M.+Breneman&title=Rapid+comparison+of+protein+binding+site+surfaces+with+property+encoded+shape+distributions&doi=10.1021%2Fci900317x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Comparison of Protein Binding Site Surfaces with Property Encoded Shape Distributions</span></div><div class="casAuthors">Das, Sourav; Kokardekar, Arshad; Breneman, Curt M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2863-2872</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Patterns in shape and property distributions on the surface of binding sites are often conserved across functional proteins without significant conservation of the underlying amino-acid residues.  To explore similarities of these sites from the viewpoint of a ligand, a sequence and fold-independent method was created to rapidly and accurately compare binding sites of proteins represented by property-mapped triangulated Gauss-Connolly surfaces.  Within this paradigm, signatures for each binding site surface are produced by calcg. their property-encoded shape distributions (PESD), a measure of the probability that a particular property will be at a specific distance to another on the mol. surface.  Similarity between the signatures can then be treated as a measure of similarity between binding sites.  As postulated, the PESD method rapidly detected high levels of similarity in binding site surface characteristics even in cases where there was very low similarity at the sequence level.  In a screening expt. involving each member of the PDBBind 2005 data set as a query against the rest of the set, PESD was able to retrieve a binding site with identical E.C. (Enzyme Commission) nos. as the top match in 79.5% of cases.  The ability of the method in detecting similarity in binding sites with low sequence conservations was compared to state-of-the-art binding site comparison methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlwxGcrBNQpLVg90H21EOLACvtfcHk0lghjBEBwjviwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsValtLnK&md5=7793378407753379d12ebfe70e05a1c1</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fci900317x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900317x%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DS.%26aulast%3DKokardekar%26aufirst%3DA.%26aulast%3DBreneman%26aufirst%3DC.%2BM.%26atitle%3DRapid%2520comparison%2520of%2520protein%2520binding%2520site%2520surfaces%2520with%2520property%2520encoded%2520shape%2520distributions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26issue%3D12%26spage%3D2863%26epage%3D2872%26doi%3D10.1021%2Fci900317x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Das, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krein, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breneman, C. M.</span><span> </span><span class="NLM_article-title">Binding affinity prediction with property-encoded shape distribution signatures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">298</span><span class="NLM_x">–</span> <span class="NLM_lpage">308</span><span class="refDoi"> DOI: 10.1021/ci9004139</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9004139" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVagsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=298-308&issue=2&author=S.+Dasauthor=M.+P.+Kreinauthor=C.+M.+Breneman&title=Binding+affinity+prediction+with+property-encoded+shape+distribution+signatures&doi=10.1021%2Fci9004139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Affinity Prediction with Property-Encoded Shape Distribution Signatures</span></div><div class="casAuthors">Das, Sourav; Krein, Michael P.; Breneman, Curt M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">298-308</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the use of the mol. signatures known as "property-encoded shape distributions" (PESD) together with std. support vector machine (SVM) techniques to produce validated models that can predict the binding affinity of a large no. of protein ligand complexes.  This "PESD-SVM" method uses PESD signatures that encode mol. shapes and property distributions on protein and ligand surfaces as features to build SVM models that require no subjective feature selection.  A simple protocol was employed for tuning the SVM models during their development, and the results were compared to SFCscore, a regression-based method that was previously shown to perform better than 14 other scoring functions.  Although the PESD-SVM method is based on only two surface property maps, the overall results were comparable.  For most complexes with a dominant enthalpic contribution to binding (ΔH/-TΔS >3), a good correlation between true and predicted affinities was obsd.  Entropy and solvent were not considered in the present approach, and further improvement in accuracy would require accounting for these components rigorously.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd5pJea-jHILVg90H21EOLACvtfcHk0lghjBEBwjviwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVagsQ%253D%253D&md5=ad40d31b520e68d6b9cb9fa903bcd209</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fci9004139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9004139%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DS.%26aulast%3DKrein%26aufirst%3DM.%2BP.%26aulast%3DBreneman%26aufirst%3DC.%2BM.%26atitle%3DBinding%2520affinity%2520prediction%2520with%2520property-encoded%2520shape%2520distribution%2520signatures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D2%26spage%3D298%26epage%3D308%26doi%3D10.1021%2Fci9004139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, R. B.</span><span> </span><span class="NLM_article-title">Using multiple microenvironments to find similar ligand-binding sites: application to kinase inhibitor binding</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002326</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1002326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1002326" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002326&issue=12&author=T.+Liuauthor=R.+B.+Altman&title=Using+multiple+microenvironments+to+find+similar+ligand-binding+sites%3A+application+to+kinase+inhibitor+binding&doi=10.1371%2Fjournal.pcbi.1002326"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002326%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DAltman%26aufirst%3DR.%2BB.%26atitle%3DUsing%2520multiple%2520microenvironments%2520to%2520find%2520similar%2520ligand-binding%2520sites%253A%2520application%2520to%2520kinase%2520inhibitor%2520binding%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D12%26spage%3De1002326%26doi%3D10.1371%2Fjournal.pcbi.1002326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Miguel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K. C.</span><span> </span><span class="NLM_article-title">Variations in the binding pocket of an inhibitor of the bacterial division protein FtsZ across genotypes and species</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1004117</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1004117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1004117" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=e1004117&issue=3&author=A.+Miguelauthor=J.+Hsinauthor=T.+Liuauthor=G.+Tangauthor=R.+B.+Altmanauthor=K.+C.+Huang&title=Variations+in+the+binding+pocket+of+an+inhibitor+of+the+bacterial+division+protein+FtsZ+across+genotypes+and+species&doi=10.1371%2Fjournal.pcbi.1004117"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004117%26sid%3Dliteratum%253Aachs%26aulast%3DMiguel%26aufirst%3DA.%26aulast%3DHsin%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DTang%26aufirst%3DG.%26aulast%3DAltman%26aufirst%3DR.%2BB.%26aulast%3DHuang%26aufirst%3DK.%2BC.%26atitle%3DVariations%2520in%2520the%2520binding%2520pocket%2520of%2520an%2520inhibitor%2520of%2520the%2520bacterial%2520division%2520protein%2520FtsZ%2520across%2520genotypes%2520and%2520species%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D3%26spage%3De1004117%26doi%3D10.1371%2Fjournal.pcbi.1004117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Yeturu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandra, N.</span><span> </span><span class="NLM_article-title">PocketMatch: a new algorithm to compare binding sites in protein structures</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">543</span><span class="refDoi"> DOI: 10.1186/1471-2105-9-543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-9-543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=19091072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BD1M7jtVKksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=543&author=K.+Yeturuauthor=N.+Chandra&title=PocketMatch%3A+a+new+algorithm+to+compare+binding+sites+in+protein+structures&doi=10.1186%2F1471-2105-9-543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">PocketMatch: a new algorithm to compare binding sites in protein structures</span></div><div class="casAuthors">Yeturu Kalidas; Chandra Nagasuma</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">543</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Recognizing similarities and deriving relationships among protein molecules is a fundamental requirement in present-day biology.  Similarities can be present at various levels which can be detected through comparison of protein sequences or their structural folds.  In some cases similarities obscure at these levels could be present merely in the substructures at their binding sites.  Inferring functional similarities between protein molecules by comparing their binding sites is still largely exploratory and not as yet a routine protocol.  One of the main reasons for this is the limitation in the choice of appropriate analytical tools that can compare binding sites with high sensitivity.  To benefit from the enormous amount of structural data that is being rapidly accumulated, it is essential to have high throughput tools that enable large scale binding site comparison.  RESULTS:  Here we present a new algorithm PocketMatch for comparison of binding sites in a frame invariant manner.  Each binding site is represented by 90 lists of sorted distances capturing shape and chemical nature of the site.  The sorted arrays are then aligned using an incremental alignment method and scored to obtain PMScores for pairs of sites.  A comprehensive sensitivity analysis and an extensive validation of the algorithm have been carried out.  A comparison with other site matching algorithms is also presented.  Perturbation studies where the geometry of a given site was retained but the residue types were changed randomly, indicated that chance similarities were virtually non-existent.  Our analysis also demonstrates that shape information alone is insufficient to discriminate between diverse binding sites, unless combined with chemical nature of amino acids.  CONCLUSION:  A new algorithm has been developed to compare binding sites in accurate, efficient and high-throughput manner.  Though the representation used is conceptually simplistic, we demonstrate that along with the new alignment strategy used, it is sufficient to enable binding comparison with high sensitivity.  Novel methodology has also been presented for validating the algorithm for accuracy and sensitivity with respect to geometry and chemical nature of the site.  The method is also fast and takes about 1/250th second for one comparison on a single processor.  A parallel version on BlueGene has also been implemented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQG4iooRU0PkP9pSBfyDvp7fW6udTcc2ebjuqOZgRcI27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M7jtVKksg%253D%253D&md5=b5f9edf7278e1d6f5dca62178c8c1a03</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-543%26sid%3Dliteratum%253Aachs%26aulast%3DYeturu%26aufirst%3DK.%26aulast%3DChandra%26aufirst%3DN.%26atitle%3DPocketMatch%253A%2520a%2520new%2520algorithm%2520to%2520compare%2520binding%2520sites%2520in%2520protein%2520structures%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26spage%3D543%26doi%3D10.1186%2F1471-2105-9-543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Anand, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sankaran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeturu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhardwaj, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhagavat, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brahmachari, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandra, N.</span><span> </span><span class="NLM_article-title">Structural annotation of Mycobacterium tuberculosis proteome</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">e27044</span><span class="refDoi"> DOI: 10.1371/journal.pone.0027044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0027044" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e27044&issue=10&author=P.+Anandauthor=S.+Sankaranauthor=S.+Mukherjeeauthor=K.+Yeturuauthor=R.+Laskowskiauthor=A.+Bhardwajauthor=R.+Bhagavatauthor=S.+K.+Brahmachariauthor=N.+Chandra&title=Structural+annotation+of+Mycobacterium+tuberculosis+proteome&doi=10.1371%2Fjournal.pone.0027044"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027044%26sid%3Dliteratum%253Aachs%26aulast%3DAnand%26aufirst%3DP.%26aulast%3DSankaran%26aufirst%3DS.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DYeturu%26aufirst%3DK.%26aulast%3DLaskowski%26aufirst%3DR.%26aulast%3DBhardwaj%26aufirst%3DA.%26aulast%3DBhagavat%26aufirst%3DR.%26aulast%3DBrahmachari%26aufirst%3DS.%2BK.%26aulast%3DChandra%26aufirst%3DN.%26atitle%3DStructural%2520annotation%2520of%2520Mycobacterium%2520tuberculosis%2520proteome%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26issue%3D10%26spage%3De27044%26doi%3D10.1371%2Fjournal.pone.0027044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Möller-Acuña, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contreras-Riquelme, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rojas-Fuentes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuñez-Vivanco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alzate-Morales, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iturriaga-Vásquez, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arias, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reyes-Parada, M.</span><span> </span><span class="NLM_article-title">Similarities between the binding sites of SB-206553 at serotonin type 2 and alpha7 acetylcholine nicotinic receptors: rationale for its polypharmacological profile</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e0134444</span><span class="refDoi"> DOI: 10.1371/journal.pone.0134444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0134444" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0134444&issue=8&author=P.+M%C3%B6ller-Acu%C3%B1aauthor=J.+S.+Contreras-Riquelmeauthor=C.+Rojas-Fuentesauthor=G.+Nu%C3%B1ez-Vivancoauthor=J.+Alzate-Moralesauthor=P.+Iturriaga-V%C3%A1squezauthor=H.+R.+Ariasauthor=M.+Reyes-Parada&title=Similarities+between+the+binding+sites+of+SB-206553+at+serotonin+type+2+and+alpha7+acetylcholine+nicotinic+receptors%3A+rationale+for+its+polypharmacological+profile&doi=10.1371%2Fjournal.pone.0134444"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0134444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0134444%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25B6ller-Acu%25C3%25B1a%26aufirst%3DP.%26aulast%3DContreras-Riquelme%26aufirst%3DJ.%2BS.%26aulast%3DRojas-Fuentes%26aufirst%3DC.%26aulast%3DNu%25C3%25B1ez-Vivanco%26aufirst%3DG.%26aulast%3DAlzate-Morales%26aufirst%3DJ.%26aulast%3DIturriaga-V%25C3%25A1squez%26aufirst%3DP.%26aulast%3DArias%26aufirst%3DH.%2BR.%26aulast%3DReyes-Parada%26aufirst%3DM.%26atitle%3DSimilarities%2520between%2520the%2520binding%2520sites%2520of%2520SB-206553%2520at%2520serotonin%2520type%25202%2520and%2520alpha7%2520acetylcholine%2520nicotinic%2520receptors%253A%2520rationale%2520for%2520its%2520polypharmacological%2520profile%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3De0134444%26doi%3D10.1371%2Fjournal.pone.0134444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Fierro, A.; Montecinos, A.; Gmez-Molina, C.; Nez, G.; Aldeco, M.; Edmondson, D. E.; Vilches Herrera, M.; Luhr, S.; Iturriaga-Vsquez, P.; Reyes Par, M.</span><span> </span><span class="NLM_article-title">Similarities between the binding sites of monoamine oxidase (MAO) from different species—Is zebrafish a useful model for the discovery of novel MAO inhibitors?</span> In  <span class="citation_source-book">An Integrated View of the Molecular Recognition and Toxinology: From Analytical Procedures to Biomedical Applications</span>; <span class="NLM_contrib-group">Radis-Baptista, G.</span>, Ed.; <span class="NLM_publisher-name">InTech</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>DOI: <span class="refDoi"> DOI: 10.5772/35874</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.5772%2F35874" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+Fierro&author=A.+Montecinos&author=C.+Gmez-Molina&author=G.+Nez&author=M.+Aldeco&author=D.+E.+Edmondson&author=M.+Vilches+Herrera&author=S.+Luhr&author=P.+Iturriaga-Vsquez&author=M.+Reyes+Parauthor=G.+Radis-Baptista&title=An+Integrated+View+of+the+Molecular+Recognition+and+Toxinology%3A+From+Analytical+Procedures+to+Biomedical+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.5772%2F35874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.5772%252F35874%26sid%3Dliteratum%253Aachs%26aulast%3DFierro%26aufirst%3DA.%26atitle%3DSimilarities%2520between%2520the%2520binding%2520sites%2520of%2520monoamine%2520oxidase%2520%2528MAO%2529%2520from%2520different%2520species%25E2%2580%2594Is%2520zebrafish%2520a%2520useful%2520model%2520for%2520the%2520discovery%2520of%2520novel%2520MAO%2520inhibitors%253F%26btitle%3DAn%2520Integrated%2520View%2520of%2520the%2520Molecular%2520Recognition%2520and%2520Toxinology%253A%2520From%2520Analytical%2520Procedures%2520to%2520Biomedical%2520Applications%26aulast%3DRadis-Baptista%26aufirst%3DG.%26pub%3DInTech%26date%3D2013%26doi%3D10.5772%2F35874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Reisen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weisel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kriegl, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span> </span><span class="NLM_article-title">Self-organizing fuzzy graphs for structure-based comparison of protein pockets</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">6498</span><span class="NLM_x">–</span> <span class="NLM_lpage">6510</span><span class="refDoi"> DOI: 10.1021/pr100719n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr100719n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlaltbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=6498-6510&issue=12&author=F.+Reisenauthor=M.+Weiselauthor=J.+M.+Krieglauthor=G.+Schneider&title=Self-organizing+fuzzy+graphs+for+structure-based+comparison+of+protein+pockets&doi=10.1021%2Fpr100719n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Organizing Fuzzy Graphs for Structure-Based Comparison of Protein Pockets</span></div><div class="casAuthors">Reisen, Felix; Weisel, Martin; Kriegl, Jan M.; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6498-6510</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Patterns of receptor-ligand interaction can be conserved in functionally equiv. proteins even in the absence of sequence homol.  Therefore, structural comparison of ligand-binding pockets and their pharmacophoric features allow for the characterization of so-called "orphan" proteins with known three-dimensional structure but unknown function, and predict ligand promiscuity of binding pockets.  We present an algorithm for rapid pocket comparison (PoLiMorph), in which protein pockets are represented by self-organizing graphs that fill the vol. of the cavity.  Vertices in these three-dimensional frameworks contain information about the local ligand-receptor interaction potential coded by fuzzy property labels.  For framework matching, we developed a fast heuristic based on the max. dispersion problem, as an alternative to techniques utilizing clique detection or geometric hashing algorithms.  A sophisticated scoring function was applied that incorporates knowledge about property distributions and ligand-receptor interaction patterns.  In an all-against-all virtual screening expt. with 207 pocket frameworks extd. from a subset of PDBbind, PoLiMorph correctly assigned 81% of 69 distinct structural classes and demonstrated sustained ability to group pockets accommodating the same ligand chemotype.  We detd. a score threshold that indicates "true" pocket similarity with high reliability, which not only supports structure-based drug design but also allows for sequence-independent studies of the proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjH1-VYRUmObVg90H21EOLACvtfcHk0ljcQUbSCsXuSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlaltbvL&md5=8da973a8e1097e456ed9bffa7040c4b9</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fpr100719n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr100719n%26sid%3Dliteratum%253Aachs%26aulast%3DReisen%26aufirst%3DF.%26aulast%3DWeisel%26aufirst%3DM.%26aulast%3DKriegl%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DSelf-organizing%2520fuzzy%2520graphs%2520for%2520structure-based%2520comparison%2520of%2520protein%2520pockets%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2010%26volume%3D9%26issue%3D12%26spage%3D6498%26epage%3D6510%26doi%3D10.1021%2Fpr100719n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Lanig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reisen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitley, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banting, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, T.</span><span> </span><span class="NLM_article-title">In silico adoption of an orphan nuclear receptor NR4A1</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e0135246</span><span class="refDoi"> DOI: 10.1371/journal.pone.0135246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0135246" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0135246&issue=8&author=H.+Lanigauthor=F.+Reisenauthor=D.+Whitleyauthor=G.+Schneiderauthor=L.+Bantingauthor=T.+Clark&title=In+silico+adoption+of+an+orphan+nuclear+receptor+NR4A1&doi=10.1371%2Fjournal.pone.0135246"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0135246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0135246%26sid%3Dliteratum%253Aachs%26aulast%3DLanig%26aufirst%3DH.%26aulast%3DReisen%26aufirst%3DF.%26aulast%3DWhitley%26aufirst%3DD.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DBanting%26aufirst%3DL.%26aulast%3DClark%26aufirst%3DT.%26atitle%3DIn%2520silico%2520adoption%2520of%2520an%2520orphan%2520nuclear%2520receptor%2520NR4A1%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3De0135246%26doi%3D10.1371%2Fjournal.pone.0135246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janežič, D.</span><span> </span><span class="NLM_article-title">ProBiS algorithm for detection of structurally similar protein binding sites by local structural alignment</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">1160</span><span class="NLM_x">–</span> <span class="NLM_lpage">1168</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btq100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtq100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=20305268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1160-1168&issue=9&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS+algorithm+for+detection+of+structurally+similar+protein+binding+sites+by+local+structural+alignment&doi=10.1093%2Fbioinformatics%2Fbtq100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS algorithm for detection of structurally similar protein binding sites by local structural alignment</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1160-1168</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Exploitation of locally similar 3D patterns of physicochem. properties on the surface of a protein for detection of binding sites that may lack sequence and global structural conservation.  An algorithm, ProBiS is described that detects structurally similar sites on protein surfaces by local surface structure alignment.  It compares the query protein to members of a database of protein 3D structures and detects with sub-residue precision, structurally similar sites as patterns of physicochem. properties on the protein surface.  Using an efficient max. clique algorithm, the program identifies proteins that share local structural similarities with the query protein and generates structure-based alignments of these proteins with the query.  Structural similarity scores are calcd. for the query protein's surface residues, and are expressed as different colors on the query protein surface.  The algorithm has been used successfully for the detection of protein-protein, protein-small ligand and protein-DNA binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproKvL-3-CSLVg90H21EOLACvtfcHk0ljcQUbSCsXuSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D&md5=7ff0eecdf0ea5f7d9b82fd721ca88156</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq100%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%2520algorithm%2520for%2520detection%2520of%2520structurally%2520similar%2520protein%2520binding%2520sites%2520by%2520local%2520structural%2520alignment%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26issue%3D9%26spage%3D1160%26epage%3D1168%26doi%3D10.1093%2Fbioinformatics%2Fbtq100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodošček, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogrizek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trykowska Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janežič, D.</span><span> </span><span class="NLM_article-title">Structure-based function prediction of uncharacterized protein using binding sites comparison</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1003341</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1003341" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003341&issue=11&author=J.+Koncauthor=M.+Hodo%C5%A1%C4%8Dekauthor=M.+Ogrizekauthor=J.+Trykowska+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=Structure-based+function+prediction+of+uncharacterized+protein+using+binding+sites+comparison&doi=10.1371%2Fjournal.pcbi.1003341"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003341%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DHodo%25C5%25A1%25C4%258Dek%26aufirst%3DM.%26aulast%3DOgrizek%26aufirst%3DM.%26aulast%3DTrykowska%2BKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DStructure-based%2520function%2520prediction%2520of%2520uncharacterized%2520protein%2520using%2520binding%2520sites%2520comparison%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D11%26spage%3De1003341%26doi%3D10.1371%2Fjournal.pcbi.1003341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Kakisaka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamada, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kondoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hikono, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomii, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aida, Y.</span><span> </span><span class="NLM_article-title">A novel antiviral target structure involved in the RNA binding, dimerization, and nuclear export functions of the influenza A virus nucleoprotein</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1005062</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1005062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.ppat.1005062" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=e1005062&issue=7&author=M.+Kakisakaauthor=Y.+Sasakiauthor=K.+Yamadaauthor=Y.+Kondohauthor=H.+Hikonoauthor=H.+Osadaauthor=K.+Tomiiauthor=T.+Saitoauthor=Y.+Aida&title=A+novel+antiviral+target+structure+involved+in+the+RNA+binding%2C+dimerization%2C+and+nuclear+export+functions+of+the+influenza+A+virus+nucleoprotein&doi=10.1371%2Fjournal.ppat.1005062"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005062%26sid%3Dliteratum%253Aachs%26aulast%3DKakisaka%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DY.%26aulast%3DYamada%26aufirst%3DK.%26aulast%3DKondoh%26aufirst%3DY.%26aulast%3DHikono%26aufirst%3DH.%26aulast%3DOsada%26aufirst%3DH.%26aulast%3DTomii%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DT.%26aulast%3DAida%26aufirst%3DY.%26atitle%3DA%2520novel%2520antiviral%2520target%2520structure%2520involved%2520in%2520the%2520RNA%2520binding%252C%2520dimerization%252C%2520and%2520nuclear%2520export%2520functions%2520of%2520the%2520influenza%2520A%2520virus%2520nucleoprotein%26jtitle%3DPLoS%2520Pathog.%26date%3D2015%26volume%3D11%26issue%3D7%26spage%3De1005062%26doi%3D10.1371%2Fjournal.ppat.1005062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Spitzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleves, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Surface-based protein binding pocket similarity</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">2746</span><span class="NLM_x">–</span> <span class="NLM_lpage">2763</span><span class="refDoi"> DOI: 10.1002/prot.23103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.23103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2011&pages=2746-2763&issue=9&author=R.+Spitzerauthor=A.+E.+Clevesauthor=A.+N.+Jain&title=Surface-based+protein+binding+pocket+similarity&doi=10.1002%2Fprot.23103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fprot.23103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.23103%26sid%3Dliteratum%253Aachs%26aulast%3DSpitzer%26aufirst%3DR.%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurface-based%2520protein%2520binding%2520pocket%2520similarity%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2011%26volume%3D79%26issue%3D9%26spage%3D2746%26epage%3D2763%26doi%3D10.1002%2Fprot.23103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Cleves, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Chemical and protein structural basis for biological crosstalk between PPARα and COX enzymes</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span><span class="refDoi"> DOI: 10.1007/s10822-014-9815-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2Fs10822-014-9815-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=101-112&issue=2&author=A.+E.+Clevesauthor=A.+N.+Jain&title=Chemical+and+protein+structural+basis+for+biological+crosstalk+between+PPAR%CE%B1+and+COX+enzymes&doi=10.1007%2Fs10822-014-9815-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9815-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9815-2%26sid%3Dliteratum%253Aachs%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DChemical%2520and%2520protein%2520structural%2520basis%2520for%2520biological%2520crosstalk%2520between%2520PPAR%25CE%25B1%2520and%2520COX%2520enzymes%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D2%26spage%3D101%26epage%3D112%26doi%3D10.1007%2Fs10822-014-9815-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Ausiello, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Via, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmer-Citterich, M.</span><span> </span><span class="NLM_article-title">Query3d: a new method for high-throughput analysis of functional residues in protein structures</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 4</span><span class="NLM_x">) </span> <span class="NLM_fpage">S5</span><span class="refDoi"> DOI: 10.1186/1471-2105-6-S4-S5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-6-S4-S5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=S5&issue=Suppl.+4&author=G.+Ausielloauthor=A.+Viaauthor=M.+Helmer-Citterich&title=Query3d%3A+a+new+method+for+high-throughput+analysis+of+functional+residues+in+protein+structures&doi=10.1186%2F1471-2105-6-S4-S5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-6-S4-S5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-6-S4-S5%26sid%3Dliteratum%253Aachs%26aulast%3DAusiello%26aufirst%3DG.%26aulast%3DVia%26aufirst%3DA.%26aulast%3DHelmer-Citterich%26aufirst%3DM.%26atitle%3DQuery3d%253A%2520a%2520new%2520method%2520for%2520high-throughput%2520analysis%2520of%2520functional%2520residues%2520in%2520protein%2520structures%26jtitle%3DBMC%2520Bioinf.%26date%3D2005%26volume%3D6%26issue%3DSuppl.%25204%26spage%3DS5%26doi%3D10.1186%2F1471-2105-6-S4-S5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Ausiello, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peluso, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Via, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmer-Citterich, M.</span><span> </span><span class="NLM_article-title">Local comparison of protein structures highlights cases of convergent evolution in analogous functional sites</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S24</span><span class="refDoi"> DOI: 10.1186/1471-2105-8-S1-S24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-8-S1-S24" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=S24&issue=Suppl.+1&author=G.+Ausielloauthor=D.+Pelusoauthor=A.+Viaauthor=M.+Helmer-Citterich&title=Local+comparison+of+protein+structures+highlights+cases+of+convergent+evolution+in+analogous+functional+sites&doi=10.1186%2F1471-2105-8-S1-S24"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-S1-S24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-8-S1-S24%26sid%3Dliteratum%253Aachs%26aulast%3DAusiello%26aufirst%3DG.%26aulast%3DPeluso%26aufirst%3DD.%26aulast%3DVia%26aufirst%3DA.%26aulast%3DHelmer-Citterich%26aufirst%3DM.%26atitle%3DLocal%2520comparison%2520of%2520protein%2520structures%2520highlights%2520cases%2520of%2520convergent%2520evolution%2520in%2520analogous%2520functional%2520sites%26jtitle%3DBMC%2520Bioinf.%26date%3D2007%26volume%3D8%26issue%3DSuppl.%25201%26spage%3DS24%26doi%3D10.1186%2F1471-2105-8-S1-S24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">De Franchi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Messa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onofri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benfenati, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Binding of protein kinase inhibitors to synapsin I inferred from pair-wise binding site similarity measurements</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e12214</span><span class="refDoi"> DOI: 10.1371/journal.pone.0012214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0012214" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e12214&issue=8&author=E.+De+Franchiauthor=C.+Schalonauthor=M.+Messaauthor=F.+Onofriauthor=F.+Benfenatiauthor=D.+Rognan&title=Binding+of+protein+kinase+inhibitors+to+synapsin+I+inferred+from+pair-wise+binding+site+similarity+measurements&doi=10.1371%2Fjournal.pone.0012214"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0012214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0012214%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BFranchi%26aufirst%3DE.%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DMessa%26aufirst%3DM.%26aulast%3DOnofri%26aufirst%3DF.%26aulast%3DBenfenati%26aufirst%3DF.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DBinding%2520of%2520protein%2520kinase%2520inhibitors%2520to%2520synapsin%2520I%2520inferred%2520from%2520pair-wise%2520binding%2520site%2520similarity%2520measurements%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26issue%3D8%26spage%3De12214%26doi%3D10.1371%2Fjournal.pone.0012214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Shulman-Peleg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfson, H. J.</span><span> </span><span class="NLM_article-title">Recognition of functional sites in protein structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">339</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">633</span><span class="refDoi"> DOI: 10.1016/j.jmb.2004.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.jmb.2004.04.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2004&pages=607-633&issue=3&author=A.+Shulman-Pelegauthor=R.+Nussinovauthor=H.+J.+Wolfson&title=Recognition+of+functional+sites+in+protein+structures&doi=10.1016%2Fj.jmb.2004.04.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DShulman-Peleg%26aufirst%3DA.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DWolfson%26aufirst%3DH.%2BJ.%26atitle%3DRecognition%2520of%2520functional%2520sites%2520in%2520protein%2520structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D339%26issue%3D3%26spage%3D607%26epage%3D633%26doi%3D10.1016%2Fj.jmb.2004.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group">Keren-Kaplan, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Attali, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estrin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuo, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farkash, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jerabek-Willemsen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blutraich, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artzi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freed, E. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfson, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prag, G.</span><span> </span><span class="NLM_article-title">Structure-based in silico identification of ubiquitin-binding domains provides insights into the ALIX-V:ubiquitin complex and retrovirus budding</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">551</span><span class="refDoi"> DOI: 10.1038/emboj.2013.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Femboj.2013.4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=538-551&issue=4&author=T.+Keren-Kaplanauthor=I.+Attaliauthor=M.+Estrinauthor=L.+S.+Kuoauthor=E.+Farkashauthor=M.+Jerabek-Willemsenauthor=N.+Blutraichauthor=S.+Artziauthor=A.+Periauthor=E.+O.+Freedauthor=H.+J.+Wolfsonauthor=G.+Prag&title=Structure-based+in+silico+identification+of+ubiquitin-binding+domains+provides+insights+into+the+ALIX-V%3Aubiquitin+complex+and+retrovirus+budding&doi=10.1038%2Femboj.2013.4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1038%2Femboj.2013.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2013.4%26sid%3Dliteratum%253Aachs%26aulast%3DKeren-Kaplan%26aufirst%3DT.%26aulast%3DAttali%26aufirst%3DI.%26aulast%3DEstrin%26aufirst%3DM.%26aulast%3DKuo%26aufirst%3DL.%2BS.%26aulast%3DFarkash%26aufirst%3DE.%26aulast%3DJerabek-Willemsen%26aufirst%3DM.%26aulast%3DBlutraich%26aufirst%3DN.%26aulast%3DArtzi%26aufirst%3DS.%26aulast%3DPeri%26aufirst%3DA.%26aulast%3DFreed%26aufirst%3DE.%2BO.%26aulast%3DWolfson%26aufirst%3DH.%2BJ.%26aulast%3DPrag%26aufirst%3DG.%26atitle%3DStructure-based%2520in%2520silico%2520identification%2520of%2520ubiquitin-binding%2520domains%2520provides%2520insights%2520into%2520the%2520ALIX-V%253Aubiquitin%2520complex%2520and%2520retrovirus%2520budding%26jtitle%3DEMBO%2520J.%26date%3D2013%26volume%3D32%26issue%3D4%26spage%3D538%26epage%3D551%26doi%3D10.1038%2Femboj.2013.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A unified statistical model to support local sequence order independent similarity searching for ligand-binding sites and its application to genome-based drug discovery</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">i305</span><span class="NLM_x">–</span> <span class="NLM_lpage">i312</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtp220" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=i305-i312&issue=12&author=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=A+unified+statistical+model+to+support+local+sequence+order+independent+similarity+searching+for+ligand-binding+sites+and+its+application+to+genome-based+drug+discovery&doi=10.1093%2Fbioinformatics%2Fbtp220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp220%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520unified%2520statistical%2520model%2520to%2520support%2520local%2520sequence%2520order%2520independent%2520similarity%2520searching%2520for%2520ligand-binding%2520sites%2520and%2520its%2520application%2520to%2520genome-based%2520drug%2520discovery%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26issue%3D12%26spage%3Di305%26epage%3Di312%26doi%3D10.1093%2Fbioinformatics%2Fbtp220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evangelidis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: weak inhibition of multiple kinases may contribute to the anti-cancer effect of nelfinavir</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002037</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1002037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1002037" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002037&issue=4&author=L.+Xieauthor=T.+Evangelidisauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+weak+inhibition+of+multiple+kinases+may+contribute+to+the+anti-cancer+effect+of+nelfinavir&doi=10.1371%2Fjournal.pcbi.1002037"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002037%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DEvangelidis%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520weak%2520inhibition%2520of%2520multiple%2520kinases%2520may%2520contribute%2520to%2520the%2520anti-cancer%2520effect%2520of%2520nelfinavir%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D4%26spage%3De1002037%26doi%3D10.1371%2Fjournal.pcbi.1002037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group">Niu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiaoe, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span> </span><span class="NLM_article-title">Structural bioinformatics-based identification of EGFR inhibitor gefitinib as a putative lead compound for BACE</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.1111/cbdd.12200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1111%2Fcbdd.12200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24516878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOntbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=81-88&issue=1&author=M.+Niuauthor=J.+Huauthor=S.+Wuauthor=Z.+Xiaoeauthor=H.+Xuauthor=Y.+Zhangauthor=J.+Zhangauthor=Y.+Yang&title=Structural+bioinformatics-based+identification+of+EGFR+inhibitor+gefitinib+as+a+putative+lead+compound+for+BACE&doi=10.1111%2Fcbdd.12200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Bioinformatics-Based Identification of EGFR Inhibitor Gefitinib as a Putative Lead Compound for BACE</span></div><div class="casAuthors">Niu, Mingshan; Hu, Jin; Wu, Sijin; Zhang, Xiaoe; Xu, Huaxi; Zhang, Yunwu; Zhang, Jie; Yang, Yongliang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-88</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">β-Secretase (BACE-1) is a potential target for the treatment of Alzheimer's disease (AD).  Despite its potential, only few compds. targeting BACE have entered the clin. trials.  Herein, the authors describe the identification of Gefitinib as a potential lead compd. for BACE through an integrated approach of structural bioinformatics anal., exptl. assessment and computational anal.  In particular, the authors performed ELISA and western anal. to assess the effect of Gefitinib using N2a human APP695 cells.  In addn., the authors investigated the binding mechanism of Gefitinib with BACE through mol. docking coupled with mol. dynamics simulations.  The computational analyses revealed that hydrophobic contact is a major contributing factor to the binding of Gefitinib with BACE.  The results obtained in the study have rendered Gefitinib as a putative lead compd. for BACE.  Further optimization studies are warranted to improve its potency and pharmacol. properties against BACE for potential AD treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaB-v5A6zsELVg90H21EOLACvtfcHk0lhJCc8Yu_vHxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOntbjJ&md5=229f742fee427820a7910f72c11815cf</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12200%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DXiaoe%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DStructural%2520bioinformatics-based%2520identification%2520of%2520EGFR%2520inhibitor%2520gefitinib%2520as%2520a%2520putative%2520lead%2520compound%2520for%2520BACE%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2014%26volume%3D83%26issue%3D1%26spage%3D81%26epage%3D88%26doi%3D10.1111%2Fcbdd.12200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Detecting evolutionary relationships across existing fold space, using sequence order-independent profile-profile alignments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">5441</span><span class="NLM_x">–</span> <span class="NLM_lpage">5446</span><span class="refDoi"> DOI: 10.1073/pnas.0704422105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1073%2Fpnas.0704422105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18385384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslSnsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=5441-5446&issue=14&author=L.+Xieauthor=P.+E.+Bourne&title=Detecting+evolutionary+relationships+across+existing+fold+space%2C+using+sequence+order-independent+profile-profile+alignments&doi=10.1073%2Fpnas.0704422105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Detecting evolutionary relationships across existing fold space, using sequence order-independent profile-profile alignments</span></div><div class="casAuthors">Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5441-5446</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Here, a scalable, accurate, reliable, and robust protein functional site comparison algorithm is presented.  The key components of the algorithm consist of a reduced representation of the protein structure and a sequence order-independent profile-profile alignment (SOIPPA).  We show that SOIPPA is able to detect distant evolutionary relationships in cases where both a global sequence and structure relationship remains obscure.  Results suggest evolutionary relationships across several previously evolutionary distinct protein structure superfamilies.  SOIPPA, along with an increased coverage of-protein fold space afforded by the structural genomics initiative, can be used to further test the notion that fold space is continuous rather than discrete.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwuIQ_fI-0v7Vg90H21EOLACvtfcHk0lhJCc8Yu_vHxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslSnsLY%253D&md5=a3a5a216f18aff3038d987bb6e4c6443</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0704422105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0704422105%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDetecting%2520evolutionary%2520relationships%2520across%2520existing%2520fold%2520space%252C%2520using%2520sequence%2520order-independent%2520profile-profile%2520alignments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26issue%3D14%26spage%3D5441%26epage%3D5446%26doi%3D10.1073%2Fpnas.0704422105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchmeier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: repositioning the safe medicine Comtan to treat multi-drug and extensively drug resistant tuberculosis</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1000423</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1000423" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000423&issue=7&author=S.+L.+Kinningsauthor=N.+Liuauthor=N.+Buchmeierauthor=P.+J.+Tongeauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+repositioning+the+safe+medicine+Comtan+to+treat+multi-drug+and+extensively+drug+resistant+tuberculosis&doi=10.1371%2Fjournal.pcbi.1000423"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000423%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DBuchmeier%26aufirst%3DN.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520repositioning%2520the%2520safe%2520medicine%2520Comtan%2520to%2520treat%2520multi-drug%2520and%2520extensively%2520drug%2520resistant%2520tuberculosis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26issue%3D7%26spage%3De1000423%26doi%3D10.1371%2Fjournal.pcbi.1000423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">In silico elucidation of the molecular mechanism defining the adverse effect of selective estrogen receptor modulators</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">e217</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.0030217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.0030217" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=e217&issue=11&author=L.+Xieauthor=J.+Wangauthor=P.+E.+Bourne&title=In+silico+elucidation+of+the+molecular+mechanism+defining+the+adverse+effect+of+selective+estrogen+receptor+modulators&doi=10.1371%2Fjournal.pcbi.0030217"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.0030217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.0030217%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DIn%2520silico%2520elucidation%2520of%2520the%2520molecular%2520mechanism%2520defining%2520the%2520adverse%2520effect%2520of%2520selective%2520estrogen%2520receptor%2520modulators%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2007%26volume%3D3%26issue%3D11%26spage%3De217%26doi%3D10.1371%2Fjournal.pcbi.0030217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group">Binkowski, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joachimiak, A.</span><span> </span><span class="NLM_article-title">Protein functional surfaces: global shape matching and local spatial alignments of ligand binding sites</span> <span class="citation_source-journal">BMC Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">45</span><span class="refDoi"> DOI: 10.1186/1472-6807-8-45</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1472-6807-8-45" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18954462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FntVOjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=45&author=T.+A.+Binkowskiauthor=A.+Joachimiak&title=Protein+functional+surfaces%3A+global+shape+matching+and+local+spatial+alignments+of+ligand+binding+sites&doi=10.1186%2F1472-6807-8-45"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Protein functional surfaces: global shape matching and local spatial alignments of ligand binding sites</span></div><div class="casAuthors">Binkowski T Andrew; Joachimiak Andrzej</div><div class="citationInfo"><span class="NLM_cas:title">BMC structural biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Protein surfaces comprise only a fraction of the total residues but are the most conserved functional features of proteins.  Surfaces performing identical functions are found in proteins absent of any sequence or fold similarity.  While biochemical activity can be attributed to a few key residues, the broader surrounding environment plays an equally important role.  RESULTS:  We describe a methodology that attempts to optimize two components, global shape and local physicochemical texture, for evaluating the similarity between a pair of surfaces.  Surface shape similarity is assessed using a three-dimensional object recognition algorithm and physicochemical texture similarity is assessed through a spatial alignment of conserved residues between the surfaces.  The comparisons are used in tandem to efficiently search the Global Protein Surface Survey (GPSS), a library of annotated surfaces derived from structures in the PDB, for studying evolutionary relationships and uncovering novel similarities between proteins.  CONCLUSION:  We provide an assessment of our method using library retrieval experiments for identifying functionally homologous surfaces binding different ligands, functionally diverse surfaces binding the same ligand, and binding surfaces of ubiquitous and conformationally flexible ligands.  Results using surface similarity to predict function for proteins of unknown function are reported.  Additionally, an automated analysis of the ATP binding surface landscape is presented to provide insight into the correlation between surface similarity and function for structures in the PDB and for the subset of protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRwk9DbnSjneTToTC3aBaesfW6udTcc2eYPOI9tPJtB07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FntVOjsg%253D%253D&md5=e991996eb22a4f3fcf5f236cf06f76ac</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1186%2F1472-6807-8-45&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6807-8-45%26sid%3Dliteratum%253Aachs%26aulast%3DBinkowski%26aufirst%3DT.%2BA.%26aulast%3DJoachimiak%26aufirst%3DA.%26atitle%3DProtein%2520functional%2520surfaces%253A%2520global%2520shape%2520matching%2520and%2520local%2520spatial%2520alignments%2520of%2520ligand%2520binding%2520sites%26jtitle%3DBMC%2520Struct.%2520Biol.%26date%3D2008%26volume%3D8%26spage%3D45%26doi%3D10.1186%2F1472-6807-8-45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group">Babu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beloglazova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skarina, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocek, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagarinova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pogoutse, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binkowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phanse, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joachimiak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koonin, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savchenko, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emili, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yakunin, A. F.</span><span> </span><span class="NLM_article-title">A dual function of the CRISPR-Cas system in bacterial antivirus immunity and DNA repair</span> <span class="citation_source-journal">Mol. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">484</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1111/j.1365-2958.2010.07465.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1111%2Fj.1365-2958.2010.07465.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21219465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsl2qtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2011&pages=484-502&issue=2&author=M.+Babuauthor=N.+Beloglazovaauthor=R.+Flickauthor=C.+Grahamauthor=T.+Skarinaauthor=B.+Nocekauthor=A.+Gagarinovaauthor=O.+Pogoutseauthor=G.+Brownauthor=A.+Binkowskiauthor=S.+Phanseauthor=A.+Joachimiakauthor=E.+V.+Kooninauthor=A.+Savchenkoauthor=A.+Emiliauthor=J.+Greenblattauthor=A.+M.+Edwardsauthor=A.+F.+Yakunin&title=A+dual+function+of+the+CRISPR-Cas+system+in+bacterial+antivirus+immunity+and+DNA+repair&doi=10.1111%2Fj.1365-2958.2010.07465.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">A dual function of the CRISPR-Cas system in bacterial antivirus immunity and DNA repair</span></div><div class="casAuthors">Babu, Mohan; Beloglazova, Natalia; Flick, Robert; Graham, Chris; Skarina, Tatiana; Nocek, Boguslaw; Gagarinova, Alla; Pogoutse, Oxana; Brown, Greg; Binkowski, Andrew; Phanse, Sadhna; Joachimiak, Andrzej; Koonin, Eugene V.; Savchenko, Alexei; Emili, Andrew; Greenblatt, Jack; Edwards, Aled M.; Yakunin, Alexander F.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">484-502</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Clustered Regularly Interspaced Short Palindromic Repeats (CRISPRs) and the assocd. proteins (Cas) comprise a system of adaptive immunity against viruses and plasmids in prokaryotes.  Cas1 is a CRISPR-assocd. protein that is common to all CRISPR-contg. prokaryotes but its function remains obscure.  Here we show that the purified Cas1 protein of Escherichia coli (YgbT) exhibits nuclease activity against single-stranded and branched DNAs including Holliday junctions, replication forks and 5'-flaps.  The crystal structure of YgbT and site-directed mutagenesis have revealed the potential active site.  Genome-wide screens show that YgbT phys. and genetically interacts with key components of DNA repair systems, including recB, recC and ruvB.  Consistent with these findings, the ygbT deletion strain showed increased sensitivity to DNA damage and impaired chromosomal segregation.  Similar phenotypes were obsd. in strains with deletion of CRISPR clusters, suggesting that the function of YgbT in repair involves interaction with the CRISPRs.  These results show that YgbT belongs to a novel, structurally distinct family of nucleases acting on branched DNAs and suggest that, in addn. to antiviral immunity, at least some components of the CRISPR-Cas system have a function in DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6QELoEg3Wi7Vg90H21EOLACvtfcHk0liqbncSaxHXzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsl2qtLs%253D&md5=d3f0b4ed7ebcc92e5199f403da227819</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2010.07465.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2010.07465.x%26sid%3Dliteratum%253Aachs%26aulast%3DBabu%26aufirst%3DM.%26aulast%3DBeloglazova%26aufirst%3DN.%26aulast%3DFlick%26aufirst%3DR.%26aulast%3DGraham%26aufirst%3DC.%26aulast%3DSkarina%26aufirst%3DT.%26aulast%3DNocek%26aufirst%3DB.%26aulast%3DGagarinova%26aufirst%3DA.%26aulast%3DPogoutse%26aufirst%3DO.%26aulast%3DBrown%26aufirst%3DG.%26aulast%3DBinkowski%26aufirst%3DA.%26aulast%3DPhanse%26aufirst%3DS.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DKoonin%26aufirst%3DE.%2BV.%26aulast%3DSavchenko%26aufirst%3DA.%26aulast%3DEmili%26aufirst%3DA.%26aulast%3DGreenblatt%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DYakunin%26aufirst%3DA.%2BF.%26atitle%3DA%2520dual%2520function%2520of%2520the%2520CRISPR-Cas%2520system%2520in%2520bacterial%2520antivirus%2520immunity%2520and%2520DNA%2520repair%26jtitle%3DMol.%2520Microbiol.%26date%3D2011%26volume%3D79%26issue%3D2%26spage%3D484%26epage%3D502%26doi%3D10.1111%2Fj.1365-2958.2010.07465.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">TM-align: a protein structure alignment algorithm based on the TM-score</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">2302</span><span class="NLM_x">–</span> <span class="NLM_lpage">2309</span><span class="refDoi"> DOI: 10.1093/nar/gki524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgki524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=15849316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsl2gsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=2302-2309&issue=7&author=Y.+Zhangauthor=J.+Skolnick&title=TM-align%3A+a+protein+structure+alignment+algorithm+based+on+the+TM-score&doi=10.1093%2Fnar%2Fgki524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">TM-align: a protein structure alignment algorithm based on the TM-score</span></div><div class="casAuthors">Zhang, Yang; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2302-2309</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have developed TM-align, a new algorithm to identify the best structural alignment between protein pairs that combines the TM-score rotation matrix and Dynamic Programming (DP).  The algorithm is ∼4 times faster than CE and 20 times faster than DALI and SAL.  On av., the resulting structure alignments have higher accuracy and coverage than those provided by these most often-used methods.  TM-align is applied to an all-against-all structure comparison of 10 515 representative protein chains from the Protein Data Bank (PDB) with a sequence identity cutoff <95%: 1996 distinct folds are found when a TM-score threshold of 0.5 is used.  We also use TM-align to match the models predicted by TASSER for solved non-homologous proteins in PDB.  For both folded and misfolded models, TM-align can almost always find close structural analogs, with an av. root mean square deviation, RMSD, of 3 Å and 87% alignment coverage.  Nevertheless, there exists a significant correlation between the correctness of the predicted structure and the structural similarity of the model to the other proteins in the PDB.  This correlation could be used to assist in model selection in blind protein structure predictions.  The TM-align program is freely downloadable at http://bioinformatics.buffalo.edu/TM-align.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonPrM9Mg2FZ7Vg90H21EOLACvtfcHk0liqbncSaxHXzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsl2gsLY%253D&md5=15689409a6d9281184a7f943f5cc45d3</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki524%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DTM-align%253A%2520a%2520protein%2520structure%2520alignment%2520algorithm%2520based%2520on%2520the%2520TM-score%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26issue%3D7%26spage%3D2302%26epage%3D2309%26doi%3D10.1093%2Fnar%2Fgki524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">Computational discovery and experimental verification of tyrosine kinase inhibitor pazopanib for the reversal of memory and cognitive deficits in rat model neurodegeneration</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">2812</span><span class="NLM_x">–</span> <span class="NLM_lpage">2821</span><span class="refDoi"> DOI: 10.1039/C4SC03416C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1039%2FC4SC03416C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntVGjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=2812-2821&issue=5&author=Y.+Yangauthor=G.+Liauthor=D.+Zhaoauthor=H.+Yuauthor=X.+Zhengauthor=X.+Pengauthor=X.+Zhangauthor=T.+Fuauthor=X.+Huauthor=M.+Niuauthor=X.+Jiauthor=L.+Zouauthor=J.+Wang&title=Computational+discovery+and+experimental+verification+of+tyrosine+kinase+inhibitor+pazopanib+for+the+reversal+of+memory+and+cognitive+deficits+in+rat+model+neurodegeneration&doi=10.1039%2FC4SC03416C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Computational discovery and experimental verification of tyrosine kinase inhibitor pazopanib for the reversal of memory and cognitive deficits in rat model neurodegeneration</span></div><div class="casAuthors">Yang, Yongliang; Li, Guohui; Zhao, Dongyu; Yu, Haoyang; Zheng, Xiliang; Peng, Xiangda; Zhang, Xiaoe; Fu, Ting; Hu, Xiaoqing; Niu, Mingshan; Ji, Xuefei; Zou, Libo; Wang, Jin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2812-2821</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cognition and memory impairment are hallmarks of the pathol. cascade of various neurodegenerative disorders.  Herein, we developed a novel computational strategy with two-dimensional virtual screening for not only affinity but also specificity.  We integrated the two-dimensional virtual screening with ligand screening for 3D shape, electrostatic similarity and local binding site similarity to find existing drugs that may reduce the signs of cognitive deficits.  For the first time, we found that pazopanib, a tyrosine kinase inhibitor marketed for cancer treatment, inhibits acetylcholinesterase (AchE) activities at sub-micromolar concn.  We evaluated and compared the effects of intragastrically-administered pazopanib with donepezil, a marketed AchE inhibitor, in cognitive and behavioral assays including the novel object recognition test, Y maze and Morris water maze test.  Surprisingly, we found that pazopanib can restore memory loss and cognitive dysfunction to a similar extent as donepezil in a dosage of 15 mg kg-1, only one fifth of the equiv. clin. dosage for cancer treatment.  Furthermore, we demonstrated that pazopanib dramatically enhances the hippocampal Ach levels and increases the expression of the synaptic marker SYP.  These findings suggest that pazopanib may become a viable treatment option for memory and cognitive deficits with a good safety profile in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouyZIk9hoxDLVg90H21EOLACvtfcHk0ljwD-tmURIvKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntVGjtw%253D%253D&md5=5d6efdb9187c21310061df26971e8ba7</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1039%2FC4SC03416C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4SC03416C%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DNiu%26aufirst%3DM.%26aulast%3DJi%26aufirst%3DX.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DComputational%2520discovery%2520and%2520experimental%2520verification%2520of%2520tyrosine%2520kinase%2520inhibitor%2520pazopanib%2520for%2520the%2520reversal%2520of%2520memory%2520and%2520cognitive%2520deficits%2520in%2520rat%2520model%2520neurodegeneration%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26issue%3D5%26spage%3D2812%26epage%3D2821%26doi%3D10.1039%2FC4SC03416C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">A computational procedure for determining energetically favorable binding sites on biologically important macromolecules</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">849</span><span class="NLM_x">–</span> <span class="NLM_lpage">857</span><span class="refDoi"> DOI: 10.1021/jm00145a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00145a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaL2MXktVeit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=849-857&issue=7&author=P.+J.+Goodford&title=A+computational+procedure+for+determining+energetically+favorable+binding+sites+on+biologically+important+macromolecules&doi=10.1021%2Fjm00145a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">A computational procedure for determining energetically favorable binding sites on biologically important macromolecules</span></div><div class="casAuthors">Goodford, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">849-57</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The interaction of a probe group with a protein of known structure is computed at sample positions throughout and around the macromol., giving an array of energy values.  The probe include H2O, the Me group, amine-N, carboxy-O, and OH.  Contour surfaces at appropriate energy levels are calcd. for each probe and displayed by computer graphics together with the protein structure.  Contours at neg. energy levels delineate regions of attraction between probe and protein and are found at known ligand binding clefts in particular.  The contours also enable other regions of attraction to be identified and facilitate the interpretation of protein-ligand energetics.  They may, therefore, be of value for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3tYN3Fc76dLVg90H21EOLACvtfcHk0ljwD-tmURIvKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXktVeit7c%253D&md5=03e9d22787f16c760ef0dc29ca02aa88</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fjm00145a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00145a002%26sid%3Dliteratum%253Aachs%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DA%2520computational%2520procedure%2520for%2520determining%2520energetically%2520favorable%2520binding%2520sites%2520on%2520biologically%2520important%2520macromolecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26issue%3D7%26spage%3D849%26epage%3D857%26doi%3D10.1021%2Fjm00145a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group">Feldman, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labute, P.</span><span> </span><span class="NLM_article-title">Pocket similarity: are alpha carbons enough?</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1466</span><span class="NLM_x">–</span> <span class="NLM_lpage">1475</span><span class="refDoi"> DOI: 10.1021/ci100210c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100210c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVSgsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1466-1475&issue=8&author=H.+J.+Feldmanauthor=P.+Labute&title=Pocket+similarity%3A+are+alpha+carbons+enough%3F&doi=10.1021%2Fci100210c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Pocket Similarity: Are α Carbons Enough?</span></div><div class="casAuthors">Feldman, Howard J.; Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1466-1475</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for measuring protein pocket similarity was devised, using only the α carbon positions of the pocket residues.  Pockets were compared pairwise using an exhaustive three-dimensional Cα common subset search, grouping residues by physicochem. properties.  At least five Cα matches were required for each hit, and distances between corresponding points were fit to an Extreme Value Distribution resulting in a probabilistic score or likelihood for any given superposition.  A set of 85 structures from 13 diverse protein families was clustered based on binding sites alone, using this score.  It was also successfully used to cluster 25 kinases into a no. of subfamilies.  Using a test kinase query to retrieve other kinase pockets, a specificity of 99.2% and sensitivity of 97.5% could be achieved using an appropriate cutoff score.  The search itself took from 2 to 10 min on a single 3.4 GHz CPU to search the entire Protein Data Bank (133,800 pockets), depending on the no. of hits returned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowvtYX4jdPnrVg90H21EOLACvtfcHk0ljwD-tmURIvKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVSgsbk%253D&md5=a89757babe6ecdcd012be4913a15b419</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Fci100210c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100210c%26sid%3Dliteratum%253Aachs%26aulast%3DFeldman%26aufirst%3DH.%2BJ.%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DPocket%2520similarity%253A%2520are%2520alpha%2520carbons%2520enough%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D8%26spage%3D1466%26epage%3D1475%26doi%3D10.1021%2Fci100210c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group">de Brevern, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etchebest, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazout, S.</span><span> </span><span class="NLM_article-title">Bayesian probabilistic approach for predicting backbone structures in terms of protein blocks</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1002/1097-0134(20001115)41:3<271::AID-PROT10>3.0.CO;2-Z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2F1097-0134%2820001115%2941%3A3%3C271%3A%3AAID-PROT10%3E3.0.CO%3B2-Z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2000&pages=271-287&issue=3&author=A.+G.+de+Brevernauthor=C.+Etchebestauthor=S.+Hazout&title=Bayesian+probabilistic+approach+for+predicting+backbone+structures+in+terms+of+protein+blocks&doi=10.1002%2F1097-0134%2820001115%2941%3A3%3C271%3A%3AAID-PROT10%3E3.0.CO%3B2-Z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1002%2F1097-0134%2820001115%2941%3A3%3C271%3A%3AAID-PROT10%3E3.0.CO%3B2-Z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1097-0134%252820001115%252941%253A3%253C271%253A%253AAID-PROT10%253E3.0.CO%253B2-Z%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBrevern%26aufirst%3DA.%2BG.%26aulast%3DEtchebest%26aufirst%3DC.%26aulast%3DHazout%26aufirst%3DS.%26atitle%3DBayesian%2520probabilistic%2520approach%2520for%2520predicting%2520backbone%2520structures%2520in%2520terms%2520of%2520protein%2520blocks%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2000%26volume%3D41%26issue%3D3%26spage%3D271%26epage%3D287%26doi%3D10.1002%2F1097-0134%2820001115%2941%3A3%3C271%3A%3AAID-PROT10%3E3.0.CO%3B2-Z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">Atomic property fields: generalized 3D pharmacophoric potential for automated ligand superposition, pharmacophore elucidation and 3D QSAR</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00605.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1111%2Fj.1747-0285.2007.00605.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18069986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvF2lsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=15-27&issue=1&author=M.+Totrov&title=Atomic+property+fields%3A+generalized+3D+pharmacophoric+potential+for+automated+ligand+superposition%2C+pharmacophore+elucidation+and+3D+QSAR&doi=10.1111%2Fj.1747-0285.2007.00605.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Atomic property fields: generalized 3D pharmacophoric potential for automated ligand superposition, pharmacophore elucidation and 3D QSAR</span></div><div class="casAuthors">Totrov, Maxim</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-27</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The at. property fields (APF) concept is introduced as a continuous, multi-component 3D potential that reflects preferences for various at. properties at each point in space.  At. property field-based approaches to several key problems in the field of ligand structure-based rational drug discovery and design are investigated.  The superposition of ligands on one or multiple mol. templates is performed by Monte-Carlo minimization in the at. property fields potentials combined with std. force-field energy.  The approach is extended to multiple flexible ligand alignments using an iterative procedure, Self-Consistent at. Property Fields by Optimization (SCAPFOld).  The application of at. property fields and SCAPFOld for virtual ligand screening and 3D Quantitive Structure-Activity Relationship (QSAR) is tested on published benchmarks.  The new methods are shown to perform competitively in comparison to current state-of-the-art methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQFMXqW3d8ErVg90H21EOLACvtfcHk0lgxCplaQh3Pgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvF2lsL8%253D&md5=d66ce22225d4d9dfeed56196af4025d1</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00605.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00605.x%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DAtomic%2520property%2520fields%253A%2520generalized%25203D%2520pharmacophoric%2520potential%2520for%2520automated%2520ligand%2520superposition%252C%2520pharmacophore%2520elucidation%2520and%25203D%2520QSAR%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2008%26volume%3D71%26issue%3D1%26spage%3D15%26epage%3D27%26doi%3D10.1111%2Fj.1747-0285.2007.00605.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Morphological similarity: a 3D molecular similarity method correlated with protein-ligand recognition</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">213</span><span class="refDoi"> DOI: 10.1023/A:1008100132405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1023%2FA%3A1008100132405" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2000&pages=199-213&issue=2&author=A.+N.+Jain&title=Morphological+similarity%3A+a+3D+molecular+similarity+method+correlated+with+protein-ligand+recognition&doi=10.1023%2FA%3A1008100132405"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008100132405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008100132405%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DMorphological%2520similarity%253A%2520a%25203D%2520molecular%2520similarity%2520method%2520correlated%2520with%2520protein-ligand%2520recognition%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2000%26volume%3D14%26issue%3D2%26spage%3D199%26epage%3D213%26doi%3D10.1023%2FA%3A1008100132405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group">Spitzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleves, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varela, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Protein function annotation by local binding site surface similarity</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">694</span><span class="refDoi"> DOI: 10.1002/prot.24450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.24450" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2014&pages=679-694&issue=4&author=R.+Spitzerauthor=A.+E.+Clevesauthor=R.+Varelaauthor=A.+N.+Jain&title=Protein+function+annotation+by+local+binding+site+surface+similarity&doi=10.1002%2Fprot.24450"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1002%2Fprot.24450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24450%26sid%3Dliteratum%253Aachs%26aulast%3DSpitzer%26aufirst%3DR.%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DVarela%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DProtein%2520function%2520annotation%2520by%2520local%2520binding%2520site%2520surface%2520similarity%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2014%26volume%3D82%26issue%3D4%26spage%3D679%26epage%3D694%26doi%3D10.1002%2Fprot.24450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Relibase: design and development of a database for comprehensive analysis of protein-ligand interactions</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)01408-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2802%2901408-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2003&pages=607-620&issue=2&author=M.+Hendlichauthor=A.+Bergnerauthor=J.+G%C3%BCntherauthor=G.+Klebe&title=Relibase%3A+design+and+development+of+a+database+for+comprehensive+analysis+of+protein-ligand+interactions&doi=10.1016%2FS0022-2836%2802%2901408-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2901408-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252901408-0%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DBergner%26aufirst%3DA.%26aulast%3DG%25C3%25BCnther%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DRelibase%253A%2520design%2520and%2520development%2520of%2520a%2520database%2520for%2520comprehensive%2520analysis%2520of%2520protein-ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D326%26issue%3D2%26spage%3D607%26epage%3D620%26doi%3D10.1016%2FS0022-2836%2802%2901408-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group">Günther, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Utilising structural knowledge in drug design strategies: applications using Relibase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">621</span><span class="NLM_x">–</span> <span class="NLM_lpage">636</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)01409-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2802%2901409-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2003&pages=621-636&issue=2&author=J.+G%C3%BCntherauthor=A.+Bergnerauthor=M.+Hendlichauthor=G.+Klebe&title=Utilising+structural+knowledge+in+drug+design+strategies%3A+applications+using+Relibase&doi=10.1016%2FS0022-2836%2802%2901409-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2901409-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252901409-2%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25BCnther%26aufirst%3DJ.%26aulast%3DBergner%26aufirst%3DA.%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DUtilising%2520structural%2520knowledge%2520in%2520drug%2520design%2520strategies%253A%2520applications%2520using%2520Relibase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D326%26issue%3D2%26spage%3D621%26epage%3D636%26doi%3D10.1016%2FS0022-2836%2802%2901409-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span> </span><span class="NLM_article-title">SURFNET: a program for visualizing molecular surfaces, cavities, and intermolecular interactions</span> <span class="citation_source-journal">J. Mol. Graphics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">323</span><span class="NLM_x">–</span> <span class="NLM_lpage">330, 307–308</span><span class="refDoi"> DOI: 10.1016/0263-7855(95)00073-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2F0263-7855%2895%2900073-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=8603061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2MXpslentbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1995&pages=323-330%2C+307%E2%80%93308&issue=5&author=R.+A.+Laskowski&title=SURFNET%3A+a+program+for+visualizing+molecular+surfaces%2C+cavities%2C+and+intermolecular+interactions&doi=10.1016%2F0263-7855%2895%2900073-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">SURFNET: a program for visualizing molecular surfaces, cavities, and intermolecular interactions</span></div><div class="casAuthors">Laskowski, Roman A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">323-30</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The SURFNET program generates mol. surfaces and gaps between surfaces from 3D coordinates supplied in a PDB-format file.  The gap regions can correspond to the voids between two or more mols., or to the internal cavities and surface grooves within a single mol.  The program is particularly useful in clearly delineating the regions of the active site of a protein.  It can also generate 3D contour surfaces of the d. distributions of any set of 3D data points.  All output surfaces can be viewed interactively, along with the mols. or data points in question, using some of the best-known mol. modeling packages.  In addn., PostScript output is available, and the generated surfaces can be rendered using various other graphics packages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwU6uVvAFySbVg90H21EOLACvtfcHk0lgxCplaQh3Pgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpslentbs%253D&md5=cd68217416ecf26be347bec9a1788837</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2895%2900073-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252895%252900073-9%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26atitle%3DSURFNET%253A%2520a%2520program%2520for%2520visualizing%2520molecular%2520surfaces%252C%2520cavities%252C%2520and%2520intermolecular%2520interactions%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1995%26volume%3D13%26issue%3D5%26spage%3D323%26epage%3D330%252C%2520307%25E2%2580%2593308%26doi%3D10.1016%2F0263-7855%2895%2900073-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group">Glaser, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pupko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ben-Tal, N.</span><span> </span><span class="NLM_article-title">The ConSurf-HSSP database: the mapping of evolutionary conservation among homologs onto PDB structures</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">610</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.1002/prot.20305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.20305" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2005&pages=610-617&issue=3&author=F.+Glaserauthor=Y.+Rosenbergauthor=A.+Kesselauthor=T.+Pupkoauthor=N.+Ben-Tal&title=The+ConSurf-HSSP+database%3A+the+mapping+of+evolutionary+conservation+among+homologs+onto+PDB+structures&doi=10.1002%2Fprot.20305"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1002%2Fprot.20305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20305%26sid%3Dliteratum%253Aachs%26aulast%3DGlaser%26aufirst%3DF.%26aulast%3DRosenberg%26aufirst%3DY.%26aulast%3DKessel%26aufirst%3DA.%26aulast%3DPupko%26aufirst%3DT.%26aulast%3DBen-Tal%26aufirst%3DN.%26atitle%3DThe%2520ConSurf-HSSP%2520database%253A%2520the%2520mapping%2520of%2520evolutionary%2520conservation%2520among%2520homologs%2520onto%2520PDB%2520structures%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D58%26issue%3D3%26spage%3D610%26epage%3D617%26doi%3D10.1002%2Fprot.20305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group">McLachlan, A. D.</span><span> </span><span class="NLM_article-title">Repeating sequences and gene duplication in proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">417</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1016/0022-2836(72)90508-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2F0022-2836%2872%2990508-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=1972&pages=417-437&issue=2&author=A.+D.+McLachlan&title=Repeating+sequences+and+gene+duplication+in+proteins&doi=10.1016%2F0022-2836%2872%2990508-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2872%2990508-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252872%252990508-6%26sid%3Dliteratum%253Aachs%26aulast%3DMcLachlan%26aufirst%3DA.%2BD.%26atitle%3DRepeating%2520sequences%2520and%2520gene%2520duplication%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1972%26volume%3D64%26issue%3D2%26spage%3D417%26epage%3D437%26doi%3D10.1016%2F0022-2836%2872%2990508-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group">Henikoff, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henikoff, J. G.</span><span> </span><span class="NLM_article-title">Amino acid substitution matrices from protein blocks</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x"> (</span><span class="NLM_issue">22</span><span class="NLM_x">) </span> <span class="NLM_fpage">10915</span><span class="NLM_x">–</span> <span class="NLM_lpage">10919</span><span class="refDoi"> DOI: 10.1073/pnas.89.22.10915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1073%2Fpnas.89.22.10915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1438297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK3sXjsFCgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=10915-10919&issue=22&author=S.+Henikoffauthor=J.+G.+Henikoff&title=Amino+acid+substitution+matrices+from+protein+blocks&doi=10.1073%2Fpnas.89.22.10915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid substitution matrixes from protein blocks</span></div><div class="casAuthors">Henikoff, Steven; Henikoff, Jorja G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10915-19</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Methods for alignment of protein sequences typically measure similarity by using a substitution matrix with scores for all possible exchanges of one amino acid with another.  The most widely used matrixes are based on the Dayhoff model of evolutionary rates.  Using a different approach, the authors derived substitution matrixes from about 2000 blocks of aligned sequence segments characterizing more than 500 groups of related proteins.  This led to marked improvements in alignments and in searches using queries from each of the groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY5y4-FlgzWrVg90H21EOLACvtfcHk0lgzjVOhf15R_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXjsFCgsQ%253D%253D&md5=3c4bee915654ac0b98b8d4aaafd44ebc</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1073%2Fpnas.89.22.10915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.89.22.10915%26sid%3Dliteratum%253Aachs%26aulast%3DHenikoff%26aufirst%3DS.%26aulast%3DHenikoff%26aufirst%3DJ.%2BG.%26atitle%3DAmino%2520acid%2520substitution%2520matrices%2520from%2520protein%2520blocks%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1992%26volume%3D89%26issue%3D22%26spage%3D10915%26epage%3D10919%26doi%3D10.1073%2Fpnas.89.22.10915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group">Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janežič, D.</span><span> </span><span class="NLM_article-title">An improved branch and bound algorithm for the maximum clique problem</span> <span class="citation_source-journal">MATCH</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">590</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2007&pages=569-590&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=An+improved+branch+and+bound+algorithm+for+the+maximum+clique+problem"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DAn%2520improved%2520branch%2520and%2520bound%2520algorithm%2520for%2520the%2520maximum%2520clique%2520problem%26jtitle%3DMATCH%26date%3D2007%26volume%3D58%26spage%3D569%26epage%3D590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group">Binkowski, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span> </span><span class="NLM_article-title">Inferring functional relationships of proteins from local sequence and spatial surface patterns</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">526</span><span class="refDoi"> DOI: 10.1016/S0022-2836(03)00882-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2803%2900882-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2003&pages=505-526&issue=2&author=T.+A.+Binkowskiauthor=L.+Adamianauthor=J.+Liang&title=Inferring+functional+relationships+of+proteins+from+local+sequence+and+spatial+surface+patterns&doi=10.1016%2FS0022-2836%2803%2900882-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2803%2900882-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252803%252900882-9%26sid%3Dliteratum%253Aachs%26aulast%3DBinkowski%26aufirst%3DT.%2BA.%26aulast%3DAdamian%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DJ.%26atitle%3DInferring%2520functional%2520relationships%2520of%2520proteins%2520from%2520local%2520sequence%2520and%2520spatial%2520surface%2520patterns%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D332%26issue%3D2%26spage%3D505%26epage%3D526%26doi%3D10.1016%2FS0022-2836%2803%2900882-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group">Binkowski, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naghibzadeh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span> </span><span class="NLM_article-title">CASTp: Computed Atlas of Surface Topography of proteins</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x"> (</span><span class="NLM_issue">13</span><span class="NLM_x">) </span> <span class="NLM_fpage">3352</span><span class="NLM_x">–</span> <span class="NLM_lpage">3355</span><span class="refDoi"> DOI: 10.1093/nar/gkg512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkg512" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=3352-3355&issue=13&author=T.+A.+Binkowskiauthor=S.+Naghibzadehauthor=J.+Liang&title=CASTp%3A+Computed+Atlas+of+Surface+Topography+of+proteins&doi=10.1093%2Fnar%2Fgkg512"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkg512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkg512%26sid%3Dliteratum%253Aachs%26aulast%3DBinkowski%26aufirst%3DT.%2BA.%26aulast%3DNaghibzadeh%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DJ.%26atitle%3DCASTp%253A%2520Computed%2520Atlas%2520of%2520Surface%2520Topography%2520of%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2003%26volume%3D31%26issue%3D13%26spage%3D3352%26epage%3D3355%26doi%3D10.1093%2Fnar%2Fgkg512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group">Pearson, W. R.</span><span> </span><span class="NLM_article-title">Searching protein sequence libraries: comparison of the sensitivity and selectivity of the Smith-Waterman and FASTA algorithms</span> <span class="citation_source-journal">Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">635</span><span class="NLM_x">–</span> <span class="NLM_lpage">650</span><span class="refDoi"> DOI: 10.1016/0888-7543(91)90071-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2F0888-7543%2891%2990071-L" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=635-650&issue=3&author=W.+R.+Pearson&title=Searching+protein+sequence+libraries%3A+comparison+of+the+sensitivity+and+selectivity+of+the+Smith-Waterman+and+FASTA+algorithms&doi=10.1016%2F0888-7543%2891%2990071-L"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1016%2F0888-7543%2891%2990071-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0888-7543%252891%252990071-L%26sid%3Dliteratum%253Aachs%26aulast%3DPearson%26aufirst%3DW.%2BR.%26atitle%3DSearching%2520protein%2520sequence%2520libraries%253A%2520comparison%2520of%2520the%2520sensitivity%2520and%2520selectivity%2520of%2520the%2520Smith-Waterman%2520and%2520FASTA%2520algorithms%26jtitle%3DGenomics%26date%3D1991%26volume%3D11%26issue%3D3%26spage%3D635%26epage%3D650%26doi%3D10.1016%2F0888-7543%2891%2990071-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group">Shulman-Peleg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfson, H. J.</span><span> </span><span class="NLM_article-title">SiteEngines: recognition and comparison of binding sites and protein-protein interfaces</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">W337</span><span class="NLM_x">–</span> <span class="NLM_lpage">W341</span><div class="note"><p class="first last">(Web Server issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gki482</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgki482" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=W337-W341&issue=Suppl.+2&author=A.+Shulman-Pelegauthor=R.+Nussinovauthor=H.+J.+Wolfson&title=SiteEngines%3A+recognition+and+comparison+of+binding+sites+and+protein-protein+interfaces&doi=10.1093%2Fnar%2Fgki482"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki482%26sid%3Dliteratum%253Aachs%26aulast%3DShulman-Peleg%26aufirst%3DA.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DWolfson%26aufirst%3DH.%2BJ.%26atitle%3DSiteEngines%253A%2520recognition%2520and%2520comparison%2520of%2520binding%2520sites%2520and%2520protein-protein%2520interfaces%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26issue%3DSuppl.%25202%26spage%3DW337%26epage%3DW341%26doi%3D10.1093%2Fnar%2Fgki482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group">Kabsch, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, C.</span><span> </span><span class="NLM_article-title">Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2577</span><span class="NLM_x">–</span> <span class="NLM_lpage">2637</span><span class="refDoi"> DOI: 10.1002/bip.360221211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fbip.360221211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=6667333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaL2cXkslegtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1983&pages=2577-2637&issue=12&author=W.+Kabschauthor=C.+Sander&title=Dictionary+of+protein+secondary+structure%3A+pattern+recognition+of+hydrogen-bonded+and+geometrical+features&doi=10.1002%2Fbip.360221211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Dictionary of protein secondary structure:  pattern recognition of hydrogen-bonded and geometrical features</span></div><div class="casAuthors">Kabsch, Wolfgang; Sander, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2577-637</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">For a successful anal. of the relation between amino acid sequence and protein structure, an unambiguous and phys. meaningful definition of secondary structure is essential.  A set of simple and phys. motivated criteria for secondary structure, programmed as a pattern-recognition process of H-bonded and geometrical features extd. from x-ray coordinates were developed.  Cooperative secondary structure is recognized as repeats of the elementary H-bonding patterns turn and bridge.  Repeating turns are helixes, repeating bridges are ladders, connected ladders are sheets.  Geometric structure is defined in terms of the concepts torsion and curvature of differential geometry.  Local chain chirality is the torsional handedness of 4 consecutive Cα positions and is pos. for right-handed helixes and neg. for ideal twisted β-sheets.  Curved pieces are defined as bends.  Solvent exposure is given as the no. of H2O mols. in possible contact with a residue.  The end result is a compilation of the primary structure, including SS bonds, secondary structure, and solvent exposure of 62 different globular proteins.  The presentation is in linear form: strip graphs for an overall view and strip tables for the details of each of 10,925 residues.  The dictionary is also available in computer-readable form for protein structure prediction work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph0hYRRkAqkbVg90H21EOLACvtfcHk0liAu8kRs3wvkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXkslegtQ%253D%253D&md5=a146e923e6c49cb542d0ad24a399d0f0</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1002%2Fbip.360221211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360221211%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26aulast%3DSander%26aufirst%3DC.%26atitle%3DDictionary%2520of%2520protein%2520secondary%2520structure%253A%2520pattern%2520recognition%2520of%2520hydrogen-bonded%2520and%2520geometrical%2520features%26jtitle%3DBiopolymers%26date%3D1983%26volume%3D22%26issue%3D12%26spage%3D2577%26epage%3D2637%26doi%3D10.1002%2Fbip.360221211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group">Kihara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">The PDB is a covering set of small protein structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">334</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">793</span><span class="NLM_x">–</span> <span class="NLM_lpage">802</span><span class="refDoi"> DOI: 10.1016/j.jmb.2003.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.jmb.2003.10.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2003&pages=793-802&issue=4&author=D.+Kiharaauthor=J.+Skolnick&title=The+PDB+is+a+covering+set+of+small+protein+structures&doi=10.1016%2Fj.jmb.2003.10.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2003.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2003.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DKihara%26aufirst%3DD.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DThe%2520PDB%2520is%2520a%2520covering%2520set%2520of%2520small%2520protein%2520structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D334%26issue%3D4%26spage%3D793%26epage%3D802%26doi%3D10.1016%2Fj.jmb.2003.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group">Dayhoff, M. O.; Schwartz, R. M.; Orcutt, B. C.</span><span> </span><span class="NLM_article-title">A model of evolutionary change in proteins</span>. In  <span class="citation_source-book">Atlas of Protein Sequence and Structure</span>; <span class="NLM_contrib-group">Dayhoff, M. O.</span>, Ed.; <span class="NLM_publisher-name">National Biomedical Research Foundation</span>: <span class="NLM_publisher-loc">Washington, DC</span>,<span class="NLM_x"> </span><span class="NLM_year">1978</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">345</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1978&pages=345-352&author=M.+O.+Dayhoff&author=R.+M.+Schwartz&author=B.+C.+Orcuttauthor=M.+O.+Dayhoff&title=Atlas+of+Protein+Sequence+and+Structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDayhoff%26aufirst%3DM.%2BO.%26atitle%3DA%2520model%2520of%2520evolutionary%2520change%2520in%2520proteins%26btitle%3DAtlas%2520of%2520Protein%2520Sequence%2520and%2520Structure%26aulast%3DDayhoff%26aufirst%3DM.%2BO.%26pub%3DNational%2520Biomedical%2520Research%2520Foundation%26date%3D1978%26spage%3D345%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group">Tseng, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupree, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.-H.</span><span> </span><span class="NLM_article-title">SplitPocket: identification of protein functional surfaces and characterization of their spatial patterns</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">W384</span><span class="NLM_x">–</span> <span class="NLM_lpage">W389</span><div class="note"><p class="first last">(Web Server issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkp308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkp308" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=W384-W389&issue=Suppl.+2&author=Y.+Y.+Tsengauthor=C.+Dupreeauthor=Z.+J.+Chenauthor=W.-H.+Li&title=SplitPocket%3A+identification+of+protein+functional+surfaces+and+characterization+of+their+spatial+patterns&doi=10.1093%2Fnar%2Fgkp308"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp308%26sid%3Dliteratum%253Aachs%26aulast%3DTseng%26aufirst%3DY.%2BY.%26aulast%3DDupree%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26aulast%3DLi%26aufirst%3DW.-H.%26atitle%3DSplitPocket%253A%2520identification%2520of%2520protein%2520functional%2520surfaces%2520and%2520characterization%2520of%2520their%2520spatial%2520patterns%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3DSuppl.%25202%26spage%3DW384%26epage%3DW389%26doi%3D10.1093%2Fnar%2Fgkp308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group">Wei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J. T.</span><span> </span><span class="NLM_article-title">Using the radial distributions of physical features to compare amino acid environments and align amino acid sequences</span> <span class="citation_source-journal">Pac. Symp. Biocomput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=9390315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2sXns1Sru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=465-476&author=L.+Weiauthor=R.+B.+Altmanauthor=J.+T.+Chang&title=Using+the+radial+distributions+of+physical+features+to+compare+amino+acid+environments+and+align+amino+acid+sequences"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Using the radial distributions of physical features to compare amino acid environments and align amino acid sequences</span></div><div class="casAuthors">Wei, Liping; Altman, Russ B.; Chang, Jeffrey T.</div><div class="citationInfo"><span class="NLM_cas:title">Pacific Symposium on Biocomputing '97, Maui, Hawaii, Jan. 6-9, 1997</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">465-476</span>CODEN:
                <span class="NLM_cas:coden">65JZAR</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">World Scientific</span>)
        </div><div class="casAbstract">We have performed a comprehensive anal. of the microenvironments surrounding the twenty amino acids.  Our anal. includes comparison of amino acid environments with random control environments as well as with each of the other amino acid environments.  We describe the amino acid environments with a set of 21 features summarizing at., chem. group, residue, and secondary structural features.  The environments are divided into radial shells of 1 Å thickness to represent the distance of the features from the amino acid Cβ atoms.  We make the results of our anal. available graphically over the world wide web.  To illustrate the validity and utility of our anal., we used the amino acid comparative profiles to construct a substitution matrix, the WAC matrix, based on a simple summary of the computed environmental differences.  We compared our matrix to BLOSUM62 and PAM250 in BLAST searches with query sequences selected from 39 protein families found in the PROSITE database.  Although BLOSUM62 was the most sensitive matrix overall, our matrix was more sensitive for some families, and exhibited overall performance similar to PAM250.  Our results suggest that the radial distribution of biochem. and biophys. features is useful for comparing amino acid environments, and that similarity matrixes based on the geometric distribution of features around amino acids may produce improved search sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp77neBQFhd17Vg90H21EOLACvtfcHk0liAu8kRs3wvkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXns1Sru7o%253D&md5=3c662d5cf77b2e4ab9cb344a4cf319ee</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DL.%26aulast%3DAltman%26aufirst%3DR.%2BB.%26aulast%3DChang%26aufirst%3DJ.%2BT.%26atitle%3DUsing%2520the%2520radial%2520distributions%2520of%2520physical%2520features%2520to%2520compare%2520amino%2520acid%2520environments%2520and%2520align%2520amino%2520acid%2520sequences%26jtitle%3DPac.%2520Symp.%2520Biocomput.%26date%3D1997%26spage%3D465%26epage%3D476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group">Yoon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebert, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, E.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Micheli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, R. B.</span><span> </span><span class="NLM_article-title">Clustering protein environments for function prediction: finding PROSITE motifs in 3D</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 4</span><span class="NLM_x">) </span> <span class="NLM_fpage">S10</span><span class="refDoi"> DOI: 10.1186/1471-2105-8-S4-S10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-8-S4-S10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=S10&issue=Suppl.+4&author=S.+Yoonauthor=J.+C.+Ebertauthor=E.-Y.+Chungauthor=G.+De+Micheliauthor=R.+B.+Altman&title=Clustering+protein+environments+for+function+prediction%3A+finding+PROSITE+motifs+in+3D&doi=10.1186%2F1471-2105-8-S4-S10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-S4-S10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-8-S4-S10%26sid%3Dliteratum%253Aachs%26aulast%3DYoon%26aufirst%3DS.%26aulast%3DEbert%26aufirst%3DJ.%2BC.%26aulast%3DChung%26aufirst%3DE.-Y.%26aulast%3DDe%2BMicheli%26aufirst%3DG.%26aulast%3DAltman%26aufirst%3DR.%2BB.%26atitle%3DClustering%2520protein%2520environments%2520for%2520function%2520prediction%253A%2520finding%2520PROSITE%2520motifs%2520in%25203D%26jtitle%3DBMC%2520Bioinf.%26date%3D2007%26volume%3D8%26issue%3DSuppl.%25204%26spage%3DS10%26doi%3D10.1186%2F1471-2105-8-S4-S10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Protein structure alignment by incremental combinatorial extension (CE) of the optimal path</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">739</span><span class="NLM_x">–</span> <span class="NLM_lpage">747</span><span class="refDoi"> DOI: 10.1093/protein/11.9.739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fprotein%2F11.9.739" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=739-747&issue=9&author=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=Protein+structure+alignment+by+incremental+combinatorial+extension+%28CE%29+of+the+optimal+path&doi=10.1093%2Fprotein%2F11.9.739"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F11.9.739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F11.9.739%26sid%3Dliteratum%253Aachs%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProtein%2520structure%2520alignment%2520by%2520incremental%2520combinatorial%2520extension%2520%2528CE%2529%2520of%2520the%2520optimal%2520path%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1998%26volume%3D11%26issue%3D9%26spage%3D739%26epage%3D747%26doi%3D10.1093%2Fprotein%2F11.9.739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x"> (</span><span class="NLM_issue">15</span><span class="NLM_x">) </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span><span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&issue=15&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lj8VLO_IJNckA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26issue%3D15%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group">Das, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krein, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breneman, C. M.</span><span> </span><span class="NLM_article-title">PESDserv: a server for high-throughput comparison of protein binding site surfaces</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x"> (</span><span class="NLM_issue">15</span><span class="NLM_x">) </span> <span class="NLM_fpage">1913</span><span class="NLM_x">–</span> <span class="NLM_lpage">1914</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btq288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtq288" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1913-1914&issue=15&author=S.+Dasauthor=M.+P.+Kreinauthor=C.+M.+Breneman&title=PESDserv%3A+a+server+for+high-throughput+comparison+of+protein+binding+site+surfaces&doi=10.1093%2Fbioinformatics%2Fbtq288"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq288%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DS.%26aulast%3DKrein%26aufirst%3DM.%2BP.%26aulast%3DBreneman%26aufirst%3DC.%2BM.%26atitle%3DPESDserv%253A%2520a%2520server%2520for%2520high-throughput%2520comparison%2520of%2520protein%2520binding%2520site%2520surfaces%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26issue%3D15%26spage%3D1913%26epage%3D1914%26doi%3D10.1093%2Fbioinformatics%2Fbtq288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group">Zhu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Gharabli, S. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rademann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilgenfeld, R.</span><span> </span><span class="NLM_article-title">Peptide aldehyde inhibitors challenge the substrate specificity of the SARS-coronavirus main protease</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">204</span><span class="NLM_x">–</span> <span class="NLM_lpage">212</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2011.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.antiviral.2011.08.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2011&pages=204-212&issue=2&author=L.+Zhuauthor=S.+Georgeauthor=M.+F.+Schmidtauthor=S.+I.+Al-Gharabliauthor=J.+Rademannauthor=R.+Hilgenfeld&title=Peptide+aldehyde+inhibitors+challenge+the+substrate+specificity+of+the+SARS-coronavirus+main+protease&doi=10.1016%2Fj.antiviral.2011.08.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2011.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2011.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DGeorge%26aufirst%3DS.%26aulast%3DSchmidt%26aufirst%3DM.%2BF.%26aulast%3DAl-Gharabli%26aufirst%3DS.%2BI.%26aulast%3DRademann%26aufirst%3DJ.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DPeptide%2520aldehyde%2520inhibitors%2520challenge%2520the%2520substrate%2520specificity%2520of%2520the%2520SARS-coronavirus%2520main%2520protease%26jtitle%3DAntiviral%2520Res.%26date%3D2011%26volume%3D92%26issue%3D2%26spage%3D204%26epage%3D212%26doi%3D10.1016%2Fj.antiviral.2011.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group">Willmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jandausch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forne, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imhof, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">2704</span><span class="NLM_x">–</span> <span class="NLM_lpage">2709</span><span class="refDoi"> DOI: 10.1002/ijc.27555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fijc.27555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22447389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2012&pages=2704-2709&issue=11&author=D.+Willmannauthor=S.+Limauthor=S.+Wetzelauthor=E.+Metzgerauthor=A.+Jandauschauthor=W.+Wilkauthor=M.+Jungauthor=I.+Forneauthor=A.+Imhofauthor=A.+Janzerauthor=J.+Kirfelauthor=H.+Waldmannauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Impairment+of+prostate+cancer+cell+growth+by+a+selective+and+reversible+lysine-specific+demethylase+1+inhibitor&doi=10.1002%2Fijc.27555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span></div><div class="casAuthors">Willmann, Dominica; Lim, Soyoung; Wetzel, Stefan; Metzger, Eric; Jandausch, Anett; Wilk, Wolfram; Jung, Manfred; Forne, Ignasi; Imhof, Axel; Janzer, Andreas; Kirfel, Jutta; Waldmann, Herbert; Schuele, Roland; Buettner, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2704-2709</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Post-translational modifications of histones by chromatin modifying enzymes regulate chromatin structure and gene expression.  As deregulation of histone modifications contributes to cancer progression, inhibition of chromatin modifying enzymes such as histone demethylases is an attractive therapeutic strategy to impair cancer growth.  Lysine-specific demethylase 1 (LSD1) removes mono- and di-Me marks from lysine 4 or 9 of histone H3.  LSD1 in assocn. with the androgen receptor (AR) controls androgen-dependent gene expression and prostate tumor cell proliferation, thus highlighting LSD1 as a drug target.  By combining protein structure similarity clustering and in vitro screening, we identified Namoline, a γ-pyrone, as a novel, selective and reversible LSD1 inhibitor.  Namoline blocks LSD1 demethylase activity in vitro and in vivo.  Inhibition of LSD1 by Namoline leads to silencing of AR-regulated gene expression and severely impairs androgen-dependent proliferation in vitro and in vivo.  Thus, Namoline is a novel promising starting compd. for the development of therapeutics to treat androgen-dependent prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKxYi0f1zoOrVg90H21EOLACvtfcHk0lj8VLO_IJNckA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D&md5=3104057823357b199b635e1a22071b2a</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1002%2Fijc.27555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27555%26sid%3Dliteratum%253Aachs%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DJandausch%26aufirst%3DA.%26aulast%3DWilk%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DForne%26aufirst%3DI.%26aulast%3DImhof%26aufirst%3DA.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DImpairment%2520of%2520prostate%2520cancer%2520cell%2520growth%2520by%2520a%2520selective%2520and%2520reversible%2520lysine-specific%2520demethylase%25201%2520inhibitor%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D131%26issue%3D11%26spage%3D2704%26epage%3D2709%26doi%3D10.1002%2Fijc.27555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group">Holm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenström, P.</span><span> </span><span class="NLM_article-title">Dali server: conservation mapping in 3D</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">W545</span><span class="NLM_x">–</span> <span class="NLM_lpage">W549</span><div class="note"><p class="first last">(Web Server issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkq366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkq366" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W545-W549&issue=Suppl.+2&author=L.+Holmauthor=P.+Rosenstr%C3%B6m&title=Dali+server%3A+conservation+mapping+in+3D&doi=10.1093%2Fnar%2Fgkq366"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq366%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DRosenstr%25C3%25B6m%26aufirst%3DP.%26atitle%3DDali%2520server%253A%2520conservation%2520mapping%2520in%25203D%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26issue%3DSuppl.%25202%26spage%3DW545%26epage%3DW549%26doi%3D10.1093%2Fnar%2Fgkq366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chammaa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sperl, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yektaoglu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arenz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">A scaffold-tree-merging strategy for prospective bioactivity annotation of gamma-pyrones</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">21</span><span class="NLM_x">) </span> <span class="NLM_fpage">3666</span><span class="NLM_x">–</span> <span class="NLM_lpage">3670</span><span class="refDoi"> DOI: 10.1002/anie.200906555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fanie.200906555" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=3666-3670&issue=21&author=S.+Wetzelauthor=W.+Wilkauthor=S.+Chammaaauthor=B.+Sperlauthor=A.+G.+Rothauthor=A.+Yektaogluauthor=S.+Rennerauthor=T.+Bergauthor=C.+Arenzauthor=A.+Giannisauthor=T.+I.+Opreaauthor=D.+Rauhauthor=M.+Kaiserauthor=H.+Waldmann&title=A+scaffold-tree-merging+strategy+for+prospective+bioactivity+annotation+of+gamma-pyrones&doi=10.1002%2Fanie.200906555"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1002%2Fanie.200906555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200906555%26sid%3Dliteratum%253Aachs%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DWilk%26aufirst%3DW.%26aulast%3DChammaa%26aufirst%3DS.%26aulast%3DSperl%26aufirst%3DB.%26aulast%3DRoth%26aufirst%3DA.%2BG.%26aulast%3DYektaoglu%26aufirst%3DA.%26aulast%3DRenner%26aufirst%3DS.%26aulast%3DBerg%26aufirst%3DT.%26aulast%3DArenz%26aufirst%3DC.%26aulast%3DGiannis%26aufirst%3DA.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DKaiser%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DA%2520scaffold-tree-merging%2520strategy%2520for%2520prospective%2520bioactivity%2520annotation%2520of%2520gamma-pyrones%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26issue%3D21%26spage%3D3666%26epage%3D3670%26doi%3D10.1002%2Fanie.200906555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span> <span class="citation_source-book">UCSF Chimera: A Visualization System for Exploratory Research and Analysis</span>; <span class="NLM_publisher-name">Resource for Biocomputing, Visualization, and Informatics. Computer Graphics Laboratory, Department of Pharmaceutical Chemistry, University of California</span> (<span class="NLM_publisher-loc">600 16th Street, San Francisco, California 94143-2240, U.S.</span>),<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=UCSF+Chimera%3A+A+Visualization+System+for+Exploratory+Research+and+Analysis%3B+Resource+for+Biocomputing%2C+Visualization%2C+and+Informatics.+Computer+Graphics+Laboratory%2C+Department+of+Pharmaceutical+Chemistry%2C+University+of+California+%28600+16th+Street%2C+San+Francisco%2C+California+94143-2240%2C+U.S.%29%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DUCSF%2520Chimera%253A%2520A%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26pub%3DResource%2520for%2520Biocomputing%252C%2520Visualization%252C%2520and%2520Informatics.%2520Computer%2520Graphics%2520Laboratory%252C%2520Department%2520of%2520Pharmaceutical%2520Chemistry%252C%2520University%2520of%2520California%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group">Rajagopal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajagopal, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefkowitz, R. J.</span><span> </span><span class="NLM_article-title">Teaching old receptors new tricks: biasing seven-transmembrane receptors</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">373</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1038/nrd3024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fnrd3024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=20431569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVWjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=373-386&issue=5&author=S.+Rajagopalauthor=K.+Rajagopalauthor=R.+J.+Lefkowitz&title=Teaching+old+receptors+new+tricks%3A+biasing+seven-transmembrane+receptors&doi=10.1038%2Fnrd3024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Teaching old receptors new tricks: biasing seven-transmembrane receptors</span></div><div class="casAuthors">Rajagopal, Sudarshan; Rajagopal, Keshava; Lefkowitz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">373-386</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Seven-transmembrane receptors (7TMRs; also known as G protein-coupled receptors) are the largest class of receptors in the human genome and are common targets for therapeutics.  Originally identified as mediators of 7TMR desensitization, β-arrestins (arrestin 2 and arrestin 3) are now recognized as true adaptor proteins that transduce signals to multiple effector pathways.  Signaling that is mediated by β-arrestins has distinct biochem. and functional consequences from those mediated by G proteins, and several biased ligands and receptors have been identified that preferentially signal through either G protein- or β-arrestin-mediated pathways.  These ligands are not only useful tools for investigating the biochem. of 7TMR signaling, but they also have the potential to be developed into new classes of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGuN6jivmOErVg90H21EOLACvtfcHk0lgrE-mLHXMedA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVWjs7k%253D&md5=ff525016c0ba5aa0ebc4ed86f87594d0</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1038%2Fnrd3024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3024%26sid%3Dliteratum%253Aachs%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DRajagopal%26aufirst%3DK.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26atitle%3DTeaching%2520old%2520receptors%2520new%2520tricks%253A%2520biasing%2520seven-transmembrane%2520receptors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26issue%3D5%26spage%3D373%26epage%3D386%26doi%3D10.1038%2Fnrd3024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group">Issa, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byers, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakshanamurthy, S.</span><span> </span><span class="NLM_article-title">Drug repurposing: translational pharmacology, chemistry, computers and the clinic</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">18</span><span class="NLM_x">) </span> <span class="NLM_fpage">2328</span><span class="NLM_x">–</span> <span class="NLM_lpage">2336</span><span class="refDoi"> DOI: 10.2174/15680266113136660163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2174%2F15680266113136660163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24059462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CgurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=2328-2336&issue=18&author=N.+T.+Issaauthor=S.+W.+Byersauthor=S.+Dakshanamurthy&title=Drug+repurposing%3A+translational+pharmacology%2C+chemistry%2C+computers+and+the+clinic&doi=10.2174%2F15680266113136660163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Repurposing: Translational Pharmacology, Chemistry, Computers and the Clinic</span></div><div class="casAuthors">Issa, Naiem T.; Byers, Stephen W.; Dakshanamurthy, Sivanesan</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2328-2336</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The process of discovering a pharmacol. compd. that elicits a desired clin. effect with minimal side effects is a challenge.  Prior to the advent of high-performance computing and large-scale screening technologies, drug discovery was largely a serendipitous endeavor, as in the case of thalidomide for erythema nodosum leprosum or cancer drugs in general derived from flora located in far-reaching geog. locations.  More recently, de novo drug discovery has become a more rationalized process where drug-target-effect hypotheses are formulated on the basis of already known compds./protein targets and their structures.  Although this approach is hypothesis-driven, the actual success has been very low, contributing to the soaring costs of research and development as well as the diminished pharmaceutical pipeline in the United States.  In this review, we discuss the evolution in computational pharmacol. as the next generation of successful drug discovery and implementation in the clinic where high-performance computing (HPC) is used to generate and validate drug-target-effect hypotheses completely in silico.  The use of HPC would decrease development time and errors while increasing productivity prior to in vitro, animal and human testing.  We highlight approaches in chemoinformatics, bioinformatics as well as network biopharmacol. to illustrate potential avenues from which to design clin. efficacious drugs.  We further discuss the implications of combining these approaches into an integrative methodol. for high-accuracy computational predictions within the context of drug repositioning for the efficient streamlining of currently approved drugs back into clin. trials for possible new indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZiiRtN0LnbLVg90H21EOLACvtfcHk0lgrE-mLHXMedA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CgurzL&md5=4c952bdfdd8171df353c0d3fe0778ad6</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.2174%2F15680266113136660163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F15680266113136660163%26sid%3Dliteratum%253Aachs%26aulast%3DIssa%26aufirst%3DN.%2BT.%26aulast%3DByers%26aufirst%3DS.%2BW.%26aulast%3DDakshanamurthy%26aufirst%3DS.%26atitle%3DDrug%2520repurposing%253A%2520translational%2520pharmacology%252C%2520chemistry%252C%2520computers%2520and%2520the%2520clinic%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26issue%3D18%26spage%3D2328%26epage%3D2336%26doi%3D10.2174%2F15680266113136660163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group">Haupt, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M.</span><span> </span><span class="NLM_article-title">Old friends in new guise: repositioning of known drugs with structural bioinformatics</span> <span class="citation_source-journal">Briefings Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span><span class="refDoi"> DOI: 10.1093/bib/bbr011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbib%2Fbbr011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21441562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptl2jsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=312-326&issue=4&author=V.+J.+Hauptauthor=M.+Schroeder&title=Old+friends+in+new+guise%3A+repositioning+of+known+drugs+with+structural+bioinformatics&doi=10.1093%2Fbib%2Fbbr011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Old friends in new guise: repositioning of known drugs with structural bioinformatics</span></div><div class="casAuthors">Haupt, V. Joachim; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-326</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1467-5463</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Developing a drug de novo is a laborious and costly endeavor.  Thus, the repositioning of already approved drugs for the treatment of new diseases is promising and valuable.  One computational approach to repositioning exploits the structural similarity of binding sites of known and new targets.  Here, we review computational methods to represent and align binding sites.  We review available tools, present success stories and discuss limits of the approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeO0NDH5GZrrVg90H21EOLACvtfcHk0lg2feqD0DoKVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptl2jsrc%253D&md5=e50e0f4752302d91263af4803ef0f14d</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbr011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbr011%26sid%3Dliteratum%253Aachs%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DOld%2520friends%2520in%2520new%2520guise%253A%2520repositioning%2520of%2520known%2520drugs%2520with%2520structural%2520bioinformatics%26jtitle%3DBriefings%2520Bioinf.%26date%3D2011%26volume%3D12%26issue%3D4%26spage%3D312%26epage%3D326%26doi%3D10.1093%2Fbib%2Fbbr011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group">Baek, M.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bae, J.-S.</span><span> </span><span class="NLM_article-title">Novel insight into drug repositioning: methylthiouracil as a case in point</span> <span class="citation_source-journal">Pharmacol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">185</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1016/j.phrs.2015.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.phrs.2015.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=26117428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BC2Mbnslaitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=185-193&author=M.-C.+Baekauthor=B.+Jungauthor=H.+Kangauthor=H.-S.+Leeauthor=J.-S.+Bae&title=Novel+insight+into+drug+repositioning%3A+methylthiouracil+as+a+case+in+point&doi=10.1016%2Fj.phrs.2015.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Novel insight into drug repositioning: Methylthiouracil as a case in point</span></div><div class="casAuthors">Baek Moon-Chang; Jung Byeongjin; Kang Hyejin; Lee Hyun-Shik; Bae Jong-Sup</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">185-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug repositioning refers to the development of existing drugs for new indications.  These drugs may have (I) failed to show efficacy in late stage clinical trials without safety issues; (II) stalled in the development for commercial reasons; (III) passed the point of patent expiry; or (IV) are being explored in new geographic markets.  Over the past decade, pressure on the pharmaceutical industry caused by the 'innovation gap' owing to rising development costs and stagnant product output have become major reasons for the growing interest in drug repositioning.  Companies that offer a variety of broad platforms for identifying new indications have emerged; some have been successful in building their own pipelines of candidates with reduced risks and timelines associated with further clinical development.  The business models and platforms offered by these companies will be validated if they are able to generate positive proof-of-concept clinical data for their repositioned compounds.  This review describes the strategy of biomarker-guided repositioning of chemotherapeutic drugs for inflammation therapy, considering the repositioning of methylthiouracil (MTU), an antithyroid drug, as a potential anti-inflammatory reagent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2LxaOWNLBmNbHMBt0mMIXfW6udTcc2ebERmT55K18Jrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mbnslaitw%253D%253D&md5=2094e593085461842f9ebfec9cc7c29f</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DBaek%26aufirst%3DM.-C.%26aulast%3DJung%26aufirst%3DB.%26aulast%3DKang%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DBae%26aufirst%3DJ.-S.%26atitle%3DNovel%2520insight%2520into%2520drug%2520repositioning%253A%2520methylthiouracil%2520as%2520a%2520case%2520in%2520point%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D185%26epage%3D193%26doi%3D10.1016%2Fj.phrs.2015.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group">O’Boyle, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeersch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, G. R.</span><span> </span><span class="NLM_article-title">Open Babel: An open chemical toolbox</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">33</span><span class="refDoi"> DOI: 10.1186/1758-2946-3-33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21982300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&issue=1&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+An+open+chemical+toolbox&doi=10.1186%2F1758-2946-3-33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Open Babel: an open chemical toolbox</span></div><div class="casAuthors">O'Boyle, Noel M.; Banck, Michael; James, Craig A.; Morley, Chris; Vandermeersch, Tim; Hutchison, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: A frequent problem in computational modeling is the interconversion of chem. structures between different formats.  While std. interchange formats exist (for example, Chem. Markup Language) and de facto stds. have arisen (for example, SMILES format), the need to interconvert formats is a continuing problem due to the multitude of different application areas for chem. data, differences in the data stored by different formats (0D vs. 3D, for example), and competition between software along with a lack of vendor-neutral formats.  Results: We discuss, for the first time, Open Babel, an open-source chem. toolbox that speaks the many languages of chem. data.  Open Babel version 2.3 interconverts over 110 formats.  The need to represent such a wide variety of chem. and mol. data requires a library that implements a wide range of cheminformatics algorithms, from partial charge assignment and aromaticity detection, to bond order perception and canonicalization.  We detail the implementation of Open Babel, describe key advances in the 2.3 release, and outline a variety of uses both in terms of software products and scientific research, including applications far beyond simple format interconversion.  Conclusions: Open Babel presents a soln. to the proliferation of multiple chem. file formats.  In addn., it provides a variety of useful utilities from conformer searching and 2D depiction, to filtering, batch conversion, and substructure and similarity searching.  For developers, it can be used as a programming library to handle chem. data in areas such as org. chem., drug design, materials science, and computational chem.  It is freely available under an open-source license.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfQ1lvaS5767Vg90H21EOLACvtfcHk0lg2feqD0DoKVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF&md5=74e4f19b7f87417f916d57f7abcfb761</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520An%2520open%2520chemical%2520toolbox%26jtitle%3DJ.%2520Cheminf.%26date%3D2011%26volume%3D3%26issue%3D1%26spage%3D33%26doi%3D10.1186%2F1758-2946-3-33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group">Rahman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bashton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holliday, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrader, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">12</span><span class="refDoi"> DOI: 10.1186/1758-2946-1-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1758-2946-1-12" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Rahman%2C+S.+A.%3B+Bashton%2C+M.%3B+Holliday%2C+G.+L.%3B+Schrader%2C+R.%3B+Thornton%2C+J.+M.J.+Cheminf.+2009%2C+1+%281%29+12+10.1186%2F1758-2946-1-12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-1-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-1-12%26sid%3Dliteratum%253Aachs%26aulast%3DRahman%26aufirst%3DS.%2BA.%26aulast%3DBashton%26aufirst%3DM.%26aulast%3DHolliday%26aufirst%3DG.%2BL.%26aulast%3DSchrader%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26jtitle%3DJ.%2520Cheminf.%26date%3D2009%26volume%3D1%26issue%3D1%26spage%3D12%26doi%3D10.1186%2F1758-2946-1-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">BindingDB: a web-accessible molecular recognition database</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">719</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.2174/1386207013330670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2174%2F1386207013330670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=11812264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Orsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2001&pages=719-725&issue=8&author=X.+Chenauthor=M.+Liuauthor=M.+K.+Gilson&title=BindingDB%3A+a+web-accessible+molecular+recognition+database&doi=10.2174%2F1386207013330670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: A Web-accessible molecular recognition database</span></div><div class="casAuthors">Chen, Xi; Liu, Ming; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">719-725</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">The BindingDB, a public web-accessible database of measured binding affinities for various mol. types, is described.  It allows queries based upon a range of criteria, including chem. similarity or substructure, sequence homol., numerical criteria and reactant names.  Principles of Human-Computer Interactions are being employed in creating the query interface and user-feedback is being solicited.  The data specification includes significant exptl. detail.  A full dictionary has been created for isothermal titrn. calorimetry data in consultation with experimentalists and data dictionaries for enzyme inhibition and other measurement techniques are being developed.  Currently, the BindingDB contains several data sets of broad interest, such as antigen-antibody binding and cyclodextrin/small-mol. binding.  However, it is anticipated that online deposition by experimentalists will ultimately contribute a larger flow of data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzRKVfJZtKwrVg90H21EOLACvtfcHk0lg2feqD0DoKVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Orsg%253D%253D&md5=9db61dc1ad10e8191ee11afcfa9e337e</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.2174%2F1386207013330670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207013330670%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520web-accessible%2520molecular%2520recognition%2520database%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2001%26volume%3D4%26issue%3D8%26spage%3D719%26epage%3D725%26doi%3D10.2174%2F1386207013330670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group">Lang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span> </span><span class="NLM_article-title">Identification of putative molecular imaging probes for BACE-1 by accounting for protein flexibility in virtual screening</span> <span class="citation_source-journal">J. Alzheimer’s Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">351</span><span class="NLM_x">–</span> <span class="NLM_lpage">359</span><span class="refDoi"> DOI: 10.3233/JAD-2011-111787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.3233%2FJAD-2011-111787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22232010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFWltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=351-359&issue=2&author=H.+Langauthor=X.+Huangauthor=Y.+Yang&title=Identification+of+putative+molecular+imaging+probes+for+BACE-1+by+accounting+for+protein+flexibility+in+virtual+screening&doi=10.3233%2FJAD-2011-111787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Putative Molecular Imaging Probes for BACE-1 by Accounting for Protein Flexibility in Virtual Screening</span></div><div class="casAuthors">Lang, Haijing; Huang, Xudong; Yang, Yongliang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Alzheimer's Disease</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-359</span>CODEN:
                <span class="NLM_cas:coden">JADIF9</span>;
        ISSN:<span class="NLM_cas:issn">1387-2877</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">β-Secretase (BACE-1), an enzyme crit. in the process of amyloid-β (Aβ) peptides deposition in human brain, is closely assocd. with the onset and progression of Alzheimer's disease (AD).  A strong need exists, therefore, to identify mol. imaging probes homing at BACE-1 for use with positron emission tomog. (PET) that is recognized as an effective tool for detecting AD.  Through this imaging, an early diagnosis of AD could be made.  Herein, to identify suitable mol. probes for use with PET, we searched the Mol. Imaging and Contrast Agent Database (MICAD), an online database warehousing scientific information regarding mol. imaging and contrast agents, and applied a virtual screening approach against the different confirmations of BACE-1 obtained from the World Wide Protein Database.  The lack of considering receptor flexibility is a key drawback in virtual screening for drug discovery.  Therefore, we incorporated protein flexibility into the virtual screening by using an ensemble of 143 exptl. BACE-1 structures derived from the Protein Data Bank.  Finally, the best performing affinity was recorded and used in the ranking of each ligand.  To the best of our knowledge, this is the first virtual screening approach used to identify four new mol. probes that could target BACE-1 with favorable affinity, a discovery that can lead to the development of new PET probes for the early detection and therapy of AD.  However, the actual utility of these probes can only be ascertained after in vitro and in vivo investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYAMyfzjvfF7Vg90H21EOLACvtfcHk0ljgRaJNnjLx-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFWltLs%253D&md5=689a65e19129942234976a61464e006a</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.3233%2FJAD-2011-111787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJAD-2011-111787%26sid%3Dliteratum%253Aachs%26aulast%3DLang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DIdentification%2520of%2520putative%2520molecular%2520imaging%2520probes%2520for%2520BACE-1%2520by%2520accounting%2520for%2520protein%2520flexibility%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Alzheimer%25E2%2580%2599s%2520Dis.%26date%3D2012%26volume%3D29%26issue%3D2%26spage%3D351%26epage%3D359%26doi%3D10.3233%2FJAD-2011-111787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinzi, L.</span><span> </span><span class="NLM_article-title">Computational polypharmacology comes of age</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">7874</span><span class="refDoi"> DOI: 10.3389/fphar.2015.00157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.3389%2Ffphar.2015.00157" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7874&author=G.+Rastelliauthor=L.+Pinzi&title=Computational+polypharmacology+comes+of+age&doi=10.3389%2Ffphar.2015.00157"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2015.00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2015.00157%26sid%3Dliteratum%253Aachs%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DPinzi%26aufirst%3DL.%26atitle%3DComputational%2520polypharmacology%2520comes%2520of%2520age%26jtitle%3DFront.%2520Pharmacol.%26date%3D2015%26volume%3D6%26spage%3D7874%26doi%3D10.3389%2Ffphar.2015.00157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group">Salentin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haupt, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daminelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M.</span><span> </span><span class="NLM_article-title">Polypharmacology rescored: protein-ligand interaction profiles for remote binding site similarity assessment</span> <span class="citation_source-journal">Prog. Biophys. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x"> (</span><span class="NLM_issue">2–3</span><span class="NLM_x">) </span> <span class="NLM_fpage">174</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2014.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.pbiomolbio.2014.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24923864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jt7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2014&pages=174-186&issue=2%E2%80%933&author=S.+Salentinauthor=V.+J.+Hauptauthor=S.+Daminelliauthor=M.+Schroeder&title=Polypharmacology+rescored%3A+protein-ligand+interaction+profiles+for+remote+binding+site+similarity+assessment&doi=10.1016%2Fj.pbiomolbio.2014.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology rescored: Protein-ligand interaction profiles for remote binding site similarity assessment</span></div><div class="casAuthors">Salentin, Sebastian; Haupt, V. Joachim; Daminelli, Simone; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">174-186</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Detection of remote binding site similarity in proteins plays an important role for drug repositioning and off-target effect prediction.  Various non-covalent interactions such as hydrogen bonds and van-der-Waals forces drive ligands' mol. recognition by binding sites in proteins.  The increasing amt. of available structures of protein-small mol. complexes enabled the development of comparative approaches.  Several methods have been developed to characterize and compare protein-ligand interaction patterns.  Usually implemented as fingerprints, these are mainly used for post processing docking scores and (off-)target prediction.  In the latter application, interaction profiles detect similarities in the bound interactions of different ligands and thus identify essential interactions between a protein and its small mol. ligands.  Interaction pattern similarity correlates with binding site similarity and is thus contributing to a higher precision in binding site similarity assessment of proteins with distinct global structure.  This renders it valuable for existing drug repositioning approaches in structural bioinformatics.  Current methods to characterize and compare structure-based interaction patterns - both for protein-small-mol. and protein-protein interactions - as well as their potential in target prediction will be reviewed in this article.  The question of how the set of interaction types, flexibility or water-mediated interactions, influence the comparison of interaction patterns will be discussed.  Due to the wealth of protein-ligand structures available today, predicted targets can be ranked by comparing their ligand interaction pattern to patterns of the known target.  Such knowledge-based methods offer high precision in comparison to methods comparing whole binding sites based on shape and amino acid physicochem. similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeHhHCtFDIlLVg90H21EOLACvtfcHk0ljgRaJNnjLx-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jt7nN&md5=6c83751ccdf7cadd879df9aefff29af7</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2014.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2014.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DSalentin%26aufirst%3DS.%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DDaminelli%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DPolypharmacology%2520rescored%253A%2520protein-ligand%2520interaction%2520profiles%2520for%2520remote%2520binding%2520site%2520similarity%2520assessment%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D116%26issue%3D2%25E2%2580%25933%26spage%3D174%26epage%3D186%26doi%3D10.1016%2Fj.pbiomolbio.2014.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group">Gills, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopiccolo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsurutani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Best, C. J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abu-Asab, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borojerdi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warfel, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danish, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollander, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawabata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsokos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figg, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steeg, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennis, P. A.</span><span> </span><span class="NLM_article-title">Nelfinavir, A lead HIV protease inhibitor, is a broad-spectrum, anticancer agent that induces endoplasmic reticulum stress, autophagy, and apoptosis in vitro and in vivo</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">5183</span><span class="NLM_x">–</span> <span class="NLM_lpage">5194</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-07-0161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1158%2F1078-0432.CCR-07-0161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=17785575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFGqs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=5183-5194&issue=17&author=J.+J.+Gillsauthor=J.+Lopiccoloauthor=J.+Tsurutaniauthor=R.+H.+Shoemakerauthor=C.+J.+M.+Bestauthor=M.+S.+Abu-Asabauthor=J.+Borojerdiauthor=N.+A.+Warfelauthor=E.+R.+Gardnerauthor=M.+Danishauthor=M.+C.+Hollanderauthor=S.+Kawabataauthor=M.+Tsokosauthor=W.+D.+Figgauthor=P.+S.+Steegauthor=P.+A.+Dennis&title=Nelfinavir%2C+A+lead+HIV+protease+inhibitor%2C+is+a+broad-spectrum%2C+anticancer+agent+that+induces+endoplasmic+reticulum+stress%2C+autophagy%2C+and+apoptosis+in+vitro+and+in+vivo&doi=10.1158%2F1078-0432.CCR-07-0161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Nelfinavir, A Lead HIV Protease Inhibitor, Is a Broad-Spectrum, Anticancer Agent that Induces Endoplasmic Reticulum Stress, Autophagy, and Apoptosis In vitro and In vivo</span></div><div class="casAuthors">Gills, Joell J.; LoPiccolo, Jaclyn; Tsurutani, Junji; Shoemaker, Robert H.; Best, Carolyn J. M.; Abu-Asab, Mones S.; Borojerdi, Jennifer; Warfel, Noel A.; Gardner, Erin R.; Danish, Matthew; Hollander, M. Christine; Kawabata, Shigeru; Tsokos, Maria; Figg, William D.; Steeg, Patricia S.; Dennis, Phillip A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5183-5194</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The development of new cancer drugs is slow and costly.  HIV protease inhibitors are Food and Drug Administration approved for HIV patients.  Because these drugs cause toxicities that can be assocd. with inhibition of Akt, an emerging target in cancer, we assessed the potential of HIV protease inhibitors as anticancer agents.  Exptl. Design: HIV protease inhibitors were screened in vitro using assays that measure cellular proliferation, apoptotic and nonapoptotic cell death, endoplasmic reticulum (ER) stress, autophagy, and activation of Akt.  Nelfinavir was tested in non-small cell lung carcinoma (NSCLC) xenografts with biomarker assessment.  Results: Three of six HIV protease inhibitors, nelfinavir, ritonavir, and saquinavir, inhibited proliferation of NSCLC cells, as well as every cell line in the NCI60 cell line panel.  Nelfinavir was most potent with a mean 50% growth inhibition of 5.2 μmol/L, a concn. achievable in HIV patients.  Nelfinavir caused two types of cell death, caspase-dependent apoptosis and caspase-independent death that was characterized by induction of ER stress and autophagy.  Autophagy was protective because an inhibitor of autophagy increased nelfinavir-induced death.  Akt was variably inhibited by HIV protease inhibitors, but nelfinavir caused the greatest inhibition of endogenous and growth factor-induced Akt activation.  Nelfinavir decreased the viability of a panel of drug-resistant breast cancer cell lines and inhibited the growth of NSCLC xenografts that was assocd. with induction of ER stress, autophagy, and apoptosis.  Conclusions: Nelfinavir is a lead HIV protease inhibitor with pleiotropic effects in cancer cells.  Given its wide spectrum of activity, oral availability, and familiarity of administration, nelfinavir is a Food and Drug Administration-approved drug that could be repositioned as a cancer therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLMrqFw84X_7Vg90H21EOLACvtfcHk0lj_QhMUghrxig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFGqs7o%253D&md5=b66804003472d635f209e49d4017387d</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-07-0161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-07-0161%26sid%3Dliteratum%253Aachs%26aulast%3DGills%26aufirst%3DJ.%2BJ.%26aulast%3DLopiccolo%26aufirst%3DJ.%26aulast%3DTsurutani%26aufirst%3DJ.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DBest%26aufirst%3DC.%2BJ.%2BM.%26aulast%3DAbu-Asab%26aufirst%3DM.%2BS.%26aulast%3DBorojerdi%26aufirst%3DJ.%26aulast%3DWarfel%26aufirst%3DN.%2BA.%26aulast%3DGardner%26aufirst%3DE.%2BR.%26aulast%3DDanish%26aufirst%3DM.%26aulast%3DHollander%26aufirst%3DM.%2BC.%26aulast%3DKawabata%26aufirst%3DS.%26aulast%3DTsokos%26aufirst%3DM.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26aulast%3DSteeg%26aufirst%3DP.%2BS.%26aulast%3DDennis%26aufirst%3DP.%2BA.%26atitle%3DNelfinavir%252C%2520A%2520lead%2520HIV%2520protease%2520inhibitor%252C%2520is%2520a%2520broad-spectrum%252C%2520anticancer%2520agent%2520that%2520induces%2520endoplasmic%2520reticulum%2520stress%252C%2520autophagy%252C%2520and%2520apoptosis%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2007%26volume%3D13%26issue%3D17%26spage%3D5183%26epage%3D5194%26doi%3D10.1158%2F1078-0432.CCR-07-0161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group">Dunlop, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lock, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jow, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitzia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grauer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jow, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowlby, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randall, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowal, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comery, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larocque, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soloveva, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roncarati, R.</span><span> </span><span class="NLM_article-title">Old and new pharmacology: positive allosteric modulation of the alpha7 nicotinic acetylcholine receptor by the 5-hydroxytryptamine(2B/C) receptor antagonist SB-206553 (3,5-dihydro-5-methyl-N-3-pyridinylbenzo[1,2-b:4,5-b′]di pyrrole-1(2H)-carboxamide</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">328</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">766</span><span class="NLM_x">–</span> <span class="NLM_lpage">776</span><span class="refDoi"> DOI: 10.1124/jpet.108.146514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1124%2Fjpet.108.146514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=19050173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXislals7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=328&publication_year=2009&pages=766-776&issue=3&author=J.+Dunlopauthor=T.+Lockauthor=B.+Jowauthor=F.+Sitziaauthor=S.+Grauerauthor=F.+Jowauthor=A.+Kramerauthor=M.+R.+Bowlbyauthor=A.+Randallauthor=D.+Kowalauthor=A.+Gilbertauthor=T.+A.+Comeryauthor=J.+Larocqueauthor=V.+Solovevaauthor=J.+Brownauthor=R.+Roncarati&title=Old+and+new+pharmacology%3A+positive+allosteric+modulation+of+the+alpha7+nicotinic+acetylcholine+receptor+by+the+5-hydroxytryptamine%282B%2FC%29+receptor+antagonist+SB-206553+%283%2C5-dihydro-5-methyl-N-3-pyridinylbenzo%5B1%2C2-b%3A4%2C5-b%E2%80%B2%5Ddi+pyrrole-1%282H%29-carboxamide&doi=10.1124%2Fjpet.108.146514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Old and new pharmacology: positive allosteric modulation of the α7 nicotinic acetylcholine receptor by the 5-hydroxytryptamine2B/C receptor antagonist SB-206553 (3,5-dihydro-5-methyl-N-3-pyridinylbenzo[1,2-b:4,5-b']di pyrrole-1(2H)-carboxamide)</span></div><div class="casAuthors">Dunlop, John; Lock, Tim; Jow, Brian; Sitzia, Fabrizio; Grauer, Steven; Jow, Flora; Kramer, Angela; Bowlby, Mark R.; Randall, Andrew; Kowal, Dianne; Gilbert, Adam; Comery, Thomas A.; LaRocque, James; Soloveva, Veronica; Brown, Jon; Roncarati, Renza</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">328</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">766-776</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The α7 nicotinic acetylcholine receptor (nAChR) has been implicated in Alzheimer's disease and schizophrenia, leading to efforts targeted toward discovering agonists and pos. allosteric modulators (PAMs) of this receptor.  In a Ca2+ flux fluorometric imaging plate reader assay, SB-206553 (3,5-dihydro-5-methyl-N-3-pyridinylbenzo[1,2-b:4,5-b']di pyrrole-1(2H)-carboxamide), a compd. known as a 5-hydroxytryptamine2B/2C receptor antagonist, produced an 8-fold potentiation of the evoked calcium signal in the presence of an EC20 concn. of nicotine and a corresponding EC50 of 1.5 μM for potentiation of EC20 nicotine responses in GH4C1 cells expressing the α7 receptor.  SB-206553 was devoid of direct α7 receptor agonist activity and selective against other nicotinic receptors.  Confirmation of the PAM activity of SB-206553 on the α7 nAChR was obtained in patch-clamp electrophysiol. expts. in GH4C1 cells, where it failed to evoke any detectable currents when applied alone, yet dramatically potentiated the currents evoked by an EC20 (17 μM) and EC100 (124 μM) of acetylcholine (ACh).  Native nicotinic receptors in CA1 stratum radiatum interneurons of rat hippocampal slices could also be activated by ACh (200 μM), an effect that was entirely blocked by the α7-selective antagonist methyllycaconitine (MLA).  These ACh currents were potentiated by SB-206553, which increased the area of the current response significantly, resulting in a 40-fold enhancement at 100 μM.  In behavioral expts. in rats, SB-206553 reversed an MK-801 (dizocilpine maleate)-induced deficit in the prepulse inhibition of acoustic startle response, an effect attenuated in the presence of MLA.  This latter observation provides further evidence in support of the potential therapeutic utility of α7 nAChR PAMs in schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomM1BUzvmG6LVg90H21EOLACvtfcHk0lj_QhMUghrxig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXislals7w%253D&md5=d1783c43f8d4e58af504614e13126012</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1124%2Fjpet.108.146514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.108.146514%26sid%3Dliteratum%253Aachs%26aulast%3DDunlop%26aufirst%3DJ.%26aulast%3DLock%26aufirst%3DT.%26aulast%3DJow%26aufirst%3DB.%26aulast%3DSitzia%26aufirst%3DF.%26aulast%3DGrauer%26aufirst%3DS.%26aulast%3DJow%26aufirst%3DF.%26aulast%3DKramer%26aufirst%3DA.%26aulast%3DBowlby%26aufirst%3DM.%2BR.%26aulast%3DRandall%26aufirst%3DA.%26aulast%3DKowal%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DA.%26aulast%3DComery%26aufirst%3DT.%2BA.%26aulast%3DLarocque%26aufirst%3DJ.%26aulast%3DSoloveva%26aufirst%3DV.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DRoncarati%26aufirst%3DR.%26atitle%3DOld%2520and%2520new%2520pharmacology%253A%2520positive%2520allosteric%2520modulation%2520of%2520the%2520alpha7%2520nicotinic%2520acetylcholine%2520receptor%2520by%2520the%25205-hydroxytryptamine%25282B%252FC%2529%2520receptor%2520antagonist%2520SB-206553%2520%25283%252C5-dihydro-5-methyl-N-3-pyridinylbenzo%255B1%252C2-b%253A4%252C5-b%25E2%2580%25B2%255Ddi%2520pyrrole-1%25282H%2529-carboxamide%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2009%26volume%3D328%26issue%3D3%26spage%3D766%26epage%3D776%26doi%3D10.1124%2Fjpet.108.146514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evrard, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murdoch, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Droste, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nissen, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenck, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fludzinski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucaites, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, M. L.</span><span> </span><span class="NLM_article-title">Potent, selective tetrahydro-beta-carboline antagonists of the serotonin 2B (5HT2B) contractile receptor in the rat stomach fundus</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">2773</span><span class="NLM_x">–</span> <span class="NLM_lpage">2780</span><span class="refDoi"> DOI: 10.1021/jm960062t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm960062t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2773-2780&issue=14&author=J.+E.+Audiaauthor=D.+A.+Evrardauthor=G.+R.+Murdochauthor=J.+J.+Drosteauthor=J.+S.+Nissenauthor=K.+W.+Schenckauthor=P.+Fludzinskiauthor=V.+L.+Lucaitesauthor=D.+L.+Nelsonauthor=M.+L.+Cohen&title=Potent%2C+selective+tetrahydro-beta-carboline+antagonists+of+the+serotonin+2B+%285HT2B%29+contractile+receptor+in+the+rat+stomach+fundus&doi=10.1021%2Fjm960062t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fjm960062t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm960062t%26sid%3Dliteratum%253Aachs%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DEvrard%26aufirst%3DD.%2BA.%26aulast%3DMurdoch%26aufirst%3DG.%2BR.%26aulast%3DDroste%26aufirst%3DJ.%2BJ.%26aulast%3DNissen%26aufirst%3DJ.%2BS.%26aulast%3DSchenck%26aufirst%3DK.%2BW.%26aulast%3DFludzinski%26aufirst%3DP.%26aulast%3DLucaites%26aufirst%3DV.%2BL.%26aulast%3DNelson%26aufirst%3DD.%2BL.%26aulast%3DCohen%26aufirst%3DM.%2BL.%26atitle%3DPotent%252C%2520selective%2520tetrahydro-beta-carboline%2520antagonists%2520of%2520the%2520serotonin%25202B%2520%25285HT2B%2529%2520contractile%2520receptor%2520in%2520the%2520rat%2520stomach%2520fundus%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26issue%3D14%26spage%3D2773%26epage%3D2780%26doi%3D10.1021%2Fjm960062t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group">Petrey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Structural bioinformatics of the interactome</span> <span class="citation_source-journal">Annu. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.1146/annurev-biophys-051013-022726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1146%2Fannurev-biophys-051013-022726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24895853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOht7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=193-210&author=D.+Petreyauthor=B.+Honig&title=Structural+bioinformatics+of+the+interactome&doi=10.1146%2Fannurev-biophys-051013-022726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Structural bioinformatics of the interactome</span></div><div class="casAuthors">Petrey, Donald; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">193-210</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The past decade has seen a dramatic expansion in the no. and range of techniques available to obtain genome-wide information and to analyze this information so as to infer both the functions of individual mols. and how they interact to modulate the behavior of biol. systems.  Here, we review these techniques, focusing on the construction of phys. protein-protein interaction networks, and highlighting approaches that incorporate protein structure, which is becoming an increasingly important component of systems-level computational techniques.  We also discuss how network analyses are being applied to enhance our basic understanding of biol. systems and their disregulation, as well as how these networks are being used in drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrju2I3vXKYM7Vg90H21EOLACvtfcHk0lj_QhMUghrxig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOht7rL&md5=f50bea829841525b0c3b19321aae5fb6</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-051013-022726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-051013-022726%26sid%3Dliteratum%253Aachs%26aulast%3DPetrey%26aufirst%3DD.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DStructural%2520bioinformatics%2520of%2520the%2520interactome%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2014%26volume%3D43%26spage%3D193%26epage%3D210%26doi%3D10.1146%2Fannurev-biophys-051013-022726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group">Haydon, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokes, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ure, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galbraith, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barynin, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sedelnikova, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heal, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aiwale, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauhan, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taneja, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Errington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czaplewski, L. G.</span><span> </span><span class="NLM_article-title">An inhibitor of FtsZ with potent and selective anti-staphylococcal activity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">321</span><span class="NLM_x"> (</span><span class="NLM_issue">5896</span><span class="NLM_x">) </span> <span class="NLM_fpage">1673</span><span class="NLM_x">–</span> <span class="NLM_lpage">1675</span><span class="refDoi"> DOI: 10.1126/science.1159961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1126%2Fscience.1159961" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=1673-1675&issue=5896&author=D.+J.+Haydonauthor=N.+R.+Stokesauthor=R.+Ureauthor=G.+Galbraithauthor=J.+M.+Bennettauthor=D.+R.+Brownauthor=P.+J.+Bakerauthor=V.+V.+Baryninauthor=D.+W.+Riceauthor=S.+E.+Sedelnikovaauthor=J.+R.+Healauthor=J.+M.+Sheridanauthor=S.+T.+Aiwaleauthor=P.+K.+Chauhanauthor=A.+Srivastavaauthor=A.+Tanejaauthor=I.+Collinsauthor=J.+Erringtonauthor=L.+G.+Czaplewski&title=An+inhibitor+of+FtsZ+with+potent+and+selective+anti-staphylococcal+activity&doi=10.1126%2Fscience.1159961"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1126%2Fscience.1159961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1159961%26sid%3Dliteratum%253Aachs%26aulast%3DHaydon%26aufirst%3DD.%2BJ.%26aulast%3DStokes%26aufirst%3DN.%2BR.%26aulast%3DUre%26aufirst%3DR.%26aulast%3DGalbraith%26aufirst%3DG.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBrown%26aufirst%3DD.%2BR.%26aulast%3DBaker%26aufirst%3DP.%2BJ.%26aulast%3DBarynin%26aufirst%3DV.%2BV.%26aulast%3DRice%26aufirst%3DD.%2BW.%26aulast%3DSedelnikova%26aufirst%3DS.%2BE.%26aulast%3DHeal%26aufirst%3DJ.%2BR.%26aulast%3DSheridan%26aufirst%3DJ.%2BM.%26aulast%3DAiwale%26aufirst%3DS.%2BT.%26aulast%3DChauhan%26aufirst%3DP.%2BK.%26aulast%3DSrivastava%26aufirst%3DA.%26aulast%3DTaneja%26aufirst%3DA.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DErrington%26aufirst%3DJ.%26aulast%3DCzaplewski%26aufirst%3DL.%2BG.%26atitle%3DAn%2520inhibitor%2520of%2520FtsZ%2520with%2520potent%2520and%2520selective%2520anti-staphylococcal%2520activity%26jtitle%3DScience%26date%3D2008%26volume%3D321%26issue%3D5896%26spage%3D1673%26epage%3D1675%26doi%3D10.1126%2Fscience.1159961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group">Anderson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorto, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackner, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ames, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, J. T.</span><span> </span><span class="NLM_article-title">Comparison of small molecule inhibitors of the bacterial cell division protein FtsZ and identification of a reliable cross-species inhibitor</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">1918</span><span class="NLM_x">–</span> <span class="NLM_lpage">1928</span><span class="refDoi"> DOI: 10.1021/cb300340j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300340j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWmur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1918-1928&issue=11&author=D.+E.+Andersonauthor=M.+B.+Kimauthor=J.+T.+Mooreauthor=T.+E.+O%E2%80%99Brienauthor=N.+A.+Sortoauthor=C.+I.+Groveauthor=L.+L.+Lacknerauthor=J.+B.+Amesauthor=J.+T.+Shaw&title=Comparison+of+small+molecule+inhibitors+of+the+bacterial+cell+division+protein+FtsZ+and+identification+of+a+reliable+cross-species+inhibitor&doi=10.1021%2Fcb300340j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Small Molecule Inhibitors of the Bacterial Cell Division Protein FtsZ and Identification of a Reliable Cross-Species Inhibitor</span></div><div class="casAuthors">Anderson, David E.; Kim, Michelle B.; Moore, Jared T.; O'Brien, Terrence E.; Sorto, Nohemy A.; Grove, Charles I.; Lackner, Laura L.; Ames, James B.; Shaw, Jared T.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1918-1928</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">FtsZ is a guanosine triphosphatase (GTPase) that mediates cytokinesis in bacteria.  FtsZ is homologous in structure to eukaryotic tubulin and polymerizes in a similar head-to-tail fashion.  The study of tubulin's function in eukaryotic cells has benefited greatly from specific and potent small mol. inhibitors, including colchicine and taxol.  Although many small mol. inhibitors of FtsZ have been reported, none has emerged as a generally useful probe for modulating bacterial cell division.  With the goal of establishing a useful and reliable small mol. inhibitor of FtsZ, a broad biochem. cross-comparison of reported FtsZ inhibitors was undertaken.  Several of these mols., including phenolic natural products, are unselective inhibitors that seem to derive their activity from the formation of microscopic colloids or aggregates.  Other compds., including the natural product viriditoxin and the drug development candidate PC190723, exhibit no inhibition of GTPase activity using protocols or under published conditions.  Of the compds. studied, only zantrin Z3 exhibits good levels of inhibition, maintains activity under conditions that disrupt small mol. aggregates, and provides a platform for exploration of structure-activity relationships (SAR).  Preliminary SAR studies have identified slight modifications to the two side chains of this structure that modulate the inhibitory activity of zantrin Z3.  Collectively, these studies will help focus future investigations toward the establishment of probes for FtsZ that fill the roles of colchicine and taxol in studies of tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvE8_fy6gEwLVg90H21EOLACvtfcHk0liAciCloO4UHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWmur%252FL&md5=ab69ea62af74ad72f6177db5f556c990</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1021%2Fcb300340j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300340j%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BE.%26aulast%3DKim%26aufirst%3DM.%2BB.%26aulast%3DMoore%26aufirst%3DJ.%2BT.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%2BE.%26aulast%3DSorto%26aufirst%3DN.%2BA.%26aulast%3DGrove%26aufirst%3DC.%2BI.%26aulast%3DLackner%26aufirst%3DL.%2BL.%26aulast%3DAmes%26aufirst%3DJ.%2BB.%26aulast%3DShaw%26aufirst%3DJ.%2BT.%26atitle%3DComparison%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520bacterial%2520cell%2520division%2520protein%2520FtsZ%2520and%2520identification%2520of%2520a%2520reliable%2520cross-species%2520inhibitor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26issue%3D11%26spage%3D1918%26epage%3D1928%26doi%3D10.1021%2Fcb300340j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group">Andreu, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaffner-Barbero, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huecas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alonso, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Rodriguez, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz-Avila, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Núñez-Ramírez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llorca, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martín-Galiano, A. J.</span><span> </span><span class="NLM_article-title">The antibacterial cell division inhibitor PC190723 is an FtsZ polymer-stabilizing agent that induces filament assembly and condensation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x"> (</span><span class="NLM_issue">19</span><span class="NLM_x">) </span> <span class="NLM_fpage">14239</span><span class="NLM_x">–</span> <span class="NLM_lpage">14246</span><span class="refDoi"> DOI: 10.1074/jbc.M109.094722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1074%2Fjbc.M109.094722" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=14239-14246&issue=19&author=J.+M.+Andreuauthor=C.+Schaffner-Barberoauthor=S.+Huecasauthor=D.+Alonsoauthor=M.+L.+Lopez-Rodriguezauthor=L.+B.+Ruiz-Avilaauthor=R.+N%C3%BA%C3%B1ez-Ram%C3%ADrezauthor=O.+Llorcaauthor=A.+J.+Mart%C3%ADn-Galiano&title=The+antibacterial+cell+division+inhibitor+PC190723+is+an+FtsZ+polymer-stabilizing+agent+that+induces+filament+assembly+and+condensation&doi=10.1074%2Fjbc.M109.094722"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.094722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.094722%26sid%3Dliteratum%253Aachs%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DAlonso%26aufirst%3DD.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DN%25C3%25BA%25C3%25B1ez-Ram%25C3%25ADrez%26aufirst%3DR.%26aulast%3DLlorca%26aufirst%3DO.%26aulast%3DMart%25C3%25ADn-Galiano%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520antibacterial%2520cell%2520division%2520inhibitor%2520PC190723%2520is%2520an%2520FtsZ%2520polymer-stabilizing%2520agent%2520that%2520induces%2520filament%2520assembly%2520and%2520condensation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D19%26spage%3D14239%26epage%3D14246%26doi%3D10.1074%2Fjbc.M109.094722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group">Sudha, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srinivasan, N.</span><span> </span><span class="NLM_article-title">An overview of recent advances in structural bioinformatics of protein–protein interactions and a guide to their principles</span> <span class="citation_source-journal">Prog. Biophys. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x"> (</span><span class="NLM_issue">2–3</span><span class="NLM_x">) </span> <span class="NLM_fpage">141</span><span class="NLM_x">–</span> <span class="NLM_lpage">150</span><span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2014.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.pbiomolbio.2014.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25077409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlaku7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2014&pages=141-150&issue=2%E2%80%933&author=G.+Sudhaauthor=R.+Nussinovauthor=N.+Srinivasan&title=An+overview+of+recent+advances+in+structural+bioinformatics+of+protein%E2%80%93protein+interactions+and+a+guide+to+their+principles&doi=10.1016%2Fj.pbiomolbio.2014.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of recent advances in structural bioinformatics of protein-protein interactions and a guide to their principles</span></div><div class="casAuthors">Sudha, Govindarajan; Nussinov, Ruth; Srinivasan, Narayanaswamy</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">141-150</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Rich data bearing on the structural and evolutionary principles of protein-protein interactions are paving the way to a better understanding of the regulation of function in the cell.  This is particularly the case when these interactions are considered in the framework of key pathways.  Knowledge of the interactions may provide insights into the mechanisms of crucial 'driver' mutations in oncogenesis.  They also provide the foundation toward the design of protein-protein interfaces and inhibitors that can abrogate their formation or enhance them.  The main features to learn from known 3-D structures of protein-protein complexes and the extensive literature which analyzes them computationally and exptl. include the interaction details which permit undertaking structure-based drug discovery, the evolution of complexes and their interactions, the consequences of alterations such as post-translational modifications, ligand binding, disease causing mutations, host pathogen interactions, oligomerization, aggregation and the roles of disorder, dynamics, allostery and more to the protein and the cell.  This review highlights some of the recent advances in these areas, including design, inhibition and prediction of protein-protein complexes.  The field is broad, and much work has been carried out in these areas, making it challenging to cover it in its entirety.  Much of this is due to the fast increase in the no. of mols. whose structures have been detd. exptl. and the vast increase in computational power.  Here we provide a concise overview.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhVXrmxxifZbVg90H21EOLACvtfcHk0liAciCloO4UHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlaku7jN&md5=09fb901ba27805f4807f162e5c613c24</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2014.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2014.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DSudha%26aufirst%3DG.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DSrinivasan%26aufirst%3DN.%26atitle%3DAn%2520overview%2520of%2520recent%2520advances%2520in%2520structural%2520bioinformatics%2520of%2520protein%25E2%2580%2593protein%2520interactions%2520and%2520a%2520guide%2520to%2520their%2520principles%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D116%26issue%3D2%25E2%2580%25933%26spage%3D141%26epage%3D150%26doi%3D10.1016%2Fj.pbiomolbio.2014.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group">Xue, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobbs, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonvin, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honavar, V.</span><span> </span><span class="NLM_article-title">Computational prediction of protein interfaces: A review of data driven methods</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">589</span><span class="NLM_x"> (</span><span class="NLM_issue">23</span><span class="NLM_x">) </span> <span class="NLM_fpage">3516</span><span class="NLM_x">–</span> <span class="NLM_lpage">3526</span><span class="refDoi"> DOI: 10.1016/j.febslet.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.febslet.2015.10.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=3516-3526&issue=23&author=L.+C.+Xueauthor=D.+Dobbsauthor=A.+M.+Bonvinauthor=V.+Honavar&title=Computational+prediction+of+protein+interfaces%3A+A+review+of+data+driven+methods&doi=10.1016%2Fj.febslet.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DL.%2BC.%26aulast%3DDobbs%26aufirst%3DD.%26aulast%3DBonvin%26aufirst%3DA.%2BM.%26aulast%3DHonavar%26aufirst%3DV.%26atitle%3DComputational%2520prediction%2520of%2520protein%2520interfaces%253A%2520A%2520review%2520of%2520data%2520driven%2520methods%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26issue%3D23%26spage%3D3516%26epage%3D3526%26doi%3D10.1016%2Fj.febslet.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group">Dowlatshahi, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandrin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vivona, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaler, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melandri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundquist, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopito, R. R.</span><span> </span><span class="NLM_article-title">ALIX is a Lys63-specific polyubiquitin binding protein that functions in retrovirus budding</span> <span class="citation_source-journal">Dev. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1247</span><span class="NLM_x">–</span> <span class="NLM_lpage">1254</span><span class="refDoi"> DOI: 10.1016/j.devcel.2012.10.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.devcel.2012.10.023" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=1247-1254&issue=6&author=D.+P.+Dowlatshahiauthor=V.+Sandrinauthor=S.+Vivonaauthor=T.+A.+Shalerauthor=S.+E.+Kaiserauthor=F.+Melandriauthor=W.+I.+Sundquistauthor=R.+R.+Kopito&title=ALIX+is+a+Lys63-specific+polyubiquitin+binding+protein+that+functions+in+retrovirus+budding&doi=10.1016%2Fj.devcel.2012.10.023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2012.10.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2012.10.023%26sid%3Dliteratum%253Aachs%26aulast%3DDowlatshahi%26aufirst%3DD.%2BP.%26aulast%3DSandrin%26aufirst%3DV.%26aulast%3DVivona%26aufirst%3DS.%26aulast%3DShaler%26aufirst%3DT.%2BA.%26aulast%3DKaiser%26aufirst%3DS.%2BE.%26aulast%3DMelandri%26aufirst%3DF.%26aulast%3DSundquist%26aufirst%3DW.%2BI.%26aulast%3DKopito%26aufirst%3DR.%2BR.%26atitle%3DALIX%2520is%2520a%2520Lys63-specific%2520polyubiquitin%2520binding%2520protein%2520that%2520functions%2520in%2520retrovirus%2520budding%26jtitle%3DDev.%2520Cell%26date%3D2012%26volume%3D23%26issue%3D6%26spage%3D1247%26epage%3D1254%26doi%3D10.1016%2Fj.devcel.2012.10.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref152"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref152'); return false;" data-citation="" class="refNumLink">152</a></strong><div class="NLM_citation" id="cit152"><span><span class="NLM_contrib-group">Pashkova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gakhar, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winistorfer, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunshine, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunham, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piper, R. C.</span><span> </span><span class="NLM_article-title">The yeast Alix homolog Bro1 functions as a ubiquitin receptor for protein sorting into multivesicular endosomes</span> <span class="citation_source-journal">Dev. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">520</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span><span class="refDoi"> DOI: 10.1016/j.devcel.2013.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.devcel.2013.04.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2013&pages=520-533&issue=5&author=N.+Pashkovaauthor=L.+Gakharauthor=S.+C.+Winistorferauthor=A.+B.+Sunshineauthor=M.+Richauthor=M.+J.+Dunhamauthor=L.+Yuauthor=R.+C.+Piper&title=The+yeast+Alix+homolog+Bro1+functions+as+a+ubiquitin+receptor+for+protein+sorting+into+multivesicular+endosomes&doi=10.1016%2Fj.devcel.2013.04.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2013.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2013.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DPashkova%26aufirst%3DN.%26aulast%3DGakhar%26aufirst%3DL.%26aulast%3DWinistorfer%26aufirst%3DS.%2BC.%26aulast%3DSunshine%26aufirst%3DA.%2BB.%26aulast%3DRich%26aufirst%3DM.%26aulast%3DDunham%26aufirst%3DM.%2BJ.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DPiper%26aufirst%3DR.%2BC.%26atitle%3DThe%2520yeast%2520Alix%2520homolog%2520Bro1%2520functions%2520as%2520a%2520ubiquitin%2520receptor%2520for%2520protein%2520sorting%2520into%2520multivesicular%2520endosomes%26jtitle%3DDev.%2520Cell%26date%3D2013%26volume%3D25%26issue%3D5%26spage%3D520%26epage%3D533%26doi%3D10.1016%2Fj.devcel.2013.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref153"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref153'); return false;" data-citation="" class="refNumLink">153</a></strong><div class="NLM_citation" id="cit153"><span><span class="NLM_contrib-group">Schumann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armen, R. S.</span><span> </span><span class="NLM_article-title">Identification of distant drug off-targets by direct superposition of binding pocket surfaces</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">e83533</span><span class="refDoi"> DOI: 10.1371/journal.pone.0083533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0083533" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e83533&issue=12&author=M.+Schumannauthor=R.+S.+Armen&title=Identification+of+distant+drug+off-targets+by+direct+superposition+of+binding+pocket+surfaces&doi=10.1371%2Fjournal.pone.0083533"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083533%26sid%3Dliteratum%253Aachs%26aulast%3DSchumann%26aufirst%3DM.%26aulast%3DArmen%26aufirst%3DR.%2BS.%26atitle%3DIdentification%2520of%2520distant%2520drug%2520off-targets%2520by%2520direct%2520superposition%2520of%2520binding%2520pocket%2520surfaces%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26issue%3D12%26spage%3De83533%26doi%3D10.1371%2Fjournal.pone.0083533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref154"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref154'); return false;" data-citation="" class="refNumLink">154</a></strong><div class="NLM_citation" id="cit154"><span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapland, R. H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoorn, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Global mapping of pharmacological space</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">805</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span><span class="refDoi"> DOI: 10.1038/nbt1228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&issue=7&author=G.+V.+Paoliniauthor=R.+H.+B.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+mapping+of+pharmacological+space&doi=10.1038%2Fnbt1228"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%2BB.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520mapping%2520of%2520pharmacological%2520space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26issue%3D7%26spage%3D805%26epage%3D815%26doi%3D10.1038%2Fnbt1228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref155"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref155'); return false;" data-citation="" class="refNumLink">155</a></strong><div class="NLM_citation" id="cit155"><span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">A survey of across-target bioactivity results of small molecules in PubChem</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">2251</span><span class="NLM_x">–</span> <span class="NLM_lpage">2255</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtp380" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2251-2255&issue=17&author=L.+Hanauthor=Y.+Wangauthor=S.+H.+Bryant&title=A+survey+of+across-target+bioactivity+results+of+small+molecules+in+PubChem&doi=10.1093%2Fbioinformatics%2Fbtp380"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp380%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DA%2520survey%2520of%2520across-target%2520bioactivity%2520results%2520of%2520small%2520molecules%2520in%2520PubChem%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26issue%3D17%26spage%3D2251%26epage%3D2255%26doi%3D10.1093%2Fbioinformatics%2Fbtp380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref156"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref156'); return false;" data-citation="" class="refNumLink">156</a></strong><div class="NLM_citation" id="cit156"><span><span class="NLM_contrib-group">LaBute, M. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenderman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennion, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lightstone, F. C.</span><span> </span><span class="NLM_article-title">Adverse drug reaction prediction using scores produced by large-scale drug-protein target docking on high-performance computing machines</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">e106298</span><span class="refDoi"> DOI: 10.1371/journal.pone.0106298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0106298" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e106298&issue=9&author=M.+X.+LaButeauthor=X.+Zhangauthor=J.+Lendermanauthor=B.+J.+Bennionauthor=S.+E.+Wongauthor=F.+C.+Lightstone&title=Adverse+drug+reaction+prediction+using+scores+produced+by+large-scale+drug-protein+target+docking+on+high-performance+computing+machines&doi=10.1371%2Fjournal.pone.0106298"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0106298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0106298%26sid%3Dliteratum%253Aachs%26aulast%3DLaBute%26aufirst%3DM.%2BX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLenderman%26aufirst%3DJ.%26aulast%3DBennion%26aufirst%3DB.%2BJ.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26atitle%3DAdverse%2520drug%2520reaction%2520prediction%2520using%2520scores%2520produced%2520by%2520large-scale%2520drug-protein%2520target%2520docking%2520on%2520high-performance%2520computing%2520machines%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3De106298%26doi%3D10.1371%2Fjournal.pone.0106298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref157"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref157'); return false;" data-citation="" class="refNumLink">157</a></strong><div class="NLM_citation" id="cit157"><span><span class="NLM_contrib-group">Sonnhammer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eddy, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durbin, R.</span><span> </span><span class="NLM_article-title">Pfam: a comprehensive database of protein domain families based on seed alignments</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">420</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(199707)28:3<405::AID-PROT10>3.0.CO;2-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=9223186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2sXksVCgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1997&pages=405-420&issue=3&author=E.+L.+Sonnhammerauthor=S.+R.+Eddyauthor=R.+Durbin&title=Pfam%3A+a+comprehensive+database+of+protein+domain+families+based+on+seed+alignments&doi=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Pfam: a comprehensive database of protein domain families based on seed alignments</span></div><div class="casAuthors">Sonnhammer, Erik L. L.; Eddy, Sean R.; Durbin, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-420</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss</span>)
        </div><div class="casAbstract">Databases of multiple sequence alignments are a valuable aid to protein sequence classification and anal.  One of the main challenges when constructing such a database is to simultaneously satisfy the conflicting demands of completeness on the one hand and quality of alignment and domain definitions on the other.  The latter properties are best dealt with by manual approaches, whereas completeness in practice is only amenable to automatic methods.  Here, the authors present a database based on hidden Markov model profiles (HMMs), which combines high quality and completeness.  This database, Pfam, consists of parts A and B.  Pfam-A is curated and contains well-characterized protein domain families with high quality alignments, which are maintained by using manually checked seed alignments and HMMs to find and align all members.  Pfam-B contains sequence families that were generated automatically by applying the Domainer algorithm to cluster and align the remaining protein sequences after removal of Pfam-A domains.  By using Pfam, a large no. of previously unannotated proteins from the Caenorhabditis elegans genome project were classified.  The authors also identified many novel family memberships in known proteins, including new Kazal, fibronectin type III, and response regulator receiver domains.  Pfam-A families have permanent accession nos. and form a library of HMMs available for searching and automatic annotation of new protein sequences.  There are World Wide Web servers at http://www.sanger.ac.uk/Pfam and http://genome.wustl.edu/Pfam, which allow browsing and HMM searching against Pfam-A with a query sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre6VakACw-_rVg90H21EOLACvtfcHk0lhU3jYqWy7B-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksVCgsLo%253D&md5=8fe760b424180de92f0c3816443d2d8d</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%2528199707%252928%253A3%253C405%253A%253AAID-PROT10%253E3.0.CO%253B2-L%26sid%3Dliteratum%253Aachs%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DDurbin%26aufirst%3DR.%26atitle%3DPfam%253A%2520a%2520comprehensive%2520database%2520of%2520protein%2520domain%2520families%2520based%2520on%2520seed%2520alignments%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1997%26volume%3D28%26issue%3D3%26spage%3D405%26epage%3D420%26doi%3D10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref158"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref158'); return false;" data-citation="" class="refNumLink">158</a></strong><div class="NLM_citation" id="cit158"><span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weskamp, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hüllermeier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Functional classification of protein kinase binding sites using Cavbase</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">1432</span><span class="NLM_x">–</span> <span class="NLM_lpage">1447</span><span class="refDoi"> DOI: 10.1002/cmdc.200700075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fcmdc.200700075" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=1432-1447&issue=10&author=D.+Kuhnauthor=N.+Weskampauthor=E.+H%C3%BCllermeierauthor=G.+Klebe&title=Functional+classification+of+protein+kinase+binding+sites+using+Cavbase&doi=10.1002%2Fcmdc.200700075"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200700075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200700075%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DWeskamp%26aufirst%3DN.%26aulast%3DH%25C3%25BCllermeier%26aufirst%3DE.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFunctional%2520classification%2520of%2520protein%2520kinase%2520binding%2520sites%2520using%2520Cavbase%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26issue%3D10%26spage%3D1432%26epage%3D1447%26doi%3D10.1002%2Fcmdc.200700075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref159"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref159'); return false;" data-citation="" class="refNumLink">159</a></strong><div class="NLM_citation" id="cit159"><span><span class="NLM_contrib-group">Liu, C.-G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, K.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, H.-P.</span><span> </span><span class="NLM_article-title">17Beta-oestradiol regulates the expression of Na+/K+-ATPase beta1-subunit, sarcoplasmic reticulum Ca2+-ATPase and carbonic anhydrase iv in H9C2 cells</span> <span class="citation_source-journal">Clin. Exp. Pharmacol. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">998</span><span class="NLM_x">–</span> <span class="NLM_lpage">1004</span><span class="refDoi"> DOI: 10.1111/j.1440-1681.2007.04675.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1111%2Fj.1440-1681.2007.04675.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=17714085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2007&pages=998-1004&issue=10&author=C.-G.+Liuauthor=K.-Q.+Xuauthor=X.+Xuauthor=J.-J.+Huangauthor=J.-C.+Xiaoauthor=J.-P.+Zhangauthor=H.-P.+Song&title=17Beta-oestradiol+regulates+the+expression+of+Na%2B%2FK%2B-ATPase+beta1-subunit%2C+sarcoplasmic+reticulum+Ca2%2B-ATPase+and+carbonic+anhydrase+iv+in+H9C2+cells&doi=10.1111%2Fj.1440-1681.2007.04675.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">17β-Oestradiol regulates the expression of Na+/K+-ATPase β1-subunit, sarcoplasmic reticulum Ca2+-ATPase and carbonic anhydrase IV in H9C2 cells</span></div><div class="casAuthors">Liu, Chen-Geng; Xu, Ke-Qian; Xu, Xia; Huang, Jian-Jun; Xiao, Jing-Chuan; Zhang, Jie-Ping; Song, Hui-Ping</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Experimental Pharmacology and Physiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">998-1004</span>CODEN:
                <span class="NLM_cas:coden">CEXPB9</span>;
        ISSN:<span class="NLM_cas:issn">0305-1870</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">It is necessary to improve our understanding of the effect of 17β-estradiol (E2) on the heart at a mol. and cellular level.  In the present study, the effects of E2 on Na+/K+-ATPase, sarcoplasmic/endoplasmic reticulum Ca2+-ATPase (SERCA) and carbonic anhydrase IV (CAIV) in H9C2 cells were investigated.  To identify the mechanism of action of E2 on these proteins, the estrogen receptor (ER) antagonist tamoxifen was used.  The results indicated that 1 and 100 nmol/L E2 can enhance the activity of Na+/K+-ATPase and SERCA and upregulate the expression of the Na+/K+-ATPase β1-subunit, SERCA2a and CAIV at both the mRNA and protein level compared with 0 and 0.01 nmol/L E2.  E2 had the greatest effect at 100 nmol/L; 1 μmol/L E2 did not further protein expression compared with 100 nmol/L E2.  Tamoxifen (10 nmol/L) significantly decreased the activity of SERCA, as well as the expression of the Na+/K+-ATPase β1-subunit and SERCA at the mRNA and protein level, in H9C2 cells cultured with 1 nmol/L E2.  Tamoxifen alone had no significant effect on these proteins in H9C2 cells.  It may be hypothesized that a suitable E2 concn. has a protective effect on the heart and that the actual dose of E2 used in hormone-replacement therapy is important in menopausal women.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphXyGJ0J5KNbVg90H21EOLACvtfcHk0lhU3jYqWy7B-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtrbO&md5=5751a59fa260e70ccf08f004e7e133ed</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1111%2Fj.1440-1681.2007.04675.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1440-1681.2007.04675.x%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DC.-G.%26aulast%3DXu%26aufirst%3DK.-Q.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DJ.-J.%26aulast%3DXiao%26aufirst%3DJ.-C.%26aulast%3DZhang%26aufirst%3DJ.-P.%26aulast%3DSong%26aufirst%3DH.-P.%26atitle%3D17Beta-oestradiol%2520regulates%2520the%2520expression%2520of%2520Na%252B%252FK%252B-ATPase%2520beta1-subunit%252C%2520sarcoplasmic%2520reticulum%2520Ca2%252B-ATPase%2520and%2520carbonic%2520anhydrase%2520iv%2520in%2520H9C2%2520cells%26jtitle%3DClin.%2520Exp.%2520Pharmacol.%2520Physiol.%26date%3D2007%26volume%3D34%26issue%3D10%26spage%3D998%26epage%3D1004%26doi%3D10.1111%2Fj.1440-1681.2007.04675.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref160"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref160'); return false;" data-citation="" class="refNumLink">160</a></strong><div class="NLM_citation" id="cit160"><span><span class="NLM_contrib-group">Beca, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavlov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kargacin, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aschar-Sobbi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kargacin, G. J.</span><span> </span><span class="NLM_article-title">Inhibition of a cardiac sarcoplasmic reticulum chloride channel by tamoxifen</span> <span class="citation_source-journal">Pfluegers Arch.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">457</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1007/s00424-008-0510-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2Fs00424-008-0510-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18458943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCiur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=457&publication_year=2008&pages=121-135&issue=1&author=S.+Becaauthor=E.+Pavlovauthor=M.+E.+Kargacinauthor=R.+Aschar-Sobbiauthor=R.+J.+Frenchauthor=G.+J.+Kargacin&title=Inhibition+of+a+cardiac+sarcoplasmic+reticulum+chloride+channel+by+tamoxifen&doi=10.1007%2Fs00424-008-0510-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of a cardiac sarcoplasmic reticulum chloride channel by tamoxifen</span></div><div class="casAuthors">Beca, Sanja; Pavlov, Evgeny; Kargacin, Margaret E.; Aschar-Sobbi, Roozbeh; French, Robert J.; Kargacin, Gary J.</div><div class="citationInfo"><span class="NLM_cas:title">Pfluegers Archiv</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">457</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">121-135</span>CODEN:
                <span class="NLM_cas:coden">PFLABK</span>;
        ISSN:<span class="NLM_cas:issn">0031-6768</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Anion and cation channels present in the sarcoplasmic reticulum (SR) are believed to be necessary to maintain the electroneutrality of SR membrane during Ca2+ uptake by the SR Ca2+ pump (SERCA).  Here the authors incorporated canine cardiac SR ion channels into lipid bilayers and studied the effects of tamoxifen and other antiestrogens on these channels.  A Cl- channel was identified exhibiting multiple subconductance levels which could be divided into two primary conductance bands.  Tamoxifen decreases the time the channel spends in its higher, voltage-sensitive band and the mean channel current.  The lower, voltage-insensitive, conductance band is not affected by tamoxifen, nor is a K+ channel present in the cardiac SR prepn.  By examg. SR Ca2+ uptake, SERCA ATPase activity, and SR ion channels in the same prepn., the authors also estd. SERCA transport current, SR Cl- and K+ currents, and the d. of SERCA, Cl-, and K+ channels in cardiac SR membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzceMjb_UAvrVg90H21EOLACvtfcHk0lhU3jYqWy7B-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCiur3K&md5=f4fc47985e319b9649e5d927a7472568</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1007%2Fs00424-008-0510-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00424-008-0510-9%26sid%3Dliteratum%253Aachs%26aulast%3DBeca%26aufirst%3DS.%26aulast%3DPavlov%26aufirst%3DE.%26aulast%3DKargacin%26aufirst%3DM.%2BE.%26aulast%3DAschar-Sobbi%26aufirst%3DR.%26aulast%3DFrench%26aufirst%3DR.%2BJ.%26aulast%3DKargacin%26aufirst%3DG.%2BJ.%26atitle%3DInhibition%2520of%2520a%2520cardiac%2520sarcoplasmic%2520reticulum%2520chloride%2520channel%2520by%2520tamoxifen%26jtitle%3DPfluegers%2520Arch.%26date%3D2008%26volume%3D457%26issue%3D1%26spage%3D121%26epage%3D135%26doi%3D10.1007%2Fs00424-008-0510-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref161"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref161'); return false;" data-citation="" class="refNumLink">161</a></strong><div class="NLM_citation" id="cit161"><span><span class="NLM_contrib-group">Wang, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.-H.</span><span> </span><span class="NLM_article-title">Scoring functions for prediction of protein-ligand interactions</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2174</span><span class="NLM_x">–</span> <span class="NLM_lpage">2182</span><span class="refDoi"> DOI: 10.2174/1381612811319120005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2174%2F1381612811319120005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=23016847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2174-2182&issue=12&author=J.-C.+Wangauthor=J.-H.+Lin&title=Scoring+functions+for+prediction+of+protein-ligand+interactions&doi=10.2174%2F1381612811319120005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Scoring functions for prediction of protein-ligand interactions</span></div><div class="casAuthors">Wang, Jui-Chih; Lin, Jung-Hsin</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2174-2182</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The scoring functions for protein-ligand interactions plays central roles in computational drug design, virtual screening of chem. libraries for new lead identification, and prediction of possible binding targets of small chem. mols.  An ideal scoring function for protein-ligand interactions is expected to be able to recognize the native binding pose of a ligand on the protein surface among decoy poses, and to accurately predict the binding affinity (or binding free energy) so that the active mols. can be discriminated from the non-active ones.  Due to the empirical nature of most, if not all, scoring functions for protein-ligand interactions, the general applicability of empirical scoring functions, esp. to domains far outside training sets, is a major concern.  In this review article, we will explore the foundations of different classes of scoring functions, their possible limitations, and their suitable application domains.  We also provide assessments of several scoring functions on weakly-interacting protein-ligand complexes, which will be useful information in computational fragment-based drug design or virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgdoR3oNlND7Vg90H21EOLACvtfcHk0lh0oIT1ji_Hxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtLs%253D&md5=0dbef427d5e9465beb60bd25a5882789</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.2174%2F1381612811319120005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319120005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.-C.%26aulast%3DLin%26aufirst%3DJ.-H.%26atitle%3DScoring%2520functions%2520for%2520prediction%2520of%2520protein-ligand%2520interactions%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26issue%3D12%26spage%3D2174%26epage%3D2182%26doi%3D10.2174%2F1381612811319120005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref162"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref162'); return false;" data-citation="" class="refNumLink">162</a></strong><div class="NLM_citation" id="cit162"><span><span class="NLM_contrib-group">Wikberg, J. E. S.; Lapinsh, M.; Prusis, P.</span><span> </span><span class="NLM_article-title">Proteochemometrics: a tool for modeling the molecular interaction space</span>. In  <span class="citation_source-book">Chemogenomics in Drug Discovery</span>; <span class="NLM_contrib-group">Kubinyi, H.; Müller, G.</span>, Eds.; Methods and Principles in Medicinal Chemistry; <span class="NLM_publisher-name">Wiley-VCH Verlag GmbH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">289</span><span class="NLM_x">–</span> <span class="NLM_lpage">309</span>, DOI: <span class="refDoi"> DOI: 10.1002/3527603948.ch10</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2F3527603948.ch10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=289-309&author=J.+E.+S.+Wikberg&author=M.+Lapinsh&author=P.+Prusisauthor=H.+Kubinyi&author=G.+M%C3%BCller&title=Chemogenomics+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1002%2F3527603948.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527603948.ch10%26sid%3Dliteratum%253Aachs%26aulast%3DWikberg%26aufirst%3DJ.%2BE.%2BS.%26atitle%3DProteochemometrics%253A%2520a%2520tool%2520for%2520modeling%2520the%2520molecular%2520interaction%2520space%26btitle%3DChemogenomics%2520in%2520Drug%2520Discovery%26aulast%3DKubinyi%26aufirst%3DH.%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26date%3D2004%26spage%3D289%26epage%3D309%26doi%3D10.1002%2F3527603948.ch10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref163"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref163'); return false;" data-citation="" class="refNumLink">163</a></strong><div class="NLM_citation" id="cit163"><span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">The PDBbind database: collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2977</span><span class="NLM_x">–</span> <span class="NLM_lpage">2980</span><span class="refDoi"> DOI: 10.1021/jm030580l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030580l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2977-2980&issue=12&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=S.+Wang&title=The+PDBbind+database%3A+collection+of+binding+affinities+for+protein-ligand+complexes+with+known+three-dimensional+structures&doi=10.1021%2Fjm030580l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span></div><div class="casAuthors">Wang, Renxiao; Fang, Xueliang; Lu, Yipin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2977-2980</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have screened the entire Protein Data Bank (Release No. 103, Jan. 2003) and identified 5671 protein-ligand complexes out of 19 621 exptl. structures.  A systematic examn. of the primary refs. of these entries has led to a collection of binding affinity data (Kd, Ki, and IC50) for a total of 1359 complexes.  The outcomes of this project have been organized into a Web-accessible database named the PDBbind database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1eUdeWioYBbVg90H21EOLACvtfcHk0lh0oIT1ji_Hxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D&md5=86e609172307402d8b0d4589b1270a2f</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Fjm030580l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030580l%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520collection%2520of%2520binding%2520affinities%2520for%2520protein-ligand%2520complexes%2520with%2520known%2520three-dimensional%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D12%26spage%3D2977%26epage%3D2980%26doi%3D10.1021%2Fjm030580l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref164"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref164'); return false;" data-citation="" class="refNumLink">164</a></strong><div class="NLM_citation" id="cit164"><span><span class="NLM_contrib-group">Olsson, T. S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladbury, J. E.</span><span> </span><span class="NLM_article-title">The thermodynamics of protein-ligand interaction and solvation: insights for ligand design</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1002</span><span class="NLM_x">–</span> <span class="NLM_lpage">1017</span><span class="refDoi"> DOI: 10.1016/j.jmb.2008.09.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.jmb.2008.09.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18930735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWls7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2008&pages=1002-1017&issue=4&author=T.+S.+G.+Olssonauthor=M.+A.+Williamsauthor=W.+R.+Pittauthor=J.+E.+Ladbury&title=The+thermodynamics+of+protein-ligand+interaction+and+solvation%3A+insights+for+ligand+design&doi=10.1016%2Fj.jmb.2008.09.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">The Thermodynamics of Protein-Ligand Interaction and Solvation: Insights for Ligand Design</span></div><div class="casAuthors">Olsson, Tjelvar S. G.; Williams, Mark A.; Pitt, William R.; Ladbury, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1002-1017</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry is able to provide accurate information on the thermodn. contributions of enthalpy and entropy changes to free energies of binding.  The Structure/Calorimetry of Reported Protein Interactions Online (SCORPIO) database of published isothermal titrn. calorimetry studies and structural information on the interactions between proteins and small-mol. ligands is used here to reveal general thermodn. properties of protein-ligand interactions and to investigate correlations with changes in solvation.  The overwhelming majority of interactions are found to be enthalpically favored.  Synthetic inhibitors and biol. ligands form two distinct subpopulations in the data, with the former having greater av. affinity due to more favorable entropy changes on binding.  The greatest correlation is found between the binding free energy and apolar surface burial upon complex formation.  However, the free energy contribution per unit area buried is only 30-50% of that expected from earlier studies of transfer free energies of small mols.  A simple probability-based estimator for the maximal affinity of a binding site in terms of its apolar surface area is proposed.  Polar surface area burial also contributes substantially to affinity but is difficult to express in terms of unit area due to the small variation in the amt. of polar surface buried and a tendency for cancellation of its enthalpic and entropic contributions.  Conventionally, the contribution of apolar desolvation to affinity is attributed to gain of entropy due to solvent release.  Although data presented here are supportive of this notion, because the correlation of entropy change with apolar surface burial is relatively weak, it cannot, on present evidence, be confidently considered to be correct.  Further, thermodn. changes arising from small differences between ligands binding to individual proteins are relatively large and, in general, uncorrelated with changes in solvation, suggesting that trends identified across widely differing proteins are of limited use in explaining or predicting the effects of ligand modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xdclY7Rk_bVg90H21EOLACvtfcHk0lh0oIT1ji_Hxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWls7zO&md5=f9d6d3e8a7fd85d7ff7a6dc6acf7309b</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.09.073%26sid%3Dliteratum%253Aachs%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DWilliams%26aufirst%3DM.%2BA.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520thermodynamics%2520of%2520protein-ligand%2520interaction%2520and%2520solvation%253A%2520insights%2520for%2520ligand%2520design%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D384%26issue%3D4%26spage%3D1002%26epage%3D1017%26doi%3D10.1016%2Fj.jmb.2008.09.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref165"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref165'); return false;" data-citation="" class="refNumLink">165</a></strong><div class="NLM_citation" id="cit165"><span><span class="NLM_contrib-group">Wu, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwa, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C.</span><span> </span><span class="NLM_article-title">Hidden relationship between conserved residues and locally conserved phosphate-binding structures in NAD(P)-binding proteins</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x"> (</span><span class="NLM_issue">19</span><span class="NLM_x">) </span> <span class="NLM_fpage">5644</span><span class="NLM_x">–</span> <span class="NLM_lpage">5652</span><span class="refDoi"> DOI: 10.1021/jp3014332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp3014332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2012&pages=5644-5652&issue=19&author=C.+Y.+Wuauthor=Y.+H.+Hwaauthor=Y.+C.+Chenauthor=C.+Lim&title=Hidden+relationship+between+conserved+residues+and+locally+conserved+phosphate-binding+structures+in+NAD%28P%29-binding+proteins&doi=10.1021%2Fjp3014332"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1021%2Fjp3014332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp3014332%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DC.%2BY.%26aulast%3DHwa%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DLim%26aufirst%3DC.%26atitle%3DHidden%2520relationship%2520between%2520conserved%2520residues%2520and%2520locally%2520conserved%2520phosphate-binding%2520structures%2520in%2520NAD%2528P%2529-binding%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2012%26volume%3D116%26issue%3D19%26spage%3D5644%26epage%3D5652%26doi%3D10.1021%2Fjp3014332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref166"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref166'); return false;" data-citation="" class="refNumLink">166</a></strong><div class="NLM_citation" id="cit166"><span><span class="NLM_contrib-group">Huang, B.</span><span> </span><span class="NLM_article-title">Identification of pockets on protein surface to predict protein–ligand binding sites</span>. In  <span class="citation_source-book">Identification of Ligand Binding Site and Protein–Protein Interaction Area</span>; <span class="NLM_contrib-group">Roterman-Konieczna, I.</span>, Ed.; Focus on Structural Biology; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Dordrecht, The Netherlands</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2F978-94-007-5285-6_2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=25-39&author=B.+Huangauthor=I.+Roterman-Konieczna&title=Identification+of+Ligand+Binding+Site+and+Protein%E2%80%93Protein+Interaction+Area"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1007%2F978-94-007-5285-6_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-007-5285-6_2%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520pockets%2520on%2520protein%2520surface%2520to%2520predict%2520protein%25E2%2580%2593ligand%2520binding%2520sites%26btitle%3DIdentification%2520of%2520Ligand%2520Binding%2520Site%2520and%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Area%26aulast%3DRoterman-Konieczna%26aufirst%3DI.%26pub%3DSpringer%26date%3D2013%26spage%3D25%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref167"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref167'); return false;" data-citation="" class="refNumLink">167</a></strong><div class="NLM_citation" id="cit167"><span><span class="NLM_contrib-group">Holm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, C.</span><span> </span><span class="NLM_article-title">Dali: a network tool for protein structure comparison</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">478</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span><span class="refDoi"> DOI: 10.1016/S0968-0004(00)89105-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0968-0004%2800%2989105-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=8578593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2MXpsF2murY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1995&pages=478-480&issue=11&author=L.+Holmauthor=C.+Sander&title=Dali%3A+a+network+tool+for+protein+structure+comparison&doi=10.1016%2FS0968-0004%2800%2989105-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Dali: a network tool for protein structure comparison</span></div><div class="casAuthors">Holm, Liisa; Sander, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">478-80</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Trends Journals</span>)
        </div><div class="casAbstract">The recent surge in newly solved 3-dimensional protein structures has whetted the appetite for systematic structure-structure comparison with the potential for evolutionary discoveries.  The EMBL is now providing Internet (<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="4a2e2b26230a2f27282667222f232e2f26282f382d642e2f">[email protected]</a>) and World Wide Web access to the Dali method for protein structure comparison.  The Dali method measures structural relatedness in terms of similarities of intramol. distance matrixes (i.e., the alignments are detd. only by comparing 3-dimensional coordinates).  The services include a database of precalcd. structural neighbors for all public structures and an e-mail server for searches with newly solved ones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmJryprgsB8LVg90H21EOLACvtfcHk0lgkwOIcUG2gmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsF2murY%253D&md5=78c8ba072569f09ec94e32bbbd8bfed1</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2800%2989105-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252800%252989105-7%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DSander%26aufirst%3DC.%26atitle%3DDali%253A%2520a%2520network%2520tool%2520for%2520protein%2520structure%2520comparison%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1995%26volume%3D20%26issue%3D11%26spage%3D478%26epage%3D480%26doi%3D10.1016%2FS0968-0004%2800%2989105-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref168"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref168'); return false;" data-citation="" class="refNumLink">168</a></strong><div class="NLM_citation" id="cit168"><span><span class="NLM_contrib-group">Nadzirin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Firdaus-Raih, M.</span><span> </span><span class="NLM_article-title">Proteins of unknown function in the Protein Data Bank (PDB): an inventory of true uncharacterized proteins and computational tools for their analysis</span> <span class="citation_source-journal">Int. J. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">12761</span><span class="NLM_x">–</span> <span class="NLM_lpage">12772</span><span class="refDoi"> DOI: 10.3390/ijms131012761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.3390%2Fijms131012761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=23202924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaqtbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=12761-12772&issue=10&author=N.+Nadzirinauthor=M.+Firdaus-Raih&title=Proteins+of+unknown+function+in+the+Protein+Data+Bank+%28PDB%29%3A+an+inventory+of+true+uncharacterized+proteins+and+computational+tools+for+their+analysis&doi=10.3390%2Fijms131012761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">Proteins of unknown function in the Protein Data Bank (PDB): an inventory of true uncharacterized proteins and computational tools for their analysis</span></div><div class="casAuthors">Nadzirin, Nurul; Firdaus-Raih, Mohd</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12761-12772</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Proteins of uncharacterized functions form a large part of many of the currently available biol. databases and this situation exists even in the Protein Data Bank (PDB).  Our anal. of recent PDB data revealed that only 42.53% of PDB entries (1084 coordinate files) that were categorized under "unknown function" are true examples of proteins of unknown function at this point in time.  The remainder 1465 entries also annotated as such appear to be able to have their annotations re-assessed, based on the availability of direct functional characterization expts. for the protein itself, or for homologous sequences or structures thus enabling computational function inference.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotBQpdM4xXubVg90H21EOLACvtfcHk0lgkwOIcUG2gmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaqtbnO&md5=f67070c41c38363b79caa52e11000a37</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.3390%2Fijms131012761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms131012761%26sid%3Dliteratum%253Aachs%26aulast%3DNadzirin%26aufirst%3DN.%26aulast%3DFirdaus-Raih%26aufirst%3DM.%26atitle%3DProteins%2520of%2520unknown%2520function%2520in%2520the%2520Protein%2520Data%2520Bank%2520%2528PDB%2529%253A%2520an%2520inventory%2520of%2520true%2520uncharacterized%2520proteins%2520and%2520computational%2520tools%2520for%2520their%2520analysis%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2012%26volume%3D13%26issue%3D10%26spage%3D12761%26epage%3D12772%26doi%3D10.3390%2Fijms131012761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref169"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref169'); return false;" data-citation="" class="refNumLink">169</a></strong><div class="NLM_citation" id="cit169"><span><span class="NLM_contrib-group">Petrey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garzon, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasso, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silkov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Template-based prediction of protein function</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1016/j.sbi.2015.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.sbi.2015.01.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25678152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVCqu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2015&pages=33-38&author=D.+Petreyauthor=T.+S.+Chenauthor=L.+Dengauthor=J.+I.+Garzonauthor=H.+Hwangauthor=G.+Lassoauthor=H.+Leeauthor=A.+Silkovauthor=B.+Honig&title=Template-based+prediction+of+protein+function&doi=10.1016%2Fj.sbi.2015.01.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">Template-based prediction of protein function</span></div><div class="casAuthors">Petrey, Donald; Chen, T. Scott; Deng, Lei; Garzon, Jose Ignacio; Hwang, Howook; Lasso, Gorka; Lee, Hunjoong; Silkov, Antonina; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33-38</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  We discuss recent approaches for structure-based protein function annotation.  We focus on template-based methods where the function of a query protein is deduced from that of a template for which both the structure and function are known.  We describe the different ways of identifying a template.  These are typically based on sequence anal. but new methods based on purely structural similarity are also being developed that allow function annotation based on structural relationships that cannot be recognized by sequence.  The growing no. of available structures of known function, improved homol. modeling techniques and new developments in the use of structure allow template-based methods to be applied on a proteome-wide scale and in many different biol. contexts.  This progress significantly expands the range of applicability of structural information in function annotation to a level that previously was only achievable by sequence comparison.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobUm3rclLCLbVg90H21EOLACvtfcHk0lgi35w61SKTWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVCqu7Y%253D&md5=fa08b4069309cbd6bcc350a0158ed591</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2015.01.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2015.01.007%26sid%3Dliteratum%253Aachs%26aulast%3DPetrey%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%2BS.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DGarzon%26aufirst%3DJ.%2BI.%26aulast%3DHwang%26aufirst%3DH.%26aulast%3DLasso%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DSilkov%26aufirst%3DA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DTemplate-based%2520prediction%2520of%2520protein%2520function%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2015%26volume%3D32%26spage%3D33%26epage%3D38%26doi%3D10.1016%2Fj.sbi.2015.01.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref170"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref170'); return false;" data-citation="" class="refNumLink">170</a></strong><div class="NLM_citation" id="cit170"><span><span class="NLM_contrib-group">Hwang, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, Y.</span><span> </span><span class="NLM_article-title">Structure-based identification of a novel NTPase from Methanococcus jannaschii</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">696</span><span class="refDoi"> DOI: 10.1038/10745</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2F10745" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=691-696&issue=7&author=K.+Y.+Hwangauthor=J.+H.+Chungauthor=S.+H.+Kimauthor=Y.+S.+Hanauthor=Y.+Cho&title=Structure-based+identification+of+a+novel+NTPase+from+Methanococcus+jannaschii&doi=10.1038%2F10745"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1038%2F10745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F10745%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DK.%2BY.%26aulast%3DChung%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DHan%26aufirst%3DY.%2BS.%26aulast%3DCho%26aufirst%3DY.%26atitle%3DStructure-based%2520identification%2520of%2520a%2520novel%2520NTPase%2520from%2520Methanococcus%2520jannaschii%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26issue%3D7%26spage%3D691%26epage%3D696%26doi%3D10.1038%2F10745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref171"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref171'); return false;" data-citation="" class="refNumLink">171</a></strong><div class="NLM_citation" id="cit171"><span><span class="NLM_contrib-group">Pieper, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, G. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneidman-Duhovny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spill, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinkam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">ModBase, a database of annotated comparative protein structure models and associated resources</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D336</span><span class="NLM_x">–</span> <span class="NLM_lpage">D346</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkt1144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkt1144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24271400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Gj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D336-D346&issue=D1&author=U.+Pieperauthor=B.+M.+Webbauthor=G.+Q.+Dongauthor=D.+Schneidman-Duhovnyauthor=H.+Fanauthor=S.+J.+Kimauthor=N.+Khuriauthor=Y.+G.+Spillauthor=P.+Weinkamauthor=M.+Hammelauthor=J.+A.+Tainerauthor=M.+Nilgesauthor=A.+Sali&title=ModBase%2C+a+database+of+annotated+comparative+protein+structure+models+and+associated+resources&doi=10.1093%2Fnar%2Fgkt1144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">ModBase, a database of annotated comparative protein structure models and associated resources</span></div><div class="casAuthors">Pieper, Ursula; Webb, Benjamin M.; Dong, Guang Qiang; Schneidman-Duhovny, Dina; Fan, Hao; Kim, Seung Joong; Khuri, Natalia; Spill, Yannick G.; Weinkam, Patrick; Hammel, Michal; Tainer, John A.; Nilges, Michael; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D336-D346</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ModBase http://salilab.org/modbase is a database of annotated comparative protein structure models.  The models are calcd. by ModPipe, an automated modeling pipeline that relies primarily on Modeller for fold assignment, sequence-structure alignment, model building and model assessment http://salilab.org/modeller/.  ModBase currently contains almost 30 million reliable models for domains in 4.7 million unique protein sequences.  ModBase allows users to compute or update comparative models on demand, through an interface to the ModWeb modeling server http://salilab.org/modweb.  ModBase models are also available through the Protein Model Portal http://www.proteinmodelportal.org/.  Recently developed assocd. resources include the AllosMod server for modeling ligand-induced protein dynamics http://salilab.org/allosmod, the AllosMod-FoXS server for predicting a structural ensemble that fits an SAXS profile http://salilab.org/allosmod-foxs, the FoXSDock server for protein-protein docking filtered by an SAXS profile http://salilab.org/foxsdock, the SAXS Merge server for automatic merging of SAXS profiles http://salilab.org/saxsmerge and the Pose & Rank server for scoring protein-ligand complexes http://salilab.org/poseandrank. In this update, we also highlight two applications of ModBase: a PSI:Biol. initiative to maximize the structural coverage of the human alpha-helical transmembrane proteome and a detn. of structural determinants of human immunodeficiency virus-1 protease specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomlF6VKj_t4bVg90H21EOLACvtfcHk0lgi35w61SKTWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Gj&md5=1a0809c46145a0a783ad5d5b2a6e3326</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1144%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DU.%26aulast%3DWebb%26aufirst%3DB.%2BM.%26aulast%3DDong%26aufirst%3DG.%2BQ.%26aulast%3DSchneidman-Duhovny%26aufirst%3DD.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DKhuri%26aufirst%3DN.%26aulast%3DSpill%26aufirst%3DY.%2BG.%26aulast%3DWeinkam%26aufirst%3DP.%26aulast%3DHammel%26aufirst%3DM.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModBase%252C%2520a%2520database%2520of%2520annotated%2520comparative%2520protein%2520structure%2520models%2520and%2520associated%2520resources%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD336%26epage%3DD346%26doi%3D10.1093%2Fnar%2Fgkt1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref172"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref172'); return false;" data-citation="" class="refNumLink">172</a></strong><div class="NLM_citation" id="cit172"><span><span class="NLM_contrib-group">Yin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garen, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyon, E. Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-w.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, M. N. G.</span><span> </span><span class="NLM_article-title">Expression, purification and preliminary crystallographic analysis of O-acetylhomoserine sulfhydrylase from Mycobacterium tuberculosis</span> <span class="citation_source-journal">Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x"> (</span><span class="NLM_issue">Part 8</span><span class="NLM_x">) </span> <span class="NLM_fpage">959</span><span class="NLM_x">–</span> <span class="NLM_lpage">963</span><span class="refDoi"> DOI: 10.1107/S1744309111017611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1107%2FS1744309111017611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21821905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVGkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=959-963&issue=Part+8&author=J.+Yinauthor=C.+R.+Garenauthor=K.+Batemanauthor=M.+Yuauthor=E.+Z.+A.+Lyonauthor=J.+Habelauthor=H.+Kimauthor=L.-w.+Hungauthor=C.-Y.+Kimauthor=M.+N.+G.+James&title=Expression%2C+purification+and+preliminary+crystallographic+analysis+of+O-acetylhomoserine+sulfhydrylase+from+Mycobacterium+tuberculosis&doi=10.1107%2FS1744309111017611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification and preliminary crystallographic analysis of O-acetylhomoserine sulfhydrylase from Mycobacterium tuberculosis</span></div><div class="casAuthors">Yin, Jiang; Garen, Craig R.; Bateman, Katherine; Yu, Minmin; Alipio Lyon, Emily Z.; Habel, Jeff; Kim, Heungbok; Hung, Li-wei; Kim, Chang-Yub; James, Michael N. G.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology and Crystallization Communications</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">959-963</span>CODEN:
                <span class="NLM_cas:coden">ACSFCL</span>;
        ISSN:<span class="NLM_cas:issn">1744-3091</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The gene product of open reading frame Rv3340 of M. tuberculosis was annotated as encoding a probable O-acetylhomoserine sulfhydrylase (MetC), an enzyme that catalyzes the last step in the biosynthesis of methionine, which is an essential amino acid in bacteria and plants.  Following overexpression in Escherichia coli, the M. tuberculosis MetC enzyme was purified and crystd. using the hanging-drop vapor-diffusion method.  Native diffraction data were collected from 2 crystal forms belonging to space group P21 and were processed to a resoln. of 2.1 Å.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcFq5HnJcd8rVg90H21EOLACvtfcHk0lgi35w61SKTWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVGkurY%253D&md5=782fae003f14f408db2660e18e5ece49</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1107%2FS1744309111017611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309111017611%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DGaren%26aufirst%3DC.%2BR.%26aulast%3DBateman%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DLyon%26aufirst%3DE.%2BZ.%2BA.%26aulast%3DHabel%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DHung%26aufirst%3DL.-w.%26aulast%3DKim%26aufirst%3DC.-Y.%26aulast%3DJames%26aufirst%3DM.%2BN.%2BG.%26atitle%3DExpression%252C%2520purification%2520and%2520preliminary%2520crystallographic%2520analysis%2520of%2520O-acetylhomoserine%2520sulfhydrylase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2011%26volume%3D67%26issue%3DPart%25208%26spage%3D959%26epage%3D963%26doi%3D10.1107%2FS1744309111017611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref173"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref173'); return false;" data-citation="" class="refNumLink">173</a></strong><div class="NLM_citation" id="cit173"><span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">I-TASSER server for protein 3D structure prediction</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="refDoi"> DOI: 10.1186/1471-2105-9-40</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-9-40" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18215316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BD1c7itFaksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=40&author=Y.+Zhang&title=I-TASSER+server+for+protein+3D+structure+prediction&doi=10.1186%2F1471-2105-9-40"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">I-TASSER server for protein 3D structure prediction</span></div><div class="casAuthors">Zhang Yang</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Prediction of 3-dimensional protein structures from amino acid sequences represents one of the most important problems in computational structural biology.  The community-wide Critical Assessment of Structure Prediction (CASP) experiments have been designed to obtain an objective assessment of the state-of-the-art of the field, where I-TASSER was ranked as the best method in the server section of the recent 7th CASP experiment.  Our laboratory has since then received numerous requests about the public availability of the I-TASSER algorithm and the usage of the I-TASSER predictions.  RESULTS:  An on-line version of I-TASSER is developed at the KU Center for Bioinformatics which has generated protein structure predictions for thousands of modeling requests from more than 35 countries.  A scoring function (C-score) based on the relative clustering structural density and the consensus significance score of multiple threading templates is introduced to estimate the accuracy of the I-TASSER predictions.  A large-scale benchmark test demonstrates a strong correlation between the C-score and the TM-score (a structural similarity measurement with values in [0, 1]) of the first models with a correlation coefficient of 0.91.  Using a C-score cutoff > -1.5 for the models of correct topology, both false positive and false negative rates are below 0.1.  Combining C-score and protein length, the accuracy of the I-TASSER models can be predicted with an average error of 0.08 for TM-score and 2 A for RMSD.  CONCLUSION:  The I-TASSER server has been developed to generate automated full-length 3D protein structural predictions where the benchmarked scoring system helps users to obtain quantitative assessments of the I-TASSER models.  The output of the I-TASSER server for each query includes up to five full-length models, the confidence score, the estimated TM-score and RMSD, and the standard deviation of the estimations.  The I-TASSER server is freely available to the academic community at http://zhang.bioinformatics.ku.edu/I-TASSER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUJ3biR9wC1IvKgEt20y1XfW6udTcc2eZ0ZNdkpNIehbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7itFaksQ%253D%253D&md5=48ded5b2e40af27acf1e5753f827080c</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-40&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-40%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DI-TASSER%2520server%2520for%2520protein%25203D%2520structure%2520prediction%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26spage%3D40%26doi%3D10.1186%2F1471-2105-9-40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref174"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref174'); return false;" data-citation="" class="refNumLink">174</a></strong><div class="NLM_citation" id="cit174"><span><span class="NLM_contrib-group">Farrah, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutsch, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoopmann, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallows, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moritz, R. L.</span><span> </span><span class="NLM_article-title">The state of the human proteome in 2012 as viewed through PeptideAtlas</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">162</span><span class="NLM_x">–</span> <span class="NLM_lpage">171</span><span class="refDoi"> DOI: 10.1021/pr301012j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr301012j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOjsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=162-171&issue=1&author=T.+Farrahauthor=E.+W.+Deutschauthor=M.+R.+Hoopmannauthor=J.+L.+Hallowsauthor=Z.+Sunauthor=C.-Y.+Huangauthor=R.+L.+Moritz&title=The+state+of+the+human+proteome+in+2012+as+viewed+through+PeptideAtlas&doi=10.1021%2Fpr301012j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">The State of the Human Proteome in 2012 as Viewed through PeptideAtlas</span></div><div class="casAuthors">Farrah, Terry; Deutsch, Eric W.; Hoopmann, Michael R.; Hallows, Janice L.; Sun, Zhi; Huang, Chung-Ying; Moritz, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">162-171</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Human Proteome Project was launched in Sept. 2010 with the goal of characterizing at least one protein product from each protein-coding gene.  Here we assess how much of the proteome has been detected to date via tandem mass spectrometry by analyzing PeptideAtlas, a compendium of human derived LC-MS/MS proteomics data from many labs. around the world.  All data sets are processed with a consistent set of parameters using the Trans-Proteomic Pipeline and subjected to a 1% protein FDR filter before inclusion in PeptideAtlas.  Therefore, PeptideAtlas contains only high confidence protein identifications.  To increase proteome coverage, we explored new comprehensive public data sources for data likely to add new proteins to the Human PeptideAtlas.  We then folded these data into a Human PeptideAtlas 2012 build and mapped it to Swiss-Prot, a protein sequence database curated to contain one entry per human protein coding gene.  We find that this latest PeptideAtlas build includes at least one peptide for each of ∼12500 Swiss-Prot entries, leaving ∼7500 gene products yet to be confidently cataloged.  We characterize these "PA-unseen" proteins in terms of tissue localization, transcript abundance, and Gene Ontol. enrichment, and propose reasons for their absence from PeptideAtlas and strategies for detecting them in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMvIy0oVVhlrVg90H21EOLACvtfcHk0ljo5cvDJLDxKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOjsb7N&md5=49221bf15c990d2ea6566baf7f4b2961</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1021%2Fpr301012j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr301012j%26sid%3Dliteratum%253Aachs%26aulast%3DFarrah%26aufirst%3DT.%26aulast%3DDeutsch%26aufirst%3DE.%2BW.%26aulast%3DHoopmann%26aufirst%3DM.%2BR.%26aulast%3DHallows%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DC.-Y.%26aulast%3DMoritz%26aufirst%3DR.%2BL.%26atitle%3DThe%2520state%2520of%2520the%2520human%2520proteome%2520in%25202012%2520as%2520viewed%2520through%2520PeptideAtlas%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2013%26volume%3D12%26issue%3D1%26spage%3D162%26epage%3D171%26doi%3D10.1021%2Fpr301012j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref175"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref175'); return false;" data-citation="" class="refNumLink">175</a></strong><div class="NLM_citation" id="cit175"><span><span class="NLM_contrib-group">Caetano-Anollés, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caetano-Anollés, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittenthal, J. E.</span><span> </span><span class="NLM_article-title">The origin, evolution and structure of the protein world</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">417</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">621</span><span class="NLM_x">–</span> <span class="NLM_lpage">637</span><span class="refDoi"> DOI: 10.1042/BJ20082063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1042%2FBJ20082063" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2009&pages=621-637&issue=3&author=G.+Caetano-Anoll%C3%A9sauthor=M.+Wangauthor=D.+Caetano-Anoll%C3%A9sauthor=J.+E.+Mittenthal&title=The+origin%2C+evolution+and+structure+of+the+protein+world&doi=10.1042%2FBJ20082063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1042%2FBJ20082063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20082063%26sid%3Dliteratum%253Aachs%26aulast%3DCaetano-Anoll%25C3%25A9s%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCaetano-Anoll%25C3%25A9s%26aufirst%3DD.%26aulast%3DMittenthal%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520origin%252C%2520evolution%2520and%2520structure%2520of%2520the%2520protein%2520world%26jtitle%3DBiochem.%2520J.%26date%3D2009%26volume%3D417%26issue%3D3%26spage%3D621%26epage%3D637%26doi%3D10.1042%2FBJ20082063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref176"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref176'); return false;" data-citation="" class="refNumLink">176</a></strong><div class="NLM_citation" id="cit176"><span><span class="NLM_contrib-group">Naumann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matter, H.</span><span> </span><span class="NLM_article-title">Structural classification of protein kinases using 3D molecular interaction field analysis of their ligand binding sites: target family landscapes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2366</span><span class="NLM_x">–</span> <span class="NLM_lpage">2378</span><span class="refDoi"> DOI: 10.1021/jm011002c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm011002c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2366-2378&issue=12&author=T.+Naumannauthor=H.+Matter&title=Structural+classification+of+protein+kinases+using+3D+molecular+interaction+field+analysis+of+their+ligand+binding+sites%3A+target+family+landscapes&doi=10.1021%2Fjm011002c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1021%2Fjm011002c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm011002c%26sid%3Dliteratum%253Aachs%26aulast%3DNaumann%26aufirst%3DT.%26aulast%3DMatter%26aufirst%3DH.%26atitle%3DStructural%2520classification%2520of%2520protein%2520kinases%2520using%25203D%2520molecular%2520interaction%2520field%2520analysis%2520of%2520their%2520ligand%2520binding%2520sites%253A%2520target%2520family%2520landscapes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26issue%3D12%26spage%3D2366%26epage%3D2378%26doi%3D10.1021%2Fjm011002c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref177"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref177'); return false;" data-citation="" class="refNumLink">177</a></strong><div class="NLM_citation" id="cit177"><span><span class="NLM_contrib-group">Altschul, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schäffer, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipman, D. J.</span><span> </span><span class="NLM_article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">3389</span><span class="NLM_x">–</span> <span class="NLM_lpage">3402</span><span class="refDoi"> DOI: 10.1093/nar/25.17.3389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2F25.17.3389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=9254694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=3389-3402&issue=17&author=S.+F.+Altschulauthor=T.+L.+Maddenauthor=A.+A.+Sch%C3%A4fferauthor=J.+Zhangauthor=Z.+Zhangauthor=W.+Millerauthor=D.+J.+Lipman&title=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs&doi=10.1093%2Fnar%2F25.17.3389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span></div><div class="casAuthors">Altschul, Stephen F.; Madden, Thomas L.; Schaffer, Alejandro A.; Zhang, Jinghui; Zhang, Zheng; Miller, Webb; Lipman, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3389-3402</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The BLAST programs are widely used tools for searching protein and DNA databases for sequence similarities.  For protein comparisons, a variety of definitional, algorithmic and statistical refinements described here permits the execution time of the BLAST programs to be decreased substantially while enhancing their sensitivity to weak similarities.  A new criterion for triggering the extension of word hits, combined with a new heuristic for generating gapped alignments, yields a gapped BLAST program that runs at approx. three times the speed of the original.  In addn., a method is introduced for automatically combining statistically significant alignments produced by BLAST into a position-specific score matrix, and searching the database using this matrix.  The resulting Position-Specific Iterated BLAST (PSI-BLAST) program runs at approx. the same speed per iteration as gapped BLAST, but in many cases is much more sensitive to weak but biol. relevant sequence similarities.  PSI-BLAST is used to uncover several new and interesting members of the BRCT superfamily.  The source code for the new BLAST programs is available by anonymous ftp from the machine ncbi.nlm.nih.gov, within the directory 'blast', and the programs may be run from NCBIs web site at http://www.ncbi.nlm.nih.gov/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfl-GGLfjeoLVg90H21EOLACvtfcHk0lgArvSRX0g2Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D&md5=4e44123e5984e4aca46a9899d347a176</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F25.17.3389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F25.17.3389%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26aulast%3DMadden%26aufirst%3DT.%2BL.%26aulast%3DSch%25C3%25A4ffer%26aufirst%3DA.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DMiller%26aufirst%3DW.%26aulast%3DLipman%26aufirst%3DD.%2BJ.%26atitle%3DGapped%2520BLAST%2520and%2520PSI-BLAST%253A%2520a%2520new%2520generation%2520of%2520protein%2520database%2520search%2520programs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26issue%3D17%26spage%3D3389%26epage%3D3402%26doi%3D10.1093%2Fnar%2F25.17.3389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref178"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref178'); return false;" data-citation="" class="refNumLink">178</a></strong><div class="NLM_citation" id="cit178"><span><span class="NLM_contrib-group">Bairoch, A.</span><span> </span><span class="NLM_article-title">PROSITE: a dictionary of sites and patterns in proteins</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl</span><span class="NLM_x">) </span> <span class="NLM_fpage">2241</span><span class="NLM_x">–</span> <span class="NLM_lpage">2245</span><span class="refDoi"> DOI: 10.1093/nar/19.suppl.2241</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2F19.suppl.2241" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1991&pages=2241-2245&issue=Suppl&author=A.+Bairoch&title=PROSITE%3A+a+dictionary+of+sites+and+patterns+in+proteins&doi=10.1093%2Fnar%2F19.suppl.2241"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F19.suppl.2241&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F19.suppl.2241%26sid%3Dliteratum%253Aachs%26aulast%3DBairoch%26aufirst%3DA.%26atitle%3DPROSITE%253A%2520a%2520dictionary%2520of%2520sites%2520and%2520patterns%2520in%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1991%26volume%3D19%26issue%3DSuppl%26spage%3D2241%26epage%3D2245%26doi%3D10.1093%2Fnar%2F19.suppl.2241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref179"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref179'); return false;" data-citation="" class="refNumLink">179</a></strong><div class="NLM_citation" id="cit179"><span><span class="NLM_contrib-group">Chupakhin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaspar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varnek, A.</span><span> </span><span class="NLM_article-title">Simple Ligand-Receptor Interaction Descriptor (SILIRID) for alignment-free binding site comparison</span> <span class="citation_source-journal">Comput. Struct. Biotechnol. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">16</span><span class="NLM_x">) </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">37</span><span class="refDoi"> DOI: 10.1016/j.csbj.2014.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.csbj.2014.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25210596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BC2M7gsVSqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=33-37&issue=16&author=V.+Chupakhinauthor=G.+Marcouauthor=H.+Gasparauthor=A.+Varnek&title=Simple+Ligand-Receptor+Interaction+Descriptor+%28SILIRID%29+for+alignment-free+binding+site+comparison&doi=10.1016%2Fj.csbj.2014.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">Simple Ligand-Receptor Interaction Descriptor (SILIRID) for alignment-free binding site comparison</span></div><div class="casAuthors">Chupakhin Vladimir; Marcou Gilles; Gaspar Helena; Varnek Alexandre</div><div class="citationInfo"><span class="NLM_cas:title">Computational and structural biotechnology journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">33-7</span>
        ISSN:<span class="NLM_cas:issn">2001-0370</span>.
    </div><div class="casAbstract">We describe SILIRID (Simple Ligand-Receptor Interaction Descriptor), a novel fixed size descriptor characterizing protein-ligand interactions.  SILIRID can be obtained from the binary interaction fingerprints (IFPs) by summing up the bits corresponding to identical amino acids.  This results in a vector of 168 integer numbers corresponding to the product of the number of entries (20 amino acids and one cofactor) and 8 interaction types per amino acid (hydrophobic, aromatic face to face, aromatic edge to face, H-bond donated by the protein, H-bond donated by the ligand, ionic bond with protein cation and protein anion, and interaction with metal ion).  Efficiency of SILIRID to distinguish different protein binding sites has been examined in similarity search in sc-PDB database, a druggable portion of the Protein Data Bank, using various protein-ligand complexes as queries.  The performance of retrieval of structurally and evolutionary related classes of proteins was comparable to that of state-of-the-art approaches (ROC AUC ≈ 0.91).  SILIRID can efficiently be used to visualize chemogenomic space covered by sc-PDB using Generative Topographic Mapping (GTM): sc-PDB SILIRID data form clusters corresponding to different protein types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSrYGB8Py17ciA3UY9EwTNrfW6udTcc2eYdW48VNgEf_bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7gsVSqug%253D%253D&md5=6c4111f9f23033e5365f1c46ca40226f</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1016%2Fj.csbj.2014.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.csbj.2014.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DChupakhin%26aufirst%3DV.%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DGaspar%26aufirst%3DH.%26aulast%3DVarnek%26aufirst%3DA.%26atitle%3DSimple%2520Ligand-Receptor%2520Interaction%2520Descriptor%2520%2528SILIRID%2529%2520for%2520alignment-free%2520binding%2520site%2520comparison%26jtitle%3DComput.%2520Struct.%2520Biotechnol.%2520J.%26date%3D2014%26volume%3D10%26issue%3D16%26spage%3D33%26epage%3D37%26doi%3D10.1016%2Fj.csbj.2014.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref180"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref180'); return false;" data-citation="" class="refNumLink">180</a></strong><div class="NLM_citation" id="cit180"><span><span class="NLM_contrib-group">Chartier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R.</span><span> </span><span class="NLM_article-title">Detection of binding site molecular interaction field similarities</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1600</span><span class="NLM_x">–</span> <span class="NLM_lpage">1615</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00333</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00333" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1600-1615&issue=8&author=M.+Chartierauthor=R.+Najmanovich&title=Detection+of+binding+site+molecular+interaction+field+similarities&doi=10.1021%2Facs.jcim.5b00333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of Binding Site Molecular Interaction Field Similarities</span></div><div class="casAuthors">Chartier, Matthieu; Najmanovich, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1600-1615</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein binding-site similarity detection methods can be used to predict protein function and understand mol. recognition, as a tool in drug design for drug repurposing and polypharmacol., and for the prediction of the mol. determinants of drug toxicity.  Here, the authors present IsoMIF, a method able to identify binding site mol. interaction field similarities across protein families.  IsoMIF utilizes six chem. probes and the detection of subgraph isomorphisms to identify geometrically and chem. equiv. sections of protein cavity pairs.  The method is validated using six distinct data sets, four of those previously used in the validation of other methods.  The mean area under the receiver operator curve (AUC) obtained across data sets for IsoMIF is higher than those of other methods.  Furthermore, while IsoMIF obtains consistently high AUC values across data sets, other methods perform more erratically across data sets.  IsoMIF can be used to predict function from structure, to detect potential cross-reactivity or polypharmacol. targets, and to help suggest bioisosteric replacements to known binding mols.  Given that IsoMIF detects spatial patterns of mol. interaction field similarities, its predictions are directly related to pharmacophores and may be readily translated into modeling decisions in structure-based drug design.  IsoMIF may in principle detect similar binding sites with distinct amino acid arrangements that lead to equiv. interactions within the cavity.  The source code to calc. and visualize MIFs and MIF similarities are freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-qggIKAgadbVg90H21EOLACvtfcHk0libcUkuYLlQYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfK&md5=4fdab34c9233108eb40b9a0e7052c357</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00333%26sid%3Dliteratum%253Aachs%26aulast%3DChartier%26aufirst%3DM.%26aulast%3DNajmanovich%26aufirst%3DR.%26atitle%3DDetection%2520of%2520binding%2520site%2520molecular%2520interaction%2520field%2520similarities%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D8%26spage%3D1600%26epage%3D1615%26doi%3D10.1021%2Facs.jcim.5b00333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref181"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref181'); return false;" data-citation="" class="refNumLink">181</a></strong><div class="NLM_citation" id="cit181"><span><span class="NLM_contrib-group">Kinjo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Similarity search for local protein structures at atomic resolution by exploiting a database management system</span> <span class="citation_source-journal">Biophysics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.2142/biophysics.3.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2142%2Fbiophysics.3.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1agu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=75-84&author=A.+R.+Kinjoauthor=H.+Nakamura&title=Similarity+search+for+local+protein+structures+at+atomic+resolution+by+exploiting+a+database+management+system&doi=10.2142%2Fbiophysics.3.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit181R"><div class="casContent"><span class="casTitleNuber">181</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity search for local protein structures at atomic resolution by exploiting a database management system</span></div><div class="casAuthors">Kinjo, Akira R.; Nakamura, Haruki</div><div class="citationInfo"><span class="NLM_cas:title">Biophysics (Biophysical Society of Japan)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-84</span>CODEN:
                <span class="NLM_cas:coden">BBSJC3</span>;
        ISSN:<span class="NLM_cas:issn">1349-2942</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society of Japan</span>)
        </div><div class="casAbstract">A method to search for local structural similarities in proteins at at. resoln. is presented.  It is demonstrated that a huge amt. of structural data can be handled within a reasonable CPU time by using a conventional relational database management system with appropriate indexing of geometric data.  This method, which we call geometric indexing, can enumerate ligand binding sites that are structurally similar to sub-structures of a query protein among more than 160,000 possible candidates within a few hours of CPU time on an ordinary desktop computer.  After detecting a set of high scoring ligand binding sites by the geometric indexing search, structural alignments at at. resoln. are constructed by iteratively applying the Hungarian algorithm, and the statistical significance of the final score is estd. from an empirical model based on a gamma distribution.  Applications of this method to several protein structures clearly shows that significant similarities can be detected between local structures of non-homologous as well as homologous proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7yLbA2BOqCrVg90H21EOLACvtfcHk0libcUkuYLlQYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1agu7Y%253D&md5=a9c57dd41e31f99177bd977def356283</span></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.2142%2Fbiophysics.3.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2142%252Fbiophysics.3.75%26sid%3Dliteratum%253Aachs%26aulast%3DKinjo%26aufirst%3DA.%2BR.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DSimilarity%2520search%2520for%2520local%2520protein%2520structures%2520at%2520atomic%2520resolution%2520by%2520exploiting%2520a%2520database%2520management%2520system%26jtitle%3DBiophysics%26date%3D2007%26volume%3D3%26spage%3D75%26epage%3D84%26doi%3D10.2142%2Fbiophysics.3.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref182"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref182'); return false;" data-citation="" class="refNumLink">182</a></strong><div class="NLM_citation" id="cit182"><span><span class="NLM_contrib-group">Kinjo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Comprehensive structural classification of ligand-binding motifs in proteins</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">234</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1016/j.str.2008.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.str.2008.11.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=234-246&issue=2&author=A.+R.+Kinjoauthor=H.+Nakamura&title=Comprehensive+structural+classification+of+ligand-binding+motifs+in+proteins&doi=10.1016%2Fj.str.2008.11.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2008.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2008.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DKinjo%26aufirst%3DA.%2BR.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DComprehensive%2520structural%2520classification%2520of%2520ligand-binding%2520motifs%2520in%2520proteins%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26issue%3D2%26spage%3D234%26epage%3D246%26doi%3D10.1016%2Fj.str.2008.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref183"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref183'); return false;" data-citation="" class="refNumLink">183</a></strong><div class="NLM_citation" id="cit183"><span><span class="NLM_contrib-group">Friedberg, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godzik, A.</span><span> </span><span class="NLM_article-title">Connecting the protein structure universe by using sparse recurring fragments</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1213</span><span class="NLM_x">–</span> <span class="NLM_lpage">1224</span><span class="refDoi"> DOI: 10.1016/j.str.2005.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.str.2005.05.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=1213-1224&issue=8&author=I.+Friedbergauthor=A.+Godzik&title=Connecting+the+protein+structure+universe+by+using+sparse+recurring+fragments&doi=10.1016%2Fj.str.2005.05.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2005.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2005.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DFriedberg%26aufirst%3DI.%26aulast%3DGodzik%26aufirst%3DA.%26atitle%3DConnecting%2520the%2520protein%2520structure%2520universe%2520by%2520using%2520sparse%2520recurring%2520fragments%26jtitle%3DStructure%26date%3D2005%26volume%3D13%26issue%3D8%26spage%3D1213%26epage%3D1224%26doi%3D10.1016%2Fj.str.2005.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref184"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref184'); return false;" data-citation="" class="refNumLink">184</a></strong><div class="NLM_citation" id="cit184"><span><span class="NLM_contrib-group">Godshall, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B. Y.</span><span> </span><span class="NLM_article-title">An aggregate analysis of many predicted structures to reduce errors in protein structure comparison caused by conformational flexibility</span> <span class="citation_source-journal">BMC Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S10</span><span class="refDoi"> DOI: 10.1186/1472-6807-13-S1-S10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1472-6807-13-S1-S10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=S10&issue=Suppl.+1&author=B.+G.+Godshallauthor=Y.+Tangauthor=W.+Yangauthor=B.+Y.+Chen&title=An+aggregate+analysis+of+many+predicted+structures+to+reduce+errors+in+protein+structure+comparison+caused+by+conformational+flexibility&doi=10.1186%2F1472-6807-13-S1-S10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1186%2F1472-6807-13-S1-S10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6807-13-S1-S10%26sid%3Dliteratum%253Aachs%26aulast%3DGodshall%26aufirst%3DB.%2BG.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DB.%2BY.%26atitle%3DAn%2520aggregate%2520analysis%2520of%2520many%2520predicted%2520structures%2520to%2520reduce%2520errors%2520in%2520protein%2520structure%2520comparison%2520caused%2520by%2520conformational%2520flexibility%26jtitle%3DBMC%2520Struct.%2520Biol.%26date%3D2013%26volume%3D13%26issue%3DSuppl.%25201%26spage%3DS10%26doi%3D10.1186%2F1472-6807-13-S1-S10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref185"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref185'); return false;" data-citation="" class="refNumLink">185</a></strong><div class="NLM_citation" id="cit185"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Exploiting structural information for drug-target assessment</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">331</span><span class="refDoi"> DOI: 10.4155/fmc.14.3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.4155%2Ffmc.14.3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24575967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlCqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=319-331&issue=3&author=A.+Volkamerauthor=M.+Rarey&title=Exploiting+structural+information+for+drug-target+assessment&doi=10.4155%2Ffmc.14.3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting structural information for drug-target assessment</span></div><div class="casAuthors">Volkamer, Andrea; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">319-331</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The amt. of known protein structures is continuously growing, exhibited in >95,000 3-dimensional structures freely available via the PDB.  Over the last decade, pharmaceutical research has sparked interest in computationally extg. information from this large data pool, resulting in a homol.-driven knowledge transfer from annotated to new structures.  Studying protein structures with respect to understanding and modulating their functional behavior means analyzing their centers of action.  Therefore, the detection and description of potential binding sites on the protein surface is a major step toward protein classification and assessment.  Subsequently, these representations can be incorporated to compare proteins, and to predict their druggability or function.  Esp. in the context of target identification and polypharmacol., automated tools for large-scale target comparisons are highly needed.  Here, developments for automated structure-based target assessment are reviewed and remaining challenges as well as future perspectives are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEy7ASYbcMYrVg90H21EOLACvtfcHk0libcUkuYLlQYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlCqsr4%253D&md5=f15483dd54f1ad555c4d19781b98a8de</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.3%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DExploiting%2520structural%2520information%2520for%2520drug-target%2520assessment%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26issue%3D3%26spage%3D319%26epage%3D331%26doi%3D10.4155%2Ffmc.14.3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151'],'ref152':['cit152'],'ref153':['cit153'],'ref154':['cit154'],'ref155':['cit155'],'ref156':['cit156'],'ref157':['cit157'],'ref158':['cit158'],'ref159':['cit159'],'ref160':['cit160'],'ref161':['cit161'],'ref162':['cit162'],'ref163':['cit163'],'ref164':['cit164'],'ref165':['cit165'],'ref166':['cit166'],'ref167':['cit167'],'ref168':['cit168'],'ref169':['cit169'],'ref170':['cit170'],'ref171':['cit171'],'ref172':['cit172'],'ref173':['cit173'],'ref174':['cit174'],'ref175':['cit175'],'ref176':['cit176'],'ref177':['cit177'],'ref178':['cit178'],'ref179':['cit179'],'ref180':['cit180'],'ref181':['cit181'],'ref182':['cit182'],'ref183':['cit183'],'ref184':['cit184'],'ref185':['cit185']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 58 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Merveille Eguida, <span class="NLM_string-name hlFld-ContribAuthor">Didier Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">A Computer Vision Approach to Align and Compare Protein Cavities: Application to Fragment-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (13)
                                     , 7127-7142. <a href="https://doi.org/10.1021/acs.jmedchem.0c00422" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00422</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00422%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BComputer%252BVision%252BApproach%252Bto%252BAlign%252Band%252BCompare%252BProtein%252BCavities%25253A%252BApplication%252Bto%252BFragment-Based%252BDrug%252BDesign%26aulast%3DEguida%26aufirst%3DMerveille%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12032020%26date%3D18062020%26date%3D04062020%26volume%3D63%26issue%3D13%26spage%3D7127%26epage%3D7142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martin Simonovsky, <span class="NLM_string-name hlFld-ContribAuthor">Joshua Meyers</span>. </span><span class="cited-content_cbyCitation_article-title">DeeplyTough: Learning Structural Comparison of Protein Binding Sites. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 2356-2366. <a href="https://doi.org/10.1021/acs.jcim.9b00554" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00554</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00554%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDeeplyTough%25253A%252BLearning%252BStructural%252BComparison%252Bof%252BProtein%252BBinding%252BSites%26aulast%3DSimonovsky%26aufirst%3DMartin%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D07072019%26date%3D18032020%26date%3D05022020%26volume%3D60%26issue%3D4%26spage%3D2356%26epage%3D2366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Franck Da Silva, Guillaume Bret, Leandro Teixeira, Claudio F. Gonzalez, <span class="NLM_string-name hlFld-ContribAuthor">Didier Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">Exhaustive Repertoire of Druggable Cavities at Protein–Protein Interfaces of Known Three-Dimensional Structure. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9732-9742. <a href="https://doi.org/10.1021/acs.jmedchem.9b01184" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01184</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01184%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExhaustive%252BRepertoire%252Bof%252BDruggable%252BCavities%252Bat%252BProtein%2525E2%252580%252593Protein%252BInterfaces%252Bof%252BKnown%252BThree-Dimensional%252BStructure%26aulast%3DDa%2BSilva%26aufirst%3DFranck%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D22072019%26date%3D25102019%26date%3D11102019%26volume%3D62%26issue%3D21%26spage%3D9732%26epage%3D9742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dominique Sydow, Lindsey Burggraaff, Angelika Szengel, Herman W. T. van Vlijmen, Adriaan P. IJzerman, Gerard J. P. van Westen, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Volkamer</span>. </span><span class="cited-content_cbyCitation_article-title">Advances and Challenges in Computational Target Prediction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (5)
                                     , 1728-1742. <a href="https://doi.org/10.1021/acs.jcim.8b00832" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00832</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00832%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAdvances%252Band%252BChallenges%252Bin%252BComputational%252BTarget%252BPrediction%26aulast%3DSydow%26aufirst%3DDominique%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D21112018%26date%3D28022019%26volume%3D59%26issue%3D5%26spage%3D1728%26epage%3D1742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mauro
S. Nogueira, <span class="NLM_string-name hlFld-ContribAuthor">Oliver Koch</span>. </span><span class="cited-content_cbyCitation_article-title">The Development of Target-Specific Machine Learning Models as Scoring Functions for Docking-Based Target Prediction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1238-1252. <a href="https://doi.org/10.1021/acs.jcim.8b00773" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00773</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00773%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BDevelopment%252Bof%252BTarget-Specific%252BMachine%252BLearning%252BModels%252Bas%252BScoring%252BFunctions%252Bfor%252BDocking-Based%252BTarget%252BPrediction%26aulast%3DNogueira%26aufirst%3DMauro%2BS.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D31102018%26date%3D18032019%26date%3D25022019%26volume%3D59%26issue%3D3%26spage%3D1238%26epage%3D1252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dávid  Bajusz</span>, <span class="hlFld-ContribAuthor ">Warren S.  Wade</span>, <span class="hlFld-ContribAuthor ">Grzegorz  Satała</span>, <span class="hlFld-ContribAuthor ">Andrzej J.  Bojarski</span>, <span class="hlFld-ContribAuthor ">Janez  Ilaš</span>, <span class="hlFld-ContribAuthor ">Jessica  Ebner</span>, <span class="hlFld-ContribAuthor ">Florian  Grebien</span>, <span class="hlFld-ContribAuthor ">Henrietta  Papp</span>, <span class="hlFld-ContribAuthor ">Ferenc  Jakab</span>, <span class="hlFld-ContribAuthor ">Alice  Douangamath</span>, <span class="hlFld-ContribAuthor ">Daren  Fearon</span>, <span class="hlFld-ContribAuthor ">Frank  von Delft</span>, <span class="hlFld-ContribAuthor ">Marion  Schuller</span>, <span class="hlFld-ContribAuthor ">Ivan  Ahel</span>, <span class="hlFld-ContribAuthor ">Amanda  Wakefield</span>, <span class="hlFld-ContribAuthor ">Sándor  Vajda</span>, <span class="hlFld-ContribAuthor ">János  Gerencsér</span>, <span class="hlFld-ContribAuthor ">Péter  Pallai</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring protein hotspots by optimized fragment pharmacophores. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-021-23443-y" title="DOI URL">https://doi.org/10.1038/s41467-021-23443-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-021-23443-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-021-23443-y%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DExploring%252Bprotein%252Bhotspots%252Bby%252Boptimized%252Bfragment%252Bpharmacophores%26aulast%3DBajusz%26aufirst%3DD%25C3%25A1vid%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shiliang  Li</span>, <span class="hlFld-ContribAuthor ">Chaoqian  Cai</span>, <span class="hlFld-ContribAuthor ">Jiayu  Gong</span>, <span class="hlFld-ContribAuthor ">Xiaofeng  Liu</span>, <span class="hlFld-ContribAuthor ">Honglin  Li</span>. </span><span class="cited-content_cbyCitation_article-title">A fast protein binding site comparison algorithm for proteome‐wide protein function prediction and drug repurposing. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2021,</strong> <em>44 </em><a href="https://doi.org/10.1002/prot.26176" title="DOI URL">https://doi.org/10.1002/prot.26176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.26176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.26176%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DA%252Bfast%252Bprotein%252Bbinding%252Bsite%252Bcomparison%252Balgorithm%252Bfor%252Bproteome%2525E2%252580%252590wide%252Bprotein%252Bfunction%252Bprediction%252Band%252Bdrug%252Brepurposing%26aulast%3DLi%26aufirst%3DShiliang%26date%3D2021%26date%3D2021%26volume%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Rémy  Marchand</span>, <span class="hlFld-ContribAuthor ">Bernard  Pirard</span>, <span class="hlFld-ContribAuthor ">Peter  Ertl</span>, <span class="hlFld-ContribAuthor ">Finton  Sirockin</span>. </span><span class="cited-content_cbyCitation_article-title">CAVIAR: a method for automatic cavity detection, description and decomposition into subcavities. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>35 </em>
                                    (6)
                                     , 737-750. <a href="https://doi.org/10.1007/s10822-021-00390-w" title="DOI URL">https://doi.org/10.1007/s10822-021-00390-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-021-00390-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-021-00390-w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DCAVIAR%25253A%252Ba%252Bmethod%252Bfor%252Bautomatic%252Bcavity%252Bdetection%25252C%252Bdescription%252Band%252Bdecomposition%252Binto%252Bsubcavities%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2021%26date%3D2021%26volume%3D35%26issue%3D6%26spage%3D737%26epage%3D750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natesh  Singh</span>, <span class="hlFld-ContribAuthor ">Ludovic  Chaput</span>, <span class="hlFld-ContribAuthor ">Bruno O  Villoutreix</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual screening web servers: designing chemical probes and drug candidates in the cyberspace. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2021,</strong> <em>22 </em>
                                    (2)
                                     , 1790-1818. <a href="https://doi.org/10.1093/bib/bbaa034" title="DOI URL">https://doi.org/10.1093/bib/bbaa034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbaa034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbaa034%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DVirtual%252Bscreening%252Bweb%252Bservers%25253A%252Bdesigning%252Bchemical%252Bprobes%252Band%252Bdrug%252Bcandidates%252Bin%252Bthe%252Bcyberspace%26aulast%3DSingh%26aufirst%3DNatesh%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D2%26spage%3D1790%26epage%3D1818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling Protein-Ligand Interactions: Are We Ready for Deep Learning?. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 163-173. <a href="https://doi.org/10.1016/B978-0-12-801238-3.11521-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-801238-3.11521-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801238-3.11521-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801238-3.11521-1%26sid%3Dliteratum%253Aachs%26atitle%3DModeling%252BProtein-Ligand%252BInteractions%25253A%252BAre%252BWe%252BReady%252Bfor%252BDeep%252BLearning%25253F%26aulast%3DRognan%26aufirst%3DDidier%26date%3D2021%26spage%3D163%26epage%3D173%26pub%3DElsevier%26atitle%3DSystems%252BMedicine%26date%3D2021%26volume%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joyce Villa Verde Bastos  Borba</span>, <span class="hlFld-ContribAuthor ">Arthur Carvalho  Silva</span>, <span class="hlFld-ContribAuthor ">Marilia Nunes Nascimento  Lima</span>, <span class="hlFld-ContribAuthor ">Sabrina Silva  Mendonca</span>, <span class="hlFld-ContribAuthor ">Nicholas  Furnham</span>, <span class="hlFld-ContribAuthor ">Fabio Trindade Maranhão  Costa</span>, <span class="hlFld-ContribAuthor ">Carolina Horta  Andrade</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomics and bioinformatics approaches for prioritizing kinases as drug targets for neglected tropical diseases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 187-223. <a href="https://doi.org/10.1016/bs.apcsb.2020.10.006" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.10.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.10.006%26sid%3Dliteratum%253Aachs%26atitle%3DChemogenomics%252Band%252Bbioinformatics%252Bapproaches%252Bfor%252Bprioritizing%252Bkinases%252Bas%252Bdrug%252Btargets%252Bfor%252Bneglected%252Btropical%252Bdiseases%26aulast%3DBorba%26aufirst%3DJoyce%2BVilla%2BVerde%2BBastos%26date%3D2021%26spage%3D187%26epage%3D223%26pub%3DElsevier%26atitle%3DProtein%252BKinases%252Bin%252BDrug%252BDiscovery%26date%3D2021%26volume%3D124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shun  Zhang</span>, <span class="hlFld-ContribAuthor ">Yan  Zhao</span>, <span class="hlFld-ContribAuthor ">Shuyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Min  Li</span>, <span class="hlFld-ContribAuthor ">Yue  Xu</span>, <span class="hlFld-ContribAuthor ">Jianhua  Ran</span>, <span class="hlFld-ContribAuthor ">Xiaoqiang  Geng</span>, <span class="hlFld-ContribAuthor ">Jinzhao  He</span>, <span class="hlFld-ContribAuthor ">Jia  Meng</span>, <span class="hlFld-ContribAuthor ">Guangying  Shao</span>, <span class="hlFld-ContribAuthor ">Hong  Zhou</span>, <span class="hlFld-ContribAuthor ">Zemei  Ge</span>, <span class="hlFld-ContribAuthor ">Guangping  Chen</span>, <span class="hlFld-ContribAuthor ">Runtao  Li</span>, <span class="hlFld-ContribAuthor ">Baoxue  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel diarylamides as orally active diuretics targeting urea transporters. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     , 181-202. <a href="https://doi.org/10.1016/j.apsb.2020.06.001" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.06.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.06.001%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Bnovel%252Bdiarylamides%252Bas%252Borally%252Bactive%252Bdiuretics%252Btargeting%252Burea%252Btransporters%26aulast%3DZhang%26aufirst%3DShun%26date%3D2021%26volume%3D11%26issue%3D1%26spage%3D181%26epage%3D202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elyas  Mohammadi</span>, <span class="hlFld-ContribAuthor ">Rui  Benfeitas</span>, <span class="hlFld-ContribAuthor ">Hasan  Turkez</span>, <span class="hlFld-ContribAuthor ">Jan  Boren</span>, <span class="hlFld-ContribAuthor ">Jens  Nielsen</span>, <span class="hlFld-ContribAuthor ">Mathias  Uhlen</span>, <span class="hlFld-ContribAuthor ">Adil  Mardinoglu</span>. </span><span class="cited-content_cbyCitation_article-title">Applications of Genome-Wide Screening and Systems Biology Approaches in Drug Repositioning. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (9)
                                     , 2694. <a href="https://doi.org/10.3390/cancers12092694" title="DOI URL">https://doi.org/10.3390/cancers12092694</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12092694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12092694%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DApplications%252Bof%252BGenome-Wide%252BScreening%252Band%252BSystems%252BBiology%252BApproaches%252Bin%252BDrug%252BRepositioning%26aulast%3DMohammadi%26aufirst%3DElyas%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D9%26spage%3D2694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paolo  Gelosa</span>, <span class="hlFld-ContribAuthor ">Laura  Castiglioni</span>, <span class="hlFld-ContribAuthor ">Marina  Camera</span>, <span class="hlFld-ContribAuthor ">Luigi  Sironi</span>. </span><span class="cited-content_cbyCitation_article-title">Drug repurposing in cardiovascular diseases: Opportunity or hopeless dream?. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2020,</strong> <em>177 </em>, 113894. <a href="https://doi.org/10.1016/j.bcp.2020.113894" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.113894</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.113894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.113894%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DDrug%252Brepurposing%252Bin%252Bcardiovascular%252Bdiseases%25253A%252BOpportunity%252Bor%252Bhopeless%252Bdream%25253F%26aulast%3DGelosa%26aufirst%3DPaolo%26date%3D2020%26volume%3D177%26spage%3D113894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dehua  Chen</span>, <span class="hlFld-ContribAuthor ">Amir  Jalilifard</span>, <span class="hlFld-ContribAuthor ">Adriano  Veloso</span>, <span class="hlFld-ContribAuthor ">Nivio  Ziviani</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling Pharmacological Effects with Multi-Relation Unsupervised Graph Embedding. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 1-7. <a href="https://doi.org/10.1109/IJCNN48605.2020.9206668" title="DOI URL">https://doi.org/10.1109/IJCNN48605.2020.9206668</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1109/IJCNN48605.2020.9206668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1109%2FIJCNN48605.2020.9206668%26sid%3Dliteratum%253Aachs%26atitle%3DModeling%252BPharmacological%252BEffects%252Bwith%252BMulti-Relation%252BUnsupervised%252BGraph%252BEmbedding%26aulast%3DChen%26aufirst%3DDehua%26date%3D2020%26spage%3D1%26epage%3D7%26pub%3DIEEE%26atitle%3D2020%252BInternational%252BJoint%252BConference%252Bon%252BNeural%252BNetworks%252B%252528IJCNN%252529%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carolina L.  Bellera</span>, <span class="hlFld-ContribAuthor ">Lucas N.  Alberca</span>, <span class="hlFld-ContribAuthor ">María L.  Sbaraglini</span>, <span class="hlFld-ContribAuthor ">Alan  Talevi</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Drug Repositioning for Chagas Disease. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (5)
                                     , 662-675. <a href="https://doi.org/10.2174/0929867326666191016114839" title="DOI URL">https://doi.org/10.2174/0929867326666191016114839</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867326666191016114839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867326666191016114839%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DIn%252BSilico%252BDrug%252BRepositioning%252Bfor%252BChagas%252BDisease%26aulast%3DBellera%26aufirst%3DCarolina%2BL.%26date%3D2020%26volume%3D27%26issue%3D5%26spage%3D662%26epage%3D675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lorraine  Yamurai Bishi</span>, <span class="hlFld-ContribAuthor ">Sundeep  Chaitanya Vedithi</span>, <span class="hlFld-ContribAuthor ">Tom  L. Blundell</span>, <span class="hlFld-ContribAuthor ">Grace  Chitima Mugumbate</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Deorphaning of
              Mycobacterium tuberculosis
              Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.5772/intechopen.82374" title="DOI URL">https://doi.org/10.5772/intechopen.82374</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5772/intechopen.82374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5772%2Fintechopen.82374%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BDeorphaning%252Bof%252BMycobacterium%252Btuberculosis%252BTargets%26aulast%3DYamurai%2BBishi%26aufirst%3DLorraine%26date%3D2020%26date%3D2020%26pub%3DIntechOpen%26atitle%3DDrug%252BDiscovery%252Band%252BDevelopment%252B-%252BNew%252BAdvances%26aulast%3DGaitonde%26aufirst%3DVishwanath%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renzo  Angles</span>, <span class="hlFld-ContribAuthor ">Mauricio  Arenas-Salinas</span>, <span class="hlFld-ContribAuthor ">Roberto  García</span>, <span class="hlFld-ContribAuthor ">Jose Antonio  Reyes-Suarez</span>, <span class="hlFld-ContribAuthor ">Ehmke  Pohl</span>. </span><span class="cited-content_cbyCitation_article-title">GSP4PDB: a web tool to visualize, search and explore protein-ligand structural patterns. </span><span class="cited-content_cbyCitation_journal-name">BMC Bioinformatics</span><span> <strong>2020,</strong> <em>21 </em>
                                    (S2)
                                     <a href="https://doi.org/10.1186/s12859-020-3352-x" title="DOI URL">https://doi.org/10.1186/s12859-020-3352-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12859-020-3352-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12859-020-3352-x%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Bioinformatics%26atitle%3DGSP4PDB%25253A%252Ba%252Bweb%252Btool%252Bto%252Bvisualize%25252C%252Bsearch%252Band%252Bexplore%252Bprotein-ligand%252Bstructural%252Bpatterns%26aulast%3DAngles%26aufirst%3DRenzo%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3DS2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucas N.  Alberca</span>, <span class="hlFld-ContribAuthor ">Alan  Talevi</span>. </span><span class="cited-content_cbyCitation_article-title">The Efficiency of Multi-target Drugs: A Network Approach. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 63-75. <a href="https://doi.org/10.1007/978-3-030-32857-3_3" title="DOI URL">https://doi.org/10.1007/978-3-030-32857-3_3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-32857-3_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-32857-3_3%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BEfficiency%252Bof%252BMulti-target%252BDrugs%25253A%252BA%252BNetwork%252BApproach%26aulast%3DAlberca%26aufirst%3DLucas%2BN.%26date%3D2020%26date%3D2020%26spage%3D63%26epage%3D75%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DApproaching%252BComplex%252BDiseases%26aulast%3DBizzarri%26aufirst%3DMariano%26date%3D2020%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alan  Talevi</span>, <span class="hlFld-ContribAuthor ">Carolina  Carrillo</span>, <span class="hlFld-ContribAuthor ">Marcelo  Comini</span>. </span><span class="cited-content_cbyCitation_article-title">The Thiol-polyamine Metabolism of Trypanosoma cruzi: Molecular Targets and Drug Repurposing Strategies. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2019,</strong> <em>26 </em>
                                    (36)
                                     , 6614-6635. <a href="https://doi.org/10.2174/0929867325666180926151059" title="DOI URL">https://doi.org/10.2174/0929867325666180926151059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666180926151059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666180926151059%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DThe%252BThiol-polyamine%252BMetabolism%252Bof%252BTrypanosoma%252Bcruzi%25253A%252BMolecular%252BTargets%252Band%252BDrug%252BRepurposing%252BStrategies%26aulast%3DTalevi%26aufirst%3DAlan%26date%3D2019%26volume%3D26%26issue%3D36%26spage%3D6614%26epage%3D6635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dominique  Sydow</span>, <span class="hlFld-ContribAuthor ">Andrea  Morger</span>, <span class="hlFld-ContribAuthor ">Maximilian  Driller</span>, <span class="hlFld-ContribAuthor ">Andrea  Volkamer</span>. </span><span class="cited-content_cbyCitation_article-title">TeachOpenCADD: a teaching platform for computer-aided drug design using open source packages and data. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-019-0351-x" title="DOI URL">https://doi.org/10.1186/s13321-019-0351-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-019-0351-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-019-0351-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DTeachOpenCADD%25253A%252Ba%252Bteaching%252Bplatform%252Bfor%252Bcomputer-aided%252Bdrug%252Bdesign%252Busing%252Bopen%252Bsource%252Bpackages%252Band%252Bdata%26aulast%3DSydow%26aufirst%3DDominique%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Misagh  Naderi</span>, <span class="hlFld-ContribAuthor ">Jeffrey Mitchell  Lemoine</span>, <span class="hlFld-ContribAuthor ">Rajiv Gandhi  Govindaraj</span>, <span class="hlFld-ContribAuthor ">Omar Zade  Kana</span>, <span class="hlFld-ContribAuthor ">Wei Pan  Feinstein</span>, <span class="hlFld-ContribAuthor ">Michal  Brylinski</span>. </span><span class="cited-content_cbyCitation_article-title">Binding site matching in rational drug design: algorithms and applications. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2019,</strong> <em>20 </em>
                                    (6)
                                     , 2167-2184. <a href="https://doi.org/10.1093/bib/bby078" title="DOI URL">https://doi.org/10.1093/bib/bby078</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bby078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbby078%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DBinding%252Bsite%252Bmatching%252Bin%252Brational%252Bdrug%252Bdesign%25253A%252Balgorithms%252Band%252Bapplications%26aulast%3DNaderi%26aufirst%3DMisagh%26date%3D2019%26date%3D2018%26volume%3D20%26issue%3D6%26spage%3D2167%26epage%3D2184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carolina Horta  Andrade</span>, <span class="hlFld-ContribAuthor ">Bruno Junior  Neves</span>, <span class="hlFld-ContribAuthor ">Cleber Camilo  Melo-Filho</span>, <span class="hlFld-ContribAuthor ">Juliana  Rodrigues</span>, <span class="hlFld-ContribAuthor ">Diego Cabral  Silva</span>, <span class="hlFld-ContribAuthor ">Rodolpho Campos  Braga</span>, <span class="hlFld-ContribAuthor ">Pedro Vitor Lemos  Cravo</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Chemogenomics Drug Repositioning Strategies for Neglected Tropical Diseases. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2019,</strong> <em>26 </em>
                                    (23)
                                     , 4355-4379. <a href="https://doi.org/10.2174/0929867325666180309114824" title="DOI URL">https://doi.org/10.2174/0929867325666180309114824</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666180309114824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666180309114824%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DIn%252BSilico%252BChemogenomics%252BDrug%252BRepositioning%252BStrategies%252Bfor%252BNeglected%252BTropical%252BDiseases%26aulast%3DAndrade%26aufirst%3DCarolina%2BHorta%26date%3D2019%26volume%3D26%26issue%3D23%26spage%3D4355%26epage%3D4379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guodong  Yang</span>, <span class="hlFld-ContribAuthor ">Aiqun  Ma</span>, <span class="hlFld-ContribAuthor ">Zhaohui S.  Qin</span>. </span><span class="cited-content_cbyCitation_article-title">An Integrated System Biology Approach Yields Drug Repositioning Candidates for the Treatment of Heart Failure. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2019,</strong> <em>10 </em><a href="https://doi.org/10.3389/fgene.2019.00916" title="DOI URL">https://doi.org/10.3389/fgene.2019.00916</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2019.00916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2019.00916%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DAn%252BIntegrated%252BSystem%252BBiology%252BApproach%252BYields%252BDrug%252BRepositioning%252BCandidates%252Bfor%252Bthe%252BTreatment%252Bof%252BHeart%252BFailure%26aulast%3DYang%26aufirst%3DGuodong%26date%3D2019%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manish D.  Paranjpe</span>, <span class="hlFld-ContribAuthor ">Alice  Taubes</span>, <span class="hlFld-ContribAuthor ">Marina  Sirota</span>. </span><span class="cited-content_cbyCitation_article-title">Insights into Computational Drug Repurposing for Neurodegenerative Disease. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2019,</strong> <em>40 </em>
                                    (8)
                                     , 565-576. <a href="https://doi.org/10.1016/j.tips.2019.06.003" title="DOI URL">https://doi.org/10.1016/j.tips.2019.06.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2019.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2019.06.003%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DInsights%252Binto%252BComputational%252BDrug%252BRepurposing%252Bfor%252BNeurodegenerative%252BDisease%26aulast%3DParanjpe%26aufirst%3DManish%2BD.%26date%3D2019%26volume%3D40%26issue%3D8%26spage%3D565%26epage%3D576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christiane  Ehrt</span>, <span class="hlFld-ContribAuthor ">Tobias  Brinkjost</span>, <span class="hlFld-ContribAuthor ">Oliver  Koch</span>. </span><span class="cited-content_cbyCitation_article-title">Binding site characterization – similarity, promiscuity, and druggability. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (7)
                                     , 1145-1159. <a href="https://doi.org/10.1039/C9MD00102F" title="DOI URL">https://doi.org/10.1039/C9MD00102F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00102F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00102F%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DBinding%252Bsite%252Bcharacterization%252B%2525E2%252580%252593%252Bsimilarity%25252C%252Bpromiscuity%25252C%252Band%252Bdruggability%26aulast%3DEhrt%26aufirst%3DChristiane%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D7%26spage%3D1145%26epage%3D1159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alejandro  Valdés-Jiménez</span>, <span class="hlFld-ContribAuthor ">Josep-L.  Larriba-Pey</span>, <span class="hlFld-ContribAuthor ">Gabriel  Núñez-Vivanco</span>, <span class="hlFld-ContribAuthor ">Miguel  Reyes-Parada</span>. </span><span class="cited-content_cbyCitation_article-title">3D-PP: A Tool for Discovering Conserved Three-Dimensional Protein Patterns. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (13)
                                     , 3174. <a href="https://doi.org/10.3390/ijms20133174" title="DOI URL">https://doi.org/10.3390/ijms20133174</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20133174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20133174%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3D3D-PP%25253A%252BA%252BTool%252Bfor%252BDiscovering%252BConserved%252BThree-Dimensional%252BProtein%252BPatterns%26aulast%3DVald%25C3%25A9s-Jim%25C3%25A9nez%26aufirst%3DAlejandro%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D13%26spage%3D3174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Célien  Jacquemard</span>, <span class="hlFld-ContribAuthor ">Esther  Kellenberger</span>. </span><span class="cited-content_cbyCitation_article-title">A bright future for fragment-based drug discovery: what does it hold?. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (5)
                                     , 413-416. <a href="https://doi.org/10.1080/17460441.2019.1583643" title="DOI URL">https://doi.org/10.1080/17460441.2019.1583643</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1583643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1583643%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DA%252Bbright%252Bfuture%252Bfor%252Bfragment-based%252Bdrug%252Bdiscovery%25253A%252Bwhat%252Bdoes%252Bit%252Bhold%25253F%26aulast%3DJacquemard%26aufirst%3DC%25C3%25A9lien%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D5%26spage%3D413%26epage%3D416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Janez  Konc</span>. </span><span class="cited-content_cbyCitation_article-title">Binding site comparisons for target-centered drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (5)
                                     , 445-454. <a href="https://doi.org/10.1080/17460441.2019.1588883" title="DOI URL">https://doi.org/10.1080/17460441.2019.1588883</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1588883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1588883%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DBinding%252Bsite%252Bcomparisons%252Bfor%252Btarget-centered%252Bdrug%252Bdiscovery%26aulast%3DKonc%26aufirst%3DJanez%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D5%26spage%3D445%26epage%3D454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Limeng  Pu</span>, <span class="hlFld-ContribAuthor ">Rajiv Gandhi  Govindaraj</span>, <span class="hlFld-ContribAuthor ">Jeffrey Mitchell  Lemoine</span>, <span class="hlFld-ContribAuthor ">Hsiao-Chun  Wu</span>, <span class="hlFld-ContribAuthor ">Michal  Brylinski</span>, . </span><span class="cited-content_cbyCitation_article-title">DeepDrug3D: Classification of ligand-binding pockets in proteins with a convolutional neural network. </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2019,</strong> <em>15 </em>
                                    (2)
                                     , e1006718. <a href="https://doi.org/10.1371/journal.pcbi.1006718" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1006718</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1006718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1006718%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DDeepDrug3D%25253A%252BClassification%252Bof%252Bligand-binding%252Bpockets%252Bin%252Bproteins%252Bwith%252Ba%252Bconvolutional%252Bneural%252Bnetwork%26aulast%3DPu%26aufirst%3DLimeng%26date%3D2019%26date%3D2019%26volume%3D15%26issue%3D2%26spage%3De1006718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elizabeth  Sam</span>, <span class="hlFld-ContribAuthor ">Prashanth  Athri</span>. </span><span class="cited-content_cbyCitation_article-title">Web-based drug repurposing tools: a survey. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2019,</strong> <em>20 </em>
                                    (1)
                                     , 299-316. <a href="https://doi.org/10.1093/bib/bbx125" title="DOI URL">https://doi.org/10.1093/bib/bbx125</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbx125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbx125%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DWeb-based%252Bdrug%252Brepurposing%252Btools%25253A%252Ba%252Bsurvey%26aulast%3DSam%26aufirst%3DElizabeth%26date%3D2019%26date%3D2017%26volume%3D20%26issue%3D1%26spage%3D299%26epage%3D316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oliver  Koch</span>, <span class="hlFld-ContribAuthor ">Christiane  Ehrt</span>, <span class="hlFld-ContribAuthor ">Tobias  Brinkjost</span>. </span><span class="cited-content_cbyCitation_article-title">Binding Site Comparison – Software and Applications. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 650-660. <a href="https://doi.org/10.1016/B978-0-12-809633-8.20196-9" title="DOI URL">https://doi.org/10.1016/B978-0-12-809633-8.20196-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-809633-8.20196-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-809633-8.20196-9%26sid%3Dliteratum%253Aachs%26atitle%3DBinding%252BSite%252BComparison%252B%2525E2%252580%252593%252BSoftware%252Band%252BApplications%26aulast%3DKoch%26aufirst%3DOliver%26date%3D2019%26spage%3D650%26epage%3D660%26pub%3DElsevier%26atitle%3DEncyclopedia%252Bof%252BBioinformatics%252Band%252BComputational%252BBiology%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bashir Akhlaq  Akhoon</span>, <span class="hlFld-ContribAuthor ">Harshita  Tiwari</span>, <span class="hlFld-ContribAuthor ">Amit  Nargotra</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Drug Design Methods for Drug Repurposing. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 47-84. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00003-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00003-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00003-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00003-1%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BDrug%252BDesign%252BMethods%252Bfor%252BDrug%252BRepurposing%26aulast%3DAkhoon%26aufirst%3DBashir%2BAkhlaq%26date%3D2019%26spage%3D47%26epage%3D84%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pawan  Kumar</span>, <span class="hlFld-ContribAuthor ">Indira  Ghosh</span>. </span><span class="cited-content_cbyCitation_article-title">Application of In Silico Drug Repurposing in Infectious Diseases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 427-462. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00015-8" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00015-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00015-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00015-8%26sid%3Dliteratum%253Aachs%26atitle%3DApplication%252Bof%252BIn%252BSilico%252BDrug%252BRepurposing%252Bin%252BInfectious%252BDiseases%26aulast%3DKumar%26aufirst%3DPawan%26date%3D2019%26spage%3D427%26epage%3D462%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anu R.  Melge</span>, <span class="hlFld-ContribAuthor ">K.  Manzoor</span>, <span class="hlFld-ContribAuthor ">Shantikumar V.  Nair</span>, <span class="hlFld-ContribAuthor ">C. Gopi  Mohan</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Modeling of FDA-Approved Drugs for Discovery of Anti-Cancer Agents: A Drug-Repurposing Approach. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 577-608. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00019-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00019-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00019-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00019-5%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BModeling%252Bof%252BFDA-Approved%252BDrugs%252Bfor%252BDiscovery%252Bof%252BAnti-Cancer%252BAgents%25253A%252BA%252BDrug-Repurposing%252BApproach%26aulast%3DMelge%26aufirst%3DAnu%2BR.%26date%3D2019%26spage%3D577%26epage%3D608%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019%26volume%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jameel  Iqbal</span>, <span class="hlFld-ContribAuthor ">Tony  Yuen</span>, <span class="hlFld-ContribAuthor ">Neeha  Zaidi</span>, <span class="hlFld-ContribAuthor ">Se-Min  Kim</span>, <span class="hlFld-ContribAuthor ">Samir  Zaidi</span>, <span class="hlFld-ContribAuthor ">Alberta  Zallone</span>, <span class="hlFld-ContribAuthor ">Mone  Zaidi</span>, <span class="hlFld-ContribAuthor ">Li  Sun</span>, <span class="hlFld-ContribAuthor ">Shozeb  Haider</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Repurposing by Connectivity Mapping and Structural Modeling. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 609-623. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00020-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00020-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00020-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00020-1%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BRepurposing%252Bby%252BConnectivity%252BMapping%252Band%252BStructural%252BModeling%26aulast%3DIqbal%26aufirst%3DJameel%26date%3D2019%26spage%3D609%26epage%3D623%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carolina L.  Belllera</span>, <span class="hlFld-ContribAuthor ">María L.  Sbaraglini</span>, <span class="hlFld-ContribAuthor ">Lucas N.  Alberca</span>, <span class="hlFld-ContribAuthor ">Juan I.  Alice</span>, <span class="hlFld-ContribAuthor ">Alan  Talevi</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Modeling of FDA-Approved Drugs for Discovery of Therapies Against Neglected Diseases: A Drug Repurposing Approach. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 625-648. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00021-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00021-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00021-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00021-3%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BModeling%252Bof%252BFDA-Approved%252BDrugs%252Bfor%252BDiscovery%252Bof%252BTherapies%252BAgainst%252BNeglected%252BDiseases%25253A%252BA%252BDrug%252BRepurposing%252BApproach%26aulast%3DBelllera%26aufirst%3DCarolina%2BL.%26date%3D2019%26spage%3D625%26epage%3D648%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019%26volume%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajiv Gandhi  Govindaraj</span>, <span class="hlFld-ContribAuthor ">Misagh  Naderi</span>, <span class="hlFld-ContribAuthor ">Manali  Singha</span>, <span class="hlFld-ContribAuthor ">Jeffrey  Lemoine</span>, <span class="hlFld-ContribAuthor ">Michal  Brylinski</span>. </span><span class="cited-content_cbyCitation_article-title">Large-scale computational drug repositioning to find treatments for rare diseases. </span><span class="cited-content_cbyCitation_journal-name">npj Systems Biology and Applications</span><span> <strong>2018,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41540-018-0050-7" title="DOI URL">https://doi.org/10.1038/s41540-018-0050-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41540-018-0050-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41540-018-0050-7%26sid%3Dliteratum%253Aachs%26jtitle%3Dnpj%2520Systems%2520Biology%2520and%2520Applications%26atitle%3DLarge-scale%252Bcomputational%252Bdrug%252Brepositioning%252Bto%252Bfind%252Btreatments%252Bfor%252Brare%252Bdiseases%26aulast%3DGovindaraj%26aufirst%3DRajiv%2BGandhi%26date%3D2018%26date%3D2018%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajiv Gandhi  Govindaraj</span>, <span class="hlFld-ContribAuthor ">Michal  Brylinski</span>. </span><span class="cited-content_cbyCitation_article-title">Comparative assessment of strategies to identify similar ligand-binding pockets in proteins. </span><span class="cited-content_cbyCitation_journal-name">BMC Bioinformatics</span><span> <strong>2018,</strong> <em>19 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12859-018-2109-2" title="DOI URL">https://doi.org/10.1186/s12859-018-2109-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12859-018-2109-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12859-018-2109-2%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Bioinformatics%26atitle%3DComparative%252Bassessment%252Bof%252Bstrategies%252Bto%252Bidentify%252Bsimilar%252Bligand-binding%252Bpockets%252Bin%252Bproteins%26aulast%3DGovindaraj%26aufirst%3DRajiv%2BGandhi%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christiane  Ehrt</span>, <span class="hlFld-ContribAuthor ">Tobias  Brinkjost</span>, <span class="hlFld-ContribAuthor ">Oliver  Koch</span>, . </span><span class="cited-content_cbyCitation_article-title">A benchmark driven guide to binding site comparison: An exhaustive evaluation using tailor-made data sets (ProSPECCTs). </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2018,</strong> <em>14 </em>
                                    (11)
                                     , e1006483. <a href="https://doi.org/10.1371/journal.pcbi.1006483" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1006483</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1006483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1006483%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DA%252Bbenchmark%252Bdriven%252Bguide%252Bto%252Bbinding%252Bsite%252Bcomparison%25253A%252BAn%252Bexhaustive%252Bevaluation%252Busing%252Btailor-made%252Bdata%252Bsets%252B%252528ProSPECCTs%252529%26aulast%3DEhrt%26aufirst%3DChristiane%26date%3D2018%26date%3D2018%26volume%3D14%26issue%3D11%26spage%3De1006483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Pinzi</span>, <span class="hlFld-ContribAuthor ">Fabiana  Caporuscio</span>, <span class="hlFld-ContribAuthor ">Giulio  Rastelli</span>. </span><span class="cited-content_cbyCitation_article-title">Selection of protein conformations for structure-based polypharmacology studies. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2018,</strong> <em>23 </em>
                                    (11)
                                     , 1889-1896. <a href="https://doi.org/10.1016/j.drudis.2018.08.007" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.08.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.08.007%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DSelection%252Bof%252Bprotein%252Bconformations%252Bfor%252Bstructure-based%252Bpolypharmacology%252Bstudies%26aulast%3DPinzi%26aufirst%3DLuca%26date%3D2018%26volume%3D23%26issue%3D11%26spage%3D1889%26epage%3D1896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Birgit J.  Waldner</span>, <span class="hlFld-ContribAuthor ">Johannes  Kraml</span>, <span class="hlFld-ContribAuthor ">Ursula  Kahler</span>, <span class="hlFld-ContribAuthor ">Alexander  Spinn</span>, <span class="hlFld-ContribAuthor ">Michael  Schauperl</span>, <span class="hlFld-ContribAuthor ">Maren  Podewitz</span>, <span class="hlFld-ContribAuthor ">Julian E.  Fuchs</span>, <span class="hlFld-ContribAuthor ">Gabriele  Cruciani</span>, <span class="hlFld-ContribAuthor ">Klaus R.  Liedl</span>. </span><span class="cited-content_cbyCitation_article-title">Electrostatic recognition in substrate binding to serine proteases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Recognition</span><span> <strong>2018,</strong> <em>31 </em>
                                    (10)
                                     , e2727. <a href="https://doi.org/10.1002/jmr.2727" title="DOI URL">https://doi.org/10.1002/jmr.2727</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jmr.2727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjmr.2727%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Recognition%26atitle%3DElectrostatic%252Brecognition%252Bin%252Bsubstrate%252Bbinding%252Bto%252Bserine%252Bproteases%26aulast%3DWaldner%26aufirst%3DBirgit%2BJ.%26date%3D2018%26date%3D2018%26volume%3D31%26issue%3D10%26spage%3De2727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moira M  Rachman</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>, <span class="hlFld-ContribAuthor ">Roderick E  Hubbard</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting how drug molecules bind to their protein targets. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Pharmacology</span><span> <strong>2018,</strong> <em>42 </em>, 34-39. <a href="https://doi.org/10.1016/j.coph.2018.07.001" title="DOI URL">https://doi.org/10.1016/j.coph.2018.07.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coph.2018.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coph.2018.07.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Pharmacology%26atitle%3DPredicting%252Bhow%252Bdrug%252Bmolecules%252Bbind%252Bto%252Btheir%252Bprotein%252Btargets%26aulast%3DRachman%26aufirst%3DMoira%2BM%26date%3D2018%26volume%3D42%26spage%3D34%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea  Volkamer</span>, <span class="hlFld-ContribAuthor ">Mathias M.  von Behren</span>, <span class="hlFld-ContribAuthor ">Stefan  Bietz</span>, <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction, Analysis, and Comparison of Active Sites. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 283-311. <a href="https://doi.org/10.1002/9783527806539.ch6g" title="DOI URL">https://doi.org/10.1002/9783527806539.ch6g</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527806539.ch6g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527806539.ch6g%26sid%3Dliteratum%253Aachs%26atitle%3DPrediction%25252C%252BAnalysis%25252C%252Band%252BComparison%252Bof%252BActive%252BSites%26aulast%3DVolkamer%26aufirst%3DAndrea%26date%3D2018%26date%3D2018%26spage%3D283%26epage%3D311%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DApplied%252BChemoinformatics%26aulast%3DEngel%26aufirst%3DThomas%26date%3D2018%26volume%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Franck  Da Silva</span>, <span class="hlFld-ContribAuthor ">Jeremy  Desaphy</span>, <span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">IChem: A Versatile Toolkit for Detecting, Comparing, and Predicting Protein-Ligand Interactions. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (6)
                                     , 507-510. <a href="https://doi.org/10.1002/cmdc.201700505" title="DOI URL">https://doi.org/10.1002/cmdc.201700505</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700505%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DIChem%25253A%252BA%252BVersatile%252BToolkit%252Bfor%252BDetecting%25252C%252BComparing%25252C%252Band%252BPredicting%252BProtein-Ligand%252BInteractions%26aulast%3DDa%25E2%2580%2585Silva%26aufirst%3DFranck%26date%3D2018%26date%3D2017%26volume%3D13%26issue%3D6%26spage%3D507%26epage%3D510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alan  Talevi</span>. </span><span class="cited-content_cbyCitation_article-title">Drug repositioning: current approaches and their implications in the precision medicine era. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Precision Medicine and Drug Development</span><span> <strong>2018,</strong> <em>3 </em>
                                    (1)
                                     , 49-61. <a href="https://doi.org/10.1080/23808993.2018.1424535" title="DOI URL">https://doi.org/10.1080/23808993.2018.1424535</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/23808993.2018.1424535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F23808993.2018.1424535%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Precision%2520Medicine%2520and%2520Drug%2520Development%26atitle%3DDrug%252Brepositioning%25253A%252Bcurrent%252Bapproaches%252Band%252Btheir%252Bimplications%252Bin%252Bthe%252Bprecision%252Bmedicine%252Bera%26aulast%3DTalevi%26aufirst%3DAlan%26date%3D2018%26date%3D2018%26volume%3D3%26issue%3D1%26spage%3D49%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronika  Temml</span>, <span class="hlFld-ContribAuthor ">Daniela  Schuster</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Studies on Natural Products for the Development of Multi-target Drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 187-201. <a href="https://doi.org/10.1007/7653_2018_18" title="DOI URL">https://doi.org/10.1007/7653_2018_18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7653_2018_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7653_2018_18%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BStudies%252Bon%252BNatural%252BProducts%252Bfor%252Bthe%252BDevelopment%252Bof%252BMulti-target%252BDrugs%26aulast%3DTemml%26aufirst%3DVeronika%26date%3D2018%26date%3D2018%26spage%3D187%26epage%3D201%26pub%3DSpringer%2520New%2520York%26atitle%3DMulti-Target%252BDrug%252BDesign%252BUsing%252BChem-Bioinformatic%252BApproaches%26aulast%3DRoy%26aufirst%3DKunal%26date%3D2019%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Géraldine  Caumes</span>, <span class="hlFld-ContribAuthor ">Alexandre  Borrel</span>, <span class="hlFld-ContribAuthor ">Hiba  Abi Hussein</span>, <span class="hlFld-ContribAuthor ">Anne-Claude  Camproux</span>, <span class="hlFld-ContribAuthor ">Leslie  Regad</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the Importance of the Pocket-estimation Method in Pocket-based Approaches: An Illustration Using Pocket-ligand Classification. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2017,</strong> <em>36 </em>
                                    (9)
                                     , 1700025. <a href="https://doi.org/10.1002/minf.201700025" title="DOI URL">https://doi.org/10.1002/minf.201700025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201700025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201700025%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DInvestigating%252Bthe%252BImportance%252Bof%252Bthe%252BPocket-estimation%252BMethod%252Bin%252BPocket-based%252BApproaches%25253A%252BAn%252BIllustration%252BUsing%252BPocket-ligand%252BClassification%26aulast%3DCaumes%26aufirst%3DG%25C3%25A9raldine%26date%3D2017%26date%3D2017%26volume%3D36%26issue%3D9%26spage%3D1700025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">The impact of in silico screening in the discovery of novel and safer drug candidates. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2017,</strong> <em>175 </em>, 47-66. <a href="https://doi.org/10.1016/j.pharmthera.2017.02.034" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2017.02.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2017.02.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2017.02.034%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DThe%252Bimpact%252Bof%252Bin%252Bsilico%252Bscreening%252Bin%252Bthe%252Bdiscovery%252Bof%252Bnovel%252Band%252Bsafer%252Bdrug%252Bcandidates%26aulast%3DRognan%26aufirst%3DDidier%26date%3D2017%26volume%3D175%26spage%3D47%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  March-Vila</span>, <span class="hlFld-ContribAuthor ">Luca  Pinzi</span>, <span class="hlFld-ContribAuthor ">Noé  Sturm</span>, <span class="hlFld-ContribAuthor ">Annachiara  Tinivella</span>, <span class="hlFld-ContribAuthor ">Ola  Engkvist</span>, <span class="hlFld-ContribAuthor ">Hongming  Chen</span>, <span class="hlFld-ContribAuthor ">Giulio  Rastelli</span>. </span><span class="cited-content_cbyCitation_article-title">On the Integration of In Silico Drug Design Methods for Drug Repurposing. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2017,</strong> <em>8 </em><a href="https://doi.org/10.3389/fphar.2017.00298" title="DOI URL">https://doi.org/10.3389/fphar.2017.00298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2017.00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2017.00298%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DOn%252Bthe%252BIntegration%252Bof%252BIn%252BSilico%252BDrug%252BDesign%252BMethods%252Bfor%252BDrug%252BRepurposing%26aulast%3DMarch-Vila%26aufirst%3DEric%26date%3D2017%26date%3D2017%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan M.  Miller</span>, <span class="hlFld-ContribAuthor ">Walter H.  Moos</span>, <span class="hlFld-ContribAuthor ">Barbara H.  Munk</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Munk</span>. </span><span class="cited-content_cbyCitation_article-title">Drug discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 281-420. <a href="https://doi.org/10.1016/B978-0-08-100625-2.00011-8" title="DOI URL">https://doi.org/10.1016/B978-0-08-100625-2.00011-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-100625-2.00011-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-100625-2.00011-8%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252Bdiscovery%26aulast%3DMiller%26aufirst%3DSusan%2BM.%26date%3D2017%26spage%3D281%26epage%3D420%26pub%3DElsevier%26atitle%3DManaging%252Bthe%252BDrug%252BDiscovery%252BProcess%26date%3D2017%26volume%3D301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">G.K.  Kanev</span>, <span class="hlFld-ContribAuthor ">A.J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">I.J.P  de Esch</span>, <span class="hlFld-ContribAuthor ">C.  de Graaf</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Chemogenomics Databases to Navigate Protein–Ligand Interaction Space. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 444-471. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12298-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12298-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12298-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12298-X%26sid%3Dliteratum%253Aachs%26atitle%3DStructural%252BChemogenomics%252BDatabases%252Bto%252BNavigate%252BProtein%2525E2%252580%252593Ligand%252BInteraction%252BSpace%26aulast%3DKanev%26aufirst%3DG.K.%26date%3D2017%26spage%3D444%26epage%3D471%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gerry  Maggiora</span>, <span class="hlFld-ContribAuthor ">Vijay  Gokhale</span>. </span><span class="cited-content_cbyCitation_article-title">A simple mathematical approach to the analysis of polypharmacology and polyspecificity data. </span><span class="cited-content_cbyCitation_journal-name">F1000Research</span><span> <strong>2017,</strong> <em>6 </em>, 788. <a href="https://doi.org/10.12688/f1000research.11517.1" title="DOI URL">https://doi.org/10.12688/f1000research.11517.1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.12688/f1000research.11517.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.12688%2Ff1000research.11517.1%26sid%3Dliteratum%253Aachs%26jtitle%3DF1000Research%26atitle%3DA%252Bsimple%252Bmathematical%252Bapproach%252Bto%252Bthe%252Banalysis%252Bof%252Bpolypharmacology%252Band%252Bpolyspecificity%252Bdata%26aulast%3DMaggiora%26aufirst%3DGerry%26date%3D2017%26date%3D2017%26volume%3D6%26spage%3D788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joshua  Meyers</span>, <span class="hlFld-ContribAuthor ">Nathan  Brown</span>, <span class="hlFld-ContribAuthor ">Julian  Blagg</span>. </span><span class="cited-content_cbyCitation_article-title">Mapping the 3D structures of small molecule binding sites. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-016-0180-0" title="DOI URL">https://doi.org/10.1186/s13321-016-0180-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-016-0180-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-016-0180-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DMapping%252Bthe%252B3D%252Bstructures%252Bof%252Bsmall%252Bmolecule%252Bbinding%252Bsites%26aulast%3DMeyers%26aufirst%3DJoshua%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Noé  Sturm</span>, <span class="hlFld-ContribAuthor ">Didier  Rognan</span>, <span class="hlFld-ContribAuthor ">Ronald J  Quinn</span>, <span class="hlFld-ContribAuthor ">Esther  Kellenberger</span>. </span><span class="cited-content_cbyCitation_article-title">Comparing atom-based with residue-based descriptors in predicting binding site similarity: do backbone atoms matter?. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (15)
                                     , 1871-1885. <a href="https://doi.org/10.4155/fmc-2016-0077" title="DOI URL">https://doi.org/10.4155/fmc-2016-0077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0077%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DComparing%252Batom-based%252Bwith%252Bresidue-based%252Bdescriptors%252Bin%252Bpredicting%252Bbinding%252Bsite%252Bsimilarity%25253A%252Bdo%252Bbackbone%252Batoms%252Bmatter%25253F%26aulast%3DSturm%26aufirst%3DNo%25C3%25A9%26date%3D2016%26volume%3D8%26issue%3D15%26spage%3D1871%26epage%3D1885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jaroslav  Koča</span>, <span class="hlFld-ContribAuthor ">Radka  Svobodová Vařeková</span>, <span class="hlFld-ContribAuthor ">Lukáš  Pravda</span>, <span class="hlFld-ContribAuthor ">Karel  Berka</span>, <span class="hlFld-ContribAuthor ">Stanislav  Geidl</span>, <span class="hlFld-ContribAuthor ">David  Sehnal</span>, <span class="hlFld-ContribAuthor ">Michal  Otyepka</span>. </span><span class="cited-content_cbyCitation_article-title">Biomacromolecular Fragments and Patterns. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 7-15. <a href="https://doi.org/10.1007/978-3-319-47388-8_2" title="DOI URL">https://doi.org/10.1007/978-3-319-47388-8_2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-47388-8_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-47388-8_2%26sid%3Dliteratum%253Aachs%26atitle%3DBiomacromolecular%252BFragments%252Band%252BPatterns%26aulast%3DKo%25C4%258Da%26aufirst%3DJaroslav%26date%3D2016%26date%3D2017%26spage%3D7%26epage%3D15%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DStructural%252BBioinformatics%252BTools%252Bfor%252BDrug%252BDesign%26aulast%3DKo%25C4%258Da%26aufirst%3DJaroslav%26date%3D2016%26volume%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeffrey E.  Arle</span>, <span class="hlFld-ContribAuthor ">Kristen W.  Carlson</span>. </span><span class="cited-content_cbyCitation_article-title">The use of dynamic computational models of neural circuitry to streamline new drug development. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Disease Models</span><span> <strong>2016,</strong> <em>19 </em>, 69-75. <a href="https://doi.org/10.1016/j.ddmod.2017.01.002" title="DOI URL">https://doi.org/10.1016/j.ddmod.2017.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddmod.2017.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddmod.2017.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Disease%2520Models%26atitle%3DThe%252Buse%252Bof%252Bdynamic%252Bcomputational%252Bmodels%252Bof%252Bneural%252Bcircuitry%252Bto%252Bstreamline%252Bnew%252Bdrug%252Bdevelopment%26aulast%3DArle%26aufirst%3DJeffrey%2BE.%26date%3D2016%26volume%3D19%26spage%3D69%26epage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stevan W.  Djuric</span>, <span class="hlFld-ContribAuthor ">Charles W.  Hutchins</span>, <span class="hlFld-ContribAuthor ">Nari N.  Talaty</span>. </span><span class="cited-content_cbyCitation_article-title">Current status and future prospects for enabling chemistry technology in the drug discovery process. </span><span class="cited-content_cbyCitation_journal-name">F1000Research</span><span> <strong>2016,</strong> <em>5 </em>, 2426. <a href="https://doi.org/10.12688/f1000research.9515.1" title="DOI URL">https://doi.org/10.12688/f1000research.9515.1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.12688/f1000research.9515.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.12688%2Ff1000research.9515.1%26sid%3Dliteratum%253Aachs%26jtitle%3DF1000Research%26atitle%3DCurrent%252Bstatus%252Band%252Bfuture%252Bprospects%252Bfor%252Benabling%252Bchemistry%252Btechnology%252Bin%252Bthe%252Bdrug%252Bdiscovery%252Bprocess%26aulast%3DDjuric%26aufirst%3DStevan%2BW.%26date%3D2016%26date%3D2016%26volume%3D5%26spage%3D2426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. PDB statistics on the number of released PDB entries (blue bars) and the number of new CATH<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> folds (red bars) and SCOP<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> folds (green bars) per year.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. sc-PDB<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> statistics on the number of unique targets per ligand illustrating the impact of ligand promiscuity.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Cofactors and ions are not considered as ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Different levels of binding site representation for subsequent comparisons. The figure is based on a crystal structure of matriptase in complex with an inhibitor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r0i">4r0i</a>) and was generated using MOE 2013.08.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Common ligand-sensing core of LSD1 and MAO-B: (a) structure of compound <b>1</b>; (b) common ligand-sensing core of LSD1 in green (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ejr">2ejr</a>) and MAO-A with bound inhibitor in orange in ball-and-sticks representation (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2bxr">2bxr</a>); (c) active sites of both enzymes with cavity-lining residues represented as sticks and names according to the crystal structures. Parts b and c were generated using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of compounds analyzed in repurposing and polypharmacology studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Alignment of synapsin I to the protein kinase Pim-1 with respect to the ATP-binding site. The structure of Pim-1 is shown in forest green (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a99">3a99</a>) with bound ATP analog represented as sticks. Synapsin I (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aux">1aux</a>) is colored orange. (a) Alignment of both proteins which differ in fold. (b) Detailed view of the aligned ATP-binding sites and the cavity-lining residues. Parts a and b were generated using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Comparison of FtsZ from <i>S. aureus</i>, <i>S. epidermidis</i>, and <i>B. subtilis</i> with respect to the <i>Sa</i>FtsZ binding site of compound <b>15</b>: (a) structure of <b>15</b>; (b) overall alignment of <i>Sa</i>FtsZ with bound <b>15</b> in ball-and-sticks representation in green (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dxd">4dxd</a>), <i>Se</i>FtsZ in purple (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4m8i">4m8i</a>), and <i>Bs</i>FtsZ in orange (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2rhl">2rhl</a>); (c) binding site alignment of all three proteins. Their structures share highly similar binding site residues (represented as sticks, names according to crystal structures). Nevertheless, they differ with respect to their binding site geometry. Parts b and c were created using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Comparison between three mycobacterial MeaB enzymes and human MMAA: (a) overall alignment of the different enzymes’ structures (forest green, <i>Mtb</i> MeaB with bound GDP molecule in sticks representation, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3md0">3md0</a>; orange, <i>M. smegmatis</i> MeaB, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nxs">3nxs</a>; purple, <i>M. thermoresistibile</i> MeaB, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3tk1">3tk1</a>; magenta, human MMAA protein, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2www">2www</a>); (b) detailed view of the major differences between the mycobacterial proteins and the human MMAA protein in the nucleotide binding site. Respective residues are represented as sticks and named according to the crystal structures. This figure was created using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of compounds analyzed in off-target prediction studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Comparison of the nucleotide binding surfaces of the pyrophosphatase and pyruvate kinase after an alignment according to bound nucleotides: (a) overall structure of the <i>Methanocaldococcus jannaschii</i> pyrophosphatase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2mjp">2mjp</a>); (b) overall structure of pyruvate kinase from <i>Oryctolagus cuniculus</i> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1a49">1a49</a>); (c) binding site surface of pyrophosphatase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2mjp">2mjp</a>) colored according to the electrostatic potential with bound ATP analog (sticks); (d) binding site surface of pyruvate kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1a49">1a49</a>) colored according to the electrostatic potential with bound ATP (sticks). Parts a–d were created using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Illustrative example of convergent evolution as observed for a member of the ABC transporter family and a protein of the Hpr kinase/phosphorylase family: (a) alignment between the unique phosphate binding motif of the ABC transport family member (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b0u">1b0u</a>, green) with bound ATP in green sticks representation and Hpr kinase/phosphorylase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kkl">1kkl</a>, orange); (b) detailed view of the common phosphate binding motif of both proteins. Two differently oriented β-strands contain essential binding site residues which are represented as sticks. Parts a and b were created using UCSF Chimera 1.10.1.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/medium/jm-2016-00078d_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Impact of binding site comparison methods on rational drug design and medicinal chemistry with focus on the importance of its combination with other structure-based drug design methods. Helpful computational tools are shown for different scenarios and are presented in green boxes, while orange boxes highlight steps in a screening workflow which have to be done in the lab. BSC marks steps in those workflows where binding site comparison can help to accelerate the search for promising hits. The superscripts refer to a summary of methods that were already successfully used in this stage of hit identification as discussed in the text. The reader has to bear in mind that the application domains partially overlap and cannot be clearly distinguished.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.6b00078/20160506/images/large/jm-2016-00078d_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00078&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85836" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85836" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 185 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&issue=1&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lgs5OJkyNiZcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26issue%3D1%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Johnson, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karanicolas, J.</span><span> </span><span class="NLM_article-title">Druggable protein interaction sites are more predisposed to surface pocket formation than the rest of the protein surface</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002951</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1002951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1002951" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1002951&issue=3&author=D.+K.+Johnsonauthor=J.+Karanicolas&title=Druggable+protein+interaction+sites+are+more+predisposed+to+surface+pocket+formation+than+the+rest+of+the+protein+surface&doi=10.1371%2Fjournal.pcbi.1002951"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002951%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DD.%2BK.%26aulast%3DKaranicolas%26aufirst%3DJ.%26atitle%3DDruggable%2520protein%2520interaction%2520sites%2520are%2520more%2520predisposed%2520to%2520surface%2520pocket%2520formation%2520than%2520the%2520rest%2520of%2520the%2520protein%2520surface%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D3%26spage%3De1002951%26doi%3D10.1371%2Fjournal.pcbi.1002951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Loving, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, A. C.</span><span> </span><span class="NLM_article-title">Structure-based druggability assessment of the mammalian structural proteome with inclusion of light protein flexibility</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1003741</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1003741" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003741&issue=7&author=K.+A.+Lovingauthor=A.+Linauthor=A.+C.+Cheng&title=Structure-based+druggability+assessment+of+the+mammalian+structural+proteome+with+inclusion+of+light+protein+flexibility&doi=10.1371%2Fjournal.pcbi.1003741"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003741%26sid%3Dliteratum%253Aachs%26aulast%3DLoving%26aufirst%3DK.%2BA.%26aulast%3DLin%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DA.%2BC.%26atitle%3DStructure-based%2520druggability%2520assessment%2520of%2520the%2520mammalian%2520structural%2520proteome%2520with%2520inclusion%2520of%2520light%2520protein%2520flexibility%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26issue%3D7%26spage%3De1003741%26doi%3D10.1371%2Fjournal.pcbi.1003741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Orengo, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michie, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swindells, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">CATH—a hierarchic classification of protein domain structures</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1016/S0969-2126(97)00260-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0969-2126%2897%2900260-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=1093-1108&issue=8&author=C.+A.+Orengoauthor=A.+D.+Michieauthor=S.+Jonesauthor=D.+T.+Jonesauthor=M.+B.+Swindellsauthor=J.+M.+Thornton&title=CATH%E2%80%94a+hierarchic+classification+of+protein+domain+structures&doi=10.1016%2FS0969-2126%2897%2900260-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2897%2900260-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252897%252900260-8%26sid%3Dliteratum%253Aachs%26aulast%3DOrengo%26aufirst%3DC.%2BA.%26aulast%3DMichie%26aufirst%3DA.%2BD.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DD.%2BT.%26aulast%3DSwindells%26aufirst%3DM.%2BB.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DCATH%25E2%2580%2594a%2520hierarchic%2520classification%2520of%2520protein%2520domain%2520structures%26jtitle%3DStructure%26date%3D1997%26volume%3D5%26issue%3D8%26spage%3D1093%26epage%3D1108%26doi%3D10.1016%2FS0969-2126%2897%2900260-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Murzin, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenner, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chothia, C.</span><span> </span><span class="NLM_article-title">SCOP: a structural classification of proteins database for the investigation of sequences and structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">247</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span><span class="refDoi"> DOI: 10.1016/S0022-2836(05)80134-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2805%2980134-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=7723011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2MXltVGgsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=1995&pages=536-540&issue=4&author=A.+G.+Murzinauthor=S.+E.+Brennerauthor=T.+Hubbardauthor=C.+Chothia&title=SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures&doi=10.1016%2FS0022-2836%2805%2980134-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">SCOP: a structural classification of proteins database for the investigation of sequences and structures</span></div><div class="casAuthors">Murzin, Alexey G.; Brenner, Steven E.; Hubbard, Tim; Chothia, Cyrus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">247</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">536-40</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">To facilitate understanding of, and access to, the information available for protein structures, we have constructed the Structural Classification of Proteins (SCOP) database.  This database provides a detailed and comprehensive description of the structural and evolutionary relationships of the proteins of known structure.  It also provides for each entry links to coordinates, images of the structure, interactive viewers, sequence data and literature refs.  Two search facilitates are available.  The homol. search permits users to enter a sequence and obtain a list of any structures to which it has significant levels of sequence similarity.  The key word search finds, for a word entered by the user, matches from both the text of the SCOP database and the headers of Brookhaven Protein Databank structure files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-9_Oh4CusCbVg90H21EOLACvtfcHk0lj25nq4BfKX6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXltVGgsr4%253D&md5=65de3f2464aa75ba7efabbd183507913</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980134-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980134-2%26sid%3Dliteratum%253Aachs%26aulast%3DMurzin%26aufirst%3DA.%2BG.%26aulast%3DBrenner%26aufirst%3DS.%2BE.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DChothia%26aufirst%3DC.%26atitle%3DSCOP%253A%2520a%2520structural%2520classification%2520of%2520proteins%2520database%2520for%2520the%2520investigation%2520of%2520sequences%2520and%2520structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1995%26volume%3D247%26issue%3D4%26spage%3D536%26epage%3D540%26doi%3D10.1016%2FS0022-2836%2805%2980134-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span>RCSB PDB. <a href="http://www.rcsb.org/pdb/static.do?p=general_information/pdb_statistics/index.html" class="extLink">http://www.rcsb.org/pdb/static.do?p=general_information/pdb_statistics/index.html</a> (accessed March 21,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RCSB+PDB.+http%3A%2F%2Fwww.rcsb.org%2Fpdb%2Fstatic.do%3Fp%3Dgeneral_information%2Fpdb_statistics%2Findex.html+%28accessed+March+21%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Skolnick, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srinivasan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, H.</span><span> </span><span class="NLM_article-title">Implications of the small number of distinct ligand binding pockets in proteins for drug discovery, evolution and biochemical function</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1163</span><span class="NLM_x">–</span> <span class="NLM_lpage">1170</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.01.059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.bmcl.2015.01.059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1163-1170&issue=6&author=J.+Skolnickauthor=M.+Gaoauthor=A.+Royauthor=B.+Srinivasanauthor=H.+Zhou&title=Implications+of+the+small+number+of+distinct+ligand+binding+pockets+in+proteins+for+drug+discovery%2C+evolution+and+biochemical+function&doi=10.1016%2Fj.bmcl.2015.01.059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.01.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.01.059%26sid%3Dliteratum%253Aachs%26aulast%3DSkolnick%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DSrinivasan%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DH.%26atitle%3DImplications%2520of%2520the%2520small%2520number%2520of%2520distinct%2520ligand%2520binding%2520pockets%2520in%2520proteins%2520for%2520drug%2520discovery%252C%2520evolution%2520and%2520biochemical%2520function%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26issue%3D6%26spage%3D1163%26epage%3D1170%26doi%3D10.1016%2Fj.bmcl.2015.01.059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Paul, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Recovering the true targets of specific ligands by virtual screening of the protein data bank</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">680</span><span class="refDoi"> DOI: 10.1002/prot.10625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.10625" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2004&pages=671-680&issue=4&author=N.+Paulauthor=E.+Kellenbergerauthor=G.+Bretauthor=P.+M%C3%BCllerauthor=D.+Rognan&title=Recovering+the+true+targets+of+specific+ligands+by+virtual+screening+of+the+protein+data+bank&doi=10.1002%2Fprot.10625"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fprot.10625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10625%26sid%3Dliteratum%253Aachs%26aulast%3DPaul%26aufirst%3DN.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DM%25C3%25BCller%26aufirst%3DP.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DRecovering%2520the%2520true%2520targets%2520of%2520specific%2520ligands%2520by%2520virtual%2520screening%2520of%2520the%2520protein%2520data%2520bank%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D54%26issue%3D4%26spage%3D671%26epage%3D680%26doi%3D10.1002%2Fprot.10625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">sc-PDB: an annotated database of druggable binding sites from the Protein Data Bank</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">717</span><span class="NLM_x">–</span> <span class="NLM_lpage">727</span><span class="refDoi"> DOI: 10.1021/ci050372x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050372x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFCgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=717-727&issue=2&author=E.+Kellenbergerauthor=P.+Mullerauthor=C.+Schalonauthor=G.+Bretauthor=N.+Foataauthor=D.+Rognan&title=sc-PDB%3A+an+annotated+database+of+druggable+binding+sites+from+the+Protein+Data+Bank&doi=10.1021%2Fci050372x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">sc-PDB: An Annotated Database of Druggable Binding Sites from the Protein Data Bank</span></div><div class="casAuthors">Kellenberger, Esther; Muller, Pascal; Schalon, Claire; Bret, Guillaume; Foata, Nicolas; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">717-727</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The s.c.-PDB is a collection of 6,415 three-dimensional structures of binding sites found in the Protein Data Bank (PDB).  Binding sites were extd. from all high-resoln. crystal structures in which a complex between a protein cavity and a small-mol.-wt. ligand could be identified.  Importantly, ligands are considered from a pharmacol. and not a structural point of view.  Therefore, solvents, detergents, and most metal ions are not stored in the s.c.-PDB.  Ligands are classified into four main categories: nucleotides (< 4-mer), peptides (< 9-mer), cofactors, and org. compds.  The corresponding binding site is formed by all protein residues (including amino acids, cofactors, and important metal ions) with at least one atom within 6.5 Å of any ligand atom.  The database was carefully annotated by browsing several protein databases (PDB, UniProt, and GO) and storing, for every s.c.-PDB entry, the following features: protein name, function, source, domain and mutations, ligand name, and structure.  The repository of ligands has also been archived by diversity anal. of mol. scaffolds, and several chemoinformatics descriptors were computed to better understand the chem. space covered by stored ligands.  The s.c.-PDB may be used for several purposes: (i) screening a collection of binding sites for predicting the most likely target(s) of any ligand, (ii) analyzing the mol. similarity between different cavities, and (iii) deriving rules that describe the relationship between ligand pharmacophoric points and active-site properties.  The database is periodically updated and accessible on the web at http://bioinfo-pharma.u-strasbg.fr/scPDB/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNlZ_WkpBUQrVg90H21EOLACvtfcHk0lj25nq4BfKX6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFCgtQ%253D%253D&md5=d035f998192ff55026c619b38f4b82ab</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fci050372x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050372x%26sid%3Dliteratum%253Aachs%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DMuller%26aufirst%3DP.%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DFoata%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3Dsc-PDB%253A%2520an%2520annotated%2520database%2520of%2520druggable%2520binding%2520sites%2520from%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26issue%3D2%26spage%3D717%26epage%3D727%26doi%3D10.1021%2Fci050372x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Sturm, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span> </span><span class="NLM_article-title">Structural insights into the molecular basis of the ligand promiscuity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">2410</span><span class="NLM_x">–</span> <span class="NLM_lpage">2421</span><span class="refDoi"> DOI: 10.1021/ci300196g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300196g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1GhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2410-2421&issue=9&author=N.+Sturmauthor=J.+Desaphyauthor=R.+J.+Quinnauthor=D.+Rognanauthor=E.+Kellenberger&title=Structural+insights+into+the+molecular+basis+of+the+ligand+promiscuity&doi=10.1021%2Fci300196g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Insights into the Molecular Basis of the Ligand Promiscuity</span></div><div class="casAuthors">Sturm, Noe; Desaphy, Jeremy; Quinn, Ronald J.; Rognan, Didier; Kellenberger, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2410-2421</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selectivity is a key factor in drug development.  In this paper, we questioned the Protein Data Bank to better understand the reasons for the promiscuity of bioactive compds.  We assembled a data set of >1000 pairs of three-dimensional structures of complexes between a "drug-like" ligand (as its physicochem. properties overlap that of approved drugs) and two distinct "druggable" protein targets (as their binding sites are likely to accommodate "drug-like" ligands).  Studying the similarity between the ligand-binding sites in the different targets revealed that the lack of selectivity of a ligand can be due (i) to the fact that Nature has created the same binding pocket in different proteins, which do not necessarily have otherwise sequence or fold similarity, or (ii) to specific characteristics of the ligand itself.  In particular, we demonstrated that many ligands can adapt to different protein environments by changing their conformation, by using different chem. moieties to anchor to different targets, or by adopting unusual extreme binding modes (e.g., only apolar contact between the ligand and the protein, even though polar groups are present on the ligand or at the protein surface).  Lastly, we provided new elements in support to the recent studies which suggest that the promiscuity of a ligand might be inferred from its mol. complexity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaq6HYzXDUl7Vg90H21EOLACvtfcHk0liTQqc2DscS_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1GhurbF&md5=808b3192a4110ec1bb26ddd41165d60d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci300196g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300196g%26sid%3Dliteratum%253Aachs%26aulast%3DSturm%26aufirst%3DN.%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DQuinn%26aufirst%3DR.%2BJ.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DStructural%2520insights%2520into%2520the%2520molecular%2520basis%2520of%2520the%2520ligand%2520promiscuity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D9%26spage%3D2410%26epage%3D2421%26doi%3D10.1021%2Fci300196g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surgand, J.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">A simple and fuzzy method to align and compare druggable ligand-binding sites</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1755</span><span class="NLM_x">–</span> <span class="NLM_lpage">1778</span><span class="refDoi"> DOI: 10.1002/prot.21858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.21858" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=1755-1778&issue=4&author=C.+Schalonauthor=J.-S.+Surgandauthor=E.+Kellenbergerauthor=D.+Rognan&title=A+simple+and+fuzzy+method+to+align+and+compare+druggable+ligand-binding+sites&doi=10.1002%2Fprot.21858"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fprot.21858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21858%26sid%3Dliteratum%253Aachs%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DSurgand%26aufirst%3DJ.-S.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DA%2520simple%2520and%2520fuzzy%2520method%2520to%2520align%2520and%2520compare%2520druggable%2520ligand-binding%2520sites%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2008%26volume%3D71%26issue%3D4%26spage%3D1755%26epage%3D1778%26doi%3D10.1002%2Fprot.21858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azdimousa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Comparison and druggability prediction of protein-ligand binding sites from pharmacophore-annotated cavity shapes</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">2287</span><span class="NLM_x">–</span> <span class="NLM_lpage">2299</span><span class="refDoi"> DOI: 10.1021/ci300184x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300184x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2isb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2287-2299&issue=8&author=J.+Desaphyauthor=K.+Azdimousaauthor=E.+Kellenbergerauthor=D.+Rognan&title=Comparison+and+druggability+prediction+of+protein-ligand+binding+sites+from+pharmacophore-annotated+cavity+shapes&doi=10.1021%2Fci300184x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison and Druggability Prediction of Protein-Ligand Binding Sites from Pharmacophore-Annotated Cavity Shapes</span></div><div class="casAuthors">Desaphy, Jeremy; Azdimousa, Karima; Kellenberger, Esther; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2287-2299</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Estg. the pairwise similarity of protein-ligand binding sites is a fast and efficient way of predicting cross-reactivity and putative side effects of drug candidates.  Among the many tools available, three-dimensional (3D) alignment-dependent methods are usually slow and based on simplified representations of binding site atoms or surfaces.  On the other hand, fast and efficient alignment-free methods have recently been described but suffer from a lack of interpretability.  We herewith present a novel binding site description (VolSite), coupled to an alignment and comparison tool (Shaper) combining the speed of alignment-free methods with the interpretability of alignment-dependent approaches.  It is based on the comparison of neg. images of binding cavities encoding both shape and pharmacophoric properties at regularly spaced grid points.  Shaper approximates the resulting mol. shape with a smooth Gaussian function and aligns protein binding sites by optimizing their vol. overlap.  Volsite and Shaper were successfully applied to compare protein-ligand binding sites and to predict their structural druggability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjUMu2riLDLbVg90H21EOLACvtfcHk0liTQqc2DscS_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2isb7I&md5=d019f05891c51085323078e5a880e30b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci300184x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300184x%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DAzdimousa%26aufirst%3DK.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DComparison%2520and%2520druggability%2520prediction%2520of%2520protein-ligand%2520binding%2520sites%2520from%2520pharmacophore-annotated%2520cavity%2520shapes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D8%26spage%3D2287%26epage%3D2299%26doi%3D10.1021%2Fci300184x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Weill, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Alignment-free ultra-high-throughput comparison of druggable protein-ligand binding sites</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1021/ci900349y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900349y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1yjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=123-135&issue=1&author=N.+Weillauthor=D.+Rognan&title=Alignment-free+ultra-high-throughput+comparison+of+druggable+protein-ligand+binding+sites&doi=10.1021%2Fci900349y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Alignment-Free Ultra-High-Throughput Comparison of Druggable Protein-Ligand Binding Sites</span></div><div class="casAuthors">Weill, Nathanael; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">123-135</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inferring the biol. function of a protein from its three-dimensional structure as well as explaining why a drug may bind to various targets is of crucial importance to modern drug discovery.  Here we present a generic 4833-integer vector describing druggable protein-ligand binding sites that can be applied to any protein and any binding cavity.  The fingerprint registers counts of pharmacophoric triplets from the Cα at. coordinates of binding-site-lining residues.  Starting from a customized data set of diverse protein-ligand binding site pairs, the most appropriate metric and a similarity threshold could be defined for similar binding sites.  The method (FuzCav) has been used in various scenarios: (i) screening a collection of 6000 binding sites for similarity to different queries; (ii) classifying protein families (serine endopeptidases, protein kinases) by binding site diversity; (iii) discriminating adenine-binding cavities from decoys.  The fingerprint generation and comparison supports ultra-high throughput (ca. 1000 measures/s), does not require prior alignment of protein binding sites, and is able to detect local similarity among subpockets.  It is thus particularly well suited to the functional annotation of novel genomic structures with low sequence identity to known X-ray templates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7OYklk3JFxLVg90H21EOLACvtfcHk0ljK2kJLmmgFOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1yjsg%253D%253D&md5=541a9800a3aa0891f179316412f1df24</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fci900349y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900349y%26sid%3Dliteratum%253Aachs%26aulast%3DWeill%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DAlignment-free%2520ultra-high-throughput%2520comparison%2520of%2520druggable%2520protein-ligand%2520binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D1%26spage%3D123%26epage%3D135%26doi%3D10.1021%2Fci900349y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span>scPDB: An Annotated Database of Druggable Binding Sites from the Protein DataBank. <a href="http://cheminfo.u-strasbg.fr/scPDB/ABOUT" class="extLink">http://cheminfo.u-strasbg.fr/scPDB/ABOUT</a> (accessed March 21,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=scPDB%3A+An+Annotated+Database+of+Druggable+Binding+Sites+from+the+Protein+DataBank.+http%3A%2F%2Fcheminfo.u-strasbg.fr%2FscPDB%2FABOUT+%28accessed+March+21%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Martin, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orengo, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchinson, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karmirantzou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taroni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">Protein folds and functions</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">875</span><span class="NLM_x">–</span> <span class="NLM_lpage">884</span><span class="refDoi"> DOI: 10.1016/S0969-2126(98)00089-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0969-2126%2898%2900089-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=875-884&issue=7&author=A.+C.+Martinauthor=C.+A.+Orengoauthor=E.+G.+Hutchinsonauthor=S.+Jonesauthor=M.+Karmirantzouauthor=R.+A.+Laskowskiauthor=J.+B.+Mitchellauthor=C.+Taroniauthor=J.+M.+Thornton&title=Protein+folds+and+functions&doi=10.1016%2FS0969-2126%2898%2900089-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2898%2900089-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252898%252900089-6%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DA.%2BC.%26aulast%3DOrengo%26aufirst%3DC.%2BA.%26aulast%3DHutchinson%26aufirst%3DE.%2BG.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DKarmirantzou%26aufirst%3DM.%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%26aulast%3DTaroni%26aufirst%3DC.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DProtein%2520folds%2520and%2520functions%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26issue%3D7%26spage%3D875%26epage%3D884%26doi%3D10.1016%2FS0969-2126%2898%2900089-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Meara, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">The recognition of identical ligands by unrelated proteins</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2772</span><span class="NLM_x">–</span> <span class="NLM_lpage">2784</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2772-2784&issue=12&author=S.+Barelierauthor=T.+Sterlingauthor=M.+J.+O%E2%80%99Mearaauthor=B.+K.+Shoichet&title=The+recognition+of+identical+ligands+by+unrelated+proteins&doi=10.1021%2Facschembio.5b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The recognition of identical ligands by unrelated proteins</span></div><div class="casAuthors">Barelier, Sarah; Sterling, Teague; O'Meara, Matthew J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2772-2784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of drugs and reagents to off-targets is well-known.  Whereas many off-targets are related to the primary target by sequence and fold, many ligands bind to unrelated pairs of proteins, and these are harder to anticipate.  If the binding site in the off-target can be related to that of the primary target, this challenge resolves into aligning the 2 pockets.  However, other cases are possible: the ligand might interact with entirely different residues and environments in the off-target, or wholly different ligand atoms may be implicated in the 2 complexes.  To investigate these scenarios at at. resoln., the structures of 59 ligands in 116 complexes (62 pairs in total), where the protein pairs were unrelated by fold but bound an identical ligand, were examd.  In almost half of the pairs, the ligand interacted with unrelated residues in the 2 proteins (29 pairs), and in 14 of the pairs wholly different ligand moieties were implicated in each complex.  Even in those 19 pairs of complexes that presented similar environments to the ligand, ligand superposition rarely resulted in the overlap of related residues.  There appears to be no single pattern-matching "code" for identifying binding sites in unrelated proteins that bind identical ligands, although modeling suggests that there might be a limited no. of different patterns that suffice to recognize different ligand functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1_lQJDNMCrVg90H21EOLACvtfcHk0ljK2kJLmmgFOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J&md5=9d6ac31c2a58bc85f77e7eb34649c195</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00683%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DThe%2520recognition%2520of%2520identical%2520ligands%2520by%2520unrelated%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D12%26spage%3D2772%26epage%3D2784%26doi%3D10.1021%2Facschembio.5b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span><span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&issue=2&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0ljK2kJLmmgFOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26issue%3D2%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Li, G.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.-X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, F.-Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lou, M.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.-H.</span><span> </span><span class="NLM_article-title">In silico analysis and experimental validation of active compounds from Cichorium intybus L. ameliorating liver injury</span> <span class="citation_source-journal">Int. J. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">22190</span><span class="NLM_x">–</span> <span class="NLM_lpage">22204</span><span class="refDoi"> DOI: 10.3390/ijms160922190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.3390%2Fijms160922190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=26389883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1yht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=22190-22204&issue=9&author=G.-Y.+Liauthor=Y.-X.+Zhengauthor=F.-Z.+Sunauthor=J.+Huangauthor=M.-M.+Louauthor=J.-K.+Guauthor=J.-H.+Wang&title=In+silico+analysis+and+experimental+validation+of+active+compounds+from+Cichorium+intybus+L.+ameliorating+liver+injury&doi=10.3390%2Fijms160922190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">In silico analysis and experimental validation of active compounds from Cichorium intybus L. ameliorating liver injury</span></div><div class="casAuthors">Li, Guo-Yu; Zheng, Ya-Xin; Sun, Fu-Zhou; Huang, Jian; Lou, Meng-Meng; Gu, Jing-Kai; Wang, Jin-Hui</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">22190-22204</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">This study aimed at investigating the possible mechanisms of hepatic protective activity of Cichorium intybus L. (chicory) in acute liver injury.  Pathol. observation, reactive oxygen species (ROS) detection and measurements of biochem. indexes on mouse models proved hepatic protective effect of Cichorium intybus L.  Identification of active compds. in Cichorium intybus L. was executed through several methods including ultra performance liq. chromatog./time of flight mass spectrometry (UPLC-TOF-MS).  Similarity ensemble approach (SEA) docking, mol. modeling, mol. docking, and mol. dynamics (MD) simulation were applied in this study to explore possible mechanisms of the hepato-protective potential of Cichorium intybus L.  We then analyzed the chem. compn. of Cichorium intybus L., and found their key targets.  Furthermore, in vitro cytol. examn. and western blot were used for validating the efficacy of the selected compds.  In silico anal. and western blot together demonstrated that selected compd. 10 in Cichorium intybus L. targeted Akt-1 in hepatocytes.  Besides, compd. 13 targeted both caspase-1 and Akt-1.  These small compds. may ameliorate liver injury by acting on their targets, which are related to apoptosis or autophagy.  The conclusions above may shed light on the complex mol. mechanisms of Cichorium intybus L. acting on hepatocytes and ameliorating liver injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQb1mIlt--7bVg90H21EOLACvtfcHk0lhJeFx3aCZN6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1yht7c%253D&md5=8954a2e5f7ad0c125b3ab6a1d6631b80</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3390%2Fijms160922190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms160922190%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-Y.%26aulast%3DZheng%26aufirst%3DY.-X.%26aulast%3DSun%26aufirst%3DF.-Z.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLou%26aufirst%3DM.-M.%26aulast%3DGu%26aufirst%3DJ.-K.%26aulast%3DWang%26aufirst%3DJ.-H.%26atitle%3DIn%2520silico%2520analysis%2520and%2520experimental%2520validation%2520of%2520active%2520compounds%2520from%2520Cichorium%2520intybus%2520L.%2520ameliorating%2520liver%2520injury%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2015%26volume%3D16%26issue%3D9%26spage%3D22190%26epage%3D22204%26doi%3D10.3390%2Fijms160922190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krüger, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowotka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papadatos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">The ChEMBL bioactivity database: an update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D1083</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1090</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-D1090&issue=D1&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+bioactivity+database%3A+an+update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0lhJeFx3aCZN6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520bioactivity%2520database%253A%2520an%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD1083%26epage%3DD1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Wishart, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shrivastava, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassanali, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stothard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolsey, J.</span><span> </span><span class="NLM_article-title">DrugBank: a comprehensive resource for in silico drug discovery and exploration</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D668</span><span class="NLM_x">–</span> <span class="NLM_lpage">D672</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkj067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkj067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=D668-D672&issue=Suppl.+1&author=D.+S.+Wishartauthor=C.+Knoxauthor=A.+C.+Guoauthor=S.+Shrivastavaauthor=M.+Hassanaliauthor=P.+Stothardauthor=Z.+Changauthor=J.+Woolsey&title=DrugBank%3A+a+comprehensive+resource+for+in+silico+drug+discovery+and+exploration&doi=10.1093%2Fnar%2Fgkj067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkj067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkj067%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DShrivastava%26aufirst%3DS.%26aulast%3DHassanali%26aufirst%3DM.%26aulast%3DStothard%26aufirst%3DP.%26aulast%3DChang%26aufirst%3DZ.%26aulast%3DWoolsey%26aufirst%3DJ.%26atitle%3DDrugBank%253A%2520a%2520comprehensive%2520resource%2520for%2520in%2520silico%2520drug%2520discovery%2520and%2520exploration%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26issue%3DSuppl.%25201%26spage%3DD668%26epage%3DD672%26doi%3D10.1093%2Fnar%2Fgkj067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzek, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gindulyte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">PubChem BioAssay: 2014 update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D1075</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1082</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkt978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkt978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24198245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslaq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1075-D1082&issue=D1&author=Y.+Wangauthor=T.+Suzekauthor=J.+Zhangauthor=J.+Wangauthor=S.+Heauthor=T.+Chengauthor=B.+A.+Shoemakerauthor=A.+Gindulyteauthor=S.+H.+Bryant&title=PubChem+BioAssay%3A+2014+update&doi=10.1093%2Fnar%2Fgkt978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem BioAssay: 2014 update</span></div><div class="casAuthors">Wang, Yanli; Suzek, Tugba; Zhang, Jian; Wang, Jiyao; He, Siqian; Cheng, Tiejun; Shoemaker, Benjamin A.; Gindulyte, Asta; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1075-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem's BioAssay database (http://pubchem.ncbi.nlm.nih.gov) is a public repository for archiving biol. tests of small mols. generated through high-throughput screening expts., medicinal chem. studies, chem. biol. research and drug discovery programs.  In addn., the BioAssay database contains data from high-throughput RNA interference screening aimed at identifying crit. genes responsible for a biol. process or disease condition.  The mission of PubChem is to serve the community by providing free and easy access to all deposited data.  To this end, PubChem BioAssay is integrated into the National Center for Biotechnol. Information retrieval system, making them searchable by Entrez queries and cross-linked to other biomedical information archived at National Center for Biotechnol. Information.  Moreover, PubChem BioAssay provides web-based and programmatic tools allowing users to search, access and analyze bioassay test results and metadata.  In this work, we provide an update for the PubChem BioAssay resource, such as information content growth, new developments supporting data integration and search, and the recently deployed PubChem Upload to streamline chem. structure and bioassay submissions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1tnO28g86-rVg90H21EOLACvtfcHk0lhJeFx3aCZN6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslaq&md5=66caa053197bcaf40b49535b0a3e0c27</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt978%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSuzek%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubChem%2520BioAssay%253A%25202014%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD1075%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkt978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Günther, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunkel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campillos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsalaki, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urdiales, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gewiess, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skoblo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preissner, R.</span><span> </span><span class="NLM_article-title">SuperTarget and Matador: resources for exploring drug-target relationships</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D919</span><span class="NLM_x">–</span> <span class="NLM_lpage">D922</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkm862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkm862" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D919-D922&issue=Suppl.+1&author=S.+G%C3%BCntherauthor=M.+Kuhnauthor=M.+Dunkelauthor=M.+Campillosauthor=C.+Sengerauthor=E.+Petsalakiauthor=J.+Ahmedauthor=E.+G.+Urdialesauthor=A.+Gewiessauthor=L.+J.+Jensenauthor=R.+Schneiderauthor=R.+Skobloauthor=R.+B.+Russellauthor=P.+E.+Bourneauthor=P.+Borkauthor=R.+Preissner&title=SuperTarget+and+Matador%3A+resources+for+exploring+drug-target+relationships&doi=10.1093%2Fnar%2Fgkm862"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm862%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DDunkel%26aufirst%3DM.%26aulast%3DCampillos%26aufirst%3DM.%26aulast%3DSenger%26aufirst%3DC.%26aulast%3DPetsalaki%26aufirst%3DE.%26aulast%3DAhmed%26aufirst%3DJ.%26aulast%3DUrdiales%26aufirst%3DE.%2BG.%26aulast%3DGewiess%26aufirst%3DA.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DSkoblo%26aufirst%3DR.%26aulast%3DRussell%26aufirst%3DR.%2BB.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DBork%26aufirst%3DP.%26aulast%3DPreissner%26aufirst%3DR.%26atitle%3DSuperTarget%2520and%2520Matador%253A%2520resources%2520for%2520exploring%2520drug-target%2520relationships%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26issue%3DSuppl.%25201%26spage%3DD919%26epage%3DD922%26doi%3D10.1093%2Fnar%2Fgkm862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Kufareva, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilatovskiy, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Pocketome: an encyclopedia of small-molecule binding sites in 4D</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D535</span><span class="NLM_x">–</span> <span class="NLM_lpage">D540</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkr825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkr825" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D535-D540&issue=D1&author=I.+Kufarevaauthor=A.+V.+Ilatovskiyauthor=R.+Abagyan&title=Pocketome%3A+an+encyclopedia+of+small-molecule+binding+sites+in+4D&doi=10.1093%2Fnar%2Fgkr825"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr825%26sid%3Dliteratum%253Aachs%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DIlatovskiy%26aufirst%3DA.%2BV.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DPocketome%253A%2520an%2520encyclopedia%2520of%2520small-molecule%2520binding%2520sites%2520in%25204D%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DD1%26spage%3DD535%26epage%3DD540%26doi%3D10.1093%2Fnar%2Fgkr825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnickel, G.</span><span> </span><span class="NLM_article-title">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">63, 389</span><span class="refDoi"> DOI: 10.1016/S1093-3263(98)00002-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS1093-3263%2898%2900002-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=9704298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADyaK1czmvF2hsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=359-63%2C+389&issue=6&author=M.+Hendlichauthor=F.+Rippmannauthor=G.+Barnickel&title=LIGSITE%3A+automatic+and+efficient+detection+of+potential+small+molecule-binding+sites+in+proteins&doi=10.1016%2FS1093-3263%2898%2900002-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span></div><div class="casAuthors">Hendlich M; Rippmann F; Barnickel G</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular graphics & modelling</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">359-63, 389</span>
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    </div><div class="casAbstract">LIGSITE is a new program for the automatic and time-efficient detection of pockets on the surface of proteins that may act as binding sites for small molecule ligands.  Pockets are identified with a series of simple operations on a cubic grid.  Using a set of receptor-ligand complexes we show that LIGSITE is able to identify the binding sites of small molecule ligands with high precision.  The main advantage of LIGSITE is its speed.  Typical search times are in the range of 5 to 20 s for medium-sized proteins.  LIGSITE is therefore well suited for identification of pockets in large sets of proteins (e.g., protein families) for comparative studies.  For graphical display LIGSITE produces VRML representations of the protein-ligand complex and the binding site for display with a VRML viewer such as WebSpace from SGI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRvgmm5GX6HIgaK-_ZudimnfW6udTcc2ebVVqPLEwQZrLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czmvF2hsw%253D%253D&md5=b7174632bdeaebcfbedec66824ec5cb6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2898%2900002-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252898%252900002-3%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DBarnickel%26aufirst%3DG.%26atitle%3DLIGSITE%253A%2520automatic%2520and%2520efficient%2520detection%2520of%2520potential%2520small%2520molecule-binding%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D1997%26volume%3D15%26issue%3D6%26spage%3D359%26epage%3D63%252C%2520389%26doi%3D10.1016%2FS1093-3263%2898%2900002-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Laurie, A. T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Q-SiteFinder: an energy-based method for the prediction of protein-ligand binding sites</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">1908</span><span class="NLM_x">–</span> <span class="NLM_lpage">1916</span><span class="refDoi"> DOI: 10.1093/bioinformatics/bti315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbti315" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2005&pages=1908-1916&issue=9&author=A.+T.+R.+Laurieauthor=R.+M.+Jackson&title=Q-SiteFinder%3A+an+energy-based+method+for+the+prediction+of+protein-ligand+binding+sites&doi=10.1093%2Fbioinformatics%2Fbti315"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbti315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbti315%26sid%3Dliteratum%253Aachs%26aulast%3DLaurie%26aufirst%3DA.%2BT.%2BR.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DQ-SiteFinder%253A%2520an%2520energy-based%2520method%2520for%2520the%2520prediction%2520of%2520protein-ligand%2520binding%2520sites%26jtitle%3DBioinformatics%26date%3D2005%26volume%3D21%26issue%3D9%26spage%3D1908%26epage%3D1916%26doi%3D10.1093%2Fbioinformatics%2Fbti315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Fukunishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Prediction of ligand-binding sites of proteins by molecular docking calculation for a random ligand library</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">95</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1002/pro.540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fpro.540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21064162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVKjsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=95-106&issue=1&author=Y.+Fukunishiauthor=H.+Nakamura&title=Prediction+of+ligand-binding+sites+of+proteins+by+molecular+docking+calculation+for+a+random+ligand+library&doi=10.1002%2Fpro.540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of ligand-binding sites of proteins by molecular docking calculation for a random ligand library</span></div><div class="casAuthors">Fukunishi, Yoshifumi; Nakamura, Haruki</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-106</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A new approach to predicting the ligand-binding sites of proteins was developed, using protein-ligand docking computation.  In this method, many compds. in a random library are docked onto the whole protein surface.  We assumed that the true ligand-binding site would exhibit stronger affinity to the compds. in the random library than the other sites, even if the random library did not include the ligand corresponding to the true binding site.  We also assumed that the affinity of the true ligand-binding site would be correlated to the docking scores of the compds. in the random library, if the ligand-binding site was correctly predicted.  We call this method the mol.-docking binding-site finding (MolSite) method.  The MolSite method was applied to 89 known protein-ligand complex structures extd. from the Protein Data Bank, and it predicted the correct binding sites with about 80-99% accuracy, when only the single top-ranked site was adopted.  In addn., the av. docking score was weakly correlated to the exptl. protein-ligand binding free energy, with a correlation coeff. of 0.44.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoOFDTHjqTZ7Vg90H21EOLACvtfcHk0lgAZUai47RuYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVKjsbw%253D&md5=77b4a7942208fa4b50b382332664ba6f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fpro.540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.540%26sid%3Dliteratum%253Aachs%26aulast%3DFukunishi%26aufirst%3DY.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DPrediction%2520of%2520ligand-binding%2520sites%2520of%2520proteins%2520by%2520molecular%2520docking%2520calculation%2520for%2520a%2520random%2520ligand%2520library%26jtitle%3DProtein%2520Sci.%26date%3D2011%26volume%3D20%26issue%3D1%26spage%3D95%26epage%3D106%26doi%3D10.1002%2Fpro.540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Pupko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayrose, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glaser, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ben-Tal, N.</span><span> </span><span class="NLM_article-title">Rate4Site: an algorithmic tool for the identification of functional regions in proteins by surface mapping of evolutionary determinants within their homologues</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S71</span><span class="NLM_x">–</span> <span class="NLM_lpage">S77</span><span class="refDoi"> DOI: 10.1093/bioinformatics/18.suppl_1.S71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2F18.suppl_1.S71" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2002&pages=S71-S77&issue=Suppl.+1&author=T.+Pupkoauthor=R.+E.+Bellauthor=I.+Mayroseauthor=F.+Glaserauthor=N.+Ben-Tal&title=Rate4Site%3A+an+algorithmic+tool+for+the+identification+of+functional+regions+in+proteins+by+surface+mapping+of+evolutionary+determinants+within+their+homologues&doi=10.1093%2Fbioinformatics%2F18.suppl_1.S71"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2F18.suppl_1.S71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252F18.suppl_1.S71%26sid%3Dliteratum%253Aachs%26aulast%3DPupko%26aufirst%3DT.%26aulast%3DBell%26aufirst%3DR.%2BE.%26aulast%3DMayrose%26aufirst%3DI.%26aulast%3DGlaser%26aufirst%3DF.%26aulast%3DBen-Tal%26aufirst%3DN.%26atitle%3DRate4Site%253A%2520an%2520algorithmic%2520tool%2520for%2520the%2520identification%2520of%2520functional%2520regions%2520in%2520proteins%2520by%2520surface%2520mapping%2520of%2520evolutionary%2520determinants%2520within%2520their%2520homologues%26jtitle%3DBioinformatics%26date%3D2002%26volume%3D18%26issue%3DSuppl.%25201%26spage%3DS71%26epage%3DS77%26doi%3D10.1093%2Fbioinformatics%2F18.suppl_1.S71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Huang, B.</span><span> </span><span class="NLM_article-title">MetaPocket: a meta approach to improve protein ligand binding site prediction</span> <span class="citation_source-journal">OMICS</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">330</span><span class="refDoi"> DOI: 10.1089/omi.2009.0045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1089%2Fomi.2009.0045" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=325-330&issue=4&author=B.+Huang&title=MetaPocket%3A+a+meta+approach+to+improve+protein+ligand+binding+site+prediction&doi=10.1089%2Fomi.2009.0045"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1089%2Fomi.2009.0045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fomi.2009.0045%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26atitle%3DMetaPocket%253A%2520a%2520meta%2520approach%2520to%2520improve%2520protein%2520ligand%2520binding%2520site%2520prediction%26jtitle%3DOMICS%26date%3D2009%26volume%3D13%26issue%3D4%26spage%3D325%26epage%3D330%26doi%3D10.1089%2Fomi.2009.0045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Xie, Z.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, M.-J.</span><span> </span><span class="NLM_article-title">Methods for predicting protein-ligand binding sites</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1215</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">398</span><span class="refDoi"> DOI: 10.1007/978-1-4939-1465-4_17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2F978-1-4939-1465-4_17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25330972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVOku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1215&publication_year=2015&pages=383-398&author=Z.-R.+Xieauthor=M.-J.+Hwang&title=Methods+for+predicting+protein-ligand+binding+sites&doi=10.1007%2F978-1-4939-1465-4_17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for Predicting Protein-Ligand Binding Sites</span></div><div class="casAuthors">Xie, Zhong-Ru; Hwang, Ming-Jing</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1215</span>
        (<span class="NLM_cas:issue">Molecular Modeling of Proteins</span>),
    <span class="NLM_cas:pages">383-398</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Ligand binding is required for many proteins to function properly.  A large no. of bioinformatics tools have been developed to predict ligand binding sites as a first step in understanding a protein's function or to facilitate docking computations in virtual screening based drug design.  The prediction usually requires only the three-dimensional structure (exptl. detd. or computationally modeled) of the target protein to be searched for ligand binding site(s), and Web servers have been built, allowing the free and simple use of prediction tools.  In this chapter, we review the underlying concepts of the methods used by various tools, and discuss their different features and the related issues of ligand binding site prediction.  Some cautionary notes about the use of these tools are also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjGl8HOBerlbVg90H21EOLACvtfcHk0lieo0ZOckrbig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVOku7k%253D&md5=5214d7bbc66d968d823d50f34503ffd4</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-1465-4_17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-1465-4_17%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DZ.-R.%26aulast%3DHwang%26aufirst%3DM.-J.%26atitle%3DMethods%2520for%2520predicting%2520protein-ligand%2520binding%2520sites%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1215%26spage%3D383%26epage%3D398%26doi%3D10.1007%2F978-1-4939-1465-4_17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Andersson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linusson, A.</span><span> </span><span class="NLM_article-title">Mapping of ligand-binding cavities in proteins</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1408</span><span class="NLM_x">–</span> <span class="NLM_lpage">1422</span><span class="refDoi"> DOI: 10.1002/prot.22655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.22655" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=1408-1422&issue=6&author=C.+D.+Anderssonauthor=B.+Y.+Chenauthor=A.+Linusson&title=Mapping+of+ligand-binding+cavities+in+proteins&doi=10.1002%2Fprot.22655"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fprot.22655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22655%26sid%3Dliteratum%253Aachs%26aulast%3DAndersson%26aufirst%3DC.%2BD.%26aulast%3DChen%26aufirst%3DB.%2BY.%26aulast%3DLinusson%26aufirst%3DA.%26atitle%3DMapping%2520of%2520ligand-binding%2520cavities%2520in%2520proteins%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2010%26volume%3D78%26issue%3D6%26spage%3D1408%26epage%3D1422%26doi%3D10.1002%2Fprot.22655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Boareto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamagishi, M. E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caticha, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leite, V. B. P.</span><span> </span><span class="NLM_article-title">Relationship between global structural parameters and Enzyme Commission hierarchy: implications for function prediction</span> <span class="citation_source-journal">Comput. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">19</span><span class="refDoi"> DOI: 10.1016/j.compbiolchem.2012.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.compbiolchem.2012.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22926016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWisbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=15-19&author=M.+Boaretoauthor=M.+E.+B.+Yamagishiauthor=N.+Catichaauthor=V.+B.+P.+Leite&title=Relationship+between+global+structural+parameters+and+Enzyme+Commission+hierarchy%3A+implications+for+function+prediction&doi=10.1016%2Fj.compbiolchem.2012.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Relationship between global structural parameters and Enzyme Commission hierarchy: Implications for function prediction</span></div><div class="casAuthors">Boareto, Marcelo; Yamagishi, Michel E. B.; Caticha, Nestor; Leite, Vitor B. P.</div><div class="citationInfo"><span class="NLM_cas:title">Computational Biology and Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-19</span>CODEN:
                <span class="NLM_cas:coden">CBCOCH</span>;
        ISSN:<span class="NLM_cas:issn">1476-9271</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In protein databases there is a substantial no. of proteins structurally detd. but without function annotation.  Understanding the relationship between function and structure can be useful to predict function on a large scale.  We have analyzed the similarities in global physicochem. parameters for a set of enzymes which were classified according to the four Enzyme Commission (EC) hierarchical levels.  Using relevance theory we introduced a distance between proteins in the space of physicochem. characteristics.  This was done by minimizing a cost function of the metric tensor built to reflect the EC classification system.  Using an unsupervised clustering method on a set of 1025 enzymes, we obtained no relevant clustering formation compatible with EC classification.  The distance distributions between enzymes from the same EC group and from different EC groups were compared by histograms.  Such anal. was also performed using sequence alignment similarity as a distance.  Our results suggest that global structure parameters are not sufficient to segregate enzymes according to EC hierarchy.  This indicates that features essential for function are rather local than global.  Consequently, methods for predicting function based on global attributes should not obtain high accuracy in main EC classes prediction without relying on similarities between enzymes from training and validation datasets.  Furthermore, these results are consistent with a substantial no. of studies suggesting that function evolves fundamentally by recruitment, i.e., a same protein motif or fold can be used to perform different enzymic functions and a few specific amino acids (AAs) are actually responsible for enzyme activity.  These essential amino acids should belong to active sites and an effective method for predicting function should be able to recognize them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSJ7tit54xL7Vg90H21EOLACvtfcHk0lieo0ZOckrbig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFWisbnF&md5=8f88e06f1c98c42ee778d08a8e46e0d0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.compbiolchem.2012.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.compbiolchem.2012.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DBoareto%26aufirst%3DM.%26aulast%3DYamagishi%26aufirst%3DM.%2BE.%2BB.%26aulast%3DCaticha%26aufirst%3DN.%26aulast%3DLeite%26aufirst%3DV.%2BB.%2BP.%26atitle%3DRelationship%2520between%2520global%2520structural%2520parameters%2520and%2520Enzyme%2520Commission%2520hierarchy%253A%2520implications%2520for%2520function%2520prediction%26jtitle%3DComput.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D40%26spage%3D15%26epage%3D19%26doi%3D10.1016%2Fj.compbiolchem.2012.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to rational drug design</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">38</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0707307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fsj.bjp.0707307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=17533416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=38-52&issue=1&author=D.+Rognan&title=Chemogenomic+approaches+to+rational+drug+design&doi=10.1038%2Fsj.bjp.0707307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to rational drug design</span></div><div class="casAuthors">Rognan, D.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-52</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Paradigms in drug design and discovery are changing at a significant pace.  Concomitant to the sequencing of over 180 several genomes, the high-throughput miniaturization of chem. synthesis and biol. evaluation of a multiple compds. on gene/protein expression and function opens the way to global drug-discovery approaches, no more focused on a single target but on an entire family of related proteins or on a full metabolic pathway.  Chemogenomics is this emerging research field aimed at systematically studying the biol. effect of a wide array of small mol.-wt. ligands on a wide array of macromol. targets.  Since the quantity of existing data (compds., targets and assays) and of produced information (gene/protein expression levels and binding consts.) are too large for manual manipulation, information technologies play a crucial role in planning, analyzing and predicting chemogenomic data.  The present review will focus on predictive in silico chemogenomic approaches to foster rational drug design and derive information from the simultaneous biol. evaluation of multiple compds. on multiple targets.  State-of-the-art methods for navigating in either ligand or target space will be presented and concrete drug design applications will be mentioned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPERITBubs0LVg90H21EOLACvtfcHk0lh7n0QE1U_pkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjurw%253D&md5=47b3b2a5155a079b486e56d31e0ca59c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707307%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DChemogenomic%2520approaches%2520to%2520rational%2520drug%2520design%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26issue%3D1%26spage%3D38%26epage%3D52%26doi%3D10.1038%2Fsj.bjp.0707307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Haupt, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daminelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M.</span><span> </span><span class="NLM_article-title">Drug promiscuity in PDB: protein binding site similarity is key</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">e65894</span><span class="refDoi"> DOI: 10.1371/journal.pone.0065894</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0065894" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=23805191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVOhsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e65894&issue=6&author=V.+J.+Hauptauthor=S.+Daminelliauthor=M.+Schroeder&title=Drug+promiscuity+in+PDB%3A+protein+binding+site+similarity+is+key&doi=10.1371%2Fjournal.pone.0065894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Drug promiscuity in PDB: protein binding site similarity is key</span></div><div class="casAuthors">Haupt, V. Joachim; Daminelli, Simone; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e65894</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Drug repositioning applies established drugs to new disease indications with increasing success.  A pre-requisite for drug repurposing is drug promiscuity (polypharmacol.) - a drug's ability to bind to several targets.  There is a long standing debate on the reasons for drug promiscuity.  Based on large compd. screens, hydrophobicity and mol. wt. have been suggested as key reasons.  However, the results are sometimes contradictory and leave space for further anal.  Protein structures offer a structural dimension to explain promiscuity: Can a drug bind multiple targets because the drug is flexible or because the targets are structurally similar or even share similar binding sites.  We present a systematic study of drug promiscuity based on structural data of PDB target proteins with a set of 164 promiscuous drugs.  We show that there is no correlation between the degree of promiscuity and ligand properties such as hydrophobicity or mol. wt. but a weak correlation to conformational flexibility.  However, we do find a correlation between promiscuity and structural similarity as well as binding site similarity of protein targets.  In particular, 71% of the drugs have at least two targets with similar binding sites.  In order to overcome issues in detection of remotely similar binding sites, we employed a score for binding site similarity: Ligand RMSD measures the similarity of the aligned ligands and uncovers remote local similarities in proteins.  It can be applied to arbitrary structural binding site alignments.  Three representative examples, namely the anti-cancer drug methotrexate, the natural product quercetin and the anti-diabetic drug acarbose are discussed in detail.  Our findings suggest that global structural and binding site similarity play a more important role to explain the obsd. drug promiscuity in the PDB than physicochem. drug properties like hydrophobicity or mol. wt.  Addnl., we find ligand flexibility to have a minor influence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSXd_1yYo5lLVg90H21EOLACvtfcHk0lh7n0QE1U_pkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVOhsLnP&md5=99574519028e547ebec5a4c2ed27f6a5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0065894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0065894%26sid%3Dliteratum%253Aachs%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DDaminelli%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DDrug%2520promiscuity%2520in%2520PDB%253A%2520protein%2520binding%2520site%2520similarity%2520is%2520key%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26issue%3D6%26spage%3De65894%26doi%3D10.1371%2Fjournal.pone.0065894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Bajusz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rácz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Héberger, K.</span><span> </span><span class="NLM_article-title">Why is Tanimoto index an appropriate choice for fingerprint-based similarity calculations?</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="refDoi"> DOI: 10.1186/s13321-015-0069-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2Fs13321-015-0069-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=20&author=D.+Bajuszauthor=A.+R%C3%A1czauthor=K.+H%C3%A9berger&title=Why+is+Tanimoto+index+an+appropriate+choice+for+fingerprint-based+similarity+calculations%3F&doi=10.1186%2Fs13321-015-0069-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0069-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0069-3%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DR%25C3%25A1cz%26aufirst%3DA.%26aulast%3DH%25C3%25A9berger%26aufirst%3DK.%26atitle%3DWhy%2520is%2520Tanimoto%2520index%2520an%2520appropriate%2520choice%2520for%2520fingerprint-based%2520similarity%2520calculations%253F%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D20%26doi%3D10.1186%2Fs13321-015-0069-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Dekker, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">Protein structure similarity clustering (PSSC) and natural product structure as inspiration sources for drug development and chemical genomics</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">232</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2005.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.cbpa.2005.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=15939324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltVaqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=232-239&issue=3&author=F.+J.+Dekkerauthor=M.+A.+Kochauthor=H.+Waldmann&title=Protein+structure+similarity+clustering+%28PSSC%29+and+natural+product+structure+as+inspiration+sources+for+drug+development+and+chemical+genomics&doi=10.1016%2Fj.cbpa.2005.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Protein structure similarity clustering (PSSC) and natural product structure as inspiration sources for drug development and chemical genomics</span></div><div class="casAuthors">Dekker, Frank J.; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">232-239</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding small mols. that modulate protein function is of primary importance in drug development and in the emerging field of chem. genomics.  To facilitate the identification of such mols., we developed a novel strategy making use of structural conservatism found in protein domain architecture and natural product inspired compd. library design.  Domains and proteins identified as being structurally similar in their ligand-sensing cores are grouped in a protein structure similarity cluster (PSSC).  Natural products can be considered as evolutionary pre-validated ligands for multiple proteins and therefore natural products that are known to interact with one of the PSSC member proteins are selected as guiding structures for compd. library synthesis.  Application of this novel strategy for compd. library design provided enhanced hit rates in small compd. libraries for structurally similar proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk6ALeUcX2kLVg90H21EOLACvtfcHk0lh7n0QE1U_pkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltVaqtrk%253D&md5=87b55a2206ba7d0e9170e3d909142743</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DDekker%26aufirst%3DF.%2BJ.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DProtein%2520structure%2520similarity%2520clustering%2520%2528PSSC%2529%2520and%2520natural%2520product%2520structure%2520as%2520inspiration%2520sources%2520for%2520drug%2520development%2520and%2520chemical%2520genomics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26issue%3D3%26spage%3D232%26epage%3D239%26doi%3D10.1016%2Fj.cbpa.2005.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Koch, O.</span><span> </span><span class="NLM_article-title">Use of secondary structure element information in drug design: polypharmacology and conserved motifs in protein-ligand binding and protein-protein interfaces</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">699</span><span class="NLM_x">–</span> <span class="NLM_lpage">708</span><span class="refDoi"> DOI: 10.4155/fmc.11.26</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.4155%2Ffmc.11.26" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21554076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlvFGisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=699-708&issue=6&author=O.+Koch&title=Use+of+secondary+structure+element+information+in+drug+design%3A+polypharmacology+and+conserved+motifs+in+protein-ligand+binding+and+protein-protein+interfaces&doi=10.4155%2Ffmc.11.26"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Use of secondary structure element information in drug design: polypharmacology and conserved motifs in protein--ligand binding and protein--protein interfaces</span></div><div class="casAuthors">Koch, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">699-708</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The structure-based design of small-mol. inhibitors of protein--ligand and protein--protein interfaces is a key component of drug discovery.  The underlying protein interactions can be regarded based on structural similarity of the secondary structure elements: similarities around the binding site ( ligand-sensing cores'') or in the protein interface ( interface-sensing surfaces'') in otherwise unrelated proteins can be useful in predicting polypharmacol. and identifying new lead structures.  Even small conserved motifs can provide similar interaction patterns in proteins with a completely different fold and function.  The identification of these structural similarities can help in the design of new drugs by guiding further optimization.  Here, the concepts and ideas based on secondary structure element similarities and their successful applications in drug design are reviewed and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXOVETHrI5CLVg90H21EOLACvtfcHk0lhr-bgB_5K7oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlvFGisLo%253D&md5=64425bac5c12f615b8302b0f3b4ad7f4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.26%26sid%3Dliteratum%253Aachs%26aulast%3DKoch%26aufirst%3DO.%26atitle%3DUse%2520of%2520secondary%2520structure%2520element%2520information%2520in%2520drug%2520design%253A%2520polypharmacology%2520and%2520conserved%2520motifs%2520in%2520protein-ligand%2520binding%2520and%2520protein-protein%2520interfaces%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26issue%3D6%26spage%3D699%26epage%3D708%26doi%3D10.4155%2Ffmc.11.26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Henrich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salo-Ahen, O. M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wade, R. C.</span><span> </span><span class="NLM_article-title">Computational approaches to identifying and characterizing protein binding sites for ligand design</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1002/jmr.984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fjmr.984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=19746440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2010&pages=209-219&issue=2&author=S.+Henrichauthor=O.+M.+H.+Salo-Ahenauthor=B.+Huangauthor=F.+F.+Rippmannauthor=G.+Crucianiauthor=R.+C.+Wade&title=Computational+approaches+to+identifying+and+characterizing+protein+binding+sites+for+ligand+design&doi=10.1002%2Fjmr.984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Computational approaches to identifying and characterizing protein binding sites for ligand design</span></div><div class="casAuthors">Henrich, Stefan; Salo-Ahen, Outi M. H.; Huang, Bingding; Rippmann, Friedrich F.; Cruciani, Gabriele; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">209-219</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  Given the 3-dimensional structure of a protein, how can 1 find the sites where other mols. might bind to it.  Do these sites have the properties necessary for high affinity binding.  Is this protein a suitable target for drug design.  Here, the authors discuss recent developments in computational methods to address these and related questions.  Geometric methods to identify pockets on protein surfaces have been developed over many years but, with new algorithms, their performance is still improving.  Simulation methods show promise in accounting for protein conformational variability to identify transient pockets but lack the ease of use of many of the (rigid) shape-based tools.  Sequence and structure comparison approaches are benefiting from the constantly increasing size of sequence and structure databases.  Energetic methods can aid identification and characterization of binding pockets, and have undergone recent improvements in the treatment of solvation and hydrophobicity.  The "druggability" of a binding site is still difficult to predict with an automated procedure.  The methodologies available for this purpose range from simple shape and hydrophobicity scores to computationally demanding free energy simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCcgqE6phTYbVg90H21EOLACvtfcHk0lhr-bgB_5K7oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSht70%253D&md5=a114c8ffea54495e020c8908a6b092b6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjmr.984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.984%26sid%3Dliteratum%253Aachs%26aulast%3DHenrich%26aufirst%3DS.%26aulast%3DSalo-Ahen%26aufirst%3DO.%2BM.%2BH.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DRippmann%26aufirst%3DF.%2BF.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DComputational%2520approaches%2520to%2520identifying%2520and%2520characterizing%2520protein%2520binding%2520sites%2520for%2520ligand%2520design%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2010%26volume%3D23%26issue%3D2%26spage%3D209%26epage%3D219%26doi%3D10.1002%2Fjmr.984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">How to measure the similarity between protein ligand-binding sites?</span> <span class="citation_source-journal">Curr. Comput.-Aided Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.2174/157340908785747401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2174%2F157340908785747401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=209-220&issue=3&author=E.+Kellenbergerauthor=C.+Schalonauthor=D.+Rognan&title=How+to+measure+the+similarity+between+protein+ligand-binding+sites%3F&doi=10.2174%2F157340908785747401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">How to measure the similarity between protein ligand-binding sites?</span></div><div class="casAuthors">Kellenberger, Esther; Schalon, Claire; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">209-220</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Quantification of local similarity between protein 3D structures is a promising tool in computer-aided drug design and prediction of biol. function.  Over the last ten years, several computational methods were proposed, mostly based on geometrical comparisons.  This review summarizes the recent literature and gives an overview of available programs.  A particular interest is given to the underlying methodologies.  Our anal. points out strengths and weaknesses of the various approaches.  If all described methods work relatively well when two binding sites obviously resemble each other, scoring potential solns. remains a difficult issue, esp. if the similarity is low.  The other challenging question is the protein flexibility, which is indeed difficult to evaluate from a static representation.  Last, most of recently developed techniques are fast and can be applied to large amts. of data.  Examples were carefully chosen to illustrate the wide applicability domain of the most popular methods: detection of common structural motifs, identification of secondary targets for a drug-like compd., comparison of binding sites across a functional family, comparison of homol. models, database screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmk10IU2NMu7Vg90H21EOLACvtfcHk0lhr-bgB_5K7oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKqt7g%253D&md5=d10c65eb50f41ea52c826ebba3693c95</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2174%2F157340908785747401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340908785747401%26sid%3Dliteratum%253Aachs%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DHow%2520to%2520measure%2520the%2520similarity%2520between%2520protein%2520ligand-binding%2520sites%253F%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2008%26volume%3D4%26issue%3D3%26spage%3D209%26epage%3D220%26doi%3D10.2174%2F157340908785747401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span>MOE (Molecular Operating Environment); <span class="NLM_publisher-name">Chemical Computing Group Inc.</span> (<span class="NLM_publisher-loc">1010 Sherbooke St. West, Suite 910, Montreal, QC, H3A 2R7, Canada</span>),<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MOE+%28Molecular+Operating+Environment%29%3B+Chemical+Computing+Group+Inc.+%281010+Sherbooke+St.+West%2C+Suite+910%2C+Montreal%2C+QC%2C+H3A+2R7%2C+Canada%29%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMOE%2520%2528Molecular%2520Operating%2520Environment%2529%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wittenberg, L.-O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeyaraj, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gourzoulidou, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinecke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Odermatt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">Compound library development guided by protein structure similarity clustering and natural product structure</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x"> (</span><span class="NLM_issue">48</span><span class="NLM_x">) </span> <span class="NLM_fpage">16721</span><span class="NLM_x">–</span> <span class="NLM_lpage">16726</span><span class="refDoi"> DOI: 10.1073/pnas.0404719101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1073%2Fpnas.0404719101" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=16721-16726&issue=48&author=M.+A.+Kochauthor=L.-O.+Wittenbergauthor=S.+Basuauthor=D.+A.+Jeyarajauthor=E.+Gourzoulidouauthor=K.+Reineckeauthor=A.+Odermattauthor=H.+Waldmann&title=Compound+library+development+guided+by+protein+structure+similarity+clustering+and+natural+product+structure&doi=10.1073%2Fpnas.0404719101"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0404719101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0404719101%26sid%3Dliteratum%253Aachs%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWittenberg%26aufirst%3DL.-O.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DJeyaraj%26aufirst%3DD.%2BA.%26aulast%3DGourzoulidou%26aufirst%3DE.%26aulast%3DReinecke%26aufirst%3DK.%26aulast%3DOdermatt%26aufirst%3DA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DCompound%2520library%2520development%2520guided%2520by%2520protein%2520structure%2520similarity%2520clustering%2520and%2520natural%2520product%2520structure%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26issue%3D48%26spage%3D16721%26epage%3D16726%26doi%3D10.1073%2Fpnas.0404719101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">Ligand binding site superposition and comparison based on Atomic Property Fields: identification of distant homologues, convergent evolution and PDB-wide clustering of binding sites</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S35</span><span class="refDoi"> DOI: 10.1186/1471-2105-12-S1-S35</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-12-S1-S35" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=S35&issue=Suppl.+1&author=M.+Totrov&title=Ligand+binding+site+superposition+and+comparison+based+on+Atomic+Property+Fields%3A+identification+of+distant+homologues%2C+convergent+evolution+and+PDB-wide+clustering+of+binding+sites&doi=10.1186%2F1471-2105-12-S1-S35"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-S1-S35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-S1-S35%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DLigand%2520binding%2520site%2520superposition%2520and%2520comparison%2520based%2520on%2520Atomic%2520Property%2520Fields%253A%2520identification%2520of%2520distant%2520homologues%252C%2520convergent%2520evolution%2520and%2520PDB-wide%2520clustering%2520of%2520binding%2520sites%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26issue%3DSuppl.%25201%26spage%3DS35%26doi%3D10.1186%2F1471-2105-12-S1-S35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Edwards, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baugh, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baydo, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witte, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompkins, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, I. Q. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abendroth, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sankaran, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Voorhis, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myler, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staker, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorimer, D. D.</span><span> </span><span class="NLM_article-title">Crystal structures of Mycobacterial MeaB and MMAA-like GTPases</span> <span class="citation_source-journal">J. Struct. Funct. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span><span class="refDoi"> DOI: 10.1007/s10969-015-9197-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2Fs10969-015-9197-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25832174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlslGjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=91-99&issue=2&author=T.+E.+Edwardsauthor=L.+Baughauthor=J.+Bullenauthor=R.+O.+Baydoauthor=P.+Witteauthor=K.+Thompkinsauthor=I.+Q.+H.+Phanauthor=J.+Abendrothauthor=M.+C.+Cliftonauthor=B.+Sankaranauthor=W.+C.+van+Voorhisauthor=P.+J.+Mylerauthor=B.+L.+Stakerauthor=C.+Grundnerauthor=D.+D.+Lorimer&title=Crystal+structures+of+Mycobacterial+MeaB+and+MMAA-like+GTPases&doi=10.1007%2Fs10969-015-9197-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of Mycobacterial MeaB and MMAA-like GTPases</span></div><div class="casAuthors">Edwards, Thomas E.; Baugh, Loren; Bullen, Jameson; Baydo, Ruth O.; Witte, Pam; Thompkins, Kaitlin; Phan, Isabelle Q. H.; Abendroth, Jan; Clifton, Matthew C.; Sankaran, Banumathi; Van Voorhis, Wesley C.; Myler, Peter J.; Staker, Bart L.; Grundner, Christoph; Lorimer, Donald D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural and Functional Genomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">91-99</span>CODEN:
                <span class="NLM_cas:coden">JSFGAW</span>;
        ISSN:<span class="NLM_cas:issn">1345-711X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The methylmalonyl Co-A mutase-assocd. GTPase MeaB from Methylobacterium extorquens is involved in glyoxylate regulation and required for growth.  In humans, mutations in the homolog methylmalonic aciduria assocd. protein (MMAA) cause methylmalonic aciduria, which is often fatal.  The central role of MeaB from bacteria to humans suggests that MeaB is also important in other, pathogenic bacteria such as Mycobacterium tuberculosis.  However, the identity of the mycobacterial MeaB homolog is presently unclear.  Here, we identify the M. tuberculosis protein Rv1496 and its homologs in M. smegmatis and M. thermoresistibile as MeaB.  The crystal structures of all three homologs are highly similar to MeaB and MMAA structures and reveal a characteristic three-domain homodimer with GDP bound in the G domain active site.  A structure of Rv1496 obtained from a crystal grown in the presence of GTP exhibited electron d. for GDP, suggesting GTPase activity.  These structures identify the mycobacterial MeaB and provide a structural framework for therapeutic targeting of M. tuberculosis MeaB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHyvqCau3g77Vg90H21EOLACvtfcHk0li3kIEnZN_lsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlslGjs7k%253D&md5=834f0ee305e48e493f1dd22dff711dd3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2Fs10969-015-9197-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10969-015-9197-2%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DT.%2BE.%26aulast%3DBaugh%26aufirst%3DL.%26aulast%3DBullen%26aufirst%3DJ.%26aulast%3DBaydo%26aufirst%3DR.%2BO.%26aulast%3DWitte%26aufirst%3DP.%26aulast%3DThompkins%26aufirst%3DK.%26aulast%3DPhan%26aufirst%3DI.%2BQ.%2BH.%26aulast%3DAbendroth%26aufirst%3DJ.%26aulast%3DClifton%26aufirst%3DM.%2BC.%26aulast%3DSankaran%26aufirst%3DB.%26aulast%3Dvan%2BVoorhis%26aufirst%3DW.%2BC.%26aulast%3DMyler%26aufirst%3DP.%2BJ.%26aulast%3DStaker%26aufirst%3DB.%2BL.%26aulast%3DGrundner%26aufirst%3DC.%26aulast%3DLorimer%26aufirst%3DD.%2BD.%26atitle%3DCrystal%2520structures%2520of%2520Mycobacterial%2520MeaB%2520and%2520MMAA-like%2520GTPases%26jtitle%3DJ.%2520Struct.%2520Funct.%2520Genomics%26date%3D2015%26volume%3D16%26issue%3D2%26spage%3D91%26epage%3D99%26doi%3D10.1007%2Fs10969-015-9197-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Baugh, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Begley, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armour, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dranow, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muruthi, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abendroth, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairman, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dieterich, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staker, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardberg, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napuli, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mundt, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompkins, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyons-Abbott, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abramov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serbzhinskiy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorimer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchko, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stacy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Voorhis, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myler, P. J.</span><span> </span><span class="NLM_article-title">Increasing the structural coverage of tuberculosis drug targets</span> <span class="citation_source-journal">Tuberculosis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">142</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span><span class="refDoi"> DOI: 10.1016/j.tube.2014.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.tube.2014.12.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2015&pages=142-148&issue=2&author=L.+Baughauthor=I.+Phanauthor=D.+W.+Begleyauthor=M.+C.+Cliftonauthor=B.+Armourauthor=D.+M.+Dranowauthor=B.+M.+Taylorauthor=M.+M.+Muruthiauthor=J.+Abendrothauthor=J.+W.+Fairmanauthor=D.+Foxauthor=S.+H.+Dieterichauthor=B.+L.+Stakerauthor=A.+S.+Gardbergauthor=R.+Choiauthor=S.+N.+Hewittauthor=A.+J.+Napuliauthor=J.+Myersauthor=L.+K.+Barrettauthor=Y.+Zhangauthor=M.+Ferrellauthor=E.+Mundtauthor=K.+Thompkinsauthor=N.+Tranauthor=S.+Lyons-Abbottauthor=A.+Abramovauthor=A.+Sekarauthor=D.+Serbzhinskiyauthor=D.+Lorimerauthor=G.+W.+Buchkoauthor=R.+Stacyauthor=L.+J.+Stewartauthor=T.+E.+Edwardsauthor=W.+C.+van+Voorhisauthor=P.+J.+Myler&title=Increasing+the+structural+coverage+of+tuberculosis+drug+targets&doi=10.1016%2Fj.tube.2014.12.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.tube.2014.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tube.2014.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DBaugh%26aufirst%3DL.%26aulast%3DPhan%26aufirst%3DI.%26aulast%3DBegley%26aufirst%3DD.%2BW.%26aulast%3DClifton%26aufirst%3DM.%2BC.%26aulast%3DArmour%26aufirst%3DB.%26aulast%3DDranow%26aufirst%3DD.%2BM.%26aulast%3DTaylor%26aufirst%3DB.%2BM.%26aulast%3DMuruthi%26aufirst%3DM.%2BM.%26aulast%3DAbendroth%26aufirst%3DJ.%26aulast%3DFairman%26aufirst%3DJ.%2BW.%26aulast%3DFox%26aufirst%3DD.%26aulast%3DDieterich%26aufirst%3DS.%2BH.%26aulast%3DStaker%26aufirst%3DB.%2BL.%26aulast%3DGardberg%26aufirst%3DA.%2BS.%26aulast%3DChoi%26aufirst%3DR.%26aulast%3DHewitt%26aufirst%3DS.%2BN.%26aulast%3DNapuli%26aufirst%3DA.%2BJ.%26aulast%3DMyers%26aufirst%3DJ.%26aulast%3DBarrett%26aufirst%3DL.%2BK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DFerrell%26aufirst%3DM.%26aulast%3DMundt%26aufirst%3DE.%26aulast%3DThompkins%26aufirst%3DK.%26aulast%3DTran%26aufirst%3DN.%26aulast%3DLyons-Abbott%26aufirst%3DS.%26aulast%3DAbramov%26aufirst%3DA.%26aulast%3DSekar%26aufirst%3DA.%26aulast%3DSerbzhinskiy%26aufirst%3DD.%26aulast%3DLorimer%26aufirst%3DD.%26aulast%3DBuchko%26aufirst%3DG.%2BW.%26aulast%3DStacy%26aufirst%3DR.%26aulast%3DStewart%26aufirst%3DL.%2BJ.%26aulast%3DEdwards%26aufirst%3DT.%2BE.%26aulast%3Dvan%2BVoorhis%26aufirst%3DW.%2BC.%26aulast%3DMyler%26aufirst%3DP.%2BJ.%26atitle%3DIncreasing%2520the%2520structural%2520coverage%2520of%2520tuberculosis%2520drug%2520targets%26jtitle%3DTuberculosis%26date%3D2015%26volume%3D95%26issue%3D2%26spage%3D142%26epage%3D148%26doi%3D10.1016%2Fj.tube.2014.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A new method to detect related function among proteins independent of sequence and fold homology</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">323</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">387</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)00811-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2802%2900811-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2002&pages=387-406&issue=2&author=S.+Schmittauthor=D.+Kuhnauthor=G.+Klebe&title=A+new+method+to+detect+related+function+among+proteins+independent+of+sequence+and+fold+homology&doi=10.1016%2FS0022-2836%2802%2900811-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900811-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900811-2%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520new%2520method%2520to%2520detect%2520related%2520function%2520among%2520proteins%2520independent%2520of%2520sequence%2520and%2520fold%2520homology%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D323%26issue%3D2%26spage%3D387%26epage%3D406%26doi%3D10.1016%2FS0022-2836%2802%2900811-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Yang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudock, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, L.</span><span> </span><span class="NLM_article-title">Identifying unexpected therapeutic targets via chemical-protein interactome</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">e9568</span><span class="refDoi"> DOI: 10.1371/journal.pone.0009568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0009568" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e9568&issue=3&author=L.+Yangauthor=J.+Chenauthor=L.+Shiauthor=M.+P.+Hudockauthor=K.+Wangauthor=L.+He&title=Identifying+unexpected+therapeutic+targets+via+chemical-protein+interactome&doi=10.1371%2Fjournal.pone.0009568"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0009568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0009568%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DHudock%26aufirst%3DM.%2BP.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DL.%26atitle%3DIdentifying%2520unexpected%2520therapeutic%2520targets%2520via%2520chemical-protein%2520interactome%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26issue%3D3%26spage%3De9568%26doi%3D10.1371%2Fjournal.pone.0009568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Al-Gharabli, S. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S. T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesters, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilgenfeld, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rademann, J.</span><span> </span><span class="NLM_article-title">An efficient method for the synthesis of peptide aldehyde libraries employed in the discovery of reversible SARS coronavirus main protease (SARS-CoV Mpro) inhibitors</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">1048</span><span class="NLM_x">–</span> <span class="NLM_lpage">1055</span><span class="refDoi"> DOI: 10.1002/cbic.200500533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fcbic.200500533" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=1048-1055&issue=7&author=S.+I.+Al-Gharabliauthor=S.+T.+A.+Shahauthor=S.+Weikauthor=M.+F.+Schmidtauthor=J.+R.+Mestersauthor=D.+Kuhnauthor=G.+Klebeauthor=R.+Hilgenfeldauthor=J.+Rademann&title=An+efficient+method+for+the+synthesis+of+peptide+aldehyde+libraries+employed+in+the+discovery+of+reversible+SARS+coronavirus+main+protease+%28SARS-CoV+Mpro%29+inhibitors&doi=10.1002%2Fcbic.200500533"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200500533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200500533%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Gharabli%26aufirst%3DS.%2BI.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DWeik%26aufirst%3DS.%26aulast%3DSchmidt%26aufirst%3DM.%2BF.%26aulast%3DMesters%26aufirst%3DJ.%2BR.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DHilgenfeld%26aufirst%3DR.%26aulast%3DRademann%26aufirst%3DJ.%26atitle%3DAn%2520efficient%2520method%2520for%2520the%2520synthesis%2520of%2520peptide%2520aldehyde%2520libraries%2520employed%2520in%2520the%2520discovery%2520of%2520reversible%2520SARS%2520coronavirus%2520main%2520protease%2520%2528SARS-CoV%2520Mpro%2529%2520inhibitors%26jtitle%3DChemBioChem%26date%3D2006%26volume%3D7%26issue%3D7%26spage%3D1048%26epage%3D1055%26doi%3D10.1002%2Fcbic.200500533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weskamp, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hüllermeier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">From the similarity analysis of protein cavities to the functional classification of protein families using cavbase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1023</span><span class="NLM_x">–</span> <span class="NLM_lpage">1044</span><span class="refDoi"> DOI: 10.1016/j.jmb.2006.04.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.jmb.2006.04.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2006&pages=1023-1044&issue=4&author=D.+Kuhnauthor=N.+Weskampauthor=S.+Schmittauthor=E.+H%C3%BCllermeierauthor=G.+Klebe&title=From+the+similarity+analysis+of+protein+cavities+to+the+functional+classification+of+protein+families+using+cavbase&doi=10.1016%2Fj.jmb.2006.04.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.04.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.04.024%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DWeskamp%26aufirst%3DN.%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DH%25C3%25BCllermeier%26aufirst%3DE.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFrom%2520the%2520similarity%2520analysis%2520of%2520protein%2520cavities%2520to%2520the%2520functional%2520classification%2520of%2520protein%2520families%2520using%2520cavbase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D359%26issue%3D4%26spage%3D1023%26epage%3D1044%26doi%3D10.1016%2Fj.jmb.2006.04.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Weber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Supuran, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scozzafava, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Unexpected nanomolar inhibition of carbonic anhydrase by COX-2-selective celecoxib: new pharmacological opportunities due to related binding site recognition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">557</span><span class="refDoi"> DOI: 10.1021/jm030912m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030912m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFGltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=550-557&issue=3&author=A.+Weberauthor=A.+Casiniauthor=A.+Heineauthor=D.+Kuhnauthor=C.+T.+Supuranauthor=A.+Scozzafavaauthor=G.+Klebe&title=Unexpected+nanomolar+inhibition+of+carbonic+anhydrase+by+COX-2-selective+celecoxib%3A+new+pharmacological+opportunities+due+to+related+binding+site+recognition&doi=10.1021%2Fjm030912m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Unexpected Nanomolar Inhibition of Carbonic Anhydrase by COX-2-Selective Celecoxib: New Pharmacological Opportunities Due to Related Binding Site Recognition</span></div><div class="casAuthors">Weber, Alexander; Casini, Angela; Heine, Andreas; Kuhn, Daniel; Supuran, Claudiu T.; Scozzafava, Andrea; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">550-557</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By optimizing binding to a selected target protein, modern drug research strives to develop safe and efficacious agents for the treatment of disease.  Selective drug action is intended to minimize undesirable side effects from scatter pharmacol.  Celecoxib (Celebrex), valdecoxib (Bextra), and rofecoxib (Vioxx) are nonsteroidal antiinflammatory drugs (NSAIDs) due to selective inhibition of inducible cyclooxygenase COX-2 while sparing inhibition of constitutive COX-1.  While rofecoxib contains a Me sulfone constituent, celecoxib and valdecoxib possess an unsubstituted arylsulfonamide moiety.  The latter group is common to many carbonic anhydrase (CA) inhibitors.  Using enzyme kinetics and X-ray crystallog., we demonstrate an unexpected nanomolar affinity of the COX-2 specific arylsulfonamide-type celecoxib and valdecoxib for isoenzymes of the totally unrelated carbonic anhydrase (CA) family, such as CA I, II, IV, and IX, whereas the rofecoxib Me sulfone-type has no effect.  When administered orally to glaucomatous rabbits, celecoxib and valdecoxib lowered intraocular pressure, suggesting that these agents may have utility in the treatment of this disorder.  The crystal structure of celecoxib in complex with CA II reveals part of this inhibition to be mediated via binding of the sulfonamide group to the catalytic zinc of CA II.  To investigate the structural basis for cross-reactivity of these compds. between COX-2 and CA II, we compared the mol. recognition properties of both protein binding pockets in terms of local physicochem. similarities among binding site-exposed amino acids accommodating different portions of the drug mols.  Our approach Cavbase, implemented into Relibase, detects similarities between the sites, suggesting some potential to predict unexpected cross-reactivity of drugs among functionally unrelated target proteins.  The obsd. cross-reactivity with CAs may also contribute to differences in the pharmacol. profiles, in particular with respect to glaucoma and anticancer therapy and may suggest new opportunities of these COX-2 selective NSAIDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSHRdlhB-jn7Vg90H21EOLACvtfcHk0lhQDJS9Ow5IDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFGltLc%253D&md5=608b04daff89563144029daea8bd91b7</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm030912m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030912m%26sid%3Dliteratum%253Aachs%26aulast%3DWeber%26aufirst%3DA.%26aulast%3DCasini%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DSupuran%26aufirst%3DC.%2BT.%26aulast%3DScozzafava%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DUnexpected%2520nanomolar%2520inhibition%2520of%2520carbonic%2520anhydrase%2520by%2520COX-2-selective%2520celecoxib%253A%2520new%2520pharmacological%2520opportunities%2520due%2520to%2520related%2520binding%2520site%2520recognition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D3%26spage%3D550%26epage%3D557%26doi%3D10.1021%2Fjm030912m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Li, G.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.-F.</span><span> </span><span class="NLM_article-title">CMASA: an accurate algorithm for detecting local protein structural similarity and its application to enzyme catalytic site annotation</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="refDoi"> DOI: 10.1186/1471-2105-11-439</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-11-439" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=20796320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BC3cfht1ylsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=439&author=G.-H.+Liauthor=J.-F.+Huang&title=CMASA%3A+an+accurate+algorithm+for+detecting+local+protein+structural+similarity+and+its+application+to+enzyme+catalytic+site+annotation&doi=10.1186%2F1471-2105-11-439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">CMASA: an accurate algorithm for detecting local protein structural similarity and its application to enzyme catalytic site annotation</span></div><div class="casAuthors">Li Gong-Hua; Huang Jing-Fei</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">439</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The rapid development of structural genomics has resulted in many "unknown function" proteins being deposited in Protein Data Bank (PDB), thus, the functional prediction of these proteins has become a challenge for structural bioinformatics.  Several sequence-based and structure-based methods have been developed to predict protein function, but these methods need to be improved further, such as, enhancing the accuracy, sensitivity, and the computational speed.  Here, an accurate algorithm, the CMASA (Contact MAtrix based local Structural Alignment algorithm), has been developed to predict unknown functions of proteins based on the local protein structural similarity.  This algorithm has been evaluated by building a test set including 164 enzyme families, and also been compared to other methods.  RESULTS:  The evaluation of CMASA shows that the CMASA is highly accurate (0.96), sensitive (0.86), and fast enough to be used in the large-scale functional annotation.  Comparing to both sequence-based and global structure-based methods, not only the CMASA can find remote homologous proteins, but also can find the active site convergence.  Comparing to other local structure comparison-based methods, the CMASA can obtain the better performance than both FFF (a method using geometry to predict protein function) and SPASM (a local structure alignment method); and the CMASA is more sensitive than PINTS and is more accurate than JESS (both are local structure alignment methods).  The CMASA was applied to annotate the enzyme catalytic sites of the non-redundant PDB, and at least 166 putative catalytic sites have been suggested, these sites can not be observed by the Catalytic Site Atlas (CSA).  CONCLUSIONS:  The CMASA is an accurate algorithm for detecting local protein structural similarity, and it holds several advantages in predicting enzyme active sites.  The CMASA can be used in large-scale enzyme active site annotation.  The CMASA can be available by the mail-based server (http://159.226.149.45/other1/CMASA/CMASA.htm).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRHQqnT9KEcsgvupmecNw_lfW6udTcc2eZL94h4dH7NUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfht1ylsA%253D%253D&md5=08ddacbde64685046c8a2c7a71ada06b</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-439%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-H.%26aulast%3DHuang%26aufirst%3DJ.-F.%26atitle%3DCMASA%253A%2520an%2520accurate%2520algorithm%2520for%2520detecting%2520local%2520protein%2520structural%2520similarity%2520and%2520its%2520application%2520to%2520enzyme%2520catalytic%2520site%2520annotation%26jtitle%3DBMC%2520Bioinf.%26date%3D2010%26volume%3D11%26spage%3D439%26doi%3D10.1186%2F1471-2105-11-439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span> </span><span class="NLM_article-title">A comparative analysis of protein targets of withdrawn cardiovascular drugs in human and mouse</span> <span class="citation_source-journal">J. Clin. Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">10</span><span class="refDoi"> DOI: 10.1186/2043-9113-2-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F2043-9113-2-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22548699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOiurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=10&issue=1&author=Y.+Zhaoauthor=J.+Wangauthor=Y.+Wangauthor=J.+Huang&title=A+comparative+analysis+of+protein+targets+of+withdrawn+cardiovascular+drugs+in+human+and+mouse&doi=10.1186%2F2043-9113-2-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A comparative analysis of protein targets of withdrawn cardiovascular drugs in human and mouse</span></div><div class="casAuthors">Zhao, Yuqi; Wang, Jingwen; Wang, Yanjie; Huang, Jingfei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>CODEN:
                <span class="NLM_cas:coden">JCBOBR</span>;
        ISSN:<span class="NLM_cas:issn">2043-9113</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Mouse is widely used in animal testing of cardiovascular disease.  However, a large no. of cardiovascular drugs that have been exptl. proved to work well on mouse were withdrawn because they caused adverse side effects in human.  In this study, we investigate whether binding patterns of withdrawn cardiovascular drugs are conserved between mouse and human through computational dockings and mol. dynamic simulations.  In addn., we also measured the level of conservation of gene expression patterns of the drug targets and their interacting partners by analyzing the microarray data.  The results show that target proteins of withdrawn cardiovascular drugs are functionally conserved between human and mouse.  However, all the binding patterns of withdrawn drugs we retrieved show striking difference due to sequence divergence in drug-binding pocket, mainly through loss or gain of hydrogen bond donors and distinct drug-binding pockets.  The binding affinities of withdrawn drugs to their receptors tend to be reduced from mouse to human.  In contrast, the FDA-approved and best-selling drugs are little affected.  Our anal. suggests that sequence divergence in drug-binding pocket may be a reasonable explanation for the discrepancy of drug effects between animal models and human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKl0p5QoPUGLVg90H21EOLACvtfcHk0li96Sye43dB4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOiurvI&md5=683ea56930f7baaea685d327b2efa92d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1186%2F2043-9113-2-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2043-9113-2-10%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DJ.%26atitle%3DA%2520comparative%2520analysis%2520of%2520protein%2520targets%2520of%2520withdrawn%2520cardiovascular%2520drugs%2520in%2520human%2520and%2520mouse%26jtitle%3DJ.%2520Clin.%2520Bioinf.%26date%3D2012%26volume%3D2%26issue%3D1%26spage%3D10%26doi%3D10.1186%2F2043-9113-2-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Roy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">COFACTOR: an accurate comparative algorithm for structure-based protein function annotation</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x"> (</span><span class="NLM_issue">W1</span><span class="NLM_x">) </span> <span class="NLM_fpage">W471</span><span class="NLM_x">–</span> <span class="NLM_lpage">W477</span><div class="note"><p class="first last">(Web Server issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gks372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgks372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22570420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCrtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=W471-W477&issue=W1&author=A.+Royauthor=J.+Yangauthor=Y.+Zhang&title=COFACTOR%3A+an+accurate+comparative+algorithm+for+structure-based+protein+function+annotation&doi=10.1093%2Fnar%2Fgks372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">COFACTOR: an accurate comparative algorithm for structure-based protein function annotation</span></div><div class="casAuthors">Roy, Ambrish; Yang, Jianyi; Zhang, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W471-W477</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have developed a new COFACTOR webserver for automated structure-based protein function annotation.  Starting from a structural model, given by either exptl. detn. or computational modeling, COFACTOR first identifies template proteins of similar folds and functional sites by threading the target structure through three representative template libraries that have known protein-ligand binding interactions, Enzyme Commission no. or Gene Ontol. terms.  The biol. function insights in these three aspects are then deduced from the functional templates, the confidence of which is evaluated by a scoring function that combines both global and local structural similarities.  The algorithm has been extensively benchmarked by large-scale benchmarking tests and demonstrated significant advantages compared to traditional sequence-based methods.  In the recent community-wide CASP9 expt., COFACTOR was ranked as the best method for protein-ligand binding site predictions.  The COFACTOR sever and the template libraries are freely available at http://zhanglab.ccmb.med.umich.edu/COFACTOR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmrygdjGKk-LVg90H21EOLACvtfcHk0li96Sye43dB4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCrtL0%253D&md5=174b2a49ad059bf00b318d2a96b7a8aa</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks372%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DCOFACTOR%253A%2520an%2520accurate%2520comparative%2520algorithm%2520for%2520structure-based%2520protein%2520function%2520annotation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DW1%26spage%3DW471%26epage%3DW477%26doi%3D10.1093%2Fnar%2Fgks372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Dong, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omenn, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Structural bioinformatics inspection of neXtProt PE5 proteins in the human proteome</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">3750</span><span class="NLM_x">–</span> <span class="NLM_lpage">3761</span><span class="refDoi"> DOI: 10.1021/acs.jproteome.5b00516</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.5b00516" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2msbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=3750-3761&issue=9&author=Q.+Dongauthor=R.+Menonauthor=G.+S.+Omennauthor=Y.+Zhang&title=Structural+bioinformatics+inspection+of+neXtProt+PE5+proteins+in+the+human+proteome&doi=10.1021%2Facs.jproteome.5b00516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Bioinformatics Inspection of neXtProt PE5 Proteins in the Human Proteome</span></div><div class="casAuthors">Dong, Qiwen; Menon, Rajasree; Omenn, Gilbert S.; Zhang, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3750-3761</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One goal of the Human Proteome Project is to identify at least one protein product for each of the ∼20,000 human protein-coding genes.  As of Oct. 2014, however, there are 3564 genes (18%) that have no or insufficient evidence of protein existence (PE), as curated by neXtProt; these comprise 2647 PE2-4 missing proteins and 616 PE5 dubious protein entries.  The authors conducted a systematic examn. of the 616 PE5 protein entries using cutting-edge protein structure and function modeling methods.  Compared to a random sample of high-confidence PE1 proteins, the putative PE5 proteins are over-represented in the membrane and cell surface proteins and peptides fold families.  Detailed functional analyses show that most PE5 proteins, if expressed, would belong to transporters and receptors localized in the plasma membrane compartment.  The results suggest that exptl. difficulty in identifying membrane-bound proteins and peptides could have precluded their detection in mass spectrometry and that special enrichment techniques with improved sensitivity for membrane proteins could be important for the characterization of the PE5 "dark matter" of the human proteome.  Finally, the authors identify 66 high scoring PE5 protein entries and find that six of them are reported in recent mass spectrometry databases; an illustrative annotation of these six is provided.  This work illustrates a new approach to examine the potential folding and function of the dubious proteins comprising PE5, which the authors will next apply to the far larger group of missing proteins comprising PE2-4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgK1Q6ycr73bVg90H21EOLACvtfcHk0li96Sye43dB4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2msbzK&md5=0dd45b807344b901fdec7c1b26b0af1a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.5b00516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.5b00516%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DQ.%26aulast%3DMenon%26aufirst%3DR.%26aulast%3DOmenn%26aufirst%3DG.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DStructural%2520bioinformatics%2520inspection%2520of%2520neXtProt%2520PE5%2520proteins%2520in%2520the%2520human%2520proteome%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2015%26volume%3D14%26issue%3D9%26spage%3D3750%26epage%3D3761%26doi%3D10.1021%2Facs.jproteome.5b00516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Powers, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercier, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramanathan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revesz, P.</span><span> </span><span class="NLM_article-title">Comparison of protein active site structures for functional annotation of proteins and drug design</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1002/prot.21092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.21092" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=124-135&issue=1&author=R.+Powersauthor=J.+C.+Copelandauthor=K.+Germerauthor=K.+A.+Mercierauthor=V.+Ramanathanauthor=P.+Revesz&title=Comparison+of+protein+active+site+structures+for+functional+annotation+of+proteins+and+drug+design&doi=10.1002%2Fprot.21092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fprot.21092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21092%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DJ.%2BC.%26aulast%3DGermer%26aufirst%3DK.%26aulast%3DMercier%26aufirst%3DK.%2BA.%26aulast%3DRamanathan%26aufirst%3DV.%26aulast%3DRevesz%26aufirst%3DP.%26atitle%3DComparison%2520of%2520protein%2520active%2520site%2520structures%2520for%2520functional%2520annotation%2520of%2520proteins%2520and%2520drug%2520design%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D65%26issue%3D1%26spage%3D124%26epage%3D135%26doi%3D10.1002%2Fprot.21092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Shortridge, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, R.</span><span> </span><span class="NLM_article-title">Structural and functional similarity between the bacterial type III secretion system needle protein PrgI and the eukaryotic apoptosis Bcl-2 proteins</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">e7442</span><span class="refDoi"> DOI: 10.1371/journal.pone.0007442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0007442" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e7442&issue=10&author=M.+D.+Shortridgeauthor=R.+Powers&title=Structural+and+functional+similarity+between+the+bacterial+type+III+secretion+system+needle+protein+PrgI+and+the+eukaryotic+apoptosis+Bcl-2+proteins&doi=10.1371%2Fjournal.pone.0007442"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0007442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0007442%26sid%3Dliteratum%253Aachs%26aulast%3DShortridge%26aufirst%3DM.%2BD.%26aulast%3DPowers%26aufirst%3DR.%26atitle%3DStructural%2520and%2520functional%2520similarity%2520between%2520the%2520bacterial%2520type%2520III%2520secretion%2520system%2520needle%2520protein%2520PrgI%2520and%2520the%2520eukaryotic%2520apoptosis%2520Bcl-2%2520proteins%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26issue%3D10%26spage%3De7442%26doi%3D10.1371%2Fjournal.pone.0007442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Kinoshita, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furui, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Identification of protein functions from a molecular surface database, eF-site</span> <span class="citation_source-journal">J. Struct. Funct. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">9</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1023/A:1011318527094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1023%2FA%3A1011318527094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=12836670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtFaqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=9-22&issue=1&author=K.+Kinoshitaauthor=J.+Furuiauthor=H.+Nakamura&title=Identification+of+protein+functions+from+a+molecular+surface+database%2C+eF-site&doi=10.1023%2FA%3A1011318527094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of protein functions from a molecular surface database, eF-site</span></div><div class="casAuthors">Kinoshita, Kengo; Furui, Jun'ichi; Nakamura, Haruki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural and Functional Genomics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-22</span>CODEN:
                <span class="NLM_cas:coden">JSFGAW</span>;
        ISSN:<span class="NLM_cas:issn">1345-711X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A bioinformatics method was developed to identify the protein surface around the functional site and to est. the biochem. function, using a newly constructed mol. surface database named the eF-site (electrostatic surface of Functional site.).  Mol. surfaces of protein mols. were computed based on the atom coordinates, and the eF-site database was prepd. by adding the phys. properties on the constructed mol. surfaces.  The electrostatic potential on each mol. surface was individually calcd. solving the Poisson-Boltzmann equation numerically for the precise continuum model, and the hydrophobicity information of each residue was also included.  The eF-site database is accessed by the internet.  We have prepd. four different databases, eF-site/antibody, eF-site/prosite, eF-site/P-site, and eF-site/ActiveSite, corresponding to the antigen binding sites of antibodies with the same orientations, the mol. surfaces for the individual motifs in PROSITE database, the phosphate binding sites, and the active site surfaces for the representatives of the individual protein family, resp.  An algorithm using the clique detection method as an applied graph theory was developed to search of the eF-site database, so as to recognize and discriminate the characteristic mol. surfaces of the proteins.  The method identifies the active site having the similar function to those of the known proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7LC-mYJpiR7Vg90H21EOLACvtfcHk0livXy5wdfBN0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtFaqsw%253D%253D&md5=8d7cb21e099b8f48ff35fb77225119ec</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1023%2FA%3A1011318527094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1011318527094%26sid%3Dliteratum%253Aachs%26aulast%3DKinoshita%26aufirst%3DK.%26aulast%3DFurui%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520protein%2520functions%2520from%2520a%2520molecular%2520surface%2520database%252C%2520eF-site%26jtitle%3DJ.%2520Struct.%2520Funct.%2520Genomics%26date%3D2002%26volume%3D2%26issue%3D1%26spage%3D9%26epage%3D22%26doi%3D10.1023%2FA%3A1011318527094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Kinoshita, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Identification of protein biochemical functions by similarity search using the molecular surface database eF-site</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1589</span><span class="NLM_x">–</span> <span class="NLM_lpage">1595</span><span class="refDoi"> DOI: 10.1110/ps.0368703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1110%2Fps.0368703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=12876308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvFSjtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=1589-1595&issue=8&author=K.+Kinoshitaauthor=H.+Nakamura&title=Identification+of+protein+biochemical+functions+by+similarity+search+using+the+molecular+surface+database+eF-site&doi=10.1110%2Fps.0368703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of protein biochemical functions by similarity search using the molecular surface database eF-site</span></div><div class="casAuthors">Kinoshita, Kengo; Nakamura, Haruki</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1589-1595</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The identification of protein biochem. functions based on their three-dimensional structures is strongly required in the post-genome-sequencing era.  We have developed a new method to identify and predict protein biochem. functions using the similarity information of mol. surface geometries and electrostatic potentials on the surfaces.  Our prediction system consists of a similarity search method based on a clique search algorithm and the mol. surface database eF-site (electrostatic surface of functional-site in proteins).  Using this system, functional sites similar to those of phosphoenoylpyruvate carboxy kinase were detected in several mononucleotide-binding proteins, which have different folds.  We also applied our method to a hypothetical protein, MJ0226 from Methanococcus jannaschii, and detected the mononucleotide binding site from the similarity to other proteins having different folds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojqDyJFOxO4bVg90H21EOLACvtfcHk0livXy5wdfBN0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvFSjtb4%253D&md5=4bc8bd3eb49558b44273487d08680215</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1110%2Fps.0368703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.0368703%26sid%3Dliteratum%253Aachs%26aulast%3DKinoshita%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520protein%2520biochemical%2520functions%2520by%2520similarity%2520search%2520using%2520the%2520molecular%2520surface%2520database%2520eF-site%26jtitle%3DProtein%2520Sci.%26date%3D2003%26volume%3D12%26issue%3D8%26spage%3D1589%26epage%3D1595%26doi%3D10.1110%2Fps.0368703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Baroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciabola, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perruccio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span> </span><span class="NLM_article-title">A common reference framework for analyzing/comparing proteins and ligands. Fingerprints for Ligands and Proteins (FLAP): theory and application</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">279</span><span class="NLM_x">–</span> <span class="NLM_lpage">294</span><span class="refDoi"> DOI: 10.1021/ci600253e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600253e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtVagsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=279-294&issue=2&author=M.+Baroniauthor=G.+Crucianiauthor=S.+Sciabolaauthor=F.+Perruccioauthor=J.+S.+Mason&title=A+common+reference+framework+for+analyzing%2Fcomparing+proteins+and+ligands.+Fingerprints+for+Ligands+and+Proteins+%28FLAP%29%3A+theory+and+application&doi=10.1021%2Fci600253e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A common reference framework for analyzing/comparing proteins and ligands. Fingerprints for ligands and proteins (FLAP): theory and application</span></div><div class="casAuthors">Baroni, Massimo; Cruciani, Gabriele; Sciabola, Simone; Perruccio, Francesca; Mason, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-294</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast new algorithm (fingerprints for ligands and proteins or FLAP) able to describe small mols. and protein structures using a common ref. framework of four-point pharmacophore fingerprints and a mol.-cavity shape is described in detail.  The procedure starts by using the GRID force field to calc. mol. interaction fields, which are then used to identify particular target locations where an energetic interaction with small mol. features would be very favorable.  The target points thus calcd. are then used by FLAP to build all possible four-point pharmacophores present in the given target site.  A related approach can be applied to small mols., using directly the GRID atom types to identify pharmacophoric features, and this complementary description of the target and ligand then leads to several novel applications.  FLAP can be used for selectivity studies or similarity analyses in order to compare macromols. without superposing them.  Protein families can be compared and clustered into target classes, without bias from previous knowledge and without requiring protein superposition, alignment, or knowledge-based comparison.  FLAP can be used effectively for ligand-based virtual screening and structure-based virtual screening, with the pharmacophore mol. recognition.  Finally, the new method can calc. descriptors for chemometric anal. and can initiate a docking procedure.  This paper presents the background to the new procedure and includes case studies illustrating several relevant applications of the new approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppMc0fYDNn7LVg90H21EOLACvtfcHk0livXy5wdfBN0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtVagsA%253D%253D&md5=9facb99c5741c7158afab7a3ac387ec3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci600253e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600253e%26sid%3Dliteratum%253Aachs%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DSciabola%26aufirst%3DS.%26aulast%3DPerruccio%26aufirst%3DF.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DA%2520common%2520reference%2520framework%2520for%2520analyzing%252Fcomparing%2520proteins%2520and%2520ligands.%2520Fingerprints%2520for%2520Ligands%2520and%2520Proteins%2520%2528FLAP%2529%253A%2520theory%2520and%2520application%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D2%26spage%3D279%26epage%3D294%26doi%3D10.1021%2Fci600253e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Sciabola, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flocco, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perruccio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span> </span><span class="NLM_article-title">High-throughput virtual screening of proteins using GRID molecular interaction fields</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">155</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1021/ci9003317</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9003317" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsValurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=155-169&issue=1&author=S.+Sciabolaauthor=R.+V.+Stantonauthor=J.+E.+Millsauthor=M.+M.+Floccoauthor=M.+Baroniauthor=G.+Crucianiauthor=F.+Perruccioauthor=J.+S.+Mason&title=High-throughput+virtual+screening+of+proteins+using+GRID+molecular+interaction+fields&doi=10.1021%2Fci9003317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening of Proteins Using GRID Molecular Interaction Fields</span></div><div class="casAuthors">Sciabola, Simone; Stanton, Robert V.; Mills, James E.; Flocco, Maria M.; Baroni, Massimo; Cruciani, Gabriele; Perruccio, Francesca; Mason, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">155-169</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new computational algorithm for protein binding sites characterization and comparison has been developed, which uses a common ref. framework of the projected ligand-space four-point pharmacophore fingerprints, includes cavity shape, and can be used with diverse proteins as no structural alignment is required.  Protein binding sites are first described using GRID mol. interaction fields (GRID-MIFs), and the FLAP (fingerprints for ligands and proteins) method is then used to encode and compare this information.  The discriminating power of the algorithm and its applicability for large-scale protein anal. was validated by analyzing various scenarios: clustering of kinase protein families in a relevant manner, predicting ligand activity across related targets, and protein-protein virtual screening.  In all cases the results showed the effectiveness of the GRID-FLAP method and its potential use in applications such as identifying selectivity targets and tools/hits for new targets via the identification of other proteins with pharmacophorically similar binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6BvC5pevsZ7Vg90H21EOLACvtfcHk0liZ_EhprfudQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsValurvN&md5=f7b9f08b95e584728625eea3f308d305</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fci9003317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003317%26sid%3Dliteratum%253Aachs%26aulast%3DSciabola%26aufirst%3DS.%26aulast%3DStanton%26aufirst%3DR.%2BV.%26aulast%3DMills%26aufirst%3DJ.%2BE.%26aulast%3DFlocco%26aufirst%3DM.%2BM.%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DPerruccio%26aufirst%3DF.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520of%2520proteins%2520using%2520GRID%2520molecular%2520interaction%2520fields%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D1%26spage%3D155%26epage%3D169%26doi%3D10.1021%2Fci9003317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Tseng, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.-H.</span><span> </span><span class="NLM_article-title">fPOP: footprinting functional pockets of proteins by comparative spatial patterns</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D288</span><span class="NLM_x">–</span> <span class="NLM_lpage">D295</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkp900</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkp900" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D288-D295&issue=Suppl.+1&author=Y.+Y.+Tsengauthor=Z.+J.+Chenauthor=W.-H.+Li&title=fPOP%3A+footprinting+functional+pockets+of+proteins+by+comparative+spatial+patterns&doi=10.1093%2Fnar%2Fgkp900"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp900%26sid%3Dliteratum%253Aachs%26aulast%3DTseng%26aufirst%3DY.%2BY.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26aulast%3DLi%26aufirst%3DW.-H.%26atitle%3DfPOP%253A%2520footprinting%2520functional%2520pockets%2520of%2520proteins%2520by%2520comparative%2520spatial%2520patterns%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26issue%3DSuppl.%25201%26spage%3DD288%26epage%3DD295%26doi%3D10.1093%2Fnar%2Fgkp900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Tseng, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.-H.</span><span> </span><span class="NLM_article-title">Classification of protein functional surfaces using structural characteristics</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1170</span><span class="NLM_x">–</span> <span class="NLM_lpage">1175</span><span class="refDoi"> DOI: 10.1073/pnas.1119684109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1073%2Fpnas.1119684109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=1170-1175&issue=4&author=Y.+Y.+Tsengauthor=W.-H.+Li&title=Classification+of+protein+functional+surfaces+using+structural+characteristics&doi=10.1073%2Fpnas.1119684109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1119684109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1119684109%26sid%3Dliteratum%253Aachs%26aulast%3DTseng%26aufirst%3DY.%2BY.%26aulast%3DLi%26aufirst%3DW.-H.%26atitle%3DClassification%2520of%2520protein%2520functional%2520surfaces%2520using%2520structural%2520characteristics%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D4%26spage%3D1170%26epage%3D1175%26doi%3D10.1073%2Fpnas.1119684109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">207</span><span class="refDoi"> DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&issue=1&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0liZ_EhprfudQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">Structure-based prediction of G-protein-coupled receptor ligand function: a β-adrenoceptor case study</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1045</span><span class="NLM_x">–</span> <span class="NLM_lpage">1061</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFenuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1045-1061&issue=5&author=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+P.+de+Eschauthor=C.+de+Graaf&title=Structure-based+prediction+of+G-protein-coupled+receptor+ligand+function%3A+a+%CE%B2-adrenoceptor+case+study&doi=10.1021%2Facs.jcim.5b00066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based prediction of G-protein-coupled receptor ligand function: A β-adrenoceptor case study</span></div><div class="casAuthors">Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1045-1061</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The spectacular advances in G-protein-coupled receptor (GPCR) structure detn. have opened up new possibilities for structure-based GPCR ligand discovery.  The structure-based prediction of whether a ligand stimulates (full/partial agonist), blocks (antagonist), or reduces (inverse agonist) GPCR signaling activity is, however, still challenging.  A total of 31 β1 (β1R) and β2 (β2R) adrenoceptor crystal structures, including antagonist, inverse agonist, and partial/full agonist-bound structures, allowed the authors to explore the possibilities and limitations of structure-based prediction of GPCR ligand function.  The authors used all unique protein-ligand interaction fingerprints (IFPs) derived from all ligand-bound β-adrenergic crystal structure monomers to post-process the docking poses of known β1R/β2R partial/full agonists, antagonists/inverse agonists, and physicochem. similar decoys in each of the β1R/β2R structures.  The systematic anal. of these 1920 unique IFP-structure combinations offered new insights into the relative impact of protein conformation and IFP scoring on selective virtual screening (VS) for ligands with a specific functional effect.  The authors' studies show that ligands with the same function can be efficiently classified on the basis of their protein-ligand interaction profile.  Small differences between the receptor conformation (used for docking) and ref. IFP (used for scoring of the docking poses) det., however, the enrichment of specific ligand types in VS hit lists.  Interestingly, the selective enrichment of partial/full agonists can be achieved by using agonist IFPs to post-process docking poses in agonist-bound as well as antagonist-bound structures.  The authors have identified optimal structure-IFP combinations for the identification and discrimination of antagonists/inverse agonist and partial/full agonists, and defined a predicted IFP for the small full agonist norepinephrine that gave the highest retrieval rate of agonists over antagonists for all structures (with an enrichment factor of 46 for agonists and 8 for antagonists on av. at a 1% false-pos. rate).  This β-adrenoceptor case study provides new insights into the opportunities for selective structure-based discovery of GPCR ligands with a desired function and emphasizes the importance of IFPs in scoring docking poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV5NA6pChaObVg90H21EOLACvtfcHk0liZ_EhprfudQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFenuro%253D&md5=cc51b89775d0e5443d193b2527741bca</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00066%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DStructure-based%2520prediction%2520of%2520G-protein-coupled%2520receptor%2520ligand%2520function%253A%2520a%2520%25CE%25B2-adrenoceptor%2520case%2520study%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D5%26spage%3D1045%26epage%3D1061%26doi%3D10.1021%2Facs.jcim.5b00066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Najmanovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurbatova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J.</span><span> </span><span class="NLM_article-title">Detection of 3D atomic similarities and their use in the discrimination of small molecule protein-binding sites</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x"> (</span><span class="NLM_issue">16</span><span class="NLM_x">) </span> <span class="NLM_fpage">i105</span><span class="NLM_x">–</span> <span class="NLM_lpage">i111</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btn263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtn263" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=i105-i111&issue=16&author=R.+Najmanovichauthor=N.+Kurbatovaauthor=J.+Thornton&title=Detection+of+3D+atomic+similarities+and+their+use+in+the+discrimination+of+small+molecule+protein-binding+sites&doi=10.1093%2Fbioinformatics%2Fbtn263"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtn263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtn263%26sid%3Dliteratum%253Aachs%26aulast%3DNajmanovich%26aufirst%3DR.%26aulast%3DKurbatova%26aufirst%3DN.%26aulast%3DThornton%26aufirst%3DJ.%26atitle%3DDetection%2520of%25203D%2520atomic%2520similarities%2520and%2520their%2520use%2520in%2520the%2520discrimination%2520of%2520small%2520molecule%2520protein-binding%2520sites%26jtitle%3DBioinformatics%26date%3D2008%26volume%3D24%26issue%3D16%26spage%3Di105%26epage%3Di111%26doi%3D10.1093%2Fbioinformatics%2Fbtn263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Larocque, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chénard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R. A.</span><span> </span><span class="NLM_article-title">A curated C. difficile strain 630 metabolic network: prediction of essential targets and inhibitors</span> <span class="citation_source-journal">BMC Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">117</span><span class="refDoi"> DOI: 10.1186/s12918-014-0117-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2Fs12918-014-0117-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25315994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFWhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=117&author=M.+Larocqueauthor=T.+Ch%C3%A9nardauthor=R.+A.+Najmanovich&title=A+curated+C.+difficile+strain+630+metabolic+network%3A+prediction+of+essential+targets+and+inhibitors&doi=10.1186%2Fs12918-014-0117-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">A curated C. difficile strain 630 metabolic network: prediction of essential targets and inhibitors</span></div><div class="casAuthors">Larocque, Mathieu; Chenard, Thierry; Najmanovich, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">BMC Systems Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">117/1-117/17</span>CODEN:
                <span class="NLM_cas:coden">BSBMCC</span>;
        ISSN:<span class="NLM_cas:issn">1752-0509</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background:Clostridium difficile is the leading cause of hospital-borne infections occurring when the natural intestinal flora is depleted following antibiotic treatment.  Current treatments for Clostridium difficile infections present high relapse rates and new hyper-virulent and multi-resistant strains are emerging, making the study of this nosocomial pathogen necessary to find novel therapeutic targets.  Results: We present iMLTC806cdf, an extensively curated reconstructed metabolic network for the C. difficile pathogenic strain 630. iMLTC806cdf contains 806 genes, 703 metabolites and 769 metabolic, 117 exchange and 145 transport reactions. iMLTC806cdf is the most complete and accurate metabolic reconstruction of a gram-pos. anaerobic bacteria to date.  We validate the model with simulated growth assays in different media and carbon sources and use it to predict essential genes.  We obtain 89.2% accuracy in the prediction of gene essentiality when compared to exptl. data for B. subtilis homologs (the closest organism for which such data exists).  We predict the existence of 76 essential genes and 39 essential gene pairs, a no. of which are unique to C. difficile and have non-existing or predicted non-essential human homologs.  For 29 of these potential therapeutic targets, we find 125 inhibitors of homologous proteins including approved drugs with the potential for drug repositioning, that when validated exptl. could serve as starting points in the development of new antibiotics.  Conclusions: We created a highly curated metabolic network model of C. difficile strain 630 and used it to predict essential genes as potential new therapeutic targets in the fight against Clostridium difficile infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1JjHlRdFNaLVg90H21EOLACvtfcHk0lhbSdahnodpjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFWhsA%253D%253D&md5=83a0d4ccc76523a40ecbcf2399792541</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1186%2Fs12918-014-0117-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12918-014-0117-z%26sid%3Dliteratum%253Aachs%26aulast%3DLarocque%26aufirst%3DM.%26aulast%3DCh%25C3%25A9nard%26aufirst%3DT.%26aulast%3DNajmanovich%26aufirst%3DR.%2BA.%26atitle%3DA%2520curated%2520C.%2520difficile%2520strain%2520630%2520metabolic%2520network%253A%2520prediction%2520of%2520essential%2520targets%2520and%2520inhibitors%26jtitle%3DBMC%2520Syst.%2520Biol.%26date%3D2014%26volume%3D8%26spage%3D117%26doi%3D10.1186%2Fs12918-014-0117-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Wood, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vlieg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagener, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritschel, T.</span><span> </span><span class="NLM_article-title">Pharmacophore fingerprint-based approach to binding site subpocket similarity and its application to bioisostere replacement</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">2031</span><span class="NLM_x">–</span> <span class="NLM_lpage">2043</span><span class="refDoi"> DOI: 10.1021/ci3000776</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3000776" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVylur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2031-2043&issue=8&author=D.+J.+Woodauthor=J.+de+Vliegauthor=M.+Wagenerauthor=T.+Ritschel&title=Pharmacophore+fingerprint-based+approach+to+binding+site+subpocket+similarity+and+its+application+to+bioisostere+replacement&doi=10.1021%2Fci3000776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore Fingerprint-Based Approach to Binding Site Subpocket Similarity and Its Application to Bioisostere Replacement</span></div><div class="casAuthors">Wood, David J.; Vlieg, Jacob de; Wagener, Markus; Ritschel, Tina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2031-2043</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bioisosteres have been defined as structurally different mols. or substructures that can form comparable intermol. interactions, and therefore, fragments that bind to similar protein structures exhibit a degree of bioisosterism.  We present KRIPO (Key Representation of Interaction in POckets): a new method for quantifying the similarities of binding site subpockets based on pharmacophore fingerprints.  The binding site fingerprints have been optimized to improve their performance for both intra- and interprotein family comparisons.  A range of attributes of the fingerprints was considered in the optimization, including the placement of pharmacophore features, whether or not the fingerprints are fuzzified, and the resoln. and complexity of the pharmacophore fingerprints (2-, 3-, and 4-point fingerprints).  Fuzzy 3-point pharmacophore fingerprints were found to represent the optimal balance between computational resource requirements and the identification of potential replacements.  The complete PDB was converted into a database comprising almost 300 000 optimized fingerprints of local binding sites together with their assocd. ligand fragments.  The value of the approach is demonstrated by application to two crystal structures from the Protein Data Bank: (1) a MAP kinase P38 structure in complex with a pyridinylimidazole inhibitor (1A9U) and (2) a complex of thrombin with melagatran (1K22).  Potentially valuable bioisosteric replacements for all subpockets of the two studied protein are identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRQTi8-dp_rVg90H21EOLACvtfcHk0lhbSdahnodpjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVylur%252FF&md5=f344aa0423546f08ea347327f56ef7cb</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fci3000776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3000776%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DD.%2BJ.%26aulast%3Dde%2BVlieg%26aufirst%3DJ.%26aulast%3DWagener%26aufirst%3DM.%26aulast%3DRitschel%26aufirst%3DT.%26atitle%3DPharmacophore%2520fingerprint-based%2520approach%2520to%2520binding%2520site%2520subpocket%2520similarity%2520and%2520its%2520application%2520to%2520bioisostere%2520replacement%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D8%26spage%3D2031%26epage%3D2043%26doi%3D10.1021%2Fci3000776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Schirris, T. J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritschel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herma Renkema, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willems</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peter, H. G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smeitink, J. A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russel, F. G. M.</span><span> </span><span class="NLM_article-title">Mitochondrial ADP/ATP exchange inhibition: a novel off-target mechanism underlying ibipinabant-induced myotoxicity</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">14533</span><span class="refDoi"> DOI: 10.1038/srep14533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fsrep14533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=26416158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFOrsrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=14533&author=T.+J.+J.+Schirrisauthor=T.+Ritschelauthor=G.+Herma+Renkemaauthor=+Willemsauthor=H.+G.+M.+Peterauthor=J.+A.+M.+Smeitinkauthor=F.+G.+M.+Russel&title=Mitochondrial+ADP%2FATP+exchange+inhibition%3A+a+novel+off-target+mechanism+underlying+ibipinabant-induced+myotoxicity&doi=10.1038%2Fsrep14533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Mitochondrial ADP/ATP exchange inhibition: a novel off-target mechanism underlying ibipinabant-induced myotoxicity</span></div><div class="casAuthors">Schirris, Tom J. J.; Ritschel, Tina; Renkema, G. Herma; Willems, Peter H. G. M.; Smeitink, Jan A. M.; Russel, Frans G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14533</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cannabinoid receptor 1 (CB1R) antagonists appear to be promising drugs for the treatment of obesity, however, serious side effects have hampered their clin. application.  Rimonabant, the first in class CB1R antagonist, was withdrawn from the market because of psychiatric side effects.  This has led to the search for more peripherally restricted CB1R antagonists, one of which is ibipinabant.  However, this 3,4-diarylpyrazoline deriv. showed muscle toxicity in a pre-clin. dog study with mitochondrial dysfunction.  Here, we studied the mol. mechanism by which ibipinabant induces mitochondrial toxicity.  We obsd. a strong cytotoxic potency of ibipinabant in C2C12 myoblasts.  Functional characterization of mitochondria revealed increased cellular reactive oxygen species generation and a decreased ATP prodn. capacity, without effects on the catalytic activities of mitochondrial enzyme complexes I-V or the complex specific-driven oxygen consumption.  Using in silico off-target prediction modeling, combined with in vitro validation in isolated mitochondria and mitoplasts, we identified adenine nucleotide translocase (ANT)-dependent mitochondrial ADP/ATP exchange as a novel mol. mechanism underlying ibipinabant-induced toxicity.  Minor structural modification of ibipinabant could abolish ANT inhibition leading to a decreased cytotoxic potency, as obsd. with the ibipinabant deriv. CB23.  Our results will be instrumental in the development of new types of safer CB1R antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF2M9vQEdYyrVg90H21EOLACvtfcHk0ljmUtukVFFA1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFOrsrrM&md5=ac01a690f1c65777e537dc597ad1e46f</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fsrep14533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep14533%26sid%3Dliteratum%253Aachs%26aulast%3DSchirris%26aufirst%3DT.%2BJ.%2BJ.%26aulast%3DRitschel%26aufirst%3DT.%26aulast%3DHerma%2BRenkema%26aufirst%3DG.%26aulast%3DWillems%26aulast%3DPeter%26aufirst%3DH.%2BG.%2BM.%26aulast%3DSmeitink%26aufirst%3DJ.%2BA.%2BM.%26aulast%3DRussel%26aufirst%3DF.%2BG.%2BM.%26atitle%3DMitochondrial%2520ADP%252FATP%2520exchange%2520inhibition%253A%2520a%2520novel%2520off-target%2520mechanism%2520underlying%2520ibipinabant-induced%2520myotoxicity%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D14533%26doi%3D10.1038%2Fsrep14533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Brakoulias, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Towards a structural classification of phosphate binding sites in protein-nucleotide complexes: an automated all-against-all structural comparison using geometric matching</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">250</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1002/prot.20123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.20123" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2004&pages=250-260&issue=2&author=A.+Brakouliasauthor=R.+M.+Jackson&title=Towards+a+structural+classification+of+phosphate+binding+sites+in+protein-nucleotide+complexes%3A+an+automated+all-against-all+structural+comparison+using+geometric+matching&doi=10.1002%2Fprot.20123"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1002%2Fprot.20123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20123%26sid%3Dliteratum%253Aachs%26aulast%3DBrakoulias%26aufirst%3DA.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DTowards%2520a%2520structural%2520classification%2520of%2520phosphate%2520binding%2520sites%2520in%2520protein-nucleotide%2520complexes%253A%2520an%2520automated%2520all-against-all%2520structural%2520comparison%2520using%2520geometric%2520matching%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D56%26issue%3D2%26spage%3D250%26epage%3D260%26doi%3D10.1002%2Fprot.20123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. M.</span><span> </span><span class="NLM_article-title">Binding site similarity analysis for the functional classification of the protein kinase family</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">318</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1021/ci800289y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800289y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=318-329&issue=2&author=S.+L.+Kinningsauthor=R.+M.+Jackson&title=Binding+site+similarity+analysis+for+the+functional+classification+of+the+protein+kinase+family&doi=10.1021%2Fci800289y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Similarity Analysis for the Functional Classification of the Protein Kinase Family</span></div><div class="casAuthors">Kinnings, Sarah L.; Jackson, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">318-329</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methods for analyzing complete gene families are becoming of increasing importance to the drug discovery process, because similarities and differences within a family are often the key to understanding functional differences that can be exploited in drug design.  We undertake a large-scale structural comparison of protein kinase ATP-binding sites using a geometric hashing method.  Subsequently, we propose a relevant classification of the protein kinase family based on the structural similarity of its binding sites.  Our classification is not only able to reveal the great diversity of different protein kinases and therefore their different potential for inhibitor selectivity, but it is also able to distinguish subtle differences within binding site conformation reflecting the protein activation state.  Furthermore, using exptl. inhibition profiling, we demonstrate that our classification can be used to identify protein kinase binding sites that are known exptl. to bind the same drug, demonstrating that it has potential as an inverse (protein) virtual screening tool, by identifying which other sites have the potential to bind a given drug.  In this way the cross-reactivities of the anticancer drugs Tarceva and Gleevec are rationalized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6kpf80r8s5LVg90H21EOLACvtfcHk0ljmUtukVFFA1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltF2qsg%253D%253D&md5=742343601ca1e3c0549fa4f383c2566b</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fci800289y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800289y%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DBinding%2520site%2520similarity%2520analysis%2520for%2520the%2520functional%2520classification%2520of%2520the%2520protein%2520kinase%2520family%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26issue%3D2%26spage%3D318%26epage%3D329%26doi%3D10.1021%2Fci800289y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Wu, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C.</span><span> </span><span class="NLM_article-title">A structural-alphabet-based strategy for finding structural motifs across protein families</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">e150</span><span class="refDoi"> DOI: 10.1093/nar/gkq478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkq478" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=e150&issue=14&author=C.+Y.+Wuauthor=Y.+C.+Chenauthor=C.+Lim&title=A+structural-alphabet-based+strategy+for+finding+structural+motifs+across+protein+families&doi=10.1093%2Fnar%2Fgkq478"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq478%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DC.%2BY.%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DLim%26aufirst%3DC.%26atitle%3DA%2520structural-alphabet-based%2520strategy%2520for%2520finding%2520structural%2520motifs%2520across%2520protein%2520families%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26issue%3D14%26spage%3De150%26doi%3D10.1093%2Fnar%2Fgkq478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Hua, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sargsyan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C.</span><span> </span><span class="NLM_article-title">Sequence-motif detection of NAD(P)-binding proteins: discovery of a unique antibacterial drug target</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">6471</span><span class="refDoi"> DOI: 10.1038/srep06471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fsrep06471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25253464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVygu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=6471&author=Y.+H.+Huaauthor=C.+Y.+Wuauthor=K.+Sargsyanauthor=C.+Lim&title=Sequence-motif+detection+of+NAD%28P%29-binding+proteins%3A+discovery+of+a+unique+antibacterial+drug+target&doi=10.1038%2Fsrep06471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-motif Detection of NAD(P)-binding Proteins: Discovery of a Unique Antibacterial Drug Target</span></div><div class="casAuthors">Hua, Yun Hao; Wu, Chih Yuan; Sargsyan, Karen; Lim, Carmay</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6471</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many enzymes use NAD or NADP (NAD(P)) as essential coenzymes.  These enzymes often do not share significant sequence identity and cannot be easily detected by sequence homol.  Previously, we detd. all distinct locally conserved pyrophosphate-binding structures (3d motifs) from NAD(P)-bound protein structures, from which 1d sequence motifs were derived.  Here, we aim to establish the precision of these 3d and 1d motifs to annotate NAD(P)-binding proteins.  We show that the pyrophosphate-binding 3d motifs are characteristic of NAD(P)-binding proteins, as they are rarely found in nonNAD(P)-binding proteins.  Furthermore, several 1d motifs could distinguish between proteins that bind only NAD and those that bind only NADP.  They could also distinguish between NAD(P)-binding proteins from nonNAD(P)-binding ones.  Interestingly, one of the pyrophosphate-binding 3d and corresponding 1d motifs was found only in enoyl-acyl carrier protein reductases, which are enzymes essential for bacterial fatty acid biosynthesis.  This unique 3d motif serves as an attractive novel drug target, as it is conserved across many bacterial species and is not found in human proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwcv4XrSagTbVg90H21EOLACvtfcHk0ljmUtukVFFA1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVygu7g%253D&md5=088f74d56a30ab954eb4636482198031</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fsrep06471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep06471%26sid%3Dliteratum%253Aachs%26aulast%3DHua%26aufirst%3DY.%2BH.%26aulast%3DWu%26aufirst%3DC.%2BY.%26aulast%3DSargsyan%26aufirst%3DK.%26aulast%3DLim%26aufirst%3DC.%26atitle%3DSequence-motif%2520detection%2520of%2520NAD%2528P%2529-binding%2520proteins%253A%2520discovery%2520of%2520a%2520unique%2520antibacterial%2520drug%2520target%26jtitle%3DSci.%2520Rep.%26date%3D2014%26volume%3D4%26spage%3D6471%26doi%3D10.1038%2Fsrep06471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Das, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kokardekar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breneman, C. M.</span><span> </span><span class="NLM_article-title">Rapid comparison of protein binding site surfaces with property encoded shape distributions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2863</span><span class="NLM_x">–</span> <span class="NLM_lpage">2872</span><span class="refDoi"> DOI: 10.1021/ci900317x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900317x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsValtLnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2863-2872&issue=12&author=S.+Dasauthor=A.+Kokardekarauthor=C.+M.+Breneman&title=Rapid+comparison+of+protein+binding+site+surfaces+with+property+encoded+shape+distributions&doi=10.1021%2Fci900317x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Comparison of Protein Binding Site Surfaces with Property Encoded Shape Distributions</span></div><div class="casAuthors">Das, Sourav; Kokardekar, Arshad; Breneman, Curt M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2863-2872</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Patterns in shape and property distributions on the surface of binding sites are often conserved across functional proteins without significant conservation of the underlying amino-acid residues.  To explore similarities of these sites from the viewpoint of a ligand, a sequence and fold-independent method was created to rapidly and accurately compare binding sites of proteins represented by property-mapped triangulated Gauss-Connolly surfaces.  Within this paradigm, signatures for each binding site surface are produced by calcg. their property-encoded shape distributions (PESD), a measure of the probability that a particular property will be at a specific distance to another on the mol. surface.  Similarity between the signatures can then be treated as a measure of similarity between binding sites.  As postulated, the PESD method rapidly detected high levels of similarity in binding site surface characteristics even in cases where there was very low similarity at the sequence level.  In a screening expt. involving each member of the PDBBind 2005 data set as a query against the rest of the set, PESD was able to retrieve a binding site with identical E.C. (Enzyme Commission) nos. as the top match in 79.5% of cases.  The ability of the method in detecting similarity in binding sites with low sequence conservations was compared to state-of-the-art binding site comparison methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlwxGcrBNQpLVg90H21EOLACvtfcHk0lj3Jopk5yAjfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsValtLnK&md5=7793378407753379d12ebfe70e05a1c1</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fci900317x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900317x%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DS.%26aulast%3DKokardekar%26aufirst%3DA.%26aulast%3DBreneman%26aufirst%3DC.%2BM.%26atitle%3DRapid%2520comparison%2520of%2520protein%2520binding%2520site%2520surfaces%2520with%2520property%2520encoded%2520shape%2520distributions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26issue%3D12%26spage%3D2863%26epage%3D2872%26doi%3D10.1021%2Fci900317x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Das, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krein, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breneman, C. M.</span><span> </span><span class="NLM_article-title">Binding affinity prediction with property-encoded shape distribution signatures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">298</span><span class="NLM_x">–</span> <span class="NLM_lpage">308</span><span class="refDoi"> DOI: 10.1021/ci9004139</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9004139" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVagsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=298-308&issue=2&author=S.+Dasauthor=M.+P.+Kreinauthor=C.+M.+Breneman&title=Binding+affinity+prediction+with+property-encoded+shape+distribution+signatures&doi=10.1021%2Fci9004139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Affinity Prediction with Property-Encoded Shape Distribution Signatures</span></div><div class="casAuthors">Das, Sourav; Krein, Michael P.; Breneman, Curt M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">298-308</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the use of the mol. signatures known as "property-encoded shape distributions" (PESD) together with std. support vector machine (SVM) techniques to produce validated models that can predict the binding affinity of a large no. of protein ligand complexes.  This "PESD-SVM" method uses PESD signatures that encode mol. shapes and property distributions on protein and ligand surfaces as features to build SVM models that require no subjective feature selection.  A simple protocol was employed for tuning the SVM models during their development, and the results were compared to SFCscore, a regression-based method that was previously shown to perform better than 14 other scoring functions.  Although the PESD-SVM method is based on only two surface property maps, the overall results were comparable.  For most complexes with a dominant enthalpic contribution to binding (ΔH/-TΔS >3), a good correlation between true and predicted affinities was obsd.  Entropy and solvent were not considered in the present approach, and further improvement in accuracy would require accounting for these components rigorously.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd5pJea-jHILVg90H21EOLACvtfcHk0lj3Jopk5yAjfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVagsQ%253D%253D&md5=ad40d31b520e68d6b9cb9fa903bcd209</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fci9004139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9004139%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DS.%26aulast%3DKrein%26aufirst%3DM.%2BP.%26aulast%3DBreneman%26aufirst%3DC.%2BM.%26atitle%3DBinding%2520affinity%2520prediction%2520with%2520property-encoded%2520shape%2520distribution%2520signatures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D2%26spage%3D298%26epage%3D308%26doi%3D10.1021%2Fci9004139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, R. B.</span><span> </span><span class="NLM_article-title">Using multiple microenvironments to find similar ligand-binding sites: application to kinase inhibitor binding</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002326</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1002326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1002326" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002326&issue=12&author=T.+Liuauthor=R.+B.+Altman&title=Using+multiple+microenvironments+to+find+similar+ligand-binding+sites%3A+application+to+kinase+inhibitor+binding&doi=10.1371%2Fjournal.pcbi.1002326"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002326%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DAltman%26aufirst%3DR.%2BB.%26atitle%3DUsing%2520multiple%2520microenvironments%2520to%2520find%2520similar%2520ligand-binding%2520sites%253A%2520application%2520to%2520kinase%2520inhibitor%2520binding%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D12%26spage%3De1002326%26doi%3D10.1371%2Fjournal.pcbi.1002326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Miguel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K. C.</span><span> </span><span class="NLM_article-title">Variations in the binding pocket of an inhibitor of the bacterial division protein FtsZ across genotypes and species</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1004117</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1004117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1004117" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=e1004117&issue=3&author=A.+Miguelauthor=J.+Hsinauthor=T.+Liuauthor=G.+Tangauthor=R.+B.+Altmanauthor=K.+C.+Huang&title=Variations+in+the+binding+pocket+of+an+inhibitor+of+the+bacterial+division+protein+FtsZ+across+genotypes+and+species&doi=10.1371%2Fjournal.pcbi.1004117"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004117%26sid%3Dliteratum%253Aachs%26aulast%3DMiguel%26aufirst%3DA.%26aulast%3DHsin%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DTang%26aufirst%3DG.%26aulast%3DAltman%26aufirst%3DR.%2BB.%26aulast%3DHuang%26aufirst%3DK.%2BC.%26atitle%3DVariations%2520in%2520the%2520binding%2520pocket%2520of%2520an%2520inhibitor%2520of%2520the%2520bacterial%2520division%2520protein%2520FtsZ%2520across%2520genotypes%2520and%2520species%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D3%26spage%3De1004117%26doi%3D10.1371%2Fjournal.pcbi.1004117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Yeturu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandra, N.</span><span> </span><span class="NLM_article-title">PocketMatch: a new algorithm to compare binding sites in protein structures</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">543</span><span class="refDoi"> DOI: 10.1186/1471-2105-9-543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-9-543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=19091072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BD1M7jtVKksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=543&author=K.+Yeturuauthor=N.+Chandra&title=PocketMatch%3A+a+new+algorithm+to+compare+binding+sites+in+protein+structures&doi=10.1186%2F1471-2105-9-543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">PocketMatch: a new algorithm to compare binding sites in protein structures</span></div><div class="casAuthors">Yeturu Kalidas; Chandra Nagasuma</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">543</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Recognizing similarities and deriving relationships among protein molecules is a fundamental requirement in present-day biology.  Similarities can be present at various levels which can be detected through comparison of protein sequences or their structural folds.  In some cases similarities obscure at these levels could be present merely in the substructures at their binding sites.  Inferring functional similarities between protein molecules by comparing their binding sites is still largely exploratory and not as yet a routine protocol.  One of the main reasons for this is the limitation in the choice of appropriate analytical tools that can compare binding sites with high sensitivity.  To benefit from the enormous amount of structural data that is being rapidly accumulated, it is essential to have high throughput tools that enable large scale binding site comparison.  RESULTS:  Here we present a new algorithm PocketMatch for comparison of binding sites in a frame invariant manner.  Each binding site is represented by 90 lists of sorted distances capturing shape and chemical nature of the site.  The sorted arrays are then aligned using an incremental alignment method and scored to obtain PMScores for pairs of sites.  A comprehensive sensitivity analysis and an extensive validation of the algorithm have been carried out.  A comparison with other site matching algorithms is also presented.  Perturbation studies where the geometry of a given site was retained but the residue types were changed randomly, indicated that chance similarities were virtually non-existent.  Our analysis also demonstrates that shape information alone is insufficient to discriminate between diverse binding sites, unless combined with chemical nature of amino acids.  CONCLUSION:  A new algorithm has been developed to compare binding sites in accurate, efficient and high-throughput manner.  Though the representation used is conceptually simplistic, we demonstrate that along with the new alignment strategy used, it is sufficient to enable binding comparison with high sensitivity.  Novel methodology has also been presented for validating the algorithm for accuracy and sensitivity with respect to geometry and chemical nature of the site.  The method is also fast and takes about 1/250th second for one comparison on a single processor.  A parallel version on BlueGene has also been implemented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQG4iooRU0PkP9pSBfyDvp7fW6udTcc2eajy2U20K_eWbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M7jtVKksg%253D%253D&md5=b5f9edf7278e1d6f5dca62178c8c1a03</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-543%26sid%3Dliteratum%253Aachs%26aulast%3DYeturu%26aufirst%3DK.%26aulast%3DChandra%26aufirst%3DN.%26atitle%3DPocketMatch%253A%2520a%2520new%2520algorithm%2520to%2520compare%2520binding%2520sites%2520in%2520protein%2520structures%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26spage%3D543%26doi%3D10.1186%2F1471-2105-9-543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Anand, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sankaran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeturu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhardwaj, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhagavat, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brahmachari, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandra, N.</span><span> </span><span class="NLM_article-title">Structural annotation of Mycobacterium tuberculosis proteome</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">e27044</span><span class="refDoi"> DOI: 10.1371/journal.pone.0027044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0027044" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e27044&issue=10&author=P.+Anandauthor=S.+Sankaranauthor=S.+Mukherjeeauthor=K.+Yeturuauthor=R.+Laskowskiauthor=A.+Bhardwajauthor=R.+Bhagavatauthor=S.+K.+Brahmachariauthor=N.+Chandra&title=Structural+annotation+of+Mycobacterium+tuberculosis+proteome&doi=10.1371%2Fjournal.pone.0027044"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027044%26sid%3Dliteratum%253Aachs%26aulast%3DAnand%26aufirst%3DP.%26aulast%3DSankaran%26aufirst%3DS.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DYeturu%26aufirst%3DK.%26aulast%3DLaskowski%26aufirst%3DR.%26aulast%3DBhardwaj%26aufirst%3DA.%26aulast%3DBhagavat%26aufirst%3DR.%26aulast%3DBrahmachari%26aufirst%3DS.%2BK.%26aulast%3DChandra%26aufirst%3DN.%26atitle%3DStructural%2520annotation%2520of%2520Mycobacterium%2520tuberculosis%2520proteome%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26issue%3D10%26spage%3De27044%26doi%3D10.1371%2Fjournal.pone.0027044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Möller-Acuña, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contreras-Riquelme, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rojas-Fuentes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuñez-Vivanco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alzate-Morales, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iturriaga-Vásquez, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arias, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reyes-Parada, M.</span><span> </span><span class="NLM_article-title">Similarities between the binding sites of SB-206553 at serotonin type 2 and alpha7 acetylcholine nicotinic receptors: rationale for its polypharmacological profile</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e0134444</span><span class="refDoi"> DOI: 10.1371/journal.pone.0134444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0134444" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0134444&issue=8&author=P.+M%C3%B6ller-Acu%C3%B1aauthor=J.+S.+Contreras-Riquelmeauthor=C.+Rojas-Fuentesauthor=G.+Nu%C3%B1ez-Vivancoauthor=J.+Alzate-Moralesauthor=P.+Iturriaga-V%C3%A1squezauthor=H.+R.+Ariasauthor=M.+Reyes-Parada&title=Similarities+between+the+binding+sites+of+SB-206553+at+serotonin+type+2+and+alpha7+acetylcholine+nicotinic+receptors%3A+rationale+for+its+polypharmacological+profile&doi=10.1371%2Fjournal.pone.0134444"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0134444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0134444%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25B6ller-Acu%25C3%25B1a%26aufirst%3DP.%26aulast%3DContreras-Riquelme%26aufirst%3DJ.%2BS.%26aulast%3DRojas-Fuentes%26aufirst%3DC.%26aulast%3DNu%25C3%25B1ez-Vivanco%26aufirst%3DG.%26aulast%3DAlzate-Morales%26aufirst%3DJ.%26aulast%3DIturriaga-V%25C3%25A1squez%26aufirst%3DP.%26aulast%3DArias%26aufirst%3DH.%2BR.%26aulast%3DReyes-Parada%26aufirst%3DM.%26atitle%3DSimilarities%2520between%2520the%2520binding%2520sites%2520of%2520SB-206553%2520at%2520serotonin%2520type%25202%2520and%2520alpha7%2520acetylcholine%2520nicotinic%2520receptors%253A%2520rationale%2520for%2520its%2520polypharmacological%2520profile%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3De0134444%26doi%3D10.1371%2Fjournal.pone.0134444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Fierro, A.; Montecinos, A.; Gmez-Molina, C.; Nez, G.; Aldeco, M.; Edmondson, D. E.; Vilches Herrera, M.; Luhr, S.; Iturriaga-Vsquez, P.; Reyes Par, M.</span><span> </span><span class="NLM_article-title">Similarities between the binding sites of monoamine oxidase (MAO) from different species—Is zebrafish a useful model for the discovery of novel MAO inhibitors?</span> In  <span class="citation_source-book">An Integrated View of the Molecular Recognition and Toxinology: From Analytical Procedures to Biomedical Applications</span>; <span class="NLM_contrib-group">Radis-Baptista, G.</span>, Ed.; <span class="NLM_publisher-name">InTech</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>DOI: <span class="refDoi"> DOI: 10.5772/35874</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.5772%2F35874" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+Fierro&author=A.+Montecinos&author=C.+Gmez-Molina&author=G.+Nez&author=M.+Aldeco&author=D.+E.+Edmondson&author=M.+Vilches+Herrera&author=S.+Luhr&author=P.+Iturriaga-Vsquez&author=M.+Reyes+Parauthor=G.+Radis-Baptista&title=An+Integrated+View+of+the+Molecular+Recognition+and+Toxinology%3A+From+Analytical+Procedures+to+Biomedical+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.5772%2F35874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.5772%252F35874%26sid%3Dliteratum%253Aachs%26aulast%3DFierro%26aufirst%3DA.%26atitle%3DSimilarities%2520between%2520the%2520binding%2520sites%2520of%2520monoamine%2520oxidase%2520%2528MAO%2529%2520from%2520different%2520species%25E2%2580%2594Is%2520zebrafish%2520a%2520useful%2520model%2520for%2520the%2520discovery%2520of%2520novel%2520MAO%2520inhibitors%253F%26btitle%3DAn%2520Integrated%2520View%2520of%2520the%2520Molecular%2520Recognition%2520and%2520Toxinology%253A%2520From%2520Analytical%2520Procedures%2520to%2520Biomedical%2520Applications%26aulast%3DRadis-Baptista%26aufirst%3DG.%26pub%3DInTech%26date%3D2013%26doi%3D10.5772%2F35874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Reisen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weisel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kriegl, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span> </span><span class="NLM_article-title">Self-organizing fuzzy graphs for structure-based comparison of protein pockets</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">6498</span><span class="NLM_x">–</span> <span class="NLM_lpage">6510</span><span class="refDoi"> DOI: 10.1021/pr100719n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr100719n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlaltbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=6498-6510&issue=12&author=F.+Reisenauthor=M.+Weiselauthor=J.+M.+Krieglauthor=G.+Schneider&title=Self-organizing+fuzzy+graphs+for+structure-based+comparison+of+protein+pockets&doi=10.1021%2Fpr100719n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Organizing Fuzzy Graphs for Structure-Based Comparison of Protein Pockets</span></div><div class="casAuthors">Reisen, Felix; Weisel, Martin; Kriegl, Jan M.; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6498-6510</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Patterns of receptor-ligand interaction can be conserved in functionally equiv. proteins even in the absence of sequence homol.  Therefore, structural comparison of ligand-binding pockets and their pharmacophoric features allow for the characterization of so-called "orphan" proteins with known three-dimensional structure but unknown function, and predict ligand promiscuity of binding pockets.  We present an algorithm for rapid pocket comparison (PoLiMorph), in which protein pockets are represented by self-organizing graphs that fill the vol. of the cavity.  Vertices in these three-dimensional frameworks contain information about the local ligand-receptor interaction potential coded by fuzzy property labels.  For framework matching, we developed a fast heuristic based on the max. dispersion problem, as an alternative to techniques utilizing clique detection or geometric hashing algorithms.  A sophisticated scoring function was applied that incorporates knowledge about property distributions and ligand-receptor interaction patterns.  In an all-against-all virtual screening expt. with 207 pocket frameworks extd. from a subset of PDBbind, PoLiMorph correctly assigned 81% of 69 distinct structural classes and demonstrated sustained ability to group pockets accommodating the same ligand chemotype.  We detd. a score threshold that indicates "true" pocket similarity with high reliability, which not only supports structure-based drug design but also allows for sequence-independent studies of the proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjH1-VYRUmObVg90H21EOLACvtfcHk0linliMn9PCdYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlaltbvL&md5=8da973a8e1097e456ed9bffa7040c4b9</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fpr100719n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr100719n%26sid%3Dliteratum%253Aachs%26aulast%3DReisen%26aufirst%3DF.%26aulast%3DWeisel%26aufirst%3DM.%26aulast%3DKriegl%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DSelf-organizing%2520fuzzy%2520graphs%2520for%2520structure-based%2520comparison%2520of%2520protein%2520pockets%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2010%26volume%3D9%26issue%3D12%26spage%3D6498%26epage%3D6510%26doi%3D10.1021%2Fpr100719n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Lanig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reisen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitley, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banting, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, T.</span><span> </span><span class="NLM_article-title">In silico adoption of an orphan nuclear receptor NR4A1</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e0135246</span><span class="refDoi"> DOI: 10.1371/journal.pone.0135246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0135246" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0135246&issue=8&author=H.+Lanigauthor=F.+Reisenauthor=D.+Whitleyauthor=G.+Schneiderauthor=L.+Bantingauthor=T.+Clark&title=In+silico+adoption+of+an+orphan+nuclear+receptor+NR4A1&doi=10.1371%2Fjournal.pone.0135246"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0135246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0135246%26sid%3Dliteratum%253Aachs%26aulast%3DLanig%26aufirst%3DH.%26aulast%3DReisen%26aufirst%3DF.%26aulast%3DWhitley%26aufirst%3DD.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DBanting%26aufirst%3DL.%26aulast%3DClark%26aufirst%3DT.%26atitle%3DIn%2520silico%2520adoption%2520of%2520an%2520orphan%2520nuclear%2520receptor%2520NR4A1%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3De0135246%26doi%3D10.1371%2Fjournal.pone.0135246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janežič, D.</span><span> </span><span class="NLM_article-title">ProBiS algorithm for detection of structurally similar protein binding sites by local structural alignment</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">1160</span><span class="NLM_x">–</span> <span class="NLM_lpage">1168</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btq100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtq100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=20305268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1160-1168&issue=9&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS+algorithm+for+detection+of+structurally+similar+protein+binding+sites+by+local+structural+alignment&doi=10.1093%2Fbioinformatics%2Fbtq100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS algorithm for detection of structurally similar protein binding sites by local structural alignment</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1160-1168</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Exploitation of locally similar 3D patterns of physicochem. properties on the surface of a protein for detection of binding sites that may lack sequence and global structural conservation.  An algorithm, ProBiS is described that detects structurally similar sites on protein surfaces by local surface structure alignment.  It compares the query protein to members of a database of protein 3D structures and detects with sub-residue precision, structurally similar sites as patterns of physicochem. properties on the protein surface.  Using an efficient max. clique algorithm, the program identifies proteins that share local structural similarities with the query protein and generates structure-based alignments of these proteins with the query.  Structural similarity scores are calcd. for the query protein's surface residues, and are expressed as different colors on the query protein surface.  The algorithm has been used successfully for the detection of protein-protein, protein-small ligand and protein-DNA binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproKvL-3-CSLVg90H21EOLACvtfcHk0li-yHanvuwz8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D&md5=7ff0eecdf0ea5f7d9b82fd721ca88156</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq100%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%2520algorithm%2520for%2520detection%2520of%2520structurally%2520similar%2520protein%2520binding%2520sites%2520by%2520local%2520structural%2520alignment%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26issue%3D9%26spage%3D1160%26epage%3D1168%26doi%3D10.1093%2Fbioinformatics%2Fbtq100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodošček, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogrizek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trykowska Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janežič, D.</span><span> </span><span class="NLM_article-title">Structure-based function prediction of uncharacterized protein using binding sites comparison</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1003341</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1003341" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003341&issue=11&author=J.+Koncauthor=M.+Hodo%C5%A1%C4%8Dekauthor=M.+Ogrizekauthor=J.+Trykowska+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=Structure-based+function+prediction+of+uncharacterized+protein+using+binding+sites+comparison&doi=10.1371%2Fjournal.pcbi.1003341"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003341%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DHodo%25C5%25A1%25C4%258Dek%26aufirst%3DM.%26aulast%3DOgrizek%26aufirst%3DM.%26aulast%3DTrykowska%2BKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DStructure-based%2520function%2520prediction%2520of%2520uncharacterized%2520protein%2520using%2520binding%2520sites%2520comparison%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D11%26spage%3De1003341%26doi%3D10.1371%2Fjournal.pcbi.1003341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Kakisaka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamada, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kondoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hikono, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomii, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aida, Y.</span><span> </span><span class="NLM_article-title">A novel antiviral target structure involved in the RNA binding, dimerization, and nuclear export functions of the influenza A virus nucleoprotein</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1005062</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1005062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.ppat.1005062" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=e1005062&issue=7&author=M.+Kakisakaauthor=Y.+Sasakiauthor=K.+Yamadaauthor=Y.+Kondohauthor=H.+Hikonoauthor=H.+Osadaauthor=K.+Tomiiauthor=T.+Saitoauthor=Y.+Aida&title=A+novel+antiviral+target+structure+involved+in+the+RNA+binding%2C+dimerization%2C+and+nuclear+export+functions+of+the+influenza+A+virus+nucleoprotein&doi=10.1371%2Fjournal.ppat.1005062"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005062%26sid%3Dliteratum%253Aachs%26aulast%3DKakisaka%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DY.%26aulast%3DYamada%26aufirst%3DK.%26aulast%3DKondoh%26aufirst%3DY.%26aulast%3DHikono%26aufirst%3DH.%26aulast%3DOsada%26aufirst%3DH.%26aulast%3DTomii%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DT.%26aulast%3DAida%26aufirst%3DY.%26atitle%3DA%2520novel%2520antiviral%2520target%2520structure%2520involved%2520in%2520the%2520RNA%2520binding%252C%2520dimerization%252C%2520and%2520nuclear%2520export%2520functions%2520of%2520the%2520influenza%2520A%2520virus%2520nucleoprotein%26jtitle%3DPLoS%2520Pathog.%26date%3D2015%26volume%3D11%26issue%3D7%26spage%3De1005062%26doi%3D10.1371%2Fjournal.ppat.1005062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Spitzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleves, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Surface-based protein binding pocket similarity</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">2746</span><span class="NLM_x">–</span> <span class="NLM_lpage">2763</span><span class="refDoi"> DOI: 10.1002/prot.23103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.23103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2011&pages=2746-2763&issue=9&author=R.+Spitzerauthor=A.+E.+Clevesauthor=A.+N.+Jain&title=Surface-based+protein+binding+pocket+similarity&doi=10.1002%2Fprot.23103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fprot.23103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.23103%26sid%3Dliteratum%253Aachs%26aulast%3DSpitzer%26aufirst%3DR.%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurface-based%2520protein%2520binding%2520pocket%2520similarity%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2011%26volume%3D79%26issue%3D9%26spage%3D2746%26epage%3D2763%26doi%3D10.1002%2Fprot.23103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Cleves, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Chemical and protein structural basis for biological crosstalk between PPARα and COX enzymes</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span><span class="refDoi"> DOI: 10.1007/s10822-014-9815-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2Fs10822-014-9815-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=101-112&issue=2&author=A.+E.+Clevesauthor=A.+N.+Jain&title=Chemical+and+protein+structural+basis+for+biological+crosstalk+between+PPAR%CE%B1+and+COX+enzymes&doi=10.1007%2Fs10822-014-9815-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9815-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9815-2%26sid%3Dliteratum%253Aachs%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DChemical%2520and%2520protein%2520structural%2520basis%2520for%2520biological%2520crosstalk%2520between%2520PPAR%25CE%25B1%2520and%2520COX%2520enzymes%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26issue%3D2%26spage%3D101%26epage%3D112%26doi%3D10.1007%2Fs10822-014-9815-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Ausiello, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Via, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmer-Citterich, M.</span><span> </span><span class="NLM_article-title">Query3d: a new method for high-throughput analysis of functional residues in protein structures</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 4</span><span class="NLM_x">) </span> <span class="NLM_fpage">S5</span><span class="refDoi"> DOI: 10.1186/1471-2105-6-S4-S5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-6-S4-S5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=S5&issue=Suppl.+4&author=G.+Ausielloauthor=A.+Viaauthor=M.+Helmer-Citterich&title=Query3d%3A+a+new+method+for+high-throughput+analysis+of+functional+residues+in+protein+structures&doi=10.1186%2F1471-2105-6-S4-S5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-6-S4-S5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-6-S4-S5%26sid%3Dliteratum%253Aachs%26aulast%3DAusiello%26aufirst%3DG.%26aulast%3DVia%26aufirst%3DA.%26aulast%3DHelmer-Citterich%26aufirst%3DM.%26atitle%3DQuery3d%253A%2520a%2520new%2520method%2520for%2520high-throughput%2520analysis%2520of%2520functional%2520residues%2520in%2520protein%2520structures%26jtitle%3DBMC%2520Bioinf.%26date%3D2005%26volume%3D6%26issue%3DSuppl.%25204%26spage%3DS5%26doi%3D10.1186%2F1471-2105-6-S4-S5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Ausiello, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peluso, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Via, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmer-Citterich, M.</span><span> </span><span class="NLM_article-title">Local comparison of protein structures highlights cases of convergent evolution in analogous functional sites</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S24</span><span class="refDoi"> DOI: 10.1186/1471-2105-8-S1-S24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-8-S1-S24" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=S24&issue=Suppl.+1&author=G.+Ausielloauthor=D.+Pelusoauthor=A.+Viaauthor=M.+Helmer-Citterich&title=Local+comparison+of+protein+structures+highlights+cases+of+convergent+evolution+in+analogous+functional+sites&doi=10.1186%2F1471-2105-8-S1-S24"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-S1-S24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-8-S1-S24%26sid%3Dliteratum%253Aachs%26aulast%3DAusiello%26aufirst%3DG.%26aulast%3DPeluso%26aufirst%3DD.%26aulast%3DVia%26aufirst%3DA.%26aulast%3DHelmer-Citterich%26aufirst%3DM.%26atitle%3DLocal%2520comparison%2520of%2520protein%2520structures%2520highlights%2520cases%2520of%2520convergent%2520evolution%2520in%2520analogous%2520functional%2520sites%26jtitle%3DBMC%2520Bioinf.%26date%3D2007%26volume%3D8%26issue%3DSuppl.%25201%26spage%3DS24%26doi%3D10.1186%2F1471-2105-8-S1-S24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">De Franchi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schalon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Messa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onofri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benfenati, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Binding of protein kinase inhibitors to synapsin I inferred from pair-wise binding site similarity measurements</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">e12214</span><span class="refDoi"> DOI: 10.1371/journal.pone.0012214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0012214" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e12214&issue=8&author=E.+De+Franchiauthor=C.+Schalonauthor=M.+Messaauthor=F.+Onofriauthor=F.+Benfenatiauthor=D.+Rognan&title=Binding+of+protein+kinase+inhibitors+to+synapsin+I+inferred+from+pair-wise+binding+site+similarity+measurements&doi=10.1371%2Fjournal.pone.0012214"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0012214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0012214%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BFranchi%26aufirst%3DE.%26aulast%3DSchalon%26aufirst%3DC.%26aulast%3DMessa%26aufirst%3DM.%26aulast%3DOnofri%26aufirst%3DF.%26aulast%3DBenfenati%26aufirst%3DF.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DBinding%2520of%2520protein%2520kinase%2520inhibitors%2520to%2520synapsin%2520I%2520inferred%2520from%2520pair-wise%2520binding%2520site%2520similarity%2520measurements%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26issue%3D8%26spage%3De12214%26doi%3D10.1371%2Fjournal.pone.0012214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Shulman-Peleg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfson, H. J.</span><span> </span><span class="NLM_article-title">Recognition of functional sites in protein structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">339</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">633</span><span class="refDoi"> DOI: 10.1016/j.jmb.2004.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.jmb.2004.04.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2004&pages=607-633&issue=3&author=A.+Shulman-Pelegauthor=R.+Nussinovauthor=H.+J.+Wolfson&title=Recognition+of+functional+sites+in+protein+structures&doi=10.1016%2Fj.jmb.2004.04.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DShulman-Peleg%26aufirst%3DA.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DWolfson%26aufirst%3DH.%2BJ.%26atitle%3DRecognition%2520of%2520functional%2520sites%2520in%2520protein%2520structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D339%26issue%3D3%26spage%3D607%26epage%3D633%26doi%3D10.1016%2Fj.jmb.2004.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group">Keren-Kaplan, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Attali, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estrin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuo, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farkash, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jerabek-Willemsen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blutraich, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artzi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freed, E. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfson, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prag, G.</span><span> </span><span class="NLM_article-title">Structure-based in silico identification of ubiquitin-binding domains provides insights into the ALIX-V:ubiquitin complex and retrovirus budding</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">551</span><span class="refDoi"> DOI: 10.1038/emboj.2013.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Femboj.2013.4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=538-551&issue=4&author=T.+Keren-Kaplanauthor=I.+Attaliauthor=M.+Estrinauthor=L.+S.+Kuoauthor=E.+Farkashauthor=M.+Jerabek-Willemsenauthor=N.+Blutraichauthor=S.+Artziauthor=A.+Periauthor=E.+O.+Freedauthor=H.+J.+Wolfsonauthor=G.+Prag&title=Structure-based+in+silico+identification+of+ubiquitin-binding+domains+provides+insights+into+the+ALIX-V%3Aubiquitin+complex+and+retrovirus+budding&doi=10.1038%2Femboj.2013.4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1038%2Femboj.2013.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2013.4%26sid%3Dliteratum%253Aachs%26aulast%3DKeren-Kaplan%26aufirst%3DT.%26aulast%3DAttali%26aufirst%3DI.%26aulast%3DEstrin%26aufirst%3DM.%26aulast%3DKuo%26aufirst%3DL.%2BS.%26aulast%3DFarkash%26aufirst%3DE.%26aulast%3DJerabek-Willemsen%26aufirst%3DM.%26aulast%3DBlutraich%26aufirst%3DN.%26aulast%3DArtzi%26aufirst%3DS.%26aulast%3DPeri%26aufirst%3DA.%26aulast%3DFreed%26aufirst%3DE.%2BO.%26aulast%3DWolfson%26aufirst%3DH.%2BJ.%26aulast%3DPrag%26aufirst%3DG.%26atitle%3DStructure-based%2520in%2520silico%2520identification%2520of%2520ubiquitin-binding%2520domains%2520provides%2520insights%2520into%2520the%2520ALIX-V%253Aubiquitin%2520complex%2520and%2520retrovirus%2520budding%26jtitle%3DEMBO%2520J.%26date%3D2013%26volume%3D32%26issue%3D4%26spage%3D538%26epage%3D551%26doi%3D10.1038%2Femboj.2013.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A unified statistical model to support local sequence order independent similarity searching for ligand-binding sites and its application to genome-based drug discovery</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">i305</span><span class="NLM_x">–</span> <span class="NLM_lpage">i312</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtp220" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=i305-i312&issue=12&author=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=A+unified+statistical+model+to+support+local+sequence+order+independent+similarity+searching+for+ligand-binding+sites+and+its+application+to+genome-based+drug+discovery&doi=10.1093%2Fbioinformatics%2Fbtp220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp220%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520unified%2520statistical%2520model%2520to%2520support%2520local%2520sequence%2520order%2520independent%2520similarity%2520searching%2520for%2520ligand-binding%2520sites%2520and%2520its%2520application%2520to%2520genome-based%2520drug%2520discovery%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26issue%3D12%26spage%3Di305%26epage%3Di312%26doi%3D10.1093%2Fbioinformatics%2Fbtp220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evangelidis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: weak inhibition of multiple kinases may contribute to the anti-cancer effect of nelfinavir</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002037</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1002037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1002037" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002037&issue=4&author=L.+Xieauthor=T.+Evangelidisauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+weak+inhibition+of+multiple+kinases+may+contribute+to+the+anti-cancer+effect+of+nelfinavir&doi=10.1371%2Fjournal.pcbi.1002037"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002037%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DEvangelidis%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520weak%2520inhibition%2520of%2520multiple%2520kinases%2520may%2520contribute%2520to%2520the%2520anti-cancer%2520effect%2520of%2520nelfinavir%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D4%26spage%3De1002037%26doi%3D10.1371%2Fjournal.pcbi.1002037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group">Niu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiaoe, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span> </span><span class="NLM_article-title">Structural bioinformatics-based identification of EGFR inhibitor gefitinib as a putative lead compound for BACE</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.1111/cbdd.12200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1111%2Fcbdd.12200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24516878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOntbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=81-88&issue=1&author=M.+Niuauthor=J.+Huauthor=S.+Wuauthor=Z.+Xiaoeauthor=H.+Xuauthor=Y.+Zhangauthor=J.+Zhangauthor=Y.+Yang&title=Structural+bioinformatics-based+identification+of+EGFR+inhibitor+gefitinib+as+a+putative+lead+compound+for+BACE&doi=10.1111%2Fcbdd.12200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Bioinformatics-Based Identification of EGFR Inhibitor Gefitinib as a Putative Lead Compound for BACE</span></div><div class="casAuthors">Niu, Mingshan; Hu, Jin; Wu, Sijin; Zhang, Xiaoe; Xu, Huaxi; Zhang, Yunwu; Zhang, Jie; Yang, Yongliang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-88</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">β-Secretase (BACE-1) is a potential target for the treatment of Alzheimer's disease (AD).  Despite its potential, only few compds. targeting BACE have entered the clin. trials.  Herein, the authors describe the identification of Gefitinib as a potential lead compd. for BACE through an integrated approach of structural bioinformatics anal., exptl. assessment and computational anal.  In particular, the authors performed ELISA and western anal. to assess the effect of Gefitinib using N2a human APP695 cells.  In addn., the authors investigated the binding mechanism of Gefitinib with BACE through mol. docking coupled with mol. dynamics simulations.  The computational analyses revealed that hydrophobic contact is a major contributing factor to the binding of Gefitinib with BACE.  The results obtained in the study have rendered Gefitinib as a putative lead compd. for BACE.  Further optimization studies are warranted to improve its potency and pharmacol. properties against BACE for potential AD treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaB-v5A6zsELVg90H21EOLACvtfcHk0lg2pWA2gaiqfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOntbjJ&md5=229f742fee427820a7910f72c11815cf</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12200%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DXiaoe%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DStructural%2520bioinformatics-based%2520identification%2520of%2520EGFR%2520inhibitor%2520gefitinib%2520as%2520a%2520putative%2520lead%2520compound%2520for%2520BACE%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2014%26volume%3D83%26issue%3D1%26spage%3D81%26epage%3D88%26doi%3D10.1111%2Fcbdd.12200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Detecting evolutionary relationships across existing fold space, using sequence order-independent profile-profile alignments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">5441</span><span class="NLM_x">–</span> <span class="NLM_lpage">5446</span><span class="refDoi"> DOI: 10.1073/pnas.0704422105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1073%2Fpnas.0704422105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18385384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslSnsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=5441-5446&issue=14&author=L.+Xieauthor=P.+E.+Bourne&title=Detecting+evolutionary+relationships+across+existing+fold+space%2C+using+sequence+order-independent+profile-profile+alignments&doi=10.1073%2Fpnas.0704422105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Detecting evolutionary relationships across existing fold space, using sequence order-independent profile-profile alignments</span></div><div class="casAuthors">Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5441-5446</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Here, a scalable, accurate, reliable, and robust protein functional site comparison algorithm is presented.  The key components of the algorithm consist of a reduced representation of the protein structure and a sequence order-independent profile-profile alignment (SOIPPA).  We show that SOIPPA is able to detect distant evolutionary relationships in cases where both a global sequence and structure relationship remains obscure.  Results suggest evolutionary relationships across several previously evolutionary distinct protein structure superfamilies.  SOIPPA, along with an increased coverage of-protein fold space afforded by the structural genomics initiative, can be used to further test the notion that fold space is continuous rather than discrete.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwuIQ_fI-0v7Vg90H21EOLACvtfcHk0lg2pWA2gaiqfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslSnsLY%253D&md5=a3a5a216f18aff3038d987bb6e4c6443</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0704422105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0704422105%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDetecting%2520evolutionary%2520relationships%2520across%2520existing%2520fold%2520space%252C%2520using%2520sequence%2520order-independent%2520profile-profile%2520alignments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26issue%3D14%26spage%3D5441%26epage%3D5446%26doi%3D10.1073%2Fpnas.0704422105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group">Kinnings, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchmeier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: repositioning the safe medicine Comtan to treat multi-drug and extensively drug resistant tuberculosis</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1000423</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.1000423" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000423&issue=7&author=S.+L.+Kinningsauthor=N.+Liuauthor=N.+Buchmeierauthor=P.+J.+Tongeauthor=L.+Xieauthor=P.+E.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+repositioning+the+safe+medicine+Comtan+to+treat+multi-drug+and+extensively+drug+resistant+tuberculosis&doi=10.1371%2Fjournal.pcbi.1000423"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000423%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%2BL.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DBuchmeier%26aufirst%3DN.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520repositioning%2520the%2520safe%2520medicine%2520Comtan%2520to%2520treat%2520multi-drug%2520and%2520extensively%2520drug%2520resistant%2520tuberculosis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26issue%3D7%26spage%3De1000423%26doi%3D10.1371%2Fjournal.pcbi.1000423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">In silico elucidation of the molecular mechanism defining the adverse effect of selective estrogen receptor modulators</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">e217</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.0030217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pcbi.0030217" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=e217&issue=11&author=L.+Xieauthor=J.+Wangauthor=P.+E.+Bourne&title=In+silico+elucidation+of+the+molecular+mechanism+defining+the+adverse+effect+of+selective+estrogen+receptor+modulators&doi=10.1371%2Fjournal.pcbi.0030217"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.0030217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.0030217%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DIn%2520silico%2520elucidation%2520of%2520the%2520molecular%2520mechanism%2520defining%2520the%2520adverse%2520effect%2520of%2520selective%2520estrogen%2520receptor%2520modulators%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2007%26volume%3D3%26issue%3D11%26spage%3De217%26doi%3D10.1371%2Fjournal.pcbi.0030217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group">Binkowski, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joachimiak, A.</span><span> </span><span class="NLM_article-title">Protein functional surfaces: global shape matching and local spatial alignments of ligand binding sites</span> <span class="citation_source-journal">BMC Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">45</span><span class="refDoi"> DOI: 10.1186/1472-6807-8-45</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1472-6807-8-45" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18954462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FntVOjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=45&author=T.+A.+Binkowskiauthor=A.+Joachimiak&title=Protein+functional+surfaces%3A+global+shape+matching+and+local+spatial+alignments+of+ligand+binding+sites&doi=10.1186%2F1472-6807-8-45"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Protein functional surfaces: global shape matching and local spatial alignments of ligand binding sites</span></div><div class="casAuthors">Binkowski T Andrew; Joachimiak Andrzej</div><div class="citationInfo"><span class="NLM_cas:title">BMC structural biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Protein surfaces comprise only a fraction of the total residues but are the most conserved functional features of proteins.  Surfaces performing identical functions are found in proteins absent of any sequence or fold similarity.  While biochemical activity can be attributed to a few key residues, the broader surrounding environment plays an equally important role.  RESULTS:  We describe a methodology that attempts to optimize two components, global shape and local physicochemical texture, for evaluating the similarity between a pair of surfaces.  Surface shape similarity is assessed using a three-dimensional object recognition algorithm and physicochemical texture similarity is assessed through a spatial alignment of conserved residues between the surfaces.  The comparisons are used in tandem to efficiently search the Global Protein Surface Survey (GPSS), a library of annotated surfaces derived from structures in the PDB, for studying evolutionary relationships and uncovering novel similarities between proteins.  CONCLUSION:  We provide an assessment of our method using library retrieval experiments for identifying functionally homologous surfaces binding different ligands, functionally diverse surfaces binding the same ligand, and binding surfaces of ubiquitous and conformationally flexible ligands.  Results using surface similarity to predict function for proteins of unknown function are reported.  Additionally, an automated analysis of the ATP binding surface landscape is presented to provide insight into the correlation between surface similarity and function for structures in the PDB and for the subset of protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRwk9DbnSjneTToTC3aBaesfW6udTcc2eaY1wcKVcenGLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FntVOjsg%253D%253D&md5=e991996eb22a4f3fcf5f236cf06f76ac</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1186%2F1472-6807-8-45&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6807-8-45%26sid%3Dliteratum%253Aachs%26aulast%3DBinkowski%26aufirst%3DT.%2BA.%26aulast%3DJoachimiak%26aufirst%3DA.%26atitle%3DProtein%2520functional%2520surfaces%253A%2520global%2520shape%2520matching%2520and%2520local%2520spatial%2520alignments%2520of%2520ligand%2520binding%2520sites%26jtitle%3DBMC%2520Struct.%2520Biol.%26date%3D2008%26volume%3D8%26spage%3D45%26doi%3D10.1186%2F1472-6807-8-45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group">Babu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beloglazova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skarina, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocek, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagarinova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pogoutse, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binkowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phanse, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joachimiak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koonin, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savchenko, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emili, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yakunin, A. F.</span><span> </span><span class="NLM_article-title">A dual function of the CRISPR-Cas system in bacterial antivirus immunity and DNA repair</span> <span class="citation_source-journal">Mol. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">484</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1111/j.1365-2958.2010.07465.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1111%2Fj.1365-2958.2010.07465.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21219465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsl2qtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2011&pages=484-502&issue=2&author=M.+Babuauthor=N.+Beloglazovaauthor=R.+Flickauthor=C.+Grahamauthor=T.+Skarinaauthor=B.+Nocekauthor=A.+Gagarinovaauthor=O.+Pogoutseauthor=G.+Brownauthor=A.+Binkowskiauthor=S.+Phanseauthor=A.+Joachimiakauthor=E.+V.+Kooninauthor=A.+Savchenkoauthor=A.+Emiliauthor=J.+Greenblattauthor=A.+M.+Edwardsauthor=A.+F.+Yakunin&title=A+dual+function+of+the+CRISPR-Cas+system+in+bacterial+antivirus+immunity+and+DNA+repair&doi=10.1111%2Fj.1365-2958.2010.07465.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">A dual function of the CRISPR-Cas system in bacterial antivirus immunity and DNA repair</span></div><div class="casAuthors">Babu, Mohan; Beloglazova, Natalia; Flick, Robert; Graham, Chris; Skarina, Tatiana; Nocek, Boguslaw; Gagarinova, Alla; Pogoutse, Oxana; Brown, Greg; Binkowski, Andrew; Phanse, Sadhna; Joachimiak, Andrzej; Koonin, Eugene V.; Savchenko, Alexei; Emili, Andrew; Greenblatt, Jack; Edwards, Aled M.; Yakunin, Alexander F.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">484-502</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Clustered Regularly Interspaced Short Palindromic Repeats (CRISPRs) and the assocd. proteins (Cas) comprise a system of adaptive immunity against viruses and plasmids in prokaryotes.  Cas1 is a CRISPR-assocd. protein that is common to all CRISPR-contg. prokaryotes but its function remains obscure.  Here we show that the purified Cas1 protein of Escherichia coli (YgbT) exhibits nuclease activity against single-stranded and branched DNAs including Holliday junctions, replication forks and 5'-flaps.  The crystal structure of YgbT and site-directed mutagenesis have revealed the potential active site.  Genome-wide screens show that YgbT phys. and genetically interacts with key components of DNA repair systems, including recB, recC and ruvB.  Consistent with these findings, the ygbT deletion strain showed increased sensitivity to DNA damage and impaired chromosomal segregation.  Similar phenotypes were obsd. in strains with deletion of CRISPR clusters, suggesting that the function of YgbT in repair involves interaction with the CRISPRs.  These results show that YgbT belongs to a novel, structurally distinct family of nucleases acting on branched DNAs and suggest that, in addn. to antiviral immunity, at least some components of the CRISPR-Cas system have a function in DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6QELoEg3Wi7Vg90H21EOLACvtfcHk0lgYiKa6G68YHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsl2qtLs%253D&md5=d3f0b4ed7ebcc92e5199f403da227819</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2010.07465.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2010.07465.x%26sid%3Dliteratum%253Aachs%26aulast%3DBabu%26aufirst%3DM.%26aulast%3DBeloglazova%26aufirst%3DN.%26aulast%3DFlick%26aufirst%3DR.%26aulast%3DGraham%26aufirst%3DC.%26aulast%3DSkarina%26aufirst%3DT.%26aulast%3DNocek%26aufirst%3DB.%26aulast%3DGagarinova%26aufirst%3DA.%26aulast%3DPogoutse%26aufirst%3DO.%26aulast%3DBrown%26aufirst%3DG.%26aulast%3DBinkowski%26aufirst%3DA.%26aulast%3DPhanse%26aufirst%3DS.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DKoonin%26aufirst%3DE.%2BV.%26aulast%3DSavchenko%26aufirst%3DA.%26aulast%3DEmili%26aufirst%3DA.%26aulast%3DGreenblatt%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DYakunin%26aufirst%3DA.%2BF.%26atitle%3DA%2520dual%2520function%2520of%2520the%2520CRISPR-Cas%2520system%2520in%2520bacterial%2520antivirus%2520immunity%2520and%2520DNA%2520repair%26jtitle%3DMol.%2520Microbiol.%26date%3D2011%26volume%3D79%26issue%3D2%26spage%3D484%26epage%3D502%26doi%3D10.1111%2Fj.1365-2958.2010.07465.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">TM-align: a protein structure alignment algorithm based on the TM-score</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">2302</span><span class="NLM_x">–</span> <span class="NLM_lpage">2309</span><span class="refDoi"> DOI: 10.1093/nar/gki524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgki524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=15849316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsl2gsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=2302-2309&issue=7&author=Y.+Zhangauthor=J.+Skolnick&title=TM-align%3A+a+protein+structure+alignment+algorithm+based+on+the+TM-score&doi=10.1093%2Fnar%2Fgki524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">TM-align: a protein structure alignment algorithm based on the TM-score</span></div><div class="casAuthors">Zhang, Yang; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2302-2309</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have developed TM-align, a new algorithm to identify the best structural alignment between protein pairs that combines the TM-score rotation matrix and Dynamic Programming (DP).  The algorithm is ∼4 times faster than CE and 20 times faster than DALI and SAL.  On av., the resulting structure alignments have higher accuracy and coverage than those provided by these most often-used methods.  TM-align is applied to an all-against-all structure comparison of 10 515 representative protein chains from the Protein Data Bank (PDB) with a sequence identity cutoff <95%: 1996 distinct folds are found when a TM-score threshold of 0.5 is used.  We also use TM-align to match the models predicted by TASSER for solved non-homologous proteins in PDB.  For both folded and misfolded models, TM-align can almost always find close structural analogs, with an av. root mean square deviation, RMSD, of 3 Å and 87% alignment coverage.  Nevertheless, there exists a significant correlation between the correctness of the predicted structure and the structural similarity of the model to the other proteins in the PDB.  This correlation could be used to assist in model selection in blind protein structure predictions.  The TM-align program is freely downloadable at http://bioinformatics.buffalo.edu/TM-align.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonPrM9Mg2FZ7Vg90H21EOLACvtfcHk0lgYiKa6G68YHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsl2gsLY%253D&md5=15689409a6d9281184a7f943f5cc45d3</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki524%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DTM-align%253A%2520a%2520protein%2520structure%2520alignment%2520algorithm%2520based%2520on%2520the%2520TM-score%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26issue%3D7%26spage%3D2302%26epage%3D2309%26doi%3D10.1093%2Fnar%2Fgki524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">Computational discovery and experimental verification of tyrosine kinase inhibitor pazopanib for the reversal of memory and cognitive deficits in rat model neurodegeneration</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">2812</span><span class="NLM_x">–</span> <span class="NLM_lpage">2821</span><span class="refDoi"> DOI: 10.1039/C4SC03416C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1039%2FC4SC03416C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntVGjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=2812-2821&issue=5&author=Y.+Yangauthor=G.+Liauthor=D.+Zhaoauthor=H.+Yuauthor=X.+Zhengauthor=X.+Pengauthor=X.+Zhangauthor=T.+Fuauthor=X.+Huauthor=M.+Niuauthor=X.+Jiauthor=L.+Zouauthor=J.+Wang&title=Computational+discovery+and+experimental+verification+of+tyrosine+kinase+inhibitor+pazopanib+for+the+reversal+of+memory+and+cognitive+deficits+in+rat+model+neurodegeneration&doi=10.1039%2FC4SC03416C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Computational discovery and experimental verification of tyrosine kinase inhibitor pazopanib for the reversal of memory and cognitive deficits in rat model neurodegeneration</span></div><div class="casAuthors">Yang, Yongliang; Li, Guohui; Zhao, Dongyu; Yu, Haoyang; Zheng, Xiliang; Peng, Xiangda; Zhang, Xiaoe; Fu, Ting; Hu, Xiaoqing; Niu, Mingshan; Ji, Xuefei; Zou, Libo; Wang, Jin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2812-2821</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cognition and memory impairment are hallmarks of the pathol. cascade of various neurodegenerative disorders.  Herein, we developed a novel computational strategy with two-dimensional virtual screening for not only affinity but also specificity.  We integrated the two-dimensional virtual screening with ligand screening for 3D shape, electrostatic similarity and local binding site similarity to find existing drugs that may reduce the signs of cognitive deficits.  For the first time, we found that pazopanib, a tyrosine kinase inhibitor marketed for cancer treatment, inhibits acetylcholinesterase (AchE) activities at sub-micromolar concn.  We evaluated and compared the effects of intragastrically-administered pazopanib with donepezil, a marketed AchE inhibitor, in cognitive and behavioral assays including the novel object recognition test, Y maze and Morris water maze test.  Surprisingly, we found that pazopanib can restore memory loss and cognitive dysfunction to a similar extent as donepezil in a dosage of 15 mg kg-1, only one fifth of the equiv. clin. dosage for cancer treatment.  Furthermore, we demonstrated that pazopanib dramatically enhances the hippocampal Ach levels and increases the expression of the synaptic marker SYP.  These findings suggest that pazopanib may become a viable treatment option for memory and cognitive deficits with a good safety profile in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouyZIk9hoxDLVg90H21EOLACvtfcHk0lhp3qDZ9xDmSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntVGjtw%253D%253D&md5=5d6efdb9187c21310061df26971e8ba7</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1039%2FC4SC03416C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4SC03416C%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DNiu%26aufirst%3DM.%26aulast%3DJi%26aufirst%3DX.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DComputational%2520discovery%2520and%2520experimental%2520verification%2520of%2520tyrosine%2520kinase%2520inhibitor%2520pazopanib%2520for%2520the%2520reversal%2520of%2520memory%2520and%2520cognitive%2520deficits%2520in%2520rat%2520model%2520neurodegeneration%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26issue%3D5%26spage%3D2812%26epage%3D2821%26doi%3D10.1039%2FC4SC03416C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">A computational procedure for determining energetically favorable binding sites on biologically important macromolecules</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">849</span><span class="NLM_x">–</span> <span class="NLM_lpage">857</span><span class="refDoi"> DOI: 10.1021/jm00145a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00145a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaL2MXktVeit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=849-857&issue=7&author=P.+J.+Goodford&title=A+computational+procedure+for+determining+energetically+favorable+binding+sites+on+biologically+important+macromolecules&doi=10.1021%2Fjm00145a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">A computational procedure for determining energetically favorable binding sites on biologically important macromolecules</span></div><div class="casAuthors">Goodford, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">849-57</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The interaction of a probe group with a protein of known structure is computed at sample positions throughout and around the macromol., giving an array of energy values.  The probe include H2O, the Me group, amine-N, carboxy-O, and OH.  Contour surfaces at appropriate energy levels are calcd. for each probe and displayed by computer graphics together with the protein structure.  Contours at neg. energy levels delineate regions of attraction between probe and protein and are found at known ligand binding clefts in particular.  The contours also enable other regions of attraction to be identified and facilitate the interpretation of protein-ligand energetics.  They may, therefore, be of value for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3tYN3Fc76dLVg90H21EOLACvtfcHk0lhp3qDZ9xDmSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXktVeit7c%253D&md5=03e9d22787f16c760ef0dc29ca02aa88</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fjm00145a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00145a002%26sid%3Dliteratum%253Aachs%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DA%2520computational%2520procedure%2520for%2520determining%2520energetically%2520favorable%2520binding%2520sites%2520on%2520biologically%2520important%2520macromolecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26issue%3D7%26spage%3D849%26epage%3D857%26doi%3D10.1021%2Fjm00145a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group">Feldman, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labute, P.</span><span> </span><span class="NLM_article-title">Pocket similarity: are alpha carbons enough?</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1466</span><span class="NLM_x">–</span> <span class="NLM_lpage">1475</span><span class="refDoi"> DOI: 10.1021/ci100210c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100210c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVSgsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1466-1475&issue=8&author=H.+J.+Feldmanauthor=P.+Labute&title=Pocket+similarity%3A+are+alpha+carbons+enough%3F&doi=10.1021%2Fci100210c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Pocket Similarity: Are α Carbons Enough?</span></div><div class="casAuthors">Feldman, Howard J.; Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1466-1475</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for measuring protein pocket similarity was devised, using only the α carbon positions of the pocket residues.  Pockets were compared pairwise using an exhaustive three-dimensional Cα common subset search, grouping residues by physicochem. properties.  At least five Cα matches were required for each hit, and distances between corresponding points were fit to an Extreme Value Distribution resulting in a probabilistic score or likelihood for any given superposition.  A set of 85 structures from 13 diverse protein families was clustered based on binding sites alone, using this score.  It was also successfully used to cluster 25 kinases into a no. of subfamilies.  Using a test kinase query to retrieve other kinase pockets, a specificity of 99.2% and sensitivity of 97.5% could be achieved using an appropriate cutoff score.  The search itself took from 2 to 10 min on a single 3.4 GHz CPU to search the entire Protein Data Bank (133,800 pockets), depending on the no. of hits returned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowvtYX4jdPnrVg90H21EOLACvtfcHk0lhp3qDZ9xDmSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVSgsbk%253D&md5=a89757babe6ecdcd012be4913a15b419</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Fci100210c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100210c%26sid%3Dliteratum%253Aachs%26aulast%3DFeldman%26aufirst%3DH.%2BJ.%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DPocket%2520similarity%253A%2520are%2520alpha%2520carbons%2520enough%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D8%26spage%3D1466%26epage%3D1475%26doi%3D10.1021%2Fci100210c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group">de Brevern, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etchebest, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazout, S.</span><span> </span><span class="NLM_article-title">Bayesian probabilistic approach for predicting backbone structures in terms of protein blocks</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1002/1097-0134(20001115)41:3<271::AID-PROT10>3.0.CO;2-Z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2F1097-0134%2820001115%2941%3A3%3C271%3A%3AAID-PROT10%3E3.0.CO%3B2-Z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2000&pages=271-287&issue=3&author=A.+G.+de+Brevernauthor=C.+Etchebestauthor=S.+Hazout&title=Bayesian+probabilistic+approach+for+predicting+backbone+structures+in+terms+of+protein+blocks&doi=10.1002%2F1097-0134%2820001115%2941%3A3%3C271%3A%3AAID-PROT10%3E3.0.CO%3B2-Z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1002%2F1097-0134%2820001115%2941%3A3%3C271%3A%3AAID-PROT10%3E3.0.CO%3B2-Z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1097-0134%252820001115%252941%253A3%253C271%253A%253AAID-PROT10%253E3.0.CO%253B2-Z%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBrevern%26aufirst%3DA.%2BG.%26aulast%3DEtchebest%26aufirst%3DC.%26aulast%3DHazout%26aufirst%3DS.%26atitle%3DBayesian%2520probabilistic%2520approach%2520for%2520predicting%2520backbone%2520structures%2520in%2520terms%2520of%2520protein%2520blocks%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2000%26volume%3D41%26issue%3D3%26spage%3D271%26epage%3D287%26doi%3D10.1002%2F1097-0134%2820001115%2941%3A3%3C271%3A%3AAID-PROT10%3E3.0.CO%3B2-Z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">Atomic property fields: generalized 3D pharmacophoric potential for automated ligand superposition, pharmacophore elucidation and 3D QSAR</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00605.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1111%2Fj.1747-0285.2007.00605.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18069986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvF2lsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=15-27&issue=1&author=M.+Totrov&title=Atomic+property+fields%3A+generalized+3D+pharmacophoric+potential+for+automated+ligand+superposition%2C+pharmacophore+elucidation+and+3D+QSAR&doi=10.1111%2Fj.1747-0285.2007.00605.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Atomic property fields: generalized 3D pharmacophoric potential for automated ligand superposition, pharmacophore elucidation and 3D QSAR</span></div><div class="casAuthors">Totrov, Maxim</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-27</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The at. property fields (APF) concept is introduced as a continuous, multi-component 3D potential that reflects preferences for various at. properties at each point in space.  At. property field-based approaches to several key problems in the field of ligand structure-based rational drug discovery and design are investigated.  The superposition of ligands on one or multiple mol. templates is performed by Monte-Carlo minimization in the at. property fields potentials combined with std. force-field energy.  The approach is extended to multiple flexible ligand alignments using an iterative procedure, Self-Consistent at. Property Fields by Optimization (SCAPFOld).  The application of at. property fields and SCAPFOld for virtual ligand screening and 3D Quantitive Structure-Activity Relationship (QSAR) is tested on published benchmarks.  The new methods are shown to perform competitively in comparison to current state-of-the-art methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQFMXqW3d8ErVg90H21EOLACvtfcHk0lg6AY-mYl-zlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvF2lsL8%253D&md5=d66ce22225d4d9dfeed56196af4025d1</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00605.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00605.x%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DAtomic%2520property%2520fields%253A%2520generalized%25203D%2520pharmacophoric%2520potential%2520for%2520automated%2520ligand%2520superposition%252C%2520pharmacophore%2520elucidation%2520and%25203D%2520QSAR%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2008%26volume%3D71%26issue%3D1%26spage%3D15%26epage%3D27%26doi%3D10.1111%2Fj.1747-0285.2007.00605.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Morphological similarity: a 3D molecular similarity method correlated with protein-ligand recognition</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">213</span><span class="refDoi"> DOI: 10.1023/A:1008100132405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1023%2FA%3A1008100132405" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2000&pages=199-213&issue=2&author=A.+N.+Jain&title=Morphological+similarity%3A+a+3D+molecular+similarity+method+correlated+with+protein-ligand+recognition&doi=10.1023%2FA%3A1008100132405"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008100132405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008100132405%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DMorphological%2520similarity%253A%2520a%25203D%2520molecular%2520similarity%2520method%2520correlated%2520with%2520protein-ligand%2520recognition%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2000%26volume%3D14%26issue%3D2%26spage%3D199%26epage%3D213%26doi%3D10.1023%2FA%3A1008100132405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group">Spitzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleves, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varela, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Protein function annotation by local binding site surface similarity</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">694</span><span class="refDoi"> DOI: 10.1002/prot.24450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.24450" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2014&pages=679-694&issue=4&author=R.+Spitzerauthor=A.+E.+Clevesauthor=R.+Varelaauthor=A.+N.+Jain&title=Protein+function+annotation+by+local+binding+site+surface+similarity&doi=10.1002%2Fprot.24450"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1002%2Fprot.24450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24450%26sid%3Dliteratum%253Aachs%26aulast%3DSpitzer%26aufirst%3DR.%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DVarela%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DProtein%2520function%2520annotation%2520by%2520local%2520binding%2520site%2520surface%2520similarity%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2014%26volume%3D82%26issue%3D4%26spage%3D679%26epage%3D694%26doi%3D10.1002%2Fprot.24450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Relibase: design and development of a database for comprehensive analysis of protein-ligand interactions</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)01408-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2802%2901408-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2003&pages=607-620&issue=2&author=M.+Hendlichauthor=A.+Bergnerauthor=J.+G%C3%BCntherauthor=G.+Klebe&title=Relibase%3A+design+and+development+of+a+database+for+comprehensive+analysis+of+protein-ligand+interactions&doi=10.1016%2FS0022-2836%2802%2901408-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2901408-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252901408-0%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DBergner%26aufirst%3DA.%26aulast%3DG%25C3%25BCnther%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DRelibase%253A%2520design%2520and%2520development%2520of%2520a%2520database%2520for%2520comprehensive%2520analysis%2520of%2520protein-ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D326%26issue%3D2%26spage%3D607%26epage%3D620%26doi%3D10.1016%2FS0022-2836%2802%2901408-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group">Günther, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Utilising structural knowledge in drug design strategies: applications using Relibase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">621</span><span class="NLM_x">–</span> <span class="NLM_lpage">636</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)01409-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2802%2901409-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2003&pages=621-636&issue=2&author=J.+G%C3%BCntherauthor=A.+Bergnerauthor=M.+Hendlichauthor=G.+Klebe&title=Utilising+structural+knowledge+in+drug+design+strategies%3A+applications+using+Relibase&doi=10.1016%2FS0022-2836%2802%2901409-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2901409-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252901409-2%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25BCnther%26aufirst%3DJ.%26aulast%3DBergner%26aufirst%3DA.%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DUtilising%2520structural%2520knowledge%2520in%2520drug%2520design%2520strategies%253A%2520applications%2520using%2520Relibase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D326%26issue%3D2%26spage%3D621%26epage%3D636%26doi%3D10.1016%2FS0022-2836%2802%2901409-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span> </span><span class="NLM_article-title">SURFNET: a program for visualizing molecular surfaces, cavities, and intermolecular interactions</span> <span class="citation_source-journal">J. Mol. Graphics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">323</span><span class="NLM_x">–</span> <span class="NLM_lpage">330, 307–308</span><span class="refDoi"> DOI: 10.1016/0263-7855(95)00073-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2F0263-7855%2895%2900073-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=8603061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2MXpslentbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1995&pages=323-330%2C+307%E2%80%93308&issue=5&author=R.+A.+Laskowski&title=SURFNET%3A+a+program+for+visualizing+molecular+surfaces%2C+cavities%2C+and+intermolecular+interactions&doi=10.1016%2F0263-7855%2895%2900073-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">SURFNET: a program for visualizing molecular surfaces, cavities, and intermolecular interactions</span></div><div class="casAuthors">Laskowski, Roman A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">323-30</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The SURFNET program generates mol. surfaces and gaps between surfaces from 3D coordinates supplied in a PDB-format file.  The gap regions can correspond to the voids between two or more mols., or to the internal cavities and surface grooves within a single mol.  The program is particularly useful in clearly delineating the regions of the active site of a protein.  It can also generate 3D contour surfaces of the d. distributions of any set of 3D data points.  All output surfaces can be viewed interactively, along with the mols. or data points in question, using some of the best-known mol. modeling packages.  In addn., PostScript output is available, and the generated surfaces can be rendered using various other graphics packages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwU6uVvAFySbVg90H21EOLACvtfcHk0lg6AY-mYl-zlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpslentbs%253D&md5=cd68217416ecf26be347bec9a1788837</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2895%2900073-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252895%252900073-9%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26atitle%3DSURFNET%253A%2520a%2520program%2520for%2520visualizing%2520molecular%2520surfaces%252C%2520cavities%252C%2520and%2520intermolecular%2520interactions%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1995%26volume%3D13%26issue%3D5%26spage%3D323%26epage%3D330%252C%2520307%25E2%2580%2593308%26doi%3D10.1016%2F0263-7855%2895%2900073-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group">Glaser, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pupko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ben-Tal, N.</span><span> </span><span class="NLM_article-title">The ConSurf-HSSP database: the mapping of evolutionary conservation among homologs onto PDB structures</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">610</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.1002/prot.20305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fprot.20305" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2005&pages=610-617&issue=3&author=F.+Glaserauthor=Y.+Rosenbergauthor=A.+Kesselauthor=T.+Pupkoauthor=N.+Ben-Tal&title=The+ConSurf-HSSP+database%3A+the+mapping+of+evolutionary+conservation+among+homologs+onto+PDB+structures&doi=10.1002%2Fprot.20305"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1002%2Fprot.20305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20305%26sid%3Dliteratum%253Aachs%26aulast%3DGlaser%26aufirst%3DF.%26aulast%3DRosenberg%26aufirst%3DY.%26aulast%3DKessel%26aufirst%3DA.%26aulast%3DPupko%26aufirst%3DT.%26aulast%3DBen-Tal%26aufirst%3DN.%26atitle%3DThe%2520ConSurf-HSSP%2520database%253A%2520the%2520mapping%2520of%2520evolutionary%2520conservation%2520among%2520homologs%2520onto%2520PDB%2520structures%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D58%26issue%3D3%26spage%3D610%26epage%3D617%26doi%3D10.1002%2Fprot.20305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group">McLachlan, A. D.</span><span> </span><span class="NLM_article-title">Repeating sequences and gene duplication in proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">417</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1016/0022-2836(72)90508-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2F0022-2836%2872%2990508-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=1972&pages=417-437&issue=2&author=A.+D.+McLachlan&title=Repeating+sequences+and+gene+duplication+in+proteins&doi=10.1016%2F0022-2836%2872%2990508-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2872%2990508-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252872%252990508-6%26sid%3Dliteratum%253Aachs%26aulast%3DMcLachlan%26aufirst%3DA.%2BD.%26atitle%3DRepeating%2520sequences%2520and%2520gene%2520duplication%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1972%26volume%3D64%26issue%3D2%26spage%3D417%26epage%3D437%26doi%3D10.1016%2F0022-2836%2872%2990508-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group">Henikoff, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henikoff, J. G.</span><span> </span><span class="NLM_article-title">Amino acid substitution matrices from protein blocks</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x"> (</span><span class="NLM_issue">22</span><span class="NLM_x">) </span> <span class="NLM_fpage">10915</span><span class="NLM_x">–</span> <span class="NLM_lpage">10919</span><span class="refDoi"> DOI: 10.1073/pnas.89.22.10915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1073%2Fpnas.89.22.10915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1438297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK3sXjsFCgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=10915-10919&issue=22&author=S.+Henikoffauthor=J.+G.+Henikoff&title=Amino+acid+substitution+matrices+from+protein+blocks&doi=10.1073%2Fpnas.89.22.10915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid substitution matrixes from protein blocks</span></div><div class="casAuthors">Henikoff, Steven; Henikoff, Jorja G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10915-19</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Methods for alignment of protein sequences typically measure similarity by using a substitution matrix with scores for all possible exchanges of one amino acid with another.  The most widely used matrixes are based on the Dayhoff model of evolutionary rates.  Using a different approach, the authors derived substitution matrixes from about 2000 blocks of aligned sequence segments characterizing more than 500 groups of related proteins.  This led to marked improvements in alignments and in searches using queries from each of the groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY5y4-FlgzWrVg90H21EOLACvtfcHk0li01NqnxcTDFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXjsFCgsQ%253D%253D&md5=3c4bee915654ac0b98b8d4aaafd44ebc</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1073%2Fpnas.89.22.10915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.89.22.10915%26sid%3Dliteratum%253Aachs%26aulast%3DHenikoff%26aufirst%3DS.%26aulast%3DHenikoff%26aufirst%3DJ.%2BG.%26atitle%3DAmino%2520acid%2520substitution%2520matrices%2520from%2520protein%2520blocks%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1992%26volume%3D89%26issue%3D22%26spage%3D10915%26epage%3D10919%26doi%3D10.1073%2Fpnas.89.22.10915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group">Konc, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janežič, D.</span><span> </span><span class="NLM_article-title">An improved branch and bound algorithm for the maximum clique problem</span> <span class="citation_source-journal">MATCH</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">590</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2007&pages=569-590&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=An+improved+branch+and+bound+algorithm+for+the+maximum+clique+problem"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DAn%2520improved%2520branch%2520and%2520bound%2520algorithm%2520for%2520the%2520maximum%2520clique%2520problem%26jtitle%3DMATCH%26date%3D2007%26volume%3D58%26spage%3D569%26epage%3D590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group">Binkowski, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span> </span><span class="NLM_article-title">Inferring functional relationships of proteins from local sequence and spatial surface patterns</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">526</span><span class="refDoi"> DOI: 10.1016/S0022-2836(03)00882-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0022-2836%2803%2900882-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2003&pages=505-526&issue=2&author=T.+A.+Binkowskiauthor=L.+Adamianauthor=J.+Liang&title=Inferring+functional+relationships+of+proteins+from+local+sequence+and+spatial+surface+patterns&doi=10.1016%2FS0022-2836%2803%2900882-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2803%2900882-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252803%252900882-9%26sid%3Dliteratum%253Aachs%26aulast%3DBinkowski%26aufirst%3DT.%2BA.%26aulast%3DAdamian%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DJ.%26atitle%3DInferring%2520functional%2520relationships%2520of%2520proteins%2520from%2520local%2520sequence%2520and%2520spatial%2520surface%2520patterns%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D332%26issue%3D2%26spage%3D505%26epage%3D526%26doi%3D10.1016%2FS0022-2836%2803%2900882-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group">Binkowski, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naghibzadeh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span> </span><span class="NLM_article-title">CASTp: Computed Atlas of Surface Topography of proteins</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x"> (</span><span class="NLM_issue">13</span><span class="NLM_x">) </span> <span class="NLM_fpage">3352</span><span class="NLM_x">–</span> <span class="NLM_lpage">3355</span><span class="refDoi"> DOI: 10.1093/nar/gkg512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkg512" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=3352-3355&issue=13&author=T.+A.+Binkowskiauthor=S.+Naghibzadehauthor=J.+Liang&title=CASTp%3A+Computed+Atlas+of+Surface+Topography+of+proteins&doi=10.1093%2Fnar%2Fgkg512"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkg512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkg512%26sid%3Dliteratum%253Aachs%26aulast%3DBinkowski%26aufirst%3DT.%2BA.%26aulast%3DNaghibzadeh%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DJ.%26atitle%3DCASTp%253A%2520Computed%2520Atlas%2520of%2520Surface%2520Topography%2520of%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2003%26volume%3D31%26issue%3D13%26spage%3D3352%26epage%3D3355%26doi%3D10.1093%2Fnar%2Fgkg512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group">Pearson, W. R.</span><span> </span><span class="NLM_article-title">Searching protein sequence libraries: comparison of the sensitivity and selectivity of the Smith-Waterman and FASTA algorithms</span> <span class="citation_source-journal">Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">635</span><span class="NLM_x">–</span> <span class="NLM_lpage">650</span><span class="refDoi"> DOI: 10.1016/0888-7543(91)90071-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2F0888-7543%2891%2990071-L" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=635-650&issue=3&author=W.+R.+Pearson&title=Searching+protein+sequence+libraries%3A+comparison+of+the+sensitivity+and+selectivity+of+the+Smith-Waterman+and+FASTA+algorithms&doi=10.1016%2F0888-7543%2891%2990071-L"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1016%2F0888-7543%2891%2990071-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0888-7543%252891%252990071-L%26sid%3Dliteratum%253Aachs%26aulast%3DPearson%26aufirst%3DW.%2BR.%26atitle%3DSearching%2520protein%2520sequence%2520libraries%253A%2520comparison%2520of%2520the%2520sensitivity%2520and%2520selectivity%2520of%2520the%2520Smith-Waterman%2520and%2520FASTA%2520algorithms%26jtitle%3DGenomics%26date%3D1991%26volume%3D11%26issue%3D3%26spage%3D635%26epage%3D650%26doi%3D10.1016%2F0888-7543%2891%2990071-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group">Shulman-Peleg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfson, H. J.</span><span> </span><span class="NLM_article-title">SiteEngines: recognition and comparison of binding sites and protein-protein interfaces</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">W337</span><span class="NLM_x">–</span> <span class="NLM_lpage">W341</span><div class="note"><p class="first last">(Web Server issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gki482</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgki482" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=W337-W341&issue=Suppl.+2&author=A.+Shulman-Pelegauthor=R.+Nussinovauthor=H.+J.+Wolfson&title=SiteEngines%3A+recognition+and+comparison+of+binding+sites+and+protein-protein+interfaces&doi=10.1093%2Fnar%2Fgki482"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki482%26sid%3Dliteratum%253Aachs%26aulast%3DShulman-Peleg%26aufirst%3DA.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DWolfson%26aufirst%3DH.%2BJ.%26atitle%3DSiteEngines%253A%2520recognition%2520and%2520comparison%2520of%2520binding%2520sites%2520and%2520protein-protein%2520interfaces%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26issue%3DSuppl.%25202%26spage%3DW337%26epage%3DW341%26doi%3D10.1093%2Fnar%2Fgki482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group">Kabsch, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, C.</span><span> </span><span class="NLM_article-title">Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2577</span><span class="NLM_x">–</span> <span class="NLM_lpage">2637</span><span class="refDoi"> DOI: 10.1002/bip.360221211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fbip.360221211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=6667333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaL2cXkslegtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1983&pages=2577-2637&issue=12&author=W.+Kabschauthor=C.+Sander&title=Dictionary+of+protein+secondary+structure%3A+pattern+recognition+of+hydrogen-bonded+and+geometrical+features&doi=10.1002%2Fbip.360221211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Dictionary of protein secondary structure:  pattern recognition of hydrogen-bonded and geometrical features</span></div><div class="casAuthors">Kabsch, Wolfgang; Sander, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2577-637</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">For a successful anal. of the relation between amino acid sequence and protein structure, an unambiguous and phys. meaningful definition of secondary structure is essential.  A set of simple and phys. motivated criteria for secondary structure, programmed as a pattern-recognition process of H-bonded and geometrical features extd. from x-ray coordinates were developed.  Cooperative secondary structure is recognized as repeats of the elementary H-bonding patterns turn and bridge.  Repeating turns are helixes, repeating bridges are ladders, connected ladders are sheets.  Geometric structure is defined in terms of the concepts torsion and curvature of differential geometry.  Local chain chirality is the torsional handedness of 4 consecutive Cα positions and is pos. for right-handed helixes and neg. for ideal twisted β-sheets.  Curved pieces are defined as bends.  Solvent exposure is given as the no. of H2O mols. in possible contact with a residue.  The end result is a compilation of the primary structure, including SS bonds, secondary structure, and solvent exposure of 62 different globular proteins.  The presentation is in linear form: strip graphs for an overall view and strip tables for the details of each of 10,925 residues.  The dictionary is also available in computer-readable form for protein structure prediction work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph0hYRRkAqkbVg90H21EOLACvtfcHk0lgkNBeVYJbJGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXkslegtQ%253D%253D&md5=a146e923e6c49cb542d0ad24a399d0f0</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1002%2Fbip.360221211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360221211%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26aulast%3DSander%26aufirst%3DC.%26atitle%3DDictionary%2520of%2520protein%2520secondary%2520structure%253A%2520pattern%2520recognition%2520of%2520hydrogen-bonded%2520and%2520geometrical%2520features%26jtitle%3DBiopolymers%26date%3D1983%26volume%3D22%26issue%3D12%26spage%3D2577%26epage%3D2637%26doi%3D10.1002%2Fbip.360221211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group">Kihara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">The PDB is a covering set of small protein structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">334</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">793</span><span class="NLM_x">–</span> <span class="NLM_lpage">802</span><span class="refDoi"> DOI: 10.1016/j.jmb.2003.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.jmb.2003.10.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2003&pages=793-802&issue=4&author=D.+Kiharaauthor=J.+Skolnick&title=The+PDB+is+a+covering+set+of+small+protein+structures&doi=10.1016%2Fj.jmb.2003.10.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2003.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2003.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DKihara%26aufirst%3DD.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DThe%2520PDB%2520is%2520a%2520covering%2520set%2520of%2520small%2520protein%2520structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D334%26issue%3D4%26spage%3D793%26epage%3D802%26doi%3D10.1016%2Fj.jmb.2003.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group">Dayhoff, M. O.; Schwartz, R. M.; Orcutt, B. C.</span><span> </span><span class="NLM_article-title">A model of evolutionary change in proteins</span>. In  <span class="citation_source-book">Atlas of Protein Sequence and Structure</span>; <span class="NLM_contrib-group">Dayhoff, M. O.</span>, Ed.; <span class="NLM_publisher-name">National Biomedical Research Foundation</span>: <span class="NLM_publisher-loc">Washington, DC</span>,<span class="NLM_x"> </span><span class="NLM_year">1978</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">345</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1978&pages=345-352&author=M.+O.+Dayhoff&author=R.+M.+Schwartz&author=B.+C.+Orcuttauthor=M.+O.+Dayhoff&title=Atlas+of+Protein+Sequence+and+Structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDayhoff%26aufirst%3DM.%2BO.%26atitle%3DA%2520model%2520of%2520evolutionary%2520change%2520in%2520proteins%26btitle%3DAtlas%2520of%2520Protein%2520Sequence%2520and%2520Structure%26aulast%3DDayhoff%26aufirst%3DM.%2BO.%26pub%3DNational%2520Biomedical%2520Research%2520Foundation%26date%3D1978%26spage%3D345%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group">Tseng, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupree, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.-H.</span><span> </span><span class="NLM_article-title">SplitPocket: identification of protein functional surfaces and characterization of their spatial patterns</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">W384</span><span class="NLM_x">–</span> <span class="NLM_lpage">W389</span><div class="note"><p class="first last">(Web Server issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkp308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkp308" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=W384-W389&issue=Suppl.+2&author=Y.+Y.+Tsengauthor=C.+Dupreeauthor=Z.+J.+Chenauthor=W.-H.+Li&title=SplitPocket%3A+identification+of+protein+functional+surfaces+and+characterization+of+their+spatial+patterns&doi=10.1093%2Fnar%2Fgkp308"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp308%26sid%3Dliteratum%253Aachs%26aulast%3DTseng%26aufirst%3DY.%2BY.%26aulast%3DDupree%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26aulast%3DLi%26aufirst%3DW.-H.%26atitle%3DSplitPocket%253A%2520identification%2520of%2520protein%2520functional%2520surfaces%2520and%2520characterization%2520of%2520their%2520spatial%2520patterns%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3DSuppl.%25202%26spage%3DW384%26epage%3DW389%26doi%3D10.1093%2Fnar%2Fgkp308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group">Wei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J. T.</span><span> </span><span class="NLM_article-title">Using the radial distributions of physical features to compare amino acid environments and align amino acid sequences</span> <span class="citation_source-journal">Pac. Symp. Biocomput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=9390315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2sXns1Sru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=465-476&author=L.+Weiauthor=R.+B.+Altmanauthor=J.+T.+Chang&title=Using+the+radial+distributions+of+physical+features+to+compare+amino+acid+environments+and+align+amino+acid+sequences"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Using the radial distributions of physical features to compare amino acid environments and align amino acid sequences</span></div><div class="casAuthors">Wei, Liping; Altman, Russ B.; Chang, Jeffrey T.</div><div class="citationInfo"><span class="NLM_cas:title">Pacific Symposium on Biocomputing '97, Maui, Hawaii, Jan. 6-9, 1997</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">465-476</span>CODEN:
                <span class="NLM_cas:coden">65JZAR</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">World Scientific</span>)
        </div><div class="casAbstract">We have performed a comprehensive anal. of the microenvironments surrounding the twenty amino acids.  Our anal. includes comparison of amino acid environments with random control environments as well as with each of the other amino acid environments.  We describe the amino acid environments with a set of 21 features summarizing at., chem. group, residue, and secondary structural features.  The environments are divided into radial shells of 1 Å thickness to represent the distance of the features from the amino acid Cβ atoms.  We make the results of our anal. available graphically over the world wide web.  To illustrate the validity and utility of our anal., we used the amino acid comparative profiles to construct a substitution matrix, the WAC matrix, based on a simple summary of the computed environmental differences.  We compared our matrix to BLOSUM62 and PAM250 in BLAST searches with query sequences selected from 39 protein families found in the PROSITE database.  Although BLOSUM62 was the most sensitive matrix overall, our matrix was more sensitive for some families, and exhibited overall performance similar to PAM250.  Our results suggest that the radial distribution of biochem. and biophys. features is useful for comparing amino acid environments, and that similarity matrixes based on the geometric distribution of features around amino acids may produce improved search sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp77neBQFhd17Vg90H21EOLACvtfcHk0lgkNBeVYJbJGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXns1Sru7o%253D&md5=3c662d5cf77b2e4ab9cb344a4cf319ee</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DL.%26aulast%3DAltman%26aufirst%3DR.%2BB.%26aulast%3DChang%26aufirst%3DJ.%2BT.%26atitle%3DUsing%2520the%2520radial%2520distributions%2520of%2520physical%2520features%2520to%2520compare%2520amino%2520acid%2520environments%2520and%2520align%2520amino%2520acid%2520sequences%26jtitle%3DPac.%2520Symp.%2520Biocomput.%26date%3D1997%26spage%3D465%26epage%3D476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group">Yoon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebert, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, E.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Micheli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, R. B.</span><span> </span><span class="NLM_article-title">Clustering protein environments for function prediction: finding PROSITE motifs in 3D</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 4</span><span class="NLM_x">) </span> <span class="NLM_fpage">S10</span><span class="refDoi"> DOI: 10.1186/1471-2105-8-S4-S10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-8-S4-S10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=S10&issue=Suppl.+4&author=S.+Yoonauthor=J.+C.+Ebertauthor=E.-Y.+Chungauthor=G.+De+Micheliauthor=R.+B.+Altman&title=Clustering+protein+environments+for+function+prediction%3A+finding+PROSITE+motifs+in+3D&doi=10.1186%2F1471-2105-8-S4-S10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-S4-S10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-8-S4-S10%26sid%3Dliteratum%253Aachs%26aulast%3DYoon%26aufirst%3DS.%26aulast%3DEbert%26aufirst%3DJ.%2BC.%26aulast%3DChung%26aufirst%3DE.-Y.%26aulast%3DDe%2BMicheli%26aufirst%3DG.%26aulast%3DAltman%26aufirst%3DR.%2BB.%26atitle%3DClustering%2520protein%2520environments%2520for%2520function%2520prediction%253A%2520finding%2520PROSITE%2520motifs%2520in%25203D%26jtitle%3DBMC%2520Bioinf.%26date%3D2007%26volume%3D8%26issue%3DSuppl.%25204%26spage%3DS10%26doi%3D10.1186%2F1471-2105-8-S4-S10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Protein structure alignment by incremental combinatorial extension (CE) of the optimal path</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">739</span><span class="NLM_x">–</span> <span class="NLM_lpage">747</span><span class="refDoi"> DOI: 10.1093/protein/11.9.739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fprotein%2F11.9.739" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=739-747&issue=9&author=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=Protein+structure+alignment+by+incremental+combinatorial+extension+%28CE%29+of+the+optimal+path&doi=10.1093%2Fprotein%2F11.9.739"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F11.9.739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F11.9.739%26sid%3Dliteratum%253Aachs%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProtein%2520structure%2520alignment%2520by%2520incremental%2520combinatorial%2520extension%2520%2528CE%2529%2520of%2520the%2520optimal%2520path%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1998%26volume%3D11%26issue%3D9%26spage%3D739%26epage%3D747%26doi%3D10.1093%2Fprotein%2F11.9.739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x"> (</span><span class="NLM_issue">15</span><span class="NLM_x">) </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span><span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&issue=15&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lgWR5QBbjgyBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26issue%3D15%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group">Das, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krein, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breneman, C. M.</span><span> </span><span class="NLM_article-title">PESDserv: a server for high-throughput comparison of protein binding site surfaces</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x"> (</span><span class="NLM_issue">15</span><span class="NLM_x">) </span> <span class="NLM_fpage">1913</span><span class="NLM_x">–</span> <span class="NLM_lpage">1914</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btq288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtq288" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1913-1914&issue=15&author=S.+Dasauthor=M.+P.+Kreinauthor=C.+M.+Breneman&title=PESDserv%3A+a+server+for+high-throughput+comparison+of+protein+binding+site+surfaces&doi=10.1093%2Fbioinformatics%2Fbtq288"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq288%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DS.%26aulast%3DKrein%26aufirst%3DM.%2BP.%26aulast%3DBreneman%26aufirst%3DC.%2BM.%26atitle%3DPESDserv%253A%2520a%2520server%2520for%2520high-throughput%2520comparison%2520of%2520protein%2520binding%2520site%2520surfaces%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26issue%3D15%26spage%3D1913%26epage%3D1914%26doi%3D10.1093%2Fbioinformatics%2Fbtq288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group">Zhu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Gharabli, S. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rademann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilgenfeld, R.</span><span> </span><span class="NLM_article-title">Peptide aldehyde inhibitors challenge the substrate specificity of the SARS-coronavirus main protease</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">204</span><span class="NLM_x">–</span> <span class="NLM_lpage">212</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2011.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.antiviral.2011.08.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2011&pages=204-212&issue=2&author=L.+Zhuauthor=S.+Georgeauthor=M.+F.+Schmidtauthor=S.+I.+Al-Gharabliauthor=J.+Rademannauthor=R.+Hilgenfeld&title=Peptide+aldehyde+inhibitors+challenge+the+substrate+specificity+of+the+SARS-coronavirus+main+protease&doi=10.1016%2Fj.antiviral.2011.08.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2011.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2011.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DGeorge%26aufirst%3DS.%26aulast%3DSchmidt%26aufirst%3DM.%2BF.%26aulast%3DAl-Gharabli%26aufirst%3DS.%2BI.%26aulast%3DRademann%26aufirst%3DJ.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DPeptide%2520aldehyde%2520inhibitors%2520challenge%2520the%2520substrate%2520specificity%2520of%2520the%2520SARS-coronavirus%2520main%2520protease%26jtitle%3DAntiviral%2520Res.%26date%3D2011%26volume%3D92%26issue%3D2%26spage%3D204%26epage%3D212%26doi%3D10.1016%2Fj.antiviral.2011.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group">Willmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jandausch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forne, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imhof, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">2704</span><span class="NLM_x">–</span> <span class="NLM_lpage">2709</span><span class="refDoi"> DOI: 10.1002/ijc.27555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fijc.27555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22447389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2012&pages=2704-2709&issue=11&author=D.+Willmannauthor=S.+Limauthor=S.+Wetzelauthor=E.+Metzgerauthor=A.+Jandauschauthor=W.+Wilkauthor=M.+Jungauthor=I.+Forneauthor=A.+Imhofauthor=A.+Janzerauthor=J.+Kirfelauthor=H.+Waldmannauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Impairment+of+prostate+cancer+cell+growth+by+a+selective+and+reversible+lysine-specific+demethylase+1+inhibitor&doi=10.1002%2Fijc.27555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span></div><div class="casAuthors">Willmann, Dominica; Lim, Soyoung; Wetzel, Stefan; Metzger, Eric; Jandausch, Anett; Wilk, Wolfram; Jung, Manfred; Forne, Ignasi; Imhof, Axel; Janzer, Andreas; Kirfel, Jutta; Waldmann, Herbert; Schuele, Roland; Buettner, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2704-2709</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Post-translational modifications of histones by chromatin modifying enzymes regulate chromatin structure and gene expression.  As deregulation of histone modifications contributes to cancer progression, inhibition of chromatin modifying enzymes such as histone demethylases is an attractive therapeutic strategy to impair cancer growth.  Lysine-specific demethylase 1 (LSD1) removes mono- and di-Me marks from lysine 4 or 9 of histone H3.  LSD1 in assocn. with the androgen receptor (AR) controls androgen-dependent gene expression and prostate tumor cell proliferation, thus highlighting LSD1 as a drug target.  By combining protein structure similarity clustering and in vitro screening, we identified Namoline, a γ-pyrone, as a novel, selective and reversible LSD1 inhibitor.  Namoline blocks LSD1 demethylase activity in vitro and in vivo.  Inhibition of LSD1 by Namoline leads to silencing of AR-regulated gene expression and severely impairs androgen-dependent proliferation in vitro and in vivo.  Thus, Namoline is a novel promising starting compd. for the development of therapeutics to treat androgen-dependent prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKxYi0f1zoOrVg90H21EOLACvtfcHk0lgWR5QBbjgyBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D&md5=3104057823357b199b635e1a22071b2a</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1002%2Fijc.27555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27555%26sid%3Dliteratum%253Aachs%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DJandausch%26aufirst%3DA.%26aulast%3DWilk%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DForne%26aufirst%3DI.%26aulast%3DImhof%26aufirst%3DA.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DImpairment%2520of%2520prostate%2520cancer%2520cell%2520growth%2520by%2520a%2520selective%2520and%2520reversible%2520lysine-specific%2520demethylase%25201%2520inhibitor%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D131%26issue%3D11%26spage%3D2704%26epage%3D2709%26doi%3D10.1002%2Fijc.27555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group">Holm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenström, P.</span><span> </span><span class="NLM_article-title">Dali server: conservation mapping in 3D</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">W545</span><span class="NLM_x">–</span> <span class="NLM_lpage">W549</span><div class="note"><p class="first last">(Web Server issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkq366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkq366" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W545-W549&issue=Suppl.+2&author=L.+Holmauthor=P.+Rosenstr%C3%B6m&title=Dali+server%3A+conservation+mapping+in+3D&doi=10.1093%2Fnar%2Fgkq366"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq366%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DRosenstr%25C3%25B6m%26aufirst%3DP.%26atitle%3DDali%2520server%253A%2520conservation%2520mapping%2520in%25203D%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26issue%3DSuppl.%25202%26spage%3DW545%26epage%3DW549%26doi%3D10.1093%2Fnar%2Fgkq366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chammaa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sperl, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yektaoglu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arenz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">A scaffold-tree-merging strategy for prospective bioactivity annotation of gamma-pyrones</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x"> (</span><span class="NLM_issue">21</span><span class="NLM_x">) </span> <span class="NLM_fpage">3666</span><span class="NLM_x">–</span> <span class="NLM_lpage">3670</span><span class="refDoi"> DOI: 10.1002/anie.200906555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fanie.200906555" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=3666-3670&issue=21&author=S.+Wetzelauthor=W.+Wilkauthor=S.+Chammaaauthor=B.+Sperlauthor=A.+G.+Rothauthor=A.+Yektaogluauthor=S.+Rennerauthor=T.+Bergauthor=C.+Arenzauthor=A.+Giannisauthor=T.+I.+Opreaauthor=D.+Rauhauthor=M.+Kaiserauthor=H.+Waldmann&title=A+scaffold-tree-merging+strategy+for+prospective+bioactivity+annotation+of+gamma-pyrones&doi=10.1002%2Fanie.200906555"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1002%2Fanie.200906555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200906555%26sid%3Dliteratum%253Aachs%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DWilk%26aufirst%3DW.%26aulast%3DChammaa%26aufirst%3DS.%26aulast%3DSperl%26aufirst%3DB.%26aulast%3DRoth%26aufirst%3DA.%2BG.%26aulast%3DYektaoglu%26aufirst%3DA.%26aulast%3DRenner%26aufirst%3DS.%26aulast%3DBerg%26aufirst%3DT.%26aulast%3DArenz%26aufirst%3DC.%26aulast%3DGiannis%26aufirst%3DA.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DKaiser%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DA%2520scaffold-tree-merging%2520strategy%2520for%2520prospective%2520bioactivity%2520annotation%2520of%2520gamma-pyrones%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26issue%3D21%26spage%3D3666%26epage%3D3670%26doi%3D10.1002%2Fanie.200906555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span> <span class="citation_source-book">UCSF Chimera: A Visualization System for Exploratory Research and Analysis</span>; <span class="NLM_publisher-name">Resource for Biocomputing, Visualization, and Informatics. Computer Graphics Laboratory, Department of Pharmaceutical Chemistry, University of California</span> (<span class="NLM_publisher-loc">600 16th Street, San Francisco, California 94143-2240, U.S.</span>),<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=UCSF+Chimera%3A+A+Visualization+System+for+Exploratory+Research+and+Analysis%3B+Resource+for+Biocomputing%2C+Visualization%2C+and+Informatics.+Computer+Graphics+Laboratory%2C+Department+of+Pharmaceutical+Chemistry%2C+University+of+California+%28600+16th+Street%2C+San+Francisco%2C+California+94143-2240%2C+U.S.%29%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DUCSF%2520Chimera%253A%2520A%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26pub%3DResource%2520for%2520Biocomputing%252C%2520Visualization%252C%2520and%2520Informatics.%2520Computer%2520Graphics%2520Laboratory%252C%2520Department%2520of%2520Pharmaceutical%2520Chemistry%252C%2520University%2520of%2520California%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group">Rajagopal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajagopal, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefkowitz, R. J.</span><span> </span><span class="NLM_article-title">Teaching old receptors new tricks: biasing seven-transmembrane receptors</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">373</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1038/nrd3024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fnrd3024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=20431569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVWjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=373-386&issue=5&author=S.+Rajagopalauthor=K.+Rajagopalauthor=R.+J.+Lefkowitz&title=Teaching+old+receptors+new+tricks%3A+biasing+seven-transmembrane+receptors&doi=10.1038%2Fnrd3024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Teaching old receptors new tricks: biasing seven-transmembrane receptors</span></div><div class="casAuthors">Rajagopal, Sudarshan; Rajagopal, Keshava; Lefkowitz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">373-386</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Seven-transmembrane receptors (7TMRs; also known as G protein-coupled receptors) are the largest class of receptors in the human genome and are common targets for therapeutics.  Originally identified as mediators of 7TMR desensitization, β-arrestins (arrestin 2 and arrestin 3) are now recognized as true adaptor proteins that transduce signals to multiple effector pathways.  Signaling that is mediated by β-arrestins has distinct biochem. and functional consequences from those mediated by G proteins, and several biased ligands and receptors have been identified that preferentially signal through either G protein- or β-arrestin-mediated pathways.  These ligands are not only useful tools for investigating the biochem. of 7TMR signaling, but they also have the potential to be developed into new classes of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGuN6jivmOErVg90H21EOLACvtfcHk0liW4oul6dOqqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVWjs7k%253D&md5=ff525016c0ba5aa0ebc4ed86f87594d0</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1038%2Fnrd3024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3024%26sid%3Dliteratum%253Aachs%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DRajagopal%26aufirst%3DK.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26atitle%3DTeaching%2520old%2520receptors%2520new%2520tricks%253A%2520biasing%2520seven-transmembrane%2520receptors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26issue%3D5%26spage%3D373%26epage%3D386%26doi%3D10.1038%2Fnrd3024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group">Issa, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byers, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakshanamurthy, S.</span><span> </span><span class="NLM_article-title">Drug repurposing: translational pharmacology, chemistry, computers and the clinic</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">18</span><span class="NLM_x">) </span> <span class="NLM_fpage">2328</span><span class="NLM_x">–</span> <span class="NLM_lpage">2336</span><span class="refDoi"> DOI: 10.2174/15680266113136660163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2174%2F15680266113136660163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24059462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CgurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=2328-2336&issue=18&author=N.+T.+Issaauthor=S.+W.+Byersauthor=S.+Dakshanamurthy&title=Drug+repurposing%3A+translational+pharmacology%2C+chemistry%2C+computers+and+the+clinic&doi=10.2174%2F15680266113136660163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Repurposing: Translational Pharmacology, Chemistry, Computers and the Clinic</span></div><div class="casAuthors">Issa, Naiem T.; Byers, Stephen W.; Dakshanamurthy, Sivanesan</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2328-2336</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The process of discovering a pharmacol. compd. that elicits a desired clin. effect with minimal side effects is a challenge.  Prior to the advent of high-performance computing and large-scale screening technologies, drug discovery was largely a serendipitous endeavor, as in the case of thalidomide for erythema nodosum leprosum or cancer drugs in general derived from flora located in far-reaching geog. locations.  More recently, de novo drug discovery has become a more rationalized process where drug-target-effect hypotheses are formulated on the basis of already known compds./protein targets and their structures.  Although this approach is hypothesis-driven, the actual success has been very low, contributing to the soaring costs of research and development as well as the diminished pharmaceutical pipeline in the United States.  In this review, we discuss the evolution in computational pharmacol. as the next generation of successful drug discovery and implementation in the clinic where high-performance computing (HPC) is used to generate and validate drug-target-effect hypotheses completely in silico.  The use of HPC would decrease development time and errors while increasing productivity prior to in vitro, animal and human testing.  We highlight approaches in chemoinformatics, bioinformatics as well as network biopharmacol. to illustrate potential avenues from which to design clin. efficacious drugs.  We further discuss the implications of combining these approaches into an integrative methodol. for high-accuracy computational predictions within the context of drug repositioning for the efficient streamlining of currently approved drugs back into clin. trials for possible new indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZiiRtN0LnbLVg90H21EOLACvtfcHk0liW4oul6dOqqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CgurzL&md5=4c952bdfdd8171df353c0d3fe0778ad6</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.2174%2F15680266113136660163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F15680266113136660163%26sid%3Dliteratum%253Aachs%26aulast%3DIssa%26aufirst%3DN.%2BT.%26aulast%3DByers%26aufirst%3DS.%2BW.%26aulast%3DDakshanamurthy%26aufirst%3DS.%26atitle%3DDrug%2520repurposing%253A%2520translational%2520pharmacology%252C%2520chemistry%252C%2520computers%2520and%2520the%2520clinic%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26issue%3D18%26spage%3D2328%26epage%3D2336%26doi%3D10.2174%2F15680266113136660163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group">Haupt, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M.</span><span> </span><span class="NLM_article-title">Old friends in new guise: repositioning of known drugs with structural bioinformatics</span> <span class="citation_source-journal">Briefings Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span><span class="refDoi"> DOI: 10.1093/bib/bbr011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbib%2Fbbr011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21441562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptl2jsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=312-326&issue=4&author=V.+J.+Hauptauthor=M.+Schroeder&title=Old+friends+in+new+guise%3A+repositioning+of+known+drugs+with+structural+bioinformatics&doi=10.1093%2Fbib%2Fbbr011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Old friends in new guise: repositioning of known drugs with structural bioinformatics</span></div><div class="casAuthors">Haupt, V. Joachim; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-326</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1467-5463</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Developing a drug de novo is a laborious and costly endeavor.  Thus, the repositioning of already approved drugs for the treatment of new diseases is promising and valuable.  One computational approach to repositioning exploits the structural similarity of binding sites of known and new targets.  Here, we review computational methods to represent and align binding sites.  We review available tools, present success stories and discuss limits of the approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeO0NDH5GZrrVg90H21EOLACvtfcHk0lgkAcqj6BsWOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptl2jsrc%253D&md5=e50e0f4752302d91263af4803ef0f14d</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbr011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbr011%26sid%3Dliteratum%253Aachs%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DOld%2520friends%2520in%2520new%2520guise%253A%2520repositioning%2520of%2520known%2520drugs%2520with%2520structural%2520bioinformatics%26jtitle%3DBriefings%2520Bioinf.%26date%3D2011%26volume%3D12%26issue%3D4%26spage%3D312%26epage%3D326%26doi%3D10.1093%2Fbib%2Fbbr011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group">Baek, M.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bae, J.-S.</span><span> </span><span class="NLM_article-title">Novel insight into drug repositioning: methylthiouracil as a case in point</span> <span class="citation_source-journal">Pharmacol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">185</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1016/j.phrs.2015.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.phrs.2015.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=26117428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BC2Mbnslaitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=185-193&author=M.-C.+Baekauthor=B.+Jungauthor=H.+Kangauthor=H.-S.+Leeauthor=J.-S.+Bae&title=Novel+insight+into+drug+repositioning%3A+methylthiouracil+as+a+case+in+point&doi=10.1016%2Fj.phrs.2015.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Novel insight into drug repositioning: Methylthiouracil as a case in point</span></div><div class="casAuthors">Baek Moon-Chang; Jung Byeongjin; Kang Hyejin; Lee Hyun-Shik; Bae Jong-Sup</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">185-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug repositioning refers to the development of existing drugs for new indications.  These drugs may have (I) failed to show efficacy in late stage clinical trials without safety issues; (II) stalled in the development for commercial reasons; (III) passed the point of patent expiry; or (IV) are being explored in new geographic markets.  Over the past decade, pressure on the pharmaceutical industry caused by the 'innovation gap' owing to rising development costs and stagnant product output have become major reasons for the growing interest in drug repositioning.  Companies that offer a variety of broad platforms for identifying new indications have emerged; some have been successful in building their own pipelines of candidates with reduced risks and timelines associated with further clinical development.  The business models and platforms offered by these companies will be validated if they are able to generate positive proof-of-concept clinical data for their repositioned compounds.  This review describes the strategy of biomarker-guided repositioning of chemotherapeutic drugs for inflammation therapy, considering the repositioning of methylthiouracil (MTU), an antithyroid drug, as a potential anti-inflammatory reagent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2LxaOWNLBmNbHMBt0mMIXfW6udTcc2eY8nLniZUSSgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mbnslaitw%253D%253D&md5=2094e593085461842f9ebfec9cc7c29f</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DBaek%26aufirst%3DM.-C.%26aulast%3DJung%26aufirst%3DB.%26aulast%3DKang%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DBae%26aufirst%3DJ.-S.%26atitle%3DNovel%2520insight%2520into%2520drug%2520repositioning%253A%2520methylthiouracil%2520as%2520a%2520case%2520in%2520point%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D185%26epage%3D193%26doi%3D10.1016%2Fj.phrs.2015.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group">O’Boyle, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeersch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, G. R.</span><span> </span><span class="NLM_article-title">Open Babel: An open chemical toolbox</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">33</span><span class="refDoi"> DOI: 10.1186/1758-2946-3-33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21982300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&issue=1&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+An+open+chemical+toolbox&doi=10.1186%2F1758-2946-3-33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Open Babel: an open chemical toolbox</span></div><div class="casAuthors">O'Boyle, Noel M.; Banck, Michael; James, Craig A.; Morley, Chris; Vandermeersch, Tim; Hutchison, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: A frequent problem in computational modeling is the interconversion of chem. structures between different formats.  While std. interchange formats exist (for example, Chem. Markup Language) and de facto stds. have arisen (for example, SMILES format), the need to interconvert formats is a continuing problem due to the multitude of different application areas for chem. data, differences in the data stored by different formats (0D vs. 3D, for example), and competition between software along with a lack of vendor-neutral formats.  Results: We discuss, for the first time, Open Babel, an open-source chem. toolbox that speaks the many languages of chem. data.  Open Babel version 2.3 interconverts over 110 formats.  The need to represent such a wide variety of chem. and mol. data requires a library that implements a wide range of cheminformatics algorithms, from partial charge assignment and aromaticity detection, to bond order perception and canonicalization.  We detail the implementation of Open Babel, describe key advances in the 2.3 release, and outline a variety of uses both in terms of software products and scientific research, including applications far beyond simple format interconversion.  Conclusions: Open Babel presents a soln. to the proliferation of multiple chem. file formats.  In addn., it provides a variety of useful utilities from conformer searching and 2D depiction, to filtering, batch conversion, and substructure and similarity searching.  For developers, it can be used as a programming library to handle chem. data in areas such as org. chem., drug design, materials science, and computational chem.  It is freely available under an open-source license.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfQ1lvaS5767Vg90H21EOLACvtfcHk0lgkAcqj6BsWOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF&md5=74e4f19b7f87417f916d57f7abcfb761</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520An%2520open%2520chemical%2520toolbox%26jtitle%3DJ.%2520Cheminf.%26date%3D2011%26volume%3D3%26issue%3D1%26spage%3D33%26doi%3D10.1186%2F1758-2946-3-33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group">Rahman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bashton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holliday, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrader, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">12</span><span class="refDoi"> DOI: 10.1186/1758-2946-1-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1758-2946-1-12" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Rahman%2C+S.+A.%3B+Bashton%2C+M.%3B+Holliday%2C+G.+L.%3B+Schrader%2C+R.%3B+Thornton%2C+J.+M.J.+Cheminf.+2009%2C+1+%281%29+12+10.1186%2F1758-2946-1-12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-1-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-1-12%26sid%3Dliteratum%253Aachs%26aulast%3DRahman%26aufirst%3DS.%2BA.%26aulast%3DBashton%26aufirst%3DM.%26aulast%3DHolliday%26aufirst%3DG.%2BL.%26aulast%3DSchrader%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26jtitle%3DJ.%2520Cheminf.%26date%3D2009%26volume%3D1%26issue%3D1%26spage%3D12%26doi%3D10.1186%2F1758-2946-1-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">BindingDB: a web-accessible molecular recognition database</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">719</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.2174/1386207013330670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2174%2F1386207013330670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=11812264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Orsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2001&pages=719-725&issue=8&author=X.+Chenauthor=M.+Liuauthor=M.+K.+Gilson&title=BindingDB%3A+a+web-accessible+molecular+recognition+database&doi=10.2174%2F1386207013330670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: A Web-accessible molecular recognition database</span></div><div class="casAuthors">Chen, Xi; Liu, Ming; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">719-725</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">The BindingDB, a public web-accessible database of measured binding affinities for various mol. types, is described.  It allows queries based upon a range of criteria, including chem. similarity or substructure, sequence homol., numerical criteria and reactant names.  Principles of Human-Computer Interactions are being employed in creating the query interface and user-feedback is being solicited.  The data specification includes significant exptl. detail.  A full dictionary has been created for isothermal titrn. calorimetry data in consultation with experimentalists and data dictionaries for enzyme inhibition and other measurement techniques are being developed.  Currently, the BindingDB contains several data sets of broad interest, such as antigen-antibody binding and cyclodextrin/small-mol. binding.  However, it is anticipated that online deposition by experimentalists will ultimately contribute a larger flow of data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzRKVfJZtKwrVg90H21EOLACvtfcHk0ljVuu5h7EBfjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Orsg%253D%253D&md5=9db61dc1ad10e8191ee11afcfa9e337e</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.2174%2F1386207013330670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207013330670%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520web-accessible%2520molecular%2520recognition%2520database%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2001%26volume%3D4%26issue%3D8%26spage%3D719%26epage%3D725%26doi%3D10.2174%2F1386207013330670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group">Lang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span> </span><span class="NLM_article-title">Identification of putative molecular imaging probes for BACE-1 by accounting for protein flexibility in virtual screening</span> <span class="citation_source-journal">J. Alzheimer’s Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">351</span><span class="NLM_x">–</span> <span class="NLM_lpage">359</span><span class="refDoi"> DOI: 10.3233/JAD-2011-111787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.3233%2FJAD-2011-111787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=22232010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFWltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=351-359&issue=2&author=H.+Langauthor=X.+Huangauthor=Y.+Yang&title=Identification+of+putative+molecular+imaging+probes+for+BACE-1+by+accounting+for+protein+flexibility+in+virtual+screening&doi=10.3233%2FJAD-2011-111787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Putative Molecular Imaging Probes for BACE-1 by Accounting for Protein Flexibility in Virtual Screening</span></div><div class="casAuthors">Lang, Haijing; Huang, Xudong; Yang, Yongliang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Alzheimer's Disease</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-359</span>CODEN:
                <span class="NLM_cas:coden">JADIF9</span>;
        ISSN:<span class="NLM_cas:issn">1387-2877</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">β-Secretase (BACE-1), an enzyme crit. in the process of amyloid-β (Aβ) peptides deposition in human brain, is closely assocd. with the onset and progression of Alzheimer's disease (AD).  A strong need exists, therefore, to identify mol. imaging probes homing at BACE-1 for use with positron emission tomog. (PET) that is recognized as an effective tool for detecting AD.  Through this imaging, an early diagnosis of AD could be made.  Herein, to identify suitable mol. probes for use with PET, we searched the Mol. Imaging and Contrast Agent Database (MICAD), an online database warehousing scientific information regarding mol. imaging and contrast agents, and applied a virtual screening approach against the different confirmations of BACE-1 obtained from the World Wide Protein Database.  The lack of considering receptor flexibility is a key drawback in virtual screening for drug discovery.  Therefore, we incorporated protein flexibility into the virtual screening by using an ensemble of 143 exptl. BACE-1 structures derived from the Protein Data Bank.  Finally, the best performing affinity was recorded and used in the ranking of each ligand.  To the best of our knowledge, this is the first virtual screening approach used to identify four new mol. probes that could target BACE-1 with favorable affinity, a discovery that can lead to the development of new PET probes for the early detection and therapy of AD.  However, the actual utility of these probes can only be ascertained after in vitro and in vivo investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYAMyfzjvfF7Vg90H21EOLACvtfcHk0ljVuu5h7EBfjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFWltLs%253D&md5=689a65e19129942234976a61464e006a</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.3233%2FJAD-2011-111787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJAD-2011-111787%26sid%3Dliteratum%253Aachs%26aulast%3DLang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DIdentification%2520of%2520putative%2520molecular%2520imaging%2520probes%2520for%2520BACE-1%2520by%2520accounting%2520for%2520protein%2520flexibility%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Alzheimer%25E2%2580%2599s%2520Dis.%26date%3D2012%26volume%3D29%26issue%3D2%26spage%3D351%26epage%3D359%26doi%3D10.3233%2FJAD-2011-111787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinzi, L.</span><span> </span><span class="NLM_article-title">Computational polypharmacology comes of age</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">7874</span><span class="refDoi"> DOI: 10.3389/fphar.2015.00157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.3389%2Ffphar.2015.00157" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7874&author=G.+Rastelliauthor=L.+Pinzi&title=Computational+polypharmacology+comes+of+age&doi=10.3389%2Ffphar.2015.00157"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2015.00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2015.00157%26sid%3Dliteratum%253Aachs%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DPinzi%26aufirst%3DL.%26atitle%3DComputational%2520polypharmacology%2520comes%2520of%2520age%26jtitle%3DFront.%2520Pharmacol.%26date%3D2015%26volume%3D6%26spage%3D7874%26doi%3D10.3389%2Ffphar.2015.00157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group">Salentin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haupt, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daminelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M.</span><span> </span><span class="NLM_article-title">Polypharmacology rescored: protein-ligand interaction profiles for remote binding site similarity assessment</span> <span class="citation_source-journal">Prog. Biophys. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x"> (</span><span class="NLM_issue">2–3</span><span class="NLM_x">) </span> <span class="NLM_fpage">174</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2014.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.pbiomolbio.2014.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24923864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jt7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2014&pages=174-186&issue=2%E2%80%933&author=S.+Salentinauthor=V.+J.+Hauptauthor=S.+Daminelliauthor=M.+Schroeder&title=Polypharmacology+rescored%3A+protein-ligand+interaction+profiles+for+remote+binding+site+similarity+assessment&doi=10.1016%2Fj.pbiomolbio.2014.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology rescored: Protein-ligand interaction profiles for remote binding site similarity assessment</span></div><div class="casAuthors">Salentin, Sebastian; Haupt, V. Joachim; Daminelli, Simone; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">174-186</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Detection of remote binding site similarity in proteins plays an important role for drug repositioning and off-target effect prediction.  Various non-covalent interactions such as hydrogen bonds and van-der-Waals forces drive ligands' mol. recognition by binding sites in proteins.  The increasing amt. of available structures of protein-small mol. complexes enabled the development of comparative approaches.  Several methods have been developed to characterize and compare protein-ligand interaction patterns.  Usually implemented as fingerprints, these are mainly used for post processing docking scores and (off-)target prediction.  In the latter application, interaction profiles detect similarities in the bound interactions of different ligands and thus identify essential interactions between a protein and its small mol. ligands.  Interaction pattern similarity correlates with binding site similarity and is thus contributing to a higher precision in binding site similarity assessment of proteins with distinct global structure.  This renders it valuable for existing drug repositioning approaches in structural bioinformatics.  Current methods to characterize and compare structure-based interaction patterns - both for protein-small-mol. and protein-protein interactions - as well as their potential in target prediction will be reviewed in this article.  The question of how the set of interaction types, flexibility or water-mediated interactions, influence the comparison of interaction patterns will be discussed.  Due to the wealth of protein-ligand structures available today, predicted targets can be ranked by comparing their ligand interaction pattern to patterns of the known target.  Such knowledge-based methods offer high precision in comparison to methods comparing whole binding sites based on shape and amino acid physicochem. similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeHhHCtFDIlLVg90H21EOLACvtfcHk0ljVuu5h7EBfjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jt7nN&md5=6c83751ccdf7cadd879df9aefff29af7</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2014.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2014.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DSalentin%26aufirst%3DS.%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DDaminelli%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DPolypharmacology%2520rescored%253A%2520protein-ligand%2520interaction%2520profiles%2520for%2520remote%2520binding%2520site%2520similarity%2520assessment%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D116%26issue%3D2%25E2%2580%25933%26spage%3D174%26epage%3D186%26doi%3D10.1016%2Fj.pbiomolbio.2014.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group">Gills, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopiccolo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsurutani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Best, C. J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abu-Asab, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borojerdi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warfel, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danish, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollander, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawabata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsokos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figg, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steeg, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennis, P. A.</span><span> </span><span class="NLM_article-title">Nelfinavir, A lead HIV protease inhibitor, is a broad-spectrum, anticancer agent that induces endoplasmic reticulum stress, autophagy, and apoptosis in vitro and in vivo</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">5183</span><span class="NLM_x">–</span> <span class="NLM_lpage">5194</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-07-0161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1158%2F1078-0432.CCR-07-0161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=17785575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFGqs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=5183-5194&issue=17&author=J.+J.+Gillsauthor=J.+Lopiccoloauthor=J.+Tsurutaniauthor=R.+H.+Shoemakerauthor=C.+J.+M.+Bestauthor=M.+S.+Abu-Asabauthor=J.+Borojerdiauthor=N.+A.+Warfelauthor=E.+R.+Gardnerauthor=M.+Danishauthor=M.+C.+Hollanderauthor=S.+Kawabataauthor=M.+Tsokosauthor=W.+D.+Figgauthor=P.+S.+Steegauthor=P.+A.+Dennis&title=Nelfinavir%2C+A+lead+HIV+protease+inhibitor%2C+is+a+broad-spectrum%2C+anticancer+agent+that+induces+endoplasmic+reticulum+stress%2C+autophagy%2C+and+apoptosis+in+vitro+and+in+vivo&doi=10.1158%2F1078-0432.CCR-07-0161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Nelfinavir, A Lead HIV Protease Inhibitor, Is a Broad-Spectrum, Anticancer Agent that Induces Endoplasmic Reticulum Stress, Autophagy, and Apoptosis In vitro and In vivo</span></div><div class="casAuthors">Gills, Joell J.; LoPiccolo, Jaclyn; Tsurutani, Junji; Shoemaker, Robert H.; Best, Carolyn J. M.; Abu-Asab, Mones S.; Borojerdi, Jennifer; Warfel, Noel A.; Gardner, Erin R.; Danish, Matthew; Hollander, M. Christine; Kawabata, Shigeru; Tsokos, Maria; Figg, William D.; Steeg, Patricia S.; Dennis, Phillip A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5183-5194</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The development of new cancer drugs is slow and costly.  HIV protease inhibitors are Food and Drug Administration approved for HIV patients.  Because these drugs cause toxicities that can be assocd. with inhibition of Akt, an emerging target in cancer, we assessed the potential of HIV protease inhibitors as anticancer agents.  Exptl. Design: HIV protease inhibitors were screened in vitro using assays that measure cellular proliferation, apoptotic and nonapoptotic cell death, endoplasmic reticulum (ER) stress, autophagy, and activation of Akt.  Nelfinavir was tested in non-small cell lung carcinoma (NSCLC) xenografts with biomarker assessment.  Results: Three of six HIV protease inhibitors, nelfinavir, ritonavir, and saquinavir, inhibited proliferation of NSCLC cells, as well as every cell line in the NCI60 cell line panel.  Nelfinavir was most potent with a mean 50% growth inhibition of 5.2 μmol/L, a concn. achievable in HIV patients.  Nelfinavir caused two types of cell death, caspase-dependent apoptosis and caspase-independent death that was characterized by induction of ER stress and autophagy.  Autophagy was protective because an inhibitor of autophagy increased nelfinavir-induced death.  Akt was variably inhibited by HIV protease inhibitors, but nelfinavir caused the greatest inhibition of endogenous and growth factor-induced Akt activation.  Nelfinavir decreased the viability of a panel of drug-resistant breast cancer cell lines and inhibited the growth of NSCLC xenografts that was assocd. with induction of ER stress, autophagy, and apoptosis.  Conclusions: Nelfinavir is a lead HIV protease inhibitor with pleiotropic effects in cancer cells.  Given its wide spectrum of activity, oral availability, and familiarity of administration, nelfinavir is a Food and Drug Administration-approved drug that could be repositioned as a cancer therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLMrqFw84X_7Vg90H21EOLACvtfcHk0lhK5mGaSe4fIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFGqs7o%253D&md5=b66804003472d635f209e49d4017387d</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-07-0161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-07-0161%26sid%3Dliteratum%253Aachs%26aulast%3DGills%26aufirst%3DJ.%2BJ.%26aulast%3DLopiccolo%26aufirst%3DJ.%26aulast%3DTsurutani%26aufirst%3DJ.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DBest%26aufirst%3DC.%2BJ.%2BM.%26aulast%3DAbu-Asab%26aufirst%3DM.%2BS.%26aulast%3DBorojerdi%26aufirst%3DJ.%26aulast%3DWarfel%26aufirst%3DN.%2BA.%26aulast%3DGardner%26aufirst%3DE.%2BR.%26aulast%3DDanish%26aufirst%3DM.%26aulast%3DHollander%26aufirst%3DM.%2BC.%26aulast%3DKawabata%26aufirst%3DS.%26aulast%3DTsokos%26aufirst%3DM.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26aulast%3DSteeg%26aufirst%3DP.%2BS.%26aulast%3DDennis%26aufirst%3DP.%2BA.%26atitle%3DNelfinavir%252C%2520A%2520lead%2520HIV%2520protease%2520inhibitor%252C%2520is%2520a%2520broad-spectrum%252C%2520anticancer%2520agent%2520that%2520induces%2520endoplasmic%2520reticulum%2520stress%252C%2520autophagy%252C%2520and%2520apoptosis%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2007%26volume%3D13%26issue%3D17%26spage%3D5183%26epage%3D5194%26doi%3D10.1158%2F1078-0432.CCR-07-0161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group">Dunlop, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lock, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jow, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitzia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grauer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jow, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowlby, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randall, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowal, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comery, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larocque, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soloveva, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roncarati, R.</span><span> </span><span class="NLM_article-title">Old and new pharmacology: positive allosteric modulation of the alpha7 nicotinic acetylcholine receptor by the 5-hydroxytryptamine(2B/C) receptor antagonist SB-206553 (3,5-dihydro-5-methyl-N-3-pyridinylbenzo[1,2-b:4,5-b′]di pyrrole-1(2H)-carboxamide</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">328</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">766</span><span class="NLM_x">–</span> <span class="NLM_lpage">776</span><span class="refDoi"> DOI: 10.1124/jpet.108.146514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1124%2Fjpet.108.146514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=19050173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1MXislals7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=328&publication_year=2009&pages=766-776&issue=3&author=J.+Dunlopauthor=T.+Lockauthor=B.+Jowauthor=F.+Sitziaauthor=S.+Grauerauthor=F.+Jowauthor=A.+Kramerauthor=M.+R.+Bowlbyauthor=A.+Randallauthor=D.+Kowalauthor=A.+Gilbertauthor=T.+A.+Comeryauthor=J.+Larocqueauthor=V.+Solovevaauthor=J.+Brownauthor=R.+Roncarati&title=Old+and+new+pharmacology%3A+positive+allosteric+modulation+of+the+alpha7+nicotinic+acetylcholine+receptor+by+the+5-hydroxytryptamine%282B%2FC%29+receptor+antagonist+SB-206553+%283%2C5-dihydro-5-methyl-N-3-pyridinylbenzo%5B1%2C2-b%3A4%2C5-b%E2%80%B2%5Ddi+pyrrole-1%282H%29-carboxamide&doi=10.1124%2Fjpet.108.146514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Old and new pharmacology: positive allosteric modulation of the α7 nicotinic acetylcholine receptor by the 5-hydroxytryptamine2B/C receptor antagonist SB-206553 (3,5-dihydro-5-methyl-N-3-pyridinylbenzo[1,2-b:4,5-b']di pyrrole-1(2H)-carboxamide)</span></div><div class="casAuthors">Dunlop, John; Lock, Tim; Jow, Brian; Sitzia, Fabrizio; Grauer, Steven; Jow, Flora; Kramer, Angela; Bowlby, Mark R.; Randall, Andrew; Kowal, Dianne; Gilbert, Adam; Comery, Thomas A.; LaRocque, James; Soloveva, Veronica; Brown, Jon; Roncarati, Renza</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">328</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">766-776</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The α7 nicotinic acetylcholine receptor (nAChR) has been implicated in Alzheimer's disease and schizophrenia, leading to efforts targeted toward discovering agonists and pos. allosteric modulators (PAMs) of this receptor.  In a Ca2+ flux fluorometric imaging plate reader assay, SB-206553 (3,5-dihydro-5-methyl-N-3-pyridinylbenzo[1,2-b:4,5-b']di pyrrole-1(2H)-carboxamide), a compd. known as a 5-hydroxytryptamine2B/2C receptor antagonist, produced an 8-fold potentiation of the evoked calcium signal in the presence of an EC20 concn. of nicotine and a corresponding EC50 of 1.5 μM for potentiation of EC20 nicotine responses in GH4C1 cells expressing the α7 receptor.  SB-206553 was devoid of direct α7 receptor agonist activity and selective against other nicotinic receptors.  Confirmation of the PAM activity of SB-206553 on the α7 nAChR was obtained in patch-clamp electrophysiol. expts. in GH4C1 cells, where it failed to evoke any detectable currents when applied alone, yet dramatically potentiated the currents evoked by an EC20 (17 μM) and EC100 (124 μM) of acetylcholine (ACh).  Native nicotinic receptors in CA1 stratum radiatum interneurons of rat hippocampal slices could also be activated by ACh (200 μM), an effect that was entirely blocked by the α7-selective antagonist methyllycaconitine (MLA).  These ACh currents were potentiated by SB-206553, which increased the area of the current response significantly, resulting in a 40-fold enhancement at 100 μM.  In behavioral expts. in rats, SB-206553 reversed an MK-801 (dizocilpine maleate)-induced deficit in the prepulse inhibition of acoustic startle response, an effect attenuated in the presence of MLA.  This latter observation provides further evidence in support of the potential therapeutic utility of α7 nAChR PAMs in schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomM1BUzvmG6LVg90H21EOLACvtfcHk0lhK5mGaSe4fIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXislals7w%253D&md5=d1783c43f8d4e58af504614e13126012</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1124%2Fjpet.108.146514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.108.146514%26sid%3Dliteratum%253Aachs%26aulast%3DDunlop%26aufirst%3DJ.%26aulast%3DLock%26aufirst%3DT.%26aulast%3DJow%26aufirst%3DB.%26aulast%3DSitzia%26aufirst%3DF.%26aulast%3DGrauer%26aufirst%3DS.%26aulast%3DJow%26aufirst%3DF.%26aulast%3DKramer%26aufirst%3DA.%26aulast%3DBowlby%26aufirst%3DM.%2BR.%26aulast%3DRandall%26aufirst%3DA.%26aulast%3DKowal%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DA.%26aulast%3DComery%26aufirst%3DT.%2BA.%26aulast%3DLarocque%26aufirst%3DJ.%26aulast%3DSoloveva%26aufirst%3DV.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DRoncarati%26aufirst%3DR.%26atitle%3DOld%2520and%2520new%2520pharmacology%253A%2520positive%2520allosteric%2520modulation%2520of%2520the%2520alpha7%2520nicotinic%2520acetylcholine%2520receptor%2520by%2520the%25205-hydroxytryptamine%25282B%252FC%2529%2520receptor%2520antagonist%2520SB-206553%2520%25283%252C5-dihydro-5-methyl-N-3-pyridinylbenzo%255B1%252C2-b%253A4%252C5-b%25E2%2580%25B2%255Ddi%2520pyrrole-1%25282H%2529-carboxamide%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2009%26volume%3D328%26issue%3D3%26spage%3D766%26epage%3D776%26doi%3D10.1124%2Fjpet.108.146514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evrard, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murdoch, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Droste, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nissen, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenck, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fludzinski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucaites, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, M. L.</span><span> </span><span class="NLM_article-title">Potent, selective tetrahydro-beta-carboline antagonists of the serotonin 2B (5HT2B) contractile receptor in the rat stomach fundus</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x"> (</span><span class="NLM_issue">14</span><span class="NLM_x">) </span> <span class="NLM_fpage">2773</span><span class="NLM_x">–</span> <span class="NLM_lpage">2780</span><span class="refDoi"> DOI: 10.1021/jm960062t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm960062t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2773-2780&issue=14&author=J.+E.+Audiaauthor=D.+A.+Evrardauthor=G.+R.+Murdochauthor=J.+J.+Drosteauthor=J.+S.+Nissenauthor=K.+W.+Schenckauthor=P.+Fludzinskiauthor=V.+L.+Lucaitesauthor=D.+L.+Nelsonauthor=M.+L.+Cohen&title=Potent%2C+selective+tetrahydro-beta-carboline+antagonists+of+the+serotonin+2B+%285HT2B%29+contractile+receptor+in+the+rat+stomach+fundus&doi=10.1021%2Fjm960062t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fjm960062t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm960062t%26sid%3Dliteratum%253Aachs%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DEvrard%26aufirst%3DD.%2BA.%26aulast%3DMurdoch%26aufirst%3DG.%2BR.%26aulast%3DDroste%26aufirst%3DJ.%2BJ.%26aulast%3DNissen%26aufirst%3DJ.%2BS.%26aulast%3DSchenck%26aufirst%3DK.%2BW.%26aulast%3DFludzinski%26aufirst%3DP.%26aulast%3DLucaites%26aufirst%3DV.%2BL.%26aulast%3DNelson%26aufirst%3DD.%2BL.%26aulast%3DCohen%26aufirst%3DM.%2BL.%26atitle%3DPotent%252C%2520selective%2520tetrahydro-beta-carboline%2520antagonists%2520of%2520the%2520serotonin%25202B%2520%25285HT2B%2529%2520contractile%2520receptor%2520in%2520the%2520rat%2520stomach%2520fundus%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26issue%3D14%26spage%3D2773%26epage%3D2780%26doi%3D10.1021%2Fjm960062t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group">Petrey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Structural bioinformatics of the interactome</span> <span class="citation_source-journal">Annu. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.1146/annurev-biophys-051013-022726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1146%2Fannurev-biophys-051013-022726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24895853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOht7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=193-210&author=D.+Petreyauthor=B.+Honig&title=Structural+bioinformatics+of+the+interactome&doi=10.1146%2Fannurev-biophys-051013-022726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Structural bioinformatics of the interactome</span></div><div class="casAuthors">Petrey, Donald; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">193-210</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The past decade has seen a dramatic expansion in the no. and range of techniques available to obtain genome-wide information and to analyze this information so as to infer both the functions of individual mols. and how they interact to modulate the behavior of biol. systems.  Here, we review these techniques, focusing on the construction of phys. protein-protein interaction networks, and highlighting approaches that incorporate protein structure, which is becoming an increasingly important component of systems-level computational techniques.  We also discuss how network analyses are being applied to enhance our basic understanding of biol. systems and their disregulation, as well as how these networks are being used in drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrju2I3vXKYM7Vg90H21EOLACvtfcHk0limf7DVqNTebg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOht7rL&md5=f50bea829841525b0c3b19321aae5fb6</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-051013-022726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-051013-022726%26sid%3Dliteratum%253Aachs%26aulast%3DPetrey%26aufirst%3DD.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DStructural%2520bioinformatics%2520of%2520the%2520interactome%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2014%26volume%3D43%26spage%3D193%26epage%3D210%26doi%3D10.1146%2Fannurev-biophys-051013-022726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group">Haydon, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokes, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ure, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galbraith, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barynin, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sedelnikova, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heal, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aiwale, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauhan, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taneja, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Errington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czaplewski, L. G.</span><span> </span><span class="NLM_article-title">An inhibitor of FtsZ with potent and selective anti-staphylococcal activity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">321</span><span class="NLM_x"> (</span><span class="NLM_issue">5896</span><span class="NLM_x">) </span> <span class="NLM_fpage">1673</span><span class="NLM_x">–</span> <span class="NLM_lpage">1675</span><span class="refDoi"> DOI: 10.1126/science.1159961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1126%2Fscience.1159961" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=1673-1675&issue=5896&author=D.+J.+Haydonauthor=N.+R.+Stokesauthor=R.+Ureauthor=G.+Galbraithauthor=J.+M.+Bennettauthor=D.+R.+Brownauthor=P.+J.+Bakerauthor=V.+V.+Baryninauthor=D.+W.+Riceauthor=S.+E.+Sedelnikovaauthor=J.+R.+Healauthor=J.+M.+Sheridanauthor=S.+T.+Aiwaleauthor=P.+K.+Chauhanauthor=A.+Srivastavaauthor=A.+Tanejaauthor=I.+Collinsauthor=J.+Erringtonauthor=L.+G.+Czaplewski&title=An+inhibitor+of+FtsZ+with+potent+and+selective+anti-staphylococcal+activity&doi=10.1126%2Fscience.1159961"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1126%2Fscience.1159961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1159961%26sid%3Dliteratum%253Aachs%26aulast%3DHaydon%26aufirst%3DD.%2BJ.%26aulast%3DStokes%26aufirst%3DN.%2BR.%26aulast%3DUre%26aufirst%3DR.%26aulast%3DGalbraith%26aufirst%3DG.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBrown%26aufirst%3DD.%2BR.%26aulast%3DBaker%26aufirst%3DP.%2BJ.%26aulast%3DBarynin%26aufirst%3DV.%2BV.%26aulast%3DRice%26aufirst%3DD.%2BW.%26aulast%3DSedelnikova%26aufirst%3DS.%2BE.%26aulast%3DHeal%26aufirst%3DJ.%2BR.%26aulast%3DSheridan%26aufirst%3DJ.%2BM.%26aulast%3DAiwale%26aufirst%3DS.%2BT.%26aulast%3DChauhan%26aufirst%3DP.%2BK.%26aulast%3DSrivastava%26aufirst%3DA.%26aulast%3DTaneja%26aufirst%3DA.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DErrington%26aufirst%3DJ.%26aulast%3DCzaplewski%26aufirst%3DL.%2BG.%26atitle%3DAn%2520inhibitor%2520of%2520FtsZ%2520with%2520potent%2520and%2520selective%2520anti-staphylococcal%2520activity%26jtitle%3DScience%26date%3D2008%26volume%3D321%26issue%3D5896%26spage%3D1673%26epage%3D1675%26doi%3D10.1126%2Fscience.1159961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group">Anderson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorto, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackner, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ames, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, J. T.</span><span> </span><span class="NLM_article-title">Comparison of small molecule inhibitors of the bacterial cell division protein FtsZ and identification of a reliable cross-species inhibitor</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">1918</span><span class="NLM_x">–</span> <span class="NLM_lpage">1928</span><span class="refDoi"> DOI: 10.1021/cb300340j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300340j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWmur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1918-1928&issue=11&author=D.+E.+Andersonauthor=M.+B.+Kimauthor=J.+T.+Mooreauthor=T.+E.+O%E2%80%99Brienauthor=N.+A.+Sortoauthor=C.+I.+Groveauthor=L.+L.+Lacknerauthor=J.+B.+Amesauthor=J.+T.+Shaw&title=Comparison+of+small+molecule+inhibitors+of+the+bacterial+cell+division+protein+FtsZ+and+identification+of+a+reliable+cross-species+inhibitor&doi=10.1021%2Fcb300340j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Small Molecule Inhibitors of the Bacterial Cell Division Protein FtsZ and Identification of a Reliable Cross-Species Inhibitor</span></div><div class="casAuthors">Anderson, David E.; Kim, Michelle B.; Moore, Jared T.; O'Brien, Terrence E.; Sorto, Nohemy A.; Grove, Charles I.; Lackner, Laura L.; Ames, James B.; Shaw, Jared T.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1918-1928</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">FtsZ is a guanosine triphosphatase (GTPase) that mediates cytokinesis in bacteria.  FtsZ is homologous in structure to eukaryotic tubulin and polymerizes in a similar head-to-tail fashion.  The study of tubulin's function in eukaryotic cells has benefited greatly from specific and potent small mol. inhibitors, including colchicine and taxol.  Although many small mol. inhibitors of FtsZ have been reported, none has emerged as a generally useful probe for modulating bacterial cell division.  With the goal of establishing a useful and reliable small mol. inhibitor of FtsZ, a broad biochem. cross-comparison of reported FtsZ inhibitors was undertaken.  Several of these mols., including phenolic natural products, are unselective inhibitors that seem to derive their activity from the formation of microscopic colloids or aggregates.  Other compds., including the natural product viriditoxin and the drug development candidate PC190723, exhibit no inhibition of GTPase activity using protocols or under published conditions.  Of the compds. studied, only zantrin Z3 exhibits good levels of inhibition, maintains activity under conditions that disrupt small mol. aggregates, and provides a platform for exploration of structure-activity relationships (SAR).  Preliminary SAR studies have identified slight modifications to the two side chains of this structure that modulate the inhibitory activity of zantrin Z3.  Collectively, these studies will help focus future investigations toward the establishment of probes for FtsZ that fill the roles of colchicine and taxol in studies of tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvE8_fy6gEwLVg90H21EOLACvtfcHk0limf7DVqNTebg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWmur%252FL&md5=ab69ea62af74ad72f6177db5f556c990</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1021%2Fcb300340j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300340j%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BE.%26aulast%3DKim%26aufirst%3DM.%2BB.%26aulast%3DMoore%26aufirst%3DJ.%2BT.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%2BE.%26aulast%3DSorto%26aufirst%3DN.%2BA.%26aulast%3DGrove%26aufirst%3DC.%2BI.%26aulast%3DLackner%26aufirst%3DL.%2BL.%26aulast%3DAmes%26aufirst%3DJ.%2BB.%26aulast%3DShaw%26aufirst%3DJ.%2BT.%26atitle%3DComparison%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520bacterial%2520cell%2520division%2520protein%2520FtsZ%2520and%2520identification%2520of%2520a%2520reliable%2520cross-species%2520inhibitor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26issue%3D11%26spage%3D1918%26epage%3D1928%26doi%3D10.1021%2Fcb300340j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group">Andreu, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaffner-Barbero, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huecas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alonso, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Rodriguez, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz-Avila, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Núñez-Ramírez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llorca, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martín-Galiano, A. J.</span><span> </span><span class="NLM_article-title">The antibacterial cell division inhibitor PC190723 is an FtsZ polymer-stabilizing agent that induces filament assembly and condensation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x"> (</span><span class="NLM_issue">19</span><span class="NLM_x">) </span> <span class="NLM_fpage">14239</span><span class="NLM_x">–</span> <span class="NLM_lpage">14246</span><span class="refDoi"> DOI: 10.1074/jbc.M109.094722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1074%2Fjbc.M109.094722" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=14239-14246&issue=19&author=J.+M.+Andreuauthor=C.+Schaffner-Barberoauthor=S.+Huecasauthor=D.+Alonsoauthor=M.+L.+Lopez-Rodriguezauthor=L.+B.+Ruiz-Avilaauthor=R.+N%C3%BA%C3%B1ez-Ram%C3%ADrezauthor=O.+Llorcaauthor=A.+J.+Mart%C3%ADn-Galiano&title=The+antibacterial+cell+division+inhibitor+PC190723+is+an+FtsZ+polymer-stabilizing+agent+that+induces+filament+assembly+and+condensation&doi=10.1074%2Fjbc.M109.094722"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.094722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.094722%26sid%3Dliteratum%253Aachs%26aulast%3DAndreu%26aufirst%3DJ.%2BM.%26aulast%3DSchaffner-Barbero%26aufirst%3DC.%26aulast%3DHuecas%26aufirst%3DS.%26aulast%3DAlonso%26aufirst%3DD.%26aulast%3DLopez-Rodriguez%26aufirst%3DM.%2BL.%26aulast%3DRuiz-Avila%26aufirst%3DL.%2BB.%26aulast%3DN%25C3%25BA%25C3%25B1ez-Ram%25C3%25ADrez%26aufirst%3DR.%26aulast%3DLlorca%26aufirst%3DO.%26aulast%3DMart%25C3%25ADn-Galiano%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520antibacterial%2520cell%2520division%2520inhibitor%2520PC190723%2520is%2520an%2520FtsZ%2520polymer-stabilizing%2520agent%2520that%2520induces%2520filament%2520assembly%2520and%2520condensation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D19%26spage%3D14239%26epage%3D14246%26doi%3D10.1074%2Fjbc.M109.094722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group">Sudha, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srinivasan, N.</span><span> </span><span class="NLM_article-title">An overview of recent advances in structural bioinformatics of protein–protein interactions and a guide to their principles</span> <span class="citation_source-journal">Prog. Biophys. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x"> (</span><span class="NLM_issue">2–3</span><span class="NLM_x">) </span> <span class="NLM_fpage">141</span><span class="NLM_x">–</span> <span class="NLM_lpage">150</span><span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2014.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.pbiomolbio.2014.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25077409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlaku7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2014&pages=141-150&issue=2%E2%80%933&author=G.+Sudhaauthor=R.+Nussinovauthor=N.+Srinivasan&title=An+overview+of+recent+advances+in+structural+bioinformatics+of+protein%E2%80%93protein+interactions+and+a+guide+to+their+principles&doi=10.1016%2Fj.pbiomolbio.2014.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of recent advances in structural bioinformatics of protein-protein interactions and a guide to their principles</span></div><div class="casAuthors">Sudha, Govindarajan; Nussinov, Ruth; Srinivasan, Narayanaswamy</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">141-150</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Rich data bearing on the structural and evolutionary principles of protein-protein interactions are paving the way to a better understanding of the regulation of function in the cell.  This is particularly the case when these interactions are considered in the framework of key pathways.  Knowledge of the interactions may provide insights into the mechanisms of crucial 'driver' mutations in oncogenesis.  They also provide the foundation toward the design of protein-protein interfaces and inhibitors that can abrogate their formation or enhance them.  The main features to learn from known 3-D structures of protein-protein complexes and the extensive literature which analyzes them computationally and exptl. include the interaction details which permit undertaking structure-based drug discovery, the evolution of complexes and their interactions, the consequences of alterations such as post-translational modifications, ligand binding, disease causing mutations, host pathogen interactions, oligomerization, aggregation and the roles of disorder, dynamics, allostery and more to the protein and the cell.  This review highlights some of the recent advances in these areas, including design, inhibition and prediction of protein-protein complexes.  The field is broad, and much work has been carried out in these areas, making it challenging to cover it in its entirety.  Much of this is due to the fast increase in the no. of mols. whose structures have been detd. exptl. and the vast increase in computational power.  Here we provide a concise overview.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhVXrmxxifZbVg90H21EOLACvtfcHk0lhYfyvqH_hp8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlaku7jN&md5=09fb901ba27805f4807f162e5c613c24</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2014.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2014.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DSudha%26aufirst%3DG.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DSrinivasan%26aufirst%3DN.%26atitle%3DAn%2520overview%2520of%2520recent%2520advances%2520in%2520structural%2520bioinformatics%2520of%2520protein%25E2%2580%2593protein%2520interactions%2520and%2520a%2520guide%2520to%2520their%2520principles%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D116%26issue%3D2%25E2%2580%25933%26spage%3D141%26epage%3D150%26doi%3D10.1016%2Fj.pbiomolbio.2014.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group">Xue, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobbs, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonvin, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honavar, V.</span><span> </span><span class="NLM_article-title">Computational prediction of protein interfaces: A review of data driven methods</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">589</span><span class="NLM_x"> (</span><span class="NLM_issue">23</span><span class="NLM_x">) </span> <span class="NLM_fpage">3516</span><span class="NLM_x">–</span> <span class="NLM_lpage">3526</span><span class="refDoi"> DOI: 10.1016/j.febslet.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.febslet.2015.10.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=3516-3526&issue=23&author=L.+C.+Xueauthor=D.+Dobbsauthor=A.+M.+Bonvinauthor=V.+Honavar&title=Computational+prediction+of+protein+interfaces%3A+A+review+of+data+driven+methods&doi=10.1016%2Fj.febslet.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DL.%2BC.%26aulast%3DDobbs%26aufirst%3DD.%26aulast%3DBonvin%26aufirst%3DA.%2BM.%26aulast%3DHonavar%26aufirst%3DV.%26atitle%3DComputational%2520prediction%2520of%2520protein%2520interfaces%253A%2520A%2520review%2520of%2520data%2520driven%2520methods%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26issue%3D23%26spage%3D3516%26epage%3D3526%26doi%3D10.1016%2Fj.febslet.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group">Dowlatshahi, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandrin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vivona, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaler, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melandri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundquist, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopito, R. R.</span><span> </span><span class="NLM_article-title">ALIX is a Lys63-specific polyubiquitin binding protein that functions in retrovirus budding</span> <span class="citation_source-journal">Dev. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1247</span><span class="NLM_x">–</span> <span class="NLM_lpage">1254</span><span class="refDoi"> DOI: 10.1016/j.devcel.2012.10.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.devcel.2012.10.023" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=1247-1254&issue=6&author=D.+P.+Dowlatshahiauthor=V.+Sandrinauthor=S.+Vivonaauthor=T.+A.+Shalerauthor=S.+E.+Kaiserauthor=F.+Melandriauthor=W.+I.+Sundquistauthor=R.+R.+Kopito&title=ALIX+is+a+Lys63-specific+polyubiquitin+binding+protein+that+functions+in+retrovirus+budding&doi=10.1016%2Fj.devcel.2012.10.023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2012.10.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2012.10.023%26sid%3Dliteratum%253Aachs%26aulast%3DDowlatshahi%26aufirst%3DD.%2BP.%26aulast%3DSandrin%26aufirst%3DV.%26aulast%3DVivona%26aufirst%3DS.%26aulast%3DShaler%26aufirst%3DT.%2BA.%26aulast%3DKaiser%26aufirst%3DS.%2BE.%26aulast%3DMelandri%26aufirst%3DF.%26aulast%3DSundquist%26aufirst%3DW.%2BI.%26aulast%3DKopito%26aufirst%3DR.%2BR.%26atitle%3DALIX%2520is%2520a%2520Lys63-specific%2520polyubiquitin%2520binding%2520protein%2520that%2520functions%2520in%2520retrovirus%2520budding%26jtitle%3DDev.%2520Cell%26date%3D2012%26volume%3D23%26issue%3D6%26spage%3D1247%26epage%3D1254%26doi%3D10.1016%2Fj.devcel.2012.10.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref152"><div class="reference"><strong class="refLabel"><a href="#ref152" class="rightTabRefNumLink">152</a></strong><div class="NLM_citation" id="rightTab-cit152"><span><span class="NLM_contrib-group">Pashkova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gakhar, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winistorfer, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunshine, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunham, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piper, R. C.</span><span> </span><span class="NLM_article-title">The yeast Alix homolog Bro1 functions as a ubiquitin receptor for protein sorting into multivesicular endosomes</span> <span class="citation_source-journal">Dev. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">520</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span><span class="refDoi"> DOI: 10.1016/j.devcel.2013.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.devcel.2013.04.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2013&pages=520-533&issue=5&author=N.+Pashkovaauthor=L.+Gakharauthor=S.+C.+Winistorferauthor=A.+B.+Sunshineauthor=M.+Richauthor=M.+J.+Dunhamauthor=L.+Yuauthor=R.+C.+Piper&title=The+yeast+Alix+homolog+Bro1+functions+as+a+ubiquitin+receptor+for+protein+sorting+into+multivesicular+endosomes&doi=10.1016%2Fj.devcel.2013.04.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2013.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2013.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DPashkova%26aufirst%3DN.%26aulast%3DGakhar%26aufirst%3DL.%26aulast%3DWinistorfer%26aufirst%3DS.%2BC.%26aulast%3DSunshine%26aufirst%3DA.%2BB.%26aulast%3DRich%26aufirst%3DM.%26aulast%3DDunham%26aufirst%3DM.%2BJ.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DPiper%26aufirst%3DR.%2BC.%26atitle%3DThe%2520yeast%2520Alix%2520homolog%2520Bro1%2520functions%2520as%2520a%2520ubiquitin%2520receptor%2520for%2520protein%2520sorting%2520into%2520multivesicular%2520endosomes%26jtitle%3DDev.%2520Cell%26date%3D2013%26volume%3D25%26issue%3D5%26spage%3D520%26epage%3D533%26doi%3D10.1016%2Fj.devcel.2013.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref153"><div class="reference"><strong class="refLabel"><a href="#ref153" class="rightTabRefNumLink">153</a></strong><div class="NLM_citation" id="rightTab-cit153"><span><span class="NLM_contrib-group">Schumann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armen, R. S.</span><span> </span><span class="NLM_article-title">Identification of distant drug off-targets by direct superposition of binding pocket surfaces</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">e83533</span><span class="refDoi"> DOI: 10.1371/journal.pone.0083533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0083533" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e83533&issue=12&author=M.+Schumannauthor=R.+S.+Armen&title=Identification+of+distant+drug+off-targets+by+direct+superposition+of+binding+pocket+surfaces&doi=10.1371%2Fjournal.pone.0083533"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083533%26sid%3Dliteratum%253Aachs%26aulast%3DSchumann%26aufirst%3DM.%26aulast%3DArmen%26aufirst%3DR.%2BS.%26atitle%3DIdentification%2520of%2520distant%2520drug%2520off-targets%2520by%2520direct%2520superposition%2520of%2520binding%2520pocket%2520surfaces%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26issue%3D12%26spage%3De83533%26doi%3D10.1371%2Fjournal.pone.0083533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref154"><div class="reference"><strong class="refLabel"><a href="#ref154" class="rightTabRefNumLink">154</a></strong><div class="NLM_citation" id="rightTab-cit154"><span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapland, R. H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoorn, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Global mapping of pharmacological space</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">805</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span><span class="refDoi"> DOI: 10.1038/nbt1228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&issue=7&author=G.+V.+Paoliniauthor=R.+H.+B.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+mapping+of+pharmacological+space&doi=10.1038%2Fnbt1228"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%2BB.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520mapping%2520of%2520pharmacological%2520space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26issue%3D7%26spage%3D805%26epage%3D815%26doi%3D10.1038%2Fnbt1228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref155"><div class="reference"><strong class="refLabel"><a href="#ref155" class="rightTabRefNumLink">155</a></strong><div class="NLM_citation" id="rightTab-cit155"><span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">A survey of across-target bioactivity results of small molecules in PubChem</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">2251</span><span class="NLM_x">–</span> <span class="NLM_lpage">2255</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btp380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fbioinformatics%2Fbtp380" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2251-2255&issue=17&author=L.+Hanauthor=Y.+Wangauthor=S.+H.+Bryant&title=A+survey+of+across-target+bioactivity+results+of+small+molecules+in+PubChem&doi=10.1093%2Fbioinformatics%2Fbtp380"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp380%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DA%2520survey%2520of%2520across-target%2520bioactivity%2520results%2520of%2520small%2520molecules%2520in%2520PubChem%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26issue%3D17%26spage%3D2251%26epage%3D2255%26doi%3D10.1093%2Fbioinformatics%2Fbtp380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref156"><div class="reference"><strong class="refLabel"><a href="#ref156" class="rightTabRefNumLink">156</a></strong><div class="NLM_citation" id="rightTab-cit156"><span><span class="NLM_contrib-group">LaBute, M. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenderman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennion, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lightstone, F. C.</span><span> </span><span class="NLM_article-title">Adverse drug reaction prediction using scores produced by large-scale drug-protein target docking on high-performance computing machines</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">e106298</span><span class="refDoi"> DOI: 10.1371/journal.pone.0106298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1371%2Fjournal.pone.0106298" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e106298&issue=9&author=M.+X.+LaButeauthor=X.+Zhangauthor=J.+Lendermanauthor=B.+J.+Bennionauthor=S.+E.+Wongauthor=F.+C.+Lightstone&title=Adverse+drug+reaction+prediction+using+scores+produced+by+large-scale+drug-protein+target+docking+on+high-performance+computing+machines&doi=10.1371%2Fjournal.pone.0106298"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0106298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0106298%26sid%3Dliteratum%253Aachs%26aulast%3DLaBute%26aufirst%3DM.%2BX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLenderman%26aufirst%3DJ.%26aulast%3DBennion%26aufirst%3DB.%2BJ.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26atitle%3DAdverse%2520drug%2520reaction%2520prediction%2520using%2520scores%2520produced%2520by%2520large-scale%2520drug-protein%2520target%2520docking%2520on%2520high-performance%2520computing%2520machines%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3De106298%26doi%3D10.1371%2Fjournal.pone.0106298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref157"><div class="reference"><strong class="refLabel"><a href="#ref157" class="rightTabRefNumLink">157</a></strong><div class="NLM_citation" id="rightTab-cit157"><span><span class="NLM_contrib-group">Sonnhammer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eddy, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durbin, R.</span><span> </span><span class="NLM_article-title">Pfam: a comprehensive database of protein domain families based on seed alignments</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">420</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(199707)28:3<405::AID-PROT10>3.0.CO;2-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=9223186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2sXksVCgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1997&pages=405-420&issue=3&author=E.+L.+Sonnhammerauthor=S.+R.+Eddyauthor=R.+Durbin&title=Pfam%3A+a+comprehensive+database+of+protein+domain+families+based+on+seed+alignments&doi=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Pfam: a comprehensive database of protein domain families based on seed alignments</span></div><div class="casAuthors">Sonnhammer, Erik L. L.; Eddy, Sean R.; Durbin, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-420</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss</span>)
        </div><div class="casAbstract">Databases of multiple sequence alignments are a valuable aid to protein sequence classification and anal.  One of the main challenges when constructing such a database is to simultaneously satisfy the conflicting demands of completeness on the one hand and quality of alignment and domain definitions on the other.  The latter properties are best dealt with by manual approaches, whereas completeness in practice is only amenable to automatic methods.  Here, the authors present a database based on hidden Markov model profiles (HMMs), which combines high quality and completeness.  This database, Pfam, consists of parts A and B.  Pfam-A is curated and contains well-characterized protein domain families with high quality alignments, which are maintained by using manually checked seed alignments and HMMs to find and align all members.  Pfam-B contains sequence families that were generated automatically by applying the Domainer algorithm to cluster and align the remaining protein sequences after removal of Pfam-A domains.  By using Pfam, a large no. of previously unannotated proteins from the Caenorhabditis elegans genome project were classified.  The authors also identified many novel family memberships in known proteins, including new Kazal, fibronectin type III, and response regulator receiver domains.  Pfam-A families have permanent accession nos. and form a library of HMMs available for searching and automatic annotation of new protein sequences.  There are World Wide Web servers at http://www.sanger.ac.uk/Pfam and http://genome.wustl.edu/Pfam, which allow browsing and HMM searching against Pfam-A with a query sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre6VakACw-_rVg90H21EOLACvtfcHk0lhvwXP188w85w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksVCgsLo%253D&md5=8fe760b424180de92f0c3816443d2d8d</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%2528199707%252928%253A3%253C405%253A%253AAID-PROT10%253E3.0.CO%253B2-L%26sid%3Dliteratum%253Aachs%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DDurbin%26aufirst%3DR.%26atitle%3DPfam%253A%2520a%2520comprehensive%2520database%2520of%2520protein%2520domain%2520families%2520based%2520on%2520seed%2520alignments%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1997%26volume%3D28%26issue%3D3%26spage%3D405%26epage%3D420%26doi%3D10.1002%2F%28SICI%291097-0134%28199707%2928%3A3%3C405%3A%3AAID-PROT10%3E3.0.CO%3B2-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref158"><div class="reference"><strong class="refLabel"><a href="#ref158" class="rightTabRefNumLink">158</a></strong><div class="NLM_citation" id="rightTab-cit158"><span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weskamp, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hüllermeier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Functional classification of protein kinase binding sites using Cavbase</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">1432</span><span class="NLM_x">–</span> <span class="NLM_lpage">1447</span><span class="refDoi"> DOI: 10.1002/cmdc.200700075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2Fcmdc.200700075" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=1432-1447&issue=10&author=D.+Kuhnauthor=N.+Weskampauthor=E.+H%C3%BCllermeierauthor=G.+Klebe&title=Functional+classification+of+protein+kinase+binding+sites+using+Cavbase&doi=10.1002%2Fcmdc.200700075"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200700075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200700075%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DWeskamp%26aufirst%3DN.%26aulast%3DH%25C3%25BCllermeier%26aufirst%3DE.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFunctional%2520classification%2520of%2520protein%2520kinase%2520binding%2520sites%2520using%2520Cavbase%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26issue%3D10%26spage%3D1432%26epage%3D1447%26doi%3D10.1002%2Fcmdc.200700075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref159"><div class="reference"><strong class="refLabel"><a href="#ref159" class="rightTabRefNumLink">159</a></strong><div class="NLM_citation" id="rightTab-cit159"><span><span class="NLM_contrib-group">Liu, C.-G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, K.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, H.-P.</span><span> </span><span class="NLM_article-title">17Beta-oestradiol regulates the expression of Na+/K+-ATPase beta1-subunit, sarcoplasmic reticulum Ca2+-ATPase and carbonic anhydrase iv in H9C2 cells</span> <span class="citation_source-journal">Clin. Exp. Pharmacol. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">998</span><span class="NLM_x">–</span> <span class="NLM_lpage">1004</span><span class="refDoi"> DOI: 10.1111/j.1440-1681.2007.04675.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1111%2Fj.1440-1681.2007.04675.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=17714085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2007&pages=998-1004&issue=10&author=C.-G.+Liuauthor=K.-Q.+Xuauthor=X.+Xuauthor=J.-J.+Huangauthor=J.-C.+Xiaoauthor=J.-P.+Zhangauthor=H.-P.+Song&title=17Beta-oestradiol+regulates+the+expression+of+Na%2B%2FK%2B-ATPase+beta1-subunit%2C+sarcoplasmic+reticulum+Ca2%2B-ATPase+and+carbonic+anhydrase+iv+in+H9C2+cells&doi=10.1111%2Fj.1440-1681.2007.04675.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">17β-Oestradiol regulates the expression of Na+/K+-ATPase β1-subunit, sarcoplasmic reticulum Ca2+-ATPase and carbonic anhydrase IV in H9C2 cells</span></div><div class="casAuthors">Liu, Chen-Geng; Xu, Ke-Qian; Xu, Xia; Huang, Jian-Jun; Xiao, Jing-Chuan; Zhang, Jie-Ping; Song, Hui-Ping</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Experimental Pharmacology and Physiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">998-1004</span>CODEN:
                <span class="NLM_cas:coden">CEXPB9</span>;
        ISSN:<span class="NLM_cas:issn">0305-1870</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">It is necessary to improve our understanding of the effect of 17β-estradiol (E2) on the heart at a mol. and cellular level.  In the present study, the effects of E2 on Na+/K+-ATPase, sarcoplasmic/endoplasmic reticulum Ca2+-ATPase (SERCA) and carbonic anhydrase IV (CAIV) in H9C2 cells were investigated.  To identify the mechanism of action of E2 on these proteins, the estrogen receptor (ER) antagonist tamoxifen was used.  The results indicated that 1 and 100 nmol/L E2 can enhance the activity of Na+/K+-ATPase and SERCA and upregulate the expression of the Na+/K+-ATPase β1-subunit, SERCA2a and CAIV at both the mRNA and protein level compared with 0 and 0.01 nmol/L E2.  E2 had the greatest effect at 100 nmol/L; 1 μmol/L E2 did not further protein expression compared with 100 nmol/L E2.  Tamoxifen (10 nmol/L) significantly decreased the activity of SERCA, as well as the expression of the Na+/K+-ATPase β1-subunit and SERCA at the mRNA and protein level, in H9C2 cells cultured with 1 nmol/L E2.  Tamoxifen alone had no significant effect on these proteins in H9C2 cells.  It may be hypothesized that a suitable E2 concn. has a protective effect on the heart and that the actual dose of E2 used in hormone-replacement therapy is important in menopausal women.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphXyGJ0J5KNbVg90H21EOLACvtfcHk0lhvwXP188w85w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSmtrbO&md5=5751a59fa260e70ccf08f004e7e133ed</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1111%2Fj.1440-1681.2007.04675.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1440-1681.2007.04675.x%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DC.-G.%26aulast%3DXu%26aufirst%3DK.-Q.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DJ.-J.%26aulast%3DXiao%26aufirst%3DJ.-C.%26aulast%3DZhang%26aufirst%3DJ.-P.%26aulast%3DSong%26aufirst%3DH.-P.%26atitle%3D17Beta-oestradiol%2520regulates%2520the%2520expression%2520of%2520Na%252B%252FK%252B-ATPase%2520beta1-subunit%252C%2520sarcoplasmic%2520reticulum%2520Ca2%252B-ATPase%2520and%2520carbonic%2520anhydrase%2520iv%2520in%2520H9C2%2520cells%26jtitle%3DClin.%2520Exp.%2520Pharmacol.%2520Physiol.%26date%3D2007%26volume%3D34%26issue%3D10%26spage%3D998%26epage%3D1004%26doi%3D10.1111%2Fj.1440-1681.2007.04675.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref160"><div class="reference"><strong class="refLabel"><a href="#ref160" class="rightTabRefNumLink">160</a></strong><div class="NLM_citation" id="rightTab-cit160"><span><span class="NLM_contrib-group">Beca, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavlov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kargacin, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aschar-Sobbi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kargacin, G. J.</span><span> </span><span class="NLM_article-title">Inhibition of a cardiac sarcoplasmic reticulum chloride channel by tamoxifen</span> <span class="citation_source-journal">Pfluegers Arch.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">457</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1007/s00424-008-0510-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2Fs00424-008-0510-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18458943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCiur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=457&publication_year=2008&pages=121-135&issue=1&author=S.+Becaauthor=E.+Pavlovauthor=M.+E.+Kargacinauthor=R.+Aschar-Sobbiauthor=R.+J.+Frenchauthor=G.+J.+Kargacin&title=Inhibition+of+a+cardiac+sarcoplasmic+reticulum+chloride+channel+by+tamoxifen&doi=10.1007%2Fs00424-008-0510-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of a cardiac sarcoplasmic reticulum chloride channel by tamoxifen</span></div><div class="casAuthors">Beca, Sanja; Pavlov, Evgeny; Kargacin, Margaret E.; Aschar-Sobbi, Roozbeh; French, Robert J.; Kargacin, Gary J.</div><div class="citationInfo"><span class="NLM_cas:title">Pfluegers Archiv</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">457</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">121-135</span>CODEN:
                <span class="NLM_cas:coden">PFLABK</span>;
        ISSN:<span class="NLM_cas:issn">0031-6768</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Anion and cation channels present in the sarcoplasmic reticulum (SR) are believed to be necessary to maintain the electroneutrality of SR membrane during Ca2+ uptake by the SR Ca2+ pump (SERCA).  Here the authors incorporated canine cardiac SR ion channels into lipid bilayers and studied the effects of tamoxifen and other antiestrogens on these channels.  A Cl- channel was identified exhibiting multiple subconductance levels which could be divided into two primary conductance bands.  Tamoxifen decreases the time the channel spends in its higher, voltage-sensitive band and the mean channel current.  The lower, voltage-insensitive, conductance band is not affected by tamoxifen, nor is a K+ channel present in the cardiac SR prepn.  By examg. SR Ca2+ uptake, SERCA ATPase activity, and SR ion channels in the same prepn., the authors also estd. SERCA transport current, SR Cl- and K+ currents, and the d. of SERCA, Cl-, and K+ channels in cardiac SR membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzceMjb_UAvrVg90H21EOLACvtfcHk0lhvwXP188w85w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCiur3K&md5=f4fc47985e319b9649e5d927a7472568</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1007%2Fs00424-008-0510-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00424-008-0510-9%26sid%3Dliteratum%253Aachs%26aulast%3DBeca%26aufirst%3DS.%26aulast%3DPavlov%26aufirst%3DE.%26aulast%3DKargacin%26aufirst%3DM.%2BE.%26aulast%3DAschar-Sobbi%26aufirst%3DR.%26aulast%3DFrench%26aufirst%3DR.%2BJ.%26aulast%3DKargacin%26aufirst%3DG.%2BJ.%26atitle%3DInhibition%2520of%2520a%2520cardiac%2520sarcoplasmic%2520reticulum%2520chloride%2520channel%2520by%2520tamoxifen%26jtitle%3DPfluegers%2520Arch.%26date%3D2008%26volume%3D457%26issue%3D1%26spage%3D121%26epage%3D135%26doi%3D10.1007%2Fs00424-008-0510-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref161"><div class="reference"><strong class="refLabel"><a href="#ref161" class="rightTabRefNumLink">161</a></strong><div class="NLM_citation" id="rightTab-cit161"><span><span class="NLM_contrib-group">Wang, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.-H.</span><span> </span><span class="NLM_article-title">Scoring functions for prediction of protein-ligand interactions</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2174</span><span class="NLM_x">–</span> <span class="NLM_lpage">2182</span><span class="refDoi"> DOI: 10.2174/1381612811319120005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2174%2F1381612811319120005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=23016847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2174-2182&issue=12&author=J.-C.+Wangauthor=J.-H.+Lin&title=Scoring+functions+for+prediction+of+protein-ligand+interactions&doi=10.2174%2F1381612811319120005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Scoring functions for prediction of protein-ligand interactions</span></div><div class="casAuthors">Wang, Jui-Chih; Lin, Jung-Hsin</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2174-2182</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The scoring functions for protein-ligand interactions plays central roles in computational drug design, virtual screening of chem. libraries for new lead identification, and prediction of possible binding targets of small chem. mols.  An ideal scoring function for protein-ligand interactions is expected to be able to recognize the native binding pose of a ligand on the protein surface among decoy poses, and to accurately predict the binding affinity (or binding free energy) so that the active mols. can be discriminated from the non-active ones.  Due to the empirical nature of most, if not all, scoring functions for protein-ligand interactions, the general applicability of empirical scoring functions, esp. to domains far outside training sets, is a major concern.  In this review article, we will explore the foundations of different classes of scoring functions, their possible limitations, and their suitable application domains.  We also provide assessments of several scoring functions on weakly-interacting protein-ligand complexes, which will be useful information in computational fragment-based drug design or virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgdoR3oNlND7Vg90H21EOLACvtfcHk0lhRp_DXn07rwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtLs%253D&md5=0dbef427d5e9465beb60bd25a5882789</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.2174%2F1381612811319120005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319120005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.-C.%26aulast%3DLin%26aufirst%3DJ.-H.%26atitle%3DScoring%2520functions%2520for%2520prediction%2520of%2520protein-ligand%2520interactions%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26issue%3D12%26spage%3D2174%26epage%3D2182%26doi%3D10.2174%2F1381612811319120005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref162"><div class="reference"><strong class="refLabel"><a href="#ref162" class="rightTabRefNumLink">162</a></strong><div class="NLM_citation" id="rightTab-cit162"><span><span class="NLM_contrib-group">Wikberg, J. E. S.; Lapinsh, M.; Prusis, P.</span><span> </span><span class="NLM_article-title">Proteochemometrics: a tool for modeling the molecular interaction space</span>. In  <span class="citation_source-book">Chemogenomics in Drug Discovery</span>; <span class="NLM_contrib-group">Kubinyi, H.; Müller, G.</span>, Eds.; Methods and Principles in Medicinal Chemistry; <span class="NLM_publisher-name">Wiley-VCH Verlag GmbH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">289</span><span class="NLM_x">–</span> <span class="NLM_lpage">309</span>, DOI: <span class="refDoi"> DOI: 10.1002/3527603948.ch10</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1002%2F3527603948.ch10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=289-309&author=J.+E.+S.+Wikberg&author=M.+Lapinsh&author=P.+Prusisauthor=H.+Kubinyi&author=G.+M%C3%BCller&title=Chemogenomics+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1002%2F3527603948.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527603948.ch10%26sid%3Dliteratum%253Aachs%26aulast%3DWikberg%26aufirst%3DJ.%2BE.%2BS.%26atitle%3DProteochemometrics%253A%2520a%2520tool%2520for%2520modeling%2520the%2520molecular%2520interaction%2520space%26btitle%3DChemogenomics%2520in%2520Drug%2520Discovery%26aulast%3DKubinyi%26aufirst%3DH.%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26date%3D2004%26spage%3D289%26epage%3D309%26doi%3D10.1002%2F3527603948.ch10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref163"><div class="reference"><strong class="refLabel"><a href="#ref163" class="rightTabRefNumLink">163</a></strong><div class="NLM_citation" id="rightTab-cit163"><span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">The PDBbind database: collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2977</span><span class="NLM_x">–</span> <span class="NLM_lpage">2980</span><span class="refDoi"> DOI: 10.1021/jm030580l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030580l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2977-2980&issue=12&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=S.+Wang&title=The+PDBbind+database%3A+collection+of+binding+affinities+for+protein-ligand+complexes+with+known+three-dimensional+structures&doi=10.1021%2Fjm030580l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">The PDBbind database: Collection of binding affinities for protein-ligand complexes with known three-dimensional structures</span></div><div class="casAuthors">Wang, Renxiao; Fang, Xueliang; Lu, Yipin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2977-2980</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have screened the entire Protein Data Bank (Release No. 103, Jan. 2003) and identified 5671 protein-ligand complexes out of 19 621 exptl. structures.  A systematic examn. of the primary refs. of these entries has led to a collection of binding affinity data (Kd, Ki, and IC50) for a total of 1359 complexes.  The outcomes of this project have been organized into a Web-accessible database named the PDBbind database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1eUdeWioYBbVg90H21EOLACvtfcHk0lhRp_DXn07rwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjs1Sjs74%253D&md5=86e609172307402d8b0d4589b1270a2f</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Fjm030580l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030580l%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520collection%2520of%2520binding%2520affinities%2520for%2520protein-ligand%2520complexes%2520with%2520known%2520three-dimensional%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D12%26spage%3D2977%26epage%3D2980%26doi%3D10.1021%2Fjm030580l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref164"><div class="reference"><strong class="refLabel"><a href="#ref164" class="rightTabRefNumLink">164</a></strong><div class="NLM_citation" id="rightTab-cit164"><span><span class="NLM_contrib-group">Olsson, T. S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladbury, J. E.</span><span> </span><span class="NLM_article-title">The thermodynamics of protein-ligand interaction and solvation: insights for ligand design</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">1002</span><span class="NLM_x">–</span> <span class="NLM_lpage">1017</span><span class="refDoi"> DOI: 10.1016/j.jmb.2008.09.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.jmb.2008.09.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18930735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWls7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2008&pages=1002-1017&issue=4&author=T.+S.+G.+Olssonauthor=M.+A.+Williamsauthor=W.+R.+Pittauthor=J.+E.+Ladbury&title=The+thermodynamics+of+protein-ligand+interaction+and+solvation%3A+insights+for+ligand+design&doi=10.1016%2Fj.jmb.2008.09.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">The Thermodynamics of Protein-Ligand Interaction and Solvation: Insights for Ligand Design</span></div><div class="casAuthors">Olsson, Tjelvar S. G.; Williams, Mark A.; Pitt, William R.; Ladbury, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1002-1017</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry is able to provide accurate information on the thermodn. contributions of enthalpy and entropy changes to free energies of binding.  The Structure/Calorimetry of Reported Protein Interactions Online (SCORPIO) database of published isothermal titrn. calorimetry studies and structural information on the interactions between proteins and small-mol. ligands is used here to reveal general thermodn. properties of protein-ligand interactions and to investigate correlations with changes in solvation.  The overwhelming majority of interactions are found to be enthalpically favored.  Synthetic inhibitors and biol. ligands form two distinct subpopulations in the data, with the former having greater av. affinity due to more favorable entropy changes on binding.  The greatest correlation is found between the binding free energy and apolar surface burial upon complex formation.  However, the free energy contribution per unit area buried is only 30-50% of that expected from earlier studies of transfer free energies of small mols.  A simple probability-based estimator for the maximal affinity of a binding site in terms of its apolar surface area is proposed.  Polar surface area burial also contributes substantially to affinity but is difficult to express in terms of unit area due to the small variation in the amt. of polar surface buried and a tendency for cancellation of its enthalpic and entropic contributions.  Conventionally, the contribution of apolar desolvation to affinity is attributed to gain of entropy due to solvent release.  Although data presented here are supportive of this notion, because the correlation of entropy change with apolar surface burial is relatively weak, it cannot, on present evidence, be confidently considered to be correct.  Further, thermodn. changes arising from small differences between ligands binding to individual proteins are relatively large and, in general, uncorrelated with changes in solvation, suggesting that trends identified across widely differing proteins are of limited use in explaining or predicting the effects of ligand modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xdclY7Rk_bVg90H21EOLACvtfcHk0lhRp_DXn07rwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWls7zO&md5=f9d6d3e8a7fd85d7ff7a6dc6acf7309b</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.09.073%26sid%3Dliteratum%253Aachs%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DWilliams%26aufirst%3DM.%2BA.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520thermodynamics%2520of%2520protein-ligand%2520interaction%2520and%2520solvation%253A%2520insights%2520for%2520ligand%2520design%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D384%26issue%3D4%26spage%3D1002%26epage%3D1017%26doi%3D10.1016%2Fj.jmb.2008.09.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref165"><div class="reference"><strong class="refLabel"><a href="#ref165" class="rightTabRefNumLink">165</a></strong><div class="NLM_citation" id="rightTab-cit165"><span><span class="NLM_contrib-group">Wu, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwa, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, C.</span><span> </span><span class="NLM_article-title">Hidden relationship between conserved residues and locally conserved phosphate-binding structures in NAD(P)-binding proteins</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x"> (</span><span class="NLM_issue">19</span><span class="NLM_x">) </span> <span class="NLM_fpage">5644</span><span class="NLM_x">–</span> <span class="NLM_lpage">5652</span><span class="refDoi"> DOI: 10.1021/jp3014332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp3014332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2012&pages=5644-5652&issue=19&author=C.+Y.+Wuauthor=Y.+H.+Hwaauthor=Y.+C.+Chenauthor=C.+Lim&title=Hidden+relationship+between+conserved+residues+and+locally+conserved+phosphate-binding+structures+in+NAD%28P%29-binding+proteins&doi=10.1021%2Fjp3014332"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1021%2Fjp3014332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp3014332%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DC.%2BY.%26aulast%3DHwa%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DLim%26aufirst%3DC.%26atitle%3DHidden%2520relationship%2520between%2520conserved%2520residues%2520and%2520locally%2520conserved%2520phosphate-binding%2520structures%2520in%2520NAD%2528P%2529-binding%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2012%26volume%3D116%26issue%3D19%26spage%3D5644%26epage%3D5652%26doi%3D10.1021%2Fjp3014332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref166"><div class="reference"><strong class="refLabel"><a href="#ref166" class="rightTabRefNumLink">166</a></strong><div class="NLM_citation" id="rightTab-cit166"><span><span class="NLM_contrib-group">Huang, B.</span><span> </span><span class="NLM_article-title">Identification of pockets on protein surface to predict protein–ligand binding sites</span>. In  <span class="citation_source-book">Identification of Ligand Binding Site and Protein–Protein Interaction Area</span>; <span class="NLM_contrib-group">Roterman-Konieczna, I.</span>, Ed.; Focus on Structural Biology; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Dordrecht, The Netherlands</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1007%2F978-94-007-5285-6_2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=25-39&author=B.+Huangauthor=I.+Roterman-Konieczna&title=Identification+of+Ligand+Binding+Site+and+Protein%E2%80%93Protein+Interaction+Area"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1007%2F978-94-007-5285-6_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-007-5285-6_2%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520pockets%2520on%2520protein%2520surface%2520to%2520predict%2520protein%25E2%2580%2593ligand%2520binding%2520sites%26btitle%3DIdentification%2520of%2520Ligand%2520Binding%2520Site%2520and%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Area%26aulast%3DRoterman-Konieczna%26aufirst%3DI.%26pub%3DSpringer%26date%3D2013%26spage%3D25%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref167"><div class="reference"><strong class="refLabel"><a href="#ref167" class="rightTabRefNumLink">167</a></strong><div class="NLM_citation" id="rightTab-cit167"><span><span class="NLM_contrib-group">Holm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, C.</span><span> </span><span class="NLM_article-title">Dali: a network tool for protein structure comparison</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">478</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span><span class="refDoi"> DOI: 10.1016/S0968-0004(00)89105-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2FS0968-0004%2800%2989105-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=8578593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2MXpsF2murY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1995&pages=478-480&issue=11&author=L.+Holmauthor=C.+Sander&title=Dali%3A+a+network+tool+for+protein+structure+comparison&doi=10.1016%2FS0968-0004%2800%2989105-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Dali: a network tool for protein structure comparison</span></div><div class="casAuthors">Holm, Liisa; Sander, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">478-80</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Trends Journals</span>)
        </div><div class="casAbstract">The recent surge in newly solved 3-dimensional protein structures has whetted the appetite for systematic structure-structure comparison with the potential for evolutionary discoveries.  The EMBL is now providing Internet (<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="7612171a1f36131b141a5b1e131f12131a14130411581213">[email protected]</a>) and World Wide Web access to the Dali method for protein structure comparison.  The Dali method measures structural relatedness in terms of similarities of intramol. distance matrixes (i.e., the alignments are detd. only by comparing 3-dimensional coordinates).  The services include a database of precalcd. structural neighbors for all public structures and an e-mail server for searches with newly solved ones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmJryprgsB8LVg90H21EOLACvtfcHk0lj2PBrPJwCRpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsF2murY%253D&md5=78c8ba072569f09ec94e32bbbd8bfed1</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2800%2989105-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252800%252989105-7%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DSander%26aufirst%3DC.%26atitle%3DDali%253A%2520a%2520network%2520tool%2520for%2520protein%2520structure%2520comparison%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1995%26volume%3D20%26issue%3D11%26spage%3D478%26epage%3D480%26doi%3D10.1016%2FS0968-0004%2800%2989105-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref168"><div class="reference"><strong class="refLabel"><a href="#ref168" class="rightTabRefNumLink">168</a></strong><div class="NLM_citation" id="rightTab-cit168"><span><span class="NLM_contrib-group">Nadzirin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Firdaus-Raih, M.</span><span> </span><span class="NLM_article-title">Proteins of unknown function in the Protein Data Bank (PDB): an inventory of true uncharacterized proteins and computational tools for their analysis</span> <span class="citation_source-journal">Int. J. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">12761</span><span class="NLM_x">–</span> <span class="NLM_lpage">12772</span><span class="refDoi"> DOI: 10.3390/ijms131012761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.3390%2Fijms131012761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=23202924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaqtbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=12761-12772&issue=10&author=N.+Nadzirinauthor=M.+Firdaus-Raih&title=Proteins+of+unknown+function+in+the+Protein+Data+Bank+%28PDB%29%3A+an+inventory+of+true+uncharacterized+proteins+and+computational+tools+for+their+analysis&doi=10.3390%2Fijms131012761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">Proteins of unknown function in the Protein Data Bank (PDB): an inventory of true uncharacterized proteins and computational tools for their analysis</span></div><div class="casAuthors">Nadzirin, Nurul; Firdaus-Raih, Mohd</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12761-12772</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Proteins of uncharacterized functions form a large part of many of the currently available biol. databases and this situation exists even in the Protein Data Bank (PDB).  Our anal. of recent PDB data revealed that only 42.53% of PDB entries (1084 coordinate files) that were categorized under "unknown function" are true examples of proteins of unknown function at this point in time.  The remainder 1465 entries also annotated as such appear to be able to have their annotations re-assessed, based on the availability of direct functional characterization expts. for the protein itself, or for homologous sequences or structures thus enabling computational function inference.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotBQpdM4xXubVg90H21EOLACvtfcHk0lj2PBrPJwCRpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaqtbnO&md5=f67070c41c38363b79caa52e11000a37</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.3390%2Fijms131012761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms131012761%26sid%3Dliteratum%253Aachs%26aulast%3DNadzirin%26aufirst%3DN.%26aulast%3DFirdaus-Raih%26aufirst%3DM.%26atitle%3DProteins%2520of%2520unknown%2520function%2520in%2520the%2520Protein%2520Data%2520Bank%2520%2528PDB%2529%253A%2520an%2520inventory%2520of%2520true%2520uncharacterized%2520proteins%2520and%2520computational%2520tools%2520for%2520their%2520analysis%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2012%26volume%3D13%26issue%3D10%26spage%3D12761%26epage%3D12772%26doi%3D10.3390%2Fijms131012761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref169"><div class="reference"><strong class="refLabel"><a href="#ref169" class="rightTabRefNumLink">169</a></strong><div class="NLM_citation" id="rightTab-cit169"><span><span class="NLM_contrib-group">Petrey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garzon, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasso, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silkov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Template-based prediction of protein function</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1016/j.sbi.2015.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.sbi.2015.01.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25678152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVCqu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2015&pages=33-38&author=D.+Petreyauthor=T.+S.+Chenauthor=L.+Dengauthor=J.+I.+Garzonauthor=H.+Hwangauthor=G.+Lassoauthor=H.+Leeauthor=A.+Silkovauthor=B.+Honig&title=Template-based+prediction+of+protein+function&doi=10.1016%2Fj.sbi.2015.01.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">Template-based prediction of protein function</span></div><div class="casAuthors">Petrey, Donald; Chen, T. Scott; Deng, Lei; Garzon, Jose Ignacio; Hwang, Howook; Lasso, Gorka; Lee, Hunjoong; Silkov, Antonina; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33-38</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  We discuss recent approaches for structure-based protein function annotation.  We focus on template-based methods where the function of a query protein is deduced from that of a template for which both the structure and function are known.  We describe the different ways of identifying a template.  These are typically based on sequence anal. but new methods based on purely structural similarity are also being developed that allow function annotation based on structural relationships that cannot be recognized by sequence.  The growing no. of available structures of known function, improved homol. modeling techniques and new developments in the use of structure allow template-based methods to be applied on a proteome-wide scale and in many different biol. contexts.  This progress significantly expands the range of applicability of structural information in function annotation to a level that previously was only achievable by sequence comparison.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobUm3rclLCLbVg90H21EOLACvtfcHk0licXKUXMEIr9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVCqu7Y%253D&md5=fa08b4069309cbd6bcc350a0158ed591</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2015.01.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2015.01.007%26sid%3Dliteratum%253Aachs%26aulast%3DPetrey%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%2BS.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DGarzon%26aufirst%3DJ.%2BI.%26aulast%3DHwang%26aufirst%3DH.%26aulast%3DLasso%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DSilkov%26aufirst%3DA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DTemplate-based%2520prediction%2520of%2520protein%2520function%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2015%26volume%3D32%26spage%3D33%26epage%3D38%26doi%3D10.1016%2Fj.sbi.2015.01.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref170"><div class="reference"><strong class="refLabel"><a href="#ref170" class="rightTabRefNumLink">170</a></strong><div class="NLM_citation" id="rightTab-cit170"><span><span class="NLM_contrib-group">Hwang, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, Y.</span><span> </span><span class="NLM_article-title">Structure-based identification of a novel NTPase from Methanococcus jannaschii</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">696</span><span class="refDoi"> DOI: 10.1038/10745</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1038%2F10745" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=691-696&issue=7&author=K.+Y.+Hwangauthor=J.+H.+Chungauthor=S.+H.+Kimauthor=Y.+S.+Hanauthor=Y.+Cho&title=Structure-based+identification+of+a+novel+NTPase+from+Methanococcus+jannaschii&doi=10.1038%2F10745"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1038%2F10745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F10745%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DK.%2BY.%26aulast%3DChung%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DHan%26aufirst%3DY.%2BS.%26aulast%3DCho%26aufirst%3DY.%26atitle%3DStructure-based%2520identification%2520of%2520a%2520novel%2520NTPase%2520from%2520Methanococcus%2520jannaschii%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26issue%3D7%26spage%3D691%26epage%3D696%26doi%3D10.1038%2F10745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref171"><div class="reference"><strong class="refLabel"><a href="#ref171" class="rightTabRefNumLink">171</a></strong><div class="NLM_citation" id="rightTab-cit171"><span><span class="NLM_contrib-group">Pieper, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, G. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneidman-Duhovny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spill, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinkam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">ModBase, a database of annotated comparative protein structure models and associated resources</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D336</span><span class="NLM_x">–</span> <span class="NLM_lpage">D346</span><div class="note"><p class="first last">(Database issue)</p></div><span class="refDoi"> DOI: 10.1093/nar/gkt1144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2Fgkt1144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24271400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Gj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D336-D346&issue=D1&author=U.+Pieperauthor=B.+M.+Webbauthor=G.+Q.+Dongauthor=D.+Schneidman-Duhovnyauthor=H.+Fanauthor=S.+J.+Kimauthor=N.+Khuriauthor=Y.+G.+Spillauthor=P.+Weinkamauthor=M.+Hammelauthor=J.+A.+Tainerauthor=M.+Nilgesauthor=A.+Sali&title=ModBase%2C+a+database+of+annotated+comparative+protein+structure+models+and+associated+resources&doi=10.1093%2Fnar%2Fgkt1144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">ModBase, a database of annotated comparative protein structure models and associated resources</span></div><div class="casAuthors">Pieper, Ursula; Webb, Benjamin M.; Dong, Guang Qiang; Schneidman-Duhovny, Dina; Fan, Hao; Kim, Seung Joong; Khuri, Natalia; Spill, Yannick G.; Weinkam, Patrick; Hammel, Michal; Tainer, John A.; Nilges, Michael; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D336-D346</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ModBase http://salilab.org/modbase is a database of annotated comparative protein structure models.  The models are calcd. by ModPipe, an automated modeling pipeline that relies primarily on Modeller for fold assignment, sequence-structure alignment, model building and model assessment http://salilab.org/modeller/.  ModBase currently contains almost 30 million reliable models for domains in 4.7 million unique protein sequences.  ModBase allows users to compute or update comparative models on demand, through an interface to the ModWeb modeling server http://salilab.org/modweb.  ModBase models are also available through the Protein Model Portal http://www.proteinmodelportal.org/.  Recently developed assocd. resources include the AllosMod server for modeling ligand-induced protein dynamics http://salilab.org/allosmod, the AllosMod-FoXS server for predicting a structural ensemble that fits an SAXS profile http://salilab.org/allosmod-foxs, the FoXSDock server for protein-protein docking filtered by an SAXS profile http://salilab.org/foxsdock, the SAXS Merge server for automatic merging of SAXS profiles http://salilab.org/saxsmerge and the Pose & Rank server for scoring protein-ligand complexes http://salilab.org/poseandrank. In this update, we also highlight two applications of ModBase: a PSI:Biol. initiative to maximize the structural coverage of the human alpha-helical transmembrane proteome and a detn. of structural determinants of human immunodeficiency virus-1 protease specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomlF6VKj_t4bVg90H21EOLACvtfcHk0licXKUXMEIr9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Gj&md5=1a0809c46145a0a783ad5d5b2a6e3326</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1144%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DU.%26aulast%3DWebb%26aufirst%3DB.%2BM.%26aulast%3DDong%26aufirst%3DG.%2BQ.%26aulast%3DSchneidman-Duhovny%26aufirst%3DD.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DKhuri%26aufirst%3DN.%26aulast%3DSpill%26aufirst%3DY.%2BG.%26aulast%3DWeinkam%26aufirst%3DP.%26aulast%3DHammel%26aufirst%3DM.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModBase%252C%2520a%2520database%2520of%2520annotated%2520comparative%2520protein%2520structure%2520models%2520and%2520associated%2520resources%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD336%26epage%3DD346%26doi%3D10.1093%2Fnar%2Fgkt1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref172"><div class="reference"><strong class="refLabel"><a href="#ref172" class="rightTabRefNumLink">172</a></strong><div class="NLM_citation" id="rightTab-cit172"><span><span class="NLM_contrib-group">Yin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garen, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyon, E. Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-w.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, M. N. G.</span><span> </span><span class="NLM_article-title">Expression, purification and preliminary crystallographic analysis of O-acetylhomoserine sulfhydrylase from Mycobacterium tuberculosis</span> <span class="citation_source-journal">Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x"> (</span><span class="NLM_issue">Part 8</span><span class="NLM_x">) </span> <span class="NLM_fpage">959</span><span class="NLM_x">–</span> <span class="NLM_lpage">963</span><span class="refDoi"> DOI: 10.1107/S1744309111017611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1107%2FS1744309111017611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=21821905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVGkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=959-963&issue=Part+8&author=J.+Yinauthor=C.+R.+Garenauthor=K.+Batemanauthor=M.+Yuauthor=E.+Z.+A.+Lyonauthor=J.+Habelauthor=H.+Kimauthor=L.-w.+Hungauthor=C.-Y.+Kimauthor=M.+N.+G.+James&title=Expression%2C+purification+and+preliminary+crystallographic+analysis+of+O-acetylhomoserine+sulfhydrylase+from+Mycobacterium+tuberculosis&doi=10.1107%2FS1744309111017611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification and preliminary crystallographic analysis of O-acetylhomoserine sulfhydrylase from Mycobacterium tuberculosis</span></div><div class="casAuthors">Yin, Jiang; Garen, Craig R.; Bateman, Katherine; Yu, Minmin; Alipio Lyon, Emily Z.; Habel, Jeff; Kim, Heungbok; Hung, Li-wei; Kim, Chang-Yub; James, Michael N. G.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology and Crystallization Communications</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">959-963</span>CODEN:
                <span class="NLM_cas:coden">ACSFCL</span>;
        ISSN:<span class="NLM_cas:issn">1744-3091</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The gene product of open reading frame Rv3340 of M. tuberculosis was annotated as encoding a probable O-acetylhomoserine sulfhydrylase (MetC), an enzyme that catalyzes the last step in the biosynthesis of methionine, which is an essential amino acid in bacteria and plants.  Following overexpression in Escherichia coli, the M. tuberculosis MetC enzyme was purified and crystd. using the hanging-drop vapor-diffusion method.  Native diffraction data were collected from 2 crystal forms belonging to space group P21 and were processed to a resoln. of 2.1 Å.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcFq5HnJcd8rVg90H21EOLACvtfcHk0ljSElsxIzpWiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVGkurY%253D&md5=782fae003f14f408db2660e18e5ece49</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1107%2FS1744309111017611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309111017611%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DGaren%26aufirst%3DC.%2BR.%26aulast%3DBateman%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DLyon%26aufirst%3DE.%2BZ.%2BA.%26aulast%3DHabel%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DHung%26aufirst%3DL.-w.%26aulast%3DKim%26aufirst%3DC.-Y.%26aulast%3DJames%26aufirst%3DM.%2BN.%2BG.%26atitle%3DExpression%252C%2520purification%2520and%2520preliminary%2520crystallographic%2520analysis%2520of%2520O-acetylhomoserine%2520sulfhydrylase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2011%26volume%3D67%26issue%3DPart%25208%26spage%3D959%26epage%3D963%26doi%3D10.1107%2FS1744309111017611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref173"><div class="reference"><strong class="refLabel"><a href="#ref173" class="rightTabRefNumLink">173</a></strong><div class="NLM_citation" id="rightTab-cit173"><span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">I-TASSER server for protein 3D structure prediction</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="refDoi"> DOI: 10.1186/1471-2105-9-40</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1471-2105-9-40" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=18215316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BD1c7itFaksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=40&author=Y.+Zhang&title=I-TASSER+server+for+protein+3D+structure+prediction&doi=10.1186%2F1471-2105-9-40"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">I-TASSER server for protein 3D structure prediction</span></div><div class="casAuthors">Zhang Yang</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Prediction of 3-dimensional protein structures from amino acid sequences represents one of the most important problems in computational structural biology.  The community-wide Critical Assessment of Structure Prediction (CASP) experiments have been designed to obtain an objective assessment of the state-of-the-art of the field, where I-TASSER was ranked as the best method in the server section of the recent 7th CASP experiment.  Our laboratory has since then received numerous requests about the public availability of the I-TASSER algorithm and the usage of the I-TASSER predictions.  RESULTS:  An on-line version of I-TASSER is developed at the KU Center for Bioinformatics which has generated protein structure predictions for thousands of modeling requests from more than 35 countries.  A scoring function (C-score) based on the relative clustering structural density and the consensus significance score of multiple threading templates is introduced to estimate the accuracy of the I-TASSER predictions.  A large-scale benchmark test demonstrates a strong correlation between the C-score and the TM-score (a structural similarity measurement with values in [0, 1]) of the first models with a correlation coefficient of 0.91.  Using a C-score cutoff > -1.5 for the models of correct topology, both false positive and false negative rates are below 0.1.  Combining C-score and protein length, the accuracy of the I-TASSER models can be predicted with an average error of 0.08 for TM-score and 2 A for RMSD.  CONCLUSION:  The I-TASSER server has been developed to generate automated full-length 3D protein structural predictions where the benchmarked scoring system helps users to obtain quantitative assessments of the I-TASSER models.  The output of the I-TASSER server for each query includes up to five full-length models, the confidence score, the estimated TM-score and RMSD, and the standard deviation of the estimations.  The I-TASSER server is freely available to the academic community at http://zhang.bioinformatics.ku.edu/I-TASSER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUJ3biR9wC1IvKgEt20y1XfW6udTcc2eb1eGJlco7IDLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7itFaksQ%253D%253D&md5=48ded5b2e40af27acf1e5753f827080c</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-40&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-40%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DI-TASSER%2520server%2520for%2520protein%25203D%2520structure%2520prediction%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26spage%3D40%26doi%3D10.1186%2F1471-2105-9-40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref174"><div class="reference"><strong class="refLabel"><a href="#ref174" class="rightTabRefNumLink">174</a></strong><div class="NLM_citation" id="rightTab-cit174"><span><span class="NLM_contrib-group">Farrah, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutsch, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoopmann, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallows, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moritz, R. L.</span><span> </span><span class="NLM_article-title">The state of the human proteome in 2012 as viewed through PeptideAtlas</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">162</span><span class="NLM_x">–</span> <span class="NLM_lpage">171</span><span class="refDoi"> DOI: 10.1021/pr301012j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr301012j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOjsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=162-171&issue=1&author=T.+Farrahauthor=E.+W.+Deutschauthor=M.+R.+Hoopmannauthor=J.+L.+Hallowsauthor=Z.+Sunauthor=C.-Y.+Huangauthor=R.+L.+Moritz&title=The+state+of+the+human+proteome+in+2012+as+viewed+through+PeptideAtlas&doi=10.1021%2Fpr301012j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">The State of the Human Proteome in 2012 as Viewed through PeptideAtlas</span></div><div class="casAuthors">Farrah, Terry; Deutsch, Eric W.; Hoopmann, Michael R.; Hallows, Janice L.; Sun, Zhi; Huang, Chung-Ying; Moritz, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">162-171</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Human Proteome Project was launched in Sept. 2010 with the goal of characterizing at least one protein product from each protein-coding gene.  Here we assess how much of the proteome has been detected to date via tandem mass spectrometry by analyzing PeptideAtlas, a compendium of human derived LC-MS/MS proteomics data from many labs. around the world.  All data sets are processed with a consistent set of parameters using the Trans-Proteomic Pipeline and subjected to a 1% protein FDR filter before inclusion in PeptideAtlas.  Therefore, PeptideAtlas contains only high confidence protein identifications.  To increase proteome coverage, we explored new comprehensive public data sources for data likely to add new proteins to the Human PeptideAtlas.  We then folded these data into a Human PeptideAtlas 2012 build and mapped it to Swiss-Prot, a protein sequence database curated to contain one entry per human protein coding gene.  We find that this latest PeptideAtlas build includes at least one peptide for each of ∼12500 Swiss-Prot entries, leaving ∼7500 gene products yet to be confidently cataloged.  We characterize these "PA-unseen" proteins in terms of tissue localization, transcript abundance, and Gene Ontol. enrichment, and propose reasons for their absence from PeptideAtlas and strategies for detecting them in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMvIy0oVVhlrVg90H21EOLACvtfcHk0ljSElsxIzpWiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOjsb7N&md5=49221bf15c990d2ea6566baf7f4b2961</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1021%2Fpr301012j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr301012j%26sid%3Dliteratum%253Aachs%26aulast%3DFarrah%26aufirst%3DT.%26aulast%3DDeutsch%26aufirst%3DE.%2BW.%26aulast%3DHoopmann%26aufirst%3DM.%2BR.%26aulast%3DHallows%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DC.-Y.%26aulast%3DMoritz%26aufirst%3DR.%2BL.%26atitle%3DThe%2520state%2520of%2520the%2520human%2520proteome%2520in%25202012%2520as%2520viewed%2520through%2520PeptideAtlas%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2013%26volume%3D12%26issue%3D1%26spage%3D162%26epage%3D171%26doi%3D10.1021%2Fpr301012j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref175"><div class="reference"><strong class="refLabel"><a href="#ref175" class="rightTabRefNumLink">175</a></strong><div class="NLM_citation" id="rightTab-cit175"><span><span class="NLM_contrib-group">Caetano-Anollés, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caetano-Anollés, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittenthal, J. E.</span><span> </span><span class="NLM_article-title">The origin, evolution and structure of the protein world</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">417</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">621</span><span class="NLM_x">–</span> <span class="NLM_lpage">637</span><span class="refDoi"> DOI: 10.1042/BJ20082063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1042%2FBJ20082063" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2009&pages=621-637&issue=3&author=G.+Caetano-Anoll%C3%A9sauthor=M.+Wangauthor=D.+Caetano-Anoll%C3%A9sauthor=J.+E.+Mittenthal&title=The+origin%2C+evolution+and+structure+of+the+protein+world&doi=10.1042%2FBJ20082063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1042%2FBJ20082063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20082063%26sid%3Dliteratum%253Aachs%26aulast%3DCaetano-Anoll%25C3%25A9s%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCaetano-Anoll%25C3%25A9s%26aufirst%3DD.%26aulast%3DMittenthal%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520origin%252C%2520evolution%2520and%2520structure%2520of%2520the%2520protein%2520world%26jtitle%3DBiochem.%2520J.%26date%3D2009%26volume%3D417%26issue%3D3%26spage%3D621%26epage%3D637%26doi%3D10.1042%2FBJ20082063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref176"><div class="reference"><strong class="refLabel"><a href="#ref176" class="rightTabRefNumLink">176</a></strong><div class="NLM_citation" id="rightTab-cit176"><span><span class="NLM_contrib-group">Naumann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matter, H.</span><span> </span><span class="NLM_article-title">Structural classification of protein kinases using 3D molecular interaction field analysis of their ligand binding sites: target family landscapes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2366</span><span class="NLM_x">–</span> <span class="NLM_lpage">2378</span><span class="refDoi"> DOI: 10.1021/jm011002c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm011002c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2366-2378&issue=12&author=T.+Naumannauthor=H.+Matter&title=Structural+classification+of+protein+kinases+using+3D+molecular+interaction+field+analysis+of+their+ligand+binding+sites%3A+target+family+landscapes&doi=10.1021%2Fjm011002c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1021%2Fjm011002c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm011002c%26sid%3Dliteratum%253Aachs%26aulast%3DNaumann%26aufirst%3DT.%26aulast%3DMatter%26aufirst%3DH.%26atitle%3DStructural%2520classification%2520of%2520protein%2520kinases%2520using%25203D%2520molecular%2520interaction%2520field%2520analysis%2520of%2520their%2520ligand%2520binding%2520sites%253A%2520target%2520family%2520landscapes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26issue%3D12%26spage%3D2366%26epage%3D2378%26doi%3D10.1021%2Fjm011002c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref177"><div class="reference"><strong class="refLabel"><a href="#ref177" class="rightTabRefNumLink">177</a></strong><div class="NLM_citation" id="rightTab-cit177"><span><span class="NLM_contrib-group">Altschul, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schäffer, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipman, D. J.</span><span> </span><span class="NLM_article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">3389</span><span class="NLM_x">–</span> <span class="NLM_lpage">3402</span><span class="refDoi"> DOI: 10.1093/nar/25.17.3389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2F25.17.3389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=9254694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=3389-3402&issue=17&author=S.+F.+Altschulauthor=T.+L.+Maddenauthor=A.+A.+Sch%C3%A4fferauthor=J.+Zhangauthor=Z.+Zhangauthor=W.+Millerauthor=D.+J.+Lipman&title=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs&doi=10.1093%2Fnar%2F25.17.3389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span></div><div class="casAuthors">Altschul, Stephen F.; Madden, Thomas L.; Schaffer, Alejandro A.; Zhang, Jinghui; Zhang, Zheng; Miller, Webb; Lipman, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3389-3402</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The BLAST programs are widely used tools for searching protein and DNA databases for sequence similarities.  For protein comparisons, a variety of definitional, algorithmic and statistical refinements described here permits the execution time of the BLAST programs to be decreased substantially while enhancing their sensitivity to weak similarities.  A new criterion for triggering the extension of word hits, combined with a new heuristic for generating gapped alignments, yields a gapped BLAST program that runs at approx. three times the speed of the original.  In addn., a method is introduced for automatically combining statistically significant alignments produced by BLAST into a position-specific score matrix, and searching the database using this matrix.  The resulting Position-Specific Iterated BLAST (PSI-BLAST) program runs at approx. the same speed per iteration as gapped BLAST, but in many cases is much more sensitive to weak but biol. relevant sequence similarities.  PSI-BLAST is used to uncover several new and interesting members of the BRCT superfamily.  The source code for the new BLAST programs is available by anonymous ftp from the machine ncbi.nlm.nih.gov, within the directory 'blast', and the programs may be run from NCBIs web site at http://www.ncbi.nlm.nih.gov/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfl-GGLfjeoLVg90H21EOLACvtfcHk0lhkg9wPcuMtbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D&md5=4e44123e5984e4aca46a9899d347a176</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F25.17.3389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F25.17.3389%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26aulast%3DMadden%26aufirst%3DT.%2BL.%26aulast%3DSch%25C3%25A4ffer%26aufirst%3DA.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DMiller%26aufirst%3DW.%26aulast%3DLipman%26aufirst%3DD.%2BJ.%26atitle%3DGapped%2520BLAST%2520and%2520PSI-BLAST%253A%2520a%2520new%2520generation%2520of%2520protein%2520database%2520search%2520programs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26issue%3D17%26spage%3D3389%26epage%3D3402%26doi%3D10.1093%2Fnar%2F25.17.3389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref178"><div class="reference"><strong class="refLabel"><a href="#ref178" class="rightTabRefNumLink">178</a></strong><div class="NLM_citation" id="rightTab-cit178"><span><span class="NLM_contrib-group">Bairoch, A.</span><span> </span><span class="NLM_article-title">PROSITE: a dictionary of sites and patterns in proteins</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl</span><span class="NLM_x">) </span> <span class="NLM_fpage">2241</span><span class="NLM_x">–</span> <span class="NLM_lpage">2245</span><span class="refDoi"> DOI: 10.1093/nar/19.suppl.2241</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1093%2Fnar%2F19.suppl.2241" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1991&pages=2241-2245&issue=Suppl&author=A.+Bairoch&title=PROSITE%3A+a+dictionary+of+sites+and+patterns+in+proteins&doi=10.1093%2Fnar%2F19.suppl.2241"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F19.suppl.2241&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F19.suppl.2241%26sid%3Dliteratum%253Aachs%26aulast%3DBairoch%26aufirst%3DA.%26atitle%3DPROSITE%253A%2520a%2520dictionary%2520of%2520sites%2520and%2520patterns%2520in%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1991%26volume%3D19%26issue%3DSuppl%26spage%3D2241%26epage%3D2245%26doi%3D10.1093%2Fnar%2F19.suppl.2241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref179"><div class="reference"><strong class="refLabel"><a href="#ref179" class="rightTabRefNumLink">179</a></strong><div class="NLM_citation" id="rightTab-cit179"><span><span class="NLM_contrib-group">Chupakhin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaspar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varnek, A.</span><span> </span><span class="NLM_article-title">Simple Ligand-Receptor Interaction Descriptor (SILIRID) for alignment-free binding site comparison</span> <span class="citation_source-journal">Comput. Struct. Biotechnol. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">16</span><span class="NLM_x">) </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">37</span><span class="refDoi"> DOI: 10.1016/j.csbj.2014.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.csbj.2014.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=25210596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A280%3ADC%252BC2M7gsVSqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=33-37&issue=16&author=V.+Chupakhinauthor=G.+Marcouauthor=H.+Gasparauthor=A.+Varnek&title=Simple+Ligand-Receptor+Interaction+Descriptor+%28SILIRID%29+for+alignment-free+binding+site+comparison&doi=10.1016%2Fj.csbj.2014.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">Simple Ligand-Receptor Interaction Descriptor (SILIRID) for alignment-free binding site comparison</span></div><div class="casAuthors">Chupakhin Vladimir; Marcou Gilles; Gaspar Helena; Varnek Alexandre</div><div class="citationInfo"><span class="NLM_cas:title">Computational and structural biotechnology journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">33-7</span>
        ISSN:<span class="NLM_cas:issn">2001-0370</span>.
    </div><div class="casAbstract">We describe SILIRID (Simple Ligand-Receptor Interaction Descriptor), a novel fixed size descriptor characterizing protein-ligand interactions.  SILIRID can be obtained from the binary interaction fingerprints (IFPs) by summing up the bits corresponding to identical amino acids.  This results in a vector of 168 integer numbers corresponding to the product of the number of entries (20 amino acids and one cofactor) and 8 interaction types per amino acid (hydrophobic, aromatic face to face, aromatic edge to face, H-bond donated by the protein, H-bond donated by the ligand, ionic bond with protein cation and protein anion, and interaction with metal ion).  Efficiency of SILIRID to distinguish different protein binding sites has been examined in similarity search in sc-PDB database, a druggable portion of the Protein Data Bank, using various protein-ligand complexes as queries.  The performance of retrieval of structurally and evolutionary related classes of proteins was comparable to that of state-of-the-art approaches (ROC AUC ≈ 0.91).  SILIRID can efficiently be used to visualize chemogenomic space covered by sc-PDB using Generative Topographic Mapping (GTM): sc-PDB SILIRID data form clusters corresponding to different protein types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSrYGB8Py17ciA3UY9EwTNrfW6udTcc2eamgedSEeKOjrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7gsVSqug%253D%253D&md5=6c4111f9f23033e5365f1c46ca40226f</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1016%2Fj.csbj.2014.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.csbj.2014.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DChupakhin%26aufirst%3DV.%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DGaspar%26aufirst%3DH.%26aulast%3DVarnek%26aufirst%3DA.%26atitle%3DSimple%2520Ligand-Receptor%2520Interaction%2520Descriptor%2520%2528SILIRID%2529%2520for%2520alignment-free%2520binding%2520site%2520comparison%26jtitle%3DComput.%2520Struct.%2520Biotechnol.%2520J.%26date%3D2014%26volume%3D10%26issue%3D16%26spage%3D33%26epage%3D37%26doi%3D10.1016%2Fj.csbj.2014.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref180"><div class="reference"><strong class="refLabel"><a href="#ref180" class="rightTabRefNumLink">180</a></strong><div class="NLM_citation" id="rightTab-cit180"><span><span class="NLM_contrib-group">Chartier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R.</span><span> </span><span class="NLM_article-title">Detection of binding site molecular interaction field similarities</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1600</span><span class="NLM_x">–</span> <span class="NLM_lpage">1615</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00333</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00333" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1600-1615&issue=8&author=M.+Chartierauthor=R.+Najmanovich&title=Detection+of+binding+site+molecular+interaction+field+similarities&doi=10.1021%2Facs.jcim.5b00333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of Binding Site Molecular Interaction Field Similarities</span></div><div class="casAuthors">Chartier, Matthieu; Najmanovich, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1600-1615</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein binding-site similarity detection methods can be used to predict protein function and understand mol. recognition, as a tool in drug design for drug repurposing and polypharmacol., and for the prediction of the mol. determinants of drug toxicity.  Here, the authors present IsoMIF, a method able to identify binding site mol. interaction field similarities across protein families.  IsoMIF utilizes six chem. probes and the detection of subgraph isomorphisms to identify geometrically and chem. equiv. sections of protein cavity pairs.  The method is validated using six distinct data sets, four of those previously used in the validation of other methods.  The mean area under the receiver operator curve (AUC) obtained across data sets for IsoMIF is higher than those of other methods.  Furthermore, while IsoMIF obtains consistently high AUC values across data sets, other methods perform more erratically across data sets.  IsoMIF can be used to predict function from structure, to detect potential cross-reactivity or polypharmacol. targets, and to help suggest bioisosteric replacements to known binding mols.  Given that IsoMIF detects spatial patterns of mol. interaction field similarities, its predictions are directly related to pharmacophores and may be readily translated into modeling decisions in structure-based drug design.  IsoMIF may in principle detect similar binding sites with distinct amino acid arrangements that lead to equiv. interactions within the cavity.  The source code to calc. and visualize MIFs and MIF similarities are freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-qggIKAgadbVg90H21EOLACvtfcHk0liR0eSHwo5-BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfK&md5=4fdab34c9233108eb40b9a0e7052c357</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00333%26sid%3Dliteratum%253Aachs%26aulast%3DChartier%26aufirst%3DM.%26aulast%3DNajmanovich%26aufirst%3DR.%26atitle%3DDetection%2520of%2520binding%2520site%2520molecular%2520interaction%2520field%2520similarities%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D8%26spage%3D1600%26epage%3D1615%26doi%3D10.1021%2Facs.jcim.5b00333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref181"><div class="reference"><strong class="refLabel"><a href="#ref181" class="rightTabRefNumLink">181</a></strong><div class="NLM_citation" id="rightTab-cit181"><span><span class="NLM_contrib-group">Kinjo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Similarity search for local protein structures at atomic resolution by exploiting a database management system</span> <span class="citation_source-journal">Biophysics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.2142/biophysics.3.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.2142%2Fbiophysics.3.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1agu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=75-84&author=A.+R.+Kinjoauthor=H.+Nakamura&title=Similarity+search+for+local+protein+structures+at+atomic+resolution+by+exploiting+a+database+management+system&doi=10.2142%2Fbiophysics.3.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit181R"><div class="casContent"><span class="casTitleNuber">181</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity search for local protein structures at atomic resolution by exploiting a database management system</span></div><div class="casAuthors">Kinjo, Akira R.; Nakamura, Haruki</div><div class="citationInfo"><span class="NLM_cas:title">Biophysics (Biophysical Society of Japan)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-84</span>CODEN:
                <span class="NLM_cas:coden">BBSJC3</span>;
        ISSN:<span class="NLM_cas:issn">1349-2942</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society of Japan</span>)
        </div><div class="casAbstract">A method to search for local structural similarities in proteins at at. resoln. is presented.  It is demonstrated that a huge amt. of structural data can be handled within a reasonable CPU time by using a conventional relational database management system with appropriate indexing of geometric data.  This method, which we call geometric indexing, can enumerate ligand binding sites that are structurally similar to sub-structures of a query protein among more than 160,000 possible candidates within a few hours of CPU time on an ordinary desktop computer.  After detecting a set of high scoring ligand binding sites by the geometric indexing search, structural alignments at at. resoln. are constructed by iteratively applying the Hungarian algorithm, and the statistical significance of the final score is estd. from an empirical model based on a gamma distribution.  Applications of this method to several protein structures clearly shows that significant similarities can be detected between local structures of non-homologous as well as homologous proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7yLbA2BOqCrVg90H21EOLACvtfcHk0liR0eSHwo5-BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1agu7Y%253D&md5=a9c57dd41e31f99177bd977def356283</span></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.2142%2Fbiophysics.3.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2142%252Fbiophysics.3.75%26sid%3Dliteratum%253Aachs%26aulast%3DKinjo%26aufirst%3DA.%2BR.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DSimilarity%2520search%2520for%2520local%2520protein%2520structures%2520at%2520atomic%2520resolution%2520by%2520exploiting%2520a%2520database%2520management%2520system%26jtitle%3DBiophysics%26date%3D2007%26volume%3D3%26spage%3D75%26epage%3D84%26doi%3D10.2142%2Fbiophysics.3.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref182"><div class="reference"><strong class="refLabel"><a href="#ref182" class="rightTabRefNumLink">182</a></strong><div class="NLM_citation" id="rightTab-cit182"><span><span class="NLM_contrib-group">Kinjo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Comprehensive structural classification of ligand-binding motifs in proteins</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">234</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1016/j.str.2008.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.str.2008.11.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=234-246&issue=2&author=A.+R.+Kinjoauthor=H.+Nakamura&title=Comprehensive+structural+classification+of+ligand-binding+motifs+in+proteins&doi=10.1016%2Fj.str.2008.11.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2008.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2008.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DKinjo%26aufirst%3DA.%2BR.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DComprehensive%2520structural%2520classification%2520of%2520ligand-binding%2520motifs%2520in%2520proteins%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26issue%3D2%26spage%3D234%26epage%3D246%26doi%3D10.1016%2Fj.str.2008.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref183"><div class="reference"><strong class="refLabel"><a href="#ref183" class="rightTabRefNumLink">183</a></strong><div class="NLM_citation" id="rightTab-cit183"><span><span class="NLM_contrib-group">Friedberg, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godzik, A.</span><span> </span><span class="NLM_article-title">Connecting the protein structure universe by using sparse recurring fragments</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1213</span><span class="NLM_x">–</span> <span class="NLM_lpage">1224</span><span class="refDoi"> DOI: 10.1016/j.str.2005.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1016%2Fj.str.2005.05.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=1213-1224&issue=8&author=I.+Friedbergauthor=A.+Godzik&title=Connecting+the+protein+structure+universe+by+using+sparse+recurring+fragments&doi=10.1016%2Fj.str.2005.05.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2005.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2005.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DFriedberg%26aufirst%3DI.%26aulast%3DGodzik%26aufirst%3DA.%26atitle%3DConnecting%2520the%2520protein%2520structure%2520universe%2520by%2520using%2520sparse%2520recurring%2520fragments%26jtitle%3DStructure%26date%3D2005%26volume%3D13%26issue%3D8%26spage%3D1213%26epage%3D1224%26doi%3D10.1016%2Fj.str.2005.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref184"><div class="reference"><strong class="refLabel"><a href="#ref184" class="rightTabRefNumLink">184</a></strong><div class="NLM_citation" id="rightTab-cit184"><span><span class="NLM_contrib-group">Godshall, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B. Y.</span><span> </span><span class="NLM_article-title">An aggregate analysis of many predicted structures to reduce errors in protein structure comparison caused by conformational flexibility</span> <span class="citation_source-journal">BMC Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S10</span><span class="refDoi"> DOI: 10.1186/1472-6807-13-S1-S10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.1186%2F1472-6807-13-S1-S10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=S10&issue=Suppl.+1&author=B.+G.+Godshallauthor=Y.+Tangauthor=W.+Yangauthor=B.+Y.+Chen&title=An+aggregate+analysis+of+many+predicted+structures+to+reduce+errors+in+protein+structure+comparison+caused+by+conformational+flexibility&doi=10.1186%2F1472-6807-13-S1-S10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1186%2F1472-6807-13-S1-S10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6807-13-S1-S10%26sid%3Dliteratum%253Aachs%26aulast%3DGodshall%26aufirst%3DB.%2BG.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DB.%2BY.%26atitle%3DAn%2520aggregate%2520analysis%2520of%2520many%2520predicted%2520structures%2520to%2520reduce%2520errors%2520in%2520protein%2520structure%2520comparison%2520caused%2520by%2520conformational%2520flexibility%26jtitle%3DBMC%2520Struct.%2520Biol.%26date%3D2013%26volume%3D13%26issue%3DSuppl.%25201%26spage%3DS10%26doi%3D10.1186%2F1472-6807-13-S1-S10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref185"><div class="reference"><strong class="refLabel"><a href="#ref185" class="rightTabRefNumLink">185</a></strong><div class="NLM_citation" id="rightTab-cit185"><span><span class="NLM_contrib-group">Volkamer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Exploiting structural information for drug-target assessment</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">331</span><span class="refDoi"> DOI: 10.4155/fmc.14.3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=10.4155%2Ffmc.14.3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=24575967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlCqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=319-331&issue=3&author=A.+Volkamerauthor=M.+Rarey&title=Exploiting+structural+information+for+drug-target+assessment&doi=10.4155%2Ffmc.14.3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting structural information for drug-target assessment</span></div><div class="casAuthors">Volkamer, Andrea; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">319-331</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The amt. of known protein structures is continuously growing, exhibited in >95,000 3-dimensional structures freely available via the PDB.  Over the last decade, pharmaceutical research has sparked interest in computationally extg. information from this large data pool, resulting in a homol.-driven knowledge transfer from annotated to new structures.  Studying protein structures with respect to understanding and modulating their functional behavior means analyzing their centers of action.  Therefore, the detection and description of potential binding sites on the protein surface is a major step toward protein classification and assessment.  Subsequently, these representations can be incorporated to compare proteins, and to predict their druggability or function.  Esp. in the context of target identification and polypharmacol., automated tools for large-scale target comparisons are highly needed.  Here, developments for automated structure-based target assessment are reviewed and remaining challenges as well as future perspectives are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEy7ASYbcMYrVg90H21EOLACvtfcHk0liR0eSHwo5-BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlCqsr4%253D&md5=f15483dd54f1ad555c4d19781b98a8de</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.3%26sid%3Dliteratum%253Aachs%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DExploiting%2520structural%2520information%2520for%2520drug-target%2520assessment%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26issue%3D3%26spage%3D319%26epage%3D331%26doi%3D10.4155%2Ffmc.14.3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r0i" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r0i','PDB','4r0i'); return false;">PDB: 4r0i</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uk4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1uk4','PDB','1uk4'); return false;">PDB: 1uk4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ejr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ejr','PDB','2ejr'); return false;">PDB: 2ejr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2bxr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2bxr','PDB','2bxr'); return false;">PDB: 2bxr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1gos" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1gos','PDB','1gos'); return false;">PDB: 1gos</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1oq5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1oq5','PDB','1oq5'); return false;">PDB: 1oq5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6cox" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6cox','PDB','6cox'); return false;">PDB: 6cox</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bn4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bn4','PDB','1bn4'); return false;">PDB: 1bn4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4tzk" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4tzk','PDB','4tzk'); return false;">PDB: 4tzk</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cl5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2cl5','PDB','2cl5'); return false;">PDB: 2cl5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1eve" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1eve','PDB','1eve'); return false;">PDB: 1eve</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ohr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ohr','PDB','1ohr'); return false;">PDB: 1ohr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3c0y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3c0y','PDB','3c0y'); return false;">PDB: 3c0y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f8u" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1f8u','PDB','1f8u'); return false;">PDB: 1f8u</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yhs" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1yhs','PDB','1yhs'); return false;">PDB: 1yhs</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aux" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1aux','PDB','1aux'); return false;">PDB: 1aux</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1px2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1px2','PDB','1px2'); return false;">PDB: 1px2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a99" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3a99','PDB','3a99'); return false;">PDB: 3a99</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dxd" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dxd','PDB','4dxd'); return false;">PDB: 4dxd</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vo8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vo8','PDB','3vo8'); return false;">PDB: 3vo8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4m8i" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4m8i','PDB','4m8i'); return false;">PDB: 4m8i</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2rhl" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2rhl','PDB','2rhl'); return false;">PDB: 2rhl</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k9p" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k9p','PDB','3k9p'); return false;">PDB: 3k9p</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ojq" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ojq','PDB','2ojq'); return false;">PDB: 2ojq</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ysn" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ysn','PDB','1ysn'); return false;">PDB: 1ysn</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1atp" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1atp','PDB','1atp'); return false;">PDB: 1atp</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2c3e" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2c3e','PDB','2c3e'); return false;">PDB: 2c3e</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2oyu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2oyu','PDB','2oyu'); return false;">PDB: 2oyu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kk6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kk6','PDB','3kk6'); return false;">PDB: 3kk6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3n8x" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3n8x','PDB','3n8x'); return false;">PDB: 3n8x</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3n8z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3n8z','PDB','3n8z'); return false;">PDB: 3n8z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pxx" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1pxx','PDB','1pxx'); return false;">PDB: 1pxx</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ln1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ln1','PDB','3ln1'); return false;">PDB: 3ln1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nt1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nt1','PDB','3nt1'); return false;">PDB: 3nt1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3rr3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3rr3','PDB','3rr3'); return false;">PDB: 3rr3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4cox" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4cox','PDB','4cox'); return false;">PDB: 4cox</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i7g" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i7g','PDB','1i7g'); return false;">PDB: 1i7g</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1k7l" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1k7l','PDB','1k7l'); return false;">PDB: 1k7l</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kkq" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kkq','PDB','1kkq'); return false;">PDB: 1kkq</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2npa" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2npa','PDB','2npa'); return false;">PDB: 2npa</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2p54" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2p54','PDB','2p54'); return false;">PDB: 2p54</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2rew" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2rew','PDB','2rew'); return false;">PDB: 2rew</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2znn" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2znn','PDB','2znn'); return false;">PDB: 2znn</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3et1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3et1','PDB','3et1'); return false;">PDB: 3et1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3fei" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3fei','PDB','3fei'); return false;">PDB: 3fei</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3g8i" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3g8i','PDB','3g8i'); return false;">PDB: 3g8i</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kdt" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kdt','PDB','3kdt'); return false;">PDB: 3kdt</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kdu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kdu','PDB','3kdu'); return false;">PDB: 3kdu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3sp6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3sp6','PDB','3sp6'); return false;">PDB: 3sp6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vi8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vi8','PDB','3vi8'); return false;">PDB: 3vi8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2iqh" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2iqh','PDB','2iqh'); return false;">PDB: 2iqh</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e5e" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e5e','PDB','4e5e'); return false;">PDB: 4e5e</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3md0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3md0','PDB','3md0'); return false;">PDB: 3md0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nxs" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nxs','PDB','3nxs'); return false;">PDB: 3nxs</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3tk1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3tk1','PDB','3tk1'); return false;">PDB: 3tk1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2www" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2www','PDB','2www'); return false;">PDB: 2www</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1xpc" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1xpc','PDB','1xpc'); return false;">PDB: 1xpc</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2zbd" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2zbd','PDB','2zbd'); return false;">PDB: 2zbd</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2agv" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2agv','PDB','2agv'); return false;">PDB: 2agv</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nkd" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nkd','PDB','3nkd'); return false;">PDB: 3nkd</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i7d" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1i7d','PDB','1i7d'); return false;">PDB: 1i7d</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5dlj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5dlj','PDB','5dlj'); return false;">PDB: 5dlj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2qw4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2qw4','PDB','2qw4'); return false;">PDB: 2qw4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2bxs" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2bxs','PDB','2bxs'); return false;">PDB: 2bxs</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2byb" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2byb','PDB','2byb'); return false;">PDB: 2byb</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r20" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r20','PDB','3r20'); return false;">PDB: 3r20</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4die" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4die','PDB','4die'); return false;">PDB: 4die</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b78" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b78','PDB','1b78'); return false;">PDB: 1b78</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jbv" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jbv','PDB','1jbv'); return false;">PDB: 1jbv</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1a49" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1a49','PDB','1a49'); return false;">PDB: 1a49</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2mjp" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2mjp','PDB','2mjp'); return false;">PDB: 2mjp</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1vpq" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1vpq','PDB','1vpq'); return false;">PDB: 1vpq</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2nqj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2nqj','PDB','2nqj'); return false;">PDB: 2nqj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e5f" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1e5f','PDB','1e5f'); return false;">PDB: 1e5f</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4kam" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4kam','PDB','4kam'); return false;">PDB: 4kam</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b0u" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1b0u','PDB','1b0u'); return false;">PDB: 1b0u</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kkl" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1kkl','PDB','1kkl'); return false;">PDB: 1kkl</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gka" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gka','PDB','3gka'); return false;">PDB: 3gka</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1gwj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1gwj','PDB','1gwj'); return false;">PDB: 1gwj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2r14" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2r14','PDB','2r14'); return false;">PDB: 2r14</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kru" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3kru','PDB','3kru'); return false;">PDB: 3kru</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3krz" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3krz','PDB','3krz'); return false;">PDB: 3krz</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00078">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_47636"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00078">10.1021/acs.jmedchem.6b00078</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Summary of different binding site comparison methods and their availability (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00078/suppl_file/jm6b00078_si_001.pdf">jm6b00078_si_001.pdf (235.45 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00078&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00078%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-9%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00078" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677acf3669952374","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
