
<html lang="en"     class="pb-page"  data-request-id="d01f1653-de9f-47f7-ae89-4f7cc7628960"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00598;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2019.62.issue-15;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Exploring the Natural Piericidins as Anti-Renal Cell Carcinoma Agents Targeting Peroxiredoxin 1" /></meta><meta name="dc.Creator" content="Xuefeng  Zhou" /></meta><meta name="dc.Creator" content="Zhi  Liang" /></meta><meta name="dc.Creator" content="Kunlong  Li" /></meta><meta name="dc.Creator" content="Wei  Fang" /></meta><meta name="dc.Creator" content="Yuanxin  Tian" /></meta><meta name="dc.Creator" content="Xiaowei  Luo" /></meta><meta name="dc.Creator" content="Yulian  Chen" /></meta><meta name="dc.Creator" content="Zhikun  Zhan" /></meta><meta name="dc.Creator" content="Tao  Zhang" /></meta><meta name="dc.Creator" content="Shengrong  Liao" /></meta><meta name="dc.Creator" content="Shuwen  Liu" /></meta><meta name="dc.Creator" content="Yonghong  Liu" /></meta><meta name="dc.Creator" content="William  Fenical" /></meta><meta name="dc.Creator" content="Lan  Tang" /></meta><meta name="dc.Description" content="Anti-renal cell carcinoma (RCC) agents with new mechanisms of action are urgently needed. Twenty-seven natural products of the piericidin class, including 17 new ones, are obtained from a marine-de..." /></meta><meta name="Description" content="Anti-renal cell carcinoma (RCC) agents with new mechanisms of action are urgently needed. Twenty-seven natural products of the piericidin class, including 17 new ones, are obtained from a marine-de..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 12, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00598" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00598" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00598" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00598" /></link>
        
    
    

<title>Exploring the Natural Piericidins as Anti-Renal Cell Carcinoma Agents Targeting Peroxiredoxin 1 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00598" /></meta><meta property="og:title" content="Exploring the Natural Piericidins as Anti-Renal Cell Carcinoma Agents Targeting Peroxiredoxin 1" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0006.jpeg" /></meta><meta property="og:description" content="Anti-renal cell carcinoma (RCC) agents with new mechanisms of action are urgently needed. Twenty-seven natural products of the piericidin class, including 17 new ones, are obtained from a marine-derived Streptomyces strain, and several of them show strong inhibitory activities against ACHN renal carcinoma cells. By exploring the mechanisms of two representative natural piericidin compounds, piericidin A (PA) and glucopiericidin A (GPA), peroxiredoxin 1 (PRDX1) is detected as a potential target by transcriptome data of PA-treated ACHN cells, as well as the paired RCC tumor versus adjacent nontumor tissues. PA and GPA induce cell apoptosis through reducing the reactive oxygen species level caused by upregulated PRDX1 mRNA and protein level subsequently and exhibit potent antitumor efficacy in nude mice bearing ACHN xenografts, with increasing PRDX1 expression in tumor. The interaction between PA/GPA and PRDX1 was supported by the docking analysis and surface plasmon resonance. Moreover, the translocation of PRDX1 into the nucleus forced by PA/GPA is proposed to be a key factor for the anti-RCC procedure. Piericidins provide a novel scaffold for further development of potent anti-RCC agents, and the new action mechanism of these agents targeting PRDX1 may improve upon the limitations of existing targeted drugs for the treatment of renal cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00598"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00598">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00598&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00598&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00598&amp;href=/doi/10.1021/acs.jmedchem.9b00598" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 7058-7069</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00516" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00624" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Exploring the Natural Piericidins as Anti-Renal Cell Carcinoma Agents Targeting Peroxiredoxin 1</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xuefeng Zhou</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xuefeng Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CAS Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong Key Laboratory of Marine Materia Medica, South China Sea Institute of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#2a524c5042455f6a5949594345044b49044944"><span class="__cf_email__" data-cfemail="651d031f0d0a10251606160c0a4b04064b060b">[email protected]</span></a> (X.Z.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xuefeng++Zhou">Xuefeng Zhou</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9601-4869" title="Orcid link">http://orcid.org/0000-0001-9601-4869</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhi Liang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhi Liang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biopharmaceutics, Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou 510515, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhi++Liang">Zhi Liang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kunlong Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kunlong Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CAS Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong Key Laboratory of Marine Materia Medica, South China Sea Institute of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kunlong++Li">Kunlong Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Fang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Fang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Biopesticide Engineering Research Center, Hubei Academy of Agricultural Sciences, Wuhan 430064, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Fang">Wei Fang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuanxin Tian</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuanxin Tian</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biopharmaceutics, Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou 510515, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuanxin++Tian">Yuanxin Tian</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3847-559X" title="Orcid link">http://orcid.org/0000-0003-3847-559X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaowei Luo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaowei Luo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CAS Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong Key Laboratory of Marine Materia Medica, South China Sea Institute of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaowei++Luo">Xiaowei Luo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2114-1609" title="Orcid link">http://orcid.org/0000-0002-2114-1609</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yulian Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yulian Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biopharmaceutics, Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou 510515, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yulian++Chen">Yulian Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhikun Zhan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhikun Zhan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biopharmaceutics, Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou 510515, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhikun++Zhan">Zhikun Zhan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tao Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tao Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biopharmaceutics, Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou 510515, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tao++Zhang">Tao Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shengrong Liao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shengrong Liao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CAS Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong Key Laboratory of Marine Materia Medica, South China Sea Institute of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shengrong++Liao">Shengrong Liao</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4393-3823" title="Orcid link">http://orcid.org/0000-0003-4393-3823</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shuwen Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuwen Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biopharmaceutics, Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences,  State Key Laboratory of Organ Failure Research, Southern Medical University, Guangzhou 510515, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuwen++Liu">Shuwen Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6346-5006" title="Orcid link">http://orcid.org/0000-0001-6346-5006</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yonghong Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yonghong Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CAS Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong Key Laboratory of Marine Materia Medica, South China Sea Institute of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yonghong++Liu">Yonghong Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8327-3108" title="Orcid link">http://orcid.org/0000-0001-8327-3108</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William Fenical</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William Fenical</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Marine Biotechnology and Biomedicine, Scripps Institution of Oceanography, University of California at San Diego, La Jolla, California 92093, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William++Fenical">William Fenical</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8955-1735" title="Orcid link">http://orcid.org/0000-0002-8955-1735</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Lan Tang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lan Tang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biopharmaceutics, Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences,  State Key Laboratory of Organ Failure Research, Southern Medical University, Guangzhou 510515, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#abdfc79f9b9eebd8c6de85cecfde85c8c5"><span class="__cf_email__" data-cfemail="493d257d797c093a243c672c2d3c672a27">[email protected]</span></a> (L.T.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lan++Tang">Lan Tang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2345-0886" title="Orcid link">http://orcid.org/0000-0002-2345-0886</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00598&amp;href=/doi/10.1021%2Facs.jmedchem.9b00598" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 7058–7069</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 12, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 April 2019</li><li><span class="item_label"><b>Published</b> online</span>12 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 August 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00598" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00598</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7058%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXuefeng%2BZhou%252C%2BZhi%2BLiang%252C%2BKunlong%2BLi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D15%26contentID%3Dacs.jmedchem.9b00598%26title%3DExploring%2Bthe%2BNatural%2BPiericidins%2Bas%2BAnti-Renal%2BCell%2BCarcinoma%2BAgents%2BTargeting%2BPeroxiredoxin%2B1%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7069%26publicationDate%3DAugust%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00598"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1243</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">18</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00598" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Exploring the Natural Piericidins as Anti-Renal Cell Carcinoma Agents Targeting Peroxiredoxin 1&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xuefeng&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Zhi&quot;,&quot;last_name&quot;:&quot;Liang&quot;},{&quot;first_name&quot;:&quot;Kunlong&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Fang&quot;},{&quot;first_name&quot;:&quot;Yuanxin&quot;,&quot;last_name&quot;:&quot;Tian&quot;},{&quot;first_name&quot;:&quot;Xiaowei&quot;,&quot;last_name&quot;:&quot;Luo&quot;},{&quot;first_name&quot;:&quot;Yulian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Zhikun&quot;,&quot;last_name&quot;:&quot;Zhan&quot;},{&quot;first_name&quot;:&quot;Tao&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Shengrong&quot;,&quot;last_name&quot;:&quot;Liao&quot;},{&quot;first_name&quot;:&quot;Shuwen&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Yonghong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;Fenical&quot;},{&quot;first_name&quot;:&quot;Lan&quot;,&quot;last_name&quot;:&quot;Tang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7058-7069&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00598&quot;},&quot;abstract&quot;:&quot;Anti-renal cell carcinoma (RCC) agents with new mechanisms of action are urgently needed. Twenty-seven natural products of the piericidin class, including 17 new ones, are obtained from a marine-derived Streptomyces strain, and several of them show strong inhibitory activities against ACHN renal carcinoma cells. By exploring the mechanisms of two representative natural piericidin compounds, piericidin A (PA) and glucopiericidin A (GPA), peroxiredoxin 1 (PRDX1) is detected as a potential target by transcriptome data of PA-treated ACHN cells, as well as the paired RCC tumor versus adjacent nontumor tissues. PA and GPA induce cell apoptosis through reducing the reactive oxygen species level caused by upregulated PRDX1 mRNA and protein level subsequently and exhibit potent antitumor efficacy in nude mice bearing ACHN xenografts, with increasing PRDX1 expression in tumor. The interaction between PA/GPA and PRDX1 was supported by the docking analysis and surface plasmon resonance. Moreover, the translocation of PRD&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00598&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00598" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00598&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00598" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00598&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00598" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00598&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00598&amp;href=/doi/10.1021/acs.jmedchem.9b00598" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00598" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00598" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00598%26sid%3Dliteratum%253Aachs%26pmid%3D31298537%26genre%3Darticle%26aulast%3DZhou%26date%3D2019%26atitle%3DExploring%2Bthe%2BNatural%2BPiericidins%2Bas%2BAnti-Renal%2BCell%2BCarcinoma%2BAgents%2BTargeting%2BPeroxiredoxin%2B1%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D15%26spage%3D7058%26epage%3D7069%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291137" title="Lipids">Lipids</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291376" title="Free radicals">Free radicals</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292153" title="Tumors">Tumors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/jmcmar.2019.62.issue-15/20190808/jmcmar.2019.62.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Anti-renal cell carcinoma (RCC) agents with new mechanisms of action are urgently needed. Twenty-seven natural products of the piericidin class, including 17 new ones, are obtained from a marine-derived <i>Streptomyces</i> strain, and several of them show strong inhibitory activities against ACHN renal carcinoma cells. By exploring the mechanisms of two representative natural piericidin compounds, piericidin A (PA) and glucopiericidin A (GPA), peroxiredoxin 1 (PRDX1) is detected as a potential target by transcriptome data of PA-treated ACHN cells, as well as the paired RCC tumor versus adjacent nontumor tissues. PA and GPA induce cell apoptosis through reducing the reactive oxygen species level caused by upregulated PRDX1 mRNA and protein level subsequently and exhibit potent antitumor efficacy in nude mice bearing ACHN xenografts, with increasing PRDX1 expression in tumor. The interaction between PA/GPA and PRDX1 was supported by the docking analysis and surface plasmon resonance. Moreover, the translocation of PRDX1 into the nucleus forced by PA/GPA is proposed to be a key factor for the anti-RCC procedure. Piericidins provide a novel scaffold for further development of potent anti-RCC agents, and the new action mechanism of these agents targeting PRDX1 may improve upon the limitations of existing targeted drugs for the treatment of renal cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38561" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38561" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Renal cell carcinoma (RCC) is the most common form of renal cancer, whose incidence rates have been gradually increasing by 2–4% every year over the past few decades.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Chemotherapy and radiotherapy are not as successful in the case of RCC, although significant advances have been derived from several target drugs approved by the FDA, such as sorafenib, sunitinib, and pazopanib as tyrosine kinase inhibitors.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> In order to curb drug resistance, unwanted side effects, and further improve efficacy, the discovery of more effective anti-RCC agents, especially those with novel mechanisms of action, is warranted.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The revolutionary technology employed in medicine to interrogate human cancer is next-generation sequencing, which enabled a hitherto unknown insight into the molecular and cellular machineries in RCC.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Further studies to dissect out the role of the mutated genes or key biomarkers in RCC tumorigenesis revealed by different genomic platforms will hopefully provide the foundation for the development of effective forms of therapy for this disease.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Microorganisms remain unrivalled in their potential to produce highly potent natural products for drug development, and the actinomycetes are well-known as an inexhaustible source for new chemical structures. The piericidin class metabolites, that feature a 4-pyridinol core linked with a variable polyene side chain, are commonly produced by actinomycetes isolated from soil, insects, and marine samples.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> To date, there are about 40 natural piericidin derivatives reported, including 11 piericidin glycosides.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> Piericidin A (PA) is known as a potent inhibitor of NADH–ubiquinone oxidoreductase (complex I). In addition to broad antimicrobial and insecticidal activities, some piericidins are reported to have potential as antitumor agents.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The chemical diversity of the natural piericidins has not been reached, thus the pharmaceutical significance of this class will require further refinement and examination, especially regarding their anticancer mechanisms of action.</div><div class="NLM_p last">Described herein is the isolation and structure determination of 27 natural piericidins from a <i>Streptomyces</i> strain, with inhibitory activities against RCC ACHN cells. Driven by transcriptomic analysis, a new action mechanism for the piericidins, targeting the antioxidative protein peroxiredoxin 1 (PRDX1), has been observed and confirmed. Anti-RCC evaluation in vitro and in vivo confirmed that the piericidins behave as potent anti-RCC agents.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57353" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57353" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Natural Piericidins and Their Activities against Renal Cancer Cells</h3><div class="NLM_p">In order to explore natural piericidins compounds, chemical dereplication by high-performance liquid chromatography/mass spectrometry (HPLC/MS) was performed to screen the strains with piericidins in marine <i>Streptomyces</i> strains in our lab. The HPLC/MS analysis uncovered abundant and divers piericidin glycosides in the strain <i>Streptomyces psammoticus</i> SCSIO NS126 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S1), an actinomycete strain isolated from a mangrove sediment sample collected from the Pearl River estuary to South China Sea. Chemical study of the culture (60 L for liquid fermentation) extract of strain SCSIO NS126 provided 27 natural piericidins (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of the piericidins obtained from the <i>Streptomyces</i> strain SCSIO NS126.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Among the natural piericidins obtained, seventeen (<b>2–5</b>, <b>12</b>, <b>13</b>, <b>15</b>, <b>17–26</b>) were identified as new natural products, by extensive NMR and HRMS analyses, including their configurations as defined by NOESY data, coupling constant analysis, and electronic circular dichroism (ECD) comparisons, along with their biosynthesis taken into consideration.</div><div class="NLM_p">Comparison of the NMR date of <b>2</b> with those of PA (<b>1</b>) indicated that the only difference was the absence of the 14-methyl group in <b>2</b>, which was confirmed by the HMBC and COSY correlations. Compound <b>2</b> was identified as 11-demethyl-piericidin A (<b>2</b>). Two hydroxyl groups on C-13 and C-19 were confirmed in the structures of <b>3</b> and <b>4</b>, and their absolute configurations were determined by calculated and experimental ECD curves (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Figure S9</a>). Compound <b>5</b> was identified as the same structure and same absolute configuration (9<i>R</i>) as synthetic C-10 ketone PA,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and obtained as a natural product first here. Compound <b>12</b> was suggested to be a glycoside of 7-demethylpiericidin A1 (<b>9</b>), and 10-<i>O</i>-β-<span class="smallcaps smallerCapital">d</span>-glucoside linkage was determined by the NMR analysis, as well as the acid hydrolysis. Compound <b>13</b> was characterized as 7-demethyl-13-hydroxyglucopiericidin A (<b>13</b>). Compounds <b>14–19</b> were shown to be diglycosides of piericidin. Compounds <b>14</b> and <b>16</b> were characterized as PA 10-<i>O</i>-α-<span class="smallcaps smallerCapital">d</span>-galactose (1→6)-β-<span class="smallcaps smallerCapital">d</span>-glucoside (<b>14</b>) and PA 10-<i>O</i>-α-<span class="smallcaps smallerCapital">d</span>-glucose (1→6)-β-<span class="smallcaps smallerCapital">d</span>-glucoside (<b>16</b>), respectively, by acid hydrolysis and NMR analysis. The additional β-<span class="smallcaps smallerCapital">d</span>-glucose group of <b>18</b> was linked on OH-4′ of the pyridine ring, so the structure was characterized as 4′-<i>O</i>-β-<span class="smallcaps smallerCapital">d</span>-glucose glucopiericidin A (<b>18</b>). Compounds <b>15</b>, <b>17</b>, and <b>19</b> were shown to be 13-hydroxy derivatives of <b>14</b>, <b>16</b>, and <b>18</b>, respectively. Compound <b>20</b> was indicated as a triglycoside of piericidin by HR-ESIMS and NMR date, and characterized as 4′-<i>O</i>-β-<span class="smallcaps smallerCapital">d</span>-glucose PA 10-<i>O</i>-α-<span class="smallcaps smallerCapital">d</span>-glucose (1→6)-β-<span class="smallcaps smallerCapital">d</span>-glucoside (<b>20</b>). In the structure of <b>21</b>, the C-5/C-6 double bond was replaced by the 5-hydroxy-6-hydroxymethyl group, with their configurations determined by coupling constant and NOESY correlation. Compound <b>22</b>, the 13-hydroxy derivatives of <b>21</b>, was characterized as 5-hydroxy-6-hydroxymethyl-13-hydroxyglucopiericidin A (<b>22</b>). The conventional C-2/C-3 double bond was moved to C-3/C-4 in glycoside <b>23</b>, and an additional hydroxymethyl was linked on C-2. Compound <b>23</b> was characterized as 2-hydroxymethyl-Δ3,4-glucopiericidin A (<b>23</b>), with the absolute configuration of C-2 remained to be defined. Glycosides <b>24</b> and <b>25</b> have the same planar structure, including the C-11/C-12 epoxy ring. The configurations of the C-11/C-12 in <b>24</b> and <b>25</b> were determined to be 11<i>S</i>,12<i>R</i> and 11<i>R</i>,12<i>S</i>, respectively, by NOESY and ECD analysis. The chemical shifts of CH-13 and HR-ESIMS data suggested two methoxy groups linked on C-13 in <b>26</b>, which were supported by HMBC correlations. Therefore, <b>26</b> was characterized as 13-dimethoxy glucopiericidin A (<b>26</b>). The other reported compounds were identified by comparison of their spectroscopic data with those in the literature. The details of the structure elucidation are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p">From the genome sequence of this actinomycete strain, we identified the biosynthetic gene cluster of PA (<b>1</b>), which displays the same genetic organization as that in <i>Streptomyces piomogeues</i> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Table S7</a>),<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> together with several glycosyltransferases. Diverse oxygenated piericidin derivatives, such as the seven 13-hydroxy piericidin glycosides and several derivatives with 5,6-dihydroxy, 13,19-dihydroxy or those with the C-11/C-12 epoxide ring, suggest that there are additional oxygenases taking part in the biosynthesis of these piericidin derivatives. The 13,19-dihydroxy functionality (in <b>3</b> and <b>4</b>), 5,6-dihydroxy functionality (in <b>21</b> and <b>22</b>), and also the 2-hydroxymethyl group (in <b>23</b>) are proposed to be formed by epoxidation and ring opening, the latter of which is followed by dehydration. Eighteen glycosides (<b>10–27</b>), composed of 11 monoglycosides, six diglycosides, and one triglycoside, expand the diversity of natural piericidin glycosides. Diglycosides of piericidin have been rarely reported, and the triglycoside (<b>20</b>) here is the first piericidin triglycoside reported from nature.</div><div class="NLM_p">Most of the piericidins were evaluated for their cytotoxicities against three renal carcinoma cell lines, ACHN, OS-RC-2, and 786-O, as well as a normal renal cell line, HK-2 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Most of the piericidins showed strong to moderate cytotoxicities toward ACHN, and several analogs, such as PA (<b>1</b>), glucopiericidin A (GPA, <b>10</b>), and <b>12</b>, showed significant activities with IC<sub>50</sub> values less than 1 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). OS-RC-2, 786-O, and HK-2 cells were not as sensitive as ACHN to most of the piericidins.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Cytotoxicity of the Piericidins (IC<sub>50</sub>, μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">ACHN</th><th class="colsep0 rowsep0" align="center">OS-RC-2</th><th class="colsep0 rowsep0" align="center">786-O</th><th class="colsep0 rowsep0" align="center">HK-2</th><th class="colsep0 rowsep0" align="center">shPRDX1 ACHN</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">0.40</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">5.3</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">44</td><td class="colsep0 rowsep0" align="left">8.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">63</td><td class="colsep0 rowsep0" align="left">13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">45</td><td class="colsep0 rowsep0" align="left">68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">0.21</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">0.96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">7.21</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">0.31</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">0.99</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">4.5</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">9.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Sora</td><td class="colsep0 rowsep0" align="left">3.3</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">–</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Sora: sorafenib; – unrecorded.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.2.  PA/GPA Up-Regulates PRDX1 That Is Lower in Clear Cell RCC</h3><div class="NLM_p">To investigate the underlying molecular mechanisms of PA (<b>1</b>) on ACHN cells, we analyzed transcriptome alterations using high-depth next-generation sequencing with an Illumina HiSeq 4000. In total, more than 291 million stranded 150 basepairs paired-end reads were sequenced from 3 groups of ACHN cells treated with two doses of PA (25 and 50 nM) for 24 h along with untreated cells. All mRNA sequencing data has been submitted to the GEO repository (GSE116158) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Table S1</a>). Next, DEseq was used to identify the statistically significant differentially expressed genes (DEGs) for the pairwise comparisons between these three groups, respectively. Among 17 847 detected unigenes, 6035 genes (2862 up regulated and 3173 down regulated) had significantly altered expression in cells treated with low-dose PA, while more DEGs (8889 genes total, 4305 up regulated and 4584 down regulated) were identified in high-dose PA treatment (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Table S2</a>). Given the significant effect of high-dose PA treatment, an intersection of DEGs in two experimental groups was performed to locate the key genes affected. Then, KEGG pathway enrichment analysis, according to 1066 DEGs from the intersection of three groups (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Table S3</a>), was employed to identify the biological characterization of the significantly regulated genes. The results showed that the regulated genes were involved in endoplasmic reticulum, pathways in cancer, HIF-1 signalling pathway, and renal carcinoma pathways (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Table S4</a>). Then supervised weighted correlation network analysis (sWGCNA) was employed to define hub genes and the relationship among the selected DEGs from those pathways (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Figure S4</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Transcriptome analysis of the gene expression profile in ACHN cells treated with PA. (A) Volcano plot: showed numbers of DEGs (FDR < 0.05 according to DEseq). (B) Intersection of DEGs (the statistical criteria using fold change > 1.5 and FDR < 0.05) affected by PA and those DEGs from pairs of ccRCC tumors and adjacent non-tumor renal tissues from GSE40435 and GSE76207. (C) KEGG pathways enrichment analysis. The bars represent the enrichment scores, −log 10 (<i>P</i> value). (D) PA (25 and 50 nM) and GPA (25 and 50 nM) reduced the activation of key genes in ACHN cells. (E) mRNA level of PRDX1 in ACHN cells after treatment of PA/GPA. (F) Immunoblot analysis of PRDX1 protein of extracts of paired tumour (T) vs adjacent non-tumor (N) samples from 6 human ccRCC patients.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Consequently, multiple interaction genes (<i>EGFR</i>, <i>SOD2</i>, <i>ETS1</i> and <i>HK2</i>, <i>SLC2A1</i>, <i>LDHA</i>, <i>MET</i>, <i>HIF1A</i>, etc.), among the above mentioned signaling pathways were chosen to be certificated treated with PA and GPA (25 and 50 nM) in ACHN cells by reverse transcription-polymerase chain reaction (RT-PCR) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). GPA (<b>10</b>), with similar inhibitory activity toward ACHN as PA, is the glycoside of PA (<b>1</b>). As expected, PA and GPA could both reduce the activation of those key genes. Thus, there are reasons to believe that PA and GPA exert their tumor inhibition activity through the same pathway. To further clarify the potential molecular targets, we examined the intersection of DEGs (the statistical criteria using fold change ≥ 1.5 and FDR < 0.05) affected by PA and those DEGs between pairs of clear cell RCC (ccRCC) tumor and the adjacent non-tumor renal tissues in the GEO database (GSE40435 and GSE76207) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). There were 129 common genes upregulated in the PA treatment group but down-regulated in the ccRCC tumor tissues compared to the adjacent nontumor renal tissues, including PRDX1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). RT-PCR was employed to certify the upregulation of PRDX1 in ACHN cells after treatment of PA/GPA (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E).</div><div class="NLM_p last">As a redox-regulating protein belongs to the ubiquitous PRDXs family, PRDX1 was implicated in regulating cell proliferation, differentiation, and apoptosis.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> All peroxiredoxins have been shown to have altered expression in human cancer, being upregulated in breast cancer, oesophageal cancer, and lung cancer, but down-regulated in thyroid cancer as a tumor suppressor.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> The PRDX1 protein was decreased in ccRCC as compared to adjacent normal renal tissue.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Remarkably, we make the same findings of the lower PRDX1 protein levels in six ccRCC tumor samples (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Table S10</a>). Along with the analysis of different GEO datasets, it is reasonable to believe that the mRNA and protein level of PRDX1 are lower in renal tumor tissues, and PRDX1 could be considered as a potential target in RCC. However, the mechanism of PRDX1 as an anti-RCC target still needs to be further explored.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">2.3.  PA/GPA Induce Cell Apoptosis through Increasing PRDX1 and Reducing Reactive Oxygen Species</h3><div class="NLM_p">PRDX1 was first reported as an antioxidant enzyme because of its functions to eliminate peroxide in vivo and regulate reactive oxygen species (ROS).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Aggressive cancer cells depend on elevated intracellular levels of ROS to proliferate, self-renew, metastasize, and so forth. These aggressive cancers maintain high basal levels of ROS compared to normal cells.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> With a high ROS level and low PRDX1 expression, RCC becomes less sensitive to radiation and chemotherapy. Thus, ROS elimination played a vital role in RCC treatment. To explore the molecular mechanisms of piericidins (PA/GPA) on ACHN cells, clarifying the relationship between PRDX1 and ROS levels in RCC is crucial.</div><div class="NLM_p">To further illustrate the mechanism by which PRDX1 exerts its tumor-suppressing effect, we created knockdown (Kd) PRDX1 ACHN cells (PRDX1-Kd) using lentivirus (Lv) PRDX1 shRNA (shPRDX1) to speculate on the role of ROS (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Figure S5</a>). As a result, the Kd of PRDX1 raised ROS levels compared to normal ACHN cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C1). ROS reduction in normal ACHN cells was observed when treated with PA/GPA (25 and 50 nM) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C2). Notably, Kd of PRDX1 eliminated these effects, which indicated that PRDX1 was the target of PA/GPA (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C3). By decreasing ROS levels, PA showed significant proliferation suppression as well as inducing apoptosis of ACHN cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A1,A2). In addition, the cytotoxic IC<sub>50</sub> values of PA/GPA against shPRDX1 ACHN cells were almost 4-fold higher than normal ACHN cells with slight toxicity toward the HK2 cell (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These results indicated that PA/GPA targets PRDX1 mainly increasing mRNA and protein levels rather than improving the antioxidant activity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Figure S6</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. PA/GPA inhibits cell proliferation and induced cell apoptosis through increasing PRDX1 and reducing ROS in ACHN. (A1) Phase holographic imaging of ACHN cells treated with PA (50 nM) for 72 h. (A2) Annexin V/PI staining and flow cytometry showed the percentages of apoptosis in ACHN cells treated with PA (50 nM). (B) Immunoblot analysis of the PRDX1 protein in the normal ACHN cells and PRDX1-Kd ACHN cells; (C1) Intracellular ROS detection with flow cytometer in the normal ACHN cells and PRDX1-Kd ACHN cells. (C2) Intracellular ROS detection in the normal ACHN cell treated by PA/GPA (25 and 50 nM) in 24 h; (C3) Intracellular ROS detection in the PRDX1-Kd ACHN cell treated by PA/GPA (25 and 50 nM) in 24 h. (D1) Immunoblot analysis of the PRDX1 protein in nuclear and cytosol extracts of ACHN cell treated with PA/GPA (25 and 50 nM). (D2) Fold change of protein expression (PRDX1/β-Actin). (E1) Representative images for immunofluorescent staining of ACHN treated with PA/GPA (25 and 50 nM) after co-incubating for 24 h. PRDX1 (green), nucleus (blue), <i>n</i> = 9. Scale bar = 10 μm. (E2) Mean fluorescence intensity of ACHN cell treated with PA/GPA (25 and 50 nM) in 24 h. All data represent mean ± standard deviation. Statistical significance was determined using Student’s <i>t</i>-test or one-way analysis of variance with Tukey’s post-hoc test. *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.005, ****<i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Not only serving as antioxidant enzyme, PRDX1 can also act as a molecular chaperone with the ability to modulate actions of numerous molecules or as a regulator of transcription.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> PRDX1 could directly bind with nuclear transcriptional factors and modulate gene expression such as c-Myc and NF-κB.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Therefore, we further investigated the PRDX1 protein level in nuclear and cytosol of ACHN cell. Along with increasing total PRDX1, obvious translocation of PRDX1 toward the nucleus was also observed, indicated that PA/GPA not only raised the expression of mRNA and protein of PRDX1 but also forced it into the nucleus (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D,E). Some studies have shown that PRDX1 in the nucleus leads to inhibition of NF-κB activation, a typical oncogene involved in most cancers.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> These data further suggested that the protein level of PRDX1, but not antioxidant activity, is critical in regulating the ROS level and the inactivation of NF-κB.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.4.  Docking Analysis of PA/GPA with PRDX1</h3><div class="NLM_p">The interconversion between the dimeric and decameric forms of PRDX1 is influenced by redox state or protein concentration, and the dimeric form is preferred for translocation into the nucleus.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In order to further understand the interaction between PA/GPA and the potential target, especially regulatory mechanisms of dimer–oligomer interconversion of PRDX1, the induce-fit module in the Schrödinger suite was employed to perform the in silico molecular docking analysis. The chains a, b, c, and d, as a dimer–dimer interface unit, from the reduced decameric form of PRDX1 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S">2Z9S</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Figure S7</a>) was used as the receptor structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A1). Because the redox status CYS83 might influence the oligomeric structure and consequently the functions of PRDX1 in regulating the peroxidase and chaperone activities,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> it was selected as trim residue and the center of the active pocket, which composed of the PHE48, TRP87 from the dimer–dimer interface.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> As the results, the pyridine ring of PA/GPA played a key role to form a π–π stacking interaction with PHE48. Moreover, the π cation interaction was formed between pyridine ring and LYS192, and hydrophobic interaction was formed between side chain and dimer–dimer interface (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A2,A3). The hydroxyls at side chain and pyridine ring formed hydrogen bonds with the residues GLN94, GLY95 in chain a and LYS192 in chain c, respectively. It is suggested that those interactions inhibit the formation of the oligomeric state of PRDX1 and forced the active dimer into the nucleus, which is consistent with the experimental findings about the increase of PRDX1 in nucleus (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D,E).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Molecular docking and SPR analysis of PA/GPA with PRDX1. (A1) Dimer–dimer interface region (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S">2Z9S</a>) was used as the receptor structure. Each chain of monomers is shown in different colors. Chains a, b as a dimer and chains c, d as another dimer. The CYS83–CYS83 disulfide bridge is located between chains a and c. (A2) Binding sites and the details of the predicted binding mode of PA in the dimer–dimer interface of PRDX1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S">2Z9S</a>), 90° rotation from (A1). (A3) Binding sites and the details of the predicted binding mode of GPA in the dimer–dimer interface of PRDX1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S">2Z9S</a>). The contact residues are shown and labeled by type and number. (B) Affinity activities of PA (B1) and GPA (B2) to the PRDX1 protein were analyzed using the SPR assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The docking study showed that the interactions between PA/GPA and single dimer (docking scores about −6) of PRDX1 were much weaker than with the dimer–dimer interface (docking scores about −10). The dissociation equilibrium constants (<i>K</i><sub>D</sub>) of the PRDX1 protein with PA (2.61 × 10<sup>–6</sup> M) and GPA (1.83 × 10<sup>–6</sup> M), measured by a surface plasmon resonance (SPR) assay (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B), revealed the affinity interaction between PA/GPA and PRDX1 protein (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B1,B2). Therefore, it is suggested that, not only combined with the PRDX1, PA/GPA could force PRDX1 translocate into nucleus is also proposed to be a key factor for the anti-RCC procedure of PA/GPA.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">2.5.  Antitumor Efficacy of PA and GPA in Nude Mice Bearing ACHN Xenografts</h3><div class="NLM_p">Because PA and GPA showed significant inhibitory effects against ACHN in vitro, the in vivo anticancer therapeutic potential were further investigated. Nude mice bearing ACHN tumor xenografts were divided into four groups (control group with only phosphate-buffered saline (PBS) injection, PA and GPA groups with both 0.8 mg/kg, sorafenib group with 50 mg/kg),<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and PA and GPA were administered through intraperitoneal (ip) injection daily for three weeks. The treatment outcome was assessed by monitoring relative tumor volumes and weights of the mice, and tumor tissue ablation was also evaluated by haematoxylin and eosin (HE) staining and terminal deoxynucleotidyl transferase-mediated dUTP nick end labeling (TUNEL) staining on tissue sections.</div><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, administration of PA or GPA significantly reduced the final tumor weight of the mice (mean 0.28 g to PA, and mean 0.23 g to GPA) as compared to the control group (mean 0.55 g), basically matching the tumor volume curves (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B). All the mice in the treatment groups survived more than three weeks, the mice body weights of PA and GPA groups did not decrease as observed in the sorafenib group (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). PA and GPA showed significant antitumor efficacy in nude mice bearing ACHN xenografts with low effective dose and without serious side effects.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. PA and GPA show antitumor efficacy in nude mice bearing ACHN xenografts. (A) Images of mice bearing ACHN xenografts and injected ip once a day at day 21 post injection. (B) Images of tumor (B1) and progression of body weights (B2) and tumor volumes (B3) in mice during 21 days treatment; <i>n</i> = 4, ±SE. (C) Immunoblot analysis of the PRDX1 protein in tumor tissues after 21 days treatment with PBS, PA, and GPA. (D) TUNEL and HE staining of tumor sections after treatment, respectively. Scale bar, 100 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">After administration of PA and GPA for 21 days, the mRNA and protein expression of PRDX1 in tumor tissues increased (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C), consistent with the results in vitro. In addition, TUNEL staining indicated by green color staining and HE staining results in tumor tissues treated with PA or GPA further demonstrated the inhibition of tumor growth (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">3.  Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40753" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40753" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, 27 natural piericidins were obtained from a <i>Streptomyces</i> strain isolated from the mangrove sediments. Seventeen were identified as new natural products, including 13 new piericidin glycosides. This work has extended the chemical diversity of this family, although several piericidin glycosides have been detected and structurally predicted in situ in the symbiotic <i>Streptomyces</i> strains of beewolf host species reported recently.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The availability of these diverse piericidin natural products creates an opportunity to fully understand the importance of these compounds. Although the cytotoxicity of the piericidins are commonly reported, the potential to treat renal cancer or related cancer cells was not realized. As one of the high-incidence cancers with poor prognosis, our findings highlight the inhibition activity of piericidins against renal cancer in vitro and in vivo.</div><div class="NLM_p">Cancer cells rely on the signaling capabilities of ROS for cell migration, proliferation, and survival.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Studies have found that the persistent high level of ROS can act as mitogenic or genotoxic signals, inhibiting the sensitivity of radiotherapy and chemotherapy in cancer cells.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> As a member of the ubiquitous family of redox-regulating proteins, PRDX1 is reported to potentially eliminate various ROS. The dimeric form is considered to be the active and functional form of PRDX1, and is preferred for translocation into the nucleus.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> PRDX1 can directly combine with some transcription factors in the nucleus such as c-Myc and NF-κB, affecting their bioactivities upon gene regulation, which in turn induces or suppresses cell death. Using bioinformatics analysis of clinical samples, ccRCC is thought to be a malignant tumor with a lower PRDX1 level. Remarkably, piericidins (PA/GPA) target PRDX1, increasing the mRNA and protein levels of PRDX1, which reduces ROS levels in ACHN cells along with obvious cell apoptosis induction and proliferation inhibition. Besides, PA/GPA can interact directly with PRDX1 protein and increase the nucleus co-localization, thereby inhibiting the activation of key genes in the renal caner pathway. Moreover, our study revealed that ACHN cells were less sensitive to piericidins when PRDX1 was down-regulated. Therefore, our direct evidence shows that PA/GPA targets PRDX1 and reduces ROS in ACHN cells.</div><div class="NLM_p last">In those 27 natural piericidins obtained in this study, PA and GPA were investigated with their molecular mechanisms on renal carcinoma cells, as well as the in vivo anticancer evaluation. PA and GPA were being chosen as representative molecules of piericidin aglycones and glycosides, respectively because of their relative high yield in the <i>Streptomyces</i> strain and their significant inhibitory ability against ACHN cells in vitro. There were no obvious differences between the piericidin glycosides and aglycones in the cytotoxicity against ACHN cells. Moreover, the nearly indistinguishable antitumor effects in vivo between PA and GPA in our study suggest that, the same metabolic substances might contribute to the efficacy after drug metabolism in vivo. However, the upregulation of PRDX1 protein expression by GPA in the nucleus of ACHN cells appears stronger than that of PA (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D), and the affinity interaction measured by the SPR test also showed glycoside GPA bound more effectively than the aglycone PA with PRDX1 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). In this study, we found that PRDX1 could be used as an important new target for anti-RCC agents, and piericidins such as PA and GPA could be considered as lead compounds targeting PRDX1.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">4.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, 27 natural piericidins were discovered by cultivation of a marine-derived <i>Streptomyces</i> strain, and their potential in anti-RCC was demonstrated. The low effective dose and relative safety of PA and GPA in vivo suggests these may be developed as new anti-RCC agents. The site of action of these agents targeting PRDX1, may improve upon the limitations of mechanisms of existing targeted drugs for the treatment of RCC. Although more investigations are needed, our current studies reveal that piericidins could conceivably provide a novel scaffold for further development of potent and mechanistically novel anti-RCC agents.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">5.  Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33406" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33406" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">5.1.  Chemistry</h3><div class="NLM_p last">IR spectra were measured on an IRAffinity-1 spectrometer (Shimadzu). Optical rotations were performed on a PerkinElmer 341 polarimeter. ECD spectra were measured with a Chirascan circular dichroism spectrometer (Applied Photophysics). The NMR spectra including 1D and 2D NMR were recorded on Bruker AC 500/700 MHz spectrometers. HR-ESIMS were determined with a Bruker maXis Q-TOF in positive/negative ion mode. Column chromatography was performed on silica gel (200–300, 300–400 mesh) and Sephadex LH-20 (Amersham Biosciences), respectively. All solvents used were of analytical grade (Tianjin Fuyu Chemical and Industry Factory). The semipreparative HPLC was performed on an HPLC (Hitachi-L2130, diode array detector, Hitachi L-2455, Tokyo, Japan) using a Phenomenex ODS column (250 mm × 10.0 mm i.d., 5 μm; Phenomenex, USA).</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">5.2.  Strain Material</h3><div class="NLM_p last">The strain SCSIO NS126 was isolated from a mangrove sediment sample collected from the Pearl River estuary (E 113°33′11.15″, N22°53′40.16″) to South China Sea in May 2015, by incubation at 28 °C for two weeks on ISP-2 medium (yeast extract 4 g, malt extract 4 g, glucose 4 g, crude sea salt 30 g, agar powder 20 g, distilled water 1000 mL, pH, 7.2–7.4). The strain was identified as a <i>S. psammoticus</i> species, by the 16S rRNA gene sequence analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Table S8</a>). It was deposited at the China Center for Type Culture Collection (Wuhan) as CCTCC M2017099 (<i>S. psammoticus</i> SCSIO NS126).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">5.3.  Fermentation and Extraction</h3><div class="NLM_p last">The strain SCSIO NS126 was fermented for 60 L. A few loop of cells of the strains were inoculated into a 250 mL Erlenmeyer flask containing 50 mL of seed medium (mannitol 1 g, soya peptone 0.5 g, soya-bean oil 0.125 g, K<sub>2</sub>HPO<sub>3</sub> 0.02 g, pH 7.0, 50 mL distilled water), and then cultivated on a rotary shaker at 120 rpm, 28 °C for 48 h as seed culture. Then, 2 mL of the seed culture was inoculated into a 500 mL Erlenmeyer flask containing 100 mL media (cotton seed meal 2.5 g, soluble starch 1 g, glucose 1 g, yeast extract 0.3 g, CaCO<sub>3</sub> 0.5 g, sea salt, 0.2 g; in 100 mL distilled water, pH 7.2). After cultivation on a rotary shaker at 180 rpm and 28 °C for 120 h, each bacterial culture broth was broken with ultrasonic treatment apparatus for 10 min. Then, culture broth was extracted with an equal volume of ethyl acetate three times. The organic extract was then concentrated under vacuum to afford the crude extract.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">5.4.  Compounds Isolation</h3><div class="NLM_p">The extract of strain SCSIO NS126 (38.2 g) was chromatographed on silica gel to give eight fractions (Frs.1–8). With the guide of HPLC analysis, Frs.2 and Frs.3 were combined and further purified by silica gel to obtain pure compound <b>1</b> (230 mg), <b>2</b> (13 mg), <b>5</b> (16 mg), <b>6</b> (12 mg), <b>8</b> (14 mg), and <b>9</b> (3.7 mg). Frs.4 was further purified by semipreparative HPLC to get compound <b>3</b> (0.7 mg) and <b>4</b> (0.8 mg) and <b>7</b> (2.5 mg). Frs.5 was further purified by semipreparative HPLC to get <b>10</b> (78 mg), <b>12</b> (2.9 mg), <b>23</b> (1.9 mg), <b>26</b> (1.1 mg), and <b>27</b> (2.1 mg). The compound <b>10</b> (250 mg) with the highest content was purified and obtained by semipreparative HPLC from Frs.6, together with <b>14</b> (6.5 mg), <b>16</b> (3.2 mg), <b>18</b> (2.5 mg), <b>21</b> (1.8 mg), <b>24</b> (1.6 mg), and <b>25</b> (0.6 mg). Frs.7 was also purified by semipreparative HPLC to get <b>11</b> (28 mg) <b>13</b> (1.8 mg), <b>15</b> (6.9 mg), <b>17</b> (4.1 mg), <b>19</b> (1.5 mg), <b>20</b> (1.5 mg), and <b>22</b> (1.9 mg). All obtained compounds were determined to have ≥95% purity by analytical HPLC.</div><div class="NLM_p">11-Demethyl-piericidin A (<b>2</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3335, 2945, 1449, 1020, 667 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-1 and S2-2</a>; HR-ESIMS <i>m</i>/<i>z</i>: 402.2649 [M + H]<sup>+</sup> (calcd for C<sub>24</sub>H<sub>36</sub>NO<sub>4</sub><sup>+</sup>, 402.2639).</div><div class="NLM_p">(2<i>E</i>,5<i>E</i>,7<i>E</i>,11<i>E</i>,9<i>R</i>,10<i>R</i>,13<i>S</i>,19<i>R</i>)-13,19-Dihydroxyl-IT-143-A (<b>3</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3335, 2945, 1449, 1020, 667 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-1 and S2-2</a>; HR-ESIMS <i>m</i>/<i>z</i>: 504.3332 [M + H]<sup>+</sup> (calcd for C<sub>29</sub>H<sub>46</sub>NO<sub>6</sub><sup>+</sup>, 504.3320).</div><div class="NLM_p">(2<i>E</i>,5<i>E</i>,7<i>E</i>,11<i>E</i>,9<i>R</i>,10<i>R</i>,13<i>R</i>,19<i>S</i>)-13,19-Dihydroxyl-IT-143-A (<b>4</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3329, 2945, 1446, 1018, 667 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-1 and S2-2</a>; HR-ESIMS <i>m</i>/<i>z</i>: 504.3320 [M + H]<sup>+</sup> (calcd for C<sub>29</sub>H<sub>46</sub>NO<sub>6</sub><sup>+</sup>, 504.3320).</div><div class="NLM_p">10-Ketone piericidin A (<b>5</b>). Clear oil; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −5.5, <i>c</i> 0.2, CHCl<sub>3</sub>; IR (film) ν<sub>max</sub>: 3332, 2945, 1472, 1018, 667 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-1 and S2-2</a>; HR-ESIMS <i>m</i>/<i>z</i>: 414.2657 [M + H]<sup>+</sup> (calcd for C<sub>25</sub>H<sub>36</sub>NO<sub>4</sub><sup>+</sup>, 414.2639).</div><div class="NLM_p">7-Demethylglucopiericidin A (<b>12</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3420, 2928, 1472, 1456, 1125, 1016, 652 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-3 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 564.3185 [M + H]<sup>+</sup> (calcd for C<sub>30</sub>H<sub>46</sub>NO<sub>9</sub><sup>+</sup>, 564.3167).</div><div class="NLM_p">7-Demethyl-13-hydroxyglucopiericidin A (<b>13</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3302, 2930, 1585, 1464, 1412, 1125, 1022, 650 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-3 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 580.3126 [M + H]<sup>+</sup> (calcd for C<sub>30</sub>H<sub>46</sub>NO<sub>10</sub><sup>+</sup>, 580.3116).</div><div class="NLM_p">13-Hydroxypiericidin A 10-<i>O</i>-α-<span class="smallcaps smallerCapital">d</span>-galactose (1→6)-β-<span class="smallcaps smallerCapital">d</span>-glucoside (<b>15</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3319, 2930, 1650, 1412, 1126, 1022, 824, 764, 667 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-3 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 754.3646 [M – H]<sup>−</sup> (calcd for C<sub>37</sub>H<sub>56</sub>NO<sub>15</sub><sup>–</sup>, 754.3655).</div><div class="NLM_p">13-Hydroxypiericidin A 10-<i>O</i>-α-<span class="smallcaps smallerCapital">d</span>-glucose (1→6)-β-<span class="smallcaps smallerCapital">d</span>-glucoside (<b>17</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3358, 2920, 1585, 1412, 1125, 1022, 824, 764, 667 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-3 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 754.3660 [M – H]<sup>−</sup> (calcd for C<sub>37</sub>H<sub>56</sub>NO<sub>15</sub><sup>–</sup>, 754.3655).</div><div class="NLM_p">4′-<i>O</i>-β-<span class="smallcaps smallerCapital">d</span>-Glucose glucopiericidin A (<b>18</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3366, 2920, 1582, 1404, 1047, 669 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-3 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 738.3684 [M – H]<sup>−</sup> (calcd for C<sub>37</sub>H<sub>56</sub>NO<sub>14</sub><sup>–</sup>, 738.3706).</div><div class="NLM_p">4′-<i>O</i>-β-<span class="smallcaps smallerCapital">d</span>-Glucose 13-hydroxyglucopiericidin A (<b>19</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3292, 2943, 1456, 1404, 1020, 660 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-3 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 756.3829 [M + H]<sup>+</sup> (calcd for C<sub>37</sub>H<sub>58</sub>NO<sub>15</sub><sup>+</sup>, 756.3801).</div><div class="NLM_p">4′-<i>O</i>-β-<span class="smallcaps smallerCapital">d</span>-Glucose piericidin A 10-<i>O</i>-α-<span class="smallcaps smallerCapital">d</span>-glucose (1→6)-β-<span class="smallcaps smallerCapital">d</span>-glucoside (<b>20</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3342, 2930, 1568, 1456, 1018, 650 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-4 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 902.4390 [M + H]<sup>+</sup> (calcd for C<sub>43</sub>H<sub>68</sub>NO<sub>19</sub><sup>+</sup>, 902.4380).</div><div class="NLM_p">5-Hydroxy-6-hydroxymethyl glucopiericidin A (<b>21</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3356, 2916, 1568, 1471, 1412, 1016, 667 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-4 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 626.3543 [M + H]<sup>+</sup> (calcd for C<sub>32</sub>H<sub>52</sub>NO<sub>11</sub><sup>+</sup>, 626.3534).</div><div class="NLM_p">5-Hydroxy-6-hydroxymethyl-13-hydroxyglucopiericidin A (<b>22</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3392, 1670, 1200, 1134, 723 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-4 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 642.3481 [M + H]<sup>+</sup> (calcd for C<sub>32</sub>H<sub>52</sub>NO<sub>12</sub><sup>+</sup>, 642.3484).</div><div class="NLM_p">2-Hydroxymethyl-Δ3, 4-glucopiericidin A (<b>23</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3356, 2926, 1471, 1411, 1124, 1016, 648 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-4 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 608.3444 [M + H]<sup>+</sup> (calcd for C<sub>32</sub>H<sub>50</sub>NO<sub>10</sub><sup>+</sup>, 608.3429).</div><div class="NLM_p">(11<i>S</i>,12<i>R</i>)Piericidin C1 10-<i>O</i>-β-<span class="smallcaps smallerCapital">d</span>-glucoside (<b>24</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3354, 2932, 1587, 1468, 1412, 1126, 1074 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-4 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 594.3269 [M + H]<sup>+</sup> (calcd for C<sub>31</sub>H<sub>48</sub>NO<sub>10</sub><sup>+</sup>, 594.3273).</div><div class="NLM_p">(11<i>R</i>,12<i>S</i>)Piericidin C1 10-<i>O</i>-β-<span class="smallcaps smallerCapital">d</span>-glucoside (<b>25</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3366, 2926, 1585, 1464, 1412, 1125, 1076, 721 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-4 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 594.3288 [M + H]<sup>+</sup> (calcd for C<sub>31</sub>H<sub>48</sub>NO<sub>10</sub><sup>+</sup>, 594.3273).</div><div class="NLM_p last">13-Dimethoxy glucopiericidin A (<b>26</b>). Pale yellow oil; IR (film) ν<sub>max</sub>: 3358, 2926, 1670, 1200, 1136, 802, 723 cm<sup>–1</sup>; <sup>1</sup>H and <sup>13</sup>C NMR data, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">Tables S2-4 and S2-5</a>; HR-ESIMS <i>m</i>/<i>z</i>: 638.3535 [M + H]<sup>+</sup> (calcd for C<sub>33</sub>H<sub>52</sub>NO<sub>11</sub><sup>+</sup>, 638.3535).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">5.5.  Cell Culture and Cytotoxic Bioassay</h3><div class="NLM_p last">ACHN (cat. TCHu199), OS-RC-2 (cat. TCHu40), and 786-O (cat. TCHu186) cells, along with HK-2 (cat. SCSP-511) cells were purchased from Type Culture Collection of the Chinese Academy of Sciences, Shanghai, China. ACHN cells were grown and maintained in MEM medium with 10% fetal bovine serum (FBS), while OS-RC-2 and 786-O cells were grown in RPMI1640 medium with 10% FBS. HK-2 cells were grown in Dulbecco’s modified Eagle medium with 10% FBS. Cell viability was determined by the CCK-8 (Dojindo) assay.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Cells were seeded at a density of 400–800 cells/well in 384-well plates and then treated with various concentrations of compounds or solvent control. Sorafenib was used as the positive control. After 72 h of incubation, CCK-8 reagent was added, and absorbance of the triplicate tests were measured at 450 nm by an Envision 2104 multilabel reader (PerkinElmer). Dose-response curves were plotted to determine IC<sub>50</sub> using Prism 5.0 (GraphPad Software Inc.).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">5.6.  Cell Cycle and Apoptosis Assay</h3><div class="NLM_p last">Cell cycle arrest was analyzed by propidium iodide (PI) DNA staining using flow cytometry.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Briefly, after treatment with piericidins for 24, 48, and 72 h, respectively, cells were harvested, prepared, and then fixed overnight. The fixed cells were harvested, washed, resuspended, and finally stained with PI (Sigma-Aldrich). Cell cycle distribution was studied using an Accuri C6 (BD) flow cytometer. Cell apoptosis was analyzed using a fluorescein isothiocyanate (FITC) annexin V apoptosis detection kit (BD), according to the manufacturer’s protocol. Cells were treated with piericidins for 24, 48, and 72 h, stained with annexin V-FITC and PI solution, examined and analyzed quantitatively using an Accuri C6 (BD) flow cytometer.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">5.7.  Digital Holographic Microscopy</h3><div class="NLM_p last">The HoloMonitor M4 microscope (Phase Holographic Imaging AB, Lund, Sweden) was used for imaging and tracking ACHN cell movement and morphological changes induced by PA (<b>1</b>). This technique provides long-term kinetic cellular analysis without any cell labeling. The principle behind this method is the detection of the phase shift of the probing laser light transmitting or reflecting through a cell sample and comparison to the reference light.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The ACHN cells were planted (2 × 10<sup>5</sup> cells) on a 6-well plate and adhered for 24 h. A concentration of 50 nM PA (<b>1</b>) was added and then the field of each sample was detected under the microscope automatically. For the morphometry analysis, the color bar means the thickness of cells.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20">5.8.  Intracellular ROS Detection</h3><div class="NLM_p last">The intracellular ROS were measured by flow cytometry in a BD Accuri C6 flow cytometer (BD Biosciences) using CellROX oxidative stress reagents (cat no. C10422; Life Technologies, USA). ACHN cells were cultured in MEM medium with 10% FBS by 24 h. Then, the medium was replaced and the cells (2 × 10<sup>4</sup> cells/well) were cultured in 6-well plates, together with PA/GPA (0, 25 nM, 50 nM) in medium and incubated by 24 h. Next, the cells were collected and incubated with CellROX oxidative stress reagents 5 μM for 30 min/37 °C in darkness. BD Accuri C6 flow cytometer was used to analyze the cells with ROS.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21">5.9.  Lv-Mediated RNA Interference</h3><div class="NLM_p last">Lv expressing short hairpin RNA (shRNA) targeting PRDX1 was designed and chemically synthesized by Shanghai GeneChem Co. Ltd. (Shanghai, China), and the nontarget shRNA expressing green fluorescence protein was only used as the RNA inference control. Target sequences are as followed: -TGAGACTTTGAGACTAGTT- for PRDX1; -TTCTCCGAACGTGTCACGT- for vector. The ACHN cell at 50–70% confluency was transfected and then selected in medium containing puromycin (1 μg/mL). After confirming the stable transfection efficiency in 72 h, cells were treated with 25 nM and 50 nM PA or GPA for 24 h.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22">5.10.  Immunofluorescent Staining and Confocal Microscopy</h3><div class="NLM_p last">ACHN cells were co-incubated with PA/GPA in 24 h. After washing three times in PBS, ACHN cells were fixed in 4% paraformaldehyde for 10 min, and permeabilized with 0.5% Triton X-100 in PBS. After washing, cells were blocked with 10% FBS in 4 °C for 12 h and incubated with anti-PRDX1 primary antibody overnight. Cells were washed and then incubated with FITC-conjugated secondary antibody (green imaging) for 2 h at room temperature. DAPI staining was also performed for nucleus localization and shown in blue. Merge imaging was shown to indicate the subcellular localization of PRDX1. Cell morphology was visualized under a Zeiss fluorescence microscope (Carl Zeiss, LSM 880 with Airyscan, Germany).</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23">5.11.  SPR Assay</h3><div class="NLM_p last">The SPR measurement was performed on the Biacore T100 system (GE Healthcare). Full-length human PRDX1 was immobilized on a CM5 chip (GE healthcare). Diluted in 10 mM HEPES (pH 7.4) for giving a concentration series, PA/GPA samples were dispensed into single-used snap-capped vials and injected across the protein and the protein-free (control) surfaces in a single step. Compound binding with PRDX1 was analyzed in a single-cycle kinetic analysis range from 1 to 10 μM PA and 1 to 20 μM GPA, respectively. The <i>K</i><sub>D</sub> values were calculated using dynamic fitting equation.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24">5.12.  Molecular Docking</h3><div class="NLM_p last">The Schrödinger 2017-1 suite (Schrödinger Inc., New York, NY) was employed to perform dock analysis. The PRDX1 structure was retrieved from the available crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S">2Z9S</a>, chains a, b, c and d) and constructed following the Protein Prepare Wizard workflow in Maestro 11-1. The prepared ligands were then flexibly docked into the receptor using induce-fit module with default parameters.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25">5.13.  Tumorigenesis Assays</h3><div class="NLM_p last">Four-week old, male athymic nude (Balb/c nu/nu) mice were purchased from Guangdong Medical Lab Animal Center (China) and allowed to acclimatize for 1 week. Ten million ACHN cells in logarithmic growth phase, suspended in 0.2 mL PBS on ice, were inoculated into the right flank of the nude mice. After the tumors became palpable (around 150 mm<sup>3</sup>), mice were randomly divided into four groups, when the average weights and tumors volumes distributed equally in each group. Tumor-bearing mice were received PA (0.8 mg/kg) or GPA (0.8 mg/kg) daily by ip injection. Sorafenib tosylate (50 mg/kg) was dissolved in Cremophor EL (Sigma) as a 10× stock solution, further diluted in PBS before use. During the exposure period, the animals were kept at a temperature of 21 °C at a humidity of 55% under a 12 h/12 h light/dark cycle with free access to food and water. Tumor growth was measured through calipers, and tumor volumes and body weight were in record. The calculation of tumor volumes was according to the formula 0.5 × length × width<sup>2</sup>.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26">5.14.  Histology</h3><div class="NLM_p last">When nude mice were sacrificed at day 21, tumor tissues were rapidly fixed in 4% paraformaldehyde (Sigma) at room temperature overnight. After fixation, tissues were dehydrated in ethanol with increasing concentrations up to 100%. Dehydrated specimens were subsequently infiltrated with 100% xylene and then embedded in paraffin wax. Five μm-thick sections were cut for immune-histochemical studies, which were performed using standard techniques.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27">5.15.  Western Blot Analysis</h3><div class="NLM_p last">ACHN cells, shPRDX1-ACHN cells, and tumor tissues were lysed in RIPA buffer (Beyotime), including 50 mM Tris (pH 7.4), 150 mM NaCl, 1% Triton X-100, 1% sodium deoxycholate, 0.1% SDS, sodium orthovanadate, EDTA, and a protease inhibitor cocktail (Sigma). The supernatants were collected after centrifugation at 13 000 rpm at 4 °C for 30 min, and the quantification of protein content was estimated by the BCA protein assay kit (Beyotime), detected with a spectrophotometer using absorption at 562 nm. Proteins (30–50 μg) were separated on 10 or 12% polyacrylamide gels and transferred to polyvinylidene fluoride membranes using a Bio-Rad Trans-Blot wet transfer apparatus. Subsequently, the membranes were incubated with primary antibody (1:1000 for PRDX1, 1:3000 for β-actin, 1:3000 for GAPDH) overnight after blocking with 5% nonfat milk for 2 h at room temperature. Then the appropriate HRP-conjugated secondary antibody (1:3000) was used for detection by enhanced chemiluminescence substrates.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28">5.16.  Compliance with Ethical Standards</h3><div class="NLM_p last">Animal used was permitted by the animal ethics committee of Southern Medical University. The clinical samples were collected from the Nanfang Hospital of Southern Medical University, Guangzhou, China. Informed consents were obtained from all participants. Approval for tissue collection was obtained from the ethics committee of Nanfang Hospital of Southern Medical University. Baseline clinical characteristics of human kidney samples were listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">Table S10</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00598" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44226" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44226" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00598" class="ext-link">10.1021/acs.jmedchem.9b00598</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary figures and tables, structure elucidation and spectral data of compounds <b>1–27</b>, as well as their spectra (IR, MS, and NMR) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Transcriptome data and bioinformatic analysis (Tables S1–S6) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip" class="ext-link">ZIP</a>)</p></li><li><p class="inline">Molecular formula strings for <b>1–27</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf">jm9b00598_si_001.pdf (20.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip">jm9b00598_si_002.zip (976.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_003.csv">jm9b00598_si_003.csv (3.63 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00598" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23458" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23458" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xuefeng Zhou</span> - <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong
Key Laboratory of Marine Materia Medica, South China Sea Institute
of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9601-4869" title="Orcid link">http://orcid.org/0000-0001-9601-4869</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#49312f3321263c093a2a3a202667282a672a27"><span class="__cf_email__" data-cfemail="f8809e8290978db88b9b8b9197d6999bd69b96">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lan Tang</span> - <span class="hlFld-Affiliation affiliation">Biopharmaceutics, Guangdong Provincial
Key Laboratory of New Drug
Screening, School of Pharmaceutical Sciences, 
    State Key Laboratory of Organ Failure
Research, 
    , , Southern Medical University, Guangzhou 510515, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2345-0886" title="Orcid link">http://orcid.org/0000-0002-2345-0886</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#23574f17131663504e560d4647560d404d"><span class="__cf_email__" data-cfemail="f3879fc7c3c6b3809e86dd969786dd909d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhi Liang</span> - <span class="hlFld-Affiliation affiliation">Biopharmaceutics, Guangdong Provincial
Key Laboratory of New Drug
Screening, School of Pharmaceutical Sciences, , Southern Medical University, Guangzhou 510515, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kunlong Li</span> - <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong
Key Laboratory of Marine Materia Medica, South China Sea Institute
of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Fang</span> - <span class="hlFld-Affiliation affiliation">Hubei
Biopesticide Engineering Research Center, Hubei Academy of Agricultural Sciences, Wuhan 430064, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuanxin Tian</span> - <span class="hlFld-Affiliation affiliation">Biopharmaceutics, Guangdong Provincial
Key Laboratory of New Drug
Screening, School of Pharmaceutical Sciences, , Southern Medical University, Guangzhou 510515, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3847-559X" title="Orcid link">http://orcid.org/0000-0003-3847-559X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaowei Luo</span> - <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong
Key Laboratory of Marine Materia Medica, South China Sea Institute
of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2114-1609" title="Orcid link">http://orcid.org/0000-0002-2114-1609</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yulian Chen</span> - <span class="hlFld-Affiliation affiliation">Biopharmaceutics, Guangdong Provincial
Key Laboratory of New Drug
Screening, School of Pharmaceutical Sciences, , Southern Medical University, Guangzhou 510515, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhikun Zhan</span> - <span class="hlFld-Affiliation affiliation">Biopharmaceutics, Guangdong Provincial
Key Laboratory of New Drug
Screening, School of Pharmaceutical Sciences, , Southern Medical University, Guangzhou 510515, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tao Zhang</span> - <span class="hlFld-Affiliation affiliation">Biopharmaceutics, Guangdong Provincial
Key Laboratory of New Drug
Screening, School of Pharmaceutical Sciences, , Southern Medical University, Guangzhou 510515, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shengrong Liao</span> - <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong
Key Laboratory of Marine Materia Medica, South China Sea Institute
of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4393-3823" title="Orcid link">http://orcid.org/0000-0003-4393-3823</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuwen Liu</span> - <span class="hlFld-Affiliation affiliation">Biopharmaceutics, Guangdong Provincial
Key Laboratory of New Drug
Screening, School of Pharmaceutical Sciences, 
    State Key Laboratory of Organ Failure
Research, 
    , , Southern Medical University, Guangzhou 510515, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6346-5006" title="Orcid link">http://orcid.org/0000-0001-6346-5006</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yonghong Liu</span> - <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Tropical Marine Bio-Resources and Ecology, Guangdong
Key Laboratory of Marine Materia Medica, South China Sea Institute
of Oceanology, Chinese Academy of Sciences, Guangzhou 510301, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8327-3108" title="Orcid link">http://orcid.org/0000-0001-8327-3108</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William Fenical</span> - <span class="hlFld-Affiliation affiliation">Center
for Marine Biotechnology and Biomedicine, Scripps Institution of Oceanography, University of California at San Diego, La Jolla, California 92093, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8955-1735" title="Orcid link">http://orcid.org/0000-0002-8955-1735</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>X.Z. and Z.L. contributed equally.</p><span class="author-information-subsection-header">Author Contributions</span><p>X.Z. and Z.L. contributed equally. X.Z., K.L., W.F., X.L., S. Liao conducted chemical experiments; and Z.L., Y.C., Z.Z., T.Z., L.T. performed the biological experiments; Y.T., K.L., L.T. performed the docking analysis; X.Z., S. Liu, Y.L., W.F., L.T. conceived and designed the project; X.Z., Z.L., and L.T. wrote the manuscript with the assistance of all authors. All authors participated in data analyses and discussions.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by grants from the Guangdong MEPP Funds (GDME-2018C010, 2019-017), National Natural Science Foundation of China (81673677, 21772210), Drug Innovation Major Project of China (2018ZX09735001-002-003), the Natural Science Foundation of Guangdong Province (2016A030313592), the Applied Science and Technology Research Foundation of Guangdong Province (2016B020237005), and the Open Project of State Key Laboratory of Organ Failure Research (201802). We are grateful to Z. Xiao, A. Sun, and Y. Zhang from the analytical facilities of SCSIO.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">RCC</td><td class="NLM_def"><p class="first last">renal cell carcinoma</p></td></tr><tr><td class="NLM_term">ccRCC</td><td class="NLM_def"><p class="first last">clear cell renal cell carcinoma</p></td></tr><tr><td class="NLM_term">PA</td><td class="NLM_def"><p class="first last">piericidin A</p></td></tr><tr><td class="NLM_term">GPA</td><td class="NLM_def"><p class="first last">glucopiericidin A</p></td></tr><tr><td class="NLM_term">PRDX1</td><td class="NLM_def"><p class="first last">peroxiredoxin 1</p></td></tr><tr><td class="NLM_term">ROS</td><td class="NLM_def"><p class="first last">reactive oxygen species</p></td></tr><tr><td class="NLM_term">ECD</td><td class="NLM_def"><p class="first last">electronic circular dichroism</p></td></tr><tr><td class="NLM_term">HR-ESIMS</td><td class="NLM_def"><p class="first last">high resolution electrospray ionization mass spectrometer</p></td></tr><tr><td class="NLM_term">Sora</td><td class="NLM_def"><p class="first last">sorafenib</p></td></tr><tr><td class="NLM_term">bp</td><td class="NLM_def"><p class="first last">basepairs</p></td></tr><tr><td class="NLM_term">DEGs</td><td class="NLM_def"><p class="first last">differentially expressed genes</p></td></tr><tr><td class="NLM_term">sWGCNA</td><td class="NLM_def"><p class="first last">supervised weighted correlation network analysis</p></td></tr><tr><td class="NLM_term">KEGG</td><td class="NLM_def"><p class="first last">kyoto encyclopedia of genes and genomes</p></td></tr><tr><td class="NLM_term">FDR</td><td class="NLM_def"><p class="first last">false discovery rate</p></td></tr><tr><td class="NLM_term">RT-PCR</td><td class="NLM_def"><p class="first last">reverse transcription-polymerase chain reaction</p></td></tr><tr><td class="NLM_term">GEO</td><td class="NLM_def"><p class="first last">gene expression omnibus</p></td></tr><tr><td class="NLM_term">Kd</td><td class="NLM_def"><p class="first last">knockdown</p></td></tr><tr><td class="NLM_term">shRNA</td><td class="NLM_def"><p class="first last">short hairpin RNA</p></td></tr><tr><td class="NLM_term">Lv</td><td class="NLM_def"><p class="first last">lentivirus</p></td></tr><tr><td class="NLM_term">shPRDX1</td><td class="NLM_def"><p class="first last">lentivirus PRDX1 shRNA</p></td></tr><tr><td class="NLM_term">RNAi</td><td class="NLM_def"><p class="first last">RNA inference</p></td></tr><tr><td class="NLM_term">PI</td><td class="NLM_def"><p class="first last">propidium iodide</p></td></tr><tr><td class="NLM_term">K<sub>D</sub></td><td class="NLM_def"><p class="first last">dissociation equilibrium constants</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">HE</td><td class="NLM_def"><p class="first last">haematoxylin and eosin</p></td></tr><tr><td class="NLM_term">TUNEL</td><td class="NLM_def"><p class="first last">terminal deoxynucleotidyl transferase-mediated dUTP nick end labelling</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle medium</p></td></tr><tr><td class="NLM_term">PVDF</td><td class="NLM_def"><p class="first last">polyvinylidene fluoride</p></td></tr><tr><td class="NLM_term">ECL</td><td class="NLM_def"><p class="first last">enhanced chemiluminescence</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44622" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44622" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 30 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shroff, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlin, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouw, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellovin, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philbrick, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Ocana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casey, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zare, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felsher, D. W.</span></span> <span> </span><span class="NLM_article-title">MYC oncogene overexpression drives renal cell carcinoma in a mouse model through glutamine metabolism</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">6539</span>– <span class="NLM_lpage">6544</span>, <span class="refDoi"> DOI: 10.1073/pnas.1507228112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1073%2Fpnas.1507228112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=25964345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFCmtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=6539-6544&author=E.+H.+Shroffauthor=L.+S.+Eberlinauthor=V.+M.+Dangauthor=A.+M.+Gouwauthor=M.+Gabayauthor=S.+J.+Adamauthor=D.+I.+Bellovinauthor=P.+T.+Tranauthor=W.+M.+Philbrickauthor=A.+Garcia-Ocanaauthor=S.+C.+Caseyauthor=Y.+Liauthor=C.+V.+Dangauthor=R.+N.+Zareauthor=D.+W.+Felsher&title=MYC+oncogene+overexpression+drives+renal+cell+carcinoma+in+a+mouse+model+through+glutamine+metabolism&doi=10.1073%2Fpnas.1507228112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MYC oncogene overexpression drives renal cell carcinoma in a mouse model through glutamine metabolism</span></div><div class="casAuthors">Shroff, Emelyn H.; Eberlin, Livia S.; Dang, Vanessa M.; Gouw, Arvin M.; Gabay, Meital; Adam, Stacey J.; Bellovin, David I.; Tran, Phuoc T.; Philbrick, William M.; Garcia-Ocana, Adolfo; Casey, Stephanie C.; Li, Yulin; Dang, Chi V.; Zare, Richard N.; Felsher, Dean W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6539-6544</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The MYC oncogene is frequently mutated and overexpressed in human renal cell carcinoma (RCC).  However, there have been no studies on the causative role of MYC or any other oncogene in the initiation or maintenance of kidney tumorigenesis.  Here, we show through a conditional transgenic mouse model that the MYC oncogene, but not the RAS oncogene, initiates and maintains RCC.  Desorption electrospray ionization-mass-spectrometric imaging was used to obtain chem. maps of metabolites and lipids in the mouse RCC samples.  Gene expression anal. revealed that the mouse tumors mimicked human RCC.  The data suggested that MYC-induced RCC up-regulated the glutaminolytic pathway instead of the glycolytic pathway.  The pharmacol. inhibition of glutamine metab. with bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide impeded MYC-mediated RCC tumor progression.  Our studies demonstrate that MYC overexpression causes RCC and points to the inhibition of glutamine metab. as a potential therapeutic approach for the treatment of this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqokovxXkOTurVg90H21EOLACvtfcHk0liZQocGd9j3Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFCmtrw%253D&md5=e063f432422de8a2e485f998cc5b7b8c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1507228112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1507228112%26sid%3Dliteratum%253Aachs%26aulast%3DShroff%26aufirst%3DE.%2BH.%26aulast%3DEberlin%26aufirst%3DL.%2BS.%26aulast%3DDang%26aufirst%3DV.%2BM.%26aulast%3DGouw%26aufirst%3DA.%2BM.%26aulast%3DGabay%26aufirst%3DM.%26aulast%3DAdam%26aufirst%3DS.%2BJ.%26aulast%3DBellovin%26aufirst%3DD.%2BI.%26aulast%3DTran%26aufirst%3DP.%2BT.%26aulast%3DPhilbrick%26aufirst%3DW.%2BM.%26aulast%3DGarcia-Ocana%26aufirst%3DA.%26aulast%3DCasey%26aufirst%3DS.%2BC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DZare%26aufirst%3DR.%2BN.%26aulast%3DFelsher%26aufirst%3DD.%2BW.%26atitle%3DMYC%2520oncogene%2520overexpression%2520drives%2520renal%2520cell%2520carcinoma%2520in%2520a%2520mouse%2520model%2520through%2520glutamine%2520metabolism%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3D6539%26epage%3D6544%26doi%3D10.1073%2Fpnas.1507228112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varella, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rini, B. I.</span></span> <span> </span><span class="NLM_article-title">Emerging drugs for renal cell carcinoma</span>. <i>Expert Opin. Emerging Drugs</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1517/14728214.2010.486788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1517%2F14728214.2010.486788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=20443750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCnurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=343-353&author=L.+Varellaauthor=B.+I.+Rini&title=Emerging+drugs+for+renal+cell+carcinoma&doi=10.1517%2F14728214.2010.486788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging drugs for renal cell carcinoma</span></div><div class="casAuthors">Varella, Leticia; Rini, Brian I.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Emerging Drugs</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">343-353</span>CODEN:
                <span class="NLM_cas:coden">EOEDA3</span>;
        ISSN:<span class="NLM_cas:issn">1472-8214</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: The treatment of metastatic renal cell carcinoma (RCC) has evolved significantly over the past 5 years and targeted therapy has become the std. of care.  However, complete and lasting responses to these drugs are still uncommon.  Currently, novel agents are being tested in clin. trials.Areas covered in this review: We discuss approved targeted agents in RCC and emphasize on emerging therapies.  We searched the US National Library of Medicine (PubMed) using the terms renal cell carcinoma', targeted therapy' and novel agents', from 1990 to the present.  We also searched for abstrs. from the meetings of the American Society of Clin. Oncol. and Genitourinary Cancers Symposium from 2007 to the present.What the reader will gain: This paper provides understanding of the approved treatments for advanced RCC as well as the novel agents and their targeted biol. pathways.Take home message: Despite dynamic and recent advances in the management of metastatic RCC much about the mol. biol. of this tumor has yet to be delineated.  Even more so, strategies of sequential treatment, combination of targeted drugs, mechanisms of resistance, optimal dosages and scheduling remain areas of potential development interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJjrGQcWT-HbVg90H21EOLACvtfcHk0lgb0HgLwbSZeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCnurrI&md5=06e8d93a467b9836c1870e80361cec6b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1517%2F14728214.2010.486788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728214.2010.486788%26sid%3Dliteratum%253Aachs%26aulast%3DVarella%26aufirst%3DL.%26aulast%3DRini%26aufirst%3DB.%2BI.%26atitle%3DEmerging%2520drugs%2520for%2520renal%2520cell%2520carcinoma%26jtitle%3DExpert%2520Opin.%2520Emerging%2520Drugs%26date%3D2010%26volume%3D15%26spage%3D343%26epage%3D353%26doi%3D10.1517%2F14728214.2010.486788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C.</span></span> <span> </span><span class="NLM_article-title">Targeted therapy: An elusive cancer target</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>537</i></span>,  <span class="NLM_fpage">S106</span>– <span class="NLM_lpage">S108</span>, <span class="refDoi"> DOI: 10.1038/537s106a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2F537s106a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=27626779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=537&publication_year=2016&pages=S106-S108&author=C.+Brown&title=Targeted+therapy%3A+An+elusive+cancer+target&doi=10.1038%2F537s106a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted therapy: An elusive cancer target</span></div><div class="casAuthors">Brown, Carolyn</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">537</span>
        (<span class="NLM_cas:issue">7620</span>),
    <span class="NLM_cas:pages">S106-S108</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Advanced tumors may have met their match with new drugs, but why have these treatments proved ineffective at stopping early-stage tumors from coming back.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmzsizLsmhwrVg90H21EOLACvtfcHk0lgU1K12JgBNYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiurjM&md5=c6c8f781dc6040098118dfaa6e68b430</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F537s106a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F537s106a%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DC.%26atitle%3DTargeted%2520therapy%253A%2520An%2520elusive%2520cancer%2520target%26jtitle%3DNature%26date%3D2016%26volume%3D537%26spage%3DS106%26epage%3DS108%26doi%3D10.1038%2F537s106a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mehra, R.</span></span> <span> </span><span class="NLM_article-title">An Insight into the “Dark Matter” of Kidney Cancer</span>. <i>Eur. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">766</span>, <span class="refDoi"> DOI: 10.1016/j.eururo.2018.08.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1016%2Fj.eururo.2018.08.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=30177289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ltVeiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=764-766&author=R.+Mehra&title=An+Insight+into+the+%E2%80%9CDark+Matter%E2%80%9D+of+Kidney+Cancer&doi=10.1016%2Fj.eururo.2018.08.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An Insight into the "Dark Matter" of Kidney Cancer</span></div><div class="casAuthors">Mehra Rohit</div><div class="citationInfo"><span class="NLM_cas:title">European urology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">764-766</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTh0SbA2MTWTn6oPH0mYWL1fW6udTcc2eYq9vQ_u5UUerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ltVeiuw%253D%253D&md5=da71b5363e6c42c85e7d7b49a331e849</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.eururo.2018.08.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.eururo.2018.08.023%26sid%3Dliteratum%253Aachs%26aulast%3DMehra%26aufirst%3DR.%26atitle%3DAn%2520Insight%2520into%2520the%2520%25E2%2580%259CDark%2520Matter%25E2%2580%259D%2520of%2520Kidney%2520Cancer%26jtitle%3DEur.%2520Urol.%26date%3D2018%26volume%3D74%26spage%3D764%26epage%3D766%26doi%3D10.1016%2Fj.eururo.2018.08.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Cancer
Genome Atlas Research Network</span> <span> </span><span class="NLM_article-title">Comprehensive molecular characterization of clear cell renal cell
carcinoma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>499</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1038/nature12222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Fnature12222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=23792563" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&author=Cancer%0AGenome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell%0Acarcinoma&doi=10.1038%2Fnature12222"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature12222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12222%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%2520molecular%2520characterization%2520of%2520clear%2520cell%2520renal%2520cell%250Acarcinoma%26jtitle%3DNature%26date%3D2013%26volume%3D499%26spage%3D43%26epage%3D49%26doi%3D10.1038%2Fnature12222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenical, W.</span></span> <span> </span><span class="NLM_article-title">The unique chemistry and biology of the piericidins</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">582</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1038/ja.2016.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Fja.2016.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=27301663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWjtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2016&pages=582-593&author=X.+Zhouauthor=W.+Fenical&title=The+unique+chemistry+and+biology+of+the+piericidins&doi=10.1038%2Fja.2016.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The unique chemistry and biology of the piericidins</span></div><div class="casAuthors">Zhou, Xuefeng; Fenical, William</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">582-593</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The piericidin family of microbial metabolites features a 4-pyridinol core linked with a methylated polyketide side chain.  Piericidins are exclusively produced by actinomycetes, esp. members of the genus Streptomyces.  The close structural similarity with coenzyme Q renders the piericidins important NADH-ubiquinone oxidoreductase (complex I) inhibitors in the mitochondrial electron transport chain.  Because of the significant activities of the piericidins, which include insecticidal, antimicrobial and antitumor effects, total syntheses of the piericidins were developed using various synthetic strategies.  The biosynthetic origin of this class has also been the subject of investigation.  This review covers the isolation and structure detn. of the natural piericidins, their chem. modification, the total syntheses of natural and unnatural analogs, their biosynthesis, and reported biol. activities together with structure-activity relationships.  Given the fundamental biol. of this class of metabolites, the piericidin family will likely continue to attract attention as biol. probes of important biosynthetic processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Nh2kFRxqVbVg90H21EOLACvtfcHk0lg-5FPdRTy8Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWjtbfK&md5=f72687698ec9ccb3a067be1a398c5bf4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fja.2016.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2016.71%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DFenical%26aufirst%3DW.%26atitle%3DThe%2520unique%2520chemistry%2520and%2520biology%2520of%2520the%2520piericidins%26jtitle%3DJ.%2520Antibiot.%26date%3D2016%26volume%3D69%26spage%3D582%26epage%3D593%26doi%3D10.1038%2Fja.2016.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span> <span> </span><span class="NLM_article-title">Geranylpyrrol A and Piericidin F from Streptomyces sp. CHQ-64 ΔrdmF</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">1684</span>– <span class="NLM_lpage">1687</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.7b00016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.7b00016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVyqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2017&pages=1684-1687&author=X.+Hanauthor=Z.+Liuauthor=Z.+Zhangauthor=X.+Zhangauthor=T.+Zhuauthor=Q.+Guauthor=W.+Liauthor=Q.+Cheauthor=D.+Li&title=Geranylpyrrol+A+and+Piericidin+F+from+Streptomyces+sp.+CHQ-64+%CE%94rdmF&doi=10.1021%2Facs.jnatprod.7b00016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Geranylpyrrol A and Piericidin F from Streptomyces sp. CHQ-64 ΔrdmF</span></div><div class="casAuthors">Han, Xiaoning; Liu, Zengzhi; Zhang, Zhenzhen; Zhang, Xiaomin; Zhu, Tianjiao; Gu, Qianqun; Li, Wenli; Che, Qian; Li, Dehai</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1684-1687</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Two new compds., geranylpyrrol A (I) and piericidin F (II), were isolated from a reedsmycins nonproducing mutant strain of Streptomyces sp. CHQ-64.  Their structures including abs. configurations, were elucidated by extensive NMR, MS, NOESY and ECD analyses.  Geranylpyrrol A is an unusual naturally occurring 2,3,4-trisubstituted pyrrole and piericidin F showed cytotoxicity against Hela, NB4, A549 and H1975 cell lines with IC50 values of 0.003, 0.037, 0.56, and 0.49 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi73p1d_i8k7Vg90H21EOLACvtfcHk0lh1WVa_MbKutw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVyqtbY%253D&md5=e476f3fe691aa4cdbb30d74b1c748042</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.7b00016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.7b00016%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DGu%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DChe%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DD.%26atitle%3DGeranylpyrrol%2520A%2520and%2520Piericidin%2520F%2520from%2520Streptomyces%2520sp.%2520CHQ-64%2520%25CE%2594rdmF%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2017%26volume%3D80%26spage%3D1684%26epage%3D1687%26doi%3D10.1021%2Facs.jnatprod.7b00016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shang, N.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.-X.</span></span> <span> </span><span class="NLM_article-title">Glycosylated piericidins from an endophytic streptomyces with cytotoxicity and antimicrobial activity</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1038/s41429-018-0051-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Fs41429-018-0051-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29651143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Wmtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2018&pages=672-676&author=N.-N.+Shangauthor=Z.+Zhangauthor=J.-P.+Huangauthor=L.+Wangauthor=J.+Luoauthor=J.+Yangauthor=T.+Pengauthor=Y.+Yanauthor=Y.-T.+Maauthor=S.-X.+Huang&title=Glycosylated+piericidins+from+an+endophytic+streptomyces+with+cytotoxicity+and+antimicrobial+activity&doi=10.1038%2Fs41429-018-0051-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Glycosylated piericidins from an endophytic streptomyces with cytotoxicity and antimicrobial activity</span></div><div class="casAuthors">Shang, Ning-Ning; Zhang, Zhouxin; Huang, Jian-Ping; Wang, Li; Luo, Jianying; Yang, Jing; Peng, Teng; Yan, Yijun; Ma, Ya-Tuan; Huang, Sheng-Xiong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-676</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Two new glycosylated piericidins, glucopiericidinol A3 (1) and 7-demethyl-glucopiericidin A (2), along with four known analogs were isolated from the culture broth of Streptomyces sp. KIB-H1083.  The chem. structures of new compds. were elucidated by spectroscopic analyses.  Their cytotoxicity on HL-60, SMMC-772, A-549, MCF-7, and SW480 cell lines, as well as antimicrobial activities was evaluated.  The results showed that glucopiericidin A (4) has potent cytotoxicity against HL-60, SMMC-772, A-549, and MCF-7 cell lines with IC50 values of 0.34, 0.65, 0.60, and 0.50 μM, resp.  For the antimicrobial activity, piericidin A (6) showed most powerful inhibitory activities against Xanthomonas oryzae pv. oryzicola, and Penicillium decumbens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKVUVFi1kV_bVg90H21EOLACvtfcHk0lgbR3EH1MJLyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Wmtbo%253D&md5=64c4563cc846c0f3b73e7242bb2da127</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41429-018-0051-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41429-018-0051-1%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DN.-N.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DJ.-P.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.-T.%26aulast%3DHuang%26aufirst%3DS.-X.%26atitle%3DGlycosylated%2520piericidins%2520from%2520an%2520endophytic%2520streptomyces%2520with%2520cytotoxicity%2520and%2520antimicrobial%2520activity%26jtitle%3DJ.%2520Antibiot.%26date%3D2018%26volume%3D71%26spage%3D672%26epage%3D676%26doi%3D10.1038%2Fs41429-018-0051-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schnermann, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamaru-Ogiso, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Total synthesis of piericidin A1 and B1 and key analogues</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">11799</span>– <span class="NLM_lpage">11807</span>, <span class="refDoi"> DOI: 10.1021/ja0632862</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0632862" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD28XosVCit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=11799-11807&author=M.+J.+Schnermannauthor=F.+A.+Romeroauthor=I.+Hwangauthor=E.+Nakamaru-Ogisoauthor=T.+Yagiauthor=D.+L.+Boger&title=Total+synthesis+of+piericidin+A1+and+B1+and+key+analogues&doi=10.1021%2Fja0632862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis of Piericidin A1 and B1 and Key Analogues</span></div><div class="casAuthors">Schnermann, Martin J.; Romero, F. Anthony; Hwang, Inkyu; Nakamaru-Ogiso, Eiko; Yagi, Takao; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">11799-11807</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Piericidin A1 and B1 I (R = H, Me) and analogs with modified side chains and pyridine cores (and the enantiomer of piericidin A1) are prepd. enantioselectively; the effect of structural changes on the inhibition of mitochondrial complex I (ubiquinone reductase) and on their cytotoxicities in mouse leukemia cell line L1210 are detd.  The key step in the synthesis of I (R = H, Me) and their analogs is the inverse electron-demand hetero-Diels-Alder reactions of α-(sulfonylimino)-β,γ-unsatd. esters with tetramethoxyethene followed by elimination of the sulfonyl group, a proton, and two methoxy groups in the presence of boron trifluoride etherate to yield functionalized dimethoxypyridinecarboxylates which are oxidized to the piericidin core by a lithiation/boronation/oxidn. sequence.  The side chains of I (R = H, Me) and their nonracemic analogs are prepd. by use of stereoselective aldol addns. of nonracemic propanoyloxazolidinones and by stereoselective Julia couplings of allylic phenyltetrazolyl sulfones with nonracemic aldehydes.  The fully elaborated side chains are prepd. as alkenylstannanes; Stille coupling reactions with bromomethylpyridines in the presence of palladium catalysts generates protected piericidins and their analogs which are converted to the desired products.  While simplified analogs of I inhibit mitochondrial complex I with similar affinities, the cytotoxicities of analogs of I in mouse leukemia cell line L1210 vary with structural changes in the side chain and in the pyridine core,.  The crystal structures of a tetrahydropyridinecarboxylate and of an α-(triisopropylsilyl)pyridinemethanol, intermediates in the prepn. of I (R = H, Me), are detd. by X-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg7FxLd3_uxbVg90H21EOLACvtfcHk0lgbR3EH1MJLyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosVCit7w%253D&md5=f18bfee48e8b1136445a28548016e677</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fja0632862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0632862%26sid%3Dliteratum%253Aachs%26aulast%3DSchnermann%26aufirst%3DM.%2BJ.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DHwang%26aufirst%3DI.%26aulast%3DNakamaru-Ogiso%26aufirst%3DE.%26aulast%3DYagi%26aufirst%3DT.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DTotal%2520synthesis%2520of%2520piericidin%2520A1%2520and%2520B1%2520and%2520key%2520analogues%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D11799%26epage%3D11807%26doi%3D10.1021%2Fja0632862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chooi, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, D.</span></span> <span> </span><span class="NLM_article-title">Elucidation of Piericidin A1 Biosynthetic Locus Revealed a Thioesterase-Dependent Mechanism of α-Pyridone Ring Formation</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">243</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1016%2Fj.chembiol.2011.12.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=22365607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWgt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=243-253&author=Q.+Liuauthor=F.+Yaoauthor=Y.+H.+Chooiauthor=Q.+Kangauthor=W.+Xuauthor=Y.+Liauthor=Y.+Shaoauthor=Y.+Shiauthor=Z.+Dengauthor=Y.+Tangauthor=D.+You&title=Elucidation+of+Piericidin+A1+Biosynthetic+Locus+Revealed+a+Thioesterase-Dependent+Mechanism+of+%CE%B1-Pyridone+Ring+Formation&doi=10.1016%2Fj.chembiol.2011.12.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidation of Piericidin A1 Biosynthetic Locus Revealed a Thioesterase-Dependent Mechanism of α-Pyridone Ring Formation</span></div><div class="casAuthors">Liu, Qian; Yao, Fen; Chooi, Yit Heng; Kang, Qianjin; Xu, Wei; Li, Yanran; Shao, Yucheng; Shi, Yuefeng; Deng, Zixin; Tang, Yi; You, Delin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">243-253</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Piericidins are a class of α-pyridone antibiotics that inhibit mitochondrial respiratory chain and exhibit antimicrobial, antifungal, and antitumor activities.  Sequential anal. of Streptomyces piomogenus var. Hangzhouwanensis genome revealed six modular polyketide synthases, an amidotransferase, two methyltransferases, and a monooxygenase for piericidin A1 prodn.  Gene functional anal. and deletion results provide overview of the biosynthesis pathway.  Furthermore, in vitro characterization of the terminal polyketide synthase module with the thioesterase domain using β-ketoacyl substrates was performed.  That revealed a pathway where the α-pyridone ring formation is dependent on hydrolysis of the product β, δ-diketo carboxylic acid by the C-terminal thioesterase followed by amidation and cyclization.  These findings set the stage to investigate unusual enzymic mechanisms in α-pyridone antibiotics biosynthesis, provide a foundation for genome mining of α-pyridone antibiotics, and produce analogs by mol. engineering.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZmRvP-V-_ObVg90H21EOLACvtfcHk0liR2qWhMATLow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWgt78%253D&md5=3b54f93840529bad2fcd121529a40dbf</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.018%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DYao%26aufirst%3DF.%26aulast%3DChooi%26aufirst%3DY.%2BH.%26aulast%3DKang%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DD.%26atitle%3DElucidation%2520of%2520Piericidin%2520A1%2520Biosynthetic%2520Locus%2520Revealed%2520a%2520Thioesterase-Dependent%2520Mechanism%2520of%2520%25CE%25B1-Pyridone%2520Ring%2520Formation%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D243%26epage%3D253%26doi%3D10.1016%2Fj.chembiol.2011.12.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span> <span> </span><span class="NLM_article-title">Elucidating hydroxylation and methylation steps tailoring piericidin A1 biosynthesis</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1021/ol4034176</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol4034176" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFWiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=736-739&author=Y.+Chenauthor=W.+Zhangauthor=Y.+Zhuauthor=Q.+Zhangauthor=X.+Tianauthor=S.+Zhangauthor=C.+Zhang&title=Elucidating+hydroxylation+and+methylation+steps+tailoring+piericidin+A1+biosynthesis&doi=10.1021%2Fol4034176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidating Hydroxylation and Methylation Steps Tailoring Piericidin A1 Biosynthesis</span></div><div class="casAuthors">Chen, Yaolong; Zhang, Wenjun; Zhu, Yiguang; Zhang, Qingbo; Tian, Xinpeng; Zhang, Si; Zhang, Changsheng</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">736-739</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The piericidin A1 (1) gene cluster was identified from the deep-sea derived Streptomyces sp. SCSIO 03032.  Our in vivo and in vitro expts. verified PieE as a 4'-hydroxylase and PieB2 as a 4'-O-methyltransferase, allowing the elucidation of the post-PKS modification steps involved in biosynthesis of 1.  In addn., the shunt metabolite piericidin E1 (7) was identified as a novel analog featuring a C-2/C-3 epoxy ring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3z39sbcVjqrVg90H21EOLACvtfcHk0liyuQjNPHQFlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFWiuw%253D%253D&md5=9971b699f7510764070b94d0ca93df8d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fol4034176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol4034176%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DC.%26atitle%3DElucidating%2520hydroxylation%2520and%2520methylation%2520steps%2520tailoring%2520piericidin%2520A1%2520biosynthesis%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D736%26epage%3D739%26doi%3D10.1021%2Fol4034176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carman, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubey, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Etten, R. A.</span></span> <span> </span><span class="NLM_article-title">Essential role for the peroxiredoxin Prdx1 in erythrocyte antioxidant defence and tumour suppression</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">565</span>, <span class="refDoi"> DOI: 10.1038/nature01819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Fnature01819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=12891360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvVGis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2003&pages=561-565&author=C.+A.+Neumannauthor=D.+S.+Krauseauthor=C.+V.+Carmanauthor=S.+Dasauthor=D.+P.+Dubeyauthor=J.+L.+Abrahamauthor=R.+T.+Bronsonauthor=Y.+Fujiwaraauthor=S.+H.+Orkinauthor=R.+A.+Van+Etten&title=Essential+role+for+the+peroxiredoxin+Prdx1+in+erythrocyte+antioxidant+defence+and+tumour+suppression&doi=10.1038%2Fnature01819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role for the peroxiredoxin Prdx1 in erythrocyte antioxidant defense and tumor suppression</span></div><div class="casAuthors">Neumann, Carola A.; Krause, Daniela S.; Carman, Christopher V.; Das, Shampa; Dubey, Devendra P.; Abraham, Jennifer L.; Bronson, Roderick T.; Fujiwara, Yuko; Orkin, Stuart H.; Van Etten, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">424</span>
        (<span class="NLM_cas:issue">6948</span>),
    <span class="NLM_cas:pages">561-565</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reactive oxygen species are involved in many cellular metabolic and signaling processes and are thought to have a role in disease, particularly in carcinogenesis and aging.  We have generated mice with targeted inactivation of Prdx1, a member of the peroxiredoxin family of antioxidant enzymes.  Here we show that mice lacking Prdx1 are viable and fertile but have a shortened lifespan owing to the development beginning at about 9 mo of severe hemolytic anemia and several malignant cancers, both of which are also obsd. at increased frequency in heterozygotes.  The hemolytic anemia is characterized by an increase in erythrocyte reactive oxygen species, leading to protein oxidn., Hb instability, Heinz body formation and decreased erythrocyte lifespan.  The malignancies include lymphomas, sarcomas and carcinomas, and are frequently assocd. with loss of Prdx1 expression in heterozygotes, which suggests that this protein functions as a tumor suppressor.  Prdx1-deficient fibroblasts show decreased proliferation and increased sensitivity to oxidative DNA damage, whereas Prdx1-null mice have abnormalities in nos., phenotype and function of natural killer cells.  Our results implicate Prdx1 as an important defense against oxidants in aging mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxSf4jPolWPrVg90H21EOLACvtfcHk0liyuQjNPHQFlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvVGis7g%253D&md5=4f49676ed87d35bb42a2c51a907ba8ac</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature01819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01819%26sid%3Dliteratum%253Aachs%26aulast%3DNeumann%26aufirst%3DC.%2BA.%26aulast%3DKrause%26aufirst%3DD.%2BS.%26aulast%3DCarman%26aufirst%3DC.%2BV.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DDubey%26aufirst%3DD.%2BP.%26aulast%3DAbraham%26aufirst%3DJ.%2BL.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DVan%2BEtten%26aufirst%3DR.%2BA.%26atitle%3DEssential%2520role%2520for%2520the%2520peroxiredoxin%2520Prdx1%2520in%2520erythrocyte%2520antioxidant%2520defence%2520and%2520tumour%2520suppression%26jtitle%3DNature%26date%3D2003%26volume%3D424%26spage%3D561%26epage%3D565%26doi%3D10.1038%2Fnature01819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span> <span> </span><span class="NLM_article-title">Peroxiredoxin 1 - an antioxidant enzyme in cancer</span>. <i>J. Cell. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1111/jcmm.12955</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1111%2Fjcmm.12955" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=27653015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis12l" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=193-202&author=C.+Dingauthor=X.+Fanauthor=G.+Wu&title=Peroxiredoxin+1+-+an+antioxidant+enzyme+in+cancer&doi=10.1111%2Fjcmm.12955"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peroxiredoxin 1 - an antioxidant enzyme in cancer</span></div><div class="casAuthors">Ding, Chenbo; Fan, Xiaobo; Wu, Guoqiu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular and Molecular Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">193-202</span>CODEN:
                <span class="NLM_cas:coden">JCMMC9</span>;
        ISSN:<span class="NLM_cas:issn">1582-4934</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Peroxiredoxins (PRDXs), a ubiquitous family of redox-regulating proteins, are reported of potential to eliminate various reactive oxygen species (ROS).  As a major member of the antioxidant enzymes, PRDX1 can become easily over-oxidized on its catalytically active cysteine induced by a variety of stimuli in vitro and in vivo.  In nucleus, oligomeric PRDX1 directly assocs. with p53 or transcription factors such as c-Myc, NF-κB and AR, and thus affects their bioactivities upon gene regulation, which in turn induces or suppresses cell death.  Addnl., PRDX1 in cytoplasm has anti-apoptotic potential through direct or indirect interactions with several ROS-dependent (redox regulation) effectors, including ASK1, p66Shc, GSTpi/JNK and c-Abl kinase.  PRDX1 is proven to be a versatile mol. regulating cell growth, differentiation and apoptosis.  Recent studies have found that PRDX1 and/or PRDX1-regulated ROS-dependent signalling pathways play an important role in the progression and metastasis of human tumors, particularly in breast, oesophageal and lung cancers.  In this paper, we review the structure, effector functions of PRDX1, its role in cancer and the pivotal role of ROS in anticancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRWU1Ys5zsRrVg90H21EOLACvtfcHk0lgcdO0IrPJY-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis12l&md5=8dd9793091969a8123a923ef4b30a119</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1111%2Fjcmm.12955&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjcmm.12955%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DC.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DG.%26atitle%3DPeroxiredoxin%25201%2520-%2520an%2520antioxidant%2520enzyme%2520in%2520cancer%26jtitle%3DJ.%2520Cell.%2520Mol.%2520Med.%26date%3D2017%26volume%3D21%26spage%3D193%26epage%3D202%26doi%3D10.1111%2Fjcmm.12955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hampton, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vick, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoko, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, C. A.</span></span> <span> </span><span class="NLM_article-title">Peroxiredoxin involvement in the initiation and progression of human cancer</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1089/ars.2017.7422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1089%2Fars.2017.7422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29237274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVOqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=591-608&author=M.+B.+Hamptonauthor=K.+A.+Vickauthor=J.+J.+Skokoauthor=C.+A.+Neumann&title=Peroxiredoxin+involvement+in+the+initiation+and+progression+of+human+cancer&doi=10.1089%2Fars.2017.7422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Peroxiredoxin Involvement in the Initiation and Progression of Human Cancer</span></div><div class="casAuthors">Hampton, Mark B.; Vick, Kate A.; Skoko, John J.; Neumann, Carola A.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">591-608</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Significance: It has been proposed that cancer cells are heavily dependent on their antioxidant defenses for survival and growth.  Peroxiredoxins are a family of abundant thiol-dependent peroxidases that break down hydrogen peroxide, and they have a central role in the maintenance and response of cells to alterations in redox homeostasis.  As such, they are potential targets for disrupting tumor growth.  Recent Advances: Genetic disruption of peroxiredoxin expression in mice leads to an increased incidence of neoplastic disease, consistent with a role for peroxiredoxins in protecting genomic integrity.  In contrast, many human tumors display increased levels of peroxiredoxin expression, suggesting that strengthened antioxidant defenses provide a survival advantage for tumor progression.  Peroxiredoxin inhibitors are being developed and explored as therapeutic agents in different cancer models.  Crit. Issues: It is important to complement peroxiredoxin knockout and expression studies with an improved understanding of the biol. function of the peroxiredoxins.  Although current results can be interpreted within the context that peroxiredoxins scavenge hydroperoxides, some peroxiredoxin family members appear to have more complex roles in regulating the response of cells to oxidative stress through protein interactions with constituents of other signaling pathways.  Future Directions: Further mechanistic information is required for understanding the role of oxidative stress in cancer, the function of peroxiredoxins in normal vs. cancer cells, and for the design and testing of specific peroxiredoxin inhibitors that display selectivity to malignant cells.  Antioxid. Redox Signal. 28, 591-608.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc3-4Dxu0zIbVg90H21EOLACvtfcHk0lhSAho0PGB0ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVOqsLc%253D&md5=66472e56b6f808a6edac98b49fa849fd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1089%2Fars.2017.7422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2017.7422%26sid%3Dliteratum%253Aachs%26aulast%3DHampton%26aufirst%3DM.%2BB.%26aulast%3DVick%26aufirst%3DK.%2BA.%26aulast%3DSkoko%26aufirst%3DJ.%2BJ.%26aulast%3DNeumann%26aufirst%3DC.%2BA.%26atitle%3DPeroxiredoxin%2520involvement%2520in%2520the%2520initiation%2520and%2520progression%2520of%2520human%2520cancer%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2018%26volume%3D28%26spage%3D591%26epage%3D608%26doi%3D10.1089%2Fars.2017.7422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perroud, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valkova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhirapong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiehn, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kültz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, R. H.</span></span> <span> </span><span class="NLM_article-title">Pathway analysis of kidney cancer using proteomics and metabolic profiling</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-5-64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1186%2F1476-4598-5-64" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=17123452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A280%3ADC%252BD28npsFejuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=64&author=B.+Perroudauthor=J.+Leeauthor=N.+Valkovaauthor=A.+Dhirapongauthor=P.-Y.+Linauthor=O.+Fiehnauthor=D.+K%C3%BCltzauthor=R.+H.+Weiss&title=Pathway+analysis+of+kidney+cancer+using+proteomics+and+metabolic+profiling&doi=10.1186%2F1476-4598-5-64"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Pathway analysis of kidney cancer using proteomics and metabolic profiling</span></div><div class="casAuthors">Perroud Bertrand; Lee Jinoo; Valkova Nelly; Dhirapong Amy; Lin Pei-Yin; Fiehn Oliver; Kultz Dietmar; Weiss Robert H</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Renal cell carcinoma (RCC) is the sixth leading cause of cancer death and is responsible for 11,000 deaths per year in the US.  Approximately one-third of patients present with disease which is already metastatic and for which there is currently no adequate treatment, and no biofluid screening tests exist for RCC.  In this study, we have undertaken a comprehensive proteomic analysis and subsequently a pathway and network approach to identify biological processes involved in clear cell RCC (ccRCC).  We have used these data to investigate urinary markers of RCC which could be applied to high-risk patients, or to those being followed for recurrence, for early diagnosis and treatment, thereby substantially reducing mortality of this disease.  RESULTS:  Using 2-dimensional electrophoresis and mass spectrometric analysis, we identified 31 proteins which were differentially expressed with a high degree of significance in ccRCC as compared to adjacent non-malignant tissue, and we confirmed some of these by immunoblotting, immunohistochemistry, and comparison to published transcriptomic data.  When evaluated by several pathway and biological process analysis programs, these proteins are demonstrated to be involved with a high degree of confidence (p values < 2.0 E-05) in glycolysis, propanoate metabolism, pyruvate metabolism, urea cycle and arginine/proline metabolism, as well as in the non-metabolic p53 and FAS pathways.  In a pilot study using random urine samples from both ccRCC and control patients, we performed metabolic profiling and found that only sorbitol, a component of an alternative glycolysis pathway, is significantly elevated at 5.4-fold in RCC patients as compared to controls.  CONCLUSION:  Extensive pathway and network analysis allowed for the discovery of highly significant pathways from a set of clear cell RCC samples.  Knowledge of activation of these processes will lead to novel assays identifying their proteomic and/or metabolomic signatures in biofluids of patient at high risk for this disease; we provide pilot data for such a urinary bioassay.  Furthermore, we demonstrate how the knowledge of networks, processes, and pathways altered in kidney cancer may be used to influence the choice of optimal therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTw5QbiB5WoQIJKJbLTgsvCfW6udTcc2eZqg4Wbddn1sLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28npsFejuw%253D%253D&md5=8a7afd646d8309932d6cba85053cdfad</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-5-64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-5-64%26sid%3Dliteratum%253Aachs%26aulast%3DPerroud%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DValkova%26aufirst%3DN.%26aulast%3DDhirapong%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DP.-Y.%26aulast%3DFiehn%26aufirst%3DO.%26aulast%3DK%25C3%25BCltz%26aufirst%3DD.%26aulast%3DWeiss%26aufirst%3DR.%2BH.%26atitle%3DPathway%2520analysis%2520of%2520kidney%2520cancer%2520using%2520proteomics%2520and%2520metabolic%2520profiling%26jtitle%3DMol.%2520Cancer%26date%3D2006%26volume%3D5%26spage%3D64%26doi%3D10.1186%2F1476-4598-5-64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaiswing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clair, W. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clair, D. K. S.</span></span> <span> </span><span class="NLM_article-title">Redox paradox: A novel approach to therapeutics-resistant cancer</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1237</span>– <span class="NLM_lpage">1272</span>, <span class="refDoi"> DOI: 10.1089/ars.2017.7485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1089%2Fars.2017.7485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29325444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslKqtrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=1237-1272&author=L.+Chaiswingauthor=W.+H.+S.+Clairauthor=D.+K.+S.+Clair&title=Redox+paradox%3A+A+novel+approach+to+therapeutics-resistant+cancer&doi=10.1089%2Fars.2017.7485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Redox Paradox: A Novel Approach to Therapeutics-Resistant Cancer</span></div><div class="casAuthors">Chaiswing, Luksana; St. Clair, William H.; St. Clair, Daret K.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1237-1272</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Significance: Cancer cells that are resistant to radiation and chemotherapy are a major problem limiting the success of cancer therapy.  Aggressive cancer cells depend on elevated intracellular levels of reactive oxygen species (ROS) to proliferate, self-renew, and metastasize.  As a result, these aggressive cancers maintain high basal levels of ROS compared with normal cells.  The prominence of the redox state in cancer cells led us to consider whether increasing the redox state to the condition of oxidative stress could be used as a successful adjuvant therapy for aggressive cancers.  Recent Advances: Past attempts using antioxidant compds. to inhibit ROS levels in cancers as redox-based therapy have met with very limited success.  However, recent clin. trials using pro-oxidant compds. reveal noteworthy results, which could have a significant impact on the development of strategies for redox-based therapies.  Crit. Issues: The major objective of this review is to discuss the role of the redox state in aggressive cancers and how to utilize the shift in redox state to improve cancer therapy.  We also discuss the paradox of redox state parameters; i.e., hydrogen peroxide (H2O2) as the driver mol. for cancer progression as well as a target for cancer treatment.  Future directions: Based on the biol. significance of the redox state, we postulate that this system could potentially be used to create a new avenue for targeted therapy, including the potential to incorporate personalized redox therapy for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIB99mI-1zo7Vg90H21EOLACvtfcHk0lgyO2PxgEP_Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslKqtrfF&md5=47cda6fe364af1fd99425569dea0ec89</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1089%2Fars.2017.7485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2017.7485%26sid%3Dliteratum%253Aachs%26aulast%3DChaiswing%26aufirst%3DL.%26aulast%3DClair%26aufirst%3DW.%2BH.%2BS.%26aulast%3DClair%26aufirst%3DD.%2BK.%2BS.%26atitle%3DRedox%2520paradox%253A%2520A%2520novel%2520approach%2520to%2520therapeutics-resistant%2520cancer%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2018%26volume%3D29%26spage%3D1237%26epage%3D1272%26doi%3D10.1089%2Fars.2017.7485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulte, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagozdzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manevich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beeson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, C. A.</span></span> <span> </span><span class="NLM_article-title">Prdx1 inhibits tumorigenesis via regulating PTEN/AKT activity</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1505</span>– <span class="NLM_lpage">1517</span>, <span class="refDoi"> DOI: 10.1038/emboj.2009.101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Femboj.2009.101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=19369943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksFemurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=1505-1517&author=J.+Caoauthor=J.+Schulteauthor=A.+Knightauthor=N.+R.+Leslieauthor=A.+Zagozdzonauthor=R.+Bronsonauthor=Y.+Manevichauthor=C.+Beesonauthor=C.+A.+Neumann&title=Prdx1+inhibits+tumorigenesis+via+regulating+PTEN%2FAKT+activity&doi=10.1038%2Femboj.2009.101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Prdx1 inhibits tumorigenesis via regulating PTEN/AKT activity</span></div><div class="casAuthors">Cao, Juxiang; Schulte, Jennifer; Knight, Alexander; Leslie, Nicholas R.; Zagozdzon, Agnieszka; Bronson, Roderick; Manevich, Yefim; Beeson, Craig; Neumann, Carola A.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1505-1517</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">It is widely accepted that reactive oxygen species (ROS) promote tumorigenesis.  However, the exact mechanisms are still unclear.  As mice lacking the peroxidase peroxiredoxin1 (Prdx1) produce more cellular ROS and die prematurely of cancer, they offer an ideal model system to study ROS-induced tumorigenesis.  Prdx1 ablation increased the susceptibility to Ras-induced breast cancer.  We, therefore, investigated the role of Prdx1 in regulating oncogenic Ras effector pathways.  We found Akt hyperactive in fibroblasts and mammary epithelial cells lacking Prdx1.  Investigating the nature of such elevated Akt activation established a novel role for Prdx1 as a safeguard for the lipid phosphatase activity of PTEN, which is essential for its tumor suppressive function.  We found binding of the peroxidase Prdx1 to PTEN essential for protecting PTEN from oxidn.-induced inactivation.  Along those lines, Prdx1 tumor suppression of Ras- or ErbB-2-induced transformation was mediated mainly via PTEN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhIZ-cdT0k4rVg90H21EOLACvtfcHk0lgyO2PxgEP_Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksFemurY%253D&md5=5c13edd53866c75fca0a9ad75f392bc4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Femboj.2009.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2009.101%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DSchulte%26aufirst%3DJ.%26aulast%3DKnight%26aufirst%3DA.%26aulast%3DLeslie%26aufirst%3DN.%2BR.%26aulast%3DZagozdzon%26aufirst%3DA.%26aulast%3DBronson%26aufirst%3DR.%26aulast%3DManevich%26aufirst%3DY.%26aulast%3DBeeson%26aufirst%3DC.%26aulast%3DNeumann%26aufirst%3DC.%2BA.%26atitle%3DPrdx1%2520inhibits%2520tumorigenesis%2520via%2520regulating%2520PTEN%252FAKT%2520activity%26jtitle%3DEMBO%2520J.%26date%3D2009%26volume%3D28%26spage%3D1505%26epage%3D1517%26doi%3D10.1038%2Femboj.2009.101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delcuve, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, J. R.</span></span> <span> </span><span class="NLM_article-title">Selective association of peroxiredoxin 1 with genomic DNA and COX-2 upstream promoter elements in estrogen receptor negative breast cancer cells</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2987</span>– <span class="NLM_lpage">2995</span>, <span class="refDoi"> DOI: 10.1091/mbc.e10-02-0160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1091%2Fmbc.e10-02-0160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=20631257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGlt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=2987-2995&author=X.+Wangauthor=S.+Heauthor=J.-M.+Sunauthor=G.+P.+Delcuveauthor=J.+R.+Davie&title=Selective+association+of+peroxiredoxin+1+with+genomic+DNA+and+COX-2+upstream+promoter+elements+in+estrogen+receptor+negative+breast+cancer+cells&doi=10.1091%2Fmbc.e10-02-0160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective association of peroxiredoxin 1 with genomic DNA and COX-2 upstream promoter elements in estrogen receptor negative breast cancer cells</span></div><div class="casAuthors">Wang, Xuemei; He, Shihua; Sun, Jian-Min; Delcuve, Genevieve P.; Davie, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2987-2995</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">In a search for proteins differentially cross-linked to DNA by cisplatin or formaldehyde in normal breast epithelial and breast cancer cell lines, we identified peroxiredoxin 1 (PRDX1) as a protein preferentially cross-linked to DNA in estrogen receptor neg. (ER-) MDA-MB-231 but not in estrogen receptor pos. (ER+) MCF7 breast cancer cells.  Indirect immunofluorescence microscopic analyses showed that PRDX1 was located in the cytoplasm and nucleus of normal and breast cancer cells, with nuclear PRDX1 assocd. with promyelocytic leukemia protein bodies.  We demonstrated that PRDX1 assocn.- with the transcription factor nuclear factor-κB (NF-κB) in MDA-MB-231 but not in MCF7 cells contributed to PRDX1-selective recruitment to MDA-MB-231 genomic DNA.  Furthermore, PRDX1 was assocd. with the cyclooxygenase (COX)-2 upstream promoter region at sites occupied by NF-κB in ER- but not in ER+ breast cancer cells.  PRDX1 knockdown attenuated COX-2 expression by reducing NF-κB occupancy at its upstream promoter element in MDA-MB-231 but not in MCF7 cells.  A phosphorylated form of PRDX1 was only present in ER- breast cancer cells.  Because PRDX1 phosphorylation is known to inhibit its peroxidase activity and to promote PRDX1 oligomerization, we propose that PRDX1 acts as a chaperone to enhance the transactivation potential of NF-κB in ER- breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyD3iLaeWTrrVg90H21EOLACvtfcHk0li_PFMAeMIqkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGlt7vK&md5=d127e1d53bdaa5260af697bb989232b3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e10-02-0160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e10-02-0160%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DJ.-M.%26aulast%3DDelcuve%26aufirst%3DG.%2BP.%26aulast%3DDavie%26aufirst%3DJ.%2BR.%26atitle%3DSelective%2520association%2520of%2520peroxiredoxin%25201%2520with%2520genomic%2520DNA%2520and%2520COX-2%2520upstream%2520promoter%2520elements%2520in%2520estrogen%2520receptor%2520negative%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2010%26volume%3D21%26spage%3D2987%26epage%3D2995%26doi%3D10.1091%2Fmbc.e10-02-0160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alkam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semenza, G. L.</span></span> <span> </span><span class="NLM_article-title">PRDX2 and PRDX4 are negative regulators of hypoxia-inducible factors under conditions of prolonged hypoxia</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">6379</span>– <span class="NLM_lpage">6397</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.7142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.18632%2Foncotarget.7142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=26837221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A280%3ADC%252BC28nmvVKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=6379-6397&author=W.+Luoauthor=I.+Chenauthor=Y.+Chenauthor=D.+Alkamauthor=Y.+Wangauthor=G.+L.+Semenza&title=PRDX2+and+PRDX4+are+negative+regulators+of+hypoxia-inducible+factors+under+conditions+of+prolonged+hypoxia&doi=10.18632%2Foncotarget.7142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">PRDX2 and PRDX4 are negative regulators of hypoxia-inducible factors under conditions of prolonged hypoxia</span></div><div class="casAuthors">Luo Weibo; Chen Ivan; Semenza Gregg L; Luo Weibo; Semenza Gregg L; Luo Weibo; Chen Yan; Alkam Duah; Wang Yingfei; Luo Weibo; Wang Yingfei; Semenza Gregg L; Semenza Gregg L; Semenza Gregg L; Semenza Gregg L; Semenza Gregg L</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">6379-97</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hypoxia-inducible factors (HIFs) control the transcription of genes that are crucial for the pathogenesis of cancer and other human diseases.  The transcriptional activity of HIFs is rapidly increased upon exposure to hypoxia, but expression of some HIF target genes decreases during prolonged hypoxia.  However, the underlying mechanism for feedback inhibition is not completely understood.  Here, we report that peroxiredoxin 2 (PRDX2) and PRDX4 interact with HIF-1α and HIF-2α in vitro and in hypoxic HeLa cells.  Prolonged hypoxia increases the nuclear translocation of PRDX2 and PRDX4.  As a result, PRDX2 and PRDX4 impair HIF-1 and HIF-2 binding to the hypoxia response elements of a subset of HIF target genes, thereby inhibiting gene transcription in cells exposed to prolonged hypoxia.  PRDX2 and PRDX4 have no effect on the recruitment of p300 and RNA polymerase II to HIF target genes and the enzymatic activity of PRDX2 and PRDX4 is not required for inhibition of HIF-1 and HIF-2.  We also demonstrate that PRDX2 is a direct HIF target gene and that PRDX2 expression is induced by prolonged hypoxia.  These findings uncover a novel feedback mechanism for inhibition of HIF transcriptional activity under conditions of prolonged hypoxia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyKKcecMBANoYPsCkJIw00fW6udTcc2eYskpdRDoBkTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nmvVKjsA%253D%253D&md5=c21855c99d2d56991ede091f2915b4fc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7142%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DAlkam%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26atitle%3DPRDX2%2520and%2520PRDX4%2520are%2520negative%2520regulators%2520of%2520hypoxia-inducible%2520factors%2520under%2520conditions%2520of%2520prolonged%2520hypoxia%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D6379%26epage%3D6397%26doi%3D10.18632%2Foncotarget.7142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Min, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-Y.</span></span> <span> </span><span class="NLM_article-title">Inhibition of TRAF6 ubiquitin-ligase activity by PRDX1 leads to inhibition of NFKB activation and autophagy activation</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1358</span>, <span class="refDoi"> DOI: 10.1080/15548627.2018.1474995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1080%2F15548627.2018.1474995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29929436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=1347-1358&author=Y.+Minauthor=M.-J.+Kimauthor=S.+Leeauthor=E.+Chunauthor=K.-Y.+Lee&title=Inhibition+of+TRAF6+ubiquitin-ligase+activity+by+PRDX1+leads+to+inhibition+of+NFKB+activation+and+autophagy+activation&doi=10.1080%2F15548627.2018.1474995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of TRAF6 ubiquitin-ligase activity by PRDX1 leads to inhibition of NFKB activation and autophagy activation</span></div><div class="casAuthors">Min, Yoon; Kim, Mi-Jeong; Lee, Sena; Chun, Eunyoung; Lee, Ki-Young</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1347-1358</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">TRAF6 (TNF receptor assocd. factor 6) plays a pivotal role in NFKB activation and macroautphagy/autophagy activation induced by TLR4 (toll like receptor 4) signaling.  The objective of this study was to det. the functional role of PRDX1 (peroxiredoxin 1) in NFKB activation and autophagy activation.  PRDX1 interacted with the ring finger domain of TRAF6 and inhibited its ubiquitin-ligase activity.  The inhibition on TRAF6 ubiquitin-ligase activity by PRDX1 induced the suppression of ubiquitination of an evolutionarily conserved signaling intermediate in Toll pathways (ECSIT) essential for NFKB activation and BECN1 (beclin 1) required for autophagy activation.  An inhibitory effect of PRDX1 on TRAF6 was clearly evidenced in PRDX1-knockdown (PRDX1KD) THP-1, PRDX1KD MDA-MB-231, and PRDX1KD SK-HEP-1 cells.  PRDX1KD THP-1 cells showed increases of NFKB activation, pro-inflammatory cytokine prodn., NFKB-dependent gene expression induced by TLR4 stimulation, and resistance against Salmonella typhimurium infection.  Addnl., migration and invasion abilities of PRDX1KD MDA-MB-231 and PRDX1KD SK-HEP-1 cancer cells were significantly enhanced compared to those of control cancer cells.  Taken together, these results suggest that PRDX1 neg. regulates TLR4 signaling for NFKB activation and autophagy functions such as bactericidal activity, cancer cell migration, and cancer cell invasion by inhibiting TRAF6 ubiquitin-ligase activity. 3-MA: 3-methyladenine; BECN1: beclin 1; CHUK/IKKA: conserved helix-loop-helix ubiquitous kinase; ECSIT: ECSIT signalling integrator; ELISA: ELISA; NFKB: nuclear factor kappa-light-chain-enhancer of activated B cells; IB: immunoblotting; IKBKB/IKKB: inhibitor of nuclear factor kappa B kinase subunit beta; IL1B: interleukin 1 beta; IL6: interleukin 6; IP: immunopptn.; LPS: lipopolysaccharide; MAP1LC3/LC3: microtuble assocd. protein 1 light chain 3; MAP3K7/TAK1: mitogen-activated protein kinase kinase kinase 7; MAPK14/p38: mitogen-activated protein kinase 14; mROS: mitochondrial reactive oxygen species; PRDX1: peroxiredoxin 1; PRDX6: peroxiredoxin 6; RELA/p65: RELA proto-oncogene, NF-kB subunit; TRAF6 TNF: receptor assocd. factor 6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGpT9XSrqB0bVg90H21EOLACvtfcHk0liTUSes8886UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsrrJ&md5=b0f42cec5eddab01270cd5990c3b5ce3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1080%2F15548627.2018.1474995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2018.1474995%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DM.-J.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DChun%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DK.-Y.%26atitle%3DInhibition%2520of%2520TRAF6%2520ubiquitin-ligase%2520activity%2520by%2520PRDX1%2520leads%2520to%2520inhibition%2520of%2520NFKB%2520activation%2520and%2520autophagy%2520activation%26jtitle%3DAutophagy%26date%3D2018%26volume%3D14%26spage%3D1347%26epage%3D1358%26doi%3D10.1080%2F15548627.2018.1474995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwahara, S.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, Y.</span></span> <span> </span><span class="NLM_article-title">Dimer-Oligomer Interconversion of Wild-type and Mutant Rat 2-Cys Peroxiredoxin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1074/jbc.m705753200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1074%2Fjbc.m705753200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=17974571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=284-293&author=T.+Matsumuraauthor=K.+Okamotoauthor=S.-i.+Iwaharaauthor=H.+Horiauthor=Y.+Takahashiauthor=T.+Nishinoauthor=Y.+Abe&title=Dimer-Oligomer+Interconversion+of+Wild-type+and+Mutant+Rat+2-Cys+Peroxiredoxin&doi=10.1074%2Fjbc.m705753200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dimer-Oligomer Interconversion of Wild-type and Mutant Rat 2-Cys Peroxiredoxin. Disulfide Formation at Dimer-Dimer Interfaces is not Essential for Decamerization</span></div><div class="casAuthors">Matsumura, Tomohiro; Okamoto, Ken; Iwahara, Shin-ichiro; Hori, Hiroyuki; Takahashi, Yuriko; Nishino, Takeshi; Abe, Yasuko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">284-293</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Rat heme-binding protein 23 (HBP23)/peroxiredoxin (Prx I) belongs to the 2-Cys peroxiredoxin type I family and exhibits peroxidase activity coupled with reduced thioredoxin (Trx) as an electron donor.  We analyzed the dimer-oligomer interconversion of wild-type and mutant HBP23/Prx I by gel filtration and found that the C52S and C173S mutants existed mostly as decamers, whereas the wild type was a mixt. of various forms, favoring the decamer at higher protein concn. and lower ionic salt concn. and in the presence of dithiothreitol.  The C83S mutant was predominantly dimeric, in agreement with a previous crystallog. anal.  X-ray diffraction anal. of the decameric C52S mutant revealed a toroidal structure (diam., ∼130Å; inside diam., ∼55Å; thickness, ∼45Å).  In contrast to human Prx I, which was recently reported to exist predominantly as the decamer with Cys83-Cys83 disulfide bonds at all dimer-dimer interfaces, rat HBP23/Prx I has a Cys83-Cys83 disulfide bond at only one dimer-dimer interface (S-S sepn. of ∼2.1Å), whereas the interactions at the other interfaces (mean S-S sepn. of 3.6Å) appear to involve hydrophobic and van der Waals forces.  This finding is consistent with gel filtration analyses showing that the protein readily interconverts between dimer and oligomeric forms.  The C83S mutant exhibited similar peroxidase activity to the wild type, which is exclusively dimeric, in the Trx/Trx reductase system.  At higher concns., where the protein was mostly decameric, less efficient attack of reduced Trx was obsd. in a [14C]iodoacetamide incorporation expt.  We suggest that the dimer-decamer interconversion may have a regulatory role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6sGarZpmgOrVg90H21EOLACvtfcHk0lgxB5AEXCY0rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOksw%253D%253D&md5=cbf6946890e2ec363b77f5c2191b1efc</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m705753200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m705753200%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumura%26aufirst%3DT.%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DIwahara%26aufirst%3DS.-i.%26aulast%3DHori%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DY.%26aulast%3DNishino%26aufirst%3DT.%26aulast%3DAbe%26aufirst%3DY.%26atitle%3DDimer-Oligomer%2520Interconversion%2520of%2520Wild-type%2520and%2520Mutant%2520Rat%25202-Cys%2520Peroxiredoxin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D284%26epage%3D293%26doi%3D10.1074%2Fjbc.m705753200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.-M.</span></span> <span> </span><span class="NLM_article-title">Human Peroxiredoxin 1 and 2 Are Not Duplicate Proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">22011</span>– <span class="NLM_lpage">22022</span>, <span class="refDoi"> DOI: 10.1074/jbc.m610330200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1074%2Fjbc.m610330200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=17519234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVGls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=22011-22022&author=W.+Leeauthor=K.-S.+Choiauthor=J.+Riddellauthor=J.+Riddellauthor=C.+Ipauthor=D.+Ghoshauthor=J.-H.+Parkauthor=Y.-M.+Park&title=Human+Peroxiredoxin+1+and+2+Are+Not+Duplicate+Proteins&doi=10.1074%2Fjbc.m610330200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Human Peroxiredoxin 1 and 2 Are Not Duplicate Proteins: the unique presence of Cys83 in Prx1 underscores the structural and functional differences between Prx1 and Prx2</span></div><div class="casAuthors">Lee, WeonSup; Choi, Kyoung-Soo; Riddell, Jonah; Ip, Clement; Ghosh, Debashis; Park, Jong-Hoon; Park, Young-Mee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">22011-22022</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human peroxiredoxins 1 and 2, also known as Prx1 and Prx2, are more than 90% homologous in their amino acid sequences.  Prx1 and Prx2 are elevated in various cancers and are shown to influence diverse cellular processes.  Although their growth regulatory role has traditionally been attributed to the peroxidase activity, the physiol. significance of this function is unclear because the proteins are highly susceptible to inactivation by H2O2.  A chaperone activity appears to emerge when their peroxidase activity is lost.  Structural studies suggest that they may form a homodimer or doughnut-shaped homodecamer.  However, little information is available whether human Prx1 and Prx2 are duplicative in structure and function.  We noted that Prx1 contains a cysteine (Cys83) at the putative dimer-dimer interface, which is absent in Prx2.  We studied the role of Cys83 in regulating the peroxidase and chaperone activities of Prx1, because the redox status of Cys83 might influence the oligomeric structure and consequently the functions of Prx1.  We show that Prx1 is more efficient as a mol. chaperone, whereas Prx2 is better suited as a peroxidase enzyme.  Substituting Cys83 with Ser83 (Prx1C83S) results in dramatic changes in the structural and functional characteristics of Prx1 in a direction similar to those of Prx2.  Here we also report the first crystal structure of human Prx1 and the presence of the Cys83-Cys83 bond at the dimer-dimer interface of decameric Prx1.  These findings are consistent with the hypothesis that human Prx1 and Prx2 possess unique functions and regulatory mechanisms and that Cys83 bestows a distinctive identity to Prx1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlM2Il1g6njrVg90H21EOLACvtfcHk0liIZirtr7xHfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVGls7c%253D&md5=a69c123c9f2b59eb2ce521880b15d2e6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m610330200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m610330200%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DW.%26aulast%3DChoi%26aufirst%3DK.-S.%26aulast%3DRiddell%26aufirst%3DJ.%26aulast%3DRiddell%26aufirst%3DJ.%26aulast%3DIp%26aufirst%3DC.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DPark%26aufirst%3DJ.-H.%26aulast%3DPark%26aufirst%3DY.-M.%26atitle%3DHuman%2520Peroxiredoxin%25201%2520and%25202%2520Are%2520Not%2520Duplicate%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D22011%26epage%3D22022%26doi%3D10.1074%2Fjbc.m610330200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, D.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and anticancer properties of isocorydine derivatives</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">6542</span>– <span class="NLM_lpage">6553</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1016%2Fj.bmc.2017.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29103873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslKqs7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=6542-6553&author=Q.+Yanauthor=R.+Liauthor=A.+Xinauthor=Y.+Hanauthor=Y.+Zhangauthor=J.+Liuauthor=W.+Liauthor=D.+Di&title=Design%2C+synthesis%2C+and+anticancer+properties+of+isocorydine+derivatives&doi=10.1016%2Fj.bmc.2017.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and anticancer properties of isocorydine derivatives</span></div><div class="casAuthors">Yan, Qian; Li, Ruxia; Xin, Aiyi; Han, Yin; Zhang, Yanxia; Liu, Junxi; Li, Wenguang; Di, Duolong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6542-6553</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Isocorydine (ICD), an aporphine alkaloid, is widely distributed in nature.  Its ability to target side population (SP) cells found in human hepatocellular carcinoma (HCC) makes it and its deriv. 8-amino-isocorydine (NICD) promising chemotherapeutic agents for the treatment of HCC.  To improve the anticancer activity of isocorydine derivs., twenty derivs. of NICD were designed and synthesized through chem. structure modifications of the arom. amino group at C-8.  The anti-proliferative activities of all synthesized compds. against human hepatocellular (HepG2), cervical (HeLa), and gastric (MGC-803) cancer cell lines were evaluated using an MTT assay.  The results showed that all the synthetic compds. had some tumor cell growth inhibitory activity.  The compd. COM33 (24) was the most active with IC50 values under 10 μM (IC50 for HepG2 = 7.51 μM; IC50 for HeLa = 6.32 μM).  FICD (12) and COM33 (24) were selected for further investigation of their in vitro and in vivo activities due to their relatively good antiproliferative properties.  These two compds. significantly downregulated the expression of four key proteins (C-Myc, β-Catenin, CylinD1, and Ki67) in HepG2 cells.  The tumor inhibition rate of COM33 (24) in vivo was 73.8% after a dose 100 mg/kg via i.p. injection and the combined inhibition rate of COM33 (24) (50 mg/kg) with sorafenib (50 mg/kg) was 66.5%.  The results indicated that these isocorydine derivs. could potentially be used as targeted chemotherapy agents or could be further developed in combination with conventional chemotherapy drugs to target cancer stem cells (CSCs) and epithelial-to-mesenchymal transition (EMT), the main therapeutic targets in HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl2wlHCQQ_2rVg90H21EOLACvtfcHk0liIZirtr7xHfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslKqs7bM&md5=79d4ab790e2ba063b3978aa446e47e09</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DXin%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DDi%26aufirst%3DD.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520anticancer%2520properties%2520of%2520isocorydine%2520derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D6542%26epage%3D6553%26doi%3D10.1016%2Fj.bmc.2017.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroiss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nechitaylo, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svatoš, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltenpoth, M.</span></span> <span> </span><span class="NLM_article-title">Evolutionary stability of antibiotic protection in a defensive symbiosis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>, <span class="NLM_elocation-id">E2020</span> <span class="refDoi"> DOI: 10.1073/pnas.1719797115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1073%2Fpnas.1719797115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29444867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVanu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&author=T.+Englauthor=J.+Kroissauthor=M.+Kaiauthor=T.+Y.+Nechitayloauthor=A.+Svato%C5%A1author=M.+Kaltenpoth&title=Evolutionary+stability+of+antibiotic+protection+in+a+defensive+symbiosis&doi=10.1073%2Fpnas.1719797115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutionary stability of antibiotic protection in a defensive symbiosis</span></div><div class="casAuthors">Engl, Tobias; Kroiss, Johannes; Kai, Marco; Nechitaylo, Taras Y.; SvatoA, AleA; Kaltenpoth, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">E2020-E2029</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The increasing resistance of human pathogens severely limits the efficacy of antibiotics in medicine, yet many animals, including solitary beewolf wasps, successfully engage in defensive alliances with antibiotic-producing bacteria for millions of years.  Here, we report on the in situ prodn. of 49 derivs. belonging to three antibiotic compd. classes (45 piericidin derivs., 3 streptochlorin derivs., and nigericin) by the symbionts of 25 beewolf host species and subspecies, spanning 68 million years of evolution.  Despite a high degree of qual. stability in the antibiotic mixt., we found consistent quant. differences between species and across geog. localities, presumably reflecting adaptations to combat local pathogen communities.  Antimicrobial bioassays with the three main components and in silico predictions based on the structure and specificity in polyketide synthase domains of the piericidin biosynthesis gene cluster yield insights into the mechanistic basis and ecoevolutionary implications of producing a complex mixt. of antimicrobial compds. in a natural setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYpNrANw7F97Vg90H21EOLACvtfcHk0lg4iBE-uwUzEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVanu7vN&md5=b80e3912236bc65ea797a8b5f7b76054</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1719797115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1719797115%26sid%3Dliteratum%253Aachs%26aulast%3DEngl%26aufirst%3DT.%26aulast%3DKroiss%26aufirst%3DJ.%26aulast%3DKai%26aufirst%3DM.%26aulast%3DNechitaylo%26aufirst%3DT.%2BY.%26aulast%3DSvato%25C5%25A1%26aufirst%3DA.%26aulast%3DKaltenpoth%26aufirst%3DM.%26atitle%3DEvolutionary%2520stability%2520of%2520antibiotic%2520protection%2520in%2520a%2520defensive%2520symbiosis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2018%26volume%3D115%26doi%3D10.1073%2Fpnas.1719797115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reczek, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandel, N. S.</span></span> <span> </span><span class="NLM_article-title">ROS promotes cancer cell survival through calcium signaling</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">951</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1016%2Fj.ccell.2018.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29894695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlu7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=949-951&author=C.+R.+Reczekauthor=N.+S.+Chandel&title=ROS+promotes+cancer+cell+survival+through+calcium+signaling&doi=10.1016%2Fj.ccell.2018.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">ROS Promotes Cancer Cell Survival through Calcium Signaling</span></div><div class="casAuthors">Reczek, Colleen R.; Chandel, Navdeep S.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">949-951</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">To avoid reactive oxygen species (ROS)-induced cell death, cancer cells increase their antioxidant defense system.  In this issue of Cancer Cell, Takahashi et al. identify a novel, non-canonical oxidative stress defense mechanism involving TRPA1, a redox-sensitive Ca2+ channel, and the upregulation of anti-apoptotic pathways to promote cancer cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyer3tLFxEWrVg90H21EOLACvtfcHk0lgXsGomfQd7mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlu7%252FL&md5=8cc6a3ec6b3fdc48a95ce613acb2aa1c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DReczek%26aufirst%3DC.%2BR.%26aulast%3DChandel%26aufirst%3DN.%2BS.%26atitle%3DROS%2520promotes%2520cancer%2520cell%2520survival%2520through%2520calcium%2520signaling%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D949%26epage%3D951%26doi%3D10.1016%2Fj.ccell.2018.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forshaw, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmila, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowther, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furdui, C. M.</span></span> <span> </span><span class="NLM_article-title">Peroxiredoxins in cancer and response to radiation therapies</span>. <i>Antioxidants</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">11</span>, <span class="refDoi"> DOI: 10.3390/antiox8010011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.3390%2Fantiox8010011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs12rur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=11&author=T.+E.+Forshawauthor=R.+Holmilaauthor=K.+J.+Nelsonauthor=J.+E.+Lewisauthor=M.+L.+Kempauthor=A.+W.+Tsangauthor=L.+B.+Pooleauthor=W.+T.+Lowtherauthor=C.+M.+Furdui&title=Peroxiredoxins+in+cancer+and+response+to+radiation+therapies&doi=10.3390%2Fantiox8010011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Peroxiredoxins in cancer and response to radiation therapies</span></div><div class="casAuthors">Forshaw, Tom E.; Holmila, Reetta; Nelson, Kimberly J.; Lewis, Joshua E.; Kemp, Melissa L.; Tsang, Allen W.; Poole, Leslie B.; Lowther, W. Todd; Furdui, Cristina M.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11/1-11/24</span>CODEN:
                <span class="NLM_cas:coden">ANTIGE</span>;
        ISSN:<span class="NLM_cas:issn">2076-3921</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Peroxiredoxins have a long-established cellular function as regulators of redox metab. by catalyzing the redn. of peroxides (e.g., H2O2, lipid peroxides) with high catalytic efficiency.  This activity is also crit. to the initiation and relay of both phosphorylation and redox signaling in a broad range of pathophysiol. contexts.  Under normal physiol. conditions, peroxiredoxins protect normal cells from oxidative damage that could promote oncogenesis (e.g., environmental stressors).  In cancer, higher expression level of peroxiredoxins has been assocd. with both tumor growth and resistance to radiation therapies.  However, this relationship between the expression of peroxiredoxins and the response to radiation is not evident from an anal. of data in The Cancer Genome Atlas (TCGA) or NCI60 panel of cancer cell lines.  The focus of this review is to summarize the current exptl. knowledge implicating this class of proteins in cancer, and to provide a perspective on the value of targeting peroxiredoxins in the management of cancer.  Potential biases in the anal. of the TCGA data with respect to radiation resistance are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEETkUGLFnq7Vg90H21EOLACvtfcHk0lgXsGomfQd7mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs12rur4%253D&md5=3e4d0af62e112f5d1b919caebecb810f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fantiox8010011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fantiox8010011%26sid%3Dliteratum%253Aachs%26aulast%3DForshaw%26aufirst%3DT.%2BE.%26aulast%3DHolmila%26aufirst%3DR.%26aulast%3DNelson%26aufirst%3DK.%2BJ.%26aulast%3DLewis%26aufirst%3DJ.%2BE.%26aulast%3DKemp%26aufirst%3DM.%2BL.%26aulast%3DTsang%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DL.%2BB.%26aulast%3DLowther%26aufirst%3DW.%2BT.%26aulast%3DFurdui%26aufirst%3DC.%2BM.%26atitle%3DPeroxiredoxins%2520in%2520cancer%2520and%2520response%2520to%2520radiation%2520therapies%26jtitle%3DAntioxidants%26date%3D2019%26volume%3D8%26spage%3D11%26doi%3D10.3390%2Fantiox8010011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evens, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prachand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paniaqua, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gartenhaus, R. B.</span></span> <span> </span><span class="NLM_article-title">Imexon-induced apoptosis in multiple myeloma tumor cells is caspase-8 dependent</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1481</span>– <span class="NLM_lpage">1491</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-1058-03</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1158%2F1078-0432.ccr-1058-03" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=14977852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsVGjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1481-1491&author=A.+M.+Evensauthor=S.+Prachandauthor=B.+Shiauthor=M.+Paniaquaauthor=L.+I.+Gordonauthor=R.+B.+Gartenhaus&title=Imexon-induced+apoptosis+in+multiple+myeloma+tumor+cells+is+caspase-8+dependent&doi=10.1158%2F1078-0432.ccr-1058-03"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Imexon-induced apoptosis in multiple myeloma tumor cells is caspase-8 dependent</span></div><div class="casAuthors">Evens, Andrew M.; Prachand, Sheila; Shi, Bo; Paniaqua, Mary; Gordon, Leo I.; Gartenhaus, Ronald B.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1481-1491</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Imexon is a 2-cyanoaziridine agent that has been shown to inhibit growth of chemotherapy-sensitive myeloma cells through apoptosis with decreased cellular stores of glutathione and increased reactive oxygen species (ROS).  We examd. the mechanism of imexon cytotoxicity in a diverse panel of dexamethasone and chemotherapy-sensitive and -resistant myeloma cell lines.  We examd. cellular cytotoxicity, apoptosis, and changes in redox state in dexamethasone-sensitive (C2E3), dexamethasone-resistant (1-310 and 1-414), chemotherapy-sensitive (RPMI-8226), and chemotherapy-resistant (DOX-1V and DOX-10V) myeloma cell lines.  We found significant cytotoxicity after 48-h incubation with imexon (80-160 μM) in dexamethasone and chemotherapy-sensitive and -resistant myeloma cell lines in a time- and dose-dependent manner.  The mechanism of imexon cytotoxicity in all cell lines was related to induction of apoptosis with the presence of cleaved caspase-3.  Moreover, after imexon exposure in C2E3 and 1-414 cell lines, we demonstrated caspase-8-dependent apoptosis.  Bcl-2:bax was proapoptotic with imexon in C2E3, whereas bcl-2:bax was independent of steroid resistance, chemotherapy sensitivity, and chemotherapy resistance.  Depletion of intracellular glutathione was documented in RPMI-8226 at high imexon concns. (≥225 μM) but not in other cell lines.  Furthermore, ROS were found in C2E3, RPMI-8226, and 1-310 only at high imexon concns., whereas a sensitive marker of oxidative DNA damage, 8-hydroxydeoxyguanosine, was not increased in any cell line.  Our results demonstrate that imexon has significant broad antimyeloma activity that is mediated through apoptotic mechanisms that is not dependent on prodn. of ROS.  Moreover, we have identified a mechanism of cytotoxicity in dexamethasone-sensitive and -resistant myeloma cells induced by imexon that is caspase-8 dependent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHkuRSvZWB-LVg90H21EOLACvtfcHk0ljw8xCfOW6qsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsVGjtro%253D&md5=29af89d82b6335aab0310bbf4133f992</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-1058-03&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-1058-03%26sid%3Dliteratum%253Aachs%26aulast%3DEvens%26aufirst%3DA.%2BM.%26aulast%3DPrachand%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DB.%26aulast%3DPaniaqua%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DL.%2BI.%26aulast%3DGartenhaus%26aufirst%3DR.%2BB.%26atitle%3DImexon-induced%2520apoptosis%2520in%2520multiple%2520myeloma%2520tumor%2520cells%2520is%2520caspase-8%2520dependent%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2004%26volume%3D10%26spage%3D1481%26epage%3D1491%26doi%3D10.1158%2F1078-0432.ccr-1058-03" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span> <span> </span><span class="NLM_article-title">Nitrobenzoyl sesquiterpenoids with cytotoxic activities from a marine-derived Aspergillus ochraceus fungus</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">92</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.7b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.7b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFSisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=92-97&author=Y.+Tanauthor=B.+Yangauthor=X.+Linauthor=X.+Luoauthor=X.+Pangauthor=L.+Tangauthor=Y.+Liuauthor=X.+Liauthor=X.+Zhou&title=Nitrobenzoyl+sesquiterpenoids+with+cytotoxic+activities+from+a+marine-derived+Aspergillus+ochraceus+fungus&doi=10.1021%2Facs.jnatprod.7b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Nitrobenzoyl sesquiterpenoids with cytotoxic activities from a marine-derived Aspergillus ochraceus fungus</span></div><div class="casAuthors">Tan, Yanhui; Yang, Bin; Lin, Xiuping; Luo, Xiaowei; Pang, Xiaoyan; Tang, Lan; Liu, Yonghong; Li, Xiaojuan; Zhou, Xuefeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">92-97</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Nitrobenzoyl sesquiterpenoids are rare from natural sources.  Two new nitrobenzoyl sesquiterpenoids, insulicolide B (1) and insulicolide C (3), and the new natural product 14-O-acetylinsulicolide A (2) were isolated from culture exts. of the marine-derived fungus Aspergillus ochraceus Jcma1F17, together with three known nitrobenzoyl sesquiterpenoids (4-6) and a deriv. sesquiterpenoid (7).  The structures of the new compds., including their abs. configurations, were detd. by NMR and MS spectroscopic data analyses and comparison between the calcd. and exptl. ECD spectra.  The nitrobenzoyl sesquiterpenoids (1-6) were evaluated for their cytotoxicities against three renal carcinoma cell lines, ACHN, OS-RC-2, and 786-O cells, and compds. 2, 4, and 5 displayed activities with IC50 values of 0.89 to 8.2 μM.  Further studies indicated that 2 arrested the cell cycle at the G0/G1 phase at a concn. of 1 μM and induced late apoptosis at a concn. of 2 μM after a 72 h treatment of 786-O cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGLpU9yG7CrbVg90H21EOLACvtfcHk0ljnXN-OFMJQeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFSisw%253D%253D&md5=cff823de551af3679bb0b7b2c34534ef</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.7b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.7b00698%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DPang%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DX.%26atitle%3DNitrobenzoyl%2520sesquiterpenoids%2520with%2520cytotoxic%2520activities%2520from%2520a%2520marine-derived%2520Aspergillus%2520ochraceus%2520fungus%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2018%26volume%3D81%26spage%3D92%26epage%3D97%26doi%3D10.1021%2Facs.jnatprod.7b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nador, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhasz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korosi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Székács, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horvath, R.</span></span> <span> </span><span class="NLM_article-title">Incubator proof miniaturized Holomonitor toin situmonitor cancer cells exposed to green tea polyphenol and preosteoblast cells adhering on nanostructured titanate surfaces: validity of the measured parameters and their corrections</span>. <i>J. Biomed. Opt.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">067002</span>, <span class="refDoi"> DOI: 10.1117/1.jbo.20.6.067002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1117%2F1.jbo.20.6.067002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=26057033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A280%3ADC%252BC2MbitFaktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=067002&author=B.+Peterauthor=J.+Nadorauthor=K.+Juhaszauthor=A.+Dobosauthor=L.+Korosiauthor=I.+Sz%C3%A9k%C3%A1csauthor=D.+Patkoauthor=R.+Horvath&title=Incubator+proof+miniaturized+Holomonitor+toin+situmonitor+cancer+cells+exposed+to+green+tea+polyphenol+and+preosteoblast+cells+adhering+on+nanostructured+titanate+surfaces%3A+validity+of+the+measured+parameters+and+their+corrections&doi=10.1117%2F1.jbo.20.6.067002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Incubator proof miniaturized Holomonitor to in situ monitor cancer cells exposed to green tea polyphenol and preosteoblast cells adhering on nanostructured titanate surfaces: validity of the measured parameters and their corrections</span></div><div class="casAuthors">Peter Beatrix; Nador Judit; Juhasz Krisztina; Patko Daniel; Horvath Robert; Dobos Agnes; Szekacs Inna; Korosi Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomedical optics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">067002</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The in situ observation of cell movements and morphological parameters over longer periods of time under physiological conditions is critical in basic cell research and biomedical applications.  The quantitative phase-contrast microscope applied in this study has a remarkably small size, therefore it can be placed directly into a humidified incubator.  Here, we report on the successful application of this M4 Holomonitor to observe cancer cell motility, motility speed, and migration in the presence of the green tea polyphenol, epigallocatechin gallate, as well as to monitor the adhesion of preosteoblast cells on nanostructured titanate coatings, relevant for biomedical applications.  A special mechanical stage was developed to position the sample into that range of the optical arrangement where digital autofocusing works with high reproducibility and precision.  By in-depth analyzing the obtained single cell morphological parameters, we show that the limited vertical resolution of the optical setup results in underestimated single cell contact area and volume and overestimated single cell averaged thickness.  We propose a simple model to correct the recorded data to obtain more precise single cell parameters.  We compare the results with the kinetic data recorded by a surface sensitive optical biosensor, optical waveguide lightmode spectroscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTPnAsi6YzzQsRAV4D_Ins_fW6udTcc2eaiTT9zcfKFmbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbitFaktA%253D%253D&md5=7cbdf278ce6cba250d2759c3f85a8087</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1117%2F1.jbo.20.6.067002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1117%252F1.jbo.20.6.067002%26sid%3Dliteratum%253Aachs%26aulast%3DPeter%26aufirst%3DB.%26aulast%3DNador%26aufirst%3DJ.%26aulast%3DJuhasz%26aufirst%3DK.%26aulast%3DDobos%26aufirst%3DA.%26aulast%3DKorosi%26aufirst%3DL.%26aulast%3DSz%25C3%25A9k%25C3%25A1cs%26aufirst%3DI.%26aulast%3DPatko%26aufirst%3DD.%26aulast%3DHorvath%26aufirst%3DR.%26atitle%3DIncubator%2520proof%2520miniaturized%2520Holomonitor%2520toin%2520situmonitor%2520cancer%2520cells%2520exposed%2520to%2520green%2520tea%2520polyphenol%2520and%2520preosteoblast%2520cells%2520adhering%2520on%2520nanostructured%2520titanate%2520surfaces%253A%2520validity%2520of%2520the%2520measured%2520parameters%2520and%2520their%2520corrections%26jtitle%3DJ.%2520Biomed.%2520Opt.%26date%3D2015%26volume%3D20%26spage%3D067002%26doi%3D10.1117%2F1.jbo.20.6.067002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span> <span> </span><span class="NLM_article-title">Spirostaphylotrichin X from a marine-derived fungus as an anti-influenza agent targeting RNA polymerase PB2</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">2722</span>– <span class="NLM_lpage">2730</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.8b00656</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.8b00656" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVelsb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=2722-2730&author=J.+Wangauthor=F.+Chenauthor=Y.+Liuauthor=Y.+Liuauthor=K.+Liauthor=X.+Yangauthor=S.+Liuauthor=X.+Zhouauthor=J.+Yang&title=Spirostaphylotrichin+X+from+a+marine-derived+fungus+as+an+anti-influenza+agent+targeting+RNA+polymerase+PB2&doi=10.1021%2Facs.jnatprod.8b00656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Spirostaphylotrichin X from a Marine-Derived Fungus as an Anti-influenza Agent Targeting RNA Polymerase PB2</span></div><div class="casAuthors">Wang, Jianjiao; Chen, Feimin; Liu, Yunhao; Liu, Yuxuan; Li, Kunlong; Yang, Xiliang; Liu, Shuwen; Zhou, Xuefeng; Yang, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2722-2730</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">A new spirocyclic γ-lactam, named spirostaphylotrichin X (I), and three related known spirostaphylotrichins (2-4) were isolated from the marine-derived fungus Cochliobolus lunatus SCSIO41401.  Their structures were detd. by spectroscopic analyses.  Spirostaphylotrichin X displayed obvious inhibitory activities against multiple influenza virus strains, with IC50 values from 1.2 to 5.5 μM.  Investigation of the mechanism showed that I inhibited viral polymerase activity and interfered with the prodn. of progeny viral RNA.  Homogeneous time-resolved fluorescence, surface plasmon resonance assays, and a mol. docking study revealed that I could inhibit polymerase PB2 protein activity by binding to the highly conserved region of the cap-binding domain of PB2.  These results suggest that I inhibits the replication of influenza A virus by interfering with the activity of PB2 protein and that I represents a new type of potential lead compd. for the development of anti-influenza therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdkyOxXO-X5bVg90H21EOLACvtfcHk0ljoiYHiI41D-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVelsb%252FP&md5=b3e8ba9f8c6196d0c22d483a8125daee</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.8b00656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.8b00656%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DJ.%26atitle%3DSpirostaphylotrichin%2520X%2520from%2520a%2520marine-derived%2520fungus%2520as%2520an%2520anti-influenza%2520agent%2520targeting%2520RNA%2520polymerase%2520PB2%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2018%26volume%3D81%26spage%3D2722%26epage%3D2730%26doi%3D10.1021%2Facs.jnatprod.8b00656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 18 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhi Liang, Yulian Chen, Tanwei Gu, Jianglian She, Fahong Dai, Huanguo Jiang, Zhikun Zhan, Kunlong Li, Yonghong Liu, Xuefeng Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Lan Tang</span>. </span><span class="cited-content_cbyCitation_article-title">LXR-Mediated Regulation of Marine-Derived Piericidins Aggravates High-Cholesterol Diet-Induced Cholesterol Metabolism Disorder in Mice. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 9943-9959. <a href="https://doi.org/10.1021/acs.jmedchem.1c00175" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00175</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00175%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLXR-Mediated%252BRegulation%252Bof%252BMarine-Derived%252BPiericidins%252BAggravates%252BHigh-Cholesterol%252BDiet-Induced%252BCholesterol%252BMetabolism%252BDisorder%252Bin%252BMice%26aulast%3DLiang%26aufirst%3DZhi%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D31012021%26date%3D12072021%26volume%3D64%26issue%3D14%26spage%3D9943%26epage%3D9959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jinhui Wang, Weiyang Jin, Xiaoxin Zhou, Jiaqi Li, Chengdong Xu, Zhongjun Ma, Jianan Wang, Lele Qin, Biao Zhou, Wanjing Ding, Tingting Gao, Hangping Yao, <span class="NLM_string-name hlFld-ContribAuthor">Zhe Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Identification, Structure–Activity Relationships of Marine-Derived Indolocarbazoles, and a Dual PKCθ/δ Inhibitor with Potent Antipancreatic Cancer Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12978-12991. <a href="https://doi.org/10.1021/acs.jmedchem.0c01271" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01271</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01271%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%25252C%252BStructure%2525E2%252580%252593Activity%252BRelationships%252Bof%252BMarine-Derived%252BIndolocarbazoles%25252C%252Band%252Ba%252BDual%252BPKC%2525CE%2525B8%25252F%2525CE%2525B4%252BInhibitor%252Bwith%252BPotent%252BAntipancreatic%252BCancer%252BEfficacy%26aulast%3DWang%26aufirst%3DJinhui%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11082020%26date%3D25102020%26volume%3D63%26issue%3D21%26spage%3D12978%26epage%3D12991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kunlong Li, Zhi Liang, Weihao Chen, Xiaowei Luo, Wei Fang, Shengrong Liao, Xiuping Lin, Bin Yang, Junfeng Wang, Lan Tang, Yonghong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Xuefeng Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Iakyricidins A–D, Antiproliferative Piericidin Analogues Bearing a Carbonyl Group or Cyclic Skeleton from Streptomyces iakyrus SCSIO NS104. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (19)
                                     , 12626-12631. <a href="https://doi.org/10.1021/acs.joc.9b01270" title="DOI URL">https://doi.org/10.1021/acs.joc.9b01270</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b01270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b01270%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DIakyricidins%252BA%2525E2%252580%252593D%25252C%252BAntiproliferative%252BPiericidin%252BAnalogues%252BBearing%252Ba%252BCarbonyl%252BGroup%252Bor%252BCyclic%252BSkeleton%252Bfrom%252BStreptomyces%252Biakyrus%252BSCSIO%252BNS104%26aulast%3DLi%26aufirst%3DKunlong%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12052019%26date%3D07082019%26date%3D26072019%26volume%3D84%26issue%3D19%26spage%3D12626%26epage%3D12631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kunlong  Li</span>, <span class="hlFld-ContribAuthor ">Ziqi  Su</span>, <span class="hlFld-ContribAuthor ">Yongli  Gao</span>, <span class="hlFld-ContribAuthor ">Xiuping  Lin</span>, <span class="hlFld-ContribAuthor ">Xiaoyan  Pang</span>, <span class="hlFld-ContribAuthor ">Bin  Yang</span>, <span class="hlFld-ContribAuthor ">Huaming  Tao</span>, <span class="hlFld-ContribAuthor ">Xiaowei  Luo</span>, <span class="hlFld-ContribAuthor ">Yonghong  Liu</span>, <span class="hlFld-ContribAuthor ">Xuefeng  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Cytotoxic Minor Piericidin Derivatives from the Actinomycete Strain Streptomyces psammoticus SCSIO NS126. </span><span class="cited-content_cbyCitation_journal-name">Marine Drugs</span><span> <strong>2021,</strong> <em>19 </em>
                                    (8)
                                     , 428. <a href="https://doi.org/10.3390/md19080428" title="DOI URL">https://doi.org/10.3390/md19080428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/md19080428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmd19080428%26sid%3Dliteratum%253Aachs%26jtitle%3DMarine%2520Drugs%26atitle%3DCytotoxic%252BMinor%252BPiericidin%252BDerivatives%252Bfrom%252Bthe%252BActinomycete%252BStrain%252BStreptomyces%2525C2%2525A0psammoticus%252BSCSIO%252BNS126%26aulast%3DLi%26aufirst%3DKunlong%26date%3D2021%26date%3D2021%26volume%3D19%26issue%3D8%26spage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi  Liang</span>, <span class="hlFld-ContribAuthor ">Tanwei  Gu</span>, <span class="hlFld-ContribAuthor ">Jianjiao  Wang</span>, <span class="hlFld-ContribAuthor ">Jianglian  She</span>, <span class="hlFld-ContribAuthor ">Yuxiu  Ye</span>, <span class="hlFld-ContribAuthor ">Wenyu  Cao</span>, <span class="hlFld-ContribAuthor ">Xiaowei  Luo</span>, <span class="hlFld-ContribAuthor ">Jiao  Xiao</span>, <span class="hlFld-ContribAuthor ">Yonghong  Liu</span>, <span class="hlFld-ContribAuthor ">Lan  Tang</span>, <span class="hlFld-ContribAuthor ">Xuefeng  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Chromene and chromone derivatives as liver X receptors modulators from a marine-derived Pestalotiopsis neglecta fungus. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>112 </em>, 104927. <a href="https://doi.org/10.1016/j.bioorg.2021.104927" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104927</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104927%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DChromene%252Band%252Bchromone%252Bderivatives%252Bas%252Bliver%2525C2%2525A0X%2525C2%2525A0receptors%252Bmodulators%252Bfrom%252Ba%252Bmarine-derived%252BPestalotiopsis%252Bneglecta%252Bfungus%26aulast%3DLiang%26aufirst%3DZhi%26date%3D2021%26volume%3D112%26spage%3D104927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olívia  Pontes</span>, <span class="hlFld-ContribAuthor ">Sofia  Oliveira-Pinto</span>, <span class="hlFld-ContribAuthor ">Fátima  Baltazar</span>, <span class="hlFld-ContribAuthor ">Marta  Costa</span>. </span><span class="cited-content_cbyCitation_article-title">Renal cell carcinoma therapy: Current and new drug candidates. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>75 </em><a href="https://doi.org/10.1016/j.drudis.2021.07.009" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.07.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.07.009%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DRenal%252Bcell%252Bcarcinoma%252Btherapy%25253A%252BCurrent%252Band%252Bnew%252Bdrug%252Bcandidates%26aulast%3DPontes%26aufirst%3DOl%25C3%25ADvia%26date%3D2021%26volume%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zengzhi  Liu</span>, <span class="hlFld-ContribAuthor ">Fei  Xiao</span>, <span class="hlFld-ContribAuthor ">Siqi  Cai</span>, <span class="hlFld-ContribAuthor ">Chunni  Liu</span>, <span class="hlFld-ContribAuthor ">Huayue  Li</span>, <span class="hlFld-ContribAuthor ">Ting  Wu</span>, <span class="hlFld-ContribAuthor ">Yuechen  Jiang</span>, <span class="hlFld-ContribAuthor ">Xin  Wang</span>, <span class="hlFld-ContribAuthor ">Qian  Che</span>, <span class="hlFld-ContribAuthor ">Tianjiao  Zhu</span>, <span class="hlFld-ContribAuthor ">Dehai  Li</span>, <span class="hlFld-ContribAuthor ">Wenli  Li</span>, . </span><span class="cited-content_cbyCitation_article-title">Effective Generation of Glucosylpiericidins with Selective Cytotoxicities and Insights into Their Biosynthesis. </span><span class="cited-content_cbyCitation_journal-name">Applied and Environmental Microbiology</span><span> <strong>2021,</strong> <em>87 </em>
                                    (13)
                                     <a href="https://doi.org/10.1128/AEM.00294-21" title="DOI URL">https://doi.org/10.1128/AEM.00294-21</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/AEM.00294-21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FAEM.00294-21%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520and%2520Environmental%2520Microbiology%26atitle%3DEffective%252BGeneration%252Bof%252BGlucosylpiericidins%252Bwith%252BSelective%252BCytotoxicities%252Band%252BInsights%252Binto%252BTheir%252BBiosynthesis%26aulast%3DLiu%26aufirst%3DZengzhi%26date%3D2021%26volume%3D87%26issue%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mikhail V.  Pugachev</span>, <span class="hlFld-ContribAuthor ">Maria N.  Agafonova</span>, <span class="hlFld-ContribAuthor ">Oksana A.  Bastrikova</span>, <span class="hlFld-ContribAuthor ">Oleg I.  Gnezdilov</span>, <span class="hlFld-ContribAuthor ">Tatyana V.  Nikishova</span>, <span class="hlFld-ContribAuthor ">Konstantin V.  Balakin</span>, <span class="hlFld-ContribAuthor ">Yurii G.  Shtyrlin</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and antitumor activity of novel pyridoxine-based structural analogs of saccharumoside-B. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2021,</strong> <em>30 </em>
                                    (5)
                                     , 1139-1150. <a href="https://doi.org/10.1007/s00044-021-02719-4" title="DOI URL">https://doi.org/10.1007/s00044-021-02719-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-021-02719-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-021-02719-4%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DSynthesis%252Band%252Bantitumor%252Bactivity%252Bof%252Bnovel%252Bpyridoxine-based%252Bstructural%252Banalogs%252Bof%252Bsaccharumoside-B%26aulast%3DPugachev%26aufirst%3DMikhail%2BV.%26date%3D2021%26date%3D2021%26volume%3D30%26issue%3D5%26spage%3D1139%26epage%3D1150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Peng</span>, <span class="hlFld-ContribAuthor ">Qingbo  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Jiang</span>, <span class="hlFld-ContribAuthor ">Liang  Ma</span>, <span class="hlFld-ContribAuthor ">Ting  Long</span>, <span class="hlFld-ContribAuthor ">Ziqian  Cheng</span>, <span class="hlFld-ContribAuthor ">Changsheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Yiguang  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">New piericidin derivatives from the marine-derived
              streptomyces
              sp. SCSIO 40063 with cytotoxic activity. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Research</span><span> <strong>2021,</strong> <em> </em>, 1-7. <a href="https://doi.org/10.1080/14786419.2021.1901699" title="DOI URL">https://doi.org/10.1080/14786419.2021.1901699</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14786419.2021.1901699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14786419.2021.1901699%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Research%26atitle%3DNew%252Bpiericidin%252Bderivatives%252Bfrom%252Bthe%252Bmarine-derived%252Bstreptomyces%252Bsp.%252BSCSIO%252B40063%252Bwith%252Bcytotoxic%252Bactivity%26aulast%3DPeng%26aufirst%3DJing%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony R.  Carroll</span>, <span class="hlFld-ContribAuthor ">Brent R.  Copp</span>, <span class="hlFld-ContribAuthor ">Rohan A.  Davis</span>, <span class="hlFld-ContribAuthor ">Robert A.  Keyzers</span>, <span class="hlFld-ContribAuthor ">Michèle R.  Prinsep</span>. </span><span class="cited-content_cbyCitation_article-title">Marine natural products. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2021,</strong> <em>38 </em>
                                    (2)
                                     , 362-413. <a href="https://doi.org/10.1039/D0NP00089B" title="DOI URL">https://doi.org/10.1039/D0NP00089B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NP00089B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NP00089B%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DMarine%252Bnatural%252Bproducts%26aulast%3DCarroll%26aufirst%3DAnthony%2BR.%26date%3D2021%26date%3D2021%26volume%3D38%26issue%3D2%26spage%3D362%26epage%3D413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kun-Long  Li</span>, <span class="hlFld-ContribAuthor ">Yu  Dai</span>, <span class="hlFld-ContribAuthor ">Jiang-Lian  She</span>, <span class="hlFld-ContribAuthor ">Yan-Bo  Zeng</span>, <span class="hlFld-ContribAuthor ">Hao-Fu  Dai</span>, <span class="hlFld-ContribAuthor ">Shun-Ling  Ou</span>, <span class="hlFld-ContribAuthor ">Xue-Feng  Zhou</span>, <span class="hlFld-ContribAuthor ">Yong-Hong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Bisabolanoic acid A, a new polychiral sesquiterpene with AChE inhibitory activity from a mangrove-derived fungus
              Colletotrichum
              sp.. </span><span class="cited-content_cbyCitation_journal-name">Journal of Asian Natural Products Research</span><span> <strong>2021,</strong> <em>8 </em>, 1-8. <a href="https://doi.org/10.1080/10286020.2021.1873297" title="DOI URL">https://doi.org/10.1080/10286020.2021.1873297</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10286020.2021.1873297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10286020.2021.1873297%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Asian%2520Natural%2520Products%2520Research%26atitle%3DBisabolanoic%252Bacid%252BA%25252C%252Ba%252Bnew%252Bpolychiral%252Bsesquiterpene%252Bwith%252BAChE%252Binhibitory%252Bactivity%252Bfrom%252Ba%252Bmangrove-derived%252Bfungus%252BColletotrichum%252Bsp.%26aulast%3DLi%26aufirst%3DKun-Long%26date%3D2021%26date%3D2021%26volume%3D8%26spage%3D1%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liang  Ma</span>, <span class="hlFld-ContribAuthor ">Wenjun  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhiwen  Liu</span>, <span class="hlFld-ContribAuthor ">Yanbing  Huang</span>, <span class="hlFld-ContribAuthor ">Qingbo  Zhang</span>, <span class="hlFld-ContribAuthor ">Xinpeng  Tian</span>, <span class="hlFld-ContribAuthor ">Changsheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Yiguang  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Complete genome sequence of Streptomyces sp. SCSIO 03032 isolated from Indian Ocean sediment, producing diverse bioactive natural products. </span><span class="cited-content_cbyCitation_journal-name">Marine Genomics</span><span> <strong>2021,</strong> <em>55 </em>, 100803. <a href="https://doi.org/10.1016/j.margen.2020.100803" title="DOI URL">https://doi.org/10.1016/j.margen.2020.100803</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.margen.2020.100803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.margen.2020.100803%26sid%3Dliteratum%253Aachs%26jtitle%3DMarine%2520Genomics%26atitle%3DComplete%252Bgenome%252Bsequence%252Bof%252BStreptomyces%252Bsp.%252BSCSIO%252B03032%252Bisolated%252Bfrom%252BIndian%252BOcean%252Bsediment%25252C%252Bproducing%252Bdiverse%252Bbioactive%252Bnatural%252Bproducts%26aulast%3DMa%26aufirst%3DLiang%26date%3D2021%26volume%3D55%26spage%3D100803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Diego V.  Wilke</span>, <span class="hlFld-ContribAuthor ">Paula C.  Jimenez</span>, <span class="hlFld-ContribAuthor ">Paola C.  Branco</span>, <span class="hlFld-ContribAuthor ">Paula  Rezende-Teixeira</span>, <span class="hlFld-ContribAuthor ">Amaro E.  Trindade-Silva</span>, <span class="hlFld-ContribAuthor ">Anelize  Bauermeister</span>, <span class="hlFld-ContribAuthor ">Norberto Peporine  Lopes</span>, <span class="hlFld-ContribAuthor ">Leticia V.  Costa-Lotufo</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer Potential of Compounds from the Brazilian Blue Amazon. </span><span class="cited-content_cbyCitation_journal-name">Planta Medica</span><span> <strong>2021,</strong> <em>87 </em>
                                    (01/02)
                                     , 49-70. <a href="https://doi.org/10.1055/a-1257-8402" title="DOI URL">https://doi.org/10.1055/a-1257-8402</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1055/a-1257-8402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1055%2Fa-1257-8402%26sid%3Dliteratum%253Aachs%26jtitle%3DPlanta%2520Medica%26atitle%3DAnticancer%252BPotential%252Bof%252BCompounds%252Bfrom%252Bthe%252BBrazilian%252BBlue%252BAmazon%26aulast%3DWilke%26aufirst%3DDiego%2BV.%26date%3D2021%26date%3D2020%26volume%3D87%26issue%3D01%252F02%26spage%3D49%26epage%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Dai</span>, <span class="hlFld-ContribAuthor ">Kunlong  Li</span>, <span class="hlFld-ContribAuthor ">Jianglian  She</span>, <span class="hlFld-ContribAuthor ">Yanbo  Zeng</span>, <span class="hlFld-ContribAuthor ">Hao  Wang</span>, <span class="hlFld-ContribAuthor ">Shengrong  Liao</span>, <span class="hlFld-ContribAuthor ">Xiuping  Lin</span>, <span class="hlFld-ContribAuthor ">Bin  Yang</span>, <span class="hlFld-ContribAuthor ">Junfeng  Wang</span>, <span class="hlFld-ContribAuthor ">Huaming  Tao</span>, <span class="hlFld-ContribAuthor ">Haofu  Dai</span>, <span class="hlFld-ContribAuthor ">Xuefeng  Zhou</span>, <span class="hlFld-ContribAuthor ">Yonghong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Lipopeptide Epimers and a Phthalide Glycerol Ether with AChE Inhibitory Activities from the Marine-Derived Fungus Cochliobolus Lunatus SCSIO41401. </span><span class="cited-content_cbyCitation_journal-name">Marine Drugs</span><span> <strong>2020,</strong> <em>18 </em>
                                    (11)
                                     , 547. <a href="https://doi.org/10.3390/md18110547" title="DOI URL">https://doi.org/10.3390/md18110547</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/md18110547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmd18110547%26sid%3Dliteratum%253Aachs%26jtitle%3DMarine%2520Drugs%26atitle%3DLipopeptide%252BEpimers%252Band%252Ba%252BPhthalide%252BGlycerol%252BEther%252Bwith%252BAChE%252BInhibitory%252BActivities%252Bfrom%252Bthe%252BMarine-Derived%252BFungus%252BCochliobolus%252BLunatus%252BSCSIO41401%26aulast%3DDai%26aufirst%3DYu%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D11%26spage%3D547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhifei  Che</span>, <span class="hlFld-ContribAuthor ">Jinfeng  Fan</span>, <span class="hlFld-ContribAuthor ">Zhiyan  Zhou</span>, <span class="hlFld-ContribAuthor ">Qi  Li</span>, <span class="hlFld-ContribAuthor ">Zhe  Ma</span>, <span class="hlFld-ContribAuthor ">Zhanhao  Hu</span>, <span class="hlFld-ContribAuthor ">Yaoxi  Wu</span>, <span class="hlFld-ContribAuthor ">Yingxia  Jin</span>, <span class="hlFld-ContribAuthor ">Yang  Su</span>, <span class="hlFld-ContribAuthor ">Peiyu  Liang</span>, <span class="hlFld-ContribAuthor ">Haoyong  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Activation-Induced Cytidine Deaminase Expression Facilitates the Malignant Phenotype and Epithelial-to-Mesenchymal Transition in Clear Cell Renal Cell Carcinoma. </span><span class="cited-content_cbyCitation_journal-name">DNA and Cell Biology</span><span> <strong>2020,</strong> <em>39 </em>
                                    (7)
                                     , 1299-1312. <a href="https://doi.org/10.1089/dna.2019.5119" title="DOI URL">https://doi.org/10.1089/dna.2019.5119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/dna.2019.5119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fdna.2019.5119%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520and%2520Cell%2520Biology%26atitle%3DActivation-Induced%252BCytidine%252BDeaminase%252BExpression%252BFacilitates%252Bthe%252BMalignant%252BPhenotype%252Band%252BEpithelial-to-Mesenchymal%252BTransition%252Bin%252BClear%252BCell%252BRenal%252BCell%252BCarcinoma%26aulast%3DChe%26aufirst%3DZhifei%26date%3D2020%26volume%3D39%26issue%3D7%26spage%3D1299%26epage%3D1312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David E.  Williams</span>, <span class="hlFld-ContribAuthor ">Raymond J.  Andersen</span>. </span><span class="cited-content_cbyCitation_article-title">Biologically active marine natural products and their molecular targets discovered using a chemical genetics approach. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2020,</strong> <em>37 </em>
                                    (5)
                                     , 617-633. <a href="https://doi.org/10.1039/C9NP00054B" title="DOI URL">https://doi.org/10.1039/C9NP00054B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9NP00054B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9NP00054B%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DBiologically%252Bactive%252Bmarine%252Bnatural%252Bproducts%252Band%252Btheir%252Bmolecular%252Btargets%252Bdiscovered%252Busing%252Ba%252Bchemical%252Bgenetics%252Bapproach%26aulast%3DWilliams%26aufirst%3DDavid%2BE.%26date%3D2020%26date%3D2020%26volume%3D37%26issue%3D5%26spage%3D617%26epage%3D633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weiyi  Wang</span>, <span class="hlFld-ContribAuthor ">Hui  Qian</span>, <span class="hlFld-ContribAuthor ">Shengming  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Rh‐Catalyzed Reaction of Propargylic Alcohols with Aryl Boronic Acids–Switch from β‐OH Elimination to Protodemetalation. </span><span class="cited-content_cbyCitation_journal-name">Chinese Journal of Chemistry</span><span> <strong>2020,</strong> <em>38 </em>
                                    (4)
                                     , 331-345. <a href="https://doi.org/10.1002/cjoc.202000044" title="DOI URL">https://doi.org/10.1002/cjoc.202000044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cjoc.202000044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcjoc.202000044%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Journal%2520of%2520Chemistry%26atitle%3DRh%2525E2%252580%252590Catalyzed%252BReaction%252Bof%252BPropargylic%252BAlcohols%252Bwith%252BAryl%252BBoronic%252BAcids%2525E2%252580%252593Switch%252Bfrom%252B%2525CE%2525B2%2525E2%252580%252590OH%252BElimination%252Bto%252BProtodemetalation%26aulast%3DWang%26aufirst%3DWeiyi%26date%3D2020%26volume%3D38%26issue%3D4%26spage%3D331%26epage%3D345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie-Yi  LONG</span>, <span class="hlFld-ContribAuthor ">Jun-Feng  WANG</span>, <span class="hlFld-ContribAuthor ">Sheng-Rong  LIAO</span>, <span class="hlFld-ContribAuthor ">Xiu-Ping  LIN</span>, <span class="hlFld-ContribAuthor ">Xue-Feng  ZHOU</span>, <span class="hlFld-ContribAuthor ">Yun-Qiu  LI</span>, <span class="hlFld-ContribAuthor ">Bin  YANG</span>, <span class="hlFld-ContribAuthor ">Yong-Hong  LIU</span>. </span><span class="cited-content_cbyCitation_article-title">Four new steroids from the marine soft coral-derived fungus Penicillium sp. SCSIO41201. </span><span class="cited-content_cbyCitation_journal-name">Chinese Journal of Natural Medicines</span><span> <strong>2020,</strong> <em>18 </em>
                                    (4)
                                     , 250-255. <a href="https://doi.org/10.1016/S1875-5364(20)30030-3" title="DOI URL">https://doi.org/10.1016/S1875-5364(20)30030-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1875-5364(20)30030-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1875-5364%2820%2930030-3%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Journal%2520of%2520Natural%2520Medicines%26atitle%3DFour%252Bnew%252Bsteroids%252Bfrom%252Bthe%252Bmarine%252Bsoft%252Bcoral-derived%252Bfungus%252BPenicillium%252Bsp.%252BSCSIO41201%26aulast%3DLONG%26aufirst%3DJie-Yi%26date%3D2020%26volume%3D18%26issue%3D4%26spage%3D250%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of the piericidins obtained from the <i>Streptomyces</i> strain SCSIO NS126.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Transcriptome analysis of the gene expression profile in ACHN cells treated with PA. (A) Volcano plot: showed numbers of DEGs (FDR < 0.05 according to DEseq). (B) Intersection of DEGs (the statistical criteria using fold change > 1.5 and FDR < 0.05) affected by PA and those DEGs from pairs of ccRCC tumors and adjacent non-tumor renal tissues from GSE40435 and GSE76207. (C) KEGG pathways enrichment analysis. The bars represent the enrichment scores, −log 10 (<i>P</i> value). (D) PA (25 and 50 nM) and GPA (25 and 50 nM) reduced the activation of key genes in ACHN cells. (E) mRNA level of PRDX1 in ACHN cells after treatment of PA/GPA. (F) Immunoblot analysis of PRDX1 protein of extracts of paired tumour (T) vs adjacent non-tumor (N) samples from 6 human ccRCC patients.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. PA/GPA inhibits cell proliferation and induced cell apoptosis through increasing PRDX1 and reducing ROS in ACHN. (A1) Phase holographic imaging of ACHN cells treated with PA (50 nM) for 72 h. (A2) Annexin V/PI staining and flow cytometry showed the percentages of apoptosis in ACHN cells treated with PA (50 nM). (B) Immunoblot analysis of the PRDX1 protein in the normal ACHN cells and PRDX1-Kd ACHN cells; (C1) Intracellular ROS detection with flow cytometer in the normal ACHN cells and PRDX1-Kd ACHN cells. (C2) Intracellular ROS detection in the normal ACHN cell treated by PA/GPA (25 and 50 nM) in 24 h; (C3) Intracellular ROS detection in the PRDX1-Kd ACHN cell treated by PA/GPA (25 and 50 nM) in 24 h. (D1) Immunoblot analysis of the PRDX1 protein in nuclear and cytosol extracts of ACHN cell treated with PA/GPA (25 and 50 nM). (D2) Fold change of protein expression (PRDX1/β-Actin). (E1) Representative images for immunofluorescent staining of ACHN treated with PA/GPA (25 and 50 nM) after co-incubating for 24 h. PRDX1 (green), nucleus (blue), <i>n</i> = 9. Scale bar = 10 μm. (E2) Mean fluorescence intensity of ACHN cell treated with PA/GPA (25 and 50 nM) in 24 h. All data represent mean ± standard deviation. Statistical significance was determined using Student’s <i>t</i>-test or one-way analysis of variance with Tukey’s post-hoc test. *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.005, ****<i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Molecular docking and SPR analysis of PA/GPA with PRDX1. (A1) Dimer–dimer interface region (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S">2Z9S</a>) was used as the receptor structure. Each chain of monomers is shown in different colors. Chains a, b as a dimer and chains c, d as another dimer. The CYS83–CYS83 disulfide bridge is located between chains a and c. (A2) Binding sites and the details of the predicted binding mode of PA in the dimer–dimer interface of PRDX1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S">2Z9S</a>), 90° rotation from (A1). (A3) Binding sites and the details of the predicted binding mode of GPA in the dimer–dimer interface of PRDX1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S">2Z9S</a>). The contact residues are shown and labeled by type and number. (B) Affinity activities of PA (B1) and GPA (B2) to the PRDX1 protein were analyzed using the SPR assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/medium/jm-2019-005987_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. PA and GPA show antitumor efficacy in nude mice bearing ACHN xenografts. (A) Images of mice bearing ACHN xenografts and injected ip once a day at day 21 post injection. (B) Images of tumor (B1) and progression of body weights (B2) and tumor volumes (B3) in mice during 21 days treatment; <i>n</i> = 4, ±SE. (C) Immunoblot analysis of the PRDX1 protein in tumor tissues after 21 days treatment with PBS, PA, and GPA. (D) TUNEL and HE staining of tumor sections after treatment, respectively. Scale bar, 100 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00598/20190801/images/large/jm-2019-005987_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00598&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72515" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72515" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 30 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shroff, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlin, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouw, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellovin, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philbrick, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Ocana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casey, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zare, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felsher, D. W.</span></span> <span> </span><span class="NLM_article-title">MYC oncogene overexpression drives renal cell carcinoma in a mouse model through glutamine metabolism</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">6539</span>– <span class="NLM_lpage">6544</span>, <span class="refDoi"> DOI: 10.1073/pnas.1507228112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1073%2Fpnas.1507228112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=25964345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFCmtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=6539-6544&author=E.+H.+Shroffauthor=L.+S.+Eberlinauthor=V.+M.+Dangauthor=A.+M.+Gouwauthor=M.+Gabayauthor=S.+J.+Adamauthor=D.+I.+Bellovinauthor=P.+T.+Tranauthor=W.+M.+Philbrickauthor=A.+Garcia-Ocanaauthor=S.+C.+Caseyauthor=Y.+Liauthor=C.+V.+Dangauthor=R.+N.+Zareauthor=D.+W.+Felsher&title=MYC+oncogene+overexpression+drives+renal+cell+carcinoma+in+a+mouse+model+through+glutamine+metabolism&doi=10.1073%2Fpnas.1507228112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MYC oncogene overexpression drives renal cell carcinoma in a mouse model through glutamine metabolism</span></div><div class="casAuthors">Shroff, Emelyn H.; Eberlin, Livia S.; Dang, Vanessa M.; Gouw, Arvin M.; Gabay, Meital; Adam, Stacey J.; Bellovin, David I.; Tran, Phuoc T.; Philbrick, William M.; Garcia-Ocana, Adolfo; Casey, Stephanie C.; Li, Yulin; Dang, Chi V.; Zare, Richard N.; Felsher, Dean W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6539-6544</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The MYC oncogene is frequently mutated and overexpressed in human renal cell carcinoma (RCC).  However, there have been no studies on the causative role of MYC or any other oncogene in the initiation or maintenance of kidney tumorigenesis.  Here, we show through a conditional transgenic mouse model that the MYC oncogene, but not the RAS oncogene, initiates and maintains RCC.  Desorption electrospray ionization-mass-spectrometric imaging was used to obtain chem. maps of metabolites and lipids in the mouse RCC samples.  Gene expression anal. revealed that the mouse tumors mimicked human RCC.  The data suggested that MYC-induced RCC up-regulated the glutaminolytic pathway instead of the glycolytic pathway.  The pharmacol. inhibition of glutamine metab. with bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide impeded MYC-mediated RCC tumor progression.  Our studies demonstrate that MYC overexpression causes RCC and points to the inhibition of glutamine metab. as a potential therapeutic approach for the treatment of this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqokovxXkOTurVg90H21EOLACvtfcHk0lj8CtF8iY9TAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFCmtrw%253D&md5=e063f432422de8a2e485f998cc5b7b8c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1507228112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1507228112%26sid%3Dliteratum%253Aachs%26aulast%3DShroff%26aufirst%3DE.%2BH.%26aulast%3DEberlin%26aufirst%3DL.%2BS.%26aulast%3DDang%26aufirst%3DV.%2BM.%26aulast%3DGouw%26aufirst%3DA.%2BM.%26aulast%3DGabay%26aufirst%3DM.%26aulast%3DAdam%26aufirst%3DS.%2BJ.%26aulast%3DBellovin%26aufirst%3DD.%2BI.%26aulast%3DTran%26aufirst%3DP.%2BT.%26aulast%3DPhilbrick%26aufirst%3DW.%2BM.%26aulast%3DGarcia-Ocana%26aufirst%3DA.%26aulast%3DCasey%26aufirst%3DS.%2BC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DZare%26aufirst%3DR.%2BN.%26aulast%3DFelsher%26aufirst%3DD.%2BW.%26atitle%3DMYC%2520oncogene%2520overexpression%2520drives%2520renal%2520cell%2520carcinoma%2520in%2520a%2520mouse%2520model%2520through%2520glutamine%2520metabolism%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3D6539%26epage%3D6544%26doi%3D10.1073%2Fpnas.1507228112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varella, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rini, B. I.</span></span> <span> </span><span class="NLM_article-title">Emerging drugs for renal cell carcinoma</span>. <i>Expert Opin. Emerging Drugs</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1517/14728214.2010.486788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1517%2F14728214.2010.486788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=20443750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCnurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=343-353&author=L.+Varellaauthor=B.+I.+Rini&title=Emerging+drugs+for+renal+cell+carcinoma&doi=10.1517%2F14728214.2010.486788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging drugs for renal cell carcinoma</span></div><div class="casAuthors">Varella, Leticia; Rini, Brian I.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Emerging Drugs</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">343-353</span>CODEN:
                <span class="NLM_cas:coden">EOEDA3</span>;
        ISSN:<span class="NLM_cas:issn">1472-8214</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: The treatment of metastatic renal cell carcinoma (RCC) has evolved significantly over the past 5 years and targeted therapy has become the std. of care.  However, complete and lasting responses to these drugs are still uncommon.  Currently, novel agents are being tested in clin. trials.Areas covered in this review: We discuss approved targeted agents in RCC and emphasize on emerging therapies.  We searched the US National Library of Medicine (PubMed) using the terms renal cell carcinoma', targeted therapy' and novel agents', from 1990 to the present.  We also searched for abstrs. from the meetings of the American Society of Clin. Oncol. and Genitourinary Cancers Symposium from 2007 to the present.What the reader will gain: This paper provides understanding of the approved treatments for advanced RCC as well as the novel agents and their targeted biol. pathways.Take home message: Despite dynamic and recent advances in the management of metastatic RCC much about the mol. biol. of this tumor has yet to be delineated.  Even more so, strategies of sequential treatment, combination of targeted drugs, mechanisms of resistance, optimal dosages and scheduling remain areas of potential development interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJjrGQcWT-HbVg90H21EOLACvtfcHk0lj6kxicbGKRfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCnurrI&md5=06e8d93a467b9836c1870e80361cec6b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1517%2F14728214.2010.486788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728214.2010.486788%26sid%3Dliteratum%253Aachs%26aulast%3DVarella%26aufirst%3DL.%26aulast%3DRini%26aufirst%3DB.%2BI.%26atitle%3DEmerging%2520drugs%2520for%2520renal%2520cell%2520carcinoma%26jtitle%3DExpert%2520Opin.%2520Emerging%2520Drugs%26date%3D2010%26volume%3D15%26spage%3D343%26epage%3D353%26doi%3D10.1517%2F14728214.2010.486788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C.</span></span> <span> </span><span class="NLM_article-title">Targeted therapy: An elusive cancer target</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>537</i></span>,  <span class="NLM_fpage">S106</span>– <span class="NLM_lpage">S108</span>, <span class="refDoi"> DOI: 10.1038/537s106a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2F537s106a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=27626779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=537&publication_year=2016&pages=S106-S108&author=C.+Brown&title=Targeted+therapy%3A+An+elusive+cancer+target&doi=10.1038%2F537s106a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted therapy: An elusive cancer target</span></div><div class="casAuthors">Brown, Carolyn</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">537</span>
        (<span class="NLM_cas:issue">7620</span>),
    <span class="NLM_cas:pages">S106-S108</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Advanced tumors may have met their match with new drugs, but why have these treatments proved ineffective at stopping early-stage tumors from coming back.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmzsizLsmhwrVg90H21EOLACvtfcHk0lj6kxicbGKRfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiurjM&md5=c6c8f781dc6040098118dfaa6e68b430</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F537s106a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F537s106a%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DC.%26atitle%3DTargeted%2520therapy%253A%2520An%2520elusive%2520cancer%2520target%26jtitle%3DNature%26date%3D2016%26volume%3D537%26spage%3DS106%26epage%3DS108%26doi%3D10.1038%2F537s106a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mehra, R.</span></span> <span> </span><span class="NLM_article-title">An Insight into the “Dark Matter” of Kidney Cancer</span>. <i>Eur. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">766</span>, <span class="refDoi"> DOI: 10.1016/j.eururo.2018.08.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1016%2Fj.eururo.2018.08.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=30177289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ltVeiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=764-766&author=R.+Mehra&title=An+Insight+into+the+%E2%80%9CDark+Matter%E2%80%9D+of+Kidney+Cancer&doi=10.1016%2Fj.eururo.2018.08.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An Insight into the "Dark Matter" of Kidney Cancer</span></div><div class="casAuthors">Mehra Rohit</div><div class="citationInfo"><span class="NLM_cas:title">European urology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">764-766</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTh0SbA2MTWTn6oPH0mYWL1fW6udTcc2ebrpop-byal6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ltVeiuw%253D%253D&md5=da71b5363e6c42c85e7d7b49a331e849</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.eururo.2018.08.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.eururo.2018.08.023%26sid%3Dliteratum%253Aachs%26aulast%3DMehra%26aufirst%3DR.%26atitle%3DAn%2520Insight%2520into%2520the%2520%25E2%2580%259CDark%2520Matter%25E2%2580%259D%2520of%2520Kidney%2520Cancer%26jtitle%3DEur.%2520Urol.%26date%3D2018%26volume%3D74%26spage%3D764%26epage%3D766%26doi%3D10.1016%2Fj.eururo.2018.08.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Cancer
Genome Atlas Research Network</span> <span> </span><span class="NLM_article-title">Comprehensive molecular characterization of clear cell renal cell
carcinoma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>499</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1038/nature12222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Fnature12222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=23792563" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&author=Cancer%0AGenome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell%0Acarcinoma&doi=10.1038%2Fnature12222"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature12222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12222%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%2520molecular%2520characterization%2520of%2520clear%2520cell%2520renal%2520cell%250Acarcinoma%26jtitle%3DNature%26date%3D2013%26volume%3D499%26spage%3D43%26epage%3D49%26doi%3D10.1038%2Fnature12222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenical, W.</span></span> <span> </span><span class="NLM_article-title">The unique chemistry and biology of the piericidins</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">582</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1038/ja.2016.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Fja.2016.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=27301663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWjtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2016&pages=582-593&author=X.+Zhouauthor=W.+Fenical&title=The+unique+chemistry+and+biology+of+the+piericidins&doi=10.1038%2Fja.2016.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The unique chemistry and biology of the piericidins</span></div><div class="casAuthors">Zhou, Xuefeng; Fenical, William</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">582-593</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The piericidin family of microbial metabolites features a 4-pyridinol core linked with a methylated polyketide side chain.  Piericidins are exclusively produced by actinomycetes, esp. members of the genus Streptomyces.  The close structural similarity with coenzyme Q renders the piericidins important NADH-ubiquinone oxidoreductase (complex I) inhibitors in the mitochondrial electron transport chain.  Because of the significant activities of the piericidins, which include insecticidal, antimicrobial and antitumor effects, total syntheses of the piericidins were developed using various synthetic strategies.  The biosynthetic origin of this class has also been the subject of investigation.  This review covers the isolation and structure detn. of the natural piericidins, their chem. modification, the total syntheses of natural and unnatural analogs, their biosynthesis, and reported biol. activities together with structure-activity relationships.  Given the fundamental biol. of this class of metabolites, the piericidin family will likely continue to attract attention as biol. probes of important biosynthetic processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Nh2kFRxqVbVg90H21EOLACvtfcHk0lhmiEHL3VhHOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWjtbfK&md5=f72687698ec9ccb3a067be1a398c5bf4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fja.2016.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2016.71%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DFenical%26aufirst%3DW.%26atitle%3DThe%2520unique%2520chemistry%2520and%2520biology%2520of%2520the%2520piericidins%26jtitle%3DJ.%2520Antibiot.%26date%3D2016%26volume%3D69%26spage%3D582%26epage%3D593%26doi%3D10.1038%2Fja.2016.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span> <span> </span><span class="NLM_article-title">Geranylpyrrol A and Piericidin F from Streptomyces sp. CHQ-64 ΔrdmF</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">1684</span>– <span class="NLM_lpage">1687</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.7b00016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.7b00016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVyqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2017&pages=1684-1687&author=X.+Hanauthor=Z.+Liuauthor=Z.+Zhangauthor=X.+Zhangauthor=T.+Zhuauthor=Q.+Guauthor=W.+Liauthor=Q.+Cheauthor=D.+Li&title=Geranylpyrrol+A+and+Piericidin+F+from+Streptomyces+sp.+CHQ-64+%CE%94rdmF&doi=10.1021%2Facs.jnatprod.7b00016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Geranylpyrrol A and Piericidin F from Streptomyces sp. CHQ-64 ΔrdmF</span></div><div class="casAuthors">Han, Xiaoning; Liu, Zengzhi; Zhang, Zhenzhen; Zhang, Xiaomin; Zhu, Tianjiao; Gu, Qianqun; Li, Wenli; Che, Qian; Li, Dehai</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1684-1687</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Two new compds., geranylpyrrol A (I) and piericidin F (II), were isolated from a reedsmycins nonproducing mutant strain of Streptomyces sp. CHQ-64.  Their structures including abs. configurations, were elucidated by extensive NMR, MS, NOESY and ECD analyses.  Geranylpyrrol A is an unusual naturally occurring 2,3,4-trisubstituted pyrrole and piericidin F showed cytotoxicity against Hela, NB4, A549 and H1975 cell lines with IC50 values of 0.003, 0.037, 0.56, and 0.49 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi73p1d_i8k7Vg90H21EOLACvtfcHk0lhmiEHL3VhHOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVyqtbY%253D&md5=e476f3fe691aa4cdbb30d74b1c748042</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.7b00016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.7b00016%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DGu%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DChe%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DD.%26atitle%3DGeranylpyrrol%2520A%2520and%2520Piericidin%2520F%2520from%2520Streptomyces%2520sp.%2520CHQ-64%2520%25CE%2594rdmF%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2017%26volume%3D80%26spage%3D1684%26epage%3D1687%26doi%3D10.1021%2Facs.jnatprod.7b00016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shang, N.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.-X.</span></span> <span> </span><span class="NLM_article-title">Glycosylated piericidins from an endophytic streptomyces with cytotoxicity and antimicrobial activity</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1038/s41429-018-0051-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Fs41429-018-0051-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29651143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Wmtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2018&pages=672-676&author=N.-N.+Shangauthor=Z.+Zhangauthor=J.-P.+Huangauthor=L.+Wangauthor=J.+Luoauthor=J.+Yangauthor=T.+Pengauthor=Y.+Yanauthor=Y.-T.+Maauthor=S.-X.+Huang&title=Glycosylated+piericidins+from+an+endophytic+streptomyces+with+cytotoxicity+and+antimicrobial+activity&doi=10.1038%2Fs41429-018-0051-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Glycosylated piericidins from an endophytic streptomyces with cytotoxicity and antimicrobial activity</span></div><div class="casAuthors">Shang, Ning-Ning; Zhang, Zhouxin; Huang, Jian-Ping; Wang, Li; Luo, Jianying; Yang, Jing; Peng, Teng; Yan, Yijun; Ma, Ya-Tuan; Huang, Sheng-Xiong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-676</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Two new glycosylated piericidins, glucopiericidinol A3 (1) and 7-demethyl-glucopiericidin A (2), along with four known analogs were isolated from the culture broth of Streptomyces sp. KIB-H1083.  The chem. structures of new compds. were elucidated by spectroscopic analyses.  Their cytotoxicity on HL-60, SMMC-772, A-549, MCF-7, and SW480 cell lines, as well as antimicrobial activities was evaluated.  The results showed that glucopiericidin A (4) has potent cytotoxicity against HL-60, SMMC-772, A-549, and MCF-7 cell lines with IC50 values of 0.34, 0.65, 0.60, and 0.50 μM, resp.  For the antimicrobial activity, piericidin A (6) showed most powerful inhibitory activities against Xanthomonas oryzae pv. oryzicola, and Penicillium decumbens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKVUVFi1kV_bVg90H21EOLACvtfcHk0lhmiEHL3VhHOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Wmtbo%253D&md5=64c4563cc846c0f3b73e7242bb2da127</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41429-018-0051-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41429-018-0051-1%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DN.-N.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DJ.-P.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.-T.%26aulast%3DHuang%26aufirst%3DS.-X.%26atitle%3DGlycosylated%2520piericidins%2520from%2520an%2520endophytic%2520streptomyces%2520with%2520cytotoxicity%2520and%2520antimicrobial%2520activity%26jtitle%3DJ.%2520Antibiot.%26date%3D2018%26volume%3D71%26spage%3D672%26epage%3D676%26doi%3D10.1038%2Fs41429-018-0051-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schnermann, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamaru-Ogiso, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Total synthesis of piericidin A1 and B1 and key analogues</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">11799</span>– <span class="NLM_lpage">11807</span>, <span class="refDoi"> DOI: 10.1021/ja0632862</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0632862" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD28XosVCit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=11799-11807&author=M.+J.+Schnermannauthor=F.+A.+Romeroauthor=I.+Hwangauthor=E.+Nakamaru-Ogisoauthor=T.+Yagiauthor=D.+L.+Boger&title=Total+synthesis+of+piericidin+A1+and+B1+and+key+analogues&doi=10.1021%2Fja0632862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis of Piericidin A1 and B1 and Key Analogues</span></div><div class="casAuthors">Schnermann, Martin J.; Romero, F. Anthony; Hwang, Inkyu; Nakamaru-Ogiso, Eiko; Yagi, Takao; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">11799-11807</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Piericidin A1 and B1 I (R = H, Me) and analogs with modified side chains and pyridine cores (and the enantiomer of piericidin A1) are prepd. enantioselectively; the effect of structural changes on the inhibition of mitochondrial complex I (ubiquinone reductase) and on their cytotoxicities in mouse leukemia cell line L1210 are detd.  The key step in the synthesis of I (R = H, Me) and their analogs is the inverse electron-demand hetero-Diels-Alder reactions of α-(sulfonylimino)-β,γ-unsatd. esters with tetramethoxyethene followed by elimination of the sulfonyl group, a proton, and two methoxy groups in the presence of boron trifluoride etherate to yield functionalized dimethoxypyridinecarboxylates which are oxidized to the piericidin core by a lithiation/boronation/oxidn. sequence.  The side chains of I (R = H, Me) and their nonracemic analogs are prepd. by use of stereoselective aldol addns. of nonracemic propanoyloxazolidinones and by stereoselective Julia couplings of allylic phenyltetrazolyl sulfones with nonracemic aldehydes.  The fully elaborated side chains are prepd. as alkenylstannanes; Stille coupling reactions with bromomethylpyridines in the presence of palladium catalysts generates protected piericidins and their analogs which are converted to the desired products.  While simplified analogs of I inhibit mitochondrial complex I with similar affinities, the cytotoxicities of analogs of I in mouse leukemia cell line L1210 vary with structural changes in the side chain and in the pyridine core,.  The crystal structures of a tetrahydropyridinecarboxylate and of an α-(triisopropylsilyl)pyridinemethanol, intermediates in the prepn. of I (R = H, Me), are detd. by X-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg7FxLd3_uxbVg90H21EOLACvtfcHk0ljGS37HD_RV1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosVCit7w%253D&md5=f18bfee48e8b1136445a28548016e677</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fja0632862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0632862%26sid%3Dliteratum%253Aachs%26aulast%3DSchnermann%26aufirst%3DM.%2BJ.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DHwang%26aufirst%3DI.%26aulast%3DNakamaru-Ogiso%26aufirst%3DE.%26aulast%3DYagi%26aufirst%3DT.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DTotal%2520synthesis%2520of%2520piericidin%2520A1%2520and%2520B1%2520and%2520key%2520analogues%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D11799%26epage%3D11807%26doi%3D10.1021%2Fja0632862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chooi, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, D.</span></span> <span> </span><span class="NLM_article-title">Elucidation of Piericidin A1 Biosynthetic Locus Revealed a Thioesterase-Dependent Mechanism of α-Pyridone Ring Formation</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">243</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1016%2Fj.chembiol.2011.12.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=22365607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWgt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=243-253&author=Q.+Liuauthor=F.+Yaoauthor=Y.+H.+Chooiauthor=Q.+Kangauthor=W.+Xuauthor=Y.+Liauthor=Y.+Shaoauthor=Y.+Shiauthor=Z.+Dengauthor=Y.+Tangauthor=D.+You&title=Elucidation+of+Piericidin+A1+Biosynthetic+Locus+Revealed+a+Thioesterase-Dependent+Mechanism+of+%CE%B1-Pyridone+Ring+Formation&doi=10.1016%2Fj.chembiol.2011.12.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidation of Piericidin A1 Biosynthetic Locus Revealed a Thioesterase-Dependent Mechanism of α-Pyridone Ring Formation</span></div><div class="casAuthors">Liu, Qian; Yao, Fen; Chooi, Yit Heng; Kang, Qianjin; Xu, Wei; Li, Yanran; Shao, Yucheng; Shi, Yuefeng; Deng, Zixin; Tang, Yi; You, Delin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">243-253</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Piericidins are a class of α-pyridone antibiotics that inhibit mitochondrial respiratory chain and exhibit antimicrobial, antifungal, and antitumor activities.  Sequential anal. of Streptomyces piomogenus var. Hangzhouwanensis genome revealed six modular polyketide synthases, an amidotransferase, two methyltransferases, and a monooxygenase for piericidin A1 prodn.  Gene functional anal. and deletion results provide overview of the biosynthesis pathway.  Furthermore, in vitro characterization of the terminal polyketide synthase module with the thioesterase domain using β-ketoacyl substrates was performed.  That revealed a pathway where the α-pyridone ring formation is dependent on hydrolysis of the product β, δ-diketo carboxylic acid by the C-terminal thioesterase followed by amidation and cyclization.  These findings set the stage to investigate unusual enzymic mechanisms in α-pyridone antibiotics biosynthesis, provide a foundation for genome mining of α-pyridone antibiotics, and produce analogs by mol. engineering.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZmRvP-V-_ObVg90H21EOLACvtfcHk0ljGS37HD_RV1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWgt78%253D&md5=3b54f93840529bad2fcd121529a40dbf</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.018%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DYao%26aufirst%3DF.%26aulast%3DChooi%26aufirst%3DY.%2BH.%26aulast%3DKang%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DD.%26atitle%3DElucidation%2520of%2520Piericidin%2520A1%2520Biosynthetic%2520Locus%2520Revealed%2520a%2520Thioesterase-Dependent%2520Mechanism%2520of%2520%25CE%25B1-Pyridone%2520Ring%2520Formation%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D243%26epage%3D253%26doi%3D10.1016%2Fj.chembiol.2011.12.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span> <span> </span><span class="NLM_article-title">Elucidating hydroxylation and methylation steps tailoring piericidin A1 biosynthesis</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1021/ol4034176</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol4034176" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFWiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=736-739&author=Y.+Chenauthor=W.+Zhangauthor=Y.+Zhuauthor=Q.+Zhangauthor=X.+Tianauthor=S.+Zhangauthor=C.+Zhang&title=Elucidating+hydroxylation+and+methylation+steps+tailoring+piericidin+A1+biosynthesis&doi=10.1021%2Fol4034176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidating Hydroxylation and Methylation Steps Tailoring Piericidin A1 Biosynthesis</span></div><div class="casAuthors">Chen, Yaolong; Zhang, Wenjun; Zhu, Yiguang; Zhang, Qingbo; Tian, Xinpeng; Zhang, Si; Zhang, Changsheng</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">736-739</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The piericidin A1 (1) gene cluster was identified from the deep-sea derived Streptomyces sp. SCSIO 03032.  Our in vivo and in vitro expts. verified PieE as a 4'-hydroxylase and PieB2 as a 4'-O-methyltransferase, allowing the elucidation of the post-PKS modification steps involved in biosynthesis of 1.  In addn., the shunt metabolite piericidin E1 (7) was identified as a novel analog featuring a C-2/C-3 epoxy ring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3z39sbcVjqrVg90H21EOLACvtfcHk0ljGS37HD_RV1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFWiuw%253D%253D&md5=9971b699f7510764070b94d0ca93df8d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fol4034176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol4034176%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DC.%26atitle%3DElucidating%2520hydroxylation%2520and%2520methylation%2520steps%2520tailoring%2520piericidin%2520A1%2520biosynthesis%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D736%26epage%3D739%26doi%3D10.1021%2Fol4034176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carman, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubey, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Etten, R. A.</span></span> <span> </span><span class="NLM_article-title">Essential role for the peroxiredoxin Prdx1 in erythrocyte antioxidant defence and tumour suppression</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">565</span>, <span class="refDoi"> DOI: 10.1038/nature01819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Fnature01819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=12891360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvVGis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2003&pages=561-565&author=C.+A.+Neumannauthor=D.+S.+Krauseauthor=C.+V.+Carmanauthor=S.+Dasauthor=D.+P.+Dubeyauthor=J.+L.+Abrahamauthor=R.+T.+Bronsonauthor=Y.+Fujiwaraauthor=S.+H.+Orkinauthor=R.+A.+Van+Etten&title=Essential+role+for+the+peroxiredoxin+Prdx1+in+erythrocyte+antioxidant+defence+and+tumour+suppression&doi=10.1038%2Fnature01819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role for the peroxiredoxin Prdx1 in erythrocyte antioxidant defense and tumor suppression</span></div><div class="casAuthors">Neumann, Carola A.; Krause, Daniela S.; Carman, Christopher V.; Das, Shampa; Dubey, Devendra P.; Abraham, Jennifer L.; Bronson, Roderick T.; Fujiwara, Yuko; Orkin, Stuart H.; Van Etten, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">424</span>
        (<span class="NLM_cas:issue">6948</span>),
    <span class="NLM_cas:pages">561-565</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reactive oxygen species are involved in many cellular metabolic and signaling processes and are thought to have a role in disease, particularly in carcinogenesis and aging.  We have generated mice with targeted inactivation of Prdx1, a member of the peroxiredoxin family of antioxidant enzymes.  Here we show that mice lacking Prdx1 are viable and fertile but have a shortened lifespan owing to the development beginning at about 9 mo of severe hemolytic anemia and several malignant cancers, both of which are also obsd. at increased frequency in heterozygotes.  The hemolytic anemia is characterized by an increase in erythrocyte reactive oxygen species, leading to protein oxidn., Hb instability, Heinz body formation and decreased erythrocyte lifespan.  The malignancies include lymphomas, sarcomas and carcinomas, and are frequently assocd. with loss of Prdx1 expression in heterozygotes, which suggests that this protein functions as a tumor suppressor.  Prdx1-deficient fibroblasts show decreased proliferation and increased sensitivity to oxidative DNA damage, whereas Prdx1-null mice have abnormalities in nos., phenotype and function of natural killer cells.  Our results implicate Prdx1 as an important defense against oxidants in aging mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxSf4jPolWPrVg90H21EOLACvtfcHk0li_JnYIO67BxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvVGis7g%253D&md5=4f49676ed87d35bb42a2c51a907ba8ac</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature01819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01819%26sid%3Dliteratum%253Aachs%26aulast%3DNeumann%26aufirst%3DC.%2BA.%26aulast%3DKrause%26aufirst%3DD.%2BS.%26aulast%3DCarman%26aufirst%3DC.%2BV.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DDubey%26aufirst%3DD.%2BP.%26aulast%3DAbraham%26aufirst%3DJ.%2BL.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DVan%2BEtten%26aufirst%3DR.%2BA.%26atitle%3DEssential%2520role%2520for%2520the%2520peroxiredoxin%2520Prdx1%2520in%2520erythrocyte%2520antioxidant%2520defence%2520and%2520tumour%2520suppression%26jtitle%3DNature%26date%3D2003%26volume%3D424%26spage%3D561%26epage%3D565%26doi%3D10.1038%2Fnature01819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span> <span> </span><span class="NLM_article-title">Peroxiredoxin 1 - an antioxidant enzyme in cancer</span>. <i>J. Cell. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1111/jcmm.12955</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1111%2Fjcmm.12955" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=27653015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis12l" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=193-202&author=C.+Dingauthor=X.+Fanauthor=G.+Wu&title=Peroxiredoxin+1+-+an+antioxidant+enzyme+in+cancer&doi=10.1111%2Fjcmm.12955"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peroxiredoxin 1 - an antioxidant enzyme in cancer</span></div><div class="casAuthors">Ding, Chenbo; Fan, Xiaobo; Wu, Guoqiu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular and Molecular Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">193-202</span>CODEN:
                <span class="NLM_cas:coden">JCMMC9</span>;
        ISSN:<span class="NLM_cas:issn">1582-4934</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Peroxiredoxins (PRDXs), a ubiquitous family of redox-regulating proteins, are reported of potential to eliminate various reactive oxygen species (ROS).  As a major member of the antioxidant enzymes, PRDX1 can become easily over-oxidized on its catalytically active cysteine induced by a variety of stimuli in vitro and in vivo.  In nucleus, oligomeric PRDX1 directly assocs. with p53 or transcription factors such as c-Myc, NF-κB and AR, and thus affects their bioactivities upon gene regulation, which in turn induces or suppresses cell death.  Addnl., PRDX1 in cytoplasm has anti-apoptotic potential through direct or indirect interactions with several ROS-dependent (redox regulation) effectors, including ASK1, p66Shc, GSTpi/JNK and c-Abl kinase.  PRDX1 is proven to be a versatile mol. regulating cell growth, differentiation and apoptosis.  Recent studies have found that PRDX1 and/or PRDX1-regulated ROS-dependent signalling pathways play an important role in the progression and metastasis of human tumors, particularly in breast, oesophageal and lung cancers.  In this paper, we review the structure, effector functions of PRDX1, its role in cancer and the pivotal role of ROS in anticancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRWU1Ys5zsRrVg90H21EOLACvtfcHk0li_JnYIO67BxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis12l&md5=8dd9793091969a8123a923ef4b30a119</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1111%2Fjcmm.12955&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjcmm.12955%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DC.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DG.%26atitle%3DPeroxiredoxin%25201%2520-%2520an%2520antioxidant%2520enzyme%2520in%2520cancer%26jtitle%3DJ.%2520Cell.%2520Mol.%2520Med.%26date%3D2017%26volume%3D21%26spage%3D193%26epage%3D202%26doi%3D10.1111%2Fjcmm.12955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hampton, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vick, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoko, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, C. A.</span></span> <span> </span><span class="NLM_article-title">Peroxiredoxin involvement in the initiation and progression of human cancer</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1089/ars.2017.7422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1089%2Fars.2017.7422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29237274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVOqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=591-608&author=M.+B.+Hamptonauthor=K.+A.+Vickauthor=J.+J.+Skokoauthor=C.+A.+Neumann&title=Peroxiredoxin+involvement+in+the+initiation+and+progression+of+human+cancer&doi=10.1089%2Fars.2017.7422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Peroxiredoxin Involvement in the Initiation and Progression of Human Cancer</span></div><div class="casAuthors">Hampton, Mark B.; Vick, Kate A.; Skoko, John J.; Neumann, Carola A.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">591-608</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Significance: It has been proposed that cancer cells are heavily dependent on their antioxidant defenses for survival and growth.  Peroxiredoxins are a family of abundant thiol-dependent peroxidases that break down hydrogen peroxide, and they have a central role in the maintenance and response of cells to alterations in redox homeostasis.  As such, they are potential targets for disrupting tumor growth.  Recent Advances: Genetic disruption of peroxiredoxin expression in mice leads to an increased incidence of neoplastic disease, consistent with a role for peroxiredoxins in protecting genomic integrity.  In contrast, many human tumors display increased levels of peroxiredoxin expression, suggesting that strengthened antioxidant defenses provide a survival advantage for tumor progression.  Peroxiredoxin inhibitors are being developed and explored as therapeutic agents in different cancer models.  Crit. Issues: It is important to complement peroxiredoxin knockout and expression studies with an improved understanding of the biol. function of the peroxiredoxins.  Although current results can be interpreted within the context that peroxiredoxins scavenge hydroperoxides, some peroxiredoxin family members appear to have more complex roles in regulating the response of cells to oxidative stress through protein interactions with constituents of other signaling pathways.  Future Directions: Further mechanistic information is required for understanding the role of oxidative stress in cancer, the function of peroxiredoxins in normal vs. cancer cells, and for the design and testing of specific peroxiredoxin inhibitors that display selectivity to malignant cells.  Antioxid. Redox Signal. 28, 591-608.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc3-4Dxu0zIbVg90H21EOLACvtfcHk0li_JnYIO67BxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVOqsLc%253D&md5=66472e56b6f808a6edac98b49fa849fd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1089%2Fars.2017.7422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2017.7422%26sid%3Dliteratum%253Aachs%26aulast%3DHampton%26aufirst%3DM.%2BB.%26aulast%3DVick%26aufirst%3DK.%2BA.%26aulast%3DSkoko%26aufirst%3DJ.%2BJ.%26aulast%3DNeumann%26aufirst%3DC.%2BA.%26atitle%3DPeroxiredoxin%2520involvement%2520in%2520the%2520initiation%2520and%2520progression%2520of%2520human%2520cancer%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2018%26volume%3D28%26spage%3D591%26epage%3D608%26doi%3D10.1089%2Fars.2017.7422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perroud, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valkova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhirapong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiehn, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kültz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, R. H.</span></span> <span> </span><span class="NLM_article-title">Pathway analysis of kidney cancer using proteomics and metabolic profiling</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-5-64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1186%2F1476-4598-5-64" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=17123452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A280%3ADC%252BD28npsFejuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=64&author=B.+Perroudauthor=J.+Leeauthor=N.+Valkovaauthor=A.+Dhirapongauthor=P.-Y.+Linauthor=O.+Fiehnauthor=D.+K%C3%BCltzauthor=R.+H.+Weiss&title=Pathway+analysis+of+kidney+cancer+using+proteomics+and+metabolic+profiling&doi=10.1186%2F1476-4598-5-64"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Pathway analysis of kidney cancer using proteomics and metabolic profiling</span></div><div class="casAuthors">Perroud Bertrand; Lee Jinoo; Valkova Nelly; Dhirapong Amy; Lin Pei-Yin; Fiehn Oliver; Kultz Dietmar; Weiss Robert H</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Renal cell carcinoma (RCC) is the sixth leading cause of cancer death and is responsible for 11,000 deaths per year in the US.  Approximately one-third of patients present with disease which is already metastatic and for which there is currently no adequate treatment, and no biofluid screening tests exist for RCC.  In this study, we have undertaken a comprehensive proteomic analysis and subsequently a pathway and network approach to identify biological processes involved in clear cell RCC (ccRCC).  We have used these data to investigate urinary markers of RCC which could be applied to high-risk patients, or to those being followed for recurrence, for early diagnosis and treatment, thereby substantially reducing mortality of this disease.  RESULTS:  Using 2-dimensional electrophoresis and mass spectrometric analysis, we identified 31 proteins which were differentially expressed with a high degree of significance in ccRCC as compared to adjacent non-malignant tissue, and we confirmed some of these by immunoblotting, immunohistochemistry, and comparison to published transcriptomic data.  When evaluated by several pathway and biological process analysis programs, these proteins are demonstrated to be involved with a high degree of confidence (p values < 2.0 E-05) in glycolysis, propanoate metabolism, pyruvate metabolism, urea cycle and arginine/proline metabolism, as well as in the non-metabolic p53 and FAS pathways.  In a pilot study using random urine samples from both ccRCC and control patients, we performed metabolic profiling and found that only sorbitol, a component of an alternative glycolysis pathway, is significantly elevated at 5.4-fold in RCC patients as compared to controls.  CONCLUSION:  Extensive pathway and network analysis allowed for the discovery of highly significant pathways from a set of clear cell RCC samples.  Knowledge of activation of these processes will lead to novel assays identifying their proteomic and/or metabolomic signatures in biofluids of patient at high risk for this disease; we provide pilot data for such a urinary bioassay.  Furthermore, we demonstrate how the knowledge of networks, processes, and pathways altered in kidney cancer may be used to influence the choice of optimal therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTw5QbiB5WoQIJKJbLTgsvCfW6udTcc2ea7puBLbgG0ubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28npsFejuw%253D%253D&md5=8a7afd646d8309932d6cba85053cdfad</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-5-64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-5-64%26sid%3Dliteratum%253Aachs%26aulast%3DPerroud%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DValkova%26aufirst%3DN.%26aulast%3DDhirapong%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DP.-Y.%26aulast%3DFiehn%26aufirst%3DO.%26aulast%3DK%25C3%25BCltz%26aufirst%3DD.%26aulast%3DWeiss%26aufirst%3DR.%2BH.%26atitle%3DPathway%2520analysis%2520of%2520kidney%2520cancer%2520using%2520proteomics%2520and%2520metabolic%2520profiling%26jtitle%3DMol.%2520Cancer%26date%3D2006%26volume%3D5%26spage%3D64%26doi%3D10.1186%2F1476-4598-5-64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaiswing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clair, W. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clair, D. K. S.</span></span> <span> </span><span class="NLM_article-title">Redox paradox: A novel approach to therapeutics-resistant cancer</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1237</span>– <span class="NLM_lpage">1272</span>, <span class="refDoi"> DOI: 10.1089/ars.2017.7485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1089%2Fars.2017.7485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29325444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslKqtrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=1237-1272&author=L.+Chaiswingauthor=W.+H.+S.+Clairauthor=D.+K.+S.+Clair&title=Redox+paradox%3A+A+novel+approach+to+therapeutics-resistant+cancer&doi=10.1089%2Fars.2017.7485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Redox Paradox: A Novel Approach to Therapeutics-Resistant Cancer</span></div><div class="casAuthors">Chaiswing, Luksana; St. Clair, William H.; St. Clair, Daret K.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1237-1272</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Significance: Cancer cells that are resistant to radiation and chemotherapy are a major problem limiting the success of cancer therapy.  Aggressive cancer cells depend on elevated intracellular levels of reactive oxygen species (ROS) to proliferate, self-renew, and metastasize.  As a result, these aggressive cancers maintain high basal levels of ROS compared with normal cells.  The prominence of the redox state in cancer cells led us to consider whether increasing the redox state to the condition of oxidative stress could be used as a successful adjuvant therapy for aggressive cancers.  Recent Advances: Past attempts using antioxidant compds. to inhibit ROS levels in cancers as redox-based therapy have met with very limited success.  However, recent clin. trials using pro-oxidant compds. reveal noteworthy results, which could have a significant impact on the development of strategies for redox-based therapies.  Crit. Issues: The major objective of this review is to discuss the role of the redox state in aggressive cancers and how to utilize the shift in redox state to improve cancer therapy.  We also discuss the paradox of redox state parameters; i.e., hydrogen peroxide (H2O2) as the driver mol. for cancer progression as well as a target for cancer treatment.  Future directions: Based on the biol. significance of the redox state, we postulate that this system could potentially be used to create a new avenue for targeted therapy, including the potential to incorporate personalized redox therapy for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIB99mI-1zo7Vg90H21EOLACvtfcHk0lj3SibKIaK2RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslKqtrfF&md5=47cda6fe364af1fd99425569dea0ec89</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1089%2Fars.2017.7485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2017.7485%26sid%3Dliteratum%253Aachs%26aulast%3DChaiswing%26aufirst%3DL.%26aulast%3DClair%26aufirst%3DW.%2BH.%2BS.%26aulast%3DClair%26aufirst%3DD.%2BK.%2BS.%26atitle%3DRedox%2520paradox%253A%2520A%2520novel%2520approach%2520to%2520therapeutics-resistant%2520cancer%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2018%26volume%3D29%26spage%3D1237%26epage%3D1272%26doi%3D10.1089%2Fars.2017.7485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulte, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagozdzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manevich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beeson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, C. A.</span></span> <span> </span><span class="NLM_article-title">Prdx1 inhibits tumorigenesis via regulating PTEN/AKT activity</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1505</span>– <span class="NLM_lpage">1517</span>, <span class="refDoi"> DOI: 10.1038/emboj.2009.101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1038%2Femboj.2009.101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=19369943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksFemurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=1505-1517&author=J.+Caoauthor=J.+Schulteauthor=A.+Knightauthor=N.+R.+Leslieauthor=A.+Zagozdzonauthor=R.+Bronsonauthor=Y.+Manevichauthor=C.+Beesonauthor=C.+A.+Neumann&title=Prdx1+inhibits+tumorigenesis+via+regulating+PTEN%2FAKT+activity&doi=10.1038%2Femboj.2009.101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Prdx1 inhibits tumorigenesis via regulating PTEN/AKT activity</span></div><div class="casAuthors">Cao, Juxiang; Schulte, Jennifer; Knight, Alexander; Leslie, Nicholas R.; Zagozdzon, Agnieszka; Bronson, Roderick; Manevich, Yefim; Beeson, Craig; Neumann, Carola A.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1505-1517</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">It is widely accepted that reactive oxygen species (ROS) promote tumorigenesis.  However, the exact mechanisms are still unclear.  As mice lacking the peroxidase peroxiredoxin1 (Prdx1) produce more cellular ROS and die prematurely of cancer, they offer an ideal model system to study ROS-induced tumorigenesis.  Prdx1 ablation increased the susceptibility to Ras-induced breast cancer.  We, therefore, investigated the role of Prdx1 in regulating oncogenic Ras effector pathways.  We found Akt hyperactive in fibroblasts and mammary epithelial cells lacking Prdx1.  Investigating the nature of such elevated Akt activation established a novel role for Prdx1 as a safeguard for the lipid phosphatase activity of PTEN, which is essential for its tumor suppressive function.  We found binding of the peroxidase Prdx1 to PTEN essential for protecting PTEN from oxidn.-induced inactivation.  Along those lines, Prdx1 tumor suppression of Ras- or ErbB-2-induced transformation was mediated mainly via PTEN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhIZ-cdT0k4rVg90H21EOLACvtfcHk0lj3SibKIaK2RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksFemurY%253D&md5=5c13edd53866c75fca0a9ad75f392bc4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Femboj.2009.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2009.101%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DSchulte%26aufirst%3DJ.%26aulast%3DKnight%26aufirst%3DA.%26aulast%3DLeslie%26aufirst%3DN.%2BR.%26aulast%3DZagozdzon%26aufirst%3DA.%26aulast%3DBronson%26aufirst%3DR.%26aulast%3DManevich%26aufirst%3DY.%26aulast%3DBeeson%26aufirst%3DC.%26aulast%3DNeumann%26aufirst%3DC.%2BA.%26atitle%3DPrdx1%2520inhibits%2520tumorigenesis%2520via%2520regulating%2520PTEN%252FAKT%2520activity%26jtitle%3DEMBO%2520J.%26date%3D2009%26volume%3D28%26spage%3D1505%26epage%3D1517%26doi%3D10.1038%2Femboj.2009.101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delcuve, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, J. R.</span></span> <span> </span><span class="NLM_article-title">Selective association of peroxiredoxin 1 with genomic DNA and COX-2 upstream promoter elements in estrogen receptor negative breast cancer cells</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2987</span>– <span class="NLM_lpage">2995</span>, <span class="refDoi"> DOI: 10.1091/mbc.e10-02-0160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1091%2Fmbc.e10-02-0160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=20631257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGlt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=2987-2995&author=X.+Wangauthor=S.+Heauthor=J.-M.+Sunauthor=G.+P.+Delcuveauthor=J.+R.+Davie&title=Selective+association+of+peroxiredoxin+1+with+genomic+DNA+and+COX-2+upstream+promoter+elements+in+estrogen+receptor+negative+breast+cancer+cells&doi=10.1091%2Fmbc.e10-02-0160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective association of peroxiredoxin 1 with genomic DNA and COX-2 upstream promoter elements in estrogen receptor negative breast cancer cells</span></div><div class="casAuthors">Wang, Xuemei; He, Shihua; Sun, Jian-Min; Delcuve, Genevieve P.; Davie, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2987-2995</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">In a search for proteins differentially cross-linked to DNA by cisplatin or formaldehyde in normal breast epithelial and breast cancer cell lines, we identified peroxiredoxin 1 (PRDX1) as a protein preferentially cross-linked to DNA in estrogen receptor neg. (ER-) MDA-MB-231 but not in estrogen receptor pos. (ER+) MCF7 breast cancer cells.  Indirect immunofluorescence microscopic analyses showed that PRDX1 was located in the cytoplasm and nucleus of normal and breast cancer cells, with nuclear PRDX1 assocd. with promyelocytic leukemia protein bodies.  We demonstrated that PRDX1 assocn.- with the transcription factor nuclear factor-κB (NF-κB) in MDA-MB-231 but not in MCF7 cells contributed to PRDX1-selective recruitment to MDA-MB-231 genomic DNA.  Furthermore, PRDX1 was assocd. with the cyclooxygenase (COX)-2 upstream promoter region at sites occupied by NF-κB in ER- but not in ER+ breast cancer cells.  PRDX1 knockdown attenuated COX-2 expression by reducing NF-κB occupancy at its upstream promoter element in MDA-MB-231 but not in MCF7 cells.  A phosphorylated form of PRDX1 was only present in ER- breast cancer cells.  Because PRDX1 phosphorylation is known to inhibit its peroxidase activity and to promote PRDX1 oligomerization, we propose that PRDX1 acts as a chaperone to enhance the transactivation potential of NF-κB in ER- breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyD3iLaeWTrrVg90H21EOLACvtfcHk0ljUyoGwvP3tCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGlt7vK&md5=d127e1d53bdaa5260af697bb989232b3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e10-02-0160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e10-02-0160%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DJ.-M.%26aulast%3DDelcuve%26aufirst%3DG.%2BP.%26aulast%3DDavie%26aufirst%3DJ.%2BR.%26atitle%3DSelective%2520association%2520of%2520peroxiredoxin%25201%2520with%2520genomic%2520DNA%2520and%2520COX-2%2520upstream%2520promoter%2520elements%2520in%2520estrogen%2520receptor%2520negative%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2010%26volume%3D21%26spage%3D2987%26epage%3D2995%26doi%3D10.1091%2Fmbc.e10-02-0160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alkam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semenza, G. L.</span></span> <span> </span><span class="NLM_article-title">PRDX2 and PRDX4 are negative regulators of hypoxia-inducible factors under conditions of prolonged hypoxia</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">6379</span>– <span class="NLM_lpage">6397</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.7142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.18632%2Foncotarget.7142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=26837221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A280%3ADC%252BC28nmvVKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=6379-6397&author=W.+Luoauthor=I.+Chenauthor=Y.+Chenauthor=D.+Alkamauthor=Y.+Wangauthor=G.+L.+Semenza&title=PRDX2+and+PRDX4+are+negative+regulators+of+hypoxia-inducible+factors+under+conditions+of+prolonged+hypoxia&doi=10.18632%2Foncotarget.7142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">PRDX2 and PRDX4 are negative regulators of hypoxia-inducible factors under conditions of prolonged hypoxia</span></div><div class="casAuthors">Luo Weibo; Chen Ivan; Semenza Gregg L; Luo Weibo; Semenza Gregg L; Luo Weibo; Chen Yan; Alkam Duah; Wang Yingfei; Luo Weibo; Wang Yingfei; Semenza Gregg L; Semenza Gregg L; Semenza Gregg L; Semenza Gregg L; Semenza Gregg L</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">6379-97</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hypoxia-inducible factors (HIFs) control the transcription of genes that are crucial for the pathogenesis of cancer and other human diseases.  The transcriptional activity of HIFs is rapidly increased upon exposure to hypoxia, but expression of some HIF target genes decreases during prolonged hypoxia.  However, the underlying mechanism for feedback inhibition is not completely understood.  Here, we report that peroxiredoxin 2 (PRDX2) and PRDX4 interact with HIF-1α and HIF-2α in vitro and in hypoxic HeLa cells.  Prolonged hypoxia increases the nuclear translocation of PRDX2 and PRDX4.  As a result, PRDX2 and PRDX4 impair HIF-1 and HIF-2 binding to the hypoxia response elements of a subset of HIF target genes, thereby inhibiting gene transcription in cells exposed to prolonged hypoxia.  PRDX2 and PRDX4 have no effect on the recruitment of p300 and RNA polymerase II to HIF target genes and the enzymatic activity of PRDX2 and PRDX4 is not required for inhibition of HIF-1 and HIF-2.  We also demonstrate that PRDX2 is a direct HIF target gene and that PRDX2 expression is induced by prolonged hypoxia.  These findings uncover a novel feedback mechanism for inhibition of HIF transcriptional activity under conditions of prolonged hypoxia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyKKcecMBANoYPsCkJIw00fW6udTcc2eYweLVTClpIZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nmvVKjsA%253D%253D&md5=c21855c99d2d56991ede091f2915b4fc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7142%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DAlkam%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26atitle%3DPRDX2%2520and%2520PRDX4%2520are%2520negative%2520regulators%2520of%2520hypoxia-inducible%2520factors%2520under%2520conditions%2520of%2520prolonged%2520hypoxia%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D6379%26epage%3D6397%26doi%3D10.18632%2Foncotarget.7142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Min, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-Y.</span></span> <span> </span><span class="NLM_article-title">Inhibition of TRAF6 ubiquitin-ligase activity by PRDX1 leads to inhibition of NFKB activation and autophagy activation</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1358</span>, <span class="refDoi"> DOI: 10.1080/15548627.2018.1474995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1080%2F15548627.2018.1474995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29929436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=1347-1358&author=Y.+Minauthor=M.-J.+Kimauthor=S.+Leeauthor=E.+Chunauthor=K.-Y.+Lee&title=Inhibition+of+TRAF6+ubiquitin-ligase+activity+by+PRDX1+leads+to+inhibition+of+NFKB+activation+and+autophagy+activation&doi=10.1080%2F15548627.2018.1474995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of TRAF6 ubiquitin-ligase activity by PRDX1 leads to inhibition of NFKB activation and autophagy activation</span></div><div class="casAuthors">Min, Yoon; Kim, Mi-Jeong; Lee, Sena; Chun, Eunyoung; Lee, Ki-Young</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1347-1358</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">TRAF6 (TNF receptor assocd. factor 6) plays a pivotal role in NFKB activation and macroautphagy/autophagy activation induced by TLR4 (toll like receptor 4) signaling.  The objective of this study was to det. the functional role of PRDX1 (peroxiredoxin 1) in NFKB activation and autophagy activation.  PRDX1 interacted with the ring finger domain of TRAF6 and inhibited its ubiquitin-ligase activity.  The inhibition on TRAF6 ubiquitin-ligase activity by PRDX1 induced the suppression of ubiquitination of an evolutionarily conserved signaling intermediate in Toll pathways (ECSIT) essential for NFKB activation and BECN1 (beclin 1) required for autophagy activation.  An inhibitory effect of PRDX1 on TRAF6 was clearly evidenced in PRDX1-knockdown (PRDX1KD) THP-1, PRDX1KD MDA-MB-231, and PRDX1KD SK-HEP-1 cells.  PRDX1KD THP-1 cells showed increases of NFKB activation, pro-inflammatory cytokine prodn., NFKB-dependent gene expression induced by TLR4 stimulation, and resistance against Salmonella typhimurium infection.  Addnl., migration and invasion abilities of PRDX1KD MDA-MB-231 and PRDX1KD SK-HEP-1 cancer cells were significantly enhanced compared to those of control cancer cells.  Taken together, these results suggest that PRDX1 neg. regulates TLR4 signaling for NFKB activation and autophagy functions such as bactericidal activity, cancer cell migration, and cancer cell invasion by inhibiting TRAF6 ubiquitin-ligase activity. 3-MA: 3-methyladenine; BECN1: beclin 1; CHUK/IKKA: conserved helix-loop-helix ubiquitous kinase; ECSIT: ECSIT signalling integrator; ELISA: ELISA; NFKB: nuclear factor kappa-light-chain-enhancer of activated B cells; IB: immunoblotting; IKBKB/IKKB: inhibitor of nuclear factor kappa B kinase subunit beta; IL1B: interleukin 1 beta; IL6: interleukin 6; IP: immunopptn.; LPS: lipopolysaccharide; MAP1LC3/LC3: microtuble assocd. protein 1 light chain 3; MAP3K7/TAK1: mitogen-activated protein kinase kinase kinase 7; MAPK14/p38: mitogen-activated protein kinase 14; mROS: mitochondrial reactive oxygen species; PRDX1: peroxiredoxin 1; PRDX6: peroxiredoxin 6; RELA/p65: RELA proto-oncogene, NF-kB subunit; TRAF6 TNF: receptor assocd. factor 6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGpT9XSrqB0bVg90H21EOLACvtfcHk0ljUyoGwvP3tCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsrrJ&md5=b0f42cec5eddab01270cd5990c3b5ce3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1080%2F15548627.2018.1474995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2018.1474995%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DM.-J.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DChun%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DK.-Y.%26atitle%3DInhibition%2520of%2520TRAF6%2520ubiquitin-ligase%2520activity%2520by%2520PRDX1%2520leads%2520to%2520inhibition%2520of%2520NFKB%2520activation%2520and%2520autophagy%2520activation%26jtitle%3DAutophagy%26date%3D2018%26volume%3D14%26spage%3D1347%26epage%3D1358%26doi%3D10.1080%2F15548627.2018.1474995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwahara, S.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, Y.</span></span> <span> </span><span class="NLM_article-title">Dimer-Oligomer Interconversion of Wild-type and Mutant Rat 2-Cys Peroxiredoxin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1074/jbc.m705753200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1074%2Fjbc.m705753200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=17974571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=284-293&author=T.+Matsumuraauthor=K.+Okamotoauthor=S.-i.+Iwaharaauthor=H.+Horiauthor=Y.+Takahashiauthor=T.+Nishinoauthor=Y.+Abe&title=Dimer-Oligomer+Interconversion+of+Wild-type+and+Mutant+Rat+2-Cys+Peroxiredoxin&doi=10.1074%2Fjbc.m705753200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dimer-Oligomer Interconversion of Wild-type and Mutant Rat 2-Cys Peroxiredoxin. Disulfide Formation at Dimer-Dimer Interfaces is not Essential for Decamerization</span></div><div class="casAuthors">Matsumura, Tomohiro; Okamoto, Ken; Iwahara, Shin-ichiro; Hori, Hiroyuki; Takahashi, Yuriko; Nishino, Takeshi; Abe, Yasuko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">284-293</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Rat heme-binding protein 23 (HBP23)/peroxiredoxin (Prx I) belongs to the 2-Cys peroxiredoxin type I family and exhibits peroxidase activity coupled with reduced thioredoxin (Trx) as an electron donor.  We analyzed the dimer-oligomer interconversion of wild-type and mutant HBP23/Prx I by gel filtration and found that the C52S and C173S mutants existed mostly as decamers, whereas the wild type was a mixt. of various forms, favoring the decamer at higher protein concn. and lower ionic salt concn. and in the presence of dithiothreitol.  The C83S mutant was predominantly dimeric, in agreement with a previous crystallog. anal.  X-ray diffraction anal. of the decameric C52S mutant revealed a toroidal structure (diam., ∼130Å; inside diam., ∼55Å; thickness, ∼45Å).  In contrast to human Prx I, which was recently reported to exist predominantly as the decamer with Cys83-Cys83 disulfide bonds at all dimer-dimer interfaces, rat HBP23/Prx I has a Cys83-Cys83 disulfide bond at only one dimer-dimer interface (S-S sepn. of ∼2.1Å), whereas the interactions at the other interfaces (mean S-S sepn. of 3.6Å) appear to involve hydrophobic and van der Waals forces.  This finding is consistent with gel filtration analyses showing that the protein readily interconverts between dimer and oligomeric forms.  The C83S mutant exhibited similar peroxidase activity to the wild type, which is exclusively dimeric, in the Trx/Trx reductase system.  At higher concns., where the protein was mostly decameric, less efficient attack of reduced Trx was obsd. in a [14C]iodoacetamide incorporation expt.  We suggest that the dimer-decamer interconversion may have a regulatory role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6sGarZpmgOrVg90H21EOLACvtfcHk0lgQsRc538ZQJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOksw%253D%253D&md5=cbf6946890e2ec363b77f5c2191b1efc</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m705753200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m705753200%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumura%26aufirst%3DT.%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DIwahara%26aufirst%3DS.-i.%26aulast%3DHori%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DY.%26aulast%3DNishino%26aufirst%3DT.%26aulast%3DAbe%26aufirst%3DY.%26atitle%3DDimer-Oligomer%2520Interconversion%2520of%2520Wild-type%2520and%2520Mutant%2520Rat%25202-Cys%2520Peroxiredoxin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D284%26epage%3D293%26doi%3D10.1074%2Fjbc.m705753200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.-M.</span></span> <span> </span><span class="NLM_article-title">Human Peroxiredoxin 1 and 2 Are Not Duplicate Proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">22011</span>– <span class="NLM_lpage">22022</span>, <span class="refDoi"> DOI: 10.1074/jbc.m610330200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1074%2Fjbc.m610330200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=17519234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVGls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=22011-22022&author=W.+Leeauthor=K.-S.+Choiauthor=J.+Riddellauthor=J.+Riddellauthor=C.+Ipauthor=D.+Ghoshauthor=J.-H.+Parkauthor=Y.-M.+Park&title=Human+Peroxiredoxin+1+and+2+Are+Not+Duplicate+Proteins&doi=10.1074%2Fjbc.m610330200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Human Peroxiredoxin 1 and 2 Are Not Duplicate Proteins: the unique presence of Cys83 in Prx1 underscores the structural and functional differences between Prx1 and Prx2</span></div><div class="casAuthors">Lee, WeonSup; Choi, Kyoung-Soo; Riddell, Jonah; Ip, Clement; Ghosh, Debashis; Park, Jong-Hoon; Park, Young-Mee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">22011-22022</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human peroxiredoxins 1 and 2, also known as Prx1 and Prx2, are more than 90% homologous in their amino acid sequences.  Prx1 and Prx2 are elevated in various cancers and are shown to influence diverse cellular processes.  Although their growth regulatory role has traditionally been attributed to the peroxidase activity, the physiol. significance of this function is unclear because the proteins are highly susceptible to inactivation by H2O2.  A chaperone activity appears to emerge when their peroxidase activity is lost.  Structural studies suggest that they may form a homodimer or doughnut-shaped homodecamer.  However, little information is available whether human Prx1 and Prx2 are duplicative in structure and function.  We noted that Prx1 contains a cysteine (Cys83) at the putative dimer-dimer interface, which is absent in Prx2.  We studied the role of Cys83 in regulating the peroxidase and chaperone activities of Prx1, because the redox status of Cys83 might influence the oligomeric structure and consequently the functions of Prx1.  We show that Prx1 is more efficient as a mol. chaperone, whereas Prx2 is better suited as a peroxidase enzyme.  Substituting Cys83 with Ser83 (Prx1C83S) results in dramatic changes in the structural and functional characteristics of Prx1 in a direction similar to those of Prx2.  Here we also report the first crystal structure of human Prx1 and the presence of the Cys83-Cys83 bond at the dimer-dimer interface of decameric Prx1.  These findings are consistent with the hypothesis that human Prx1 and Prx2 possess unique functions and regulatory mechanisms and that Cys83 bestows a distinctive identity to Prx1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlM2Il1g6njrVg90H21EOLACvtfcHk0lgQsRc538ZQJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVGls7c%253D&md5=a69c123c9f2b59eb2ce521880b15d2e6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m610330200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m610330200%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DW.%26aulast%3DChoi%26aufirst%3DK.-S.%26aulast%3DRiddell%26aufirst%3DJ.%26aulast%3DRiddell%26aufirst%3DJ.%26aulast%3DIp%26aufirst%3DC.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DPark%26aufirst%3DJ.-H.%26aulast%3DPark%26aufirst%3DY.-M.%26atitle%3DHuman%2520Peroxiredoxin%25201%2520and%25202%2520Are%2520Not%2520Duplicate%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D22011%26epage%3D22022%26doi%3D10.1074%2Fjbc.m610330200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, D.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and anticancer properties of isocorydine derivatives</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">6542</span>– <span class="NLM_lpage">6553</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1016%2Fj.bmc.2017.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29103873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslKqs7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=6542-6553&author=Q.+Yanauthor=R.+Liauthor=A.+Xinauthor=Y.+Hanauthor=Y.+Zhangauthor=J.+Liuauthor=W.+Liauthor=D.+Di&title=Design%2C+synthesis%2C+and+anticancer+properties+of+isocorydine+derivatives&doi=10.1016%2Fj.bmc.2017.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and anticancer properties of isocorydine derivatives</span></div><div class="casAuthors">Yan, Qian; Li, Ruxia; Xin, Aiyi; Han, Yin; Zhang, Yanxia; Liu, Junxi; Li, Wenguang; Di, Duolong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6542-6553</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Isocorydine (ICD), an aporphine alkaloid, is widely distributed in nature.  Its ability to target side population (SP) cells found in human hepatocellular carcinoma (HCC) makes it and its deriv. 8-amino-isocorydine (NICD) promising chemotherapeutic agents for the treatment of HCC.  To improve the anticancer activity of isocorydine derivs., twenty derivs. of NICD were designed and synthesized through chem. structure modifications of the arom. amino group at C-8.  The anti-proliferative activities of all synthesized compds. against human hepatocellular (HepG2), cervical (HeLa), and gastric (MGC-803) cancer cell lines were evaluated using an MTT assay.  The results showed that all the synthetic compds. had some tumor cell growth inhibitory activity.  The compd. COM33 (24) was the most active with IC50 values under 10 μM (IC50 for HepG2 = 7.51 μM; IC50 for HeLa = 6.32 μM).  FICD (12) and COM33 (24) were selected for further investigation of their in vitro and in vivo activities due to their relatively good antiproliferative properties.  These two compds. significantly downregulated the expression of four key proteins (C-Myc, β-Catenin, CylinD1, and Ki67) in HepG2 cells.  The tumor inhibition rate of COM33 (24) in vivo was 73.8% after a dose 100 mg/kg via i.p. injection and the combined inhibition rate of COM33 (24) (50 mg/kg) with sorafenib (50 mg/kg) was 66.5%.  The results indicated that these isocorydine derivs. could potentially be used as targeted chemotherapy agents or could be further developed in combination with conventional chemotherapy drugs to target cancer stem cells (CSCs) and epithelial-to-mesenchymal transition (EMT), the main therapeutic targets in HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl2wlHCQQ_2rVg90H21EOLACvtfcHk0lgQsRc538ZQJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslKqs7bM&md5=79d4ab790e2ba063b3978aa446e47e09</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DXin%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DDi%26aufirst%3DD.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520anticancer%2520properties%2520of%2520isocorydine%2520derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D6542%26epage%3D6553%26doi%3D10.1016%2Fj.bmc.2017.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroiss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nechitaylo, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svatoš, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltenpoth, M.</span></span> <span> </span><span class="NLM_article-title">Evolutionary stability of antibiotic protection in a defensive symbiosis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>, <span class="NLM_elocation-id">E2020</span> <span class="refDoi"> DOI: 10.1073/pnas.1719797115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1073%2Fpnas.1719797115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29444867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVanu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&author=T.+Englauthor=J.+Kroissauthor=M.+Kaiauthor=T.+Y.+Nechitayloauthor=A.+Svato%C5%A1author=M.+Kaltenpoth&title=Evolutionary+stability+of+antibiotic+protection+in+a+defensive+symbiosis&doi=10.1073%2Fpnas.1719797115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutionary stability of antibiotic protection in a defensive symbiosis</span></div><div class="casAuthors">Engl, Tobias; Kroiss, Johannes; Kai, Marco; Nechitaylo, Taras Y.; SvatoA, AleA; Kaltenpoth, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">E2020-E2029</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The increasing resistance of human pathogens severely limits the efficacy of antibiotics in medicine, yet many animals, including solitary beewolf wasps, successfully engage in defensive alliances with antibiotic-producing bacteria for millions of years.  Here, we report on the in situ prodn. of 49 derivs. belonging to three antibiotic compd. classes (45 piericidin derivs., 3 streptochlorin derivs., and nigericin) by the symbionts of 25 beewolf host species and subspecies, spanning 68 million years of evolution.  Despite a high degree of qual. stability in the antibiotic mixt., we found consistent quant. differences between species and across geog. localities, presumably reflecting adaptations to combat local pathogen communities.  Antimicrobial bioassays with the three main components and in silico predictions based on the structure and specificity in polyketide synthase domains of the piericidin biosynthesis gene cluster yield insights into the mechanistic basis and ecoevolutionary implications of producing a complex mixt. of antimicrobial compds. in a natural setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYpNrANw7F97Vg90H21EOLACvtfcHk0lgQsRc538ZQJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVanu7vN&md5=b80e3912236bc65ea797a8b5f7b76054</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1719797115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1719797115%26sid%3Dliteratum%253Aachs%26aulast%3DEngl%26aufirst%3DT.%26aulast%3DKroiss%26aufirst%3DJ.%26aulast%3DKai%26aufirst%3DM.%26aulast%3DNechitaylo%26aufirst%3DT.%2BY.%26aulast%3DSvato%25C5%25A1%26aufirst%3DA.%26aulast%3DKaltenpoth%26aufirst%3DM.%26atitle%3DEvolutionary%2520stability%2520of%2520antibiotic%2520protection%2520in%2520a%2520defensive%2520symbiosis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2018%26volume%3D115%26doi%3D10.1073%2Fpnas.1719797115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reczek, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandel, N. S.</span></span> <span> </span><span class="NLM_article-title">ROS promotes cancer cell survival through calcium signaling</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">951</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1016%2Fj.ccell.2018.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=29894695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlu7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=949-951&author=C.+R.+Reczekauthor=N.+S.+Chandel&title=ROS+promotes+cancer+cell+survival+through+calcium+signaling&doi=10.1016%2Fj.ccell.2018.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">ROS Promotes Cancer Cell Survival through Calcium Signaling</span></div><div class="casAuthors">Reczek, Colleen R.; Chandel, Navdeep S.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">949-951</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">To avoid reactive oxygen species (ROS)-induced cell death, cancer cells increase their antioxidant defense system.  In this issue of Cancer Cell, Takahashi et al. identify a novel, non-canonical oxidative stress defense mechanism involving TRPA1, a redox-sensitive Ca2+ channel, and the upregulation of anti-apoptotic pathways to promote cancer cell survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyer3tLFxEWrVg90H21EOLACvtfcHk0livq9RvInEAaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlu7%252FL&md5=8cc6a3ec6b3fdc48a95ce613acb2aa1c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DReczek%26aufirst%3DC.%2BR.%26aulast%3DChandel%26aufirst%3DN.%2BS.%26atitle%3DROS%2520promotes%2520cancer%2520cell%2520survival%2520through%2520calcium%2520signaling%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D949%26epage%3D951%26doi%3D10.1016%2Fj.ccell.2018.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forshaw, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmila, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowther, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furdui, C. M.</span></span> <span> </span><span class="NLM_article-title">Peroxiredoxins in cancer and response to radiation therapies</span>. <i>Antioxidants</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">11</span>, <span class="refDoi"> DOI: 10.3390/antiox8010011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.3390%2Fantiox8010011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs12rur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=11&author=T.+E.+Forshawauthor=R.+Holmilaauthor=K.+J.+Nelsonauthor=J.+E.+Lewisauthor=M.+L.+Kempauthor=A.+W.+Tsangauthor=L.+B.+Pooleauthor=W.+T.+Lowtherauthor=C.+M.+Furdui&title=Peroxiredoxins+in+cancer+and+response+to+radiation+therapies&doi=10.3390%2Fantiox8010011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Peroxiredoxins in cancer and response to radiation therapies</span></div><div class="casAuthors">Forshaw, Tom E.; Holmila, Reetta; Nelson, Kimberly J.; Lewis, Joshua E.; Kemp, Melissa L.; Tsang, Allen W.; Poole, Leslie B.; Lowther, W. Todd; Furdui, Cristina M.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11/1-11/24</span>CODEN:
                <span class="NLM_cas:coden">ANTIGE</span>;
        ISSN:<span class="NLM_cas:issn">2076-3921</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Peroxiredoxins have a long-established cellular function as regulators of redox metab. by catalyzing the redn. of peroxides (e.g., H2O2, lipid peroxides) with high catalytic efficiency.  This activity is also crit. to the initiation and relay of both phosphorylation and redox signaling in a broad range of pathophysiol. contexts.  Under normal physiol. conditions, peroxiredoxins protect normal cells from oxidative damage that could promote oncogenesis (e.g., environmental stressors).  In cancer, higher expression level of peroxiredoxins has been assocd. with both tumor growth and resistance to radiation therapies.  However, this relationship between the expression of peroxiredoxins and the response to radiation is not evident from an anal. of data in The Cancer Genome Atlas (TCGA) or NCI60 panel of cancer cell lines.  The focus of this review is to summarize the current exptl. knowledge implicating this class of proteins in cancer, and to provide a perspective on the value of targeting peroxiredoxins in the management of cancer.  Potential biases in the anal. of the TCGA data with respect to radiation resistance are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEETkUGLFnq7Vg90H21EOLACvtfcHk0livq9RvInEAaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs12rur4%253D&md5=3e4d0af62e112f5d1b919caebecb810f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fantiox8010011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fantiox8010011%26sid%3Dliteratum%253Aachs%26aulast%3DForshaw%26aufirst%3DT.%2BE.%26aulast%3DHolmila%26aufirst%3DR.%26aulast%3DNelson%26aufirst%3DK.%2BJ.%26aulast%3DLewis%26aufirst%3DJ.%2BE.%26aulast%3DKemp%26aufirst%3DM.%2BL.%26aulast%3DTsang%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DL.%2BB.%26aulast%3DLowther%26aufirst%3DW.%2BT.%26aulast%3DFurdui%26aufirst%3DC.%2BM.%26atitle%3DPeroxiredoxins%2520in%2520cancer%2520and%2520response%2520to%2520radiation%2520therapies%26jtitle%3DAntioxidants%26date%3D2019%26volume%3D8%26spage%3D11%26doi%3D10.3390%2Fantiox8010011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evens, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prachand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paniaqua, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gartenhaus, R. B.</span></span> <span> </span><span class="NLM_article-title">Imexon-induced apoptosis in multiple myeloma tumor cells is caspase-8 dependent</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1481</span>– <span class="NLM_lpage">1491</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-1058-03</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1158%2F1078-0432.ccr-1058-03" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=14977852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsVGjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1481-1491&author=A.+M.+Evensauthor=S.+Prachandauthor=B.+Shiauthor=M.+Paniaquaauthor=L.+I.+Gordonauthor=R.+B.+Gartenhaus&title=Imexon-induced+apoptosis+in+multiple+myeloma+tumor+cells+is+caspase-8+dependent&doi=10.1158%2F1078-0432.ccr-1058-03"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Imexon-induced apoptosis in multiple myeloma tumor cells is caspase-8 dependent</span></div><div class="casAuthors">Evens, Andrew M.; Prachand, Sheila; Shi, Bo; Paniaqua, Mary; Gordon, Leo I.; Gartenhaus, Ronald B.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1481-1491</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Imexon is a 2-cyanoaziridine agent that has been shown to inhibit growth of chemotherapy-sensitive myeloma cells through apoptosis with decreased cellular stores of glutathione and increased reactive oxygen species (ROS).  We examd. the mechanism of imexon cytotoxicity in a diverse panel of dexamethasone and chemotherapy-sensitive and -resistant myeloma cell lines.  We examd. cellular cytotoxicity, apoptosis, and changes in redox state in dexamethasone-sensitive (C2E3), dexamethasone-resistant (1-310 and 1-414), chemotherapy-sensitive (RPMI-8226), and chemotherapy-resistant (DOX-1V and DOX-10V) myeloma cell lines.  We found significant cytotoxicity after 48-h incubation with imexon (80-160 μM) in dexamethasone and chemotherapy-sensitive and -resistant myeloma cell lines in a time- and dose-dependent manner.  The mechanism of imexon cytotoxicity in all cell lines was related to induction of apoptosis with the presence of cleaved caspase-3.  Moreover, after imexon exposure in C2E3 and 1-414 cell lines, we demonstrated caspase-8-dependent apoptosis.  Bcl-2:bax was proapoptotic with imexon in C2E3, whereas bcl-2:bax was independent of steroid resistance, chemotherapy sensitivity, and chemotherapy resistance.  Depletion of intracellular glutathione was documented in RPMI-8226 at high imexon concns. (≥225 μM) but not in other cell lines.  Furthermore, ROS were found in C2E3, RPMI-8226, and 1-310 only at high imexon concns., whereas a sensitive marker of oxidative DNA damage, 8-hydroxydeoxyguanosine, was not increased in any cell line.  Our results demonstrate that imexon has significant broad antimyeloma activity that is mediated through apoptotic mechanisms that is not dependent on prodn. of ROS.  Moreover, we have identified a mechanism of cytotoxicity in dexamethasone-sensitive and -resistant myeloma cells induced by imexon that is caspase-8 dependent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHkuRSvZWB-LVg90H21EOLACvtfcHk0livq9RvInEAaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsVGjtro%253D&md5=29af89d82b6335aab0310bbf4133f992</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-1058-03&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-1058-03%26sid%3Dliteratum%253Aachs%26aulast%3DEvens%26aufirst%3DA.%2BM.%26aulast%3DPrachand%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DB.%26aulast%3DPaniaqua%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DL.%2BI.%26aulast%3DGartenhaus%26aufirst%3DR.%2BB.%26atitle%3DImexon-induced%2520apoptosis%2520in%2520multiple%2520myeloma%2520tumor%2520cells%2520is%2520caspase-8%2520dependent%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2004%26volume%3D10%26spage%3D1481%26epage%3D1491%26doi%3D10.1158%2F1078-0432.ccr-1058-03" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span> <span> </span><span class="NLM_article-title">Nitrobenzoyl sesquiterpenoids with cytotoxic activities from a marine-derived Aspergillus ochraceus fungus</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">92</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.7b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.7b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFSisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=92-97&author=Y.+Tanauthor=B.+Yangauthor=X.+Linauthor=X.+Luoauthor=X.+Pangauthor=L.+Tangauthor=Y.+Liuauthor=X.+Liauthor=X.+Zhou&title=Nitrobenzoyl+sesquiterpenoids+with+cytotoxic+activities+from+a+marine-derived+Aspergillus+ochraceus+fungus&doi=10.1021%2Facs.jnatprod.7b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Nitrobenzoyl sesquiterpenoids with cytotoxic activities from a marine-derived Aspergillus ochraceus fungus</span></div><div class="casAuthors">Tan, Yanhui; Yang, Bin; Lin, Xiuping; Luo, Xiaowei; Pang, Xiaoyan; Tang, Lan; Liu, Yonghong; Li, Xiaojuan; Zhou, Xuefeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">92-97</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Nitrobenzoyl sesquiterpenoids are rare from natural sources.  Two new nitrobenzoyl sesquiterpenoids, insulicolide B (1) and insulicolide C (3), and the new natural product 14-O-acetylinsulicolide A (2) were isolated from culture exts. of the marine-derived fungus Aspergillus ochraceus Jcma1F17, together with three known nitrobenzoyl sesquiterpenoids (4-6) and a deriv. sesquiterpenoid (7).  The structures of the new compds., including their abs. configurations, were detd. by NMR and MS spectroscopic data analyses and comparison between the calcd. and exptl. ECD spectra.  The nitrobenzoyl sesquiterpenoids (1-6) were evaluated for their cytotoxicities against three renal carcinoma cell lines, ACHN, OS-RC-2, and 786-O cells, and compds. 2, 4, and 5 displayed activities with IC50 values of 0.89 to 8.2 μM.  Further studies indicated that 2 arrested the cell cycle at the G0/G1 phase at a concn. of 1 μM and induced late apoptosis at a concn. of 2 μM after a 72 h treatment of 786-O cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGLpU9yG7CrbVg90H21EOLACvtfcHk0lgwB9AoFd0iVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFSisw%253D%253D&md5=cff823de551af3679bb0b7b2c34534ef</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.7b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.7b00698%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DPang%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DX.%26atitle%3DNitrobenzoyl%2520sesquiterpenoids%2520with%2520cytotoxic%2520activities%2520from%2520a%2520marine-derived%2520Aspergillus%2520ochraceus%2520fungus%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2018%26volume%3D81%26spage%3D92%26epage%3D97%26doi%3D10.1021%2Facs.jnatprod.7b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nador, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhasz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korosi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Székács, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horvath, R.</span></span> <span> </span><span class="NLM_article-title">Incubator proof miniaturized Holomonitor toin situmonitor cancer cells exposed to green tea polyphenol and preosteoblast cells adhering on nanostructured titanate surfaces: validity of the measured parameters and their corrections</span>. <i>J. Biomed. Opt.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">067002</span>, <span class="refDoi"> DOI: 10.1117/1.jbo.20.6.067002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=10.1117%2F1.jbo.20.6.067002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=26057033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A280%3ADC%252BC2MbitFaktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=067002&author=B.+Peterauthor=J.+Nadorauthor=K.+Juhaszauthor=A.+Dobosauthor=L.+Korosiauthor=I.+Sz%C3%A9k%C3%A1csauthor=D.+Patkoauthor=R.+Horvath&title=Incubator+proof+miniaturized+Holomonitor+toin+situmonitor+cancer+cells+exposed+to+green+tea+polyphenol+and+preosteoblast+cells+adhering+on+nanostructured+titanate+surfaces%3A+validity+of+the+measured+parameters+and+their+corrections&doi=10.1117%2F1.jbo.20.6.067002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Incubator proof miniaturized Holomonitor to in situ monitor cancer cells exposed to green tea polyphenol and preosteoblast cells adhering on nanostructured titanate surfaces: validity of the measured parameters and their corrections</span></div><div class="casAuthors">Peter Beatrix; Nador Judit; Juhasz Krisztina; Patko Daniel; Horvath Robert; Dobos Agnes; Szekacs Inna; Korosi Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomedical optics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">067002</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The in situ observation of cell movements and morphological parameters over longer periods of time under physiological conditions is critical in basic cell research and biomedical applications.  The quantitative phase-contrast microscope applied in this study has a remarkably small size, therefore it can be placed directly into a humidified incubator.  Here, we report on the successful application of this M4 Holomonitor to observe cancer cell motility, motility speed, and migration in the presence of the green tea polyphenol, epigallocatechin gallate, as well as to monitor the adhesion of preosteoblast cells on nanostructured titanate coatings, relevant for biomedical applications.  A special mechanical stage was developed to position the sample into that range of the optical arrangement where digital autofocusing works with high reproducibility and precision.  By in-depth analyzing the obtained single cell morphological parameters, we show that the limited vertical resolution of the optical setup results in underestimated single cell contact area and volume and overestimated single cell averaged thickness.  We propose a simple model to correct the recorded data to obtain more precise single cell parameters.  We compare the results with the kinetic data recorded by a surface sensitive optical biosensor, optical waveguide lightmode spectroscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTPnAsi6YzzQsRAV4D_Ins_fW6udTcc2ebQEXuVDyutwLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbitFaktA%253D%253D&md5=7cbdf278ce6cba250d2759c3f85a8087</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1117%2F1.jbo.20.6.067002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1117%252F1.jbo.20.6.067002%26sid%3Dliteratum%253Aachs%26aulast%3DPeter%26aufirst%3DB.%26aulast%3DNador%26aufirst%3DJ.%26aulast%3DJuhasz%26aufirst%3DK.%26aulast%3DDobos%26aufirst%3DA.%26aulast%3DKorosi%26aufirst%3DL.%26aulast%3DSz%25C3%25A9k%25C3%25A1cs%26aufirst%3DI.%26aulast%3DPatko%26aufirst%3DD.%26aulast%3DHorvath%26aufirst%3DR.%26atitle%3DIncubator%2520proof%2520miniaturized%2520Holomonitor%2520toin%2520situmonitor%2520cancer%2520cells%2520exposed%2520to%2520green%2520tea%2520polyphenol%2520and%2520preosteoblast%2520cells%2520adhering%2520on%2520nanostructured%2520titanate%2520surfaces%253A%2520validity%2520of%2520the%2520measured%2520parameters%2520and%2520their%2520corrections%26jtitle%3DJ.%2520Biomed.%2520Opt.%26date%3D2015%26volume%3D20%26spage%3D067002%26doi%3D10.1117%2F1.jbo.20.6.067002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span> <span> </span><span class="NLM_article-title">Spirostaphylotrichin X from a marine-derived fungus as an anti-influenza agent targeting RNA polymerase PB2</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">2722</span>– <span class="NLM_lpage">2730</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.8b00656</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.8b00656" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVelsb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=2722-2730&author=J.+Wangauthor=F.+Chenauthor=Y.+Liuauthor=Y.+Liuauthor=K.+Liauthor=X.+Yangauthor=S.+Liuauthor=X.+Zhouauthor=J.+Yang&title=Spirostaphylotrichin+X+from+a+marine-derived+fungus+as+an+anti-influenza+agent+targeting+RNA+polymerase+PB2&doi=10.1021%2Facs.jnatprod.8b00656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Spirostaphylotrichin X from a Marine-Derived Fungus as an Anti-influenza Agent Targeting RNA Polymerase PB2</span></div><div class="casAuthors">Wang, Jianjiao; Chen, Feimin; Liu, Yunhao; Liu, Yuxuan; Li, Kunlong; Yang, Xiliang; Liu, Shuwen; Zhou, Xuefeng; Yang, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2722-2730</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">A new spirocyclic γ-lactam, named spirostaphylotrichin X (I), and three related known spirostaphylotrichins (2-4) were isolated from the marine-derived fungus Cochliobolus lunatus SCSIO41401.  Their structures were detd. by spectroscopic analyses.  Spirostaphylotrichin X displayed obvious inhibitory activities against multiple influenza virus strains, with IC50 values from 1.2 to 5.5 μM.  Investigation of the mechanism showed that I inhibited viral polymerase activity and interfered with the prodn. of progeny viral RNA.  Homogeneous time-resolved fluorescence, surface plasmon resonance assays, and a mol. docking study revealed that I could inhibit polymerase PB2 protein activity by binding to the highly conserved region of the cap-binding domain of PB2.  These results suggest that I inhibits the replication of influenza A virus by interfering with the activity of PB2 protein and that I represents a new type of potential lead compd. for the development of anti-influenza therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdkyOxXO-X5bVg90H21EOLACvtfcHk0lgwB9AoFd0iVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVelsb%252FP&md5=b3e8ba9f8c6196d0c22d483a8125daee</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.8b00656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.8b00656%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DJ.%26atitle%3DSpirostaphylotrichin%2520X%2520from%2520a%2520marine-derived%2520fungus%2520as%2520an%2520anti-influenza%2520agent%2520targeting%2520RNA%2520polymerase%2520PB2%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2018%26volume%3D81%26spage%3D2722%26epage%3D2730%26doi%3D10.1021%2Facs.jnatprod.8b00656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z9S','PDB','2Z9S'); return false;">PDB: 2Z9S</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i29"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00598">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10530"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00598">10.1021/acs.jmedchem.9b00598</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary figures and tables, structure elucidation and spectral data of compounds <b>1–27</b>, as well as their spectra (IR, MS, and NMR) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Transcriptome data and bioinformatic analysis (Tables S1–S6) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip">ZIP</a>)</p></li><li><p class="inline">Molecular formula strings for <b>1–27</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_001.pdf">jm9b00598_si_001.pdf (20.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_002.zip">jm9b00598_si_002.zip (976.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00598/suppl_file/jm9b00598_si_003.csv">jm9b00598_si_003.csv (3.63 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00598&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00598%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-15%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00598" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b26deccad3c9a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
