
<html lang="en"     class="pb-page"  data-request-id="d59046e3-9475-4e1b-87d8-51e19b8f5a45"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00774;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2019.62.issue-14;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Chemoproteomics of an Indole-Based Quinone Epoxide Identifies Druggable Vulnerabilities in Vancomycin-Resistant Staphylococcus aureus" /></meta><meta name="dc.Creator" content="Amogh  Kulkarni" /></meta><meta name="dc.Creator" content="Isha  Soni" /></meta><meta name="dc.Creator" content="Dhanashree S.  Kelkar" /></meta><meta name="dc.Creator" content="Allimuthu T.  Dharmaraja" /></meta><meta name="dc.Creator" content="Rathinam K.  Sankar" /></meta><meta name="dc.Creator" content="Gaurav  Beniwal" /></meta><meta name="dc.Creator" content="Abinaya  Rajendran" /></meta><meta name="dc.Creator" content="Sharvari  Tamhankar" /></meta><meta name="dc.Creator" content="Sidharth  Chopra" /></meta><meta name="dc.Creator" content="Siddhesh S.  Kamat" /></meta><meta name="dc.Creator" content="Harinath  Chakrapani" /></meta><meta name="dc.Description" content="The alarming global rise in fatalities from multidrug-resistant Staphylococcus aureus (S. aureus) infections has underscored a need to develop new therapies to address this epidemic. Chemoproteomic..." /></meta><meta name="Description" content="The alarming global rise in fatalities from multidrug-resistant Staphylococcus aureus (S. aureus) infections has underscored a need to develop new therapies to address this epidemic. Chemoproteomic..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 26, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00774" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00774" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00774" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00774" /></link>
        
    
    

<title>Chemoproteomics of an Indole-Based Quinone Epoxide Identifies Druggable Vulnerabilities in Vancomycin-Resistant Staphylococcus aureus | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00774" /></meta><meta property="og:title" content="Chemoproteomics of an Indole-Based Quinone Epoxide Identifies Druggable Vulnerabilities in Vancomycin-Resistant Staphylococcus aureus" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0011.jpeg" /></meta><meta property="og:description" content="The alarming global rise in fatalities from multidrug-resistant Staphylococcus aureus (S. aureus) infections has underscored a need to develop new therapies to address this epidemic. Chemoproteomics is valuable in identifying targets for new drugs in different human diseases including bacterial infections. Targeting functional cysteines is particularly attractive, as they serve critical catalytic functions that enable bacterial survival. Here, we report an indole-based quinone epoxide scaffold with a unique boat-like conformation that allows steric control in modulating thiol reactivity. We extensively characterize a lead compound (4a), which potently inhibits clinically derived vancomycin-resistant S. aureus. Leveraging diverse chemoproteomic platforms, we identify and biochemically validate important transcriptional factors as potent targets of 4a. Interestingly, each identified transcriptional factor has a conserved catalytic cysteine residue that confers antibiotic tolerance to these bacteria. Thus, the chemical tools and biological targets that we describe here prospect new therapeutic paradigms in combatting S. aureus infections." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00774"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00774">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00774&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00774&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00774&amp;href=/doi/10.1021/acs.jmedchem.9b00774" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 6785-6795</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00736" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00818" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Chemoproteomics of an Indole-Based Quinone Epoxide Identifies Druggable Vulnerabilities in Vancomycin-Resistant <i>Staphylococcus aureus</i></span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Amogh Kulkarni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amogh Kulkarni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amogh++Kulkarni">Amogh Kulkarni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Isha Soni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Isha Soni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Microbiology, CSIR-Central Drug Research Institute, Sector 10, Janakipuram Extension, Sitapur Road, Lucknow 226021, Uttar Pradesh, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Isha++Soni">Isha Soni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dhanashree S. Kelkar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dhanashree S. Kelkar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dhanashree+S.++Kelkar">Dhanashree S. Kelkar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Allimuthu T. Dharmaraja</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Allimuthu T. Dharmaraja</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Allimuthu+T.++Dharmaraja">Allimuthu T. Dharmaraja</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rathinam K. Sankar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rathinam K. Sankar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rathinam+K.++Sankar">Rathinam K. Sankar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gaurav Beniwal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gaurav Beniwal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gaurav++Beniwal">Gaurav Beniwal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Abinaya Rajendran</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Abinaya Rajendran</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Abinaya++Rajendran">Abinaya Rajendran</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sharvari Tamhankar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sharvari Tamhankar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sharvari++Tamhankar">Sharvari Tamhankar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sidharth Chopra</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sidharth Chopra</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Microbiology, CSIR-Central Drug Research Institute, Sector 10, Janakipuram Extension, Sitapur Road, Lucknow 226021, Uttar Pradesh, India</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#20534b43484f50524110101760474d41494c0e434f4d"><span class="__cf_email__" data-cfemail="621109010a0d12100352525522050f030b0e4c010d0f">[email protected]</span></a> (S.C.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sidharth++Chopra">Sidharth Chopra</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Siddhesh S. Kamat</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Siddhesh S. Kamat</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#6f1c060b0b070a1c072f06061c0a1d1f1a010a410e0c410601"><span class="__cf_email__" data-cfemail="a8dbc1ccccc0cddbc0e8c1c1dbcddad8ddc6cd86c9cb86c1c6">[email protected]</span></a> (S.K.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Siddhesh+S.++Kamat">Siddhesh S. Kamat</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6132-7574" title="Orcid link">http://orcid.org/0000-0001-6132-7574</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Harinath Chakrapani</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Harinath Chakrapani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Indian Institute of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#9ef6ffecf7f0ffeaf6def7f7edfbeceeebf0fbb0fffdb0f7f0"><span class="__cf_email__" data-cfemail="026a63706b6c63766a426b6b71677072776c672c63612c6b6c">[email protected]</span></a> (H.C.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Harinath++Chakrapani">Harinath Chakrapani</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7267-0906" title="Orcid link">http://orcid.org/0000-0002-7267-0906</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00774&amp;href=/doi/10.1021%2Facs.jmedchem.9b00774" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 6785–6795</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 26, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 May 2019</li><li><span class="item_label"><b>Published</b> online</span>26 June 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 July 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00774" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00774</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00774"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2466</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00774" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Chemoproteomics of an Indole-Based Quinone Epoxide Identifies Druggable Vulnerabilities in Vancomycin-Resistant Staphylococcus aureus&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Amogh&quot;,&quot;last_name&quot;:&quot;Kulkarni&quot;},{&quot;first_name&quot;:&quot;Isha&quot;,&quot;last_name&quot;:&quot;Soni&quot;},{&quot;first_name&quot;:&quot;Dhanashree&quot;,&quot;last_name&quot;:&quot;S. Kelkar&quot;},{&quot;first_name&quot;:&quot;Allimuthu&quot;,&quot;last_name&quot;:&quot;T. Dharmaraja&quot;},{&quot;first_name&quot;:&quot;Rathinam&quot;,&quot;last_name&quot;:&quot;K. Sankar&quot;},{&quot;first_name&quot;:&quot;Gaurav&quot;,&quot;last_name&quot;:&quot;Beniwal&quot;},{&quot;first_name&quot;:&quot;Abinaya&quot;,&quot;last_name&quot;:&quot;Rajendran&quot;},{&quot;first_name&quot;:&quot;Sharvari&quot;,&quot;last_name&quot;:&quot;Tamhankar&quot;},{&quot;first_name&quot;:&quot;Sidharth&quot;,&quot;last_name&quot;:&quot;Chopra&quot;},{&quot;first_name&quot;:&quot;Siddhesh&quot;,&quot;last_name&quot;:&quot;S. Kamat&quot;},{&quot;first_name&quot;:&quot;Harinath&quot;,&quot;last_name&quot;:&quot;Chakrapani&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;6785-6795&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00774&quot;},&quot;abstract&quot;:&quot;The alarming global rise in fatalities from multidrug-resistant Staphylococcus aureus (S. aureus) infections has underscored a need to develop new therapies to address this epidemic. Chemoproteomics is valuable in identifying targets for new drugs in different human diseases including bacterial infections. Targeting functional cysteines is particularly attractive, as they serve critical catalytic functions that enable bacterial survival. Here, we report an indole-based quinone epoxide scaffold with a unique boat-like conformation that allows steric control in modulating thiol reactivity. We extensively characterize a lead compound (4a), which potently inhibits clinically derived vancomycin-resistant S. aureus. Leveraging diverse chemoproteomic platforms, we identify and biochemically validate important transcriptional factors as potent targets of 4a. Interestingly, each identified transcriptional factor has a conserved catalytic cysteine residue that confers antibiotic tolerance to these bacteria. Thus, the c&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00774&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00774" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00774&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00774" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00774&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00774" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00774&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00774&amp;href=/doi/10.1021/acs.jmedchem.9b00774" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00774" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00774" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00774%26sid%3Dliteratum%253Aachs%26pmid%3D31241934%26genre%3Darticle%26aulast%3DKulkarni%26date%3D2019%26atitle%3DChemoproteomics%2Bof%2Ban%2BIndole-Based%2BQuinone%2BEpoxide%2BIdentifies%2BDruggable%2BVulnerabilities%2Bin%2BVancomycin-Resistant%2BStaphylococcus%2Baureus%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D14%26spage%3D6785%26epage%3D6795%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292225" title="Thiols">Thiols</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291285" title="Ethers">Ethers</a>,</li><li><a href="/action/doSearch?ConceptID=291452" title="Reactivity">Reactivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/jmcmar.2019.62.issue-14/20190725/jmcmar.2019.62.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The alarming global rise in fatalities from multidrug-resistant <i>Staphylococcus aureus</i> (<i>S. aureus</i>) infections has underscored a need to develop new therapies to address this epidemic. Chemoproteomics is valuable in identifying targets for new drugs in different human diseases including bacterial infections. Targeting functional cysteines is particularly attractive, as they serve critical catalytic functions that enable bacterial survival. Here, we report an indole-based quinone epoxide scaffold with a unique boat-like conformation that allows steric control in modulating thiol reactivity. We extensively characterize a lead compound (<b>4a</b>), which potently inhibits clinically derived vancomycin-resistant <i>S. aureus</i>. Leveraging diverse chemoproteomic platforms, we identify and biochemically validate important transcriptional factors as potent targets of <b>4a</b>. Interestingly, each identified transcriptional factor has a conserved catalytic cysteine residue that confers antibiotic tolerance to these bacteria. Thus, the chemical tools and biological targets that we describe here prospect new therapeutic paradigms in combatting <i>S. aureus</i> infections.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41261" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41261" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Clinical incidences of antimicrobial resistance (AMR) are on the rise globally and will likely render the existing pool of antimicrobial drugs ineffective. Resistance developed against these drugs makes treatment difficult and expensive.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The bacterium <i>Staphylococcus aureus</i> (<i>S. aureus</i>) is associated with numerous nosocomial infections and can hence rapidly spread among susceptible patient populations. The emergence of resistance against vancomycin [vancomycin-resistant <i>S. aureus</i> (VRSA)] has further weakened our defence against this particular pathogen.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Because only a few new classes of antibiotics are in the pipeline,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> identification of novel druggable biological targets is of high priority. The chemoproteomics technique, activity-based protein profiling (ABPP)<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> has emerged as a powerful tool to study and understand the native activity of enzymes in complex physiological settings, mainly utilizing the unique reactivity of serine,<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> lysine,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and cysteine<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> residues on the proteins. Among these, the cysteine-based ABPP has received considerable attention because thiols are excellent nucleophiles and are known to play crucial physiological roles.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> In order to study cysteines, thiol-selective probes are being designed to pharmacologically perturb the activity of such proteins in order to understand their specific biological functions. Such probes not only help identify new druggable protein sites but can also eventually emerge as potential drug candidates. An integral part of this strategy is to identify functional cysteine residues that are vital for survival, and modification of which, may result in stress and eventual lethality to bacteria. While a number of probes are reported in literature,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> a significant portion of the proteome still remains unmined.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Thus, there is an increasing need to develop new chemical tools to profile the thiol proteome. As a testament to the success of ABPP in identifying new bacterial proteins as targets, Sieber and co-workers have reported a suite of reactive probes to interrogate the proteome for druggable targets.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16">(12−16)</a> For example, using a Michael acceptor, they studied the thiol proteome of <i>S. aureus</i> and found a number of virulence factors as druggable targets.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> This and other pioneering studies<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> lay a strong foundation for our approach to overcome VRSA. Here, we report the design of a new scaffold that is cell-permeable, selective in its reactivity with thiols, and has in-built structural features for modulating thiol reactivity, which helps in profiling the <i>S. aureus</i> thiol proteome.</div><div class="NLM_p">Although several electrophilic species have been developed as thiol-selective probes, many of them suffer from diminished cell permeability and indiscriminate reactivity.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The diversity of p<i>K</i><sub>a</sub> of cysteine residues in the proteome also suggests that a single probe may likely be insufficient. The 2,3-epoxy-1,4-benzoquinone functional group has been previously reported to react with protein thiols<a onclick="showRef(event, 'ref14 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref17">(14,17)</a> and was used as the primary thiol-reactive scaffold for this study (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> We hypothesized that reactivity of this scaffold with a thiol can be modulated by systematic introduction of substituents around the epoxide (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Proposed indole-quinone epoxide (inset: an epoxide the electrophilic reactive site in the probe, and the arrows indicate the approach of the nucleophile).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In order to achieve this, a naphthoquinone epoxide (NQE) was fused with an indole ring, and the resulting indole-based quinone epoxides (IND-QEs) were designed with two major considerations. First, because of its high rates of occurrence in natural products and drugs, indole has been considered a privileged structure.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> This typically suggests enhanced permeability and bioactivity. Second, substituting around the epoxide ring provides opportunities to modulate thiol reactivity and cell permeability. These previously unutilized structural variations will provide a systematic way to study the bacterial thiol proteome. Here, we describe the synthesis of this scaffold and generation of analogues, its structure activity studies in bacteria, their chemoproteomic profiling, and subsequent identification of the biological targets of a lead compound (<b>4a</b>) for this scaffold in <i>S. aureus</i>.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Keeping the aforementioned synthetic criteria in mind, we decided to synthesize a small yet focused library of IND-QEs. In order to synthesize the desired compounds, a series of indole-based quinones (compounds <b>1b–15b</b>, <b>Q1–Q2</b>) were first synthesized using the general procedure outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Procedure for the Synthesis of Indole-based Quinone Epoxides: (a) R<sup>1</sup>CH<sub>2</sub>PPh<sub>3</sub>Br (1.15 equiv), <i>n</i>-BuLi (1 equiv), and THF, −78 °C; (b) Benzoquinone (2.5 equiv), and Ethanol, RT</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The first series of compounds had varying substituents on the indole nitrogen (substituent R). These quinones were then converted to an epoxide using NaOCl (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). We tested the ability of these compounds to inhibit <i>S. aureus</i> ATCC 29213, and the minimum inhibitory concentration (MIC) was determined using a previously described protocol (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entries 1–5).<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IND-QE Analogues with Variations in R, Their MIC Values, and Reactivity with Thiol</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0008.gif" alt="" id="GRAPHIC-d7e446-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">yield (%)</th><th class="colsep0 rowsep0" align="center">MIC<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">% remaining<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"><b>1a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"><b>2a</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left"><b>4a</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"><sup><i>i</i></sup>Pr</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">0.125–0.5</td><td class="colsep0 rowsep0" align="left">34<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left"><b>5a</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">82</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">57</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">MIC in μg/mL against <i>S. aureus</i> ATCC 29213.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Determined by HPLC analysis: the compound was treated with <span class="smallcaps smallerCapital">l</span>-cysteine (1 equiv) in pH 7.4 phosphate buffer for 60 min.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">For X-ray analysis, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S1.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">For the HPLC trace, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S2.</p></div></div></div><div class="NLM_p">Based on this data, the <i>N</i>-isopropyl derivative, <b>4a</b>, was identified as the most potent inhibitor with a MIC of 0.125–0.5 μg/mL (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entry 4). To assess the nature of bacterial growth inhibitory activity of <b>4a</b>, a time-kill method was employed, and this compound was found to have robust bactericidal activity (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S3A). Next, <b>4a</b> was tested against a panel of clinical drug-resistant <i>S. aureus</i> strains and was found to be a potent inhibitor of all of these, including VRSA (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entries 1–8; for the entire panel, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S1). Time-kill analysis with VRSA strain HIP 11714 showed that <b>4a</b> had a potent bactericidal activity comparable with daptomycin, a drug which is frequently used to treat VRSA infections (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Time-kill analysis for <b>4a</b> (10× MIC, MIC = 0.5 μg/mL) and daptomycin (Dap, 10× MIC, MIC = 0.5 μg/mL) against VRSA strain HIP 11714 showing potent bactericidal activity of <b>4a</b>. For the entire plot (with vancomycin), see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S4, (<i>n</i> = 2); (B) Time courses of independent reactions of compounds <b>4a</b>, <b>9a</b>, <b>10a</b>, and <b>11a</b> with 10 equiv <span class="smallcaps smallerCapital">l</span>-cysteine. Curve fitting to an exponential decay afforded pseudo-first-order rate constants, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S2, (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. MIC (μg/mL) of <b>4a</b> against Various Pathogens Including VRSA</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">bacterium</th><th class="colsep0 rowsep0" align="center">strain</th><th class="colsep0 rowsep0" align="center" char=".">MIC of <b>4a</b></th><th class="colsep0 rowsep0" align="center">MIC of vancomycin</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"><i>S. aureus</i></td><td class="colsep0 rowsep0" align="left">SA-HIP-14300</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"><i>S. aureus</i></td><td class="colsep0 rowsep0" align="left">SA-HIP-11983</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">32–64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"><i>S. aureus</i></td><td class="colsep0 rowsep0" align="left">AIS100050</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">32–64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left"><i>S. aureus</i></td><td class="colsep0 rowsep0" align="left">SA-HIP-15178</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">32–64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left"><i>S. aureus</i></td><td class="colsep0 rowsep0" align="left">SA-71080</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">32–64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left"><i>S. aureus</i></td><td class="colsep0 rowsep0" align="left">HIP11714</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left"><i>S. aureus</i></td><td class="colsep0 rowsep0" align="left">HIP14300</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left"><i>S. aureus</i></td><td class="colsep0 rowsep0" align="left">1002434</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left"><i>E. coli</i></td><td class="colsep0 rowsep0" align="left">ATCC 252922</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="left"><0.01<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left"><i>K. pneumoniae</i></td><td class="colsep0 rowsep0" align="left">BAA 1705</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="left">64<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left"><i>E. faecalis</i></td><td class="colsep0 rowsep0" align="left">B3119</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="left">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left"><i>P. aeruginosa</i></td><td class="colsep0 rowsep0" align="left">ATCC 27853</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="left">0.5<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left"><i>A. baumanii</i></td><td class="colsep0 rowsep0" align="left">BAA 1605</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="left">8<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Ciprofloxacin was used.</p></div></div></div><div class="NLM_p">Because IND-QE was designed to react with thiols (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Scheme S1), we tested the propensity of these compounds to react with <span class="smallcaps smallerCapital">l</span>-cysteine. High-performance liquid chromatography (HPLC) analysis of reaction mixtures containing the epoxide and <span class="smallcaps smallerCapital">l</span>-cysteine revealed comparable thiol reactivity for <b>4a</b> and its analogues during 60 min (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entries 1–5). A time course of consumption for <b>4a</b> in the presence of <span class="smallcaps smallerCapital">l</span>-cysteine was next assessed, and this reaction followed pseudo-first-order kinetics with a rate constant of 7.54 × 10<sup>–2</sup> min<sup>–1</sup> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). To further confirm the formation of a thiol-adduct, compound <b>4a</b> was reacted with 4-nitro-benzyl thiol, and HPLC analysis showed the formation of a thiolated-adduct. This result was further confirmed by co-elution with a synthetic standard (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figures S5 and S6, Scheme S2). Together, these data suggest that <b>4a</b> reacts with a thiol to form a stable covalent adduct.</div><div class="NLM_p">In order to test the ability of the IND-QE scaffold to modulate reactivity with thiols, analogues of <b>4a</b>, our lead compound, with varying R<sup>1</sup> substituents, were synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, entries 3–4). We surprisingly found that addition of these substituents resulted in diminished reactivity (example, <b>9a</b>, R<sup>1</sup> = Et, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The differences in the rate constants were about 6-fold (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S2). While this difference is not small, it is this structural tool that provides remote control over reactivity. Given the difficulty in synthesis, preparation of compounds with the increased substituent size beyond an ethyl substituent at R<sup>1</sup> was unsuccessful.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IND-QE Analogues with Variations in R<sup>1</sup>, Their Reactivity with Thiol, and MIC Values</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0009.gif" alt="" id="GRAPHIC-d7e958-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">% remaining<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">MIC<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left">>16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">93</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"><b>8a</b></td><td class="colsep0 rowsep0" align="left"><sup><i>i</i></sup>Pr</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">84</td><td class="colsep0 rowsep0" align="left">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="left"><sup><i>i</i></sup>Pr</td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">64</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Determined by HPLC analysis: the compound was treated with <span class="smallcaps smallerCapital">l</span>-cysteine (1 equiv) in pH 7.4 phosphate buffer for 60 min.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">MIC in μg/mL against <i>S. aureus</i> ATCC 29213.</p></div></div></div><div class="NLM_p">Intrigued by this remote substituent effect, we decided to further investigate the structural aspects that may contribute to diminished reactivity. X-ray diffraction analysis of crystalline <b>4a</b> was conducted, and a boat-like conformation for the quinone epoxide was observed<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). In light of this structure, it is expected that during the thiol attack (attack at C-10, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), one flagpole of the boat is expected to approach planarity; and this process is likely hindered by neighboring substituents. Similarly, when attack happens at the C-1 position, a similar flattening of the boat is expected. Our data suggests that the neighboring substituent does play a role. Analysis of X-ray crystallographic data showed overlapping van der Waals radii<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> for the carbon attached to the nitrogen and the carbonyl adjacent to it. According to our data (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), the <i>N</i>-substituent contributes to only a marginal difference in relative thiol reactivity, which is consistent with the substituent being far from the reaction center. The substituent on the C-3 (R<sup>1</sup>) and the adjacent carbonyl also has overlapping van der Waals radii. Here, however, the substituent effect is larger, that is, 6-fold. Further evidence for this trend was obtained from compounds <b>6a</b> and <b>7a</b>, which had a methyl substituent at the C-3 position with different substituents at the indole nitrogen (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, entries 1–2). Here, we observed that the reactivity with thiol decreased uniformly in these cases (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, entries 1–2) when compared with their unsubstituted counterparts (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, entries 1 and 3). Together, these data support a preference for attack on C-1 over that on C-10.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) ORTEP diagram for <b>4a</b> with van der Waals radii overlap (dotted circles), for groups at R and R<sup>1</sup> with carbonyl oxygens; (B) ORTEP diagram for <b>4a</b> showing the boat-like conformation of the quinone epoxide, and schematic showing flagpole positions of quinone epoxide during the thiol attack.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Lastly, the analogues containing a phenyl ring on the carbons bearing the epoxide (substituents R<sup>2</sup> and R<sup>3</sup>) were synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entries 1–2, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Scheme S3 for synthesis of substituted benzoquinone). As expected, addition of a phenyl group on the carbon bearing the epoxide had an impact on the rate of thiol-mediated disappearance of the epoxide. The rate constant for the reaction of <b>11a</b> was determined as 0.78 × 10<sup>–2</sup> min<sup>–1</sup> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The 10-fold rate difference between <b>4a</b> and <b>11a</b> translates to roughly 1.4 kcal/mol difference in the activation barrier (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S2). However, the analogue <b>10a</b>, which had a substituent on the 10-position showed a rate difference of about 2-fold only (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S2). Together, these data corroborate our model that the attack on C-1 is more favorable than at C-10. Recently, Diness and co-workers have reported fluorobenzene-based probes for cysteine-selective modification.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Here, they varied the electronics on the aryl ring and found that the presence of an electron-withdrawing group increased the rate of reaction with cysteines while an electron-donating group reduced reactivity.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> This elegant design allows for tuneable reaction with thiols. For example, Pan and co-workers have earlier reported ibrutinib-based irreversible Bruton’s tyrosine kinase (BTK) inhibitors and studied a series of analogues by changing the Michael acceptor group on the pyrazole.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> This study suggests that the reactivity of the cysteine residue on BTK is critical for the inhibition and can be tuned by the structural modifications on the Michael acceptor installed. Further, leveraging advanced chemoproteomics, Cravatt and co-workers have employed alkyne-tagged irreversible probes selectively targeting EGFR (epidermal growth factor receptor) and BTK. In another study, Allimuthu and Adams have utilized 2-chloropropionamide as a steric-driven low reactivity electrophile that selectively acts as a PDI (protein disulfide isomerase) inhibitor.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Recently, Ojida and co-workers have reported reversible modification of kinase cysteines using chlorofluoroacetamides and have explored new warheads for TCI (targeted covalent inhibition) by inhibiting EGFRs.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> With IND-QE, the possibility of tuning the thiol reactivity with varying substituent size offers a unique and complementary approach for interrogating cysteines with a large range of p<i>K</i><sub>a</sub>s. The precise rationale for preferential attack on one carbon of the epoxide ring over the other remains to be deciphered. However, to the best of our knowledge, this is the first report where tunability of reactivity with thiol using an epoxide warhead is achieved by modulation of sterics around the electrophile.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30−32)</a></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. IND-QE Analogues Synthesized in the Study and Their MIC Values</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0010.gif" alt="" id="GRAPHIC-d7e1205-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">MIC<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"><b>10a</b><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">0.0625<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"><b>11a</b><a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>64<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-OMePh</td><td class="colsep0 rowsep0" align="left">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left"><b>13a</b></td><td class="colsep0 rowsep0" align="left">4-OMePh</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left"><b>14a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-BrPh</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left"><b>15a</b><a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">4-BrPh</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>64</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">MIC in μg/mL against <i>S. aureus</i> ATCC 29213.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">For the X-ray structure of <b>10a</b>, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S7.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Time-kill analysis with <i>S. aureus</i> ATCC 29213 showed that this compound had a bactericidal activity (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S3B).</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">For the X-ray structure of <b>11a</b>, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S8.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">For the HPLC trace, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S9.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">For the X-ray structure of <b>15a</b>, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S10.</p></div></div></div><div class="NLM_p">We compared the effects of these substitutions on the <i>S. aureus</i> growth inhibitory activity. In general, we observed that lowering reactivity with a thiol, reduced inhibitory activity. For example, a significant lowering of potency with the increased substituent size at R<sup>1</sup> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, entries 3–4). We also found that the MICs of <b>10a</b> and <b>11a</b> against <i>S. aureus</i> were in accordance with their thiol reactivity (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entries 1–2).</div><div class="NLM_p">Similar observations were made with their analogues (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, entries 3–6), which were synthesized using similar techniques. To expand the structure–activity relationship within this scaffold and further understand the role of indole in the antibacterial activity, we synthesized NQE derivatives with and without substituent at the carbon bearing the epoxide and tested their inhibitory activity against <i>S. aureus</i> ATCC 29213 (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S3). Not surprisingly, we found that these compounds were significantly less effective in inhibiting <i>S. aureus</i> compared to their indole counterparts. Furthermore, the quinones<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> containing indoles were also relatively poor inhibitors (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S4). Previously, Sieber and co-workers have reported aminoepoxycyclohexenones with higher MICs; one analogue with an indole tether was found to have good potency against <i>S. aureus</i>.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Thus, the indole moiety plays a major role in enhancing the permeability and activity of this quinone epoxide scaffold.</div><div class="NLM_p">Based on the MIC measurements, the most active compound with comparatively easier synthetic access, <b>4a</b>, and an antibacterial activity comparable to that of vancomycin, was chosen as the lead compound for subsequent investigations. To assess the selectivity of this compound, we screened its antibacterial activity against <i>Escherichia coli</i>, <i>S. aureus</i>, <i>Klebsiella pneumoniae</i>, <i>Acinetobacter baumannii</i>, <i>Pseudomonas aeruginosa,</i> and <i>Enterococcus faecalis</i> (ESKAPE pathogens) (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, also see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S5). We found that <b>4a</b> has an exquisitely selective inhibitory activity against <i>S. aureus</i>, with no significant activity against any other pathogen. Taken together, IND-QE <b>4a</b> was a selective and potent inhibitor of VRSA, and thus, becomes an excellent starting point for chemoproteomic profiling to identify its biological targets.</div><div class="NLM_p">To further understand the pharmacology, two IND-QE alkyne probes, <b>P1</b> and <b>P2</b>, were synthesized (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A; for X-ray data of <b>P2</b>, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S11) based on <b>4a</b> and <b>11a</b> scaffolds, respectively, that would enable bioorthogonal approaches (click chemistry) in pursuit of identifying the biological targets of <b>4a</b>. The relative thiol reactivity of these two probes was evaluated using a competition experiment against monobromobimane,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and both probes were found to be comparable in reactivity with their respective parent IND-QE derivatives (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S12, Scheme S4). We confirmed that <b>P1</b> (MIC, 1 μg/mL) and <b>P2</b> (MIC, >64 μg/mL) had inhibitory potencies against <i>S. aureus</i> comparable to the parent compounds <b>4a</b> and <b>11a</b> respectively, and proceeded with chemoproteomics experiments using these alkyne compounds. First, <i>S. aureus</i> cell lysates were titrated with increasing concentrations of <b>P1</b> and <b>P2</b>. In this experiment, compound incubation was done for 1 h, following which Cu-catalyzed click reaction with rhodamine-azide was carried out, and the proteomes were resolved and visualized by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis using standard gel-based ABPP protocols.<a onclick="showRef(event, 'ref25 ref35'); return false;" href="javascript:void(0);" class="ref ref25 ref35">(25,35)</a> The gel-based ABPP experiment showed robust dose-dependent labeling of the <i>S. aureus</i> soluble protein lysate following <b>P1</b> treatment but very minimal labeling with <b>P2</b> in comparison (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Fractionated membrane lysates also suggested that <b>P1</b> has robust protein reactivity, in comparison to <b>P2</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S13a). Qualitatively, we found from these in vitro experiments that <b>P1</b> has more reactivity in the soluble lysates as compared to the membrane proteome.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Structures of IND-QE alkyne probes <b>P1</b> and <b>P2</b> synthesized in this study; (B) dose-dependent proteome profiling with <b>P1</b> (0.01–100 μM) and <b>P2</b> (0.01–100 μM) in soluble proteomic fraction of <i>S. aureus</i> ATCC 29213; (C) proteome profiling with 10 μM of <b>P1</b> in lysates of <i>S. aureus</i> ATCC 29213 compared with IAM pretreated fractions; and (D) proteome profiling with 10 μM of <b>P1</b> and <b>P2</b> in situ for <i>S. aureus</i> ATCC 29213.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, to determine if the site of labeling was indeed a protein cysteine residue, we performed a competitive gel-based ABPP experiment, where fractionated <i>S. aureus</i> cell lysates (soluble and membrane proteomes) were pretreated with excess iodoacetamide (IAM) (10 mM) and chased with <b>P1</b> (10 μM). We found that there was near complete ablation of <b>P1</b> protein labeling when lysates were pretreated with IAM, suggesting that <b>P1</b> indeed, and as per literature precedence,<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> reacted with protein cysteine residues (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). Further, to assess if <b>P1</b> and <b>P2</b> were active in situ, <i>S. aureus</i> cultures were treated with <b>P1</b> and <b>P2</b> for 1 h (10 μM), following which the cells were lysed, lysate fractionated into membrane and soluble proteomes, and the extent of <b>P1</b> and <b>P2</b> protein reactivity in each fraction was analysed by gel-based ABPP, following click reaction with rhodamine-azide. We found consistent result with the previously described in vitro studies, in which <b>P1</b> was significantly more reactive than <b>P2</b> in <i>S. aureus</i>, and the extent of reactivity was even more pronounced in the soluble proteome, suggesting that the potential biological target(s) might be soluble proteins (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S14 for all Coomassie gels).</div><div class="NLM_p">Given the exquisite selectivity of the lead compound <b>4a</b> for <i>S. aureus</i> over other ESKAPE pathogens, we tried to understand the basis of selectivity and chose to chemoproteomically profile <b>P1</b> in <i>E. coli</i>. We found that in vitro <b>P1</b> was fairly reactive against <i>E. coli</i> lysates but had virtually no reactivity when <b>P1</b> was used in an in situ treatment paradigm in <i>E. coli</i> (See <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S13b). This result suggests that this class of compounds might lack cell permeability against Gram-negative bacteria. Having obtained significantly more labeling of soluble proteins over membrane proteins from the in situ experiments, we hypothesized that the target(s) of interest would likely be present in the soluble proteome. To find the biological targets of <b>4a</b> in <i>S. aureus</i>, we performed a competitive LC–MS/MS-based ABPP experiment. In this experiment, we incubated the <i>S. aureus</i> soluble proteome with DMSO or <b>4a</b> (10 μM, 1 h) or <b>11a</b> (10 μM, 1 h) or IAM (10 mM, 1 h), and then chased these pretreated proteomes with <b>P1</b> (10 μM, 1 h) and following <b>P1</b> treatment performed click reaction using biotin–azide. Subsequently, the biotinylated proteins were enriched using avidin–agarose beads. Following enrichment, we performed on-bead trypsin digestion using established protocols.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> The tryptic peptides were subjected to LC–MS/MS analysis to identify the respective proteins enriched in each treatment group (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, proteomics dataset). We hypothesized that proteins, which were absent (or largely competed) in the <b>4a</b> and IAM treatment groups (ratio relative to DMSO group ≤0.1) but present comparable to control in the <b>11a</b> treatment group (ratio relative to DMSO group ∼1), were likely targets of <b>4a</b>. Using these filtering criteria of the 435 proteins identified by LC–MS/MS, we found only a handful of proteins (17 proteins) that were likely targets of <b>4a</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Interestingly, we found from this proteomics experiment that a significant number of these protein targets of <b>4a</b> were in fact transcriptional regulators (also known as virulence factors) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) that have implications in conferring antibiotic resistance to microbes<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and have to date been thought as “undruggable”.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Targets identified for <b>4a</b> using LC–MS/MS-based proteomics experiment; proteins that exhibited ratios of <0.1 for <b>4a</b> and IAM (red color) and ∼1 for <b>11a</b> (blue color) were considered targets of <b>4a</b>. (B) Pie-chart categorizing the identified targets of <b>4a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A bioinformatics survey of the amino acid sequences of the protein targets identified for <b>4a</b> showed, as predicted, that every protein target identified from the LC–MS/MS proteomics experiment had at least one conserved cysteine residue across ESKAPE pathogens, reinforcing our hypothesis that <b>4a</b> forms a covalent adduct with a protein cysteine residue. We were particularly interested in the two protein targets from the multiple antibiotic resistance regulon (MarR)<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39−41)</a> family of transcriptional factors, namely MarR_12840 and MarR_05815, that were identified as putative targets of <b>4a</b>, as previous studies have shown that inactivating mutations to either of these genes results in increased antibiotic vulnerabilities in <i>E. coli</i>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p">Interestingly, both these MarR proteins have only one conserved cysteine residue, Cys61 and Cys12 for MarR_12840 and MarR_05815, respectively. To validate whether these two proteins are indeed targets of <b>4a</b>, we cloned and purified these recombinant C-terminal 6X-His-tag proteins from <i>E. coli</i> by Ni-NTA affinity chromatography. We also made alanine point variants of Cys61 (C61A) and Cys12 (C12A) for MarR_12840 and MarR_05815, respectively. We then treated the wild type (WT) and the variant proteins with the increasing dose of <b>P1</b> (0–100 μM, 1 h), following which we performed click-reaction using rhodamine-azide and visualized the activity of <b>P1</b> labeling by SDS-PAGE. We found for both proteins, MarR_12840 and MarR_05815, the WT variant displayed a robust dose-dependent labeling with <b>P1</b>, while the C61A and C12A variants of both MarR_12840 and MarR_05815, respectively, showed negligible labeling for <b>P1</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Of note, the three-dimensional structures are available for both proteins,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> and the Cys61 in MarR_12840 and Cys12 in MarR_05815 are functionally important residues responsible for transcriptional activation<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and sensing cellular oxidative stress,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> respectively (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Dose-dependent labeling of targets MarR_12840 and MarR_05815 with varying concentrations of probe <b>P1</b>, as compared with their alanine point variants; crystal structures of identified targets with Cys residues at Cys61 and Cys12, respectively (in black spheres).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Taken together, we report a new class of cell active inhibitors against <i>S. aureus</i>, and leveraging chemoproteomics identify the biological targets for the lead compound <b>4a</b>. The lead compound is highly selective in modifying cysteine residues on proteins and inhibits multidrug resistant <i>S. aureus</i> with a low MIC value. The most important identified targets are transcriptional factors, which are responsible for conferring virulence and antibiotic resistance in <i>S. aureus</i>. In a complementary approach, previously, using Michael acceptor-based ABPP, Sieber and co-workers have identified cysteine residues of SarA, SarR, and MgrA that bind to DNA as targets for inhibiting virulence that reduces bacterial pathogenicity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The MarR family proteins are novel druggable targets in <i>S. aureus,</i> and this is the first study, to the best of our knowledge, that reports these two putative MarR proteins as important and novel targets. This study creates novel paradigms and possibilities in combating antibiotic resistance and provides new insights into possibly novel drug targets.</div><div class="NLM_p last">As the golden era of antibiotics is nearing its end, finding new strategies to overcome antibiotic resistance is of urgent need, as it is predicted that even routine infections are likely to be untreatable by current antibiotics in the near future. This has underscored the importance of finding new druggable targets in highly prevalent pathogens such as <i>S. aureus</i> and VRSA. Our study reveals the importance of covalently modifying cysteine residues of transcription factors, which may have a profound impact on survival of highly drug-resistant <i>S. aureus</i>. Although covalent modification of cysteine residues<a onclick="showRef(event, 'ref10 ref44'); return false;" href="javascript:void(0);" class="ref ref10 ref44">(10,44)</a> is frequently associated with idiosyncratic toxicities, it presents an attractive alternative to currently used drug design strategies.<a onclick="showRef(event, 'ref19 ref45'); return false;" href="javascript:void(0);" class="ref ref19 ref45">(19,45)</a> In this study, cytotoxicity assessment of lead compound <b>4a</b> against mammalian cell line VERO cells showed a CC<sub>50</sub> > 25 μg/mL, suggesting a higher selectivity index of >200 (the selectivity index of <b>10a</b> was also >200; see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S6). Additionally, we found no evidence for hemolysis at elevated concentrations suggesting that this compound did not adversely affect human erythrocytes (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S15).</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10474" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10474" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Projecting ahead, we present a small molecular scaffold that lays the foundation for new approaches to address the problem of AMR. It is likely that structural optimization of this scaffold will be necessary for further development and in vivo efficacy. It has been previously demonstrated that modification of cysteines in transcription factors reduces DNA binding ability<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and hence, screening for MarR inhibitors may hold promise for identification of new antibacterial candidates. However, the inhibitory effects reported in this study might likely be due to the pharmacological disruption of more than one transcriptional factor(s), and the polypharmacological effects of <b>4a</b> will need to be assessed by complementary genetic experiments (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). While it may prove challenging to assess the exact extent of polypharmacological effect from a medicinal chemistry perspective, the advantages of multiple target engagements may contribute to positive attributes of <b>4a</b>. It would be extremely challenging for <i>S. aureus</i> to simultaneously generate multiple mutations in the different protein targets to develop resistance to <b>4a</b>. With IND-QE, the structural features allowing for tunability of reactivity with cysteines coupled with excellent <i>S. aureus</i> cell permeability is a major advancement that can be leveraged to profile thiols in a variety of pathophysiologies. However, the diminished permeability of IND-QE in Gram-negative bacteria appears to be a significant impediment for profiling in such pathogens. This will be addressed in future studies by structural modifications that enhance permeability of the present scaffold.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">4.  Experimental Procedures (Chemistry)</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All reactions were performed under a nitrogen atmosphere. All the chemicals were purchased from Aldrich, Acros, TCI, Spectrochem, Rankem, Alfa Aesar and used as received unless stated otherwise. Dichloromethane, ethanol, <i>N</i>,<i>N</i>-dimethylformamide, toluene, and tetrahydrofuran (THF) for reaction were used as received, and petroleum ether and ethyl acetate (EtOAc) for chromatography were distilled before use. Column chromatography was performed on Merck silica gel (60–120 mesh). <sup>1</sup>H and <sup>13</sup>C spectra were recorded on JEOL or Bruker 400 MHz (or 100 MHz for <sup>13</sup>C) spectrometers using residual solvent signals as an internal standard (CHCl<sub>3</sub><sup>δ</sup>H, 7.26 ppm, <sup>δ</sup>C 77.2 ppm) or tetramethylsilane (<sup>δ</sup>H = 0.00, <sup>δ</sup>C = 0.0). Chemical shifts (δ) are reported in ppm and coupling constants (<i>J</i>) in Hz. The following abbreviations are used: br (broad), m (multiplet), s (singlet), d (doublet), t (triplet), q (quartet), h (heptet), ddd (doublet of doublet of doublet), dd (doublet of doublet), dt (doublet of triplet), dq (doublet of quartet), td (triplet of doublet), and ABq (AB quartet). High-resolution mass spectra were obtained from HRMS-ESI-Q-time-of-flight LC/MS. FT-IR spectra were recorded using a NICOLET 6700 FT-IR spectrometer as the KBr disc for some compounds and using Bruker ALPHA FT-IR spectrometer for other samples, reported in cm<sup>–1</sup>. Melting point was measured using a VEEGO melting point apparatus. All melting points were measured in an open glass capillary, and the values are uncorrected. HPLC was performed on an Agilent model with a Zorbax SB C-18 reversed phase column (250 mm × 4.6 mm, 5 μm) and with a Phenomenex C-18 reversed phase column (250 mm × 4.6 mm, 5 μm). Fluorimetric and luminometric measurements were performed using a Thermo Scientific Varioskan microtiter plate reader. All compounds reported were found to be ≥95% pure, as determined by elemental analysis and/or HPLC analysis. HPLC traces for key compounds are included in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">4.1.  General Procedure for Diels–Alder Reaction<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></h3><div class="NLM_p last">To a solution of the Wittig adduct in ethanol (10 mL), benzoquinone or arylated benzoquinone was added, and the reaction mixture was stirred at RT in an open round-bottom flask. Upon complete consumption of the starting material (TLC analysis), ethanol was evaporated from the reaction mixture under reduced pressure. Silica gel chromatography of the crude using a mixture of ethyl acetate and hexane (1:5, v/v) as an eluent gave the pure product. This procedure was followed for the synthesis of compounds <b>1b–15b</b>, <b>Q1–Q2</b>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">4.2.  General Procedure for Epoxidation<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></h3><div class="NLM_p last">To a solution of the quinone (compounds <b>1b–15b</b>, <b>Q1–Q2</b>) in THF, sodium hypochlorite (9–12 wt % in water, excess) was added, and the reaction mixture was stirred at RT in an open round-bottom flask. Upon complete consumption of the starting material (TLC analysis), the reaction mixture was diluted with water (10 mL) and extracted with multiple portions of ethyl acetate (5 × 5 mL). The combined organic phase was washed with brine (10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> (5 g), filtered and the filtrate was evaporated to dryness to get the crude product. Silica gel chromatography of the crude using a mixture of ethyl acetate and hexane (1:5, v/v) as an eluent gave the pure product. This procedure was followed for the synthesis of compounds <b>1a–15a</b>, <b>P1–P2</b>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">4.3.  Preparation and Characterization of Compounds</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14">4.3.1.  1a<i>H</i>-Oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>1a</b>)</h4><div class="NLM_p last">Starting from <b>1b</b> (70 mg, 0.28 mmol), <b>1a</b> (70 mg, 94%) was isolated as an orange yellow crystalline solid: mp 226–228 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 3397, 2921, 1681, 1587, 1114; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.97 (s, 1H), 8.58 (dd, <i>J</i> = 7.9, 1.6 Hz, 1H), 8.23 (d, <i>J</i> = 7.9 Hz, 1H), 7.74 (dd, <i>J</i> = 16.6, 8.2 Hz, 2H), 7.50 (t, <i>J</i> = 7.6 Hz, 1H), 7.25 (t, <i>J</i> = 7.5 Hz, 1H), 4.17 (dd, <i>J</i> = 10.4, 3.9 Hz, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 192.3, 191.6, 143.1, 137.1, 130.0, 129.1, 128.6, 127.2, 121.8, 121.2, 121.0, 117.3, 114.5, 113.4, 55.8, 55.7; HRMS (ESI) for C<sub>16</sub>H<sub>9</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: calcd, 264.0655; found, 264.0660.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15">4.3.2.  11<i>H</i>-Benzo[<i>a</i>]carbazole-1,4-dione (<b>1b</b>)</h4><div class="NLM_p last">Starting from 3-vinyl-1<i>H</i>-indole (510 mg, 3.56 mmol), <b>1b</b> was isolated as a dark red-brown solid (560 mg, 64%): mp 251–253 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1646, 1566, 1414, 1303, 1216, 1114, 1019; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.40 (br, 1H), 8.33 (d, <i>J</i> = 8.1 Hz, 1H), 8.19 (d, <i>J</i> = 7.8 Hz, 1H), 7.92 (d, <i>J</i> = 7.9 Hz, 1H), 7.52–7.54 (m, 2H), 7.28–7.32 (m, 1H), 6.93 (s, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 187.0, 185.8, 143.4, 139.4, 139.0, 137.0, 130.1, 129.3, 128.6, 126.5, 121.8, 121.3, 121.0, 119.9, 117.1, 113.4; HRMS (ESI) for C<sub>16</sub>H<sub>9</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 248.0712; found, 248.0711.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16">4.3.3.  9-Methyl-1a<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>2a</b>)</h4><div class="NLM_p last">Starting from <b>2b</b> (50 mg, 0.19 mmol), <b>2a</b> (50 mg, 94%) was isolated as a yellow crystalline solid: mp 177–179 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 2924, 1683, 1471, 1288, 1025; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.33 (d, <i>J</i> = 7.8 Hz, 1H), 8.10 (d, <i>J</i> = 7.8 Hz, 1H), 7.87 (d, <i>J</i> = 8.2 Hz, 1H), 7.60 (t, <i>J</i> = 7.8 Hz, 1H), 7.49 (d, <i>J</i> = 8.4 Hz, 1H), 7.32 (t, <i>J</i> = 7.5 Hz, 1H), 4.12 (s, 2H), 3.80 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 192.1, 191.3, 144.6, 138.6, 131.0, 130.0, 128.6, 125.3, 121.6, 121.0, 120.9, 118.2, 110.1, 56.3, 54.9, 34.6; HRMS (ESI) for C<sub>17</sub>H<sub>11</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: calcd, 278.0812; found, 278.0825.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17">4.3.4.  11-Methyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>2b</b>)</h4><div class="NLM_p last">Starting from 1-methyl-3-vinyl-1<i>H</i>-indole (1 g, 6.36 mmol), <b>2b</b> was isolated as a dark red-brown solid (220 mg, 14%): mp 184–186 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1652, 1611, 1466, 1290, 1217, 1122, 1081, 1024; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.55 (d, <i>J</i> = 8.0 Hz, 1H), 8.25 (d, <i>J</i> = 7.8 Hz, 1H), 7.84 (d, <i>J</i> = 8.0 Hz, 1H), 7.70 (d, <i>J</i> = 8.3 Hz, 1H), 7.56–7.60 (m, 1H), 7.30 (t, <i>J</i> = 7.3 Hz, 1H), 7.10 (d, <i>J</i> = 10.2 Hz, 1H), 7.00 (d, <i>J</i> = 10.3 Hz, 1H), 3.87 (s, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 185.7, 185.6, 145.3, 141.0, 138.8, 137.0, 131.0, 130.7, 128.9, 125.7, 121.6, 121.4, 118.0, 111.4, 35.8; HRMS (ESI) for C<sub>17</sub>H<sub>11</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 262.0868; found, 262.0865.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18">4.3.5.  9-Ethyl-1a<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>3a</b>)</h4><div class="NLM_p last">Starting from <b>3b</b> (50 mg, 0.18 mmol), <b>3a</b> (51 mg, 96%) was isolated as a bright yellow crystalline solid: mp 145–147 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 3042, 2964, 1682, 1626, 1432, 1202, 1131, 1026; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.29–8.32 (m, 1H), 8.07 (d, <i>J</i> = 7.7 Hz, 1H), 7.83 (dd, <i>J</i> = 9.1, 2.5 Hz, 1H), 7.49–7.58 (m, 2H), 7.30 (t, <i>J</i> = 7.4 Hz, 1H), 4.36–4.51 (m, 2H), 4.10–4.14 (m, 2H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 192.5, 191.5, 143.6, 136.8, 131.3, 130.1, 128.5, 125.3, 122.0, 121.0, 120.9, 118.1, 117.6, 110.3, 56.1, 54.7, 40.6, 13.1; Elemental analysis for C<sub>18</sub>H<sub>13</sub>NO<sub>3</sub> calcd: C, 74.22; H, 4.50; N, 4.81. Found: C, 74.18; H, 4.33; N, 4.74; HRMS (ESI) for C<sub>18</sub>H<sub>13</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 292.0968; found, 292.0983.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19">4.3.6.  11-Ethyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>3b</b>)</h4><div class="NLM_p last">Starting from <i>N</i>-ethyl-3-vinylindole (1 g, 5.83 mmol), <b>3b</b> was isolated as a dark red-brown solid (285 mg, 18%): mp 138–140 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1740, 1643, 1512, 1443, 1375, 1221, 1038; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.32 (d, <i>J</i> = 7.9 Hz, 1H), 8.07 (d, <i>J</i> = 7.7 Hz, 1H), 8.00 (d, <i>J</i> = 8.0 Hz, 1H), 7.51–7.58 (m, 2H), 7.28–7.32 (m, 1H), 6.92 (q, <i>J</i> = 14.0 Hz, 2H), 4.63 (q, <i>J</i> = 10.6 Hz, 2H), 1.40 (t, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 185.6, 183.4, 144.3, 140.2, 136.9, 131.4, 128.4, 125.1, 122.2, 121.0, 120.9, 118.6, 118.2, 110.8, 42.6, 14.4; HRMS (ESI) for C<sub>18</sub>H<sub>13</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 276.1025; found, 276.1033.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20">4.3.7.  9-Isopropyl-1a<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>4a</b>)</h4><div class="NLM_p last">Starting from <b>4b</b> (70 mg, 0.242 mmol), <b>4a</b> (74 mg, 96%) was isolated as a bright yellow crystalline solid: mp 204–206 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 2968, 1683, 1622, 1569, 1416, 1373, 1200, 1116, 1026; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.24 (d, <i>J</i> = 8.0 Hz, 1H), 8.06 (d, <i>J</i> = 7.8 Hz, 1H), 7.83 (d, <i>J</i> = 7.8 Hz, 1H), 7.72 (d, <i>J</i> = 8.5 Hz, 1H), 7.50 (td, <i>J</i> = 8.2, 1.0 Hz, 1H), 7.26 (dd, <i>J</i> = 14.1, 7.6 Hz, 1H), 4.64–4.74 (m, 1H), 4.10 (s, 2H), 1.96 (d, <i>J</i> = 6.8 Hz, 3H), 1.39 (d, <i>J</i> = 6.9 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 193.3, 191.1, 142.1, 138.6, 131.0, 130.1, 127.8, 125.0, 123.4, 121.2, 120.7, 118.4, 117.8, 113.9, 56.5, 54.6, 51.3, 20.7, 20.5; HRMS (ESI) for C<sub>19</sub>H<sub>15</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: calcd, 306.1125; found, 306.1126.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21">4.3.8.  11-Isopropyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4-(11<i>H</i>)-dione (<b>4b</b>)</h4><div class="NLM_p last">Starting from <i>N</i>-isopropyl-3-vinylindole (765 mg, 4.13 mmol), <b>4b</b> was isolated as a dark red-brown solid (280 mg, 23%): mp 138–140 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1740, 1643, 1512, 1443, 1375, 1038; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.32 (d, <i>J</i> = 7.8 Hz, 1H), 8.05 (d, <i>J</i> = 7.7 Hz, 1H), 7.96 (d, <i>J</i> = 7.8 Hz, 1H), 7.71 (d, <i>J</i> = 8.4 Hz, 1H), 7.44–7.48 (m, 1H), 7.25 (t, <i>J</i> = 7.4 Hz, 1H), 6.92 (d, 2H), 4.85–4.96 (m, 1H), 1.71 (d, <i>J</i> = 6.9 Hz, 6H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 186.3, 185.8, 142.9, 140.0, 139.7, 136.8, 136.6, 131.5, 131.4, 127.6, 124.9, 123.7, 121.2, 120.8, 118.8, 118.4, 114.4, 52.3, 21.2; HRMS (ESI) for C<sub>19</sub>H<sub>15</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 290.1181; found, 290.1184.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22">4.3.9.  9-Phenyl-1a<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>5a</b>)</h4><div class="NLM_p last">Starting from <b>5b</b> (37 mg, 0.11 mmol), <b>5a</b> (32 mg, 82%) was isolated as a bright yellow crystalline solid: mp 187–189 °C; FT-IR (ν<sub>max</sub>): 2935, 1687, 1457, 1283, 1206, 1110; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.35 (d, <i>J</i> = 7.9 Hz, 1H), 8.12 (d, <i>J</i> = 7.8 Hz, 1H), 7.90 (d, <i>J</i> = 7.9 Hz, 1H), 7.15 (s, 2H), 7.42–7.49 (m, 2H), 7.31–7.36 (m, 2H), 6.95 (s, 1H), 3.99 (d, <i>J</i> = 4.2 Hz, 1H), 3.76 (d, <i>J</i> = 4.2 Hz, 1H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 191.2, 190.4, 144.8, 139.5, 138.2, 131.3, 129.9, 128.6, 128.1, 127.4, 125.2, 125.1, 121.9, 121.7, 121.0, 118.9, 117.6, 111.2, 55.9, 54.2; HRMS (ESI) for C<sub>22</sub>H<sub>13</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 340.0968; found, 340.0974.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23">4.3.10.  11-Phenyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>5b</b>)</h4><div class="NLM_p last">Starting from <i>N</i>-phenyl-3-vinylindole (160 mg, 0.73 mmol), <b>5b</b> was isolated as a dark red-brown solid (50 mg, 21%): mp 218–220 °C; FT-IR (ν<sub>max</sub>): 1655, 1611, 1463, 1412, 1289; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.42 (d, <i>J</i> = 7.7 Hz, 1H), 8.14 (d, <i>J</i> = 7.8 Hz, 1H), 8.10 (d, <i>J</i> = 7.6 Hz, 1H), 7.54 (t, <i>J</i> = 7.5 Hz, 2H), 7.45 (t, <i>J</i> = 7.5 Hz, 2H), 7.26–7.36 (m, 4H), 6.84 (d, <i>J</i> = 10.0 Hz, 1H), 6.67 (d, <i>J</i> = 10.0 Hz, 1H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 185.6, 184.1, 145.7, 141.1, 139.6, 138.3, 137.1, 136.7, 132.0, 131.2, 129.3, 128.6, 127.7, 126.5, 125.2, 122.0, 121.8, 120.8, 119.4, 117.9, 111.9; HRMS (ESI) for C<sub>22</sub>H<sub>13</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 324.1025; found, 324.1016.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24">4.3.11.  3-Methyl-1a<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>6a</b>)</h4><div class="NLM_p last">Starting from <b>6b</b> (50 mg, 0.19 mmol), <b>6a</b> (45 mg, 85%) was isolated as an orange red crystalline solid: mp 224–226 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 2252, 1740, 1643, 1512, 1443, 1375, 1038; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.16 (s, 1H), 8.15 (s, 1H), 8.05 (d, <i>J</i> = 7.8 Hz, 1H), 7.47–7.52 (m, 2H), 7.29 (td, <i>J</i> = 5.8 Hz, 2.2 Hz, 1H), 4.03 (q, <i>J</i> = 9.5 Hz, 2H), 2.74 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 193.4, 193.2, 142.1, 137.4, 131.5, 130.2, 129.2, 128.6, 121.4, 121.0, 111.8, 55.7, 55.5, 22.0; HRMS (ESI) for C<sub>17</sub>H<sub>11</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: calcd, 278.0817; found, 278.0813.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25">4.3.12.  11-Ethyl-5-methyl-2,3-dihydro-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>6b</b>)</h4><div class="NLM_p last">Starting from 3-(1-propenyl)-indole (1 g, 6.36 mmol), <b>6b</b> was isolated as a dark red-brown solid (235 mg, 14%): mp 224–226 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1739, 1646, 1367, 1224; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.46 (s, 1H), 8.05 (s, 1H), 8.01 (d, <i>J</i> = 8.5 Hz, 1H), 7.48 (dd, <i>J</i> = 1.2, 6.2 Hz, 2H), 7.23–7.27 (m, 1H), 6.82 (d, <i>J</i> = 1.9 Hz, 2H), 2.80 (s, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 188.1, 187.1, 142.5, 140.5, 137.5, 136.9, 132.6, 129.6, 128.5, 126.3, 121.5, 121.2, 120.8, 115.2, 111.7, 23.5; HRMS (ESI) for C<sub>17</sub>H<sub>11</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 262.0868; found, 262.0863.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">4.3.13.  9-Ethyl-3-methyl-1a<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>7a</b>)</h4><div class="NLM_p last">Starting from <b>7b</b> (45 mg, 0.15 mmol), <b>7a</b> (37 mg, 71%) was isolated as a yellow solid: mp 159–161 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1680, 1427, 1258, 1199, 1124; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.17 (s, 1H), 8.08 (d, <i>J</i> = 7.8 Hz, 1H), 7.57 (t, <i>J</i> = 7.6 Hz, 1H), 7.50 (d, <i>J</i> = 8.3 Hz, 1H), 7.29 (t, <i>J</i> = 7.4 Hz, 1H), 4.40 (q, <i>J</i> = 14.2 Hz, 2H), 4.17 (d, <i>J</i> = 4.6 Hz, 1H), 4.13 (d, <i>J</i> = 4.5 Hz, 1H), 2.73 (s, 3H), 1.24 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 193.7, 193.4, 143.9, 135.5, 130.5, 130.3, 128.4, 127.8, 121.7, 121.0, 120.6, 118.4, 110.1, 56.3, 54.6, 40.1, 21.9, 12.8; HRMS (ESI) for C<sub>19</sub>H<sub>15</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: calcd, 306.1125; found, 306.1133.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27">4.3.14.  11-Ethyl-5-methyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>7b</b>)</h4><div class="NLM_p last">Starting from <i>N</i>-ethyl-3-(1-propenyl)-indole (350 mg, 1.88 mmol), <b>7b</b> was isolated as a dark red-brown solid (70 mg, 13%): mp 132–134 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1740, 1645, 1512, 1443, 1375, 1221, 1038; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.13 (d, <i>J</i> = 3.3 Hz, 1H), 8.05 (d, <i>J</i> = 7.7 Hz, 1H), 7.50–7.57 (m, 2H), 7.26–7.30 (m, 1H), 6.91 (dd, <i>J</i> = 10.1,1.3 Hz, 1H), 6.84 (dd, <i>J</i> = 10.1,1.3, 1H), 4.51 (dq, <i>J</i> = 7.1, 1.7 Hz, 2H), 2.83 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 187.7, 186.6, 145.1, 138.7, 138.0, 132.7, 131.1, 129.0, 128.4, 121.0, 120.8, 110.8, 100.0, 42.1, 23.8, 13.7; HRMS (ESI) for C<sub>19</sub>H<sub>15</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 290.1181; found, 290.1176.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28">4.3.15.  9-Isopropyl-3-methyl-1a<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>8a</b>)</h4><div class="NLM_p last">Starting from <b>8b</b> (crude, 55 mg, 0.18 mmol), <b>8a</b> (48 mg, 84%) was isolated as a yellow solid: mp 177–179 °C; FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1680, 1264, 1202, 1192, 1031; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.10 (s, 1H), 8.06 (d, <i>J</i> = 7.8 Hz, 1H), 7.69 (d, <i>J</i> = 8.5 Hz, 1H), 7.49 (dt, <i>J</i> = 6.8, 0.9 Hz, 1H), 7.27 (d, <i>J</i> = 7.4 Hz, 1H), 4.55–4.66 (m, 1H), 4.12 (ABq, <i>J</i> = 9.5 Hz, 2H), 2.72 (s, 3H), 1.92 (d, <i>J</i> = 6.8 Hz, 3H), 1.38 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 194.9, 193.0, 142.5, 137.6, 130.9, 129.9, 128.1, 127.7, 123.1, 121.2, 120.4, 118.6, 113.8, 56.6, 54.5, 51.1, 22.1, 20.7, 20.3; Elemental analysis for C<sub>20</sub>H<sub>17</sub>NO<sub>3</sub> calcd: C, 75.22; H, 5.37; N, 4.39. Found: C, 75.55; H, 5.16; N, 4.03; HRMS (ESI) for C<sub>20</sub>H<sub>17</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: calcd, 320.1281; found, 320.1286.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29">4.3.16.  3-Ethyl-9-isopropyl-1a<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>,10a<i>H</i>)-dione (<b>9a</b>)</h4><div class="NLM_p last">Starting from <b>9b</b> (250 mg, 0.79 mmol), <b>9a</b> (200 mg, 76%) was isolated as a yellow solid. FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1741, 1725, 1706, 1679, 1647, 1626, 1547, 1532, 1516, 1478, 1464, 1426, 1395, 1367, 1310, 1200; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.18 (s, 1H), 8.11 (d, <i>J</i> = 7.8 Hz, 1H), 7.71 (d, <i>J</i> = 8.4 Hz, 1H), 7.51 (t, <i>J</i> = 7.6 Hz, 1H), 7.28 (t, <i>J</i> = 7.6 Hz, 1H), 4.63 (h, <i>J</i> = 6.9 Hz, 1H), 4.16 (d, <i>J</i> = 4.6 Hz, 1H), 4.15 (d, <i>J</i> = 4 Hz, 1H), 3.21–3.03 (m, 2H), 1.94 (d, <i>J</i> = 6.9 Hz, 3H), 1.42 (d, <i>J</i> = 7 Hz, 3H), 1.34 (t, <i>J</i> = 7.4 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 195.1, 193.0, 142.5, 137.5, 137.2, 130.0, 127.7, 126.7, 123.2, 121.2, 120.4, 118.7, 113.8, 56.5, 54.4, 51.1, 27.5, 20.7, 20.3, 16.4; HRMS (ESI) for C<sub>21</sub>H<sub>19</sub>O<sub>3</sub>N [M + H]<sup>+</sup>: calcd, 334.1443; found, 334.1445.</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30">4.3.17.  5-Ethyl-11-isopropyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>9b</b>)</h4><div class="NLM_p last">Starting from <i>N</i>-isopropyl-3-(but-1-en-1-yl)-indole (1.02 g, 4.8 mmol), compound <b>9b</b> was isolated as a dark red-brown solid (261 mg, 13%). FT-IR(ν<sub>max</sub>, cm<sup>–1</sup>): 1741, 1693, 1651, 1617, 1589, 1534, 1463, 1428, 1391, 1368, 1308, 1282, 1233, 1199, 1156, 1105, 1062; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.10 (s, 1H), 8.10 (d, <i>J</i> = 7.6 Hz, 1H), 7.74 (d, <i>J</i> = 8.6 Hz, 1H), 7.5 (td, <i>J</i> = 7.2 Hz, 1.2 Hz, 1H), 7.28 (td, <i>J</i> = 7.8 Hz, 0.7 Hz, 1H), 6.87 (dd, <i>J</i> = 17.4 Hz, 10.1 Hz, 2H), 4.74 (h, <i>J</i> = 7 Hz, 1H), 3.26 (q, <i>J</i> = 7.4 Hz, 2H), 1.67 (d, <i>J</i> = 6.9 Hz 6H), 1.35 (t, <i>J</i> = 7.4 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 188.1, 186.9, 143.7, 139.8, 139.2, 138.8, 137.7, 131.2, 129.1, 127.8, 127.5, 123.6, 121.2, 120.7, 120.1, 114.4, 52.5, 28.9, 21.1, 15.7; HRMS (ESI) for C<sub>21</sub>H<sub>19</sub>O<sub>2</sub>N [M + H]<sup>+</sup>: calcd, 318.1494; found, 318.1501.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31">4.3.18.  9-Isopropyl-10<i>a</i>-phenyl-1<i>a</i>,10<i>a</i>-dihydro-2<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>)-dione (<b>10a</b>)</h4><div class="NLM_p last">Starting from <b>10b</b>, <b>11b</b> (40 mg, 0.105 mmol), the mixture of regioisomers, was isolated as a yellow solid (30 mg, 73%). Using reversed-phase semi-preparative HPLC, compound <b>10a</b> was isolated as an orange-yellow solid: FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1679, 1617, 1553, 1455, 1416, 1376, 1323, 1286, 1221, 1167, 1128, 1078, 1021; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.36 (d, <i>J</i> = 8.0 Hz, 1H), 8.15 (d, <i>J</i> = 8.0 Hz, 1H), 7.92 (d, <i>J</i> = 7.9 Hz, 1H), 7.74 (d, <i>J</i> = 8.3 Hz, 1H), 7.63–7.59 (m, 2H), 7.55–7.47 (m, 4H), 7.34–7.29 (m, 1H), 4.61 (h, <i>J</i> = 7.0 Hz, 1H), 4.06 (s, 1H), 1.90 (d, <i>J</i> = 4 Hz, 3H), 1.40 (d, <i>J</i> = 4 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 192.7, 191.4, 142.2, 139.0, 131.1, 129.9, 129.4, 128.7, 128.3, 128.2, 127.7, 124.8, 123.5, 121.3, 120.7, 119.0, 118.0, 114.0, 63.9, 63.4, 51.4, 20.8, 20.6; HRMS (ESI) for C<sub>25</sub>H<sub>19</sub>O<sub>3</sub>N [M + H]<sup>+</sup>: calcd, 382.1443; found, 382.1443.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32">4.3.19.  11-Isopropyl-3-phenyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione + 11-Isopropyl-2-phenyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>10b</b>, <b>11b</b>)</h4><div class="NLM_p last">Starting from 3-vinyl, <i>N</i>-isopropyl-indole (250 mg, 1.35 mmol), <b>10b</b>, <b>11b</b> mixture were isolated as a dark red-brown solid (100 mg, 20%); FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1643, 1590, 1469, 1418, 1294, 1180, 1139, 1099, 1044; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, <i>J</i> = 7.9 Hz, 1H), 8.13 (dd, <i>J</i> = 7.9, 2.8 Hz, 2H), 7.79 (d, <i>J</i> = 8.3 Hz, 1H), 7.63 (dd, <i>J</i> = 6.6, 2.9 Hz, 2H), 7.60–7.43 (m, 7H), 7.32 (t, <i>J</i> = 7.5 Hz, 1H), 7.07 (s, 1H), 5.09 (h, <i>J</i> = 7.0 Hz, 1H), 1.73 (d, <i>J</i> = 7.0 Hz, 6H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 187.8, 186.8, 146.1, 137.2, 136.4, 132.8, 130.3, 129.5, 129.4, 128.7, 124.8, 121.2, 120.8, 118.3, 114.3, 64.3, 52.4, 21.2; HRMS (ESI) for C<sub>25</sub>H<sub>19</sub>O<sub>2</sub>N [M + H]<sup>+</sup>: calcd, 366.1494; found, 366.1493.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33">4.3.20.  9-Isopropyl-1a-phenyl-1a,10a-dihydro-2<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>)-dione (<b>11a</b>)</h4><div class="NLM_p last">Starting from <b>10b</b>, <b>11b</b> (40 mg, 0.105 mmol), the mixture of regioisomers, was isolated as a yellow solid (30 mg, 73%). Using reversed-phase semi-preparative HPLC, compound <b>11a</b> was isolated as a yellow solid: FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1676, 1559, 1455, 1423, 1297, 1257, 1213, 1131, 1102, 1064, 1028; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, <i>J</i> = 8 Hz, 1H), 8.14 (d, <i>J</i> = 8 Hz, 1H), 8.00 (d, <i>J</i> = 8.0 Hz, 1H), 7.77 (d, <i>J</i> = 8 Hz, 1H), 7.58–7.52 (m, 3H), 7.49–7.45 (m, 3H), 7.33 (t, <i>J</i> = 7.5 Hz, 1H), 4.81 (h, <i>J</i> = 6.9 Hz, 1H), 4.09 (s, 1H), 2.00 (d, <i>J</i> = 6.9 Hz, 3H), 1.46 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 193.2, 190.7, 142.2, 138.4, 131.3, 130.9, 130.8, 129.4, 128.6, 127.9, 127.7, 125.0, 123.4, 121.3, 120.7, 119.1, 118.2, 113.9, 65.8, 62.3, 51.3, 20.8, 20.6; HRMS (ESI) for C<sub>25</sub>H<sub>19</sub>O<sub>3</sub>N [M + H]<sup>+</sup>: calculated, 382.1443; found, 382.1443.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34">4.3.21.  9-Isopropyl-10a-(4-methoxyphenyl)-1a,10a-dihydro-2<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>)-dione (<b>12a</b>)</h4><div class="NLM_p last">Starting from <b>12b</b>, <b>13b</b> (244 mg, 0.592 mmol), the mixture of regioisomers, was isolated as a yellow solid (91 mg, 36%). Using reversed-phase semi-preparative HPLC, compound <b>12a</b> was isolated as a yellow solid: FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1687, 1601, 1514, 1456, 1424, 1385, 1332, 1292, 1245, 1168, 1134, 1024; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.35 (d, <i>J</i> = 7.9 Hz, 1H), 8.14 (d, <i>J</i> = 7.7 Hz, 1H), 7.91 (d, <i>J</i> = 8.1 Hz, 1H), 7.74 (d, <i>J</i> = 8.5 Hz, 1H), 7.56–7.50 (m, 3H), 7.31 (t, <i>J</i> = 7.6 Hz, 1H), 7.04–7.00 (m, 2H), 4.61 (h, <i>J</i> = 6.9 Hz, 1H), 4.05 (s, 1H), 3.87 (s, 3H), 1.91 (d, <i>J</i> = 6.9 Hz, 3H), 1.42–1.39 (m, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 193.2, 191.7, 160.4, 142.2, 139.0, 131.1, 129.9, 129.6, 127.7, 124.8, 123.5, 122.9, 121.3, 120.6, 118.0, 114.1, 114.0, 64.0, 55.5, 51.4, 20.8, 20.6; HRMS (ESI) for C<sub>26</sub>H<sub>21</sub>O<sub>4</sub>N [M + H]<sup>+</sup>: calcd, 412.1549; found, 412.1546.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35">4.3.22.  11-Isopropyl-3-(4-methoxyphenyl)-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione + 11-Isopropyl-2-(4-methoxyphenyl)-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>12b</b>, <b>13b</b>)</h4><div class="NLM_p last">Starting from 3-vinyl, <i>N</i>-isopropyl-indole (190.2 mg, 1.03 mmol), <b>12b</b>, <b>13b</b> mixture was isolated as a dark red-brown solid (210.2 mg, 52%); FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1648, 1602, 1556, 1511, 1462, 1417, 1366, 1298, 1230, 1178, 1134, 1086, 1031; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.33 (d, <i>J</i> = 7.9 Hz, 1H), 8.13–8.04 (m, 2H), 7.88 (d, <i>J</i> = 8.4 Hz, 1H), 7.63 (d, <i>J</i> = 6.9 Hz, 2H), 7.52 (t, <i>J</i> = 7.4 Hz, 1H), 7.30 (t, <i>J</i> = 7.4 Hz, 1H), 7.05–7.00 (m, 3H), 5.0 (h, <i>J</i> = 7.0 Hz, 1H), 3.88 (s, 3H), 1.74 (d, <i>J</i> = 7 Hz, 6H) (regioisomer ratio 4:1); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 186.3, 185.4, 161.2, 145.6, 143.0, 139.4, 134.9, 132.3, 131.2, 131.0, 127.6, 125.8, 124.7, 123.8, 121.2, 120.8, 119.4, 118.7, 114.4, 114.3, 114.1, 55.5, 52.3, 21.3; MALDI-TOF for C<sub>26</sub>H<sub>21</sub>O<sub>3</sub>N: expected mass, 395.4498; found, 395.0778.</div></div><div id="sec4_3_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36">4.3.23.  9-Isopropyl-1a-(4-methoxyphenyl)-1a,10a-dihydro-2<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>)-dione (<b>13a</b>)</h4><div class="NLM_p last">Starting from <b>12b</b>, <b>13b</b> (244 mg, 0.592 mmol), the mixture of regioisomers, was isolated as a yellow solid (91 mg, 36%). Using reversed-phase semi-preparative HPLC, compound <b>13a</b> was isolated as a yellow solid: FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1682, 1606, 1513, 1454, 1425, 1379, 1293, 1245, 1169, 1026; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, <i>J</i> = 7.9 Hz, 1H), 8.14 (d, <i>J</i> = 7.8 Hz, 1H), 8.00 (d, <i>J</i> = 8.1 Hz, 1H), 7.77 (d, <i>J</i> = 8.3 Hz, 1H), 7.54 (ddd, <i>J</i> = 8.5, 7.3, 1.3 Hz, 1H), 7.50–7.47 (m, 2H), 7.35–7.30 (m, 1H), 7.02–6.98 (m, 2H), 4.80 (h, <i>J</i> = 6.9 Hz, 1H), 4.09 (s, 1H), 3.86 (s, 3H), 2.00 (d, <i>J</i> = 6.9 Hz, 3H), 1.46 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 199.0, 198.6, 198.0, 194.1, 189.2, 173.1, 159.4, 140.8, 137.1, 130.2, 128.3, 128.1, 127.8, 126.2, 123.7, 121.4, 121.3, 121.2, 120.9, 118.8, 114.7, 114.6, 114.3, 114.1, 113.9, 113.7, 84.6, 60.1, 55.4, 55.4, 51.4, 51.2, 29.9, 21.5, 20.9, 20.5; HRMS (ESI) for C<sub>26</sub>H<sub>21</sub>O<sub>4</sub>N [M + H]<sup>+</sup>: calcd, 412.1549; found, 412.1546.</div></div><div id="sec4_3_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37">4.3.24.  10a-(4-Bromophenyl)-9-isopropyl-1a,10a-dihydro-2<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>)-dione (<b>14a</b>)</h4><div class="NLM_p last">Starting from <b>14b</b>, <b>15b</b> (627.2 mg, 1.41 mmol), the mixture of regioisomers, was isolated as a yellow solid (604.5 mg, 93%). Using reversed-phase semi-preparative HPLC, compound <b>14a</b> was isolated as a yellow solid: FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1741, 1725, 1678, 1648, 1616, 1549, 1516, 1498, 1463, 1422, 1396, 1366, 1299, 1223, 1167, 1133, 1107; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.35 (d, <i>J</i> = 8.1 Hz, 1H), 8.14 (d, <i>J</i> = 7.8 Hz, 1H), 7.91 (d, <i>J</i> = 7.8 Hz, 1H), 7.74 (d, <i>J</i> = 8.3 Hz, 1H), 7.65–7.61 (m, 2H), 7.57–7.46 (m, 3H), 7.35–7.29 (m, 1H), 4.57 (h, <i>J</i> = 7.0 Hz, 1H), 4.01 (s, 1H), 1.90 (d, <i>J</i> = 6.9 Hz, 3H), 1.40 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 192.1, 190.1, 142.2, 139.1, 131.8, 131.0, 131.3, 130.0, 129.8, 127.8, 125.0, 123.7, 123.4, 121.3, 120.7, 118.6, 118.1, 114.0, 64.0, 62.9, 51.5, 22.2, 20.8, 20.6; HRMS (ESI) for C<sub>25</sub>H<sub>18</sub>O<sub>3</sub>NBr [M + H]<sup>+</sup>: calcd, 460.0548; found, 460.0548.</div></div><div id="sec4_3_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38">4.3.25.  3-(4-Bromophenyl)-11-isopropyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione + 2-(4-Bromophenyl)-11-isopropyl-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>14b</b>, <b>15b</b>)</h4><div class="NLM_p last">Starting from 3-vinyl, <i>N</i>-isopropyl-indole (400 mg, 2.16 mmol), <b>14b</b>, <b>15b</b> mixture was isolated as a dark red-brown solid (653.1 mg, 68%); FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1741, 1725, 1707, 1694, 1678, 1646, 1615, 1547, 1532, 1516, 1485, 1463, 1425, 1396, 1366, 1340, 1311, 1227, 1134; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.33 (d, <i>J</i> = 7.9 Hz, 1H), 8.14–8.04 (m, 2H), 7.88 (d, <i>J</i> = 3.6 Hz, 1H), 7.66–7.61 (m, 2H), 7.55–7.50 (m, 3H), 7.32 (t, <i>J</i> = 7.5 Hz, 1H), 7.04 (d, <i>J</i> = 9.0 Hz, 1H), 5.08 (h, <i>J</i> = 7.0 Hz, 1H), 1.73 (s, 6H) (regioisomer ratio: 1.22:1); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 186.0, 185.9, 185.7, 184.7, 148.2, 145.1, 143.0, 142.8, 139.8, 139.4, 136.4, 133.3, 133.0, 132.4, 132.0, 131.8, 131.6, 131.5, 131.1, 131.0, 127.7, 125.0, 124.8, 124.6, 123.8, 121.3, 120.9, 119.5, 119.4, 118.4, 118.4, 114.4, 114.4, 52.4, 52.3, 21.3, 21.2; HRMS (ESI) for C<sub>25</sub>H<sub>18</sub>O<sub>2</sub>NBr [M + H]<sup>+</sup>: calcd, 444.0599; found, 444.0600.</div></div><div id="sec4_3_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39">4.3.26.  1a-(4-Bromophenyl)-9-isopropyl-1a,10a-dihydro-2<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>)-dione (<b>15a</b>)</h4><div class="NLM_p last">Starting from <b>14b</b>, <b>15b</b> (627.2 mg, 1.41 mmol), the mixture of regioisomers, was isolated as a yellow solid (604.5 mg, 93%). Using reversed-phase semi-preparative HPLC, compound <b>15a</b> was isolated as a yellow solid: FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 1680, 1556, 1460, 1417, 1367, 1311, 1219, 1136, 1106, 1064, 1011; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.33 (d, <i>J</i> = 7.9 Hz, 1H), 8.14 (d, <i>J</i> = 7.8 Hz, 1H), 7.99 (d, <i>J</i> = 8.1 Hz, 1H), 7.77 (d, <i>J</i> = 8.5 Hz, 1H), 7.63–7.59 (m, 2H), 7.55 (ddd, <i>J</i> = 8.4, 7.4, 1.2 Hz, 1H), 7.46–7.41 (m, 2H), 7.33 (t, <i>J</i> = 7.6 Hz, 1H), 4.78 (h, <i>J</i> = 6.9 Hz, 1H), 4.05 (s, 1H), 2.00 (d, <i>J</i> = 6.9 Hz, 3H), 1.45 (d, <i>J</i> = 7.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 192.7, 190.3, 142.2, 138.5, 131.8, 130.9, 130.6, 130.4, 129.6, 127.8, 125.1, 123.8, 123.4, 121.3, 120.8, 119.2, 118.0, 114.0, 65.3, 62.3, 51.3, 20.8, 20.6; HRMS (ESI) for C<sub>25</sub>H<sub>18</sub>O<sub>3</sub>NBr [M + H]<sup>+</sup>: calcd, 460.0548; found, 460.0548.</div></div><div id="sec4_3_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40">4.3.27.  9-(Pent-4-yn-1-yl)-1a,10a-dihydro-2<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>)-dione (<b>P1</b>)</h4><div class="NLM_p last">Starting from <b>Q1</b> (150 mg, 0.479 mmol), <b>P1</b> was isolated as a bright yellow solid (73 mg, 46%). FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 3291, 2924, 2855, 1687, 1564, 1468, 1425, 1334, 1287, 1216, 1183, 865, 742; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.36 (d, <i>J</i> = 8.0 Hz, 1H), 8.10 (d, <i>J</i> = 7.7 Hz, 1H), 7.88 (d, <i>J</i> = 8.0 Hz, 1H), 7.60 (t, <i>J</i> = 6.0 Hz, 2H), 7.35–7.32 (m, 1H), 4.68–4.61 (m, 1H), 4.57–4.49 (m, 1H), 4.14 (dd, <i>J</i> = 6.5, 4.2 Hz, 2H), 2.18–2.01 (m, 3H), 1.97–1.88 (m, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 192.5, 191.4, 143.9, 136.9, 131.3, 130.2, 128.6, 125.4, 121.8, 121.1, 121.0, 118.4, 117.7, 110.6, 83.2, 69.7, 56.1, 54.8, 44.4, 26.6, 16.0.; HRMS (ESI): for C<sub>21</sub>H<sub>15</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: calcd, 330.1130; found, 330.1131.</div></div><div id="sec4_3_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41">4.3.28.  9-(Pent-4-yn-1-yl)-1a-phenyl-1a,10a-dihydro-2<i>H</i>-oxireno[2′,3′:4,5]benzo[1,2-<i>a</i>]carbazole-2,10(9<i>H</i>)-dione (<b>P2</b>)</h4><div class="NLM_p last">Starting from <b>Q2</b> (crude, 116 mg, 0.152 mmol), <b>P2</b> was isolated as a bright yellow solid (85 mg, 70%). FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 2922, 2861, 1687, 1465, 1335, 1287, 1205, 1126, 1066, 739; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, <i>J</i> = 8.2 Hz, 1H), 8.11 (d, <i>J</i> = 7.8 Hz, 1H), 7.97 (d, <i>J</i> = 7.9 Hz, 1H), 7.62–7.56 (m, 4H), 7.49–7.46 (m, 3H), 7.37–7.33 (m, 1H), 4.74–4.68 (m, 1H), 4.60–4.52 (m, 1H), 4.10 (s, 1H), 2.18–2.08 (m, 2H), 2.06 (t, <i>J</i> = 2.2 Hz, 1H), 1.98–1.90 (m, 2H).; <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 192.4, 191.0, 143.9, 136.7, 131.1, 131.0, 131.0, 129.4, 128.6, 128.5, 127.9, 125.3, 121.9, 121.1, 121.0, 119.1, 117.9, 110.6, 83.2, 69.7, 65.5, 62.4, 44.4, 26.7, 16.0.; HRMS (ESI): for C<sub>27</sub>H<sub>19</sub>NO<sub>3</sub> [M + H]<sup>+</sup>: calcd, 406.1443; found, 406.1443.</div></div><div id="sec4_3_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42">4.3.29.  11-(Pent-4-yn-1-yl)-1<i>H</i>-benzo[<i>a</i>]carbazole-1,4(11<i>H</i>)-dione (<b>Q1</b>)</h4><div class="NLM_p last">Starting from <i>N</i>-(pent-4-yn-1-yl)-indole-3-carboxaldehyde (1 g, 4.66 mmol), <b>Q1</b> was isolated as a dark red-brown solid (264 mg, 27%). FT-IR (ν<sub>max</sub>, cm<sup>–1</sup>): 3280, 2922, 2857, 1658, 1615, 1465, 1378, 1292, 1182, 1122, 835, 747; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.35 (d, <i>J</i> = 8.0 Hz, 1H), 8.09 (dt, <i>J</i> = 7.8, 0.8 Hz, 1H), 8.04 (d, <i>J</i> = 8.0 Hz, 1H), 7.62–7.56 (m, 2H), 7.35–7.31 (m, 1H), 6.94 (dd, <i>J</i> = 15.3, 10.1 Hz, 2H), 4.78 (t, <i>J</i> = 7.3 Hz, 2H), 2.13–2.08 (m, 2H), 2.05–2.03 (m, 1H), 2.00 (t, <i>J</i> = 2.6 Hz, 2H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 186.1, 185.6, 144.5, 140.3, 138.1, 137.0, 131.6, 131.5, 128.5, 125.3, 122.2, 121.2, 120.9, 118.9, 118.5, 111.1, 83.5, 69.4, 46.3, 27.8, 16.1; HRMS (ESI) for C<sub>21</sub>H<sub>15</sub>NO<sub>2</sub> [M + H]<sup>+</sup>: calcd, 314.1181; found, 314.1178.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i43"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00774" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00774" class="ext-link">10.1021/acs.jmedchem.9b00774</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Preparative procedures, characterization data, and protocols (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Proteomics data table (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf">jm9b00774_si_001.pdf (4.48 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_002.xlsx">jm9b00774_si_002.xlsx (659.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_003.csv">jm9b00774_si_003.csv (2.05 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00774" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02538" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02538" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sidharth Chopra</span> - <span class="hlFld-Affiliation affiliation">Division
of Microbiology, CSIR-Central Drug Research
Institute, Sector 10,
Janakipuram Extension, Sitapur Road, Lucknow 226021, Uttar Pradesh, India</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#96e5fdf5fef9e6e4f7a6a6a1d6f1fbf7fffab8f5f9fb"><span class="__cf_email__" data-cfemail="36455d555e5946445706060176515b575f5a1855595b">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Siddhesh S. Kamat</span> - <span class="hlFld-Affiliation affiliation">Department of Biology and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6132-7574" title="Orcid link">http://orcid.org/0000-0001-6132-7574</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#72011b16161a17011a321b1b01170002071c175c13115c1b1c"><span class="__cf_email__" data-cfemail="5a29333e3e323f29321a3333293f282a2f343f743b39743334">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harinath Chakrapani</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7267-0906" title="Orcid link">http://orcid.org/0000-0002-7267-0906</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#60080112090e01140820090913051210150e054e01034e090e"><span class="__cf_email__" data-cfemail="345c55465d5a55405c745d5d47514644415a511a55571a5d5a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amogh Kulkarni</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Isha Soni</span> - <span class="hlFld-Affiliation affiliation">Division
of Microbiology, CSIR-Central Drug Research
Institute, Sector 10,
Janakipuram Extension, Sitapur Road, Lucknow 226021, Uttar Pradesh, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dhanashree S. Kelkar</span> - <span class="hlFld-Affiliation affiliation">Department of Biology and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Allimuthu T. Dharmaraja</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span>; 
    <span>Present Address:
                        Department of Chemistry, Indian Institute of Technology, Kanpur, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rathinam K. Sankar</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span>; 
    <span>Present Address:
                        Faculty of Pharmaceutical Sciences, University of Iceland, 107 Reykjavik, Iceland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gaurav Beniwal</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abinaya Rajendran</span> - <span class="hlFld-Affiliation affiliation">Department of Biology and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sharvari Tamhankar</span> - <span class="hlFld-Affiliation affiliation">Department of Biology and , Indian Institute
of Science Education and Research Pune, Dr. Homi Bhabha Road, Pashan, Pune 411008, Maharashtra, India</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The authors will release the atomic coordinates and experimental data upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71272" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71272" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank the Department of Biotechnology (DBT), Government of India (BT/PR15848/MED/29/1025/2016 to H.C. and S.C.), a Wellcome Trust DBT India Alliance Intermediate Fellowship (IA/I/15/2/502058 to S.S.K.) and a DST-FIST Infrastructure Development Grant (to IISER Pune Biology) for the financial support for our research. The Council for Scientific and Industrial Research (CSIR) and the Department of Science and Technology—Innovation in Science Pursuit for Inspired Research (DST-INSPIRE) for graduate student fellowships.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AMR</td><td class="NLM_def"><p class="first last">antimicrobial resistance</p></td></tr><tr><td class="NLM_term"><i>S. aureus</i></td><td class="NLM_def"><p class="first last"><i>Staphylococcus aureus</i></p></td></tr><tr><td class="NLM_term">VRSA</td><td class="NLM_def"><p class="first last">vancomycin-resistant <i>Staphylococcus aureus</i></p></td></tr><tr><td class="NLM_term">ABPP</td><td class="NLM_def"><p class="first last">activity-based protein profiling</p></td></tr><tr><td class="NLM_term">p<i>K</i><sub>a</sub></td><td class="NLM_def"><p class="first last">negative log of acid-dissociation constant</p></td></tr><tr><td class="NLM_term">INDQE</td><td class="NLM_def"><p class="first last">indole-based quinone epoxide</p></td></tr><tr><td class="NLM_term">ATCC</td><td class="NLM_def"><p class="first last">American Type Culture Collection</p></td></tr><tr><td class="NLM_term">Dap</td><td class="NLM_def"><p class="first last">daptomycin</p></td></tr><tr><td class="NLM_term"><i>E. coli</i></td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i></p></td></tr><tr><td class="NLM_term"><i>K. pneumoniae</i></td><td class="NLM_def"><p class="first last"><i>Klebsiella pneumonia</i></p></td></tr><tr><td class="NLM_term"><i>E. faecalis</i></td><td class="NLM_def"><p class="first last"><i>Enterococcus faecalis</i></p></td></tr><tr><td class="NLM_term"><i>P. aeruginosa</i></td><td class="NLM_def"><p class="first last"><i>Pseudomonas aeruginosa</i></p></td></tr><tr><td class="NLM_term"><i>A. baumanii</i></td><td class="NLM_def"><p class="first last"><i>Acinetobacter baumanii</i></p></td></tr><tr><td class="NLM_term">PDI</td><td class="NLM_def"><p class="first last">protein disulfide-isomerase</p></td></tr><tr><td class="NLM_term">BTK</td><td class="NLM_def"><p class="first last">Bruton’s tyrosine kinase</p></td></tr><tr><td class="NLM_term">TCI</td><td class="NLM_def"><p class="first last">targeted covalent inhibitors</p></td></tr><tr><td class="NLM_term">NQE</td><td class="NLM_def"><p class="first last">naphthoquinone epoxide</p></td></tr><tr><td class="NLM_term">ESKAPE</td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i>, <i>Staphylococcus aureus</i>, <i>Klebsiella pneumoniae</i>, <i>Acinetobacter baumannii</i>, <i>Pseudomonas aeruginosa</i> and <i>Enterococcus faecalis</i></p></td></tr><tr><td class="NLM_term">mBBr</td><td class="NLM_def"><p class="first last">monobromobimane</p></td></tr><tr><td class="NLM_term">IAM</td><td class="NLM_def"><p class="first last">iodoacetamide</p></td></tr><tr><td class="NLM_term">Mar R</td><td class="NLM_def"><p class="first last">multiple antibiotic resistance regulator</p></td></tr><tr><td class="NLM_term">NTA</td><td class="NLM_def"><p class="first last">nitrilotriacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41353" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41353" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLeo, F. R.</span></span> <span> </span><span class="NLM_article-title">Waves of resistance: <i>Staphylococcus aureus</i> in the antibiotic era</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1038/nrmicro2200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnrmicro2200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=19680247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFCrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=629-641&author=H.+F.+Chambersauthor=F.+R.+DeLeo&title=Waves+of+resistance%3A+Staphylococcus+aureus+in+the+antibiotic+era&doi=10.1038%2Fnrmicro2200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Waves of resistance: Staphylococcus aureus in the antibiotic era</span></div><div class="casAuthors">Chambers, Henry F.; DeLeo, Frank R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">629-641</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Staphylococcus aureus is notorious for its ability to become resistant to antibiotics.  Infections that are caused by antibiotic-resistant strains often occur in epidemic waves that are initiated by one or a few successful clones.  Methicillin-resistant S. aureus (MRSA) features prominently in these epidemics.  Historically assocd. with hospitals and other health care settings, MRSA has now emerged as a widespread cause of community infections.  Community or community-assocd. MRSA (CA-MRSA) can spread rapidly among healthy individuals.  Outbreaks of CA-MRSA infections have been reported worldwide, and CA-MRSA strains are now epidemic in the United States.  Here, we review the mol. epidemiol. of the epidemic waves of penicillin- and methicillin-resistant strains of S. aureus that have occurred since 1940, with a focus on the clin. and mol. epidemiol. of CA-MRSA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXSOiu96U2nLVg90H21EOLACvtfcHk0ljesae511_qTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFCrsLo%253D&md5=17ae44f98bfbdbeb6f9ef9e728ea6a6b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2200%26sid%3Dliteratum%253Aachs%26aulast%3DChambers%26aufirst%3DH.%2BF.%26aulast%3DDeLeo%26aufirst%3DF.%2BR.%26atitle%3DWaves%2520of%2520resistance%253A%2520Staphylococcus%2520aureus%2520in%2520the%2520antibiotic%2520era%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2009%26volume%3D7%26spage%3D629%26epage%3D641%26doi%3D10.1038%2Fnrmicro2200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musser, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLeo, F. R.</span></span> <span> </span><span class="NLM_article-title">Genomic analysis of the emergence of vancomycin-resistant Staphylococcus aureus</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e00170</span> <span class="refDoi"> DOI: 10.1128/mbio.00170-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1128%2Fmbio.00170-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=22736541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtlaht7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&author=S.+D.+Kobayashiauthor=J.+M.+Musserauthor=F.+R.+DeLeo&title=Genomic+analysis+of+the+emergence+of+vancomycin-resistant+Staphylococcus+aureus&doi=10.1128%2Fmbio.00170-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic analysis of the emergence of vancomycin-resistant Staphylococcus aureus</span></div><div class="casAuthors">Kobayashi, Scott D.; Musser, James M.; DeLeo, Frank R.</div><div class="citationInfo"><span class="NLM_cas:title">mBio</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e00170-12</span>CODEN:
                <span class="NLM_cas:coden">MBIOCL</span>;
        ISSN:<span class="NLM_cas:issn">2150-7511</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Staphylococcus aureus is a human commensal bacterium and a prominent cause of infections globally.  The high incidence of S. aureus infections is compounded by the ability of the microbe to readily acquire resistance to antibiotics.  In the United States, methicillin-resistant S. aureus (MRSA) is a leading cause of morbidity and mortality by a single infectious agent.  Therapeutic options for severe MRSA infections are limited to a few antibiotics to which the organism is typically susceptible, including vancomycin.  Acquisition of high-level vancomycin resistance by MRSA is a major concern, but to date, there have been only 12 vancomycin-resistant S. aureus (VRSA) isolates reported in the United States and all belong to a phylogenetic lineage known as clonal complex 5.  To gain enhanced understanding of the genetic characteristics conducive to the acquisition of vancomycin resistance by S. aureus, V. N. Kos et al. performed whole-genome sequencing of all 12 VRSA isolates and compared the DNA sequences to the genomes of other S. aureus strains.  The findings provide new information about the evolutionary history of VRSA and identify genetic features that may bear on the relationship between S. aureus clonal complex 5 strains and the acquisition of vancomycin resistance genes from enterococci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE9fgRd77SMrVg90H21EOLACvtfcHk0lhAS_tR63RK-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtlaht7jO&md5=c1b8d43540068433fd682e79234cbc23</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2Fmbio.00170-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmbio.00170-12%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DS.%2BD.%26aulast%3DMusser%26aufirst%3DJ.%2BM.%26aulast%3DDeLeo%26aufirst%3DF.%2BR.%26atitle%3DGenomic%2520analysis%2520of%2520the%2520emergence%2520of%2520vancomycin-resistant%2520Staphylococcus%2520aureus%26jtitle%3DmBio%26date%3D2012%26volume%3D3%26doi%3D10.1128%2Fmbio.00170-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C.</span>; <span class="NLM_string-name">Wencewicz, T.</span></span> <i>Antibiotics: Challenges, Mechanisms, Opportunities</i>; <span class="NLM_publisher-name">American Society of Microbiology</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1128%2F9781555819316" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=C.+Walsh&author=T.+Wencewicz&title=Antibiotics%3A+Challenges%2C+Mechanisms%2C+Opportunities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1128%2F9781555819316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1128%252F9781555819316%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DC.%26btitle%3DAntibiotics%253A%2520Challenges%252C%2520Mechanisms%252C%2520Opportunities%26pub%3DAmerican%2520Society%2520of%2520Microbiology%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based profiling of enzyme families</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">3279</span>– <span class="NLM_lpage">3301</span>, <span class="refDoi"> DOI: 10.1021/cr050288g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr050288g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFejtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=3279-3301&author=M.+J.+Evansauthor=B.+F.+Cravatt&title=Mechanism-based+profiling+of+enzyme+families&doi=10.1021%2Fcr050288g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based profiling of enzyme families</span></div><div class="casAuthors">Evans, Michael J.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3279-3301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Activity-based protein profiling (ABPP) represents a chem. strategy that utilizes site-directed covalent probes to profile the functional state of enzymes in complex proteomes.  This review describes the design of prototype activity-based probes (ABPs) for functional proteomics, biol. applications of ABPs, methodol. advances, and applications of ABPs beyond comparative proteomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwF1DAf24AkrVg90H21EOLACvtfcHk0lhAS_tR63RK-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFejtrs%253D&md5=3986124d96400c9392ba562fc7fa3faf</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fcr050288g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr050288g%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DM.%2BJ.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DMechanism-based%2520profiling%2520of%2520enzyme%2520families%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D3279%26epage%3D3301%26doi%3D10.1021%2Fcr050288g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: the serine hydrolases</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">14694</span>– <span class="NLM_lpage">14699</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.26.14694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1073%2Fpnas.96.26.14694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10611275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=14694-14699&author=Y.+Liuauthor=M.+P.+Patricelliauthor=B.+F.+Cravatt&title=Activity-based+protein+profiling%3A+the+serine+hydrolases&doi=10.1073%2Fpnas.96.26.14694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: the serine hydrolases</span></div><div class="casAuthors">Liu, Yongsheng; Patricelli, Matthew P.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">14694-14699</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">With the postgenome era rapidly approaching, new strategies for the functional anal. of proteins are needed.  To date, proteomics efforts have primarily been confined to recording variations in protein level rather than activity.  The ability to profile classes of proteins on the basis of changes in their activity would greatly accelerate both the assignment of protein function and the identification of potential pharmaceutical targets.  Here, we describe the chem. synthesis and utility of an active-site directed probe for visualizing dynamics in the expression and function of an entire enzyme family, the serine hydrolases.  By reacting this probe, a biotinylated fluorophosphonate referred to as FP-biotin, with crude tissue exts., we quickly and with high sensitivity detect numerous serine hydrolases, many of which display tissue-restricted patterns of expression.  Addnl., we show that FP-biotin labels these proteins in an activity-dependent manner that can be followed kinetically, offering a powerful means to monitor dynamics simultaneously in both protein function and expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8i5m0pTTunrVg90H21EOLACvtfcHk0lhAS_tR63RK-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D&md5=d5d1a09328fc7715e9d30ca392733426</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.26.14694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.26.14694%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520the%2520serine%2520hydrolases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D14694%26epage%3D14699%26doi%3D10.1073%2Fpnas.96.26.14694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatzios, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasabe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qadri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldor, M. K.</span></span> <span> </span><span class="NLM_article-title">Chemoproteomic profiling of host and pathogen enzymes active in cholera</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">268</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio.2025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=26900865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC28XivFKgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=268&author=S.+K.+Hatziosauthor=S.+Abelauthor=J.+Martellauthor=T.+Hubbardauthor=J.+Sasabeauthor=D.+Muneraauthor=L.+Clarkauthor=D.+A.+Bachovchinauthor=F.+Qadriauthor=E.+T.+Ryanauthor=B.+M.+Davisauthor=E.+Weerapanaauthor=M.+K.+Waldor&title=Chemoproteomic+profiling+of+host+and+pathogen+enzymes+active+in+cholera&doi=10.1038%2Fnchembio.2025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoproteomic profiling of host and pathogen enzymes active in cholera</span></div><div class="casAuthors">Hatzios, Stavroula K.; Abel, Soren; Martell, Julianne; Hubbard, Troy; Sasabe, Jumpei; Munera, Diana; Clark, Lars; Bachovchin, Daniel A.; Qadri, Firdausi; Ryan, Edward T.; Davis, Brigid M.; Weerapana, Eranthie; Waldor, Matthew K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">268-274</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activity-based protein profiling (ABPP) is a chemoproteomic tool for detecting active enzymes in complex biol. systems.  We used ABPP to identify secreted bacterial and host serine hydrolases that are active in animals infected with the cholera pathogen Vibrio cholerae.  Four V. cholerae proteases were consistently active in infected rabbits, and one, VC0157 (renamed IvaP), was also active in human choleric stool.  Inactivation of IvaP influenced the activity of other secreted V. cholerae and rabbit enzymes in vivo, and genetic disruption of all four proteases increased the abundance of intelectin, an intestinal lectin, and its binding to V. cholerae in infected rabbits.  Intelectin also bound to other enteric bacterial pathogens, suggesting that it may constitute a previously unrecognized mechanism of bacterial surveillance in the intestine that is inhibited by pathogen-secreted proteases.  Our work demonstrates the power of activity-based proteomics to reveal host-pathogen enzymic dialog in an animal model of infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolhuO5dBmCbVg90H21EOLACvtfcHk0lj56-ZRNKGtMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivFKgtro%253D&md5=c07b3bb6213fe023f5916a10dc0be7db</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2025%26sid%3Dliteratum%253Aachs%26aulast%3DHatzios%26aufirst%3DS.%2BK.%26aulast%3DAbel%26aufirst%3DS.%26aulast%3DMartell%26aufirst%3DJ.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DSasabe%26aufirst%3DJ.%26aulast%3DMunera%26aufirst%3DD.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DQadri%26aufirst%3DF.%26aulast%3DRyan%26aufirst%3DE.%2BT.%26aulast%3DDavis%26aufirst%3DB.%2BM.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DWaldor%26aufirst%3DM.%2BK.%26atitle%3DChemoproteomic%2520profiling%2520of%2520host%2520and%2520pathogen%2520enzymes%2520active%2520in%2520cholera%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D268%26doi%3D10.1038%2Fnchembio.2025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hacker, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazear, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Global profiling of lysine reactivity and ligandability in the human proteome</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1038/nchem.2826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchem.2826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=29168484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1181-1190&author=S.+M.+Hackerauthor=K.+M.+Backusauthor=M.+R.+Lazearauthor=S.+Forliauthor=B.+E.+Correiaauthor=B.+F.+Cravatt&title=Global+profiling+of+lysine+reactivity+and+ligandability+in+the+human+proteome&doi=10.1038%2Fnchem.2826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Global profiling of lysine reactivity and ligandability in the human proteome</span></div><div class="casAuthors">Hacker, Stephan M.; Backus, Keriann M.; Lazear, Michael R.; Forli, Stefano; Correia, Bruno E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nucleophilic amino acids make important contributions to protein function, including performing key roles in catalysis and serving as sites for post-translational modification.  Electrophilic groups that target amino-acid nucleophiles have been used to create covalent ligands and drugs, but have, so far, been mainly limited to cysteine and serine.  Here, the authors report a chem. proteomic platform for the global and quant. anal. of lysine residues in native biol. systems.  The authors have quantified, in total, >9000 lysines in human cell proteomes and have identified several hundred residues with heightened reactivity that are enriched at protein functional sites and can frequently be targeted by electrophilic small mols.  The authors also discovered lysine-reactive fragment electrophiles that inhibit enzymes by active site and allosteric mechanisms, as well as disrupt protein-protein interactions in transcriptional regulatory complexes, emphasizing the broad potential and diverse functional consequences of liganding lysine residues throughout the human proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRKvUVcT5YRrVg90H21EOLACvtfcHk0lj56-ZRNKGtMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN&md5=1dac8f079c1caffc8ae584291480fc61</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2826%26sid%3Dliteratum%253Aachs%26aulast%3DHacker%26aufirst%3DS.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DLazear%26aufirst%3DM.%2BR.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DGlobal%2520profiling%2520of%2520lysine%2520reactivity%2520and%2520ligandability%2520in%2520the%2520human%2520proteome%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1181%26epage%3D1190%26doi%3D10.1038%2Fnchem.2826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span> <span> </span><span class="NLM_article-title">Cysteine-reactive probes and their use in chemical proteomics</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4501</span>– <span class="NLM_lpage">4512</span>, <span class="refDoi"> DOI: 10.1039/c8cc01485j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc8cc01485j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=29645055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsVegs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2018&pages=4501-4512&author=D.+G.+Hochauthor=D.+Abeggauthor=A.+Adibekian&title=Cysteine-reactive+probes+and+their+use+in+chemical+proteomics&doi=10.1039%2Fc8cc01485j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine-reactive probes and their use in chemical proteomics</span></div><div class="casAuthors">Hoch, Dominic G.; Abegg, Daniel; Adibekian, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">4501-4512</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Proteomic profiling using bioorthogonal chem. probes that selectively react with certain amino acids is now a widely used method in life sciences to investigate enzymic activities, study posttranslational modifications and discover novel covalent inhibitors.  Over the past two decades, researchers have developed selective probes for several different amino acids, including lysine, serine, cysteine, threonine, tyrosine, aspartate and glutamate.  Among these amino acids, cysteines are particularly interesting due to their highly diverse and complex biochem. role in our cells.  In this feature article, we focus on the chem. probes and methods used to study cysteines in complex proteomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGricIk8XJT4m7Vg90H21EOLACvtfcHk0lj56-ZRNKGtMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsVegs7k%253D&md5=d91bb3d96048cb6c9516d69033b2365f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2Fc8cc01485j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc8cc01485j%26sid%3Dliteratum%253Aachs%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DAdibekian%26aufirst%3DA.%26atitle%3DCysteine-reactive%2520probes%2520and%2520their%2520use%2520in%2520chemical%2520proteomics%26jtitle%3DChem.%2520Commun.%26date%3D2018%26volume%3D54%26spage%3D4501%26epage%3D4512%26doi%3D10.1039%2Fc8cc01485j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span> <span> </span><span class="NLM_article-title">Optimization of caged electrophiles for improved monitoring of cysteine reactivity in living cells</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1002/cbic.201600524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fcbic.201600524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=27813293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFyrtrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=81-84&author=M.+Aboauthor=D.+W.+Bakauthor=E.+Weerapana&title=Optimization+of+caged+electrophiles+for+improved+monitoring+of+cysteine+reactivity+in+living+cells&doi=10.1002%2Fcbic.201600524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Caged Electrophiles for Improved Monitoring of Cysteine Reactivity in Living Cells</span></div><div class="casAuthors">Abo, Masahiro; Bak, Daniel W.; Weerapana, Eranthie</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-84</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cysteine residues play crit. roles in protein function and are susceptible to numerous post-translational modifications (PTMs) that serve to modulate the activity and localization of diverse proteins.  Many of these PTMs are highly transient and labile, thus necessitating methods to study these modifications directly within the context of living cells.  We previously reported a caged electrophilic probe, CBK1, that can be activated by UV for temporally controlled covalent modification of cysteine residues in living cells.  To improve upon the no. of cysteine residues identified in cellular cysteine-profiling studies, the reactivity and uncaging efficiency of a panel of caged electrophiles were explored.  We identified an optimized caged electrophilic probe, CIK4, that affords significantly improved coverage of cellular cysteine residues.  The broader proteome coverage afforded by CIK4 renders it a useful tool for the biol. investigation of cysteine-reactivity changes and PTMs directly within living cells and highlights design elements that are crit. to optimizing photoactivatable chem. probes for cellular labeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB8LMfPRO2i7Vg90H21EOLACvtfcHk0ljL-PPY5hCK2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFyrtrfM&md5=08cb2975512fce1d9bbf773a27fef930</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201600524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201600524%26sid%3Dliteratum%253Aachs%26aulast%3DAbo%26aufirst%3DM.%26aulast%3DBak%26aufirst%3DD.%2BW.%26aulast%3DWeerapana%26aufirst%3DE.%26atitle%3DOptimization%2520of%2520caged%2520electrophiles%2520for%2520improved%2520monitoring%2520of%2520cysteine%2520reactivity%2520in%2520living%2520cells%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D81%26epage%3D84%26doi%3D10.1002%2Fcbic.201600524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, K. S.</span></span> <span> </span><span class="NLM_article-title">Cysteine-mediated redox signaling: Chemistry, Biology, and Tools for discovery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">4633</span>– <span class="NLM_lpage">4679</span>, <span class="refDoi"> DOI: 10.1021/cr300163e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr300163e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlert7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2013&pages=4633-4679&author=C.+E.+Paulsenauthor=K.+S.+Carroll&title=Cysteine-mediated+redox+signaling%3A+Chemistry%2C+Biology%2C+and+Tools+for+discovery&doi=10.1021%2Fcr300163e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine-mediated redox signaling: Chemistry, biology, and tools for discovery</span></div><div class="casAuthors">Paulsen, Candice E.; Carroll, Kate S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4633-4679</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The different oxidative post-translational modifications of protein Cys residue SH groups are reviewed, with particular emphasis on those chem. properties that differentiate one modification from another.  Recent progress in using chem. approaches to develop probes that enable selective and direct detection of individual modifications within their native cellular environment are also reviewed.  The discussion is complemented with examples from the literature that highlight ways in which Cys oxidn. can be used to control protein function and cell signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeGtY8LEzuF7Vg90H21EOLACvtfcHk0ljL-PPY5hCK2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlert7Y%253D&md5=a071177f7815ccefe7a58abe02d24906</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fcr300163e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr300163e%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DC.%2BE.%26aulast%3DCarroll%26aufirst%3DK.%2BS.%26atitle%3DCysteine-mediated%2520redox%2520signaling%253A%2520Chemistry%252C%2520Biology%252C%2520and%2520Tools%2520for%2520discovery%26jtitle%3DChem.%2520Rev.%26date%3D2013%26volume%3D113%26spage%3D4633%26epage%3D4679%26doi%3D10.1021%2Fcr300163e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerato, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad Hari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide profiling of targets of cysteine reactive small molecules by using ethynyl benziodoxolone reagents</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">10852</span>– <span class="NLM_lpage">10857</span>, <span class="refDoi"> DOI: 10.1002/anie.201505641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fanie.201505641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1CiurbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=10852-10857&author=D.+Abeggauthor=R.+Freiauthor=L.+Ceratoauthor=D.+Prasad+Hariauthor=C.+Wangauthor=J.+Waserauthor=A.+Adibekian&title=Proteome-wide+profiling+of+targets+of+cysteine+reactive+small+molecules+by+using+ethynyl+benziodoxolone+reagents&doi=10.1002%2Fanie.201505641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-Wide Profiling of Targets of Cysteine-reactive Small Molecules by using Ethynyl Benziodoxolone Reagents</span></div><div class="casAuthors">Abegg, Daniel; Frei, Reto; Cerato, Luca; Prasad Hari, Durga; Wang, Chao; Waser, Jerome; Adibekian, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">10852-10857</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In this study, we present a highly efficient method for proteomic profiling of cysteine residues in complex proteomes and in living cells.  Our method is based on alkynylation of cysteines in complex proteomes using a "clickable" alkynyl benziodoxolone bearing an azide group.  This reaction proceeds fast, under mild physiol. conditions, and with a very high degree of chemoselectivity.  The formed azide-capped alkynyl-cysteine adducts are readily detectable by LC-MS/MS, and can be further functionalized with TAMRA or biotin alkyne via CuAAC.  We demonstrate the utility of alkynyl benziodoxolones for chem. proteomics applications by identifying the proteomic targets of curcumin, a diarylheptanoid natural product that was and still is part of multiple human clin. trials as anticancer agent.  Our results demonstrate that curcumin covalently modifies several key players of cellular signaling and metab., most notably the enzyme casein kinase I gamma.  We anticipate that this new method for cysteine profiling will find broad application in chem. proteomics and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0j8t7w7eJmLVg90H21EOLACvtfcHk0ljL-PPY5hCK2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1CiurbN&md5=e033baff50f02f4ff73e70d8ac9c1106</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fanie.201505641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201505641%26sid%3Dliteratum%253Aachs%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DFrei%26aufirst%3DR.%26aulast%3DCerato%26aufirst%3DL.%26aulast%3DPrasad%2BHari%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWaser%26aufirst%3DJ.%26aulast%3DAdibekian%26aufirst%3DA.%26atitle%3DProteome-wide%2520profiling%2520of%2520targets%2520of%2520cysteine%2520reactive%2520small%2520molecules%2520by%2520using%2520ethynyl%2520benziodoxolone%2520reagents%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D10852%26epage%3D10857%26doi%3D10.1002%2Fanie.201505641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gersch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreuzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">Electrophilic natural products and their biological targets</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">659</span>– <span class="NLM_lpage">682</span>, <span class="refDoi"> DOI: 10.1039/c2np20012k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc2np20012k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=22504336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC38XntVehsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=659-682&author=M.+Gerschauthor=J.+Kreuzerauthor=S.+A.+Sieber&title=Electrophilic+natural+products+and+their+biological+targets&doi=10.1039%2Fc2np20012k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Electrophilic natural products and their biological targets</span></div><div class="casAuthors">Gersch, Malte; Kreuzer, Johannes; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">659-682</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The study of biol. active natural products has resulted in seminal contributions to the understanding of living systems.  In the case of electrophilic natural products, the covalent nature of their interaction has largely facilitated the identification of their biol. binding partners.  In this review, we provide a comprehensive compilation of electrophilic natural products from all major chem. classes together with their biol. targets.  Covering Michael acceptor systems, ring-strained compds. and other electrophiles, such as esters or carbamates, we highlight representative and instructive examples for over 20 electrophilic moieties.  The fruitful cooperation of natural product chem., medicinal chem. and chem. biol. has produced a collection of well-studied examples for how electrophilic natural products exert their biol. functions that range from antibiotic to antitumor effects.  Special emphasis is put on the elucidation of their resp. biol. targets via activity-based protein profiling, which together with the recent advancements in mass spectrometry has been crucial to the success of the field.  The wealth of naturally occurring electrophilic moieties and their chem. complexity enables binding of a large variety of biol. targets, such as enzymes of all classes, nonenzymic proteins, DNA and other cellular compds.  With approx. 30 000 genes in the human genome but only 266 confirmed protein drug targets, the study of biol. active, electrophilic natural products has the potential to provide insights into fundamental biol. processes and to greatly aid the discovery of new drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHI4qnF7oSNbVg90H21EOLACvtfcHk0ljL-PPY5hCK2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntVehsb0%253D&md5=560cb6a084f0905741109551530ed02c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1039%2Fc2np20012k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2np20012k%26sid%3Dliteratum%253Aachs%26aulast%3DGersch%26aufirst%3DM.%26aulast%3DKreuzer%26aufirst%3DJ.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3DElectrophilic%2520natural%2520products%2520and%2520their%2520biological%2520targets%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2012%26volume%3D29%26spage%3D659%26epage%3D682%26doi%3D10.1039%2Fc2np20012k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunzmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">α-Methylene-γ-butyrolactones attenuate Staphylococcus aureus virulence by inhibition of transcriptional regulation</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1158</span>, <span class="refDoi"> DOI: 10.1039/c3sc52228h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc3sc52228h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Oqtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1158&author=M.+H.+Kunzmannauthor=N.+C.+Bachauthor=B.+Bauerauthor=S.+A.+Sieber&title=%CE%B1-Methylene-%CE%B3-butyrolactones+attenuate+Staphylococcus+aureus+virulence+by+inhibition+of+transcriptional+regulation&doi=10.1039%2Fc3sc52228h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">α-Methylene-γ-butyrolactones attenuate Staphylococcus aureus virulence by inhibition of transcriptional regulation</span></div><div class="casAuthors">Kunzmann, Martin H.; Bach, Nina C.; Bauer, Bianca; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1158-1167</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Bacterial pathogenesis is triggered by complex mol. mechanisms that sense bacterial d. within an infected host and induce the expression of toxins for overriding the immune response.  Virulence is controlled by a set of transcriptional regulators that directly bind to DNA promoter regions of toxin-encoding genes.  Here, we identified an α-methylene-γ-butyrolactone as a potent inhibitor of Staphylococcus aureus virulence.  Treatment of bacteria not only resulted in a markedly decreased expression of one of the most prominent virulence factors α-hemolysin (Hla) but also caused attenuated invasion efficiency.  Mass spectrometry (MS) based target identification revealed this biol. effect originating from the consolidated binding to three important transcriptional regulators SarA, SarR and MgrA.  MS investigation of the binding site uncovered a conserved cysteine in all three proteins which gets covalently modified.  Intriguingly, investigation of DNA binding demonstrated an impaired DNA-protein interaction upon compd. treatment.  The functional correlation between target binding, inhibition and the obsd. biol. effect was proven by gene knockouts and confirmed the expected mode of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFG3nbr90xeLVg90H21EOLACvtfcHk0lg8jXmPpgzncQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Oqtr8%253D&md5=9ecf5a95ee148d3885d43f3b9bcc6123</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2Fc3sc52228h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3sc52228h%26sid%3Dliteratum%253Aachs%26aulast%3DKunzmann%26aufirst%3DM.%2BH.%26aulast%3DBach%26aufirst%3DN.%2BC.%26aulast%3DBauer%26aufirst%3DB.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3D%25CE%25B1-Methylene-%25CE%25B3-butyrolactones%2520attenuate%2520Staphylococcus%2520aureus%2520virulence%2520by%2520inhibition%2520of%2520transcriptional%2520regulation%26jtitle%3DChem.%2520Sci.%26date%3D2014%26volume%3D5%26spage%3D1158%26doi%3D10.1039%2Fc3sc52228h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandl, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirsch, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugur, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vomacka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fetzer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antes, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">Natural-product-inspired aminoepoxybenzoquinones kill members of the Gram-negative pathogen <i>Salmonella</i> by attenuating cellular stress response</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">14852</span>– <span class="NLM_lpage">14857</span>, <span class="refDoi"> DOI: 10.1002/anie.201607338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fanie.201607338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGitLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=14852-14857&author=F.+A.+Mandlauthor=V.+C.+Kirschauthor=I.+Ugurauthor=E.+Kunoldauthor=J.+Vomackaauthor=C.+Fetzerauthor=S.+Schneiderauthor=K.+Richterauthor=T.+M.+Fuchsauthor=I.+Antesauthor=S.+A.+Sieber&title=Natural-product-inspired+aminoepoxybenzoquinones+kill+members+of+the+Gram-negative+pathogen+Salmonella+by+attenuating+cellular+stress+response&doi=10.1002%2Fanie.201607338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Natural-Product-Inspired Aminoepoxybenzoquinones Kill Members of the Gram-Negative Pathogen Salmonella by Attenuating Cellular Stress Response</span></div><div class="casAuthors">Mandl, Franziska A.; Kirsch, Volker C.; Ugur, Ilke; Kunold, Elena; Vomacka, Jan; Fetzer, Christian; Schneider, Sabine; Richter, Klaus; Fuchs, Thilo M.; Antes, Iris; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">14852-14857</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Gram-neg. bacteria represent a challenging task for antibacterial drug discovery owing to their impermeable cell membrane and restricted uptake of small mols.  The authors herein describe the synthesis of natural-product-derived epoxycyclohexenones and explore their antibiotic activity against several pathogenic bacteria.  A compd. with activity against Salmonella Typhimurium was identified, and the target enzymes were unraveled by quant. chem. proteomics.  Importantly, two protein hits were linked to bacterial stress response, and corresponding assays revealed an elevated susceptibility to reactive oxygen species upon compd. treatment.  The consolidated inhibition of these targets provides a rationale for antibacterial activity and highlights epoxycyclohexenones as natural product scaffolds with suitable properties for killing Gram-neg. Salmonella.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIS15iSizD1LVg90H21EOLACvtfcHk0lg8jXmPpgzncQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGitLbP&md5=fca9cf9dc7031442ef628e169e410cc7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201607338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201607338%26sid%3Dliteratum%253Aachs%26aulast%3DMandl%26aufirst%3DF.%2BA.%26aulast%3DKirsch%26aufirst%3DV.%2BC.%26aulast%3DUgur%26aufirst%3DI.%26aulast%3DKunold%26aufirst%3DE.%26aulast%3DVomacka%26aufirst%3DJ.%26aulast%3DFetzer%26aufirst%3DC.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DRichter%26aufirst%3DK.%26aulast%3DFuchs%26aufirst%3DT.%2BM.%26aulast%3DAntes%26aufirst%3DI.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3DNatural-product-inspired%2520aminoepoxybenzoquinones%2520kill%2520members%2520of%2520the%2520Gram-negative%2520pathogen%2520Salmonella%2520by%2520attenuating%2520cellular%2520stress%2520response%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D14852%26epage%3D14857%26doi%3D10.1002%2Fanie.201607338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staub, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">β-Lactam probes as selective chemical-proteomic tools for the identification and functional characterization of resistance associated enzymes in MRSA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">6271</span>– <span class="NLM_lpage">6276</span>, <span class="refDoi"> DOI: 10.1021/ja901304n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja901304n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktF2nsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=6271-6276&author=I.+Staubauthor=S.+A.+Sieber&title=%CE%B2-Lactam+probes+as+selective+chemical-proteomic+tools+for+the+identification+and+functional+characterization+of+resistance+associated+enzymes+in+MRSA&doi=10.1021%2Fja901304n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">β-Lactam Probes As Selective Chemical-Proteomic Tools for the Identification and Functional Characterization of Resistance Associated Enzymes in MRSA</span></div><div class="casAuthors">Staub, Isabell; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6271-6276</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the development of antibiotic resistant bacterial strains, infectious diseases have become again a life threatening problem.  One of the reasons for this dilemma is the limited no. and breadth of current therapeutic targets for which several resistance strategies have evolved over time.  To identify resistance assocd. targets and to understand their function, activity, and regulation, we utilized a novel strategy based on small synthetic β-lactam mols. that were applied in activity based protein profiling expts. (ABPP) to comparatively profile in situ enzyme activities in antibiotic sensitive and resistant S. aureus strains (MRSA).  Several enzyme activities which are unique to the MRSA strain including known resistant assocd. targets, involved in cell wall biosynthesis and antibiotic sensing, could be identified.  In addn., we also identified uncharacterized enzymes which turned out to be capable of hydrolyzing β-lactam antibiotics.  This technol. could therefore represent a valuable tool to monitor the activity and function of other yet unexplored resistance assocd. enzymes in pathogenic bacteria and help to discover new drug targets for customized therapeutic interventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1GeSOXr49KbVg90H21EOLACvtfcHk0lg8jXmPpgzncQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktF2nsro%253D&md5=dc0a31cf0524ddce26d1c5e128d564b0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fja901304n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja901304n%26sid%3Dliteratum%253Aachs%26aulast%3DStaub%26aufirst%3DI.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3D%25CE%25B2-Lactam%2520probes%2520as%2520selective%2520chemical-proteomic%2520tools%2520for%2520the%2520identification%2520and%2520functional%2520characterization%2520of%2520resistance%2520associated%2520enzymes%2520in%2520MRSA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D6271%26epage%3D6276%26doi%3D10.1021%2Fja901304n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staub, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">β-Lactams as Selective Chemical Probes for the in Vivo Labeling of Bacterial Enzymes Involved in Cell Wall Biosynthesis, Antibiotic Resistance, and Virulence</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">13400</span>– <span class="NLM_lpage">13409</span>, <span class="refDoi"> DOI: 10.1021/ja803349j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja803349j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtV2lu7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=13400-13409&author=I.+Staubauthor=S.+A.+Sieber&title=%CE%B2-Lactams+as+Selective+Chemical+Probes+for+the+in+Vivo+Labeling+of+Bacterial+Enzymes+Involved+in+Cell+Wall+Biosynthesis%2C+Antibiotic+Resistance%2C+and+Virulence&doi=10.1021%2Fja803349j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">β-Lactams as Selective Chemical Probes for the in Vivo Labeling of Bacterial Enzymes Involved in Cell Wall Biosynthesis, Antibiotic Resistance, and Virulence</span></div><div class="casAuthors">Staub, Isabell; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13400-13409</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the development of antibiotic-resistant bacterial strains, infectious diseases have become again a life-threatening problem.  One of the reasons for this dilemma is the limited no. and breadth of current therapeutic targets for which several resistance strategies have evolved over time.  To expand the no. of addressable enzyme targets and to understand their function, activity, and regulation, the authors utilized a chem. proteomic strategy, called activity-based protein profiling (ABPP) pioneered by Cravatt, for the identification of β-lactam-binding enzymes under in vivo conditions.  In this two-tiered strategy, the authors first prepd. a selection of conventional antibiotics for labeling diverse penicillin binding proteins (PBPs) and second introduced a new synthetic generation of β-lactam probes, which labeled and inhibited a selection of addnl. PBP unrelated bacterial targets.  Among these, the virulence-assocd. enzyme ClpP and a resistance-assocd. β-lactamase were labeled and inhibited by selected probes, indicating that the specificity of β-lactams can be adjusted to versatile enzyme families with important cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3q8I13btJErVg90H21EOLACvtfcHk0ljVCI3TrWMeWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtV2lu7fK&md5=e20c1f362434fe948a61ebbe9260867c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja803349j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja803349j%26sid%3Dliteratum%253Aachs%26aulast%3DStaub%26aufirst%3DI.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3D%25CE%25B2-Lactams%2520as%2520Selective%2520Chemical%2520Probes%2520for%2520the%2520in%2520Vivo%2520Labeling%2520of%2520Bacterial%2520Enzymes%2520Involved%2520in%2520Cell%2520Wall%2520Biosynthesis%252C%2520Antibiotic%2520Resistance%252C%2520and%2520Virulence%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D13400%26epage%3D13409%26doi%3D10.1021%2Fja803349j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drechsel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandl, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">Chemical probe to monitor the Parkinsonism-associated protein DJ-1 in live cells</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2016</span>– <span class="NLM_lpage">2019</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00633</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00633" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2016-2019&author=J.+Drechselauthor=F.+A.+Mandlauthor=S.+A.+Sieber&title=Chemical+probe+to+monitor+the+Parkinsonism-associated+protein+DJ-1+in+live+cells&doi=10.1021%2Facschembio.8b00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Probe To Monitor the Parkinsonism-Associated Protein DJ-1 in Live Cells</span></div><div class="casAuthors">Drechsel, Jonas; Mandl, Franziska A.; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2016-2019</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Reactive oxygen species (ROS) play an important role in the onset of Parkinson's disease (PD), and deciphering protective mechanisms is a major goal for therapeutic development.  Here, DJ-1 (PARK7) gained major attention when a conserved cysteine residue with a putative role in oxidative stress sensing/protection was linked to PD.  Inspired by previous studies with a bacterial homolog of DJ-1, several amino-epoxycylcohexenones were screened for enzyme inhibition, and a chem. probe with specificity for the human ortholog was selected for cellular studies.  The probe selectively labeled the cysteine oxidn. sensor and whole proteome anal. in HeLa, A549, and SHSY5Y cell lines confirmed strong enrichment of reduced DJ-1 as the most prominent target.  Increasing levels of oxidative stress diminished this signal demonstrating the utility of the tool compd. for selective in situ monitoring of this important biomarker in its reduced state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre49Kkobp6ArVg90H21EOLACvtfcHk0ljVCI3TrWMeWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtLrI&md5=91dc30eb533205dfbe1d38a970f31960</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00633%26sid%3Dliteratum%253Aachs%26aulast%3DDrechsel%26aufirst%3DJ.%26aulast%3DMandl%26aufirst%3DF.%2BA.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3DChemical%2520probe%2520to%2520monitor%2520the%2520Parkinsonism-associated%2520protein%2520DJ-1%2520in%2520live%2520cells%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2016%26epage%3D2019%26doi%3D10.1021%2Facschembio.8b00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dharmaraja, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dash, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konkimalla, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrapani, H.</span></span> <span> </span><span class="NLM_article-title">Synthesis, thiol-mediated reactive oxygen species generation profiles and anti-proliferative activities of 2,3-epoxy-1,4-naphthoquinones</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1039/c1md00234a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc1md00234a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=219-224&author=A.+T.+Dharmarajaauthor=T.+K.+Dashauthor=V.+B.+Konkimallaauthor=H.+Chakrapani&title=Synthesis%2C+thiol-mediated+reactive+oxygen+species+generation+profiles+and+anti-proliferative+activities+of+2%2C3-epoxy-1%2C4-naphthoquinones&doi=10.1039%2Fc1md00234a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2Fc1md00234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00234a%26sid%3Dliteratum%253Aachs%26aulast%3DDharmaraja%26aufirst%3DA.%2BT.%26aulast%3DDash%26aufirst%3DT.%2BK.%26aulast%3DKonkimalla%26aufirst%3DV.%2BB.%26aulast%3DChakrapani%26aufirst%3DH.%26atitle%3DSynthesis%252C%2520thiol-mediated%2520reactive%2520oxygen%2520species%2520generation%2520profiles%2520and%2520anti-proliferative%2520activities%2520of%25202%252C3-epoxy-1%252C4-naphthoquinones%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D219%26epage%3D224%26doi%3D10.1039%2Fc1md00234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Puri, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Applications of small molecule probes in dissecting mechanisms of bacterial virulence and host responses</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">5985</span>– <span class="NLM_lpage">5996</span>, <span class="refDoi"> DOI: 10.1021/bi400854d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi400854d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Cqtb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=5985-5996&author=A.+W.+Puriauthor=M.+Bogyo&title=Applications+of+small+molecule+probes+in+dissecting+mechanisms+of+bacterial+virulence+and+host+responses&doi=10.1021%2Fbi400854d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of Small Molecule Probes in Dissecting Mechanisms of Bacterial Virulence and Host Responses</span></div><div class="casAuthors">Puri, Aaron W.; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">5985-5996</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Elucidating the mol. and biochem. details of bacterial infections can be challenging because of the many complex interactions that exist between a pathogen and its host.  Consequently, many tools have been developed to aid the study of bacterial pathogenesis.  Small mols. are a valuable complement to traditional genetic techniques because they can be used to rapidly perturb genetically intractable systems and to monitor post-translationally regulated processes.  Activity-based probes are a subset of small mols. that covalently label an enzyme of interest based on its catalytic mechanism.  These tools allow monitoring of enzyme activation within the context of a native biol. system and can be used to dissect the biochem. details of enzyme function.  This review describes the development and application of activity-based probes for examg. aspects of bacterial infection on both sides of the host-pathogen interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf6fjRlyM797Vg90H21EOLACvtfcHk0lj82B55Cp7MTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Cqtb7L&md5=df69fd0943da31227af7f13bbd746786</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fbi400854d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi400854d%26sid%3Dliteratum%253Aachs%26aulast%3DPuri%26aufirst%3DA.%2BW.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DApplications%2520of%2520small%2520molecule%2520probes%2520in%2520dissecting%2520mechanisms%2520of%2520bacterial%2520virulence%2520and%2520host%2520responses%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26spage%3D5985%26epage%3D5996%26doi%3D10.1021%2Fbi400854d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welsch, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Privileged scaffolds for library design and drug discovery</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1016%2Fj.cbpa.2010.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=20303320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=347-361&author=M.+E.+Welschauthor=S.+A.+Snyderauthor=B.+R.+Stockwell&title=Privileged+scaffolds+for+library+design+and+drug+discovery&doi=10.1016%2Fj.cbpa.2010.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged scaffolds for library design and drug discovery</span></div><div class="casAuthors">Welsch, Matthew E.; Snyder, Scott A.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-361</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review explores the concept of using privileged scaffolds to identify biol. active compds. through building chem. libraries.  We hope to accomplish three main objectives: to provide one of the most comprehensive listings of privileged scaffolds; to reveal through four selected examples the present state of the art in privileged scaffold library synthesis (in hopes of inspiring new and even more creative approaches); and also to offer some thoughts on how new privileged scaffolds might be identified and exploited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWQPxdsWawiLVg90H21EOLACvtfcHk0lj82B55Cp7MTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D&md5=fd0c2a642f063fd6f42f2c34fd186f8a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DWelsch%26aufirst%3DM.%2BE.%26aulast%3DSnyder%26aufirst%3DS.%2BA.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DPrivileged%2520scaffolds%2520for%2520library%2520design%2520and%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D347%26epage%3D361%26doi%3D10.1016%2Fj.cbpa.2010.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><a href="https://clsi.org/media/1632/m07a10_sample.pdf" class="extLink">https://clsi.org/media/1632/m07a10_sample.pdf</a> (accessed June
20, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fclsi.org%2Fmedia%2F1632%2Fm07a10_sample.pdf+%28accessed+June%0A20%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span> <span> </span><span class="NLM_article-title">Identification and bioevaluation of SRI-12742 as an antimicrobial agent against multidrug-resistant <i>Acinetobacter baumannii</i></span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2018.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1016%2Fj.ijantimicag.2018.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=29501820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFSqsrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2018&pages=22-27&author=M.+Shuklaauthor=I.+Soniauthor=K.+Matsuyamaauthor=T.+Tranauthor=M.+Tangaauthor=L.+Gongauthor=S.+Chopra&title=Identification+and+bioevaluation+of+SRI-12742+as+an+antimicrobial+agent+against+multidrug-resistant+Acinetobacter+baumannii&doi=10.1016%2Fj.ijantimicag.2018.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and bioevaluation of SRI-12742 as an antimicrobial agent against multidrug-resistant Acinetobacter baumannii</span></div><div class="casAuthors">Shukla, Manjulika; Soni, Isha; Matsuyama, Karen; Tran, Tran; Tanga, Mary; Gong, Leyi; Chopra, Sidharth</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-27</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Multidrug-resistant Acinetobacter baumannii (MDR-Ab) is one of the most significant nosocomial pathogens that is being increasingly isolated in healthcare settings worldwide.  Owing to its inherent drug-resistant nature, coupled with its ability to readily acquire resistance to other antibiotic classes, there is a real dearth of antibiotics available to treat infections with MDR-Ab.  A com. available library was screened against MDR-Ab BAA-1605 to identify novel inhibitory mols.  The selectivity index of a hit was tested against Vero cells and in vitro efficacy was profiled against a panel of clin. MDR-Ab.  The bacteriostatic or bactericidal nature was detd. by time-kill expts., and synergy with clin. approved drugs was detd. by the chequerboard method.  Addnl., in vivo efficacy was measured in a murine neutropenic A. baumannii thigh infection model.  SRI-12742 was identified as a potent active hit, with a min. inhibitory concn. (MIC) of 4 mg/L against BAA-1605.  Its activity was then profiled against a MDR-Ab clin. strain panel (MICs 4 mg/L to >64 mg/L).  SRI-12742 exhibited concn.-dependent bactericidal activity and caused an ca. 16 log10 CFU/mL redn. at 10 x MIC in 24 h, which is comparable with minocycline.  In a murine neutropenic thigh infection model of A. baumannii infection, SRI-12742 reduced CFU counts by ca. 0.9 log10 CFU, which is comparable with polymyxin B.  In addn., SRI-12742 synergised with all classes of antibiotics tested.  SRI-12742 exhibits all of the criteria necessary to be positioned as a novel lead with potential to be deployed for the treatment of infections caused by MDR-Ab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlXbBF4QAxwLVg90H21EOLACvtfcHk0lj82B55Cp7MTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFSqsrzN&md5=b96e66d537c34e9c35a00e8a3a7cad3d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2018.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2018.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DM.%26aulast%3DSoni%26aufirst%3DI.%26aulast%3DMatsuyama%26aufirst%3DK.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DTanga%26aufirst%3DM.%26aulast%3DGong%26aufirst%3DL.%26aulast%3DChopra%26aufirst%3DS.%26atitle%3DIdentification%2520and%2520bioevaluation%2520of%2520SRI-12742%2520as%2520an%2520antimicrobial%2520agent%2520against%2520multidrug-resistant%2520Acinetobacter%2520baumannii%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2018%26volume%3D52%26spage%3D22%26epage%3D27%26doi%3D10.1016%2Fj.ijantimicag.2018.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rybak, M. J.</span></span> <span> </span><span class="NLM_article-title">Bactericidal activities of daptomycin, quinupristin-dalfopristin, and linezolid against vancomycin-resistant Staphylococcus aureus in an <i>in vitro</i> pharmacodynamic model with simulated endocardial vegetations</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3960</span>– <span class="NLM_lpage">3963</span>, <span class="refDoi"> DOI: 10.1128/aac.47.12.3960-3963.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1128%2Faac.47.12.3960-3963.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=14638509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsV2msr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2003&pages=3960-3963&author=R.+Chaauthor=W.+J.+Brownauthor=M.+J.+Rybak&title=Bactericidal+activities+of+daptomycin%2C+quinupristin-dalfopristin%2C+and+linezolid+against+vancomycin-resistant+Staphylococcus+aureus+in+an+in+vitro+pharmacodynamic+model+with+simulated+endocardial+vegetations&doi=10.1128%2Faac.47.12.3960-3963.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bactericidal activities of daptomycin, quinupristin-dalfopristin, and linezolid against vancomycin-resistant Staphylococcus aureus in an in vitro pharmacodynamic model with simulated endocardial vegetations</span></div><div class="casAuthors">Cha, Raymond; Brown, William J.; Rybak, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3960-3963</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In search of treatment alternatives against vancomycin-resistant S. aureus (VRSA), an in vitro pharmacodynamic model with simulated endocardial vegetations incorporating protein and a high inoculum was used to simulate daptomycin, linezolid, quinupristin-dalfopristin, and vancomycin against the Michigan VRSA strain.  Daptomycin and quinupristin-dalfopristin exhibited the greatest bacterial redns., and all tested agents except vancomycin exhibited bactericidal activity against the VRSA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-E5EvH4C2E7Vg90H21EOLACvtfcHk0lj82B55Cp7MTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsV2msr4%253D&md5=e7ae79398cf24dbef61af4b97b8acdd2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2Faac.47.12.3960-3963.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.47.12.3960-3963.2003%26sid%3Dliteratum%253Aachs%26aulast%3DCha%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DW.%2BJ.%26aulast%3DRybak%26aufirst%3DM.%2BJ.%26atitle%3DBactericidal%2520activities%2520of%2520daptomycin%252C%2520quinupristin-dalfopristin%252C%2520and%2520linezolid%2520against%2520vancomycin-resistant%2520Staphylococcus%2520aureus%2520in%2520an%2520in%2520vitro%2520pharmacodynamic%2520model%2520with%2520simulated%2520endocardial%2520vegetations%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2003%26volume%3D47%26spage%3D3960%26epage%3D3963%26doi%3D10.1128%2Faac.47.12.3960-3963.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Housecroft, C.</span>; <span class="NLM_string-name">Sharpe, A.</span></span> <i>Inorganic Chemistry</i>, <span class="NLM_edition">4</span>th ed.; <span class="NLM_publisher-name">Prentice Hall</span>: <span class="NLM_publisher-loc">USA</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=C.+Housecroft&author=A.+Sharpe&title=Inorganic+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHousecroft%26aufirst%3DC.%26btitle%3DInorganic%2520Chemistry%26pub%3DPrentice%2520Hall%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Embaby, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoffelen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kofoed, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meldal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diness, F.</span></span> <span> </span><span class="NLM_article-title">Rational tuning of fluorobenzene probes for cysteine-selective protein modification</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8022</span>– <span class="NLM_lpage">8026</span>, <span class="refDoi"> DOI: 10.1002/anie.201712589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fanie.201712589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1els7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=8022-8026&author=A.+M.+Embabyauthor=S.+Schoffelenauthor=C.+Kofoedauthor=M.+Meldalauthor=F.+Diness&title=Rational+tuning+of+fluorobenzene+probes+for+cysteine-selective+protein+modification&doi=10.1002%2Fanie.201712589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Tuning of Fluorobenzene Probes for Cysteine-Selective Protein Modification</span></div><div class="casAuthors">Embaby, Ahmed M.; Schoffelen, Sanne; Kofoed, Christian; Meldal, Morten; Diness, Frederik</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8022-8026</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Fluorobenzene probes for protein profiling through selective cysteine labeling have been developed by rational reactivity tuning.  Tuning was achieved by selecting an electron-withdrawing para substituent in combination with variation of the no. of fluorine substituents.  Optimized probes chemoselectively arylated cysteine residues in proteins under aq. conditions.  Probes linked to azide, biotin, or a fluorophore were applicable to labeling of eGFP and albumin.  Selective inhibition of cysteine proteases was also demonstrated with the probes.  Addnl., probes were tuned for site-selective labeling of cysteine residues and for activity-based protein profiling in cell lysates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrnIOutUPylLVg90H21EOLACvtfcHk0lgL_O70abDXRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1els7c%253D&md5=a2317907c517038ce96c8f6837f37b4d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201712589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201712589%26sid%3Dliteratum%253Aachs%26aulast%3DEmbaby%26aufirst%3DA.%2BM.%26aulast%3DSchoffelen%26aufirst%3DS.%26aulast%3DKofoed%26aufirst%3DC.%26aulast%3DMeldal%26aufirst%3DM.%26aulast%3DDiness%26aufirst%3DF.%26atitle%3DRational%2520tuning%2520of%2520fluorobenzene%2520probes%2520for%2520cysteine-selective%2520protein%2520modification%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D8022%26epage%3D8026%26doi%3D10.1002%2Fanie.201712589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheerens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprengeler, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrill, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendonca, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, K. C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grothaus, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffery, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spoerke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honigberg, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalrymple, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, J. T.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective irreversible inhibitors for Bruton’s tyrosine kinase</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200600221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fcmdc.200600221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=17154430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=58-61&author=Z.+Panauthor=H.+Scheerensauthor=S.-J.+Liauthor=B.+E.+Schultzauthor=P.+A.+Sprengelerauthor=L.+C.+Burrillauthor=R.+V.+Mendoncaauthor=M.+D.+Sweeneyauthor=K.+C.+K.+Scottauthor=P.+G.+Grothausauthor=D.+A.+Jefferyauthor=J.+M.+Spoerkeauthor=L.+A.+Honigbergauthor=P.+R.+Youngauthor=S.+A.+Dalrympleauthor=J.+T.+Palmer&title=Discovery+of+selective+irreversible+inhibitors+for+Bruton%E2%80%99s+tyrosine+kinase&doi=10.1002%2Fcmdc.200600221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective irreversible inhibitors for bruton's tyrosine kinase</span></div><div class="casAuthors">Pan, Zhengying; Scheerens, Heleen; Li, Shyr-Jiann; Schultz, Brian E.; Sprengeler, Paul A.; Burrill, L. Chuck; Mendonca, Rohan V.; Sweeney, Michael D.; Scott, Keana C. K.; Grothaus, Paul G.; Jeffery, Douglas A.; Spoerke, Jill M.; Honigberg, Lee A.; Young, Peter R.; Dalrymple, Stacie A.; Palmer, James T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-61</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthesis and pharmacol. evaluation of a series of peptidomimetic quinoline derivs. was undertaken to evaluate their efficacy in acting as selective irreversible inhibitors of Bruton's tyrosine kinase (Btk).  With the relative scarcity of knowledge on the inhibition of Btk it is crucial to discover a potent and selective tool compd. for this kinase.  Herein is described the discovery of selective irreversible Btk inhibitors and their efficacy in a mouse RA model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovT3kBCYECdbVg90H21EOLACvtfcHk0lgL_O70abDXRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D&md5=023ba8bca9942ad0127d7cf32293e9d2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600221%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DScheerens%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.-J.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26aulast%3DSprengeler%26aufirst%3DP.%2BA.%26aulast%3DBurrill%26aufirst%3DL.%2BC.%26aulast%3DMendonca%26aufirst%3DR.%2BV.%26aulast%3DSweeney%26aufirst%3DM.%2BD.%26aulast%3DScott%26aufirst%3DK.%2BC.%2BK.%26aulast%3DGrothaus%26aufirst%3DP.%2BG.%26aulast%3DJeffery%26aufirst%3DD.%2BA.%26aulast%3DSpoerke%26aufirst%3DJ.%2BM.%26aulast%3DHonigberg%26aufirst%3DL.%2BA.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DDalrymple%26aufirst%3DS.%2BA.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26atitle%3DDiscovery%2520of%2520selective%2520irreversible%2520inhibitors%2520for%2520Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D58%26epage%3D61%26doi%3D10.1002%2Fcmdc.200600221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dix, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hett, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joslyn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnute, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulce, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whiteley, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">A road map to evaluate the proteome-wide selectivity of covalent kinase inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">760</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1582</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio.1582" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=25038787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=760&author=B.+R.+Lanningauthor=L.+R.+Whitbyauthor=M.+M.+Dixauthor=J.+Douhanauthor=A.+M.+Gilbertauthor=E.+C.+Hettauthor=T.+O.+Johnsonauthor=C.+Joslynauthor=J.+C.+Kathauthor=S.+Niessenauthor=L.+R.+Robertsauthor=M.+E.+Schnuteauthor=C.+Wangauthor=J.+J.+Hulceauthor=B.+Weiauthor=L.+O.+Whiteleyauthor=M.+M.+Haywardauthor=B.+F.+Cravatt&title=A+road+map+to+evaluate+the+proteome-wide+selectivity+of+covalent+kinase+inhibitors&doi=10.1038%2Fnchembio.1582"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A road map to evaluate the proteome-wide selectivity of covalent kinase inhibitors</span></div><div class="casAuthors">Lanning, Bryan R.; Whitby, Landon R.; Dix, Melissa M.; Douhan, John; Gilbert, Adam M.; Hett, Erik C.; Johnson, Theodore O.; Joslyn, Chris; Kath, John C.; Niessen, Sherry; Roberts, Lee R.; Schnute, Mark E.; Wang, Chu; Hulce, Jonathan J.; Wei, Baoxian; Whiteley, Laurence O.; Hayward, Matthew M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">760-767</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinases are principal components of signal transduction pathways and the focus of intense basic and drug discovery research.  Irreversible inhibitors that covalently modify non-catalytic cysteines in kinase active sites have emerged as valuable probes and approved drugs.  Many protein classes, however, have functional cysteines, and therefore understanding the proteome-wide selectivity of covalent kinase inhibitors is imperative.  Here, we accomplish this objective using activity-based protein profiling coupled with quant. MS to globally map the targets, both specific and nonspecific, of covalent kinase inhibitors in human cells.  Many of the specific off-targets represent nonkinase proteins that, notably, have conserved active site cysteines.  We define windows of selectivity for covalent kinase inhibitors and show that, when these windows are exceeded, rampant proteome-wide reactivity and kinase target-independent cell death conjointly occur.  Our findings, taken together, provide an exptl. road map to illuminate opportunities and surmount challenges for the development of covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMAiUlEg9R5rVg90H21EOLACvtfcHk0lgMr2Ns3_Pq-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nO&md5=6f17b0d1b00e088f37c511eea577d290</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1582%26sid%3Dliteratum%253Aachs%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DWhitby%26aufirst%3DL.%2BR.%26aulast%3DDix%26aufirst%3DM.%2BM.%26aulast%3DDouhan%26aufirst%3DJ.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DHett%26aufirst%3DE.%2BC.%26aulast%3DJohnson%26aufirst%3DT.%2BO.%26aulast%3DJoslyn%26aufirst%3DC.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26aulast%3DSchnute%26aufirst%3DM.%2BE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHulce%26aufirst%3DJ.%2BJ.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWhiteley%26aufirst%3DL.%2BO.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DA%2520road%2520map%2520to%2520evaluate%2520the%2520proteome-wide%2520selectivity%2520of%2520covalent%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D760%26doi%3D10.1038%2Fnchembio.1582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allimuthu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span> <span> </span><span class="NLM_article-title">2-Chloropropionamide as a low-reactivity electrophile for irreversible small-molecule probe identification</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2124</span>– <span class="NLM_lpage">2131</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00424</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00424" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvVGqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2124-2131&author=D.+Allimuthuauthor=D.+J.+Adams&title=2-Chloropropionamide+as+a+low-reactivity+electrophile+for+irreversible+small-molecule+probe+identification&doi=10.1021%2Facschembio.7b00424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">2-Chloropropionamide As a Low-Reactivity Electrophile for Irreversible Small-Molecule Probe Identification</span></div><div class="casAuthors">Allimuthu, Dharmaraja; Adams, Drew J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2124-2131</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Resurgent interest in covalent target engagement in drug discovery has demonstrated that small mols. contg. weakly reactive electrophiles can be safe and effective therapies.  Several recently FDA-approved drugs feature an acrylamide functionality to selectively engage cysteine side chains of kinases (Ibrutinib, Afatinib, and Neratinib).  Addnl. electrophilic functionalities whose reactivity is compatible with highly selective target engagement and in vivo application could open new avenues in covalent small mol. discovery.  Here the authors report the synthesis and evaluation of a library of small mols. contg. the 2-chloropropionamide functionality, which the authors demonstrate is less reactive than typical acrylamide electrophiles.  Although many library members do not appear to label proteins in cells, the authors identified S-CW3554 as selectively labeling protein disulfide isomerase and inhibiting its enzymic activity.  Subsequent profiling of the library against five diverse cancer cell lines showed unique cytotoxicity for S-CW3554 I in cells derived from multiple myeloma, a cancer recently reported to be sensitive to PDI inhibition.  The novel PDI inhibitor highlights the potential of 2-chloropropionamides as weak and stereochem.-tunable electrophiles for covalent drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonSvqy3pfybrVg90H21EOLACvtfcHk0lgMr2Ns3_Pq-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvVGqsrw%253D&md5=39b4794ebb55d3314a7bac5b04ddadc7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00424%26sid%3Dliteratum%253Aachs%26aulast%3DAllimuthu%26aufirst%3DD.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26atitle%3D2-Chloropropionamide%2520as%2520a%2520low-reactivity%2520electrophile%2520for%2520irreversible%2520small-molecule%2520probe%2520identification%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2124%26epage%3D2131%26doi%3D10.1021%2Facschembio.7b00424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shindo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miura, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatsuyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokunaga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiroishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caaveiro, J. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyanagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohdo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ono, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojida, A.</span></span> <span> </span><span class="NLM_article-title">Selective and reversible modification of kinase cysteines with chlorofluoroacetamides</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0204-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fs41589-018-0204-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=30643284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFSgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=250-258&author=N.+Shindoauthor=H.+Fuchidaauthor=M.+Satoauthor=K.+Watariauthor=T.+Shibataauthor=K.+Kuwataauthor=C.+Miuraauthor=K.+Okamotoauthor=Y.+Hatsuyamaauthor=K.+Tokunagaauthor=S.+Sakamotoauthor=S.+Morimotoauthor=Y.+Abeauthor=M.+Shiroishiauthor=J.+M.+M.+Caaveiroauthor=T.+Uedaauthor=T.+Tamuraauthor=N.+Matsunagaauthor=T.+Nakaoauthor=S.+Koyanagiauthor=S.+Ohdoauthor=Y.+Yamaguchiauthor=I.+Hamachiauthor=M.+Onoauthor=A.+Ojida&title=Selective+and+reversible+modification+of+kinase+cysteines+with+chlorofluoroacetamides&doi=10.1038%2Fs41589-018-0204-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Selective and reversible modification of kinase cysteines with chlorofluoroacetamides</span></div><div class="casAuthors">Shindo, Naoya; Fuchida, Hirokazu; Sato, Mami; Watari, Kosuke; Shibata, Tomohiro; Kuwata, Keiko; Miura, Chizuru; Okamoto, Kei; Hatsuyama, Yuji; Tokunaga, Keisuke; Sakamoto, Seiichi; Morimoto, Satoshi; Abe, Yoshito; Shiroishi, Mitsunori; Caaveiro, Jose M. M.; Ueda, Tadashi; Tamura, Tomonori; Matsunaga, Naoya; Nakao, Takaharu; Koyanagi, Satoru; Ohdo, Shigehiro; Yamaguchi, Yasuchika; Hamachi, Itaru; Ono, Mayumi; Ojida, Akio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">250-258</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Irreversible inhibition of disease-assocd. proteins with small mols. is a powerful approach for achieving increased and sustained pharmacol. potency.  Here, we introduce α-chlorofluoroacetamide (CFA) as a novel warhead of targeted covalent inhibitor (TCI).  Despite weak intrinsic reactivity, CFA-appended quinazoline showed high reactivity toward Cys797 of epidermal growth factor receptor (EGFR).  In cells, CFA-quinazoline showed higher target specificity for EGFR than the corresponding Michael acceptors in a wide concn. range (0.1-10 μM).  The cysteine adduct of the CFA deriv. was susceptible to hydrolysis and reversibly yielded intact thiol but was stable in solvent-sequestered ATP-binding pocket of EGFR.  This environment-dependent hydrolysis can potentially reduce off-target protein modification by CFA-based drugs.  Oral administration of CFA quinazoline, (2R)-1-(2-chloro-2-fluoroacetyl)-N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[3-(4-morpholinyl)-propoxy]quinazolin-6-yl]pyrrolidine-2-carboxamide [2226257-92-5] (NS-062, compd. 51) significantly suppressed tumor growth in a mouse xenograft model.  Further, CFA-appended pyrazolopyrimidine irreversibly inhibited Bruton's tyrosine kinase with higher target specificity.  These results demonstrate the utility of CFA as a new class warheads for TCI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_uzgSiZvaPbVg90H21EOLACvtfcHk0ljOc5zv3NhiHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFSgur4%253D&md5=f6c8886d540ea26a8abecd950feba913</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0204-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0204-3%26sid%3Dliteratum%253Aachs%26aulast%3DShindo%26aufirst%3DN.%26aulast%3DFuchida%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DM.%26aulast%3DWatari%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DT.%26aulast%3DKuwata%26aufirst%3DK.%26aulast%3DMiura%26aufirst%3DC.%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DHatsuyama%26aufirst%3DY.%26aulast%3DTokunaga%26aufirst%3DK.%26aulast%3DSakamoto%26aufirst%3DS.%26aulast%3DMorimoto%26aufirst%3DS.%26aulast%3DAbe%26aufirst%3DY.%26aulast%3DShiroishi%26aufirst%3DM.%26aulast%3DCaaveiro%26aufirst%3DJ.%2BM.%2BM.%26aulast%3DUeda%26aufirst%3DT.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DMatsunaga%26aufirst%3DN.%26aulast%3DNakao%26aufirst%3DT.%26aulast%3DKoyanagi%26aufirst%3DS.%26aulast%3DOhdo%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DHamachi%26aufirst%3DI.%26aulast%3DOno%26aufirst%3DM.%26aulast%3DOjida%26aufirst%3DA.%26atitle%3DSelective%2520and%2520reversible%2520modification%2520of%2520kinase%2520cysteines%2520with%2520chlorofluoroacetamides%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D250%26epage%3D258%26doi%3D10.1038%2Fs41589-018-0204-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sankar, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumbhare, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmaraja, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrapani, H.</span></span> <span> </span><span class="NLM_article-title">A phenacrylate scaffold for tunable thiol activation and release</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">15323</span>– <span class="NLM_lpage">15326</span>, <span class="refDoi"> DOI: 10.1039/c4cc07343f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc4cc07343f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=25347732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCitL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=15323-15326&author=R.+K.+Sankarauthor=R.+S.+Kumbhareauthor=A.+T.+Dharmarajaauthor=H.+Chakrapani&title=A+phenacrylate+scaffold+for+tunable+thiol+activation+and+release&doi=10.1039%2Fc4cc07343f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A phenacrylate scaffold for tunable thiol activation and release</span></div><div class="casAuthors">Sankar, Rathinam K.; Kumbhare, Rohan S.; Dharmaraja, Allimuthu T.; Chakrapani, Harinath</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">97</span>),
    <span class="NLM_cas:pages">15323-15326</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A thiol-selective 2-methyl-3-phenacrylate scaffold with spatiotemporal control over delivery of a cargo is reported.  The half-lives of decompn. could be tuned from 30 min to 1 day and the scaffold's utility in thiol-inducible fluorophore release in cell-free as well as within cells is demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2eFxDyIc3bLVg90H21EOLACvtfcHk0ljOc5zv3NhiHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCitL7N&md5=ed4ed0ca291eeff03907c6a6c6736bbd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2Fc4cc07343f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4cc07343f%26sid%3Dliteratum%253Aachs%26aulast%3DSankar%26aufirst%3DR.%2BK.%26aulast%3DKumbhare%26aufirst%3DR.%2BS.%26aulast%3DDharmaraja%26aufirst%3DA.%2BT.%26aulast%3DChakrapani%26aufirst%3DH.%26atitle%3DA%2520phenacrylate%2520scaffold%2520for%2520tunable%2520thiol%2520activation%2520and%2520release%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D15323%26epage%3D15326%26doi%3D10.1039%2Fc4cc07343f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serafimova, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pufall, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maglathlin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frödin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Reversible targeting of noncatalytic cysteines with chemically tuned electrophiles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">476</span>, <span class="refDoi"> DOI: 10.1038/nchembio.925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio.925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=22466421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVKitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=471-476&author=I.+M.+Serafimovaauthor=M.+A.+Pufallauthor=S.+Krishnanauthor=K.+Dudaauthor=M.+S.+Cohenauthor=R.+L.+Maglathlinauthor=J.+M.+McFarlandauthor=R.+M.+Millerauthor=M.+Fr%C3%B6dinauthor=J.+Taunton&title=Reversible+targeting+of+noncatalytic+cysteines+with+chemically+tuned+electrophiles&doi=10.1038%2Fnchembio.925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible targeting of noncatalytic cysteines with chemically tuned electrophiles</span></div><div class="casAuthors">Serafimova, Iana M.; Pufall, Miles A.; Krishnan, Shyam; Duda, Katarzyna; Cohen, Michael S.; Maglathlin, Rebecca L.; McFarland, Jesse M.; Miller, Rand M.; Froedin, Morten; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-476</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Targeting noncatalytic cysteine residues with irreversible acrylamide-based inhibitors is a powerful approach for enhancing pharmacol. potency and selectivity.  Nevertheless, concerns about off-target modification motivate the development of reversible cysteine-targeting strategies.  Here we show that electron-deficient olefins, including acrylamides, can be tuned to react with cysteine thiols in a rapidly reversible manner.  Installation of a nitrile group increased the olefins' intrinsic reactivity, but, paradoxically, eliminated the formation of irreversible adducts.  Incorporation of these electrophiles into a noncovalent kinase-recognition scaffold produced slowly dissocg., covalent inhibitors of the p90 ribosomal protein S6 kinase RSK2.  A cocrystal structure revealed specific noncovalent interactions that stabilize the complex by positioning the electrophilic carbon near the targeted cysteine.  Disruption of these interactions by protein unfolding or proteolysis promoted instantaneous cleavage of the covalent bond.  Our results establish a chem.-based framework for engineering sustained covalent inhibition without accumulating permanently modified proteins and peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlcv3GjPOB6rVg90H21EOLACvtfcHk0ljOc5zv3NhiHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVKitrw%253D&md5=6455a9a029714beb317286f71ac1326b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.925%26sid%3Dliteratum%253Aachs%26aulast%3DSerafimova%26aufirst%3DI.%2BM.%26aulast%3DPufall%26aufirst%3DM.%2BA.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DDuda%26aufirst%3DK.%26aulast%3DCohen%26aufirst%3DM.%2BS.%26aulast%3DMaglathlin%26aufirst%3DR.%2BL.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DFr%25C3%25B6din%26aufirst%3DM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DReversible%2520targeting%2520of%2520noncatalytic%2520cysteines%2520with%2520chemically%2520tuned%2520electrophiles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D471%26epage%3D476%26doi%3D10.1038%2Fnchembio.925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Design of reversible, cysteine-targeted Michael acceptors guided by kinetic and computational analysis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">12624</span>– <span class="NLM_lpage">12630</span>, <span class="refDoi"> DOI: 10.1021/ja505194w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja505194w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWrtbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=12624-12630&author=S.+Krishnanauthor=R.+M.+Millerauthor=B.+Tianauthor=R.+D.+Mullinsauthor=M.+P.+Jacobsonauthor=J.+Taunton&title=Design+of+reversible%2C+cysteine-targeted+Michael+acceptors+guided+by+kinetic+and+computational+analysis&doi=10.1021%2Fja505194w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Reversible, Cysteine-Targeted Michael Acceptors Guided by Kinetic and Computational Analysis</span></div><div class="casAuthors">Krishnan, Shyam; Miller, Rand M.; Tian, Boxue; Mullins, R. Dyche; Jacobson, Matthew P.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">12624-12630</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Electrophilic probes that covalently modify a cysteine thiol often show enhanced pharmacol. potency and selectivity.  Although reversible Michael acceptors have been reported, the structural requirements for reversibility are poorly understood.  Here, we report a novel class of acrylonitrile-based Michael acceptors, activated by aryl or heteroaryl electron-withdrawing groups.  We demonstrate that thiol adducts of these acrylonitriles undergo β-elimination at rates that span more than 3 orders of magnitude.  These rates correlate inversely with the computed proton affinity of the corresponding carbanions, enabling the intrinsic reversibility of the thiol-Michael reaction to be tuned in a predictable manner.  We apply these principles to the design of new reversible covalent kinase inhibitors with improved properties.  A cocrystal structure of one such inhibitor reveals specific noncovalent interactions between the 1,2,4-triazole activating group and the kinase.  Our exptl. and computational study enables the design of new Michael acceptors, expanding the palette of reversible, cysteine-targeted electrophiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHXljSEecz5LVg90H21EOLACvtfcHk0lgrs_rPTnuV0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWrtbvK&md5=de052bac07c8920cb807c05f71f4f95d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja505194w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja505194w%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DMullins%26aufirst%3DR.%2BD.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DDesign%2520of%2520reversible%252C%2520cysteine-targeted%2520Michael%2520acceptors%2520guided%2520by%2520kinetic%2520and%2520computational%2520analysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D12624%26epage%3D12630%26doi%3D10.1021%2Fja505194w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khodade, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharath Chandra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulipeta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangarajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrapani, H.</span></span> <span> </span><span class="NLM_article-title">Bioreductively activated reactive oxygen species (ROS) generators as MRSA inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1021/ml5001118</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5001118" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFOiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=777-781&author=V.+S.+Khodadeauthor=M.+Sharath+Chandraauthor=A.+Banerjeeauthor=S.+Lahiriauthor=M.+Pulipetaauthor=R.+Rangarajanauthor=H.+Chakrapani&title=Bioreductively+activated+reactive+oxygen+species+%28ROS%29+generators+as+MRSA+inhibitors&doi=10.1021%2Fml5001118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Bioreductively Activated Reactive Oxygen Species (ROS) Generators as MRSA Inhibitors</span></div><div class="casAuthors">Khodade, Vinayak S.; Sharath Chandra, Mallojjala; Banerjee, Ankita; Lahiri, Surobhi; Pulipeta, Mallikarjuna; Rangarajan, Radha; Chakrapani, Harinath</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">777-781</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The no. of cases of drug resistant Staphylococcus aureus infections is on the rise globally and new strategies to identify drug candidates with novel mechanisms of action are in urgent need.  Here, we report the synthesis and evaluation of a series of benzo[b]phenanthridine-5,7,12(6H)-triones, which were designed based on redox-active natural products.  We find that the in vitro inhibitory activity of 6-(prop-2-ynyl)benzo[b]phenanthridine-5,7,12(6H)-trione (I) against methicillin-resistant Staphylococcus aureus (MRSA), including a panel of patient-derived strains, is comparable or better than vancomycin.  We show that the lead compd. generates reactive oxygen species (ROS) in the cell, contributing to its antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG_I34Hm-Jx7Vg90H21EOLACvtfcHk0lgrs_rPTnuV0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFOiu74%253D&md5=b4ef0f3179a9d4bb5059c3a42d86b763</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fml5001118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5001118%26sid%3Dliteratum%253Aachs%26aulast%3DKhodade%26aufirst%3DV.%2BS.%26aulast%3DSharath%2BChandra%26aufirst%3DM.%26aulast%3DBanerjee%26aufirst%3DA.%26aulast%3DLahiri%26aufirst%3DS.%26aulast%3DPulipeta%26aufirst%3DM.%26aulast%3DRangarajan%26aufirst%3DR.%26aulast%3DChakrapani%26aufirst%3DH.%26atitle%3DBioreductively%2520activated%2520reactive%2520oxygen%2520species%2520%2528ROS%2529%2520generators%2520as%2520MRSA%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D777%26epage%3D781%26doi%3D10.1021%2Fml5001118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kosower, N. S.</span>; <span class="NLM_string-name">Kosower, E. M.</span></span> <i>Thiol Labeling with Bromobimanes</i>; <span class="NLM_series">Methods in Enzymology</span>; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">1987</span>; Vol.  <span class="NLM_volume">143</span>, pp  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">84</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1016%2F0076-6879%2887%2943015-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1987&pages=76-84&author=N.+S.+Kosower&author=E.+M.+Kosower&title=Thiol+Labeling+with+Bromobimanes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2887%2943015-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252F0076-6879%252887%252943015-2%26sid%3Dliteratum%253Aachs%26aulast%3DKosower%26aufirst%3DN.%2BS.%26btitle%3DThiol%2520Labeling%2520with%2520Bromobimanes%26pub%3DAcademic%2520Press%26date%3D1987%26volume%3D143%26spage%3D76%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Speers, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling <i>in vivo</i> using a copper(I)-catalyzed azide-alkyne [3 + 2] cycloaddition</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">4686</span>– <span class="NLM_lpage">4687</span>, <span class="refDoi"> DOI: 10.1021/ja034490h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja034490h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlylu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=4686-4687&author=A.+E.+Speersauthor=G.+C.+Adamauthor=B.+F.+Cravatt&title=Activity-based+protein+profiling+in+vivo+using+a+copper%28I%29-catalyzed+azide-alkyne+%5B3+%2B+2%5D+cycloaddition&doi=10.1021%2Fja034490h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling in vivo using a copper(I)-catalyzed azide-alkyne [3 + 2] cycloaddition</span></div><div class="casAuthors">Speers, Anna E.; Adam, Gregory C.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4686-4687</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Toward the goal of assigning function to the tens of thousands of protein products encoded by eukaryotic and prokaryotic genomes, the field of proteomics requires new technologies that can functionally characterize proteins within the dynamic environment of the cell, where these biomols. are subject to myriad posttranslational modifications and the actions of endogenous activators and inhibitors. Here, we report an advanced strategy for activity-based protein profiling (ABPP) that addresses this important need.  We show that several enzymes can be labeled in an activity-based manner both in vitro and in vivo by an azido-sulfonate ester probe and that these labeling events can be detected in whole proteomes by copper-catalyzed ligation with a rhodamine-alkyne reagent.  This click chem.-based strategy for ABPP represents a unique and versatile method for functional proteome anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGXIFycwk3N7Vg90H21EOLACvtfcHk0lgrs_rPTnuV0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlylu7o%253D&md5=ead5de3b59297fe0387d34b0949661ab</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja034490h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja034490h%26sid%3Dliteratum%253Aachs%26aulast%3DSpeers%26aufirst%3DA.%2BE.%26aulast%3DAdam%26aufirst%3DG.%2BC.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520protein%2520profiling%2520in%2520vivo%2520using%2520a%2520copper%2528I%2529-catalyzed%2520azide-alkyne%2520%255B3%2520%252B%25202%255D%2520cycloaddition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D4686%26epage%3D4687%26doi%3D10.1021%2Fja034490h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khare, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Quantitative reactivity profiling predicts functional cysteines in proteomes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">790</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.1038/nature09472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnature09472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=21085121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=790-795&author=E.+Weerapanaauthor=C.+Wangauthor=G.+M.+Simonauthor=F.+Richterauthor=S.+Khareauthor=M.+B.+D.+Dillonauthor=D.+A.+Bachovchinauthor=K.+Mowenauthor=D.+Bakerauthor=B.+F.+Cravatt&title=Quantitative+reactivity+profiling+predicts+functional+cysteines+in+proteomes&doi=10.1038%2Fnature09472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative reactivity profiling predicts functional cysteines in proteomes</span></div><div class="casAuthors">Weerapana, Eranthie; Wang, Chu; Simon, Gabriel M.; Richter, Florian; Khare, Sagar; Dillon, Myles B. D.; Bachovchin, Daniel A.; Mowen, Kerri; Baker, David; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7325</span>),
    <span class="NLM_cas:pages">790-795</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cysteine is the most intrinsically nucleophilic amino acid in proteins, where its reactivity is tuned to perform diverse biochem. functions.  The absence of a consensus sequence that defines functional cysteines in proteins has hindered their discovery and characterization.  Here we describe a proteomics method to profile quant. the intrinsic reactivity of cysteine residues en masse directly in native biol. systems.  Hyper-reactivity was a rare feature among cysteines and it was found to specify a wide range of activities, including nucleophilic and reductive catalysis and sites of oxidative modification.  Hyper-reactive cysteines were identified in several proteins of uncharacterized function, including a residue conserved across eukaryotic phylogeny that we show is required for yeast viability and is involved in iron-sulfur protein biogenesis.  We also demonstrate that quant. reactivity profiling can form the basis for screening and functional assignment of cysteines in computationally designed proteins, where it discriminated catalytically active from inactive cysteine hydrolase designs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe4hHN7L4cGbVg90H21EOLACvtfcHk0liMJz8MS4jFhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr%252FI&md5=277e436e7a8f4dfff0b548bce52ac40d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnature09472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09472%26sid%3Dliteratum%253Aachs%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DRichter%26aufirst%3DF.%26aulast%3DKhare%26aufirst%3DS.%26aulast%3DDillon%26aufirst%3DM.%2BB.%2BD.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DMowen%26aufirst%3DK.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DQuantitative%2520reactivity%2520profiling%2520predicts%2520functional%2520cysteines%2520in%2520proteomes%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D790%26epage%3D795%26doi%3D10.1038%2Fnature09472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Páez, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teijaro, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolan, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide covalent ligand discovery in native biological systems</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>534</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nature18002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnature18002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=27309814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=570-574&author=K.+M.+Backusauthor=B.+E.+Correiaauthor=K.+M.+Lumauthor=S.+Forliauthor=B.+D.+Horningauthor=G.+E.+Gonz%C3%A1lez-P%C3%A1ezauthor=S.+Chatterjeeauthor=B.+R.+Lanningauthor=J.+R.+Teijaroauthor=A.+J.+Olsonauthor=D.+W.+Wolanauthor=B.+F.+Cravatt&title=Proteome-wide+covalent+ligand+discovery+in+native+biological+systems&doi=10.1038%2Fnature18002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide covalent ligand discovery in native biological systems</span></div><div class="casAuthors">Backus, Keriann M.; Correia, Bruno E.; Lum, Kenneth M.; Forli, Stefano; Horning, Benjamin D.; Gonzalez-Paez, Gonzalo E.; Chatterjee, Sandip; Lanning, Bryan R.; Teijaro, John R.; Olson, Arthur J.; Wolan, Dennis W.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7608</span>),
    <span class="NLM_cas:pages">570-574</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small mols. are powerful tools for investigating protein function and can serve as leads for new therapeutics.  Most human proteins, however, lack small-mol. ligands, and entire protein classes are considered 'undruggable'.  Fragment-based ligand discovery can identify small-mol. probes for proteins that have proven difficult to target using high-throughput screening of complex compd. libraries.  Although reversibly binding ligands are commonly pursued, covalent fragments provide an alternative route to small-mol. probes, including those that can access regions of proteins that are difficult to target through binding affinity alone.  Here we report a quant. anal. of cysteine-reactive small-mol. fragments screened against thousands of proteins in human proteomes and cells.  Covalent ligands were identified for >700 cysteines found in both druggable proteins and proteins deficient in chem. probes, including transcription factors, adaptor/scaffolding proteins, and uncharacterized proteins.  Among the atypical ligand-protein interactions discovered were compds. that react preferentially with pro- (inactive) caspases.  We used these ligands to distinguish extrinsic apoptosis pathways in human cell lines vs. primary human T cells, showing that the former is largely mediated by caspase-8 while the latter depends on both caspase-8 and -10.  Fragment-based covalent ligand discovery provides a greatly expanded portrait of the ligandable proteome and furnishes compds. that can illuminate protein functions in native biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgNJ88KALeSLVg90H21EOLACvtfcHk0liMJz8MS4jFhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN&md5=6fab087c042de735f8a5931e9f5b946c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature18002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18002%26sid%3Dliteratum%253Aachs%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DGonz%25C3%25A1lez-P%25C3%25A1ez%26aufirst%3DG.%2BE.%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DTeijaro%26aufirst%3DJ.%2BR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProteome-wide%2520covalent%2520ligand%2520discovery%2520in%2520native%2520biological%2520systems%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D570%26epage%3D574%26doi%3D10.1038%2Fnature18002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelkar, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravikumar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehendale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mhetre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajendran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrapani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, S. S.</span></span> <span> </span><span class="NLM_article-title">A chemical–genetic screen identifies ABHD12 as an oxidized-phosphatidylserine lipase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0195-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fs41589-018-0195-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=30643283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFSgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=169-178&author=D.+S.+Kelkarauthor=G.+Ravikumarauthor=N.+Mehendaleauthor=S.+Singhauthor=A.+Joshiauthor=A.+K.+Sharmaauthor=A.+Mhetreauthor=A.+Rajendranauthor=H.+Chakrapaniauthor=S.+S.+Kamat&title=A+chemical%E2%80%93genetic+screen+identifies+ABHD12+as+an+oxidized-phosphatidylserine+lipase&doi=10.1038%2Fs41589-018-0195-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical-genetic screen identifies ABHD12 as an oxidized-phosphatidylserine lipase</span></div><div class="casAuthors">Kelkar, Dhanashree S.; Ravikumar, Govindan; Mehendale, Neelay; Singh, Shubham; Joshi, Alaumy; Sharma, Ajay Kumar; Mhetre, Amol; Rajendran, Abinaya; Chakrapani, Harinath; Kamat, Siddhesh S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-178</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Reactive oxygen species (ROS) are transient, highly reactive intermediates or byproducts produced during oxygen metab.  However, when innate mechanisms are unable to cope with sequestration of surplus ROS, oxidative stress results, in which excess ROS damage biomols.  Oxidized phosphatidylserine (PS), a proapoptotic 'eat me' signal, is produced in response to elevated ROS, yet little is known regarding its chem. compn. and metab.  Here, we report a small mol. that generates ROS in different mammalian cells.  We used this mol. to detect, characterize and study oxidized PS in mammalian cells.  We developed a chem.-genetic screen to identify enzymes that regulate oxidized PS in mammalian cells and found that the lipase ABHD12 hydrolyzes oxidized PS.  We validated these findings in different physiol. settings including primary peritoneal macrophages and brains from Abhd12-/- mice under inflammatory stress, and in the process, we functionally annotated an enzyme regulating oxidized PS in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3D9DCwfyBk7Vg90H21EOLACvtfcHk0liMJz8MS4jFhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFSgtb4%253D&md5=51d88dc955559211b9250221eb88370d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0195-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0195-0%26sid%3Dliteratum%253Aachs%26aulast%3DKelkar%26aufirst%3DD.%2BS.%26aulast%3DRavikumar%26aufirst%3DG.%26aulast%3DMehendale%26aufirst%3DN.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DJoshi%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DA.%2BK.%26aulast%3DMhetre%26aufirst%3DA.%26aulast%3DRajendran%26aufirst%3DA.%26aulast%3DChakrapani%26aufirst%3DH.%26aulast%3DKamat%26aufirst%3DS.%2BS.%26atitle%3DA%2520chemical%25E2%2580%2593genetic%2520screen%2520identifies%2520ABHD12%2520as%2520an%2520oxidized-phosphatidylserine%2520lipase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D169%26epage%3D178%26doi%3D10.1038%2Fs41589-018-0195-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P. R.</span></span> <span> </span><span class="NLM_article-title">The multiple antibiotic resistance regulator MarR is a copper sensor in Escherichia coli</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">21</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio.1380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=24185215" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=21&author=Z.+Haoauthor=H.+Louauthor=R.+Zhuauthor=J.+Zhuauthor=D.+Zhangauthor=B.+S.+Zhaoauthor=S.+Zengauthor=X.+Chenauthor=J.+Chanauthor=C.+Heauthor=P.+R.+Chen&title=The+multiple+antibiotic+resistance+regulator+MarR+is+a+copper+sensor+in+Escherichia+coli&doi=10.1038%2Fnchembio.1380"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1380%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DZ.%26aulast%3DLou%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DR.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DB.%2BS.%26aulast%3DZeng%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DP.%2BR.%26atitle%3DThe%2520multiple%2520antibiotic%2520resistance%2520regulator%2520MarR%2520is%2520a%2520copper%2520sensor%2520in%2520Escherichia%2520coli%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D10%26spage%3D21%26doi%3D10.1038%2Fnchembio.1380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alekshun, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mealy, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seaton, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head, J. F.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of MarR, a regulator of multiple antibiotic resistance, at 2.3 Å resolution</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">710</span>, <span class="refDoi"> DOI: 10.1038/90429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2F90429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=11473263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1OmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=710&author=M.+N.+Alekshunauthor=S.+B.+Levyauthor=T.+R.+Mealyauthor=B.+A.+Seatonauthor=J.+F.+Head&title=The+crystal+structure+of+MarR%2C+a+regulator+of+multiple+antibiotic+resistance%2C+at+2.3+%C3%85+resolution&doi=10.1038%2F90429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of MarR, a regulator of multiple antibiotic resistance, at 2.3 Å resolution</span></div><div class="casAuthors">Alekshun, Michael N.; Levy, Stuart B.; Mealy, Tanya R.; Seaton, Barbara A.; Head, James F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">710-714</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">MarR is a regulator of multiple antibiotic resistance in Escherichia coli.  It is the prototypical member of the MarR family of regulatory proteins found in bacteria and archaea that play important roles in the development of antibiotic resistance, a global health problem.  Here we describe the crystal structure of the MarR protein, detd. at a resoln. of 2.3 Å.  This is the first reported crystal structure of a member of this newly-described protein family.  The structure shows MarR as a dimer with each subunit contg. a winged-helix DNA binding motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtm41LVqIrZbVg90H21EOLACvtfcHk0ljt_EP2r4k_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1OmsL4%253D&md5=766fc05d4ed4fec8721803904f9c2b7d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2F90429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F90429%26sid%3Dliteratum%253Aachs%26aulast%3DAlekshun%26aufirst%3DM.%2BN.%26aulast%3DLevy%26aufirst%3DS.%2BB.%26aulast%3DMealy%26aufirst%3DT.%2BR.%26aulast%3DSeaton%26aufirst%3DB.%2BA.%26aulast%3DHead%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520crystal%2520structure%2520of%2520MarR%252C%2520a%2520regulator%2520of%2520multiple%2520antibiotic%2520resistance%252C%2520at%25202.3%2520%25C3%2585%2520resolution%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2001%26volume%3D8%26spage%3D710%26doi%3D10.1038%2F90429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Birukou, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schindler, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaatz, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, R. G.</span></span> <span> </span><span class="NLM_article-title">Structural mechanism of transcription regulation of the Staphylococcus aureus multidrug efflux operon mepRA by the MarR family repressor MepR</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2774</span>– <span class="NLM_lpage">2788</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1093%2Fnar%2Fgkt1215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=24293644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlOqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=2774-2788&author=I.+Birukouauthor=S.+M.+Seoauthor=B.+D.+Schindlerauthor=G.+W.+Kaatzauthor=R.+G.+Brennan&title=Structural+mechanism+of+transcription+regulation+of+the+Staphylococcus+aureus+multidrug+efflux+operon+mepRA+by+the+MarR+family+repressor+MepR&doi=10.1093%2Fnar%2Fgkt1215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism of transcription regulation of the Staphylococcus aureus multidrug efflux operon mepRA by the MarR family repressor MepR</span></div><div class="casAuthors">Birukou, Ivan; Seo, Susan M.; Schindler, Bryan D.; Kaatz, Glenn W.; Brennan, Richard G.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2774-2788</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The multidrug efflux pump MepA is a major contributor to multidrug resistance in Staphylococcus aureus.  MepR, a member of the multiple antibiotic resistance regulator (MarR) family, represses mepA and its own gene.  Here, we report the structure of a MepR-mepR operator complex.  Structural comparison of DNA-bound MepR with 'induced' apoMepR reveals the large conformational changes needed to allow the DNA-binding winged helix-turn-helix motifs to interact with the consecutive major and minor grooves of the GTTAG signature sequence.  Intriguingly, MepR makes no hydrogen bonds to major groove nucleobases.  Rather, recognition-helix residues Thr60, Gly61, Pro62 and Thr63 make sequence-specifying van der Waals contacts with the TTAG bases.  Removing these contacts dramatically affects MepR-DNA binding activity.  The wings insert into the flanking minor grooves, whereby residue Arg87, buttressed by Asp85, interacts with the O2 of T4 and O4' ribosyl oxygens of A23 and T4.  Mutating Asp85 and Arg87, both conserved throughout the MarR family, markedly affects MepR repressor activity.  The His14':Arg59 and Arg10':His35:Phe108 interaction networks stabilize the DNA-binding conformation of MepR thereby contributing significantly to its high affinity binding.  A structure-guided model of the MepR-mepA operator complex suggests that MepR dimers do not interact directly and cooperative binding is likely achieved by DNA-mediated allosteric effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL78XMJCN9JbVg90H21EOLACvtfcHk0ljt_EP2r4k_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlOqs78%253D&md5=652d3926ddfa003b5744a01a07b61be1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1215%26sid%3Dliteratum%253Aachs%26aulast%3DBirukou%26aufirst%3DI.%26aulast%3DSeo%26aufirst%3DS.%2BM.%26aulast%3DSchindler%26aufirst%3DB.%2BD.%26aulast%3DKaatz%26aufirst%3DG.%2BW.%26aulast%3DBrennan%26aufirst%3DR.%2BG.%26atitle%3DStructural%2520mechanism%2520of%2520transcription%2520regulation%2520of%2520the%2520Staphylococcus%2520aureus%2520multidrug%2520efflux%2520operon%2520mepRA%2520by%2520the%2520MarR%2520family%2520repressor%2520MepR%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D2774%26epage%3D2788%26doi%3D10.1093%2Fnar%2Fgkt1215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manna, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the SarS protein from Staphylococcus aureus</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">4219</span>– <span class="NLM_lpage">4225</span>, <span class="refDoi"> DOI: 10.1128/jb.185.14.4219-4225.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1128%2Fjb.185.14.4219-4225.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=12837797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlsFahtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=2003&pages=4219-4225&author=R.+Liauthor=A.+C.+Mannaauthor=S.+Daiauthor=A.+L.+Cheungauthor=G.+Zhang&title=Crystal+structure+of+the+SarS+protein+from+Staphylococcus+aureus&doi=10.1128%2Fjb.185.14.4219-4225.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the SarS protein from Staphylococcus aureus</span></div><div class="casAuthors">Li, Ronggui; Manna, Adhar C.; Dai, Shaodong; Cheung, Ambrose L.; Zhang, Gongyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4219-4225</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The expression of virulence determinants in Staphylococcus aureus is controlled by global regulatory loci (e.g., sarA and agr).  One of these determinants, protein A (spa), is activated by sarS, which encodes a 250-residue DNA-binding protein.  Genetic anal. indicated that the agr locus likely mediates spa repression by suppressing the transcription of sarS.  Contrary to SarA and SarR, which require homodimer formation for proper function, SarS is unusual within the SarA protein family in that it contains two homologous halves, with each half sharing sequence similarity to SarA and SarR.  Here we report the 2.2 Å resoln. x-ray crystal structure of the SarS protein.  SarS has folds similar to those of SarR and, quite plausibly, the native SarA structure.  Two typical winged-helix DNA-binding domains are connected by a well-ordered loop.  The interactions between the two domains are extensive and conserved.  The putative DNA-binding surface is highly pos. charged.  In contrast, neg. charged patches are located opposite to the DNA-binding surface.  Furthermore, sequence alignment and structural comparison revealed that MarR has folds similar to those of SarR and SarS.  Members of the MarR protein family have previously been implicated in the neg. regulation of an efflux pump involved in multiple antibiotic resistance in many gram-neg. species.  We propose that MarR also belongs to the winged-helix protein family and has a similar mode of DNA binding as SarR and SarS and possibly the entire SarA protein family member.  Based on the structural differences of SarR, SarS, and MarR, we further classified these winged-helix proteins to three subfamilies, SarA, SarS, and MarR.  Finally, a possible transcription regulation mechanism is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeHARYRHKRJrVg90H21EOLACvtfcHk0ljt_EP2r4k_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlsFahtr8%253D&md5=1a6331c46d063984ebdc37c67fadbe43</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2Fjb.185.14.4219-4225.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.185.14.4219-4225.2003%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DR.%26aulast%3DManna%26aufirst%3DA.%2BC.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DA.%2BL.%26aulast%3DZhang%26aufirst%3DG.%26atitle%3DCrystal%2520structure%2520of%2520the%2520SarS%2520protein%2520from%2520Staphylococcus%2520aureus%26jtitle%3DJ.%2520Bacteriol.%26date%3D2003%26volume%3D185%26spage%3D4219%26epage%3D4225%26doi%3D10.1128%2Fjb.185.14.4219-4225.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duguid, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneewind, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span> <span> </span><span class="NLM_article-title">An oxidation-sensing mechanism is used by the global regulator MgrA in Staphylococcus aureus</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1038/nchembio820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=16980961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtV2gsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=591-595&author=P.+R.+Chenauthor=T.+Baeauthor=W.+A.+Williamsauthor=E.+M.+Duguidauthor=P.+A.+Riceauthor=O.+Schneewindauthor=C.+He&title=An+oxidation-sensing+mechanism+is+used+by+the+global+regulator+MgrA+in+Staphylococcus+aureus&doi=10.1038%2Fnchembio820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">An oxidation-sensing mechanism is used by the global regulator MgrA in Staphylococcus aureus</span></div><div class="casAuthors">Chen, Peng R.; Bae, Taeok; Williams, Wade A.; Duguid, Erica M.; Rice, Phoebe A.; Schneewind, Olaf; He, Chuan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">591-595</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Staphylococcus aureus is a human pathogen responsible for most wound and hospital-acquired infections.  The protein MgrA is both an important virulence determinant during infection and a regulator of antibiotic resistance in S. aureus.  The crystal structure of the MgrA homodimer, solved at 2.86 Å, indicates the presence of a unique cysteine residue located at the interface of the protein dimer.  We discovered that this cysteine residue can be oxidized by various reactive oxygen species, such as hydrogen peroxide and org. hydroperoxide.  Cysteine oxidn. leads to dissocn. of MgrA from DNA and initiation of signaling pathways that turn on antibiotic resistance in S. aureus.  The oxidn.-sensing mechanism is typically used by bacteria to counter challenges of reactive oxygen and nitrogen species.  Our study reveals that in S. aureus, MgrA adopts a similar mechanism but uses it to globally regulate different defensive pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohu2z8sVhB2rVg90H21EOLACvtfcHk0lhO4pCNQ3U8bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtV2gsrzF&md5=8ab45df1e170c557f8506adeada6fa1a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio820%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%2BR.%26aulast%3DBae%26aufirst%3DT.%26aulast%3DWilliams%26aufirst%3DW.%2BA.%26aulast%3DDuguid%26aufirst%3DE.%2BM.%26aulast%3DRice%26aufirst%3DP.%2BA.%26aulast%3DSchneewind%26aufirst%3DO.%26aulast%3DHe%26aufirst%3DC.%26atitle%3DAn%2520oxidation-sensing%2520mechanism%2520is%2520used%2520by%2520the%2520global%2520regulator%2520MgrA%2520in%2520Staphylococcus%2520aureus%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D591%26epage%3D595%26doi%3D10.1038%2Fnchembio820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kahan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahan, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kropp, H.</span></span> <span> </span><span class="NLM_article-title">The mechanism of action of Fosfomycin (Phosphonomycin)</span>. <i>Ann. N.Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>235</i></span>,  <span class="NLM_fpage">364</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1111/j.1749-6632.1974.tb43277.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1111%2Fj.1749-6632.1974.tb43277.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=4605290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADyaE2cXkvVKnsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1974&pages=364-386&author=F.+M.+Kahanauthor=J.+S.+Kahanauthor=P.+J.+Cassidyauthor=H.+Kropp&title=The+mechanism+of+action+of+Fosfomycin+%28Phosphonomycin%29&doi=10.1111%2Fj.1749-6632.1974.tb43277.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of action of fosfomycin (phosphonomycin)</span></div><div class="casAuthors">Kahan, Frederick M.; Kahan, Jean S.; Cassidy, Patrick J.; Kropp, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">364-86</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    </div><div class="casAbstract">Fosfomycin (I) [23155-02-4] killed bacteria by blocking cell wall synthesis, specifically by inhibiting phosphoenolpyruvate: UDP-N-acetylglucosamine-3-enolpyruvyl transferase [9023-27-2].  I, which reacted covalently with a cysteinyl residue of the enzyme, mimicked the natural substrate, phosphoenolpyruvate.  I utilized the L-α-glycerophosphate transport system as the sole method of entry in almost all sensitive bacteria, and loss of this transport system was the primary mechanism of resistant mutants.  A second pathway of antibiotic accumulation involved the inducible hexose phosphate uptake system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqelIVHQfVWMbVg90H21EOLACvtfcHk0lhO4pCNQ3U8bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXkvVKnsLY%253D&md5=b27030810bde1e63ada6ee00bab30bcf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fj.1749-6632.1974.tb43277.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1749-6632.1974.tb43277.x%26sid%3Dliteratum%253Aachs%26aulast%3DKahan%26aufirst%3DF.%2BM.%26aulast%3DKahan%26aufirst%3DJ.%2BS.%26aulast%3DCassidy%26aufirst%3DP.%2BJ.%26aulast%3DKropp%26aufirst%3DH.%26atitle%3DThe%2520mechanism%2520of%2520action%2520of%2520Fosfomycin%2520%2528Phosphonomycin%2529%26jtitle%3DAnn.%2520N.Y.%2520Acad.%2520Sci.%26date%3D1974%26volume%3D235%26spage%3D364%26epage%3D386%26doi%3D10.1111%2Fj.1749-6632.1974.tb43277.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aye, Y.</span></span> <span> </span><span class="NLM_article-title">Privileged electrophile sensors: A resource for covalent drug development</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1016%2Fj.chembiol.2017.05.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=28648380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=787-800&author=M.+J.+C.+Longauthor=Y.+Aye&title=Privileged+electrophile+sensors%3A+A+resource+for+covalent+drug+development&doi=10.1016%2Fj.chembiol.2017.05.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged Electrophile Sensors: A Resource for Covalent Drug Development</span></div><div class="casAuthors">Long, Marcus John Curtis; Aye, Yimon</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">787-800</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">This Perspective delineates how redox signaling affects the activity of specific enzyme isoforms and how this property may be harnessed for rational drug design.  Covalent drugs have resurged in recent years and several reports have extolled the general virtues of developing irreversible inhibitors.  Indeed, many modern pharmaceuticals contain electrophilic appendages.  Several invoke a warhead that hijacks active-site nucleophiles whereas others take advantage of spectator nucleophilic side chains that do not participate in enzymic chem., but are poised to bind/react with electrophiles.  The latest data suggest that innate electrophile sensing-which enables rapid reaction with an endogenous signaling electrophile-is a quintessential resource for the development of covalent drugs.  For instance, based on recent work documenting isoform-specific electrophile sensing, isoenzyme non-specific drugs may be converted to isoenzyme-specific analogs by hijacking privileged first-responder electrophile-sensing cysteines.  Because this approach targets functionally relevant cysteines, we can simultaneously harness previously untapped moonlighting roles of enzymes linked to redox sensing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWQ9w0b7pay7Vg90H21EOLACvtfcHk0lhO4pCNQ3U8bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtr3E&md5=b7edc384ac48df7465303f33f7b1cca8</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.023%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BJ.%2BC.%26aulast%3DAye%26aufirst%3DY.%26atitle%3DPrivileged%2520electrophile%2520sensors%253A%2520A%2520resource%2520for%2520covalent%2520drug%2520development%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D787%26epage%3D800%26doi%3D10.1016%2Fj.chembiol.2017.05.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gioia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauville, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricci, A.</span></span> <span> </span><span class="NLM_article-title">Organocatalytic asymmetric Diels–Alder reactions of 3-vinylindoles</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">9236</span>– <span class="NLM_lpage">9239</span>, <span class="refDoi"> DOI: 10.1002/anie.200804275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fanie.200804275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVygs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=9236-9239&author=C.+Gioiaauthor=A.+Hauvilleauthor=L.+Bernardiauthor=F.+Finiauthor=A.+Ricci&title=Organocatalytic+asymmetric+Diels%E2%80%93Alder+reactions+of+3-vinylindoles&doi=10.1002%2Fanie.200804275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Organocatalytic asymmetric diels-alder reactions of 3-vinylindoles</span></div><div class="casAuthors">Gioia, Claudio; Hauville, Agnes; Bernardi, Luca; Fini, Francesco; Ricci, Alfredo</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">9236-9239</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A bifunctional org. catalyst based on the thiourea motif is able to coordinate both diene and dienophile in an unprecedented asym. Diels-Alder reaction of 3-vinylindole derivs., giving a rapid access to optically active tetra- and hexahydrocarbazoles, e.g. I, with excellent results in terms of yields, diastereoselectivities, and enantioselectivities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouMljMVhEoJLVg90H21EOLACvtfcHk0lhO4pCNQ3U8bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVygs7%252FI&md5=004af59760dc1b44a1c29a445d8bd838</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fanie.200804275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200804275%26sid%3Dliteratum%253Aachs%26aulast%3DGioia%26aufirst%3DC.%26aulast%3DHauville%26aufirst%3DA.%26aulast%3DBernardi%26aufirst%3DL.%26aulast%3DFini%26aufirst%3DF.%26aulast%3DRicci%26aufirst%3DA.%26atitle%3DOrganocatalytic%2520asymmetric%2520Diels%25E2%2580%2593Alder%2520reactions%2520of%25203-vinylindoles%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D9236%26epage%3D9239%26doi%3D10.1002%2Fanie.200804275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Kundan Kumar, Amol Mhetre, Girish S. Ratnaparkhi, <span class="NLM_string-name hlFld-ContribAuthor">Siddhesh S. Kamat</span>. </span><span class="cited-content_cbyCitation_article-title">A Superfamily-wide Activity Atlas of Serine Hydrolases in Drosophila melanogaster. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2021,</strong> <em>60 </em>
                                    (16)
                                     , 1312-1324. <a href="https://doi.org/10.1021/acs.biochem.1c00171" title="DOI URL">https://doi.org/10.1021/acs.biochem.1c00171</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.1c00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.1c00171%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DA%252BSuperfamily-wide%252BActivity%252BAtlas%252Bof%252BSerine%252BHydrolases%252Bin%252BDrosophila%252Bmelanogaster%26aulast%3DKumar%26aufirst%3DKundan%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06032021%26date%3D01042021%26date%3D07042021%26volume%3D60%26issue%3D16%26spage%3D1312%26epage%3D1324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li  Zhang</span>, <span class="hlFld-ContribAuthor ">Guiying  Zhang</span>, <span class="hlFld-ContribAuthor ">Shujing  Xu</span>, <span class="hlFld-ContribAuthor ">Yuning  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances of quinones as a privileged structure in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>223 </em>, 113632. <a href="https://doi.org/10.1016/j.ejmech.2021.113632" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113632</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113632%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bof%252Bquinones%252Bas%252Ba%252Bprivileged%252Bstructure%252Bin%252Bdrug%252Bdiscovery%26aulast%3DZhang%26aufirst%3DLi%26date%3D2021%26volume%3D223%26spage%3D113632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanshu  Jia</span>, <span class="hlFld-ContribAuthor ">Xiaoyue  Wen</span>, <span class="hlFld-ContribAuthor ">Yufeng  Gong</span>, <span class="hlFld-ContribAuthor ">Xuefeng  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Current scenario of indole derivatives with potential anti-drug-resistant cancer activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112359. <a href="https://doi.org/10.1016/j.ejmech.2020.112359" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112359%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCurrent%252Bscenario%252Bof%252Bindole%252Bderivatives%252Bwith%252Bpotential%252Banti-drug-resistant%252Bcancer%252Bactivity%26aulast%3DJia%26aufirst%3DYanshu%26date%3D2020%26volume%3D200%26spage%3D112359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongjin  Shi</span>, <span class="hlFld-ContribAuthor ">Lubin  Xu</span>, <span class="hlFld-ContribAuthor ">Didi  Ren</span>, <span class="hlFld-ContribAuthor ">Liang  Wang</span>, <span class="hlFld-ContribAuthor ">Weisi  Guo</span>, <span class="hlFld-ContribAuthor ">Shuai-Shuai  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorinated alcohol mediated
              N
              ,
              N
              ′-dialkylation of amino acid derivatives
              via
              cascade [1,5]-hydride transfer/cyclization for concise synthesis of tetrahydroquinazoline. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2020,</strong> <em>18 </em>
                                    (5)
                                     , 895-904. <a href="https://doi.org/10.1039/C9OB02498K" title="DOI URL">https://doi.org/10.1039/C9OB02498K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9OB02498K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9OB02498K%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DFluorinated%252Balcohol%252Bmediated%252BN%252B%25252C%252BN%252B%2525E2%252580%2525B2-dialkylation%252Bof%252Bamino%252Bacid%252Bderivatives%252Bvia%252Bcascade%252B%25255B1%25252C5%25255D-hydride%252Btransfer%25252Fcyclization%252Bfor%252Bconcise%252Bsynthesis%252Bof%252Btetrahydroquinazoline%26aulast%3DShi%26aufirst%3DHongjin%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D5%26spage%3D895%26epage%3D904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Proposed indole-quinone epoxide (inset: an epoxide the electrophilic reactive site in the probe, and the arrows indicate the approach of the nucleophile).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Procedure for the Synthesis of Indole-based Quinone Epoxides: (a) R<sup>1</sup>CH<sub>2</sub>PPh<sub>3</sub>Br (1.15 equiv), <i>n</i>-BuLi (1 equiv), and THF, −78 °C; (b) Benzoquinone (2.5 equiv), and Ethanol, RT</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Time-kill analysis for <b>4a</b> (10× MIC, MIC = 0.5 μg/mL) and daptomycin (Dap, 10× MIC, MIC = 0.5 μg/mL) against VRSA strain HIP 11714 showing potent bactericidal activity of <b>4a</b>. For the entire plot (with vancomycin), see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S4, (<i>n</i> = 2); (B) Time courses of independent reactions of compounds <b>4a</b>, <b>9a</b>, <b>10a</b>, and <b>11a</b> with 10 equiv <span class="smallcaps smallerCapital">l</span>-cysteine. Curve fitting to an exponential decay afforded pseudo-first-order rate constants, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf" class="ext-link">Supporting Information</a>, Table S2, (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) ORTEP diagram for <b>4a</b> with van der Waals radii overlap (dotted circles), for groups at R and R<sup>1</sup> with carbonyl oxygens; (B) ORTEP diagram for <b>4a</b> showing the boat-like conformation of the quinone epoxide, and schematic showing flagpole positions of quinone epoxide during the thiol attack.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Structures of IND-QE alkyne probes <b>P1</b> and <b>P2</b> synthesized in this study; (B) dose-dependent proteome profiling with <b>P1</b> (0.01–100 μM) and <b>P2</b> (0.01–100 μM) in soluble proteomic fraction of <i>S. aureus</i> ATCC 29213; (C) proteome profiling with 10 μM of <b>P1</b> in lysates of <i>S. aureus</i> ATCC 29213 compared with IAM pretreated fractions; and (D) proteome profiling with 10 μM of <b>P1</b> and <b>P2</b> in situ for <i>S. aureus</i> ATCC 29213.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Targets identified for <b>4a</b> using LC–MS/MS-based proteomics experiment; proteins that exhibited ratios of <0.1 for <b>4a</b> and IAM (red color) and ∼1 for <b>11a</b> (blue color) were considered targets of <b>4a</b>. (B) Pie-chart categorizing the identified targets of <b>4a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/medium/jm-2019-007749_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Dose-dependent labeling of targets MarR_12840 and MarR_05815 with varying concentrations of probe <b>P1</b>, as compared with their alanine point variants; crystal structures of identified targets with Cys residues at Cys61 and Cys12, respectively (in black spheres).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00774/20190719/images/large/jm-2019-007749_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00774&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02042" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02042" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLeo, F. R.</span></span> <span> </span><span class="NLM_article-title">Waves of resistance: <i>Staphylococcus aureus</i> in the antibiotic era</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1038/nrmicro2200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnrmicro2200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=19680247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFCrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=629-641&author=H.+F.+Chambersauthor=F.+R.+DeLeo&title=Waves+of+resistance%3A+Staphylococcus+aureus+in+the+antibiotic+era&doi=10.1038%2Fnrmicro2200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Waves of resistance: Staphylococcus aureus in the antibiotic era</span></div><div class="casAuthors">Chambers, Henry F.; DeLeo, Frank R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">629-641</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Staphylococcus aureus is notorious for its ability to become resistant to antibiotics.  Infections that are caused by antibiotic-resistant strains often occur in epidemic waves that are initiated by one or a few successful clones.  Methicillin-resistant S. aureus (MRSA) features prominently in these epidemics.  Historically assocd. with hospitals and other health care settings, MRSA has now emerged as a widespread cause of community infections.  Community or community-assocd. MRSA (CA-MRSA) can spread rapidly among healthy individuals.  Outbreaks of CA-MRSA infections have been reported worldwide, and CA-MRSA strains are now epidemic in the United States.  Here, we review the mol. epidemiol. of the epidemic waves of penicillin- and methicillin-resistant strains of S. aureus that have occurred since 1940, with a focus on the clin. and mol. epidemiol. of CA-MRSA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXSOiu96U2nLVg90H21EOLACvtfcHk0liGJrXObxntMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFCrsLo%253D&md5=17ae44f98bfbdbeb6f9ef9e728ea6a6b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2200%26sid%3Dliteratum%253Aachs%26aulast%3DChambers%26aufirst%3DH.%2BF.%26aulast%3DDeLeo%26aufirst%3DF.%2BR.%26atitle%3DWaves%2520of%2520resistance%253A%2520Staphylococcus%2520aureus%2520in%2520the%2520antibiotic%2520era%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2009%26volume%3D7%26spage%3D629%26epage%3D641%26doi%3D10.1038%2Fnrmicro2200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musser, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLeo, F. R.</span></span> <span> </span><span class="NLM_article-title">Genomic analysis of the emergence of vancomycin-resistant Staphylococcus aureus</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e00170</span> <span class="refDoi"> DOI: 10.1128/mbio.00170-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1128%2Fmbio.00170-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=22736541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtlaht7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&author=S.+D.+Kobayashiauthor=J.+M.+Musserauthor=F.+R.+DeLeo&title=Genomic+analysis+of+the+emergence+of+vancomycin-resistant+Staphylococcus+aureus&doi=10.1128%2Fmbio.00170-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic analysis of the emergence of vancomycin-resistant Staphylococcus aureus</span></div><div class="casAuthors">Kobayashi, Scott D.; Musser, James M.; DeLeo, Frank R.</div><div class="citationInfo"><span class="NLM_cas:title">mBio</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e00170-12</span>CODEN:
                <span class="NLM_cas:coden">MBIOCL</span>;
        ISSN:<span class="NLM_cas:issn">2150-7511</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Staphylococcus aureus is a human commensal bacterium and a prominent cause of infections globally.  The high incidence of S. aureus infections is compounded by the ability of the microbe to readily acquire resistance to antibiotics.  In the United States, methicillin-resistant S. aureus (MRSA) is a leading cause of morbidity and mortality by a single infectious agent.  Therapeutic options for severe MRSA infections are limited to a few antibiotics to which the organism is typically susceptible, including vancomycin.  Acquisition of high-level vancomycin resistance by MRSA is a major concern, but to date, there have been only 12 vancomycin-resistant S. aureus (VRSA) isolates reported in the United States and all belong to a phylogenetic lineage known as clonal complex 5.  To gain enhanced understanding of the genetic characteristics conducive to the acquisition of vancomycin resistance by S. aureus, V. N. Kos et al. performed whole-genome sequencing of all 12 VRSA isolates and compared the DNA sequences to the genomes of other S. aureus strains.  The findings provide new information about the evolutionary history of VRSA and identify genetic features that may bear on the relationship between S. aureus clonal complex 5 strains and the acquisition of vancomycin resistance genes from enterococci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE9fgRd77SMrVg90H21EOLACvtfcHk0liGJrXObxntMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtlaht7jO&md5=c1b8d43540068433fd682e79234cbc23</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2Fmbio.00170-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmbio.00170-12%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DS.%2BD.%26aulast%3DMusser%26aufirst%3DJ.%2BM.%26aulast%3DDeLeo%26aufirst%3DF.%2BR.%26atitle%3DGenomic%2520analysis%2520of%2520the%2520emergence%2520of%2520vancomycin-resistant%2520Staphylococcus%2520aureus%26jtitle%3DmBio%26date%3D2012%26volume%3D3%26doi%3D10.1128%2Fmbio.00170-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C.</span>; <span class="NLM_string-name">Wencewicz, T.</span></span> <i>Antibiotics: Challenges, Mechanisms, Opportunities</i>; <span class="NLM_publisher-name">American Society of Microbiology</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1128%2F9781555819316" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=C.+Walsh&author=T.+Wencewicz&title=Antibiotics%3A+Challenges%2C+Mechanisms%2C+Opportunities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1128%2F9781555819316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1128%252F9781555819316%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DC.%26btitle%3DAntibiotics%253A%2520Challenges%252C%2520Mechanisms%252C%2520Opportunities%26pub%3DAmerican%2520Society%2520of%2520Microbiology%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based profiling of enzyme families</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">3279</span>– <span class="NLM_lpage">3301</span>, <span class="refDoi"> DOI: 10.1021/cr050288g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr050288g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFejtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=3279-3301&author=M.+J.+Evansauthor=B.+F.+Cravatt&title=Mechanism-based+profiling+of+enzyme+families&doi=10.1021%2Fcr050288g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based profiling of enzyme families</span></div><div class="casAuthors">Evans, Michael J.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3279-3301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Activity-based protein profiling (ABPP) represents a chem. strategy that utilizes site-directed covalent probes to profile the functional state of enzymes in complex proteomes.  This review describes the design of prototype activity-based probes (ABPs) for functional proteomics, biol. applications of ABPs, methodol. advances, and applications of ABPs beyond comparative proteomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwF1DAf24AkrVg90H21EOLACvtfcHk0lgLijpSmbXWqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFejtrs%253D&md5=3986124d96400c9392ba562fc7fa3faf</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fcr050288g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr050288g%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DM.%2BJ.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DMechanism-based%2520profiling%2520of%2520enzyme%2520families%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D3279%26epage%3D3301%26doi%3D10.1021%2Fcr050288g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: the serine hydrolases</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">14694</span>– <span class="NLM_lpage">14699</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.26.14694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1073%2Fpnas.96.26.14694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10611275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=14694-14699&author=Y.+Liuauthor=M.+P.+Patricelliauthor=B.+F.+Cravatt&title=Activity-based+protein+profiling%3A+the+serine+hydrolases&doi=10.1073%2Fpnas.96.26.14694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: the serine hydrolases</span></div><div class="casAuthors">Liu, Yongsheng; Patricelli, Matthew P.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">14694-14699</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">With the postgenome era rapidly approaching, new strategies for the functional anal. of proteins are needed.  To date, proteomics efforts have primarily been confined to recording variations in protein level rather than activity.  The ability to profile classes of proteins on the basis of changes in their activity would greatly accelerate both the assignment of protein function and the identification of potential pharmaceutical targets.  Here, we describe the chem. synthesis and utility of an active-site directed probe for visualizing dynamics in the expression and function of an entire enzyme family, the serine hydrolases.  By reacting this probe, a biotinylated fluorophosphonate referred to as FP-biotin, with crude tissue exts., we quickly and with high sensitivity detect numerous serine hydrolases, many of which display tissue-restricted patterns of expression.  Addnl., we show that FP-biotin labels these proteins in an activity-dependent manner that can be followed kinetically, offering a powerful means to monitor dynamics simultaneously in both protein function and expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8i5m0pTTunrVg90H21EOLACvtfcHk0lgLijpSmbXWqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D&md5=d5d1a09328fc7715e9d30ca392733426</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.26.14694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.26.14694%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520the%2520serine%2520hydrolases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D14694%26epage%3D14699%26doi%3D10.1073%2Fpnas.96.26.14694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatzios, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasabe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qadri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldor, M. K.</span></span> <span> </span><span class="NLM_article-title">Chemoproteomic profiling of host and pathogen enzymes active in cholera</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">268</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio.2025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=26900865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC28XivFKgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=268&author=S.+K.+Hatziosauthor=S.+Abelauthor=J.+Martellauthor=T.+Hubbardauthor=J.+Sasabeauthor=D.+Muneraauthor=L.+Clarkauthor=D.+A.+Bachovchinauthor=F.+Qadriauthor=E.+T.+Ryanauthor=B.+M.+Davisauthor=E.+Weerapanaauthor=M.+K.+Waldor&title=Chemoproteomic+profiling+of+host+and+pathogen+enzymes+active+in+cholera&doi=10.1038%2Fnchembio.2025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoproteomic profiling of host and pathogen enzymes active in cholera</span></div><div class="casAuthors">Hatzios, Stavroula K.; Abel, Soren; Martell, Julianne; Hubbard, Troy; Sasabe, Jumpei; Munera, Diana; Clark, Lars; Bachovchin, Daniel A.; Qadri, Firdausi; Ryan, Edward T.; Davis, Brigid M.; Weerapana, Eranthie; Waldor, Matthew K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">268-274</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activity-based protein profiling (ABPP) is a chemoproteomic tool for detecting active enzymes in complex biol. systems.  We used ABPP to identify secreted bacterial and host serine hydrolases that are active in animals infected with the cholera pathogen Vibrio cholerae.  Four V. cholerae proteases were consistently active in infected rabbits, and one, VC0157 (renamed IvaP), was also active in human choleric stool.  Inactivation of IvaP influenced the activity of other secreted V. cholerae and rabbit enzymes in vivo, and genetic disruption of all four proteases increased the abundance of intelectin, an intestinal lectin, and its binding to V. cholerae in infected rabbits.  Intelectin also bound to other enteric bacterial pathogens, suggesting that it may constitute a previously unrecognized mechanism of bacterial surveillance in the intestine that is inhibited by pathogen-secreted proteases.  Our work demonstrates the power of activity-based proteomics to reveal host-pathogen enzymic dialog in an animal model of infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolhuO5dBmCbVg90H21EOLACvtfcHk0lgLijpSmbXWqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivFKgtro%253D&md5=c07b3bb6213fe023f5916a10dc0be7db</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2025%26sid%3Dliteratum%253Aachs%26aulast%3DHatzios%26aufirst%3DS.%2BK.%26aulast%3DAbel%26aufirst%3DS.%26aulast%3DMartell%26aufirst%3DJ.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DSasabe%26aufirst%3DJ.%26aulast%3DMunera%26aufirst%3DD.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DQadri%26aufirst%3DF.%26aulast%3DRyan%26aufirst%3DE.%2BT.%26aulast%3DDavis%26aufirst%3DB.%2BM.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DWaldor%26aufirst%3DM.%2BK.%26atitle%3DChemoproteomic%2520profiling%2520of%2520host%2520and%2520pathogen%2520enzymes%2520active%2520in%2520cholera%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D268%26doi%3D10.1038%2Fnchembio.2025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hacker, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazear, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Global profiling of lysine reactivity and ligandability in the human proteome</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1038/nchem.2826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchem.2826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=29168484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1181-1190&author=S.+M.+Hackerauthor=K.+M.+Backusauthor=M.+R.+Lazearauthor=S.+Forliauthor=B.+E.+Correiaauthor=B.+F.+Cravatt&title=Global+profiling+of+lysine+reactivity+and+ligandability+in+the+human+proteome&doi=10.1038%2Fnchem.2826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Global profiling of lysine reactivity and ligandability in the human proteome</span></div><div class="casAuthors">Hacker, Stephan M.; Backus, Keriann M.; Lazear, Michael R.; Forli, Stefano; Correia, Bruno E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nucleophilic amino acids make important contributions to protein function, including performing key roles in catalysis and serving as sites for post-translational modification.  Electrophilic groups that target amino-acid nucleophiles have been used to create covalent ligands and drugs, but have, so far, been mainly limited to cysteine and serine.  Here, the authors report a chem. proteomic platform for the global and quant. anal. of lysine residues in native biol. systems.  The authors have quantified, in total, >9000 lysines in human cell proteomes and have identified several hundred residues with heightened reactivity that are enriched at protein functional sites and can frequently be targeted by electrophilic small mols.  The authors also discovered lysine-reactive fragment electrophiles that inhibit enzymes by active site and allosteric mechanisms, as well as disrupt protein-protein interactions in transcriptional regulatory complexes, emphasizing the broad potential and diverse functional consequences of liganding lysine residues throughout the human proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRKvUVcT5YRrVg90H21EOLACvtfcHk0liq64DG2B1NUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GrsrrN&md5=1dac8f079c1caffc8ae584291480fc61</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2826%26sid%3Dliteratum%253Aachs%26aulast%3DHacker%26aufirst%3DS.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DLazear%26aufirst%3DM.%2BR.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DGlobal%2520profiling%2520of%2520lysine%2520reactivity%2520and%2520ligandability%2520in%2520the%2520human%2520proteome%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1181%26epage%3D1190%26doi%3D10.1038%2Fnchem.2826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span> <span> </span><span class="NLM_article-title">Cysteine-reactive probes and their use in chemical proteomics</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4501</span>– <span class="NLM_lpage">4512</span>, <span class="refDoi"> DOI: 10.1039/c8cc01485j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc8cc01485j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=29645055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsVegs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2018&pages=4501-4512&author=D.+G.+Hochauthor=D.+Abeggauthor=A.+Adibekian&title=Cysteine-reactive+probes+and+their+use+in+chemical+proteomics&doi=10.1039%2Fc8cc01485j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine-reactive probes and their use in chemical proteomics</span></div><div class="casAuthors">Hoch, Dominic G.; Abegg, Daniel; Adibekian, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">4501-4512</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Proteomic profiling using bioorthogonal chem. probes that selectively react with certain amino acids is now a widely used method in life sciences to investigate enzymic activities, study posttranslational modifications and discover novel covalent inhibitors.  Over the past two decades, researchers have developed selective probes for several different amino acids, including lysine, serine, cysteine, threonine, tyrosine, aspartate and glutamate.  Among these amino acids, cysteines are particularly interesting due to their highly diverse and complex biochem. role in our cells.  In this feature article, we focus on the chem. probes and methods used to study cysteines in complex proteomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGricIk8XJT4m7Vg90H21EOLACvtfcHk0liq64DG2B1NUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsVegs7k%253D&md5=d91bb3d96048cb6c9516d69033b2365f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2Fc8cc01485j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc8cc01485j%26sid%3Dliteratum%253Aachs%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DAdibekian%26aufirst%3DA.%26atitle%3DCysteine-reactive%2520probes%2520and%2520their%2520use%2520in%2520chemical%2520proteomics%26jtitle%3DChem.%2520Commun.%26date%3D2018%26volume%3D54%26spage%3D4501%26epage%3D4512%26doi%3D10.1039%2Fc8cc01485j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span> <span> </span><span class="NLM_article-title">Optimization of caged electrophiles for improved monitoring of cysteine reactivity in living cells</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1002/cbic.201600524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fcbic.201600524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=27813293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFyrtrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=81-84&author=M.+Aboauthor=D.+W.+Bakauthor=E.+Weerapana&title=Optimization+of+caged+electrophiles+for+improved+monitoring+of+cysteine+reactivity+in+living+cells&doi=10.1002%2Fcbic.201600524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Caged Electrophiles for Improved Monitoring of Cysteine Reactivity in Living Cells</span></div><div class="casAuthors">Abo, Masahiro; Bak, Daniel W.; Weerapana, Eranthie</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-84</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cysteine residues play crit. roles in protein function and are susceptible to numerous post-translational modifications (PTMs) that serve to modulate the activity and localization of diverse proteins.  Many of these PTMs are highly transient and labile, thus necessitating methods to study these modifications directly within the context of living cells.  We previously reported a caged electrophilic probe, CBK1, that can be activated by UV for temporally controlled covalent modification of cysteine residues in living cells.  To improve upon the no. of cysteine residues identified in cellular cysteine-profiling studies, the reactivity and uncaging efficiency of a panel of caged electrophiles were explored.  We identified an optimized caged electrophilic probe, CIK4, that affords significantly improved coverage of cellular cysteine residues.  The broader proteome coverage afforded by CIK4 renders it a useful tool for the biol. investigation of cysteine-reactivity changes and PTMs directly within living cells and highlights design elements that are crit. to optimizing photoactivatable chem. probes for cellular labeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB8LMfPRO2i7Vg90H21EOLACvtfcHk0liq64DG2B1NUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFyrtrfM&md5=08cb2975512fce1d9bbf773a27fef930</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201600524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201600524%26sid%3Dliteratum%253Aachs%26aulast%3DAbo%26aufirst%3DM.%26aulast%3DBak%26aufirst%3DD.%2BW.%26aulast%3DWeerapana%26aufirst%3DE.%26atitle%3DOptimization%2520of%2520caged%2520electrophiles%2520for%2520improved%2520monitoring%2520of%2520cysteine%2520reactivity%2520in%2520living%2520cells%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D81%26epage%3D84%26doi%3D10.1002%2Fcbic.201600524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, K. S.</span></span> <span> </span><span class="NLM_article-title">Cysteine-mediated redox signaling: Chemistry, Biology, and Tools for discovery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">4633</span>– <span class="NLM_lpage">4679</span>, <span class="refDoi"> DOI: 10.1021/cr300163e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr300163e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlert7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2013&pages=4633-4679&author=C.+E.+Paulsenauthor=K.+S.+Carroll&title=Cysteine-mediated+redox+signaling%3A+Chemistry%2C+Biology%2C+and+Tools+for+discovery&doi=10.1021%2Fcr300163e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine-mediated redox signaling: Chemistry, biology, and tools for discovery</span></div><div class="casAuthors">Paulsen, Candice E.; Carroll, Kate S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4633-4679</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The different oxidative post-translational modifications of protein Cys residue SH groups are reviewed, with particular emphasis on those chem. properties that differentiate one modification from another.  Recent progress in using chem. approaches to develop probes that enable selective and direct detection of individual modifications within their native cellular environment are also reviewed.  The discussion is complemented with examples from the literature that highlight ways in which Cys oxidn. can be used to control protein function and cell signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeGtY8LEzuF7Vg90H21EOLACvtfcHk0lhnHX6pE91yhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlert7Y%253D&md5=a071177f7815ccefe7a58abe02d24906</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fcr300163e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr300163e%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DC.%2BE.%26aulast%3DCarroll%26aufirst%3DK.%2BS.%26atitle%3DCysteine-mediated%2520redox%2520signaling%253A%2520Chemistry%252C%2520Biology%252C%2520and%2520Tools%2520for%2520discovery%26jtitle%3DChem.%2520Rev.%26date%3D2013%26volume%3D113%26spage%3D4633%26epage%3D4679%26doi%3D10.1021%2Fcr300163e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerato, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad Hari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide profiling of targets of cysteine reactive small molecules by using ethynyl benziodoxolone reagents</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">10852</span>– <span class="NLM_lpage">10857</span>, <span class="refDoi"> DOI: 10.1002/anie.201505641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fanie.201505641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1CiurbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=10852-10857&author=D.+Abeggauthor=R.+Freiauthor=L.+Ceratoauthor=D.+Prasad+Hariauthor=C.+Wangauthor=J.+Waserauthor=A.+Adibekian&title=Proteome-wide+profiling+of+targets+of+cysteine+reactive+small+molecules+by+using+ethynyl+benziodoxolone+reagents&doi=10.1002%2Fanie.201505641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-Wide Profiling of Targets of Cysteine-reactive Small Molecules by using Ethynyl Benziodoxolone Reagents</span></div><div class="casAuthors">Abegg, Daniel; Frei, Reto; Cerato, Luca; Prasad Hari, Durga; Wang, Chao; Waser, Jerome; Adibekian, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">10852-10857</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In this study, we present a highly efficient method for proteomic profiling of cysteine residues in complex proteomes and in living cells.  Our method is based on alkynylation of cysteines in complex proteomes using a "clickable" alkynyl benziodoxolone bearing an azide group.  This reaction proceeds fast, under mild physiol. conditions, and with a very high degree of chemoselectivity.  The formed azide-capped alkynyl-cysteine adducts are readily detectable by LC-MS/MS, and can be further functionalized with TAMRA or biotin alkyne via CuAAC.  We demonstrate the utility of alkynyl benziodoxolones for chem. proteomics applications by identifying the proteomic targets of curcumin, a diarylheptanoid natural product that was and still is part of multiple human clin. trials as anticancer agent.  Our results demonstrate that curcumin covalently modifies several key players of cellular signaling and metab., most notably the enzyme casein kinase I gamma.  We anticipate that this new method for cysteine profiling will find broad application in chem. proteomics and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0j8t7w7eJmLVg90H21EOLACvtfcHk0lhnHX6pE91yhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1CiurbN&md5=e033baff50f02f4ff73e70d8ac9c1106</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fanie.201505641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201505641%26sid%3Dliteratum%253Aachs%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DFrei%26aufirst%3DR.%26aulast%3DCerato%26aufirst%3DL.%26aulast%3DPrasad%2BHari%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWaser%26aufirst%3DJ.%26aulast%3DAdibekian%26aufirst%3DA.%26atitle%3DProteome-wide%2520profiling%2520of%2520targets%2520of%2520cysteine%2520reactive%2520small%2520molecules%2520by%2520using%2520ethynyl%2520benziodoxolone%2520reagents%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D10852%26epage%3D10857%26doi%3D10.1002%2Fanie.201505641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gersch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreuzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">Electrophilic natural products and their biological targets</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">659</span>– <span class="NLM_lpage">682</span>, <span class="refDoi"> DOI: 10.1039/c2np20012k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc2np20012k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=22504336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC38XntVehsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=659-682&author=M.+Gerschauthor=J.+Kreuzerauthor=S.+A.+Sieber&title=Electrophilic+natural+products+and+their+biological+targets&doi=10.1039%2Fc2np20012k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Electrophilic natural products and their biological targets</span></div><div class="casAuthors">Gersch, Malte; Kreuzer, Johannes; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">659-682</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The study of biol. active natural products has resulted in seminal contributions to the understanding of living systems.  In the case of electrophilic natural products, the covalent nature of their interaction has largely facilitated the identification of their biol. binding partners.  In this review, we provide a comprehensive compilation of electrophilic natural products from all major chem. classes together with their biol. targets.  Covering Michael acceptor systems, ring-strained compds. and other electrophiles, such as esters or carbamates, we highlight representative and instructive examples for over 20 electrophilic moieties.  The fruitful cooperation of natural product chem., medicinal chem. and chem. biol. has produced a collection of well-studied examples for how electrophilic natural products exert their biol. functions that range from antibiotic to antitumor effects.  Special emphasis is put on the elucidation of their resp. biol. targets via activity-based protein profiling, which together with the recent advancements in mass spectrometry has been crucial to the success of the field.  The wealth of naturally occurring electrophilic moieties and their chem. complexity enables binding of a large variety of biol. targets, such as enzymes of all classes, nonenzymic proteins, DNA and other cellular compds.  With approx. 30 000 genes in the human genome but only 266 confirmed protein drug targets, the study of biol. active, electrophilic natural products has the potential to provide insights into fundamental biol. processes and to greatly aid the discovery of new drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHI4qnF7oSNbVg90H21EOLACvtfcHk0lhnHX6pE91yhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntVehsb0%253D&md5=560cb6a084f0905741109551530ed02c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1039%2Fc2np20012k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2np20012k%26sid%3Dliteratum%253Aachs%26aulast%3DGersch%26aufirst%3DM.%26aulast%3DKreuzer%26aufirst%3DJ.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3DElectrophilic%2520natural%2520products%2520and%2520their%2520biological%2520targets%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2012%26volume%3D29%26spage%3D659%26epage%3D682%26doi%3D10.1039%2Fc2np20012k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunzmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">α-Methylene-γ-butyrolactones attenuate Staphylococcus aureus virulence by inhibition of transcriptional regulation</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1158</span>, <span class="refDoi"> DOI: 10.1039/c3sc52228h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc3sc52228h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Oqtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1158&author=M.+H.+Kunzmannauthor=N.+C.+Bachauthor=B.+Bauerauthor=S.+A.+Sieber&title=%CE%B1-Methylene-%CE%B3-butyrolactones+attenuate+Staphylococcus+aureus+virulence+by+inhibition+of+transcriptional+regulation&doi=10.1039%2Fc3sc52228h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">α-Methylene-γ-butyrolactones attenuate Staphylococcus aureus virulence by inhibition of transcriptional regulation</span></div><div class="casAuthors">Kunzmann, Martin H.; Bach, Nina C.; Bauer, Bianca; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1158-1167</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Bacterial pathogenesis is triggered by complex mol. mechanisms that sense bacterial d. within an infected host and induce the expression of toxins for overriding the immune response.  Virulence is controlled by a set of transcriptional regulators that directly bind to DNA promoter regions of toxin-encoding genes.  Here, we identified an α-methylene-γ-butyrolactone as a potent inhibitor of Staphylococcus aureus virulence.  Treatment of bacteria not only resulted in a markedly decreased expression of one of the most prominent virulence factors α-hemolysin (Hla) but also caused attenuated invasion efficiency.  Mass spectrometry (MS) based target identification revealed this biol. effect originating from the consolidated binding to three important transcriptional regulators SarA, SarR and MgrA.  MS investigation of the binding site uncovered a conserved cysteine in all three proteins which gets covalently modified.  Intriguingly, investigation of DNA binding demonstrated an impaired DNA-protein interaction upon compd. treatment.  The functional correlation between target binding, inhibition and the obsd. biol. effect was proven by gene knockouts and confirmed the expected mode of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFG3nbr90xeLVg90H21EOLACvtfcHk0lhnHX6pE91yhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Oqtr8%253D&md5=9ecf5a95ee148d3885d43f3b9bcc6123</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2Fc3sc52228h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3sc52228h%26sid%3Dliteratum%253Aachs%26aulast%3DKunzmann%26aufirst%3DM.%2BH.%26aulast%3DBach%26aufirst%3DN.%2BC.%26aulast%3DBauer%26aufirst%3DB.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3D%25CE%25B1-Methylene-%25CE%25B3-butyrolactones%2520attenuate%2520Staphylococcus%2520aureus%2520virulence%2520by%2520inhibition%2520of%2520transcriptional%2520regulation%26jtitle%3DChem.%2520Sci.%26date%3D2014%26volume%3D5%26spage%3D1158%26doi%3D10.1039%2Fc3sc52228h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandl, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirsch, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugur, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vomacka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fetzer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antes, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">Natural-product-inspired aminoepoxybenzoquinones kill members of the Gram-negative pathogen <i>Salmonella</i> by attenuating cellular stress response</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">14852</span>– <span class="NLM_lpage">14857</span>, <span class="refDoi"> DOI: 10.1002/anie.201607338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fanie.201607338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGitLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=14852-14857&author=F.+A.+Mandlauthor=V.+C.+Kirschauthor=I.+Ugurauthor=E.+Kunoldauthor=J.+Vomackaauthor=C.+Fetzerauthor=S.+Schneiderauthor=K.+Richterauthor=T.+M.+Fuchsauthor=I.+Antesauthor=S.+A.+Sieber&title=Natural-product-inspired+aminoepoxybenzoquinones+kill+members+of+the+Gram-negative+pathogen+Salmonella+by+attenuating+cellular+stress+response&doi=10.1002%2Fanie.201607338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Natural-Product-Inspired Aminoepoxybenzoquinones Kill Members of the Gram-Negative Pathogen Salmonella by Attenuating Cellular Stress Response</span></div><div class="casAuthors">Mandl, Franziska A.; Kirsch, Volker C.; Ugur, Ilke; Kunold, Elena; Vomacka, Jan; Fetzer, Christian; Schneider, Sabine; Richter, Klaus; Fuchs, Thilo M.; Antes, Iris; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">14852-14857</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Gram-neg. bacteria represent a challenging task for antibacterial drug discovery owing to their impermeable cell membrane and restricted uptake of small mols.  The authors herein describe the synthesis of natural-product-derived epoxycyclohexenones and explore their antibiotic activity against several pathogenic bacteria.  A compd. with activity against Salmonella Typhimurium was identified, and the target enzymes were unraveled by quant. chem. proteomics.  Importantly, two protein hits were linked to bacterial stress response, and corresponding assays revealed an elevated susceptibility to reactive oxygen species upon compd. treatment.  The consolidated inhibition of these targets provides a rationale for antibacterial activity and highlights epoxycyclohexenones as natural product scaffolds with suitable properties for killing Gram-neg. Salmonella.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIS15iSizD1LVg90H21EOLACvtfcHk0lgRqpR19Y6IQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGitLbP&md5=fca9cf9dc7031442ef628e169e410cc7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201607338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201607338%26sid%3Dliteratum%253Aachs%26aulast%3DMandl%26aufirst%3DF.%2BA.%26aulast%3DKirsch%26aufirst%3DV.%2BC.%26aulast%3DUgur%26aufirst%3DI.%26aulast%3DKunold%26aufirst%3DE.%26aulast%3DVomacka%26aufirst%3DJ.%26aulast%3DFetzer%26aufirst%3DC.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DRichter%26aufirst%3DK.%26aulast%3DFuchs%26aufirst%3DT.%2BM.%26aulast%3DAntes%26aufirst%3DI.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3DNatural-product-inspired%2520aminoepoxybenzoquinones%2520kill%2520members%2520of%2520the%2520Gram-negative%2520pathogen%2520Salmonella%2520by%2520attenuating%2520cellular%2520stress%2520response%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D14852%26epage%3D14857%26doi%3D10.1002%2Fanie.201607338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staub, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">β-Lactam probes as selective chemical-proteomic tools for the identification and functional characterization of resistance associated enzymes in MRSA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">6271</span>– <span class="NLM_lpage">6276</span>, <span class="refDoi"> DOI: 10.1021/ja901304n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja901304n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktF2nsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=6271-6276&author=I.+Staubauthor=S.+A.+Sieber&title=%CE%B2-Lactam+probes+as+selective+chemical-proteomic+tools+for+the+identification+and+functional+characterization+of+resistance+associated+enzymes+in+MRSA&doi=10.1021%2Fja901304n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">β-Lactam Probes As Selective Chemical-Proteomic Tools for the Identification and Functional Characterization of Resistance Associated Enzymes in MRSA</span></div><div class="casAuthors">Staub, Isabell; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6271-6276</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the development of antibiotic resistant bacterial strains, infectious diseases have become again a life threatening problem.  One of the reasons for this dilemma is the limited no. and breadth of current therapeutic targets for which several resistance strategies have evolved over time.  To identify resistance assocd. targets and to understand their function, activity, and regulation, we utilized a novel strategy based on small synthetic β-lactam mols. that were applied in activity based protein profiling expts. (ABPP) to comparatively profile in situ enzyme activities in antibiotic sensitive and resistant S. aureus strains (MRSA).  Several enzyme activities which are unique to the MRSA strain including known resistant assocd. targets, involved in cell wall biosynthesis and antibiotic sensing, could be identified.  In addn., we also identified uncharacterized enzymes which turned out to be capable of hydrolyzing β-lactam antibiotics.  This technol. could therefore represent a valuable tool to monitor the activity and function of other yet unexplored resistance assocd. enzymes in pathogenic bacteria and help to discover new drug targets for customized therapeutic interventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1GeSOXr49KbVg90H21EOLACvtfcHk0lgRqpR19Y6IQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktF2nsro%253D&md5=dc0a31cf0524ddce26d1c5e128d564b0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fja901304n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja901304n%26sid%3Dliteratum%253Aachs%26aulast%3DStaub%26aufirst%3DI.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3D%25CE%25B2-Lactam%2520probes%2520as%2520selective%2520chemical-proteomic%2520tools%2520for%2520the%2520identification%2520and%2520functional%2520characterization%2520of%2520resistance%2520associated%2520enzymes%2520in%2520MRSA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D6271%26epage%3D6276%26doi%3D10.1021%2Fja901304n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staub, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">β-Lactams as Selective Chemical Probes for the in Vivo Labeling of Bacterial Enzymes Involved in Cell Wall Biosynthesis, Antibiotic Resistance, and Virulence</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">13400</span>– <span class="NLM_lpage">13409</span>, <span class="refDoi"> DOI: 10.1021/ja803349j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja803349j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtV2lu7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=13400-13409&author=I.+Staubauthor=S.+A.+Sieber&title=%CE%B2-Lactams+as+Selective+Chemical+Probes+for+the+in+Vivo+Labeling+of+Bacterial+Enzymes+Involved+in+Cell+Wall+Biosynthesis%2C+Antibiotic+Resistance%2C+and+Virulence&doi=10.1021%2Fja803349j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">β-Lactams as Selective Chemical Probes for the in Vivo Labeling of Bacterial Enzymes Involved in Cell Wall Biosynthesis, Antibiotic Resistance, and Virulence</span></div><div class="casAuthors">Staub, Isabell; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13400-13409</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the development of antibiotic-resistant bacterial strains, infectious diseases have become again a life-threatening problem.  One of the reasons for this dilemma is the limited no. and breadth of current therapeutic targets for which several resistance strategies have evolved over time.  To expand the no. of addressable enzyme targets and to understand their function, activity, and regulation, the authors utilized a chem. proteomic strategy, called activity-based protein profiling (ABPP) pioneered by Cravatt, for the identification of β-lactam-binding enzymes under in vivo conditions.  In this two-tiered strategy, the authors first prepd. a selection of conventional antibiotics for labeling diverse penicillin binding proteins (PBPs) and second introduced a new synthetic generation of β-lactam probes, which labeled and inhibited a selection of addnl. PBP unrelated bacterial targets.  Among these, the virulence-assocd. enzyme ClpP and a resistance-assocd. β-lactamase were labeled and inhibited by selected probes, indicating that the specificity of β-lactams can be adjusted to versatile enzyme families with important cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3q8I13btJErVg90H21EOLACvtfcHk0lgRqpR19Y6IQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtV2lu7fK&md5=e20c1f362434fe948a61ebbe9260867c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja803349j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja803349j%26sid%3Dliteratum%253Aachs%26aulast%3DStaub%26aufirst%3DI.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3D%25CE%25B2-Lactams%2520as%2520Selective%2520Chemical%2520Probes%2520for%2520the%2520in%2520Vivo%2520Labeling%2520of%2520Bacterial%2520Enzymes%2520Involved%2520in%2520Cell%2520Wall%2520Biosynthesis%252C%2520Antibiotic%2520Resistance%252C%2520and%2520Virulence%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D13400%26epage%3D13409%26doi%3D10.1021%2Fja803349j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drechsel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandl, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieber, S. A.</span></span> <span> </span><span class="NLM_article-title">Chemical probe to monitor the Parkinsonism-associated protein DJ-1 in live cells</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2016</span>– <span class="NLM_lpage">2019</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00633</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00633" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2016-2019&author=J.+Drechselauthor=F.+A.+Mandlauthor=S.+A.+Sieber&title=Chemical+probe+to+monitor+the+Parkinsonism-associated+protein+DJ-1+in+live+cells&doi=10.1021%2Facschembio.8b00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Probe To Monitor the Parkinsonism-Associated Protein DJ-1 in Live Cells</span></div><div class="casAuthors">Drechsel, Jonas; Mandl, Franziska A.; Sieber, Stephan A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2016-2019</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Reactive oxygen species (ROS) play an important role in the onset of Parkinson's disease (PD), and deciphering protective mechanisms is a major goal for therapeutic development.  Here, DJ-1 (PARK7) gained major attention when a conserved cysteine residue with a putative role in oxidative stress sensing/protection was linked to PD.  Inspired by previous studies with a bacterial homolog of DJ-1, several amino-epoxycylcohexenones were screened for enzyme inhibition, and a chem. probe with specificity for the human ortholog was selected for cellular studies.  The probe selectively labeled the cysteine oxidn. sensor and whole proteome anal. in HeLa, A549, and SHSY5Y cell lines confirmed strong enrichment of reduced DJ-1 as the most prominent target.  Increasing levels of oxidative stress diminished this signal demonstrating the utility of the tool compd. for selective in situ monitoring of this important biomarker in its reduced state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre49Kkobp6ArVg90H21EOLACvtfcHk0lgePh9rvoh4hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtLrI&md5=91dc30eb533205dfbe1d38a970f31960</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00633%26sid%3Dliteratum%253Aachs%26aulast%3DDrechsel%26aufirst%3DJ.%26aulast%3DMandl%26aufirst%3DF.%2BA.%26aulast%3DSieber%26aufirst%3DS.%2BA.%26atitle%3DChemical%2520probe%2520to%2520monitor%2520the%2520Parkinsonism-associated%2520protein%2520DJ-1%2520in%2520live%2520cells%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2016%26epage%3D2019%26doi%3D10.1021%2Facschembio.8b00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dharmaraja, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dash, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konkimalla, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrapani, H.</span></span> <span> </span><span class="NLM_article-title">Synthesis, thiol-mediated reactive oxygen species generation profiles and anti-proliferative activities of 2,3-epoxy-1,4-naphthoquinones</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1039/c1md00234a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc1md00234a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=219-224&author=A.+T.+Dharmarajaauthor=T.+K.+Dashauthor=V.+B.+Konkimallaauthor=H.+Chakrapani&title=Synthesis%2C+thiol-mediated+reactive+oxygen+species+generation+profiles+and+anti-proliferative+activities+of+2%2C3-epoxy-1%2C4-naphthoquinones&doi=10.1039%2Fc1md00234a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2Fc1md00234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00234a%26sid%3Dliteratum%253Aachs%26aulast%3DDharmaraja%26aufirst%3DA.%2BT.%26aulast%3DDash%26aufirst%3DT.%2BK.%26aulast%3DKonkimalla%26aufirst%3DV.%2BB.%26aulast%3DChakrapani%26aufirst%3DH.%26atitle%3DSynthesis%252C%2520thiol-mediated%2520reactive%2520oxygen%2520species%2520generation%2520profiles%2520and%2520anti-proliferative%2520activities%2520of%25202%252C3-epoxy-1%252C4-naphthoquinones%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D219%26epage%3D224%26doi%3D10.1039%2Fc1md00234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Puri, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Applications of small molecule probes in dissecting mechanisms of bacterial virulence and host responses</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">5985</span>– <span class="NLM_lpage">5996</span>, <span class="refDoi"> DOI: 10.1021/bi400854d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi400854d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Cqtb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=5985-5996&author=A.+W.+Puriauthor=M.+Bogyo&title=Applications+of+small+molecule+probes+in+dissecting+mechanisms+of+bacterial+virulence+and+host+responses&doi=10.1021%2Fbi400854d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of Small Molecule Probes in Dissecting Mechanisms of Bacterial Virulence and Host Responses</span></div><div class="casAuthors">Puri, Aaron W.; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">5985-5996</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Elucidating the mol. and biochem. details of bacterial infections can be challenging because of the many complex interactions that exist between a pathogen and its host.  Consequently, many tools have been developed to aid the study of bacterial pathogenesis.  Small mols. are a valuable complement to traditional genetic techniques because they can be used to rapidly perturb genetically intractable systems and to monitor post-translationally regulated processes.  Activity-based probes are a subset of small mols. that covalently label an enzyme of interest based on its catalytic mechanism.  These tools allow monitoring of enzyme activation within the context of a native biol. system and can be used to dissect the biochem. details of enzyme function.  This review describes the development and application of activity-based probes for examg. aspects of bacterial infection on both sides of the host-pathogen interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf6fjRlyM797Vg90H21EOLACvtfcHk0lgePh9rvoh4hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Cqtb7L&md5=df69fd0943da31227af7f13bbd746786</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fbi400854d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi400854d%26sid%3Dliteratum%253Aachs%26aulast%3DPuri%26aufirst%3DA.%2BW.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DApplications%2520of%2520small%2520molecule%2520probes%2520in%2520dissecting%2520mechanisms%2520of%2520bacterial%2520virulence%2520and%2520host%2520responses%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26spage%3D5985%26epage%3D5996%26doi%3D10.1021%2Fbi400854d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welsch, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Privileged scaffolds for library design and drug discovery</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1016%2Fj.cbpa.2010.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=20303320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=347-361&author=M.+E.+Welschauthor=S.+A.+Snyderauthor=B.+R.+Stockwell&title=Privileged+scaffolds+for+library+design+and+drug+discovery&doi=10.1016%2Fj.cbpa.2010.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged scaffolds for library design and drug discovery</span></div><div class="casAuthors">Welsch, Matthew E.; Snyder, Scott A.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-361</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review explores the concept of using privileged scaffolds to identify biol. active compds. through building chem. libraries.  We hope to accomplish three main objectives: to provide one of the most comprehensive listings of privileged scaffolds; to reveal through four selected examples the present state of the art in privileged scaffold library synthesis (in hopes of inspiring new and even more creative approaches); and also to offer some thoughts on how new privileged scaffolds might be identified and exploited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWQPxdsWawiLVg90H21EOLACvtfcHk0lgePh9rvoh4hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D&md5=fd0c2a642f063fd6f42f2c34fd186f8a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DWelsch%26aufirst%3DM.%2BE.%26aulast%3DSnyder%26aufirst%3DS.%2BA.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DPrivileged%2520scaffolds%2520for%2520library%2520design%2520and%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D347%26epage%3D361%26doi%3D10.1016%2Fj.cbpa.2010.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><a href="https://clsi.org/media/1632/m07a10_sample.pdf" class="extLink">https://clsi.org/media/1632/m07a10_sample.pdf</a> (accessed June
20, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fclsi.org%2Fmedia%2F1632%2Fm07a10_sample.pdf+%28accessed+June%0A20%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span> <span> </span><span class="NLM_article-title">Identification and bioevaluation of SRI-12742 as an antimicrobial agent against multidrug-resistant <i>Acinetobacter baumannii</i></span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2018.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1016%2Fj.ijantimicag.2018.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=29501820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFSqsrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2018&pages=22-27&author=M.+Shuklaauthor=I.+Soniauthor=K.+Matsuyamaauthor=T.+Tranauthor=M.+Tangaauthor=L.+Gongauthor=S.+Chopra&title=Identification+and+bioevaluation+of+SRI-12742+as+an+antimicrobial+agent+against+multidrug-resistant+Acinetobacter+baumannii&doi=10.1016%2Fj.ijantimicag.2018.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and bioevaluation of SRI-12742 as an antimicrobial agent against multidrug-resistant Acinetobacter baumannii</span></div><div class="casAuthors">Shukla, Manjulika; Soni, Isha; Matsuyama, Karen; Tran, Tran; Tanga, Mary; Gong, Leyi; Chopra, Sidharth</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-27</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Multidrug-resistant Acinetobacter baumannii (MDR-Ab) is one of the most significant nosocomial pathogens that is being increasingly isolated in healthcare settings worldwide.  Owing to its inherent drug-resistant nature, coupled with its ability to readily acquire resistance to other antibiotic classes, there is a real dearth of antibiotics available to treat infections with MDR-Ab.  A com. available library was screened against MDR-Ab BAA-1605 to identify novel inhibitory mols.  The selectivity index of a hit was tested against Vero cells and in vitro efficacy was profiled against a panel of clin. MDR-Ab.  The bacteriostatic or bactericidal nature was detd. by time-kill expts., and synergy with clin. approved drugs was detd. by the chequerboard method.  Addnl., in vivo efficacy was measured in a murine neutropenic A. baumannii thigh infection model.  SRI-12742 was identified as a potent active hit, with a min. inhibitory concn. (MIC) of 4 mg/L against BAA-1605.  Its activity was then profiled against a MDR-Ab clin. strain panel (MICs 4 mg/L to >64 mg/L).  SRI-12742 exhibited concn.-dependent bactericidal activity and caused an ca. 16 log10 CFU/mL redn. at 10 x MIC in 24 h, which is comparable with minocycline.  In a murine neutropenic thigh infection model of A. baumannii infection, SRI-12742 reduced CFU counts by ca. 0.9 log10 CFU, which is comparable with polymyxin B.  In addn., SRI-12742 synergised with all classes of antibiotics tested.  SRI-12742 exhibits all of the criteria necessary to be positioned as a novel lead with potential to be deployed for the treatment of infections caused by MDR-Ab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlXbBF4QAxwLVg90H21EOLACvtfcHk0liImm4tnrIe4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFSqsrzN&md5=b96e66d537c34e9c35a00e8a3a7cad3d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2018.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2018.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DM.%26aulast%3DSoni%26aufirst%3DI.%26aulast%3DMatsuyama%26aufirst%3DK.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DTanga%26aufirst%3DM.%26aulast%3DGong%26aufirst%3DL.%26aulast%3DChopra%26aufirst%3DS.%26atitle%3DIdentification%2520and%2520bioevaluation%2520of%2520SRI-12742%2520as%2520an%2520antimicrobial%2520agent%2520against%2520multidrug-resistant%2520Acinetobacter%2520baumannii%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2018%26volume%3D52%26spage%3D22%26epage%3D27%26doi%3D10.1016%2Fj.ijantimicag.2018.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rybak, M. J.</span></span> <span> </span><span class="NLM_article-title">Bactericidal activities of daptomycin, quinupristin-dalfopristin, and linezolid against vancomycin-resistant Staphylococcus aureus in an <i>in vitro</i> pharmacodynamic model with simulated endocardial vegetations</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3960</span>– <span class="NLM_lpage">3963</span>, <span class="refDoi"> DOI: 10.1128/aac.47.12.3960-3963.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1128%2Faac.47.12.3960-3963.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=14638509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsV2msr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2003&pages=3960-3963&author=R.+Chaauthor=W.+J.+Brownauthor=M.+J.+Rybak&title=Bactericidal+activities+of+daptomycin%2C+quinupristin-dalfopristin%2C+and+linezolid+against+vancomycin-resistant+Staphylococcus+aureus+in+an+in+vitro+pharmacodynamic+model+with+simulated+endocardial+vegetations&doi=10.1128%2Faac.47.12.3960-3963.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bactericidal activities of daptomycin, quinupristin-dalfopristin, and linezolid against vancomycin-resistant Staphylococcus aureus in an in vitro pharmacodynamic model with simulated endocardial vegetations</span></div><div class="casAuthors">Cha, Raymond; Brown, William J.; Rybak, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3960-3963</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In search of treatment alternatives against vancomycin-resistant S. aureus (VRSA), an in vitro pharmacodynamic model with simulated endocardial vegetations incorporating protein and a high inoculum was used to simulate daptomycin, linezolid, quinupristin-dalfopristin, and vancomycin against the Michigan VRSA strain.  Daptomycin and quinupristin-dalfopristin exhibited the greatest bacterial redns., and all tested agents except vancomycin exhibited bactericidal activity against the VRSA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-E5EvH4C2E7Vg90H21EOLACvtfcHk0liImm4tnrIe4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsV2msr4%253D&md5=e7ae79398cf24dbef61af4b97b8acdd2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2Faac.47.12.3960-3963.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.47.12.3960-3963.2003%26sid%3Dliteratum%253Aachs%26aulast%3DCha%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DW.%2BJ.%26aulast%3DRybak%26aufirst%3DM.%2BJ.%26atitle%3DBactericidal%2520activities%2520of%2520daptomycin%252C%2520quinupristin-dalfopristin%252C%2520and%2520linezolid%2520against%2520vancomycin-resistant%2520Staphylococcus%2520aureus%2520in%2520an%2520in%2520vitro%2520pharmacodynamic%2520model%2520with%2520simulated%2520endocardial%2520vegetations%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2003%26volume%3D47%26spage%3D3960%26epage%3D3963%26doi%3D10.1128%2Faac.47.12.3960-3963.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Housecroft, C.</span>; <span class="NLM_string-name">Sharpe, A.</span></span> <i>Inorganic Chemistry</i>, <span class="NLM_edition">4</span>th ed.; <span class="NLM_publisher-name">Prentice Hall</span>: <span class="NLM_publisher-loc">USA</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=C.+Housecroft&author=A.+Sharpe&title=Inorganic+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHousecroft%26aufirst%3DC.%26btitle%3DInorganic%2520Chemistry%26pub%3DPrentice%2520Hall%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Embaby, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoffelen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kofoed, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meldal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diness, F.</span></span> <span> </span><span class="NLM_article-title">Rational tuning of fluorobenzene probes for cysteine-selective protein modification</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8022</span>– <span class="NLM_lpage">8026</span>, <span class="refDoi"> DOI: 10.1002/anie.201712589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fanie.201712589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1els7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=8022-8026&author=A.+M.+Embabyauthor=S.+Schoffelenauthor=C.+Kofoedauthor=M.+Meldalauthor=F.+Diness&title=Rational+tuning+of+fluorobenzene+probes+for+cysteine-selective+protein+modification&doi=10.1002%2Fanie.201712589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Tuning of Fluorobenzene Probes for Cysteine-Selective Protein Modification</span></div><div class="casAuthors">Embaby, Ahmed M.; Schoffelen, Sanne; Kofoed, Christian; Meldal, Morten; Diness, Frederik</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8022-8026</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Fluorobenzene probes for protein profiling through selective cysteine labeling have been developed by rational reactivity tuning.  Tuning was achieved by selecting an electron-withdrawing para substituent in combination with variation of the no. of fluorine substituents.  Optimized probes chemoselectively arylated cysteine residues in proteins under aq. conditions.  Probes linked to azide, biotin, or a fluorophore were applicable to labeling of eGFP and albumin.  Selective inhibition of cysteine proteases was also demonstrated with the probes.  Addnl., probes were tuned for site-selective labeling of cysteine residues and for activity-based protein profiling in cell lysates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrnIOutUPylLVg90H21EOLACvtfcHk0liImm4tnrIe4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1els7c%253D&md5=a2317907c517038ce96c8f6837f37b4d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201712589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201712589%26sid%3Dliteratum%253Aachs%26aulast%3DEmbaby%26aufirst%3DA.%2BM.%26aulast%3DSchoffelen%26aufirst%3DS.%26aulast%3DKofoed%26aufirst%3DC.%26aulast%3DMeldal%26aufirst%3DM.%26aulast%3DDiness%26aufirst%3DF.%26atitle%3DRational%2520tuning%2520of%2520fluorobenzene%2520probes%2520for%2520cysteine-selective%2520protein%2520modification%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D8022%26epage%3D8026%26doi%3D10.1002%2Fanie.201712589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheerens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprengeler, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrill, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendonca, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, K. C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grothaus, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffery, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spoerke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honigberg, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalrymple, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, J. T.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective irreversible inhibitors for Bruton’s tyrosine kinase</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200600221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fcmdc.200600221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=17154430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=58-61&author=Z.+Panauthor=H.+Scheerensauthor=S.-J.+Liauthor=B.+E.+Schultzauthor=P.+A.+Sprengelerauthor=L.+C.+Burrillauthor=R.+V.+Mendoncaauthor=M.+D.+Sweeneyauthor=K.+C.+K.+Scottauthor=P.+G.+Grothausauthor=D.+A.+Jefferyauthor=J.+M.+Spoerkeauthor=L.+A.+Honigbergauthor=P.+R.+Youngauthor=S.+A.+Dalrympleauthor=J.+T.+Palmer&title=Discovery+of+selective+irreversible+inhibitors+for+Bruton%E2%80%99s+tyrosine+kinase&doi=10.1002%2Fcmdc.200600221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective irreversible inhibitors for bruton's tyrosine kinase</span></div><div class="casAuthors">Pan, Zhengying; Scheerens, Heleen; Li, Shyr-Jiann; Schultz, Brian E.; Sprengeler, Paul A.; Burrill, L. Chuck; Mendonca, Rohan V.; Sweeney, Michael D.; Scott, Keana C. K.; Grothaus, Paul G.; Jeffery, Douglas A.; Spoerke, Jill M.; Honigberg, Lee A.; Young, Peter R.; Dalrymple, Stacie A.; Palmer, James T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-61</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthesis and pharmacol. evaluation of a series of peptidomimetic quinoline derivs. was undertaken to evaluate their efficacy in acting as selective irreversible inhibitors of Bruton's tyrosine kinase (Btk).  With the relative scarcity of knowledge on the inhibition of Btk it is crucial to discover a potent and selective tool compd. for this kinase.  Herein is described the discovery of selective irreversible Btk inhibitors and their efficacy in a mouse RA model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovT3kBCYECdbVg90H21EOLACvtfcHk0liOZAoj1YHz-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D&md5=023ba8bca9942ad0127d7cf32293e9d2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600221%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DScheerens%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.-J.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26aulast%3DSprengeler%26aufirst%3DP.%2BA.%26aulast%3DBurrill%26aufirst%3DL.%2BC.%26aulast%3DMendonca%26aufirst%3DR.%2BV.%26aulast%3DSweeney%26aufirst%3DM.%2BD.%26aulast%3DScott%26aufirst%3DK.%2BC.%2BK.%26aulast%3DGrothaus%26aufirst%3DP.%2BG.%26aulast%3DJeffery%26aufirst%3DD.%2BA.%26aulast%3DSpoerke%26aufirst%3DJ.%2BM.%26aulast%3DHonigberg%26aufirst%3DL.%2BA.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DDalrymple%26aufirst%3DS.%2BA.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26atitle%3DDiscovery%2520of%2520selective%2520irreversible%2520inhibitors%2520for%2520Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D58%26epage%3D61%26doi%3D10.1002%2Fcmdc.200600221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dix, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hett, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joslyn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnute, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulce, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whiteley, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">A road map to evaluate the proteome-wide selectivity of covalent kinase inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">760</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1582</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio.1582" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=25038787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=760&author=B.+R.+Lanningauthor=L.+R.+Whitbyauthor=M.+M.+Dixauthor=J.+Douhanauthor=A.+M.+Gilbertauthor=E.+C.+Hettauthor=T.+O.+Johnsonauthor=C.+Joslynauthor=J.+C.+Kathauthor=S.+Niessenauthor=L.+R.+Robertsauthor=M.+E.+Schnuteauthor=C.+Wangauthor=J.+J.+Hulceauthor=B.+Weiauthor=L.+O.+Whiteleyauthor=M.+M.+Haywardauthor=B.+F.+Cravatt&title=A+road+map+to+evaluate+the+proteome-wide+selectivity+of+covalent+kinase+inhibitors&doi=10.1038%2Fnchembio.1582"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A road map to evaluate the proteome-wide selectivity of covalent kinase inhibitors</span></div><div class="casAuthors">Lanning, Bryan R.; Whitby, Landon R.; Dix, Melissa M.; Douhan, John; Gilbert, Adam M.; Hett, Erik C.; Johnson, Theodore O.; Joslyn, Chris; Kath, John C.; Niessen, Sherry; Roberts, Lee R.; Schnute, Mark E.; Wang, Chu; Hulce, Jonathan J.; Wei, Baoxian; Whiteley, Laurence O.; Hayward, Matthew M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">760-767</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinases are principal components of signal transduction pathways and the focus of intense basic and drug discovery research.  Irreversible inhibitors that covalently modify non-catalytic cysteines in kinase active sites have emerged as valuable probes and approved drugs.  Many protein classes, however, have functional cysteines, and therefore understanding the proteome-wide selectivity of covalent kinase inhibitors is imperative.  Here, we accomplish this objective using activity-based protein profiling coupled with quant. MS to globally map the targets, both specific and nonspecific, of covalent kinase inhibitors in human cells.  Many of the specific off-targets represent nonkinase proteins that, notably, have conserved active site cysteines.  We define windows of selectivity for covalent kinase inhibitors and show that, when these windows are exceeded, rampant proteome-wide reactivity and kinase target-independent cell death conjointly occur.  Our findings, taken together, provide an exptl. road map to illuminate opportunities and surmount challenges for the development of covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMAiUlEg9R5rVg90H21EOLACvtfcHk0liOZAoj1YHz-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7nO&md5=6f17b0d1b00e088f37c511eea577d290</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1582&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1582%26sid%3Dliteratum%253Aachs%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DWhitby%26aufirst%3DL.%2BR.%26aulast%3DDix%26aufirst%3DM.%2BM.%26aulast%3DDouhan%26aufirst%3DJ.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DHett%26aufirst%3DE.%2BC.%26aulast%3DJohnson%26aufirst%3DT.%2BO.%26aulast%3DJoslyn%26aufirst%3DC.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26aulast%3DSchnute%26aufirst%3DM.%2BE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHulce%26aufirst%3DJ.%2BJ.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWhiteley%26aufirst%3DL.%2BO.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DA%2520road%2520map%2520to%2520evaluate%2520the%2520proteome-wide%2520selectivity%2520of%2520covalent%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D760%26doi%3D10.1038%2Fnchembio.1582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allimuthu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span> <span> </span><span class="NLM_article-title">2-Chloropropionamide as a low-reactivity electrophile for irreversible small-molecule probe identification</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2124</span>– <span class="NLM_lpage">2131</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00424</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00424" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvVGqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2124-2131&author=D.+Allimuthuauthor=D.+J.+Adams&title=2-Chloropropionamide+as+a+low-reactivity+electrophile+for+irreversible+small-molecule+probe+identification&doi=10.1021%2Facschembio.7b00424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">2-Chloropropionamide As a Low-Reactivity Electrophile for Irreversible Small-Molecule Probe Identification</span></div><div class="casAuthors">Allimuthu, Dharmaraja; Adams, Drew J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2124-2131</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Resurgent interest in covalent target engagement in drug discovery has demonstrated that small mols. contg. weakly reactive electrophiles can be safe and effective therapies.  Several recently FDA-approved drugs feature an acrylamide functionality to selectively engage cysteine side chains of kinases (Ibrutinib, Afatinib, and Neratinib).  Addnl. electrophilic functionalities whose reactivity is compatible with highly selective target engagement and in vivo application could open new avenues in covalent small mol. discovery.  Here the authors report the synthesis and evaluation of a library of small mols. contg. the 2-chloropropionamide functionality, which the authors demonstrate is less reactive than typical acrylamide electrophiles.  Although many library members do not appear to label proteins in cells, the authors identified S-CW3554 as selectively labeling protein disulfide isomerase and inhibiting its enzymic activity.  Subsequent profiling of the library against five diverse cancer cell lines showed unique cytotoxicity for S-CW3554 I in cells derived from multiple myeloma, a cancer recently reported to be sensitive to PDI inhibition.  The novel PDI inhibitor highlights the potential of 2-chloropropionamides as weak and stereochem.-tunable electrophiles for covalent drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonSvqy3pfybrVg90H21EOLACvtfcHk0liOZAoj1YHz-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvVGqsrw%253D&md5=39b4794ebb55d3314a7bac5b04ddadc7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00424%26sid%3Dliteratum%253Aachs%26aulast%3DAllimuthu%26aufirst%3DD.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26atitle%3D2-Chloropropionamide%2520as%2520a%2520low-reactivity%2520electrophile%2520for%2520irreversible%2520small-molecule%2520probe%2520identification%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2124%26epage%3D2131%26doi%3D10.1021%2Facschembio.7b00424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shindo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miura, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatsuyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokunaga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiroishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caaveiro, J. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyanagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohdo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamachi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ono, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojida, A.</span></span> <span> </span><span class="NLM_article-title">Selective and reversible modification of kinase cysteines with chlorofluoroacetamides</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0204-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fs41589-018-0204-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=30643284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFSgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=250-258&author=N.+Shindoauthor=H.+Fuchidaauthor=M.+Satoauthor=K.+Watariauthor=T.+Shibataauthor=K.+Kuwataauthor=C.+Miuraauthor=K.+Okamotoauthor=Y.+Hatsuyamaauthor=K.+Tokunagaauthor=S.+Sakamotoauthor=S.+Morimotoauthor=Y.+Abeauthor=M.+Shiroishiauthor=J.+M.+M.+Caaveiroauthor=T.+Uedaauthor=T.+Tamuraauthor=N.+Matsunagaauthor=T.+Nakaoauthor=S.+Koyanagiauthor=S.+Ohdoauthor=Y.+Yamaguchiauthor=I.+Hamachiauthor=M.+Onoauthor=A.+Ojida&title=Selective+and+reversible+modification+of+kinase+cysteines+with+chlorofluoroacetamides&doi=10.1038%2Fs41589-018-0204-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Selective and reversible modification of kinase cysteines with chlorofluoroacetamides</span></div><div class="casAuthors">Shindo, Naoya; Fuchida, Hirokazu; Sato, Mami; Watari, Kosuke; Shibata, Tomohiro; Kuwata, Keiko; Miura, Chizuru; Okamoto, Kei; Hatsuyama, Yuji; Tokunaga, Keisuke; Sakamoto, Seiichi; Morimoto, Satoshi; Abe, Yoshito; Shiroishi, Mitsunori; Caaveiro, Jose M. M.; Ueda, Tadashi; Tamura, Tomonori; Matsunaga, Naoya; Nakao, Takaharu; Koyanagi, Satoru; Ohdo, Shigehiro; Yamaguchi, Yasuchika; Hamachi, Itaru; Ono, Mayumi; Ojida, Akio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">250-258</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Irreversible inhibition of disease-assocd. proteins with small mols. is a powerful approach for achieving increased and sustained pharmacol. potency.  Here, we introduce α-chlorofluoroacetamide (CFA) as a novel warhead of targeted covalent inhibitor (TCI).  Despite weak intrinsic reactivity, CFA-appended quinazoline showed high reactivity toward Cys797 of epidermal growth factor receptor (EGFR).  In cells, CFA-quinazoline showed higher target specificity for EGFR than the corresponding Michael acceptors in a wide concn. range (0.1-10 μM).  The cysteine adduct of the CFA deriv. was susceptible to hydrolysis and reversibly yielded intact thiol but was stable in solvent-sequestered ATP-binding pocket of EGFR.  This environment-dependent hydrolysis can potentially reduce off-target protein modification by CFA-based drugs.  Oral administration of CFA quinazoline, (2R)-1-(2-chloro-2-fluoroacetyl)-N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[3-(4-morpholinyl)-propoxy]quinazolin-6-yl]pyrrolidine-2-carboxamide [2226257-92-5] (NS-062, compd. 51) significantly suppressed tumor growth in a mouse xenograft model.  Further, CFA-appended pyrazolopyrimidine irreversibly inhibited Bruton's tyrosine kinase with higher target specificity.  These results demonstrate the utility of CFA as a new class warheads for TCI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_uzgSiZvaPbVg90H21EOLACvtfcHk0liqpC_v8xsqzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFSgur4%253D&md5=f6c8886d540ea26a8abecd950feba913</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0204-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0204-3%26sid%3Dliteratum%253Aachs%26aulast%3DShindo%26aufirst%3DN.%26aulast%3DFuchida%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DM.%26aulast%3DWatari%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DT.%26aulast%3DKuwata%26aufirst%3DK.%26aulast%3DMiura%26aufirst%3DC.%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DHatsuyama%26aufirst%3DY.%26aulast%3DTokunaga%26aufirst%3DK.%26aulast%3DSakamoto%26aufirst%3DS.%26aulast%3DMorimoto%26aufirst%3DS.%26aulast%3DAbe%26aufirst%3DY.%26aulast%3DShiroishi%26aufirst%3DM.%26aulast%3DCaaveiro%26aufirst%3DJ.%2BM.%2BM.%26aulast%3DUeda%26aufirst%3DT.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DMatsunaga%26aufirst%3DN.%26aulast%3DNakao%26aufirst%3DT.%26aulast%3DKoyanagi%26aufirst%3DS.%26aulast%3DOhdo%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DHamachi%26aufirst%3DI.%26aulast%3DOno%26aufirst%3DM.%26aulast%3DOjida%26aufirst%3DA.%26atitle%3DSelective%2520and%2520reversible%2520modification%2520of%2520kinase%2520cysteines%2520with%2520chlorofluoroacetamides%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D250%26epage%3D258%26doi%3D10.1038%2Fs41589-018-0204-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sankar, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumbhare, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmaraja, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrapani, H.</span></span> <span> </span><span class="NLM_article-title">A phenacrylate scaffold for tunable thiol activation and release</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">15323</span>– <span class="NLM_lpage">15326</span>, <span class="refDoi"> DOI: 10.1039/c4cc07343f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1039%2Fc4cc07343f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=25347732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCitL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=15323-15326&author=R.+K.+Sankarauthor=R.+S.+Kumbhareauthor=A.+T.+Dharmarajaauthor=H.+Chakrapani&title=A+phenacrylate+scaffold+for+tunable+thiol+activation+and+release&doi=10.1039%2Fc4cc07343f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A phenacrylate scaffold for tunable thiol activation and release</span></div><div class="casAuthors">Sankar, Rathinam K.; Kumbhare, Rohan S.; Dharmaraja, Allimuthu T.; Chakrapani, Harinath</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">97</span>),
    <span class="NLM_cas:pages">15323-15326</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A thiol-selective 2-methyl-3-phenacrylate scaffold with spatiotemporal control over delivery of a cargo is reported.  The half-lives of decompn. could be tuned from 30 min to 1 day and the scaffold's utility in thiol-inducible fluorophore release in cell-free as well as within cells is demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2eFxDyIc3bLVg90H21EOLACvtfcHk0liqpC_v8xsqzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCitL7N&md5=ed4ed0ca291eeff03907c6a6c6736bbd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2Fc4cc07343f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4cc07343f%26sid%3Dliteratum%253Aachs%26aulast%3DSankar%26aufirst%3DR.%2BK.%26aulast%3DKumbhare%26aufirst%3DR.%2BS.%26aulast%3DDharmaraja%26aufirst%3DA.%2BT.%26aulast%3DChakrapani%26aufirst%3DH.%26atitle%3DA%2520phenacrylate%2520scaffold%2520for%2520tunable%2520thiol%2520activation%2520and%2520release%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D15323%26epage%3D15326%26doi%3D10.1039%2Fc4cc07343f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serafimova, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pufall, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maglathlin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frödin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Reversible targeting of noncatalytic cysteines with chemically tuned electrophiles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">476</span>, <span class="refDoi"> DOI: 10.1038/nchembio.925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio.925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=22466421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVKitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=471-476&author=I.+M.+Serafimovaauthor=M.+A.+Pufallauthor=S.+Krishnanauthor=K.+Dudaauthor=M.+S.+Cohenauthor=R.+L.+Maglathlinauthor=J.+M.+McFarlandauthor=R.+M.+Millerauthor=M.+Fr%C3%B6dinauthor=J.+Taunton&title=Reversible+targeting+of+noncatalytic+cysteines+with+chemically+tuned+electrophiles&doi=10.1038%2Fnchembio.925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible targeting of noncatalytic cysteines with chemically tuned electrophiles</span></div><div class="casAuthors">Serafimova, Iana M.; Pufall, Miles A.; Krishnan, Shyam; Duda, Katarzyna; Cohen, Michael S.; Maglathlin, Rebecca L.; McFarland, Jesse M.; Miller, Rand M.; Froedin, Morten; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-476</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Targeting noncatalytic cysteine residues with irreversible acrylamide-based inhibitors is a powerful approach for enhancing pharmacol. potency and selectivity.  Nevertheless, concerns about off-target modification motivate the development of reversible cysteine-targeting strategies.  Here we show that electron-deficient olefins, including acrylamides, can be tuned to react with cysteine thiols in a rapidly reversible manner.  Installation of a nitrile group increased the olefins' intrinsic reactivity, but, paradoxically, eliminated the formation of irreversible adducts.  Incorporation of these electrophiles into a noncovalent kinase-recognition scaffold produced slowly dissocg., covalent inhibitors of the p90 ribosomal protein S6 kinase RSK2.  A cocrystal structure revealed specific noncovalent interactions that stabilize the complex by positioning the electrophilic carbon near the targeted cysteine.  Disruption of these interactions by protein unfolding or proteolysis promoted instantaneous cleavage of the covalent bond.  Our results establish a chem.-based framework for engineering sustained covalent inhibition without accumulating permanently modified proteins and peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlcv3GjPOB6rVg90H21EOLACvtfcHk0li17Gn6iloFrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVKitrw%253D&md5=6455a9a029714beb317286f71ac1326b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.925%26sid%3Dliteratum%253Aachs%26aulast%3DSerafimova%26aufirst%3DI.%2BM.%26aulast%3DPufall%26aufirst%3DM.%2BA.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DDuda%26aufirst%3DK.%26aulast%3DCohen%26aufirst%3DM.%2BS.%26aulast%3DMaglathlin%26aufirst%3DR.%2BL.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DFr%25C3%25B6din%26aufirst%3DM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DReversible%2520targeting%2520of%2520noncatalytic%2520cysteines%2520with%2520chemically%2520tuned%2520electrophiles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D471%26epage%3D476%26doi%3D10.1038%2Fnchembio.925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Design of reversible, cysteine-targeted Michael acceptors guided by kinetic and computational analysis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">12624</span>– <span class="NLM_lpage">12630</span>, <span class="refDoi"> DOI: 10.1021/ja505194w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja505194w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWrtbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=12624-12630&author=S.+Krishnanauthor=R.+M.+Millerauthor=B.+Tianauthor=R.+D.+Mullinsauthor=M.+P.+Jacobsonauthor=J.+Taunton&title=Design+of+reversible%2C+cysteine-targeted+Michael+acceptors+guided+by+kinetic+and+computational+analysis&doi=10.1021%2Fja505194w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Reversible, Cysteine-Targeted Michael Acceptors Guided by Kinetic and Computational Analysis</span></div><div class="casAuthors">Krishnan, Shyam; Miller, Rand M.; Tian, Boxue; Mullins, R. Dyche; Jacobson, Matthew P.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">12624-12630</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Electrophilic probes that covalently modify a cysteine thiol often show enhanced pharmacol. potency and selectivity.  Although reversible Michael acceptors have been reported, the structural requirements for reversibility are poorly understood.  Here, we report a novel class of acrylonitrile-based Michael acceptors, activated by aryl or heteroaryl electron-withdrawing groups.  We demonstrate that thiol adducts of these acrylonitriles undergo β-elimination at rates that span more than 3 orders of magnitude.  These rates correlate inversely with the computed proton affinity of the corresponding carbanions, enabling the intrinsic reversibility of the thiol-Michael reaction to be tuned in a predictable manner.  We apply these principles to the design of new reversible covalent kinase inhibitors with improved properties.  A cocrystal structure of one such inhibitor reveals specific noncovalent interactions between the 1,2,4-triazole activating group and the kinase.  Our exptl. and computational study enables the design of new Michael acceptors, expanding the palette of reversible, cysteine-targeted electrophiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHXljSEecz5LVg90H21EOLACvtfcHk0li17Gn6iloFrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWrtbvK&md5=de052bac07c8920cb807c05f71f4f95d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja505194w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja505194w%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DMullins%26aufirst%3DR.%2BD.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DDesign%2520of%2520reversible%252C%2520cysteine-targeted%2520Michael%2520acceptors%2520guided%2520by%2520kinetic%2520and%2520computational%2520analysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D12624%26epage%3D12630%26doi%3D10.1021%2Fja505194w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khodade, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharath Chandra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulipeta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangarajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrapani, H.</span></span> <span> </span><span class="NLM_article-title">Bioreductively activated reactive oxygen species (ROS) generators as MRSA inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1021/ml5001118</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5001118" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFOiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=777-781&author=V.+S.+Khodadeauthor=M.+Sharath+Chandraauthor=A.+Banerjeeauthor=S.+Lahiriauthor=M.+Pulipetaauthor=R.+Rangarajanauthor=H.+Chakrapani&title=Bioreductively+activated+reactive+oxygen+species+%28ROS%29+generators+as+MRSA+inhibitors&doi=10.1021%2Fml5001118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Bioreductively Activated Reactive Oxygen Species (ROS) Generators as MRSA Inhibitors</span></div><div class="casAuthors">Khodade, Vinayak S.; Sharath Chandra, Mallojjala; Banerjee, Ankita; Lahiri, Surobhi; Pulipeta, Mallikarjuna; Rangarajan, Radha; Chakrapani, Harinath</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">777-781</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The no. of cases of drug resistant Staphylococcus aureus infections is on the rise globally and new strategies to identify drug candidates with novel mechanisms of action are in urgent need.  Here, we report the synthesis and evaluation of a series of benzo[b]phenanthridine-5,7,12(6H)-triones, which were designed based on redox-active natural products.  We find that the in vitro inhibitory activity of 6-(prop-2-ynyl)benzo[b]phenanthridine-5,7,12(6H)-trione (I) against methicillin-resistant Staphylococcus aureus (MRSA), including a panel of patient-derived strains, is comparable or better than vancomycin.  We show that the lead compd. generates reactive oxygen species (ROS) in the cell, contributing to its antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG_I34Hm-Jx7Vg90H21EOLACvtfcHk0lip0Nwr28anWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFOiu74%253D&md5=b4ef0f3179a9d4bb5059c3a42d86b763</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fml5001118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5001118%26sid%3Dliteratum%253Aachs%26aulast%3DKhodade%26aufirst%3DV.%2BS.%26aulast%3DSharath%2BChandra%26aufirst%3DM.%26aulast%3DBanerjee%26aufirst%3DA.%26aulast%3DLahiri%26aufirst%3DS.%26aulast%3DPulipeta%26aufirst%3DM.%26aulast%3DRangarajan%26aufirst%3DR.%26aulast%3DChakrapani%26aufirst%3DH.%26atitle%3DBioreductively%2520activated%2520reactive%2520oxygen%2520species%2520%2528ROS%2529%2520generators%2520as%2520MRSA%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D777%26epage%3D781%26doi%3D10.1021%2Fml5001118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kosower, N. S.</span>; <span class="NLM_string-name">Kosower, E. M.</span></span> <i>Thiol Labeling with Bromobimanes</i>; <span class="NLM_series">Methods in Enzymology</span>; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">1987</span>; Vol.  <span class="NLM_volume">143</span>, pp  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">84</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1016%2F0076-6879%2887%2943015-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1987&pages=76-84&author=N.+S.+Kosower&author=E.+M.+Kosower&title=Thiol+Labeling+with+Bromobimanes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2887%2943015-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252F0076-6879%252887%252943015-2%26sid%3Dliteratum%253Aachs%26aulast%3DKosower%26aufirst%3DN.%2BS.%26btitle%3DThiol%2520Labeling%2520with%2520Bromobimanes%26pub%3DAcademic%2520Press%26date%3D1987%26volume%3D143%26spage%3D76%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Speers, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling <i>in vivo</i> using a copper(I)-catalyzed azide-alkyne [3 + 2] cycloaddition</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">4686</span>– <span class="NLM_lpage">4687</span>, <span class="refDoi"> DOI: 10.1021/ja034490h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja034490h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlylu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=4686-4687&author=A.+E.+Speersauthor=G.+C.+Adamauthor=B.+F.+Cravatt&title=Activity-based+protein+profiling+in+vivo+using+a+copper%28I%29-catalyzed+azide-alkyne+%5B3+%2B+2%5D+cycloaddition&doi=10.1021%2Fja034490h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling in vivo using a copper(I)-catalyzed azide-alkyne [3 + 2] cycloaddition</span></div><div class="casAuthors">Speers, Anna E.; Adam, Gregory C.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4686-4687</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Toward the goal of assigning function to the tens of thousands of protein products encoded by eukaryotic and prokaryotic genomes, the field of proteomics requires new technologies that can functionally characterize proteins within the dynamic environment of the cell, where these biomols. are subject to myriad posttranslational modifications and the actions of endogenous activators and inhibitors. Here, we report an advanced strategy for activity-based protein profiling (ABPP) that addresses this important need.  We show that several enzymes can be labeled in an activity-based manner both in vitro and in vivo by an azido-sulfonate ester probe and that these labeling events can be detected in whole proteomes by copper-catalyzed ligation with a rhodamine-alkyne reagent.  This click chem.-based strategy for ABPP represents a unique and versatile method for functional proteome anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGXIFycwk3N7Vg90H21EOLACvtfcHk0lip0Nwr28anWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlylu7o%253D&md5=ead5de3b59297fe0387d34b0949661ab</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja034490h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja034490h%26sid%3Dliteratum%253Aachs%26aulast%3DSpeers%26aufirst%3DA.%2BE.%26aulast%3DAdam%26aufirst%3DG.%2BC.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520protein%2520profiling%2520in%2520vivo%2520using%2520a%2520copper%2528I%2529-catalyzed%2520azide-alkyne%2520%255B3%2520%252B%25202%255D%2520cycloaddition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D4686%26epage%3D4687%26doi%3D10.1021%2Fja034490h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khare, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Quantitative reactivity profiling predicts functional cysteines in proteomes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">790</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.1038/nature09472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnature09472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=21085121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=790-795&author=E.+Weerapanaauthor=C.+Wangauthor=G.+M.+Simonauthor=F.+Richterauthor=S.+Khareauthor=M.+B.+D.+Dillonauthor=D.+A.+Bachovchinauthor=K.+Mowenauthor=D.+Bakerauthor=B.+F.+Cravatt&title=Quantitative+reactivity+profiling+predicts+functional+cysteines+in+proteomes&doi=10.1038%2Fnature09472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative reactivity profiling predicts functional cysteines in proteomes</span></div><div class="casAuthors">Weerapana, Eranthie; Wang, Chu; Simon, Gabriel M.; Richter, Florian; Khare, Sagar; Dillon, Myles B. D.; Bachovchin, Daniel A.; Mowen, Kerri; Baker, David; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7325</span>),
    <span class="NLM_cas:pages">790-795</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cysteine is the most intrinsically nucleophilic amino acid in proteins, where its reactivity is tuned to perform diverse biochem. functions.  The absence of a consensus sequence that defines functional cysteines in proteins has hindered their discovery and characterization.  Here we describe a proteomics method to profile quant. the intrinsic reactivity of cysteine residues en masse directly in native biol. systems.  Hyper-reactivity was a rare feature among cysteines and it was found to specify a wide range of activities, including nucleophilic and reductive catalysis and sites of oxidative modification.  Hyper-reactive cysteines were identified in several proteins of uncharacterized function, including a residue conserved across eukaryotic phylogeny that we show is required for yeast viability and is involved in iron-sulfur protein biogenesis.  We also demonstrate that quant. reactivity profiling can form the basis for screening and functional assignment of cysteines in computationally designed proteins, where it discriminated catalytically active from inactive cysteine hydrolase designs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe4hHN7L4cGbVg90H21EOLACvtfcHk0lhiTXaUVYY9ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr%252FI&md5=277e436e7a8f4dfff0b548bce52ac40d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnature09472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09472%26sid%3Dliteratum%253Aachs%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DRichter%26aufirst%3DF.%26aulast%3DKhare%26aufirst%3DS.%26aulast%3DDillon%26aufirst%3DM.%2BB.%2BD.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DMowen%26aufirst%3DK.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DQuantitative%2520reactivity%2520profiling%2520predicts%2520functional%2520cysteines%2520in%2520proteomes%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D790%26epage%3D795%26doi%3D10.1038%2Fnature09472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Páez, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teijaro, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolan, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide covalent ligand discovery in native biological systems</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>534</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nature18002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnature18002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=27309814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=570-574&author=K.+M.+Backusauthor=B.+E.+Correiaauthor=K.+M.+Lumauthor=S.+Forliauthor=B.+D.+Horningauthor=G.+E.+Gonz%C3%A1lez-P%C3%A1ezauthor=S.+Chatterjeeauthor=B.+R.+Lanningauthor=J.+R.+Teijaroauthor=A.+J.+Olsonauthor=D.+W.+Wolanauthor=B.+F.+Cravatt&title=Proteome-wide+covalent+ligand+discovery+in+native+biological+systems&doi=10.1038%2Fnature18002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide covalent ligand discovery in native biological systems</span></div><div class="casAuthors">Backus, Keriann M.; Correia, Bruno E.; Lum, Kenneth M.; Forli, Stefano; Horning, Benjamin D.; Gonzalez-Paez, Gonzalo E.; Chatterjee, Sandip; Lanning, Bryan R.; Teijaro, John R.; Olson, Arthur J.; Wolan, Dennis W.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7608</span>),
    <span class="NLM_cas:pages">570-574</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small mols. are powerful tools for investigating protein function and can serve as leads for new therapeutics.  Most human proteins, however, lack small-mol. ligands, and entire protein classes are considered 'undruggable'.  Fragment-based ligand discovery can identify small-mol. probes for proteins that have proven difficult to target using high-throughput screening of complex compd. libraries.  Although reversibly binding ligands are commonly pursued, covalent fragments provide an alternative route to small-mol. probes, including those that can access regions of proteins that are difficult to target through binding affinity alone.  Here we report a quant. anal. of cysteine-reactive small-mol. fragments screened against thousands of proteins in human proteomes and cells.  Covalent ligands were identified for >700 cysteines found in both druggable proteins and proteins deficient in chem. probes, including transcription factors, adaptor/scaffolding proteins, and uncharacterized proteins.  Among the atypical ligand-protein interactions discovered were compds. that react preferentially with pro- (inactive) caspases.  We used these ligands to distinguish extrinsic apoptosis pathways in human cell lines vs. primary human T cells, showing that the former is largely mediated by caspase-8 while the latter depends on both caspase-8 and -10.  Fragment-based covalent ligand discovery provides a greatly expanded portrait of the ligandable proteome and furnishes compds. that can illuminate protein functions in native biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgNJ88KALeSLVg90H21EOLACvtfcHk0lhiTXaUVYY9ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN&md5=6fab087c042de735f8a5931e9f5b946c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature18002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18002%26sid%3Dliteratum%253Aachs%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DGonz%25C3%25A1lez-P%25C3%25A1ez%26aufirst%3DG.%2BE.%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DTeijaro%26aufirst%3DJ.%2BR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProteome-wide%2520covalent%2520ligand%2520discovery%2520in%2520native%2520biological%2520systems%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D570%26epage%3D574%26doi%3D10.1038%2Fnature18002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelkar, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravikumar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehendale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mhetre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajendran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrapani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, S. S.</span></span> <span> </span><span class="NLM_article-title">A chemical–genetic screen identifies ABHD12 as an oxidized-phosphatidylserine lipase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0195-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fs41589-018-0195-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=30643283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFSgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=169-178&author=D.+S.+Kelkarauthor=G.+Ravikumarauthor=N.+Mehendaleauthor=S.+Singhauthor=A.+Joshiauthor=A.+K.+Sharmaauthor=A.+Mhetreauthor=A.+Rajendranauthor=H.+Chakrapaniauthor=S.+S.+Kamat&title=A+chemical%E2%80%93genetic+screen+identifies+ABHD12+as+an+oxidized-phosphatidylserine+lipase&doi=10.1038%2Fs41589-018-0195-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical-genetic screen identifies ABHD12 as an oxidized-phosphatidylserine lipase</span></div><div class="casAuthors">Kelkar, Dhanashree S.; Ravikumar, Govindan; Mehendale, Neelay; Singh, Shubham; Joshi, Alaumy; Sharma, Ajay Kumar; Mhetre, Amol; Rajendran, Abinaya; Chakrapani, Harinath; Kamat, Siddhesh S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-178</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Reactive oxygen species (ROS) are transient, highly reactive intermediates or byproducts produced during oxygen metab.  However, when innate mechanisms are unable to cope with sequestration of surplus ROS, oxidative stress results, in which excess ROS damage biomols.  Oxidized phosphatidylserine (PS), a proapoptotic 'eat me' signal, is produced in response to elevated ROS, yet little is known regarding its chem. compn. and metab.  Here, we report a small mol. that generates ROS in different mammalian cells.  We used this mol. to detect, characterize and study oxidized PS in mammalian cells.  We developed a chem.-genetic screen to identify enzymes that regulate oxidized PS in mammalian cells and found that the lipase ABHD12 hydrolyzes oxidized PS.  We validated these findings in different physiol. settings including primary peritoneal macrophages and brains from Abhd12-/- mice under inflammatory stress, and in the process, we functionally annotated an enzyme regulating oxidized PS in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3D9DCwfyBk7Vg90H21EOLACvtfcHk0lhiTXaUVYY9ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFSgtb4%253D&md5=51d88dc955559211b9250221eb88370d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0195-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0195-0%26sid%3Dliteratum%253Aachs%26aulast%3DKelkar%26aufirst%3DD.%2BS.%26aulast%3DRavikumar%26aufirst%3DG.%26aulast%3DMehendale%26aufirst%3DN.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DJoshi%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DA.%2BK.%26aulast%3DMhetre%26aufirst%3DA.%26aulast%3DRajendran%26aufirst%3DA.%26aulast%3DChakrapani%26aufirst%3DH.%26aulast%3DKamat%26aufirst%3DS.%2BS.%26atitle%3DA%2520chemical%25E2%2580%2593genetic%2520screen%2520identifies%2520ABHD12%2520as%2520an%2520oxidized-phosphatidylserine%2520lipase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D169%26epage%3D178%26doi%3D10.1038%2Fs41589-018-0195-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P. R.</span></span> <span> </span><span class="NLM_article-title">The multiple antibiotic resistance regulator MarR is a copper sensor in Escherichia coli</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">21</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio.1380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=24185215" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=21&author=Z.+Haoauthor=H.+Louauthor=R.+Zhuauthor=J.+Zhuauthor=D.+Zhangauthor=B.+S.+Zhaoauthor=S.+Zengauthor=X.+Chenauthor=J.+Chanauthor=C.+Heauthor=P.+R.+Chen&title=The+multiple+antibiotic+resistance+regulator+MarR+is+a+copper+sensor+in+Escherichia+coli&doi=10.1038%2Fnchembio.1380"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1380%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DZ.%26aulast%3DLou%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DR.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DB.%2BS.%26aulast%3DZeng%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DP.%2BR.%26atitle%3DThe%2520multiple%2520antibiotic%2520resistance%2520regulator%2520MarR%2520is%2520a%2520copper%2520sensor%2520in%2520Escherichia%2520coli%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D10%26spage%3D21%26doi%3D10.1038%2Fnchembio.1380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alekshun, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mealy, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seaton, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head, J. F.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of MarR, a regulator of multiple antibiotic resistance, at 2.3 Å resolution</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">710</span>, <span class="refDoi"> DOI: 10.1038/90429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2F90429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=11473263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1OmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=710&author=M.+N.+Alekshunauthor=S.+B.+Levyauthor=T.+R.+Mealyauthor=B.+A.+Seatonauthor=J.+F.+Head&title=The+crystal+structure+of+MarR%2C+a+regulator+of+multiple+antibiotic+resistance%2C+at+2.3+%C3%85+resolution&doi=10.1038%2F90429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of MarR, a regulator of multiple antibiotic resistance, at 2.3 Å resolution</span></div><div class="casAuthors">Alekshun, Michael N.; Levy, Stuart B.; Mealy, Tanya R.; Seaton, Barbara A.; Head, James F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">710-714</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">MarR is a regulator of multiple antibiotic resistance in Escherichia coli.  It is the prototypical member of the MarR family of regulatory proteins found in bacteria and archaea that play important roles in the development of antibiotic resistance, a global health problem.  Here we describe the crystal structure of the MarR protein, detd. at a resoln. of 2.3 Å.  This is the first reported crystal structure of a member of this newly-described protein family.  The structure shows MarR as a dimer with each subunit contg. a winged-helix DNA binding motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtm41LVqIrZbVg90H21EOLACvtfcHk0ljGOzOZDwbFkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1OmsL4%253D&md5=766fc05d4ed4fec8721803904f9c2b7d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2F90429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F90429%26sid%3Dliteratum%253Aachs%26aulast%3DAlekshun%26aufirst%3DM.%2BN.%26aulast%3DLevy%26aufirst%3DS.%2BB.%26aulast%3DMealy%26aufirst%3DT.%2BR.%26aulast%3DSeaton%26aufirst%3DB.%2BA.%26aulast%3DHead%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520crystal%2520structure%2520of%2520MarR%252C%2520a%2520regulator%2520of%2520multiple%2520antibiotic%2520resistance%252C%2520at%25202.3%2520%25C3%2585%2520resolution%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2001%26volume%3D8%26spage%3D710%26doi%3D10.1038%2F90429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Birukou, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schindler, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaatz, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, R. G.</span></span> <span> </span><span class="NLM_article-title">Structural mechanism of transcription regulation of the Staphylococcus aureus multidrug efflux operon mepRA by the MarR family repressor MepR</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2774</span>– <span class="NLM_lpage">2788</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1093%2Fnar%2Fgkt1215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=24293644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlOqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=2774-2788&author=I.+Birukouauthor=S.+M.+Seoauthor=B.+D.+Schindlerauthor=G.+W.+Kaatzauthor=R.+G.+Brennan&title=Structural+mechanism+of+transcription+regulation+of+the+Staphylococcus+aureus+multidrug+efflux+operon+mepRA+by+the+MarR+family+repressor+MepR&doi=10.1093%2Fnar%2Fgkt1215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism of transcription regulation of the Staphylococcus aureus multidrug efflux operon mepRA by the MarR family repressor MepR</span></div><div class="casAuthors">Birukou, Ivan; Seo, Susan M.; Schindler, Bryan D.; Kaatz, Glenn W.; Brennan, Richard G.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2774-2788</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The multidrug efflux pump MepA is a major contributor to multidrug resistance in Staphylococcus aureus.  MepR, a member of the multiple antibiotic resistance regulator (MarR) family, represses mepA and its own gene.  Here, we report the structure of a MepR-mepR operator complex.  Structural comparison of DNA-bound MepR with 'induced' apoMepR reveals the large conformational changes needed to allow the DNA-binding winged helix-turn-helix motifs to interact with the consecutive major and minor grooves of the GTTAG signature sequence.  Intriguingly, MepR makes no hydrogen bonds to major groove nucleobases.  Rather, recognition-helix residues Thr60, Gly61, Pro62 and Thr63 make sequence-specifying van der Waals contacts with the TTAG bases.  Removing these contacts dramatically affects MepR-DNA binding activity.  The wings insert into the flanking minor grooves, whereby residue Arg87, buttressed by Asp85, interacts with the O2 of T4 and O4' ribosyl oxygens of A23 and T4.  Mutating Asp85 and Arg87, both conserved throughout the MarR family, markedly affects MepR repressor activity.  The His14':Arg59 and Arg10':His35:Phe108 interaction networks stabilize the DNA-binding conformation of MepR thereby contributing significantly to its high affinity binding.  A structure-guided model of the MepR-mepA operator complex suggests that MepR dimers do not interact directly and cooperative binding is likely achieved by DNA-mediated allosteric effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL78XMJCN9JbVg90H21EOLACvtfcHk0ljGOzOZDwbFkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlOqs78%253D&md5=652d3926ddfa003b5744a01a07b61be1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1215%26sid%3Dliteratum%253Aachs%26aulast%3DBirukou%26aufirst%3DI.%26aulast%3DSeo%26aufirst%3DS.%2BM.%26aulast%3DSchindler%26aufirst%3DB.%2BD.%26aulast%3DKaatz%26aufirst%3DG.%2BW.%26aulast%3DBrennan%26aufirst%3DR.%2BG.%26atitle%3DStructural%2520mechanism%2520of%2520transcription%2520regulation%2520of%2520the%2520Staphylococcus%2520aureus%2520multidrug%2520efflux%2520operon%2520mepRA%2520by%2520the%2520MarR%2520family%2520repressor%2520MepR%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D2774%26epage%3D2788%26doi%3D10.1093%2Fnar%2Fgkt1215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manna, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the SarS protein from Staphylococcus aureus</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">4219</span>– <span class="NLM_lpage">4225</span>, <span class="refDoi"> DOI: 10.1128/jb.185.14.4219-4225.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1128%2Fjb.185.14.4219-4225.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=12837797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlsFahtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=2003&pages=4219-4225&author=R.+Liauthor=A.+C.+Mannaauthor=S.+Daiauthor=A.+L.+Cheungauthor=G.+Zhang&title=Crystal+structure+of+the+SarS+protein+from+Staphylococcus+aureus&doi=10.1128%2Fjb.185.14.4219-4225.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the SarS protein from Staphylococcus aureus</span></div><div class="casAuthors">Li, Ronggui; Manna, Adhar C.; Dai, Shaodong; Cheung, Ambrose L.; Zhang, Gongyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4219-4225</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The expression of virulence determinants in Staphylococcus aureus is controlled by global regulatory loci (e.g., sarA and agr).  One of these determinants, protein A (spa), is activated by sarS, which encodes a 250-residue DNA-binding protein.  Genetic anal. indicated that the agr locus likely mediates spa repression by suppressing the transcription of sarS.  Contrary to SarA and SarR, which require homodimer formation for proper function, SarS is unusual within the SarA protein family in that it contains two homologous halves, with each half sharing sequence similarity to SarA and SarR.  Here we report the 2.2 Å resoln. x-ray crystal structure of the SarS protein.  SarS has folds similar to those of SarR and, quite plausibly, the native SarA structure.  Two typical winged-helix DNA-binding domains are connected by a well-ordered loop.  The interactions between the two domains are extensive and conserved.  The putative DNA-binding surface is highly pos. charged.  In contrast, neg. charged patches are located opposite to the DNA-binding surface.  Furthermore, sequence alignment and structural comparison revealed that MarR has folds similar to those of SarR and SarS.  Members of the MarR protein family have previously been implicated in the neg. regulation of an efflux pump involved in multiple antibiotic resistance in many gram-neg. species.  We propose that MarR also belongs to the winged-helix protein family and has a similar mode of DNA binding as SarR and SarS and possibly the entire SarA protein family member.  Based on the structural differences of SarR, SarS, and MarR, we further classified these winged-helix proteins to three subfamilies, SarA, SarS, and MarR.  Finally, a possible transcription regulation mechanism is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeHARYRHKRJrVg90H21EOLACvtfcHk0ljGOzOZDwbFkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlsFahtr8%253D&md5=1a6331c46d063984ebdc37c67fadbe43</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2Fjb.185.14.4219-4225.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.185.14.4219-4225.2003%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DR.%26aulast%3DManna%26aufirst%3DA.%2BC.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DA.%2BL.%26aulast%3DZhang%26aufirst%3DG.%26atitle%3DCrystal%2520structure%2520of%2520the%2520SarS%2520protein%2520from%2520Staphylococcus%2520aureus%26jtitle%3DJ.%2520Bacteriol.%26date%3D2003%26volume%3D185%26spage%3D4219%26epage%3D4225%26doi%3D10.1128%2Fjb.185.14.4219-4225.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duguid, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneewind, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span> <span> </span><span class="NLM_article-title">An oxidation-sensing mechanism is used by the global regulator MgrA in Staphylococcus aureus</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1038/nchembio820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1038%2Fnchembio820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=16980961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtV2gsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=591-595&author=P.+R.+Chenauthor=T.+Baeauthor=W.+A.+Williamsauthor=E.+M.+Duguidauthor=P.+A.+Riceauthor=O.+Schneewindauthor=C.+He&title=An+oxidation-sensing+mechanism+is+used+by+the+global+regulator+MgrA+in+Staphylococcus+aureus&doi=10.1038%2Fnchembio820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">An oxidation-sensing mechanism is used by the global regulator MgrA in Staphylococcus aureus</span></div><div class="casAuthors">Chen, Peng R.; Bae, Taeok; Williams, Wade A.; Duguid, Erica M.; Rice, Phoebe A.; Schneewind, Olaf; He, Chuan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">591-595</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Staphylococcus aureus is a human pathogen responsible for most wound and hospital-acquired infections.  The protein MgrA is both an important virulence determinant during infection and a regulator of antibiotic resistance in S. aureus.  The crystal structure of the MgrA homodimer, solved at 2.86 Å, indicates the presence of a unique cysteine residue located at the interface of the protein dimer.  We discovered that this cysteine residue can be oxidized by various reactive oxygen species, such as hydrogen peroxide and org. hydroperoxide.  Cysteine oxidn. leads to dissocn. of MgrA from DNA and initiation of signaling pathways that turn on antibiotic resistance in S. aureus.  The oxidn.-sensing mechanism is typically used by bacteria to counter challenges of reactive oxygen and nitrogen species.  Our study reveals that in S. aureus, MgrA adopts a similar mechanism but uses it to globally regulate different defensive pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohu2z8sVhB2rVg90H21EOLACvtfcHk0ljnDqyGBO_e1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtV2gsrzF&md5=8ab45df1e170c557f8506adeada6fa1a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio820%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%2BR.%26aulast%3DBae%26aufirst%3DT.%26aulast%3DWilliams%26aufirst%3DW.%2BA.%26aulast%3DDuguid%26aufirst%3DE.%2BM.%26aulast%3DRice%26aufirst%3DP.%2BA.%26aulast%3DSchneewind%26aufirst%3DO.%26aulast%3DHe%26aufirst%3DC.%26atitle%3DAn%2520oxidation-sensing%2520mechanism%2520is%2520used%2520by%2520the%2520global%2520regulator%2520MgrA%2520in%2520Staphylococcus%2520aureus%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D591%26epage%3D595%26doi%3D10.1038%2Fnchembio820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kahan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahan, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kropp, H.</span></span> <span> </span><span class="NLM_article-title">The mechanism of action of Fosfomycin (Phosphonomycin)</span>. <i>Ann. N.Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>235</i></span>,  <span class="NLM_fpage">364</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1111/j.1749-6632.1974.tb43277.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1111%2Fj.1749-6632.1974.tb43277.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=4605290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADyaE2cXkvVKnsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1974&pages=364-386&author=F.+M.+Kahanauthor=J.+S.+Kahanauthor=P.+J.+Cassidyauthor=H.+Kropp&title=The+mechanism+of+action+of+Fosfomycin+%28Phosphonomycin%29&doi=10.1111%2Fj.1749-6632.1974.tb43277.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of action of fosfomycin (phosphonomycin)</span></div><div class="casAuthors">Kahan, Frederick M.; Kahan, Jean S.; Cassidy, Patrick J.; Kropp, Helmut</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">364-86</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    </div><div class="casAbstract">Fosfomycin (I) [23155-02-4] killed bacteria by blocking cell wall synthesis, specifically by inhibiting phosphoenolpyruvate: UDP-N-acetylglucosamine-3-enolpyruvyl transferase [9023-27-2].  I, which reacted covalently with a cysteinyl residue of the enzyme, mimicked the natural substrate, phosphoenolpyruvate.  I utilized the L-α-glycerophosphate transport system as the sole method of entry in almost all sensitive bacteria, and loss of this transport system was the primary mechanism of resistant mutants.  A second pathway of antibiotic accumulation involved the inducible hexose phosphate uptake system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqelIVHQfVWMbVg90H21EOLACvtfcHk0ljnDqyGBO_e1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXkvVKnsLY%253D&md5=b27030810bde1e63ada6ee00bab30bcf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fj.1749-6632.1974.tb43277.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1749-6632.1974.tb43277.x%26sid%3Dliteratum%253Aachs%26aulast%3DKahan%26aufirst%3DF.%2BM.%26aulast%3DKahan%26aufirst%3DJ.%2BS.%26aulast%3DCassidy%26aufirst%3DP.%2BJ.%26aulast%3DKropp%26aufirst%3DH.%26atitle%3DThe%2520mechanism%2520of%2520action%2520of%2520Fosfomycin%2520%2528Phosphonomycin%2529%26jtitle%3DAnn.%2520N.Y.%2520Acad.%2520Sci.%26date%3D1974%26volume%3D235%26spage%3D364%26epage%3D386%26doi%3D10.1111%2Fj.1749-6632.1974.tb43277.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aye, Y.</span></span> <span> </span><span class="NLM_article-title">Privileged electrophile sensors: A resource for covalent drug development</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1016%2Fj.chembiol.2017.05.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=28648380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=787-800&author=M.+J.+C.+Longauthor=Y.+Aye&title=Privileged+electrophile+sensors%3A+A+resource+for+covalent+drug+development&doi=10.1016%2Fj.chembiol.2017.05.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged Electrophile Sensors: A Resource for Covalent Drug Development</span></div><div class="casAuthors">Long, Marcus John Curtis; Aye, Yimon</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">787-800</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">This Perspective delineates how redox signaling affects the activity of specific enzyme isoforms and how this property may be harnessed for rational drug design.  Covalent drugs have resurged in recent years and several reports have extolled the general virtues of developing irreversible inhibitors.  Indeed, many modern pharmaceuticals contain electrophilic appendages.  Several invoke a warhead that hijacks active-site nucleophiles whereas others take advantage of spectator nucleophilic side chains that do not participate in enzymic chem., but are poised to bind/react with electrophiles.  The latest data suggest that innate electrophile sensing-which enables rapid reaction with an endogenous signaling electrophile-is a quintessential resource for the development of covalent drugs.  For instance, based on recent work documenting isoform-specific electrophile sensing, isoenzyme non-specific drugs may be converted to isoenzyme-specific analogs by hijacking privileged first-responder electrophile-sensing cysteines.  Because this approach targets functionally relevant cysteines, we can simultaneously harness previously untapped moonlighting roles of enzymes linked to redox sensing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWQ9w0b7pay7Vg90H21EOLACvtfcHk0lhAiai369mYkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtr3E&md5=b7edc384ac48df7465303f33f7b1cca8</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.023%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BJ.%2BC.%26aulast%3DAye%26aufirst%3DY.%26atitle%3DPrivileged%2520electrophile%2520sensors%253A%2520A%2520resource%2520for%2520covalent%2520drug%2520development%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D787%26epage%3D800%26doi%3D10.1016%2Fj.chembiol.2017.05.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gioia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauville, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricci, A.</span></span> <span> </span><span class="NLM_article-title">Organocatalytic asymmetric Diels–Alder reactions of 3-vinylindoles</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">9236</span>– <span class="NLM_lpage">9239</span>, <span class="refDoi"> DOI: 10.1002/anie.200804275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=10.1002%2Fanie.200804275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVygs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=9236-9239&author=C.+Gioiaauthor=A.+Hauvilleauthor=L.+Bernardiauthor=F.+Finiauthor=A.+Ricci&title=Organocatalytic+asymmetric+Diels%E2%80%93Alder+reactions+of+3-vinylindoles&doi=10.1002%2Fanie.200804275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Organocatalytic asymmetric diels-alder reactions of 3-vinylindoles</span></div><div class="casAuthors">Gioia, Claudio; Hauville, Agnes; Bernardi, Luca; Fini, Francesco; Ricci, Alfredo</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">9236-9239</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A bifunctional org. catalyst based on the thiourea motif is able to coordinate both diene and dienophile in an unprecedented asym. Diels-Alder reaction of 3-vinylindole derivs., giving a rapid access to optically active tetra- and hexahydrocarbazoles, e.g. I, with excellent results in terms of yields, diastereoselectivities, and enantioselectivities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouMljMVhEoJLVg90H21EOLACvtfcHk0lgLbWrL3h2fdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVygs7%252FI&md5=004af59760dc1b44a1c29a445d8bd838</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fanie.200804275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200804275%26sid%3Dliteratum%253Aachs%26aulast%3DGioia%26aufirst%3DC.%26aulast%3DHauville%26aufirst%3DA.%26aulast%3DBernardi%26aufirst%3DL.%26aulast%3DFini%26aufirst%3DF.%26aulast%3DRicci%26aufirst%3DA.%26atitle%3DOrganocatalytic%2520asymmetric%2520Diels%25E2%2580%2593Alder%2520reactions%2520of%25203-vinylindoles%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D9236%26epage%3D9239%26doi%3D10.1002%2Fanie.200804275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i43"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00774">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_74608"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00774">10.1021/acs.jmedchem.9b00774</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Preparative procedures, characterization data, and protocols (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Proteomics data table (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_001.pdf">jm9b00774_si_001.pdf (4.48 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_002.xlsx">jm9b00774_si_002.xlsx (659.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00774/suppl_file/jm9b00774_si_003.csv">jm9b00774_si_003.csv (2.05 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00774&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00774%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-14%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00774" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b12b33fa52440","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
