
<html lang="en"     class="pb-page"  data-request-id="cc68f49c-7f1a-4e79-ab0f-e06cb3340e2f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2018.61.issue-10;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b00080;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Ligand Desolvation Steers On-Rate and Impacts Drug Residence Time of Heat Shock Protein 90 (Hsp90) Inhibitors" /></meta><meta name="dc.Creator" content="Doris A.  Schuetz" /></meta><meta name="dc.Creator" content="Lars  Richter" /></meta><meta name="dc.Creator" content="Marta  Amaral" /></meta><meta name="dc.Creator" content="Melanie  Grandits" /></meta><meta name="dc.Creator" content="Ulrich  Grädler" /></meta><meta name="dc.Creator" content="Djordje  Musil" /></meta><meta name="dc.Creator" content="Hans-Peter  Buchstaller" /></meta><meta name="dc.Creator" content="Hans-Michael  Eggenweiler" /></meta><meta name="dc.Creator" content="Matthias  Frech" /></meta><meta name="dc.Creator" content="Gerhard F.  Ecker" /></meta><meta name="dc.Description" content="Residence time and more recently the association rate constant kon are increasingly acknowledged as important parameters for in vivo efficacy and safety of drugs. However, their broader considerati..." /></meta><meta name="Description" content="Residence time and more recently the association rate constant kon are increasingly acknowledged as important parameters for in vivo efficacy and safety of drugs. However, their broader considerati..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 27, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00080" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00080" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00080" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00080" /></link>
        
    
    

<title>Ligand Desolvation Steers On-Rate and Impacts Drug Residence Time of Heat Shock Protein 90 (Hsp90) Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00080" /></meta><meta property="og:title" content="Ligand Desolvation Steers On-Rate and Impacts Drug Residence Time of Heat Shock Protein 90 (Hsp90) Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0014.jpeg" /></meta><meta property="og:description" content="Residence time and more recently the association rate constant kon are increasingly acknowledged as important parameters for in vivo efficacy and safety of drugs. However, their broader consideration in drug development is limited by a lack of knowledge of how to optimize these parameters. In this study on a set of 176 heat shock protein 90 inhibitors, structure–kinetic relationships, X-ray crystallography, and molecular dynamics simulations were combined to retrieve a concrete scheme of how to rationally slow down on-rates. We discovered that an increased ligand desolvation barrier by introducing polar substituents resulted in a significant kon decrease. The slowdown was accomplished by introducing polar moieties to those parts of the ligand that point toward a hydrophobic cavity. We validated this scheme by increasing polarity of three Hsp90 inhibitors and observed a 9-, 13-, and 45-fold slowdown of on-rates and a 9-fold prolongation in residence time. This prolongation was driven by transition state destabilization rather than ground state stabilization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00080"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00080">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00080&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00080&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00080&amp;href=/doi/10.1021/acs.jmedchem.8b00080" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 4397-4411</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00060" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00081" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Ligand Desolvation Steers On-Rate and Impacts Drug Residence Time of Heat Shock Protein 90 (Hsp90) Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Doris A. Schuetz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Doris A. Schuetz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, University of Vienna, UZA 2, Althanstrasse 14, 1090 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Doris+A.++Schuetz">Doris A. Schuetz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8615-9554" title="Orcid link">http://orcid.org/0000-0001-8615-9554</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lars Richter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lars Richter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, University of Vienna, UZA 2, Althanstrasse 14, 1090 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lars++Richter">Lars Richter</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9417-9042" title="Orcid link">http://orcid.org/0000-0002-9417-9042</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marta Amaral</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marta Amaral</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</div><div class="loa-info-affiliations-info">Instituto de Biologia Experimental e Tecnológica, Oeiras, Portugal</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marta++Amaral">Marta Amaral</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Melanie Grandits</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Melanie Grandits</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, University of Vienna, UZA 2, Althanstrasse 14, 1090 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Melanie++Grandits">Melanie Grandits</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ulrich Grädler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulrich Grädler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulrich++Gr%C3%A4dler">Ulrich Grädler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Djordje Musil</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Djordje Musil</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Djordje++Musil">Djordje Musil</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hans-Peter Buchstaller</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hans-Peter Buchstaller</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Merck KGaA, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hans-Peter++Buchstaller">Hans-Peter Buchstaller</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hans-Michael Eggenweiler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hans-Michael Eggenweiler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Merck KGaA, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hans-Michael++Eggenweiler">Hans-Michael Eggenweiler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthias Frech</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthias Frech</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthias++Frech">Matthias Frech</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Gerhard F. Ecker</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gerhard F. Ecker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, University of Vienna, UZA 2, Althanstrasse 14, 1090 Vienna, Austria</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +43(1)427755110. E-mail: <a href="/cdn-cgi/l/email-protection#4a2d2f38222b382e642c642f29212f380a3f24233c232f642b29642b3e"><span class="__cf_email__" data-cfemail="9cfbf9eef4fdeef8b2fab2f9fff7f9eedce9f2f5eaf5f9b2fdffb2fde8">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gerhard+F.++Ecker">Gerhard F. Ecker</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4209-6883" title="Orcid link">http://orcid.org/0000-0003-4209-6883</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00080&amp;href=/doi/10.1021%2Facs.jmedchem.8b00080" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 4397–4411</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 27, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 January 2018</li><li><span class="item_label"><b>Published</b> online</span>27 April 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 May 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00080" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00080</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4397%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDoris%2BA.%2BSchuetz%252C%2BLars%2BRichter%252C%2BMarta%2BAmaral%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D10%26contentID%3Dacs.jmedchem.8b00080%26title%3DLigand%2BDesolvation%2BSteers%2BOn-Rate%2Band%2BImpacts%2BDrug%2BResidence%2BTime%2Bof%2BHeat%2BShock%2BProtein%2B90%2B%2528Hsp90%2529%2BInhibitors%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4411%26publicationDate%3DMay%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00080"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2230</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">21</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00080" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Ligand Desolvation Steers On-Rate and Impacts Drug Residence Time of Heat Shock Protein 90 (Hsp90) Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Doris&quot;,&quot;last_name&quot;:&quot;A. Schuetz&quot;},{&quot;first_name&quot;:&quot;Lars&quot;,&quot;last_name&quot;:&quot;Richter&quot;},{&quot;first_name&quot;:&quot;Marta&quot;,&quot;last_name&quot;:&quot;Amaral&quot;},{&quot;first_name&quot;:&quot;Melanie&quot;,&quot;last_name&quot;:&quot;Grandits&quot;},{&quot;first_name&quot;:&quot;Ulrich&quot;,&quot;last_name&quot;:&quot;Grädler&quot;},{&quot;first_name&quot;:&quot;Djordje&quot;,&quot;last_name&quot;:&quot;Musil&quot;},{&quot;first_name&quot;:&quot;Hans-Peter&quot;,&quot;last_name&quot;:&quot;Buchstaller&quot;},{&quot;first_name&quot;:&quot;Hans-Michael&quot;,&quot;last_name&quot;:&quot;Eggenweiler&quot;},{&quot;first_name&quot;:&quot;Matthias&quot;,&quot;last_name&quot;:&quot;Frech&quot;},{&quot;first_name&quot;:&quot;Gerhard&quot;,&quot;last_name&quot;:&quot;F. Ecker&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;4397-4411&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00080&quot;},&quot;abstract&quot;:&quot;Residence time and more recently the association rate constant kon are increasingly acknowledged as important parameters for in vivo efficacy and safety of drugs. However, their broader consideration in drug development is limited by a lack of knowledge of how to optimize these parameters. In this study on a set of 176 heat shock protein 90 inhibitors, structure–kinetic relationships, X-ray crystallography, and molecular dynamics simulations were combined to retrieve a concrete scheme of how to rationally slow down on-rates. We discovered that an increased ligand desolvation barrier by introducing polar substituents resulted in a significant kon decrease. The slowdown was accomplished by introducing polar moieties to those parts of the ligand that point toward a hydrophobic cavity. We validated this scheme by increasing polarity of three Hsp90 inhibitors and observed a 9-, 13-, and 45-fold slowdown of on-rates and a 9-fold prolongation in residence time. This prolongation was driven by transition state destab&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00080&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00080" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00080&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00080" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00080&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00080" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00080&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00080&amp;href=/doi/10.1021/acs.jmedchem.8b00080" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00080" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00080" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00080%26sid%3Dliteratum%253Aachs%26pmid%3D29701469%26genre%3Darticle%26aulast%3DSchuetz%26date%3D2018%26atitle%3DLigand%2BDesolvation%2BSteers%2BOn-Rate%2Band%2BImpacts%2BDrug%2BResidence%2BTime%2Bof%2BHeat%2BShock%2BProtein%2B90%2B%2528Hsp90%2529%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D10%26spage%3D4397%26epage%3D4411%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291750" title="Hydroxyls">Hydroxyls</a>,</li><li><a href="/action/doSearch?ConceptID=290976" title="Free energy">Free energy</a>,</li><li><a href="/action/doSearch?ConceptID=291120" title="Hydrophobicity">Hydrophobicity</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290905" title="Kinetics">Kinetics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/jmcmar.2018.61.issue-10/20180524/jmcmar.2018.61.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Residence time and more recently the association rate constant <i>k</i><sub>on</sub> are increasingly acknowledged as important parameters for in vivo efficacy and safety of drugs. However, their broader consideration in drug development is limited by a lack of knowledge of how to optimize these parameters. In this study on a set of 176 heat shock protein 90 inhibitors, structure–kinetic relationships, X-ray crystallography, and molecular dynamics simulations were combined to retrieve a concrete scheme of how to rationally slow down on-rates. We discovered that an increased ligand desolvation barrier by introducing polar substituents resulted in a significant <i>k</i><sub>on</sub> decrease. The slowdown was accomplished by introducing polar moieties to those parts of the ligand that point toward a hydrophobic cavity. We validated this scheme by increasing polarity of three Hsp90 inhibitors and observed a 9-, 13-, and 45-fold slowdown of on-rates and a 9-fold prolongation in residence time. This prolongation was driven by transition state destabilization rather than ground state stabilization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Early stage drug discovery programs are still mostly guided by binding affinity optimization. However, in recent years it has been recognized that the lifetime of the binary drug target complex (the drug residence time) is a key determinant for clinical success of drug candidates.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> On the basis of the relationship between kinetic parameters and affinity, the residence time can be increased by either ground state (GS) stabilization<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> or transition state (TS) destabilization (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9">(5−9)</a> According to the transition state theory,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> the TS represents the highest barrier during the binding event. The binding free energy (Δ<i>G</i><sub>D</sub>) relates to the energy difference between the unbound and bound state and represents the GS. Improved interactions in the drug receptor complex (ΔΔ<i>G</i><sub>D</sub>) stabilize the GS and thus can increase the Δ<i>G</i><sup>⧧</sup><sub>off</sub> and the residence time (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Destabilization of the TS by unfavorable interactions along the binding pathway (ΔΔ<i>G</i><sup>⧧</sup><sub>on</sub>) can likewise increase Δ<i>G</i><sub>off</sub> and the residence time (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Effect of ground state (GS) and transition state (TS) (de)stabilization on residence time shown as free energy profiles of the binding event of a drug (D) to its receptor (R) to form a drug–receptor complex (DR). Δ<i>G</i><sup>⧧</sup><sub>on</sub> relates to the energy barrier the drug has to overcome during binding. The sum of Δ<i>G</i><sup>⧧</sup><sub>on</sub> and |Δ<i>G</i><sub>D</sub>| is the energy barrier, Δ<i>G</i><sup>⧧</sup><sub>off</sub>, the drug has to overcome during the unbinding event. The equations for <i>K</i><sub>D</sub>, <i>k</i><sub>on</sub>, and <i>k</i><sub>off</sub> show the direct relation to their respective free energies Δ<i>G</i><sub>D</sub>, Δ<i>G</i><sup>⧧</sup><sub>on</sub>, and Δ<i>G</i><sup>⧧</sup><sub>off</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Lapatinib was the first drug approved by FDA, whose label includes long residence time as mechanism of action. Lapatinib is only 1.6-fold more affine than erlotinib, hence showing similar GS stabilization. However, lapatinib dissociates 70-fold slower from the epidermal growth factor receptor (EGFR). The slower dissociation derives from the considerably higher energy barrier (TS) along the binding pathway, which lapatinib has to overcome (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). As the energy barrier, and therefore the destabilization of the TS, is directly linked to <i>k</i><sub>on</sub>, lapatinib shows a 43-fold slower on-rate.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> According to the principle of microscopic reversibility, barriers along the unbinding pathway are the same as on the binding pathway. Hence a better molecular understanding of the physicochemical features determining <i>k</i><sub>on</sub> will eventually contribute to a more detailed model of the dissociation process.</div><div class="NLM_p">The importance of <i>k</i><sub>on</sub> has been neglected for a long time, as it was considered to be invariant within lead-like series. Furthermore, any limitations could in theory be overcome by dosing at higher concentrations. Within the public–private partnership K4DD (Kinetics for Drug Discovery)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> so far 387 drug-like compounds have been tested against pharmaceutically relevant targets resulting in 847 sets of <i>k</i><sub>on</sub>, <i>k</i><sub>off</sub>, and <i>K</i><sub>D</sub> values.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Interestingly, the variance in affinities and on-rates was nearly identical. Both parameters cover 7 orders of magnitude, which is in good agreement with Schoop and Dey’s observation of an overall on-rate range of 10<sup>2</sup>–10<sup>8</sup> M<sup>–1</sup> s<sup>–1</sup>.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">Apart from its contribution to residence time, <i>k</i><sub>on</sub> itself is emerging as a key parameter in drug design. Several studies have indicated that <i>k</i><sub>on</sub> can be equally important for target occupancy as <i>k</i><sub>off</sub>.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Most recently Sykes et al. published the indisputable clinical relevance of <i>k</i><sub>on</sub>. Measured on- and off-rates for antipsychotic D<sub>2</sub> receptor antagonists showed a significant relation between <i>k</i><sub>on</sub> and extrapyramidal side effects (EPS) in the clinic. Interestingly, no correlation between <i>k</i><sub>off</sub> and EPS was found.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Furthermore, fast <i>k</i><sub>on</sub> has been shown to be of critical importance for the efficacy of the anticoagulants ximelagatran and dabigatran,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> which is related to their rapid onset of action. Moreover, Walkup et al. introduced a mechanistic PD model, which relies on <i>k</i><sub>off</sub> as well as <i>k</i><sub>on</sub> values.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_p">These findings ask for a deeper understanding of structure–kinetic relationship (SKR) in order to rationally optimize the on-rate. One great obstacle for a detailed analysis is the nature of the elusive, not directly observable TS that determines association kinetics.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In this context, molecular dynamics (MD) simulations are used to gain information about energy barriers during the binding event. These simulations have identified energetically costly conformational changes of the protein, including loop–helix conversions, during the binding event.<a onclick="showRef(event, 'ref5 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref5 ref21 ref22 ref23">(5,21−23)</a> MD studies have also identified desolvation effects as a critical barrier in the binding event. Dror et al.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> reported hydrophobic ligand and receptor desolvation of the extracellular vestibule of the β<sub>2</sub>-adrenergic receptor as the main rate-limiting step in the ligand–receptor association. In another comprehensive analysis, Spagnuolo et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> introduced steric clashes based on a TS model to successfully slow down the on-rate. In contrast to binding affinity data, which is abundantly accessible in the public domain, only limited sets of kinetic data from lead-like series are available for comprehensive SKR analysis.</div><div class="NLM_p last">To contribute to a better SKR understanding, we analyzed a large collection of 176 kinetically characterized inhibitors of the model protein Hsp90, generated within the K4DD consortium.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Those inhibitors were derived from three lead-like series, with X-ray resolved binding modes. From SKR analysis and MD simulations we inferred a polar desolvation scheme to rationally slow down on-rates. Following that scheme we chemically modified three ligands and in fact observed a 9-, 13-, and 45-fold slowdown in on-rates. Most interestingly, in addition to the slow on-rate, one compound showed a 9-fold prolongation in residence time, despite loss of affinity (GS destabilization).</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40518" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40518" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Hsp90 Inhibitor Data Set</h3><div class="NLM_p">The oncology target heat shock protein 90 (Hsp90) was an early example for the importance of binding kinetics for in vivo activity.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This target, among others, has been chosen by the K4DD consortium to shed light on the phenomenon of binding kinetics.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The N-terminal part of Hsp90 forms a flexible nucleotide binding site, which is in the focus of our study.</div><div class="NLM_p">In the framework of the K4DD project<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> we had the opportunity to work with a highly consistent data set of 176 Hsp90 inhibitors provided by Merck KGaA. In addition to SPR data, also eight newly resolved X-ray structures were available for structural insight. This comprises a unique data set, which offers the opportunity to systematically analyze molecular features driving binding kinetics. Chemistrywise the data set covers 73 indazole-, 74 resorcinol-, and 29 quinazoline-inhibitors. The indazole<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and resorcinol<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> scaffolds are displayed in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B. In general, the molecular scaffolds vary on two substitution sites, R<sup>1</sup> and R<sup>2</sup>. Cocrystals of the various scaffolds show good overlay of the R<sup>1</sup> and R<sup>2</sup> substitution sites (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S1 in Supporting Information</a>), with R<sup>1</sup> pointing into the hydrophobic binding cavity and R<sup>2</sup> pointing toward the hydrophilic pocket entrance (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Hsp90 inhibitor sets of two different scaffolds showing similar binding mode. (A) The indazole scaffold<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and (B) the resorcinol scaffold.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (C) Bound complexes of an indazole (<b>1g</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OCI">5OCI</a>) and (D) a resorcinol (<b>4i</b> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J86">5J86</a>) show the R<sup>1</sup> substituents embedded in the hydrophobic cavity and the R<sup>2</sup> substituents at the entrance of the protein binding site. R<sup>2</sup>-substitutions are facing toward the hydrophilic pocket entrance (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S4</a>). Hydrophobic areas are displayed in orange, hydrophilic areas in blue. (E) Kinetic map of the three data sets to show ranges of <i>k</i><sub>on</sub>, <i>k</i><sub>off</sub>, and <i>K</i><sub>D</sub>. <i>k</i><sub>on</sub> is plotted against <i>k</i><sub>off</sub>, displaying <i>K</i><sub>D</sub> in the diagonal. (F) Correlation between desolvation free energy of R<sup>1</sup> and R<sup>2</sup> substituents of the indazole and resorcinole data sets, and the log<sub>10</sub>-transformed on-rates of the “cavity-varying” data sets (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) show significant correlation, while “entrance-varying” indazoles (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) do not. <i>R</i> is the Pearson correlation coefficient (two-tailed <i>t</i> test).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Global SKR Analysis on the Hsp90 Inhibitor Data Set</h3><div class="NLM_p last">A set of four easily interpretable physicochemical descriptors was computed for the entire Hsp90 data set (<i>n</i> = 176) (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods in Supporting Information</a>). We conducted Pearson correlation analysis of the descriptor set against on-rates. The analysis exhibited a significant negative correlation for molecular weight (<i>R</i> = −0.64; <i>P</i> < 0.001) and polar surface area (<i>R</i> = −0.52; <i>P</i> < 0.001), whereas no correlation was observed for compound lipophilicity and molecular flexibility (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S2</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Comparative, Local SKR Analysis of “Cavity-Varying” and “Entrance-Varying” Inhibitors</h3><div class="NLM_p">We selected 13 indazoles (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and nine resorcinols (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), which vary only at one substitution site (R<sup>1</sup>). The substitution on R<sup>1</sup> points into the hydrophobic cavity, once the inhibitor is bound to Hsp90 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D). Local SKR analysis of these “cavity-varying” subsets revealed significant correlations of the on-rate with molecular weight (<i>R</i> = −0.76; <i>P</i> < 0.01) and polarity (<i>R</i> = −0.72; <i>P</i> < 0.01) for the indazoles (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S3</a>). Furthermore, we performed similar Pearson correlation analysis on 25 indazoles (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) possessing a varying substitution on R<sup>2</sup>- the site facing toward the hydrophilic pocket entrance (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D). In contrast to the “cavity-varying” subsets, these “entrance-varying” data set showed very poor correlation (<i>R</i> = −0.20; <i>P</i> = 0.33 for polarity and <i>R</i> = −0.37; <i>P</i> = 0.07 for molecular weight) between chosen descriptors and the on-rate (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S3</a>). The kinetic maps of these series can be found in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E. The negative correlation between polarity and the on-rate, observed within the “cavity-varying” subsets, prompted us to extend our descriptor set, employing desolvation free energy (Δ<i>G</i><sub>desolv</sub>) (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods</a>). Pearson correlation analysis between Δ<i>G</i><sub>desolv</sub> and the log<sub>10</sub>-transformed on-rate showed a strong discrepancy between the “cavity-varying” subsets (indazoles, <i>R</i> = −0.66, <i>P</i> < 0.05; resorcinols, <i>R</i> = −0.85, <i>P</i> < 0.01), on the one hand, and the “entrance-varying” indazoles (<i>R</i> = −0.10; <i>P</i> = 0.62) on the other hand (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F). The correlation results suggest that stripping off water molecules from polar ligand moieties while entering a hydrophobic cavity or traversing a hydrophobic entry pathway might present a considerable enthalpic barrier that slows down the on-rate.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Chemical and Biological Data for the “Cavity-Varying Indazoles”<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a><b>1a</b>–<b>j</b>, <b>2a</b>, <b>3a</b>,<b>b</b> (<i>n</i> = 13), Possessing a Variable R<sup>1</sup> Substitution and a Constant 4-(4-Morpholinyl)phenyl Substitution at R<sup>2</sup> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,C)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0006.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0007.gif" alt="" id="GRAPHIC-d113e892-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Calculated desolvation free energy (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supporting Information</a>). SPR measurements were performed on Biacore 4000 instruments (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). All experimental data are shown as the mean ± 1 s.d.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Resorcinol Data Set<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a><b>4a</b>–<b>i</b> (<i>n</i> = 9) Owning Variation in the R<sup>1</sup> Substitution and a Constant 2-Tolyl as a Substitution in R<sup>2</sup>, at the Hydrophilic Pocket Entrance (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,D)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0008.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0009.gif" alt="" id="GRAPHIC-d113e934-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Calculated desolvation free energy (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supporting Information</a>). SPR measurements were performed on Biacore 4000 instruments (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). All experimental data are shown as the mean ± 1 sd.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. “Entrance-Varying” Indazoles<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a><b>1g, 5a</b>–<b>x</b> (<i>n</i> = 25) Showing a 3-Methylbenzyl Substituent in R<sup>1</sup> and Various Substituents in R<sup>2</sup> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,C)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0010.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0011.gif" alt="" id="GRAPHIC-d113e976-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0012.gif" alt="" id="GRAPHIC-d113e977-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0013.gif" alt="" id="GRAPHIC-d113e978-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Calculated desolvation free energy (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supporting Information</a>). SPR measurements were performed on Biacore 4000 instruments (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). All experimental data are shown as the mean ± 1 sd.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Polar Desolvation Scheme Identifies Slowest Associating Inhibitor</h3><div class="NLM_p">To validate the desolvation scheme, we screened the Merck in-house chemical library for validation compounds. We extracted three indazoles carrying R<sup>1</sup> substituents, with gradually increasing desolvation barriers. Starting from compound <b>2a</b>, we selected its kinetically uncharacterized polar analogues <b>2b</b> and <b>2c</b> to complete a matched molecular triplet, the “five-membered ring triplet“ (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). In the same manner, starting from compounds <b>3a</b> and <b>3b</b>, we picked compound <b>3c</b> to constitute the “six-membered ring triplet” (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Subsequently, the respective desolvation free energy (Δ<i>G</i><sub>desolv</sub>) of those compounds was calculated (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods</a>) and SPR analysis was performed (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Polar desolvation scheme validated on two matched triplets. (A) Two matched triplets ordered by their R<sup>1</sup>-substituents’ desolvation free energy (Δ<i>G</i><sub>desolv</sub>). Structures in black are from the “cavity-varying” indazole data set (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Structures in red were prospectively selected for SPR measurements (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). (B) <i>k</i><sub>on</sub> value distribution of the five- and six-membered ring triplets and the overall Hsp90 data set. (C) Representative sensograms obtained by SPR for interaction between N-Hsp90 with compounds <b>2b</b>, <b>2c</b>, and <b>3c</b>. Fitted model traces are shown in black lines. (D) Overlay of <b>3b</b> and <b>3c</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNY">5LNY</a> and PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ODX">5ODX</a>) shows a concordant overlay of their indazole scaffolds and their respective pocket geometries. (E) Residence time distribution of the five- and six-membered ring triplets and the overall Hsp90 data set. (F) GS and TS contributions to compound residence times. Changes in ΔΔ<i>G</i><sub>D</sub> and ΔΔ<i>G</i><sup>⧧</sup><sub>on</sub> were computed relative to a reference compound with an affinity of 1000 nM and a <i>k</i><sub>on</sub> rate of 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup> according to Lu and Tonge.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Free energies have been calculated as shown in Schoop and Dey<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> from <i>K</i><sub>D</sub> and <i>k</i><sub>on</sub> values. GS and TS contributions are shown in blue and red, respectively. Statistical difference between the mean values was determined by a one-tailed unpaired Student’s <i>t</i> test: (∗) <i>P</i> < 0.05, (∗∗) <i>P</i> < 0.01, (∗∗∗) <i>P</i> < 0.001; error bars indicate ±1 sd.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Chemical and Biological Data for the Predicted Compounds <b>2b</b>, <b>2c</b>, <b>3c</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) and <b>6a</b>–<b>c</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cmp</th><th class="colsep0 rowsep0" align="center">Desolv. (kJ/mol)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>on</sub> (M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>off</sub> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub> (nM)</th><th class="colsep0 rowsep0" align="center">residence time (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2b</b></td><td class="colsep0 rowsep0" align="left">8.9</td><td class="colsep0 rowsep0" align="left">1.3 × 10<sup>05</sup> ± 6.2 × 10<sup>03</sup></td><td class="colsep0 rowsep0" align="left">2.4 × 10<sup>–03</sup> ± 7.9 × 10<sup>–05</sup></td><td class="colsep0 rowsep0" align="left">18.7 ± 0.5</td><td class="colsep0 rowsep0" align="left">6.9 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2c</b></td><td class="colsep0 rowsep0" align="left">11.4</td><td class="colsep0 rowsep0" align="left">1.7 × 10<sup>04</sup> ± 2.5 × 10<sup>03</sup></td><td class="colsep0 rowsep0" align="left">1.4 × 10<sup>–02</sup> ± 2.0 × 10<sup>–04</sup></td><td class="colsep0 rowsep0" align="left">82.3 ± 1.3</td><td class="colsep0 rowsep0" align="left">12.1 ± 1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3c</b></td><td class="colsep0 rowsep0" align="left">9.8</td><td class="colsep0 rowsep0" align="left">1.7 × 10<sup>03</sup> ± 7.4 × 10<sup>02</sup></td><td class="colsep0 rowsep0" align="left">3.0 × 10<sup>–04</sup> ± 1.5 × 10<sup>–05</sup></td><td class="colsep0 rowsep0" align="left">195 ± 78</td><td class="colsep0 rowsep0" align="left">56.7 ± 3.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">18.0</td><td class="colsep0 rowsep0" align="left">2.6 × 10<sup>05</sup> ± 2.2 × 10<sup>04</sup></td><td class="colsep0 rowsep0" align="left">5.5 × 10<sup>–01</sup> ± 5.3 × 10<sup>–02</sup></td><td class="colsep0 rowsep0" align="left">2,153 ± 23</td><td class="colsep0 rowsep0" align="left">0.03 ± 0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="left">22.4</td><td class="colsep0 rowsep0" align="left">5.8 × 10<sup>05</sup> ± 4.3 × 10<sup>04</sup></td><td class="colsep0 rowsep0" align="left">5.7 × 10<sup>–01</sup> ± 2.0 × 10<sup>–02</sup></td><td class="colsep0 rowsep0" align="left">999 ± 38</td><td class="colsep0 rowsep0" align="left">0.03 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">22.3</td><td class="colsep0 rowsep0" align="left">2.8 × 10<sup>04</sup> ± 1.2 × 10<sup>04</sup></td><td class="colsep0 rowsep0" align="left">5.5 × 10<sup>–01</sup> ± 1.5 × 10<sup>–01</sup></td><td class="colsep0 rowsep0" align="left">20205 ± 3146</td><td class="colsep0 rowsep0" align="left">0.03 ± 0.008</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Calculated desolvation free energy (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supporting Information</a>). SPR measurements were performed on a Biacore 4000 instrument from GE Healthcare. All experimental data are shown as the mean ± 1 sd.</p></div></div></div><div class="NLM_p">In both triplets we observed a stepwise reduction of on-rates (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), as the desolvation penalty of the R<sup>1</sup> substitution increased. In total, <i>k</i><sub>on</sub> decelerated stepwise 13-fold within the “five-membered ring triplet”. The most striking slowdown was the observed 45-fold deceleration within the “six-membered ring triplet”. Compound <b>3c</b> turned out to be the slowest of all 176 Hsp90 inhibitors in the entire data set, showing a <i>k</i><sub>on</sub> as slow as 1653 M<sup>–1</sup> s<sup>–1</sup> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B).</div><div class="NLM_p last">Slowdown of on-rates has frequently been related to structural protein adaptations during the binding event.<a onclick="showRef(event, 'ref5 ref21'); return false;" href="javascript:void(0);" class="ref ref5 ref21">(5,21)</a> Thus, we calculated the rmsd differences of the pocket shaping residues for the cocrystals of compound <b>3b</b> and its methoxy derivative <b>3c</b>. The pocket shapes turned out to be close to identical with a pocket atom rmsd of 0.3 Å (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D), which rules out that the change of 1 order of magnitude in association rate is related to different pocket rearrangements. The carbonyl oxygens in <b>3b</b> and <b>3c</b> show crystal water interaction, proposing unchanged solvation of the oxygen throughout the binding event (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S5 and Supporting Information</a>). This interaction was also found among other Hsp90 inhibitor cocrystals (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S6</a>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Nine-Fold Residence Time Slowdown Due to TS Destabilization</h3><div class="NLM_p">Within the “six-membered ring triplet“, we also observed a 9-fold decrease in residence time (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Similar to the on-rates, the residence times decreased stepwise with increased polar desolvation penalty. To investigate the influences of GS stabilization and TS destabilization on the residence times, we calculated the free energy values for <i>k</i><sub>on</sub> and <i>K</i><sub>D</sub>, respectively, according to Schoop and Dey<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). The results show that the increased residence time in <b>3c</b> was due to a TS destabilization, while the GS was unchanged.</div><div class="NLM_p last">Recently, Spagnuolo et al.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> described the discovery of a slow dissociating inhibitor class on InhA in which the slow residence time was also driven by TS destabilization (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Their study as well as our observations demonstrates the contribution of TS destabilization to a compound’s residence time. However, despite TS destabilization, we also observed invariant residence time when comparing compound <b>2a</b> to <b>2c</b>. In these cases, the destabilization of the TS was accompanied by a simultaneous destabilization of the GS, resulting in similar residence time (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Quinazoline Regioisomers Validate the Influence of a Hydrophobic Pocket Environment</h3><div class="NLM_p">For a further strengthening of the polar desolvation hypothesis, regioisomers with similar physicochemical properties were selected from the Merck in-house compound library. The new compound set comprised three quinazolines possessing local variation of their polar substituent in either R<sup>1</sup> or R<sup>2</sup> (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A for chemical structures of <b>6a</b>–<b>c</b>). On one derivative (<b>6b</b>) the hydroxyl group is attached to the quinazoline scaffold (R<sup>2</sup> substitution site); in the other (<b>6c</b>), the hydroxyl group is attached to the isoindoline moiety (R<sup>1</sup> substitution site). Assuming a common binding mode of quinazolines (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S7</a>), the hydroxyl group of <b>6b</b> points toward the hydrophilic pocket entrance and the hydroxyl group of <b>6c</b> points into the hydrophobic cavity. SPR measurements showed a <i>k</i><sub>on</sub> of 2.56 × 10<sup>5</sup> M<sup>–1</sup> s<sup>–1</sup> for the unsubstituted quinazoline <b>6a</b>. The additional hydroxyl group on the isoindoline in R<sup>1</sup> (compound <b>6c</b>) slowed down the association rate 9-fold (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, red chart). Despite sharing identical physicochemical properties with <b>6c</b>, compound <b>6b</b> showed a 2-fold acceleration of association rate (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, red chart and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). However, the slowdown of <i>k</i><sub>on</sub> comparing <b>6a</b> to <b>6c</b> did not influence the residence time of the compound pair (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, gray chart). The destabilization of the TS barrier was compensated by GS destabilization (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, blue chart), most probably due to the unfavorable placement of a hydroxyl group in a hydrophobic environment.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Quinazoline triplet <b>6a</b>, <b>6b</b>, and <b>6c</b> and their calculated desolvation free energies (Δ<i>G</i><sub>desolv</sub>). (B) X-ray structure of a quinazoline (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7">5OD7</a>) shows orientation of the isoindoline moiety (R<sup>1</sup>) into the hydrophobic cavity. The scaffold substitution R<sup>2</sup> is reaching toward the hydrophilic pocket entrance. (C) Bar charts display the <i>k</i><sub>on</sub> values (red), residence time (gray), and <i>K</i><sub>D</sub> (blue) for compounds <b>6a</b>, <b>6b</b>, and <b>6c</b>. (D) Potential of mean force (PMF) for <b>6a</b>, <b>6b</b>, and <b>6c</b> (black lines) and profile of number of water molecules (red lines) surrounding the hydroxyl group (within 3.6 Å) as a function of the distance ξ. The center-of-mass (COM) pulling simulation starts from the bound complex and terminates at 30 Å distance from the starting point (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods in Supporting Information</a>). Shadow areas around the free energy profile indicate statistical uncertainties. Red arrow indicates the highest energy barrier (TS). Statistical difference between the mean values was determined by a one-tailed unpaired Student’s <i>t</i> test: (∗∗) <i>P</i> < 0.01, (∗∗∗) <i>P</i> < 0.001; error bars indicate ±1 sd.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> MD Simulations Confirm a Polar Desolvation Barrier as the Rate-Limiting Step along the Binding Event</h3><div class="NLM_p">To get a better understanding about the barriers along the binding event, a set of MD simulations<a onclick="showRef(event, 'ref5 ref24 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref5 ref24 ref28 ref29 ref30 ref31">(5,24,28−31)</a> using the quinazoline <b>6a</b> (R<sup>1</sup> = H, R<sup>2</sup> = H) and two polar hydroxyl analogues <b>6b</b> (R<sup>1</sup> = H, R<sup>2</sup> = OH) and <b>6c</b> (R<sup>1</sup> = OH, R<sup>2</sup> = H) were performed. Starting coordinates were derived from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7">5OD7</a> (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods</a>). We chose a combination of steered MD simulations and umbrella sampling to determine the TS.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Employing steered MD simulations was one of the first attempts for detection of unbinding pathways in kinetic studies.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a></div><div class="NLM_p">The potential of mean force (PMF) was calculated, which discloses energy changes along the binding pathway (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D, <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supporting Information</a>). In addition, we examined the solvation state of the hydroxyl group of <b>6b</b> and <b>6c</b> during the simulation path (red line in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). While <b>6b</b> showed interactions of the hydroxyl group with at least two water molecules at all times, the hydroxyl group of <b>6c</b> had to strip off all its water molecules to enter the hydrophobic binding pocket.</div><div class="NLM_p">Starting from the bound complex of the quinazoline to Hsp90 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A), the simulations showed a distinct energy barrier at 6.2 Å. Compound <b>6c</b> shows the highest energy barrier along the pathway (red arrow in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D), which is in accordance with <b>6c</b> possessing 9-fold and 20-fold slower on-rate than <b>6a</b> and <b>6b</b>, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Compound <b>6c</b> is the only quinazoline with a hydroxyl substitution on the isoindoline moiety, which is embedded inside the hydrophobic pocket when bound. The high energy barrier detected using MD corresponds to the isoindoline moiety entering the hydrophobic binding pocket during the association process (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Putative binding pathway of <b>6c</b> derived from MD simulation. (A) Compound <b>6c</b> bound to the binding site (modeled from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7">5OD7</a>; see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods in Supporting Information</a>), the isoindoline fully embedded in the hydrophobic cavity. (B) The isoindoline has moved out of the cavity while the quinazoline scaffold is still “in place”. The hydroxyl group is fully desolvated to traverse the narrow, hydrophobic passage. (C) Compound <b>6c</b> has overcome the highest energy barrier (TS), and the hydroxyl group is solvated. Hydrophobic areas are displayed in orange, hydrophilic areas in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To put the TS (6.2 Å) into context, we compared it to a position of the ligand before entering the hydrophobic pocket. At this position (8.5 Å), <b>6c</b>’s hydroxyl group on the isoindoline shows hydrogen interactions with water molecules (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Consequently, <b>6c</b>’s hydroxyl moiety becomes desolvated upon entering the hydrophobic pocket (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, TS). We conclude that the disruption of hydrogen bonds between the hydroxyl group of <b>6c</b> and surrounding water molecules poses a considerable desolvation penalty. This penalty cannot be compensated by polar interactions of <b>6c</b> at the TS as it lacks interaction partners in the hydrophobic pocket (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Broken uncompensated water interactions typically render as enthalpic barriers in thermodynamic decomposition.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Hence, we assume that the increased energy barrier (Δ<i>G</i><sup>⧧</sup><sub>on</sub>) observed in <b>2c</b>, <b>3c</b>, and <b>6c</b> is of enthalpic nature. Our MD findings provide a mechanistic model for the observed <i>k</i><sub>on</sub> slowdown suggested by our SKR studies (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F). Desolvation has been recognized as an important factor influencing the kinetics of ligand binding.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> However, these studies focused on kinetic barriers due to water reorganization, including “pocket dewetting” upon ligand binding.<a onclick="showRef(event, 'ref24 ref32 ref37'); return false;" href="javascript:void(0);" class="ref ref24 ref32 ref37">(24,32,37)</a> In our study we aimed to increase our understanding for the <i>k</i><sub>on</sub> variance within a group of highly similar compounds, rather than a single binding event. In this respect, pocket desolvation does not provide a rationale for the large <i>k</i><sub>on</sub> derivations in the data set (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S8</a>). Our analysis led to the conclusion that enthalpic barriers due to polar ligand desolvation in an unfavorable hydrophobic pocket greatly impact the association kinetics. A literature search for enthalpic (Δ<i>H</i><sup>⧧</sup><sub>on</sub>) and entropic (−<i>T</i>Δ<i>S</i><sup>⧧</sup><sub>on</sub>) contributions to the association barrier Δ<i>G</i><sup>⧧</sup><sub>on</sub> identified experimental data on various drugs and drug-like molecules for the drug targets MAP38α,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> thrombin,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> gpH3 receptor,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> HIV1 protease,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and FGFR1<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Table S1</a>). Interestingly, in all those cases the enthalpic contribution to the association barrier was dominant.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66064" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66064" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, we demonstrate a guideline that introduces enthalpic desolvation penalties to rationally slow down the on-rate of drug like compounds binding to the hydrophobic pocket of the heat shock protein 90. We prospectively validated the scheme by introducing polar moieties to indazoles and quinazolines. In fact, their corresponding polar analogues showed a <i>k</i><sub>on</sub> slowdown by 13-, 45-, and 9-fold, respectively. One of the compounds turned out to be the slowest associating inhibitor among the 176 compounds encompassing Hsp90 data set. In addition, the introduced desolvation barrier, resembling a TS destabilization, led to a 9-fold prolongation in residence time, despite affinity loss.</div><div class="NLM_p last">These findings are of substantial interest, as hydrophobic areas constitute an integral part of druggable sites in proteins. To our knowledge, this is the first validated scheme to prospectively optimize <i>k</i><sub>on</sub> by chemical modification of drug-like molecules within a lead-like series. The work presented here provides guidance to chemists to optimize on-rates and therefore enables rational drug design of binding kinetics of drugs.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06732" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06732" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Molecular Dynamics Simulations</h3><div class="NLM_p">For this study, a center-of-mass (COM) pulling simulation followed by umbrella sampling was used to study three complex structures. The crystallographic structures of these complexes were obtained from Merck and modified using PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7">5OD7</a>, resolution of 1.62 Å (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods</a>). For the simulations, the GROMACS 5.0.4 package<a onclick="showRef(event, 'ref43 ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45 ref46">(43−46)</a> was used in combination with the GROMOS54a7 force field<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> to define the parameters for the protein and the compounds. The force field has proven successful in a number of studies, including studies to quantify protein–ligand interactions.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The parameters for the compounds were derived based on the analogy of functional groups of the force field (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Table S2</a>).<a onclick="showRef(event, 'ref49 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51">(49−51)</a> For the protein, the protonation state of charged amino acids was chosen to describe them at neutral pH. Therefore, Lys and Arg were protonated whereas Glu and Asp were deprotonated. The tautomeric form of His was selected based on a geometric criterion to allow for an optimal hydrogen bonding conformation.</div><div class="NLM_p last">The complexes were positioned in a rectangular box using periodic boundary conditions. For the box dimensions a minimum protein-wall distance was set (1.0 nm), and in the pulling direction, the pulling distance (2 times the pulling distance) was additionally taken into account in order to avoid periodicity artifacts. Sodium and chloride ions were added to gain a neutral system, and the simple point charge (SPC) water model<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> was used. The system was energy minimized with the steepest descent method, followed by a two-step equilibration cycle. The first step was performed in an <i>NVT</i> ensemble for 100 ps followed by a second equilibration step under constant pressure (<i>NpT</i>, 1 bar, 1 ns). For the regulation of the pressure the Berendsen barostat<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> was chosen with a coupling constant of 5.0 ps and the V-rescale temperature coupling<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> (0.5 ps) was used to maintain the temperature constant at 300 K. During these steps, the atoms of the complexes were position restrained. To constrain the covalent bonds, the LINCS algorithm<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> was used. For the nonbonded interactions the short-range cutoff was set to 1.2 nm and the particle mesh Ewald algorithm<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> was applied to calculate the electrostatic interactions beyond this point. For the COM pulling simulation the protein was kept rigid and the compound was pulled out of the active site with a pull rate of 0.001 nm·ps<sup>–1</sup> and a force constant of 1000 kJ mol<sup>–1</sup> nm<sup>–2</sup>. The pulling distance was set to 3.0 nm to ensure that at the end of the simulation the compound is not in contact with the protein anymore. The compound was pulled out of the protein along a single reaction coordinate (<i>y</i>-axis). For the pulling a harmonic potential was used. Umbrella sampling was used to find energy changes along the pathway of the compounds. To sample all relevant configurations along the pathway, the spacing between the sampling windows was set to approximately 0.06 nm. For each window an <i>NPT</i> equilibration step (100 ps) and an umbrella sampling simulation (10 ns) were performed. At least 470 ns of umbrella sampling simulations were performed for each compound to calculate the potential of mean force (PMF). The PMF was computed with the weighted histogram analysis method (WHAM)<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> by selecting a series of overlapping umbrella sampling simulations with a COM distance between 0.3 and 3.0 nm. The free energy was set to be zero at 0.3 nm, and the PMF curves were smoothed by applying the Savitzky–Golay filter<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> in R. Additionally the Bayesian bootstrapping (200 bootstraps) of complete histograms was used to determine the statistical uncertainties of the PMF.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Further analyses include the calculation of the amount of water molecules in the vicinity of the compounds (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S9</a>). Water molecules having the oxygen atom within a distance of 0.36 nm to all atoms of the compounds were taken into account. Additionally, the surrounding water molecules of the substituted hydroxyl group were determined.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Compound Synthesis</h3><div class="NLM_p last">Information on the synthesis of Merck chemical compounds is provided in WIPO (2006), WO2006/087077, WO 2008/003396, WO 2009/010139, WO 2010/066324, WO 2011/060873. Analytical data are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Surface Plasmon Resonance</h3><div class="NLM_p last">SPR measurements were performed on a Biacore 4000 instrument from GE Healthcare as previously described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Briefly, recombinant N-Hsp90 was immobilized on a Biacore CM5 chip at 25 °C at a flow rate of 10 μL/min using amine coupling at pH 4.50 according to Biacore’s standard protocol. HBS-N (10 mM HEPES, pH 7.40, 0.15 M NaCl) served as running buffer during immobilization, and all SPR binding kinetics measurements assays were performed in 20 mM HEPES, pH 7.50, 150 mM NaCl, 0.05% Tween 20, 1 mM DTT, 0.1 mM EDTA, 2% DMSO. Data sets were processed and analyzed using the Biacore 4000 Evaluation software, version 1.1. Solvent corrected and double-referenced association and dissociation phase data were fitted to a simple 1:1 interaction model with mass transport limitations.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <sup>1</sup>H NMR</h3><div class="NLM_p last"><sup>1</sup>H NMR spectra were recorded at 300 K unless otherwise specified using a Bruker Avance DPX 300, AV 400, DPX 500 spectrometer (TMS as an internal standard). <sup>1</sup>H NMR chemical shifts are reported in parts per million (ppm). <sup>1</sup>H NMR data are reported as chemical shift (dH), relative integral, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, dd = doublet of doublets, ddd = doublet of doublet of doublets, dt = doublet of triplets, td = triplet of doublets, tt = triplet of triplets, qd = quartet of doublets), and coupling constant (<i>J</i>, Hz). All of the compounds reported in the manuscript have a purity of ≥95% unless noted otherwise.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> HPLC/MS Conditions</h3><div class="NLM_p last">LC/MS spectra of the products were recorded on an Agilent 1100 HPLC system (1100 high pressure gradient pump, 1100 diode array detector) interfaced to an Agilent 1100 mass spectrometer detector using a Chromolith SpeedROD RP 18e50-4.6 column. Polar gradient: water (0.05% HCOOH) and acetonitrile (0.04% HCOOH) were used as eluent in mixtures as follows: 0 min, 4% ACN; 2.8 min, 100% ACN; 3.3 min, 100% ACN. Gradient: 5.5 min. Flow rate: 2.4 mL/min. UV detection: 220 nm. HPLC/MS spectra of the compounds were recorded on an Agilent 1100 HPLC system (column, Chromolith SpeedROD RP-18e, 50 × 4.6 mm<sup>2</sup>; wavelength, 220 nm) interfaced to an Agilent 1100 mass spectrometer detector (positive mode) using different eluent systems and gradients. For all compounds purity was above 95% and the expected molecular ion was confirmed with the LC coupled mass spectrometry.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Protein Expression and Purification</h3><div class="NLM_p last">A hexahistidine tagged N-terminal fragment of Hsp90 (9–236) (NP_005339) was expressed and purified by Instituto de Biologia Experimental e Tecnológica, according to previously described protocols.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Crystallization and Structure Determination</h3><div class="NLM_p last">The crystallization conditions are essentially the same as those described by Amaral et al.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and Buchstaller et al.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Data sets were collected at SLS synchotron and processed with the XDS software package.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The structures were solved by molecular replacement with either CNX<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> or BUSTER (<contrib-group>Bricogne et al.</contrib-group>  <cite><i>BUSTER</i></cite>, version 2.11.6; Global Phasing Ltd.: Cambridge, U.K., <span class="NLM_year">2016</span>). Model building was performed in Coot, with compounds and waters fitted into the initial |<i>F</i><sub>o</sub>| – |<i>F</i><sub>c</sub>| map. The coordinates of the new holo-structures of N-Hsp90 have been deposited in the RCSB Protein Data Bank. (Data collection and refinement statistics are to be found in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Table S3</a>.)</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> In Silico Pan Assay Interference Compounds (PAINS) Screen</h3><div class="NLM_p last">A KNIME<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> workflow was conducted to search for potential PAIN substructures in compounds <b>1a</b>–<b>6c</b>. An “anil_di_alk_A(478)” alert was found for compounds <b>1a</b>–<b>3c</b>, <b>5h</b>, <b>5m</b>, and <b>5p</b>. However, these analogues were considered as non-PAIN in this study as compounds show (i) a logical, consistent SAR, (ii) <i>k</i><sub>on</sub>, <i>k</i><sub>off</sub>, and <i>K</i><sub>D</sub> derived from detailed biophysical testing using surface plasmon resonance, and (iii) the availability of X-ray resolved bound complexes of analogues <b>1g</b>, <b>2b</b>, <b>3b</b>, and <b>3c</b>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00080" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15635" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15635" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00080" class="ext-link">10.1021/acs.jmedchem.8b00080</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">SKR analysis of complete Hsp90 data set; structural analysis of cocrystallized inhibitors (<b>2b</b>, <b>3b</b>, <b>3c</b>, <b>4i</b>, and three quinazolines); experimentally derived enthalpic and entropic contributions to the association barrier of drug-like molecules from literature; analytical data on <b>1a</b>–<b>6c</b>; supplementary methods; data collection and refinement statistic of X-ray structures (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Parameter settings for MD simulation (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf">jm8b00080_si_001.pdf (12.22 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_002.xlsx">jm8b00080_si_002.xlsx (26.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_003.csv">jm8b00080_si_003.csv (6.88 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates have been deposited in the ProteinData Bank (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OCI">5OCI</a> for <b>1g</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ODX">5ODX</a> for <b>3c</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNY">5LNY</a> for <b>3b</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NYH">5NYH</a> for <b>2b</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNZ">5LNZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7">5OD7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LO0">5LO0</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LO1">5LO1</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b00080" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63018" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63018" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gerhard F. Ecker</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, University of Vienna, UZA 2, Althanstrasse 14, 1090 Vienna, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4209-6883" title="Orcid link">http://orcid.org/0000-0003-4209-6883</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#187f7d6a70796a7c367e367d7b737d6a586d76716e717d36797b36796c"><span class="__cf_email__" data-cfemail="640301160c0516004a024a01070f011624110a0d120d014a05074a0510">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Doris A. Schuetz</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, University of Vienna, UZA 2, Althanstrasse 14, 1090 Vienna, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8615-9554" title="Orcid link">http://orcid.org/0000-0001-8615-9554</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lars Richter</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, University of Vienna, UZA 2, Althanstrasse 14, 1090 Vienna, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9417-9042" title="Orcid link">http://orcid.org/0000-0002-9417-9042</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marta Amaral</span> - <span class="hlFld-Affiliation affiliation">Discovery Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Instituto de Biologia Experimental e Tecnológica, Oeiras, Portugal</span>; 
    <span>Present Address:
                        M.A.: Sanofi-Aventis Deutschland GmbH, Biologics Research/Protein Therapeutics, Industriepark Höchst, 65926 Frankfurt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Melanie Grandits</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, University of Vienna, UZA 2, Althanstrasse 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulrich Grädler</span> - <span class="hlFld-Affiliation affiliation">Discovery Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Djordje Musil</span> - <span class="hlFld-Affiliation affiliation">Discovery Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hans-Peter Buchstaller</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Merck KGaA, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hans-Michael Eggenweiler</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Merck KGaA, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthias Frech</span> - <span class="hlFld-Affiliation affiliation">Discovery Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>D.A.S. and L.R. contributed equally to this work. D.A.S., L.R., and M.G. performed all computational studies. M.A. designed and performed SPR and X-ray studies. U.G. and D.M. solved selected crystal structures. H.-P.B. and H.-M.E. developed the Hsp90 inhibitors described in this article. G.F.E. and M.F. supervised the studies. All authors contributed to writing of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d113e2334-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27413" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27413" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the EU/EFPIA Innovative Medicines Initiative (IMI) Joint Undertaking, K4DD (Grant 1115366). This paper reflects only the authors’ views, and neither the IMI nor the European Commission is liable for any use that may be made of the information contained herein. The computational results presented have been achieved in part using the Vienna Scientific Cluster (VSC). D.A.S. thanks Bernhard Knasmueller for support with the K4DD database and Wilbert de Witte for help with in vivo models and useful discussions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">COM</td><td class="NLM_def"><p class="first last">center of mass</p></td></tr><tr><td class="NLM_term">GS</td><td class="NLM_def"><p class="first last">ground state</p></td></tr><tr><td class="NLM_term">Hsp90</td><td class="NLM_def"><p class="first last">heat shock protein 90</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">TS</td><td class="NLM_def"><p class="first last">transition state</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35355" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35355" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 63 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span> <span> </span><span class="NLM_article-title">Biochemical Mechanisms of Drug Action: What Does It Take for Success?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1038/nrd1500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnrd1500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=15340390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFCnsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=801-808&issue=9&author=D.+C.+Swinney&title=Biochemical+Mechanisms+of+Drug+Action%3A+What+Does+It+Take+for+Success%3F&doi=10.1038%2Fnrd1500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Biochemical mechanisms of drug action: what does it take for success?</span></div><div class="casAuthors">Swinney, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">801-808</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Drug discovery is extremely difficult.  There are many unanticipated scientific, medical and business challenges to every drug discovery program.  It is important to increase our understanding of the fundamental properties of effective drugs so that we can anticipate potential problems in developing new agents.  This article addresses potential drug discovery and development risks assocd. with the biochem. mechanism of drug action, and proposes simple rules to minimize these risks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXhyDgozFUdrVg90H21EOLACvtfcHk0ljKhRMVObH8GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFCnsL4%253D&md5=b06bdc6ace3168dcb011679b43b72aed</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1500%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26atitle%3DBiochemical%2520Mechanisms%2520of%2520Drug%2520Action%253A%2520What%2520Does%2520It%2520Take%2520for%2520Success%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26issue%3D9%26spage%3D801%26epage%3D808%26doi%3D10.1038%2Fnrd1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Residence Time and Its Implications for Lead Optimization</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">730</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&issue=9&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-Target+Residence+Time+and+Its+Implications+for+Lead+Optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0ljKhRMVObH8GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-Target%2520Residence%2520Time%2520and%2520Its%2520Implications%2520for%2520Lead%2520Optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26issue%3D9%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R.
A.</span></span> <span> </span><span class="NLM_article-title">The Drug-Target Residence Time Model: A 10-Year Retrospective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1038/nrd.2015.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnrd.2015.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=26678621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=87-95&issue=2&author=R.%0AA.+Copeland&title=The+Drug-Target+Residence+Time+Model%3A+A+10-Year+Retrospective&doi=10.1038%2Fnrd.2015.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-target residence time model: a 10-year retrospective</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">87-95</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The drug-target residence time model was first introduced in 2006 and has been broadly adopted across the chem. biol., biotechnol. and pharmaceutical communities.  While traditional in vitro methods view drug-target interactions exclusively in terms of equil. affinity, the residence time model takes into account the conformational dynamics of target macromols. that affect drug binding and dissocn.  The key tenet of this model is that the lifetime (or residence time) of the binary drug-target complex, and not the binding affinity per se, dictates much of the in vivo pharmacol. activity.  Here, this model is revisited and key applications of it over the past 10 years are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwNrhp9U8kwLVg90H21EOLACvtfcHk0ljKhRMVObH8GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE&md5=23f4a6973d77804292ff0af6c81358c0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.18%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520Drug-Target%2520Residence%2520Time%2520Model%253A%2520A%252010-Year%2520Retrospective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26issue%3D2%26spage%3D87%26epage%3D95%26doi%3D10.1038%2Fnrd.2015.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">The Use of Thermodynamic and Kinetic Data in Drug Discovery: Decisive Insight or Increasing the Puzzlement?</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">229</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201402521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2Fcmdc.201402521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=25537429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFejsrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=229-231&issue=2&author=G.+Klebe&title=The+Use+of+Thermodynamic+and+Kinetic+Data+in+Drug+Discovery%3A+Decisive+Insight+or+Increasing+the+Puzzlement%3F&doi=10.1002%2Fcmdc.201402521"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The Use of Thermodynamic and Kinetic Data in Drug Discovery: Decisive Insight or Increasing the Puzzlement?</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">229-231</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The prime property to rate the success of hit-to-lead-to-drug optimization in drug discovery is binding affinity.  Rational approaches try to relate this property with structure.  Affinity can be linked to the thermodn. property, Gibbs free energy of binding, which itself factorizes into enthalpy and entropy.  With respect to kinetic properties, affinity can be assocd. with the ratio of koff and kon of complex formation.  Do these features help to obtain better insight into affinity.  The present viewpoint assesses our current understanding of thermodn.- or kinetics-structure relationships and questions the accuracy of data collected to learn about the thermodn. and kinetic basis to comprehend affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS9VXbE-qOU7Vg90H21EOLACvtfcHk0ljKhRMVObH8GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFejsrfL&md5=315bae98f81db9dcbcd0a4bc4a5da4f9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201402521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201402521%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DThe%2520Use%2520of%2520Thermodynamic%2520and%2520Kinetic%2520Data%2520in%2520Drug%2520Discovery%253A%2520Decisive%2520Insight%2520or%2520Increasing%2520the%2520Puzzlement%253F%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26issue%3D2%26spage%3D229%26epage%3D231%26doi%3D10.1002%2Fcmdc.201402521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosemond, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure and Dynamics of the M3 Muscarinic Acetylcholine Receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>482</i></span> (<span class="NLM_issue">7386</span>),  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">556</span>, <span class="refDoi"> DOI: 10.1038/nature10867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnature10867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=22358844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC38Xis1eqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2012&pages=552-556&issue=7386&author=A.+C.+Kruseauthor=J.+Huauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=D.+M.+Rosenbaumauthor=E.+Rosemondauthor=H.+F.+Greenauthor=T.+Liuauthor=P.+S.+Chaeauthor=R.+O.+Drorauthor=D.+E.+Shawauthor=W.+I.+Weisauthor=J.+Wessauthor=B.+K.+Kobilka&title=Structure+and+Dynamics+of+the+M3+Muscarinic+Acetylcholine+Receptor&doi=10.1038%2Fnature10867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and dynamics of the M3 muscarinic acetylcholine receptor</span></div><div class="casAuthors">Kruse, Andrew C.; Hu, Jianxin; Pan, Albert C.; Arlow, Daniel H.; Rosenbaum, Daniel M.; Rosemond, Erica; Green, Hillary F.; Liu, Tong; Chae, Pil Seok; Dror, Ron O.; Shaw, David E.; Weis, William I.; Wess, Jurgen; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">7386</span>),
    <span class="NLM_cas:pages">552-556</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acetylcholine, the first neurotransmitter to be identified, exerts many of its physiol. actions via activation of a family of G-protein-coupled receptors (GPCRs) known as muscarinic acetylcholine receptors (mAChRs).  Although the five mAChR subtypes (M1-M5) share a high degree of sequence homol., they show pronounced differences in G-protein coupling preference and the physiol. responses they mediate.  Unfortunately, despite decades of effort, no therapeutic agents endowed with clear mAChR subtype selectivity have been developed to exploit these differences.  We describe here the structure of the Gq/11-coupled M3 mAChR ('M3 receptor', from rat) bound to the bronchodilator drug tiotropium and identify the binding mode for this clin. important drug.  This structure, together with that of the Gi/o-coupled M2 receptor, offers possibilities for the design of mAChR subtype-selective ligands.  Importantly, the M3 receptor structure allows a structural comparison between two members of a mammalian GPCR subfamily displaying different G-protein coupling selectivities.  Furthermore, mol. dynamics simulations suggest that tiotropium binds transiently to an allosteric site en route to the binding pocket of both receptors.  These simulations offer a structural view of an allosteric binding mode for an orthosteric GPCR ligand and provide addnl. opportunities for the design of ligands with different affinities or binding kinetics for different mAChR subtypes.  Our findings not only offer insights into the structure and function of one of the most important GPCR families, but may also facilitate the design of improved therapeutics targeting these crit. receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozveVSiaQvz7Vg90H21EOLACvtfcHk0ljOz9aXd9Jf-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xis1eqs7k%253D&md5=31bb89b1d14cf122353fced8f905e996</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature10867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10867%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DRosemond%26aufirst%3DE.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520and%2520Dynamics%2520of%2520the%2520M3%2520Muscarinic%2520Acetylcholine%2520Receptor%26jtitle%3DNature%26date%3D2012%26volume%3D482%26issue%3D7386%26spage%3D552%26epage%3D556%26doi%3D10.1038%2Fnature10867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span></span> <span> </span><span class="NLM_article-title">On-Rate Based Optimization of Structure-Kinetic Relationship--Surfing the Kinetic Map</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.ddtec.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=26724331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC28rms1egtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=9-15&author=A.+Schoopauthor=F.+Dey&title=On-Rate+Based+Optimization+of+Structure-Kinetic+Relationship%2D%2DSurfing+the+Kinetic+Map&doi=10.1016%2Fj.ddtec.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">On-rate based optimization of structure-kinetic relationship--surfing the kinetic map</span></div><div class="casAuthors">Schoop Andreas; Dey Fabian</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9-15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In the lead discovery process residence time has become an important parameter for the identification and characterization of the most efficacious compounds in vivo.  To enable the success of compound optimization by medicinal chemistry toward a desired residence time the understanding of structure-kinetic relationship (SKR) is essential.  This article reviews various approaches to monitor SKR and suggests using the on-rate as the key monitoring parameter.  The literature is reviewed and examples of compound series with low variability as well as with significant changes in on-rates are highlighted.  Furthermore, findings of kinetic on-rate changes are presented and potential underlying rationales are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwRMy18rtTE3T4ugvCDgnZfW6udTcc2eaN0fV3TesVfrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rms1egtw%253D%253D&md5=a7f0dee9cd02f38a945123e7faa9bc1c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DSchoop%26aufirst%3DA.%26aulast%3DDey%26aufirst%3DF.%26atitle%3DOn-Rate%2520Based%2520Optimization%2520of%2520Structure-Kinetic%2520Relationship--Surfing%2520the%2520Kinetic%2520Map%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2015%26volume%3D17%26spage%3D9%26epage%3D15%26doi%3D10.1016%2Fj.ddtec.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Residence Time: Critical Information for Lead Optimization</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.06.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.cbpa.2010.06.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=20663707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=467-474&issue=4&author=H.+Luauthor=P.+J.+Tonge&title=Drug-Target+Residence+Time%3A+Critical+Information+for+Lead+Optimization&doi=10.1016%2Fj.cbpa.2010.06.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time: Critical information for lead optimization</span></div><div class="casAuthors">Lu, Hao; Tonge, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Failure due to poor in vivo efficacy is a primary contributor to attrition during the development of new chemotherapeutics.  Lead optimization programs that in their quest for efficacy focus solely on improving the affinity of drug-target binding are flawed, since this approach ignores the fluctuations in drug concn. that occur in vivo.  Instead the lifetime of the drug-target complex must also be considered, since drugs only act when they are bound to their targets.  Consequently, to improve the correlation between the in vitro and in vivo activity of drugs, measurements of drug-target residence time must be incorporated into the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodk2kN_txuMLVg90H21EOLACvtfcHk0ljOz9aXd9Jf-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D&md5=3189f56032106db858cfb4269d3fc11f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.06.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.06.176%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DH.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26atitle%3DDrug-Target%2520Residence%2520Time%253A%2520Critical%2520Information%2520for%2520Lead%2520Optimization%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26issue%3D4%26spage%3D467%26epage%3D474%26doi%3D10.1016%2Fj.cbpa.2010.06.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spagnuolo, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eltschkner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daryaee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davoodi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, E. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pschibul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moree, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thivalapill, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truglio, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salafsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slayden, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kisker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span> <span> </span><span class="NLM_article-title">Evaluating the Contribution of Transition-State Destabilization to Changes in the Residence Time of Triazole-Based InhA Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3417</span>– <span class="NLM_lpage">3429</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b11148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=3417-3429&issue=9&author=L.+A.+Spagnuoloauthor=S.+Eltschknerauthor=W.+Yuauthor=F.+Daryaeeauthor=S.+Davoodiauthor=S.+E.+Knudsonauthor=E.+K.+H.+Allenauthor=J.+Merinoauthor=A.+Pschibulauthor=B.+Moreeauthor=N.+Thivalapillauthor=J.+J.+Truglioauthor=J.+Salafskyauthor=R.+A.+Slaydenauthor=C.+Kiskerauthor=P.+J.+Tonge&title=Evaluating+the+Contribution+of+Transition-State+Destabilization+to+Changes+in+the+Residence+Time+of+Triazole-Based+InhA+Inhibitors&doi=10.1021%2Fjacs.6b11148"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11148%26sid%3Dliteratum%253Aachs%26aulast%3DSpagnuolo%26aufirst%3DL.%2BA.%26aulast%3DEltschkner%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DDaryaee%26aufirst%3DF.%26aulast%3DDavoodi%26aufirst%3DS.%26aulast%3DKnudson%26aufirst%3DS.%2BE.%26aulast%3DAllen%26aufirst%3DE.%2BK.%2BH.%26aulast%3DMerino%26aufirst%3DJ.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DMoree%26aufirst%3DB.%26aulast%3DThivalapill%26aufirst%3DN.%26aulast%3DTruglio%26aufirst%3DJ.%2BJ.%26aulast%3DSalafsky%26aufirst%3DJ.%26aulast%3DSlayden%26aufirst%3DR.%2BA.%26aulast%3DKisker%26aufirst%3DC.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26atitle%3DEvaluating%2520the%2520Contribution%2520of%2520Transition-State%2520Destabilization%2520to%2520Changes%2520in%2520the%2520Residence%2520Time%2520of%2520Triazole-Based%2520InhA%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D9%26spage%3D3417%26epage%3D3429%26doi%3D10.1021%2Fjacs.6b11148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollica, L.</span></span> <span> </span><span class="NLM_article-title">Protein–ligand (Un)Binding Kinetics as a New Paradigm for Drug Discovery at the Crossroad between Experiments and Modelling</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1039/C6MD00581K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1039%2FC6MD00581K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=30108770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsleitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=534-550&issue=3&author=M.+Bernettiauthor=A.+Cavalliauthor=L.+Mollica&title=Protein%E2%80%93ligand+%28Un%29Binding+Kinetics+as+a+New+Paradigm+for+Drug+Discovery+at+the+Crossroad+between+Experiments+and+Modelling&doi=10.1039%2FC6MD00581K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-ligand (un)binding kinetics as a new paradigm for drug discovery at the crossroad between experiments and modelling</span></div><div class="casAuthors">Bernetti, M.; Cavalli, A.; Mollica, L.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">534-550</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last three decades, protein and nucleic acid structure detn. and comprehension of the mechanisms, leading to their physiol. and pathol. functions, have become a cornerstone of biomedical sciences.  A deep understanding of the principles governing the fates of cells and tissue at the mol. level has been gained over the years, offering a solid basis for the rational design of drugs aimed at the pharmacol. treatment of numerous diseases.  Historically, affinity indicators (i.e. Kd and IC50/EC50) have been assumed to be valid indicators of the in vivo efficacy of a drug.  However, recent studies pointed out that the kinetics of the drug-receptor binding process could be as important or even more important than affinity in detg. the drug efficacy.  This eventually led to a growing interest in the characterization and prediction of the rate consts. of protein-ligand assocn. and dissocn.  For instance, a drug with a longer residence time can kinetically select a given receptor over another, even if the affinity for both receptors is comparable, thus increasing its therapeutic index.  Therefore, understanding the mol. features underlying binding and unbinding processes is of central interest towards the rational control of drug binding kinetics.  In this review, we report the theor. framework behind protein-ligand assocn. and highlight the latest advances in the exptl. and computational approaches exploited to investigate the binding kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrO85mDP2s17Vg90H21EOLACvtfcHk0ljl90xfEqgndg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsleitbw%253D&md5=8538a9e13d7e1d2c873da236be95a811</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1039%2FC6MD00581K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00581K%26sid%3Dliteratum%253Aachs%26aulast%3DBernetti%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DMollica%26aufirst%3DL.%26atitle%3DProtein%25E2%2580%2593ligand%2520%2528Un%2529Binding%2520Kinetics%2520as%2520a%2520New%2520Paradigm%2520for%2520Drug%2520Discovery%2520at%2520the%2520Crossroad%2520between%2520Experiments%2520and%2520Modelling%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26issue%3D3%26spage%3D534%26epage%3D550%26doi%3D10.1039%2FC6MD00581K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Truhlar, D.
G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klippenstein, S. J.</span></span> <span> </span><span class="NLM_article-title">Current Status of Transition-State Theory</span>. <i>J. Phys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>100</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">12771</span>– <span class="NLM_lpage">12800</span>, <span class="refDoi"> DOI: 10.1021/jp953748q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp953748q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaK28Xkt1ansr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1996&pages=12771-12800&issue=31&author=D.%0AG.+Truhlarauthor=B.+C.+Garrettauthor=S.+J.+Klippenstein&title=Current+Status+of+Transition-State+Theory&doi=10.1021%2Fjp953748q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Current Status of Transition-State Theory</span></div><div class="casAuthors">Truhlar, Donald G.; Garrett, Bruce C.; Klippenstein, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">12771-12800</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 843 refs.; we present an overview of the current status of transition-state theory and its generalizations.  We emphasize (i) recent improvements in available methodol. for calcns. on complex systems, including the interface with electronic structure theory, (ii) progress in the theory and application of transition-state theory to condensed-phase reactions, and (iii) insight into the relation of transition-state theory to accurate quantum dynamics and tests of its accuracy via comparisons with both exptl. and other theor. dynamical approxns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnTDhjv-GhGbVg90H21EOLACvtfcHk0ljl90xfEqgndg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xkt1ansr8%253D&md5=5663e2f23815cdc1c0bbb6bbb91adabe</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjp953748q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp953748q%26sid%3Dliteratum%253Aachs%26aulast%3DTruhlar%26aufirst%3DD.%2BG.%26aulast%3DGarrett%26aufirst%3DB.%2BC.%26aulast%3DKlippenstein%26aufirst%3DS.%2BJ.%26atitle%3DCurrent%2520Status%2520of%2520Transition-State%2520Theory%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1996%26volume%3D100%26issue%3D31%26spage%3D12771%26epage%3D12800%26doi%3D10.1021%2Fjp953748q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willemsen-Seegers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uitdehaag, J. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinsen, M. B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vetter, J. R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Man, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buijsman, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman, G. J. R.</span></span> <span> </span><span class="NLM_article-title">Compound Selectivity and Target Residence Time of Kinase Inhibitors Studied with Surface Plasmon Resonance</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">574</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2016.12.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.jmb.2016.12.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=574-586&issue=4&author=N.+Willemsen-Seegersauthor=J.+C.+M.+Uitdehaagauthor=M.+B.+W.+Prinsenauthor=J.+R.+F.+de+Vetterauthor=J.+de+Manauthor=M.+Sawaauthor=Y.+Kawaseauthor=R.+C.+Buijsmanauthor=G.+J.+R.+Zaman&title=Compound+Selectivity+and+Target+Residence+Time+of+Kinase+Inhibitors+Studied+with+Surface+Plasmon+Resonance&doi=10.1016%2Fj.jmb.2016.12.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.12.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.12.019%26sid%3Dliteratum%253Aachs%26aulast%3DWillemsen-Seegers%26aufirst%3DN.%26aulast%3DUitdehaag%26aufirst%3DJ.%2BC.%2BM.%26aulast%3DPrinsen%26aufirst%3DM.%2BB.%2BW.%26aulast%3Dde%2BVetter%26aufirst%3DJ.%2BR.%2BF.%26aulast%3Dde%2BMan%26aufirst%3DJ.%26aulast%3DSawa%26aufirst%3DM.%26aulast%3DKawase%26aufirst%3DY.%26aulast%3DBuijsman%26aufirst%3DR.%2BC.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%2BR.%26atitle%3DCompound%2520Selectivity%2520and%2520Target%2520Residence%2520Time%2520of%2520Kinase%2520Inhibitors%2520Studied%2520with%2520Surface%2520Plasmon%2520Resonance%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26issue%3D4%26spage%3D574%26epage%3D586%26doi%3D10.1016%2Fj.jmb.2016.12.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuetz, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Witte, W. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knasmueller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokh, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadiq, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nederpelt, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segala, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaral, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoddart, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frech, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Lange, E. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span> <span> </span><span class="NLM_article-title">Kinetics for Drug Discovery: An Industry-Driven Effort to Target Drug Residence Time</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">896</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.drudis.2017.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=28412474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt1Kjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=896-911&issue=6&author=D.+A.+Schuetzauthor=W.+E.+A.+de+Witteauthor=Y.+C.+Wongauthor=B.+Knasmuellerauthor=L.+Richterauthor=D.+B.+Kokhauthor=S.+K.+Sadiqauthor=R.+Bosmaauthor=I.+Nederpeltauthor=L.+H.+Heitmanauthor=E.+Segalaauthor=M.+Amaralauthor=D.+Guoauthor=D.+Andresauthor=V.+Georgiauthor=L.+A.+Stoddartauthor=S.+Hillauthor=R.+M.+Cookeauthor=C.+De+Graafauthor=R.+Leursauthor=M.+Frechauthor=R.+C.+Wadeauthor=E.+C.+M.+de+Langeauthor=A.+P.+Ijzermanauthor=A.+M%C3%BCller-Fahrnowauthor=G.+F.+Ecker&title=Kinetics+for+Drug+Discovery%3A+An+Industry-Driven+Effort+to+Target+Drug+Residence+Time&doi=10.1016%2Fj.drudis.2017.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics for Drug Discovery: an industry-driven effort to target drug residence time</span></div><div class="casAuthors">Schuetz, Doris A.; de Witte, Wilhelmus Egbertus Arnout; Wong, Yin Cheong; Knasmueller, Bernhard; Richter, Lars; Kokh, Daria B.; Sadiq, S. Kashif; Bosma, Reggie; Nederpelt, Indira; Heitman, Laura H.; Segala, Elena; Amaral, Marta; Guo, Dong; Andres, Dorothee; Georgi, Victoria; Stoddart, Leigh A.; Hill, Steve; Cooke, Robert M.; De Graaf, Chris; Leurs, Rob; Frech, Matthias; Wade, Rebecca C.; de Lange, Elizabeth Cunera Maria; IJzerman, Adriaan P.; Mueller-Fahrnow, Anke; Ecker, Gerhard F.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">896-911</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  A considerable no. of approved drugs show non-equil. binding characteristics, emphasizing the potential role of drug residence times for in vivo efficacy.  Therefore, a detailed understanding of the kinetics of assocn. and dissocn. of a target-ligand complex might provide crucial insight into the mol. mechanism-of-action of a compd.  This deeper understanding will help to improve decision making in drug discovery, thus leading to a better selection of interesting compds. to be profiled further.  In this review, we highlight the contributions of the Kinetics for Drug Discovery (K4DD) Consortium, which targets major open questions related to binding kinetics in an industry-driven public-private partnership.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRmkYejg8ObVg90H21EOLACvtfcHk0lj99LrVC3tb-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmt1Kjs7k%253D&md5=78fce4f850718075ba44b28a20f79195</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DSchuetz%26aufirst%3DD.%2BA.%26aulast%3Dde%2BWitte%26aufirst%3DW.%2BE.%2BA.%26aulast%3DWong%26aufirst%3DY.%2BC.%26aulast%3DKnasmueller%26aufirst%3DB.%26aulast%3DRichter%26aufirst%3DL.%26aulast%3DKokh%26aufirst%3DD.%2BB.%26aulast%3DSadiq%26aufirst%3DS.%2BK.%26aulast%3DBosma%26aufirst%3DR.%26aulast%3DNederpelt%26aufirst%3DI.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DSegala%26aufirst%3DE.%26aulast%3DAmaral%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DAndres%26aufirst%3DD.%26aulast%3DGeorgi%26aufirst%3DV.%26aulast%3DStoddart%26aufirst%3DL.%2BA.%26aulast%3DHill%26aufirst%3DS.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DDe%2BGraaf%26aufirst%3DC.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3DFrech%26aufirst%3DM.%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3Dde%2BLange%26aufirst%3DE.%2BC.%2BM.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DM%25C3%25BCller-Fahrnow%26aufirst%3DA.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26atitle%3DKinetics%2520for%2520Drug%2520Discovery%253A%2520An%2520Industry-Driven%2520Effort%2520to%2520Target%2520Drug%2520Residence%2520Time%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26issue%3D6%26spage%3D896%26epage%3D911%26doi%3D10.1016%2Fj.drudis.2017.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span>K4DD - Kinetics for Drug Discovery. <a href="http://db.k4dd.eu" class="extLink">http://db.k4dd.eu</a> (accessed December
6, <span class="NLM_year">2017</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=K4DD+-+Kinetics+for+Drug+Discovery.+http%3A%2F%2Fdb.k4dd.eu+%28accessed+December%0A6%2C+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vauquelin, G.</span></span> <span> </span><span class="NLM_article-title">Effects of Target Binding Kinetics on in Vivo Drug Efficacy: Koff, Kon and Rebinding</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>173</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">2319</span>– <span class="NLM_lpage">2334</span>, <span class="refDoi"> DOI: 10.1111/bph.13504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1111%2Fbph.13504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=27129075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSiurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2016&pages=2319-2334&issue=15&author=G.+Vauquelin&title=Effects+of+Target+Binding+Kinetics+on+in+Vivo+Drug+Efficacy%3A+Koff%2C+Kon+and+Rebinding&doi=10.1111%2Fbph.13504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of target binding kinetics on in vivo drug efficacy: koff, kon and rebinding</span></div><div class="casAuthors">Vauquelin, Georges</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2319-2334</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and Purpose : Optimal drug therapy often requires continuing high levels of target occupancy.  Besides the traditional pharmacokinetic contribution, target binding kinetics is increasingly considered to play an important role as well.  While most attention has been focused on the dissocn. rate of the complex, recent reports expressed doubt about the unreserved translatability of this pharmacodynamic property into clin. efficacy. 'Micro'-pharmacokinetic mechanisms like drug rebinding and partitioning into the cell membrane may constitute a potential fix.  Exptl. Approach : Simulations were based on solving differential equations.  Key Results : Based on a selected range of assocn. and dissocn. rate consts., kon and koff, and rebinding potencies of the drugs as variables, their effects on the temporal in vivo occupancy profile of their targets, after one or multiple repetitive dosings, have here been simulated.  Conclusions and Implications : Most strikingly, the simulations show that, when rebinding is also taken into account, increasing kon may produce closely the same outcome as decreasing koff when dosing is performed in accordance with the therapeutically most relevant const. [Lmax]/KD ratio paradigm.  Also, under certain conditions, rebinding may produce closely the same outcome as invoking slow diffusion of the drug between the plasma compartment and a target-contg. 'effect' compartment.  Although the present simulations should only be regarded as a 'proof of principle', these findings may help pharmacologists and medicinal chemists to devise ex vivo and in vitro binding kinetic assays that are more relevant and translatable to in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5-dbZNJetv7Vg90H21EOLACvtfcHk0li4HLVYPyzI-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSiurvI&md5=5d1b13ef5786a0c661118ce9be503d50</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1111%2Fbph.13504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13504%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DEffects%2520of%2520Target%2520Binding%2520Kinetics%2520on%2520in%2520Vivo%2520Drug%2520Efficacy%253A%2520Koff%252C%2520Kon%2520and%2520Rebinding%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2016%26volume%3D173%26issue%3D15%26spage%3D2319%26epage%3D2334%26doi%3D10.1111%2Fbph.13504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vauquelin, G.</span></span> <span> </span><span class="NLM_article-title">Rebinding: Or Why Drugs May Act Longer in Vivo than Expected from Their in Vitro Target Residence Time</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">941</span>, <span class="refDoi"> DOI: 10.1517/17460441.2010.512037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1517%2F17460441.2010.512037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=22823988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFKmtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=927-941&issue=10&author=G.+Vauquelin&title=Rebinding%3A+Or+Why+Drugs+May+Act+Longer+in+Vivo+than+Expected+from+Their+in+Vitro+Target+Residence+Time&doi=10.1517%2F17460441.2010.512037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Rebinding: or why drugs may act longer in vivo than expected from their in vitro target residence time</span></div><div class="casAuthors">Vauquelin, Georges</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">927-941</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: It is well established that the in vivo duration of drug action not only depends on macroscopic pharmacokinetic properties like its plasma half-life, but also on the residence time of the drug-target complexes.  However, drug 'rebinding' (i.e., the consecutive binding of dissocd. drug mols. to the original target and/or targets nearby) can be influential in vivo as well.  Areas covered in this review: Information about rebinding is available since the 1980s but it is dispersed in the life sciences literature.  This review compiles this information.  In this respect, neurochemists and biophysicians advance the same equations to describe drug rebinding.  What the reader will gain: The rebinding mechanism is explained according to the prevailing viewpoint in different life science disciplines.  There is a general consensus that high target densities, high assocn. rates and local phenomena that hinder the diffusion of free drug mols. away from their target all promote rebinding.  Take home message: Simulations presented here for the first time suggest that rebinding may increase the duration and even the constancy of the drug's clin. action.  Intact cell radioligand dissocn. and related ex vivo expts. offer useful indications about a drug's aptitude to experience target rebinding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSRqDC7FEY8bVg90H21EOLACvtfcHk0li4HLVYPyzI-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFKmtLbM&md5=928b8861b47515c10db3116bfcd47e4b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1517%2F17460441.2010.512037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2010.512037%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DRebinding%253A%2520Or%2520Why%2520Drugs%2520May%2520Act%2520Longer%2520in%2520Vivo%2520than%2520Expected%2520from%2520Their%2520in%2520Vitro%2520Target%2520Residence%2520Time%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26issue%3D10%26spage%3D927%26epage%3D941%26doi%3D10.1517%2F17460441.2010.512037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de Witte, W. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danhof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Graaf, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Lange, E. C. M.</span></span> <span> </span><span class="NLM_article-title">In Vivo Target Residence Time and Kinetic Selectivity: The Association Rate Constant as Determinant</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">831</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2016.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.tips.2016.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=27394919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVynur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=831-842&issue=10&author=W.+E.+A.+de+Witteauthor=M.+Danhofauthor=P.+H.+van+der%0AGraafauthor=E.+C.+M.+de+Lange&title=In+Vivo+Target+Residence+Time+and+Kinetic+Selectivity%3A+The+Association+Rate+Constant+as+Determinant&doi=10.1016%2Fj.tips.2016.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo Target Residence Time and Kinetic Selectivity: The Association Rate Constant as Determinant</span></div><div class="casAuthors">de Witte, Wilhelmus E. A.; Danhof, Meindert; van der Graaf, Piet H.; de Lange, Elizabeth C. M.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">831-842</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">It is generally accepted that, in conjunction with pharmacokinetics, the first-order rate const. of target dissocn. is a major determinant of the time course and duration of in vivo target occupancy.  Here we show that the second-order rate const. of target assocn. can be equally important.  On the basis of the commonly used math. models for drug-target binding, it is shown that a high target assocn. rate const. can increase the (local) concn. of the drug, which decreases the rate of decline of target occupancy.  The increased drug concn. can also lead to increased off-target binding and decreased selectivity.  Therefore, the kinetics of both target assocn. and dissocn. need to be taken into account in the selection of drug candidates with optimal pharmacodynamic properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri0cm4NKLv-7Vg90H21EOLACvtfcHk0li4HLVYPyzI-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVynur%252FE&md5=e2c077f2c8636214e2f8a77ee5734ad4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2016.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2016.06.008%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BWitte%26aufirst%3DW.%2BE.%2BA.%26aulast%3DDanhof%26aufirst%3DM.%26aulast%3Dvan%2Bder%2BGraaf%26aufirst%3DP.%2BH.%26aulast%3Dde%2BLange%26aufirst%3DE.%2BC.%2BM.%26atitle%3DIn%2520Vivo%2520Target%2520Residence%2520Time%2520and%2520Kinetic%2520Selectivity%253A%2520The%2520Association%2520Rate%2520Constant%2520as%2520Determinant%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2016%26volume%3D37%26issue%3D10%26spage%3D831%26epage%3D842%26doi%3D10.1016%2Fj.tips.2016.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javitch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlton, S. J.</span></span> <span> </span><span class="NLM_article-title">Extrapyramidal Side Effects of Antipsychotics Are Linked to Their Association Kinetics at Dopamine D 2 Receptors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">763</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-00716-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fs41467-017-00716-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=28970469" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FksFGgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=763&issue=1&author=D.+A.+Sykesauthor=H.+Mooreauthor=L.+Stottauthor=N.+Hollidayauthor=J.+A.+Javitchauthor=J.+R.+Laneauthor=S.+J.+Charlton&title=Extrapyramidal+Side+Effects+of+Antipsychotics+Are+Linked+to+Their+Association+Kinetics+at+Dopamine+D+2+Receptors&doi=10.1038%2Fs41467-017-00716-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Extrapyramidal side effects of antipsychotics are linked to their association kinetics at dopamine D2 receptors</span></div><div class="casAuthors">Sykes David A; Stott Lisa; Holliday Nicholas; Charlton Steven J; Moore Holly; Javitch Jonathan A; Moore Holly; Javitch Jonathan A; Javitch Jonathan A; Lane J Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">763</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Atypical antipsychotic drugs (APDs) have been hypothesized to show reduced extrapyramidal side effects (EPS) due to their rapid dissociation from the dopamine D2 receptor.  However, support for this hypothesis is limited to a relatively small number of observations made across several decades and under different experimental conditions.  Here we show that association rates, but not dissociation rates, correlate with EPS.  We measured the kinetic binding properties of a series of typical and atypical APDs in a novel time-resolved fluorescence resonance energy transfer assay, and correlated these properties with their EPS and prolactin-elevating liabilities at therapeutic doses.  EPS are robustly predicted by a rebinding model that considers the microenvironment of postsynaptic D2 receptors and integrates association and dissociation rates to calculate the net rate of reversal of receptor blockade.  Thus, optimizing binding kinetics at the D2 receptor may result in APDs with improved therapeutic profile.Atypical antipsychotics show reduced extrapyramidal side effects compared to first generation drugs.  Here the authors use time-resolved FRET to measure binding kinetics, and show that side effects correlate with drug association rates to the D2 receptor, while dissociation rates correlate with prolactin elevation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSAv7HAvl2UWlVwdslNBF-tfW6udTcc2ea5jWo27SzMHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FksFGgsg%253D%253D&md5=6d5c8596da9a3565b7a95b55eceac3cf</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00716-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00716-z%26sid%3Dliteratum%253Aachs%26aulast%3DSykes%26aufirst%3DD.%2BA.%26aulast%3DMoore%26aufirst%3DH.%26aulast%3DStott%26aufirst%3DL.%26aulast%3DHolliday%26aufirst%3DN.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DCharlton%26aufirst%3DS.%2BJ.%26atitle%3DExtrapyramidal%2520Side%2520Effects%2520of%2520Antipsychotics%2520Are%2520Linked%2520to%2520Their%2520Association%2520Kinetics%2520at%2520Dopamine%2520D%25202%2520Receptors%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D763%26doi%3D10.1038%2Fs41467-017-00716-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansell, J. E.</span></span> <span> </span><span class="NLM_article-title">Direct Thrombin Inhibitors</span>. <i>Br. J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>72</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2125.2011.03916.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1111%2Fj.1365-2125.2011.03916.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=21241354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlers7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2011&pages=581-592&issue=4&author=C.+J.+Leeauthor=J.+E.+Ansell&title=Direct+Thrombin+Inhibitors&doi=10.1111%2Fj.1365-2125.2011.03916.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Direct thrombin inhibitors</span></div><div class="casAuthors">Lee, Catherine J.; Ansell, Jack E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">581-592</span>CODEN:
                <span class="NLM_cas:coden">BCPHBM</span>;
        ISSN:<span class="NLM_cas:issn">0306-5251</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Heparins and vitamin K antagonists have been the primary agents used for anticoagulation in certain cardiovascular and thromboembolic diseases for over 50 years.  However, they can be difficult to administer and are fraught with limitations.  In response to the need for new anticoagulants, direct thrombin inhibitors (DTIs) have been developed and investigated for their utility in prophylaxis and treatment of venous thromboembolism (VTE), heparin-induced thrombocytopenia (HIT), acute coronary syndromes (ACS), secondary prevention of coronary events after ACS, and nonvalvular atrial fibrillation.  Currently, four parenteral direct inhibitors of thrombin activity are FDA-approved in North America: lepirudin, desirudin, bivalirudin and argatroban.  Of the new oral DTIs, dabigatran etexilate is the most studied and promising of these agents.  This review discusses the clin. indications and efficacy of these direct thrombin inhibitors as well as future directions in anticoagulant therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsZGu46hTy8rVg90H21EOLACvtfcHk0li44e9A6seYcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlers7rM&md5=32c9ea4decff589c5f2d453c9a322f6b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2125.2011.03916.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2125.2011.03916.x%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DAnsell%26aufirst%3DJ.%2BE.%26atitle%3DDirect%2520Thrombin%2520Inhibitors%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D2011%26volume%3D72%26issue%3D4%26spage%3D581%26epage%3D592%26doi%3D10.1111%2Fj.1365-2125.2011.03916.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walkup, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, E. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daryaee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, V. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buurman, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajec, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy-Benenato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patey, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosser, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, S. L.</span></span> <span> </span><span class="NLM_article-title">Translating Slow-Binding Inhibition Kinetics into Cellular and in Vivo Effects</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">416</span>– <span class="NLM_lpage">423</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnchembio.1796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=25894085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=416-423&issue=6&author=G.+K.+Walkupauthor=Z.+Youauthor=P.+L.+Rossauthor=E.+K.+H.+Allenauthor=F.+Daryaeeauthor=M.+R.+Haleauthor=J.+O%E2%80%99Donnellauthor=D.+E.+Ehmannauthor=V.+J.+A.+Schuckauthor=E.+T.+Buurmanauthor=A.+L.+Choyauthor=L.+Hajecauthor=K.+Murphy-Benenatoauthor=V.+Maroneauthor=S.+A.+Pateyauthor=L.+A.+Grosserauthor=M.+Johnstoneauthor=S.+G.+Walkerauthor=P.+J.+Tongeauthor=S.+L.+Fisher&title=Translating+Slow-Binding+Inhibition+Kinetics+into+Cellular+and+in+Vivo+Effects&doi=10.1038%2Fnchembio.1796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Translating slow-binding inhibition kinetics into cellular and in vivo effects</span></div><div class="casAuthors">Walkup, Grant K.; You, Zhiping; Ross, Philip L.; Allen, Eleanor K. H.; Daryaee, Fereidoon; Hale, Michael R.; O'Donnell, John; Ehmann, David E.; Schuck, Virna J. A.; Buurman, Ed T.; Choy, Allison L.; Hajec, Laurel; Murphy-Benenato, Kerry; Marone, Valerie; Patey, Sara A.; Grosser, Lena A.; Johnstone, Michele; Walker, Stephen G.; Tonge, Peter J.; Fisher, Stewart L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">416-423</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many drug candidates fail in clin. trials owing to a lack of efficacy from limited target engagement or an insufficient therapeutic index.  Minimizing off-target effects while retaining the desired pharmacodynamic (PD) response can be achieved by reduced exposure for drugs that display kinetic selectivity in which the drug-target complex has a longer half-life than off-target-drug complexes.  However, though slow-binding inhibition kinetics are a key feature of many marketed drugs, prospective tools that integrate drug-target residence time into predictions of drug efficacy are lacking, hindering the integration of drug-target kinetics into the drug discovery cascade.  Here we describe a mechanistic PD model that includes drug-target kinetic parameters, including the on- and off-rates for the formation and breakdown of the drug-target complex.  We demonstrate the utility of this model by using it to predict dose response curves for inhibitors of the LpxC enzyme from Pseudomonas aeruginosa in an animal model of infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS0SazAerLJLVg90H21EOLACvtfcHk0li44e9A6seYcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartb0%253D&md5=87c25ffbcbb74aaa3ecd80aba26d2f13</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1796%26sid%3Dliteratum%253Aachs%26aulast%3DWalkup%26aufirst%3DG.%2BK.%26aulast%3DYou%26aufirst%3DZ.%26aulast%3DRoss%26aufirst%3DP.%2BL.%26aulast%3DAllen%26aufirst%3DE.%2BK.%2BH.%26aulast%3DDaryaee%26aufirst%3DF.%26aulast%3DHale%26aufirst%3DM.%2BR.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%26aulast%3DEhmann%26aufirst%3DD.%2BE.%26aulast%3DSchuck%26aufirst%3DV.%2BJ.%2BA.%26aulast%3DBuurman%26aufirst%3DE.%2BT.%26aulast%3DChoy%26aufirst%3DA.%2BL.%26aulast%3DHajec%26aufirst%3DL.%26aulast%3DMurphy-Benenato%26aufirst%3DK.%26aulast%3DMarone%26aufirst%3DV.%26aulast%3DPatey%26aufirst%3DS.%2BA.%26aulast%3DGrosser%26aufirst%3DL.%2BA.%26aulast%3DJohnstone%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DS.%2BG.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DFisher%26aufirst%3DS.%2BL.%26atitle%3DTranslating%2520Slow-Binding%2520Inhibition%2520Kinetics%2520into%2520Cellular%2520and%2520in%2520Vivo%2520Effects%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D6%26spage%3D416%26epage%3D423%26doi%3D10.1038%2Fnchembio.1796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bommineni, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Diaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium Tuberculosis InhA</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">4683</span>– <span class="NLM_lpage">4691</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.5b00284</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b00284" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFWqtb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=4683-4691&issue=30&author=C.-T.+Laiauthor=H.-J.+Liauthor=W.+Yuauthor=S.+Shahauthor=G.+R.+Bommineniauthor=V.+Perroneauthor=M.+Garcia-Diazauthor=P.+J.+Tongeauthor=C.+Simmerling&title=Rational+Modulation+of+the+Induced-Fit+Conformational+Change+for+Slow-Onset+Inhibition+in+Mycobacterium+Tuberculosis+InhA&doi=10.1021%2Facs.biochem.5b00284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium tuberculosis InhA</span></div><div class="casAuthors">Lai, Cheng-Tsung; Li, Huei-Jiun; Yu, Weixuan; Shah, Sonam; Bommineni, Gopal R.; Perrone, Victoria; Garcia-Diaz, Miguel; Tonge, Peter J.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">4683-4691</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Slow-onset enzyme inhibitors are the subject of considerable interest as an approach to increasing the potency of pharmaceutical compds. by extending the residence time of the inhibitor on the target (the lifetime of the drug-receptor complex).  However, rational modulation of residence time presents significant challenges because it requires addnl. mechanistic insight, such as the nature of the transition state for postbinding isomerization.  Our previous work, based on X-ray crystallog., enzyme kinetics, and mol. dynamics simulation, suggested that the slow step in inhibition of the Mycobacterium tuberculosis enoyl-ACP reductase InhA involves a change in the conformation of the substrate binding loop from an open state in the initial enzyme-inhibitor complex to a closed state in the final enzyme-inhibitor complex.  Here, we use multidimensional free energy landscapes for loop isomerization to obtain a computational model for the transition state.  The results suggest that slow-onset inhibitors crowd key side chains on helixes that slide past each other during isomerization, resulting in a steric clash.  The landscapes become significantly flatter when residues involved in the steric clash are replaced with alanine.  Importantly, this lower barrier can be increased by rational inhibitor redesign to restore the steric clash.  Crystallog. studies and enzyme kinetics confirm the predicted effects on loop structure and flexibility, as well as inhibitor residence time.  These loss and regain of function studies validate our mechanistic hypothesis for interactions controlling substrate binding loop isomerization, providing a platform for the future design of inhibitors with longer residence times and better in vivo potency.  Similar opportunities for slow-onset inhibition via the same mechanism are identified in other pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6OmewK8DIh7Vg90H21EOLACvtfcHk0lhL_JL8mY4KmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWqtb%252FM&md5=2c014493a57205f99e2fbc698ff64488</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b00284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b00284%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DC.-T.%26aulast%3DLi%26aufirst%3DH.-J.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DBommineni%26aufirst%3DG.%2BR.%26aulast%3DPerrone%26aufirst%3DV.%26aulast%3DGarcia-Diaz%26aufirst%3DM.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DRational%2520Modulation%2520of%2520the%2520Induced-Fit%2520Conformational%2520Change%2520for%2520Slow-Onset%2520Inhibition%2520in%2520Mycobacterium%2520Tuberculosis%2520InhA%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26issue%3D30%26spage%3D4683%26epage%3D4691%26doi%3D10.1021%2Facs.biochem.5b00284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luckner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">am Ende, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kisker, C.</span></span> <span> </span><span class="NLM_article-title">A Slow, Tight Binding Inhibitor of InhA, the Enoyl-Acyl Carrier Protein Reductase from Mycobacterium Tuberculosis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">14330</span>– <span class="NLM_lpage">14337</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.090373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1074%2Fjbc.M109.090373" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=14330-14337&issue=19&author=S.+R.+Lucknerauthor=N.+Liuauthor=C.+W.+am+Endeauthor=P.+J.+Tongeauthor=C.+Kisker&title=A+Slow%2C+Tight+Binding+Inhibitor+of+InhA%2C+the+Enoyl-Acyl+Carrier+Protein+Reductase+from+Mycobacterium+Tuberculosis&doi=10.1074%2Fjbc.M109.090373"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.090373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.090373%26sid%3Dliteratum%253Aachs%26aulast%3DLuckner%26aufirst%3DS.%2BR.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3Dam%2BEnde%26aufirst%3DC.%2BW.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DKisker%26aufirst%3DC.%26atitle%3DA%2520Slow%252C%2520Tight%2520Binding%2520Inhibitor%2520of%2520InhA%252C%2520the%2520Enoyl-Acyl%2520Carrier%2520Protein%2520Reductase%2520from%2520Mycobacterium%2520Tuberculosis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D19%26spage%3D14330%26epage%3D14337%26doi%3D10.1074%2Fjbc.M109.090373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Conformational Adaptation in Drug-Target Interactions and Residence Time</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1491</span>– <span class="NLM_lpage">1501</span>, <span class="refDoi"> DOI: 10.4155/fmc.11.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.4155%2Ffmc.11.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=21882942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFSksLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1491-1501&issue=12&author=R.+A.+Copeland&title=Conformational+Adaptation+in+Drug-Target+Interactions+and+Residence+Time&doi=10.4155%2Ffmc.11.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational adaptation in drug-target interactions and residence time</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1491-1501</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Although drug-target interactions are commonly illustrated in terms of structurally static binding and dissocn. events, such descriptions are inadequate to explain the impact of conformational dynamics on these processes.  For high-affinity interactions, both the assocn. and dissocn. of drug mols. to and from their targets are often controlled by conformational changes of the target.  Conformational adaptation can greatly influence the residence time of a drug on its target (i.e., the lifetime of the binary drug-target complex); long residence time can lead to sustained pharmacol. and may also mitigate off-target toxicity.  In this perspective, the kinetics of drug-target assocn. and dissocn. reactions are explored, with particular emphasis on the impact of conformational adaptation on drug-target residence time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4Bp8rlLqFa7Vg90H21EOLACvtfcHk0lhL_JL8mY4KmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFSksLfO&md5=f995e1a9d854831af67a5d96eb14cac4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.112%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DConformational%2520Adaptation%2520in%2520Drug-Target%2520Interactions%2520and%2520Residence%2520Time%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26issue%3D12%26spage%3D1491%26epage%3D1501%26doi%3D10.4155%2Ffmc.11.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amaral, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokh, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggenweiler, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matias, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirrenberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frech, M.</span></span> <span> </span><span class="NLM_article-title">Protein Conformational Flexibility Modulates Kinetics and Thermodynamics of Drug Binding</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">2276</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-02258-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fs41467-017-02258-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=29273709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC1Mzksleqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2276&issue=1&author=M.+Amaralauthor=D.+B.+Kokhauthor=J.+Bomkeauthor=A.+Wegenerauthor=H.+P.+Buchstallerauthor=H.+M.+Eggenweilerauthor=P.+Matiasauthor=C.+Sirrenbergauthor=R.+C.+Wadeauthor=M.+Frech&title=Protein+Conformational+Flexibility+Modulates+Kinetics+and+Thermodynamics+of+Drug+Binding&doi=10.1038%2Fs41467-017-02258-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Protein conformational flexibility modulates kinetics and thermodynamics of drug binding</span></div><div class="casAuthors">Amaral M; Matias P; Amaral M; Wegener A; Frech M; Amaral M; Kokh D B; Wade R C; Bomke J; Buchstaller H P; Eggenweiler H M; Matias P; Sirrenberg C; Wade R C; Wade R C</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2276</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based drug design has often been restricted by the rather static picture of protein-ligand complexes presented by crystal structures, despite the widely accepted importance of protein flexibility in biomolecular recognition.  Here we report a detailed experimental and computational study of the drug target, human heat shock protein 90, to explore the contribution of protein dynamics to the binding thermodynamics and kinetics of drug-like compounds.  We observe that their binding properties depend on whether the protein has a loop or a helical conformation in the binding site of the ligand-bound state.  Compounds bound to the helical conformation display slow association and dissociation rates, high-affinity and high cellular efficacy, and predominantly entropically driven binding.  An important entropic contribution comes from the greater flexibility of the helical relative to the loop conformation in the ligand-bound state.  This unusual mechanism suggests increasing target flexibility in the bound state by ligand design as a new strategy for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLPI8jjKBQEQvupmecNw_lfW6udTcc2ebooB8_U00G5Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mzksleqsg%253D%253D&md5=2ccf76a6569363af27b6f27cb1be5b7a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02258-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02258-w%26sid%3Dliteratum%253Aachs%26aulast%3DAmaral%26aufirst%3DM.%26aulast%3DKokh%26aufirst%3DD.%2BB.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DBuchstaller%26aufirst%3DH.%2BP.%26aulast%3DEggenweiler%26aufirst%3DH.%2BM.%26aulast%3DMatias%26aufirst%3DP.%26aulast%3DSirrenberg%26aufirst%3DC.%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DFrech%26aufirst%3DM.%26atitle%3DProtein%2520Conformational%2520Flexibility%2520Modulates%2520Kinetics%2520and%2520Thermodynamics%2520of%2520Drug%2520Binding%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D2276%26doi%3D10.1038%2Fs41467-017-02258-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maragakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Pathway and Mechanism of Drug Binding to G-Protein-Coupled Receptors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span> (<span class="NLM_issue">32</span>),  <span class="NLM_fpage">13118</span>– <span class="NLM_lpage">13123</span>, <span class="refDoi"> DOI: 10.1073/pnas.1104614108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1073%2Fpnas.1104614108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=21778406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKit7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13118-13123&issue=32&author=R.+O.+Drorauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=D.+W.+Borhaniauthor=P.+Maragakisauthor=Y.+Shanauthor=H.+Xuauthor=D.+E.+Shaw&title=Pathway+and+Mechanism+of+Drug+Binding+to+G-Protein-Coupled+Receptors&doi=10.1073%2Fpnas.1104614108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Pathway and mechanism of drug binding to G-protein-coupled receptors</span></div><div class="casAuthors">Dror, Ron O.; Pan, Albert C.; Arlow, Daniel H.; Borhani, David W.; Maragakis, Paul; Shan, Yibing; Xu, Huafeng; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">13118-13123, S13118/1-S13118/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">How drugs bind to their receptors-from initial assocn., through drug entry into the binding pocket, to adoption of the final bound conformation, or "pose"-has remained unknown, even for G-protein-coupled receptor modulators, which constitute 1/3 of all marketed drugs.  Here, the authors captured this pharmaceutically crit. process in at. detail using the 1st unbiased mol. dynamics simulations in which drug mols. spontaneously assoc. with G-protein-coupled receptors to achieve final poses matching those detd. crystallog.  The authors found that several beta blockers and a beta agonist all traversed the same well-defined, dominant pathway as they bound to the β1- and β2-adrenergic receptors, initially making contact with a vestibule on each receptor's extracellular surface.  Surprisingly, assocn. with this vestibule, at a distance of 15 Å from the binding pocket, often presented the largest energetic barrier to binding, despite the fact that subsequent entry into the binding pocket required the receptor to deform and the drug to squeeze through a narrow passage.  The early barrier appeared to reflect the substantial dehydration that takes place as the drug assocs. with the vestibule.  The at.-level description of the binding process suggests opportunities for allosteric modulation and provides a structural foundation for future optimization of drug-receptor binding and unbinding rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKUYFtkTGwf7Vg90H21EOLACvtfcHk0lgehc-5KJUlLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKit7bE&md5=9cfa8673217931cacd586321309fa72e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1104614108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1104614108%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DMaragakis%26aufirst%3DP.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DPathway%2520and%2520Mechanism%2520of%2520Drug%2520Binding%2520to%2520G-Protein-Coupled%2520Receptors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D32%26spage%3D13118%26epage%3D13123%26doi%3D10.1073%2Fpnas.1104614108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gooljarsingh, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grooms, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johanson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinnamon, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span> <span> </span><span class="NLM_article-title">A Biochemical Rationale for the Anticancer Effects of Hsp90 Inhibitors: Slow, Tight Binding Inhibition by Geldanamycin and Its Analogues</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">7625</span>– <span class="NLM_lpage">7630</span>, <span class="refDoi"> DOI: 10.1073/pnas.0602650103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1073%2Fpnas.0602650103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=16684877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD28XlsVGlurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=7625-7630&issue=20&author=L.+T.+Gooljarsinghauthor=C.+Fernandesauthor=K.+Yanauthor=H.+Zhangauthor=M.+Groomsauthor=K.+Johansonauthor=R.+H.+Sinnamonauthor=R.+B.+Kirkpatrickauthor=J.+Kerriganauthor=T.+Lewisauthor=M.+Arnoneauthor=A.+J.+Kingauthor=Z.+Laiauthor=R.+A.+Copelandauthor=P.+J.+Tummino&title=A+Biochemical+Rationale+for+the+Anticancer+Effects+of+Hsp90+Inhibitors%3A+Slow%2C+Tight+Binding+Inhibition+by+Geldanamycin+and+Its+Analogues&doi=10.1073%2Fpnas.0602650103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A biochemical rationale for the anticancer effects of Hsp90 inhibitors: slow, tight binding inhibition by geldanamycin and its analogues</span></div><div class="casAuthors">Gooljarsingh, Lata T.; Fernandes, Christine; Yan, Kang; Zhang, Hong; Grooms, Michael; Johanson, Kyung; Sinnamon, Robert H.; Kirkpatrick, Robert B.; Kerrigan, John; Lewis, Tia; Arnone, Marc; King, Alastair J.; Lai, Zhihong; Copeland, Robert A.; Tummino, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7625-7630</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Heat shock protein (Hsp)90 is emerging as an important therapeutic target for the treatment of cancer.  Two analogs of the Hsp90 inhibitor geldanamycin are currently in clin. trials.  Geldanamycin (GA) and its analogs have been reported to bind purified Hsp90 with low micromolar potency, in stark contrast to their low nanomolar antiproliferative activity in cell culture and their potent antitumor activity in animal models.  Several models have been proposed to account for the ≈100-fold-greater potency in cell culture, including that GA analogs bind with greater affinity to a five-protein Hsp90 complex than to Hsp90 alone.  We have detd. that GA and the fluorescent analog BODIPY-GA (BDGA) both demonstrate slow, tight binding to purified Hsp90.  BDGA, used to characterize the kinetics of ligand-Hsp90 interactions, was found to bind Hsp90α with koff = 2.5×10-3 min-1, t1/2 = 4.6 h, and Ki* = 10 nM.  It was found that BDGA binds to a functional multiprotein Hsp90 complex with kinetics and affinity identical to that of Hsp90 alone.  Also, BDGA binds to Hsp90 from multiple cell lyzates in a time-dependent manner with similar kinetics.  Therefore, our results indicate that the high potency of GA in cell culture and in vivo can be accounted for by its time-dependent, tight binding to Hsp90 alone.  In the broader context, these studies highlight the essentiality of detailed biochem. characterization of drug-target interactions for the effective translation of in vitro pharmacol. to cellular and in vivo efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYOm9dd7WlmLVg90H21EOLACvtfcHk0lgehc-5KJUlLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlsVGlurY%253D&md5=10e3cc3af47d06242dd621b8c64b3b34</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0602650103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0602650103%26sid%3Dliteratum%253Aachs%26aulast%3DGooljarsingh%26aufirst%3DL.%2BT.%26aulast%3DFernandes%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DGrooms%26aufirst%3DM.%26aulast%3DJohanson%26aufirst%3DK.%26aulast%3DSinnamon%26aufirst%3DR.%2BH.%26aulast%3DKirkpatrick%26aufirst%3DR.%2BB.%26aulast%3DKerrigan%26aufirst%3DJ.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DArnone%26aufirst%3DM.%26aulast%3DKing%26aufirst%3DA.%2BJ.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26atitle%3DA%2520Biochemical%2520Rationale%2520for%2520the%2520Anticancer%2520Effects%2520of%2520Hsp90%2520Inhibitors%253A%2520Slow%252C%2520Tight%2520Binding%2520Inhibition%2520by%2520Geldanamycin%2520and%2520Its%2520Analogues%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26issue%3D20%26spage%3D7625%26epage%3D7630%26doi%3D10.1073%2Fpnas.0602650103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H.-P.</span>; <span class="NLM_string-name">Eggenweiler, H.-M.</span>; <span class="NLM_string-name">Wolf, M.</span>; <span class="NLM_string-name">Sirrenberg, C.</span></span> <span> </span><span class="NLM_article-title">Indazole Derivatives for Treating Hsp90-Induced Diseases</span>. Patent <span class="NLM_patent">WO 2008/003396</span>, June 18, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=H.-P.+Buchstaller&author=H.-M.+Eggenweiler&author=M.+Wolf&author=C.+Sirrenberg&title=Indazole+Derivatives+for+Treating+Hsp90-Induced+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBuchstaller%26aufirst%3DH.-P.%26atitle%3DIndazole%2520Derivatives%2520for%2520Treating%2520Hsp90-Induced%2520Diseases%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eggenweiler, H.-M.</span>; <span class="NLM_string-name">Wolf, M.</span>; <span class="NLM_string-name">Buchstaller, H.-P.</span>; <span class="NLM_string-name">Sirrenberg, C.</span></span> <span> </span><span class="NLM_article-title">Triazole Derivatives II</span>. CA Patent <span class="NLM_patent">CA 2,652,394</span>, April 20, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=H.-M.+Eggenweiler&author=M.+Wolf&author=H.-P.+Buchstaller&author=C.+Sirrenberg&title=Triazole+Derivatives+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEggenweiler%26aufirst%3DH.-M.%26atitle%3DTriazole%2520Derivatives%2520II%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rechlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocchia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, H.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Kinetic and Structural Insights into the Mechanism of Binding of Sulfonamides to Human Carbonic Anhydrase by Computational and Experimental Studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4245</span>– <span class="NLM_lpage">4256</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01643</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01643" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4245-4256&issue=9&author=R.+Gaspariauthor=C.+Rechlinauthor=A.+Heineauthor=G.+Bottegoniauthor=W.+Rocchiaauthor=D.+Schwarzauthor=J.+Bomkeauthor=H.-D.+Gerberauthor=G.+Klebeauthor=A.+Cavalli&title=Kinetic+and+Structural+Insights+into+the+Mechanism+of+Binding+of+Sulfonamides+to+Human+Carbonic+Anhydrase+by+Computational+and+Experimental+Studies&doi=10.1021%2Facs.jmedchem.5b01643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and Structural Insights into the Mechanism of Binding of Sulfonamides to Human Carbonic Anhydrase by Computational and Experimental Studies</span></div><div class="casAuthors">Gaspari, Roberto; Rechlin, Chris; Heine, Andreas; Bottegoni, Giovanni; Rocchia, Walter; Schwarz, Daniel; Bomke, Jorg; Gerber, Hans-Dieter; Klebe, Gerhard; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4245-4256</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of sulfonamides to human carbonic anhydrase II (hCAII) is a complex and long-debated example of protein-ligand recognition and interaction.  In this study, we investigate the para-substituted n-alkyl and hydroxyethylene-benzenesulfonamides, providing a complete reconstruction of their binding pathway to hCAII by means of large-scale mol. dynamics simulations, d. functional calcns., surface plasmon resonance (SPR) measurements, and X-ray crystallog. expts.  Our anal. shows that the protein-ligand assocn. rate (kon) dramatically increases with the ligand's hydrophobicity, pointing to the existence of a prebinding stage largely stabilized by a favorable packing of the ligand's apolar moieties with the hCAII "hydrophobic wall".  The characterization of the binding pathway allows an unprecedented understanding of the structure-kinetic relationship in hCAII/benzenesulfonamide complexes, depicting a paradigmatic scenario for the multistep binding process in protein-ligand systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWXT8lIkKp8LVg90H21EOLACvtfcHk0lhCFmWxX3zxsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVygsbbF&md5=433134a082e1b9e33e0dae0af2cc07c8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01643%26sid%3Dliteratum%253Aachs%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DRechlin%26aufirst%3DC.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DSchwarz%26aufirst%3DD.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DGerber%26aufirst%3DH.-D.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DKinetic%2520and%2520Structural%2520Insights%2520into%2520the%2520Mechanism%2520of%2520Binding%2520of%2520Sulfonamides%2520to%2520Human%2520Carbonic%2520Anhydrase%2520by%2520Computational%2520and%2520Experimental%2520Studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4245%26epage%3D4256%26doi%3D10.1021%2Facs.jmedchem.5b01643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capelli, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entrena
Guadix, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costantino, G.</span></span> <span> </span><span class="NLM_article-title">Unbinding Pathways from the Glucocorticoid Receptor Shed Light on the Reduced Sensitivity of Glucocorticoid Ligands to a Naturally Occurring, Clinically Relevant Mutant Receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">7003</span>– <span class="NLM_lpage">7014</span>, <span class="refDoi"> DOI: 10.1021/jm400802b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400802b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7003-7014&issue=17&author=A.+M.+Capelliauthor=A.+Brunoauthor=A.+Entrena%0AGuadixauthor=G.+Costantino&title=Unbinding+Pathways+from+the+Glucocorticoid+Receptor+Shed+Light+on+the+Reduced+Sensitivity+of+Glucocorticoid+Ligands+to+a+Naturally+Occurring%2C+Clinically+Relevant+Mutant+Receptor&doi=10.1021%2Fjm400802b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm400802b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400802b%26sid%3Dliteratum%253Aachs%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DBruno%26aufirst%3DA.%26aulast%3DEntrena%2BGuadix%26aufirst%3DA.%26aulast%3DCostantino%26aufirst%3DG.%26atitle%3DUnbinding%2520Pathways%2520from%2520the%2520Glucocorticoid%2520Receptor%2520Shed%2520Light%2520on%2520the%2520Reduced%2520Sensitivity%2520of%2520Glucocorticoid%2520Ligands%2520to%2520a%2520Naturally%2520Occurring%252C%2520Clinically%2520Relevant%2520Mutant%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D17%26spage%3D7003%26epage%3D7014%26doi%3D10.1021%2Fjm400802b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Vivo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Role of Molecular Dynamics and Related Methods in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4035</span>– <span class="NLM_lpage">4061</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4035-4061&issue=9&author=M.+De+Vivoauthor=M.+Masettiauthor=G.+Bottegoniauthor=A.+Cavalli&title=Role+of+Molecular+Dynamics+and+Related+Methods+in+Drug+Discovery&doi=10.1021%2Facs.jmedchem.5b01684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></div><div class="casAuthors">De Vivo, Marco; Masetti, Matteo; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4035-4061</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery.  Their main advantage is in explicitly treating structural flexibility and entropic effects.  This allows a more accurate est. of the thermodn. and kinetics assocd. with drug-target recognition and binding, as better algorithms and hardware architectures increase their use.  Here, we review the theor. background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug-target binding.  We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand-target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy.  Further issues discussed include allosteric modulation and the role of water mols. in ligand binding and optimization.  We conclude by calling for more prospective studies to attest to these methods' utility in discovering novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQb_02a7Nmh7Vg90H21EOLACvtfcHk0ljIwpbu-bILyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D&md5=b3d8a4fb5705a3555c2bb2a962420396</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRole%2520of%2520Molecular%2520Dynamics%2520and%2520Related%2520Methods%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4035%26epage%3D4061%26doi%3D10.1021%2Facs.jmedchem.5b01684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mollica, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decherchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zia, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocchia, W.</span></span> <span> </span><span class="NLM_article-title">Kinetics of Protein-Ligand Unbinding via Smoothed Potential Molecular Dynamics Simulations</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11539</span>, <span class="refDoi"> DOI: 10.1038/srep11539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fsrep11539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=26103621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC2MbmsFKmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11539&author=L.+Mollicaauthor=S.+Decherchiauthor=S.+R.+Ziaauthor=R.+Gaspariauthor=A.+Cavalliauthor=W.+Rocchia&title=Kinetics+of+Protein-Ligand+Unbinding+via+Smoothed+Potential+Molecular+Dynamics+Simulations&doi=10.1038%2Fsrep11539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of protein-ligand unbinding via smoothed potential molecular dynamics simulations</span></div><div class="casAuthors">Mollica Luca; Decherchi Sergio; Zia Syeda Rehana; Gaspari Roberto; Rocchia Walter; Cavalli Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug discovery is expensive and high-risk.  Its main reasons of failure are lack of efficacy and toxicity of a drug candidate.  Binding affinity for the biological target has been usually considered one of the most relevant figures of merit to judge a drug candidate along with bioavailability, selectivity and metabolic properties, which could depend on off-target interactions.  Nevertheless, affinity does not always satisfactorily correlate with in vivo drug efficacy.  It is indeed becoming increasingly evident that the time a drug spends in contact with its target (aka residence time) can be a more reliable figure of merit.  Experimental kinetic measurements are operatively limited by the cost and the time needed to synthesize compounds to be tested, to express and purify the target, and to setup the assays.  We present here a simple and efficient molecular-dynamics-based computational approach to prioritize compounds according to their residence time.  We devised a multiple-replica scaled molecular dynamics protocol with suitably defined harmonic restraints to accelerate the unbinding events while preserving the native fold.  Ligands are ranked according to the mean observed scaled unbinding time.  The approach, trivially parallel and easily implementable, was validated against experimental information available on biological systems of pharmacological relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxJkC0jZN2SEH3Rlp_k91zfW6udTcc2eZViVvHuVFkCLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbmsFKmsg%253D%253D&md5=8e3963bce57cf49954ea50fc004283d6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fsrep11539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11539%26sid%3Dliteratum%253Aachs%26aulast%3DMollica%26aufirst%3DL.%26aulast%3DDecherchi%26aufirst%3DS.%26aulast%3DZia%26aufirst%3DS.%2BR.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DRocchia%26aufirst%3DW.%26atitle%3DKinetics%2520of%2520Protein-Ligand%2520Unbinding%2520via%2520Smoothed%2520Potential%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11539%26doi%3D10.1038%2Fsrep11539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span> <span> </span><span class="NLM_article-title">Role of Desolvation in Thermodynamics and Kinetics of Ligand Binding to a Kinase</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">5696</span>– <span class="NLM_lpage">5705</span>, <span class="refDoi"> DOI: 10.1021/ct500584n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct500584n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFWlu7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=5696-5705&issue=12&author=J.+Mondalauthor=R.+A.+Friesnerauthor=B.+J.+Berne&title=Role+of+Desolvation+in+Thermodynamics+and+Kinetics+of+Ligand+Binding+to+a+Kinase&doi=10.1021%2Fct500584n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Role of desolvation in thermodynamics and kinetics of ligand binding to a kinase</span></div><div class="casAuthors">Mondal, Jagannath; Friesner, Richard A.; Berne, B. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5696-5705</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computer simulations were used to det. the free energy landscape for the binding of the anticancer drug, Dasatinib, to its src kinase receptor and showed that before settling into a free energy basin the ligand must surmount a free energy barrier.  An anal. based on using both the ligand-pocket sepn. and the pocket-water occupancy as reaction coordinates showed that the free energy barrier was a result of the free energy cost for almost complete desolvation of the binding pocket.  The simulations further showed that the barrier was not a result of the reorganization free energy of the binding pocket.  Although a continuum solvent model gives the location of free energy min., it is not able to reproduce the intermediate free energy barrier.  Finally, it was shown that a kinetic model for the on-rate const. in which the ligand diffuses up to a doorway state and then surmounts the desolvation free energy barrier was consistent with published microsecond time-scale simulations of the ligand binding kinetics for this system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT08E1EfK8nLVg90H21EOLACvtfcHk0ljIwpbu-bILyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWlu7nE&md5=e7d134fead395023a230489455d61e69</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fct500584n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct500584n%26sid%3Dliteratum%253Aachs%26aulast%3DMondal%26aufirst%3DJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26atitle%3DRole%2520of%2520Desolvation%2520in%2520Thermodynamics%2520and%2520Kinetics%2520of%2520Ligand%2520Binding%2520to%2520a%2520Kinase%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26issue%3D12%26spage%3D5696%26epage%3D5705%26doi%3D10.1021%2Fct500584n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Colizzi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perozzo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scapozza, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recanatini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Single-Molecule Pulling Simulations Can Discern Active from Inactive Enzyme Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">7361</span>– <span class="NLM_lpage">7371</span>, <span class="refDoi"> DOI: 10.1021/ja100259r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja100259r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=7361-7371&issue=21&author=F.+Colizziauthor=R.+Perozzoauthor=L.+Scapozzaauthor=M.+Recanatiniauthor=A.+Cavalli&title=Single-Molecule+Pulling+Simulations+Can+Discern+Active+from+Inactive+Enzyme+Inhibitors&doi=10.1021%2Fja100259r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fja100259r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja100259r%26sid%3Dliteratum%253Aachs%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DPerozzo%26aufirst%3DR.%26aulast%3DScapozza%26aufirst%3DL.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSingle-Molecule%2520Pulling%2520Simulations%2520Can%2520Discern%2520Active%2520from%2520Inactive%2520Enzyme%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26issue%3D21%26spage%3D7361%26epage%3D7371%26doi%3D10.1021%2Fja100259r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery: Pulled from a Protein’s Embrace</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span> (<span class="NLM_issue">7302</span>),  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1038/466042a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2F466042a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=42-43&issue=7302&author=W.+L.+Jorgensen&title=Drug+Discovery%3A+Pulled+from+a+Protein%E2%80%99s+Embrace&doi=10.1038%2F466042a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F466042a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F466042a%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDrug%2520Discovery%253A%2520Pulled%2520from%2520a%2520Protein%25E2%2580%2599s%2520Embrace%26jtitle%3DNature%26date%3D2010%26volume%3D466%26issue%3D7302%26spage%3D42%26epage%3D43%26doi%3D10.1038%2F466042a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G.</span>, <span class="NLM_string-name">Swinney, D. C.</span></span>, Eds.  <i>Thermodynamics and Kinetics of Drug Binding</i>; Methods and Principles in Medicinal Chemistry, Vol. 65; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">329</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2F9783527673025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=315-329&author=G.+Keseru&author=D.+C.+Swinney&title=Thermodynamics+and+Kinetics+of+Drug+Binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2F9783527673025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527673025%26sid%3Dliteratum%253Aachs%26aulast%3DKeseru%26aufirst%3DG.%26btitle%3DThermodynamics%2520and%2520Kinetics%2520of%2520Drug%2520Binding%26pub%3DWiley%26date%3D2015%26spage%3D315%26epage%3D329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Molecular Determinants of Drug-Receptor Binding Kinetics</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">13–14</span>),  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">673</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.drudis.2013.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=23454741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVyrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=667-673&issue=13%E2%80%9314&author=A.+C.+Panauthor=D.+W.+Borhaniauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Molecular+Determinants+of+Drug-Receptor+Binding+Kinetics&doi=10.1016%2Fj.drudis.2013.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular determinants of drug-receptor binding kinetics</span></div><div class="casAuthors">Pan, Albert C.; Borhani, David W.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">667-673</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  It is increasingly appreciated that the rates at which drugs assoc. with and dissoc. from receptors - the binding kinetics - directly impact drug efficacy and safety.  The mol. determinants of drug-receptor binding kinetics remain poorly understood, however, esp. when compared with the well-known factors that affect binding affinity.  The rational modulation of kinetics during lead optimization thus remains challenging.  We review some of the key factors thought to control drug-receptor binding kinetics at the mol. level - mol. size, conformational fluctuations, electrostatic interactions and hydrophobic effects - and discuss several possible approaches for the rational design of drugs with desired binding kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAvEt3tx06Z7Vg90H21EOLACvtfcHk0lj87GqvUKKKtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVyrtr0%253D&md5=df094bb86b1e280b9ddf5a90031b3f32</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DMolecular%2520Determinants%2520of%2520Drug-Receptor%2520Binding%2520Kinetics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26issue%3D13%25E2%2580%259314%26spage%3D667%26epage%3D673%26doi%3D10.1016%2Fj.drudis.2013.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Setny, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzubiella, J.</span></span> <span> </span><span class="NLM_article-title">Dewetting-Controlled Binding of Ligands to Hydrophobic Pockets</span>. <i>Phys. Rev. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">187801</span>, <span class="refDoi"> DOI: 10.1103/PhysRevLett.103.187801</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1103%2FPhysRevLett.103.187801" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=19905832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlKit7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2009&pages=187801&issue=18&author=P.+Setnyauthor=Z.+Wangauthor=L.-T.+Chengauthor=B.+Liauthor=J.+A.+McCammonauthor=J.+Dzubiella&title=Dewetting-Controlled+Binding+of+Ligands+to+Hydrophobic+Pockets&doi=10.1103%2FPhysRevLett.103.187801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Dewetting-Controlled Binding of Ligands to Hydrophobic Pockets</span></div><div class="casAuthors">Setny, P.; Wang, Z.; Cheng, L.-T.; Li, B.; McCammon, J. A.; Dzubiella, J.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">187801/1-187801/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">We report on a combined atomistic mol. dynamics simulation and implicit solvent anal. of a generic hydrophobic pocket-ligand (host-guest) system.  The approaching ligand induces complex wetting-dewetting transitions in the weakly solvated pocket.  The transitions lead to bimodal solvent fluctuations which govern magnitude and range of the pocket-ligand attraction.  A recently developed implicit water model, based on the minimization of a geometric functional, captures the sensitive aq. interface response to the concave-convex pocket-ligand configuration semiquant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ1xmGB8ESgrVg90H21EOLACvtfcHk0lhN7lM59H6p-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlKit7zP&md5=f387b92e561428e0c3cb860eefa3d530</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.103.187801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.103.187801%26sid%3Dliteratum%253Aachs%26aulast%3DSetny%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DCheng%26aufirst%3DL.-T.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DDzubiella%26aufirst%3DJ.%26atitle%3DDewetting-Controlled%2520Binding%2520of%2520Ligands%2520to%2520Hydrophobic%2520Pockets%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2009%26volume%3D103%26issue%3D18%26spage%3D187801%26doi%3D10.1103%2FPhysRevLett.103.187801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhtiyarova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springman, E. B.</span></span> <span> </span><span class="NLM_article-title">A Biacore Biosensor Method for Detailed Kinetic Binding Analysis of Small Molecule Inhibitors of P38α Mitogen-Activated Protein Kinase</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>325</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2003.10.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.ab.2003.10.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2004&pages=126-136&issue=1&author=D.+Casperauthor=M.+Bukhtiyarovaauthor=E.+B.+Springman&title=A+Biacore+Biosensor+Method+for+Detailed+Kinetic+Binding+Analysis+of+Small+Molecule+Inhibitors+of+P38%CE%B1+Mitogen-Activated+Protein+Kinase&doi=10.1016%2Fj.ab.2003.10.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2003.10.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2003.10.025%26sid%3Dliteratum%253Aachs%26aulast%3DCasper%26aufirst%3DD.%26aulast%3DBukhtiyarova%26aufirst%3DM.%26aulast%3DSpringman%26aufirst%3DE.%2BB.%26atitle%3DA%2520Biacore%2520Biosensor%2520Method%2520for%2520Detailed%2520Kinetic%2520Binding%2520Analysis%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520P38%25CE%25B1%2520Mitogen-Activated%2520Protein%2520Kinase%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D325%26issue%3D1%26spage%3D126%26epage%3D136%26doi%3D10.1016%2Fj.ab.2003.10.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winquist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavsson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deinum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span> <span> </span><span class="NLM_article-title">Identification of Structural–Kinetic and Structural–Thermodynamic Relationships for Thrombin Inhibitors</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">613</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1021/bi301333z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi301333z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=613-626&issue=4&author=J.+Winquistauthor=S.+Geschwindnerauthor=Y.+Xueauthor=L.+Gustavssonauthor=D.+Musilauthor=J.+Deinumauthor=U.+H.+Danielson&title=Identification+of+Structural%E2%80%93Kinetic+and+Structural%E2%80%93Thermodynamic+Relationships+for+Thrombin+Inhibitors&doi=10.1021%2Fbi301333z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fbi301333z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi301333z%26sid%3Dliteratum%253Aachs%26aulast%3DWinquist%26aufirst%3DJ.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DXue%26aufirst%3DY.%26aulast%3DGustavsson%26aufirst%3DL.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DDeinum%26aufirst%3DJ.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26atitle%3DIdentification%2520of%2520Structural%25E2%2580%2593Kinetic%2520and%2520Structural%25E2%2580%2593Thermodynamic%2520Relationships%2520for%2520Thrombin%2520Inhibitors%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26issue%3D4%26spage%3D613%26epage%3D626%26doi%3D10.1021%2Fbi301333z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, H.-J.</span></span> <span> </span><span class="NLM_article-title">Binding of Ligands to GPCRs – How Valid Is a Thermodynamic Discrimination of Antagonists and Agonists?</span>. <i>J. Phys. Chem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>S1</i></span>,  <span class="NLM_fpage">001</span>, <span class="refDoi"> DOI: 10.4172/2161-0398.S1-001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.4172%2F2161-0398.S1-001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=S1&publication_year=2012&pages=001&author=A.+Strasserauthor=H.-J.+Wittmann&title=Binding+of+Ligands+to+GPCRs+%E2%80%93+How+Valid+Is+a+Thermodynamic+Discrimination+of+Antagonists+and+Agonists%3F&doi=10.4172%2F2161-0398.S1-001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.4172%2F2161-0398.S1-001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F2161-0398.S1-001%26sid%3Dliteratum%253Aachs%26aulast%3DStrasser%26aufirst%3DA.%26aulast%3DWittmann%26aufirst%3DH.-J.%26atitle%3DBinding%2520of%2520Ligands%2520to%2520GPCRs%2520%25E2%2580%2593%2520How%2520Valid%2520Is%2520a%2520Thermodynamic%2520Discrimination%2520of%2520Antagonists%2520and%2520Agonists%253F%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Biophys.%26date%3D2012%26volume%3DS1%26spage%3D001%26doi%3D10.4172%2F2161-0398.S1-001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shuman, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hämäläinen, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span> <span> </span><span class="NLM_article-title">Kinetic and Thermodynamic Characterization of HIV-1 Protease Inhibitors</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1002/jmr.655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2Fjmr.655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=15027031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1altLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2004&pages=106-119&issue=2&author=C.+F.+Shumanauthor=M.+D.+H%C3%A4m%C3%A4l%C3%A4inenauthor=U.+H.+Danielson&title=Kinetic+and+Thermodynamic+Characterization+of+HIV-1+Protease+Inhibitors&doi=10.1002%2Fjmr.655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and thermodynamic characterization of HIV-1 protease inhibitors</span></div><div class="casAuthors">Shuman, Cynthia F.; Haemaelaeinen, Markku D.; Danielson, U. Helena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">106-119</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Interaction kinetic and thermodn. analyses provide information beyond that obtained in general inhibition studies, and may contribute to the design of improved inhibitors and increased understanding of mol. interactions.  Thus, a biosensor-based method was used to characterize the interactions between HIV-1 protease and seven inhibitors, revealing distinguishing kinetic and thermodn. characteristics for the inhibitors.  Lopinavir had fast assocn. and the highest affinity of the tested compds., and the interaction kinetics were less temp.-dependent as compared with the other inhibitors.  Amprenavir, indinavir and ritonavir showed non-linear temp. dependencies of the kinetics.  The free energy, enthalpy and entropy (ΔG, ΔH, ΔS) were detd., and the energetics of complex assocn. (ΔGon, ΔHon, ΔSon) and dissocn. (ΔGoff, ΔHoff, ΔSoff) were resolved.  In general, the energetics for the studied inhibitors was in the same range, with the neg. free energy change (ΔG < 0) due primarily to increased entropy (ΔS > 0).  Thus, the driving force of the interaction was increased degrees of freedom in the system (entropy) rather than the formation of bonds between the enzyme and inhibitor (enthalpy).  Although the ΔGon and ΔGoff were in the same range for all inhibitors, the enthalpy and entropy terms contributed differently to assocn. and dissocn., distinguishing these phases energetically.  Dissocn. was accompanied by pos. enthalpy (ΔHoff > 0) and neg. entropy (ΔSoff < 0) changes, whereas assocn. for all inhibitors except lopinavir had pos. entropy changes (ΔSon > 0), demonstrating unique energetic characteristics for lopinavir.  This study indicates that this type of data will be useful for the characterization of target-ligand interactions and the development of new inhibitors of HIV-1 protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr6QEHUdEX6bVg90H21EOLACvtfcHk0ljum-ARP78L5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1altLk%253D&md5=a5e3e6273f1cc03d2454ecfce598da30</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjmr.655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.655%26sid%3Dliteratum%253Aachs%26aulast%3DShuman%26aufirst%3DC.%2BF.%26aulast%3DH%25C3%25A4m%25C3%25A4l%25C3%25A4inen%26aufirst%3DM.%2BD.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26atitle%3DKinetic%2520and%2520Thermodynamic%2520Characterization%2520of%2520HIV-1%2520Protease%2520Inhibitors%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2004%26volume%3D17%26issue%3D2%26spage%3D106%26epage%3D119%26doi%3D10.1002%2Fjmr.655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajpai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higazi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span> <span> </span><span class="NLM_article-title">Structural and Dynamic Insights into the Energetics of Activation Loop Rearrangement in FGFR1 Kinase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">7877</span>, <span class="refDoi"> DOI: 10.1038/ncomms8877</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fncomms8877" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=26203596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FmvVOktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7877&author=T.+Kleinauthor=N.+Vajpaiauthor=J.+J.+Phillipsauthor=G.+Daviesauthor=G.+A.+Holdgateauthor=C.+Phillipsauthor=J.+A.+Tuckerauthor=R.+A.+Normanauthor=A.+D.+Scottauthor=D.+R.+Higaziauthor=D.+Loweauthor=G.+S.+Thompsonauthor=A.+L.+Breeze&title=Structural+and+Dynamic+Insights+into+the+Energetics+of+Activation+Loop+Rearrangement+in+FGFR1+Kinase&doi=10.1038%2Fncomms8877"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and dynamic insights into the energetics of activation loop rearrangement in FGFR1 kinase</span></div><div class="casAuthors">Klein Tobias; Vajpai Navratna; Davies Gareth; Holdgate Geoffrey A; Phillips Chris; Tucker Julie A; Norman Richard A; Scott Andrew D; Breeze Alexander L; Phillips Jonathan J; Higazi Daniel R; Lowe David; Thompson Gary S; Breeze Alexander L</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7877</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein tyrosine kinases differ widely in their propensity to undergo rearrangements of the N-terminal Asp-Phe-Gly (DFG) motif of the activation loop, with some, including FGFR1 kinase, appearing refractory to this so-called 'DFG flip'.  Recent inhibitor-bound structures have unexpectedly revealed FGFR1 for the first time in a 'DFG-out' state.  Here we use conformationally selective inhibitors as chemical probes for interrogation of the structural and dynamic features that appear to govern the DFG flip in FGFR1.  Our detailed structural and biophysical insights identify contributions from altered dynamics in distal elements, including the αH helix, towards the outstanding stability of the DFG-out complex with the inhibitor ponatinib.  We conclude that the αC-β4 loop and 'molecular brake' regions together impose a high energy barrier for this conformational rearrangement, and that this may have significance for maintaining autoinhibition in the non-phosphorylated basal state of FGFR1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSmhfFatniy2WmbXvSEQle5fW6udTcc2eYhmktrcpyX9bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FmvVOktQ%253D%253D&md5=ddc5f597e82c3432831a86c65dac2f26</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fncomms8877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8877%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DT.%26aulast%3DVajpai%26aufirst%3DN.%26aulast%3DPhillips%26aufirst%3DJ.%2BJ.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DHoldgate%26aufirst%3DG.%2BA.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DScott%26aufirst%3DA.%2BD.%26aulast%3DHigazi%26aufirst%3DD.%2BR.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DThompson%26aufirst%3DG.%2BS.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26atitle%3DStructural%2520and%2520Dynamic%2520Insights%2520into%2520the%2520Energetics%2520of%2520Activation%2520Loop%2520Rearrangement%2520in%2520FGFR1%2520Kinase%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7877%26doi%3D10.1038%2Fncomms8877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutzner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span> <span> </span><span class="NLM_article-title">GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1021/ct700301q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700301q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=435-447&issue=3&author=B.+Hessauthor=C.+Kutznerauthor=D.+van+der%0ASpoelauthor=E.+Lindahl&title=GROMACS+4%3A+Algorithms+for+Highly+Efficient%2C+Load-Balanced%2C+and+Scalable+Molecular+Simulation&doi=10.1021%2Fct700301q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation</span></div><div class="casAuthors">Hess, Berk; Kutzner, Carsten; van der Spoel, David; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">435-447</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. simulation is an extremely useful, but computationally very expensive tool for studies of chem. and biomol. systems.  Here, we present a new implementation of our mol. simulation toolkit GROMACS which now both achieves extremely high performance on single processors from algorithmic optimizations and hand-coded routines and simultaneously scales very well on parallel machines.  The code encompasses a minimal-communication domain decompn. algorithm, full dynamic load balancing, a state-of-the-art parallel constraint solver, and efficient virtual site algorithms that allow removal of hydrogen atom degrees of freedom to enable integration time steps up to 5 fs for atomistic simulations also in parallel.  To improve the scaling properties of the common particle mesh Ewald electrostatics algorithms, we have in addn. used a Multiple-Program, Multiple-Data approach, with sep. node domains responsible for direct and reciprocal space interactions.  Not only does this combination of algorithms enable extremely long simulations of large systems but also it provides that simulation performance on quite modest nos. of std. cluster nodes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrajjWdk6mUFrVg90H21EOLACvtfcHk0lgFeZft8WLhdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D&md5=d53c94901386260221792ea30f151c5f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fct700301q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700301q%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DKutzner%26aufirst%3DC.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGROMACS%25204%253A%2520Algorithms%2520for%2520Highly%2520Efficient%252C%2520Load-Balanced%252C%2520and%2520Scalable%2520Molecular%2520Simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26issue%3D3%26spage%3D435%26epage%3D447%26doi%3D10.1021%2Fct700301q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Der Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groenhof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">GROMACS: Fast, Flexible, and Free</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&issue=16&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+C.+Berendsen&title=GROMACS%3A+Fast%2C+Flexible%2C+and+Free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0lgFeZft8WLhdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DGROMACS%253A%2520Fast%252C%2520Flexible%252C%2520and%2520Free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26issue%3D16%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span> <span> </span><span class="NLM_article-title">GROMACS 3.0: A Package for Molecular Simulation and Trajectory Analysis</span>. <i>Mol. Model. Annu.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1007/s008940100045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1007%2Fs008940100045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsVGjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=306-317&issue=8&author=E.+Lindahlauthor=B.+Hessauthor=D.+van+der%0ASpoel&title=GROMACS+3.0%3A+A+Package+for+Molecular+Simulation+and+Trajectory+Analysis&doi=10.1007%2Fs008940100045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 3.0: a package for molecular simulation and trajectory analysis</span></div><div class="casAuthors">Lindahl, Erik; Hess, Berk; van der Spoel, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling [online computer file]</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">306-317</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">GROMACS 3.0 is the latest release of a versatile and very well optimized package for mol. simulation.  Much effort has been devoted to achieving extremely high performance on both workstations and parallel comouters.  The design includes an extn. of virial and periodic boundary conditions from the loops over pairwise interactions, and special software routines to enable rapid calcn. of x-1/2.  Inner loops are generated automatically in C or Fortran at compile time, with optimizations adapted to each architecture.  Assembly loops using SSE and 3DNow! multimedia instructions are provided for x86 processors, resulting in exceptional performance on inexpensive PC workstations.  The interface is simple and easy to use, based on std. command line arguments with self-explanatory functionality and integrated documentation.  A large collection of flexible tools for trajectory anal. is included, with output in the form of finished Xmgr/Grace graphs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2xBKmKIYW3bVg90H21EOLACvtfcHk0lgFeZft8WLhdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsVGjsL4%253D&md5=fca1fafdbbab2f42b3ab9c789b8a048e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2Fs008940100045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs008940100045%26sid%3Dliteratum%253Aachs%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26atitle%3DGROMACS%25203.0%253A%2520A%2520Package%2520for%2520Molecular%2520Simulation%2520and%2520Trajectory%2520Analysis%26jtitle%3DMol.%2520Model.%2520Annu.%26date%3D2001%26volume%3D7%26issue%3D8%26spage%3D306%26epage%3D317%26doi%3D10.1007%2Fs008940100045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Drunen, R.</span></span> <span> </span><span class="NLM_article-title">GROMACS: A Message-Passing Parallel Molecular Dynamics Implementation</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>91</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/0010-4655(95)00042-E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2F0010-4655%2895%2900042-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaK2MXps1Wrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1995&pages=43-56&issue=1&author=H.+J.+C.+Berendsenauthor=D.+van+der+Spoelauthor=R.+van+Drunen&title=GROMACS%3A+A+Message-Passing+Parallel+Molecular+Dynamics+Implementation&doi=10.1016%2F0010-4655%2895%2900042-E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: A message-passing parallel molecular dynamics implementation</span></div><div class="casAuthors">Berendsen, H. J. C.; van der Spoel, D.; van Drunen, R.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">43-56</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A parallel message-passing implementation of a mol. dynamics (MD) program that is useful for bio(macro)mols. in aq. environment is described.  The software has been developed for a custom-designed 32-processor ring GROMACS (Groningen MAchine for Chem. Simulation) with communication to and from left and right neighbors, but can run on any parallel system onto which a a ring of processors can be mapped and which supports PVM-like block send and receive calls.  The GROMACS software consists of a preprocessor, a parallel MD and energy minimization program that can use an arbitrary no. of processors (including one), an optional monitor, and several anal. tools.  The programs are written in ANSI C and available by ftp (information: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="bfd8cdd0d2dedcccffdcd7dad291cdcad891d1d3">[email protected]</a>).  The functionality is based on the GROMOS (Groningen Mol. Simulation) package (van Gunsteren and Berendsen, 1987; BIOMOS B.V., Nijenborgh 4, 9747 AG Groningen).  Conversion programs between GROMOS and GROMACS formats are included.The MD program can handle rectangular periodic boundary conditions with temp. and pressure scaling.  The interactions that can be handled without modification are variable non-bonded pair interactions with Coulomb and Lennard-Jones or Buckingham potentials, using a twin-range cut-off based on charge groups, and fixed bonded interactions of either harmonic or constraint type for bonds and bond angles and either periodic or cosine power series interactions for dihedral angles.  Special forces can be added to groups of particles (for non-equil. dynamics or for position restraining) or between particles (for distance restraints).  The parallelism is based on particle decompn.  Interprocessor communication is largely limited to position and force distribution over the ring once per time step.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovcOlT-z9eKLVg90H21EOLACvtfcHk0li3_W9v5_B1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXps1Wrtr0%253D&md5=04d823aeab28ca374efb86839c705179</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2F0010-4655%2895%2900042-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0010-4655%252895%252900042-E%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3Dvan%2BDrunen%26aufirst%3DR.%26atitle%3DGROMACS%253A%2520A%2520Message-Passing%2520Parallel%2520Molecular%2520Dynamics%2520Implementation%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D1995%26volume%3D91%26issue%3D1%26spage%3D43%26epage%3D56%26doi%3D10.1016%2F0010-4655%2895%2900042-E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichenberger, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choutko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">Definition and Testing of the GROMOS Force-Field Versions 54A7 and 54B7</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">843</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1007/s00249-011-0700-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1007%2Fs00249-011-0700-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=21533652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVCgs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=843-856&issue=7&author=N.+Schmidauthor=A.+P.+Eichenbergerauthor=A.+Choutkoauthor=S.+Rinikerauthor=M.+Wingerauthor=A.+E.+Markauthor=W.+F.+van+Gunsteren&title=Definition+and+Testing+of+the+GROMOS+Force-Field+Versions+54A7+and+54B7&doi=10.1007%2Fs00249-011-0700-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Definition and testing of the GROMOS force-field versions 54A7 and 54B7</span></div><div class="casAuthors">Schmid, Nathan; Eichenberger, Andreas P.; Choutko, Alexandra; Riniker, Sereina; Winger, Moritz; Mark, Alan E.; Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">843-856</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">New parameter sets of the GROMOS biomol. force field, 54A7 and 54B7, are introduced.  These parameter sets summarize some previously published force field modifications: The 53A6 helical propensities are cor. through new φ/ψ torsional angle terms and a modification of the N-H, C=O repulsion, a new atom type for a charged -CH3 in the choline moiety is added, the Na+ and Cl- ions are modified to reproduce the free energy of hydration, and addnl. improper torsional angle types for free energy calcns. involving a chirality change are introduced.  The new helical propensity modification is tested using the benchmark proteins hen egg-white lysozyme, fox1 RNA binding domain, chorismate mutase and the GCN4-p1 peptide.  The stability of the proteins is improved in comparison with the 53A6 force field, and good agreement with a range of primary exptl. data is obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3tjHc1auDa7Vg90H21EOLACvtfcHk0li3_W9v5_B1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVCgs74%253D&md5=a1cd133f1e663da949324060e00c950d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs00249-011-0700-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00249-011-0700-9%26sid%3Dliteratum%253Aachs%26aulast%3DSchmid%26aufirst%3DN.%26aulast%3DEichenberger%26aufirst%3DA.%2BP.%26aulast%3DChoutko%26aufirst%3DA.%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DWinger%26aufirst%3DM.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DDefinition%2520and%2520Testing%2520of%2520the%2520GROMOS%2520Force-Field%2520Versions%252054A7%2520and%252054B7%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D2011%26volume%3D40%26issue%3D7%26spage%3D843%26epage%3D856%26doi%3D10.1007%2Fs00249-011-0700-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bermudez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgueitio, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolber, G.</span></span> <span> </span><span class="NLM_article-title">The Impact of Molecular Dynamics on Drug Design: Applications for the Characterization of Ligand-Macromolecule Complexes</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.drudis.2015.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=25615716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFSjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=686-702&issue=6&author=J.+Mortierauthor=C.+Rakersauthor=M.+Bermudezauthor=M.+S.+Murgueitioauthor=S.+Rinikerauthor=G.+Wolber&title=The+Impact+of+Molecular+Dynamics+on+Drug+Design%3A+Applications+for+the+Characterization+of+Ligand-Macromolecule+Complexes&doi=10.1016%2Fj.drudis.2015.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of molecular dynamics on drug design: applications for the characterization of ligand-macromolecule complexes</span></div><div class="casAuthors">Mortier, Jeremie; Rakers, Christin; Bermudez, Marcel; Murgueitio, Manuela S.; Riniker, Sereina; Wolber, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">686-702</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Among all tools available to design new drugs, mol. dynamics (MD) simulations have become an essential technique.  Initially developed to investigate mol. models with a limited no. of atoms, computers now enable investigations of large macromol. systems with a simulation time reaching the microsecond range.  The reviewed articles cover four years of research to give an overview on the actual impact of MD on the current medicinal chem. landscape with a particular emphasis on studies of ligand-protein interactions.  With a special focus on studies combining computational approaches with data gained from other techniques, this review shows how deeply embedded MD simulations are in drug design strategies and articulates what the future of this technique could be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaq-ZlpVGoG7Vg90H21EOLACvtfcHk0lgYgnxKJca_8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFSjtbk%253D&md5=36aba5288f374baf50b2831225047c7c</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DMortier%26aufirst%3DJ.%26aulast%3DRakers%26aufirst%3DC.%26aulast%3DBermudez%26aufirst%3DM.%26aulast%3DMurgueitio%26aufirst%3DM.%2BS.%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DWolber%26aufirst%3DG.%26atitle%3DThe%2520Impact%2520of%2520Molecular%2520Dynamics%2520on%2520Drug%2520Design%253A%2520Applications%2520for%2520the%2520Characterization%2520of%2520Ligand-Macromolecule%2520Complexes%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26issue%3D6%26spage%3D686%26epage%3D702%26doi%3D10.1016%2Fj.drudis.2015.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristòfol-Clough, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Efficient Round-Trip Time Optimization for Replica-Exchange Enveloping Distribution Sampling (RE-EDS)</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">3020</span>– <span class="NLM_lpage">3030</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.7b00286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.7b00286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFChtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=3020-3030&issue=6&author=D.+Sidlerauthor=M.+Crist%C3%B2fol-Cloughauthor=S.+Riniker&title=Efficient+Round-Trip+Time+Optimization+for+Replica-Exchange+Enveloping+Distribution+Sampling+%28RE-EDS%29&doi=10.1021%2Facs.jctc.7b00286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Round-Trip Time Optimization for Replica-Exchange Enveloping Distribution Sampling (RE-EDS)</span></div><div class="casAuthors">Sidler, Dominik; Cristofol-Clough, Michael; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3020-3030</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Replica-exchange enveloping distribution sampling (RE-EDS) allows the efficient estn. of free-energy differences between multiple end-states from a single mol. dynamics (MD) simulation.  In EDS, a ref. state is sampled, which can be tuned by two types of parameters, i.e., smoothness parameters(s) and energy offsets, such that all end-states are sufficiently sampled.  However, the choice of these parameters is not trivial.  Replica exchange (RE) or parallel tempering is a widely applied technique to enhance sampling.  By combining EDS with the RE technique, the parameter choice problem could be simplified and the challenge shifted toward an optimal distribution of the replicas in the smoothness-parameter space.  The choice of a certain replica distribution can alter the sampling efficiency significantly.  In this work, global round-trip time optimization (GRTO) algorithms are tested for the use in RE-EDS simulations.  In addn., a local round-trip time optimization (LRTO) algorithm is proposed for systems with slowly adapting environments, where a reliable est. for the round-trip time is challenging to obtain.  The optimization algorithms were applied to RE-EDS simulations of a system of nine small-mol. inhibitors of phenylethanolamine N-methyltransferase (PNMT).  The energy offsets were detd. using the recently proposed parallel energy-offset (PEOE) estn. scheme.  While the multi-state GRTO algorithm yielded the best replica distribution for the ligands in water, the multi-state LRTO algorithm was found to be the method of choice for the ligands in complex with PNMT.  With this, the 36 alchem. free-energy differences between the nine ligands were calcd. successfully from a single RE-EDS simulation 10 ns in length.  Thus, RE-EDS presents an efficient method for the estn. of relative binding free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKivaoB2NS_7Vg90H21EOLACvtfcHk0lgYgnxKJca_8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFChtrk%253D&md5=97e910746ea8e746ba56ec05c5061e85</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.7b00286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.7b00286%26sid%3Dliteratum%253Aachs%26aulast%3DSidler%26aufirst%3DD.%26aulast%3DCrist%25C3%25B2fol-Clough%26aufirst%3DM.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DEfficient%2520Round-Trip%2520Time%2520Optimization%2520for%2520Replica-Exchange%2520Enveloping%2520Distribution%2520Sampling%2520%2528RE-EDS%2529%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26issue%3D6%26spage%3D3020%26epage%3D3030%26doi%3D10.1021%2Facs.jctc.7b00286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grandits, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oostenbrink, C.</span></span> <span> </span><span class="NLM_article-title">Selectivity of Cytosolic Phospholipase A2 Type IV toward Arachidonyl Phospholipids</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1002/jmr.2462</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2Fjmr.2462" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=25703463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps12qtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=447-457&issue=7&author=M.+Granditsauthor=C.+Oostenbrink&title=Selectivity+of+Cytosolic+Phospholipase+A2+Type+IV+toward+Arachidonyl+Phospholipids&doi=10.1002%2Fjmr.2462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity of cytosolic phospholipase A2 type IV toward arachidonyl phospholipids</span></div><div class="casAuthors">Grandits, Melanie; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">447-457</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cytosolic phospholipase A2 (cPLA2) is an interesting protein involved in inflammatory processes and various diseases.  Its catalytic mechanism as well as its substrate specificity for arachidonyl phospholipids is not typical for other phospolipases.  Furthermore, a lid structure, which ensures a hydrophilic surface of the protein without any substrate bound and the movement of this flexible loop to make the hydrophobic active site accessible, is of high interest.  Therefore, the focus of this work was to det. the binding mode of cPLA2 with various substrates, such as arachidonic acid, a synthetic inhibitor, a satd. phospholipid, and most importantly an arachidonyl phospholipid.  To understand the selectivity of the protein toward the arachidonyl phospholipid and the interaction in a protein-ligand complex, mol. dynamics (MD) simulations were performed using the GROMOS suite of simulation programs.  The MD simulations provided insight into the protein and showed that selective binding of arachidonyl phospholipids was because of the shape of the sn-2 tail.  The cPLA2 residues Asn-555 and Ala-578 were involved in the strongest interactions obsd. in the protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9uTIv9KWpFrVg90H21EOLACvtfcHk0lgYgnxKJca_8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps12qtb4%253D&md5=935bd5fca598ecc56684cd3917a06a9e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fjmr.2462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.2462%26sid%3Dliteratum%253Aachs%26aulast%3DGrandits%26aufirst%3DM.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DSelectivity%2520of%2520Cytosolic%2520Phospholipase%2520A2%2520Type%2520IV%2520toward%2520Arachidonyl%2520Phospholipids%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2015%26volume%3D28%26issue%3D7%26spage%3D447%26epage%3D457%26doi%3D10.1002%2Fjmr.2462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pechlaner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oostenbrink, C.</span></span> <span> </span><span class="NLM_article-title">Multiple Binding Poses in the Hydrophobic Cavity of Bee Odorant Binding Protein AmelOBP14</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2633</span>– <span class="NLM_lpage">2643</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFClsrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2633-2643&issue=12&author=M.+Pechlanerauthor=C.+Oostenbrink&title=Multiple+Binding+Poses+in+the+Hydrophobic+Cavity+of+Bee+Odorant+Binding+Protein+AmelOBP14&doi=10.1021%2Facs.jcim.5b00673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Binding Poses in the Hydrophobic Cavity of Bee Odorant Binding Protein AmelOBP14</span></div><div class="casAuthors">Pechlaner, Maria; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2633-2643</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the first step of olfaction, odorants are bound and solubilized by small globular odorant binding proteins (OBPs) which shuttle them to the membrane of a sensory neuron.  Low ligand affinity and selectivity at this step enable the recognition of a wide range of chems.  Honey bee Apis mellifera's OBP14 (AmelOBP14) binds different plant odorants in a largely hydrophobic cavity.  In long mol. dynamics simulations in the presence and absence of ligand eugenol, we observe a highly dynamic C-terminal region which forms one side of the ligand-binding cavity, and the ligand drifts away from its crystd. orientation.  Hamiltonian replica exchange simulations, allowing exchanges of conformations sampled by the real ligand with those sampled by a noninteracting dummy mol. and several intermediates, suggest an alternative, quite different ligand pose which is adopted immediately and which is stable in long simulations.  Thermodn. integration yields binding free energies which are in reasonable agreement with exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXxIhY6DqduLVg90H21EOLACvtfcHk0lgYgnxKJca_8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFClsrnJ&md5=6466eb870ed2d9b153797d64dd3092c9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00673%26sid%3Dliteratum%253Aachs%26aulast%3DPechlaner%26aufirst%3DM.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DMultiple%2520Binding%2520Poses%2520in%2520the%2520Hydrophobic%2520Cavity%2520of%2520Bee%2520Odorant%2520Binding%2520Protein%2520AmelOBP14%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D12%26spage%3D2633%26epage%3D2643%26doi%3D10.1021%2Facs.jcim.5b00673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span>; <span class="NLM_string-name">Postma, J. P. M.</span>; <span class="NLM_string-name">van Gunsteren, W. F.</span>; <span class="NLM_string-name">Hermans, J.</span></span> <span> </span><span class="NLM_article-title">Interaction Models for Water in Relation to Protein Hydration</span>. In  <i>Intermolecular Forces</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pullman, B.</span></span>, Ed.; The Jerusalem Symposia on Quantum Chemistry and Biochemistry; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Dordrecht, The Netherlands</span>, <span class="NLM_year">1981</span>; pp  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">342</span>, DOI: <span class="refDoi"> DOI: 10.1007/978-94-015-7658-1_21</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1007%2F978-94-015-7658-1_21" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1981&pages=331-342&author=H.+J.+C.+Berendsen&author=J.+P.+M.+Postma&author=W.+F.+van+Gunsteren&author=J.+Hermansauthor=B.+Pullman&title=Intermolecular+Forces"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2F978-94-015-7658-1_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-015-7658-1_21%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DInteraction%2520Models%2520for%2520Water%2520in%2520Relation%2520to%2520Protein%2520Hydration%26btitle%3DIntermolecular%2520Forces%26aulast%3DPullman%26aufirst%3DB.%26pub%3DSpringer%26date%3D1981%26spage%3D331%26epage%3D342%26doi%3D10.1007%2F978-94-015-7658-1_21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, J. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haak, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics with Coupling to an External Bath</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">3684</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&issue=8&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van+Gunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+Dynamics+with+Coupling+to+an+External+Bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0liUk3q78eSNrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520Dynamics%2520with%2520Coupling%2520to%2520an%2520External%2520Bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26issue%3D8%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donadio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrinello, M.</span></span> <span> </span><span class="NLM_article-title">Canonical Sampling through Velocity Rescaling</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>126</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">014101</span>, <span class="refDoi"> DOI: 10.1063/1.2408420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1063%2F1.2408420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=17212484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD2sXosVCltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2007&pages=014101&issue=1&author=G.+Bussiauthor=D.+Donadioauthor=M.+Parrinello&title=Canonical+Sampling+through+Velocity+Rescaling&doi=10.1063%2F1.2408420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Canonical sampling through velocity rescaling</span></div><div class="casAuthors">Bussi, Giovanni; Donadio, Davide; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">014101/1-014101/7</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The authors present a new mol. dynamics algorithm for sampling the canonical distribution.  In this approach the velocities of all the particles are rescaled by a properly chosen random factor.  The algorithm is formally justified and it is shown that, in spite of its stochastic nature, a quantity can still be defined that remains const. during the evolution.  In numerical applications this quantity can be used to measure the accuracy of the sampling.  The authors illustrate the properties of this new method on Lennard-Jones and TIP4P water models in the solid and liq. phases.  Its performance is excellent and largely independent of the thermostat parameter also with regard to the dynamic properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgtMYfaJjM3rVg90H21EOLACvtfcHk0liUk3q78eSNrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXosVCltg%253D%253D&md5=9c182b57bfc65bca6be23c8c76b4be77</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1063%2F1.2408420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2408420%26sid%3Dliteratum%253Aachs%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DDonadio%26aufirst%3DD.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DCanonical%2520Sampling%2520through%2520Velocity%2520Rescaling%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2007%26volume%3D126%26issue%3D1%26spage%3D014101%26doi%3D10.1063%2F1.2408420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraaije, J. G. E. M.</span></span> <span> </span><span class="NLM_article-title">LINCS: A Linear Constraint Solver for Molecular Simulations</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1463</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199709)18:12<1463::AID-JCC4>3.0.CO;2-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=1463-1472&issue=12&author=B.+Hessauthor=H.+Bekkerauthor=H.+J.+C.+Berendsenauthor=J.+G.+E.+M.+Fraaije&title=LINCS%3A+A+Linear+Constraint+Solver+for+Molecular+Simulations&doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">LINCS: a linear constraint solver for molecular simulations</span></div><div class="casAuthors">Hess, Berk; Bekker, Henk; Berendsen, Herman J. C.; Fraaije, Johannes G. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1463-1472</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">We present a new LINear Constraint Solver (LINCS) for mol. simulations with bond constraints using the enzyme lysozyme and a 32-residue peptide as test systems.  The algorithm is inherently stable, as the constraints themselves are reset instead of derivs. of the constraints, thereby eliminating drift.  Although the derivation of the algorithm is presented in terms of matrixes, no matrix matrix multiplications are needed and only the nonzero matrix elements have to be stored, making the method useful for very large mols.  At the same accuracy, the LINCS algorithm is 3-4 times faster than the SHAKE algorithm.  Parallelization of the algorithm is straightforward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk6h4pdV0Nx7Vg90H21EOLACvtfcHk0liUk3q78eSNrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D&md5=890f8af0d2ca1f65aa93db5a3a0bacf2</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.0.CO%253B2-H%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DBekker%26aufirst%3DH.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DFraaije%26aufirst%3DJ.%2BG.%2BE.%2BM.%26atitle%3DLINCS%253A%2520A%2520Linear%2520Constraint%2520Solver%2520for%2520Molecular%2520Simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1997%26volume%3D18%26issue%3D12%26spage%3D1463%26epage%3D1472%26doi%3D10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Essmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L. G.</span></span> <span> </span><span class="NLM_article-title">A Smooth Particle Mesh Ewald Method</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8577</span>– <span class="NLM_lpage">8593</span>, <span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&issue=19&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+Smooth+Particle+Mesh+Ewald+Method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0lio6ZVrPHRdaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520Smooth%2520Particle%2520Mesh%2520Ewald%2520Method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26issue%3D19%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hub, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Groot, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span> <span> </span><span class="NLM_article-title">G_wham—A Free Weighted Histogram Analysis Implementation Including Robust Error and Autocorrelation Estimates</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3713</span>– <span class="NLM_lpage">3720</span>, <span class="refDoi"> DOI: 10.1021/ct100494z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100494z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVegu7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=3713-3720&issue=12&author=J.+S.+Hubauthor=B.+L.+de+Grootauthor=D.+van+der%0ASpoel&title=G_wham%E2%80%94A+Free+Weighted+Histogram+Analysis+Implementation+Including+Robust+Error+and+Autocorrelation+Estimates&doi=10.1021%2Fct100494z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">g_wham: A Free Weighted Histogram Analysis Implementation Including Robust Error and Autocorrelation Estimates</span></div><div class="casAuthors">Hub, Jochen S.; de Groot, Bert L.; van der Spoel, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3713-3720</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Weighted Histogram Anal. Method (WHAM) is a std. technique used to compute potentials of mean force (PMFs) from a set of umbrella sampling simulations.  Here, the authors present a new WHAM implementation, termed g_wham, which is distributed freely with the GROMACS mol. simulation suite.  G_wham ests. statistical errors using the technique of bootstrap anal.  Three bootstrap methods are supported: (i) bootstrapping new trajectories based on the umbrella histograms, (ii) bootstrapping of complete histograms, and (iii) Bayesian bootstrapping of complete histograms, i.e., bootstrapping via the assignment of random wts. to the histograms.  Because methods ii and iii consider only complete histograms as independent data points, these methods do not require the accurate calcn. of autocorrelation times.  The authors demonstrate that, given sufficient sampling, bootstrapping new trajectories allows for an accurate error est.  In the presence of long autocorrelations, however, (Bayesian) bootstrapping of complete histograms yields a more reliable error est., whereas bootstrapping of new trajectories may underestimate the error.  In addn., the authors emphasize that the incorporation of autocorrelations into WHAM reduces the bias from limited sampling, in particular, when computing periodic PMFs in inhomogeneous systems such as solvated lipid membranes or protein channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3sB6LvdBAELVg90H21EOLACvtfcHk0lio6ZVrPHRdaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVegu7bI&md5=c798afe576b97471e29040069e434028</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fct100494z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100494z%26sid%3Dliteratum%253Aachs%26aulast%3DHub%26aufirst%3DJ.%2BS.%26aulast%3Dde%2BGroot%26aufirst%3DB.%2BL.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26atitle%3DG_wham%25E2%2580%2594A%2520Free%2520Weighted%2520Histogram%2520Analysis%2520Implementation%2520Including%2520Robust%2520Error%2520and%2520Autocorrelation%2520Estimates%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26issue%3D12%26spage%3D3713%26epage%3D3720%26doi%3D10.1021%2Fct100494z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savitzky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golay, M. J. E.</span></span> <span> </span><span class="NLM_article-title">Smoothing and Differentiation of Data by Simplified Least Squares Procedures</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1627</span>– <span class="NLM_lpage">1639</span>, <span class="refDoi"> DOI: 10.1021/ac60214a047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac60214a047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaF2cXksVCjur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1964&pages=1627-1639&issue=8&author=A.+Savitzkyauthor=M.+J.+E.+Golay&title=Smoothing+and+Differentiation+of+Data+by+Simplified+Least+Squares+Procedures&doi=10.1021%2Fac60214a047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Smoothing and differentiation of data by simplified least squares procedures</span></div><div class="casAuthors">Savitzky, Abraham; Golay, Marcel J. E.</div><div class="citationInfo"><span class="NLM_cas:title"></span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1627-39</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    </div><div class="casAbstract">The operation may be carried out on a computer by convolution of the data.  Two examples are presented as subroutines in the FORTRAN language.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGF5gFu4oYrbVg90H21EOLACvtfcHk0lio6ZVrPHRdaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXksVCjur8%253D&md5=8406f1500e608f8f89eada4d7949ed77</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fac60214a047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac60214a047%26sid%3Dliteratum%253Aachs%26aulast%3DSavitzky%26aufirst%3DA.%26aulast%3DGolay%26aufirst%3DM.%2BJ.%2BE.%26atitle%3DSmoothing%2520and%2520Differentiation%2520of%2520Data%2520by%2520Simplified%2520Least%2520Squares%2520Procedures%26jtitle%3DAnal.%2520Chem.%26date%3D1964%26volume%3D36%26issue%3D8%26spage%3D1627%26epage%3D1639%26doi%3D10.1021%2Fac60214a047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, D. B.</span></span> <span> </span><span class="NLM_article-title">The Bayesian Bootstrap</span>. <i>Ann. Stat.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1214/aos/1176345338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1214%2Faos%2F1176345338" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1981&pages=130-134&issue=1&author=D.+B.+Rubin&title=The+Bayesian+Bootstrap&doi=10.1214%2Faos%2F1176345338"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1214%2Faos%2F1176345338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1214%252Faos%252F1176345338%26sid%3Dliteratum%253Aachs%26aulast%3DRubin%26aufirst%3DD.%2BB.%26atitle%3DThe%2520Bayesian%2520Bootstrap%26jtitle%3DAnn.%2520Stat.%26date%3D1981%26volume%3D9%26issue%3D1%26spage%3D130%26epage%3D134%26doi%3D10.1214%2Faos%2F1176345338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggenweiler, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirrenberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grädler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoppe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">März, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, M.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of Hydroxy-Indazole-Carboxamides as Novel Small Molecule Inhibitors of Hsp90</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">4396</span>– <span class="NLM_lpage">4403</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.04.121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.bmcl.2012.04.121" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4396-4403&issue=13&author=H.-P.+Buchstallerauthor=H.-M.+Eggenweilerauthor=C.+Sirrenbergauthor=U.+Gr%C3%A4dlerauthor=D.+Musilauthor=E.+Hoppeauthor=A.+Zimmermannauthor=H.+Schwartzauthor=J.+M%C3%A4rzauthor=J.+Bomkeauthor=A.+Wegenerauthor=M.+Wolf&title=Fragment-Based+Discovery+of+Hydroxy-Indazole-Carboxamides+as+Novel+Small+Molecule+Inhibitors+of+Hsp90&doi=10.1016%2Fj.bmcl.2012.04.121"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.04.121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.04.121%26sid%3Dliteratum%253Aachs%26aulast%3DBuchstaller%26aufirst%3DH.-P.%26aulast%3DEggenweiler%26aufirst%3DH.-M.%26aulast%3DSirrenberg%26aufirst%3DC.%26aulast%3DGr%25C3%25A4dler%26aufirst%3DU.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DHoppe%26aufirst%3DE.%26aulast%3DZimmermann%26aufirst%3DA.%26aulast%3DSchwartz%26aufirst%3DH.%26aulast%3DM%25C3%25A4rz%26aufirst%3DJ.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DM.%26atitle%3DFragment-Based%2520Discovery%2520of%2520Hydroxy-Indazole-Carboxamides%2520as%2520Novel%2520Small%2520Molecule%2520Inhibitors%2520of%2520Hsp90%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26issue%3D13%26spage%3D4396%26epage%3D4403%26doi%3D10.1016%2Fj.bmcl.2012.04.121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&issue=2&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgQArFK7J87FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuszewski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span> <span> </span><span class="NLM_article-title">Crystallography & NMR System: A New Software Suite for Macromolecular Structure Determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">905</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1107/S0907444998003254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1107%2FS0907444998003254" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=905-921&issue=5&author=A.+T.+Br%C3%BCngerauthor=P.+D.+Adamsauthor=G.+M.+Cloreauthor=W.+L.+DeLanoauthor=P.+Grosauthor=R.+W.+Grosse-Kunstleveauthor=J.+S.+Jiangauthor=J.+Kuszewskiauthor=M.+Nilgesauthor=N.+S.+Pannuauthor=R.+J.+Readauthor=L.+M.+Riceauthor=T.+Simonsonauthor=G.+L.+Warren&title=Crystallography+%26+NMR+System%3A+A+New+Software+Suite+for+Macromolecular+Structure+Determination&doi=10.1107%2FS0907444998003254"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444998003254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998003254%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DJiang%26aufirst%3DJ.%2BS.%26aulast%3DKuszewski%26aufirst%3DJ.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRice%26aufirst%3DL.%2BM.%26aulast%3DSimonson%26aufirst%3DT.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DCrystallography%2520%2526%2520NMR%2520System%253A%2520A%2520New%2520Software%2520Suite%2520for%2520Macromolecular%2520Structure%2520Determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1998%26volume%3D54%26issue%3D5%26spage%3D905%26epage%3D921%26doi%3D10.1107%2FS0907444998003254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berthold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cebron, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dill, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kötter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieb, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswedel, B.</span></span> <span> </span><span class="NLM_article-title">KNIME: The Konstanz Information Miner</span>. <i>SpringerLink</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1007/978-3-540-78246-9_38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1007%2F978-3-540-78246-9_38" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=319-326&author=M.+R.+Bertholdauthor=N.+Cebronauthor=F.+Dillauthor=T.+R.+Gabrielauthor=T.+K%C3%B6tterauthor=T.+Meinlauthor=P.+Ohlauthor=C.+Siebauthor=K.+Thielauthor=B.+Wiswedel&title=KNIME%3A+The+Konstanz+Information+Miner&doi=10.1007%2F978-3-540-78246-9_38"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2F978-3-540-78246-9_38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-540-78246-9_38%26sid%3Dliteratum%253Aachs%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26aulast%3DCebron%26aufirst%3DN.%26aulast%3DDill%26aufirst%3DF.%26aulast%3DGabriel%26aufirst%3DT.%2BR.%26aulast%3DK%25C3%25B6tter%26aufirst%3DT.%26aulast%3DMeinl%26aufirst%3DT.%26aulast%3DOhl%26aufirst%3DP.%26aulast%3DSieb%26aufirst%3DC.%26aulast%3DThiel%26aufirst%3DK.%26aulast%3DWiswedel%26aufirst%3DB.%26atitle%3DKNIME%253A%2520The%2520Konstanz%2520Information%2520Miner%26jtitle%3DSpringerLink%26date%3D2008%26spage%3D319%26epage%3D326%26doi%3D10.1007%2F978-3-540-78246-9_38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 21 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Johannes R. Loeffler, Monica L. Fernández-Quintero, Michael Schauperl, <span class="NLM_string-name hlFld-ContribAuthor">Klaus R. Liedl</span>. </span><span class="cited-content_cbyCitation_article-title">STACKED – Solvation Theory of Aromatic Complexes as Key for Estimating Drug Binding. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 2304-2313. <a href="https://doi.org/10.1021/acs.jcim.9b01165" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01165</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01165%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSTACKED%252B%2525E2%252580%252593%252BSolvation%252BTheory%252Bof%252BAromatic%252BComplexes%252Bas%252BKey%252Bfor%252BEstimating%252BDrug%252BBinding%26aulast%3DLoeffler%26aufirst%3DJohannes%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D17122019%26date%3D19032020%26date%3D06032020%26volume%3D60%26issue%3D4%26spage%3D2304%26epage%3D2313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tobias Hüfner-Wulsdorf, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Klebe</span>. </span><span class="cited-content_cbyCitation_article-title">Role of Water Molecules in Protein–Ligand Dissociation and Selectivity Discrimination: Analysis of the Mechanisms and Kinetics of Biomolecular Solvation Using Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (3)
                                     , 1818-1832. <a href="https://doi.org/10.1021/acs.jcim.0c00156" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00156%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRole%252Bof%252BWater%252BMolecules%252Bin%252BProtein%2525E2%252580%252593Ligand%252BDissociation%252Band%252BSelectivity%252BDiscrimination%25253A%252BAnalysis%252Bof%252Bthe%252BMechanisms%252Band%252BKinetics%252Bof%252BBiomolecular%252BSolvation%252BUsing%252BMolecular%252BDynamics%26aulast%3DH%25C3%25BCfner-Wulsdorf%26aufirst%3DTobias%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12022020%26date%3D06032020%26date%3D20022020%26volume%3D60%26issue%3D3%26spage%3D1818%26epage%3D1832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Steffen Glöckner, Khang Ngo, Christoph P. Sager, Tobias Hüfner-Wulsdorf, Andreas Heine, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Klebe</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational Changes in Alkyl Chains Determine the Thermodynamic and Kinetic Binding Profiles of Carbonic Anhydrase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (3)
                                     , 675-685. <a href="https://doi.org/10.1021/acschembio.9b00895" title="DOI URL">https://doi.org/10.1021/acschembio.9b00895</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00895%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DConformational%252BChanges%252Bin%252BAlkyl%252BChains%252BDetermine%252Bthe%252BThermodynamic%252Band%252BKinetic%252BBinding%252BProfiles%252Bof%252BCarbonic%252BAnhydrase%252BInhibitors%26aulast%3DGl%25C3%25B6ckner%26aufirst%3DSteffen%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D04112019%26date%3D06022020%26date%3D19022020%26date%3D06022020%26volume%3D15%26issue%3D3%26spage%3D675%26epage%3D685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Steffen Wolf, Marta Amaral, Maryse Lowinski, Francois Vallée, Djordje Musil, Jörn Güldenhaupt, Matthias K. Dreyer, Jörg Bomke, Matthias Frech, Jürgen Schlitter, <span class="NLM_string-name hlFld-ContribAuthor">Klaus Gerwert</span>. </span><span class="cited-content_cbyCitation_article-title">Estimation of Protein–Ligand Unbinding Kinetics Using Non-Equilibrium Targeted Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (12)
                                     , 5135-5147. <a href="https://doi.org/10.1021/acs.jcim.9b00592" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00592</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00592%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DEstimation%252Bof%252BProtein%2525E2%252580%252593Ligand%252BUnbinding%252BKinetics%252BUsing%252BNon-Equilibrium%252BTargeted%252BMolecular%252BDynamics%252BSimulations%26aulast%3DWolf%26aufirst%3DSteffen%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D19072019%26date%3D22112019%26date%3D07112019%26volume%3D59%26issue%3D12%26spage%3D5135%26epage%3D5147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Doris
A. Schuetz, Mattia Bernetti, Martina Bertazzo, Djordje Musil, Hans-Michael Eggenweiler, Maurizio Recanatini, Matteo Masetti, Gerhard F. Ecker, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Cavalli</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Residence Time and Drug Unbinding Pathway through Scaled Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (1)
                                     , 535-549. <a href="https://doi.org/10.1021/acs.jcim.8b00614" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00614%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredicting%252BResidence%252BTime%252Band%252BDrug%252BUnbinding%252BPathway%252Bthrough%252BScaled%252BMolecular%252BDynamics%26aulast%3DSchuetz%26aufirst%3DDoris%2BA.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D11092018%26date%3D13122018%26date%3D30112018%26volume%3D59%26issue%3D1%26spage%3D535%26epage%3D549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Doris
A. Schuetz, Thomas Seidel, Arthur Garon, Riccardo Martini, Markus Körbel, Gerhard F. Ecker, <span class="NLM_string-name hlFld-ContribAuthor">Thierry Langer</span>. </span><span class="cited-content_cbyCitation_article-title">GRAIL: GRids of phArmacophore Interaction fieLds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2018,</strong> <em>14 </em>
                                    (9)
                                     , 4958-4970. <a href="https://doi.org/10.1021/acs.jctc.8b00495" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b00495</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b00495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b00495%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DGRAIL%25253A%252BGRids%252Bof%252BphArmacophore%252BInteraction%252BfieLds%26aulast%3DSchuetz%26aufirst%3DDoris%2BA.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22052018%26date%3D20082018%26date%3D03082018%26volume%3D14%26issue%3D9%26spage%3D4958%26epage%3D4970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daria B. Kokh, Marta Amaral, Joerg Bomke, Ulrich Grädler, Djordje Musil, Hans-Peter Buchstaller, Matthias K. Dreyer, Matthias Frech, Maryse Lowinski, Francois Vallee, Marc Bianciotto, Alexey Rak, <span class="NLM_string-name hlFld-ContribAuthor">Rebecca C. Wade</span>. </span><span class="cited-content_cbyCitation_article-title">Estimation of Drug-Target Residence Times by τ-Random Acceleration Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2018,</strong> <em>14 </em>
                                    (7)
                                     , 3859-3869. <a href="https://doi.org/10.1021/acs.jctc.8b00230" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b00230</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b00230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b00230%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DEstimation%252Bof%252BDrug-Target%252BResidence%252BTimes%252Bby%252B%2525CF%252584-Random%252BAcceleration%252BMolecular%252BDynamics%252BSimulations%26aulast%3DKokh%26aufirst%3DDaria%2BB.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01032018%26date%3D04062018%26date%3D16052018%26volume%3D14%26issue%3D7%26spage%3D3859%26epage%3D3869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saba  Rezvani</span>, <span class="hlFld-ContribAuthor ">Ahmad  Ebadi</span>, <span class="hlFld-ContribAuthor ">Nima  Razzaghi-Asl</span>. </span><span class="cited-content_cbyCitation_article-title">In silico identification of potential Hsp90 inhibitors via ensemble docking, DFT and molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em>26 </em>, 1-12. <a href="https://doi.org/10.1080/07391102.2021.1947383" title="DOI URL">https://doi.org/10.1080/07391102.2021.1947383</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2021.1947383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2021.1947383%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DIn%252Bsilico%252Bidentification%252Bof%252Bpotential%252BHsp90%252Binhibitors%252Bvia%252Bensemble%252Bdocking%25252C%252BDFT%252Band%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DRezvani%26aufirst%3DSaba%26date%3D2021%26date%3D2021%26volume%3D26%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Doris A.  Schuetz</span>, <span class="hlFld-ContribAuthor ">Lars  Richter</span>, <span class="hlFld-ContribAuthor ">Riccardo  Martini</span>, <span class="hlFld-ContribAuthor ">Gerhard F.  Ecker</span>. </span><span class="cited-content_cbyCitation_article-title">A structure–kinetic relationship study using matched molecular pair analysis. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (11)
                                     , 1285-1294. <a href="https://doi.org/10.1039/D0MD00178C" title="DOI URL">https://doi.org/10.1039/D0MD00178C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00178C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00178C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DA%252Bstructure%2525E2%252580%252593kinetic%252Brelationship%252Bstudy%252Busing%252Bmatched%252Bmolecular%252Bpair%252Banalysis%26aulast%3DSchuetz%26aufirst%3DDoris%2BA.%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D11%26spage%3D1285%26epage%3D1294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ariane  Nunes-Alves</span>, <span class="hlFld-ContribAuthor ">Daria B  Kokh</span>, <span class="hlFld-ContribAuthor ">Rebecca C  Wade</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in molecular simulation methods for drug binding kinetics. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2020,</strong> <em>64 </em>, 126-133. <a href="https://doi.org/10.1016/j.sbi.2020.06.022" title="DOI URL">https://doi.org/10.1016/j.sbi.2020.06.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2020.06.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2020.06.022%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DRecent%252Bprogress%252Bin%252Bmolecular%252Bsimulation%252Bmethods%252Bfor%252Bdrug%252Bbinding%252Bkinetics%26aulast%3DNunes-Alves%26aufirst%3DAriane%26date%3D2020%26volume%3D64%26spage%3D126%26epage%3D133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daria B.  Kokh</span>, <span class="hlFld-ContribAuthor ">Bernd  Doser</span>, <span class="hlFld-ContribAuthor ">Stefan  Richter</span>, <span class="hlFld-ContribAuthor ">Fabian  Ormersbach</span>, <span class="hlFld-ContribAuthor ">Xingyi  Cheng</span>, <span class="hlFld-ContribAuthor ">Rebecca C.  Wade</span>. </span><span class="cited-content_cbyCitation_article-title">A workflow for exploring ligand dissociation from a macromolecule: Efficient random acceleration molecular dynamics simulation and interaction fingerprint analysis of ligand trajectories. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Chemical Physics</span><span> <strong>2020,</strong> <em>153 </em>
                                    (12)
                                     , 125102. <a href="https://doi.org/10.1063/5.0019088" title="DOI URL">https://doi.org/10.1063/5.0019088</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/5.0019088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F5.0019088%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Chemical%2520Physics%26atitle%3DA%252Bworkflow%252Bfor%252Bexploring%252Bligand%252Bdissociation%252Bfrom%252Ba%252Bmacromolecule%25253A%252BEfficient%252Brandom%252Bacceleration%252Bmolecular%252Bdynamics%252Bsimulation%252Band%252Binteraction%252Bfingerprint%252Banalysis%252Bof%252Bligand%252Btrajectories%26aulast%3DKokh%26aufirst%3DDaria%2BB.%26date%3D2020%26volume%3D153%26issue%3D12%26spage%3D125102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunji  Liu</span>, <span class="hlFld-ContribAuthor ">Leyi  Xia</span>, <span class="hlFld-ContribAuthor ">Kequan  Fu</span>, <span class="hlFld-ContribAuthor ">Xudong  Cao</span>, <span class="hlFld-ContribAuthor ">Wenzhong  Yan</span>, <span class="hlFld-ContribAuthor ">Jianjun  Cheng</span>, <span class="hlFld-ContribAuthor ">Thomas  Roux</span>, <span class="hlFld-ContribAuthor ">Lambertus A.  Peletier</span>, <span class="hlFld-ContribAuthor ">Xiaoxing  Yin</span>, <span class="hlFld-ContribAuthor ">Dong  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Revisit ligand-receptor interaction at the human vasopressin V2 receptor: A kinetic perspective. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2020,</strong> <em>880 </em>, 173157. <a href="https://doi.org/10.1016/j.ejphar.2020.173157" title="DOI URL">https://doi.org/10.1016/j.ejphar.2020.173157</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2020.173157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2020.173157%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DRevisit%252Bligand-receptor%252Binteraction%252Bat%252Bthe%252Bhuman%252Bvasopressin%252BV2%252Breceptor%25253A%252BA%252Bkinetic%252Bperspective%26aulast%3DLiu%26aufirst%3DChunji%26date%3D2020%26volume%3D880%26spage%3D173157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D. V.  Borisov</span>, <span class="hlFld-ContribAuthor ">A.V.  Veselovsky</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand–Receptor Binding Kinetics in Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry (Moscow), Supplement Series B: Biomedical Chemistry</span><span> <strong>2020,</strong> <em>14 </em>
                                    (3)
                                     , 228-240. <a href="https://doi.org/10.1134/S199075082003004X" title="DOI URL">https://doi.org/10.1134/S199075082003004X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S199075082003004X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS199075082003004X%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%2520%2528Moscow%2529%252C%2520Supplement%2520Series%2520B%253A%2520Biomedical%2520Chemistry%26atitle%3DLigand%2525E2%252580%252593Receptor%252BBinding%252BKinetics%252Bin%252BDrug%252BDesign%26aulast%3DBorisov%26aufirst%3DD.%2BV.%26date%3D2020%26date%3D2020%26volume%3D14%26issue%3D3%26spage%3D228%26epage%3D240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuheng  Huang</span>, <span class="hlFld-ContribAuthor ">Linxin  Chen</span>, <span class="hlFld-ContribAuthor ">Hu  Mei</span>, <span class="hlFld-ContribAuthor ">Duo  Zhang</span>, <span class="hlFld-ContribAuthor ">Tingting  Shi</span>, <span class="hlFld-ContribAuthor ">Zuyin  Kuang</span>, <span class="hlFld-ContribAuthor ">Yu  Heng</span>, <span class="hlFld-ContribAuthor ">Lei  Xu</span>, <span class="hlFld-ContribAuthor ">Xianchao  Pan</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Prediction of the Dissociation Rate Constants of Small Chemical Ligands by 3D-Grid-Based VolSurf Method. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (7)
                                     , 2456. <a href="https://doi.org/10.3390/ijms21072456" title="DOI URL">https://doi.org/10.3390/ijms21072456</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21072456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21072456%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DIn%252BSilico%252BPrediction%252Bof%252Bthe%252BDissociation%252BRate%252BConstants%252Bof%252BSmall%252BChemical%252BLigands%252Bby%252B3D-Grid-Based%252BVolSurf%252BMethod%26aulast%3DHuang%26aufirst%3DShuheng%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D7%26spage%3D2456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D.V.  Borisov</span>, <span class="hlFld-ContribAuthor ">A.V.  Veselovsky</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-receptor binding kinetics in drug design. </span><span class="cited-content_cbyCitation_journal-name">Biomeditsinskaya Khimiya</span><span> <strong>2020,</strong> <em>66 </em>
                                    (1)
                                     , 42-53. <a href="https://doi.org/10.18097/pbmc20206601042" title="DOI URL">https://doi.org/10.18097/pbmc20206601042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18097/pbmc20206601042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18097%2Fpbmc20206601042%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomeditsinskaya%2520Khimiya%26atitle%3DLigand-receptor%252Bbinding%252Bkinetics%252Bin%252Bdrug%252Bdesign%26aulast%3DBorisov%26aufirst%3DD.V.%26date%3D2020%26date%3D2020%26volume%3D66%26issue%3D1%26spage%3D42%26epage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maciej  Majewski</span>, <span class="hlFld-ContribAuthor ">Sergio  Ruiz-Carmona</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>. </span><span class="cited-content_cbyCitation_article-title">An investigation of structural stability in protein-ligand complexes reveals the balance between order and disorder. </span><span class="cited-content_cbyCitation_journal-name">Communications Chemistry</span><span> <strong>2019,</strong> <em>2 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s42004-019-0205-5" title="DOI URL">https://doi.org/10.1038/s42004-019-0205-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42004-019-0205-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42004-019-0205-5%26sid%3Dliteratum%253Aachs%26jtitle%3DCommunications%2520Chemistry%26atitle%3DAn%252Binvestigation%252Bof%252Bstructural%252Bstability%252Bin%252Bprotein-ligand%252Bcomplexes%252Breveals%252Bthe%252Bbalance%252Bbetween%252Border%252Band%252Bdisorder%26aulast%3DMajewski%26aufirst%3DMaciej%26date%3D2019%26date%3D2019%26volume%3D2%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adriaan P.  IJzerman</span>, <span class="hlFld-ContribAuthor ">Dong  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Drug–Target Association Kinetics in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Trends in Biochemical Sciences</span><span> <strong>2019,</strong> <em>44 </em>
                                    (10)
                                     , 861-871. <a href="https://doi.org/10.1016/j.tibs.2019.04.004" title="DOI URL">https://doi.org/10.1016/j.tibs.2019.04.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tibs.2019.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tibs.2019.04.004%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Biochemical%2520Sciences%26atitle%3DDrug%2525E2%252580%252593Target%252BAssociation%252BKinetics%252Bin%252BDrug%252BDiscovery%26aulast%3DIJzerman%26aufirst%3DAdriaan%2BP.%26date%3D2019%26volume%3D44%26issue%3D10%26spage%3D861%26epage%3D871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daria B.  Kokh</span>, <span class="hlFld-ContribAuthor ">Tom  Kaufmann</span>, <span class="hlFld-ContribAuthor ">Bastian  Kister</span>, <span class="hlFld-ContribAuthor ">Rebecca C.  Wade</span>. </span><span class="cited-content_cbyCitation_article-title">Machine Learning Analysis of τRAMD Trajectories to Decipher Molecular Determinants of Drug-Target Residence Times. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2019,</strong> <em>6 </em><a href="https://doi.org/10.3389/fmolb.2019.00036" title="DOI URL">https://doi.org/10.3389/fmolb.2019.00036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2019.00036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2019.00036%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DMachine%252BLearning%252BAnalysis%252Bof%252B%2525CF%252584RAMD%252BTrajectories%252Bto%252BDecipher%252BMolecular%252BDeterminants%252Bof%252BDrug-Target%252BResidence%252BTimes%26aulast%3DKokh%26aufirst%3DDaria%2BB.%26date%3D2019%26date%3D2019%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David A.  Sykes</span>, <span class="hlFld-ContribAuthor ">Leigh A.  Stoddart</span>, <span class="hlFld-ContribAuthor ">Laura E.  Kilpatrick</span>, <span class="hlFld-ContribAuthor ">Stephen J.  Hill</span>. </span><span class="cited-content_cbyCitation_article-title">Binding kinetics of ligands acting at GPCRs. </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Endocrinology</span><span> <strong>2019,</strong> <em>485 </em>, 9-19. <a href="https://doi.org/10.1016/j.mce.2019.01.018" title="DOI URL">https://doi.org/10.1016/j.mce.2019.01.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mce.2019.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mce.2019.01.018%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Endocrinology%26atitle%3DBinding%252Bkinetics%252Bof%252Bligands%252Bacting%252Bat%252BGPCRs%26aulast%3DSykes%26aufirst%3DDavid%2BA.%26date%3D2019%26volume%3D485%26spage%3D9%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shu-Long  Wang</span>, <span class="hlFld-ContribAuthor ">Zhen-Feng  Wang</span>, <span class="hlFld-ContribAuthor ">Qi-Pin  Qin</span>, <span class="hlFld-ContribAuthor ">Ming-Xiong  Tan</span>, <span class="hlFld-ContribAuthor ">Dong-Mei  Luo</span>, <span class="hlFld-ContribAuthor ">Bi-Qun  Zou</span>, <span class="hlFld-ContribAuthor ">Yan-Cheng  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">A 9‑chloro‑5,6,7,8‑tetrahydroacridine Pt(II) complex induces apoptosis of Hep‑G2 cells via inhibiting telomerase activity and disrupting mitochondrial pathway. </span><span class="cited-content_cbyCitation_journal-name">Inorganic Chemistry Communications</span><span> <strong>2019,</strong> <em>99 </em>, 77-81. <a href="https://doi.org/10.1016/j.inoche.2018.11.012" title="DOI URL">https://doi.org/10.1016/j.inoche.2018.11.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.inoche.2018.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.inoche.2018.11.012%26sid%3Dliteratum%253Aachs%26jtitle%3DInorganic%2520Chemistry%2520Communications%26atitle%3DA%252B9%2525E2%252580%252591chloro%2525E2%252580%2525915%25252C6%25252C7%25252C8%2525E2%252580%252591tetrahydroacridine%252BPt%252528II%252529%252Bcomplex%252Binduces%252Bapoptosis%252Bof%252BHep%2525E2%252580%252591G2%252Bcells%252Bvia%252Binhibiting%252Btelomerase%252Bactivity%252Band%252Bdisrupting%252Bmitochondrial%252Bpathway%26aulast%3DWang%26aufirst%3DShu-Long%26date%3D2019%26volume%3D99%26spage%3D77%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Lu</span>, <span class="hlFld-ContribAuthor ">James N  Iuliano</span>, <span class="hlFld-ContribAuthor ">Peter J  Tonge</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–kinetic relationships that control the residence time of drug–target complexes: insights from molecular structure and dynamics. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>44 </em>, 101-109. <a href="https://doi.org/10.1016/j.cbpa.2018.06.002" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.06.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DStructure%2525E2%252580%252593kinetic%252Brelationships%252Bthat%252Bcontrol%252Bthe%252Bresidence%252Btime%252Bof%252Bdrug%2525E2%252580%252593target%252Bcomplexes%25253A%252Binsights%252Bfrom%252Bmolecular%252Bstructure%252Band%252Bdynamics%26aulast%3DLu%26aufirst%3DHao%26date%3D2018%26volume%3D44%26spage%3D101%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Effect of ground state (GS) and transition state (TS) (de)stabilization on residence time shown as free energy profiles of the binding event of a drug (D) to its receptor (R) to form a drug–receptor complex (DR). Δ<i>G</i><sup>⧧</sup><sub>on</sub> relates to the energy barrier the drug has to overcome during binding. The sum of Δ<i>G</i><sup>⧧</sup><sub>on</sub> and |Δ<i>G</i><sub>D</sub>| is the energy barrier, Δ<i>G</i><sup>⧧</sup><sub>off</sub>, the drug has to overcome during the unbinding event. The equations for <i>K</i><sub>D</sub>, <i>k</i><sub>on</sub>, and <i>k</i><sub>off</sub> show the direct relation to their respective free energies Δ<i>G</i><sub>D</sub>, Δ<i>G</i><sup>⧧</sup><sub>on</sub>, and Δ<i>G</i><sup>⧧</sup><sub>off</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Hsp90 inhibitor sets of two different scaffolds showing similar binding mode. (A) The indazole scaffold<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and (B) the resorcinol scaffold.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (C) Bound complexes of an indazole (<b>1g</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OCI">5OCI</a>) and (D) a resorcinol (<b>4i</b> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J86">5J86</a>) show the R<sup>1</sup> substituents embedded in the hydrophobic cavity and the R<sup>2</sup> substituents at the entrance of the protein binding site. R<sup>2</sup>-substitutions are facing toward the hydrophilic pocket entrance (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Figure S4</a>). Hydrophobic areas are displayed in orange, hydrophilic areas in blue. (E) Kinetic map of the three data sets to show ranges of <i>k</i><sub>on</sub>, <i>k</i><sub>off</sub>, and <i>K</i><sub>D</sub>. <i>k</i><sub>on</sub> is plotted against <i>k</i><sub>off</sub>, displaying <i>K</i><sub>D</sub> in the diagonal. (F) Correlation between desolvation free energy of R<sup>1</sup> and R<sup>2</sup> substituents of the indazole and resorcinole data sets, and the log<sub>10</sub>-transformed on-rates of the “cavity-varying” data sets (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) show significant correlation, while “entrance-varying” indazoles (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) do not. <i>R</i> is the Pearson correlation coefficient (two-tailed <i>t</i> test).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Polar desolvation scheme validated on two matched triplets. (A) Two matched triplets ordered by their R<sup>1</sup>-substituents’ desolvation free energy (Δ<i>G</i><sub>desolv</sub>). Structures in black are from the “cavity-varying” indazole data set (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Structures in red were prospectively selected for SPR measurements (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). (B) <i>k</i><sub>on</sub> value distribution of the five- and six-membered ring triplets and the overall Hsp90 data set. (C) Representative sensograms obtained by SPR for interaction between N-Hsp90 with compounds <b>2b</b>, <b>2c</b>, and <b>3c</b>. Fitted model traces are shown in black lines. (D) Overlay of <b>3b</b> and <b>3c</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNY">5LNY</a> and PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ODX">5ODX</a>) shows a concordant overlay of their indazole scaffolds and their respective pocket geometries. (E) Residence time distribution of the five- and six-membered ring triplets and the overall Hsp90 data set. (F) GS and TS contributions to compound residence times. Changes in ΔΔ<i>G</i><sub>D</sub> and ΔΔ<i>G</i><sup>⧧</sup><sub>on</sub> were computed relative to a reference compound with an affinity of 1000 nM and a <i>k</i><sub>on</sub> rate of 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup> according to Lu and Tonge.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Free energies have been calculated as shown in Schoop and Dey<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> from <i>K</i><sub>D</sub> and <i>k</i><sub>on</sub> values. GS and TS contributions are shown in blue and red, respectively. Statistical difference between the mean values was determined by a one-tailed unpaired Student’s <i>t</i> test: (∗) <i>P</i> < 0.05, (∗∗) <i>P</i> < 0.01, (∗∗∗) <i>P</i> < 0.001; error bars indicate ±1 sd.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Quinazoline triplet <b>6a</b>, <b>6b</b>, and <b>6c</b> and their calculated desolvation free energies (Δ<i>G</i><sub>desolv</sub>). (B) X-ray structure of a quinazoline (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7">5OD7</a>) shows orientation of the isoindoline moiety (R<sup>1</sup>) into the hydrophobic cavity. The scaffold substitution R<sup>2</sup> is reaching toward the hydrophilic pocket entrance. (C) Bar charts display the <i>k</i><sub>on</sub> values (red), residence time (gray), and <i>K</i><sub>D</sub> (blue) for compounds <b>6a</b>, <b>6b</b>, and <b>6c</b>. (D) Potential of mean force (PMF) for <b>6a</b>, <b>6b</b>, and <b>6c</b> (black lines) and profile of number of water molecules (red lines) surrounding the hydroxyl group (within 3.6 Å) as a function of the distance ξ. The center-of-mass (COM) pulling simulation starts from the bound complex and terminates at 30 Å distance from the starting point (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods in Supporting Information</a>). Shadow areas around the free energy profile indicate statistical uncertainties. Red arrow indicates the highest energy barrier (TS). Statistical difference between the mean values was determined by a one-tailed unpaired Student’s <i>t</i> test: (∗∗) <i>P</i> < 0.01, (∗∗∗) <i>P</i> < 0.001; error bars indicate ±1 sd.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/medium/jm-2018-00080t_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Putative binding pathway of <b>6c</b> derived from MD simulation. (A) Compound <b>6c</b> bound to the binding site (modeled from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7">5OD7</a>; see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf" class="ext-link">Supplementary Methods in Supporting Information</a>), the isoindoline fully embedded in the hydrophobic cavity. (B) The isoindoline has moved out of the cavity while the quinazoline scaffold is still “in place”. The hydroxyl group is fully desolvated to traverse the narrow, hydrophobic passage. (C) Compound <b>6c</b> has overcome the highest energy barrier (TS), and the hydroxyl group is solvated. Hydrophobic areas are displayed in orange, hydrophilic areas in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-10/acs.jmedchem.8b00080/20180517/images/large/jm-2018-00080t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00080&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06073" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06073" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 63 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span> <span> </span><span class="NLM_article-title">Biochemical Mechanisms of Drug Action: What Does It Take for Success?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1038/nrd1500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnrd1500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=15340390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFCnsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=801-808&issue=9&author=D.+C.+Swinney&title=Biochemical+Mechanisms+of+Drug+Action%3A+What+Does+It+Take+for+Success%3F&doi=10.1038%2Fnrd1500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Biochemical mechanisms of drug action: what does it take for success?</span></div><div class="casAuthors">Swinney, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">801-808</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Drug discovery is extremely difficult.  There are many unanticipated scientific, medical and business challenges to every drug discovery program.  It is important to increase our understanding of the fundamental properties of effective drugs so that we can anticipate potential problems in developing new agents.  This article addresses potential drug discovery and development risks assocd. with the biochem. mechanism of drug action, and proposes simple rules to minimize these risks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXhyDgozFUdrVg90H21EOLACvtfcHk0ljaZ1pF_aXwIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFCnsL4%253D&md5=b06bdc6ace3168dcb011679b43b72aed</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1500%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26atitle%3DBiochemical%2520Mechanisms%2520of%2520Drug%2520Action%253A%2520What%2520Does%2520It%2520Take%2520for%2520Success%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26issue%3D9%26spage%3D801%26epage%3D808%26doi%3D10.1038%2Fnrd1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Residence Time and Its Implications for Lead Optimization</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">730</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&issue=9&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-Target+Residence+Time+and+Its+Implications+for+Lead+Optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0ljaZ1pF_aXwIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-Target%2520Residence%2520Time%2520and%2520Its%2520Implications%2520for%2520Lead%2520Optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26issue%3D9%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R.
A.</span></span> <span> </span><span class="NLM_article-title">The Drug-Target Residence Time Model: A 10-Year Retrospective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1038/nrd.2015.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnrd.2015.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=26678621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=87-95&issue=2&author=R.%0AA.+Copeland&title=The+Drug-Target+Residence+Time+Model%3A+A+10-Year+Retrospective&doi=10.1038%2Fnrd.2015.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-target residence time model: a 10-year retrospective</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">87-95</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The drug-target residence time model was first introduced in 2006 and has been broadly adopted across the chem. biol., biotechnol. and pharmaceutical communities.  While traditional in vitro methods view drug-target interactions exclusively in terms of equil. affinity, the residence time model takes into account the conformational dynamics of target macromols. that affect drug binding and dissocn.  The key tenet of this model is that the lifetime (or residence time) of the binary drug-target complex, and not the binding affinity per se, dictates much of the in vivo pharmacol. activity.  Here, this model is revisited and key applications of it over the past 10 years are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwNrhp9U8kwLVg90H21EOLACvtfcHk0ljaZ1pF_aXwIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE&md5=23f4a6973d77804292ff0af6c81358c0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.18%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520Drug-Target%2520Residence%2520Time%2520Model%253A%2520A%252010-Year%2520Retrospective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26issue%3D2%26spage%3D87%26epage%3D95%26doi%3D10.1038%2Fnrd.2015.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">The Use of Thermodynamic and Kinetic Data in Drug Discovery: Decisive Insight or Increasing the Puzzlement?</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">229</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201402521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2Fcmdc.201402521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=25537429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFejsrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=229-231&issue=2&author=G.+Klebe&title=The+Use+of+Thermodynamic+and+Kinetic+Data+in+Drug+Discovery%3A+Decisive+Insight+or+Increasing+the+Puzzlement%3F&doi=10.1002%2Fcmdc.201402521"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The Use of Thermodynamic and Kinetic Data in Drug Discovery: Decisive Insight or Increasing the Puzzlement?</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">229-231</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The prime property to rate the success of hit-to-lead-to-drug optimization in drug discovery is binding affinity.  Rational approaches try to relate this property with structure.  Affinity can be linked to the thermodn. property, Gibbs free energy of binding, which itself factorizes into enthalpy and entropy.  With respect to kinetic properties, affinity can be assocd. with the ratio of koff and kon of complex formation.  Do these features help to obtain better insight into affinity.  The present viewpoint assesses our current understanding of thermodn.- or kinetics-structure relationships and questions the accuracy of data collected to learn about the thermodn. and kinetic basis to comprehend affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS9VXbE-qOU7Vg90H21EOLACvtfcHk0liJC9UTjqQeIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFejsrfL&md5=315bae98f81db9dcbcd0a4bc4a5da4f9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201402521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201402521%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DThe%2520Use%2520of%2520Thermodynamic%2520and%2520Kinetic%2520Data%2520in%2520Drug%2520Discovery%253A%2520Decisive%2520Insight%2520or%2520Increasing%2520the%2520Puzzlement%253F%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26issue%3D2%26spage%3D229%26epage%3D231%26doi%3D10.1002%2Fcmdc.201402521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosemond, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure and Dynamics of the M3 Muscarinic Acetylcholine Receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>482</i></span> (<span class="NLM_issue">7386</span>),  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">556</span>, <span class="refDoi"> DOI: 10.1038/nature10867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnature10867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=22358844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC38Xis1eqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2012&pages=552-556&issue=7386&author=A.+C.+Kruseauthor=J.+Huauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=D.+M.+Rosenbaumauthor=E.+Rosemondauthor=H.+F.+Greenauthor=T.+Liuauthor=P.+S.+Chaeauthor=R.+O.+Drorauthor=D.+E.+Shawauthor=W.+I.+Weisauthor=J.+Wessauthor=B.+K.+Kobilka&title=Structure+and+Dynamics+of+the+M3+Muscarinic+Acetylcholine+Receptor&doi=10.1038%2Fnature10867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and dynamics of the M3 muscarinic acetylcholine receptor</span></div><div class="casAuthors">Kruse, Andrew C.; Hu, Jianxin; Pan, Albert C.; Arlow, Daniel H.; Rosenbaum, Daniel M.; Rosemond, Erica; Green, Hillary F.; Liu, Tong; Chae, Pil Seok; Dror, Ron O.; Shaw, David E.; Weis, William I.; Wess, Jurgen; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">7386</span>),
    <span class="NLM_cas:pages">552-556</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acetylcholine, the first neurotransmitter to be identified, exerts many of its physiol. actions via activation of a family of G-protein-coupled receptors (GPCRs) known as muscarinic acetylcholine receptors (mAChRs).  Although the five mAChR subtypes (M1-M5) share a high degree of sequence homol., they show pronounced differences in G-protein coupling preference and the physiol. responses they mediate.  Unfortunately, despite decades of effort, no therapeutic agents endowed with clear mAChR subtype selectivity have been developed to exploit these differences.  We describe here the structure of the Gq/11-coupled M3 mAChR ('M3 receptor', from rat) bound to the bronchodilator drug tiotropium and identify the binding mode for this clin. important drug.  This structure, together with that of the Gi/o-coupled M2 receptor, offers possibilities for the design of mAChR subtype-selective ligands.  Importantly, the M3 receptor structure allows a structural comparison between two members of a mammalian GPCR subfamily displaying different G-protein coupling selectivities.  Furthermore, mol. dynamics simulations suggest that tiotropium binds transiently to an allosteric site en route to the binding pocket of both receptors.  These simulations offer a structural view of an allosteric binding mode for an orthosteric GPCR ligand and provide addnl. opportunities for the design of ligands with different affinities or binding kinetics for different mAChR subtypes.  Our findings not only offer insights into the structure and function of one of the most important GPCR families, but may also facilitate the design of improved therapeutics targeting these crit. receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozveVSiaQvz7Vg90H21EOLACvtfcHk0liJC9UTjqQeIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xis1eqs7k%253D&md5=31bb89b1d14cf122353fced8f905e996</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature10867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10867%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DRosemond%26aufirst%3DE.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520and%2520Dynamics%2520of%2520the%2520M3%2520Muscarinic%2520Acetylcholine%2520Receptor%26jtitle%3DNature%26date%3D2012%26volume%3D482%26issue%3D7386%26spage%3D552%26epage%3D556%26doi%3D10.1038%2Fnature10867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span></span> <span> </span><span class="NLM_article-title">On-Rate Based Optimization of Structure-Kinetic Relationship--Surfing the Kinetic Map</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.ddtec.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=26724331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC28rms1egtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=9-15&author=A.+Schoopauthor=F.+Dey&title=On-Rate+Based+Optimization+of+Structure-Kinetic+Relationship%2D%2DSurfing+the+Kinetic+Map&doi=10.1016%2Fj.ddtec.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">On-rate based optimization of structure-kinetic relationship--surfing the kinetic map</span></div><div class="casAuthors">Schoop Andreas; Dey Fabian</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9-15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In the lead discovery process residence time has become an important parameter for the identification and characterization of the most efficacious compounds in vivo.  To enable the success of compound optimization by medicinal chemistry toward a desired residence time the understanding of structure-kinetic relationship (SKR) is essential.  This article reviews various approaches to monitor SKR and suggests using the on-rate as the key monitoring parameter.  The literature is reviewed and examples of compound series with low variability as well as with significant changes in on-rates are highlighted.  Furthermore, findings of kinetic on-rate changes are presented and potential underlying rationales are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwRMy18rtTE3T4ugvCDgnZfW6udTcc2eYbQNjMcyFHZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rms1egtw%253D%253D&md5=a7f0dee9cd02f38a945123e7faa9bc1c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DSchoop%26aufirst%3DA.%26aulast%3DDey%26aufirst%3DF.%26atitle%3DOn-Rate%2520Based%2520Optimization%2520of%2520Structure-Kinetic%2520Relationship--Surfing%2520the%2520Kinetic%2520Map%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2015%26volume%3D17%26spage%3D9%26epage%3D15%26doi%3D10.1016%2Fj.ddtec.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Residence Time: Critical Information for Lead Optimization</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.06.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.cbpa.2010.06.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=20663707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=467-474&issue=4&author=H.+Luauthor=P.+J.+Tonge&title=Drug-Target+Residence+Time%3A+Critical+Information+for+Lead+Optimization&doi=10.1016%2Fj.cbpa.2010.06.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time: Critical information for lead optimization</span></div><div class="casAuthors">Lu, Hao; Tonge, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Failure due to poor in vivo efficacy is a primary contributor to attrition during the development of new chemotherapeutics.  Lead optimization programs that in their quest for efficacy focus solely on improving the affinity of drug-target binding are flawed, since this approach ignores the fluctuations in drug concn. that occur in vivo.  Instead the lifetime of the drug-target complex must also be considered, since drugs only act when they are bound to their targets.  Consequently, to improve the correlation between the in vitro and in vivo activity of drugs, measurements of drug-target residence time must be incorporated into the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodk2kN_txuMLVg90H21EOLACvtfcHk0liH2j1ie0CWQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVyqtr4%253D&md5=3189f56032106db858cfb4269d3fc11f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.06.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.06.176%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DH.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26atitle%3DDrug-Target%2520Residence%2520Time%253A%2520Critical%2520Information%2520for%2520Lead%2520Optimization%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26issue%3D4%26spage%3D467%26epage%3D474%26doi%3D10.1016%2Fj.cbpa.2010.06.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spagnuolo, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eltschkner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daryaee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davoodi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, E. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pschibul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moree, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thivalapill, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truglio, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salafsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slayden, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kisker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span> <span> </span><span class="NLM_article-title">Evaluating the Contribution of Transition-State Destabilization to Changes in the Residence Time of Triazole-Based InhA Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3417</span>– <span class="NLM_lpage">3429</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b11148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=3417-3429&issue=9&author=L.+A.+Spagnuoloauthor=S.+Eltschknerauthor=W.+Yuauthor=F.+Daryaeeauthor=S.+Davoodiauthor=S.+E.+Knudsonauthor=E.+K.+H.+Allenauthor=J.+Merinoauthor=A.+Pschibulauthor=B.+Moreeauthor=N.+Thivalapillauthor=J.+J.+Truglioauthor=J.+Salafskyauthor=R.+A.+Slaydenauthor=C.+Kiskerauthor=P.+J.+Tonge&title=Evaluating+the+Contribution+of+Transition-State+Destabilization+to+Changes+in+the+Residence+Time+of+Triazole-Based+InhA+Inhibitors&doi=10.1021%2Fjacs.6b11148"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11148%26sid%3Dliteratum%253Aachs%26aulast%3DSpagnuolo%26aufirst%3DL.%2BA.%26aulast%3DEltschkner%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DDaryaee%26aufirst%3DF.%26aulast%3DDavoodi%26aufirst%3DS.%26aulast%3DKnudson%26aufirst%3DS.%2BE.%26aulast%3DAllen%26aufirst%3DE.%2BK.%2BH.%26aulast%3DMerino%26aufirst%3DJ.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DMoree%26aufirst%3DB.%26aulast%3DThivalapill%26aufirst%3DN.%26aulast%3DTruglio%26aufirst%3DJ.%2BJ.%26aulast%3DSalafsky%26aufirst%3DJ.%26aulast%3DSlayden%26aufirst%3DR.%2BA.%26aulast%3DKisker%26aufirst%3DC.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26atitle%3DEvaluating%2520the%2520Contribution%2520of%2520Transition-State%2520Destabilization%2520to%2520Changes%2520in%2520the%2520Residence%2520Time%2520of%2520Triazole-Based%2520InhA%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D9%26spage%3D3417%26epage%3D3429%26doi%3D10.1021%2Fjacs.6b11148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollica, L.</span></span> <span> </span><span class="NLM_article-title">Protein–ligand (Un)Binding Kinetics as a New Paradigm for Drug Discovery at the Crossroad between Experiments and Modelling</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1039/C6MD00581K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1039%2FC6MD00581K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=30108770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsleitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=534-550&issue=3&author=M.+Bernettiauthor=A.+Cavalliauthor=L.+Mollica&title=Protein%E2%80%93ligand+%28Un%29Binding+Kinetics+as+a+New+Paradigm+for+Drug+Discovery+at+the+Crossroad+between+Experiments+and+Modelling&doi=10.1039%2FC6MD00581K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-ligand (un)binding kinetics as a new paradigm for drug discovery at the crossroad between experiments and modelling</span></div><div class="casAuthors">Bernetti, M.; Cavalli, A.; Mollica, L.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">534-550</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last three decades, protein and nucleic acid structure detn. and comprehension of the mechanisms, leading to their physiol. and pathol. functions, have become a cornerstone of biomedical sciences.  A deep understanding of the principles governing the fates of cells and tissue at the mol. level has been gained over the years, offering a solid basis for the rational design of drugs aimed at the pharmacol. treatment of numerous diseases.  Historically, affinity indicators (i.e. Kd and IC50/EC50) have been assumed to be valid indicators of the in vivo efficacy of a drug.  However, recent studies pointed out that the kinetics of the drug-receptor binding process could be as important or even more important than affinity in detg. the drug efficacy.  This eventually led to a growing interest in the characterization and prediction of the rate consts. of protein-ligand assocn. and dissocn.  For instance, a drug with a longer residence time can kinetically select a given receptor over another, even if the affinity for both receptors is comparable, thus increasing its therapeutic index.  Therefore, understanding the mol. features underlying binding and unbinding processes is of central interest towards the rational control of drug binding kinetics.  In this review, we report the theor. framework behind protein-ligand assocn. and highlight the latest advances in the exptl. and computational approaches exploited to investigate the binding kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrO85mDP2s17Vg90H21EOLACvtfcHk0liH2j1ie0CWQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsleitbw%253D&md5=8538a9e13d7e1d2c873da236be95a811</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1039%2FC6MD00581K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00581K%26sid%3Dliteratum%253Aachs%26aulast%3DBernetti%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DMollica%26aufirst%3DL.%26atitle%3DProtein%25E2%2580%2593ligand%2520%2528Un%2529Binding%2520Kinetics%2520as%2520a%2520New%2520Paradigm%2520for%2520Drug%2520Discovery%2520at%2520the%2520Crossroad%2520between%2520Experiments%2520and%2520Modelling%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26issue%3D3%26spage%3D534%26epage%3D550%26doi%3D10.1039%2FC6MD00581K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Truhlar, D.
G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klippenstein, S. J.</span></span> <span> </span><span class="NLM_article-title">Current Status of Transition-State Theory</span>. <i>J. Phys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>100</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">12771</span>– <span class="NLM_lpage">12800</span>, <span class="refDoi"> DOI: 10.1021/jp953748q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp953748q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaK28Xkt1ansr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1996&pages=12771-12800&issue=31&author=D.%0AG.+Truhlarauthor=B.+C.+Garrettauthor=S.+J.+Klippenstein&title=Current+Status+of+Transition-State+Theory&doi=10.1021%2Fjp953748q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Current Status of Transition-State Theory</span></div><div class="casAuthors">Truhlar, Donald G.; Garrett, Bruce C.; Klippenstein, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">12771-12800</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 843 refs.; we present an overview of the current status of transition-state theory and its generalizations.  We emphasize (i) recent improvements in available methodol. for calcns. on complex systems, including the interface with electronic structure theory, (ii) progress in the theory and application of transition-state theory to condensed-phase reactions, and (iii) insight into the relation of transition-state theory to accurate quantum dynamics and tests of its accuracy via comparisons with both exptl. and other theor. dynamical approxns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnTDhjv-GhGbVg90H21EOLACvtfcHk0lhfZM4TNlQnLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xkt1ansr8%253D&md5=5663e2f23815cdc1c0bbb6bbb91adabe</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjp953748q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp953748q%26sid%3Dliteratum%253Aachs%26aulast%3DTruhlar%26aufirst%3DD.%2BG.%26aulast%3DGarrett%26aufirst%3DB.%2BC.%26aulast%3DKlippenstein%26aufirst%3DS.%2BJ.%26atitle%3DCurrent%2520Status%2520of%2520Transition-State%2520Theory%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1996%26volume%3D100%26issue%3D31%26spage%3D12771%26epage%3D12800%26doi%3D10.1021%2Fjp953748q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willemsen-Seegers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uitdehaag, J. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinsen, M. B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vetter, J. R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Man, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buijsman, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman, G. J. R.</span></span> <span> </span><span class="NLM_article-title">Compound Selectivity and Target Residence Time of Kinase Inhibitors Studied with Surface Plasmon Resonance</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">574</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2016.12.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.jmb.2016.12.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=574-586&issue=4&author=N.+Willemsen-Seegersauthor=J.+C.+M.+Uitdehaagauthor=M.+B.+W.+Prinsenauthor=J.+R.+F.+de+Vetterauthor=J.+de+Manauthor=M.+Sawaauthor=Y.+Kawaseauthor=R.+C.+Buijsmanauthor=G.+J.+R.+Zaman&title=Compound+Selectivity+and+Target+Residence+Time+of+Kinase+Inhibitors+Studied+with+Surface+Plasmon+Resonance&doi=10.1016%2Fj.jmb.2016.12.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.12.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.12.019%26sid%3Dliteratum%253Aachs%26aulast%3DWillemsen-Seegers%26aufirst%3DN.%26aulast%3DUitdehaag%26aufirst%3DJ.%2BC.%2BM.%26aulast%3DPrinsen%26aufirst%3DM.%2BB.%2BW.%26aulast%3Dde%2BVetter%26aufirst%3DJ.%2BR.%2BF.%26aulast%3Dde%2BMan%26aufirst%3DJ.%26aulast%3DSawa%26aufirst%3DM.%26aulast%3DKawase%26aufirst%3DY.%26aulast%3DBuijsman%26aufirst%3DR.%2BC.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%2BR.%26atitle%3DCompound%2520Selectivity%2520and%2520Target%2520Residence%2520Time%2520of%2520Kinase%2520Inhibitors%2520Studied%2520with%2520Surface%2520Plasmon%2520Resonance%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26issue%3D4%26spage%3D574%26epage%3D586%26doi%3D10.1016%2Fj.jmb.2016.12.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuetz, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Witte, W. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knasmueller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokh, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadiq, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nederpelt, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segala, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaral, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoddart, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frech, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Lange, E. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span> <span> </span><span class="NLM_article-title">Kinetics for Drug Discovery: An Industry-Driven Effort to Target Drug Residence Time</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">896</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.drudis.2017.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=28412474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt1Kjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=896-911&issue=6&author=D.+A.+Schuetzauthor=W.+E.+A.+de+Witteauthor=Y.+C.+Wongauthor=B.+Knasmuellerauthor=L.+Richterauthor=D.+B.+Kokhauthor=S.+K.+Sadiqauthor=R.+Bosmaauthor=I.+Nederpeltauthor=L.+H.+Heitmanauthor=E.+Segalaauthor=M.+Amaralauthor=D.+Guoauthor=D.+Andresauthor=V.+Georgiauthor=L.+A.+Stoddartauthor=S.+Hillauthor=R.+M.+Cookeauthor=C.+De+Graafauthor=R.+Leursauthor=M.+Frechauthor=R.+C.+Wadeauthor=E.+C.+M.+de+Langeauthor=A.+P.+Ijzermanauthor=A.+M%C3%BCller-Fahrnowauthor=G.+F.+Ecker&title=Kinetics+for+Drug+Discovery%3A+An+Industry-Driven+Effort+to+Target+Drug+Residence+Time&doi=10.1016%2Fj.drudis.2017.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics for Drug Discovery: an industry-driven effort to target drug residence time</span></div><div class="casAuthors">Schuetz, Doris A.; de Witte, Wilhelmus Egbertus Arnout; Wong, Yin Cheong; Knasmueller, Bernhard; Richter, Lars; Kokh, Daria B.; Sadiq, S. Kashif; Bosma, Reggie; Nederpelt, Indira; Heitman, Laura H.; Segala, Elena; Amaral, Marta; Guo, Dong; Andres, Dorothee; Georgi, Victoria; Stoddart, Leigh A.; Hill, Steve; Cooke, Robert M.; De Graaf, Chris; Leurs, Rob; Frech, Matthias; Wade, Rebecca C.; de Lange, Elizabeth Cunera Maria; IJzerman, Adriaan P.; Mueller-Fahrnow, Anke; Ecker, Gerhard F.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">896-911</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  A considerable no. of approved drugs show non-equil. binding characteristics, emphasizing the potential role of drug residence times for in vivo efficacy.  Therefore, a detailed understanding of the kinetics of assocn. and dissocn. of a target-ligand complex might provide crucial insight into the mol. mechanism-of-action of a compd.  This deeper understanding will help to improve decision making in drug discovery, thus leading to a better selection of interesting compds. to be profiled further.  In this review, we highlight the contributions of the Kinetics for Drug Discovery (K4DD) Consortium, which targets major open questions related to binding kinetics in an industry-driven public-private partnership.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRmkYejg8ObVg90H21EOLACvtfcHk0lhfZM4TNlQnLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmt1Kjs7k%253D&md5=78fce4f850718075ba44b28a20f79195</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DSchuetz%26aufirst%3DD.%2BA.%26aulast%3Dde%2BWitte%26aufirst%3DW.%2BE.%2BA.%26aulast%3DWong%26aufirst%3DY.%2BC.%26aulast%3DKnasmueller%26aufirst%3DB.%26aulast%3DRichter%26aufirst%3DL.%26aulast%3DKokh%26aufirst%3DD.%2BB.%26aulast%3DSadiq%26aufirst%3DS.%2BK.%26aulast%3DBosma%26aufirst%3DR.%26aulast%3DNederpelt%26aufirst%3DI.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DSegala%26aufirst%3DE.%26aulast%3DAmaral%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DAndres%26aufirst%3DD.%26aulast%3DGeorgi%26aufirst%3DV.%26aulast%3DStoddart%26aufirst%3DL.%2BA.%26aulast%3DHill%26aufirst%3DS.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DDe%2BGraaf%26aufirst%3DC.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3DFrech%26aufirst%3DM.%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3Dde%2BLange%26aufirst%3DE.%2BC.%2BM.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DM%25C3%25BCller-Fahrnow%26aufirst%3DA.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26atitle%3DKinetics%2520for%2520Drug%2520Discovery%253A%2520An%2520Industry-Driven%2520Effort%2520to%2520Target%2520Drug%2520Residence%2520Time%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26issue%3D6%26spage%3D896%26epage%3D911%26doi%3D10.1016%2Fj.drudis.2017.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span>K4DD - Kinetics for Drug Discovery. <a href="http://db.k4dd.eu" class="extLink">http://db.k4dd.eu</a> (accessed December
6, <span class="NLM_year">2017</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=K4DD+-+Kinetics+for+Drug+Discovery.+http%3A%2F%2Fdb.k4dd.eu+%28accessed+December%0A6%2C+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vauquelin, G.</span></span> <span> </span><span class="NLM_article-title">Effects of Target Binding Kinetics on in Vivo Drug Efficacy: Koff, Kon and Rebinding</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>173</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">2319</span>– <span class="NLM_lpage">2334</span>, <span class="refDoi"> DOI: 10.1111/bph.13504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1111%2Fbph.13504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=27129075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSiurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2016&pages=2319-2334&issue=15&author=G.+Vauquelin&title=Effects+of+Target+Binding+Kinetics+on+in+Vivo+Drug+Efficacy%3A+Koff%2C+Kon+and+Rebinding&doi=10.1111%2Fbph.13504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of target binding kinetics on in vivo drug efficacy: koff, kon and rebinding</span></div><div class="casAuthors">Vauquelin, Georges</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2319-2334</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and Purpose : Optimal drug therapy often requires continuing high levels of target occupancy.  Besides the traditional pharmacokinetic contribution, target binding kinetics is increasingly considered to play an important role as well.  While most attention has been focused on the dissocn. rate of the complex, recent reports expressed doubt about the unreserved translatability of this pharmacodynamic property into clin. efficacy. 'Micro'-pharmacokinetic mechanisms like drug rebinding and partitioning into the cell membrane may constitute a potential fix.  Exptl. Approach : Simulations were based on solving differential equations.  Key Results : Based on a selected range of assocn. and dissocn. rate consts., kon and koff, and rebinding potencies of the drugs as variables, their effects on the temporal in vivo occupancy profile of their targets, after one or multiple repetitive dosings, have here been simulated.  Conclusions and Implications : Most strikingly, the simulations show that, when rebinding is also taken into account, increasing kon may produce closely the same outcome as decreasing koff when dosing is performed in accordance with the therapeutically most relevant const. [Lmax]/KD ratio paradigm.  Also, under certain conditions, rebinding may produce closely the same outcome as invoking slow diffusion of the drug between the plasma compartment and a target-contg. 'effect' compartment.  Although the present simulations should only be regarded as a 'proof of principle', these findings may help pharmacologists and medicinal chemists to devise ex vivo and in vitro binding kinetic assays that are more relevant and translatable to in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5-dbZNJetv7Vg90H21EOLACvtfcHk0lgeN2k2c91VLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSiurvI&md5=5d1b13ef5786a0c661118ce9be503d50</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1111%2Fbph.13504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13504%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DEffects%2520of%2520Target%2520Binding%2520Kinetics%2520on%2520in%2520Vivo%2520Drug%2520Efficacy%253A%2520Koff%252C%2520Kon%2520and%2520Rebinding%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2016%26volume%3D173%26issue%3D15%26spage%3D2319%26epage%3D2334%26doi%3D10.1111%2Fbph.13504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vauquelin, G.</span></span> <span> </span><span class="NLM_article-title">Rebinding: Or Why Drugs May Act Longer in Vivo than Expected from Their in Vitro Target Residence Time</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">941</span>, <span class="refDoi"> DOI: 10.1517/17460441.2010.512037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1517%2F17460441.2010.512037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=22823988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFKmtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=927-941&issue=10&author=G.+Vauquelin&title=Rebinding%3A+Or+Why+Drugs+May+Act+Longer+in+Vivo+than+Expected+from+Their+in+Vitro+Target+Residence+Time&doi=10.1517%2F17460441.2010.512037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Rebinding: or why drugs may act longer in vivo than expected from their in vitro target residence time</span></div><div class="casAuthors">Vauquelin, Georges</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">927-941</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: It is well established that the in vivo duration of drug action not only depends on macroscopic pharmacokinetic properties like its plasma half-life, but also on the residence time of the drug-target complexes.  However, drug 'rebinding' (i.e., the consecutive binding of dissocd. drug mols. to the original target and/or targets nearby) can be influential in vivo as well.  Areas covered in this review: Information about rebinding is available since the 1980s but it is dispersed in the life sciences literature.  This review compiles this information.  In this respect, neurochemists and biophysicians advance the same equations to describe drug rebinding.  What the reader will gain: The rebinding mechanism is explained according to the prevailing viewpoint in different life science disciplines.  There is a general consensus that high target densities, high assocn. rates and local phenomena that hinder the diffusion of free drug mols. away from their target all promote rebinding.  Take home message: Simulations presented here for the first time suggest that rebinding may increase the duration and even the constancy of the drug's clin. action.  Intact cell radioligand dissocn. and related ex vivo expts. offer useful indications about a drug's aptitude to experience target rebinding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSRqDC7FEY8bVg90H21EOLACvtfcHk0lgeN2k2c91VLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFKmtLbM&md5=928b8861b47515c10db3116bfcd47e4b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1517%2F17460441.2010.512037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2010.512037%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DRebinding%253A%2520Or%2520Why%2520Drugs%2520May%2520Act%2520Longer%2520in%2520Vivo%2520than%2520Expected%2520from%2520Their%2520in%2520Vitro%2520Target%2520Residence%2520Time%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26issue%3D10%26spage%3D927%26epage%3D941%26doi%3D10.1517%2F17460441.2010.512037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de Witte, W. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danhof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Graaf, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Lange, E. C. M.</span></span> <span> </span><span class="NLM_article-title">In Vivo Target Residence Time and Kinetic Selectivity: The Association Rate Constant as Determinant</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">831</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2016.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.tips.2016.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=27394919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVynur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=831-842&issue=10&author=W.+E.+A.+de+Witteauthor=M.+Danhofauthor=P.+H.+van+der%0AGraafauthor=E.+C.+M.+de+Lange&title=In+Vivo+Target+Residence+Time+and+Kinetic+Selectivity%3A+The+Association+Rate+Constant+as+Determinant&doi=10.1016%2Fj.tips.2016.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo Target Residence Time and Kinetic Selectivity: The Association Rate Constant as Determinant</span></div><div class="casAuthors">de Witte, Wilhelmus E. A.; Danhof, Meindert; van der Graaf, Piet H.; de Lange, Elizabeth C. M.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">831-842</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">It is generally accepted that, in conjunction with pharmacokinetics, the first-order rate const. of target dissocn. is a major determinant of the time course and duration of in vivo target occupancy.  Here we show that the second-order rate const. of target assocn. can be equally important.  On the basis of the commonly used math. models for drug-target binding, it is shown that a high target assocn. rate const. can increase the (local) concn. of the drug, which decreases the rate of decline of target occupancy.  The increased drug concn. can also lead to increased off-target binding and decreased selectivity.  Therefore, the kinetics of both target assocn. and dissocn. need to be taken into account in the selection of drug candidates with optimal pharmacodynamic properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri0cm4NKLv-7Vg90H21EOLACvtfcHk0lgeN2k2c91VLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVynur%252FE&md5=e2c077f2c8636214e2f8a77ee5734ad4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2016.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2016.06.008%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BWitte%26aufirst%3DW.%2BE.%2BA.%26aulast%3DDanhof%26aufirst%3DM.%26aulast%3Dvan%2Bder%2BGraaf%26aufirst%3DP.%2BH.%26aulast%3Dde%2BLange%26aufirst%3DE.%2BC.%2BM.%26atitle%3DIn%2520Vivo%2520Target%2520Residence%2520Time%2520and%2520Kinetic%2520Selectivity%253A%2520The%2520Association%2520Rate%2520Constant%2520as%2520Determinant%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2016%26volume%3D37%26issue%3D10%26spage%3D831%26epage%3D842%26doi%3D10.1016%2Fj.tips.2016.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javitch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlton, S. J.</span></span> <span> </span><span class="NLM_article-title">Extrapyramidal Side Effects of Antipsychotics Are Linked to Their Association Kinetics at Dopamine D 2 Receptors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">763</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-00716-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fs41467-017-00716-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=28970469" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FksFGgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=763&issue=1&author=D.+A.+Sykesauthor=H.+Mooreauthor=L.+Stottauthor=N.+Hollidayauthor=J.+A.+Javitchauthor=J.+R.+Laneauthor=S.+J.+Charlton&title=Extrapyramidal+Side+Effects+of+Antipsychotics+Are+Linked+to+Their+Association+Kinetics+at+Dopamine+D+2+Receptors&doi=10.1038%2Fs41467-017-00716-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Extrapyramidal side effects of antipsychotics are linked to their association kinetics at dopamine D2 receptors</span></div><div class="casAuthors">Sykes David A; Stott Lisa; Holliday Nicholas; Charlton Steven J; Moore Holly; Javitch Jonathan A; Moore Holly; Javitch Jonathan A; Javitch Jonathan A; Lane J Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">763</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Atypical antipsychotic drugs (APDs) have been hypothesized to show reduced extrapyramidal side effects (EPS) due to their rapid dissociation from the dopamine D2 receptor.  However, support for this hypothesis is limited to a relatively small number of observations made across several decades and under different experimental conditions.  Here we show that association rates, but not dissociation rates, correlate with EPS.  We measured the kinetic binding properties of a series of typical and atypical APDs in a novel time-resolved fluorescence resonance energy transfer assay, and correlated these properties with their EPS and prolactin-elevating liabilities at therapeutic doses.  EPS are robustly predicted by a rebinding model that considers the microenvironment of postsynaptic D2 receptors and integrates association and dissociation rates to calculate the net rate of reversal of receptor blockade.  Thus, optimizing binding kinetics at the D2 receptor may result in APDs with improved therapeutic profile.Atypical antipsychotics show reduced extrapyramidal side effects compared to first generation drugs.  Here the authors use time-resolved FRET to measure binding kinetics, and show that side effects correlate with drug association rates to the D2 receptor, while dissociation rates correlate with prolactin elevation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSAv7HAvl2UWlVwdslNBF-tfW6udTcc2ebvKYqrhxyLRrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FksFGgsg%253D%253D&md5=6d5c8596da9a3565b7a95b55eceac3cf</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00716-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00716-z%26sid%3Dliteratum%253Aachs%26aulast%3DSykes%26aufirst%3DD.%2BA.%26aulast%3DMoore%26aufirst%3DH.%26aulast%3DStott%26aufirst%3DL.%26aulast%3DHolliday%26aufirst%3DN.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DCharlton%26aufirst%3DS.%2BJ.%26atitle%3DExtrapyramidal%2520Side%2520Effects%2520of%2520Antipsychotics%2520Are%2520Linked%2520to%2520Their%2520Association%2520Kinetics%2520at%2520Dopamine%2520D%25202%2520Receptors%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D763%26doi%3D10.1038%2Fs41467-017-00716-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansell, J. E.</span></span> <span> </span><span class="NLM_article-title">Direct Thrombin Inhibitors</span>. <i>Br. J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>72</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2125.2011.03916.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1111%2Fj.1365-2125.2011.03916.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=21241354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlers7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2011&pages=581-592&issue=4&author=C.+J.+Leeauthor=J.+E.+Ansell&title=Direct+Thrombin+Inhibitors&doi=10.1111%2Fj.1365-2125.2011.03916.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Direct thrombin inhibitors</span></div><div class="casAuthors">Lee, Catherine J.; Ansell, Jack E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">581-592</span>CODEN:
                <span class="NLM_cas:coden">BCPHBM</span>;
        ISSN:<span class="NLM_cas:issn">0306-5251</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Heparins and vitamin K antagonists have been the primary agents used for anticoagulation in certain cardiovascular and thromboembolic diseases for over 50 years.  However, they can be difficult to administer and are fraught with limitations.  In response to the need for new anticoagulants, direct thrombin inhibitors (DTIs) have been developed and investigated for their utility in prophylaxis and treatment of venous thromboembolism (VTE), heparin-induced thrombocytopenia (HIT), acute coronary syndromes (ACS), secondary prevention of coronary events after ACS, and nonvalvular atrial fibrillation.  Currently, four parenteral direct inhibitors of thrombin activity are FDA-approved in North America: lepirudin, desirudin, bivalirudin and argatroban.  Of the new oral DTIs, dabigatran etexilate is the most studied and promising of these agents.  This review discusses the clin. indications and efficacy of these direct thrombin inhibitors as well as future directions in anticoagulant therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsZGu46hTy8rVg90H21EOLACvtfcHk0lgtNIG_qQhstw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlers7rM&md5=32c9ea4decff589c5f2d453c9a322f6b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2125.2011.03916.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2125.2011.03916.x%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DAnsell%26aufirst%3DJ.%2BE.%26atitle%3DDirect%2520Thrombin%2520Inhibitors%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D2011%26volume%3D72%26issue%3D4%26spage%3D581%26epage%3D592%26doi%3D10.1111%2Fj.1365-2125.2011.03916.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walkup, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, E. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daryaee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, V. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buurman, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajec, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy-Benenato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patey, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosser, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, S. L.</span></span> <span> </span><span class="NLM_article-title">Translating Slow-Binding Inhibition Kinetics into Cellular and in Vivo Effects</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">416</span>– <span class="NLM_lpage">423</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fnchembio.1796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=25894085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=416-423&issue=6&author=G.+K.+Walkupauthor=Z.+Youauthor=P.+L.+Rossauthor=E.+K.+H.+Allenauthor=F.+Daryaeeauthor=M.+R.+Haleauthor=J.+O%E2%80%99Donnellauthor=D.+E.+Ehmannauthor=V.+J.+A.+Schuckauthor=E.+T.+Buurmanauthor=A.+L.+Choyauthor=L.+Hajecauthor=K.+Murphy-Benenatoauthor=V.+Maroneauthor=S.+A.+Pateyauthor=L.+A.+Grosserauthor=M.+Johnstoneauthor=S.+G.+Walkerauthor=P.+J.+Tongeauthor=S.+L.+Fisher&title=Translating+Slow-Binding+Inhibition+Kinetics+into+Cellular+and+in+Vivo+Effects&doi=10.1038%2Fnchembio.1796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Translating slow-binding inhibition kinetics into cellular and in vivo effects</span></div><div class="casAuthors">Walkup, Grant K.; You, Zhiping; Ross, Philip L.; Allen, Eleanor K. H.; Daryaee, Fereidoon; Hale, Michael R.; O'Donnell, John; Ehmann, David E.; Schuck, Virna J. A.; Buurman, Ed T.; Choy, Allison L.; Hajec, Laurel; Murphy-Benenato, Kerry; Marone, Valerie; Patey, Sara A.; Grosser, Lena A.; Johnstone, Michele; Walker, Stephen G.; Tonge, Peter J.; Fisher, Stewart L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">416-423</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many drug candidates fail in clin. trials owing to a lack of efficacy from limited target engagement or an insufficient therapeutic index.  Minimizing off-target effects while retaining the desired pharmacodynamic (PD) response can be achieved by reduced exposure for drugs that display kinetic selectivity in which the drug-target complex has a longer half-life than off-target-drug complexes.  However, though slow-binding inhibition kinetics are a key feature of many marketed drugs, prospective tools that integrate drug-target residence time into predictions of drug efficacy are lacking, hindering the integration of drug-target kinetics into the drug discovery cascade.  Here we describe a mechanistic PD model that includes drug-target kinetic parameters, including the on- and off-rates for the formation and breakdown of the drug-target complex.  We demonstrate the utility of this model by using it to predict dose response curves for inhibitors of the LpxC enzyme from Pseudomonas aeruginosa in an animal model of infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS0SazAerLJLVg90H21EOLACvtfcHk0lj_z6-hbZNmWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartb0%253D&md5=87c25ffbcbb74aaa3ecd80aba26d2f13</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1796%26sid%3Dliteratum%253Aachs%26aulast%3DWalkup%26aufirst%3DG.%2BK.%26aulast%3DYou%26aufirst%3DZ.%26aulast%3DRoss%26aufirst%3DP.%2BL.%26aulast%3DAllen%26aufirst%3DE.%2BK.%2BH.%26aulast%3DDaryaee%26aufirst%3DF.%26aulast%3DHale%26aufirst%3DM.%2BR.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%26aulast%3DEhmann%26aufirst%3DD.%2BE.%26aulast%3DSchuck%26aufirst%3DV.%2BJ.%2BA.%26aulast%3DBuurman%26aufirst%3DE.%2BT.%26aulast%3DChoy%26aufirst%3DA.%2BL.%26aulast%3DHajec%26aufirst%3DL.%26aulast%3DMurphy-Benenato%26aufirst%3DK.%26aulast%3DMarone%26aufirst%3DV.%26aulast%3DPatey%26aufirst%3DS.%2BA.%26aulast%3DGrosser%26aufirst%3DL.%2BA.%26aulast%3DJohnstone%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DS.%2BG.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DFisher%26aufirst%3DS.%2BL.%26atitle%3DTranslating%2520Slow-Binding%2520Inhibition%2520Kinetics%2520into%2520Cellular%2520and%2520in%2520Vivo%2520Effects%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D6%26spage%3D416%26epage%3D423%26doi%3D10.1038%2Fnchembio.1796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bommineni, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Diaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium Tuberculosis InhA</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">4683</span>– <span class="NLM_lpage">4691</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.5b00284</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b00284" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFWqtb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=4683-4691&issue=30&author=C.-T.+Laiauthor=H.-J.+Liauthor=W.+Yuauthor=S.+Shahauthor=G.+R.+Bommineniauthor=V.+Perroneauthor=M.+Garcia-Diazauthor=P.+J.+Tongeauthor=C.+Simmerling&title=Rational+Modulation+of+the+Induced-Fit+Conformational+Change+for+Slow-Onset+Inhibition+in+Mycobacterium+Tuberculosis+InhA&doi=10.1021%2Facs.biochem.5b00284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium tuberculosis InhA</span></div><div class="casAuthors">Lai, Cheng-Tsung; Li, Huei-Jiun; Yu, Weixuan; Shah, Sonam; Bommineni, Gopal R.; Perrone, Victoria; Garcia-Diaz, Miguel; Tonge, Peter J.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">4683-4691</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Slow-onset enzyme inhibitors are the subject of considerable interest as an approach to increasing the potency of pharmaceutical compds. by extending the residence time of the inhibitor on the target (the lifetime of the drug-receptor complex).  However, rational modulation of residence time presents significant challenges because it requires addnl. mechanistic insight, such as the nature of the transition state for postbinding isomerization.  Our previous work, based on X-ray crystallog., enzyme kinetics, and mol. dynamics simulation, suggested that the slow step in inhibition of the Mycobacterium tuberculosis enoyl-ACP reductase InhA involves a change in the conformation of the substrate binding loop from an open state in the initial enzyme-inhibitor complex to a closed state in the final enzyme-inhibitor complex.  Here, we use multidimensional free energy landscapes for loop isomerization to obtain a computational model for the transition state.  The results suggest that slow-onset inhibitors crowd key side chains on helixes that slide past each other during isomerization, resulting in a steric clash.  The landscapes become significantly flatter when residues involved in the steric clash are replaced with alanine.  Importantly, this lower barrier can be increased by rational inhibitor redesign to restore the steric clash.  Crystallog. studies and enzyme kinetics confirm the predicted effects on loop structure and flexibility, as well as inhibitor residence time.  These loss and regain of function studies validate our mechanistic hypothesis for interactions controlling substrate binding loop isomerization, providing a platform for the future design of inhibitors with longer residence times and better in vivo potency.  Similar opportunities for slow-onset inhibition via the same mechanism are identified in other pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6OmewK8DIh7Vg90H21EOLACvtfcHk0lj_z6-hbZNmWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWqtb%252FM&md5=2c014493a57205f99e2fbc698ff64488</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b00284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b00284%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DC.-T.%26aulast%3DLi%26aufirst%3DH.-J.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DBommineni%26aufirst%3DG.%2BR.%26aulast%3DPerrone%26aufirst%3DV.%26aulast%3DGarcia-Diaz%26aufirst%3DM.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DRational%2520Modulation%2520of%2520the%2520Induced-Fit%2520Conformational%2520Change%2520for%2520Slow-Onset%2520Inhibition%2520in%2520Mycobacterium%2520Tuberculosis%2520InhA%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26issue%3D30%26spage%3D4683%26epage%3D4691%26doi%3D10.1021%2Facs.biochem.5b00284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luckner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">am Ende, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kisker, C.</span></span> <span> </span><span class="NLM_article-title">A Slow, Tight Binding Inhibitor of InhA, the Enoyl-Acyl Carrier Protein Reductase from Mycobacterium Tuberculosis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">14330</span>– <span class="NLM_lpage">14337</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.090373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1074%2Fjbc.M109.090373" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=14330-14337&issue=19&author=S.+R.+Lucknerauthor=N.+Liuauthor=C.+W.+am+Endeauthor=P.+J.+Tongeauthor=C.+Kisker&title=A+Slow%2C+Tight+Binding+Inhibitor+of+InhA%2C+the+Enoyl-Acyl+Carrier+Protein+Reductase+from+Mycobacterium+Tuberculosis&doi=10.1074%2Fjbc.M109.090373"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.090373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.090373%26sid%3Dliteratum%253Aachs%26aulast%3DLuckner%26aufirst%3DS.%2BR.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3Dam%2BEnde%26aufirst%3DC.%2BW.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DKisker%26aufirst%3DC.%26atitle%3DA%2520Slow%252C%2520Tight%2520Binding%2520Inhibitor%2520of%2520InhA%252C%2520the%2520Enoyl-Acyl%2520Carrier%2520Protein%2520Reductase%2520from%2520Mycobacterium%2520Tuberculosis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D19%26spage%3D14330%26epage%3D14337%26doi%3D10.1074%2Fjbc.M109.090373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Conformational Adaptation in Drug-Target Interactions and Residence Time</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1491</span>– <span class="NLM_lpage">1501</span>, <span class="refDoi"> DOI: 10.4155/fmc.11.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.4155%2Ffmc.11.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=21882942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFSksLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1491-1501&issue=12&author=R.+A.+Copeland&title=Conformational+Adaptation+in+Drug-Target+Interactions+and+Residence+Time&doi=10.4155%2Ffmc.11.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational adaptation in drug-target interactions and residence time</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1491-1501</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Although drug-target interactions are commonly illustrated in terms of structurally static binding and dissocn. events, such descriptions are inadequate to explain the impact of conformational dynamics on these processes.  For high-affinity interactions, both the assocn. and dissocn. of drug mols. to and from their targets are often controlled by conformational changes of the target.  Conformational adaptation can greatly influence the residence time of a drug on its target (i.e., the lifetime of the binary drug-target complex); long residence time can lead to sustained pharmacol. and may also mitigate off-target toxicity.  In this perspective, the kinetics of drug-target assocn. and dissocn. reactions are explored, with particular emphasis on the impact of conformational adaptation on drug-target residence time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4Bp8rlLqFa7Vg90H21EOLACvtfcHk0lhsys4IhpXhoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFSksLfO&md5=f995e1a9d854831af67a5d96eb14cac4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.112%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DConformational%2520Adaptation%2520in%2520Drug-Target%2520Interactions%2520and%2520Residence%2520Time%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26issue%3D12%26spage%3D1491%26epage%3D1501%26doi%3D10.4155%2Ffmc.11.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amaral, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokh, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggenweiler, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matias, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirrenberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frech, M.</span></span> <span> </span><span class="NLM_article-title">Protein Conformational Flexibility Modulates Kinetics and Thermodynamics of Drug Binding</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">2276</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-02258-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fs41467-017-02258-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=29273709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC1Mzksleqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2276&issue=1&author=M.+Amaralauthor=D.+B.+Kokhauthor=J.+Bomkeauthor=A.+Wegenerauthor=H.+P.+Buchstallerauthor=H.+M.+Eggenweilerauthor=P.+Matiasauthor=C.+Sirrenbergauthor=R.+C.+Wadeauthor=M.+Frech&title=Protein+Conformational+Flexibility+Modulates+Kinetics+and+Thermodynamics+of+Drug+Binding&doi=10.1038%2Fs41467-017-02258-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Protein conformational flexibility modulates kinetics and thermodynamics of drug binding</span></div><div class="casAuthors">Amaral M; Matias P; Amaral M; Wegener A; Frech M; Amaral M; Kokh D B; Wade R C; Bomke J; Buchstaller H P; Eggenweiler H M; Matias P; Sirrenberg C; Wade R C; Wade R C</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2276</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based drug design has often been restricted by the rather static picture of protein-ligand complexes presented by crystal structures, despite the widely accepted importance of protein flexibility in biomolecular recognition.  Here we report a detailed experimental and computational study of the drug target, human heat shock protein 90, to explore the contribution of protein dynamics to the binding thermodynamics and kinetics of drug-like compounds.  We observe that their binding properties depend on whether the protein has a loop or a helical conformation in the binding site of the ligand-bound state.  Compounds bound to the helical conformation display slow association and dissociation rates, high-affinity and high cellular efficacy, and predominantly entropically driven binding.  An important entropic contribution comes from the greater flexibility of the helical relative to the loop conformation in the ligand-bound state.  This unusual mechanism suggests increasing target flexibility in the bound state by ligand design as a new strategy for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLPI8jjKBQEQvupmecNw_lfW6udTcc2eYZ8KvhLLpXOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mzksleqsg%253D%253D&md5=2ccf76a6569363af27b6f27cb1be5b7a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02258-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02258-w%26sid%3Dliteratum%253Aachs%26aulast%3DAmaral%26aufirst%3DM.%26aulast%3DKokh%26aufirst%3DD.%2BB.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DBuchstaller%26aufirst%3DH.%2BP.%26aulast%3DEggenweiler%26aufirst%3DH.%2BM.%26aulast%3DMatias%26aufirst%3DP.%26aulast%3DSirrenberg%26aufirst%3DC.%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DFrech%26aufirst%3DM.%26atitle%3DProtein%2520Conformational%2520Flexibility%2520Modulates%2520Kinetics%2520and%2520Thermodynamics%2520of%2520Drug%2520Binding%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D2276%26doi%3D10.1038%2Fs41467-017-02258-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maragakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Pathway and Mechanism of Drug Binding to G-Protein-Coupled Receptors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span> (<span class="NLM_issue">32</span>),  <span class="NLM_fpage">13118</span>– <span class="NLM_lpage">13123</span>, <span class="refDoi"> DOI: 10.1073/pnas.1104614108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1073%2Fpnas.1104614108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=21778406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKit7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13118-13123&issue=32&author=R.+O.+Drorauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=D.+W.+Borhaniauthor=P.+Maragakisauthor=Y.+Shanauthor=H.+Xuauthor=D.+E.+Shaw&title=Pathway+and+Mechanism+of+Drug+Binding+to+G-Protein-Coupled+Receptors&doi=10.1073%2Fpnas.1104614108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Pathway and mechanism of drug binding to G-protein-coupled receptors</span></div><div class="casAuthors">Dror, Ron O.; Pan, Albert C.; Arlow, Daniel H.; Borhani, David W.; Maragakis, Paul; Shan, Yibing; Xu, Huafeng; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">13118-13123, S13118/1-S13118/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">How drugs bind to their receptors-from initial assocn., through drug entry into the binding pocket, to adoption of the final bound conformation, or "pose"-has remained unknown, even for G-protein-coupled receptor modulators, which constitute 1/3 of all marketed drugs.  Here, the authors captured this pharmaceutically crit. process in at. detail using the 1st unbiased mol. dynamics simulations in which drug mols. spontaneously assoc. with G-protein-coupled receptors to achieve final poses matching those detd. crystallog.  The authors found that several beta blockers and a beta agonist all traversed the same well-defined, dominant pathway as they bound to the β1- and β2-adrenergic receptors, initially making contact with a vestibule on each receptor's extracellular surface.  Surprisingly, assocn. with this vestibule, at a distance of 15 Å from the binding pocket, often presented the largest energetic barrier to binding, despite the fact that subsequent entry into the binding pocket required the receptor to deform and the drug to squeeze through a narrow passage.  The early barrier appeared to reflect the substantial dehydration that takes place as the drug assocs. with the vestibule.  The at.-level description of the binding process suggests opportunities for allosteric modulation and provides a structural foundation for future optimization of drug-receptor binding and unbinding rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKUYFtkTGwf7Vg90H21EOLACvtfcHk0lith88BqN97GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKit7bE&md5=9cfa8673217931cacd586321309fa72e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1104614108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1104614108%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DMaragakis%26aufirst%3DP.%26aulast%3DShan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DPathway%2520and%2520Mechanism%2520of%2520Drug%2520Binding%2520to%2520G-Protein-Coupled%2520Receptors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D32%26spage%3D13118%26epage%3D13123%26doi%3D10.1073%2Fpnas.1104614108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gooljarsingh, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grooms, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johanson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinnamon, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span> <span> </span><span class="NLM_article-title">A Biochemical Rationale for the Anticancer Effects of Hsp90 Inhibitors: Slow, Tight Binding Inhibition by Geldanamycin and Its Analogues</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">7625</span>– <span class="NLM_lpage">7630</span>, <span class="refDoi"> DOI: 10.1073/pnas.0602650103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1073%2Fpnas.0602650103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=16684877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD28XlsVGlurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=7625-7630&issue=20&author=L.+T.+Gooljarsinghauthor=C.+Fernandesauthor=K.+Yanauthor=H.+Zhangauthor=M.+Groomsauthor=K.+Johansonauthor=R.+H.+Sinnamonauthor=R.+B.+Kirkpatrickauthor=J.+Kerriganauthor=T.+Lewisauthor=M.+Arnoneauthor=A.+J.+Kingauthor=Z.+Laiauthor=R.+A.+Copelandauthor=P.+J.+Tummino&title=A+Biochemical+Rationale+for+the+Anticancer+Effects+of+Hsp90+Inhibitors%3A+Slow%2C+Tight+Binding+Inhibition+by+Geldanamycin+and+Its+Analogues&doi=10.1073%2Fpnas.0602650103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A biochemical rationale for the anticancer effects of Hsp90 inhibitors: slow, tight binding inhibition by geldanamycin and its analogues</span></div><div class="casAuthors">Gooljarsingh, Lata T.; Fernandes, Christine; Yan, Kang; Zhang, Hong; Grooms, Michael; Johanson, Kyung; Sinnamon, Robert H.; Kirkpatrick, Robert B.; Kerrigan, John; Lewis, Tia; Arnone, Marc; King, Alastair J.; Lai, Zhihong; Copeland, Robert A.; Tummino, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7625-7630</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Heat shock protein (Hsp)90 is emerging as an important therapeutic target for the treatment of cancer.  Two analogs of the Hsp90 inhibitor geldanamycin are currently in clin. trials.  Geldanamycin (GA) and its analogs have been reported to bind purified Hsp90 with low micromolar potency, in stark contrast to their low nanomolar antiproliferative activity in cell culture and their potent antitumor activity in animal models.  Several models have been proposed to account for the ≈100-fold-greater potency in cell culture, including that GA analogs bind with greater affinity to a five-protein Hsp90 complex than to Hsp90 alone.  We have detd. that GA and the fluorescent analog BODIPY-GA (BDGA) both demonstrate slow, tight binding to purified Hsp90.  BDGA, used to characterize the kinetics of ligand-Hsp90 interactions, was found to bind Hsp90α with koff = 2.5×10-3 min-1, t1/2 = 4.6 h, and Ki* = 10 nM.  It was found that BDGA binds to a functional multiprotein Hsp90 complex with kinetics and affinity identical to that of Hsp90 alone.  Also, BDGA binds to Hsp90 from multiple cell lyzates in a time-dependent manner with similar kinetics.  Therefore, our results indicate that the high potency of GA in cell culture and in vivo can be accounted for by its time-dependent, tight binding to Hsp90 alone.  In the broader context, these studies highlight the essentiality of detailed biochem. characterization of drug-target interactions for the effective translation of in vitro pharmacol. to cellular and in vivo efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYOm9dd7WlmLVg90H21EOLACvtfcHk0lith88BqN97GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlsVGlurY%253D&md5=10e3cc3af47d06242dd621b8c64b3b34</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0602650103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0602650103%26sid%3Dliteratum%253Aachs%26aulast%3DGooljarsingh%26aufirst%3DL.%2BT.%26aulast%3DFernandes%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DGrooms%26aufirst%3DM.%26aulast%3DJohanson%26aufirst%3DK.%26aulast%3DSinnamon%26aufirst%3DR.%2BH.%26aulast%3DKirkpatrick%26aufirst%3DR.%2BB.%26aulast%3DKerrigan%26aufirst%3DJ.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DArnone%26aufirst%3DM.%26aulast%3DKing%26aufirst%3DA.%2BJ.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26atitle%3DA%2520Biochemical%2520Rationale%2520for%2520the%2520Anticancer%2520Effects%2520of%2520Hsp90%2520Inhibitors%253A%2520Slow%252C%2520Tight%2520Binding%2520Inhibition%2520by%2520Geldanamycin%2520and%2520Its%2520Analogues%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26issue%3D20%26spage%3D7625%26epage%3D7630%26doi%3D10.1073%2Fpnas.0602650103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H.-P.</span>; <span class="NLM_string-name">Eggenweiler, H.-M.</span>; <span class="NLM_string-name">Wolf, M.</span>; <span class="NLM_string-name">Sirrenberg, C.</span></span> <span> </span><span class="NLM_article-title">Indazole Derivatives for Treating Hsp90-Induced Diseases</span>. Patent <span class="NLM_patent">WO 2008/003396</span>, June 18, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=H.-P.+Buchstaller&author=H.-M.+Eggenweiler&author=M.+Wolf&author=C.+Sirrenberg&title=Indazole+Derivatives+for+Treating+Hsp90-Induced+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBuchstaller%26aufirst%3DH.-P.%26atitle%3DIndazole%2520Derivatives%2520for%2520Treating%2520Hsp90-Induced%2520Diseases%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eggenweiler, H.-M.</span>; <span class="NLM_string-name">Wolf, M.</span>; <span class="NLM_string-name">Buchstaller, H.-P.</span>; <span class="NLM_string-name">Sirrenberg, C.</span></span> <span> </span><span class="NLM_article-title">Triazole Derivatives II</span>. CA Patent <span class="NLM_patent">CA 2,652,394</span>, April 20, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=H.-M.+Eggenweiler&author=M.+Wolf&author=H.-P.+Buchstaller&author=C.+Sirrenberg&title=Triazole+Derivatives+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEggenweiler%26aufirst%3DH.-M.%26atitle%3DTriazole%2520Derivatives%2520II%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rechlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocchia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, H.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Kinetic and Structural Insights into the Mechanism of Binding of Sulfonamides to Human Carbonic Anhydrase by Computational and Experimental Studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4245</span>– <span class="NLM_lpage">4256</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01643</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01643" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4245-4256&issue=9&author=R.+Gaspariauthor=C.+Rechlinauthor=A.+Heineauthor=G.+Bottegoniauthor=W.+Rocchiaauthor=D.+Schwarzauthor=J.+Bomkeauthor=H.-D.+Gerberauthor=G.+Klebeauthor=A.+Cavalli&title=Kinetic+and+Structural+Insights+into+the+Mechanism+of+Binding+of+Sulfonamides+to+Human+Carbonic+Anhydrase+by+Computational+and+Experimental+Studies&doi=10.1021%2Facs.jmedchem.5b01643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and Structural Insights into the Mechanism of Binding of Sulfonamides to Human Carbonic Anhydrase by Computational and Experimental Studies</span></div><div class="casAuthors">Gaspari, Roberto; Rechlin, Chris; Heine, Andreas; Bottegoni, Giovanni; Rocchia, Walter; Schwarz, Daniel; Bomke, Jorg; Gerber, Hans-Dieter; Klebe, Gerhard; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4245-4256</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of sulfonamides to human carbonic anhydrase II (hCAII) is a complex and long-debated example of protein-ligand recognition and interaction.  In this study, we investigate the para-substituted n-alkyl and hydroxyethylene-benzenesulfonamides, providing a complete reconstruction of their binding pathway to hCAII by means of large-scale mol. dynamics simulations, d. functional calcns., surface plasmon resonance (SPR) measurements, and X-ray crystallog. expts.  Our anal. shows that the protein-ligand assocn. rate (kon) dramatically increases with the ligand's hydrophobicity, pointing to the existence of a prebinding stage largely stabilized by a favorable packing of the ligand's apolar moieties with the hCAII "hydrophobic wall".  The characterization of the binding pathway allows an unprecedented understanding of the structure-kinetic relationship in hCAII/benzenesulfonamide complexes, depicting a paradigmatic scenario for the multistep binding process in protein-ligand systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWXT8lIkKp8LVg90H21EOLACvtfcHk0lhFx6gTomYIew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVygsbbF&md5=433134a082e1b9e33e0dae0af2cc07c8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01643%26sid%3Dliteratum%253Aachs%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DRechlin%26aufirst%3DC.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DSchwarz%26aufirst%3DD.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DGerber%26aufirst%3DH.-D.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DKinetic%2520and%2520Structural%2520Insights%2520into%2520the%2520Mechanism%2520of%2520Binding%2520of%2520Sulfonamides%2520to%2520Human%2520Carbonic%2520Anhydrase%2520by%2520Computational%2520and%2520Experimental%2520Studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4245%26epage%3D4256%26doi%3D10.1021%2Facs.jmedchem.5b01643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capelli, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entrena
Guadix, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costantino, G.</span></span> <span> </span><span class="NLM_article-title">Unbinding Pathways from the Glucocorticoid Receptor Shed Light on the Reduced Sensitivity of Glucocorticoid Ligands to a Naturally Occurring, Clinically Relevant Mutant Receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">7003</span>– <span class="NLM_lpage">7014</span>, <span class="refDoi"> DOI: 10.1021/jm400802b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400802b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7003-7014&issue=17&author=A.+M.+Capelliauthor=A.+Brunoauthor=A.+Entrena%0AGuadixauthor=G.+Costantino&title=Unbinding+Pathways+from+the+Glucocorticoid+Receptor+Shed+Light+on+the+Reduced+Sensitivity+of+Glucocorticoid+Ligands+to+a+Naturally+Occurring%2C+Clinically+Relevant+Mutant+Receptor&doi=10.1021%2Fjm400802b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm400802b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400802b%26sid%3Dliteratum%253Aachs%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DBruno%26aufirst%3DA.%26aulast%3DEntrena%2BGuadix%26aufirst%3DA.%26aulast%3DCostantino%26aufirst%3DG.%26atitle%3DUnbinding%2520Pathways%2520from%2520the%2520Glucocorticoid%2520Receptor%2520Shed%2520Light%2520on%2520the%2520Reduced%2520Sensitivity%2520of%2520Glucocorticoid%2520Ligands%2520to%2520a%2520Naturally%2520Occurring%252C%2520Clinically%2520Relevant%2520Mutant%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D17%26spage%3D7003%26epage%3D7014%26doi%3D10.1021%2Fjm400802b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Vivo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Role of Molecular Dynamics and Related Methods in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4035</span>– <span class="NLM_lpage">4061</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4035-4061&issue=9&author=M.+De+Vivoauthor=M.+Masettiauthor=G.+Bottegoniauthor=A.+Cavalli&title=Role+of+Molecular+Dynamics+and+Related+Methods+in+Drug+Discovery&doi=10.1021%2Facs.jmedchem.5b01684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></div><div class="casAuthors">De Vivo, Marco; Masetti, Matteo; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4035-4061</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery.  Their main advantage is in explicitly treating structural flexibility and entropic effects.  This allows a more accurate est. of the thermodn. and kinetics assocd. with drug-target recognition and binding, as better algorithms and hardware architectures increase their use.  Here, we review the theor. background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug-target binding.  We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand-target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy.  Further issues discussed include allosteric modulation and the role of water mols. in ligand binding and optimization.  We conclude by calling for more prospective studies to attest to these methods' utility in discovering novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQb_02a7Nmh7Vg90H21EOLACvtfcHk0lhFx6gTomYIew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D&md5=b3d8a4fb5705a3555c2bb2a962420396</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRole%2520of%2520Molecular%2520Dynamics%2520and%2520Related%2520Methods%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4035%26epage%3D4061%26doi%3D10.1021%2Facs.jmedchem.5b01684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mollica, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decherchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zia, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocchia, W.</span></span> <span> </span><span class="NLM_article-title">Kinetics of Protein-Ligand Unbinding via Smoothed Potential Molecular Dynamics Simulations</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11539</span>, <span class="refDoi"> DOI: 10.1038/srep11539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fsrep11539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=26103621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC2MbmsFKmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11539&author=L.+Mollicaauthor=S.+Decherchiauthor=S.+R.+Ziaauthor=R.+Gaspariauthor=A.+Cavalliauthor=W.+Rocchia&title=Kinetics+of+Protein-Ligand+Unbinding+via+Smoothed+Potential+Molecular+Dynamics+Simulations&doi=10.1038%2Fsrep11539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of protein-ligand unbinding via smoothed potential molecular dynamics simulations</span></div><div class="casAuthors">Mollica Luca; Decherchi Sergio; Zia Syeda Rehana; Gaspari Roberto; Rocchia Walter; Cavalli Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug discovery is expensive and high-risk.  Its main reasons of failure are lack of efficacy and toxicity of a drug candidate.  Binding affinity for the biological target has been usually considered one of the most relevant figures of merit to judge a drug candidate along with bioavailability, selectivity and metabolic properties, which could depend on off-target interactions.  Nevertheless, affinity does not always satisfactorily correlate with in vivo drug efficacy.  It is indeed becoming increasingly evident that the time a drug spends in contact with its target (aka residence time) can be a more reliable figure of merit.  Experimental kinetic measurements are operatively limited by the cost and the time needed to synthesize compounds to be tested, to express and purify the target, and to setup the assays.  We present here a simple and efficient molecular-dynamics-based computational approach to prioritize compounds according to their residence time.  We devised a multiple-replica scaled molecular dynamics protocol with suitably defined harmonic restraints to accelerate the unbinding events while preserving the native fold.  Ligands are ranked according to the mean observed scaled unbinding time.  The approach, trivially parallel and easily implementable, was validated against experimental information available on biological systems of pharmacological relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxJkC0jZN2SEH3Rlp_k91zfW6udTcc2eaYypcPYKwxwbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbmsFKmsg%253D%253D&md5=8e3963bce57cf49954ea50fc004283d6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fsrep11539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11539%26sid%3Dliteratum%253Aachs%26aulast%3DMollica%26aufirst%3DL.%26aulast%3DDecherchi%26aufirst%3DS.%26aulast%3DZia%26aufirst%3DS.%2BR.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DRocchia%26aufirst%3DW.%26atitle%3DKinetics%2520of%2520Protein-Ligand%2520Unbinding%2520via%2520Smoothed%2520Potential%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11539%26doi%3D10.1038%2Fsrep11539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span> <span> </span><span class="NLM_article-title">Role of Desolvation in Thermodynamics and Kinetics of Ligand Binding to a Kinase</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">5696</span>– <span class="NLM_lpage">5705</span>, <span class="refDoi"> DOI: 10.1021/ct500584n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct500584n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFWlu7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=5696-5705&issue=12&author=J.+Mondalauthor=R.+A.+Friesnerauthor=B.+J.+Berne&title=Role+of+Desolvation+in+Thermodynamics+and+Kinetics+of+Ligand+Binding+to+a+Kinase&doi=10.1021%2Fct500584n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Role of desolvation in thermodynamics and kinetics of ligand binding to a kinase</span></div><div class="casAuthors">Mondal, Jagannath; Friesner, Richard A.; Berne, B. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5696-5705</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computer simulations were used to det. the free energy landscape for the binding of the anticancer drug, Dasatinib, to its src kinase receptor and showed that before settling into a free energy basin the ligand must surmount a free energy barrier.  An anal. based on using both the ligand-pocket sepn. and the pocket-water occupancy as reaction coordinates showed that the free energy barrier was a result of the free energy cost for almost complete desolvation of the binding pocket.  The simulations further showed that the barrier was not a result of the reorganization free energy of the binding pocket.  Although a continuum solvent model gives the location of free energy min., it is not able to reproduce the intermediate free energy barrier.  Finally, it was shown that a kinetic model for the on-rate const. in which the ligand diffuses up to a doorway state and then surmounts the desolvation free energy barrier was consistent with published microsecond time-scale simulations of the ligand binding kinetics for this system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT08E1EfK8nLVg90H21EOLACvtfcHk0lhPkO8VPA47sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWlu7nE&md5=e7d134fead395023a230489455d61e69</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fct500584n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct500584n%26sid%3Dliteratum%253Aachs%26aulast%3DMondal%26aufirst%3DJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26atitle%3DRole%2520of%2520Desolvation%2520in%2520Thermodynamics%2520and%2520Kinetics%2520of%2520Ligand%2520Binding%2520to%2520a%2520Kinase%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26issue%3D12%26spage%3D5696%26epage%3D5705%26doi%3D10.1021%2Fct500584n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Colizzi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perozzo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scapozza, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recanatini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Single-Molecule Pulling Simulations Can Discern Active from Inactive Enzyme Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">7361</span>– <span class="NLM_lpage">7371</span>, <span class="refDoi"> DOI: 10.1021/ja100259r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja100259r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=7361-7371&issue=21&author=F.+Colizziauthor=R.+Perozzoauthor=L.+Scapozzaauthor=M.+Recanatiniauthor=A.+Cavalli&title=Single-Molecule+Pulling+Simulations+Can+Discern+Active+from+Inactive+Enzyme+Inhibitors&doi=10.1021%2Fja100259r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fja100259r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja100259r%26sid%3Dliteratum%253Aachs%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DPerozzo%26aufirst%3DR.%26aulast%3DScapozza%26aufirst%3DL.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSingle-Molecule%2520Pulling%2520Simulations%2520Can%2520Discern%2520Active%2520from%2520Inactive%2520Enzyme%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26issue%3D21%26spage%3D7361%26epage%3D7371%26doi%3D10.1021%2Fja100259r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery: Pulled from a Protein’s Embrace</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span> (<span class="NLM_issue">7302</span>),  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1038/466042a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2F466042a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=42-43&issue=7302&author=W.+L.+Jorgensen&title=Drug+Discovery%3A+Pulled+from+a+Protein%E2%80%99s+Embrace&doi=10.1038%2F466042a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F466042a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F466042a%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDrug%2520Discovery%253A%2520Pulled%2520from%2520a%2520Protein%25E2%2580%2599s%2520Embrace%26jtitle%3DNature%26date%3D2010%26volume%3D466%26issue%3D7302%26spage%3D42%26epage%3D43%26doi%3D10.1038%2F466042a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G.</span>, <span class="NLM_string-name">Swinney, D. C.</span></span>, Eds.  <i>Thermodynamics and Kinetics of Drug Binding</i>; Methods and Principles in Medicinal Chemistry, Vol. 65; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">329</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2F9783527673025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=315-329&author=G.+Keseru&author=D.+C.+Swinney&title=Thermodynamics+and+Kinetics+of+Drug+Binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2F9783527673025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527673025%26sid%3Dliteratum%253Aachs%26aulast%3DKeseru%26aufirst%3DG.%26btitle%3DThermodynamics%2520and%2520Kinetics%2520of%2520Drug%2520Binding%26pub%3DWiley%26date%3D2015%26spage%3D315%26epage%3D329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Molecular Determinants of Drug-Receptor Binding Kinetics</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">13–14</span>),  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">673</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.drudis.2013.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=23454741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVyrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=667-673&issue=13%E2%80%9314&author=A.+C.+Panauthor=D.+W.+Borhaniauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Molecular+Determinants+of+Drug-Receptor+Binding+Kinetics&doi=10.1016%2Fj.drudis.2013.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular determinants of drug-receptor binding kinetics</span></div><div class="casAuthors">Pan, Albert C.; Borhani, David W.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">667-673</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  It is increasingly appreciated that the rates at which drugs assoc. with and dissoc. from receptors - the binding kinetics - directly impact drug efficacy and safety.  The mol. determinants of drug-receptor binding kinetics remain poorly understood, however, esp. when compared with the well-known factors that affect binding affinity.  The rational modulation of kinetics during lead optimization thus remains challenging.  We review some of the key factors thought to control drug-receptor binding kinetics at the mol. level - mol. size, conformational fluctuations, electrostatic interactions and hydrophobic effects - and discuss several possible approaches for the rational design of drugs with desired binding kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAvEt3tx06Z7Vg90H21EOLACvtfcHk0ljoMZj3c7fs_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVyrtr0%253D&md5=df094bb86b1e280b9ddf5a90031b3f32</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DMolecular%2520Determinants%2520of%2520Drug-Receptor%2520Binding%2520Kinetics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26issue%3D13%25E2%2580%259314%26spage%3D667%26epage%3D673%26doi%3D10.1016%2Fj.drudis.2013.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Setny, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzubiella, J.</span></span> <span> </span><span class="NLM_article-title">Dewetting-Controlled Binding of Ligands to Hydrophobic Pockets</span>. <i>Phys. Rev. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">187801</span>, <span class="refDoi"> DOI: 10.1103/PhysRevLett.103.187801</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1103%2FPhysRevLett.103.187801" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=19905832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlKit7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2009&pages=187801&issue=18&author=P.+Setnyauthor=Z.+Wangauthor=L.-T.+Chengauthor=B.+Liauthor=J.+A.+McCammonauthor=J.+Dzubiella&title=Dewetting-Controlled+Binding+of+Ligands+to+Hydrophobic+Pockets&doi=10.1103%2FPhysRevLett.103.187801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Dewetting-Controlled Binding of Ligands to Hydrophobic Pockets</span></div><div class="casAuthors">Setny, P.; Wang, Z.; Cheng, L.-T.; Li, B.; McCammon, J. A.; Dzubiella, J.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">187801/1-187801/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">We report on a combined atomistic mol. dynamics simulation and implicit solvent anal. of a generic hydrophobic pocket-ligand (host-guest) system.  The approaching ligand induces complex wetting-dewetting transitions in the weakly solvated pocket.  The transitions lead to bimodal solvent fluctuations which govern magnitude and range of the pocket-ligand attraction.  A recently developed implicit water model, based on the minimization of a geometric functional, captures the sensitive aq. interface response to the concave-convex pocket-ligand configuration semiquant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ1xmGB8ESgrVg90H21EOLACvtfcHk0ljoMZj3c7fs_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlKit7zP&md5=f387b92e561428e0c3cb860eefa3d530</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.103.187801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.103.187801%26sid%3Dliteratum%253Aachs%26aulast%3DSetny%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DCheng%26aufirst%3DL.-T.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DDzubiella%26aufirst%3DJ.%26atitle%3DDewetting-Controlled%2520Binding%2520of%2520Ligands%2520to%2520Hydrophobic%2520Pockets%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2009%26volume%3D103%26issue%3D18%26spage%3D187801%26doi%3D10.1103%2FPhysRevLett.103.187801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhtiyarova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springman, E. B.</span></span> <span> </span><span class="NLM_article-title">A Biacore Biosensor Method for Detailed Kinetic Binding Analysis of Small Molecule Inhibitors of P38α Mitogen-Activated Protein Kinase</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>325</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2003.10.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.ab.2003.10.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2004&pages=126-136&issue=1&author=D.+Casperauthor=M.+Bukhtiyarovaauthor=E.+B.+Springman&title=A+Biacore+Biosensor+Method+for+Detailed+Kinetic+Binding+Analysis+of+Small+Molecule+Inhibitors+of+P38%CE%B1+Mitogen-Activated+Protein+Kinase&doi=10.1016%2Fj.ab.2003.10.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2003.10.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2003.10.025%26sid%3Dliteratum%253Aachs%26aulast%3DCasper%26aufirst%3DD.%26aulast%3DBukhtiyarova%26aufirst%3DM.%26aulast%3DSpringman%26aufirst%3DE.%2BB.%26atitle%3DA%2520Biacore%2520Biosensor%2520Method%2520for%2520Detailed%2520Kinetic%2520Binding%2520Analysis%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520P38%25CE%25B1%2520Mitogen-Activated%2520Protein%2520Kinase%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D325%26issue%3D1%26spage%3D126%26epage%3D136%26doi%3D10.1016%2Fj.ab.2003.10.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winquist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavsson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deinum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span> <span> </span><span class="NLM_article-title">Identification of Structural–Kinetic and Structural–Thermodynamic Relationships for Thrombin Inhibitors</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">613</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1021/bi301333z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi301333z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=613-626&issue=4&author=J.+Winquistauthor=S.+Geschwindnerauthor=Y.+Xueauthor=L.+Gustavssonauthor=D.+Musilauthor=J.+Deinumauthor=U.+H.+Danielson&title=Identification+of+Structural%E2%80%93Kinetic+and+Structural%E2%80%93Thermodynamic+Relationships+for+Thrombin+Inhibitors&doi=10.1021%2Fbi301333z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fbi301333z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi301333z%26sid%3Dliteratum%253Aachs%26aulast%3DWinquist%26aufirst%3DJ.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DXue%26aufirst%3DY.%26aulast%3DGustavsson%26aufirst%3DL.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DDeinum%26aufirst%3DJ.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26atitle%3DIdentification%2520of%2520Structural%25E2%2580%2593Kinetic%2520and%2520Structural%25E2%2580%2593Thermodynamic%2520Relationships%2520for%2520Thrombin%2520Inhibitors%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26issue%3D4%26spage%3D613%26epage%3D626%26doi%3D10.1021%2Fbi301333z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, H.-J.</span></span> <span> </span><span class="NLM_article-title">Binding of Ligands to GPCRs – How Valid Is a Thermodynamic Discrimination of Antagonists and Agonists?</span>. <i>J. Phys. Chem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>S1</i></span>,  <span class="NLM_fpage">001</span>, <span class="refDoi"> DOI: 10.4172/2161-0398.S1-001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.4172%2F2161-0398.S1-001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=S1&publication_year=2012&pages=001&author=A.+Strasserauthor=H.-J.+Wittmann&title=Binding+of+Ligands+to+GPCRs+%E2%80%93+How+Valid+Is+a+Thermodynamic+Discrimination+of+Antagonists+and+Agonists%3F&doi=10.4172%2F2161-0398.S1-001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.4172%2F2161-0398.S1-001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F2161-0398.S1-001%26sid%3Dliteratum%253Aachs%26aulast%3DStrasser%26aufirst%3DA.%26aulast%3DWittmann%26aufirst%3DH.-J.%26atitle%3DBinding%2520of%2520Ligands%2520to%2520GPCRs%2520%25E2%2580%2593%2520How%2520Valid%2520Is%2520a%2520Thermodynamic%2520Discrimination%2520of%2520Antagonists%2520and%2520Agonists%253F%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Biophys.%26date%3D2012%26volume%3DS1%26spage%3D001%26doi%3D10.4172%2F2161-0398.S1-001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shuman, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hämäläinen, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span> <span> </span><span class="NLM_article-title">Kinetic and Thermodynamic Characterization of HIV-1 Protease Inhibitors</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1002/jmr.655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2Fjmr.655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=15027031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1altLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2004&pages=106-119&issue=2&author=C.+F.+Shumanauthor=M.+D.+H%C3%A4m%C3%A4l%C3%A4inenauthor=U.+H.+Danielson&title=Kinetic+and+Thermodynamic+Characterization+of+HIV-1+Protease+Inhibitors&doi=10.1002%2Fjmr.655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and thermodynamic characterization of HIV-1 protease inhibitors</span></div><div class="casAuthors">Shuman, Cynthia F.; Haemaelaeinen, Markku D.; Danielson, U. Helena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">106-119</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Interaction kinetic and thermodn. analyses provide information beyond that obtained in general inhibition studies, and may contribute to the design of improved inhibitors and increased understanding of mol. interactions.  Thus, a biosensor-based method was used to characterize the interactions between HIV-1 protease and seven inhibitors, revealing distinguishing kinetic and thermodn. characteristics for the inhibitors.  Lopinavir had fast assocn. and the highest affinity of the tested compds., and the interaction kinetics were less temp.-dependent as compared with the other inhibitors.  Amprenavir, indinavir and ritonavir showed non-linear temp. dependencies of the kinetics.  The free energy, enthalpy and entropy (ΔG, ΔH, ΔS) were detd., and the energetics of complex assocn. (ΔGon, ΔHon, ΔSon) and dissocn. (ΔGoff, ΔHoff, ΔSoff) were resolved.  In general, the energetics for the studied inhibitors was in the same range, with the neg. free energy change (ΔG < 0) due primarily to increased entropy (ΔS > 0).  Thus, the driving force of the interaction was increased degrees of freedom in the system (entropy) rather than the formation of bonds between the enzyme and inhibitor (enthalpy).  Although the ΔGon and ΔGoff were in the same range for all inhibitors, the enthalpy and entropy terms contributed differently to assocn. and dissocn., distinguishing these phases energetically.  Dissocn. was accompanied by pos. enthalpy (ΔHoff > 0) and neg. entropy (ΔSoff < 0) changes, whereas assocn. for all inhibitors except lopinavir had pos. entropy changes (ΔSon > 0), demonstrating unique energetic characteristics for lopinavir.  This study indicates that this type of data will be useful for the characterization of target-ligand interactions and the development of new inhibitors of HIV-1 protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr6QEHUdEX6bVg90H21EOLACvtfcHk0ljg79hyZvhmIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1altLk%253D&md5=a5e3e6273f1cc03d2454ecfce598da30</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjmr.655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.655%26sid%3Dliteratum%253Aachs%26aulast%3DShuman%26aufirst%3DC.%2BF.%26aulast%3DH%25C3%25A4m%25C3%25A4l%25C3%25A4inen%26aufirst%3DM.%2BD.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26atitle%3DKinetic%2520and%2520Thermodynamic%2520Characterization%2520of%2520HIV-1%2520Protease%2520Inhibitors%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2004%26volume%3D17%26issue%3D2%26spage%3D106%26epage%3D119%26doi%3D10.1002%2Fjmr.655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajpai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higazi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span> <span> </span><span class="NLM_article-title">Structural and Dynamic Insights into the Energetics of Activation Loop Rearrangement in FGFR1 Kinase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">7877</span>, <span class="refDoi"> DOI: 10.1038/ncomms8877</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1038%2Fncomms8877" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=26203596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FmvVOktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7877&author=T.+Kleinauthor=N.+Vajpaiauthor=J.+J.+Phillipsauthor=G.+Daviesauthor=G.+A.+Holdgateauthor=C.+Phillipsauthor=J.+A.+Tuckerauthor=R.+A.+Normanauthor=A.+D.+Scottauthor=D.+R.+Higaziauthor=D.+Loweauthor=G.+S.+Thompsonauthor=A.+L.+Breeze&title=Structural+and+Dynamic+Insights+into+the+Energetics+of+Activation+Loop+Rearrangement+in+FGFR1+Kinase&doi=10.1038%2Fncomms8877"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and dynamic insights into the energetics of activation loop rearrangement in FGFR1 kinase</span></div><div class="casAuthors">Klein Tobias; Vajpai Navratna; Davies Gareth; Holdgate Geoffrey A; Phillips Chris; Tucker Julie A; Norman Richard A; Scott Andrew D; Breeze Alexander L; Phillips Jonathan J; Higazi Daniel R; Lowe David; Thompson Gary S; Breeze Alexander L</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7877</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein tyrosine kinases differ widely in their propensity to undergo rearrangements of the N-terminal Asp-Phe-Gly (DFG) motif of the activation loop, with some, including FGFR1 kinase, appearing refractory to this so-called 'DFG flip'.  Recent inhibitor-bound structures have unexpectedly revealed FGFR1 for the first time in a 'DFG-out' state.  Here we use conformationally selective inhibitors as chemical probes for interrogation of the structural and dynamic features that appear to govern the DFG flip in FGFR1.  Our detailed structural and biophysical insights identify contributions from altered dynamics in distal elements, including the αH helix, towards the outstanding stability of the DFG-out complex with the inhibitor ponatinib.  We conclude that the αC-β4 loop and 'molecular brake' regions together impose a high energy barrier for this conformational rearrangement, and that this may have significance for maintaining autoinhibition in the non-phosphorylated basal state of FGFR1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSmhfFatniy2WmbXvSEQle5fW6udTcc2ea0Z9BAwd2jG7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FmvVOktQ%253D%253D&md5=ddc5f597e82c3432831a86c65dac2f26</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fncomms8877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8877%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DT.%26aulast%3DVajpai%26aufirst%3DN.%26aulast%3DPhillips%26aufirst%3DJ.%2BJ.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DHoldgate%26aufirst%3DG.%2BA.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DScott%26aufirst%3DA.%2BD.%26aulast%3DHigazi%26aufirst%3DD.%2BR.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DThompson%26aufirst%3DG.%2BS.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26atitle%3DStructural%2520and%2520Dynamic%2520Insights%2520into%2520the%2520Energetics%2520of%2520Activation%2520Loop%2520Rearrangement%2520in%2520FGFR1%2520Kinase%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7877%26doi%3D10.1038%2Fncomms8877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutzner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span> <span> </span><span class="NLM_article-title">GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1021/ct700301q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700301q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=435-447&issue=3&author=B.+Hessauthor=C.+Kutznerauthor=D.+van+der%0ASpoelauthor=E.+Lindahl&title=GROMACS+4%3A+Algorithms+for+Highly+Efficient%2C+Load-Balanced%2C+and+Scalable+Molecular+Simulation&doi=10.1021%2Fct700301q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation</span></div><div class="casAuthors">Hess, Berk; Kutzner, Carsten; van der Spoel, David; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">435-447</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. simulation is an extremely useful, but computationally very expensive tool for studies of chem. and biomol. systems.  Here, we present a new implementation of our mol. simulation toolkit GROMACS which now both achieves extremely high performance on single processors from algorithmic optimizations and hand-coded routines and simultaneously scales very well on parallel machines.  The code encompasses a minimal-communication domain decompn. algorithm, full dynamic load balancing, a state-of-the-art parallel constraint solver, and efficient virtual site algorithms that allow removal of hydrogen atom degrees of freedom to enable integration time steps up to 5 fs for atomistic simulations also in parallel.  To improve the scaling properties of the common particle mesh Ewald electrostatics algorithms, we have in addn. used a Multiple-Program, Multiple-Data approach, with sep. node domains responsible for direct and reciprocal space interactions.  Not only does this combination of algorithms enable extremely long simulations of large systems but also it provides that simulation performance on quite modest nos. of std. cluster nodes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrajjWdk6mUFrVg90H21EOLACvtfcHk0ljg79hyZvhmIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D&md5=d53c94901386260221792ea30f151c5f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fct700301q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700301q%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DKutzner%26aufirst%3DC.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGROMACS%25204%253A%2520Algorithms%2520for%2520Highly%2520Efficient%252C%2520Load-Balanced%252C%2520and%2520Scalable%2520Molecular%2520Simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26issue%3D3%26spage%3D435%26epage%3D447%26doi%3D10.1021%2Fct700301q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Der Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groenhof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">GROMACS: Fast, Flexible, and Free</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&issue=16&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+C.+Berendsen&title=GROMACS%3A+Fast%2C+Flexible%2C+and+Free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0lgmW-vN7OO0ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DGROMACS%253A%2520Fast%252C%2520Flexible%252C%2520and%2520Free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26issue%3D16%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span> <span> </span><span class="NLM_article-title">GROMACS 3.0: A Package for Molecular Simulation and Trajectory Analysis</span>. <i>Mol. Model. Annu.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1007/s008940100045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1007%2Fs008940100045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsVGjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=306-317&issue=8&author=E.+Lindahlauthor=B.+Hessauthor=D.+van+der%0ASpoel&title=GROMACS+3.0%3A+A+Package+for+Molecular+Simulation+and+Trajectory+Analysis&doi=10.1007%2Fs008940100045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 3.0: a package for molecular simulation and trajectory analysis</span></div><div class="casAuthors">Lindahl, Erik; Hess, Berk; van der Spoel, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling [online computer file]</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">306-317</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">GROMACS 3.0 is the latest release of a versatile and very well optimized package for mol. simulation.  Much effort has been devoted to achieving extremely high performance on both workstations and parallel comouters.  The design includes an extn. of virial and periodic boundary conditions from the loops over pairwise interactions, and special software routines to enable rapid calcn. of x-1/2.  Inner loops are generated automatically in C or Fortran at compile time, with optimizations adapted to each architecture.  Assembly loops using SSE and 3DNow! multimedia instructions are provided for x86 processors, resulting in exceptional performance on inexpensive PC workstations.  The interface is simple and easy to use, based on std. command line arguments with self-explanatory functionality and integrated documentation.  A large collection of flexible tools for trajectory anal. is included, with output in the form of finished Xmgr/Grace graphs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2xBKmKIYW3bVg90H21EOLACvtfcHk0lgmW-vN7OO0ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsVGjsL4%253D&md5=fca1fafdbbab2f42b3ab9c789b8a048e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2Fs008940100045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs008940100045%26sid%3Dliteratum%253Aachs%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26atitle%3DGROMACS%25203.0%253A%2520A%2520Package%2520for%2520Molecular%2520Simulation%2520and%2520Trajectory%2520Analysis%26jtitle%3DMol.%2520Model.%2520Annu.%26date%3D2001%26volume%3D7%26issue%3D8%26spage%3D306%26epage%3D317%26doi%3D10.1007%2Fs008940100045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Drunen, R.</span></span> <span> </span><span class="NLM_article-title">GROMACS: A Message-Passing Parallel Molecular Dynamics Implementation</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>91</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/0010-4655(95)00042-E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2F0010-4655%2895%2900042-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaK2MXps1Wrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1995&pages=43-56&issue=1&author=H.+J.+C.+Berendsenauthor=D.+van+der+Spoelauthor=R.+van+Drunen&title=GROMACS%3A+A+Message-Passing+Parallel+Molecular+Dynamics+Implementation&doi=10.1016%2F0010-4655%2895%2900042-E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: A message-passing parallel molecular dynamics implementation</span></div><div class="casAuthors">Berendsen, H. J. C.; van der Spoel, D.; van Drunen, R.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">43-56</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A parallel message-passing implementation of a mol. dynamics (MD) program that is useful for bio(macro)mols. in aq. environment is described.  The software has been developed for a custom-designed 32-processor ring GROMACS (Groningen MAchine for Chem. Simulation) with communication to and from left and right neighbors, but can run on any parallel system onto which a a ring of processors can be mapped and which supports PVM-like block send and receive calls.  The GROMACS software consists of a preprocessor, a parallel MD and energy minimization program that can use an arbitrary no. of processors (including one), an optional monitor, and several anal. tools.  The programs are written in ANSI C and available by ftp (information: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="0c6b7e63616d6f7f4c6f646961227e796b226260">[email protected]</a>).  The functionality is based on the GROMOS (Groningen Mol. Simulation) package (van Gunsteren and Berendsen, 1987; BIOMOS B.V., Nijenborgh 4, 9747 AG Groningen).  Conversion programs between GROMOS and GROMACS formats are included.The MD program can handle rectangular periodic boundary conditions with temp. and pressure scaling.  The interactions that can be handled without modification are variable non-bonded pair interactions with Coulomb and Lennard-Jones or Buckingham potentials, using a twin-range cut-off based on charge groups, and fixed bonded interactions of either harmonic or constraint type for bonds and bond angles and either periodic or cosine power series interactions for dihedral angles.  Special forces can be added to groups of particles (for non-equil. dynamics or for position restraining) or between particles (for distance restraints).  The parallelism is based on particle decompn.  Interprocessor communication is largely limited to position and force distribution over the ring once per time step.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovcOlT-z9eKLVg90H21EOLACvtfcHk0lgmW-vN7OO0ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXps1Wrtr0%253D&md5=04d823aeab28ca374efb86839c705179</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2F0010-4655%2895%2900042-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0010-4655%252895%252900042-E%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3Dvan%2BDrunen%26aufirst%3DR.%26atitle%3DGROMACS%253A%2520A%2520Message-Passing%2520Parallel%2520Molecular%2520Dynamics%2520Implementation%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D1995%26volume%3D91%26issue%3D1%26spage%3D43%26epage%3D56%26doi%3D10.1016%2F0010-4655%2895%2900042-E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichenberger, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choutko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">Definition and Testing of the GROMOS Force-Field Versions 54A7 and 54B7</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">843</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1007/s00249-011-0700-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1007%2Fs00249-011-0700-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=21533652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVCgs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=843-856&issue=7&author=N.+Schmidauthor=A.+P.+Eichenbergerauthor=A.+Choutkoauthor=S.+Rinikerauthor=M.+Wingerauthor=A.+E.+Markauthor=W.+F.+van+Gunsteren&title=Definition+and+Testing+of+the+GROMOS+Force-Field+Versions+54A7+and+54B7&doi=10.1007%2Fs00249-011-0700-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Definition and testing of the GROMOS force-field versions 54A7 and 54B7</span></div><div class="casAuthors">Schmid, Nathan; Eichenberger, Andreas P.; Choutko, Alexandra; Riniker, Sereina; Winger, Moritz; Mark, Alan E.; Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">843-856</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">New parameter sets of the GROMOS biomol. force field, 54A7 and 54B7, are introduced.  These parameter sets summarize some previously published force field modifications: The 53A6 helical propensities are cor. through new φ/ψ torsional angle terms and a modification of the N-H, C=O repulsion, a new atom type for a charged -CH3 in the choline moiety is added, the Na+ and Cl- ions are modified to reproduce the free energy of hydration, and addnl. improper torsional angle types for free energy calcns. involving a chirality change are introduced.  The new helical propensity modification is tested using the benchmark proteins hen egg-white lysozyme, fox1 RNA binding domain, chorismate mutase and the GCN4-p1 peptide.  The stability of the proteins is improved in comparison with the 53A6 force field, and good agreement with a range of primary exptl. data is obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3tjHc1auDa7Vg90H21EOLACvtfcHk0lgmW-vN7OO0ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVCgs74%253D&md5=a1cd133f1e663da949324060e00c950d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs00249-011-0700-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00249-011-0700-9%26sid%3Dliteratum%253Aachs%26aulast%3DSchmid%26aufirst%3DN.%26aulast%3DEichenberger%26aufirst%3DA.%2BP.%26aulast%3DChoutko%26aufirst%3DA.%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DWinger%26aufirst%3DM.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DDefinition%2520and%2520Testing%2520of%2520the%2520GROMOS%2520Force-Field%2520Versions%252054A7%2520and%252054B7%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D2011%26volume%3D40%26issue%3D7%26spage%3D843%26epage%3D856%26doi%3D10.1007%2Fs00249-011-0700-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bermudez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgueitio, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolber, G.</span></span> <span> </span><span class="NLM_article-title">The Impact of Molecular Dynamics on Drug Design: Applications for the Characterization of Ligand-Macromolecule Complexes</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.drudis.2015.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=25615716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFSjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=686-702&issue=6&author=J.+Mortierauthor=C.+Rakersauthor=M.+Bermudezauthor=M.+S.+Murgueitioauthor=S.+Rinikerauthor=G.+Wolber&title=The+Impact+of+Molecular+Dynamics+on+Drug+Design%3A+Applications+for+the+Characterization+of+Ligand-Macromolecule+Complexes&doi=10.1016%2Fj.drudis.2015.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of molecular dynamics on drug design: applications for the characterization of ligand-macromolecule complexes</span></div><div class="casAuthors">Mortier, Jeremie; Rakers, Christin; Bermudez, Marcel; Murgueitio, Manuela S.; Riniker, Sereina; Wolber, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">686-702</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Among all tools available to design new drugs, mol. dynamics (MD) simulations have become an essential technique.  Initially developed to investigate mol. models with a limited no. of atoms, computers now enable investigations of large macromol. systems with a simulation time reaching the microsecond range.  The reviewed articles cover four years of research to give an overview on the actual impact of MD on the current medicinal chem. landscape with a particular emphasis on studies of ligand-protein interactions.  With a special focus on studies combining computational approaches with data gained from other techniques, this review shows how deeply embedded MD simulations are in drug design strategies and articulates what the future of this technique could be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaq-ZlpVGoG7Vg90H21EOLACvtfcHk0liK54sp_Lu6oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFSjtbk%253D&md5=36aba5288f374baf50b2831225047c7c</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DMortier%26aufirst%3DJ.%26aulast%3DRakers%26aufirst%3DC.%26aulast%3DBermudez%26aufirst%3DM.%26aulast%3DMurgueitio%26aufirst%3DM.%2BS.%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DWolber%26aufirst%3DG.%26atitle%3DThe%2520Impact%2520of%2520Molecular%2520Dynamics%2520on%2520Drug%2520Design%253A%2520Applications%2520for%2520the%2520Characterization%2520of%2520Ligand-Macromolecule%2520Complexes%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26issue%3D6%26spage%3D686%26epage%3D702%26doi%3D10.1016%2Fj.drudis.2015.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristòfol-Clough, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Efficient Round-Trip Time Optimization for Replica-Exchange Enveloping Distribution Sampling (RE-EDS)</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">3020</span>– <span class="NLM_lpage">3030</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.7b00286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.7b00286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFChtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=3020-3030&issue=6&author=D.+Sidlerauthor=M.+Crist%C3%B2fol-Cloughauthor=S.+Riniker&title=Efficient+Round-Trip+Time+Optimization+for+Replica-Exchange+Enveloping+Distribution+Sampling+%28RE-EDS%29&doi=10.1021%2Facs.jctc.7b00286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Round-Trip Time Optimization for Replica-Exchange Enveloping Distribution Sampling (RE-EDS)</span></div><div class="casAuthors">Sidler, Dominik; Cristofol-Clough, Michael; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3020-3030</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Replica-exchange enveloping distribution sampling (RE-EDS) allows the efficient estn. of free-energy differences between multiple end-states from a single mol. dynamics (MD) simulation.  In EDS, a ref. state is sampled, which can be tuned by two types of parameters, i.e., smoothness parameters(s) and energy offsets, such that all end-states are sufficiently sampled.  However, the choice of these parameters is not trivial.  Replica exchange (RE) or parallel tempering is a widely applied technique to enhance sampling.  By combining EDS with the RE technique, the parameter choice problem could be simplified and the challenge shifted toward an optimal distribution of the replicas in the smoothness-parameter space.  The choice of a certain replica distribution can alter the sampling efficiency significantly.  In this work, global round-trip time optimization (GRTO) algorithms are tested for the use in RE-EDS simulations.  In addn., a local round-trip time optimization (LRTO) algorithm is proposed for systems with slowly adapting environments, where a reliable est. for the round-trip time is challenging to obtain.  The optimization algorithms were applied to RE-EDS simulations of a system of nine small-mol. inhibitors of phenylethanolamine N-methyltransferase (PNMT).  The energy offsets were detd. using the recently proposed parallel energy-offset (PEOE) estn. scheme.  While the multi-state GRTO algorithm yielded the best replica distribution for the ligands in water, the multi-state LRTO algorithm was found to be the method of choice for the ligands in complex with PNMT.  With this, the 36 alchem. free-energy differences between the nine ligands were calcd. successfully from a single RE-EDS simulation 10 ns in length.  Thus, RE-EDS presents an efficient method for the estn. of relative binding free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKivaoB2NS_7Vg90H21EOLACvtfcHk0liK54sp_Lu6oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFChtrk%253D&md5=97e910746ea8e746ba56ec05c5061e85</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.7b00286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.7b00286%26sid%3Dliteratum%253Aachs%26aulast%3DSidler%26aufirst%3DD.%26aulast%3DCrist%25C3%25B2fol-Clough%26aufirst%3DM.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DEfficient%2520Round-Trip%2520Time%2520Optimization%2520for%2520Replica-Exchange%2520Enveloping%2520Distribution%2520Sampling%2520%2528RE-EDS%2529%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26issue%3D6%26spage%3D3020%26epage%3D3030%26doi%3D10.1021%2Facs.jctc.7b00286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grandits, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oostenbrink, C.</span></span> <span> </span><span class="NLM_article-title">Selectivity of Cytosolic Phospholipase A2 Type IV toward Arachidonyl Phospholipids</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1002/jmr.2462</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2Fjmr.2462" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=25703463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps12qtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=447-457&issue=7&author=M.+Granditsauthor=C.+Oostenbrink&title=Selectivity+of+Cytosolic+Phospholipase+A2+Type+IV+toward+Arachidonyl+Phospholipids&doi=10.1002%2Fjmr.2462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity of cytosolic phospholipase A2 type IV toward arachidonyl phospholipids</span></div><div class="casAuthors">Grandits, Melanie; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">447-457</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cytosolic phospholipase A2 (cPLA2) is an interesting protein involved in inflammatory processes and various diseases.  Its catalytic mechanism as well as its substrate specificity for arachidonyl phospholipids is not typical for other phospolipases.  Furthermore, a lid structure, which ensures a hydrophilic surface of the protein without any substrate bound and the movement of this flexible loop to make the hydrophobic active site accessible, is of high interest.  Therefore, the focus of this work was to det. the binding mode of cPLA2 with various substrates, such as arachidonic acid, a synthetic inhibitor, a satd. phospholipid, and most importantly an arachidonyl phospholipid.  To understand the selectivity of the protein toward the arachidonyl phospholipid and the interaction in a protein-ligand complex, mol. dynamics (MD) simulations were performed using the GROMOS suite of simulation programs.  The MD simulations provided insight into the protein and showed that selective binding of arachidonyl phospholipids was because of the shape of the sn-2 tail.  The cPLA2 residues Asn-555 and Ala-578 were involved in the strongest interactions obsd. in the protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9uTIv9KWpFrVg90H21EOLACvtfcHk0liK54sp_Lu6oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps12qtb4%253D&md5=935bd5fca598ecc56684cd3917a06a9e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fjmr.2462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.2462%26sid%3Dliteratum%253Aachs%26aulast%3DGrandits%26aufirst%3DM.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DSelectivity%2520of%2520Cytosolic%2520Phospholipase%2520A2%2520Type%2520IV%2520toward%2520Arachidonyl%2520Phospholipids%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2015%26volume%3D28%26issue%3D7%26spage%3D447%26epage%3D457%26doi%3D10.1002%2Fjmr.2462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pechlaner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oostenbrink, C.</span></span> <span> </span><span class="NLM_article-title">Multiple Binding Poses in the Hydrophobic Cavity of Bee Odorant Binding Protein AmelOBP14</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2633</span>– <span class="NLM_lpage">2643</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFClsrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2633-2643&issue=12&author=M.+Pechlanerauthor=C.+Oostenbrink&title=Multiple+Binding+Poses+in+the+Hydrophobic+Cavity+of+Bee+Odorant+Binding+Protein+AmelOBP14&doi=10.1021%2Facs.jcim.5b00673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Binding Poses in the Hydrophobic Cavity of Bee Odorant Binding Protein AmelOBP14</span></div><div class="casAuthors">Pechlaner, Maria; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2633-2643</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the first step of olfaction, odorants are bound and solubilized by small globular odorant binding proteins (OBPs) which shuttle them to the membrane of a sensory neuron.  Low ligand affinity and selectivity at this step enable the recognition of a wide range of chems.  Honey bee Apis mellifera's OBP14 (AmelOBP14) binds different plant odorants in a largely hydrophobic cavity.  In long mol. dynamics simulations in the presence and absence of ligand eugenol, we observe a highly dynamic C-terminal region which forms one side of the ligand-binding cavity, and the ligand drifts away from its crystd. orientation.  Hamiltonian replica exchange simulations, allowing exchanges of conformations sampled by the real ligand with those sampled by a noninteracting dummy mol. and several intermediates, suggest an alternative, quite different ligand pose which is adopted immediately and which is stable in long simulations.  Thermodn. integration yields binding free energies which are in reasonable agreement with exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXxIhY6DqduLVg90H21EOLACvtfcHk0lgmi5B5DrsULQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFClsrnJ&md5=6466eb870ed2d9b153797d64dd3092c9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00673%26sid%3Dliteratum%253Aachs%26aulast%3DPechlaner%26aufirst%3DM.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DMultiple%2520Binding%2520Poses%2520in%2520the%2520Hydrophobic%2520Cavity%2520of%2520Bee%2520Odorant%2520Binding%2520Protein%2520AmelOBP14%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26issue%3D12%26spage%3D2633%26epage%3D2643%26doi%3D10.1021%2Facs.jcim.5b00673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span>; <span class="NLM_string-name">Postma, J. P. M.</span>; <span class="NLM_string-name">van Gunsteren, W. F.</span>; <span class="NLM_string-name">Hermans, J.</span></span> <span> </span><span class="NLM_article-title">Interaction Models for Water in Relation to Protein Hydration</span>. In  <i>Intermolecular Forces</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pullman, B.</span></span>, Ed.; The Jerusalem Symposia on Quantum Chemistry and Biochemistry; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Dordrecht, The Netherlands</span>, <span class="NLM_year">1981</span>; pp  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">342</span>, DOI: <span class="refDoi"> DOI: 10.1007/978-94-015-7658-1_21</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1007%2F978-94-015-7658-1_21" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1981&pages=331-342&author=H.+J.+C.+Berendsen&author=J.+P.+M.+Postma&author=W.+F.+van+Gunsteren&author=J.+Hermansauthor=B.+Pullman&title=Intermolecular+Forces"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2F978-94-015-7658-1_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-015-7658-1_21%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DInteraction%2520Models%2520for%2520Water%2520in%2520Relation%2520to%2520Protein%2520Hydration%26btitle%3DIntermolecular%2520Forces%26aulast%3DPullman%26aufirst%3DB.%26pub%3DSpringer%26date%3D1981%26spage%3D331%26epage%3D342%26doi%3D10.1007%2F978-94-015-7658-1_21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, J. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haak, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics with Coupling to an External Bath</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">3684</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&issue=8&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van+Gunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+Dynamics+with+Coupling+to+an+External+Bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0lgmi5B5DrsULQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520Dynamics%2520with%2520Coupling%2520to%2520an%2520External%2520Bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26issue%3D8%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donadio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrinello, M.</span></span> <span> </span><span class="NLM_article-title">Canonical Sampling through Velocity Rescaling</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>126</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">014101</span>, <span class="refDoi"> DOI: 10.1063/1.2408420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1063%2F1.2408420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=17212484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BD2sXosVCltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2007&pages=014101&issue=1&author=G.+Bussiauthor=D.+Donadioauthor=M.+Parrinello&title=Canonical+Sampling+through+Velocity+Rescaling&doi=10.1063%2F1.2408420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Canonical sampling through velocity rescaling</span></div><div class="casAuthors">Bussi, Giovanni; Donadio, Davide; Parrinello, Michele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">014101/1-014101/7</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The authors present a new mol. dynamics algorithm for sampling the canonical distribution.  In this approach the velocities of all the particles are rescaled by a properly chosen random factor.  The algorithm is formally justified and it is shown that, in spite of its stochastic nature, a quantity can still be defined that remains const. during the evolution.  In numerical applications this quantity can be used to measure the accuracy of the sampling.  The authors illustrate the properties of this new method on Lennard-Jones and TIP4P water models in the solid and liq. phases.  Its performance is excellent and largely independent of the thermostat parameter also with regard to the dynamic properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgtMYfaJjM3rVg90H21EOLACvtfcHk0lgmi5B5DrsULQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXosVCltg%253D%253D&md5=9c182b57bfc65bca6be23c8c76b4be77</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1063%2F1.2408420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2408420%26sid%3Dliteratum%253Aachs%26aulast%3DBussi%26aufirst%3DG.%26aulast%3DDonadio%26aufirst%3DD.%26aulast%3DParrinello%26aufirst%3DM.%26atitle%3DCanonical%2520Sampling%2520through%2520Velocity%2520Rescaling%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2007%26volume%3D126%26issue%3D1%26spage%3D014101%26doi%3D10.1063%2F1.2408420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraaije, J. G. E. M.</span></span> <span> </span><span class="NLM_article-title">LINCS: A Linear Constraint Solver for Molecular Simulations</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1463</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199709)18:12<1463::AID-JCC4>3.0.CO;2-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=1463-1472&issue=12&author=B.+Hessauthor=H.+Bekkerauthor=H.+J.+C.+Berendsenauthor=J.+G.+E.+M.+Fraaije&title=LINCS%3A+A+Linear+Constraint+Solver+for+Molecular+Simulations&doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">LINCS: a linear constraint solver for molecular simulations</span></div><div class="casAuthors">Hess, Berk; Bekker, Henk; Berendsen, Herman J. C.; Fraaije, Johannes G. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1463-1472</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">We present a new LINear Constraint Solver (LINCS) for mol. simulations with bond constraints using the enzyme lysozyme and a 32-residue peptide as test systems.  The algorithm is inherently stable, as the constraints themselves are reset instead of derivs. of the constraints, thereby eliminating drift.  Although the derivation of the algorithm is presented in terms of matrixes, no matrix matrix multiplications are needed and only the nonzero matrix elements have to be stored, making the method useful for very large mols.  At the same accuracy, the LINCS algorithm is 3-4 times faster than the SHAKE algorithm.  Parallelization of the algorithm is straightforward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk6h4pdV0Nx7Vg90H21EOLACvtfcHk0lgh7XTdwbtVUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D&md5=890f8af0d2ca1f65aa93db5a3a0bacf2</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.0.CO%253B2-H%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DBekker%26aufirst%3DH.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DFraaije%26aufirst%3DJ.%2BG.%2BE.%2BM.%26atitle%3DLINCS%253A%2520A%2520Linear%2520Constraint%2520Solver%2520for%2520Molecular%2520Simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1997%26volume%3D18%26issue%3D12%26spage%3D1463%26epage%3D1472%26doi%3D10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Essmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L. G.</span></span> <span> </span><span class="NLM_article-title">A Smooth Particle Mesh Ewald Method</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8577</span>– <span class="NLM_lpage">8593</span>, <span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&issue=19&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+Smooth+Particle+Mesh+Ewald+Method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0lgh7XTdwbtVUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520Smooth%2520Particle%2520Mesh%2520Ewald%2520Method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26issue%3D19%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hub, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Groot, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span> <span> </span><span class="NLM_article-title">G_wham—A Free Weighted Histogram Analysis Implementation Including Robust Error and Autocorrelation Estimates</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3713</span>– <span class="NLM_lpage">3720</span>, <span class="refDoi"> DOI: 10.1021/ct100494z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100494z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVegu7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=3713-3720&issue=12&author=J.+S.+Hubauthor=B.+L.+de+Grootauthor=D.+van+der%0ASpoel&title=G_wham%E2%80%94A+Free+Weighted+Histogram+Analysis+Implementation+Including+Robust+Error+and+Autocorrelation+Estimates&doi=10.1021%2Fct100494z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">g_wham: A Free Weighted Histogram Analysis Implementation Including Robust Error and Autocorrelation Estimates</span></div><div class="casAuthors">Hub, Jochen S.; de Groot, Bert L.; van der Spoel, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3713-3720</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Weighted Histogram Anal. Method (WHAM) is a std. technique used to compute potentials of mean force (PMFs) from a set of umbrella sampling simulations.  Here, the authors present a new WHAM implementation, termed g_wham, which is distributed freely with the GROMACS mol. simulation suite.  G_wham ests. statistical errors using the technique of bootstrap anal.  Three bootstrap methods are supported: (i) bootstrapping new trajectories based on the umbrella histograms, (ii) bootstrapping of complete histograms, and (iii) Bayesian bootstrapping of complete histograms, i.e., bootstrapping via the assignment of random wts. to the histograms.  Because methods ii and iii consider only complete histograms as independent data points, these methods do not require the accurate calcn. of autocorrelation times.  The authors demonstrate that, given sufficient sampling, bootstrapping new trajectories allows for an accurate error est.  In the presence of long autocorrelations, however, (Bayesian) bootstrapping of complete histograms yields a more reliable error est., whereas bootstrapping of new trajectories may underestimate the error.  In addn., the authors emphasize that the incorporation of autocorrelations into WHAM reduces the bias from limited sampling, in particular, when computing periodic PMFs in inhomogeneous systems such as solvated lipid membranes or protein channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3sB6LvdBAELVg90H21EOLACvtfcHk0lgh7XTdwbtVUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVegu7bI&md5=c798afe576b97471e29040069e434028</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fct100494z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100494z%26sid%3Dliteratum%253Aachs%26aulast%3DHub%26aufirst%3DJ.%2BS.%26aulast%3Dde%2BGroot%26aufirst%3DB.%2BL.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26atitle%3DG_wham%25E2%2580%2594A%2520Free%2520Weighted%2520Histogram%2520Analysis%2520Implementation%2520Including%2520Robust%2520Error%2520and%2520Autocorrelation%2520Estimates%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26issue%3D12%26spage%3D3713%26epage%3D3720%26doi%3D10.1021%2Fct100494z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savitzky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golay, M. J. E.</span></span> <span> </span><span class="NLM_article-title">Smoothing and Differentiation of Data by Simplified Least Squares Procedures</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1627</span>– <span class="NLM_lpage">1639</span>, <span class="refDoi"> DOI: 10.1021/ac60214a047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac60214a047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADyaF2cXksVCjur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1964&pages=1627-1639&issue=8&author=A.+Savitzkyauthor=M.+J.+E.+Golay&title=Smoothing+and+Differentiation+of+Data+by+Simplified+Least+Squares+Procedures&doi=10.1021%2Fac60214a047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Smoothing and differentiation of data by simplified least squares procedures</span></div><div class="casAuthors">Savitzky, Abraham; Golay, Marcel J. E.</div><div class="citationInfo"><span class="NLM_cas:title"></span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1627-39</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    </div><div class="casAbstract">The operation may be carried out on a computer by convolution of the data.  Two examples are presented as subroutines in the FORTRAN language.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGF5gFu4oYrbVg90H21EOLACvtfcHk0lgh7XTdwbtVUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXksVCjur8%253D&md5=8406f1500e608f8f89eada4d7949ed77</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fac60214a047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac60214a047%26sid%3Dliteratum%253Aachs%26aulast%3DSavitzky%26aufirst%3DA.%26aulast%3DGolay%26aufirst%3DM.%2BJ.%2BE.%26atitle%3DSmoothing%2520and%2520Differentiation%2520of%2520Data%2520by%2520Simplified%2520Least%2520Squares%2520Procedures%26jtitle%3DAnal.%2520Chem.%26date%3D1964%26volume%3D36%26issue%3D8%26spage%3D1627%26epage%3D1639%26doi%3D10.1021%2Fac60214a047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, D. B.</span></span> <span> </span><span class="NLM_article-title">The Bayesian Bootstrap</span>. <i>Ann. Stat.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1214/aos/1176345338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1214%2Faos%2F1176345338" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1981&pages=130-134&issue=1&author=D.+B.+Rubin&title=The+Bayesian+Bootstrap&doi=10.1214%2Faos%2F1176345338"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1214%2Faos%2F1176345338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1214%252Faos%252F1176345338%26sid%3Dliteratum%253Aachs%26aulast%3DRubin%26aufirst%3DD.%2BB.%26atitle%3DThe%2520Bayesian%2520Bootstrap%26jtitle%3DAnn.%2520Stat.%26date%3D1981%26volume%3D9%26issue%3D1%26spage%3D130%26epage%3D134%26doi%3D10.1214%2Faos%2F1176345338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggenweiler, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirrenberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grädler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoppe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">März, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, M.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of Hydroxy-Indazole-Carboxamides as Novel Small Molecule Inhibitors of Hsp90</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">4396</span>– <span class="NLM_lpage">4403</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.04.121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1016%2Fj.bmcl.2012.04.121" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4396-4403&issue=13&author=H.-P.+Buchstallerauthor=H.-M.+Eggenweilerauthor=C.+Sirrenbergauthor=U.+Gr%C3%A4dlerauthor=D.+Musilauthor=E.+Hoppeauthor=A.+Zimmermannauthor=H.+Schwartzauthor=J.+M%C3%A4rzauthor=J.+Bomkeauthor=A.+Wegenerauthor=M.+Wolf&title=Fragment-Based+Discovery+of+Hydroxy-Indazole-Carboxamides+as+Novel+Small+Molecule+Inhibitors+of+Hsp90&doi=10.1016%2Fj.bmcl.2012.04.121"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.04.121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.04.121%26sid%3Dliteratum%253Aachs%26aulast%3DBuchstaller%26aufirst%3DH.-P.%26aulast%3DEggenweiler%26aufirst%3DH.-M.%26aulast%3DSirrenberg%26aufirst%3DC.%26aulast%3DGr%25C3%25A4dler%26aufirst%3DU.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DHoppe%26aufirst%3DE.%26aulast%3DZimmermann%26aufirst%3DA.%26aulast%3DSchwartz%26aufirst%3DH.%26aulast%3DM%25C3%25A4rz%26aufirst%3DJ.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DM.%26atitle%3DFragment-Based%2520Discovery%2520of%2520Hydroxy-Indazole-Carboxamides%2520as%2520Novel%2520Small%2520Molecule%2520Inhibitors%2520of%2520Hsp90%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26issue%3D13%26spage%3D4396%26epage%3D4403%26doi%3D10.1016%2Fj.bmcl.2012.04.121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&issue=2&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgl6XOChvYeDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuszewski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span> <span> </span><span class="NLM_article-title">Crystallography & NMR System: A New Software Suite for Macromolecular Structure Determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">905</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1107/S0907444998003254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1107%2FS0907444998003254" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=905-921&issue=5&author=A.+T.+Br%C3%BCngerauthor=P.+D.+Adamsauthor=G.+M.+Cloreauthor=W.+L.+DeLanoauthor=P.+Grosauthor=R.+W.+Grosse-Kunstleveauthor=J.+S.+Jiangauthor=J.+Kuszewskiauthor=M.+Nilgesauthor=N.+S.+Pannuauthor=R.+J.+Readauthor=L.+M.+Riceauthor=T.+Simonsonauthor=G.+L.+Warren&title=Crystallography+%26+NMR+System%3A+A+New+Software+Suite+for+Macromolecular+Structure+Determination&doi=10.1107%2FS0907444998003254"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444998003254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998003254%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DJiang%26aufirst%3DJ.%2BS.%26aulast%3DKuszewski%26aufirst%3DJ.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRice%26aufirst%3DL.%2BM.%26aulast%3DSimonson%26aufirst%3DT.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DCrystallography%2520%2526%2520NMR%2520System%253A%2520A%2520New%2520Software%2520Suite%2520for%2520Macromolecular%2520Structure%2520Determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1998%26volume%3D54%26issue%3D5%26spage%3D905%26epage%3D921%26doi%3D10.1107%2FS0907444998003254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berthold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cebron, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dill, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kötter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieb, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswedel, B.</span></span> <span> </span><span class="NLM_article-title">KNIME: The Konstanz Information Miner</span>. <i>SpringerLink</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1007/978-3-540-78246-9_38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;key=10.1007%2F978-3-540-78246-9_38" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=319-326&author=M.+R.+Bertholdauthor=N.+Cebronauthor=F.+Dillauthor=T.+R.+Gabrielauthor=T.+K%C3%B6tterauthor=T.+Meinlauthor=P.+Ohlauthor=C.+Siebauthor=K.+Thielauthor=B.+Wiswedel&title=KNIME%3A+The+Konstanz+Information+Miner&doi=10.1007%2F978-3-540-78246-9_38"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2F978-3-540-78246-9_38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-540-78246-9_38%26sid%3Dliteratum%253Aachs%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26aulast%3DCebron%26aufirst%3DN.%26aulast%3DDill%26aufirst%3DF.%26aulast%3DGabriel%26aufirst%3DT.%2BR.%26aulast%3DK%25C3%25B6tter%26aufirst%3DT.%26aulast%3DMeinl%26aufirst%3DT.%26aulast%3DOhl%26aufirst%3DP.%26aulast%3DSieb%26aufirst%3DC.%26aulast%3DThiel%26aufirst%3DK.%26aulast%3DWiswedel%26aufirst%3DB.%26atitle%3DKNIME%253A%2520The%2520Konstanz%2520Information%2520Miner%26jtitle%3DSpringerLink%26date%3D2008%26spage%3D319%26epage%3D326%26doi%3D10.1007%2F978-3-540-78246-9_38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OCI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OCI','PDB','5OCI'); return false;">PDB: 5OCI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J86" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J86','PDB','5J86'); return false;">PDB: 5J86</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNY','PDB','5LNY'); return false;">PDB: 5LNY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ODX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ODX','PDB','5ODX'); return false;">PDB: 5ODX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7','PDB','5OD7'); return false;">PDB: 5OD7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NYH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NYH','PDB','5NYH'); return false;">PDB: 5NYH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNZ','PDB','5LNZ'); return false;">PDB: 5LNZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LO0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LO0','PDB','5LO0'); return false;">PDB: 5LO0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LO1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LO1','PDB','5LO1'); return false;">PDB: 5LO1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i25"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00080">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_08853"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00080">10.1021/acs.jmedchem.8b00080</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">SKR analysis of complete Hsp90 data set; structural analysis of cocrystallized inhibitors (<b>2b</b>, <b>3b</b>, <b>3c</b>, <b>4i</b>, and three quinazolines); experimentally derived enthalpic and entropic contributions to the association barrier of drug-like molecules from literature; analytical data on <b>1a</b>–<b>6c</b>; supplementary methods; data collection and refinement statistic of X-ray structures (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Parameter settings for MD simulation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_001.pdf">jm8b00080_si_001.pdf (12.22 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_002.xlsx">jm8b00080_si_002.xlsx (26.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00080/suppl_file/jm8b00080_si_003.csv">jm8b00080_si_003.csv (6.88 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates have been deposited in the ProteinData Bank (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OCI">5OCI</a> for <b>1g</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ODX">5ODX</a> for <b>3c</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNY">5LNY</a> for <b>3b</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NYH">5NYH</a> for <b>2b</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LNZ">5LNZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OD7">5OD7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LO0">5LO0</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LO1">5LO1</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00080&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-10%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00080%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00080" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b111d9eb63dac","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
