
<html lang="en"     class="pb-page"  data-request-id="209f5ca9-263d-4c29-afda-cfb3a9f544fb"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-5;article:article:10.1021/acs.jmedchem.9b00868"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Developing and Characterization of Chemically Modified RNA Aptamers for Targeting Wild Type and Mutated c-KIT Receptor Tyrosine Kinases" /></meta><meta name="dc.Creator" content="Ala’a S.  Shraim" /></meta><meta name="dc.Creator" content="Abdelrahim  Hunaiti" /></meta><meta name="dc.Creator" content="Abdalla  Awidi" /></meta><meta name="dc.Creator" content="Walhan  Alshaer" /></meta><meta name="dc.Creator" content="Nidaa A.  Ababneh" /></meta><meta name="dc.Creator" content="Bashaer  Abu-Irmaileh" /></meta><meta name="dc.Creator" content="Fadwa  Odeh" /></meta><meta name="dc.Creator" content="Said  Ismail" /></meta><meta name="dc.Description" content="The c-KIT receptor represents an attractive target for cancer therapy. Aptamers are emerging as a new promising class of nucleic acid therapeutics. In this study, a conventional SELEX approach was ..." /></meta><meta name="Description" content="The c-KIT receptor represents an attractive target for cancer therapy. Aptamers are emerging as a new promising class of nucleic acid therapeutics. In this study, a conventional SELEX approach was ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 1, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00868" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00868" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00868" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00868" /></link>
        
    
    

<title>Developing and Characterization of Chemically Modified RNA Aptamers for Targeting Wild Type and Mutated c-KIT Receptor Tyrosine Kinases | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00868" /></meta><meta property="og:title" content="Developing and Characterization of Chemically Modified RNA Aptamers for Targeting Wild Type and Mutated c-KIT Receptor Tyrosine Kinases" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0012.jpeg" /></meta><meta property="og:description" content="The c-KIT receptor represents an attractive target for cancer therapy. Aptamers are emerging as a new promising class of nucleic acid therapeutics. In this study, a conventional SELEX approach was applied against the kinase domain of a group of c-KIT proteins (c-KITWT, c-KITD816V, and c-KITD816H) to select aptamers from a random RNA pool that can bind to the kinase domain of each target with high affinity and can selectively interfere with their kinase activities. Interestingly, our data indicated that one candidate aptamer, called V15, can specifically inhibit the in vitro kinase activity of mutant c-KITD816V with an IC50 value that is 9-fold more potent than the sunitinib drug tested under the same conditions. Another aptamer, named as H5/V36, showed the potential to distinguish between the c-KIT kinases by modulating the phosphorylation activity of each in a distinct mechanism of action and in a different potency." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00868"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00868">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00868&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00868&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00868&amp;href=/doi/10.1021/acs.jmedchem.9b00868" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2209-2228</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00860" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00881" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Developing and Characterization of Chemically Modified RNA Aptamers for Targeting Wild Type and Mutated c-KIT Receptor Tyrosine Kinases</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ala’a S. Shraim</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ala’a S. Shraim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences, School of Science, The University of Jordan, Amman JO 11942, Jordan</div><div class="loa-info-affiliations-info">Department of Medical Laboratory Sciences, Faculty of Allied Medical Sciences, Al-Ahliyya Amman University, Amman JO 19328, Jordan</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#3b5a644853495a52567b425a53545415585456"><span class="__cf_email__" data-cfemail="54350b273c26353d39142d353c3b3b7a373b39">[email protected]</span></a> or <a href="/cdn-cgi/l/email-protection#b0c3d8c2d1d9ddd1dcd1d1f0d7ddd1d9dc9ed3dfdd"><span class="__cf_email__" data-cfemail="5e2d362c3f37333f323f3f1e39333f3732703d3133">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ala%E2%80%99a+S.++Shraim">Ala’a S. Shraim</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7021-8847" title="Orcid link">http://orcid.org/0000-0001-7021-8847</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Abdelrahim Hunaiti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Abdelrahim Hunaiti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Clinical Laboratory Sciences, School of Science, The University of Jordan, Amman JO 11942, Jordan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Abdelrahim++Hunaiti">Abdelrahim Hunaiti</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Abdalla Awidi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Abdalla Awidi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cell Therapy Center, The University of Jordan, Amman JO 11942, Jordan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Abdalla++Awidi">Abdalla Awidi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Walhan Alshaer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Walhan Alshaer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cell Therapy Center, The University of Jordan, Amman JO 11942, Jordan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Walhan++Alshaer">Walhan Alshaer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nidaa A. Ababneh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nidaa A. Ababneh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cell Therapy Center, The University of Jordan, Amman JO 11942, Jordan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nidaa+A.++Ababneh">Nidaa A. Ababneh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bashaer Abu-Irmaileh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bashaer Abu-Irmaileh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hamdi Mango Center for Scientific Research, The University of Jordan, Amman JO 11942, Jordan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bashaer++Abu-Irmaileh">Bashaer Abu-Irmaileh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fadwa Odeh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fadwa Odeh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, School of Science, The University of Jordan, Amman JO 11942, Jordan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fadwa++Odeh">Fadwa Odeh</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Said Ismail</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Said Ismail</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Physiology, School of Medicine, The University of Jordan, Amman JO 11942, Jordan</div><div class="loa-info-affiliations-info">Qatar Genome Project, Qatar Foundation, Doha, Qatar</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Said++Ismail">Said Ismail</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00868&amp;href=/doi/10.1021%2Facs.jmedchem.9b00868" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2209–2228</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 1, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 May 2019</li><li><span class="item_label"><b>Published</b> online</span>1 August 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00868" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00868</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2209%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAla%25E2%2580%2599a%2BS.%2BShraim%252C%2BAbdelrahim%2BHunaiti%252C%2BAbdalla%2BAwidi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D5%26contentID%3Dacs.jmedchem.9b00868%26title%3DDeveloping%2Band%2BCharacterization%2Bof%2BChemically%2BModified%2BRNA%2BAptamers%2Bfor%2BTargeting%2BWild%2BType%2Band%2BMutated%2Bc-KIT%2BReceptor%2BTyrosine%2BKinases%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2228%26publicationDate%3DMarch%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00868"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">868</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00868" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Developing and Characterization of Chemically Modified RNA Aptamers for Targeting Wild Type and Mutated c-KIT Receptor Tyrosine Kinases&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ala’a&quot;,&quot;last_name&quot;:&quot;S. Shraim&quot;},{&quot;first_name&quot;:&quot;Abdelrahim&quot;,&quot;last_name&quot;:&quot;Hunaiti&quot;},{&quot;first_name&quot;:&quot;Abdalla&quot;,&quot;last_name&quot;:&quot;Awidi&quot;},{&quot;first_name&quot;:&quot;Walhan&quot;,&quot;last_name&quot;:&quot;Alshaer&quot;},{&quot;first_name&quot;:&quot;Nidaa&quot;,&quot;last_name&quot;:&quot;A. Ababneh&quot;},{&quot;first_name&quot;:&quot;Bashaer&quot;,&quot;last_name&quot;:&quot;Abu-Irmaileh&quot;},{&quot;first_name&quot;:&quot;Fadwa&quot;,&quot;last_name&quot;:&quot;Odeh&quot;},{&quot;first_name&quot;:&quot;Said&quot;,&quot;last_name&quot;:&quot;Ismail&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;2209-2228&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00868&quot;},&quot;abstract&quot;:&quot;The c-KIT receptor represents an attractive target for cancer therapy. Aptamers are emerging as a new promising class of nucleic acid therapeutics. In this study, a conventional SELEX approach was applied against the kinase domain of a group of c-KIT proteins (c-KITWT, c-KITD816V, and c-KITD816H) to select aptamers from a random RNA pool that can bind to the kinase domain of each target with high affinity and can selectively interfere with their kinase activities. Interestingly, our data indicated that one candidate aptamer, called V15, can specifically inhibit the in vitro kinase activity of mutant c-KITD816V with an IC50 value that is 9-fold more potent than the sunitinib drug tested under the same conditions. Another aptamer, named as H5/V36, showed the potential to distinguish between the c-KIT kinases by modulating the phosphorylation activity of each in a distinct mechanism of action and in a different potency.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00868&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00868" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00868&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00868" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00868&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00868" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00868&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00868&amp;href=/doi/10.1021/acs.jmedchem.9b00868" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00868" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00868" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00868%26sid%3Dliteratum%253Aachs%26pmid%3D31369705%26genre%3Darticle%26aulast%3DShraim%26date%3D2020%26atitle%3DDeveloping%2Band%2BCharacterization%2Bof%2BChemically%2BModified%2BRNA%2BAptamers%2Bfor%2BTargeting%2BWild%2BType%2Band%2BMutated%2Bc-KIT%2BReceptor%2BTyrosine%2BKinases%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D5%26spage%3D2209%26epage%3D2228%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291993" title="Nucleic acid structure">Nucleic acid structure</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/jmcmar.2020.63.issue-5/20200312/jmcmar.2020.63.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The c-KIT receptor represents an attractive target for cancer therapy. Aptamers are emerging as a new promising class of nucleic acid therapeutics. In this study, a conventional SELEX approach was applied against the kinase domain of a group of c-KIT proteins (c-KIT<sup>WT</sup>, c-KIT<sup>D816V</sup>, and c-KIT<sup>D816H</sup>) to select aptamers from a random RNA pool that can bind to the kinase domain of each target with high affinity and can selectively interfere with their kinase activities. Interestingly, our data indicated that one candidate aptamer, called V15, can specifically inhibit the <i>in vitro</i> kinase activity of mutant c-KIT<sup>D816V</sup> with an IC<sub>50</sub> value that is 9-fold more potent than the sunitinib drug tested under the same conditions. Another aptamer, named as H5/V36, showed the potential to distinguish between the c-KIT kinases by modulating the phosphorylation activity of each in a distinct mechanism of action and in a different potency.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/63/5"><issue-title>Women in Medicinal Chemistry</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82478" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82478" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The c-KIT receptor tyrosine kinase is a member of class III receptor tyrosine kinase (RTK) that catalyzes the transfer of γ-phosphate from ATP to tyrosine residues in polypeptides. Structurally, these receptors are divided into an extracellular ligand binding domain (ECD), a single transmembrane segment, and an intracellular cytoplasmic tail. The intracellular domain composed of a juxta-membrane domain (JMD), a highly conserved intracellulartyrosine kinase domain (KD), and a C-terminal tail.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The kinase domain has the characteristic bilobed architecture observed in all protein kinases and comprises two subdomains that are interrupted by a kinase insert sequence (KID) of 77 amino acids long.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">The catalytic site of the c-KIT kinase lies in the cleft between the small and large lobes.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The large C-lobe includes an activation loop (A-loop; 810–839), which provides a platform for the peptide substrate and whose orientation determines the active or inactive state of the kinase domain.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The catalytic domain of kinases shows a high degree of sequence homology, especially for kinases that belong to the same family. They share a common ATP binding site with a conserved activation loop and similar three-dimensional structures.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The c-KIT kinase domain exists in active (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG">1PKG</a>) and inactive conformations (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1T45">1T45</a>) as determined by the X-ray crystallography.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> c-KIT receptors are expressed in different types of normal cells such as hematopoietic stem cells from the bone marrow, melanocytes, the differentiated mast cells, the interstitial cells of Cajal (ICC) in the gastrointestinal tract, and the germ cells in the fertility system.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> Under physiological conditions, the activity of c-KIT protein kinases and the downstream signaling events are stringently regulated. Deregulation of c-KIT activity can result in cancer. More than 500 different mutations of c-KIT have been described in human tumors as reported by Sanger Institute Catalogue of Somatic Mutations in Cancer; <a href="http://www.sanger.ac.uk/genetics/CGP/cosmic/" class="extLink">http://www.sanger.ac.uk/genetics/CGP/cosmic/</a>.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In some tumors, such as melanoma, thyroid carcinoma, and breast cancer, loss of function mutations of the c-KIT was observed.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Conversely, in certain cancer types, such as gastrointestinal stromal tumors (GIST) and human mast cell leukemia, the main cause of tumorigenesis is the activating mutations in the c-KIT. Indeed, 75–80% of GIST cases and greater than 90% of all cases of systemic mastocytosis bear gain of function mutations in the c-KIT protein.<a onclick="showRef(event, 'ref3 ref12'); return false;" href="javascript:void(0);" class="ref ref3 ref12">(3,12)</a> The most common mutation was the missense substitution mutation in D816 residue located in the A-loop domain, which is encoded by exon 17.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In fact, in the inactive state, D816 residue occurs within an important regulatory α-helix segment and its mutation to a hydrophobic or aromatic residue would eliminate a negative charge from the A-loop region and potentially destabilize this helical segment.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In consequence, A-loop mutants can become activated more than 100 times faster than wild-type because the active conformation is so energetically favored.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Thus, substitution of D816 with V816, H816, Y816, or F816 would promote the conformational conversion of c-KIT kinase from inactive DFG-out to active DFG-in conformation, conferring a constitutive activation to the c-KIT kinase.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Noteworthy, the amino acid sequence similarity of the A-loop in class III RTKs is greater than 70%. Moreover, the protein structure of this region is almost identical between c-KIT, PDGFRA, and FLT3 proteins.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In addition, the A-loop mutants of the c-KIT kinase have been largely resistant to inhibition by the majority of conventional small-molecule ATP-competitive inhibitors that target the receptor’s inactive conformation.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> In a complementary fashion, accumulated shreds of evidence indicated that the frequency of these mutations is increasing after the second line therapy (sunitinib), which also has inadequate activity on the A-loop mutants.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p">Recently, an interest has been growing in an effort to discover an approach to only inhibit the mutant therapeutic target, thereby minimizing the risk of potential side effects associated with the inhibition of normal function.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Thus, the identification of antagonist that more specifically inhibits the activity of oncogenic mutant c-KIT kinase, but not the wild type protein in normal tissues, would be very useful. The future of novel and selective kinase inhibitors rests on the discovery of compounds that bind outside of the ATP site while retaining a high level of potency and efficacy.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Indeed, the last three decades have witnessed the progression in the development and application of the nucleic acid aptamers in a wide variety of fields, including target analysis, disease therapy, and molecular and cellular engineering.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><div class="NLM_p last">Aptamers are defined as a short, typically with 25–100 nucleotides, single-stranded oligonucleotides (RNA or DNA) that can fold into specific three-dimensional structures in order to recognize target molecules.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Recent discoveries reveal that these specific sequences of nucleic acids possess unique binding characteristics to their targets and can be easily synthesized and selected with high specificity and affinity, which make them emerge as a promising innovative tool for therapeutic strategies specifically for cancer therapy. The current study aimed to select chemically modified RNA aptamers, using the systematic evolution of ligands by exponential enrichment (SELEX) technique, targeting the intracellular kinase domain of the wild type and two mutated D816H and D816V c-KIT kinases, in an attempt to identify novel therapeutic aptamers that can bind with high affinity and can functionally discriminate and interfere with the biological activity of the wild type and the mutant drug-resistant variants of c-KIT protein.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76667" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76667" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> <i>In Vitro</i> Selection of Chemically Modified Anti c-KIT Aptamers</h3><div class="NLM_p last">In this study, a branched SELEX-based approach was employed for the enrichment of 2′-fluoropyrimidine modified RNA ligands able to bind to the active kinase domain of the wild type and mutated c-KIT proteins with high affinity and specificity. After 13 iterative rounds of selection, the enriched RNA aptamers from each final pool were cloned, and a total of 194 cloned plasmids were sent for sequencing to Macrogen Incorporation, South Korea. Accordingly, 34 successful sequences of anti c-KIT<sup>WT</sup> aptamers were identified and grouped into six families based on sequence similarities in the variable regions. These sequences are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">Table S1</a> according to their abundance. Eighteen different sequences of anti-c-KIT<sup>D816H</sup> aptamers were identified and listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">Table S2</a>. For anti c-KIT<sup>D816V</sup> aptamers, 59 successful sequences were identified and classified into 14 families with different sequences according to their abundance as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">Table S3</a>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Sequences Analysis and Secondary Structures Prediction</h3><div class="NLM_p">Sequences analysis revealed that W3 anti c-KIT<sup>WT</sup> aptamer is the most abundant sequence that constitutes around half of the input pool that was selected against the wild type form of c-KIT kinase domain. In the same approach, H31 and V5 are the most prevalent sequences against c-KIT<sup>D816H</sup> and c-KIT<sup>D816V</sup> proteins, respectively. There is a general perception that the most abundant sequence of the final round of a selection experiment is the ones with the highest target affinity.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> However, kinetic analysis of representative aptamers revealed that the aptamers with the highest affinity are not necessarily the ones that dominate the clone library. Indeed, an informative study provides convincing pieces of evidence that aptamers with fast association rates dominate the final pool.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In fact, the nature of certain protein areas may be particularly well suited for aptamer binding, and such hotspot region represents a dominant recognition site for aptamers.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In supporting fashion, our data indicate that W3, H31, and V5 dominant sequences are unique to their targets, suggesting that the substituted point mutation in D816 residue induces different conformational changes that could result in the disappearance of such dominant recognition epitope for which a given aptamer can recognize and bind. Noteworthy, the researchers found that aspartic acids interacted more frequently with the RNA loops in which the nucleotide bases had flipped out to form hydrogen bonds with the protein.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Moreover, cross-competition between aptamers bound to distinct sites through partial overlap are also possible given that the size of our aptamers is not small compared to small molecules. Overall, the principles that guide the <i>in vitro</i> selection of aptamers are similar to those that govern the natural selection of biopolymers. As described by Ellington and Conrad, it is truly the survival of the “fittest” sequences from the random sequence pools that are extracted in the final enriched pool.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Finally, in order to gain insight into the conservation of nucleotide sequences, multiple sequence alignments were performed to determine the homologous sequences and to identify the conserved sequences among all listed aptamers (data not shown). Accordingly, a conserved motif consisting of <u class="uu"><b>UUU</b></u> nucleotides was identified in almost all sequences isolated against the c-KIT kinase proteins, which might indicate a structurally important motif that can recognize a particular prominent structural epitope in the c-KIT kinase domain. Moreover, the multiple sequence alignment of variable regions indicated the presence of conserved G<sub>20</sub>G<sub>21</sub>, U<sub>29</sub>, and U<sub>45</sub> residues at the same position between all listed anti c-KIT<sup>D816H</sup> aptamers. Additionally, three sequences (termed W9/V22, W16/V28, and W21/V37) were found to occur in different frequencies in both the wild type and mutated c-KIT<sup>D816V</sup> clones. W38 sequence was also identical to H31 anti c-KIT<sup>D816H</sup> clone sequence. These shared sequences are most probably bound to the same epitopes existing in the kinase domain of the wild type and mutated c-KIT proteins and hence may recognize regions outside the mutation site. Moreover, shared H5/V36 homologous sequences were found, which could bind to dominant regions structurally shared by both c-KIT<sup>D816H</sup> and c-KIT<sup>D816V</sup> mutated proteins but distinguished from that of the WT protein. Interestingly, H5/V36 sequence contains two copies of <u class="uu"><b>UUU</b></u> conserved motif, which might further highlight the importance of this motif in recognizing and binding to the c-KIT kinase domain. Noticeably, the multiple sequence alignment indicated that the location of this structural motif in the primary structure of a given aptamer and the type of flanked sequences are not fixed within the anti c-KIT aptamer sequences. A unique UUUU motif was identified in three anti c-KIT<sup>D816V</sup> clones termed as V15. For further analysis, the two predominant aptamers <b>W3</b> and <b>V5</b>, the shared anti c-KIT <sup>D816H/V</sup> aptamer (<b>H5/V36</b>), and the unique <b>V15</b> aptamer were chosen to determine their binding affinities and to assess their potential inhibitory effects. These potential candidates are defined in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Candidate Anti c-KIT Aptamers<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">clone ID</th><th class="colsep0 rowsep0" align="center">sequence of aptamers from 5′ to 3′</th><th class="colsep0 rowsep0" align="center">frequency of sequence (%)</th><th class="colsep0 rowsep0" align="center">length nt</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>W3</b></td><td class="colsep0 rowsep0" align="left">GGGAUGGAUCCAAGCUUACUGGAGAAUGAGUG<u class="uu">UUU</u>ACGCAACUGGUGUGUCGUAUGGACGCAAUUCUCGGGAAGCUUCGAUAGGAAUUCGG</td><td class="colsep0 rowsep0" align="left">50.0</td><td class="colsep0 rowsep0" align="left">91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>H5/V36</b></td><td class="colsep0 rowsep0" align="left">GGGAUGGAUCCAAGCUUACUGGCUUGCG<u class="uu">UUU</u>ACAACGAUCCGGAUCAUUGUCGA<u class="uu">UUU</u>CUACUUGUGUGGGAAGCUUCGAUAGGAAUUCGG</td><td class="colsep0 rowsep0" align="left">19.4</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>V5</b></td><td class="colsep0 rowsep0" align="left">GGGAUGGAUCCAAGCUUACUGGCACGGGCUUGGAUGGUGGGAAAGUGUCCAGACGAAGUUCUCC<u class="uu">UUU</u>GGGAAGCUUCGAUAGGAAUUCGG</td><td class="colsep0 rowsep0" align="left">47.4</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>V15</b></td><td class="colsep0 rowsep0" align="left">GGGAUGGAUCCAAGCUUACUGGAUGACUAUAAAGCGUCUGUGAAG<u class="uu">UUUU</u>GGGAGGGUCAAGUGAGGUGGGAAGCUUCGAUAGGAAUUCGG</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left">90</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The nucleotide sequence of candidate aptamers and their frequency in the final selection pool. The underlined nucleotides represent the conserved <u class="uu"><b>UUU</b></u> motif.</p></div></div></div><div class="NLM_p">To further characterize these candidate aptamers, the secondary structures were predicted using Mfold web server program. In fact, a distinguishing feature of aptamers is their ability to form a pronounced secondary structure.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Based on the secondary structures analysis, different conformations among candidate aptamers have been observed as illustrated in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, which suggests that these candidate sequences may have different binding properties. However, it is important to keep in mind that the best currently available prediction methods can only give a rough model of the secondary structure.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In addition, the RNA folding does not always occur on the path toward the lowest free energy structure.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> As demonstrated in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, the <b>H5/V36</b> anti c-KIT <sup>D816H/V</sup> aptamer has distinct predicted secondary structure in its free state, composed of trifoliate stem-loop structures that are connected by a multiway single-stranded junction. Two stem-loop structures are located at the 5′-fixed sequences while the third stem–triloop structure is located at the variable sequences and contained the detected conserved <u class="uu"><b>UUU</b></u> motif. On the other side, a long duplex extended from the junction and ended with a bulge–stem–tetraloop structure located at the variable sequences and contained the consensus nucleotides. This duplex is formed by sequences common to all library members. The predicted secondary structure of <b>W3</b> anti c-KIT<sup>WT</sup> aptamer contains one terminal stem-loop structure located at the 5′-, 3′-fixed sequences and two hairpins biforked from a common junction located at the variable sequences. Noteworthy, the highly conserved <u class="uu"><b>UUU</b></u> nucleotides arranged in a tetraloop structure that extended from a characteristic stem–bulge region. Similar secondary structures were predicted for <b>V5</b> and <b>V15</b> anti c-KIT<sup>D816V</sup> aptamers. These structures composed of 5′- interior loop–stem–apical loop structure and 3′-two successive interior loop–stem–apical loop structures connected by single-stranded junction contained a unique solvent-exposed <b>UGGGA</b> motif that was located at the same position in both <b>V5</b> and <b>V15</b> predicted secondary structures. The observed mismatch pairs in the secondary structure of <b>V15</b> aptamer (A<sub>18</sub>·G<sub>42</sub> and C<sub>19</sub>·U<sub>41</sub>) have been widely found in RNA architectures. As reported, these mismatch pairs can participate in stacking interactions and provide recognition sites.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Finally, even though the conserved structural <u class="uu"><b>UUU</b></u> motif occurs at different locations within the primary structures of these candidate aptamers, it is obviously arranged at exposed folding structures such as a terminal loop or interior loop in all predicted secondary structures. This finding strongly suggests that <u class="uu"><b>UUU</b></u> conserved nucleotides are the most likely candidates to interact with the protein target. A prevailing view is that the solvent exposure of loop nucleotides could play a role in sequence-specific molecular recognition, particularly via “induced-fit” mechanisms associated with RNA–protein complexes.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Predicted secondary structures of candidate anti c-KIT aptamers. Numbers are nucleotide positions relative to the 5′-end of the RNA sequence. Blue highlights indicate nucleotides of the flanking fixed regions; red highlights indicate <b><u class="uu">UUU</u></b> conserved motif among most of the isolated aptamers.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">More importantly, Günter Mayer and his colleagues have identified an RNA aptamer, named C13, that targets the kinase domain of human GRK2 receptor.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The binding of this aptamer competitively inhibits the binding of ATP molecule and potently inactivates the kinase activity of GRK2 enzyme. Crystallographic analysis of GRK2 complexes with C13.18 variant revealed that the key to the mechanism of inhibition is the positioning of the underlined <b>A</b><sub><b>51</b></sub> base in the identified active motif 5′-C<sub>47</sub>CA<b>U</b><u class="uu">A</u><b>CG</b>G<sub>55</sub>-3′ that projects deep into the active site and occupies the adenine-binding pocket of the kinase domain.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Intriguingly, this tetraloop structure (<sub><b>50</b></sub><b>UACG</b><sub><b>53</b></sub>) is detected in the proposed secondary structure of our candidate <b>W3</b> anti c-KIT<sup>WT</sup> aptamer (<sub><b>35</b></sub><b>UACG</b><sub><b>38</b></sub>) as part of the conserved <u class="uu"><b>UUU</b></u> containing stem–tetraloop structure demonstrated in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. Equally important, the predicted secondary structure of <b>H5/V36</b> anti c-KIT <sup>D816H/V</sup> aptamer clarifies the presence of two homologous motifs that exist in the same consensus <u class="uu"><b>UUU</b></u> containing stem-loop structures that are 5′-<sub>28</sub>G<u class="uu"><b>UU</b></u><u class="uu">U</u><b>ACA</b>AC<sub>36-</sub>3′ and 5′-<sub>54</sub>A<u class="uu"><b>UUU</b></u>C<b>UACU</b>UG<sub>64-</sub>3′. Most probably, the first motif is the active one because, as illustrated in the proposed secondary structure, the <b>UAC</b> nucleotides in this motif are unpaired in the predicted triloop domain. In line with this finding, the structural properties of <b>W3</b> and <b>H5/V36</b> aptamers could be consistent with their binding to the same general region of the protein. Indeed, the existence of the ATP-mimicking motif of C13 in our aptamers may pave the way toward the development of other specific aptamers. In the same context, Coulson et al. identified by serendipity a 15-mer phosphorothioate (PS) aptamer that significantly reduces the autophosphorylation and hence inhibits the tyrosine kinase activity of the EGFRs highly expressed on A431 cells.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> A site directed mutational study revealed that the 5′-terminal <u class="uu"><b>GGAGGG</b></u> hexamer sequence in this aptamer was essential for the anti-tyrosine kinase activity in A431 cells.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Sequence analysis of our selected aptamers curiously revealed that this hexamer motif is present in the variable region of our <b>V15</b> anti c-KIT<sup>D816V</sup> candidate aptamer 5′-<sub><b>51</b></sub><b>GGAGGG</b><sub><b>56</b></sub>-3′. Noticeably, this hexamer motif is repeated twice in the variable region of our examined <b>H30</b> anti c-KIT<sup>D816H</sup> selected aptamer (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">Table S2</a>). On the other hand, the structural analysis of <b>V5</b> aptamer revealed that it has two repeated <b>GGC</b> motifs that were previously identified in the ODN1 aptamer described by Bergan et al. As indicated in this study, ODN1 aptamer shows high specificity as it potently inhibits the kinase activity of p210<sup>bcr-abl</sup>PTK and, to a lesser extent, the PDGF receptor.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In the same approach, only one quadruplex structure is predicted in our <b>V5</b> anti c-KIT<sup>D816V</sup> aptamer. It has been shown that G-rich oligodeoxynucleotides, especially those forming G-quadruplex structures, are known to bind and/or interfere with the functions of DNA polymerases and transcription factors.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> However, the binding specificity between quadruplexes and binding proteins will be affected by the loops and the surrounding sequences.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In summary, hence aptamers that inhibit the kinase activity might employ versatile motifs for kinase recognition and specificity; elucidating these motifs will not only clarify the aptamers mode of action but also may reveal new unresolved inhibitory entities that serve as scaffolds for the identification of novel kinase inhibitors.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> <i>In Vitro</i> Characterization of Candidate Aptamers</h3><div id="sec3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Qualitative Assessment of Aptamer–Kinase Interaction Using EMSA</h4><div class="NLM_p">Polyacrylamide gel electrophoresis (PAGE) under native conditions is a well-established and versatile method for probing nucleic acid conformation and nucleic acid–protein interactions.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Commonly in EMSA, the binding is detected through the appearance of one or more protein–nucleic acid complexes and a corresponding reduction of the intensity of the free nucleic acid band.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Accordingly, and as shown in the EMSA results, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, it is clear at first glance that the addition of our candidate aptamers to a given c-KIT kinase target, under the <i>in vitro</i> binding conditions, results in an observed reduction in the intensity of unbound aptamers in a concentration-dependent manner. Luckily, a characteristic discrete stable band with decreased gel mobility was also detected with the mutant forms of c-KIT protein as demonstrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, which represents the corresponding bound aptamer when compared with the free aptamer. However, the retardation pattern in the c-KIT<sup>WT</sup> is not exactly as predicted, and the exact location of the resulting complex was not detected. Nevertheless, the EMSA results can still point out clearly whether a given candidate aptamer is interacting with the c-KIT<sup>WT</sup> or not. Additionally, the observed faint RNA smear in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> (wells 1 and 2) that appears between the wells and unbound RNA aptamers could indeed represent a newly released RNA from the RNA-c-KIT<sup>WT</sup> complex, which occurs as a result of the dissociation of the complex during gel running.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> This supports our conclusion that the resulting RNA-c-KIT<sup>WT</sup> complex is most likely retarded in the gel wells. As a negative control, we used the chemically modified RNA transcript of our previously published anti-EpCAM DNA aptamer (Ep-1), which shares the same length and the same fixed regions.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> By comparing the EMSA results of our candidate aptamers with the negative control, clearly, no specific interaction was detected between the negative sequence and the target proteins as demonstrated by the absence of the characteristic shifted band and the uniformed unaffected intensity of the migrated RNA band when compared with the free RNA. In addition, the migratory manner of the RNA negative sequence is almost the same regardless of the presence or absence of different ratios and even types of a given c-KIT protein. The observed smear that extends from a given well to the detected RNA band could reflect a sort of nonspecific, weak, and random interaction that can be predicted between protein and any nucleic acid.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. EMSA of the selected aptamers with the c-KIT kinases using different aptamer/protein mixture ratios: (1) 2:1 ratio; (2) 1:1 ratio; (3) 0.5:1 ratio; (4) 0.25:1 ratio; (F) free aptamer, 1:0 ratio; (U) unbound aptamer; (B) bound aptamer, (NS) negative sequence, and (Wt.) free wild type protein, (Mut.V) free mutant c-KIT<sup>D816V</sup> protein, (Mut.H) free mutant c-KIT<sup>D816H</sup> protein, 400 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In view of that, our optimized EMSA method succeeded in studying the interaction between the c-KIT protein targets and the candidate aptamers even, in assessing the characteristic conformation of each aptamer. Indeed, the two observed distinct bands, corresponding to the aptamer–protein complex and the unbound aptamer, reflect a strong interaction between the target protein and a given aptamer with an association constant value (<i>K</i><sub>A</sub>) larger than the dissociation constant (<i>K</i><sub>D</sub>).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> However, these results reflect a rough qualitative estimate of the binding affinity of each aptamer to a given protein target. A good rule of thumb is that equilibrium dissociation constant values greater than 1–3 μM cannot accurately be determined by a standard gel mobility shift assay.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Binding Affinity of Candidate Aptamers</h4><div class="NLM_p">In order to use aptamers in research and clinical applications, a thorough understanding of aptamer–target binding is necessary. In our experiment, the equilibrium dissociation constant (<i>K</i><sub>D</sub>) was used to quantitatively evaluate the c-KIT binding affinity of our candidate aptamers. As indicated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, apparently, all aptamers show relatively compatible binding affinities toward the tested kinases with dissociation constant values in the nanomolar range. When we considered the measured apparent affinities of each candidate aptamer toward all tested kinases, we found that aptamer <b>V5</b> binds to mutant c-KIT<sup>D816V</sup> kinase with a <i>K</i><sub>D</sub><sup>app</sup> value that is approximately two times lower than the c-KIT<sup>WT</sup> and mutant c-KIT<sup>D816H</sup> kinases. Noteworthy, the sequence of <b>V5</b> aptamer appeared only with the mutant c-KIT<sup>D816V</sup> during the SELEX experiment and showed the highest frequency compared to other anti c-KIT<sup>D816V</sup> aptamers. These results are in full agreement with the widely held view that the dominant sequence prevailed in the final pool of SELEX experiment is the one with the highest target affinity.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> Indeed, the difference in affinities between c-KIT<sup>D816V</sup> and c-KIT<sup>WT</sup> kinases observed using <b>V5</b> aptamer indicates that it derives part of its binding energy from epitopes specific to the mutated D816V activation loop domain. Most probably, the D816 site might be located at the center of the epitope or the epitope itself might surround the D816 residue. Likewise, aptamer <b>H5/V36</b>, which was only detected with both mutant proteins during the SELEX, shows similar higher binding affinities toward the mutant (c-KIT<sup>D816H</sup> and c-KIT<sup>D816V</sup>) kinases compared to the wild type c-KIT kinase. This result may suggest that <b>H5/V36</b> aptamer has the ability to bind to common regions structurally shared by both mutated proteins and distinguished from that of the wild type kinase. In contrast, <b>W3</b> anti c-KIT<sup>WT</sup> aptamer, despite being the most abundant sequence, did not exhibit the highest binding affinity to the wild type c-KIT kinase compared to the mutant forms. Most notably, many studies reported that the examined selected aptamers with the highest copy number in the final SELEX pool were not necessarily the tightest binder.<a onclick="showRef(event, 'ref25 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref25 ref45 ref46">(25,45,46)</a> Consistent with these reports, our data demonstrated that the multiplicity of our candidate aptamers did not always correlate with the determined <i>K</i><sub>D</sub> values.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Apparent Dissociation Constant (<i>K</i><sub>D</sub><sup>app</sup>) Values for Candidate Aptamers<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0010.gif" alt="" id="gr100" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Results represent the mean of three independent experiments (±SEM).</p></div></div><div></div></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding curves of candidate aptamers to c-KIT kinases. The data was fitted to one site specific binding with Hill slope using nonlinear regression analysis, Graphpad prism 6 software. Each data point represents the mean of three independent experiments ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In another aspect, the binding curves of all our candidate aptamers suggest that the binding does not fit a 1:1 binding model. Indeed, the presence of obvious steep Hill slopes (<i>h</i> ≫1) complicate the interpretation of our binding results. Early in 1994, Jellinek and collaborators observed similar binding pattern with their high affinity RNA aptamers selected against the VEGF.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> They speculated that this type of binding could be explained using a model of which a given RNA aptamer represents assemblies of at least two conformationally related species that bind to their target with different affinities.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Noticeably, in our EMSA results (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) we observed that <b>V5</b> and <b>H5/V36</b> aptamers have three characteristic migratory bands (one major and two minor bands). Therefore, these results more likely represent conformationally distinct species of an RNA aptamer that, as they postulated, may exhibit different binding affinities toward a given target. Additionally, it was suggested that generally, the steep Hill slope may indicate that the inhibitor is forming aggregates in the aqueous solution.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Such suggestion seems plausible based on the published data that RNA aptamers can form dimers and a higher order of aggregates.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In another perspective, it was documented that tight-binding inhibitors also display steep Hill slopes due to the fact that they are titrating the target protein.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Additionally, this characteristic sigmoidal binding curve was observed in several studies that employed filter binding assays to determine the binding affinities of their selected aptamers. Such as the binding curve of K61 aptamer selected against cytohesin-1 and 2,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> the binding curves of the selected aptamers 12.01 and M302 recognize the drug-resistant HIV-1 reverse transcriptase<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and many others.<a onclick="showRef(event, 'ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53">(51−53)</a></div><div class="NLM_p last">The most prevailing view implies that the sigmoidal binding curve is considered an evidence for positive cooperativity in the binding reaction. This means that the binding of one aptamer molecule to a given target influences the affinity of subsequent aptamers to the same target.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> In the same approach, Ozer and White proposed a model of density-dependent cooperative (DDC) binding, which relates the cooperative binding to both the target concentration and the target surface density on the immobilizing substrate.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> According to this model, the target immobilization might increase the surface density of target molecules to the point where one ligand could concurrently interact nonspecifically with multiple targets. The latter explanation seems more plausible to explain the cooperative binding effects observed in the binding curves of our candidate aptamers.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Inhibitory Potency of Candidate Aptamers on the Activity of Recombinant c-KIT Kinases</h4><div class="NLM_p">Standard analyses of enzyme kinetics generally preferred to read the reaction at the initial linear phase of the enzymatic reaction when 10% or less of the substrate has been depleted, which represents the first requirement for steady state conditions.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Under our experimental conditions, the reaction progress curves (data not shown) revealed that the period of time over which the product concentration increases linearly is 30 min; after that, the enzyme progression is decreased. This time interval represents the initial velocity period for all tested kinases. It is also important to ascertain that the enzyme concentration chosen should result in a low ATP to ADP conversion percentage to maintain proper kinetics for the enzyme reaction.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Thus, 75 ng of c-KIT<sup>WT</sup>, 10 ng of mutant c-KIT<sup>D816H</sup> kinase, and 6 ng of mutant c-KIT<sup>D816V</sup> kinase were used per reaction to maintain the initial velocity conditions over the 30 min reaction time. Additionally, our enzymatic screen is designed to run under pseudo-first order kinetics by running the assay under conditions where poly (Glu:Tyr) peptide substrate is set at the saturation level (200 μg/mL), well above its predetermined apparent <i>K</i><sub>m</sub> value (<i>K</i><sub>m</sub><sup>app</sup>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">Figure S1</a>). The <i>K</i><sub>m</sub><sup>app</sup> value for the synthetic poly (Glu:Tyr) peptide was estimated to be 6 μg/mL, and to our knowledge, it is reported here for the first time. The second substrate (ATP) is set at its measured <i>K</i><sub>m</sub><sup>app</sup> value for the c-KIT<sup>WT</sup> kinase and above the measured <i>K</i><sub>m</sub><sup>app</sup> values for the c-KIT<sup>D816H</sup> and c-KIT<sup>D816V</sup> kinases (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">Figure S2</a>). The <i>K</i><sub>m</sub><sup>app</sup> values for ATP were estimated for the three active c-KIT kinases, which are 51.1 μM for c-KIT<sup>WT</sup> kinase, 20.6 μM for mutant c-KIT<sup>D816H</sup>, and 26.4 μM for mutant c-KIT<sup>D816V</sup> kinases as indicated in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">Figure S2</a>. These values are close to the <i>K</i><sub>m</sub> values documented in the literature, which are 42.5, 22, and 17 μM for the active c-KIT<sup>WT</sup>, c-KIT<sup>D816H</sup>, and, c-KIT<sup>D816V</sup> recombinant kinases, respectively.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Finally, the IC<sub>50</sub> values were determined using the antagonist dose–response curve, and the aptamer concentration that provokes a response halfway between the baseline and maximum response is the IC<sub>50</sub> value.</div><div class="NLM_p">Our data demonstrate that <b>H5/V36</b> aptamer stood out as the most effective aptamer that inhibits over 82% of c-KIT<sup>WT</sup> phosphotransferase activity, <i>in vitro</i>, at a concentration of 10 μM with an estimated IC<sub>50</sub> value of 867.1 nM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), whereas <b>V15</b> aptamer showed almost two and a half times lower potency, compared to <b>H5/V36</b> aptamer, with an estimated IC<sub>50</sub> value of 2.18 μM. A similar inhibition approach was observed toward the c-KIT<sup>D816H</sup> kinase activity. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, the measured IC<sub>50</sub> values of <b>H5/V36</b> and <b>V15</b> aptamers are 635.7 nM and 1651 μM, respectively. In order to test the <i>in vitro</i> effect of our candidate aptamers on the mutant c-KIT<sup>D816V</sup> kinase, we analyzed the enzymatic activity at different aptamer concentrations. As anticipated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, the kinase activity of D816V A-loop mutant is markedly inhibited in a dose-dependent manner by preincubation of the kinase with <b>V15</b> aptamer with an IC<sub>50</sub> value equal to 134.2 nM. We observed striking differences in the response of <b>V15</b> aptamer toward the A-loop mutants and the c-KIT kinase with a wild type catalytic domain. The c-KIT kinase activity of the A-loop mutant D816V was, on average, 16-fold more sensitive to inhibition by <b>V15</b> than the wild type c-KIT and 12-fold more than the mutant D816H. These data demonstrate that this aptamer doses not only potently inhibit the kinase activity of the mutant c-KIT<sup>D816V</sup> kinase but also shows significant differential specificity between the wild type and mutated c-KIT kinases and even between the two A-loop mutants. Indeed, the weak activity of <b>V15</b> aptamer against c-KIT<sup>WT</sup> kinase is pointing to a potential safety margin for inhibiting mutant c-KIT over the endogenous c-KIT<sup>WT</sup> signaling and other members of class III RTKs. Additionally, <b>V15</b> aptamer was 9-fold more potent than the sunitinib drug; the FDA approved multitargeted RTK inhibitor when it was tested under the same experimental conditions (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E). For <b>H5/V36</b> aptamer, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A indicates that the enzymatic activity was reduced to 50% at an aptamer concentration of 2.6 μM. The most abundant <b>V5</b> aptamer showed reduced effect on the mutated c-KIT<sup>D816V</sup> kinase with only partial inhibition being observed through the highest concentration of aptamer used as indicated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D. Sequences analysis demonstrated that <b>V5</b> and <b>V15</b> anti c-KIT<sup>D816V</sup> aptamers have homologous conserved motif <b>5′-UGGGA-3</b>′ that located at the same characterized interior junctional loop in the predicted secondary structures (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). It is interesting to note that <b>V15</b> aptamer has evolved additional <b>5′-UUU-3</b>′ nucleotides that precede this examined motif. Such finding could provide a possible interpretation for the markedly different potencies of <b>V5</b> and <b>V15</b> aptamers against the c-KIT<sup>D816V</sup> kinase. Finally, <b>W3</b> aptamer showed low or no biochemical detectable effect on all tested c-KIT kinases. It should be mentioned that some aptamers do not affect the known functions of their target proteins. A research group from Ellington’s laboratory reported HIV-1 RT (drug-resistant variants of HIV-1 reverse transcriptase) binding aptamers, which did not exhibit detectable inhibition of either the polymerase or RNase activities of the protein.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> However, few studies report such “inactive aptamers” since it is usually the inhibitory aptamers that are chosen for further characterization.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In the end, the presence of “promiscuous” inhibitors or “aggregating” compounds in a chemical library has become a recent area of research. These compounds can cause false positives that can be frequently recognized by steep concentration response curves. The exact mechanism of these false positives is not known, but it is possible that these compounds induce the formation of compound–enzyme clusters, which reduce the enzyme activity.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Our data revealed that the same nonspecific behavior has been observed at the highest concentration (10 μM) of some aptamers and even the negative sequence as depicted in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B–F. As we previously mentioned, it was reported that RNA aptamers can actually form dimers and higher order of aggregates at the higher concentrations.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Measured IC<sub>50</sub> (nM) Values of Candidate Aptamers against Different c-KIT Kinases</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0011.gif" alt="" id="gr101" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><p class="first last">*Ambiguous: GraphPad Prism was unable to find a unique fit to these data.</p></div></div><div></div></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Antagonist inhibition curves of candidate aptamers against all tested c-KIT kinases. Each data point represents the average percentage of inhibition from two independent experiments performed in triplicate. Error bars depict the standard deviation of the mean. Maximum enzyme activity control sample (0% inhibition) contains all the kinase reaction components except the aptamer. For sunitinib, the final DMSO concentration was 1.5%. Data fitting was performed using the three-parameter dose–response equation, GraphPad Prism 6 software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Identifying the Modes of c-KIT Kinase Inhibition</h4><div class="NLM_p">According to the results described above, <b>V15</b> and <b>H5/V36</b> aptamers can inhibit the phosphotransferase activities of our tested kinases. Therefore, their modes of action (MOAs) were examined by measuring the velocity of a given tested kinase (in RLU/minute) as a function of ATP concentrations under two fixed concentrations of these aptamers. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>-A), <b>H5/V36</b> aptamer has resulted in a decrease of the apparent affinity of c-KIT<sup>WT</sup> kinase to its ATP substrate (<i>K</i><sub>m</sub><sup>app</sup> values increase), which is as expected by a competitive mechanism of inhibition.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Modes of action (MOAs) of aptamer <b>H5/V36</b> against tested kinases. Panels A, C, and E: Enzyme activity was determined at various ATP concentrations in the presence or absence of the aptamer. Each data point represents the average enzyme activity at each ATP concentration obtained from two independent experiments performed in triplicates. Panels B, D, and F: The Lineweaver–Burk plot of data by using the apparent <i>K</i><sub>m</sub> and the apparent <i>V</i><sub>max</sub> values obtained for each inhibitor concentration. The <i>x</i>-axis values represent (−1/<i>K</i><sub>m</sub>) and the <i>y</i>-axis values represent (1/<i>V</i><sub>max</sub>)(1 + <i>K</i><sub>m</sub>/[ATP minimum]). All data analyses were conducted using GraphPad Prism 6 Software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, the pattern of straight lines with intersecting Y-axis intercepts observed in the Lineweaver–Burk plot <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) represents the characteristic signature of a competitive inhibitor as they do not affect the apparent value of <i>V</i><sub>max</sub>. The apparent dissociation constant (<i>K</i><sub>i</sub>) value for a competitive inhibitor can be calculated using the Cheng and Prusoff equation.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Accordingly, the calculated <i>K</i><sub>i</sub> value of aptamer <b>H5/V36</b> for the wild type c-KIT is 490.1 nM. As we detailed in the previous section, the identified ATP-mimicking motif <sub><b>50</b></sub><b>UACG</b><sub><b>53</b></sub> was detected in the proposed secondary structure of our candidate <b>H5/V36</b> (<sub><b>31</b></sub><b>UACA</b><sub><b>34</b></sub>) and <b>W3</b> (<sub><b>35</b></sub><b>UACG</b><sub><b>38</b></sub>) aptamers.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> This finding is interesting because it is presumed that <b>H5/V36</b> aptamer most likely adopted the same mechanism of action, and the <b>A</b><sub><b>32</b></sub> base located at the terminal triloop structure could extend into the active site of the wild type c-KIT kinase and competitively block ATP binding. Moreover, in an excellent agreement with the previously reported data,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a><b>W3</b> anti c-KIT<sup>WT</sup> aptamer, which has the same tetraloop active motif (<sub><b>35</b></sub><b>UACG</b><sub><b>38</b></sub>) as C13 aptamer, showed no apparent inhibitory effect on the phosphotransferase activity of the c-KIT<sup>WT</sup> enzyme when it was tested under the same experimental conditions (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). The first facile explanation for this apparent discrepancy in the inhibitory effects between <b>H5/V36</b> and <b>W3</b> aptamers is that the <b>A</b><sub><b>34</b></sub><b>:U</b><sub><b>30</b></sub> Watson–Crick base pair in <b>H5/V36</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) replaces the <b>G</b><sub><b>38</b></sub><b>:U</b><sub><b>33</b></sub> Wobble base pair in the stem region of the active hairpin structure in <b>W3</b> aptamer which may affect the overall aptamer conformation, stability, and binding. This combined with the fact that <b>G</b><sub><b>38</b></sub> has an additional free functional carbonyl group, which can be involved in hydrogen bond interactions with the kinase surface. Additionally, we cannot rule out that both aptamers have other distinct secondary substructures that render them to exhibit a unique pattern of interactions with the kinase domain, and these are the key determinants of the aptamer functionality.</div><div class="NLM_p">On the basis of this approach, <b>H5/V36</b> aptamer could belong to type I kinase inhibitors that target the active form of c-KIT<sup>WT</sup> kinase characterized by an open conformation of the A-loop and referred to as “DFG in” conformation.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> In a complementary fashion, due to the conserved nature of the ATP-binding domain among protein kinases, the selectivity of <b>H5/V36</b> aptamer has been verified against other kinases. This is necessary in order to assess that it does not inhibit other nontarget kinases, resulting in undesirable adverse effects. As clearly demonstrated in the selectivity assay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>), <b>H5/V36</b> aptamer showed high selectivity against the tested kinases belonging to the same subfamily (as with PDGFRβ kinase) or to another kinase family (JNK3 kinase) despite its presumed interaction with the ATP binding site. This finding is consistent with the high selectivity profile reported for the anti-GRK2 aptamer against a panel of kinases belonging to distinct kinase groups.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> However, an important point to bear in mind is that a given mode of inhibition does not necessarily tell us where on the enzyme surface an aptamer can bind. Indeed, all we know from the kinetic data is that <b>H5/V36</b> aptamer binds to the free c-KIT<sup>WT</sup> enzyme and blocks the ATP binding. It is also possible for the aptamer to bind at a distinct site that is distal from the substrate binding site and to induce a conformational change that closes down the active site so that the substrate can no longer bind. In the case of D816H mutant, <b>H5/V36</b> aptamer apparently inhibits the enzymatic reaction without affecting the affinity toward the ATP substrate as demonstrated in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C. The figure clarifies that the binding of <b>H5/V36</b> aptamer lead to a decrease in <i>V</i><sub>max</sub> value without affecting the apparent <i>K</i><sub>m</sub> value for the ATP substrate. Accordingly, <b>H5/V36</b> aptamer apparently inhibits the activity of mutant c-KIT<sup>D816H</sup> kinase in a purely noncompetitive manner with respect to ATP substrate. The apparent dissociation constant (<i>K</i><sub>i</sub> value) of a noncompetitive inhibitor equals to its observed IC<sub>50</sub> value according to Cheng and Prusoff equation.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Antagonist inhibition curves of aptamer <b>H5/V36</b> against both PDGFRβ and JNK3 kinases. Panel A: Data fitting was performed using the three-parameter dose–response equation, GraphPad Prism 6 software; each data point represents the average percentage of inhibition from two independent experiments performed in triplicate. Error bars depict standard deviation of the mean. Panel B: Average inhibition percentages of aptamer <b>H5/V36</b> against all tested kinases at 5 μM final concentration. Data obtained from IC<sub>50</sub> experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">So, the <i>K</i><sub>i</sub> value of <b>H5/V36</b> aptamer with mutant c-KIT<sup>D816H</sup> kinase equals 635.7nM. On the other hand, the same assay was performed with <b>V15</b> aptamer and indicated that the presence of <b>V15</b> decreases both the apparent affinity of c-KIT<sup>D816H</sup> kinase for the ATP substrate (<i>K</i><sub>m</sub><sup>app</sup> values increase) and the reaction velocity as demonstrated in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A. This finding is in agreement with the mixed mechanism of inhibition.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The same mixed type of inhibition was observed with mutant c-KIT<sup>D816V</sup> enzyme in the presence of both <b>V15</b> and <b>H5/V36</b> aptamers as shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>C. According to this type of inhibition, the aptamer binds with unequal affinity to both free enzyme and the enzyme–ATP complex. The pattern of the straight lines in the Lineweaver–Burk plots and the position of the intersection points above the <i>x</i>-axis and left to the <i>y</i>-axis indicate that both aptamers bind to the free enzyme with less affinity than enzyme-ATP complex.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Mode of action (MOA) of aptamer <b>V15</b> against mutated c-KIT<sup>D816V</sup>and c-KIT<sup>D816H</sup> kinases. Panels A and C: Enzyme activity was determined at various ATP concentrations in the presence or absence of the aptamer. Each data point represents the average enzyme activity at each ATP concentration obtained from two independent experiments performed in triplicate. Panels B and D: The Lineweaver–Burk plot of data by using the apparent <i>K</i><sub>m</sub> and apparent <i>V</i><sub>max</sub> values obtained for each inhibitor concentration. The <i>x</i>-axis values represent (−1/<i>K</i><sub>m</sub>) and the <i>y</i>-axis values represent (1/<i>V</i><sub>max</sub>)(1 + <i>K</i><sub>m</sub>/[ATP minimum]). All data analyses were conducted using GraphPad Prism 6 Software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In summary, <b>H5/V36</b> aptamer can functionally distinguish between the c-KIT<sup>WT</sup>, c-KIT<sup>D816H</sup>, and c-KIT<sup>D816V</sup> kinases by modulating the phosphorylation activity of each in a distinct mechanism of action and in different potency. Structurally, the D816 residue is located at the A-loop of the c-KIT kinase domain, and hence, substituting the acidic aspartic acid residue with valine or histidine can eliminate a negative charge from the loop region.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Moreover, the conformational conversion of the kinase domain from inactive DFG-out to active DFG-in conformation pushes the D816 approaching the negatively charged D572, thus causing an electrostatic repulsion between them.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Thus, in the light of these observations, the most likely explanation is that D816 missense mutations induced conformational changes that may cause structural distortion and direct effects on aptamer binding such as steric or electrostatic hindrance that could occlude the entrance to the catalytic site and hence may impede the aptamer’s active motif from extending deep in the active site and competitively hindering the ATP binding. Nevertheless, the aptamer still can bind to the adjacent or distant sites and affects the enzyme activity with lower potency in an ATP noncompetitive manner. Basically, the binding events of noncompetitive inhibitors occur exclusively at a site distinct from the precise active site occupied by the substrate.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Taken together, <b>H5/V36</b> aptamer could most likely belong to the type V bivalent kinase inhibitors, which target two sites on the protein kinase surface including the catalytic site and another distinct allosteric site on the c-KIT kinase domain that remote from the ATP binding pocket.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Thus, <b>H5/V36</b> aptamer may perturb the kinase activity of c-KIT kinase by mediating both orthosteric and allosteric effects. Indeed, the preparation of bivalent inhibitors serves as a strategy to increase potency and selectivity and has been applied to several kinases.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> In the same approach, <b>H5/V36</b> aptamer could also be considered as type V bisubstrate inhibitor that competitively targets the ATP binding site and noncompetitively targets the peptide substrate binding site. It is worth to mention here that we found a DNA sequence (called <b>V10</b>) almost similar to the <b>H5/V36</b> RNA aptamer when we examined the sequences of DNA aptamers that were previously selected by our laboratory colleagues, independently, against the c-KIT<sup>D816V</sup> kinase using the SELEX technique (unpublished data).</div><div class="NLM_p">In another aspect, the inhibition curves shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C demonstrate that <b>V15</b> aptamer exhibits potent inhibition against c-KIT<sup>D816V</sup> kinase activity, mild inhibition against c-KIT<sup>D816H</sup> kinase activity, and weak potency against the wild type c-KIT kinase phosphotransferase activity. Our results indicated that <b>V15</b> aptamer has an unequal affinity for both free enzymes (c-KIT<sup>D816H</sup> and c-KIT<sup>D816V</sup>) and the enzyme–ATP complex (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Accordingly, we suppose that <b>V15</b> aptamer is an allosteric inhibitor that decreases the kinase activity by binding to an allosteric site other than the active site on the target kinase. It remains unclear how the binding of <b>V15</b> aptamer to this site inhibits the kinase activity. One possibility is that <b>V15</b> binding could induce a conformational change in the target enzyme that can affect the formation of the usual enzyme–substrate complex and hence modulate the kinase activity.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Currently, the literature describing RNA aptamers affecting the activities of proteins by an allosteric regulation remains limited, and hence, a clear demonstration of the allosteric regulation of protein function by an RNA aptamer would be of high interest.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Another possible scenario is that <b>V15</b> aptamer could bind to an allosteric site in close proximity to the catalytic domain, thereby allowing for steric interference with the access of substrates to the catalytic site. Indeed, the shape and the considerable large size of our nucleotide aptamer (∼29 kDa) relative to the c-KIT kinase domain (∼73 kDa) are most likely able to produce an extensive shielding of protein surfaces and long-range steric interference with the binding events even at sites distant from the identified aptamer binding region.</div><div class="NLM_p last">This effect was suggested to be caused by either the large electronegativity of the aptamer or the steric hindrance of access to a key contact surface.<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a> It is worth pointing out that the identified occluded allosteric pocket in the active conformation of kinases could be exposed in the oncogenic mutants, which may explain the significant specificity of <b>V15</b> aptamer toward the A-loop activating mutants.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Our data, however, were in contrast to a study demonstrating that 5′<b>GGAGGG</b>3′ hexamer motif is essential for the anti-EGFR tyrosine kinase activity expressed in the A431 epidermoid cancer cells.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Our findings revealed that this motif was not related to the potent inhibitory activity of <b>V15</b> aptamer as this hexamer motif is repeated twice in the variable region of our selected <b>H30</b> anti c-KIT<sup>D816H</sup> aptamer that was used as a control in the <i>in vitro</i> kinase assays (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">Figure S3</a>). As anticipated, <b>H30</b> aptamer shows mild inhibitory potencies of no more than 50% even at the highest concentration used. Taken as a whole, there is a clear need to better define the detailed molecular and structural characterization of the binding motif of <b>V15</b> aptamer on the mutant c-KIT<sup>D816V</sup> kinase, which will provide the means for the development of highly specific anticancer ligands targeting the D816V activating loop mutation in the c-KIT kinase. Indeed, our results provide a proof-of-concept example in isolating specific and functional RNA aptamers against protein targets with a single amino acid mutation. To the best of our knowledge, only two aptamers have been previously characterized in the literature that can distinguish a mutant protein with a single amino acid substitution from the wild type protein.<a onclick="showRef(event, 'ref66 ref67'); return false;" href="javascript:void(0);" class="ref ref66 ref67">(66,67)</a></div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Aptamer Stability in Human Serum</h4><div class="NLM_p">Serum stability of RNA aptamers is limited by the endonuclease cleavage. Generally, aptamer stability is directly related to the structure and chemical composition of their molecules. For example, it is expected that tightly folded molecules are more resistant to degradation by nucleases because of limited solvent exposure. In serum, the reported half-lives for 2′-hydroxy (RNA) nucleic acids are in the order of seconds.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div><div class="NLM_p">To determine the stability of <b>H5/V36</b> aptamer, the RNA aptamer was incubated in fresh human serum (total 80% human serum) for a total of 4 days at 37 °C. Then, serum–RNA samples at the indicated time-points were resolved on a 3% agarose gel electrophoresis (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The results clearly demonstrate that our synthesized aptamer was stable for up to 4 days in fresh human serum, indicating that 2′-fluoro-ribose modification is able to provide resistance against nucleases for further use in therapeutic applications.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Stability of modified <b>H5/V36</b> RNA aptamers against serum nucleases. Aptamer was incubated with fresh human serum for up to 96 h at 37 °C and visualized using 3% agarose gel electrophoresis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Inhibitory Effect of Aptamers on the Viability of H526 SCLC Cells</h4><div class="NLM_p">Human small cell lung cancer (SCLC) is an aggressive disease representing approximately 20% of lung cancers. The molecular abnormalities underlying SCLC are not well understood.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> One feature of SCLC is coexpression of the c-KIT RTK with its ligand stem cell factor (SCF). At least 70% of SCLC tumors and tumor derived cell lines coexpress the genes for SCF and c-KIT RTK.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a></div><div class="NLM_p">The expression of wild type c-KIT receptors in NCI-H526 human SCLC cell line was previously confirmed by Krystal et al.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Upon stimulation with rhSCF, the c-KIT tyrosine kinase undergoes a rapid increase in its degree of tyrosine phosphorylation.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> Several studies have demonstrated that the activation of c-KIT can lead to enhanced proliferation, inhibition of apoptosis, and enhanced motility of SCLC cells.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> To examine the inhibitory effect of our candidate aptamers on the receptor activity, H526 cells were made quiescent by overnight incubation in serum-free medium, then the cells were incubated with 200 nM of each aptamer in complex with lipofectamine (Invitrogen, USA) or with lipofectamine vehicle alone for 6 h followed by the addition of rhSCF growth factor or serum-containing medium. The cellular metabolism was assessed over a period of 72 h by using the MTT (Promega, USA) colorimetric dye reduction method.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> It was reported that MTT assay correlates very well with the viable SCLC cell number under the conditions used.<a onclick="showRef(event, 'ref70 ref74'); return false;" href="javascript:void(0);" class="ref ref70 ref74">(70,74)</a></div><div class="NLM_p">Our data demonstrated that after three days, the <b>W3</b> and <b>H5/V36</b> aptamers have significantly decreased the metabolic activity, and so, the viability of H526 treated cells stimulated by rhSCF, as the only exogenous growth factor, compared to cells treated with the negative sequence control. As anticipated, the metabolic activity was reduced to about 60% by comparison with cells mock treated or treated with the negative sequence (<b>NS</b>) (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A). No effect was observed in the presence of <b>V5</b> and <b>V15</b> aptamers. However, to be clinically relevant, inhibition of the metabolic activity should extend to the cellular activity in the presence of a complex mix of growth factors found in the serum. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B, the inhibitory effect was statistically significant in serum-containing medium when cells were treated with <b>W3</b>, <b>H5/V36</b>, and <b>V5</b> aptamers for three days. The inhibition potency was also no more than 40% at 200 nM final aptamer concentration, whereas no effect was observed in the presence of the <b>NS</b> and <b>V15</b> aptamer. This moderate inhibition potency could be probably consistent with the previously published findings showing that the growth of SCLC cells in serum-free medium was only slightly increased upon SCF stimulation and the H526 has a very low level of basal PI3K-Akt activation.<a onclick="showRef(event, 'ref72 ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref72 ref75 ref76">(72,75,76)</a> In contrast, it was reported that sunitinib (SU11248) potently inhibits the SCF-dependent proliferation of H526 cells <i>in vitro</i> after 72 h. However, its potency was reduced dramatically in the presence of complete serum.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Indeed, the ability of <b>H5/V36</b> aptamer to inhibit the metabolic activity of H526 SCLC cells is expected as these cells showed high expression of wild type c-KIT protein, which is potently inhibited by <b>H5/V36</b> aptamer as revealed from our <i>in vitro</i> cell-free biochemical kinase assay. Among the main intracellular effectors of c-KIT that mediate the induction of cell growth and resistance to apoptosis are the extracellular-signal regulated kinase 1/2 (ERK1/2) and AKT. Thus, a good correlation can be mediated between the ability of <b>H5/V36</b> aptamer to competitively bind the ATP binding pocket and its ability to inhibit receptor activation and hence subsequent signaling pathways that promote cell growth.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Cell viability assays. Cell viability was assessed using MTT assay. NCI-H526 (30000 cells/well in 96-well plates) cells were mock treated or treated with 200 nM final concentration of each candidate aptamer in complex with lipofectamine for 6 h and then stimulated with (A) rhSCF (100 ng/mL) or with (B) 5% FBS for 72 h. The cell viability was expressed as percent of metabolically active cells present after treatment with respect to untreated cells containing only the vehicle. The results from two independent experiments are shown. Data points represent the mean of eight replicates ± SD. <i>P</i> values were determined using two-tailed Student’s <i>t</i> test. *<i>P</i> < 0.05 and **<i>P</i> < 0.01 were regarded as statistically significant when compared with the negative sequence control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Unexpectedly, <b>W3</b> aptamer also shows significant inhibition in the metabolic activity of rhSCF and FBS stimulated cells. A plausible explanation for this unexpected result is that <b>W3</b> aptamer may bind to the ATP binding pocket through the identified active motif only when the c-KIT<sup>WT</sup> kinase domain is in the inactive unphosphorylated conformation. Subsequently, activation by the growth factors induces conformational changes in the kinase domain that could result in the disappearance of the epitope for <b>W3</b> aptamer binding. Noteworthy, a similar approach was reported for type II kinase inhibitors, such as imatinib (STI-571), that binds to the ATP binding pocket as well as the adjacent hydrophobic pocket created by the DFG-out inactive conformation. This pocket, referred to as the allosteric binding site, is not present in the activated kinase.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> Therefore, such inhibitors often show greatly enhanced potency in cell-based assays in which the kinase is in the nonactivated form before stimulation, rather than in the biochemical assays in which highly activated (phosphorylated) kinases are used.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> To our surprise, <b>V5</b> aptamer strongly inhibits the viability of H526 cells, which is reduced to about 60% by comparison with cells treated with the negative sequence.</div><div class="NLM_p last">This inhibitory effect was only observed when the cells were stimulated with 5% serum but not with SCF as the sole exogenous growth factor. A similar observation was reported by Bergan et al. in this study they identified 21-mer oligodeoxynucleotide 1 (ODN1; also called BCR-AS) that could inhibit p210<sup>bcr-abl</sup> kinase activity in a sequence-specific manner that depends on repeated <b>GGC</b> motifs. Using K562 chronic myelogenous leukemia-derived cell line, they showed that additional ODN (termed BCR-SCR) that did not inhibit p210<sup>bcr-abl</sup> kinase <i>in vitro</i> still appeared to alter the cell kinase activity.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> We postulate that <b>V5</b> aptamer most likely inhibits specific downstream effectors that are only activated and/or sustained active by the complex growth factors existing in the serum. Moreover, it is worth pointing out that the time course of Akt activation in H526 SCLC is growth factor dependent. As demonstrated, the rhSCF induced Akt phosphorylation that peaked at 5 to 10 min began fading between 15 to 30 min and returned back to baseline by 60 min after stimulation. The kinetics of 10% FCS-mediated activation of Akt phosphorylation was delayed and could be barely detected prior to 30 min, peaked at 1 h after stimulation, and markedly prolonged for at least 6 h after FCS stimulation.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> Interestingly, mass spectra analysis for the fully activated form of the wild type c-KIT kinase revealed that the A-loop tyrosine (Y823) is not phosphorylated until very late in the activation reaction (>90% completion), indicating that the A-loop phosphorylation is not required for c-KIT activation and for the kinase activity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> On the basis of these findings, it is reasonable to speculate that one possible interpretation of disparate differences between the biochemical and cellular assays is that our candidate <b>V5</b> aptamer could bind to a characteristic epitope around or near Y823 in the activation loop domain. In an excellent agreement with our data anticipated from the <i>in vitro</i> biochemical kinase assays, it was reported that the kinase activity of the wild type c-KIT and mutated c-KIT<sup>D816V</sup> kinase are unaffected by the Y823F mutation. As illustrated, the Y823 is not involved in the regulation of the kinase activity of both wild type and D816V mutated c-KIT kinase.<a onclick="showRef(event, 'ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref80 ref81">(80,81)</a> Such data could provide a reasonable explanation of the undetected inhibition of our tested c-KIT kinases in the biochemical kinase activity when incubated with <b>V5</b> aptamer (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). However, at the cellular level, the role of the activation loop tyrosine Y823 in the c-KIT kinase is related to the downstream signaling rather than the kinase activity. As elucidated, the Ba/F3 cell expressing the Y823F mutant of c-KIT showed almost a 60% reduction in cell survival compared with cells expressing the wild type c-KIT.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> So, the observed reduction in the H526 cell viability is likely to be attributed to a potential functional effect of <b>V5</b> aptamer binding to the A-loop in a way that blocking the Y823 residue hence affects the subsequent downstream signaling pathways related to cell viability and probably to the cell migration. As proposed by Agarwal et al., the Y823 in the c-KIT kinase appears to be a good target for cancer therapy. Thus, a therapy targeting Y823 so that its phosphorylation is prevented, presumably by an aptamer, could in combination with chemotherapy provide an improved treatment option for tumors caused by c-KIT mutations.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> Finally, under our experimental conditions, no apparent inhibitory effect was observed in the presence of 200 nM <b>V15</b> aptamer. These results are also in concordance with our previous biochemical assays. After trying to place it in the context of other findings, our results illustrate that this aptamer most probably binds specifically or with higher affinity to the mutant rather than to the c-KIT<sup>WT</sup> expressed in the H526 SCLC cells.</div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75473" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75473" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the present work, we have successfully selected RNA aptamers that showed high affinity to their respective recombinant proteins with <i>K</i><sub>D</sub> values in the nanomolar range. Two of the selected aptamers (<b>H5/V36</b> and <b>V15</b>) can potently inhibit the <i>in vitro</i> kinase activity of their respective targets in different modalities. The <b>V15</b> aptamer was effectively able to functionally inhibit the activation loop mutants of c-KIT kinase more potently than the wild type c-KIT in a noncompetitive mechanism of action. Presumably, it interacts with less conserved structural aspects to their target kinase, and hence, it could possess a much greater potential for selectivity than the small molecule inhibitors of ATP binding. Thus, the inhibitory properties of <b>V15</b> demonstrate its potential usefulness as a lead compound for the design of anticancer drugs. The <b>H5/V36</b> competitive inhibitor possesses high selectivity toward wild type c-KIT kinase as compared to other kinases and thus represents a promising candidate for functional interference and analysis of endogenous wild type c-KIT function inside the cells and living organisms. Taken together, our results suggest that these RNA aptamers may serve as a platform for the future development of novel aptamer-based targeted cancer therapies. Additionally, the potential of both aptamers to inhibit the kinase activity of their respective targets revealed the possibility of using these aptamers as specific enzymatic inhibitors for laboratory use. To the best of our knowledge, the selected aptamers described in this study are the first RNA aptamers developed against the kinase domain of the wild type and the mutated D816H/ D816V c-KIT proteins.</div><div class="NLM_p last">We believe that our RNA aptamers could serve as useful tools to define new functional sites on the c-KIT kinase that can be used to guide the rational chemical synthesis of small molecule drugs. A study demonstrated that aptamers can be converted into small molecule inhibitors using the so-called aptamer-displacement assays for rapidly identifying alternative small-molecule target sites in proteins.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> These strategies have been developed to enable the conversion of an aptamer/protein complex into a small organic inhibitor by screening small-molecule libraries for compounds that can displace the aptamer from its target and adopt its inhibitory activity.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66302" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66302" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Pool Design and Synthesis</h3><div class="NLM_p">The initial 90 nucleotides (nt) long single-stranded DNA (ssDNA) pool was purchased from Midland (Midland certified company, USA). The design of the oligonucleotides comprised a central variable region of 45 nucleotides length flanked by two fixed-sequence regions at both ends allowing primer annealing and PCR amplification.</div><div class="NLM_p">5′-<u class="uu">GGGATGGATCCAAGCTTACTGG</u>(<b>45N</b>)<u class="uu">GGGAAGCTTCGATAGGAATTCGG</u>-3′.</div><div class="NLM_p last">N is one of four nucleotides: A, T, C, or G. The initial 2′-fluoro-pyrimidine-modifiedrandom RNA pool was synthesized as previously described by Ababneh et al.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> <i>In Vitro</i> Selection Process (SELEX)</h3><div class="NLM_p last">The 2′-fluoro-modified random RNA pool was heated in folding buffer (4.2 mM Na<sub>2</sub>HPO<sub>4</sub>, 2 mM KH<sub>2</sub>PO<sub>4</sub>, 140 mM NaCl, 10 mM KCL, and 5 mM MgCl<sub>2</sub>) at 70 °C for 10 min. Then, the mixture was rapidly cooled in ice for 10 min to ensure proper folding of RNA ligands into their 3D structures, after which the pool was equilibrated at room temperature (RT) for an additional 10 min. In the first three rounds of SELEX, a negative preselection step was performed by mixing one microgram of the RNA pool with the MagneGST Glutathione Particles (Promega, USA) at RT for 20 min. After that, the flow-through of unbound sequences was incubated with an equimolar mixture of the active GST-tagged c-KIT recombinant proteins (Millipore, USA): wild type c-KIT (c-KIT<sup>WT</sup>) and two mutated proteins, c-KIT<sup>D816H</sup> and c-KIT<sup>D816V</sup>, for 1 h at 37 °C. For the partitioning purpose, the equilibrated GST magnetic beads (Promega, USA) were added and incubated with the mixture for 20 min at RT. Then, the magnetic stand separator was used to separate the beads with bound protein–RNA complexes, and the supernatant containing the unbound RNA sequences was carefully aspirated and discarded. The beads were then washed with the folding buffer to remove any nonspecific binders and to enhance partition efficiency. After proper washing, the RNA sequences bound to the GST-tagged c-KIT proteins and beads were eluted by mixing with an appropriate volume of glutathione containing elution buffer at room temperature for 15 min. The recovered RNA aptamers were reverse-transcribed into cDNA for further amplification by PCR, and the PCR products were purified by PAGE using the crush and soak method. Finally, the purified PCR products were <i>in vitro</i> transcribed to produce the second pool of 2′-F-modified RNA molecules for the second round of SELEX. After the fourth round of selection, the obtained RNA pool was split and incubated with each variant of c-KIT proteins individually for the remaining cycles of selection. Thirteen rounds of iterative selection and amplification have been performed for the selection of RNA aptamers against c-KIT<sup>WT</sup>, c-KIT<sup>D816H</sup>, and c-KIT<sup>D816V</sup> proteins. At the end of the selection experiment, the winning RNA sequences with enhanced binding to the corresponding target were isolated from the evolved last pool, reverse-transcribed into cDNA, and amplified by PCR. The double-stranded PCR products were purified using QIAquick PCR purification kit (Qiagen, USA) and cloned into pGEM-T plasmid using pGEM-T Easy vector system (Promega, USA). The <i>Escherichia coli</i> competent cells were then transformed and plated in selective media. The positive transforming colonies were isolated, and the plasmids of individual clones were extracted and purified using PureYield Plasmid Miniprep System (Promega, USA). The eluate containing the purified plasmid DNA was collected and stored at −20 °C for sequencing and characterization.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> DNA Sequencing and Sequences Analysis</h3><div class="NLM_p last">The extracted DNA plasmids were sent to Macrogen Inc., South Korea, to perform Sanger sequencing by ABI Prism 3700XL capillary sequencer instrument (Applied Biosystems, USA). The sequencing reactions were performed using the pUC/M13 forward universal sequencing primer. Each sequence was named as KIT (WX/HX/VX) based on their protein target and the sequencing serial number, where W stands for c-KIT<sup>WT</sup>, H for c-KIT<sup>D816H</sup>, and V for c-KIT<sup>D816V</sup> proteins. Sequences analysis and alignment were carried out using the software CLC Main Workbench (Version 7.8.1) from the Web site <a href="http://www.qiagenbioinformatics.com" class="extLink">www.qiagenbioinformatics.com</a>. Putative Quadruplex forming G Rich Sequences (QGRS) of candidate aptamers were predicted using QGRS Mapper software 3 from <a href="http://bioinformatics.ramapo.edu/QGRS/index.php" class="extLink">http://bioinformatics.ramapo.edu/QGRS/index.php</a>.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Synthesis of 2′-Fluoro-pyrimidine Modified RNA Aptamers</h3><div class="NLM_p last">The initial ssDNA sequences of candidate aptamers were purchased from Midland Certified Reagent Company (0.2 μM scale, Reverse Phase (RP)-HPLC; Midland, Texas. USA) and amplified by PCR using the designed forward primer (APT-FT7: <u class="uu">5′-GCTAATACGACTCACTATA</u>GGGATGGATCCAAGCTTACTGG-3′) and reverse primer (APT-R: 5′-CCGAATTCCTATCGAAGCTTCCC-3′). The forward primer (APT-FT7) contains a T7 promoter sequence (underlined) for the <i>in vitro</i> transcription. The PCR mix contained 30 ng of the initial ssDNA template, 1× GoTaq reaction buffer, 200 μM of each dNTPs mix, 1 μM of each aptamer, 2.5 mM MgCl<sub>2</sub>, and 2.5 U of <i>Taq</i> DNA polymerase (Promega, USA). The PCR program starts by heating up to 95 °C for 5 min as an initial denaturation step. Then, the cycling begins with a short denaturation step for 1 min at 95 °C and primers annealing for 1 min at 55 °C followed by 1 min extension step at 72 °C. These cycles were repeated 14 times, followed by a final elongation step of 10 min at 72 °C. The resulting 109 bp PCR products were purified by using the MinElute PCR purification system (Qiagen, Germany). Then, the eluted DNA fragments were loaded in 2% low-melting point agarose gel to be further purified using MinElute gel extraction system (Qiagen, Germany). Following the PCR purification, double-stranded T7-DNA fragments were <i>in vitro</i> transcribed into 2′-fluoro-pyrimidine-modified RNA aptamers using the DuraScribe T7 Transcription Kit (Epicentre-Lucigen, Wisconsin, USA) which produces DuraScript RNA transcripts that are resistant to RNase A degradation. According to the manufacturer’s protocol, each <i>in vitro</i> transcription reaction was carried out in 20 μL reaction volume and incubated at 37 °C for overnight. The resulting RNA transcripts were treated with DNase I for subsequent degradation of the remaining DNA template at 37 °C for 30 min. Then, the RNA transcripts were purified using miRNeasy Mini kit (Qiagen, Germany), which combines both phenol/guanidine and silica membrane-based purification systems. Finally, the purified RNA aptamers were analyzed using 4% agarose gel to assess the purity, length of the transcript, and extent of DNase I treatment. This was performed by mixing equal volume of purified RNA aptamers with 2× RNA loading dye (New England BioLabs, UK) and heated at 70 °C for 10 min before loading in the gel. Gel electrophoresis was run at a constant voltage of 70 V for 30 min. The purity of synthesized RNA aptamers was further evaluated using a NanoDrop spectrophotometer (ThermoFischer Scientific, USA) by measuring the absorbance of a given RNA aptamer at 260, 280, and 230 nm wavelengths in OD unit. Then, the OD<sub>260</sub>/OD<sub>280</sub> and OD<sub>260</sub>/OD<sub>230</sub> ratios were calculated. The purity ratios for “pure” RNA aptamer were ∼2.0 for OD<sub>260</sub>/OD<sub>280</sub> and in the range of 2.0–2.2 for the OD<sub>260</sub>/OD<sub>230</sub> ratio, which indicate that the synthesized RNA aptamer is free from proteins and other contaminants, e.g., phenol and guanidine reagents that were used in the purification kit. The purity of all tested aptamers is >95% purity.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Secondary Structures Prediction</h3><div class="NLM_p last">The secondary structures of the candidate RNA aptamers were predicted using the Mfold web server program, which is freely available at <a href="http://unafold.rna.albany.edu/?q=mfold/RNA-Folding-Form2.3" class="extLink">http://unafold.rna.albany.edu/?q=mfold/RNA-Folding-Form2.3</a> (version 2.3 energies). The most likely structure was chosen on the basis of the lowest predicted free energy of formation (Δ<i>G</i>; kcal/mol) using the default settings except for the temperature (25 °C). The Structure Display and Free Energy Determination application was also used that takes into consideration the folding conditions, which are the temperature (25 °C), and the ionic conditions, particularly Na<sup>+</sup> and Mg<sup>2+</sup> concentrations that were set at 140 mM Na<sup>+</sup> and 5 mM Mg<sup>2+</sup> (<a href="http://unafold.rna.albany.edu/?q=mfold/Structure-display-and-free-energy-determination" class="extLink">http://unafold.rna.albany.edu/?q=mfold/Structure-display-and-free-energy-determination</a>).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Electrophoresis Mobility Shift Assay (EMSA)</h3><div class="NLM_p last">Typically, 7% w/v mini nondenaturing polyacrylamide gel (19:1 acrylamide/bisacrylamide; EuroClone, Europe) of 1.00 mm thickness was prepared in 0.5× TBE buffer and 2.5 mM MgCl<sub>2</sub> final concentrations. Following polymerization, the gel was placed into the electrophoresis tank filled with prechilled 0.5× TBE running buffer supplemented with 2.5 mM MgCl<sub>2</sub>. The gel was prerun at constant 80 V in the refrigerator for at least 45 min. Aptamer–protein mixtures were prepared by mixing 400 nM fixed concentration of folded aptamers with various concentrations of purified recombinant proteins to get different ratios of aptamer–protein mixture in a reaction buffer supplemented with 2.5 mM MgCl<sub>2</sub> final concentration. Binding reactions were incubated for 1 h at 37 °C. At the end of the incubation time, each sample was mixed with 6× Blue/Orange loading dye (Promega Corporation, USA) and immediately loaded in the gel. Finally, the gel was run under constant voltage of 80 V for 60 min in the refrigerator. After that, the RNA aptamers in the gel were stained using SYBR Green II RNA Gel Stain (Invitrogen, USA). The RNA/SYBR Green II complexes were then illuminated, and the resulting fluorescence was visualized using SYBER GREEN filter of ChemiDoc MP Imaging System (Biorad, USA).</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Measurement of the Binding Affinity of RNA Aptamers</h3><div class="NLM_p last">The extent of affinity between an aptamer and its target is estimated by means of the equilibrium dissociation constant (<i>K</i><sub>D</sub>). To do so, a series of aptamer concentrations ranges from 20 to 150 nM were prepared in binding buffer (4.2 mM Na<sub>2</sub>HPO<sub>4</sub>, 2 mM KH<sub>2</sub>PO<sub>4</sub>, 140 mM NaCl, 10 mM KCl, and 2.5 mM MgCl<sub>2</sub>) and mixed with fixed concentration of diluted protein (10 nM) in a final volume of 40 μL binding reaction. Then, the binding reactions were incubated for 1 h at 37 °C. For separation of aptamer–protein complexes from unbound aptamers, 5 μL of the pre-equilibrated GST magnetic beads (Promega, USA) were added at RT with gentle mixing on a rotating platform for 30 min. Unbound aptamers in the supernatant were carefully collected and transferred to a new tube. Then, the beads were washed once again using 100 μL of binding buffer, and the resulting supernatant was also collected and combined in the same tube. This washing step was performed to remove any residual unbound aptamers. Finally, the concentration of unbound aptamers from each binding reaction was quantified using the QuantiFluor RNA system (Promega Corporation, Madison, USA). Control experiments in the absence of protein were performed at each aptamer concentration to calculate the nonspecific binding signal attributed to aptamers that may nonspecifically adhere directly to the beads. For aptamer quantification, a standard curve was prepared using the same binding buffer. For measurement, unknown samples, blank, and standard samples were transferred to a 96-well solid-black NBS microplate (Corning, USA). Then, diluted QuantiFluor RNA dye (Promega Corporation, Madison, USA) was added to each well and mixed briefly according to the manufacturer’s instructions. Finally, the fluorescence was measured using the GloMax-Multi+ Detection Systems (Promega Corporation, Madison, USA).</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Calculations and Data Analysis</h3><div class="NLM_p last">The concentrations of the unbound aptamers in all washing collections were calculated using the prepared standard curve. For affinity measurement, the concentration of bound aptamers in each reaction mixture was calculated by subtracting the measured concentration of unbound aptamers from the concentration of the total aptamers used in each reaction mixture. As illustrated in the following formula:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_m001.gif" alt="" id="_i26" /></img></span>Then, the concentration of the aptamer–protein complexes, which represents the specific binding, was calculated by subtracting the total concentration of aptamers bound to the beads from the concentration of aptamers nonspecifically bound to the beads at that particular concentration using the following formula:<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_m002.gif" alt="" id="_i27" /></img></span>For each candidate aptamer, three independent experiments were performed. After that, the <i>K</i><sub>D</sub> value was determined by plotting the calculated concentrations of aptamer–protein complexes on the <i>y</i>-axis versus the concentrations of total aptamers on <i>x</i>-axis and fitting the data using a one-site saturation binding curve with the following Hill equation:<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_m003.gif" alt="" id="_i28" /></img></span>where [<i>T</i><sub>total</sub>] is the apparent total concentration of the protein target, [TA] is the calculated concentration of aptamer–target complexes at a given concentration of aptamers used in the binding reaction [<i>A</i>], and h is the Hill slope.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> All data analyses were conducted using GraphPad Prism 6 (GraphPad Software, San Diego, CA, USA).</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Biochemical <i>in Vitro</i> Kinase Assay</h3><div class="NLM_p last">In our characterization experiment, the ADP-Glo Kinase assay kit (Promega, USA) was used. The luminescence signal, represented as relative luminescence unit (RLU), was measured using the GloMax-Multi-Detection System (Promega Corporation, Madison, USA) and an integration time of 0.5 s per well. In a 384-well solid-white NBS (Corning, USA) microplate, 1.5 μL of the diluted recombinant kinase enzyme was preincubated with 1.5 μL of the folded aptamer for 30 min in the binding buffer at RT. Then, 2.5× ATP/poly(Glu:Tyr) peptide mixture was prepared in the kinase buffer (12.5 mM Tris, pH 7.5; 6.25 mM MgCl<sub>2</sub>; 31.25 μg/mL BSA; and 125 μM DTT). The kinase reaction was started by adding 2 μL of 2.5× ATP/poly(Glu:Tyr) peptide mixture into each well to get 5 μL kinase reaction volumes. The reactions were allowed to proceed at 25 °C for 30 min. For each kinase assay, the background (blank) signal was determined by enzyme-free reaction and then subtracting this signal from all measured values. All kinase reactions were conducted in the initial velocity region of the enzymatic reaction in which less than 10% of the substrate has been converted to product.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> IC<sub>50</sub> Values Determination</h3><div class="NLM_p last">The IC<sub>50</sub> value of each candidate aptamer was determined by using an antagonist dose–response experiment in which the activity of a given tested kinase is determined in the presence or absence of different aptamer concentrations that are equally spaced on the log scale. Therefore, 10-fold dilutions of a given aptamer were prepared in folding buffer and mixed with equal volume of tested kinase enzyme using the predetermined optimized enzyme concentration for the steady state condition. Accordingly, 75 ng of c-KIT<sup>WT</sup>, 6 ng of c-KIT<sup>D816V</sup>, and 10 ng of c-KIT<sup>D816H</sup> recombinant active enzymes were used (SignalChem, Lifescience, Canada). The final prepared concentrations of a given aptamer ranged from 0.01 to 10 μM in the reaction wells. Finally, the kinase reactions were initiated using 50 μM ATP and 200 μg/mL poly(Glu:Tyr) peptide as the final concentrations. Then, the reactions were allowed to proceed for 30 min at 25 °C. The maximum enzyme activity control (0% inhibition) was prepared by mixing all components of the kinase reaction except the aptamer. The luminescent signal (RLU) for each reaction well was measured, and the percentage of inhibition (%) was calculated. The data were plotted in the antagonist dose–response curve, where the percentage of inhibition was plotted at the <i>y</i>-axis versus the logarithm of each aptamer concentration at the <i>x</i>-axis. The aptamer antagonist dose–response curves were analyzed by fitting data to a three-parameter logistic equation “dose–response curve with standard slope” from which the apparent IC<sub>50</sub> values were deduced. All data analyses were conducted using GraphPad Prism 6 (GraphPad Software, San Diego, CA, USA). All experiments were performed at least twice, and figures are representative of the results obtained. Values are expressed as means of triplicate determinations ± standard deviations (SD).</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Identification of the Mechanism of Action</h3><div class="NLM_p last">The enzyme activity of the analyzed kinase was measured at different ATP concentrations in the presence or absence of the aptamer. Two different aptamer concentrations were used, and serial dilutions of 2.5× ATP were prepared in kinase buffer that contained fixed 2.5× poly(Glu:Tyr) peptide concentration. This will result in variable concentrations of ATP substrate ranged from 0.5 to 100 μM in tested wells combined with fixed 200 μg/mL poly(Glu:Tyr) peptide final concentration. The measured kinase activity at a given ATP concentration in RLU/minute was plotted at the <i>y</i>-axis versus the ATP concentrations at the <i>x</i>-axis. Data were fitted to the Henri–Michaelis–Menten equation using nonlinear regression analysis. Finally, to create a Lineweaver–Burk line corresponding to the nonlinear regression fit, we followed the GraphPad curve fitting guide by using the predetermined <i>K</i><sub>m</sub><sup>app</sup> and <i>V</i><sub>max</sub><sup>app</sup> values (GraphPad Software, San Diego, CA, USA).</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Selectivity Determination</h3><div class="NLM_p last">The selectivity of a given candidate aptamer was determined by measuring its IC<sub>50</sub> value against PDGFRβ tyrosine kinase as well as against JNK3 serine-threonine kinase. The antagonist dose–response experiments were performed as described above. For PDGFR β kinase assay, we followed the manufacturer’s protocol by using 10 ng of active kinase enzyme, 25 μM ATP, and 200 μg/mL poly(Glu:Tyr) peptide. For JNK3 kinase, 4 ng of the active enzyme, 1 μM ATP, and 200 μg/mL of p38 peptide were used to perform the assay. Both kinase reaction mixes were incubated for 60 min at 25 °C.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Aptamer Stability Assay</h3><div class="NLM_p last">To investigate the stability of our synthesized 2′-fluoro-pyrimidine-modified RNA aptamers against nucleases, fresh human serum was mixed with a given aptamer for different time intervals extending up to 4 days. For each time point, 10 μg of aptamer was folded in 10 μL of folding buffer and then incubated with 40 μL of fresh human serum at 37 °C for the defined time point. After that, 25 μL aliquot was taken and treated with 1.5 mAU of proteinase K (Qiagen, Germany) followed by incubation at 56 °C for 10 min.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> The samples were then stored at −20 °C. Later, the stability of the treated aptamer was analyzed on 3% agarose gel stained with ethidium bromide by mixing 10 μL of each time point sample with 10 μL of 2× denaturing RNA loading dye (New England BioLabs, UK), heated for 10 min at 70 °C, and then rapidly cooled on ice for 2 min. The gel electrophoresis was run at 70 V for 25 min, and the intensity of each band was visualized using Image Lab Software, ChemiDoc MP Imaging System (BioRad, USA).</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Cell Viability Assay</h3><div class="NLM_p last">NCI-H256 cells from human SCLC cell line (obtained from American Type Culture Collection, ATCC; Manassas, VA) were cultured in RPMI 1640 supplemented with 10% FBS and maintained in a humidified chamber at 37 °C and 5% CO<sub>2</sub>. Then, cells were split in 96-well plates at density 3 × 10<sup>4</sup> cells/well in 60 μL of serum free RPMI 1640 medium for overnight. The number of cells plated was adjusted so that untreated cells were in the exponential growth phase at the time of treatment. On the next day, the cells were treated dropwise with 40 μL of aptamer–lipid complexes and incubated for 6 h at 37 °C and 5% CO<sub>2</sub> before ligand stimulation. Subsequently, cells were stimulated either with 100 μL of RPMI 1640 medium supplemented with 200 ng/mL rhSCF (R&D Systems) or with 100 μL of RPMI 1640 medium supplemented with 10% FBS and incubated for 72 h at 37 °C. After that, the number of metabolically active cells present after 3 days was assessed by using CellTiter 96 Non-Radioactive Cell Proliferation assay (Promega, USA).</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Preparation of Aptamer–Lipid Complexes</h3><div class="NLM_p last">The candidate aptamers were first diluted to 8 μM in folding buffer, folded, and then mixed with equal volume of serum-free RPMI 1640 medium. The Lipofectamine 3000 Reagent (Invitrogen, USA) was also diluted by mixing 0.25 μL with 4.75 μL of serum-free RPMI 1640 medium to get 5 μL of diluted Lipofectamine for each well. A master mix of diluted Lipofectamine was prepared and mixed thoroughly by vortexing for 2–3 s. In PCR tubes, 5 μL of the diluted Lipofectamine Reagent was transferred to each tube, and then, 5 μL of diluted aptamer was added to each tube in 1:1 ratio and incubated for 10–15 min at RT as recommended by the manufacturer’s protocol. After incubation, 30 μL of serum free RPMI 1640 medium was added to each well and mixed, and then 40 μL of aptamer–lipid complexes was added dropwise onto the cells. Mock control was prepared by mixing all the above ingredients except the aptamer. The treated cells were incubated for 3 days at 37 °C and 5% CO<sub>2</sub>. Four replicate wells were performed per each candidate aptamer, and the data were expressed as the percentage of metabolically active cells to the untreated control cells containing only the Lipofectamine vehicle. Assays were repeated twice and statistically analyzed using Student’s two tailed <i>t</i> test on GraphPad Prism 6 (GraphPad Software, San Diego, CA, USA). Data of <i>P</i> value < 0.05 was considered statistically significant when compared to the negative sequence control value.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00868" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40372" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40372" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00868" class="ext-link">10.1021/acs.jmedchem.9b00868</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplemental tables and figures: nucleotide sequences of all selected anti c-KIT<sup>WT</sup> aptamers, nucleotide sequences of all selected anti c-KIT<sup>D816H</sup> aptamers, nucleotide sequences of all selected anti c-KIT<sup>D816V</sup> aptamers, apparent <i>K</i><sub>m</sub> value for peptide substrate, apparent <i>K</i><sub>m</sub> values for ATP substrate, and antagonist inhibition curves of H30 aptamer (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf">jm9b00868_si_001.pdf (613.36 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00868" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67205" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67205" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ala’a S. Shraim</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences, School of Science, The University of Jordan, Amman JO 11942, Jordan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Laboratory Sciences, Faculty of Allied Medical Sciences, Al-Ahliyya Amman University, Amman JO 19328, Jordan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7021-8847" title="Orcid link">http://orcid.org/0000-0001-7021-8847</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e180be92899380888ca19880898e8ecf828e8cc192899380888c808d8080a1868c80888dcf828e8c"><span class="__cf_email__" data-cfemail="33526c405b41525a5e734a525b5c5c1d505c5e">[email protected]</span> <span class="__cf_email__" data-cfemail="91e2f9e3f0f8fcf0fdf0f0d1f6fcf0f8fdbff2fefc">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abdelrahim Hunaiti</span> - <span class="hlFld-Affiliation affiliation">Department
of Clinical Laboratory Sciences, School of Science, The University of Jordan, Amman JO 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abdalla Awidi</span> - <span class="hlFld-Affiliation affiliation">Cell
Therapy Center, The University of Jordan, Amman JO 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Walhan Alshaer</span> - <span class="hlFld-Affiliation affiliation">Cell
Therapy Center, The University of Jordan, Amman JO 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nidaa A. Ababneh</span> - <span class="hlFld-Affiliation affiliation">Cell
Therapy Center, The University of Jordan, Amman JO 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bashaer Abu-Irmaileh</span> - <span class="hlFld-Affiliation affiliation">Hamdi
Mango Center for Scientific Research, The
University of Jordan, Amman JO 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fadwa Odeh</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, School of Science, The University
of Jordan, Amman JO 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Said Ismail</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Physiology, School of Medicine, The University of Jordan, Amman JO 11942, Jordan</span>; 
    <span class="hlFld-Affiliation affiliation">Qatar
Genome Project, Qatar Foundation, Doha, Qatar</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.S.S.: Investigation, methodology (designed and performed the experiments), validation, visualization, and manuscript writing; A.H. and A.A.: supervision and validation; W.A.: project administration; N.A.A.: methodology (designed and performed SELEX method); B.A.-I.: methodology (involved in cell- based assay); F.O.: funding acquisition and project administration; S.I.: conceptualization and funding acquisition.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The corresponding author would like to thank Dr. Yasser K. Bustanji, the director of Hamdi Mango Center for Scientific Research; Dr. Hanan Jafar, Dr. Dana Alhattab, and Dr. Duaa Abuarqoub from Cell Therapy Center; and Mrs. Bara’a Qaqish and Mrs. Fatima Al-Akhras from the Molecular Biology Research Laboratory, Department of Biochemistry, School of Medicine, The University of Jordan, for kindly providing all facilities to perform this project. This work was supported by research grant MPH/2/08/2013 from Scientific Research Support Fund (SRF) to A.A.B., S.I., and F.O. (<a href="http://www.srf.gov.jo/" class="extLink">http://www.srf.gov.jo/</a>), the Deanship of Scientific Research, The University of Jordan, and the Cell Therapy Center, The University of Jordan, Amman, Jordan.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS</td></tr><tr><td class="NLM_term">A431 cells</td><td class="NLM_def"><p class="first last">epidermoid carcinoma cells</p></td></tr><tr><td class="NLM_term">Akt</td><td class="NLM_def"><p class="first last">serine-threonine protein kinase B</p></td></tr><tr><td class="NLM_term">A-loop</td><td class="NLM_def"><p class="first last">activation loop region of the c-KIT kinase domain</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">Ba/F3</td><td class="NLM_def"><p class="first last">murine pro-B hematopoietic cell line</p></td></tr><tr><td class="NLM_term">CML</td><td class="NLM_def"><p class="first last">chronic myelogenous leukemia</p></td></tr><tr><td class="NLM_term">EMSA</td><td class="NLM_def"><p class="first last">electrophoresis mobility shift assay</p></td></tr><tr><td class="NLM_term">EpCAM</td><td class="NLM_def"><p class="first last">epithelial cell adhesion molecules</p></td></tr><tr><td class="NLM_term">EGFRs</td><td class="NLM_def"><p class="first last">epidermal growth factor receptors</p></td></tr><tr><td class="NLM_term">ERK2</td><td class="NLM_def"><p class="first last">extracellular regulated kinase 2</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">GIST</td><td class="NLM_def"><p class="first last">gastrointestinal stromal tumor</p></td></tr><tr><td class="NLM_term">GRK2</td><td class="NLM_def"><p class="first last">G protein-coupled receptor kinase 2</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>D</sub></td><td class="NLM_def"><p class="first last">equilibrium dissociation constant</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>m</sub></td><td class="NLM_def"><p class="first last">Michaelis–Menten constant</p></td></tr><tr><td class="NLM_term">NS</td><td class="NLM_def"><p class="first last">negative sequence</p></td></tr><tr><td class="NLM_term">ODN</td><td class="NLM_def"><p class="first last">oligodeoxynucleotide</p></td></tr><tr><td class="NLM_term">PAGE</td><td class="NLM_def"><p class="first last">polyacrylamide gel electrophoresis</p></td></tr><tr><td class="NLM_term">PDGF-R</td><td class="NLM_def"><p class="first last">platelet derived growth factor-receptors</p></td></tr><tr><td class="NLM_term">PI3K</td><td class="NLM_def"><p class="first last">phosphatidylinositol 3- kinase</p></td></tr><tr><td class="NLM_term">PTK</td><td class="NLM_def"><p class="first last">protein tyrosine kinase</p></td></tr><tr><td class="NLM_term">QGRS</td><td class="NLM_def"><p class="first last">quadruplex forming G-rich sequences</p></td></tr><tr><td class="NLM_term">rhSCF</td><td class="NLM_def"><p class="first last">recombinant human stem cell factor</p></td></tr><tr><td class="NLM_term">RTK</td><td class="NLM_def"><p class="first last">receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">SCLC</td><td class="NLM_def"><p class="first last">small cell lung cancer</p></td></tr><tr><td class="NLM_term">SELEX</td><td class="NLM_def"><p class="first last">Systematic Evolution of Ligands by EXponential enrichment</p></td></tr><tr><td class="NLM_term">TEMED</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethylethylenediamine-M-<span class="smallcaps smallerCapital">d</span>-galactoside</p></td></tr><tr><td class="NLM_term">VEGF</td><td class="NLM_def"><p class="first last">vascular endothelial growth factor</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild type</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56345" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56345" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 86 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span> <span> </span><span class="NLM_article-title">The c-Kit Receptor-Mediated Signal Transduction and Tumor-Related Diseases</span>. <i>Int. J. Biol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">443</span>, <span class="refDoi"> DOI: 10.7150/ijbs.6087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.7150%2Fijbs.6087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23678293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1eku7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=435-443&issue=5&author=J.+Liangauthor=Y.+Wuauthor=B.+Chenauthor=W.+Zhangauthor=Y.+Tanakaauthor=H.+Sugiyama&title=The+c-Kit+Receptor-Mediated+Signal+Transduction+and+Tumor-Related+Diseases&doi=10.7150%2Fijbs.6087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The C-kit receptor-mediated signal transduction and tumor-related diseases</span></div><div class="casAuthors">Liang, Jing; Wu, Yan-Ling; Chen, Bing-Jia; Zhang, Wen; Tanaka, Yoshimasa; Sugiyama, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">435-443, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">IJBSB9</span>;
        ISSN:<span class="NLM_cas:issn">1449-2288</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">A review.  As an important member of tyrosine kinase family, c-kit receptor causes specific expression of certain genes, regulates cell differentiation and proliferation, resists cell apoptosis and plays a key role in tumor occurrence, development, migration and recurrence through activating the downstream signaling mols. following interaction with stem cell factor (SCF).  The abnormality of SCF/c-kit signaling pathway is closely related to some certain tumors.  The discovery of c-kit receptor-targeted drugs has promoted clin.-related cancer's diagnosis and treatment.  In this paper, we review recent research progress on c-kit receptor-mediated signal transduction and its potential therapeutic application as a target in tumor-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJDVYBTh98p7Vg90H21EOLACvtfcHk0lhnppbhwg9jTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1eku7rI&md5=1ccfda0e33eadaa4462fbd241fb81e1d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.7150%2Fijbs.6087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fijbs.6087%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DSugiyama%26aufirst%3DH.%26atitle%3DThe%2520c-Kit%2520Receptor-Mediated%2520Signal%2520Transduction%2520and%2520Tumor-Related%2520Diseases%26jtitle%3DInt.%2520J.%2520Biol.%2520Sci.%26date%3D2013%26volume%3D9%26issue%3D5%26spage%3D435%26epage%3D443%26doi%3D10.7150%2Fijbs.6087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">Structure and Regulation of Kit Protein-Tyrosine Kinase-The Stem Cell Factor Receptor</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>338</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1307</span>– <span class="NLM_lpage">1315</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2005.09.150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.bbrc.2005.09.150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=16226710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1egs77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=338&publication_year=2005&pages=1307-1315&issue=3&author=R.+Roskoski&title=Structure+and+Regulation+of+Kit+Protein-Tyrosine+Kinase-The+Stem+Cell+Factor+Receptor&doi=10.1016%2Fj.bbrc.2005.09.150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and regulation of Kit protein-tyrosine kinase-The stem cell factor receptor</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">338</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1307-1315</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Signaling by stem cell factor and Kit, its receptor, play important roles in gametogenesis, hematopoiesis, mast cell development and function, and melanogenesis.  Moreover, human and mouse embryonic stem cells express Kit transcripts.  Stem cell factor exists as both a sol. and a membrane-bound glycoprotein while Kit is a glycoprotein receptor protein-tyrosine kinase.  The complete absence of stem cell factor or Kit is lethal.  Gain-of-function mutations of Kit are assocd. with several human neoplasms including acute myelogenous leukemia, gastrointestinal stromal tumors, mastocytomas, and nasal T-cell lymphomas.  Binding of stem cell factor to Kit results in receptor dimerization and activation of protein kinase activity.  The activated receptor becomes autophosphorylated at tyrosine residues that serve as docking sites for signal transduction mols. contg. SH2 domains.  Kit activates Akt, Src family kinases, phosphatidylinositol 3-kinase, phospholipase Cγ, and Ras/mitogen-activated protein kinases.  Kit exists in active and inactive conformations as detd. by X-ray crystallog.  Kit consists of an extracellular domain, a transmembrane segment, a juxtamembrane domain, and a protein kinase domain that contains an insert of about 80 amino acid residues.  The juxtamembrane domain inhibits enzyme activity in cis by maintaining the control αC-helix and the activation loop in their inactive conformations.  The juxtamembrane domain also inhibits receptor dimerization.  STI-571, a clin. effective targeted protein-tyrosine kinase inhibitor, binds to an inactive conformation of Kit.  The majority of human gastrointestinal stromal tumors have Kit gain-of-function mutations in the juxtamembrane domain, and most people with these tumors respond to STI-571.  STI-571 binds to Kit and Bcr-Abl (the oncoprotein of chronic myelogenous leukemia) at their ATP-binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_HWJXdp926bVg90H21EOLACvtfcHk0lj2aPi2r1FhHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1egs77N&md5=e71ef40ef41dfd2d2500558ee65cc4aa</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2005.09.150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2005.09.150%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DStructure%2520and%2520Regulation%2520of%2520Kit%2520Protein-Tyrosine%2520Kinase-The%2520Stem%2520Cell%2520Factor%2520Receptor%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2005%26volume%3D338%26issue%3D3%26spage%3D1307%26epage%3D1315%26doi%3D10.1016%2Fj.bbrc.2005.09.150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">The Role of Small Molecule Kit Protein-Tyrosine Kinase Inhibitors in the Treatment of Neoplastic Disorders</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2018.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.phrs.2018.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29704617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptVagsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2018&pages=35-52&author=R.+Roskoski&title=The+Role+of+Small+Molecule+Kit+Protein-Tyrosine+Kinase+Inhibitors+in+the+Treatment+of+Neoplastic+Disorders&doi=10.1016%2Fj.phrs.2018.04.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The role of small molecule Kit protein-tyrosine kinase inhibitors in the treatment of neoplastic disorders</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-52</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Kit proto-oncogene was found as a consequence of the discovery of the feline v-kit sarcoma oncogene.  Stem cell factor (SCF) is the Kit ligand and it mediates Kit dimerization and activation.  The Kit receptor contains an extracellular segment that is made up of five Ig-like domains (D1/2/3/4/5), a transmembrane segment, a juxtamembrane segment, a protein-tyrosine kinase domain that contains an insert of 77 amino acid residues, and a carboxyterminal tail.  Activating somatic mutations in Kit have been documented in various neoplasms including gastrointestinal stromal tumors (GIST), mast cell overexpression (systemic mastocytosis), core-binding factor acute myeloid leukemias (AML), melanomas, and seminomas.  In the case of gastrointestinal stromal tumors, most activating mutations occur in the juxtamembrane segment and these mutants are initially sensitive to imatinib.  As with many targeted anticancer drugs, resistance to Kit antagonists occurs in about two years and is the result of secondary KIT mutations.  An activation segment exon 17 D816V mutation is one of the more common resistance mutations in Kit and this mutant is resistant to imatinib and sorafenib.  Type I protein kinase inhibitors interact with the active enzyme form with DFG-D of the proximal activation segment directed inward toward the active site (DFG-Din).  In contrast, type II inhibitors bind to their target with the DFG-D pointing away from the active site (DFG-Dout).  Based upon the X-ray crystallog. structures, imatinib, sunitinib, and ponatinib are Type II Kit inhibitors.  We used the Schrodinger induced fit docking protocol to model the interaction of midostaurin with Kit and the result indicates that it binds to the DFG-Din conformation of the receptor and is thus classified as type I inhibitor.  This medication inhibits the notoriously resistant Kit D816V mutant and is approved for the treatment of systemic mastocytosis and is effective against tumors bearing the D816V activation/resistance mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqteI4_qt5heLVg90H21EOLACvtfcHk0lj2aPi2r1FhHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptVagsb4%253D&md5=8a71d3b7399d53c3085fb04a87aef1a2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2018.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2018.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DThe%2520Role%2520of%2520Small%2520Molecule%2520Kit%2520Protein-Tyrosine%2520Kinase%2520Inhibitors%2520in%2520the%2520Treatment%2520of%2520Neoplastic%2520Disorders%26jtitle%3DPharmacol.%2520Res.%26date%3D2018%26volume%3D133%26spage%3D35%26epage%3D52%26doi%3D10.1016%2Fj.phrs.2018.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ségaliny, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tellez-Gabriel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymann, D.</span></span> <span> </span><span class="NLM_article-title">Receptor Tyrosine Kinases: Characterisation, Mechanism of Action and Therapeutic Interests for Bone Cancers</span>. <i>Journal of Bone Oncology</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1016/j.jbo.2015.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.jbo.2015.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=26579483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A280%3ADC%252BC28rgtlyjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=1-12&issue=1&author=A.+I.+S%C3%A9galinyauthor=M.+Tellez-Gabrielauthor=M.+Heymannauthor=D.+Heymann&title=Receptor+Tyrosine+Kinases%3A+Characterisation%2C+Mechanism+of+Action+and+Therapeutic+Interests+for+Bone+Cancers&doi=10.1016%2Fj.jbo.2015.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor tyrosine kinases: Characterisation, mechanism of action and therapeutic interests for bone cancers</span></div><div class="casAuthors">Segaliny Aude I; Tellez-Gabriel Marta; Heymann Marie-Francoise; Heymann Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of bone oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>
        ISSN:<span class="NLM_cas:issn">2212-1366</span>.
    </div><div class="casAbstract">Bone cancers are characterised by the development of tumour cells in bone sites, associated with a dysregulation of their environment.  In the last two decades, numerous therapeutic strategies have been developed to target the cancer cells or tumour niche.  As the crosstalk between these two entities is tightly controlled by the release of polypeptide mediators activating signalling pathways through several receptor tyrosine kinases (RTKs), RTK inhibitors have been designed.  These inhibitors have shown exciting clinical impacts, such as imatinib mesylate, which has become a reference treatment for chronic myeloid leukaemia and gastrointestinal tumours.  The present review gives an overview of the main molecular and functional characteristics of RTKs, and focuses on the clinical applications that are envisaged and already assessed for the treatment of bone sarcomas and bone metastases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-qtzzXrDFl42y4bB4mnvYfW6udTcc2eaX5trr7Jqi_7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rgtlyjtg%253D%253D&md5=57563bfe33ca5752242d56cc24546c1c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.jbo.2015.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbo.2015.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A9galiny%26aufirst%3DA.%2BI.%26aulast%3DTellez-Gabriel%26aufirst%3DM.%26aulast%3DHeymann%26aufirst%3DM.%26aulast%3DHeymann%26aufirst%3DD.%26atitle%3DReceptor%2520Tyrosine%2520Kinases%253A%2520Characterisation%252C%2520Mechanism%2520of%2520Action%2520and%2520Therapeutic%2520Interests%2520for%2520Bone%2520Cancers%26jtitle%3DJournal%2520of%2520Bone%2520Oncology%26date%3D2015%26volume%3D4%26issue%3D1%26spage%3D1%26epage%3D12%26doi%3D10.1016%2Fj.jbo.2015.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hojjat-Farsangi, M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors of the Receptor Tyrosine Kinases: Promising Tools for Targeted Cancer Therapies</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">13768</span>– <span class="NLM_lpage">13801</span>, <span class="refDoi"> DOI: 10.3390/ijms150813768</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.3390%2Fijms150813768" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=25110867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslyqu7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=13768-13801&issue=8&author=M.+Hojjat-Farsangi&title=Small-Molecule+Inhibitors+of+the+Receptor+Tyrosine+Kinases%3A+Promising+Tools+for+Targeted+Cancer+Therapies&doi=10.3390%2Fijms150813768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of the receptor tyrosine kinases: promising tools for targeted cancer therapies</span></div><div class="casAuthors">Hojjat-Farsangi, Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">13768-13801, 34</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Chemotherapeutic and cytotoxic drugs are widely used in the treatment of cancer.  In spite of the improvements in the life quality of patients, their effectiveness is compromised by several disadvantages.  This represents a demand for developing new effective strategies with focusing on tumor cells and min. side effects.  Targeted cancer therapies and personalized medicine have been defined as a new type of emerging treatments.  Small mol. inhibitors (SMIs) are among the most effective drugs for targeted cancer therapy.  The growing no. of approved SMIs of receptor tyrosine kinases (RTKs) i.e., tyrosine kinase inhibitors (TKIs) in the clin. oncol. imply the increasing attention and application of these therapeutic tools.  Most of the current approved RTK-TKIs in preclin. and clin. settings are multi-targeted inhibitors with several side effects.  Only a few specific/selective RTK-TKIs have been developed for the treatment of cancer patients.  Specific/selective RTK-TKIs have shown less deleterious effects compared to multi-targeted inhibitors.  This review intends to highlight the importance of specific/selective TKIs for future development with less side effects and more manageable agents.  This article provides an overview of: (1) the characteristics and function of RTKs and TKIs; (2) the recent advances in the improvement of specific/selective RTK-TKIs in preclin. or clin. settings; and (3) emerging RTKs for targeted cancer therapies by TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0oFHs0APBNrVg90H21EOLACvtfcHk0lgE5ws8Rxc1Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslyqu7zI&md5=08836c25f587f44f2843abc6b2d3587f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fijms150813768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms150813768%26sid%3Dliteratum%253Aachs%26aulast%3DHojjat-Farsangi%26aufirst%3DM.%26atitle%3DSmall-Molecule%2520Inhibitors%2520of%2520the%2520Receptor%2520Tyrosine%2520Kinases%253A%2520Promising%2520Tools%2520for%2520Targeted%2520Cancer%2520Therapies%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2014%26volume%3D15%26issue%3D8%26spage%3D13768%26epage%3D13801%26doi%3D10.3390%2Fijms150813768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mol, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheibe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. P.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for The Autoinhibition and STI-571 Inhibition of c-Kit Tyrosine Kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">31655</span>– <span class="NLM_lpage">31663</span>, <span class="refDoi"> DOI: 10.1074/jbc.M403319200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1074%2Fjbc.M403319200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=15123710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlslOrtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=31655-31663&author=C.+Molauthor=D.+Douganauthor=T.+Schneiderauthor=R.+Skeneauthor=M.+Krausauthor=D.+Scheibeauthor=G.+P.+Snellauthor=H.+Zouauthor=B.+C.+Sangauthor=K.+P.+Wilson&title=Structural+Basis+for+The+Autoinhibition+and+STI-571+Inhibition+of+c-Kit+Tyrosine+Kinase&doi=10.1074%2Fjbc.M403319200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Autoinhibition and STI-571 Inhibition of c-Kit Tyrosine Kinase</span></div><div class="casAuthors">Mol, Clifford D.; Dougan, Douglas R.; Schneider, Thomas R.; Skene, Robert J.; Kraus, Michelle L.; Scheibe, Daniel N.; Snell, Gyorgy P.; Zou, Hua; Sang, Bi-Ching; Wilson, Keith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">31655-31663</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activity of the c-Kit receptor protein-tyrosine kinase is tightly regulated in normal cells, whereas deregulated c-Kit kinase activity is implicated in the pathogenesis of human cancers.  The c-Kit juxtamembrane region is known to have an autoinhibitory function; however the precise mechanism by which c-Kit is maintained in an autoinhibited state is not known.  We report the 1.9-Å resoln. crystal structure of native c-Kit kinase in an autoinhibited conformation and compare it with active c-Kit kinase.  Autoinhibited c-Kit is stabilized by the juxtamembrane domain, which inserts into the kinase-active site and disrupts formation of the activated structure.  A 1.6-Å crystal structure of c-Kit in complex with STI-571 (Imatinib or Gleevec) demonstrates that inhibitor binding disrupts this natural mechanism for maintaining c-Kit in an autoinhibited state.  Together, these results provide a structural basis for understanding c-Kit kinase autoinhibition and will facilitate the structure-guided design of specific inhibitors that target the activated and autoinhibited conformations of c-Kit kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWJsKx1IzXQbVg90H21EOLACvtfcHk0lgE5ws8Rxc1Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlslOrtrc%253D&md5=df6065a62b44e4c2e6b020da0f797ab4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M403319200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M403319200%26sid%3Dliteratum%253Aachs%26aulast%3DMol%26aufirst%3DC.%26aulast%3DDougan%26aufirst%3DD.%26aulast%3DSchneider%26aufirst%3DT.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DKraus%26aufirst%3DM.%26aulast%3DScheibe%26aufirst%3DD.%26aulast%3DSnell%26aufirst%3DG.%2BP.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DSang%26aufirst%3DB.%2BC.%26aulast%3DWilson%26aufirst%3DK.%2BP.%26atitle%3DStructural%2520Basis%2520for%2520The%2520Autoinhibition%2520and%2520STI-571%2520Inhibition%2520of%2520c-Kit%2520Tyrosine%2520Kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D31655%26epage%3D31663%26doi%3D10.1074%2Fjbc.M403319200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mol, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowakowski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassel, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcree, D. E.</span></span> <span> </span><span class="NLM_article-title">Structure of a c-Kit Product Complex Reveals the Basis for Kinase Transactivation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span> (<span class="NLM_issue">34</span>),  <span class="NLM_fpage">31461</span>– <span class="NLM_lpage">31464</span>, <span class="refDoi"> DOI: 10.1074/jbc.C300186200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1074%2Fjbc.C300186200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=12824176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVOnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=31461-31464&issue=34&author=C.+D.+Molauthor=K.+B.+Limauthor=V.+Sridharauthor=H.+Zouauthor=E.+Y.+Chienauthor=B.+Sangauthor=J.+Nowakowskiauthor=D.+B.+Kasselauthor=C.+N.+Croninauthor=D.+E.+Mcree&title=Structure+of+a+c-Kit+Product+Complex+Reveals+the+Basis+for+Kinase+Transactivation&doi=10.1074%2Fjbc.C300186200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a c-Kit Product Complex Reveals the Basis for Kinase Transactivation</span></div><div class="casAuthors">Mol, Clifford D.; Lim, Kheng B.; Sridhar, Vandana; Zou, Hua; Chien, Ellen Y. T.; Sang, Bi-Ching; Nowakowski, Jacek; Kassel, Daniel B.; Cronin, Ciaran N.; McRee, Duncan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">31461-31464</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The c-Kit proto-oncogene is a receptor protein-tyrosine kinase assocd. with several highly malignant human cancers.  Upon binding its ligand, stem cell factor (SCF), c-Kit forms an active dimer that autophosphorylates itself and activates a signaling cascade that induces cell growth.  Disease-causing human mutations that activate SCF-independent constitutive expression of c-Kit are found in acute myelogenous leukemia, human mast cell disease, and gastrointestinal stromal tumors.  We report on the phosphorylation state and crystal structure of a c-Kit product complex.  The c-Kit structure is in a fully active form, with ordered kinase activation and phosphate-binding loops.  These results provide key insights into the mol. basis for c-Kit kinase transactivation to assist in the design of new competitive inhibitors targeting activated mutant forms of c-Kit that are resistant to current chemotherapy regimes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv0eZJ9UHNILVg90H21EOLACvtfcHk0lgE5ws8Rxc1Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVOnsb0%253D&md5=f537b5c9903adb6cc057406bd20f0f5a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C300186200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C300186200%26sid%3Dliteratum%253Aachs%26aulast%3DMol%26aufirst%3DC.%2BD.%26aulast%3DLim%26aufirst%3DK.%2BB.%26aulast%3DSridhar%26aufirst%3DV.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DChien%26aufirst%3DE.%2BY.%26aulast%3DSang%26aufirst%3DB.%26aulast%3DNowakowski%26aufirst%3DJ.%26aulast%3DKassel%26aufirst%3DD.%2BB.%26aulast%3DCronin%26aufirst%3DC.%2BN.%26aulast%3DMcree%26aufirst%3DD.%2BE.%26atitle%3DStructure%2520of%2520a%2520c-Kit%2520Product%2520Complex%2520Reveals%2520the%2520Basis%2520for%2520Kinase%2520Transactivation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26issue%3D34%26spage%3D31461%26epage%3D31464%26doi%3D10.1074%2Fjbc.C300186200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heldin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennartsson, J.</span></span> <span> </span><span class="NLM_article-title">Structural and Functional Properties of Platelet-Derived Growth Factor and Stem Cell Factor Receptors</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">8</span>), <span class="NLM_elocation-id">a009100</span> <span class="refDoi"> DOI: 10.1101/cshperspect.a009100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1101%2Fcshperspect.a009100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23906712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslGhsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&issue=8&author=C.+Heldinauthor=J.+Lennartsson&title=Structural+and+Functional+Properties+of+Platelet-Derived+Growth+Factor+and+Stem+Cell+Factor+Receptors&doi=10.1101%2Fcshperspect.a009100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional properties of platelet-derived growth factor and stem cell factor receptors</span></div><div class="casAuthors">Heldin, Carl-Henrik; Lennartsson, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">A009100/1-A009100/19</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The receptors for platelet-derived growth factor (PDGF) and stem cell factor (SCF) are members of the type III class of PTK receptors, which are characterized by five Ig-like domains extracellularly and a split kinase domain intracellularly.  The receptors are activated by ligand-induced dimerization, leading to autophosphorylation on specific tyrosine residues.  Thereby the kinase activities of the receptors are activated and docking sites for downstream SH2 domain signal transduction mols. are created; activation of these pathways promotes cell growth, survival, and migration.  These receptors mediate important signals during the embryonal development, and control tissue homeostasis in the adult.  Their over-activity is seen in malignancies and other diseases involving excessive cell proliferation, such as atherosclerosis and fibrotic diseases.  In cancer, mutations of PDGF and SCF receptors-including gene fusions, point mutations, and amplifications-drive subpopulations of certain malignancies, such as gastrointestinal stromal tumors, chronic myelomonocytic leukemia, hypereosinophilic syndrome, glioblastoma, acute myeloid leukemia, mastocytosis, and melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAYrCdbqmo8rVg90H21EOLACvtfcHk0lhhd2_x5qw2fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslGhsLfF&md5=7fb28447c9fefd66a2e429668dd62884</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a009100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a009100%26sid%3Dliteratum%253Aachs%26aulast%3DHeldin%26aufirst%3DC.%26aulast%3DLennartsson%26aufirst%3DJ.%26atitle%3DStructural%2520and%2520Functional%2520Properties%2520of%2520Platelet-Derived%2520Growth%2520Factor%2520and%2520Stem%2520Cell%2520Factor%2520Receptors%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2013%26volume%3D5%26issue%3D8%26doi%3D10.1101%2Fcshperspect.a009100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cruse, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalfe, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivera, A.</span></span> <span> </span><span class="NLM_article-title">Functional Deregulation of KIT Link to Mast Cell Proliferative Diseases and Other Neoplasms</span>. <i>Immunol. Allergy Clin. North Am.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1016/j.iac.2014.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.iac.2014.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=24745671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A280%3ADC%252BC2cjltlCktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=219-237&issue=2&author=G.+Cruseauthor=D.+D.+Metcalfeauthor=A.+Olivera&title=Functional+Deregulation+of+KIT+Link+to+Mast+Cell+Proliferative+Diseases+and+Other+Neoplasms&doi=10.1016%2Fj.iac.2014.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Functional deregulation of KIT: link to mast cell proliferative diseases and other neoplasms</span></div><div class="casAuthors">Cruse Glenn; Metcalfe Dean D; Olivera Ana</div><div class="citationInfo"><span class="NLM_cas:title">Immunology and allergy clinics of North America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">219-37</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this review, the authors discuss common gain-of-function mutations in the stem cell factor receptor KIT found in mast cell proliferation disorders and summarize the current understanding of the molecular mechanisms by which these transforming mutations may affect KIT structure and function leading to altered downstream signaling and cellular transformation.  Drugs targeting KIT have shown mixed success in the treatment of mastocytosis and other hyperproliferative diseases.  A brief overview of the most common KIT inhibitors currently used, the reasons for the varied clinical results of such inhibitors and a discussion of potential new strategies are provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSnziUwFm7S_7YtC3U5jeOifW6udTcc2eYCFaJz35OQh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cjltlCktg%253D%253D&md5=81bbecd6b864653328f5e39b041a86ad</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.iac.2014.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.iac.2014.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DCruse%26aufirst%3DG.%26aulast%3DMetcalfe%26aufirst%3DD.%2BD.%26aulast%3DOlivera%26aufirst%3DA.%26atitle%3DFunctional%2520Deregulation%2520of%2520KIT%2520Link%2520to%2520Mast%2520Cell%2520Proliferative%2520Diseases%2520and%2520Other%2520Neoplasms%26jtitle%3DImmunol.%2520Allergy%2520Clin.%2520North%2520Am.%26date%3D2014%26volume%3D34%26issue%3D2%26spage%3D219%26epage%3D237%26doi%3D10.1016%2Fj.iac.2014.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lennartsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rönnstrand, L.</span></span> <span> </span><span class="NLM_article-title">Stem Cell Factor Receptor/c-Kit: From Basic Science to Clinical Implications</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1619</span>– <span class="NLM_lpage">1649</span>, <span class="refDoi"> DOI: 10.1152/physrev.00046.2011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1152%2Fphysrev.00046.2011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23073628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2lsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=1619-1649&issue=4&author=J.+Lennartssonauthor=L.+R%C3%B6nnstrand&title=Stem+Cell+Factor+Receptor%2Fc-Kit%3A+From+Basic+Science+to+Clinical+Implications&doi=10.1152%2Fphysrev.00046.2011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Stel cell factor receptor/c-Kit: from basic science to clinical implications</span></div><div class="casAuthors">Lennartsson, Johan; Roennstrand, Lars</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1619-1649</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  Stem cell factor (SCF) is a dimeric mol. that exerts its biol. functions by binding to and activating the receptor tyrosine kinase c-Kit.  Activation of c-Kit leads to its autophosphorylation and initiation of signal transduction.  Signaling proteins are recruited to activated c-Kit by certain interaction domains (e.g., SH2 and PTB) that specifically bind to phosphorylated tyrosine residues in the intracellular region of c-Kit.  Activation of c-Kit signaling has been found to mediate cell survival, migration, and proliferation depending on the cell type.  Signaling from c-Kit is crucial for normal hematopoiesis, pigmentation, fertility, gut movement, and some aspects of the nervous system.  Deregulated c-Kit kinase activity has been found in a no. of pathol. conditions, including cancer and allergy.  The observation that gain-of-function mutations in c-Kit can promote tumor formation and progression has stimulated the development of therapeutics agents targeting this receptor, e.g., the clin. used inhibitor imatinib mesylate.  Also other clin. used multiselective kinase inhibitors, for instance, sorafenib and sunitinib, have c-Kit included in their range of targets.  Furthermore, loss-of-function mutations in c-Kit have been obsd. and shown to give rise to a condition called piebaldism.  This review provides a summary of our current knowledge regarding structural and functional aspects of c-Kit signaling both under normal and pathol. conditions, as well as advances in the development of low-mol.-wt. mols. inhibiting c-Kit function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiAi_V1Wivd7Vg90H21EOLACvtfcHk0lhhd2_x5qw2fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2lsbnK&md5=b874bfb881938c505b6718732950be51</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00046.2011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00046.2011%26sid%3Dliteratum%253Aachs%26aulast%3DLennartsson%26aufirst%3DJ.%26aulast%3DR%25C3%25B6nnstrand%26aufirst%3DL.%26atitle%3DStem%2520Cell%2520Factor%2520Receptor%252Fc-Kit%253A%2520From%2520Basic%2520Science%2520to%2520Clinical%2520Implications%26jtitle%3DPhysiol.%2520Rev.%26date%3D2012%26volume%3D92%26issue%3D4%26spage%3D1619%26epage%3D1649%26doi%3D10.1152%2Fphysrev.00046.2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abbaspour
Babaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamalidehghan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleem, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman Huri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmadipour, F.</span></span> <span> </span><span class="NLM_article-title">Receptor Tyrosine Kinase (c-Kit) Inhibitors: A Potential Therapeutic Target in Cancer Cells</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2443</span>– <span class="NLM_lpage">2459</span>, <span class="refDoi"> DOI: 10.2147/DDDT.S89114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.2147%2FDDDT.S89114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=27536065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A280%3ADC%252BC2szhvV2guw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2016&pages=2443-2459&author=M.+Abbaspour%0ABabaeiauthor=B.+Kamalidehghanauthor=M.+Saleemauthor=H.+Zaman+Huriauthor=F.+Ahmadipour&title=Receptor+Tyrosine+Kinase+%28c-Kit%29+Inhibitors%3A+A+Potential+Therapeutic+Target+in+Cancer+Cells&doi=10.2147%2FDDDT.S89114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor tyrosine kinase (c-Kit) inhibitors: a potential therapeutic target in cancer cells</span></div><div class="casAuthors">Abbaspour Babaei Maryam; Ahmadipour Fatemeh; Kamalidehghan Behnam; Saleem Mohammad; Huri Hasniza Zaman</div><div class="citationInfo"><span class="NLM_cas:title">Drug design, development and therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2443-59</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">c-Kit, a receptor tyrosine kinase, is involved in intracellular signaling, and the mutated form of c-Kit plays a crucial role in occurrence of some cancers.  The function of c-Kit has led to the concept that inhibiting c-Kit kinase activity can be a target for cancer therapy.  The promising results of inhibition of c-Kit for treatment of cancers have been observed in some cancers such as gastrointestinal stromal tumor, acute myeloid leukemia, melanoma, and other tumors, and these results have encouraged attempts toward improvement of using c-Kit as a capable target for cancer therapy.  This paper presents the findings of previous studies regarding c-Kit as a receptor tyrosine kinase and an oncogene, as well as its gene targets and signaling pathways in normal and cancer cells.  The c-Kit gene location, protein structure, and the role of c-Kit in normal cell have been discussed.  Comprehending the molecular mechanism underlying c-Kit-mediated tumorogenesis is consequently essential and may lead to the identification of future novel drug targets.  The potential mechanisms by which c-Kit induces cellular transformation have been described.  This study aims to elucidate the function of c-Kit for future cancer therapy.  In addition, it has c-Kit inhibitor drug properties and their functions have been listed in tables and demonstrated in schematic pictures.  This review also has collected previous studies that targeted c-Kit as a novel strategy for cancer therapy.  This paper further emphasizes the advantages of this approach, as well as the limitations that must be addressed in the future.  Finally, although c-Kit is an attractive target for cancer therapy, based on the outcomes of treatment of patients with c-Kit inhibitors, it is unlikely that Kit inhibitors alone can lead to cure.  It seems that c-Kit mutations alone are not sufficient for tumorogenesis, but do play a crucial role in cancer occurrence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfTnK2L7idhRwLZxei1HN8fW6udTcc2ebO-cDqTmHKKLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szhvV2guw%253D%253D&md5=c45d232471e049dc8d42a98fc5aa1d76</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S89114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S89114%26sid%3Dliteratum%253Aachs%26aulast%3DAbbaspour%2BBabaei%26aufirst%3DM.%26aulast%3DKamalidehghan%26aufirst%3DB.%26aulast%3DSaleem%26aufirst%3DM.%26aulast%3DZaman%2BHuri%26aufirst%3DH.%26aulast%3DAhmadipour%26aufirst%3DF.%26atitle%3DReceptor%2520Tyrosine%2520Kinase%2520%2528c-Kit%2529%2520Inhibitors%253A%2520A%2520Potential%2520Therapeutic%2520Target%2520in%2520Cancer%2520Cells%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2016%26volume%3D10%26spage%3D2443%26epage%3D2459%26doi%3D10.2147%2FDDDT.S89114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garner, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozgit, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vodala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrock, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serrano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span> <span> </span><span class="NLM_article-title">Ponatinib Inhibits Polyclonal Drug-Resistant KIT Oncoproteins and Shows Therapeutic Potential in Heavily Pretreated Gastrointestinal Stromal Tumor (GIST) Patients</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">5745</span>– <span class="NLM_lpage">5755</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-1397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1158%2F1078-0432.CCR-14-1397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=25239608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGgsL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=5745-5755&issue=22&author=A.+P.+Garnerauthor=J.+M.+Gozgitauthor=R.+Anjumauthor=S.+Vodalaauthor=A.+Schrockauthor=T.+Zhouauthor=C.+Serranoauthor=G.+Eilersauthor=M.+Zhuauthor=J.+Ketzerauthor=S.+Wardwellauthor=Y.+Ningauthor=Y.+Songauthor=A.+Kohlmannauthor=F.+Wangauthor=T.+Clacksonauthor=M.+C.+Heinrichauthor=J.+A.+Fletcherauthor=S.+Bauerauthor=V.+M.+Rivera&title=Ponatinib+Inhibits+Polyclonal+Drug-Resistant+KIT+Oncoproteins+and+Shows+Therapeutic+Potential+in+Heavily+Pretreated+Gastrointestinal+Stromal+Tumor+%28GIST%29+Patients&doi=10.1158%2F1078-0432.CCR-14-1397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Ponatinib Inhibits Polyclonal Drug-Resistant KIT Oncoproteins and Shows Therapeutic Potential in Heavily Pretreated Gastrointestinal Stromal Tumor (GIST) Patients</span></div><div class="casAuthors">Garner, Andrew P.; Gozgit, Joseph M.; Anjum, Rana; Vodala, Sadanand; Schrock, Alexa; Zhou, Tianjun; Serrano, Cesar; Eilers, Grant; Zhu, Meijun; Ketzer, Julia; Wardwell, Scott; Ning, Yaoyu; Song, Youngchul; Kohlmann, Anna; Wang, Frank; Clackson, Tim; Heinrich, Michael C.; Fletcher, Jonathan A.; Bauer, Sebastian; Rivera, Victor M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5745-5755</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: KIT is the major oncogenic driver of gastrointestinal stromal tumors (GIST).  Imatinib, sunitinib, and regorafenib are approved therapies; however, efficacy is often limited by the acquisition of polyclonal secondary resistance mutations in KIT, with those located in the activation (A) loop (exons 17/18) being particularly problematic.  Here, we explore the KIT-inhibitory activity of ponatinib in preclin. models and describe initial characterization of its activity in patients with GIST.  Exptl. Design: The cellular and in vivo activities of ponatinib, imatinib, sunitinib, and regorafenib against mutant KIT were evaluated using an accelerated mutagenesis assay and a panel of engineered and GIST-derived cell lines.  The ponatinib-KIT costructure was also detd.  The clin. activity of ponatinib was examd. in three patients with GIST previously treated with all three FDA-approved agents.  Results: In engineered and GIST-derived cell lines, ponatinib potently inhibited KIT exon 11 primary mutants and a range of secondary mutants, including those within the A-loop.  Ponatinib also induced regression in engineered and GIST-derived tumor models contg. these secondary mutations.  In a mutagenesis screen, 40 nmol/L ponatinib was sufficient to suppress outgrowth of all secondary mutants except V654A, which was suppressed at 80 nmol/L.  This inhibitory profile could be rationalized on the basis of structural analyses.  Ponatinib (30 mg daily) displayed encouraging clin. activity in two of three patients with GIST.  Conclusion:Ponatinib possesses potent activity against most major clin. relevant KIT mutants and has demonstrated preliminary evidence of activity in patients with refractory GIST.  These data strongly support further evaluation of ponatinib in patients with GIST.  Clin Cancer Res; 20(22); 5745-55. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTGG18CSl3XLVg90H21EOLACvtfcHk0lgd4xDQsYYezA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGgsL7E&md5=7537e45f37546f9e0cb799bdf05ed847</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1397%26sid%3Dliteratum%253Aachs%26aulast%3DGarner%26aufirst%3DA.%2BP.%26aulast%3DGozgit%26aufirst%3DJ.%2BM.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DVodala%26aufirst%3DS.%26aulast%3DSchrock%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DSerrano%26aufirst%3DC.%26aulast%3DEilers%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DM.%26aulast%3DKetzer%26aufirst%3DJ.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26aulast%3DFletcher%26aufirst%3DJ.%2BA.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26atitle%3DPonatinib%2520Inhibits%2520Polyclonal%2520Drug-Resistant%2520KIT%2520Oncoproteins%2520and%2520Shows%2520Therapeutic%2520Potential%2520in%2520Heavily%2520Pretreated%2520Gastrointestinal%2520Stromal%2520Tumor%2520%2528GIST%2529%2520Patients%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26issue%3D22%26spage%3D5745%26epage%3D5755%26doi%3D10.1158%2F1078-0432.CCR-14-1397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DiNitto, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worrall, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">English, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. C.</span></span> <span> </span><span class="NLM_article-title">Function of Activation Loop Tyrosine Phosphorylation in the Mechanism of c-Kit Auto-activation and its Implication in Sunitinib Resistance</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>147</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">601</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1093/jb/mvq015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fjb%2Fmvq015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=20147452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkt1Oqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2010&pages=601-609&issue=4&author=J.+P.+DiNittoauthor=G.+D.+Deshmukhauthor=Y.+Zhangauthor=S.+L.+Jacquesauthor=R.+Coliauthor=J.+W.+Worrallauthor=W.+Diehlauthor=J.+M.+Englishauthor=J.+C.+Wu&title=Function+of+Activation+Loop+Tyrosine+Phosphorylation+in+the+Mechanism+of+c-Kit+Auto-activation+and+its+Implication+in+Sunitinib+Resistance&doi=10.1093%2Fjb%2Fmvq015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Function of activation loop tyrosine phosphorylation in the mechanism of c-Kit auto-activation and its implication in sunitinib resistance</span></div><div class="casAuthors">DiNitto, Jonathan P.; Deshmukh, Gayatri D.; Zhang, Yan; Jacques, Suzanne L.; Coli, Rocco; Worrall, Joseph W.; Diehl, Wade; English, Jessie M.; Wu, Joe C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">601-609</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">The activation of receptor tyrosine kinases (RTKs) is tightly regulated through a variety of mechanisms.  Kinetic studies show that activation of c-Kit RTK occurs through an inter-mol. autophosphorylation.  Phosphopeptide mapping of c-Kit reveals that 14-22 phosphates are added to each mol of wild-type (WT) c-Kit during the activation.  Phosphorylation sites are found on the JM, kinase insert (KID), c-terminal domains and the activation loop (A-loop), but only the sites on the JM domain contribute to the kinase activation.  The A-loop tyrosine (Y823) is not phosphorylated until very late in the activation (>90% completion), indicating that the A-loop phosphorylation is not required for c-Kit activation.  A sunitinib-resistant mutant D816H that accelerates auto-activation by 184-fold shows no phosphorylation on the A-loop tyrosine after full activation.  A loss-of-phosphorylation mutation Y823F remains fully competent in auto-activation.  Similar to WT and D816H, the unactivated Y823F mutant binds sunitinib and imatinib with high affinity (KD = 5.9 nM).  But unlike the WT and D816H where the activated enzymes lose the ability to bind the two drugs, activated Y823F binds the two inhibitors effectively.  These observations suggest that the A-loop of activated Y823F remains flexible and can readily adopt unactivated conformations to accommodate DFG-out binders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLl2NrvMAaC7Vg90H21EOLACvtfcHk0lgd4xDQsYYezA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkt1Oqurw%253D&md5=f3bcc8c20cbba329d295196983a19468</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvq015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvq015%26sid%3Dliteratum%253Aachs%26aulast%3DDiNitto%26aufirst%3DJ.%2BP.%26aulast%3DDeshmukh%26aufirst%3DG.%2BD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DColi%26aufirst%3DR.%26aulast%3DWorrall%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DW.%26aulast%3DEnglish%26aufirst%3DJ.%2BM.%26aulast%3DWu%26aufirst%3DJ.%2BC.%26atitle%3DFunction%2520of%2520Activation%2520Loop%2520Tyrosine%2520Phosphorylation%2520in%2520the%2520Mechanism%2520of%2520c-Kit%2520Auto-activation%2520and%2520its%2520Implication%2520in%2520Sunitinib%2520Resistance%26jtitle%3DJ.%2520Biochem.%26date%3D2010%26volume%3D147%26issue%3D4%26spage%3D601%26epage%3D609%26doi%3D10.1093%2Fjb%2Fmvq015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, X.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanism of D816X Mutation-Induced c-Kit Activation and Mediated Inhibitor Resistance in Gastrointestinal Stromal Tumor</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2018.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.jmgm.2018.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=30015051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlensbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2018&pages=189-196&author=H.+Jiangauthor=W.+Shaoauthor=Y.+Wangauthor=R.+Xuauthor=L.+Zhouauthor=X.+Mu&title=Molecular+Mechanism+of+D816X+Mutation-Induced+c-Kit+Activation+and+Mediated+Inhibitor+Resistance+in+Gastrointestinal+Stromal+Tumor&doi=10.1016%2Fj.jmgm.2018.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanism of D816X mutation-induced c-Kit activation and -mediated inhibitor resistance in gastrointestinal stromal tumor</span></div><div class="casAuthors">Jiang, Haohai; Shao, Weiwei; Wang, Yuanjin; Xu, Ronghui; Zhou, Linsen; Mu, Xiangming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">189-196</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The D816X (X = V, H, Y or F) missense mutation constitutively activates c-Kit kinase in gastrointestinal stromal tumor (GIST) and has been obsd. to cause acquired resistance against first-line and second-line kinase inhibitors.  In the present study, the allosteric mechanism of D816X-induced c-Kit conformational change is investigated at mol. level.  The Asp816 residue is located at the activation loop (A-loop) of c-Kit and the mutation can eliminate a neg. formal charge from the loop region by substituting the acidic asparagic acid residue with neutral valine, histidine, tyrosine or phenylalanine.  Here, we classify the c-Kit kinase into four states in terms of its mutation (wild type or mutant) and conformation (DFG-in or DFG-out).  The wild-type kinase is electrostatically stabilized in inactive DFG-out conformation, whereas the D816X mutation can promote the conformational conversion to active DFG-in and then activate the kinase.  Structural anal. reveals that the Asp816 residue in DFG-out is surrounded by a no. of polar and pos. charged residues within its first and second shells of protein context, and kinase conformational change to DFG-in brings this residue into a neg. electrostatic potential environment.  Dynamics simulation characterizes that the c-Kit conformational conversion from DFG-out to DFG-in can cause local unfavorable effect to type-II inhibitor, while the mutation-induced global structural rearrangement would participate in the favorable interaction of c-Kit with type-I inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc7O_8k3wISLVg90H21EOLACvtfcHk0lgILgR4hMivpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlensbvI&md5=c76cac50df01c3a25d0939955f75dc14</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2018.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2018.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DMu%26aufirst%3DX.%26atitle%3DMolecular%2520Mechanism%2520of%2520D816X%2520Mutation-Induced%2520c-Kit%2520Activation%2520and%2520Mediated%2520Inhibitor%2520Resistance%2520in%2520Gastrointestinal%2520Stromal%2520Tumor%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2018%26volume%3D84%26spage%3D189%26epage%3D196%26doi%3D10.1016%2Fj.jmgm.2018.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klug, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. C.</span></span> <span> </span><span class="NLM_article-title">Structural and Clinical Consequences of Activation Loop Mutations in Class III Receptor Tyrosine Kinases</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2018.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.pharmthera.2018.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29964125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1CjtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2018&pages=123-134&author=L.+R.+Klugauthor=J.+D.+Kentauthor=M.+C.+Heinrich&title=Structural+and+Clinical+Consequences+of+Activation+Loop+Mutations+in+Class+III+Receptor+Tyrosine+Kinases&doi=10.1016%2Fj.pharmthera.2018.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and clinical consequences of activation loop mutations in class III receptor tyrosine kinases</span></div><div class="casAuthors">Klug, Lillian R.; Kent, Jason D.; Heinrich, Michael C.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">123-134</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Mutations within the activation loop of members of the class III receptor tyrosine kinase (RTK) subfamily, which includes KIT, PDGFRA, and FLT3, have been obsd. in multiple human tumor types.  These mutations confer constitutive activation as well as resistance to the type II tyrosine kinase inhibitors (TKI) that are currently clin. available, such as imatinib and sunitinib.  It is now understood that activation loop mutations in class III RTKs shift the activation state equil. away from inactive states, to which type II TKIs bind, to the active state by destabilizing the inactive conformation.  Recently, type I TKIs, which can bind to active kinase conformations, have been developed with specificity for class III RTK members.  Some type I TKIs, such as crenolanib and avapritinib (BLU-285), have entered clin. studies for patients with activation loop mutations in KIT, PDGFRA, or FLT3.  Preliminary results suggest that these type I TKIs show activity in these patient populations that previously lacked effective treatments.  This article reviews the inactive and active structures of KIT, PDGFRA, and FLT3, how the mutations seen in human cancers affect kinase structure, and the clin. implications of these mutations in terms of type I vs. type II TKI binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRH_W_T1s9lLVg90H21EOLACvtfcHk0lgILgR4hMivpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1CjtL7F&md5=168ec705a8e0c19c3191255739beb483</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2018.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2018.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DKlug%26aufirst%3DL.%2BR.%26aulast%3DKent%26aufirst%3DJ.%2BD.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26atitle%3DStructural%2520and%2520Clinical%2520Consequences%2520of%2520Activation%2520Loop%2520Mutations%2520in%2520Class%2520III%2520Receptor%2520Tyrosine%2520Kinases%26jtitle%3DPharmacol.%2520Ther.%26date%3D2018%26volume%3D191%26spage%3D123%26epage%3D134%26doi%3D10.1016%2Fj.pharmthera.2018.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikemori-Kawada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jestel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">König, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funahashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuruoka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, J.</span></span> <span> </span><span class="NLM_article-title">Distinct Binding Mode of Multikinase Inhibitor Lenvatinib Revealed by Biochemical Characterization</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1021/ml500394m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500394m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=89-94&issue=1&author=K.+Okamotoauthor=M.+Ikemori-Kawadaauthor=A.+Jestelauthor=K.+V.+K%C3%B6nigauthor=Y.+Funahashiauthor=T.+Matsushimaauthor=A.+Tsuruokaauthor=A.+Inoueauthor=J.+Matsui&title=Distinct+Binding+Mode+of+Multikinase+Inhibitor+Lenvatinib+Revealed+by+Biochemical+Characterization&doi=10.1021%2Fml500394m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct Binding Mode of Multikinase Inhibitor Lenvatinib Revealed by Biochemical Characterization</span></div><div class="casAuthors">Okamoto, Kiyoshi; Ikemori-Kawada, Megumi; Jestel, Anja; von Konig, Konstanze; Funahashi, Yasuhiro; Matsushima, Tomohiro; Tsuruoka, Akihiko; Inoue, Atsushi; Matsui, Junji</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-94</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lenvatinib is an oral multikinase inhibitor that selectively inhibits vascular endothelial growth factor (VEGF) receptors 1 to 3 and other proangiogenic and oncogenic pathway-related receptor tyrosine kinases.  To elucidate the origin of the potency of lenvatinib in VEGF receptor 2 (VEGFR2) inhibition, we conducted a kinetic interaction anal. of lenvatinib with VEGFR2 and x-ray anal. of the crystal structure of VEGFR2-lenvatinib complexes.  Kinetic anal. revealed that lenvatinib had a rapid assocn. rate const. and a relatively slow dissocn. rate const. in complex with VEGFR2.  Co-crystal structure anal. demonstrated that lenvatinib binds at its ATP mimetic quinoline moiety to the ATP binding site and to the neighboring region via a cyclopropane ring, adopting an Asp-Phe-Gly (DFG)-"in" conformation.  These results suggest that lenvatinib is very distinct in its binding mode of interaction compared to the several approved VEGFR2 kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUysiHRGw3P7Vg90H21EOLACvtfcHk0lgILgR4hMivpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtb7E&md5=5461832fad0577f2ea93b47521b4bbdc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fml500394m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500394m%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DIkemori-Kawada%26aufirst%3DM.%26aulast%3DJestel%26aufirst%3DA.%26aulast%3DK%25C3%25B6nig%26aufirst%3DK.%2BV.%26aulast%3DFunahashi%26aufirst%3DY.%26aulast%3DMatsushima%26aufirst%3DT.%26aulast%3DTsuruoka%26aufirst%3DA.%26aulast%3DInoue%26aufirst%3DA.%26aulast%3DMatsui%26aufirst%3DJ.%26atitle%3DDistinct%2520Binding%2520Mode%2520of%2520Multikinase%2520Inhibitor%2520Lenvatinib%2520Revealed%2520by%2520Biochemical%2520Characterization%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26issue%3D1%26spage%3D89%26epage%3D94%26doi%3D10.1021%2Fml500394m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotlib, J.</span></span> <span> </span><span class="NLM_article-title">Clinical Validation of KIT Inhibition in Advanced Systemic Mastocytosis</span>. <i>Current Hematologic Malignancy Reports</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">416</span>, <span class="refDoi"> DOI: 10.1007/s11899-018-0469-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1007%2Fs11899-018-0469-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=30155614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ivVKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=407-416&issue=5&author=J.+H.+Bairdauthor=J.+Gotlib&title=Clinical+Validation+of+KIT+Inhibition+in+Advanced+Systemic+Mastocytosis&doi=10.1007%2Fs11899-018-0469-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Validation of KIT Inhibition in Advanced Systemic Mastocytosis</span></div><div class="casAuthors">Baird John H; Gotlib Jason</div><div class="citationInfo"><span class="NLM_cas:title">Current hematologic malignancy reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">407-416</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PURPOSE OF REVIEW:  We discuss recent developments in the treatment of advanced systemic mastocytosis (advSM) with inhibitors of the KIT receptor tyrosine kinase.  RECENT FINDINGS:  advSM is a heterogeneous group of neoplasms of poor prognosis characterized by the accumulation of neoplastic mast cells.  The canonical KIT D816V mutation is present in approximately 90% of SM patients, and its detection is critical for both diagnosis and therapeutic decision-making.  The multikinase/KIT inhibitor midostaurin was recently approved for advSM.  This agent can reverse SM-related organ damage and disease symptoms, and decrease the bone marrow mast cell burden and splenomegaly.  However, complete remissions are rare and durability of responses is variable.  Potent and selective KIT D816V inhibitors including avapritinib (BLU-285) and DCC-2618 have entered clinical trials, and rational combination strategies are under development.  The clinical efficacy of KIT inhibitors validate KIT as a key oncogenic driver in mast cell neoplasms.  An improved understanding of the genetic heterogeneity beyond KIT will help inform the dynamics of response and relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRASjErjCIGasSzjBFCN0BJfW6udTcc2eawn5dVGYLvwbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ivVKlsQ%253D%253D&md5=f0854dc0fa929a50cdfa3fb54907e368</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs11899-018-0469-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11899-018-0469-3%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DJ.%2BH.%26aulast%3DGotlib%26aufirst%3DJ.%26atitle%3DClinical%2520Validation%2520of%2520KIT%2520Inhibition%2520in%2520Advanced%2520Systemic%2520Mastocytosis%26jtitle%3DCurrent%2520Hematologic%2520Malignancy%2520Reports%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3D407%26epage%3D416%26doi%3D10.1007%2Fs11899-018-0469-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schittenhelm, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiraga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbin, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bokemeyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druker, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. C.</span></span> <span> </span><span class="NLM_article-title">Dasatinib (BMS-354825), a Dual SRC/ABL Kinase Inhibitor, Inhibits the Kinase Activity of Wild-Type, Juxtamembrane, and Activation Loop Mutant KIT Isoforms Associated with Human Malignancies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-05-2050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1158%2F0008-5472.CAN-05-2050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=16397263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFWltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=473-481&issue=1&author=M.+M.+Schittenhelmauthor=S.+Shiragaauthor=A.+Schroederauthor=A.+S.+Corbinauthor=D.+Griffithauthor=F.+Y.+Leeauthor=C.+Bokemeyerauthor=M.+W.+N.+Deiningerauthor=B.+J.+Drukerauthor=M.+C.+Heinrich&title=Dasatinib+%28BMS-354825%29%2C+a+Dual+SRC%2FABL+Kinase+Inhibitor%2C+Inhibits+the+Kinase+Activity+of+Wild-Type%2C+Juxtamembrane%2C+and+Activation+Loop+Mutant+KIT+Isoforms+Associated+with+Human+Malignancies&doi=10.1158%2F0008-5472.CAN-05-2050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Dasatinib (BMS-354825), a Dual SRC/ABL Kinase Inhibitor, Inhibits the Kinase Activity of Wild-Type, Juxtamembrane, and Activation Loop Mutant KIT Isoforms Associated with Human Malignancies</span></div><div class="casAuthors">Schittenhelm, Marcus M.; Shiraga, Sharon; Schroeder, Arin; Corbin, Amie S.; Griffith, Diana; Lee, Francis Y.; Bokemeyer, Carsten; Deininger, Michael W. N.; Druker, Brian J.; Heinrich, Michael C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Activating mutations of the activation loop of KIT are assocd. with certain human neoplasms, including the majority of patients with systemic mast cell disorders, as well as cases of seminoma, acute myelogenous leukemia (AML), and gastrointestinal stromal tumors (GISTs).  The small-mol. tyrosine kinase inhibitor imatinib mesylate is a potent inhibitor of wild-type (WT) KIT and certain mutant KIT isoforms and has become the std. of care for treating patients with metastatic GIST.  However, KIT activation loop mutations involving codon D816 that are typically found in AML, systemic mastocytosis, and seminoma are insensitive to imatinib mesylate (IC50 > 5-10 μmol/L), and acquired KIT activation loop mutations can be assocd. with imatinib mesylate resistance in GIST.  Dasatinib (formerly BMS-354825) is a small-mol., ATP-competitive inhibitor of SRC and ABL tyrosine kinases with potency in the low nanomolar range.  Some small-mol. SRC/ABL inhibitors also have potency against WT KIT kinase.  Therefore, the authors hypothesized that dasatinib might inhibit the kinase activity of both WT and mutant KIT isoforms.  The authors report herein that dasatinib potently inhibits WT KIT and juxtamembrane domain mutant KIT autophosphorylation and KIT-dependent activation of downstream pathways important for cell viability and cell survival, such as Ras/mitogen-activated protein kinase, phosphoinositide 3-kinase/Akt, and Janus-activated kinase/signal transducers and activators of transcription.  Furthermore, dasatinib is a potent inhibitor of imatinib-resistant KIT activation loop mutants and induces apoptosis in mast cell and leukemic cell lines expressing these mutations (potency against KIT D816Y » D816F > D816V).  Our studies suggest that dasatinib may have clin. efficacy against human neoplasms that are assocd. with gain-of-function KIT mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryNXWvlK-abLVg90H21EOLACvtfcHk0lhGAP4shUilBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFWltw%253D%253D&md5=dbd2d8af79e727c22197f7a273b45b41</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-2050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-2050%26sid%3Dliteratum%253Aachs%26aulast%3DSchittenhelm%26aufirst%3DM.%2BM.%26aulast%3DShiraga%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DA.%26aulast%3DCorbin%26aufirst%3DA.%2BS.%26aulast%3DGriffith%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DF.%2BY.%26aulast%3DBokemeyer%26aufirst%3DC.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%2BN.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26atitle%3DDasatinib%2520%2528BMS-354825%2529%252C%2520a%2520Dual%2520SRC%252FABL%2520Kinase%2520Inhibitor%252C%2520Inhibits%2520the%2520Kinase%2520Activity%2520of%2520Wild-Type%252C%2520Juxtamembrane%252C%2520and%2520Activation%2520Loop%2520Mutant%2520KIT%2520Isoforms%2520Associated%2520with%2520Human%2520Malignancies%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26issue%3D1%26spage%3D473%26epage%3D481%26doi%3D10.1158%2F0008-5472.CAN-05-2050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span> <span> </span><span class="NLM_article-title">Development and Biological Evaluation of Potent and Selective c-KITD816V Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6428</span>– <span class="NLM_lpage">6443</span>, <span class="refDoi"> DOI: 10.1021/jm500413g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500413g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFajsbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6428-6443&author=S.+Leeauthor=H.+Leeauthor=J.+Kimauthor=S.+Leeauthor=S.+J.+Kimauthor=B.+S.+Choiauthor=S.+Hongauthor=S.+Hong&title=Development+and+Biological+Evaluation+of+Potent+and+Selective+c-KITD816V+Inhibitors&doi=10.1021%2Fjm500413g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Biological Evaluation of Potent and Selective c-KITD816V Inhibitors</span></div><div class="casAuthors">Lee, Soyoung; Lee, Hyunseung; Kim, Jinhee; Lee, Suhyun; Kim, Soo Jung; Choi, Byong-Seok; Hong, Soon-Sun; Hong, Sungwoo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6428-6443</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The c-KIT tyrosine kinase has emerged as a potential therapeutic target for an array of diseases.  However, there exists a drug resistance that is caused by mutations in c-KIT; therefore, c-KIT remains as a clin. challenge due to limited effective treatment options for therapies.  For example, the acquired activating point mutation D816V significantly impairs the efficacy of targeted cancer therapies.  Understanding the mechanisms of drug resistance at the mol. level will aid in designing and developing particular inhibitors with the potential to overcome these resistance mutations.  The authors undertake a structure-based de novo design of 7-azaindole as the mol. core using the modified scoring function.  This approach led to an identification of new c-KIT inhibitors over 100-fold specific for the D816V mutant relative to the wild-type c-KIT with nanomolar inhibitory activity.  More importantly, these compds. potently inhibit clin. relevant D816V mutations of c-KIT in biochem. and cellular studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbp-ykbRTq7LVg90H21EOLACvtfcHk0lhGAP4shUilBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFajsbjK&md5=5b368e9fa0eb3c9d5ad5a69439347c81</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm500413g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500413g%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DChoi%26aufirst%3DB.%2BS.%26aulast%3DHong%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520Biological%2520Evaluation%2520of%2520Potent%2520and%2520Selective%2520c-KITD816V%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6428%26epage%3D6443%26doi%3D10.1021%2Fjm500413g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavrin, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiah, E.</span></span> <span> </span><span class="NLM_article-title">Approaches to Discover Non-ATP Site Kinase Inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1039/C2MD20180A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1039%2FC2MD20180A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=41-51&issue=1&author=L.+K.+Gavrinauthor=E.+Saiah&title=Approaches+to+Discover+Non-ATP+Site+Kinase+Inhibitors&doi=10.1039%2FC2MD20180A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to discover non-ATP site kinase inhibitors</span></div><div class="casAuthors">Gavrin, Lori Krim; Saiah, Eddine</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-51</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The catalytic domain of kinases shows a high degree of sequence homol., esp. for kinases that belong to the same family.  They share a common ATP binding site with a conserved activation loop and similar three-dimensional structure.  Consequently, a major challenge in kinase research exists in achieving selectivity among the >500 family members, since they all process the same substrate.  In addn. to requiring selectivity against other kinases, ATP site inhibitors must also bind tightly to overcome the high physiol. concn. of ATP in the cell.  Furthermore, the development of novel ATP site inhibitors is becoming increasingly challenging, as many ATP competitive scaffolds have previously been disclosed.  In order to develop compds. with better selectivity among kinases, inhibitors that bind outside the ATP site show great promise and are currently being explored by many groups.  This review will highlight the most commonly used methods to discover small mol. Type III and IV kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xJcf4-RJ9rVg90H21EOLACvtfcHk0liYHD4paRS95g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N&md5=97acf2247e5516c1649c1d964c867656</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC2MD20180A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20180A%26sid%3Dliteratum%253Aachs%26aulast%3DGavrin%26aufirst%3DL.%2BK.%26aulast%3DSaiah%26aufirst%3DE.%26atitle%3DApproaches%2520to%2520Discover%2520Non-ATP%2520Site%2520Kinase%2520Inhibitors%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26issue%3D1%26spage%3D41%26epage%3D51%26doi%3D10.1039%2FC2MD20180A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span> <span> </span><span class="NLM_article-title">Nucleic Acid Aptamers: An Emerging Frontier in Cancer Therapy</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">85</span>),  <span class="NLM_fpage">10472</span>, <span class="refDoi"> DOI: 10.1039/c2cc35042d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1039%2Fc2cc35042d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22951893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCqtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=10472&issue=85&author=G.+Zhuauthor=M.+Yeauthor=M.+J.+Donovanauthor=E.+Songauthor=Z.+Zhaoauthor=W.+Tan&title=Nucleic+Acid+Aptamers%3A+An+Emerging+Frontier+in+Cancer+Therapy&doi=10.1039%2Fc2cc35042d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleic acid aptamers: an emerging frontier in cancer therapy</span></div><div class="casAuthors">Zhu, Guizhi; Ye, Mao; Donovan, Michael J.; Song, Erqun; Zhao, Zilong; Tan, Weihong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">85</span>),
    <span class="NLM_cas:pages">10472-10480</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The last 2 decades have witnessed the development and application of nucleic acid aptamers in a variety of fields, including target anal., disease therapy, and mol. and cellular engineering.  The efficient and widely applicable aptamer selection, reproducible chem. synthesis and modification, generally impressive target binding selectivity and affinity, relatively rapid tissue penetration, low immunogenicity, and rapid systemic clearance make aptamers ideal recognition elements for use as therapeutics or for in vivo delivery of therapeutics.  In this feature article, the authors discuss the development and biomedical application of nucleic acid aptamers, with emphasis on cancer cell aptamer isolation, targeted cancer therapy, oncol. biomarker identification and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM2VW-GB3T7LVg90H21EOLACvtfcHk0liYHD4paRS95g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCqtLfO&md5=f4d934441bc5a9f75ca21eb30fa53e12</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2Fc2cc35042d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2cc35042d%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DDonovan%26aufirst%3DM.%2BJ.%26aulast%3DSong%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DNucleic%2520Acid%2520Aptamers%253A%2520An%2520Emerging%2520Frontier%2520in%2520Cancer%2520Therapy%26jtitle%3DChem.%2520Commun.%26date%3D2012%26volume%3D48%26issue%3D85%26spage%3D10472%26doi%3D10.1039%2Fc2cc35042d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clary, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigner, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullenger, B. A.</span></span> <span> </span><span class="NLM_article-title">Aptamers Selected Against the Unglycosylated EGFRvIII Ectodomain and Delivered Intracellularly Reduce Membrane-bound EGFRvIII and Induce Apoptosis</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1515/BC.2009.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1515%2FBC.2009.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=19040357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVajsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=137-144&issue=2&author=Y.+Liuauthor=C.+Kuanauthor=J.+Miauthor=X.+Zhangauthor=B.+M.+Claryauthor=D.+D.+Bignerauthor=B.+A.+Sullenger&title=Aptamers+Selected+Against+the+Unglycosylated+EGFRvIII+Ectodomain+and+Delivered+Intracellularly+Reduce+Membrane-bound+EGFRvIII+and+Induce+Apoptosis&doi=10.1515%2FBC.2009.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamers selected against the unglycosylated EGFRvIII ectodomain and delivered intracellularly reduce membrane-bound EGFRvIII and induce apoptosis</span></div><div class="casAuthors">Liu, Yingmiao; Kuan, Chien-Tsun; Mi, Jing; Zhang, Xiuwu; Clary, Bryan M.; Bigner, Darell D.; Sullenger, Bruce A.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">137-144</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor variant III (EGFRvIII) is a glycoprotein uniquely expressed in glioblastoma, but not in normal brain tissues.  To develop targeted therapies for brain tumors, the authors selected RNA aptamers against the histidine-tagged EGFRvIII ectodomain, using an Escherichia coli system for protein expression and purifn.  Representative aptamer E21 has a dissocn. const. (Kd) of 33 × 10-9 M, and exhibits high affinity and specificity for EGFRvIII in ELISA and surface plasmon resonance assays.  However, selected aptamers cannot bind the same protein expressed from eukaryotic cells because glycosylation, a post-translational modification present only in eukaryotic systems, significantly alters the structure of the target protein.  By transfecting EGFRvIII aptamers into cells, it was found that membrane-bound, glycosylated EGFRvIII is reduced and the percentage of cells undergoing apoptosis is increased.  It is postulated that transfected aptamers can interact with newly synthesized EGFRvIII, disrupt proper glycosylation, and reduce the amt. of mature EGFRvIII reaching the cell surface.  This work establishes the feasibility of disrupting protein post-translational modifications in situ with aptamers.  This finding is useful for elucidating the function of proteins of interest with various modifications, as well as dissecting signal transduction pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVzTIzjghlJrVg90H21EOLACvtfcHk0lioe9aCvdQzig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVajsrc%253D&md5=f0c266ecc883b4b2e1bfa7cd228ace6a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1515%2FBC.2009.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2009.022%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKuan%26aufirst%3DC.%26aulast%3DMi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DClary%26aufirst%3DB.%2BM.%26aulast%3DBigner%26aufirst%3DD.%2BD.%26aulast%3DSullenger%26aufirst%3DB.%2BA.%26atitle%3DAptamers%2520Selected%2520Against%2520the%2520Unglycosylated%2520EGFRvIII%2520Ectodomain%2520and%2520Delivered%2520Intracellularly%2520Reduce%2520Membrane-bound%2520EGFRvIII%2520and%2520Induce%2520Apoptosis%26jtitle%3DBiol.%2520Chem.%26date%3D2009%26volume%3D390%26issue%3D2%26spage%3D137%26epage%3D144%26doi%3D10.1515%2FBC.2009.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keranen, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellington, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. C.</span></span> <span> </span><span class="NLM_article-title">Isozyme-specific Inhibition of Protein Kinase C by RNA Aptamer</span>. <i>The Journal of Biological Chemistry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>269</i></span> (<span class="NLM_issue">51</span>),  <span class="NLM_fpage">32051</span>– <span class="NLM_lpage">32054</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=7528207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2MXitFSlu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=32051-32054&issue=51&author=R.+Conradauthor=L.+M.+Keranenauthor=A.+D.+Ellingtonauthor=A.+C.+Newton&title=Isozyme-specific+Inhibition+of+Protein+Kinase+C+by+RNA+Aptamer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Isoenzyme-specific inhibition of protein kinase C by RNA aptamers</span></div><div class="casAuthors">Conrad, Rick; Keranen, Lisa M.; Ellington, Andrew D.; Newton, Alexandra C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">32051-4</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In vitro selection technol. has been used to purify RNA aptamers from a random sequence pool that can bind to, and specifically inhibit, protein kinase C βII.  Two of the selected RNA aptamers bind to this isoenzyme of protein kinase C with nanomolar affinities and inhibit activation with unprecedented selectivity; the highly related, alternatively spliced βI isoenzyme, which differs by 23 residues, is inhibited with 1 order of magnitude lower potency; the next most similar isoenzyme, α, shows no detectable inhibition.  The prodn. of isoenzyme-specific inhibitors of protein kinase C opens the possibilities for dissecting the roles of specific protein kinase Cs in the myriad of intracellular signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf4rqMQJk8PLVg90H21EOLACvtfcHk0lioe9aCvdQzig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXitFSlu78%253D&md5=31b82ff246a981ed692d451122631589</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DConrad%26aufirst%3DR.%26aulast%3DKeranen%26aufirst%3DL.%2BM.%26aulast%3DEllington%26aufirst%3DA.%2BD.%26aulast%3DNewton%26aufirst%3DA.%2BC.%26atitle%3DIsozyme-specific%2520Inhibition%2520of%2520Protein%2520Kinase%2520C%2520by%2520RNA%2520Aptamer%26jtitle%3DThe%2520Journal%2520of%2520Biological%2520Chemistry%26date%3D1994%26volume%3D269%26issue%3D51%26spage%3D32051%26epage%3D32054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreasen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kjems, J.</span></span> <span> </span><span class="NLM_article-title">Characterisation of Aptamer–Target Interactions by Branched Selection and High-throughput Sequencing of SELEX Pools</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">21</span>), <span class="NLM_elocation-id">e139</span> <span class="refDoi"> DOI: 10.1093/nar/gkv700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkv700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=26163061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFCrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&issue=21&author=D.+M.+Dupontauthor=N.+Larsenauthor=J.+K.+Jensenauthor=P.+A.+Andreasenauthor=J.+Kjems&title=Characterisation+of+Aptamer%E2%80%93Target+Interactions+by+Branched+Selection+and+High-throughput+Sequencing+of+SELEX+Pools&doi=10.1093%2Fnar%2Fgkv700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Characterisation of aptamer-target interactions by branched selection and high-throughput sequencing of SELEX pools</span></div><div class="casAuthors">Dupont, Daniel M.; Larsen, Niels; Jensen, Jan. K.; Andreasen, Peter A.; Kjems, Joergen</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">e139/1-e139/13</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Nucleic acid aptamer selection by systematic evolution of ligands by exponential enrichment (SELEX) has shown great promise for use in the development of research tools, therapeutics and diagnostics.  Typically, aptamers are identified from libraries contg. up to 1016 different RNA or DNA sequences by 5-70 rounds of affinity selection towards a target of interest.  Such library screenings can result in complex pools of many target-binding aptamers.  New high-throughput sequencing techniques may potentially revolutionize aptamer selection by allowing quant. assessment of the dynamic changes in the pool compn. during the SELEX process and by facilitating large-scale post-SELEX characterization.  In the present study, we demonstrate how high-throughput sequencing of SELEX pools, before and after a single round of branched selection for binding to different target variants, can provide detailed information about aptamer binding sites, preferences for specific target conformations, and functional effects of the aptamers.  The procedure was applied on a diverse pool of 2'-fluoropyrimidine-modified RNA enriched for aptamers specific for the serpin plasminogen activator inhibitor-1 (PAI-1) through five rounds of std. selection.  The results demonstrate that it is possible to perform large-scale detailed characterization of aptamer sequences directly in the complex pools obtained from library selection methods, thus without the need to produce individual aptamers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqxqLR88dkfbVg90H21EOLACvtfcHk0lioe9aCvdQzig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFCrsLc%253D&md5=ee2b04c641854ebd211c64eb5585b096</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv700%26sid%3Dliteratum%253Aachs%26aulast%3DDupont%26aufirst%3DD.%2BM.%26aulast%3DLarsen%26aufirst%3DN.%26aulast%3DJensen%26aufirst%3DJ.%2BK.%26aulast%3DAndreasen%26aufirst%3DP.%2BA.%26aulast%3DKjems%26aufirst%3DJ.%26atitle%3DCharacterisation%2520of%2520Aptamer%25E2%2580%2593Target%2520Interactions%2520by%2520Branched%2520Selection%2520and%2520High-throughput%2520Sequencing%2520of%2520SELEX%2520Pools%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3D21%26doi%3D10.1093%2Fnar%2Fgkv700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruigrok, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duijn, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barendregt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoltenburg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strehlitz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levisson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smidt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oost, J. V.</span></span> <span> </span><span class="NLM_article-title">Kinetic and Stoichiometric Characterisation of Streptavidin-Binding Aptamers</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">829</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1002/cbic.201100774</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fcbic.201100774" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22416028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1yjur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=829-836&issue=6&author=V.+J.+Ruigrokauthor=E.+V.+Duijnauthor=A.+Barendregtauthor=K.+Dyerauthor=J.+A.+Tainerauthor=R.+Stoltenburgauthor=B.+Strehlitzauthor=M.+Levissonauthor=H.+Smidtauthor=J.+V.+Oost&title=Kinetic+and+Stoichiometric+Characterisation+of+Streptavidin-Binding+Aptamers&doi=10.1002%2Fcbic.201100774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and Stoichiometric Characterisation of Streptavidin-Binding Aptamers</span></div><div class="casAuthors">Ruigrok, Vincent J. B.; van Duijn, Esther; Barendregt, Arjan; Dyer, Kevin; Tainer, John A.; Stoltenburg, Regina; Strehlitz, Beate; Levisson, Mark; Smidt, Hauke; van der Oost, John</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">829-836</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Aptamers are oligonucleotide ligands that are selected for high-affinity binding to mol. targets.  Only limited knowledge relating to relations between structural and kinetic properties that define aptamer-target interactions is available.  To this end, streptavidin-binding aptamers were isolated and characterized by distinct anal. techniques.  Binding kinetics of five broadly similar aptamers were detd. by surface plasmon resonance (SPR); affinities ranged from 35-375 n with large differences in assocn. and dissocn. rates.  Native mass spectrometry showed that streptavidin can accommodate up to two aptamer units.  In a 3D model of one aptamer, conserved regions are exposed, strongly suggesting that they directly interact with the biotin-binding pockets of streptavidin.  Mutational studies confirmed both conserved regions to be crucial for binding.  An important result is the observation that the most abundant aptamer in the authors' selections is not the tightest binder, emphasizing the importance of having insight into the kinetics of complex formation.  To find the tightest binder it might be better to perform fewer selection rounds and to focus on post-selection characterization, through the use of complementary approaches as described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA8_hBjfMC-7Vg90H21EOLACvtfcHk0lg_2QBczTULgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1yjur8%253D&md5=d04f7b513cb0da5ec332ba2a435b3bdc</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100774%26sid%3Dliteratum%253Aachs%26aulast%3DRuigrok%26aufirst%3DV.%2BJ.%26aulast%3DDuijn%26aufirst%3DE.%2BV.%26aulast%3DBarendregt%26aufirst%3DA.%26aulast%3DDyer%26aufirst%3DK.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DStoltenburg%26aufirst%3DR.%26aulast%3DStrehlitz%26aufirst%3DB.%26aulast%3DLevisson%26aufirst%3DM.%26aulast%3DSmidt%26aufirst%3DH.%26aulast%3DOost%26aufirst%3DJ.%2BV.%26atitle%3DKinetic%2520and%2520Stoichiometric%2520Characterisation%2520of%2520Streptavidin-Binding%2520Aptamers%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26issue%3D6%26spage%3D829%26epage%3D836%26doi%3D10.1002%2Fcbic.201100774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bjerregaard, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreasen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, D. M.</span></span> <span> </span><span class="NLM_article-title">Expected and Unexpected Features of Protein-Binding RNA Aptamers</span>. <i>Wiley Interdisciplinary Reviews: RNA</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">744</span>– <span class="NLM_lpage">757</span>, <span class="refDoi"> DOI: 10.1002/wrna.1360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fwrna.1360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=27173731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Ogsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=744-757&issue=6&author=N.+Bjerregaardauthor=P.+A.+Andreasenauthor=D.+M.+Dupont&title=Expected+and+Unexpected+Features+of+Protein-Binding+RNA+Aptamers&doi=10.1002%2Fwrna.1360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Expected and unexpected features of protein-binding RNA aptamers</span></div><div class="casAuthors">Bjerregaard, Nils; Andreasen, Peter A.; Dupont, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">744-757</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">RNA mols. with high affinity to specific proteins can be isolated from libraries of up to 1016 different RNA sequences by systematic evolution of ligands by exponential enrichment (SELEX).  These so-called protein-binding RNA aptamers are often interesting, e.g., as modulators of protein function for therapeutic use, for probing the conformations of proteins, for studies of basic aspects of nucleic acid-protein interactions, etc.  Studies on the interactions between RNA aptamers and proteins display a no. of expected and unexpected features, including the chem. nature of the interacting RNA-protein surfaces, the conformation of protein-bound aptamer vs. free aptamer, the conformation of aptamer-bound protein vs. free protein, and the effects of aptamers on protein function.  Here, we review current insights into the details of RNA aptamer-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2S0mFCMa5T7Vg90H21EOLACvtfcHk0lg_2QBczTULgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Ogsr7L&md5=77e55852cfefb34e39f0f8961380b240</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fwrna.1360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.1360%26sid%3Dliteratum%253Aachs%26aulast%3DBjerregaard%26aufirst%3DN.%26aulast%3DAndreasen%26aufirst%3DP.%2BA.%26aulast%3DDupont%26aufirst%3DD.%2BM.%26atitle%3DExpected%2520and%2520Unexpected%2520Features%2520of%2520Protein-Binding%2520RNA%2520Aptamers%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520RNA%26date%3D2016%26volume%3D7%26issue%3D6%26spage%3D744%26epage%3D757%26doi%3D10.1002%2Fwrna.1360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iwakiri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tateishi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patil, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenmochi, N.</span></span> <span> </span><span class="NLM_article-title">Dissecting the Protein–RNA Interface: The Role of Protein Surface Shapes and RNA Secondary Structures in Protein–RNA Recognition</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">3299</span>– <span class="NLM_lpage">3306</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr1225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkr1225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22199255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1yqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=3299-3306&issue=8&author=J.+Iwakiriauthor=H.+Tateishiauthor=A.+Chakrabortyauthor=P.+Patilauthor=N.+Kenmochi&title=Dissecting+the+Protein%E2%80%93RNA+Interface%3A+The+Role+of+Protein+Surface+Shapes+and+RNA+Secondary+Structures+in+Protein%E2%80%93RNA+Recognition&doi=10.1093%2Fnar%2Fgkr1225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting the protein-RNA interface: the role of protein surface shapes and RNA secondary structures in protein-RNA recognition</span></div><div class="casAuthors">Iwakiri, Junichi; Tateishi, Hiroki; Chakraborty, Anirban; Patil, Prakash; Kenmochi, Naoya</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3299-3306</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein-RNA interactions are essential for many biol. processes.  However, the structural mechanisms underlying these interactions are not fully understood.  Here, the authors analyzed the protein surface shape (dented, intermediate, or protruded) and the RNA base-pairing properties (paired or unpaired nucleotides) at the interfaces of 91 protein-RNA complexes derived from the Protein Data Bank.  It was found that dented protein surfaces preferred unpaired nucleotides to paired ones at the interface, and H-bonds frequently occurred between the protein backbone and RNA bases.  In contrast, protruded protein surfaces did not show such a preference; rather, electrostatic interactions initiated the formation of H-bonds between pos. charged amino acids and RNA phosphate groups.  Interestingly, in many protein-RNA complexes that interact via an RNA loop, an Asp residue was favored at the interface.  Moreover, in most of these complexes, nucleotide bases in the RNA loop were flipped out and formed H-bonds with the protein, which suggested that the Asp residue is important for RNA loop recognition through a base-flipping process.  This study provided fundamental insights into the role of the shape of the protein surface and RNA secondary structures in mediating protein-RNA interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZUXDPF80n87Vg90H21EOLACvtfcHk0lg_2QBczTULgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1yqtbw%253D&md5=dfd4c632db6f9a33db949be61534f588</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1225%26sid%3Dliteratum%253Aachs%26aulast%3DIwakiri%26aufirst%3DJ.%26aulast%3DTateishi%26aufirst%3DH.%26aulast%3DChakraborty%26aufirst%3DA.%26aulast%3DPatil%26aufirst%3DP.%26aulast%3DKenmochi%26aufirst%3DN.%26atitle%3DDissecting%2520the%2520Protein%25E2%2580%2593RNA%2520Interface%253A%2520The%2520Role%2520of%2520Protein%2520Surface%2520Shapes%2520and%2520RNA%2520Secondary%2520Structures%2520in%2520Protein%25E2%2580%2593RNA%2520Recognition%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3D8%26spage%3D3299%26epage%3D3306%26doi%3D10.1093%2Fnar%2Fgkr1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellington, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, R.</span></span> <span> </span><span class="NLM_article-title">Aptamers as Potential Nucleic Acid Pharmaceuticals</span>. <i>Biotechnol. Annu. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/S1387-2656(08)70052-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS1387-2656%2808%2970052-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=9704089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK28XhvFCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1995&pages=185-214&author=A.+D.+Ellingtonauthor=R.+Conrad&title=Aptamers+as+Potential+Nucleic+Acid+Pharmaceuticals&doi=10.1016%2FS1387-2656%2808%2970052-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamers as potential nucleic acid pharmaceuticals</span></div><div class="casAuthors">Ellington, Andrew D.; Conrad, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Annual Review</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">185-214</span>CODEN:
                <span class="NLM_cas:coden">BAREFD</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review, with refs.  In vitro selection is a technique for the isolation of nucleic acid ligands that can bind to proteins with high affinity and specificity, and has potential applications in the development of new pharmaceuticals.  This review summarizes the protein targets that have successfully elicited nucleic acid binding species (also known as "aptamers") and explores examples of how they might be developed for clin. use.  In particular, the use of aptamers for the alleviation of blood clotting and the treatment of AIDS are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS3yBPoaoQ5LVg90H21EOLACvtfcHk0lg_2QBczTULgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvFCisrs%253D&md5=3a6c29c6dddbfafcfbe289f16018430b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1387-2656%2808%2970052-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1387-2656%252808%252970052-8%26sid%3Dliteratum%253Aachs%26aulast%3DEllington%26aufirst%3DA.%2BD.%26aulast%3DConrad%26aufirst%3DR.%26atitle%3DAptamers%2520as%2520Potential%2520Nucleic%2520Acid%2520Pharmaceuticals%26jtitle%3DBiotechnol.%2520Annu.%2520Rev.%26date%3D1995%26volume%3D1%26spage%3D185%26epage%3D214%26doi%3D10.1016%2FS1387-2656%2808%2970052-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulbachinskiy, A. V.</span></span> <span> </span><span class="NLM_article-title">Methods for Selection of Aptamers to Protein Targets</span>. <i>Biochemistry (Moscow)</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>72</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">1505</span>– <span class="NLM_lpage">1518</span>, <span class="refDoi"> DOI: 10.1134/S000629790713007X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1134%2FS000629790713007X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=18282139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitFWktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=1505-1518&issue=13&author=A.+V.+Kulbachinskiy&title=Methods+for+Selection+of+Aptamers+to+Protein+Targets&doi=10.1134%2FS000629790713007X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for selection of aptamers to protein targets</span></div><div class="casAuthors">Kulbachinskiy, A. V.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry (Moscow)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1505-1518</span>CODEN:
                <span class="NLM_cas:coden">BIORAK</span>;
        ISSN:<span class="NLM_cas:issn">0006-2979</span>.
    
            (<span class="NLM_cas:orgname">Pleiades Publishing, Ltd.</span>)
        </div><div class="casAbstract">A review.  Aptamers are synthetic single-stranded RNA or DNA mols. capable of specific binding to other target mols.  In this review, the main aptamer properties are considered and methods for selection of aptamers against various protein targets are described.  Special attention is given to the methods for directed selection of aptamers, which allow one to obtain ligands with specified properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfvwIbRfeCl7Vg90H21EOLACvtfcHk0ljY8UXDXB9nYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitFWktL8%253D&md5=f953811b8f4276a6bf3af3d12fe199aa</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1134%2FS000629790713007X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1134%252FS000629790713007X%26sid%3Dliteratum%253Aachs%26aulast%3DKulbachinskiy%26aufirst%3DA.%2BV.%26atitle%3DMethods%2520for%2520Selection%2520of%2520Aptamers%2520to%2520Protein%2520Targets%26jtitle%3DBiochemistry%2520%2528Moscow%2529%26date%3D2007%26volume%3D72%26issue%3D13%26spage%3D1505%26epage%3D1518%26doi%3D10.1134%2FS000629790713007X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Washietl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrix, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinn, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellis, M.</span></span> <span> </span><span class="NLM_article-title">Computational Analysis of Noncoding RNAs</span>. <i>Wiley Interdisciplinary Reviews: RNA</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">759</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1002/wrna.1134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fwrna.1134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22991327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyqtLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=759-778&issue=6&author=S.+Washietlauthor=S.+Willauthor=D.+A.+Hendrixauthor=L.+A.+Goffauthor=J.+L.+Rinnauthor=B.+Bergerauthor=M.+Kellis&title=Computational+Analysis+of+Noncoding+RNAs&doi=10.1002%2Fwrna.1134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of noncoding RNAs</span></div><div class="casAuthors">Washietl, Stefan; Will, Sebastian; Hendrix, David A.; Goff, Loyal A.; Rinn, John L.; Berger, Bonnie; Kellis, Manolis</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">759-778</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Noncoding RNAs have emerged as important key players in the cell.  Understanding their surprisingly diverse range of functions is challenging for exptl. and computational biol.  Here, we review computational methods to analyze noncoding RNAs.  The topics covered include basic and advanced techniques to predict RNA structures, annotation of noncoding RNAs in genomic data, mining RNA-seq data for novel transcripts and prediction of transcript structures, computational aspects of microRNAs, and database resources.  © 2012 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd17N6l4q84bVg90H21EOLACvtfcHk0ljY8UXDXB9nYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyqtLrN&md5=3b099675393e9643fc9b4d070ddcbf92</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fwrna.1134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.1134%26sid%3Dliteratum%253Aachs%26aulast%3DWashietl%26aufirst%3DS.%26aulast%3DWill%26aufirst%3DS.%26aulast%3DHendrix%26aufirst%3DD.%2BA.%26aulast%3DGoff%26aufirst%3DL.%2BA.%26aulast%3DRinn%26aufirst%3DJ.%2BL.%26aulast%3DBerger%26aufirst%3DB.%26aulast%3DKellis%26aufirst%3DM.%26atitle%3DComputational%2520Analysis%2520of%2520Noncoding%2520RNAs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520RNA%26date%3D2012%26volume%3D3%26issue%3D6%26spage%3D759%26epage%3D778%26doi%3D10.1002%2Fwrna.1134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tinoco, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bustamante, C.</span></span> <span> </span><span class="NLM_article-title">How RNA Folds</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>293</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1999.3001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1006%2Fjmbi.1999.3001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10550208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK1MXms12rtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=1999&pages=271-281&issue=2&author=I.+Tinocoauthor=C.+Bustamante&title=How+RNA+Folds&doi=10.1006%2Fjmbi.1999.3001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">How RNA Folds</span></div><div class="casAuthors">Tinoco, Ignacio, Jr.; Bustamante, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review and discussion with 55 refs.  Here, the authors describe the RNA folding problem and contrast it with the much more difficult protein folding problem.  RNA has 4 similar monomer units, whereas proteins have 20 very different residues.  The folding of RNA is hierarchical in that secondary structure is much more stable than tertiary folding.  In RNA, the 2 levels of folding (secondary and tertiary) can be exptl. sepd. by the presence or absence of Mg2+.  Secondary structure can be predicted successfully from exptl. thermodn. data on secondary structure elements: helixes, loops, and bulges.  Tertiary interactions can then be added without much distortion of the secondary structure.  These observations suggest a folding algorithm to predict the structure of an RNA from its sequence.  However, to solve the RNA folding problem one needs thermodn. data on tertiary structure interactions, and identification and characterization of metal ion-binding sites.  These data, together with force vs. extension measurements on single RNA mols., should provide the information necessary to test and refine the proposed algorithm.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL0G6bLmsiibVg90H21EOLACvtfcHk0ljY8UXDXB9nYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXms12rtLw%253D&md5=3985e0bc7b665f7e6ebc83099685130a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.3001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.3001%26sid%3Dliteratum%253Aachs%26aulast%3DTinoco%26aufirst%3DI.%26aulast%3DBustamante%26aufirst%3DC.%26atitle%3DHow%2520RNA%2520Folds%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D293%26issue%3D2%26spage%3D271%26epage%3D281%26doi%3D10.1006%2Fjmbi.1999.3001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hermann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span> <span> </span><span class="NLM_article-title">RNA Bulges as Architectural and Recognition Motifs</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">R47</span>– <span class="NLM_lpage">R54</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(00)00110-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS0969-2126%2800%2900110-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10745015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVOqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2000&pages=R47-R54&issue=3&author=T.+Hermannauthor=D.+J.+Patel&title=RNA+Bulges+as+Architectural+and+Recognition+Motifs&doi=10.1016%2FS0969-2126%2800%2900110-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">RNA bulges as architectural and recognition motifs</span></div><div class="casAuthors">Hermann, Thomas; Patel, Dinshaw J.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">R47-R54</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 64 refs.  RNA bulges constitute versatile structural motifs in the assembly of RNA architectures.  Three-dimensional structures of RNA mols. and their complexes reveal the role of bulges in RNA architectures and illustrate the mol. mechanisms by which they confer intramol. interactions and intermol. recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo73kYSWXFK7bVg90H21EOLACvtfcHk0lijVE073ijrSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVOqt7w%253D&md5=0ffdd4504fcd38e6cd46e8a9172a7cac</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2800%2900110-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252800%252900110-6%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26atitle%3DRNA%2520Bulges%2520as%2520Architectural%2520and%2520Recognition%2520Motifs%26jtitle%3DStructure%26date%3D2000%26volume%3D8%26issue%3D3%26spage%3DR47%26epage%3DR54%26doi%3D10.1016%2FS0969-2126%2800%2900110-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thapar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denmon, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikonowicz, E. P.</span></span> <span> </span><span class="NLM_article-title">Recognition Modes of RNA Tetraloops and Tetraloop-like Motifs by RNA-binding Proteins</span>. <i>Wiley Interdisciplinary Reviews: RNA</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1002/wrna.1196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fwrna.1196" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=24124096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKqu7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=49-67&issue=1&author=R.+Thaparauthor=A.+P.+Denmonauthor=E.+P.+Nikonowicz&title=Recognition+Modes+of+RNA+Tetraloops+and+Tetraloop-like+Motifs+by+RNA-binding+Proteins&doi=10.1002%2Fwrna.1196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition modes of RNA tetraloops and tetraloop-like motifs by RNA-binding proteins</span></div><div class="casAuthors">Thapar, Roopa; Denmon, Andria P.; Nikonowicz, Edward P.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-67</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  RNA hairpins are the most commonly occurring secondary structural elements in RNAs and serve as nucleation sites for RNA folding, RNA-RNA, and RNA-protein interactions.  RNA hairpins are frequently capped by tetraloops, and based on sequence similarity, three broad classes of RNA tetraloops have been defined: GNRA, UNCG, and CUYG.  Other classes such as the UYUN tetraloop in histone mRNAs, the UGAA in 16S rRNA, the AUUA tetraloop from the MS2 bacteriophage, and the AGNN tetraloop that binds RNase III have also been characterized.  The tetraloop structure is compact and is usually characterized by a paired interaction between the first and fourth nucleotides.  The two unpaired nucleotides in the loop are usually involved in base-stacking or base-phosphate hydrogen bonding interactions.  Several structures of RNA tetraloops, free and complexed to other RNAs or proteins, are now available and these studies have increased our understanding of the diverse mechanisms by which this motif is recognized.  RNA tetraloops can mediate RNA-RNA contacts via the tetraloop-receptor motif, kissing hairpin loops, A-minor interactions, and pseudoknots.  While these RNA-RNA interactions are fairly well understood, how RNA-binding proteins recognize RNA tetraloops and tetraloop-like motifs remains unclear.  In this review, we summarize the structures of RNA tetraloop-protein complexes and the general themes that have emerged on sequence- and structure-specific recognition of RNA tetraloops.  We highlight how proteins achieve mol. recognition of this nucleic acid motif, the structural adaptations obsd. in the tetraloop to accommodate the protein-binding partner, and the role of dynamics in recognition.  WIREs RNA 2014, 5:49-67. doi: 10.1002/wrna.1196 Conflict of interest: The authors declare no competing financial interest.  For further resources related to this article, please visit the WIREs website.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzkdk3dbO0jbVg90H21EOLACvtfcHk0lijVE073ijrSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKqu7nN&md5=6bc4639c4b5077e00e38c2f91798948d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fwrna.1196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.1196%26sid%3Dliteratum%253Aachs%26aulast%3DThapar%26aufirst%3DR.%26aulast%3DDenmon%26aufirst%3DA.%2BP.%26aulast%3DNikonowicz%26aufirst%3DE.%2BP.%26atitle%3DRecognition%2520Modes%2520of%2520RNA%2520Tetraloops%2520and%2520Tetraloop-like%2520Motifs%2520by%2520RNA-binding%2520Proteins%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520RNA%26date%3D2014%26volume%3D5%26issue%3D1%26spage%3D49%26epage%3D67%26doi%3D10.1002%2Fwrna.1196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulffen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brockmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flicke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafenbradl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebl, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohse, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krasel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blind, M.</span></span> <span> </span><span class="NLM_article-title">An RNA Molecule that Specifically Inhibits G-protein-Coupled Receptor Kinase 2 In Vitro</span>. <i>RNA</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1261/rna.821908</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1261%2Frna.821908" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=18230760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1cXivFaqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=524-534&issue=3&author=G.+Mayerauthor=B.+Wulffenauthor=C.+Huberauthor=J.+Brockmannauthor=B.+Flickeauthor=L.+Neumannauthor=D.+Hafenbradlauthor=B.+M.+Kleblauthor=M.+J.+Lohseauthor=C.+Kraselauthor=M.+Blind&title=An+RNA+Molecule+that+Specifically+Inhibits+G-protein-Coupled+Receptor+Kinase+2+In+Vitro&doi=10.1261%2Frna.821908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA molecule that specifically inhibits G-protein-coupled receptor kinase 2 in vitro</span></div><div class="casAuthors">Mayer, Guenter; Wulffen, Bernhard; Huber, Christian; Brockmann, Joerg; Flicke, Birgit; Neumann, Lars; Hafenbradl, Doris; Klebl, Bert M.; Lohse, Martin J.; Krasel, Cornelius; Blind, Michael</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">524-534</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">G-protein-coupled receptors are desensitized by a two-step process.  In a first step, G-protein-coupled receptor kinases (GRKs) phosphorylate agonist-activated receptors that subsequently bind to a second class of proteins, the arrestins.  GRKs can be classified into three subfamilies, which have been implicated in various diseases.  The physiol. role(s) of GRKs have been difficult to study as selective inhibitors are not available.  We have used SELEX (systematic evolution of ligands by exponential enrichment) to develop RNA aptamers that potently and selectively inhibit GRK2.  This process has yielded an aptamer, C13, which bound to GRK2 with a high affinity and inhibited GRK2-catalyzed rhodopsin phosphorylation with an IC50 of 4.1 nM.  Phosphorylation of rhodopsin catalyzed by GRK5 was also inhibited, albeit with 20-fold lower potency (IC50 of 79 nM).  Furthermore, C13 reveals significant specificity, since almost no inhibitory activity was detectable testing it against a panel of 14 other kinases.  The aptamer is two orders of magnitude more potent than the best GRK2 inhibitors described previously and shows high selectivity for the GRK family of protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDj6_B4gax2rVg90H21EOLACvtfcHk0lijVE073ijrSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXivFaqsLs%253D&md5=af9993133b975c237bc6424082a4a268</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1261%2Frna.821908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.821908%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DWulffen%26aufirst%3DB.%26aulast%3DHuber%26aufirst%3DC.%26aulast%3DBrockmann%26aufirst%3DJ.%26aulast%3DFlicke%26aufirst%3DB.%26aulast%3DNeumann%26aufirst%3DL.%26aulast%3DHafenbradl%26aufirst%3DD.%26aulast%3DKlebl%26aufirst%3DB.%2BM.%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26aulast%3DKrasel%26aufirst%3DC.%26aulast%3DBlind%26aufirst%3DM.%26atitle%3DAn%2520RNA%2520Molecule%2520that%2520Specifically%2520Inhibits%2520G-protein-Coupled%2520Receptor%2520Kinase%25202%2520In%2520Vitro%26jtitle%3DRNA%26date%3D2008%26volume%3D14%26issue%3D3%26spage%3D524%26epage%3D534%26doi%3D10.1261%2Frna.821908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennarz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanism for Inhibition of G Protein-Coupled Receptor Kinase 2 by a Selective RNA Aptamer</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1300</span>– <span class="NLM_lpage">1309</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.str.2012.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22727813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVKgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1300-1309&issue=8&author=V.+Tesmerauthor=S.+Lennarzauthor=G.+Mayerauthor=J.+Tesmer&title=Molecular+Mechanism+for+Inhibition+of+G+Protein-Coupled+Receptor+Kinase+2+by+a+Selective+RNA+Aptamer&doi=10.1016%2Fj.str.2012.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanism for Inhibition of G Protein-Coupled Receptor Kinase 2 by a Selective RNA Aptamer</span></div><div class="casAuthors">Tesmer, Valerie M.; Lennarz, Sabine; Mayer, Gunter; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1300-1309</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cardiovascular homeostasis is maintained in part by the rapid desensitization of activated heptahelical receptors that have been phosphorylated by G protein-coupled receptor kinase 2 (GRK2).  However, during chronic heart failure GRK2 is upregulated and believed to contribute to disease progression.  We have detd. crystallog. structures of GRK2 bound to an RNA aptamer that potently and selectively inhibits kinase activity.  Key to the mechanism of inhibition is the positioning of an adenine nucleotide into the ATP-binding pocket and interactions with the basic αF-αG loop region of the GRK2 kinase domain.  Constraints imposed on the RNA by the terminal stem of the aptamer also play a role.  These results highlight how a high-affinity aptamer can be used to selectively trap a novel conformational state of a protein kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNVpjLXpH2bVg90H21EOLACvtfcHk0lgRdcbENAIwgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVKgsLw%253D&md5=26119d2c9cb193d6d9cffa552540a53c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DTesmer%26aufirst%3DV.%26aulast%3DLennarz%26aufirst%3DS.%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DTesmer%26aufirst%3DJ.%26atitle%3DMolecular%2520Mechanism%2520for%2520Inhibition%2520of%2520G%2520Protein-Coupled%2520Receptor%2520Kinase%25202%2520by%2520a%2520Selective%2520RNA%2520Aptamer%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26issue%3D8%26spage%3D1300%26epage%3D1309%26doi%3D10.1016%2Fj.str.2012.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coulson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poyner, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chantry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, S.</span></span> <span> </span><span class="NLM_article-title">A Nonantisense Sequence-Selective Effect of a Phosphorothioate Oligodeoxynucleotide Directed Against the Epidermal Growth Factor Receptor in A431 cells</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">325</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=8700139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK28XkvFCqtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1996&pages=314-325&issue=2&author=J.+M.+Coulsonauthor=D.+R.+Poynerauthor=A.+Chantryauthor=W.+J.+Irwinauthor=S.+Akhtar&title=A+Nonantisense+Sequence-Selective+Effect+of+a+Phosphorothioate+Oligodeoxynucleotide+Directed+Against+the+Epidermal+Growth+Factor+Receptor+in+A431+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">A nonantisense sequence-selective effect of a phosphorothioate oligodeoxynucleotide directed against the epidermal growth factor receptor in A431 cells</span></div><div class="casAuthors">Coulson, Judy M.; Poyner, David R.; Chantry, Andrew; Irwin, William J.; Akhtar, Saghir</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">314-325</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">The overexpression of epidermal growth factor receptor (EGFr) has been implicated as a causative factor and a poor prognostic marker in a no. of carcinomas.  Therefore, strategies that down-regulate EGFr expression may be therapeutically useful.  We designed antisense ODNs complementary to the initiation codon region of the EGFr mRNA and evaluated their efficacy in several tumor-derived cells, including the A431 cell line, that express amplified levels of EGFr.  A 15-mer phosphorothioate (PS) antisense ODN (erbB1AS15) induced a concn.-dependent redn. in proliferation that was accompanied by a change in the morphol. of A431 cells into more tightly clustered and discrete colonies.  A 15-mer sense (PS) control oligodeoxynucleotide (ODN) and a phosphodiester (PS) control oligodeoxynucleotide (ODN) and a phosphodiester (PO) version of erbB1AS15 had little or no effect on cell no. or morphol., and erbB1AS15 (PS) did not induce these effects in control cell lines expressing lower levels of EGFr.  The effects of erbB1AS15 (PS) on A431 cells were not mediated by a true antisense mechanism in that there was no redn. in the level of EGFr mRNA or protein over a 24-h period, as detd. by Northern and Western blotting, resp.  However, autophosphorylation of the receptor was significantly reduced by erbB1AS15 (PS) and not by control ODNs.  The results of further studies suggested that this effect was mediated by a direct, dose-dependent inhibition of the EGFr tyrosine kinase enzyme and was not due to impairment of either ligand-binding or receptor dimerization.  These data suggest that erbB1AS15 (PS) can inhibit proliferation and alter the morphol. of A431 cells by a sequence-selective, but nonantisense, mechanism affecting receptor tyrosine kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBwByQXrQfQLVg90H21EOLACvtfcHk0lgRdcbENAIwgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XkvFCqtb4%253D&md5=969f668cf9b3fdee9a80d828cc9bc350</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCoulson%26aufirst%3DJ.%2BM.%26aulast%3DPoyner%26aufirst%3DD.%2BR.%26aulast%3DChantry%26aufirst%3DA.%26aulast%3DIrwin%26aufirst%3DW.%2BJ.%26aulast%3DAkhtar%26aufirst%3DS.%26atitle%3DA%2520Nonantisense%2520Sequence-Selective%2520Effect%2520of%2520a%2520Phosphorothioate%2520Oligodeoxynucleotide%2520Directed%2520Against%2520the%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520in%2520A431%2520cells%26jtitle%3DMol.%2520Pharmacol.%26date%3D1996%26volume%3D50%26issue%3D2%26spage%3D314%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunnion, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poyner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackroyd, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bibby, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Double, J.</span></span> <span> </span><span class="NLM_article-title">Sequence and Chemistry Requirements for a Novel Aptameric Oligonucleotide Inhibitor of EGF Receptor Tyrosine Kinase Activity</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2187</span>– <span class="NLM_lpage">2195</span>, <span class="refDoi"> DOI: 10.1016/S0006-2952(02)00985-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS0006-2952%2802%2900985-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=12110378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD38XltFSqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2002&pages=2187-2195&issue=12&author=S.+Akhtarauthor=D.+Dunnionauthor=D.+Poynerauthor=J.+Ackroydauthor=M.+Bibbyauthor=J.+Double&title=Sequence+and+Chemistry+Requirements+for+a+Novel+Aptameric+Oligonucleotide+Inhibitor+of+EGF+Receptor+Tyrosine+Kinase+Activity&doi=10.1016%2FS0006-2952%2802%2900985-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence and chemistry requirements for a novel aptameric oligonucleotide inhibitor of EGF receptor tyrosine kinase activity</span></div><div class="casAuthors">Akhtar, Saghir; Dunnion, Debbie; Poyner, David; Ackroyd, John; Bibby, Mike; Double, John</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2187-2195</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">We have previously identified a phosphorothioate oligonucleotide (PS-ODN) that inhibited epidermal growth factor receptor tyrosine kinase (TK) activity both in cell fractions and in intact A431 cells.  Since ODN-based TK inhibitors may have anti-cancer applications and may also help understand the non-antisense mediated effects of PS-ODNs, we have further studied the sequence and chem. requirements of the parent PS-ODN (sequence: 5'-GGA GGG TCG CAT CGC-3') as a sequence-dependent TK inhibitor.  Sequence deletion and substitution studies revealed that the 5'-terminal GGA GGG hexamer sequence in the parent compd. was essential for anti-TK activity in A431 cells.  Site-specific substitution of any G with a T in this 5'-terminal motif within the parent compd. caused a significant loss in anti-TK activity.  The fully PS-modified hexameric motif alone exhibited equipotent activity as the parent 15-mer whereas phosphodiester (PO) or 2'-O-methyl-modified versions of this motif had significantly reduced anti-TK activity.  Further, T substitutions within the two 5'-terminal G residues of the hexameric PS-ODN to produce a sequence, TTA GGG, representing the telomeric repeats in human chromosomes, also did not exhibit a significant anti-TK activity.  Multiple repeats of the active hexameric motif in PS-ODNs resulted in more potent inhibitors of TK activity than the parent ODN.  These results suggested that PS-ODNs, but not PO or 2'-O-Me modified ODNs, contg. the GGA GGG motif can exert potent anti-TK activity which may be desirable in some anti-tumor applications.  Addnl., the presence of this previously unidentified motif in antisense PS-ODN constructs may contribute to their biol. effects in vitro and in vivo and should be accounted for in the design of the PS-modified antisense ODNs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsp7iuJxKpw7Vg90H21EOLACvtfcHk0lgRdcbENAIwgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltFSqsbk%253D&md5=bdebe9d814b245090f32ad1487e62b7c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS0006-2952%2802%2900985-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-2952%252802%252900985-1%26sid%3Dliteratum%253Aachs%26aulast%3DAkhtar%26aufirst%3DS.%26aulast%3DDunnion%26aufirst%3DD.%26aulast%3DPoyner%26aufirst%3DD.%26aulast%3DAckroyd%26aufirst%3DJ.%26aulast%3DBibby%26aufirst%3DM.%26aulast%3DDouble%26aufirst%3DJ.%26atitle%3DSequence%2520and%2520Chemistry%2520Requirements%2520for%2520a%2520Novel%2520Aptameric%2520Oligonucleotide%2520Inhibitor%2520of%2520EGF%2520Receptor%2520Tyrosine%2520Kinase%2520Activity%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2002%26volume%3D63%26issue%3D12%26spage%3D2187%26epage%3D2195%26doi%3D10.1016%2FS0006-2952%2802%2900985-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connell, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahmy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L.</span></span> <span> </span><span class="NLM_article-title">Aptameric Inhibition of p210bcr-abl Tyrosine Kinase Autophosphorylation by Oligodeoxynucleotides of Defined Sequence and Backbone Structure</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2150</span>– <span class="NLM_lpage">2154</span>, <span class="refDoi"> DOI: 10.1093/nar/22.11.2150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2F22.11.2150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=8029025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2cXlslelt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1994&pages=2150-2154&issue=11&author=R.+Berganauthor=Y.+Connellauthor=B.+Fahmyauthor=E.+Kyleauthor=L.+Neckers&title=Aptameric+Inhibition+of+p210bcr-abl+Tyrosine+Kinase+Autophosphorylation+by+Oligodeoxynucleotides+of+Defined+Sequence+and+Backbone+Structure&doi=10.1093%2Fnar%2F22.11.2150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Aptameric inhibition of p210bcr-abl tyrosine kinase autophosphorylation by oligodeoxynucleotides of defined sequence and backbone structure</span></div><div class="casAuthors">Bergan, Raymond; Connell, Yvette; Fahmy, Brigid; Kyle, Edward; Neckers, Len</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2150-4</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    </div><div class="casAbstract">Protein tyrosine kinases play key roles in cellular physiol.  Specific inhibitors of these enzymes are important lab. tools and may prove to be novel therapeutic agents.  In this report the authors describe a new class of tyrosine kinase inhibitor, synthetic oligodeoxynucleotides (ODNs).  An ODN is described which specifically inhibits p210bc4-abl tyrosine kinase autophosphorylation in vitro with a Ki of 0.5 μM.  Inhibition is non-competitive with respect to ATP.  The effects upon inhibitory activity of ODN structure modifications are described.  The inhibition described is not mediated by classical antisense mechanisms and represents an example of the recently recognized aptameric properties of ODNs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgN4XH8VxO1LVg90H21EOLACvtfcHk0lhQw66pWOoxSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlslelt7w%253D&md5=5aaeb6bdce940448c7a9a348cc574ce3</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F22.11.2150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F22.11.2150%26sid%3Dliteratum%253Aachs%26aulast%3DBergan%26aufirst%3DR.%26aulast%3DConnell%26aufirst%3DY.%26aulast%3DFahmy%26aufirst%3DB.%26aulast%3DKyle%26aufirst%3DE.%26aulast%3DNeckers%26aufirst%3DL.%26atitle%3DAptameric%2520Inhibition%2520of%2520p210bcr-abl%2520Tyrosine%2520Kinase%2520Autophosphorylation%2520by%2520Oligodeoxynucleotides%2520of%2520Defined%2520Sequence%2520and%2520Backbone%2520Structure%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1994%26volume%3D22%26issue%3D11%26spage%3D2150%26epage%3D2154%26doi%3D10.1093%2Fnar%2F22.11.2150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fay, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, P.</span></span> <span> </span><span class="NLM_article-title">RNA G-Quadruplexes in Biology: Principles and Molecular Mechanisms</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">2127</span>– <span class="NLM_lpage">2147</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2017.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.jmb.2017.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=28554731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVKhtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2127-2147&issue=14&author=M.+M.+Fayauthor=S.+M.+Lyonsauthor=P.+Ivanov&title=RNA+G-Quadruplexes+in+Biology%3A+Principles+and+Molecular+Mechanisms&doi=10.1016%2Fj.jmb.2017.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">RNA G-quadruplexes in biology: Principles and molecular mechanisms</span></div><div class="casAuthors">Fay, Marta M.; Lyons, Shawn M.; Ivanov, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2127-2147</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes (G4s) are extremely stable DNA or RNA secondary structures formed by sequences rich in guanine.  These structures are implicated in many essential cellular processes, and the no. of biol. functions attributed to them continues to grow.  While DNA G4s are well understood on structural and, to some extent, functional levels, RNA G4s and their functions have received less attention.  The presence of bona fide RNA G4s in cells has long been a matter of debate.  The development of G4-specific antibodies and ligands hinted at their presence in vivo, but recent advances in RNA sequencing coupled with chem. footprinting suggested the opposite.  Here, we critically discuss the biol. of RNA G4s focusing on the mol. mechanisms underlying their proposed functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUMCO9XOrQVLVg90H21EOLACvtfcHk0lhQw66pWOoxSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVKhtLo%253D&md5=987b0cb1715cb621beba9808ef73d5e2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2017.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2017.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DFay%26aufirst%3DM.%2BM.%26aulast%3DLyons%26aufirst%3DS.%2BM.%26aulast%3DIvanov%26aufirst%3DP.%26atitle%3DRNA%2520G-Quadruplexes%2520in%2520Biology%253A%2520Principles%2520and%2520Molecular%2520Mechanisms%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26issue%3D14%26spage%3D2127%26epage%3D2147%26doi%3D10.1016%2Fj.jmb.2017.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woodson, S. A.</span>; <span class="NLM_string-name">Koculi, E.</span></span> <span> </span><span class="NLM_article-title">Analysis of RNA Folding by Native Polyacrylamide Gel Electrophoresis</span>. In  <i>Part B: Methods in Enzymology: Biophysical, Chemical, and Functional Probes of RNA Structure, Interactions and Folding</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span>, Ed.; <span class="NLM_publisher-name">Elsevier Inc. Publishing</span>, <span class="NLM_year">2009</span>; pp  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">208</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS0076-6879%2809%2969009-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=189-208&author=S.+A.+Woodson&author=E.+Koculiauthor=D.+Herschlag&title=Part+B%3A+Methods+in+Enzymology%3A+Biophysical%2C+Chemical%2C+and+Functional+Probes+of+RNA+Structure%2C+Interactions+and+Folding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2809%2969009-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252809%252969009-1%26sid%3Dliteratum%253Aachs%26aulast%3DWoodson%26aufirst%3DS.%2BA.%26atitle%3DAnalysis%2520of%2520RNA%2520Folding%2520by%2520Native%2520Polyacrylamide%2520Gel%2520Electrophoresis%26btitle%3DPart%2520B%253A%2520Methods%2520in%2520Enzymology%253A%2520Biophysical%252C%2520Chemical%252C%2520and%2520Functional%2520Probes%2520of%2520RNA%2520Structure%252C%2520Interactions%2520and%2520Folding%26aulast%3DHerschlag%26aufirst%3DD.%26pub%3DElsevier%2520Inc.%2520Publishing%26date%3D2009%26spage%3D189%26epage%3D208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, C. A.</span>; <span class="NLM_string-name">Fried, M. G.</span></span> <span> </span><span class="NLM_article-title">Analysis of Protein-DNA Equilibria by Native Gel Electrophoresis</span>. In  <i>Protein Interactions: Biophysical Approaches for the Study of Complex Reversible Systems</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, P.</span></span>, Ed.; <span class="NLM_publisher-name">Springer Science Publishing</span>, <span class="NLM_year">2007</span>; pp  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">446</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1007%2F978-0-387-35966-3_14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=417-446&author=C.+A.+Adams&author=M.+G.+Friedauthor=P.+Schuck&title=Protein+Interactions%3A+Biophysical+Approaches+for+the+Study+of+Complex+Reversible+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-35966-3_14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-35966-3_14%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DC.%2BA.%26atitle%3DAnalysis%2520of%2520Protein-DNA%2520Equilibria%2520by%2520Native%2520Gel%2520Electrophoresis%26btitle%3DProtein%2520Interactions%253A%2520Biophysical%2520Approaches%2520for%2520the%2520Study%2520of%2520Complex%2520Reversible%2520Systems%26aulast%3DSchuck%26aufirst%3DP.%26pub%3DSpringer%2520Science%2520Publishing%26date%3D2007%26spage%3D417%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yakhnin, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakhnin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babitzke, P.</span></span> <span> </span><span class="NLM_article-title">Gel Mobility Shift Assays to Detect Protein–RNA Interactions</span>. <i>Bacterial Regulatory RNA</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-949-5_12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1007%2F978-1-61779-949-5_12" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=201-211&author=A.+V.+Yakhninauthor=H.+Yakhninauthor=P.+Babitzke&title=Gel+Mobility+Shift+Assays+to+Detect+Protein%E2%80%93RNA+Interactions&doi=10.1007%2F978-1-61779-949-5_12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-949-5_12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-949-5_12%26sid%3Dliteratum%253Aachs%26aulast%3DYakhnin%26aufirst%3DA.%2BV.%26aulast%3DYakhnin%26aufirst%3DH.%26aulast%3DBabitzke%26aufirst%3DP.%26atitle%3DGel%2520Mobility%2520Shift%2520Assays%2520to%2520Detect%2520Protein%25E2%2580%2593RNA%2520Interactions%26jtitle%3DBacterial%2520Regulatory%2520RNA%26date%3D2012%26spage%3D201%26epage%3D211%26doi%3D10.1007%2F978-1-61779-949-5_12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alshaer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ababneh, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatmal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izmirli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choukeife, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shraim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu-Shiekah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odeh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Bawab, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span> <span> </span><span class="NLM_article-title">Selection and Targeting of EpCAM Protein by ssDNA Aptamer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>), <span class="NLM_elocation-id">e0194529</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0194529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1371%2Fjournal.pone.0194529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29538434" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&issue=3&author=W.+Alshaerauthor=N.+A.+Ababnehauthor=M.+Hatmalauthor=H.+Izmirliauthor=M.+Choukeifeauthor=A.+Shraimauthor=N.+Shararauthor=A.+Abu-Shiekahauthor=F.+Odehauthor=A.+Al+Bawabauthor=A.+Awidiauthor=S.+Ismail&title=Selection+and+Targeting+of+EpCAM+Protein+by+ssDNA+Aptamer&doi=10.1371%2Fjournal.pone.0194529"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0194529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0194529%26sid%3Dliteratum%253Aachs%26aulast%3DAlshaer%26aufirst%3DW.%26aulast%3DAbabneh%26aufirst%3DN.%2BA.%26aulast%3DHatmal%26aufirst%3DM.%26aulast%3DIzmirli%26aufirst%3DH.%26aulast%3DChoukeife%26aufirst%3DM.%26aulast%3DShraim%26aufirst%3DA.%26aulast%3DSharar%26aufirst%3DN.%26aulast%3DAbu-Shiekah%26aufirst%3DA.%26aulast%3DOdeh%26aufirst%3DF.%26aulast%3DAl%2BBawab%26aufirst%3DA.%26aulast%3DAwidi%26aufirst%3DA.%26aulast%3DIsmail%26aufirst%3DS.%26atitle%3DSelection%2520and%2520Targeting%2520of%2520EpCAM%2520Protein%2520by%2520ssDNA%2520Aptamer%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26issue%3D3%26doi%3D10.1371%2Fjournal.pone.0194529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recht, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, J. R.</span></span> <span> </span><span class="NLM_article-title">Quantitative Analysis of Protein-RNA Interactions by Gel Mobility Shift</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>488</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1007/978-1-60327-475-3_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1007%2F978-1-60327-475-3_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=18982286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1CmtLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=488&publication_year=2008&pages=99-115&author=S.+P.+Ryderauthor=M.+I.+Rechtauthor=J.+R.+Williamson&title=Quantitative+Analysis+of+Protein-RNA+Interactions+by+Gel+Mobility+Shift&doi=10.1007%2F978-1-60327-475-3_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative analysis of protein-RNA interactions by gel mobility shift</span></div><div class="casAuthors">Ryder, Sean P.; Recht, Michael I.; Williamson, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (Totowa, NJ, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">488</span>
        (<span class="NLM_cas:issue">RNA-Protein Interaction Protocols (2nd Edition)</span>),
    <span class="NLM_cas:pages">99-115</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">A review.  The gel mobility shift assay is routinely used to visualize protein-RNA interactions.  Its power resides in the ability to resolve free from bound RNA with high resoln. in a gel matrix.  We review the quant. application of this approach to elucidate thermodn. properties of protein-RNA complexes.  Assay designs for titrn., competition, and stoichiometry expts. are presented for two unrelated model complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy2Q7u9qbd4rVg90H21EOLACvtfcHk0lgvtOFXBDOKIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1CmtLfI&md5=54da92d375332a9370de762077ee46cc</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2F978-1-60327-475-3_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60327-475-3_7%26sid%3Dliteratum%253Aachs%26aulast%3DRyder%26aufirst%3DS.%2BP.%26aulast%3DRecht%26aufirst%3DM.%2BI.%26aulast%3DWilliamson%26aufirst%3DJ.%2BR.%26atitle%3DQuantitative%2520Analysis%2520of%2520Protein-RNA%2520Interactions%2520by%2520Gel%2520Mobility%2520Shift%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2008%26volume%3D488%26spage%3D99%26epage%3D115%26doi%3D10.1007%2F978-1-60327-475-3_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navani, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span> <span> </span><span class="NLM_article-title">Isolation of DNA Aptamers for CDP-Ribitol Synthase, and Characterization of Their Inhibitory and Structural Properties</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">2052</span>– <span class="NLM_lpage">2057</span>, <span class="refDoi"> DOI: 10.1002/cbic.200700257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fcbic.200700257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=17929340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsValu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=2052-2057&issue=17&author=A.+Allali-Hassaniauthor=M.+P.+Pereiraauthor=N.+K.+Navaniauthor=E.+D.+Brownauthor=Y.+Li&title=Isolation+of+DNA+Aptamers+for+CDP-Ribitol+Synthase%2C+and+Characterization+of+Their+Inhibitory+and+Structural+Properties&doi=10.1002%2Fcbic.200700257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of DNA aptamers for CDP-ribitol synthase, and characterization of their inhibitory and structural properties</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Pereira, Mark P.; Navani, Naveen K.; Brown, Eric D.; Li, Yingfu</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2052-2057</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Isolation of DNA aptamers for CDP-ribitol synthase is described.  Using the SELEX (systematic evolution of ligands by exponential enrichment) strategy, a pool of 81 nucleotide long single stranded DNA (ssDNA) mols. were produced by automated DNA synthesis.  DNA aptamer pool was then searched to isolate the aptamer for the enzyme Bcs1 from H. influenzae and to test whether any of th aptamers could inhibit either one or both the enzyme activities of this bifunctional enzyme.  The results show that the aptamers were developed for a bifunctional enzyme, and that the SELEX strategy has expanded the no. of protein binding DNA aptamers and has generated new DNA sequences and structures for functional studies of nucleic acid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIjfUKa7pokLVg90H21EOLACvtfcHk0lgvtOFXBDOKIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsValu7zK&md5=5fa40316d63671efae8f06225971bd36</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200700257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200700257%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DPereira%26aufirst%3DM.%2BP.%26aulast%3DNavani%26aufirst%3DN.%2BK.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DIsolation%2520of%2520DNA%2520Aptamers%2520for%2520CDP-Ribitol%2520Synthase%252C%2520and%2520Characterization%2520of%2520Their%2520Inhibitory%2520and%2520Structural%2520Properties%26jtitle%3DChemBioChem%26date%3D2007%26volume%3D8%26issue%3D17%26spage%3D2052%26epage%3D2057%26doi%3D10.1002%2Fcbic.200700257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reinholt, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craighead, H. G.</span></span> <span> </span><span class="NLM_article-title">Highly Multiplexed RNA Aptamer Selection Using a Microplate-based Microcolumn Device</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">29771</span>, <span class="refDoi"> DOI: 10.1038/srep29771</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1038%2Fsrep29771" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=27432610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=29771&issue=1&author=S.+J.+Reinholtauthor=A.+Ozerauthor=J.+T.+Lisauthor=H.+G.+Craighead&title=Highly+Multiplexed+RNA+Aptamer+Selection+Using+a+Microplate-based+Microcolumn+Device&doi=10.1038%2Fsrep29771"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Multiplexed RNA Aptamer Selection using a Microplate-based Microcolumn Device</span></div><div class="casAuthors">Reinholt, Sarah J.; Ozer, Abdullah; Lis, John T.; Craighead, Harold G.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29771</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe a multiplexed RNA aptamer selection to 19 different targets simultaneously using a microcolumn-based device, MEDUSA (Microplate-based Enrichment Device Used for the Selection of Aptamers), as well as a modified selection process, that significantly reduce the time and reagents needed for selections.  We exploited MEDUSA's reconfigurable design between parallel and serially-connected microcolumns to enable the use of just 2 aliquots of starting library, and its 96-well microplate compatibility to enable the continued use of high-throughput techniques in downstream processes.  Our modified selection protocol allowed us to perform the equiv. of a 10-cycle selection in the time it takes for 4 traditional selection cycles.  Several aptamers were discovered with nanomolar dissocn. consts.  Furthermore, aptamers were identified that not only bound with high affinity, but also acted as inhibitors to significantly reduce the activity of their target protein, mouse decapping exoribonuclease (DXO).  The aptamers resisted DXO's exoribonuclease activity, and in studies monitoring DXO's degrdn. of a 30-nucleotide substrate, less than 1 μM of aptamer demonstrated significant inhibition of DXO activity.  This aptamer selection method using MEDUSA helps to overcome some of the major challenges with traditional aptamer selections, and provides a platform for high-throughput selections that lends itself to process automation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprm5z4eWUm47Vg90H21EOLACvtfcHk0lgvtOFXBDOKIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVOgur4%253D&md5=dc850b0fb48e7f340ee8d3d5fb25b57c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fsrep29771&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep29771%26sid%3Dliteratum%253Aachs%26aulast%3DReinholt%26aufirst%3DS.%2BJ.%26aulast%3DOzer%26aufirst%3DA.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26aulast%3DCraighead%26aufirst%3DH.%2BG.%26atitle%3DHighly%2520Multiplexed%2520RNA%2520Aptamer%2520Selection%2520Using%2520a%2520Microplate-based%2520Microcolumn%2520Device%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26issue%3D1%26spage%3D29771%26doi%3D10.1038%2Fsrep29771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jellinek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janjic, N.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Receptor Binding by High-Affinity RNA Ligands to Vascular Endothelial Growth Factor</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">34</span>),  <span class="NLM_fpage">10450</span>– <span class="NLM_lpage">10456</span>, <span class="refDoi"> DOI: 10.1021/bi00200a028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00200a028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2cXltlWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1994&pages=10450-10456&issue=34&author=D.+Jellinekauthor=L.+S.+Greenauthor=C.+Bellauthor=N.+Janjic&title=Inhibition+of+Receptor+Binding+by+High-Affinity+RNA+Ligands+to+Vascular+Endothelial+Growth+Factor&doi=10.1021%2Fbi00200a028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of receptor binding by high-affinity RNA ligands to vascular endothelial growth factor</span></div><div class="casAuthors">Jellinek, Derek; Green, Louis S.; Bell, Carol; Janjic, Nebojsa</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">10450-6</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">The proliferation of new blood vessels (angiogenesis) is a process that accompanies many pathol. conditions including rheumatoid arthritis and solid tumor growth.  Among angiogenic cytokines that have been identified to date, vascular endothelial growth factor (VEGF) is one of the most potent.  SELEX (systematic evolution of ligands by exponential enrichment) was used to identify RNA ligands that bind to VEGF in a specific manner with affinities in the low nanomolar range.  Ligands were selected from a starting pool of ∼1014 RNA mols. contg. 30 randomized positions.  Isolates from the affinity-enriched pool were grouped into 6 distinct families on the basis of primary and secondary structure similarities.  Minimal sequence information required for high-affinity binding to VEGF is contained in 29-36-nucleotide motifs.  Binding of truncated (minimal) high-affinity ligands to VEGF is competitive with that of other truncated ligands and heparin.  Furthermore, truncated ligands from the 6 ligand families inhibit binding of [125I]VEGF to its cell-surface receptors.  Oligonucleotide ligands described here represent an initial set of lead compds. in the ongoing effort toward the development of potent and specific VEGF antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBaPkVkfhWg7Vg90H21EOLACvtfcHk0ljx5aTQZYleFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXltlWqt70%253D&md5=0ae5ff503a899515bc9ecc1ae3861054</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi00200a028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00200a028%26sid%3Dliteratum%253Aachs%26aulast%3DJellinek%26aufirst%3DD.%26aulast%3DGreen%26aufirst%3DL.%2BS.%26aulast%3DBell%26aufirst%3DC.%26aulast%3DJanjic%26aufirst%3DN.%26atitle%3DInhibition%2520of%2520Receptor%2520Binding%2520by%2520High-Affinity%2520RNA%2520Ligands%2520to%2520Vascular%2520Endothelial%2520Growth%2520Factor%26jtitle%3DBiochemistry%26date%3D1994%26volume%3D33%26issue%3D34%26spage%3D10450%26epage%3D10456%26doi%3D10.1021%2Fbi00200a028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. E.</span>; <span class="NLM_string-name">Williams, K. P.</span></span> <span> </span><span class="NLM_article-title">Validating Identity, Mass Purity and Enzymatic Purity of Enzyme Preparations</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, S.</span></span>, Ed.; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences Publishing</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">43</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=31-43&author=J.+E.+Scott&author=K.+P.+Williamsauthor=S.+Sittampalam&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DJ.%2BE.%26atitle%3DValidating%2520Identity%252C%2520Mass%2520Purity%2520and%2520Enzymatic%2520Purity%2520of%2520Enzyme%2520Preparations%26btitle%3DAssay%2520Guidance%2520Manual%26aulast%3DSittampalam%26aufirst%3DS.%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%2520Publishing%26date%3D2012%26spage%3D31%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theis, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knorre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellersch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moelleken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolanus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famulok, M.</span></span> <span> </span><span class="NLM_article-title">Discriminatory Aptamer Reveals Serum Response Element Transcription Regulated by Cytohesin-2</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">11221</span>– <span class="NLM_lpage">11226</span>, <span class="refDoi"> DOI: 10.1073/pnas.0402901101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1073%2Fpnas.0402901101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=15277685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVKgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=11221-11226&issue=31&author=M.+G.+Theisauthor=A.+Knorreauthor=B.+Kellerschauthor=J.+Moellekenauthor=F.+Wielandauthor=W.+Kolanusauthor=M.+Famulok&title=Discriminatory+Aptamer+Reveals+Serum+Response+Element+Transcription+Regulated+by+Cytohesin-2&doi=10.1073%2Fpnas.0402901101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discriminatory aptamer reveals serum response element transcription regulated by cytohesin-2</span></div><div class="casAuthors">Theis, Mirko G.; Knorre, Alexander; Kellersch, Bettina; Moelleken, Joerg; Wieland, Felix; Kolanus, Waldemar; Famulok, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">11221-11226</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cytohesins are a family of highly homologous guanine nucleotide exchange factors (GEFs) that act on ADP-ribosylation factors (ARFs).  The small ARF-GEFs are involved in integrin signaling, actin cytoskeleton remodeling, and vesicle transport.  Here, we selected and applied a specific inhibitor for ARF nucleotide-binding site opener (ARNO)/cytohesin-2, an RNA aptamer that clearly discriminates between cytohesin-1 and cytohesin-2.  This reagent bound to an N-terminal segment of cytohesin-2 and did not inhibit ARF-GEF function in vitro.  When transfected into HeLa cells, it persisted for at least 6 h without requiring stabilization.  Its effect in vivo was to down-regulate gene expression mediated through the serum-response element and knockdown mitogen-activated protein kinase activation, indicating that cytohesin-2 acts by means of mitogen-activated protein kinase signaling.  We conclude that the N-terminal coiled-coil and parts of the Sec7 domain of cytohesin-2 are required for serum-mediated transcriptional activation in nonimmune cells, whereas cytohesin-1 is not.  Our results indicate that intramer technol. can be used not only for assigning novel biol. functions to proteins or protein domains but also to prove nonredundancy of highly homologous proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2nObwn4ouJLVg90H21EOLACvtfcHk0ljx5aTQZYleFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVKgtrg%253D&md5=3641a7ee14d0040340a4b6ff2c57ba63</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0402901101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0402901101%26sid%3Dliteratum%253Aachs%26aulast%3DTheis%26aufirst%3DM.%2BG.%26aulast%3DKnorre%26aufirst%3DA.%26aulast%3DKellersch%26aufirst%3DB.%26aulast%3DMoelleken%26aufirst%3DJ.%26aulast%3DWieland%26aufirst%3DF.%26aulast%3DKolanus%26aufirst%3DW.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DDiscriminatory%2520Aptamer%2520Reveals%2520Serum%2520Response%2520Element%2520Transcription%2520Regulated%2520by%2520Cytohesin-2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26issue%3D31%26spage%3D11221%26epage%3D11226%26doi%3D10.1073%2Fpnas.0402901101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pothukuchy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Syrett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husain, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrisha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosaraju, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellington, A. D.</span></span> <span> </span><span class="NLM_article-title">Aptamers that Recognize Drug-Resistant HIV-1 Reverse Transcriptase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">5236</span>– <span class="NLM_lpage">5236</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp663</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkp663" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKntbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=5236-5236&issue=15&author=N.+Liauthor=Y.+Wangauthor=A.+Pothukuchyauthor=A.+Syrettauthor=N.+Husainauthor=S.+Gopalakrishaauthor=P.+Kosarajuauthor=A.+D.+Ellington&title=Aptamers+that+Recognize+Drug-Resistant+HIV-1+Reverse+Transcriptase&doi=10.1093%2Fnar%2Fgkp663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamers that recognize drug-resistant HIV-1 reverse transcriptase. [Erratum to document cited in CA150:159457]</span></div><div class="casAuthors">Li, Na; Wang, Yuxuan; Pothukuchy, Arti; Syrett, Angel; Husain, Naeem; Gopalakrisha, Siddharth; Kosaraju, Pradeepa; Ellington, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5236</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The authors note that although the did collaborate with the University of Texas, they wish their affiliations to be cor. to Accacia International, 2113 Wells branch Pkwy, Suite 6900, Austin, Texas 78728.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6j5-QczVWJLVg90H21EOLACvtfcHk0ljx5aTQZYleFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKntbvN&md5=2c29e4ad6a5b271087225358faca4579</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp663%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DPothukuchy%26aufirst%3DA.%26aulast%3DSyrett%26aufirst%3DA.%26aulast%3DHusain%26aufirst%3DN.%26aulast%3DGopalakrisha%26aufirst%3DS.%26aulast%3DKosaraju%26aufirst%3DP.%26aulast%3DEllington%26aufirst%3DA.%2BD.%26atitle%3DAptamers%2520that%2520Recognize%2520Drug-Resistant%2520HIV-1%2520Reverse%2520Transcriptase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3D15%26spage%3D5236%26epage%3D5236%26doi%3D10.1093%2Fnar%2Fgkp663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lennarz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heider, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blind, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span> <span> </span><span class="NLM_article-title">An Aptamer to the MAP Kinase Insert Region</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1021/cb5005756</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5005756" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVOntL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=320-327&issue=1&author=S.+Lennarzauthor=E.+Heiderauthor=M.+Blindauthor=G.+Mayer&title=An+Aptamer+to+the+MAP+Kinase+Insert+Region&doi=10.1021%2Fcb5005756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">An Aptamer to the MAP Kinase Insert Region</span></div><div class="casAuthors">Lennarz, Sabine; Heider, Elena; Blind, Michael; Mayer, Guenter</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">320-327</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting functional, non-catalytic domains of protein kinases or other proteins is an emerging field in chem. biol. research.  Non-ATP competitive kinase inhibitors allow for the investigation of active-site independent functions, e.g., the biol. role of protein-protein interactions.  These inhibitors also serve as a starting point for developing novel therapeutic strategies.  However, the identification of such inhibitors by means of conventional low mol. wt. compds. represents a great challenge in modern drug discovery.  Employing the mitogen-activated protein (MAP) kinase Erk2, we show that RNA aptamers have the capacity to be a novel, promising class of protein kinase inhibitors that can be applied to target individual subdomains and block domain specific functions without affecting kinase activity per se.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooDd55t1VYxbVg90H21EOLACvtfcHk0lii42G8FpMktA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVOntL7J&md5=dd4c086945b6d450b367aab2d3090e4e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fcb5005756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5005756%26sid%3Dliteratum%253Aachs%26aulast%3DLennarz%26aufirst%3DS.%26aulast%3DHeider%26aufirst%3DE.%26aulast%3DBlind%26aufirst%3DM.%26aulast%3DMayer%26aufirst%3DG.%26atitle%3DAn%2520Aptamer%2520to%2520the%2520MAP%2520Kinase%2520Insert%2520Region%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D1%26spage%3D320%26epage%3D327%26doi%3D10.1021%2Fcb5005756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seiwert, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahreini, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aigner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhlenbeck, O. C.</span></span> <span> </span><span class="NLM_article-title">RNA Aptamers as Pathway-specific MAP Kinase Inhibitors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">833</span>– <span class="NLM_lpage">843</span>, <span class="refDoi"> DOI: 10.1016/S1074-5521(00)00032-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS1074-5521%2800%2900032-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=11094337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVWit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=833-843&issue=11&author=S.+D.+Seiwertauthor=T.+S.+Nahreiniauthor=S.+Aignerauthor=N.+G.+Ahnauthor=O.+C.+Uhlenbeck&title=RNA+Aptamers+as+Pathway-specific+MAP+Kinase+Inhibitors&doi=10.1016%2FS1074-5521%2800%2900032-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">RNA aptamers as pathway-specific MAP kinase inhibitors</span></div><div class="casAuthors">Seiwert, Scott D.; Nahreini, Theresa Stines; Aigner, Stefan; Ahn, Natalie G.; Uhlenbeck, Olke C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">833-843</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Background: In eukaryotic cells, many intracellular signaling pathways have closely related mitogen activated protein kinase (MAPK) paralogs as central components.  Although MAPKs are therefore obvious targets to control the cellular responses resulting from the activation of these signaling pathways, the development of inhibitors which target specific cell signaling pathways involving MAPKs has proven difficult.  Results: We used an RNA combinatorial approach to isolate RNAs that inhibit the in vitro phosphorylation activity of extracellular regulated kinase 2 (ERK2).  These inhibitors block phosphorylation by ERK1 and ERK2, but do not inhibit Jun N-terminal kinase of p38 MAPKs.  Kinetic anal. indicates these inhibitors function at high picomolar concns. through the steric exclusion of substrate and ATP binding.  In one case, we identified a compact RNA structural domain responsible for inhibition.  Conclusions: RNA reagents can selectively recognize and inhibit MAPKs involved in a single signal transduction pathway.  The methodol. described here is readily generalizable, and can be used to develop inhibitors of MAPKs involved in other signal transduction pathways.  Such reagents may be valuable tools to analyze and distinguish homologous effectors which regulate distinct signaling responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfZyTtw9i0q7Vg90H21EOLACvtfcHk0lii42G8FpMktA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVWit7c%253D&md5=b0d67a6da7702b6226846505310a8363</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2800%2900032-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252800%252900032-6%26sid%3Dliteratum%253Aachs%26aulast%3DSeiwert%26aufirst%3DS.%2BD.%26aulast%3DNahreini%26aufirst%3DT.%2BS.%26aulast%3DAigner%26aufirst%3DS.%26aulast%3DAhn%26aufirst%3DN.%2BG.%26aulast%3DUhlenbeck%26aufirst%3DO.%2BC.%26atitle%3DRNA%2520Aptamers%2520as%2520Pathway-specific%2520MAP%2520Kinase%2520Inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2000%26volume%3D7%26issue%3D11%26spage%3D833%26epage%3D843%26doi%3D10.1016%2FS1074-5521%2800%2900032-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padlan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malashkevich, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenowitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girvin, M. E.</span></span> <span> </span><span class="NLM_article-title">An RNA Aptamer Possessing a Novel Monovalent Cation-Mediated Fold Inhibits Lysozyme Catalysis by Inhibiting the Binding of Long Natural Substrates</span>. <i>RNA</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1261/rna.043034.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1261%2Frna.043034.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=24570482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFCmtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=447-461&issue=4&author=C.+S.+Padlanauthor=V.+N.+Malashkevichauthor=S.+C.+Almoauthor=M.+Levyauthor=M.+Brenowitzauthor=M.+E.+Girvin&title=An+RNA+Aptamer+Possessing+a+Novel+Monovalent+Cation-Mediated+Fold+Inhibits+Lysozyme+Catalysis+by+Inhibiting+the+Binding+of+Long+Natural+Substrates&doi=10.1261%2Frna.043034.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA aptamer possessing a novel monovalent cation-mediated fold inhibits lysozyme catalysis by inhibiting the binding of long natural substrates</span></div><div class="casAuthors">Padlan, Camille S.; Malashkevich, Vladimir N.; Almo, Steve C.; Levy, Matthew; Brenowitz, Michael; Girvin, Mark E.</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-461</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">RNA aptamers are being developed as inhibitors of macromol. and cellular function, diagnostic tools, and potential therapeutics.  Our understanding of the phys. nature of this emerging class of nucleic acid-protein complexes is limited; few at. resoln. structures have been reported for aptamers bound to their protein target.  Guided by chem. mapping, we systematically minimized an RNA aptamer (Lys1) selected against hen egg white lysozyme.  The resultant 59-nucleotide compact aptamer (Lys1.2minE) retains nanomolar binding affinity and the ability to inhibit lysozyme's catalytic activity.  Our 2.0-Å crystal structure of the aptamer-protein complex reveals a helical stem stabilizing two loops to form a protein binding platform that binds lysozyme distal to the catalytic cleft.  This structure along with complementary soln. analyses illuminate a novel protein-nucleic acid interface.  (1) Only 410 Å2 of solvent accessible surface are buried by aptamer binding.  (2) An unusually small fraction (∼18%) of the RNA-protein interaction is electrostatic, consistent with the limited protein phosphate backbone contacts obsd. in the structure.  (3) A single Na+ stabilizes the loops that constitute the protein-binding platform, and consistent with this observation, Lys1.2minE-lysozyme complex formation takes up rather than displaces cations at low ionic strength.  (4) Lys1.2minE inhibits catalysis of large cell wall substrates but not catalysis of small model substrates; and (5) the helical stem of Lys1.2minE can be shortened to four base pairs (Lys1.2minF) without compromising binding affinity, yielding a 45-nucleotide aptamer whose structure may be an adaptable protein binding platform.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6gQVK4xWrXrVg90H21EOLACvtfcHk0lii42G8FpMktA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFCmtb4%253D&md5=adbc4a9ed26e7e27427fd27c4e103b31</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1261%2Frna.043034.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.043034.113%26sid%3Dliteratum%253Aachs%26aulast%3DPadlan%26aufirst%3DC.%2BS.%26aulast%3DMalashkevich%26aufirst%3DV.%2BN.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DBrenowitz%26aufirst%3DM.%26aulast%3DGirvin%26aufirst%3DM.%2BE.%26atitle%3DAn%2520RNA%2520Aptamer%2520Possessing%2520a%2520Novel%2520Monovalent%2520Cation-Mediated%2520Fold%2520Inhibits%2520Lysozyme%2520Catalysis%2520by%2520Inhibiting%2520the%2520Binding%2520of%2520Long%2520Natural%2520Substrates%26jtitle%3DRNA%26date%3D2014%26volume%3D20%26issue%3D4%26spage%3D447%26epage%3D461%26doi%3D10.1261%2Frna.043034.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, T. D.</span></span> <span> </span><span class="NLM_article-title">A Guide to Simple and Informative Binding Assays</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">4061</span>– <span class="NLM_lpage">4067</span>, <span class="refDoi"> DOI: 10.1091/mbc.e10-08-0683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1091%2Fmbc.e10-08-0683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=21115850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1OmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=4061-4067&issue=23&author=T.+D.+Pollard&title=A+Guide+to+Simple+and+Informative+Binding+Assays&doi=10.1091%2Fmbc.e10-08-0683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to simple and informative binding assays</span></div><div class="casAuthors">Pollard, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4061-4067</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">A review.  The aim of binding assays is to measure interactions between two mols., such as a protein binding another protein, a small mol., or a nucleic acid.  Hard work is required to prep. reagents, but flaws in the design of many binding expts. limit the information obtained.  In particular many expts. fail to measure the affinity of the reactants for each other.  This essay describes simple methods to get the most out of valuable reagents in binding expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKpdSYWYPwSrVg90H21EOLACvtfcHk0lhVnai3t4fk8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXis1OmtQ%253D%253D&md5=f837bb1ec71a5babcc5898080d121c27</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e10-08-0683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e10-08-0683%26sid%3Dliteratum%253Aachs%26aulast%3DPollard%26aufirst%3DT.%2BD.%26atitle%3DA%2520Guide%2520to%2520Simple%2520and%2520Informative%2520Binding%2520Assays%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2010%26volume%3D21%26issue%3D23%26spage%3D4061%26epage%3D4067%26doi%3D10.1091%2Fmbc.e10-08-0683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalloway, D.</span></span> <span> </span><span class="NLM_article-title">Density-Dependent Cooperative Non-Specific Binding in Solid-Phase SELEX Affinity Selection</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">7167</span>– <span class="NLM_lpage">7175</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkt477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23737446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1CktLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=7167-7175&issue=14&author=A.+Ozerauthor=B.+S.+Whiteauthor=J.+T.+Lisauthor=D.+Shalloway&title=Density-Dependent+Cooperative+Non-Specific+Binding+in+Solid-Phase+SELEX+Affinity+Selection&doi=10.1093%2Fnar%2Fgkt477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Density-dependent cooperative non-specific binding in solid-phase SELEX affinity selection</span></div><div class="casAuthors">Ozer, Abdullah; White, Brian S.; Lis, John T.; Shalloway, David</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7167-7175</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The non-specific binding of undesired ligands to a target is the primary factor limiting the enrichment of tight-binding ligands in affinity selection.  Soln.-phase non-specific affinity is detd. by the free-energy of ligand binding to a single target.  However, the solid-phase affinity might be higher if a ligand bound concurrently to multiple adjacent immobilized targets in a cooperative manner.  Cooperativity could emerge in this case as a simple consequence of the relationship between the free energy of binding, localization entropy and the spatial distribution of the immobilized targets.  We tested this hypothesis using a SELEX exptl. design and found that non-specific RNA aptamer ligands can concurrently bind up to four bead-immobilized peptide targets, and that this can increase their effective binding affinity by two orders-of-magnitude.  Binding curves were quant. explained by a new statistical mech. model of d.-dependent cooperative binding, which relates cooperative binding to both the target concn. and the target surface d. on the immobilizing substrate.  Target immobilization plays a key role in SELEX and other ligand enrichment methods, particularly in new multiplexed microfluidic purifn. devices, and these results have strong implications for optimizing their performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4PMA4LeNFLLVg90H21EOLACvtfcHk0lhVnai3t4fk8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1CktLbF&md5=b731565dbd11b2d4c2b861dcd3974a81</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt477%26sid%3Dliteratum%253Aachs%26aulast%3DOzer%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DB.%2BS.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26aulast%3DShalloway%26aufirst%3DD.%26atitle%3DDensity-Dependent%2520Cooperative%2520Non-Specific%2520Binding%2520in%2520Solid-Phase%2520SELEX%2520Affinity%2520Selection%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26issue%3D14%26spage%3D7167%26epage%3D7175%26doi%3D10.1093%2Fnar%2Fgkt477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, J. F.</span></span> <span> </span><span class="NLM_article-title">Assay Development for Protein Kinase Enzymes</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, S.</span></span>, Ed.; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences Publishing</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">102</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=85-102&author=J.+F.+Glickmanauthor=S.+Sittampalam&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGlickman%26aufirst%3DJ.%2BF.%26atitle%3DAssay%2520Development%2520for%2520Protein%2520Kinase%2520Enzymes%26btitle%3DAssay%2520Guidance%2520Manual%26aulast%3DSittampalam%26aufirst%3DS.%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%2520Publishing%26date%3D2012%26spage%3D85%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zegzouti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goueli, S. A.</span></span> <span> </span><span class="NLM_article-title">A Simple and Robust Automated Kinase Profiling Platform Using Luminescent ADP Accumulation Technology</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.1089/adt.2009.0216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Fadt.2009.0216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=20059362" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFejtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=573-584&issue=6&author=B.+Larsonauthor=P.+Banksauthor=H.+Zegzoutiauthor=S.+A.+Goueli&title=A+Simple+and+Robust+Automated+Kinase+Profiling+Platform+Using+Luminescent+ADP+Accumulation+Technology&doi=10.1089%2Fadt.2009.0216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A simple and robust automated kinase profiling platform using luminescent ADP accumulation technology</span></div><div class="casAuthors">Larson, Brad; Banks, Peter; Zegzouti, Hicham; Goueli, Said A.</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">573-584</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Kinases continue to be one of the most important targets in today's drug discovery efforts.  Following the identification of lead compds. through screening efforts, it is important to profile these leads against other kinases within that family, as well as from other families, to ascertain potential off-target effects.  Because many kinase assays require the use of different substrates, optimization time and costs during profiling can be prohibitive.  Here, the versatility of a luminescent ADP accumulation assay, where one set of reagents can be used for a wide variety of kinases with differing Kmapp for ATP and substrates, is demonstrated.  Assay sensitivity allows for the use of low enzyme concns. and small percent ATP conversion levels while still maintaining high signal:background ratios.  A simple, inexpensive automated pipetting system was used to automate the entire process from enzyme optimization through generation of compd. IC50 values.  Agreement with literature values proves this combination of chem. and instrumentation provides a simple, yet robust soln. for automated kinase profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoypq9mhtWGL7Vg90H21EOLACvtfcHk0lhVnai3t4fk8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFejtLw%253D&md5=0ad5fb475c052ebb86607a8bc74bd349</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1089%2Fadt.2009.0216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2009.0216%26sid%3Dliteratum%253Aachs%26aulast%3DLarson%26aufirst%3DB.%26aulast%3DBanks%26aufirst%3DP.%26aulast%3DZegzouti%26aufirst%3DH.%26aulast%3DGoueli%26aufirst%3DS.%2BA.%26atitle%3DA%2520Simple%2520and%2520Robust%2520Automated%2520Kinase%2520Profiling%2520Platform%2520Using%2520Luminescent%2520ADP%2520Accumulation%2520Technology%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2009%26volume%3D7%26issue%3D6%26spage%3D573%26epage%3D584%26doi%3D10.1089%2Fadt.2009.0216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNitto, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">English, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greig, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunney, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McTigue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quenzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmett, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demetri, G. D.</span></span> <span> </span><span class="NLM_article-title">KIT Kinase Mutants Show Unique Mechanisms of Drug Resistance to Imatinib and Sunitinib in Gastrointestinal Stromal Tumor Patients</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1542</span>– <span class="NLM_lpage">1547</span>, <span class="refDoi"> DOI: 10.1073/pnas.0812413106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1073%2Fpnas.0812413106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=19164557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVOms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1542-1547&issue=5&author=K.+S.+Gajiwalaauthor=J.+C.+Wuauthor=J.+Christensenauthor=G.+D.+Deshmukhauthor=W.+Diehlauthor=J.+P.+DiNittoauthor=J.+M.+Englishauthor=M.+J.+Greigauthor=Y.+Heauthor=S.+L.+Jacquesauthor=E.+A.+Lunneyauthor=M.+McTigueauthor=D.+Molinaauthor=T.+Quenzerauthor=P.+A.+Wellsauthor=X.+Yuauthor=Y.+Zhangauthor=A.+Zouauthor=M.+R.+Emmettauthor=A.+G.+Marshallauthor=H.+Zhangauthor=G.+D.+Demetri&title=KIT+Kinase+Mutants+Show+Unique+Mechanisms+of+Drug+Resistance+to+Imatinib+and+Sunitinib+in+Gastrointestinal+Stromal+Tumor+Patients&doi=10.1073%2Fpnas.0812413106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">KIT kinase mutants show unique mechanisms of drug resistance to imatinib and sunitinib in gastrointestinal stromal tumor patients</span></div><div class="casAuthors">Gajiwala, Ketan S.; Wu, Joe C.; Christensen, James; Deshmukh, Gayatri D.; Diehl, Wade; DiNitto, Jonathan P.; English, Jessie M.; Greig, Michael J.; He, You-Ai; Jacques, Suzanne L.; Lunney, Elizabeth A.; McTigue, Michele; Molina, Dave; Quenzer, Terri; Wells, Peter A.; Yu, Xiu; Zhang, Yan; Zou, Aihua; Emmett, Mark R.; Marshall, Alan G.; Zhang, Hui-Min; Demetri, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1547</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Most gastrointestinal stromal tumors (GISTs) exhibit aberrant activation of the receptor tyrosine kinase (RTK) KIT.  The efficacy of the inhibitors imatinib mesylate and sunitinib malate in GIST patients has been linked to their inhibition of these mutant KIT proteins.  However, patients on imatinib can acquire secondary KIT mutations that render the protein insensitive to the inhibitor.  Sunitinib has shown efficacy against certain imatinib-resistant mutants, although a subset that resides in the activation loop, including D816H/V, remains resistant.  Biochem. and structural studies were undertaken to det. the mol. basis of sunitinib resistance.  Our results show that sunitinib targets the autoinhibited conformation of WT KIT and that the D816H mutant undergoes a shift in conformational equil. toward the active state.  These findings provide a structural and enzymol. explanation for the resistance profile obsd. with the KIT inhibitors.  Prospectively, they have implications for understanding oncogenic kinase mutants and for circumventing drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkwFxa-p-iWbVg90H21EOLACvtfcHk0liiDh45xc5m5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVOms78%253D&md5=dd226c914c6f9a258426c0f4984a5258</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812413106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812413106%26sid%3Dliteratum%253Aachs%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DWu%26aufirst%3DJ.%2BC.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DDeshmukh%26aufirst%3DG.%2BD.%26aulast%3DDiehl%26aufirst%3DW.%26aulast%3DDiNitto%26aufirst%3DJ.%2BP.%26aulast%3DEnglish%26aufirst%3DJ.%2BM.%26aulast%3DGreig%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLunney%26aufirst%3DE.%2BA.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DMolina%26aufirst%3DD.%26aulast%3DQuenzer%26aufirst%3DT.%26aulast%3DWells%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DEmmett%26aufirst%3DM.%2BR.%26aulast%3DMarshall%26aufirst%3DA.%2BG.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DDemetri%26aufirst%3DG.%2BD.%26atitle%3DKIT%2520Kinase%2520Mutants%2520Show%2520Unique%2520Mechanisms%2520of%2520Drug%2520Resistance%2520to%2520Imatinib%2520and%2520Sunitinib%2520in%2520Gastrointestinal%2520Stromal%2520Tumor%2520Patients%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26issue%3D5%26spage%3D1542%26epage%3D1547%26doi%3D10.1073%2Fpnas.0812413106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <i>Enzymes: A Practical Introduction to Structure, Mechanism, and Data Analysis</i>; <span class="NLM_publisher-name">Wiley-VCH, Inc.</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2000</span>; pp  <span class="NLM_fpage">266</span>– <span class="NLM_lpage">303</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2F0471220639.ch8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&pages=266-303&author=R.+A.+Copeland&title=Enzymes%3A+A+Practical+Introduction+to+Structure%2C+Mechanism%2C+and+Data+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2F0471220639.ch8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471220639.ch8%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEnzymes%253A%2520A%2520Practical%2520Introduction%2520to%2520Structure%252C%2520Mechanism%252C%2520and%2520Data%2520Analysis%26pub%3DWiley-VCH%252C%2520Inc%26date%3D2000%26spage%3D266%26epage%3D303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Georgi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Montalvan, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span> <span> </span><span class="NLM_article-title">Binding Kinetics in Drug Discovery-A Current Perspective</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.2741/4470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.2741%2F4470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=27814600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1CktLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=21-47&issue=1&author=V.+Georgiauthor=D.+Andresauthor=A.+E.+Fernandez-Montalvanauthor=C.+M.+Stegmannauthor=A.+Beckerauthor=A.+Mueller-Fahrnow&title=Binding+Kinetics+in+Drug+Discovery-A+Current+Perspective&doi=10.2741%2F4470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Binding kinetics in drug discovery - a current perspective</span></div><div class="casAuthors">Georgi, Victoria; Andres, Dorothee; Fernandez-Montalvan, Amaury E.; Stegmann, Christian M.; Becker, Andreas; Mueller-Fahrnow, Anke</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-47</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">The impact of target binding kinetics (BK) on the clin. performance of therapeutic agents is presently a topic of intense debate in drug discovery.  While retrospective studies suggest that BK is a differentiating parameter in marketed medicines, it is yet unclear how this information could be used to prioritize drug candidates during lead optimization.  Motivated by the question whether BK can be understood and rationally optimized, we review the most relevant literature in the field, with special focus on selected examples from our organization.  First we discuss structure-kinetic relationships (SKR), and how they can be influenced by factors such as conformational changes, mol. flexibility, hydrogen bonds, hydrophobicity, water mols. and (reversible-) covalent bonds.  We then introduce the methodologies currently used for the investigation of BK parameters, briefly commenting on their strengths, weaknesses and future trends.  Finally, we present our current perspective on the integration of BK in the drug discovery process, aiming to stimulate further thoughts on this important subject.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3xgmlxpV2zbVg90H21EOLACvtfcHk0liiDh45xc5m5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1CktLvM&md5=1ca3deab8d1b2e331a4789bf2f6d2d10</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.2741%2F4470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F4470%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgi%26aufirst%3DV.%26aulast%3DAndres%26aufirst%3DD.%26aulast%3DFernandez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26atitle%3DBinding%2520Kinetics%2520in%2520Drug%2520Discovery-A%2520Current%2520Perspective%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2017%26volume%3D22%26issue%3D1%26spage%3D21%26epage%3D47%26doi%3D10.2741%2F4470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span> <span> </span><span class="NLM_article-title">Defining Balanced Conditions for Inhibitor Screening Assays That Target Bisubstrate Enzymes</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1177/1087057108328763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1177%2F1087057108328763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=19196704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFWksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=111-120&issue=2&author=J.+Yangauthor=R.+A.+Copelandauthor=Z.+Lai&title=Defining+Balanced+Conditions+for+Inhibitor+Screening+Assays+That+Target+Bisubstrate+Enzymes&doi=10.1177%2F1087057108328763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Defining balanced conditions for inhibitor screening assays that target bisubstrate enzymes</span></div><div class="casAuthors">Yang, Jingsong; Copeland, Robert A.; Lai, Zhihong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-120</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is a common mechanism for identifying lead compds. for drug discovery efforts.  Small mols. can inhibit enzymes by a variety of mechanisms, such as competitive, noncompetitive, and uncompetitive with respect to the substrate(s) of the catalytic reaction.  To optimize the chances of finding the broadest diversity of inhibitor modalities during screening, one must run assays under "balanced" conditions where the potency of inhibitors with various modes of action falls within a similar range.  When an enzyme reaction involves more than one substrate, the definition and assessment of the apparent potency of inhibitors (IC50), in relation to their true potency (Ki), can be nontrivial.  This article provides a theor. anal., on the basis of the Cheng-Prusoff derivation, of the IC50/Ki relationship of bisubstrate enzyme reactions following various sequential kinetic mechanisms, as well as the application and limitations of this information for defining optimal screening conditions for such enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM9O6ZD0r6l7Vg90H21EOLACvtfcHk0liiDh45xc5m5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFWksr4%253D&md5=076f5a6ad75c4fc0e01d59ca5ea53fe2</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1177%2F1087057108328763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108328763%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DLai%26aufirst%3DZ.%26atitle%3DDefining%2520Balanced%2520Conditions%2520for%2520Inhibitor%2520Screening%2520Assays%2520That%2520Target%2520Bisubstrate%2520Enzymes%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26issue%3D2%26spage%3D111%26epage%3D120%26doi%3D10.1177%2F1087057108328763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelow, J.</span>; <span class="NLM_string-name">Dewe, W.</span>; <span class="NLM_string-name">Iversen, P. W.</span>; <span class="NLM_string-name">Brooks, H. B.</span>; <span class="NLM_string-name">Radding, J. A.</span>; <span class="NLM_string-name">McGee, J.</span>; <span class="NLM_string-name">Weidner, J.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Action Assays for Enzymes</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, S.</span></span>, Ed.; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences Publishing</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">84</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=58-84&author=J.+Strelow&author=W.+Dewe&author=P.+W.+Iversen&author=H.+B.+Brooks&author=J.+A.+Radding&author=J.+McGee&author=J.+Weidnerauthor=S.+Sittampalam&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%26atitle%3DMechanism%2520of%2520Action%2520Assays%2520for%2520Enzymes%26btitle%3DAssay%2520Guidance%2520Manual%26aulast%3DSittampalam%26aufirst%3DS.%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%2520Publishing%26date%3D2012%26spage%3D58%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span> <span> </span><span class="NLM_article-title">New Directions in Targeting Protein Kinases: Focusing Upon True Allosteric and Bivalent Inhibitors</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">2936</span>– <span class="NLM_lpage">2945</span>, <span class="refDoi"> DOI: 10.2174/138161212800672813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.2174%2F138161212800672813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22571662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=2936-2945&issue=20&author=V.+Lambaauthor=I.+Ghosh&title=New+Directions+in+Targeting+Protein+Kinases%3A+Focusing+Upon+True+Allosteric+and+Bivalent+Inhibitors&doi=10.2174%2F138161212800672813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">New directions in targeting protein kinases: focusing upon true allosteric and bivalent inhibitors</span></div><div class="casAuthors">Lamba, Vandana; Ghosh, Indraneel</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2936-2945</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, therapeutics that target subsets of the 518 human protein kinases have played a vital role in the fight against cancer.  Protein kinases are typically targeted at the ATP-binding cleft by type I and II inhibitors; however, the high sequence and structural homol. shared by protein kinases, esp. at the ATP-binding site, inherently leads to polypharmacol.  In order to discover or design truly selective protein kinase inhibitors as both pharmacol. reagents and safer therapeutic leads, new efforts are needed to target kinases outside the ATP cleft.  Recent advances include the serendipitous discovery of type III inhibitors that bind a site proximal to the ATP pocket as well as the truly allosteric type IV inhibitors that target protein kinases distal to the substrate-binding pocket.  These new classes of inhibitors are often selective, but usually display moderate affinities.  Here, the authors discuss the different classes of inhibitors with an emphasis on bisubstrate and bivalent inhibitors (type V) that combine different inhibitor classes.  These inhibitors have the potential to couple the high affinity and potency of traditional active site-targeted small-mol. inhibitors with the selectivity of inhibitors that target the protein kinase surface outside ATP cleft.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIIgeMxhCrrVg90H21EOLACvtfcHk0ljbgZw5UAWNzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaltrk%253D&md5=36a44e36709507b5053bfdfe7d09e5e9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.2174%2F138161212800672813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212800672813%26sid%3Dliteratum%253Aachs%26aulast%3DLamba%26aufirst%3DV.%26aulast%3DGhosh%26aufirst%3DI.%26atitle%3DNew%2520Directions%2520in%2520Targeting%2520Protein%2520Kinases%253A%2520Focusing%2520Upon%2520True%2520Allosteric%2520and%2520Bivalent%2520Inhibitors%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26issue%3D20%26spage%3D2936%26epage%3D2945%26doi%3D10.2174%2F138161212800672813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavrin, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiah, E.</span></span> <span> </span><span class="NLM_article-title">Approaches to Discover Non-ATP Site Kinase Inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1039/C2MD20180A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1039%2FC2MD20180A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=41-51&issue=1&author=L.+K.+Gavrinauthor=E.+Saiah&title=Approaches+to+Discover+Non-ATP+Site+Kinase+Inhibitors&doi=10.1039%2FC2MD20180A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to discover non-ATP site kinase inhibitors</span></div><div class="casAuthors">Gavrin, Lori Krim; Saiah, Eddine</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-51</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The catalytic domain of kinases shows a high degree of sequence homol., esp. for kinases that belong to the same family.  They share a common ATP binding site with a conserved activation loop and similar three-dimensional structure.  Consequently, a major challenge in kinase research exists in achieving selectivity among the >500 family members, since they all process the same substrate.  In addn. to requiring selectivity against other kinases, ATP site inhibitors must also bind tightly to overcome the high physiol. concn. of ATP in the cell.  Furthermore, the development of novel ATP site inhibitors is becoming increasingly challenging, as many ATP competitive scaffolds have previously been disclosed.  In order to develop compds. with better selectivity among kinases, inhibitors that bind outside the ATP site show great promise and are currently being explored by many groups.  This review will highlight the most commonly used methods to discover small mol. Type III and IV kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xJcf4-RJ9rVg90H21EOLACvtfcHk0ljbgZw5UAWNzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N&md5=97acf2247e5516c1649c1d964c867656</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2FC2MD20180A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20180A%26sid%3Dliteratum%253Aachs%26aulast%3DGavrin%26aufirst%3DL.%2BK.%26aulast%3DSaiah%26aufirst%3DE.%26atitle%3DApproaches%2520to%2520Discover%2520Non-ATP%2520Site%2520Kinase%2520Inhibitors%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26issue%3D1%26spage%3D41%26epage%3D51%26doi%3D10.1039%2FC2MD20180A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatachalam, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surana, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C.</span></span> <span> </span><span class="NLM_article-title">Conformational Landscape of the Epidermal Growth Factor Receptor Kinase Reveals a Mutant Specific Allosteric Pocket</span>. <i>Chemical Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">5212</span>– <span class="NLM_lpage">5222</span>, <span class="refDoi"> DOI: 10.1039/C8SC01262H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1039%2FC8SC01262H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29997876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsFCntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=5212-5222&issue=23&author=S.+Kannanauthor=G.+Venkatachalamauthor=H.+H.+Limauthor=U.+Suranaauthor=C.+Verma&title=Conformational+Landscape+of+the+Epidermal+Growth+Factor+Receptor+Kinase+Reveals+a+Mutant+Specific+Allosteric+Pocket&doi=10.1039%2FC8SC01262H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational landscape of the epidermal growth factor receptor kinase reveals a mutant specific allosteric pocket</span></div><div class="casAuthors">Kannan, Srinivasaraghavan; Venkatachalam, Gireedhar; Lim, Hong Hwa; Surana, Uttam; Verma, Chandra</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5212-5222</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Activating mutations within the epidermal growth factor receptor (EGFR) kinase domain give rise to several cancers including Non-Small Cell Lung Cancer (NSCLC).  Small mol. inhibitors targeted at these mutants have proven to be clin. successful drugs.  These mols. are ATP competitive and rapidly result in the emergence of resistance.  Recently Jia et al. [Nature, 2016, 534, 129-132] reported a small mol. inhibitor (called EAI045) that binds at an allosteric pocket, does not compete with ATP and displays high potency and selectivity towards certain activating mutants (L858R, T790M, L858R/T790M) of EGFR, with IC50 values ranging from 3 nM to 49 nM.  We present here a study combining extensive mol. dynamics simulations with binding assays to provide a structural basis underlying the mechanism of binding of this mol.  It appears that in mutants, conformational destabilization of the short helix (that carries Leu858 in the wildtype), is key to the exposure of the allosteric pocket which otherwise is occluded by a set of sidechains including L858.  We extend this hypothesis to show that a similar mechanism would enable the mol. to inhibit EGFRL861Q which is another oncogenic mutant and validate this with binding expts.  The screening of the human structural kinome revealed at least 12 other oncogenic kinases which carry at least one activating mutant in this disorder-prone region and hence would be amenable to allosteric inhibition by mols. such as EAI045.  Our study characterizes a druggable allosteric pocket which appears to be specific to certain oncogenic mutants of the EGFR and holds therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNJMPumMkZFbVg90H21EOLACvtfcHk0ljbgZw5UAWNzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsFCntLo%253D&md5=57b221846f35482d03d19944ccc48919</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1039%2FC8SC01262H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC01262H%26sid%3Dliteratum%253Aachs%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DVenkatachalam%26aufirst%3DG.%26aulast%3DLim%26aufirst%3DH.%2BH.%26aulast%3DSurana%26aufirst%3DU.%26aulast%3DVerma%26aufirst%3DC.%26atitle%3DConformational%2520Landscape%2520of%2520the%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Kinase%2520Reveals%2520a%2520Mutant%2520Specific%2520Allosteric%2520Pocket%26jtitle%3DChemical%2520Science%26date%3D2018%26volume%3D9%26issue%3D23%26spage%3D5212%26epage%3D5222%26doi%3D10.1039%2FC8SC01262H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span> <span> </span><span class="NLM_article-title">Identification of RNA Aptamer Specific to Mutant KRAS Protein</span>. <i>Oligonucleotides</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1089/oli.2010.0231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Foli.2010.0231" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=155-161&issue=3&author=S.+Jeongauthor=S.+R.+Hanauthor=Y.+J.+Leeauthor=J.+H.+Kimauthor=S.+Lee&title=Identification+of+RNA+Aptamer+Specific+to+Mutant+KRAS+Protein&doi=10.1089%2Foli.2010.0231"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1089%2Foli.2010.0231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Foli.2010.0231%26sid%3Dliteratum%253Aachs%26aulast%3DJeong%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DS.%2BR.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520RNA%2520Aptamer%2520Specific%2520to%2520Mutant%2520KRAS%2520Protein%26jtitle%3DOligonucleotides%26date%3D2012%26volume%3D20%26issue%3D3%26spage%3D155%26epage%3D161%26doi%3D10.1089%2Foli.2010.0231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rashid, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awan, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, G.</span></span> <span> </span><span class="NLM_article-title">The Isolation of an RNA Aptamer Targeting to p53 Protein with Single Amino Acid Mutation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span> (<span class="NLM_issue">32</span>),  <span class="NLM_fpage">10002</span>– <span class="NLM_lpage">10007</span>, <span class="refDoi"> DOI: 10.1073/pnas.1502159112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1073%2Fpnas.1502159112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=26216949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Crt73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=10002-10007&issue=32&author=L.+Chenauthor=F.+Rashidauthor=A.+Shahauthor=H.+M.+Awanauthor=M.+Wuauthor=A.+Liuauthor=M.+Wuauthor=J.+Wangauthor=T.+Zhuauthor=Z.+Luoauthor=G.+Shan&title=The+Isolation+of+an+RNA+Aptamer+Targeting+to+p53+Protein+with+Single+Amino+Acid+Mutation&doi=10.1073%2Fpnas.1502159112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">The isolation of an RNA aptamer targeting to p53 protein with single amino acid mutation</span></div><div class="casAuthors">Chen, Liang; Rashid, Farooq; Shah, Abdullah; Awan, Hassaan M.; Wu, Mingming; Liu, An; Wang, Jun; Zhu, Tao; Luo, Zhaofeng; Shan, Ge</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">10002-10007</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">P53, known as a tumor suppressor, is a DNA binding protein that regulates cell cycle, activates DNA repair proteins, and triggers apoptosis in multicellular animals.  More than 50% of human cancers contain a mutation or deletion of the p53 gene, and p53R175 is one of the hot spots of p53 mutation.  Nucleic acid aptamers are short single-stranded oligonucleotides that are able to bind various targets, and they are typically isolated from an exptl. procedure called systematic evolution of ligand exponential enrichment (SELEX).  Using a previously unidentified strategy of contrast screening with SELEX, we have isolated an RNA aptamer targeting p53R175H.  This RNA aptamer (p53R175H-APT) has a significantly stronger affinity to p53R175H than to the wild-type p53 in both in vitro and in vivo assays.  P53R175H-APT decreased the growth rate, weakened the migration capability, and triggered apoptosis in human lung cancer cells harboring p53R175H.  Further anal. actually indicated that p53R175H-APT might partially rescue or correct the p53R175H to function more like the wild-type p53.  In situ injections of p53R175H-APT to the tumor xenografts confirmed the effects of this RNA aptamer on p53R175H mutation in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-EyKwnVM037Vg90H21EOLACvtfcHk0lgUc0hjJ8tzaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Crt73L&md5=f744dd150680cf325cdc83234e328b96</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1502159112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1502159112%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DRashid%26aufirst%3DF.%26aulast%3DShah%26aufirst%3DA.%26aulast%3DAwan%26aufirst%3DH.%2BM.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DShan%26aufirst%3DG.%26atitle%3DThe%2520Isolation%2520of%2520an%2520RNA%2520Aptamer%2520Targeting%2520to%2520p53%2520Protein%2520with%2520Single%2520Amino%2520Acid%2520Mutation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26issue%3D32%26spage%3D10002%26epage%3D10007%26doi%3D10.1073%2Fpnas.1502159112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kratschmer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span> <span> </span><span class="NLM_article-title">Effect of Chemical Modifications on Aptamer Stability in Serum</span>. <i>Nucleic Acid Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">335</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1089/nat.2017.0680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Fnat.2017.0680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=28945147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKqtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=335-344&issue=6&author=C.+Kratschmerauthor=M.+Levy&title=Effect+of+Chemical+Modifications+on+Aptamer+Stability+in+Serum&doi=10.1089%2Fnat.2017.0680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of Chemical Modifications on Aptamer Stability in Serum</span></div><div class="casAuthors">Kratschmer, Christina; Levy, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acid Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">335-344</span>CODEN:
                <span class="NLM_cas:coden">NATUCU</span>;
        ISSN:<span class="NLM_cas:issn">2159-3337</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">There is increasing interest in the use of aptamers for the development of therapeutics.  However, as oligonucleotides, aptamers are susceptible to nuclease degrdn.; poor serum stability is likely to neg. affect in vivo function.  Modified nucleotides have been used to thwart nuclease degrdn.  However, few studies report the serum stability of selected aptamers.  In this study, we examd. the effect of various chem. modifications (2'-deoxy, 2'-hydroxyl, 2'-fluoro, and 2'-O-methyl) on the stability of a control oligonucleotide sequence following incubation in frozen human, fresh mouse, and fresh human serum.  We also assessed the effect of the 3' inverted dT cap on stability.  Surprisingly, we found that fYrR (2'-fluoro RNA) is only roughly as stable as DNA (2'-deoxy).  Interestingly, the inclusion of a 3' inverted dT cap had only a modest effect on serum stability, if any.  In one instance, the addn. of a 3' inverted dT cap rendered a mol. composed of DNA more stable than its fYrR counterpart.  By far, fully modified oligonucleotides (100% 2-O-Me or 2'-O-Me A, C, and U in combination with 2'-fluoro G, termed fGmH) had the longest half-lives.  These compns. demonstrated little degrdn. in human serum even after prolonged incubation.  Together these results support the need for using fully modified aptamers for in vivo applications and should encourage those in the field to exploit newer polymerase variants capable of directly generating such polymers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbb3qGrP1G77Vg90H21EOLACvtfcHk0lgUc0hjJ8tzaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKqtrzI&md5=0e3c675745073ea2967e7dfedd0502dc</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1089%2Fnat.2017.0680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fnat.2017.0680%26sid%3Dliteratum%253Aachs%26aulast%3DKratschmer%26aufirst%3DC.%26aulast%3DLevy%26aufirst%3DM.%26atitle%3DEffect%2520of%2520Chemical%2520Modifications%2520on%2520Aptamer%2520Stability%2520in%2520Serum%26jtitle%3DNucleic%2520Acid%2520Ther.%26date%3D2017%26volume%3D27%26issue%3D6%26spage%3D335%26epage%3D344%26doi%3D10.1089%2Fnat.2017.0680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pryer, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherrington, J. M.</span></span> <span> </span><span class="NLM_article-title">SU11248 Inhibits KIT and Platelet-derived Growth Factor Receptor β in Preclinical Models of Human Small Cell Lung Cancer</span>. <i>Molecular Cancer Therapeutics</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">478</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=12748309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvVersbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=471-478&issue=5&author=T.+J.+Abramsauthor=L.+B.+Leeauthor=L.+J.+Murrayauthor=N.+J.+Pryerauthor=J.+M.+Cherrington&title=SU11248+Inhibits+KIT+and+Platelet-derived+Growth+Factor+Receptor+%CE%B2+in+Preclinical+Models+of+Human+Small+Cell+Lung+Cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">SU11248 inhibits KIT and platelet-derived growth factor receptor β in preclinical models of human small cell lung cancer</span></div><div class="casAuthors">Abrams, Tinya J.; Lee, Leslie B.; Murray, Lesley J.; Pryer, Nancy K.; Cherrington, Julie M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-478</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The purpose of this study was to evaluate the activity of the indolinone kinase inhibitor SU11248 against the receptor tyrosine kinase KIT in vitro and in vivo, examine the role of KIT in small cell lung cancer (SCLC), and anticipate clin. utility of SU11248 in SCLC.  SU11248 is an oral, multitargeted tyrosine kinase inhibitor with direct antitumor and antiangiogenic activity through targeting platelet-derived growth factor receptor (PDGFR), vascular endothelial growth factor receptor, KIT, and FLT3 receptors.  Treatment of the KIT-expressing SCLC-derived NCI-H526 cell line in vitro with SU11248 resulted in dose-dependent inhibition of stem cell factor-stimulated KIT phosphotyrosine levels and proliferation.  The biol. significance of KIT inhibition was evaluated in vivo by treating mice bearing s.c. NCI-H526 tumors with SU11248 or another structurally unrelated KIT inhibitor, STI571 (Gleevec), which is also known to inhibit Bcr-AbI and PDGFRβ.  SU11248 treatment resulted in significant tumor growth inhibition, whereas inhibition from STI571 treatment was less dramatic.  Both compds. reduced phospho-KIT levels in NCI-H526 tumors, with a greater redn. by SU11248, correlating with efficacy.  Likewise, phospho-PDGFRβ levels contributed by tumor stroma and with known involvement in angiogenesis were strongly inhibited by SU11248 and less so by STI571.  Because platinum-based chemotherapy is part of the std. of care for SCLC, SU11248 was combined with cisplatin, and significant tumor growth delay was measured compared with either agent alone.  These results expand the profile of SU11248 as a KIT signaling inhibitor and suggest that SU11248 may have clin. potential in the treatment of SCLC via direct antitumor activity mediated via KIT as well as tumor angiogenesis via vascular endothelial growth factor receptor FLK1/KDR and PDGFRβ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLMjnXAK7tXLVg90H21EOLACvtfcHk0lhxeg7zSSe1og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvVersbw%253D&md5=732c0ed4b392c2d7ead9b1be6f6a7927</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbrams%26aufirst%3DT.%2BJ.%26aulast%3DLee%26aufirst%3DL.%2BB.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DPryer%26aufirst%3DN.%2BJ.%26aulast%3DCherrington%26aufirst%3DJ.%2BM.%26atitle%3DSU11248%2520Inhibits%2520KIT%2520and%2520Platelet-derived%2520Growth%2520Factor%2520Receptor%2520%25CE%25B2%2520in%2520Preclinical%2520Models%2520of%2520Human%2520Small%2520Cell%2520Lung%2520Cancer%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26date%3D2003%26volume%3D2%26issue%3D5%26spage%3D471%26epage%3D478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krystal, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carison, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litz, J.</span></span> <span> </span><span class="NLM_article-title">Induction of Apoptosis and Inhibition of Small Cell Lung Cancer Growth by the Quinoxaline Tyrphostins</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2203</span>– <span class="NLM_lpage">2208</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=9187122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2sXjslGrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=2203-2208&author=G.+W.+Krystalauthor=P.+Carisonauthor=J.+Litz&title=Induction+of+Apoptosis+and+Inhibition+of+Small+Cell+Lung+Cancer+Growth+by+the+Quinoxaline+Tyrphostins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis and inhibition of small cell lung cancer growth by the quinoxaline tyrphostins</span></div><div class="casAuthors">Krystal, Geoffrey W.; Carlson, Patricia; Litz, Julie</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2203-2208</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Coexpression of the Kit receptor tyrosine kinase and its ligand, stem cell factor (SCF), occurs in a high proportion of small cell lung cancers (SCLCs) and drives an autocrine loop that enhances proliferation.  To det. whether the autocrine loop affects apoptosis, SCLC cells expressing only SCF or both SCF and Kit were deprived of growth factors for 72 h and the relative no. of cells undergoing apoptosis was assessed using nuclear DNA content and terminal deoxynucleotidyl transferase-mediated dUTP nick end labeling assays.  Coexpression of SCF and Kit inhibited apoptosis; apoptosis could, in turn, be enhanced by the addn. of the quinoxaline tyrosine kinase inhibitors (AG 1295 and AG 1296), which are specific antagonists of the platelet-derived growth factor receptor and Kit.  Treatment of the H526 cell line, which is growth-stimulated by sol. SCF, with AG 1296 resulted in a marked decrease in growth and an increase in apoptosis in a dose-dependent fashion.  Growth inhibition correlated well with the inhibition of Kit tyrosine phosphorylation.  The AG 1296 compd. at its max. sol. concn. inhibited the growth of 5 of 6 SCLC cell lines in complete medium by an av. of 50%.  These data suggest that optimized pharmacol. inhibitors of Kit activity may be a new class of compds. potentially useful in the treatment of SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHfg0XwkNAT7Vg90H21EOLACvtfcHk0lhxeg7zSSe1og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjslGrtrg%253D&md5=118e61bb7a80d453dd0b88cfdc572e2b</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrystal%26aufirst%3DG.%2BW.%26aulast%3DCarison%26aufirst%3DP.%26aulast%3DLitz%26aufirst%3DJ.%26atitle%3DInduction%2520of%2520Apoptosis%2520and%2520Inhibition%2520of%2520Small%2520Cell%2520Lung%2520Cancer%2520Growth%2520by%2520the%2520Quinoxaline%2520Tyrphostins%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D2203%26epage%3D2208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krystal, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honsawek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchdunger, E.</span></span> <span> </span><span class="NLM_article-title">The Selective Tyrosine Kinase Inhibitor STI571 Inhibits Small Cell Lung Cancer Growth</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">3319</span>– <span class="NLM_lpage">3326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10955819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsVOmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=3319-3326&author=G.+W.+Krystalauthor=S.+Honsawekauthor=J.+Litzauthor=E.+Buchdunger&title=The+Selective+Tyrosine+Kinase+Inhibitor+STI571+Inhibits+Small+Cell+Lung+Cancer+Growth"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">The selective tyrosine kinase inhibitor STI571 inhibits small cell lung cancer growth</span></div><div class="casAuthors">Krystal, Geoffrey W.; Honsawek, Sittisak; Litz, Julie; Buchdunger, Elisabeth</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3319-3326</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">At least 70% of small cell lung cancers express the Kit receptor Tyr kinase and its ligand, stem cell factor (SCF).  Numerous lines of evidence have demonstrated that this coexpression constitutes a functional autocrine loop, suggesting that inhibitors of Kit Tyr kinase activity could have therapeutic efficacy in this disease.  STI571, formerly known as CGP 57148B, is a p.o. bioavailable 2-phenylaminopyrimide deriv. that was designed as an Abl Tyr kinase inhibitor, but also has efficacy against the platelet-derived growth factor receptor and Kit in vitro.  Pretreatment of the H526 small cell lung cancer (SCLC) cell line with STI571 inhibited SCF-mediated Kit activation with an IC50 of 0.1 μM as measured by inhibition of receptor Tyr phosphorylation and 0.2 μM as measured by immune complex kinase assay.  Tis paralleled the inhibition of SCF-mediated growth by STI571, which had an IC50 of ∼0.3 μM.  Growth inhibition in SCF-contg. medium was accompanied by induction of apoptosis.  STI571 efficiently blocked SCF-mediated activation of mitogen-activated protein kinase and Akt, but did not affect insulin-like growth factor-1 or serum-mediated mitogen-activated protein kinase or Akt activation.  Growth of 5 of 6 SCLC cell lines in medium contg. 10% FCS was inhibited by STI571 with an IC50 of ∼5 μM.  Growth inhibition in serum-contg. medium appeared to be cytostatic in nature because no increase in apoptosis was obsd.  Despite this growth inhibition, STI571 failed to enhance the cytotoxicity of either carboplatinum or etoposide when coadministered.  However, taken together with the minimal toxicity that this compd. has shown in preclin. studies, these data suggest that STI571 could have a role in the treatment of SCLC, possibly to block or slow recurrence after chemotherapy-induced remissions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXlmbLvNhphbVg90H21EOLACvtfcHk0lhxeg7zSSe1og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsVOmtLo%253D&md5=25ad853eb069e2741c21a9edc8104df4</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrystal%26aufirst%3DG.%2BW.%26aulast%3DHonsawek%26aufirst%3DS.%26aulast%3DLitz%26aufirst%3DJ.%26aulast%3DBuchdunger%26aufirst%3DE.%26atitle%3DThe%2520Selective%2520Tyrosine%2520Kinase%2520Inhibitor%2520STI571%2520Inhibits%2520Small%2520Cell%2520Lung%2520Cancer%2520Growth%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2000%26volume%3D6%26spage%3D3319%26epage%3D3326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krystal, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulanke, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litz, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Phosphatidylinositol 3-Kinase-Akt Signaling Blocks Growth, Promotes Apoptosis, and Enhances Sensitivity of Small Cell Lung Cancer Cells to Chemotherapy</span>. <i>Molecular Cancer Therapeutics</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">922</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=12481412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFOnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=913-922&author=G.+W.+Krystalauthor=G.+Sulankeauthor=J.+Litz&title=Inhibition+of+Phosphatidylinositol+3-Kinase-Akt+Signaling+Blocks+Growth%2C+Promotes+Apoptosis%2C+and+Enhances+Sensitivity+of+Small+Cell+Lung+Cancer+Cells+to+Chemotherapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of phosphatidylinositol 3-kinase-Akt signaling blocks growth, promotes apoptosis, and enhances sensitivity of small cell lung cancer cells to chemotherapy</span></div><div class="casAuthors">Krystal, Geoffrey W.; Sulanke, Geoffrey; Litz, Julie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">913-922</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A promising therapeutic alternative to inhibition of growth factor receptors is the inhibition of downstream signal transduction pathways.  Such an approach may be esp. important in tumors that can use signals from multiple growth factor receptors for growth and survival.  Both stem cell factor (SCF) and insulin-like growth factor (IGF)-I, components of prominent small cell lung cancer (SCLC) autocrine loops, as well as FCS, can potently activate phosphatidylinositol 3-kinase (PI3K)-Akt signaling, albeit with different kinetics.  SCF-induced PI3K-Akt activation occurs rapidly but fades within 60 min; IGF-I and FCS-induced activation persists for at least 6 h.  SCF and IGF-I-mediated growth was potently inhibited by LY294002 in proportion to its ability to inhibit phosphatidylinositol 3-kinase (PI3K)-Akt signaling.  A panel of six SCLC cell lines grown in 10% FCS was also very sensitive to LY294002, with av. IC50 and LD50 of 5 and 25 μM, resp.  These drug concns. suppressed the growth of the MRC-5 pulmonary fibroblast cell line and primary bronchial epithelial cells but did not induce significant cell death.  Because LY294002 can also inhibit PI3K-related enzymes, we confirmed the role of the PI3K-Akt pathway in SCLC using doxycycline-regulated expression of a dominant-neg. (kinase dead) and a constitutively active (CA; myristolated) Akt allele.  Expression of dominant-neg. Akt, which could only be achieved at relatively low levels, completely inhibited growth in the absence of exogenous growth factors and inhibited SCF-mediated growth but had no effect on IGF-I-mediated growth at the expression levels attained.  Expression of CA Akt markedly augmented growth in the absence of exogenous growth factors but had minimal effect on growth in the presence of satg. concns. SCF or IGF-I.  Because PI3K-Akt signaling is known to promote survival under apoptotic stresses, we detd. the effect of this pathway on SCLC sensitivity to etoposide.  LY294002 potentiated the effect of low concns. of etoposide in inhibiting growth and inducing apoptosis.  The effect of low concns. of LY294002 could largely be reversed by expression of CA Akt, suggesting that it was mediated by inhibition of Akt signaling.  Expression of CA Akt by itself also induced resistance to etoposide-mediated apoptosis.  Taken together, these data demonstrate that PI3K-Akt signaling promotes SCLC growth, survival, and chemotherapy resistance.  Therefore, selective inhibitors of PI3K or Akt could potentially be useful as novel therapeutic agents in the treatment of SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7p1hRSeRxJrVg90H21EOLACvtfcHk0liFuXeyqOKgIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFOnsw%253D%253D&md5=16a348dbb16be9354417f47c07087276</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrystal%26aufirst%3DG.%2BW.%26aulast%3DSulanke%26aufirst%3DG.%26aulast%3DLitz%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520Phosphatidylinositol%25203-Kinase-Akt%2520Signaling%2520Blocks%2520Growth%252C%2520Promotes%2520Apoptosis%252C%2520and%2520Enhances%2520Sensitivity%2520of%2520Small%2520Cell%2520Lung%2520Cancer%2520Cells%2520to%2520Chemotherapy%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26date%3D2002%26volume%3D1%26spage%3D913%26epage%3D922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maulik, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broderick, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kijima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salgia, R.</span></span> <span> </span><span class="NLM_article-title">Modulation of c-Kit/SCF Pathway Leads to Alterations in Topoisomerase-I Activity in Small Cell Lung Cancer</span>. <i>J. Environ. Pathol., Toxicol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">251</span>, <span class="refDoi"> DOI: 10.1615/JEnvPathToxOncol.v23.i4.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1615%2FJEnvPathToxOncol.v23.i4.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=15511212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGgurbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=237-251&issue=4&author=G.+Maulikauthor=A.+Bhartiauthor=E.+Khanauthor=R.+J.+Broderickauthor=T.+Kijimaauthor=R.+Salgia&title=Modulation+of+c-Kit%2FSCF+Pathway+Leads+to+Alterations+in+Topoisomerase-I+Activity+in+Small+Cell+Lung+Cancer&doi=10.1615%2FJEnvPathToxOncol.v23.i4.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of c-Kit/SCF pathway leads to alterations in topoisomerase-I activity in small cell lung cancer</span></div><div class="casAuthors">Maulik, Gautam; Bharti, Ajit; Khan, Ehsan; Broderick, Ryan J.; Kijima, Takashi; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Environmental Pathology, Toxicology and Oncology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">237-251</span>CODEN:
                <span class="NLM_cas:coden">JEPOEC</span>;
        ISSN:<span class="NLM_cas:issn">0731-8898</span>.
    
            (<span class="NLM_cas:orgname">Begell House, Inc.</span>)
        </div><div class="casAbstract">Small cell lung cancer (SCLC) is an aggressive type of lung cancer, for which cytotoxic chemotherapy appears to have reached its maximal efficacy.  This neoplasm is characterized by the overexpression of several receptor tyrosine kinases (RTKs), esp. c-Kit.  The ligand for c-Kit is stem cell factor (SCF).  In SCLC, SCF can influence c-Kit activation by autocrine or paracrine mechanisms.  We have recently shown that the c-Kit/SCF pathway is operational in SCLC and can be inhibited by Glivec (STI571).  Because the inhibition of topoisomerase-I (topo-I) is one approach used to treat SCLC, we detd. the effects of c-Kit/SCF signaling on topo-I activity.  A unique phosphorylation of c-Kit on amino acid 823 and amino acid 703 was identified with the SCF stimulation of H526 cells.  We demonstrate that with SCF stimulation over 16 h (dose response 0-100 ng/mL) in H526 SCLC cells (c-Kit pos., SCF responsive), a decrease in topo-1 activity was obsd., whereas in H82 SCLC cells (c-Kit neg., SCF unresponsive) there was no modulation of topo-I activity by SCF.  Using STI571 (5 μM, 16 h) to inhibit the c-Kit pathway following stimulation with SCF (100 ng/mL), an upregulation of topo-I activity was obsd. in H526 cells but not in H82 cells.  Performing viability assays, we show that STI571 in combination with topo-I inhibition by camptothecin or SN38, the active metabolite of irinotecan, can cooperatively inhibit H526 cell viability (but not H82 cell viability) for 72 h.  We also show that STI571 does not directly inhibit topo-I activity in SCLC.  The combination of STI571 with topo-I inhibition could provide a useful combination in the treatment of SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv2FEqUNhT17Vg90H21EOLACvtfcHk0liFuXeyqOKgIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGgurbN&md5=73b075276d2e5b4b79a9b98c593b3b89</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1615%2FJEnvPathToxOncol.v23.i4.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1615%252FJEnvPathToxOncol.v23.i4.10%26sid%3Dliteratum%253Aachs%26aulast%3DMaulik%26aufirst%3DG.%26aulast%3DBharti%26aufirst%3DA.%26aulast%3DKhan%26aufirst%3DE.%26aulast%3DBroderick%26aufirst%3DR.%2BJ.%26aulast%3DKijima%26aufirst%3DT.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DModulation%2520of%2520c-Kit%252FSCF%2520Pathway%2520Leads%2520to%2520Alterations%2520in%2520Topoisomerase-I%2520Activity%2520in%2520Small%2520Cell%2520Lung%2520Cancer%26jtitle%3DJ.%2520Environ.%2520Pathol.%252C%2520Toxicol.%2520Oncol.%26date%3D2004%26volume%3D23%26issue%3D4%26spage%3D237%26epage%3D251%26doi%3D10.1615%2FJEnvPathToxOncol.v23.i4.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGraff, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, J. B.</span></span> <span> </span><span class="NLM_article-title">Evaluation of a Tetrazolium-based Semiautomated Colorimetric Assay: Assessment of Chemosensitivity Testing</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">936</span>– <span class="NLM_lpage">942</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=3802100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaL2sXhsVyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1987&pages=936-942&issue=4&author=J.+Carmichaelauthor=W.+J.+DeGraffauthor=A.+F.+Gazdarauthor=J.+D.+Minnaauthor=J.+B.+Mitchell&title=Evaluation+of+a+Tetrazolium-based+Semiautomated+Colorimetric+Assay%3A+Assessment+of+Chemosensitivity+Testing"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of a tetrazolium-based semiautomated colorimetric assay:  assessment of chemosensitivity testing</span></div><div class="casAuthors">Carmichael, James; DeGraff, William G.; Gazdar, Adi F.; Minna, John D.; Mitchell, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">936-42</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">Drug sensitivity assays were performed using a variation of a colorimetric thiazolyl blue (MTT)  [298-93-1] assay on V79, CHO-AuxB1, CHRC5, NCI-H460, and NCI-H249 cell lines following optimization of exptl. conditions for each cell line.  Results from this assay were compared with data assimilated simultaneously by clonogenic assay and by dye exclusion assay.  Good correlation was obsd. using the CHO-AuxB1 cell line and the pleiotropic drug-resistant mutant CHRC5, with similar degrees of relative resistance obsd. with both the MTT and clonogenic assays.  Good correlation was obsd. between the clonogenic and MTT assays for 1-h drug exposures, although the MTT assay was more sensitive to vinblastine.  In general, the clonogenic assay was more sensitive when continuous drug exposures were utilized, although this was primarily related to the increased drug exposure time.  While the use of the MTT assay in drug sensitivity testing of primary tumor samples is limited, since contaminating normal cells may also reduce the tetrazolium, the MTT assay can be semiautomated, and therefore it offers a valid, simple method of assessing chemosensitivity in established cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj92XstI7WtLVg90H21EOLACvtfcHk0liFuXeyqOKgIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXhsVyrtb0%253D&md5=2414230269c9f4d5f2910764c65fc042</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDeGraff%26aufirst%3DW.%2BJ.%26aulast%3DGazdar%26aufirst%3DA.%2BF.%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%26atitle%3DEvaluation%2520of%2520a%2520Tetrazolium-based%2520Semiautomated%2520Colorimetric%2520Assay%253A%2520Assessment%2520of%2520Chemosensitivity%2520Testing%26jtitle%3DCancer%2520Res.%26date%3D1987%26volume%3D47%26issue%3D4%26spage%3D936%26epage%3D942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcaro, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Receptor Tyrosine Kinase Signalling in Small Cell Lung Cancer (SCLC): What have we learned so far?</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2007.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.ctrv.2007.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=17368733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsF2rtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2007&pages=391-406&issue=4&author=B.+Fischerauthor=M.+Marinovauthor=A.+Arcaro&title=Targeting+Receptor+Tyrosine+Kinase+Signalling+in+Small+Cell+Lung+Cancer+%28SCLC%29%3A+What+have+we+learned+so+far%3F&doi=10.1016%2Fj.ctrv.2007.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting receptor tyrosine kinase signalling in small cell lung cancer (SCLC): What have we learned so far?</span></div><div class="casAuthors">Fischer, Barbara; Marinov, Marin; Arcaro, Alexandre</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">391-406</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Small cell lung cancer (SCLC) is an aggressive form of lung cancer, which represents 13% of all cases and is strongly assocd. with cigarette smoking.  The survival of SCLC patients is dismal and has not greatly improved in the last 20 years, despite advances in chemotherapy regimens and a better understanding of SCLC biol.  The development of resistance to chemotherapy and metastasis are commonly recognized as important causes of poor clin. outcome in SCLC.  Targeting receptor tyrosine kinase (RTK) signalling represents an attractive approach to develop new drugs for SCLC, in view of the accumulating data demonstrating that polypeptide growth factors play a key role in driving SCLC cell proliferation, chemoresistance and metastasis.  The insulin-like growth factor-I receptor (IGF-IR), c-Kit, vascular endothelial growth factor receptor (VEGFR) and epidermal growth factor receptor (EGFR) have been identified as potential drug targets in SCLC.  Moreover, downstream signalling mediators of RTKs, such as phosphoinositide 3-kinase (PI3K)/Akt and the mammalian target of rapamycin (mTOR) may also represent attractive candidate mols. for anti-cancer therapies in SCLC.  Here we will review the available data concerning results with RTK inhibitors in SCLC and the clin. trials undertaken to investigate the potential of these compds. as anti-tumor agents in SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKsJGB0yAQzbVg90H21EOLACvtfcHk0lhYf-B1y8crVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsF2rtrg%253D&md5=e3572a7e560515b1c608d0f3d8fb105c</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2007.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2007.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DB.%26aulast%3DMarinov%26aufirst%3DM.%26aulast%3DArcaro%26aufirst%3DA.%26atitle%3DTargeting%2520Receptor%2520Tyrosine%2520Kinase%2520Signalling%2520in%2520Small%2520Cell%2520Lung%2520Cancer%2520%2528SCLC%2529%253A%2520What%2520have%2520we%2520learned%2520so%2520far%253F%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2007%26volume%3D33%26issue%3D4%26spage%3D391%26epage%3D406%26doi%3D10.1016%2Fj.ctrv.2007.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sekido, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zsebo, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, T.</span></span> <span> </span><span class="NLM_article-title">Recombinant Human Stem Cell Factor Mediates Chemotaxis of Small-Cell Lung Cancer Cell Lines Aberrantly Expressing the c-Kit Protooncogene</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1709</span>– <span class="NLM_lpage">1714</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=7680956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK3sXisFansro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1993&pages=1709-1714&author=Y.+Sekidoauthor=T.+Takahashiauthor=R.+Uedaauthor=M.+Takahashiauthor=H.+Suzukiauthor=K.+Nishidaauthor=T.+Tsukamotoauthor=T.+Hidaauthor=K.+Shimokataauthor=K.+M.+Zseboauthor=T.+Takahashi&title=Recombinant+Human+Stem+Cell+Factor+Mediates+Chemotaxis+of+Small-Cell+Lung+Cancer+Cell+Lines+Aberrantly+Expressing+the+c-Kit+Protooncogene"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Recombinant human stem cell factor mediates chemotaxis of small-cell lung cancer cell lines aberrantly expressing the c-kit protooncogene</span></div><div class="casAuthors">Sekido, Yoshitaka; Takahashi, Takashi; Ueda, Ryuzo; Takahashi, Masahide; Suzuki, Hiroko; Nishida, Keiko; Tsukamoto, Tetsuya; Hida, Toyoaki; Shimokata, Kaoru; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1709-14</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">Accumulating evidence suggests that c-kit and its ligand, stem cell factor (SCF), play an important role in the regulation of growth in at least three lineages of stem cell and possibly in leukemogenesis, while only limited data are available that suggest possible involvement of c-kit/SCF in the development of human solid tumors such as lung cancer.  We have recently reported that c-kit is aberrantly expressed almost exclusively in small-cell lung cancer (SCLC) among various types of solid tumors.  The present study revealed that c-kit protein ectopically expressed in SCLC is indistinguishable from that in leukemia cell lines with megakaryocytic characteristics with respect to amt., mol. size, and autophosphorylation status in response to recombinant human SCF.  Furthermore, significant chemotactic response as well as moderate in vitro cell growth was induced in SCLC cell lines by the addn. of recombinant human SCF, suggesting that c-kit/SCF may play an important biol. role in the development of SCLC.  Our extensive search for activating mutations naturally occurring in the c-kit gene revealed an amino acid substitution in the transmembrane domain of an SCLC cell line, although the functional consequences of this variant allele are yet to be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9gtA-Ir-7LbVg90H21EOLACvtfcHk0lhYf-B1y8crVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisFansro%253D&md5=44359e197f9907ee312d840a1a7e8d88</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSekido%26aufirst%3DY.%26aulast%3DTakahashi%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DSuzuki%26aufirst%3DH.%26aulast%3DNishida%26aufirst%3DK.%26aulast%3DTsukamoto%26aufirst%3DT.%26aulast%3DHida%26aufirst%3DT.%26aulast%3DShimokata%26aufirst%3DK.%26aulast%3DZsebo%26aufirst%3DK.%2BM.%26aulast%3DTakahashi%26aufirst%3DT.%26atitle%3DRecombinant%2520Human%2520Stem%2520Cell%2520Factor%2520Mediates%2520Chemotaxis%2520of%2520Small-Cell%2520Lung%2520Cancer%2520Cell%2520Lines%2520Aberrantly%2520Expressing%2520the%2520c-Kit%2520Protooncogene%26jtitle%3DCancer%2520Res.%26date%3D1993%26volume%3D53%26spage%3D1709%26epage%3D1714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Detailed Conformational Dynamics of Juxtamembrane Region and Activation Loop in c-Kit Kinase Activation Process</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>72</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1002/prot.21928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fprot.21928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=18214972" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2008&pages=323-332&issue=1&author=J.+Zouauthor=Y.+Wangauthor=F.+Maauthor=M.+Xiangauthor=B.+Shiauthor=Y.+Weiauthor=S.+Yang&title=Detailed+Conformational+Dynamics+of+Juxtamembrane+Region+and+Activation+Loop+in+c-Kit+Kinase+Activation+Process&doi=10.1002%2Fprot.21928"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1002%2Fprot.21928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21928%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DF.%26aulast%3DXiang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DDetailed%2520Conformational%2520Dynamics%2520of%2520Juxtamembrane%2520Region%2520and%2520Activation%2520Loop%2520in%2520c-Kit%2520Kinase%2520Activation%2520Process%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2008%26volume%3D72%26issue%3D1%26spage%3D323%26epage%3D332%26doi%3D10.1002%2Fprot.21928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollok, B. A.</span></span> <span> </span><span class="NLM_article-title">Developing Assays for Kinase Drug Discovery-Where have the Advances Come From?</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1517/17460441.3.1.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1517%2F17460441.3.1.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23480143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVGntLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=115-129&issue=1&author=K.+W.+Vogelauthor=Z.+Zhongauthor=K.+Biauthor=B.+A.+Pollok&title=Developing+Assays+for+Kinase+Drug+Discovery-Where+have+the+Advances+Come+From%3F&doi=10.1517%2F17460441.3.1.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Developing assays for kinase drug discovery - where have the advances come from?</span></div><div class="casAuthors">Vogel, Kurt W.; Zhong, Zhong; Bi, Kun; Pollok, Brian A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115-129</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Over the past decade, a variety of technologies for the identification and characterization of protein kinase inhibitors have been implemented in the labs. of nearly every major pharmaceutical and biotechnol. company.  Although the majority of these assay technologies are highly robust, the ability of many assays to identify compds. that target the kinase of interest in a true biol. context remains questionable.  Because every in vitro assay represents a trade-off between biol. relevancy and factors such as cost, throughput and accuracy, it is important to acknowledge and balance these trade-offs when interrogating a kinase target in such an assay.  This review addresses some of the factors that should be considered when developing protein kinase assays, as well as strategies used to address those factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo21WVdpzY72rVg90H21EOLACvtfcHk0lhuBhtC9rAXyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVGntLw%253D&md5=f4036599a04f9523175983e132f6e2ee</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1517%2F17460441.3.1.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.3.1.115%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DK.%2BW.%26aulast%3DZhong%26aufirst%3DZ.%26aulast%3DBi%26aufirst%3DK.%26aulast%3DPollok%26aufirst%3DB.%2BA.%26atitle%3DDeveloping%2520Assays%2520for%2520Kinase%2520Drug%2520Discovery-Where%2520have%2520the%2520Advances%2520Come%2520From%253F%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2008%26volume%3D3%26issue%3D1%26spage%3D115%26epage%3D129%26doi%3D10.1517%2F17460441.3.1.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connell, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Protein-Tyrosine Kinase Activity in Intact Cells by the Aptameric Action of Oligodeoxynucleotides</span>. <i>Antisense Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1089/ard.1995.5.33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Fard.1995.5.33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=7542047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2MXlsV2it78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1995&pages=33-38&issue=1&author=R.+C.+Berganauthor=E.+Kyleauthor=Y.+Connellauthor=L.+Neckers&title=Inhibition+of+Protein-Tyrosine+Kinase+Activity+in+Intact+Cells+by+the+Aptameric+Action+of+Oligodeoxynucleotides&doi=10.1089%2Fard.1995.5.33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of protein-tyrosine kinase activity in intact cells by the aptameric action of oligodeoxynucleotides</span></div><div class="casAuthors">Bergan, Raymond C.; Kyle, Edward; Connell, Yvette; Neckers, Len</div><div class="citationInfo"><span class="NLM_cas:title">Antisense Research and Development</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8</span>CODEN:
                <span class="NLM_cas:coden">AREDEI</span>;
        ISSN:<span class="NLM_cas:issn">1050-5261</span>.
    
            (<span class="NLM_cas:orgname">Liebert</span>)
        </div><div class="casAbstract">Direct interaction of oligodeoxynucleotides (ODNs) with proteins represents one of the nonantisense-mediated effects of ODNs.  Phosphorothioate-capped ODNs have been shown to inhibit directly the in vitro kinase activity of the chronic myelogenous leukemia-assocd. protein kinase p210bcr-abl.  In this study we have detd. the efficacy of this aptameric ODN in a cellular system using the K562 chronic myelogenous leukemia-derived cell line.  Significant effects upon cellular phosphotyrosine content, as well as cellular growth in soft agar, are obsd.  These effects are sequence specific and are not mediated through changes in p210bcr-abl protein levels.  Addnl. ODNs are described that also reduce cellular phosphotyrosine levels and inhibit growth in soft agar but do not inhibit p210bcr-abl kinase activity in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrevsJE3XttKrVg90H21EOLACvtfcHk0lhuBhtC9rAXyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlsV2it78%253D&md5=b7e4d62f3e603a0459dbc2e664685c5e</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1089%2Fard.1995.5.33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fard.1995.5.33%26sid%3Dliteratum%253Aachs%26aulast%3DBergan%26aufirst%3DR.%2BC.%26aulast%3DKyle%26aufirst%3DE.%26aulast%3DConnell%26aufirst%3DY.%26aulast%3DNeckers%26aufirst%3DL.%26atitle%3DInhibition%2520of%2520Protein-Tyrosine%2520Kinase%2520Activity%2520in%2520Intact%2520Cells%2520by%2520the%2520Aptameric%2520Action%2520of%2520Oligodeoxynucleotides%26jtitle%3DAntisense%2520Res.%2520Dev.%26date%3D1995%26volume%3D5%26issue%3D1%26spage%3D33%26epage%3D38%26doi%3D10.1089%2Fard.1995.5.33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazi, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rönnstrand, L.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of the Activation Loop Tyrosine 823 in c-Kit Is Crucial for Cell Survival and Proliferation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">22460</span>– <span class="NLM_lpage">22468</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.474072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1074%2Fjbc.M113.474072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23803604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Whtr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=22460-22468&issue=31&author=S.+Agarwalauthor=J.+U.+Kaziauthor=L.+R%C3%B6nnstrand&title=Phosphorylation+of+the+Activation+Loop+Tyrosine+823+in+c-Kit+Is+Crucial+for+Cell+Survival+and+Proliferation&doi=10.1074%2Fjbc.M113.474072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of the activation loop tyrosine 823 in c-Kit Is Crucial for Cell Survival and Proliferation</span></div><div class="casAuthors">Agarwal, Shruti; Kazi, Julhash U.; Roennstrand, Lars</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">22460-22468</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The receptor tyrosine kinase c-Kit, also known as the stem cell factor receptor, plays a key role in several developmental processes.  Activating mutations in c-Kit lead to alteration of these cellular processes and have been implicated in many human cancers such as gastrointestinal stromal tumors, acute myeloid leukemia, testicular seminomas and mastocytosis.  Regulation of the catalytic activity of several kinases is known to be governed by phosphorylation of tyrosine residues in the activation loop of the kinase domain.  However, in the case of c-Kit phosphorylation of Tyr-823 has been demonstrated to be a late event that is not required for kinase activation.  However, because phosphorylation of Tyr-823 is a ligand-activated event, we sought to investigate the functional consequences of Tyr-823 phosphorylation.  By using a tyrosine-to-phenylalanine mutant of tyrosine 823, we investigated the impact of Tyr-823 on c-Kit signaling.  We demonstrate here that Tyr-823 is crucial for cell survival and proliferation and that mutation of Tyr-823 to phenylalanine leads to decreased sustained phosphorylation and ubiquitination of c-Kit as compared with the wild-type receptor.  Furthermore, the mutated receptor was, upon ligand-stimulation, quickly internalized and degraded.  Phosphorylation of the E3 ubiquitin ligase Cbl was transient, followed by a substantial redn. in phosphorylation of downstream signaling mols. such as Akt, Erk, p38, Shc, and Gab2.  Thus, we propose that activation loop tyrosine 823 is crucial for activation of both the MAPK and PI3K pathways and that its disruption leads to a destabilization of the c-Kit receptor and decreased survival of cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodqwzOEtfdkrVg90H21EOLACvtfcHk0liX7wtAaSvb8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Whtr7L&md5=e8c59e575709ed8a4137edfcb989722d</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.474072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.474072%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwal%26aufirst%3DS.%26aulast%3DKazi%26aufirst%3DJ.%2BU.%26aulast%3DR%25C3%25B6nnstrand%26aufirst%3DL.%26atitle%3DPhosphorylation%2520of%2520the%2520Activation%2520Loop%2520Tyrosine%2520823%2520in%2520c-Kit%2520Is%2520Crucial%2520for%2520Cell%2520Survival%2520and%2520Proliferation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26issue%3D31%26spage%3D22460%26epage%3D22468%26doi%3D10.1074%2Fjbc.M113.474072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazi, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohlin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Påhlman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rönnstrand, L.</span></span> <span> </span><span class="NLM_article-title">The Activation Loop Tyrosine 823 is Essential for the Transforming Capacity of the c-Kit Oncogenic Mutant D816V</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">35</span>),  <span class="NLM_fpage">4581</span>– <span class="NLM_lpage">4590</span>, <span class="refDoi"> DOI: 10.1038/onc.2014.383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1038%2Fonc.2014.383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=25435369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVGlsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=4581-4590&issue=35&author=S.+Agarwalauthor=J.+U.+Kaziauthor=S.+Mohlinauthor=S.+P%C3%A5hlmanauthor=L.+R%C3%B6nnstrand&title=The+Activation+Loop+Tyrosine+823+is+Essential+for+the+Transforming+Capacity+of+the+c-Kit+Oncogenic+Mutant+D816V&doi=10.1038%2Fonc.2014.383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">The activation loop tyrosine 823 is essential for the transforming capacity of the c-Kit oncogenic mutant D816V</span></div><div class="casAuthors">Agarwal, S.; Kazi, J. U.; Mohlin, S.; Paahlman, S.; Roennstrand, L.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">4581-4590</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oncogenic c-Kit mutations have been shown to display ligand-independent receptor activation and cell proliferation.  A substitution of aspartate to valine at amino acid 816 (D816V) is one of the most commonly found oncogenic c-Kit mutations and is found in >90% of cases of mastocytosis and less commonly in germ-cell tumors, core-binding factor acute myeloid leukemia and mucosal melanomas.  The mechanisms by which this mutation leads to constitutive activation and transformation are not fully understood.  Previous studies have shown that the D816V mutation causes a structural change in the activation loop (A-loop), resulting in weaker binding of the A-loop to the juxtamembrane domain.  In this paper, the authors have investigated the role of Y823, the only tyrosine residue in the A-loop, and its role in oncogenic transformation by c-Kit/D816V by introducing the Y823F mutation.  Although dispensable for the kinase activity of c-Kit/D816V, the presence of Y823 was crucial for cell proliferation and survival.  Furthermore, mutation of Y823 selectively downregulates the Ras/Erk and Akt pathways as well as the phosphorylation of STAT5 and reduces the transforming capacity of the D816V/c-Kit in vitro.  The authors further show that mice injected with cells expressing c-Kit/D816V/Y823F display significantly reduced tumor size as well as tumor wt. compared with controls.  Finally, microarray anal., comparing Y823F/D816V cells with cells expressing c-Kit/D816V, demonstrate that mutation of Y823 causes upregulation of proapoptotic genes, whereas genes of survival pathways are downregulated.  Thus, phosphorylation of Y823 is not necessary for kinase activation, but essential for the transforming ability of the c-Kit/D816V mutant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqntC9Ge4dmtbVg90H21EOLACvtfcHk0liX7wtAaSvb8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVGlsL%252FO&md5=c7a7838ab528b094b441da7035bdeebc</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.383%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwal%26aufirst%3DS.%26aulast%3DKazi%26aufirst%3DJ.%2BU.%26aulast%3DMohlin%26aufirst%3DS.%26aulast%3DP%25C3%25A5hlman%26aufirst%3DS.%26aulast%3DR%25C3%25B6nnstrand%26aufirst%3DL.%26atitle%3DThe%2520Activation%2520Loop%2520Tyrosine%2520823%2520is%2520Essential%2520for%2520the%2520Transforming%2520Capacity%2520of%2520the%2520c-Kit%2520Oncogenic%2520Mutant%2520D816V%26jtitle%3DOncogene%26date%3D2015%26volume%3D34%26issue%3D35%26spage%3D4581%26epage%3D4590%26doi%3D10.1038%2Fonc.2014.383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartig, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Restle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grohmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krausslich, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famulok, M.</span></span> <span> </span><span class="NLM_article-title">Aptamer Displacement Identifies Alternative Small-Molecule Target Sites that Escape Viral Resistance</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">804</span>– <span class="NLM_lpage">812</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2007.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.chembiol.2007.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=17656317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotlSnu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=804-812&issue=7&author=S.+Yamazakiauthor=L.+Tanauthor=G.+Mayerauthor=J.+S.+Hartigauthor=J.-N.+Songauthor=S.+Reuterauthor=T.+Restleauthor=S.+D.+Lauferauthor=D.+Grohmannauthor=H.-G.+Krausslichauthor=J.+Bajorathauthor=M.+Famulok&title=Aptamer+Displacement+Identifies+Alternative+Small-Molecule+Target+Sites+that+Escape+Viral+Resistance&doi=10.1016%2Fj.chembiol.2007.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamer displacement identifies alternative small-molecule target sites that escape viral resistance</span></div><div class="casAuthors">Yamazaki, Satoko; Tan, Lu; Mayer, Guenter; Hartig, Joerg S.; Song, Jin-Na; Reuter, Sandra; Restle, Tobias; Laufer, Sandra D.; Grohmann, Dina; Kraeusslich, Hans-Georg; Bajorath, Juergen; Famulok, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">804-812</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Aptamers targeting reverse transcriptase (RT) from HIV-1 inhibit viral replication in vitro, presumably by competing with binding of the primer/template complex.  This site is not targeted by the currently available small-mol. anti-HIV-1 RT inhibitors.  We have identified SY-3E4, a small-mol. inhibitor of HIV-1 RT, by applying a screening assay that utilizes a reporter-ribozyme regulated by the anti-HIV-1 RT aptamer.  SY-3E4 displaces the aptamer from the protein, selectively inhibits DNA-dependent, but not RNA-dependent, polymerase activity, and inhibits the replication of both the wild-type virus and a multidrug-resistant strain.  Anal. of available structural data of HIV-1 and HIV-2 RTs rationalizes many of the obsd. characteristics of the inhibitory profiles of SY-3E4 and the aptamer and suggests a previously not considered region in these RTs as a target for antiviral therapy.  Our study reveals unexplored ways for rapidly identifying alternative small-mol. target sites in proteins and illustrates strategies for overcoming resistance-conferring mutations with small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeHcTBKxon8bVg90H21EOLACvtfcHk0liX7wtAaSvb8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotlSnu7w%253D&md5=2f60fcb348a9016c75f0488787f43968</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2007.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2007.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DHartig%26aufirst%3DJ.%2BS.%26aulast%3DSong%26aufirst%3DJ.-N.%26aulast%3DReuter%26aufirst%3DS.%26aulast%3DRestle%26aufirst%3DT.%26aulast%3DLaufer%26aufirst%3DS.%2BD.%26aulast%3DGrohmann%26aufirst%3DD.%26aulast%3DKrausslich%26aufirst%3DH.-G.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DAptamer%2520Displacement%2520Identifies%2520Alternative%2520Small-Molecule%2520Target%2520Sites%2520that%2520Escape%2520Viral%2520Resistance%26jtitle%3DChem.%2520Biol.%26date%3D2007%26volume%3D14%26issue%3D7%26spage%3D804%26epage%3D812%26doi%3D10.1016%2Fj.chembiol.2007.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ababneh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alshaer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allozi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahafzah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Khateeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillaireau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noiray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattal, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span> <span> </span><span class="NLM_article-title">In Vitro Selection of Modified RNA Aptamers Against CD44 Cancer Stem Cell Marker</span>. <i>Nucleic Acid Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1089/nat.2013.0423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Fnat.2013.0423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=24171482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFaksLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=401-407&issue=6&author=N.+Ababnehauthor=W.+Alshaerauthor=O.+Alloziauthor=A.+Mahafzahauthor=M.+El-Khateebauthor=H.+Hillaireauauthor=M.+Noirayauthor=E.+Fattalauthor=S.+Ismail&title=In+Vitro+Selection+of+Modified+RNA+Aptamers+Against+CD44+Cancer+Stem+Cell+Marker&doi=10.1089%2Fnat.2013.0423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro Selection of Modified RNA Aptamers Against CD44 Cancer Stem Cell Marker</span></div><div class="casAuthors">Ababneh, Nidaa; Alshaer, Walhan; Allozi, Omar; Mahafzah, Azmi; El-Khateeb, Mohammed; Hillaireau, Herve; Noiray, Magali; Fattal, Elias; Ismail, Said</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acid Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-407</span>CODEN:
                <span class="NLM_cas:coden">NATUCU</span>;
        ISSN:<span class="NLM_cas:issn">2159-3337</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Cancer stem cells (CSCs) are a subset of tumor cells that has the ability to self-renew and to generate the diverse cells that comprise the tumor mass.  The cell-surface glycoprotein CD44 is one of the most common surface markers used to identify CSCs.  Aptamers are synthetic oligonucleotides selected from pools of random sequences that can bind to a wide range of targets with high affinity and specificity.  In this study, the systematic evolution of ligands by exponential enrichment (SELEX) technol. was used to isolate RNA aptamers using human recombinant full-length CD44 protein and 2'-F-pyrimidine modified RNA library with a complexity of around 1014 different mols.  Following 11 iterative rounds of SELEX, the selected aptamers were cloned and sequenced.  Three different sequences were identified.  The binding specificities for one of these RNA aptamers was assessed using representative breast cancer cell lines expressing CD44; namely, MDA-MB-231, MCF7, and T47D.  The selected RNA aptamer (Apt1) was found to interact specifically with such cancer cells when analyzed by flow cytometry and fluorescent microscopy, with different intensities of fluorescence reflecting the level of CD44 expression on the surface of these cells.  It can be concluded that the selected aptamers can be used to target CD44 pos. cells, including cancer stem cells, for detection, sorting, and enrichment and for drug delivery purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTuM3lAiwtS7Vg90H21EOLACvtfcHk0lh9K8jfKRCrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFaksLbM&md5=c1861790a4352a86d5715ce1852d8e33</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1089%2Fnat.2013.0423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fnat.2013.0423%26sid%3Dliteratum%253Aachs%26aulast%3DAbabneh%26aufirst%3DN.%26aulast%3DAlshaer%26aufirst%3DW.%26aulast%3DAllozi%26aufirst%3DO.%26aulast%3DMahafzah%26aufirst%3DA.%26aulast%3DEl-Khateeb%26aufirst%3DM.%26aulast%3DHillaireau%26aufirst%3DH.%26aulast%3DNoiray%26aufirst%3DM.%26aulast%3DFattal%26aufirst%3DE.%26aulast%3DIsmail%26aufirst%3DS.%26atitle%3DIn%2520Vitro%2520Selection%2520of%2520Modified%2520RNA%2520Aptamers%2520Against%2520CD44%2520Cancer%2520Stem%2520Cell%2520Marker%26jtitle%3DNucleic%2520Acid%2520Ther.%26date%3D2013%26volume%3D23%26issue%3D6%26spage%3D401%26epage%3D407%26doi%3D10.1089%2Fnat.2013.0423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kikin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantonio, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagga, P. S.</span></span> <span> </span><span class="NLM_article-title">QGRS Mapper: A Web-based Server for Predicting G-quadruplexes in Nucleotide Sequences</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">W676</span>– <span class="NLM_lpage">W682</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkl253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=16845096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=W676-W682&author=O.+Kikinauthor=L.+Dantonioauthor=P.+S.+Bagga&title=QGRS+Mapper%3A+A+Web-based+Server+for+Predicting+G-quadruplexes+in+Nucleotide+Sequences&doi=10.1093%2Fnar%2Fgkl253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences</span></div><div class="casAuthors">Kikin, Oleg; D'Antonio, Lawrence; Bagga, Paramjeet S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W676-W682</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The quadruplex structures formed by guanine-rich nucleic acid sequences have received significant attention recently because of growing evidence for their role in important biol. processes and as therapeutic targets.  G-quadruplex DNA has been suggested to regulate DNA replication and may control cellular proliferation.  Sequences capable of forming G-quadruplexes in the RNA have been shown to play significant roles in regulation of polyadenylation and splicing events in mammalian transcripts.  Whether quadruplex structure directly plays a role in regulating RNA processing requires investigation.  Computational approaches to study G-quadruplexes allow detailed anal. of mammalian genomes.  There are no known easily accessible user-friendly tools that can compute G-quadruplexes in the nucleotide sequences.  We have developed a web-based server, QGRS Mapper, that predicts quadruplex forming G-rich sequences (QGRS) in nucleotide sequences.  It is a user-friendly application that provides many options for defining and studying G-quadruplexes.  It performs anal. of the user provided genomic sequences, e.g. promoter and telomeric regions, as well as RNA sequences.  It is also useful for predicting G-quadruplex structures in oligonucleotides.  The program provides options to search and retrieve desired gene/nucleotide sequence entries from NCBI databases for mapping G-quadruplexes in the context of RNA processing sites.  This feature is very useful for investigating the functional relevance of G-quadruplex structure, in particular its role in regulating the gene expression by alternative processing.  In addn. to providing data on compn. and locations of QGRS relative to the processing sites in the pre-mRNA sequence, QGRS Mapper features interactive graphic representation of the data.  The user can also use the graphics module to visualize QGRS distribution patterns among all the alternative RNA products of a gene simultaneously on a single screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_yCz4ebPV-bVg90H21EOLACvtfcHk0lh9K8jfKRCrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1yhsLY%253D&md5=e8f69845c3df9217df7f346a06c3db44</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl253%26sid%3Dliteratum%253Aachs%26aulast%3DKikin%26aufirst%3DO.%26aulast%3DDantonio%26aufirst%3DL.%26aulast%3DBagga%26aufirst%3DP.%2BS.%26atitle%3DQGRS%2520Mapper%253A%2520A%2520Web-based%2520Server%2520for%2520Predicting%2520G-quadruplexes%2520in%2520Nucleotide%2520Sequences%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW676%26epage%3DW682%26doi%3D10.1093%2Fnar%2Fgkl253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quang, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestourie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cibiel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducongé, F.</span></span> <span> </span><span class="NLM_article-title">How to Measure the Affinity of Aptamers for Membrane Proteins Expressed on the Surface of Living Adherent Cells</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2015.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.ymeth.2015.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=26589188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyku7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2016&pages=35-43&author=N.+N.+Quangauthor=C.+Pestourieauthor=A.+Cibielauthor=F.+Ducong%C3%A9&title=How+to+Measure+the+Affinity+of+Aptamers+for+Membrane+Proteins+Expressed+on+the+Surface+of+Living+Adherent+Cells&doi=10.1016%2Fj.ymeth.2015.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">How to measure the affinity of aptamers for membrane proteins expressed on the surface of living adherent cells</span></div><div class="casAuthors">Quang, Nam Nguyen; Pestourie, Carine; Cibiel, Agnes; Duconge, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-43</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recently, an increasing no. of aptamers have been selected against biomarkers that are expressed at the surface of cells.  This class of targets, mostly membrane proteins, is in close contact with the intra- and extra-cellular matrixes and their three-dimensional structures are inextricably linked to their inclusion in lipid bilayers.  Therefore, although binding studies can be performed on the isolated form of these proteins, it remains crucial to measure the affinity of these aptamers in a more physiol. environment, i.e., directly on living cells.  Here, we describe a procedure for radioactive binding assays that can be adapted for measuring the affinity of aptamers against different cell lines.  This method has been semi-automated using a liq. handling robot in order to reproducibly measure the apparent dissocn. const. Kd and the apparent no. of targets per cell.  Relevant issues are discussed including the labeling of aptamers, the cells prepn., the incubation, the washings, the use of non-specific competitors, the data anal. and finally the reporting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoSnro5u8pFbVg90H21EOLACvtfcHk0lh9K8jfKRCrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyku7jF&md5=8e4c1f2acc96289f4705e46bfad19bbf</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2015.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2015.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DQuang%26aufirst%3DN.%2BN.%26aulast%3DPestourie%26aufirst%3DC.%26aulast%3DCibiel%26aufirst%3DA.%26aulast%3DDucong%25C3%25A9%26aufirst%3DF.%26atitle%3DHow%2520to%2520Measure%2520the%2520Affinity%2520of%2520Aptamers%2520for%2520Membrane%2520Proteins%2520Expressed%2520on%2520the%2520Surface%2520of%2520Living%2520Adherent%2520Cells%26jtitle%3DMethods%26date%3D2016%26volume%3D97%26spage%3D35%26epage%3D43%26doi%3D10.1016%2Fj.ymeth.2015.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanlikilicer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozpolat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aslan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayraktar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbuz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Aguayo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayraktar, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denizli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Villasana, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokesh, G. L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catuogno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haemmerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorenstein, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volk, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franciscis, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Targeting of AXL Receptor Tyrosine Kinase Inhibits Tumor Growth and Intraperitoneal Metastasis in Ovarian Cancer Models</span>. <i>Mol. Ther.--Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1016/j.omtn.2017.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.omtn.2017.06.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29246304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOnsLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=251-262&author=P.+Kanlikilicerauthor=B.+Ozpolatauthor=B.+Aslanauthor=R.+Bayraktarauthor=N.+Gurbuzauthor=C.+Rodriguez-Aguayoauthor=E.+Bayraktarauthor=M.+Denizliauthor=V.+Gonzalez-Villasanaauthor=C.+Ivanauthor=G.+L.+R.+Lokeshauthor=P.+Ameroauthor=S.+Catuognoauthor=M.+Haemmerleauthor=S.+Wuauthor=R.+Mitraauthor=D.+G.+Gorensteinauthor=D.+E.+Volkauthor=V.+D.+Franciscisauthor=A.+K.+Soodauthor=G.+Lopez-Berestein&title=Therapeutic+Targeting+of+AXL+Receptor+Tyrosine+Kinase+Inhibits+Tumor+Growth+and+Intraperitoneal+Metastasis+in+Ovarian+Cancer+Models&doi=10.1016%2Fj.omtn.2017.06.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Targeting of AXL Receptor Tyrosine Kinase Inhibits Tumor Growth and Intraperitoneal Metastasis in Ovarian Cancer Models</span></div><div class="casAuthors">Kanlikilicer, Pinar; Ozpolat, Bulent; Aslan, Burcu; Bayraktar, Recep; Gurbuz, Nilgun; Rodriguez-Aguayo, Cristian; Bayraktar, Emine; Denizli, Merve; Gonzalez-Villasana, Vianey; Ivan, Cristina; Lokesh, Ganesh L. R.; Amero, Paola; Catuogno, Silvia; Haemmerle, Monika; Wu, Sherry Yen-Yao; Mitra, Rahul; Gorenstein, David G.; Volk, David E.; de Franciscis, Vittorio; Sood, Anil K.; Lopez-Berestein, Gabriel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy--Nucleic Acids</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">251-262</span>CODEN:
                <span class="NLM_cas:coden">MTAOC5</span>;
        ISSN:<span class="NLM_cas:issn">2162-2531</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Despite substantial improvements in the treatment strategies, ovarian cancer is still the most lethal gynecol. malignancy.  Identification of drug treatable therapeutic targets and their safe and effective targeting is crit. to improve patient survival in ovarian cancer.  AXL receptor tyrosine kinase (RTK) has been proposed to be an important therapeutic target for metastatic and advanced-stage human ovarian cancer.  We found that AXL-RTK expression is assocd. with significantly shorter patient survival based on the The Cancer Genome Atlas patient database.  To target AXL-RTK, we developed a chem. modified serum nuclease-stable AXL aptamer (AXL-APTAMER), and we evaluated its in vitro and in vivo antitumor activity using in vitro assays as well as two i.p. animal models.  AXL-aptamer treatment inhibited the phosphorylation and the activity of AXL, impaired the migration and invasion ability of ovarian cancer cells, and led to the inhibition of tumor growth and no. of i.p. metastatic nodules, which was assocd. with the inhibition of AXL activity and angiogenesis in tumors.  When combined with paclitaxel, in vivo systemic (i.v. [i.v.]) administration of AXL-aptamer treatment markedly enhanced the antitumor efficacy of paclitaxel in mice.  Taken together, our data indicate that AXL-aptamers successfully target in vivo AXL-RTK and inhibit its AXL activity and tumor growth and progression, representing a promising strategy for the treatment of ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKE58FM_8OrbVg90H21EOLACvtfcHk0lj0ZWm68Wg72g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOnsLvL&md5=40c844d3d267d23f198a94658b33babe</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.omtn.2017.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.omtn.2017.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DKanlikilicer%26aufirst%3DP.%26aulast%3DOzpolat%26aufirst%3DB.%26aulast%3DAslan%26aufirst%3DB.%26aulast%3DBayraktar%26aufirst%3DR.%26aulast%3DGurbuz%26aufirst%3DN.%26aulast%3DRodriguez-Aguayo%26aufirst%3DC.%26aulast%3DBayraktar%26aufirst%3DE.%26aulast%3DDenizli%26aufirst%3DM.%26aulast%3DGonzalez-Villasana%26aufirst%3DV.%26aulast%3DIvan%26aufirst%3DC.%26aulast%3DLokesh%26aufirst%3DG.%2BL.%2BR.%26aulast%3DAmero%26aufirst%3DP.%26aulast%3DCatuogno%26aufirst%3DS.%26aulast%3DHaemmerle%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DMitra%26aufirst%3DR.%26aulast%3DGorenstein%26aufirst%3DD.%2BG.%26aulast%3DVolk%26aufirst%3DD.%2BE.%26aulast%3DFranciscis%26aufirst%3DV.%2BD.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26atitle%3DTherapeutic%2520Targeting%2520of%2520AXL%2520Receptor%2520Tyrosine%2520Kinase%2520Inhibits%2520Tumor%2520Growth%2520and%2520Intraperitoneal%2520Metastasis%2520in%2520Ovarian%2520Cancer%2520Models%26jtitle%3DMol.%2520Ther.--Nucleic%2520Acids%26date%3D2017%26volume%3D9%26spage%3D251%26epage%3D262%26doi%3D10.1016%2Fj.omtn.2017.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sudeep  Banerjee</span>, <span class="hlFld-ContribAuthor ">Hyunho  Yoon</span>, <span class="hlFld-ContribAuthor ">Mayra  Yebra</span>, <span class="hlFld-ContribAuthor ">Chih-Min  Tang</span>, <span class="hlFld-ContribAuthor ">Mara  Gilardi</span>, <span class="hlFld-ContribAuthor ">Jayanth S.  Shankara Narayanan</span>, <span class="hlFld-ContribAuthor ">Rebekah R.  White</span>, <span class="hlFld-ContribAuthor ">Jason K.  Sicklick</span>, <span class="hlFld-ContribAuthor ">Partha  Ray</span>. </span><span class="cited-content_cbyCitation_article-title">Anti-KIT DNA Aptamer for Targeted Labeling of Gastrointestinal Stromal Tumor. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2020,</strong> <em>19 </em>
                                    (5)
                                     , 1173-1182. <a href="https://doi.org/10.1158/1535-7163.MCT-19-0959" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-19-0959</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-19-0959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-19-0959%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DAnti-KIT%252BDNA%252BAptamer%252Bfor%252BTargeted%252BLabeling%252Bof%252BGastrointestinal%252BStromal%252BTumor%26aulast%3DBanerjee%26aufirst%3DSudeep%26date%3D2020%26date%3D2020%26volume%3D19%26issue%3D5%26spage%3D1173%26epage%3D1182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Predicted secondary structures of candidate anti c-KIT aptamers. Numbers are nucleotide positions relative to the 5′-end of the RNA sequence. Blue highlights indicate nucleotides of the flanking fixed regions; red highlights indicate <b><u class="uu">UUU</u></b> conserved motif among most of the isolated aptamers.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. EMSA of the selected aptamers with the c-KIT kinases using different aptamer/protein mixture ratios: (1) 2:1 ratio; (2) 1:1 ratio; (3) 0.5:1 ratio; (4) 0.25:1 ratio; (F) free aptamer, 1:0 ratio; (U) unbound aptamer; (B) bound aptamer, (NS) negative sequence, and (Wt.) free wild type protein, (Mut.V) free mutant c-KIT<sup>D816V</sup> protein, (Mut.H) free mutant c-KIT<sup>D816H</sup> protein, 400 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding curves of candidate aptamers to c-KIT kinases. The data was fitted to one site specific binding with Hill slope using nonlinear regression analysis, Graphpad prism 6 software. Each data point represents the mean of three independent experiments ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Antagonist inhibition curves of candidate aptamers against all tested c-KIT kinases. Each data point represents the average percentage of inhibition from two independent experiments performed in triplicate. Error bars depict the standard deviation of the mean. Maximum enzyme activity control sample (0% inhibition) contains all the kinase reaction components except the aptamer. For sunitinib, the final DMSO concentration was 1.5%. Data fitting was performed using the three-parameter dose–response equation, GraphPad Prism 6 software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Modes of action (MOAs) of aptamer <b>H5/V36</b> against tested kinases. Panels A, C, and E: Enzyme activity was determined at various ATP concentrations in the presence or absence of the aptamer. Each data point represents the average enzyme activity at each ATP concentration obtained from two independent experiments performed in triplicates. Panels B, D, and F: The Lineweaver–Burk plot of data by using the apparent <i>K</i><sub>m</sub> and the apparent <i>V</i><sub>max</sub> values obtained for each inhibitor concentration. The <i>x</i>-axis values represent (−1/<i>K</i><sub>m</sub>) and the <i>y</i>-axis values represent (1/<i>V</i><sub>max</sub>)(1 + <i>K</i><sub>m</sub>/[ATP minimum]). All data analyses were conducted using GraphPad Prism 6 Software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Antagonist inhibition curves of aptamer <b>H5/V36</b> against both PDGFRβ and JNK3 kinases. Panel A: Data fitting was performed using the three-parameter dose–response equation, GraphPad Prism 6 software; each data point represents the average percentage of inhibition from two independent experiments performed in triplicate. Error bars depict standard deviation of the mean. Panel B: Average inhibition percentages of aptamer <b>H5/V36</b> against all tested kinases at 5 μM final concentration. Data obtained from IC<sub>50</sub> experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Mode of action (MOA) of aptamer <b>V15</b> against mutated c-KIT<sup>D816V</sup>and c-KIT<sup>D816H</sup> kinases. Panels A and C: Enzyme activity was determined at various ATP concentrations in the presence or absence of the aptamer. Each data point represents the average enzyme activity at each ATP concentration obtained from two independent experiments performed in triplicate. Panels B and D: The Lineweaver–Burk plot of data by using the apparent <i>K</i><sub>m</sub> and apparent <i>V</i><sub>max</sub> values obtained for each inhibitor concentration. The <i>x</i>-axis values represent (−1/<i>K</i><sub>m</sub>) and the <i>y</i>-axis values represent (1/<i>V</i><sub>max</sub>)(1 + <i>K</i><sub>m</sub>/[ATP minimum]). All data analyses were conducted using GraphPad Prism 6 Software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Stability of modified <b>H5/V36</b> RNA aptamers against serum nucleases. Aptamer was incubated with fresh human serum for up to 96 h at 37 °C and visualized using 3% agarose gel electrophoresis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/medium/jm9b00868_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Cell viability assays. Cell viability was assessed using MTT assay. NCI-H526 (30000 cells/well in 96-well plates) cells were mock treated or treated with 200 nM final concentration of each candidate aptamer in complex with lipofectamine for 6 h and then stimulated with (A) rhSCF (100 ng/mL) or with (B) 5% FBS for 72 h. The cell viability was expressed as percent of metabolically active cells present after treatment with respect to untreated cells containing only the vehicle. The results from two independent experiments are shown. Data points represent the mean of eight replicates ± SD. <i>P</i> values were determined using two-tailed Student’s <i>t</i> test. *<i>P</i> < 0.05 and **<i>P</i> < 0.01 were regarded as statistically significant when compared with the negative sequence control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00868/20200306/images/large/jm9b00868_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00868&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76533" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76533" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 86 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span> <span> </span><span class="NLM_article-title">The c-Kit Receptor-Mediated Signal Transduction and Tumor-Related Diseases</span>. <i>Int. J. Biol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">443</span>, <span class="refDoi"> DOI: 10.7150/ijbs.6087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.7150%2Fijbs.6087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23678293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1eku7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=435-443&issue=5&author=J.+Liangauthor=Y.+Wuauthor=B.+Chenauthor=W.+Zhangauthor=Y.+Tanakaauthor=H.+Sugiyama&title=The+c-Kit+Receptor-Mediated+Signal+Transduction+and+Tumor-Related+Diseases&doi=10.7150%2Fijbs.6087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The C-kit receptor-mediated signal transduction and tumor-related diseases</span></div><div class="casAuthors">Liang, Jing; Wu, Yan-Ling; Chen, Bing-Jia; Zhang, Wen; Tanaka, Yoshimasa; Sugiyama, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">435-443, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">IJBSB9</span>;
        ISSN:<span class="NLM_cas:issn">1449-2288</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">A review.  As an important member of tyrosine kinase family, c-kit receptor causes specific expression of certain genes, regulates cell differentiation and proliferation, resists cell apoptosis and plays a key role in tumor occurrence, development, migration and recurrence through activating the downstream signaling mols. following interaction with stem cell factor (SCF).  The abnormality of SCF/c-kit signaling pathway is closely related to some certain tumors.  The discovery of c-kit receptor-targeted drugs has promoted clin.-related cancer's diagnosis and treatment.  In this paper, we review recent research progress on c-kit receptor-mediated signal transduction and its potential therapeutic application as a target in tumor-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJDVYBTh98p7Vg90H21EOLACvtfcHk0lidcSvDW3YVhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1eku7rI&md5=1ccfda0e33eadaa4462fbd241fb81e1d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.7150%2Fijbs.6087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fijbs.6087%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DSugiyama%26aufirst%3DH.%26atitle%3DThe%2520c-Kit%2520Receptor-Mediated%2520Signal%2520Transduction%2520and%2520Tumor-Related%2520Diseases%26jtitle%3DInt.%2520J.%2520Biol.%2520Sci.%26date%3D2013%26volume%3D9%26issue%3D5%26spage%3D435%26epage%3D443%26doi%3D10.7150%2Fijbs.6087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">Structure and Regulation of Kit Protein-Tyrosine Kinase-The Stem Cell Factor Receptor</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>338</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1307</span>– <span class="NLM_lpage">1315</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2005.09.150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.bbrc.2005.09.150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=16226710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1egs77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=338&publication_year=2005&pages=1307-1315&issue=3&author=R.+Roskoski&title=Structure+and+Regulation+of+Kit+Protein-Tyrosine+Kinase-The+Stem+Cell+Factor+Receptor&doi=10.1016%2Fj.bbrc.2005.09.150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and regulation of Kit protein-tyrosine kinase-The stem cell factor receptor</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">338</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1307-1315</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Signaling by stem cell factor and Kit, its receptor, play important roles in gametogenesis, hematopoiesis, mast cell development and function, and melanogenesis.  Moreover, human and mouse embryonic stem cells express Kit transcripts.  Stem cell factor exists as both a sol. and a membrane-bound glycoprotein while Kit is a glycoprotein receptor protein-tyrosine kinase.  The complete absence of stem cell factor or Kit is lethal.  Gain-of-function mutations of Kit are assocd. with several human neoplasms including acute myelogenous leukemia, gastrointestinal stromal tumors, mastocytomas, and nasal T-cell lymphomas.  Binding of stem cell factor to Kit results in receptor dimerization and activation of protein kinase activity.  The activated receptor becomes autophosphorylated at tyrosine residues that serve as docking sites for signal transduction mols. contg. SH2 domains.  Kit activates Akt, Src family kinases, phosphatidylinositol 3-kinase, phospholipase Cγ, and Ras/mitogen-activated protein kinases.  Kit exists in active and inactive conformations as detd. by X-ray crystallog.  Kit consists of an extracellular domain, a transmembrane segment, a juxtamembrane domain, and a protein kinase domain that contains an insert of about 80 amino acid residues.  The juxtamembrane domain inhibits enzyme activity in cis by maintaining the control αC-helix and the activation loop in their inactive conformations.  The juxtamembrane domain also inhibits receptor dimerization.  STI-571, a clin. effective targeted protein-tyrosine kinase inhibitor, binds to an inactive conformation of Kit.  The majority of human gastrointestinal stromal tumors have Kit gain-of-function mutations in the juxtamembrane domain, and most people with these tumors respond to STI-571.  STI-571 binds to Kit and Bcr-Abl (the oncoprotein of chronic myelogenous leukemia) at their ATP-binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_HWJXdp926bVg90H21EOLACvtfcHk0lidcSvDW3YVhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1egs77N&md5=e71ef40ef41dfd2d2500558ee65cc4aa</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2005.09.150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2005.09.150%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DStructure%2520and%2520Regulation%2520of%2520Kit%2520Protein-Tyrosine%2520Kinase-The%2520Stem%2520Cell%2520Factor%2520Receptor%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2005%26volume%3D338%26issue%3D3%26spage%3D1307%26epage%3D1315%26doi%3D10.1016%2Fj.bbrc.2005.09.150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">The Role of Small Molecule Kit Protein-Tyrosine Kinase Inhibitors in the Treatment of Neoplastic Disorders</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2018.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.phrs.2018.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29704617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptVagsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2018&pages=35-52&author=R.+Roskoski&title=The+Role+of+Small+Molecule+Kit+Protein-Tyrosine+Kinase+Inhibitors+in+the+Treatment+of+Neoplastic+Disorders&doi=10.1016%2Fj.phrs.2018.04.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The role of small molecule Kit protein-tyrosine kinase inhibitors in the treatment of neoplastic disorders</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-52</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Kit proto-oncogene was found as a consequence of the discovery of the feline v-kit sarcoma oncogene.  Stem cell factor (SCF) is the Kit ligand and it mediates Kit dimerization and activation.  The Kit receptor contains an extracellular segment that is made up of five Ig-like domains (D1/2/3/4/5), a transmembrane segment, a juxtamembrane segment, a protein-tyrosine kinase domain that contains an insert of 77 amino acid residues, and a carboxyterminal tail.  Activating somatic mutations in Kit have been documented in various neoplasms including gastrointestinal stromal tumors (GIST), mast cell overexpression (systemic mastocytosis), core-binding factor acute myeloid leukemias (AML), melanomas, and seminomas.  In the case of gastrointestinal stromal tumors, most activating mutations occur in the juxtamembrane segment and these mutants are initially sensitive to imatinib.  As with many targeted anticancer drugs, resistance to Kit antagonists occurs in about two years and is the result of secondary KIT mutations.  An activation segment exon 17 D816V mutation is one of the more common resistance mutations in Kit and this mutant is resistant to imatinib and sorafenib.  Type I protein kinase inhibitors interact with the active enzyme form with DFG-D of the proximal activation segment directed inward toward the active site (DFG-Din).  In contrast, type II inhibitors bind to their target with the DFG-D pointing away from the active site (DFG-Dout).  Based upon the X-ray crystallog. structures, imatinib, sunitinib, and ponatinib are Type II Kit inhibitors.  We used the Schrodinger induced fit docking protocol to model the interaction of midostaurin with Kit and the result indicates that it binds to the DFG-Din conformation of the receptor and is thus classified as type I inhibitor.  This medication inhibits the notoriously resistant Kit D816V mutant and is approved for the treatment of systemic mastocytosis and is effective against tumors bearing the D816V activation/resistance mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqteI4_qt5heLVg90H21EOLACvtfcHk0lgOYweNNezW_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptVagsb4%253D&md5=8a71d3b7399d53c3085fb04a87aef1a2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2018.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2018.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DThe%2520Role%2520of%2520Small%2520Molecule%2520Kit%2520Protein-Tyrosine%2520Kinase%2520Inhibitors%2520in%2520the%2520Treatment%2520of%2520Neoplastic%2520Disorders%26jtitle%3DPharmacol.%2520Res.%26date%3D2018%26volume%3D133%26spage%3D35%26epage%3D52%26doi%3D10.1016%2Fj.phrs.2018.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ségaliny, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tellez-Gabriel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymann, D.</span></span> <span> </span><span class="NLM_article-title">Receptor Tyrosine Kinases: Characterisation, Mechanism of Action and Therapeutic Interests for Bone Cancers</span>. <i>Journal of Bone Oncology</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1016/j.jbo.2015.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.jbo.2015.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=26579483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A280%3ADC%252BC28rgtlyjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=1-12&issue=1&author=A.+I.+S%C3%A9galinyauthor=M.+Tellez-Gabrielauthor=M.+Heymannauthor=D.+Heymann&title=Receptor+Tyrosine+Kinases%3A+Characterisation%2C+Mechanism+of+Action+and+Therapeutic+Interests+for+Bone+Cancers&doi=10.1016%2Fj.jbo.2015.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor tyrosine kinases: Characterisation, mechanism of action and therapeutic interests for bone cancers</span></div><div class="casAuthors">Segaliny Aude I; Tellez-Gabriel Marta; Heymann Marie-Francoise; Heymann Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of bone oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>
        ISSN:<span class="NLM_cas:issn">2212-1366</span>.
    </div><div class="casAbstract">Bone cancers are characterised by the development of tumour cells in bone sites, associated with a dysregulation of their environment.  In the last two decades, numerous therapeutic strategies have been developed to target the cancer cells or tumour niche.  As the crosstalk between these two entities is tightly controlled by the release of polypeptide mediators activating signalling pathways through several receptor tyrosine kinases (RTKs), RTK inhibitors have been designed.  These inhibitors have shown exciting clinical impacts, such as imatinib mesylate, which has become a reference treatment for chronic myeloid leukaemia and gastrointestinal tumours.  The present review gives an overview of the main molecular and functional characteristics of RTKs, and focuses on the clinical applications that are envisaged and already assessed for the treatment of bone sarcomas and bone metastases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-qtzzXrDFl42y4bB4mnvYfW6udTcc2ebaBzLliY3N9bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rgtlyjtg%253D%253D&md5=57563bfe33ca5752242d56cc24546c1c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.jbo.2015.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbo.2015.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A9galiny%26aufirst%3DA.%2BI.%26aulast%3DTellez-Gabriel%26aufirst%3DM.%26aulast%3DHeymann%26aufirst%3DM.%26aulast%3DHeymann%26aufirst%3DD.%26atitle%3DReceptor%2520Tyrosine%2520Kinases%253A%2520Characterisation%252C%2520Mechanism%2520of%2520Action%2520and%2520Therapeutic%2520Interests%2520for%2520Bone%2520Cancers%26jtitle%3DJournal%2520of%2520Bone%2520Oncology%26date%3D2015%26volume%3D4%26issue%3D1%26spage%3D1%26epage%3D12%26doi%3D10.1016%2Fj.jbo.2015.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hojjat-Farsangi, M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors of the Receptor Tyrosine Kinases: Promising Tools for Targeted Cancer Therapies</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">13768</span>– <span class="NLM_lpage">13801</span>, <span class="refDoi"> DOI: 10.3390/ijms150813768</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.3390%2Fijms150813768" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=25110867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslyqu7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=13768-13801&issue=8&author=M.+Hojjat-Farsangi&title=Small-Molecule+Inhibitors+of+the+Receptor+Tyrosine+Kinases%3A+Promising+Tools+for+Targeted+Cancer+Therapies&doi=10.3390%2Fijms150813768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of the receptor tyrosine kinases: promising tools for targeted cancer therapies</span></div><div class="casAuthors">Hojjat-Farsangi, Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">13768-13801, 34</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Chemotherapeutic and cytotoxic drugs are widely used in the treatment of cancer.  In spite of the improvements in the life quality of patients, their effectiveness is compromised by several disadvantages.  This represents a demand for developing new effective strategies with focusing on tumor cells and min. side effects.  Targeted cancer therapies and personalized medicine have been defined as a new type of emerging treatments.  Small mol. inhibitors (SMIs) are among the most effective drugs for targeted cancer therapy.  The growing no. of approved SMIs of receptor tyrosine kinases (RTKs) i.e., tyrosine kinase inhibitors (TKIs) in the clin. oncol. imply the increasing attention and application of these therapeutic tools.  Most of the current approved RTK-TKIs in preclin. and clin. settings are multi-targeted inhibitors with several side effects.  Only a few specific/selective RTK-TKIs have been developed for the treatment of cancer patients.  Specific/selective RTK-TKIs have shown less deleterious effects compared to multi-targeted inhibitors.  This review intends to highlight the importance of specific/selective TKIs for future development with less side effects and more manageable agents.  This article provides an overview of: (1) the characteristics and function of RTKs and TKIs; (2) the recent advances in the improvement of specific/selective RTK-TKIs in preclin. or clin. settings; and (3) emerging RTKs for targeted cancer therapies by TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0oFHs0APBNrVg90H21EOLACvtfcHk0liKrpJuaStdqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslyqu7zI&md5=08836c25f587f44f2843abc6b2d3587f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fijms150813768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms150813768%26sid%3Dliteratum%253Aachs%26aulast%3DHojjat-Farsangi%26aufirst%3DM.%26atitle%3DSmall-Molecule%2520Inhibitors%2520of%2520the%2520Receptor%2520Tyrosine%2520Kinases%253A%2520Promising%2520Tools%2520for%2520Targeted%2520Cancer%2520Therapies%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2014%26volume%3D15%26issue%3D8%26spage%3D13768%26epage%3D13801%26doi%3D10.3390%2Fijms150813768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mol, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheibe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. P.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for The Autoinhibition and STI-571 Inhibition of c-Kit Tyrosine Kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">31655</span>– <span class="NLM_lpage">31663</span>, <span class="refDoi"> DOI: 10.1074/jbc.M403319200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1074%2Fjbc.M403319200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=15123710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlslOrtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=31655-31663&author=C.+Molauthor=D.+Douganauthor=T.+Schneiderauthor=R.+Skeneauthor=M.+Krausauthor=D.+Scheibeauthor=G.+P.+Snellauthor=H.+Zouauthor=B.+C.+Sangauthor=K.+P.+Wilson&title=Structural+Basis+for+The+Autoinhibition+and+STI-571+Inhibition+of+c-Kit+Tyrosine+Kinase&doi=10.1074%2Fjbc.M403319200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Autoinhibition and STI-571 Inhibition of c-Kit Tyrosine Kinase</span></div><div class="casAuthors">Mol, Clifford D.; Dougan, Douglas R.; Schneider, Thomas R.; Skene, Robert J.; Kraus, Michelle L.; Scheibe, Daniel N.; Snell, Gyorgy P.; Zou, Hua; Sang, Bi-Ching; Wilson, Keith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">31655-31663</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activity of the c-Kit receptor protein-tyrosine kinase is tightly regulated in normal cells, whereas deregulated c-Kit kinase activity is implicated in the pathogenesis of human cancers.  The c-Kit juxtamembrane region is known to have an autoinhibitory function; however the precise mechanism by which c-Kit is maintained in an autoinhibited state is not known.  We report the 1.9-Å resoln. crystal structure of native c-Kit kinase in an autoinhibited conformation and compare it with active c-Kit kinase.  Autoinhibited c-Kit is stabilized by the juxtamembrane domain, which inserts into the kinase-active site and disrupts formation of the activated structure.  A 1.6-Å crystal structure of c-Kit in complex with STI-571 (Imatinib or Gleevec) demonstrates that inhibitor binding disrupts this natural mechanism for maintaining c-Kit in an autoinhibited state.  Together, these results provide a structural basis for understanding c-Kit kinase autoinhibition and will facilitate the structure-guided design of specific inhibitors that target the activated and autoinhibited conformations of c-Kit kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWJsKx1IzXQbVg90H21EOLACvtfcHk0lg-WTfj_rHGdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlslOrtrc%253D&md5=df6065a62b44e4c2e6b020da0f797ab4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M403319200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M403319200%26sid%3Dliteratum%253Aachs%26aulast%3DMol%26aufirst%3DC.%26aulast%3DDougan%26aufirst%3DD.%26aulast%3DSchneider%26aufirst%3DT.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DKraus%26aufirst%3DM.%26aulast%3DScheibe%26aufirst%3DD.%26aulast%3DSnell%26aufirst%3DG.%2BP.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DSang%26aufirst%3DB.%2BC.%26aulast%3DWilson%26aufirst%3DK.%2BP.%26atitle%3DStructural%2520Basis%2520for%2520The%2520Autoinhibition%2520and%2520STI-571%2520Inhibition%2520of%2520c-Kit%2520Tyrosine%2520Kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D31655%26epage%3D31663%26doi%3D10.1074%2Fjbc.M403319200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mol, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowakowski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassel, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcree, D. E.</span></span> <span> </span><span class="NLM_article-title">Structure of a c-Kit Product Complex Reveals the Basis for Kinase Transactivation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span> (<span class="NLM_issue">34</span>),  <span class="NLM_fpage">31461</span>– <span class="NLM_lpage">31464</span>, <span class="refDoi"> DOI: 10.1074/jbc.C300186200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1074%2Fjbc.C300186200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=12824176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVOnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=31461-31464&issue=34&author=C.+D.+Molauthor=K.+B.+Limauthor=V.+Sridharauthor=H.+Zouauthor=E.+Y.+Chienauthor=B.+Sangauthor=J.+Nowakowskiauthor=D.+B.+Kasselauthor=C.+N.+Croninauthor=D.+E.+Mcree&title=Structure+of+a+c-Kit+Product+Complex+Reveals+the+Basis+for+Kinase+Transactivation&doi=10.1074%2Fjbc.C300186200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a c-Kit Product Complex Reveals the Basis for Kinase Transactivation</span></div><div class="casAuthors">Mol, Clifford D.; Lim, Kheng B.; Sridhar, Vandana; Zou, Hua; Chien, Ellen Y. T.; Sang, Bi-Ching; Nowakowski, Jacek; Kassel, Daniel B.; Cronin, Ciaran N.; McRee, Duncan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">31461-31464</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The c-Kit proto-oncogene is a receptor protein-tyrosine kinase assocd. with several highly malignant human cancers.  Upon binding its ligand, stem cell factor (SCF), c-Kit forms an active dimer that autophosphorylates itself and activates a signaling cascade that induces cell growth.  Disease-causing human mutations that activate SCF-independent constitutive expression of c-Kit are found in acute myelogenous leukemia, human mast cell disease, and gastrointestinal stromal tumors.  We report on the phosphorylation state and crystal structure of a c-Kit product complex.  The c-Kit structure is in a fully active form, with ordered kinase activation and phosphate-binding loops.  These results provide key insights into the mol. basis for c-Kit kinase transactivation to assist in the design of new competitive inhibitors targeting activated mutant forms of c-Kit that are resistant to current chemotherapy regimes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv0eZJ9UHNILVg90H21EOLACvtfcHk0lg-WTfj_rHGdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVOnsb0%253D&md5=f537b5c9903adb6cc057406bd20f0f5a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C300186200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C300186200%26sid%3Dliteratum%253Aachs%26aulast%3DMol%26aufirst%3DC.%2BD.%26aulast%3DLim%26aufirst%3DK.%2BB.%26aulast%3DSridhar%26aufirst%3DV.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DChien%26aufirst%3DE.%2BY.%26aulast%3DSang%26aufirst%3DB.%26aulast%3DNowakowski%26aufirst%3DJ.%26aulast%3DKassel%26aufirst%3DD.%2BB.%26aulast%3DCronin%26aufirst%3DC.%2BN.%26aulast%3DMcree%26aufirst%3DD.%2BE.%26atitle%3DStructure%2520of%2520a%2520c-Kit%2520Product%2520Complex%2520Reveals%2520the%2520Basis%2520for%2520Kinase%2520Transactivation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26issue%3D34%26spage%3D31461%26epage%3D31464%26doi%3D10.1074%2Fjbc.C300186200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heldin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennartsson, J.</span></span> <span> </span><span class="NLM_article-title">Structural and Functional Properties of Platelet-Derived Growth Factor and Stem Cell Factor Receptors</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">8</span>), <span class="NLM_elocation-id">a009100</span> <span class="refDoi"> DOI: 10.1101/cshperspect.a009100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1101%2Fcshperspect.a009100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23906712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslGhsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&issue=8&author=C.+Heldinauthor=J.+Lennartsson&title=Structural+and+Functional+Properties+of+Platelet-Derived+Growth+Factor+and+Stem+Cell+Factor+Receptors&doi=10.1101%2Fcshperspect.a009100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional properties of platelet-derived growth factor and stem cell factor receptors</span></div><div class="casAuthors">Heldin, Carl-Henrik; Lennartsson, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">A009100/1-A009100/19</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The receptors for platelet-derived growth factor (PDGF) and stem cell factor (SCF) are members of the type III class of PTK receptors, which are characterized by five Ig-like domains extracellularly and a split kinase domain intracellularly.  The receptors are activated by ligand-induced dimerization, leading to autophosphorylation on specific tyrosine residues.  Thereby the kinase activities of the receptors are activated and docking sites for downstream SH2 domain signal transduction mols. are created; activation of these pathways promotes cell growth, survival, and migration.  These receptors mediate important signals during the embryonal development, and control tissue homeostasis in the adult.  Their over-activity is seen in malignancies and other diseases involving excessive cell proliferation, such as atherosclerosis and fibrotic diseases.  In cancer, mutations of PDGF and SCF receptors-including gene fusions, point mutations, and amplifications-drive subpopulations of certain malignancies, such as gastrointestinal stromal tumors, chronic myelomonocytic leukemia, hypereosinophilic syndrome, glioblastoma, acute myeloid leukemia, mastocytosis, and melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAYrCdbqmo8rVg90H21EOLACvtfcHk0lil2aZPsPvk9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslGhsLfF&md5=7fb28447c9fefd66a2e429668dd62884</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a009100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a009100%26sid%3Dliteratum%253Aachs%26aulast%3DHeldin%26aufirst%3DC.%26aulast%3DLennartsson%26aufirst%3DJ.%26atitle%3DStructural%2520and%2520Functional%2520Properties%2520of%2520Platelet-Derived%2520Growth%2520Factor%2520and%2520Stem%2520Cell%2520Factor%2520Receptors%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2013%26volume%3D5%26issue%3D8%26doi%3D10.1101%2Fcshperspect.a009100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cruse, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalfe, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivera, A.</span></span> <span> </span><span class="NLM_article-title">Functional Deregulation of KIT Link to Mast Cell Proliferative Diseases and Other Neoplasms</span>. <i>Immunol. Allergy Clin. North Am.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1016/j.iac.2014.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.iac.2014.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=24745671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A280%3ADC%252BC2cjltlCktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=219-237&issue=2&author=G.+Cruseauthor=D.+D.+Metcalfeauthor=A.+Olivera&title=Functional+Deregulation+of+KIT+Link+to+Mast+Cell+Proliferative+Diseases+and+Other+Neoplasms&doi=10.1016%2Fj.iac.2014.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Functional deregulation of KIT: link to mast cell proliferative diseases and other neoplasms</span></div><div class="casAuthors">Cruse Glenn; Metcalfe Dean D; Olivera Ana</div><div class="citationInfo"><span class="NLM_cas:title">Immunology and allergy clinics of North America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">219-37</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this review, the authors discuss common gain-of-function mutations in the stem cell factor receptor KIT found in mast cell proliferation disorders and summarize the current understanding of the molecular mechanisms by which these transforming mutations may affect KIT structure and function leading to altered downstream signaling and cellular transformation.  Drugs targeting KIT have shown mixed success in the treatment of mastocytosis and other hyperproliferative diseases.  A brief overview of the most common KIT inhibitors currently used, the reasons for the varied clinical results of such inhibitors and a discussion of potential new strategies are provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSnziUwFm7S_7YtC3U5jeOifW6udTcc2eZG9vhpRLRznbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cjltlCktg%253D%253D&md5=81bbecd6b864653328f5e39b041a86ad</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.iac.2014.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.iac.2014.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DCruse%26aufirst%3DG.%26aulast%3DMetcalfe%26aufirst%3DD.%2BD.%26aulast%3DOlivera%26aufirst%3DA.%26atitle%3DFunctional%2520Deregulation%2520of%2520KIT%2520Link%2520to%2520Mast%2520Cell%2520Proliferative%2520Diseases%2520and%2520Other%2520Neoplasms%26jtitle%3DImmunol.%2520Allergy%2520Clin.%2520North%2520Am.%26date%3D2014%26volume%3D34%26issue%3D2%26spage%3D219%26epage%3D237%26doi%3D10.1016%2Fj.iac.2014.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lennartsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rönnstrand, L.</span></span> <span> </span><span class="NLM_article-title">Stem Cell Factor Receptor/c-Kit: From Basic Science to Clinical Implications</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1619</span>– <span class="NLM_lpage">1649</span>, <span class="refDoi"> DOI: 10.1152/physrev.00046.2011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1152%2Fphysrev.00046.2011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23073628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2lsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=1619-1649&issue=4&author=J.+Lennartssonauthor=L.+R%C3%B6nnstrand&title=Stem+Cell+Factor+Receptor%2Fc-Kit%3A+From+Basic+Science+to+Clinical+Implications&doi=10.1152%2Fphysrev.00046.2011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Stel cell factor receptor/c-Kit: from basic science to clinical implications</span></div><div class="casAuthors">Lennartsson, Johan; Roennstrand, Lars</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1619-1649</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  Stem cell factor (SCF) is a dimeric mol. that exerts its biol. functions by binding to and activating the receptor tyrosine kinase c-Kit.  Activation of c-Kit leads to its autophosphorylation and initiation of signal transduction.  Signaling proteins are recruited to activated c-Kit by certain interaction domains (e.g., SH2 and PTB) that specifically bind to phosphorylated tyrosine residues in the intracellular region of c-Kit.  Activation of c-Kit signaling has been found to mediate cell survival, migration, and proliferation depending on the cell type.  Signaling from c-Kit is crucial for normal hematopoiesis, pigmentation, fertility, gut movement, and some aspects of the nervous system.  Deregulated c-Kit kinase activity has been found in a no. of pathol. conditions, including cancer and allergy.  The observation that gain-of-function mutations in c-Kit can promote tumor formation and progression has stimulated the development of therapeutics agents targeting this receptor, e.g., the clin. used inhibitor imatinib mesylate.  Also other clin. used multiselective kinase inhibitors, for instance, sorafenib and sunitinib, have c-Kit included in their range of targets.  Furthermore, loss-of-function mutations in c-Kit have been obsd. and shown to give rise to a condition called piebaldism.  This review provides a summary of our current knowledge regarding structural and functional aspects of c-Kit signaling both under normal and pathol. conditions, as well as advances in the development of low-mol.-wt. mols. inhibiting c-Kit function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiAi_V1Wivd7Vg90H21EOLACvtfcHk0lhzTTK415HbpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2lsbnK&md5=b874bfb881938c505b6718732950be51</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00046.2011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00046.2011%26sid%3Dliteratum%253Aachs%26aulast%3DLennartsson%26aufirst%3DJ.%26aulast%3DR%25C3%25B6nnstrand%26aufirst%3DL.%26atitle%3DStem%2520Cell%2520Factor%2520Receptor%252Fc-Kit%253A%2520From%2520Basic%2520Science%2520to%2520Clinical%2520Implications%26jtitle%3DPhysiol.%2520Rev.%26date%3D2012%26volume%3D92%26issue%3D4%26spage%3D1619%26epage%3D1649%26doi%3D10.1152%2Fphysrev.00046.2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abbaspour
Babaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamalidehghan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleem, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman Huri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmadipour, F.</span></span> <span> </span><span class="NLM_article-title">Receptor Tyrosine Kinase (c-Kit) Inhibitors: A Potential Therapeutic Target in Cancer Cells</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2443</span>– <span class="NLM_lpage">2459</span>, <span class="refDoi"> DOI: 10.2147/DDDT.S89114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.2147%2FDDDT.S89114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=27536065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A280%3ADC%252BC2szhvV2guw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2016&pages=2443-2459&author=M.+Abbaspour%0ABabaeiauthor=B.+Kamalidehghanauthor=M.+Saleemauthor=H.+Zaman+Huriauthor=F.+Ahmadipour&title=Receptor+Tyrosine+Kinase+%28c-Kit%29+Inhibitors%3A+A+Potential+Therapeutic+Target+in+Cancer+Cells&doi=10.2147%2FDDDT.S89114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor tyrosine kinase (c-Kit) inhibitors: a potential therapeutic target in cancer cells</span></div><div class="casAuthors">Abbaspour Babaei Maryam; Ahmadipour Fatemeh; Kamalidehghan Behnam; Saleem Mohammad; Huri Hasniza Zaman</div><div class="citationInfo"><span class="NLM_cas:title">Drug design, development and therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2443-59</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">c-Kit, a receptor tyrosine kinase, is involved in intracellular signaling, and the mutated form of c-Kit plays a crucial role in occurrence of some cancers.  The function of c-Kit has led to the concept that inhibiting c-Kit kinase activity can be a target for cancer therapy.  The promising results of inhibition of c-Kit for treatment of cancers have been observed in some cancers such as gastrointestinal stromal tumor, acute myeloid leukemia, melanoma, and other tumors, and these results have encouraged attempts toward improvement of using c-Kit as a capable target for cancer therapy.  This paper presents the findings of previous studies regarding c-Kit as a receptor tyrosine kinase and an oncogene, as well as its gene targets and signaling pathways in normal and cancer cells.  The c-Kit gene location, protein structure, and the role of c-Kit in normal cell have been discussed.  Comprehending the molecular mechanism underlying c-Kit-mediated tumorogenesis is consequently essential and may lead to the identification of future novel drug targets.  The potential mechanisms by which c-Kit induces cellular transformation have been described.  This study aims to elucidate the function of c-Kit for future cancer therapy.  In addition, it has c-Kit inhibitor drug properties and their functions have been listed in tables and demonstrated in schematic pictures.  This review also has collected previous studies that targeted c-Kit as a novel strategy for cancer therapy.  This paper further emphasizes the advantages of this approach, as well as the limitations that must be addressed in the future.  Finally, although c-Kit is an attractive target for cancer therapy, based on the outcomes of treatment of patients with c-Kit inhibitors, it is unlikely that Kit inhibitors alone can lead to cure.  It seems that c-Kit mutations alone are not sufficient for tumorogenesis, but do play a crucial role in cancer occurrence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfTnK2L7idhRwLZxei1HN8fW6udTcc2ebBXEueob-kYLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szhvV2guw%253D%253D&md5=c45d232471e049dc8d42a98fc5aa1d76</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S89114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S89114%26sid%3Dliteratum%253Aachs%26aulast%3DAbbaspour%2BBabaei%26aufirst%3DM.%26aulast%3DKamalidehghan%26aufirst%3DB.%26aulast%3DSaleem%26aufirst%3DM.%26aulast%3DZaman%2BHuri%26aufirst%3DH.%26aulast%3DAhmadipour%26aufirst%3DF.%26atitle%3DReceptor%2520Tyrosine%2520Kinase%2520%2528c-Kit%2529%2520Inhibitors%253A%2520A%2520Potential%2520Therapeutic%2520Target%2520in%2520Cancer%2520Cells%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2016%26volume%3D10%26spage%3D2443%26epage%3D2459%26doi%3D10.2147%2FDDDT.S89114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garner, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozgit, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vodala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrock, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serrano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span> <span> </span><span class="NLM_article-title">Ponatinib Inhibits Polyclonal Drug-Resistant KIT Oncoproteins and Shows Therapeutic Potential in Heavily Pretreated Gastrointestinal Stromal Tumor (GIST) Patients</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">5745</span>– <span class="NLM_lpage">5755</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-1397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1158%2F1078-0432.CCR-14-1397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=25239608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGgsL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=5745-5755&issue=22&author=A.+P.+Garnerauthor=J.+M.+Gozgitauthor=R.+Anjumauthor=S.+Vodalaauthor=A.+Schrockauthor=T.+Zhouauthor=C.+Serranoauthor=G.+Eilersauthor=M.+Zhuauthor=J.+Ketzerauthor=S.+Wardwellauthor=Y.+Ningauthor=Y.+Songauthor=A.+Kohlmannauthor=F.+Wangauthor=T.+Clacksonauthor=M.+C.+Heinrichauthor=J.+A.+Fletcherauthor=S.+Bauerauthor=V.+M.+Rivera&title=Ponatinib+Inhibits+Polyclonal+Drug-Resistant+KIT+Oncoproteins+and+Shows+Therapeutic+Potential+in+Heavily+Pretreated+Gastrointestinal+Stromal+Tumor+%28GIST%29+Patients&doi=10.1158%2F1078-0432.CCR-14-1397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Ponatinib Inhibits Polyclonal Drug-Resistant KIT Oncoproteins and Shows Therapeutic Potential in Heavily Pretreated Gastrointestinal Stromal Tumor (GIST) Patients</span></div><div class="casAuthors">Garner, Andrew P.; Gozgit, Joseph M.; Anjum, Rana; Vodala, Sadanand; Schrock, Alexa; Zhou, Tianjun; Serrano, Cesar; Eilers, Grant; Zhu, Meijun; Ketzer, Julia; Wardwell, Scott; Ning, Yaoyu; Song, Youngchul; Kohlmann, Anna; Wang, Frank; Clackson, Tim; Heinrich, Michael C.; Fletcher, Jonathan A.; Bauer, Sebastian; Rivera, Victor M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5745-5755</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: KIT is the major oncogenic driver of gastrointestinal stromal tumors (GIST).  Imatinib, sunitinib, and regorafenib are approved therapies; however, efficacy is often limited by the acquisition of polyclonal secondary resistance mutations in KIT, with those located in the activation (A) loop (exons 17/18) being particularly problematic.  Here, we explore the KIT-inhibitory activity of ponatinib in preclin. models and describe initial characterization of its activity in patients with GIST.  Exptl. Design: The cellular and in vivo activities of ponatinib, imatinib, sunitinib, and regorafenib against mutant KIT were evaluated using an accelerated mutagenesis assay and a panel of engineered and GIST-derived cell lines.  The ponatinib-KIT costructure was also detd.  The clin. activity of ponatinib was examd. in three patients with GIST previously treated with all three FDA-approved agents.  Results: In engineered and GIST-derived cell lines, ponatinib potently inhibited KIT exon 11 primary mutants and a range of secondary mutants, including those within the A-loop.  Ponatinib also induced regression in engineered and GIST-derived tumor models contg. these secondary mutations.  In a mutagenesis screen, 40 nmol/L ponatinib was sufficient to suppress outgrowth of all secondary mutants except V654A, which was suppressed at 80 nmol/L.  This inhibitory profile could be rationalized on the basis of structural analyses.  Ponatinib (30 mg daily) displayed encouraging clin. activity in two of three patients with GIST.  Conclusion:Ponatinib possesses potent activity against most major clin. relevant KIT mutants and has demonstrated preliminary evidence of activity in patients with refractory GIST.  These data strongly support further evaluation of ponatinib in patients with GIST.  Clin Cancer Res; 20(22); 5745-55. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTGG18CSl3XLVg90H21EOLACvtfcHk0lg9XvLSd4x1cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGgsL7E&md5=7537e45f37546f9e0cb799bdf05ed847</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1397%26sid%3Dliteratum%253Aachs%26aulast%3DGarner%26aufirst%3DA.%2BP.%26aulast%3DGozgit%26aufirst%3DJ.%2BM.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DVodala%26aufirst%3DS.%26aulast%3DSchrock%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DSerrano%26aufirst%3DC.%26aulast%3DEilers%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DM.%26aulast%3DKetzer%26aufirst%3DJ.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26aulast%3DFletcher%26aufirst%3DJ.%2BA.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26atitle%3DPonatinib%2520Inhibits%2520Polyclonal%2520Drug-Resistant%2520KIT%2520Oncoproteins%2520and%2520Shows%2520Therapeutic%2520Potential%2520in%2520Heavily%2520Pretreated%2520Gastrointestinal%2520Stromal%2520Tumor%2520%2528GIST%2529%2520Patients%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26issue%3D22%26spage%3D5745%26epage%3D5755%26doi%3D10.1158%2F1078-0432.CCR-14-1397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DiNitto, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worrall, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">English, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. C.</span></span> <span> </span><span class="NLM_article-title">Function of Activation Loop Tyrosine Phosphorylation in the Mechanism of c-Kit Auto-activation and its Implication in Sunitinib Resistance</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>147</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">601</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1093/jb/mvq015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fjb%2Fmvq015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=20147452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkt1Oqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2010&pages=601-609&issue=4&author=J.+P.+DiNittoauthor=G.+D.+Deshmukhauthor=Y.+Zhangauthor=S.+L.+Jacquesauthor=R.+Coliauthor=J.+W.+Worrallauthor=W.+Diehlauthor=J.+M.+Englishauthor=J.+C.+Wu&title=Function+of+Activation+Loop+Tyrosine+Phosphorylation+in+the+Mechanism+of+c-Kit+Auto-activation+and+its+Implication+in+Sunitinib+Resistance&doi=10.1093%2Fjb%2Fmvq015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Function of activation loop tyrosine phosphorylation in the mechanism of c-Kit auto-activation and its implication in sunitinib resistance</span></div><div class="casAuthors">DiNitto, Jonathan P.; Deshmukh, Gayatri D.; Zhang, Yan; Jacques, Suzanne L.; Coli, Rocco; Worrall, Joseph W.; Diehl, Wade; English, Jessie M.; Wu, Joe C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">601-609</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">The activation of receptor tyrosine kinases (RTKs) is tightly regulated through a variety of mechanisms.  Kinetic studies show that activation of c-Kit RTK occurs through an inter-mol. autophosphorylation.  Phosphopeptide mapping of c-Kit reveals that 14-22 phosphates are added to each mol of wild-type (WT) c-Kit during the activation.  Phosphorylation sites are found on the JM, kinase insert (KID), c-terminal domains and the activation loop (A-loop), but only the sites on the JM domain contribute to the kinase activation.  The A-loop tyrosine (Y823) is not phosphorylated until very late in the activation (>90% completion), indicating that the A-loop phosphorylation is not required for c-Kit activation.  A sunitinib-resistant mutant D816H that accelerates auto-activation by 184-fold shows no phosphorylation on the A-loop tyrosine after full activation.  A loss-of-phosphorylation mutation Y823F remains fully competent in auto-activation.  Similar to WT and D816H, the unactivated Y823F mutant binds sunitinib and imatinib with high affinity (KD = 5.9 nM).  But unlike the WT and D816H where the activated enzymes lose the ability to bind the two drugs, activated Y823F binds the two inhibitors effectively.  These observations suggest that the A-loop of activated Y823F remains flexible and can readily adopt unactivated conformations to accommodate DFG-out binders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLl2NrvMAaC7Vg90H21EOLACvtfcHk0lgEi-zH2h0aCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkt1Oqurw%253D&md5=f3bcc8c20cbba329d295196983a19468</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvq015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvq015%26sid%3Dliteratum%253Aachs%26aulast%3DDiNitto%26aufirst%3DJ.%2BP.%26aulast%3DDeshmukh%26aufirst%3DG.%2BD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DColi%26aufirst%3DR.%26aulast%3DWorrall%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DW.%26aulast%3DEnglish%26aufirst%3DJ.%2BM.%26aulast%3DWu%26aufirst%3DJ.%2BC.%26atitle%3DFunction%2520of%2520Activation%2520Loop%2520Tyrosine%2520Phosphorylation%2520in%2520the%2520Mechanism%2520of%2520c-Kit%2520Auto-activation%2520and%2520its%2520Implication%2520in%2520Sunitinib%2520Resistance%26jtitle%3DJ.%2520Biochem.%26date%3D2010%26volume%3D147%26issue%3D4%26spage%3D601%26epage%3D609%26doi%3D10.1093%2Fjb%2Fmvq015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, X.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanism of D816X Mutation-Induced c-Kit Activation and Mediated Inhibitor Resistance in Gastrointestinal Stromal Tumor</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2018.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.jmgm.2018.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=30015051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlensbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2018&pages=189-196&author=H.+Jiangauthor=W.+Shaoauthor=Y.+Wangauthor=R.+Xuauthor=L.+Zhouauthor=X.+Mu&title=Molecular+Mechanism+of+D816X+Mutation-Induced+c-Kit+Activation+and+Mediated+Inhibitor+Resistance+in+Gastrointestinal+Stromal+Tumor&doi=10.1016%2Fj.jmgm.2018.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanism of D816X mutation-induced c-Kit activation and -mediated inhibitor resistance in gastrointestinal stromal tumor</span></div><div class="casAuthors">Jiang, Haohai; Shao, Weiwei; Wang, Yuanjin; Xu, Ronghui; Zhou, Linsen; Mu, Xiangming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">189-196</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The D816X (X = V, H, Y or F) missense mutation constitutively activates c-Kit kinase in gastrointestinal stromal tumor (GIST) and has been obsd. to cause acquired resistance against first-line and second-line kinase inhibitors.  In the present study, the allosteric mechanism of D816X-induced c-Kit conformational change is investigated at mol. level.  The Asp816 residue is located at the activation loop (A-loop) of c-Kit and the mutation can eliminate a neg. formal charge from the loop region by substituting the acidic asparagic acid residue with neutral valine, histidine, tyrosine or phenylalanine.  Here, we classify the c-Kit kinase into four states in terms of its mutation (wild type or mutant) and conformation (DFG-in or DFG-out).  The wild-type kinase is electrostatically stabilized in inactive DFG-out conformation, whereas the D816X mutation can promote the conformational conversion to active DFG-in and then activate the kinase.  Structural anal. reveals that the Asp816 residue in DFG-out is surrounded by a no. of polar and pos. charged residues within its first and second shells of protein context, and kinase conformational change to DFG-in brings this residue into a neg. electrostatic potential environment.  Dynamics simulation characterizes that the c-Kit conformational conversion from DFG-out to DFG-in can cause local unfavorable effect to type-II inhibitor, while the mutation-induced global structural rearrangement would participate in the favorable interaction of c-Kit with type-I inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc7O_8k3wISLVg90H21EOLACvtfcHk0ljgnmTRA27Y6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlensbvI&md5=c76cac50df01c3a25d0939955f75dc14</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2018.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2018.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DMu%26aufirst%3DX.%26atitle%3DMolecular%2520Mechanism%2520of%2520D816X%2520Mutation-Induced%2520c-Kit%2520Activation%2520and%2520Mediated%2520Inhibitor%2520Resistance%2520in%2520Gastrointestinal%2520Stromal%2520Tumor%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2018%26volume%3D84%26spage%3D189%26epage%3D196%26doi%3D10.1016%2Fj.jmgm.2018.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klug, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. C.</span></span> <span> </span><span class="NLM_article-title">Structural and Clinical Consequences of Activation Loop Mutations in Class III Receptor Tyrosine Kinases</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2018.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.pharmthera.2018.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29964125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1CjtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2018&pages=123-134&author=L.+R.+Klugauthor=J.+D.+Kentauthor=M.+C.+Heinrich&title=Structural+and+Clinical+Consequences+of+Activation+Loop+Mutations+in+Class+III+Receptor+Tyrosine+Kinases&doi=10.1016%2Fj.pharmthera.2018.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and clinical consequences of activation loop mutations in class III receptor tyrosine kinases</span></div><div class="casAuthors">Klug, Lillian R.; Kent, Jason D.; Heinrich, Michael C.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">123-134</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Mutations within the activation loop of members of the class III receptor tyrosine kinase (RTK) subfamily, which includes KIT, PDGFRA, and FLT3, have been obsd. in multiple human tumor types.  These mutations confer constitutive activation as well as resistance to the type II tyrosine kinase inhibitors (TKI) that are currently clin. available, such as imatinib and sunitinib.  It is now understood that activation loop mutations in class III RTKs shift the activation state equil. away from inactive states, to which type II TKIs bind, to the active state by destabilizing the inactive conformation.  Recently, type I TKIs, which can bind to active kinase conformations, have been developed with specificity for class III RTK members.  Some type I TKIs, such as crenolanib and avapritinib (BLU-285), have entered clin. studies for patients with activation loop mutations in KIT, PDGFRA, or FLT3.  Preliminary results suggest that these type I TKIs show activity in these patient populations that previously lacked effective treatments.  This article reviews the inactive and active structures of KIT, PDGFRA, and FLT3, how the mutations seen in human cancers affect kinase structure, and the clin. implications of these mutations in terms of type I vs. type II TKI binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRH_W_T1s9lLVg90H21EOLACvtfcHk0ljgnmTRA27Y6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1CjtL7F&md5=168ec705a8e0c19c3191255739beb483</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2018.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2018.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DKlug%26aufirst%3DL.%2BR.%26aulast%3DKent%26aufirst%3DJ.%2BD.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26atitle%3DStructural%2520and%2520Clinical%2520Consequences%2520of%2520Activation%2520Loop%2520Mutations%2520in%2520Class%2520III%2520Receptor%2520Tyrosine%2520Kinases%26jtitle%3DPharmacol.%2520Ther.%26date%3D2018%26volume%3D191%26spage%3D123%26epage%3D134%26doi%3D10.1016%2Fj.pharmthera.2018.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikemori-Kawada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jestel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">König, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funahashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuruoka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, J.</span></span> <span> </span><span class="NLM_article-title">Distinct Binding Mode of Multikinase Inhibitor Lenvatinib Revealed by Biochemical Characterization</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1021/ml500394m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500394m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=89-94&issue=1&author=K.+Okamotoauthor=M.+Ikemori-Kawadaauthor=A.+Jestelauthor=K.+V.+K%C3%B6nigauthor=Y.+Funahashiauthor=T.+Matsushimaauthor=A.+Tsuruokaauthor=A.+Inoueauthor=J.+Matsui&title=Distinct+Binding+Mode+of+Multikinase+Inhibitor+Lenvatinib+Revealed+by+Biochemical+Characterization&doi=10.1021%2Fml500394m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct Binding Mode of Multikinase Inhibitor Lenvatinib Revealed by Biochemical Characterization</span></div><div class="casAuthors">Okamoto, Kiyoshi; Ikemori-Kawada, Megumi; Jestel, Anja; von Konig, Konstanze; Funahashi, Yasuhiro; Matsushima, Tomohiro; Tsuruoka, Akihiko; Inoue, Atsushi; Matsui, Junji</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-94</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lenvatinib is an oral multikinase inhibitor that selectively inhibits vascular endothelial growth factor (VEGF) receptors 1 to 3 and other proangiogenic and oncogenic pathway-related receptor tyrosine kinases.  To elucidate the origin of the potency of lenvatinib in VEGF receptor 2 (VEGFR2) inhibition, we conducted a kinetic interaction anal. of lenvatinib with VEGFR2 and x-ray anal. of the crystal structure of VEGFR2-lenvatinib complexes.  Kinetic anal. revealed that lenvatinib had a rapid assocn. rate const. and a relatively slow dissocn. rate const. in complex with VEGFR2.  Co-crystal structure anal. demonstrated that lenvatinib binds at its ATP mimetic quinoline moiety to the ATP binding site and to the neighboring region via a cyclopropane ring, adopting an Asp-Phe-Gly (DFG)-"in" conformation.  These results suggest that lenvatinib is very distinct in its binding mode of interaction compared to the several approved VEGFR2 kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUysiHRGw3P7Vg90H21EOLACvtfcHk0liIlHRPRxhmjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtb7E&md5=5461832fad0577f2ea93b47521b4bbdc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fml500394m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500394m%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DIkemori-Kawada%26aufirst%3DM.%26aulast%3DJestel%26aufirst%3DA.%26aulast%3DK%25C3%25B6nig%26aufirst%3DK.%2BV.%26aulast%3DFunahashi%26aufirst%3DY.%26aulast%3DMatsushima%26aufirst%3DT.%26aulast%3DTsuruoka%26aufirst%3DA.%26aulast%3DInoue%26aufirst%3DA.%26aulast%3DMatsui%26aufirst%3DJ.%26atitle%3DDistinct%2520Binding%2520Mode%2520of%2520Multikinase%2520Inhibitor%2520Lenvatinib%2520Revealed%2520by%2520Biochemical%2520Characterization%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26issue%3D1%26spage%3D89%26epage%3D94%26doi%3D10.1021%2Fml500394m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotlib, J.</span></span> <span> </span><span class="NLM_article-title">Clinical Validation of KIT Inhibition in Advanced Systemic Mastocytosis</span>. <i>Current Hematologic Malignancy Reports</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">416</span>, <span class="refDoi"> DOI: 10.1007/s11899-018-0469-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1007%2Fs11899-018-0469-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=30155614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ivVKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=407-416&issue=5&author=J.+H.+Bairdauthor=J.+Gotlib&title=Clinical+Validation+of+KIT+Inhibition+in+Advanced+Systemic+Mastocytosis&doi=10.1007%2Fs11899-018-0469-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Validation of KIT Inhibition in Advanced Systemic Mastocytosis</span></div><div class="casAuthors">Baird John H; Gotlib Jason</div><div class="citationInfo"><span class="NLM_cas:title">Current hematologic malignancy reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">407-416</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PURPOSE OF REVIEW:  We discuss recent developments in the treatment of advanced systemic mastocytosis (advSM) with inhibitors of the KIT receptor tyrosine kinase.  RECENT FINDINGS:  advSM is a heterogeneous group of neoplasms of poor prognosis characterized by the accumulation of neoplastic mast cells.  The canonical KIT D816V mutation is present in approximately 90% of SM patients, and its detection is critical for both diagnosis and therapeutic decision-making.  The multikinase/KIT inhibitor midostaurin was recently approved for advSM.  This agent can reverse SM-related organ damage and disease symptoms, and decrease the bone marrow mast cell burden and splenomegaly.  However, complete remissions are rare and durability of responses is variable.  Potent and selective KIT D816V inhibitors including avapritinib (BLU-285) and DCC-2618 have entered clinical trials, and rational combination strategies are under development.  The clinical efficacy of KIT inhibitors validate KIT as a key oncogenic driver in mast cell neoplasms.  An improved understanding of the genetic heterogeneity beyond KIT will help inform the dynamics of response and relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRASjErjCIGasSzjBFCN0BJfW6udTcc2eZgprcW0cPRa7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ivVKlsQ%253D%253D&md5=f0854dc0fa929a50cdfa3fb54907e368</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs11899-018-0469-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11899-018-0469-3%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DJ.%2BH.%26aulast%3DGotlib%26aufirst%3DJ.%26atitle%3DClinical%2520Validation%2520of%2520KIT%2520Inhibition%2520in%2520Advanced%2520Systemic%2520Mastocytosis%26jtitle%3DCurrent%2520Hematologic%2520Malignancy%2520Reports%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3D407%26epage%3D416%26doi%3D10.1007%2Fs11899-018-0469-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schittenhelm, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiraga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbin, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bokemeyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druker, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. C.</span></span> <span> </span><span class="NLM_article-title">Dasatinib (BMS-354825), a Dual SRC/ABL Kinase Inhibitor, Inhibits the Kinase Activity of Wild-Type, Juxtamembrane, and Activation Loop Mutant KIT Isoforms Associated with Human Malignancies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-05-2050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1158%2F0008-5472.CAN-05-2050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=16397263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFWltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=473-481&issue=1&author=M.+M.+Schittenhelmauthor=S.+Shiragaauthor=A.+Schroederauthor=A.+S.+Corbinauthor=D.+Griffithauthor=F.+Y.+Leeauthor=C.+Bokemeyerauthor=M.+W.+N.+Deiningerauthor=B.+J.+Drukerauthor=M.+C.+Heinrich&title=Dasatinib+%28BMS-354825%29%2C+a+Dual+SRC%2FABL+Kinase+Inhibitor%2C+Inhibits+the+Kinase+Activity+of+Wild-Type%2C+Juxtamembrane%2C+and+Activation+Loop+Mutant+KIT+Isoforms+Associated+with+Human+Malignancies&doi=10.1158%2F0008-5472.CAN-05-2050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Dasatinib (BMS-354825), a Dual SRC/ABL Kinase Inhibitor, Inhibits the Kinase Activity of Wild-Type, Juxtamembrane, and Activation Loop Mutant KIT Isoforms Associated with Human Malignancies</span></div><div class="casAuthors">Schittenhelm, Marcus M.; Shiraga, Sharon; Schroeder, Arin; Corbin, Amie S.; Griffith, Diana; Lee, Francis Y.; Bokemeyer, Carsten; Deininger, Michael W. N.; Druker, Brian J.; Heinrich, Michael C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Activating mutations of the activation loop of KIT are assocd. with certain human neoplasms, including the majority of patients with systemic mast cell disorders, as well as cases of seminoma, acute myelogenous leukemia (AML), and gastrointestinal stromal tumors (GISTs).  The small-mol. tyrosine kinase inhibitor imatinib mesylate is a potent inhibitor of wild-type (WT) KIT and certain mutant KIT isoforms and has become the std. of care for treating patients with metastatic GIST.  However, KIT activation loop mutations involving codon D816 that are typically found in AML, systemic mastocytosis, and seminoma are insensitive to imatinib mesylate (IC50 > 5-10 μmol/L), and acquired KIT activation loop mutations can be assocd. with imatinib mesylate resistance in GIST.  Dasatinib (formerly BMS-354825) is a small-mol., ATP-competitive inhibitor of SRC and ABL tyrosine kinases with potency in the low nanomolar range.  Some small-mol. SRC/ABL inhibitors also have potency against WT KIT kinase.  Therefore, the authors hypothesized that dasatinib might inhibit the kinase activity of both WT and mutant KIT isoforms.  The authors report herein that dasatinib potently inhibits WT KIT and juxtamembrane domain mutant KIT autophosphorylation and KIT-dependent activation of downstream pathways important for cell viability and cell survival, such as Ras/mitogen-activated protein kinase, phosphoinositide 3-kinase/Akt, and Janus-activated kinase/signal transducers and activators of transcription.  Furthermore, dasatinib is a potent inhibitor of imatinib-resistant KIT activation loop mutants and induces apoptosis in mast cell and leukemic cell lines expressing these mutations (potency against KIT D816Y » D816F > D816V).  Our studies suggest that dasatinib may have clin. efficacy against human neoplasms that are assocd. with gain-of-function KIT mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryNXWvlK-abLVg90H21EOLACvtfcHk0lj8-lVnQIeDsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFWltw%253D%253D&md5=dbd2d8af79e727c22197f7a273b45b41</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-2050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-2050%26sid%3Dliteratum%253Aachs%26aulast%3DSchittenhelm%26aufirst%3DM.%2BM.%26aulast%3DShiraga%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DA.%26aulast%3DCorbin%26aufirst%3DA.%2BS.%26aulast%3DGriffith%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DF.%2BY.%26aulast%3DBokemeyer%26aufirst%3DC.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%2BN.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26atitle%3DDasatinib%2520%2528BMS-354825%2529%252C%2520a%2520Dual%2520SRC%252FABL%2520Kinase%2520Inhibitor%252C%2520Inhibits%2520the%2520Kinase%2520Activity%2520of%2520Wild-Type%252C%2520Juxtamembrane%252C%2520and%2520Activation%2520Loop%2520Mutant%2520KIT%2520Isoforms%2520Associated%2520with%2520Human%2520Malignancies%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26issue%3D1%26spage%3D473%26epage%3D481%26doi%3D10.1158%2F0008-5472.CAN-05-2050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span> <span> </span><span class="NLM_article-title">Development and Biological Evaluation of Potent and Selective c-KITD816V Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6428</span>– <span class="NLM_lpage">6443</span>, <span class="refDoi"> DOI: 10.1021/jm500413g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500413g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFajsbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6428-6443&author=S.+Leeauthor=H.+Leeauthor=J.+Kimauthor=S.+Leeauthor=S.+J.+Kimauthor=B.+S.+Choiauthor=S.+Hongauthor=S.+Hong&title=Development+and+Biological+Evaluation+of+Potent+and+Selective+c-KITD816V+Inhibitors&doi=10.1021%2Fjm500413g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Biological Evaluation of Potent and Selective c-KITD816V Inhibitors</span></div><div class="casAuthors">Lee, Soyoung; Lee, Hyunseung; Kim, Jinhee; Lee, Suhyun; Kim, Soo Jung; Choi, Byong-Seok; Hong, Soon-Sun; Hong, Sungwoo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6428-6443</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The c-KIT tyrosine kinase has emerged as a potential therapeutic target for an array of diseases.  However, there exists a drug resistance that is caused by mutations in c-KIT; therefore, c-KIT remains as a clin. challenge due to limited effective treatment options for therapies.  For example, the acquired activating point mutation D816V significantly impairs the efficacy of targeted cancer therapies.  Understanding the mechanisms of drug resistance at the mol. level will aid in designing and developing particular inhibitors with the potential to overcome these resistance mutations.  The authors undertake a structure-based de novo design of 7-azaindole as the mol. core using the modified scoring function.  This approach led to an identification of new c-KIT inhibitors over 100-fold specific for the D816V mutant relative to the wild-type c-KIT with nanomolar inhibitory activity.  More importantly, these compds. potently inhibit clin. relevant D816V mutations of c-KIT in biochem. and cellular studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbp-ykbRTq7LVg90H21EOLACvtfcHk0lj8-lVnQIeDsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFajsbjK&md5=5b368e9fa0eb3c9d5ad5a69439347c81</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm500413g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500413g%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DChoi%26aufirst%3DB.%2BS.%26aulast%3DHong%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520Biological%2520Evaluation%2520of%2520Potent%2520and%2520Selective%2520c-KITD816V%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6428%26epage%3D6443%26doi%3D10.1021%2Fjm500413g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavrin, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiah, E.</span></span> <span> </span><span class="NLM_article-title">Approaches to Discover Non-ATP Site Kinase Inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1039/C2MD20180A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1039%2FC2MD20180A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=41-51&issue=1&author=L.+K.+Gavrinauthor=E.+Saiah&title=Approaches+to+Discover+Non-ATP+Site+Kinase+Inhibitors&doi=10.1039%2FC2MD20180A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to discover non-ATP site kinase inhibitors</span></div><div class="casAuthors">Gavrin, Lori Krim; Saiah, Eddine</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-51</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The catalytic domain of kinases shows a high degree of sequence homol., esp. for kinases that belong to the same family.  They share a common ATP binding site with a conserved activation loop and similar three-dimensional structure.  Consequently, a major challenge in kinase research exists in achieving selectivity among the >500 family members, since they all process the same substrate.  In addn. to requiring selectivity against other kinases, ATP site inhibitors must also bind tightly to overcome the high physiol. concn. of ATP in the cell.  Furthermore, the development of novel ATP site inhibitors is becoming increasingly challenging, as many ATP competitive scaffolds have previously been disclosed.  In order to develop compds. with better selectivity among kinases, inhibitors that bind outside the ATP site show great promise and are currently being explored by many groups.  This review will highlight the most commonly used methods to discover small mol. Type III and IV kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xJcf4-RJ9rVg90H21EOLACvtfcHk0lgTLRMuvwdE6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N&md5=97acf2247e5516c1649c1d964c867656</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC2MD20180A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20180A%26sid%3Dliteratum%253Aachs%26aulast%3DGavrin%26aufirst%3DL.%2BK.%26aulast%3DSaiah%26aufirst%3DE.%26atitle%3DApproaches%2520to%2520Discover%2520Non-ATP%2520Site%2520Kinase%2520Inhibitors%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26issue%3D1%26spage%3D41%26epage%3D51%26doi%3D10.1039%2FC2MD20180A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span> <span> </span><span class="NLM_article-title">Nucleic Acid Aptamers: An Emerging Frontier in Cancer Therapy</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">85</span>),  <span class="NLM_fpage">10472</span>, <span class="refDoi"> DOI: 10.1039/c2cc35042d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1039%2Fc2cc35042d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22951893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCqtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=10472&issue=85&author=G.+Zhuauthor=M.+Yeauthor=M.+J.+Donovanauthor=E.+Songauthor=Z.+Zhaoauthor=W.+Tan&title=Nucleic+Acid+Aptamers%3A+An+Emerging+Frontier+in+Cancer+Therapy&doi=10.1039%2Fc2cc35042d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleic acid aptamers: an emerging frontier in cancer therapy</span></div><div class="casAuthors">Zhu, Guizhi; Ye, Mao; Donovan, Michael J.; Song, Erqun; Zhao, Zilong; Tan, Weihong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">85</span>),
    <span class="NLM_cas:pages">10472-10480</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The last 2 decades have witnessed the development and application of nucleic acid aptamers in a variety of fields, including target anal., disease therapy, and mol. and cellular engineering.  The efficient and widely applicable aptamer selection, reproducible chem. synthesis and modification, generally impressive target binding selectivity and affinity, relatively rapid tissue penetration, low immunogenicity, and rapid systemic clearance make aptamers ideal recognition elements for use as therapeutics or for in vivo delivery of therapeutics.  In this feature article, the authors discuss the development and biomedical application of nucleic acid aptamers, with emphasis on cancer cell aptamer isolation, targeted cancer therapy, oncol. biomarker identification and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM2VW-GB3T7LVg90H21EOLACvtfcHk0lh8o6s1PJVIEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCqtLfO&md5=f4d934441bc5a9f75ca21eb30fa53e12</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2Fc2cc35042d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2cc35042d%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DDonovan%26aufirst%3DM.%2BJ.%26aulast%3DSong%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DNucleic%2520Acid%2520Aptamers%253A%2520An%2520Emerging%2520Frontier%2520in%2520Cancer%2520Therapy%26jtitle%3DChem.%2520Commun.%26date%3D2012%26volume%3D48%26issue%3D85%26spage%3D10472%26doi%3D10.1039%2Fc2cc35042d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clary, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigner, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullenger, B. A.</span></span> <span> </span><span class="NLM_article-title">Aptamers Selected Against the Unglycosylated EGFRvIII Ectodomain and Delivered Intracellularly Reduce Membrane-bound EGFRvIII and Induce Apoptosis</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1515/BC.2009.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1515%2FBC.2009.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=19040357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVajsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=137-144&issue=2&author=Y.+Liuauthor=C.+Kuanauthor=J.+Miauthor=X.+Zhangauthor=B.+M.+Claryauthor=D.+D.+Bignerauthor=B.+A.+Sullenger&title=Aptamers+Selected+Against+the+Unglycosylated+EGFRvIII+Ectodomain+and+Delivered+Intracellularly+Reduce+Membrane-bound+EGFRvIII+and+Induce+Apoptosis&doi=10.1515%2FBC.2009.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamers selected against the unglycosylated EGFRvIII ectodomain and delivered intracellularly reduce membrane-bound EGFRvIII and induce apoptosis</span></div><div class="casAuthors">Liu, Yingmiao; Kuan, Chien-Tsun; Mi, Jing; Zhang, Xiuwu; Clary, Bryan M.; Bigner, Darell D.; Sullenger, Bruce A.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">137-144</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor variant III (EGFRvIII) is a glycoprotein uniquely expressed in glioblastoma, but not in normal brain tissues.  To develop targeted therapies for brain tumors, the authors selected RNA aptamers against the histidine-tagged EGFRvIII ectodomain, using an Escherichia coli system for protein expression and purifn.  Representative aptamer E21 has a dissocn. const. (Kd) of 33 × 10-9 M, and exhibits high affinity and specificity for EGFRvIII in ELISA and surface plasmon resonance assays.  However, selected aptamers cannot bind the same protein expressed from eukaryotic cells because glycosylation, a post-translational modification present only in eukaryotic systems, significantly alters the structure of the target protein.  By transfecting EGFRvIII aptamers into cells, it was found that membrane-bound, glycosylated EGFRvIII is reduced and the percentage of cells undergoing apoptosis is increased.  It is postulated that transfected aptamers can interact with newly synthesized EGFRvIII, disrupt proper glycosylation, and reduce the amt. of mature EGFRvIII reaching the cell surface.  This work establishes the feasibility of disrupting protein post-translational modifications in situ with aptamers.  This finding is useful for elucidating the function of proteins of interest with various modifications, as well as dissecting signal transduction pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVzTIzjghlJrVg90H21EOLACvtfcHk0lh8o6s1PJVIEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVajsrc%253D&md5=f0c266ecc883b4b2e1bfa7cd228ace6a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1515%2FBC.2009.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2009.022%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKuan%26aufirst%3DC.%26aulast%3DMi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DClary%26aufirst%3DB.%2BM.%26aulast%3DBigner%26aufirst%3DD.%2BD.%26aulast%3DSullenger%26aufirst%3DB.%2BA.%26atitle%3DAptamers%2520Selected%2520Against%2520the%2520Unglycosylated%2520EGFRvIII%2520Ectodomain%2520and%2520Delivered%2520Intracellularly%2520Reduce%2520Membrane-bound%2520EGFRvIII%2520and%2520Induce%2520Apoptosis%26jtitle%3DBiol.%2520Chem.%26date%3D2009%26volume%3D390%26issue%3D2%26spage%3D137%26epage%3D144%26doi%3D10.1515%2FBC.2009.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keranen, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellington, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newton, A. C.</span></span> <span> </span><span class="NLM_article-title">Isozyme-specific Inhibition of Protein Kinase C by RNA Aptamer</span>. <i>The Journal of Biological Chemistry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>269</i></span> (<span class="NLM_issue">51</span>),  <span class="NLM_fpage">32051</span>– <span class="NLM_lpage">32054</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=7528207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2MXitFSlu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=32051-32054&issue=51&author=R.+Conradauthor=L.+M.+Keranenauthor=A.+D.+Ellingtonauthor=A.+C.+Newton&title=Isozyme-specific+Inhibition+of+Protein+Kinase+C+by+RNA+Aptamer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Isoenzyme-specific inhibition of protein kinase C by RNA aptamers</span></div><div class="casAuthors">Conrad, Rick; Keranen, Lisa M.; Ellington, Andrew D.; Newton, Alexandra C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">32051-4</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In vitro selection technol. has been used to purify RNA aptamers from a random sequence pool that can bind to, and specifically inhibit, protein kinase C βII.  Two of the selected RNA aptamers bind to this isoenzyme of protein kinase C with nanomolar affinities and inhibit activation with unprecedented selectivity; the highly related, alternatively spliced βI isoenzyme, which differs by 23 residues, is inhibited with 1 order of magnitude lower potency; the next most similar isoenzyme, α, shows no detectable inhibition.  The prodn. of isoenzyme-specific inhibitors of protein kinase C opens the possibilities for dissecting the roles of specific protein kinase Cs in the myriad of intracellular signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf4rqMQJk8PLVg90H21EOLACvtfcHk0liVSPf93_GkBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXitFSlu78%253D&md5=31b82ff246a981ed692d451122631589</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DConrad%26aufirst%3DR.%26aulast%3DKeranen%26aufirst%3DL.%2BM.%26aulast%3DEllington%26aufirst%3DA.%2BD.%26aulast%3DNewton%26aufirst%3DA.%2BC.%26atitle%3DIsozyme-specific%2520Inhibition%2520of%2520Protein%2520Kinase%2520C%2520by%2520RNA%2520Aptamer%26jtitle%3DThe%2520Journal%2520of%2520Biological%2520Chemistry%26date%3D1994%26volume%3D269%26issue%3D51%26spage%3D32051%26epage%3D32054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreasen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kjems, J.</span></span> <span> </span><span class="NLM_article-title">Characterisation of Aptamer–Target Interactions by Branched Selection and High-throughput Sequencing of SELEX Pools</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">21</span>), <span class="NLM_elocation-id">e139</span> <span class="refDoi"> DOI: 10.1093/nar/gkv700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkv700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=26163061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFCrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&issue=21&author=D.+M.+Dupontauthor=N.+Larsenauthor=J.+K.+Jensenauthor=P.+A.+Andreasenauthor=J.+Kjems&title=Characterisation+of+Aptamer%E2%80%93Target+Interactions+by+Branched+Selection+and+High-throughput+Sequencing+of+SELEX+Pools&doi=10.1093%2Fnar%2Fgkv700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Characterisation of aptamer-target interactions by branched selection and high-throughput sequencing of SELEX pools</span></div><div class="casAuthors">Dupont, Daniel M.; Larsen, Niels; Jensen, Jan. K.; Andreasen, Peter A.; Kjems, Joergen</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">e139/1-e139/13</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Nucleic acid aptamer selection by systematic evolution of ligands by exponential enrichment (SELEX) has shown great promise for use in the development of research tools, therapeutics and diagnostics.  Typically, aptamers are identified from libraries contg. up to 1016 different RNA or DNA sequences by 5-70 rounds of affinity selection towards a target of interest.  Such library screenings can result in complex pools of many target-binding aptamers.  New high-throughput sequencing techniques may potentially revolutionize aptamer selection by allowing quant. assessment of the dynamic changes in the pool compn. during the SELEX process and by facilitating large-scale post-SELEX characterization.  In the present study, we demonstrate how high-throughput sequencing of SELEX pools, before and after a single round of branched selection for binding to different target variants, can provide detailed information about aptamer binding sites, preferences for specific target conformations, and functional effects of the aptamers.  The procedure was applied on a diverse pool of 2'-fluoropyrimidine-modified RNA enriched for aptamers specific for the serpin plasminogen activator inhibitor-1 (PAI-1) through five rounds of std. selection.  The results demonstrate that it is possible to perform large-scale detailed characterization of aptamer sequences directly in the complex pools obtained from library selection methods, thus without the need to produce individual aptamers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqxqLR88dkfbVg90H21EOLACvtfcHk0lh__wX6jR3QQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFCrsLc%253D&md5=ee2b04c641854ebd211c64eb5585b096</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv700%26sid%3Dliteratum%253Aachs%26aulast%3DDupont%26aufirst%3DD.%2BM.%26aulast%3DLarsen%26aufirst%3DN.%26aulast%3DJensen%26aufirst%3DJ.%2BK.%26aulast%3DAndreasen%26aufirst%3DP.%2BA.%26aulast%3DKjems%26aufirst%3DJ.%26atitle%3DCharacterisation%2520of%2520Aptamer%25E2%2580%2593Target%2520Interactions%2520by%2520Branched%2520Selection%2520and%2520High-throughput%2520Sequencing%2520of%2520SELEX%2520Pools%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3D21%26doi%3D10.1093%2Fnar%2Fgkv700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruigrok, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duijn, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barendregt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoltenburg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strehlitz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levisson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smidt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oost, J. V.</span></span> <span> </span><span class="NLM_article-title">Kinetic and Stoichiometric Characterisation of Streptavidin-Binding Aptamers</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">829</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1002/cbic.201100774</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fcbic.201100774" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22416028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1yjur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=829-836&issue=6&author=V.+J.+Ruigrokauthor=E.+V.+Duijnauthor=A.+Barendregtauthor=K.+Dyerauthor=J.+A.+Tainerauthor=R.+Stoltenburgauthor=B.+Strehlitzauthor=M.+Levissonauthor=H.+Smidtauthor=J.+V.+Oost&title=Kinetic+and+Stoichiometric+Characterisation+of+Streptavidin-Binding+Aptamers&doi=10.1002%2Fcbic.201100774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and Stoichiometric Characterisation of Streptavidin-Binding Aptamers</span></div><div class="casAuthors">Ruigrok, Vincent J. B.; van Duijn, Esther; Barendregt, Arjan; Dyer, Kevin; Tainer, John A.; Stoltenburg, Regina; Strehlitz, Beate; Levisson, Mark; Smidt, Hauke; van der Oost, John</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">829-836</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Aptamers are oligonucleotide ligands that are selected for high-affinity binding to mol. targets.  Only limited knowledge relating to relations between structural and kinetic properties that define aptamer-target interactions is available.  To this end, streptavidin-binding aptamers were isolated and characterized by distinct anal. techniques.  Binding kinetics of five broadly similar aptamers were detd. by surface plasmon resonance (SPR); affinities ranged from 35-375 n with large differences in assocn. and dissocn. rates.  Native mass spectrometry showed that streptavidin can accommodate up to two aptamer units.  In a 3D model of one aptamer, conserved regions are exposed, strongly suggesting that they directly interact with the biotin-binding pockets of streptavidin.  Mutational studies confirmed both conserved regions to be crucial for binding.  An important result is the observation that the most abundant aptamer in the authors' selections is not the tightest binder, emphasizing the importance of having insight into the kinetics of complex formation.  To find the tightest binder it might be better to perform fewer selection rounds and to focus on post-selection characterization, through the use of complementary approaches as described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA8_hBjfMC-7Vg90H21EOLACvtfcHk0lgzYwyDqwMY6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1yjur8%253D&md5=d04f7b513cb0da5ec332ba2a435b3bdc</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100774%26sid%3Dliteratum%253Aachs%26aulast%3DRuigrok%26aufirst%3DV.%2BJ.%26aulast%3DDuijn%26aufirst%3DE.%2BV.%26aulast%3DBarendregt%26aufirst%3DA.%26aulast%3DDyer%26aufirst%3DK.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DStoltenburg%26aufirst%3DR.%26aulast%3DStrehlitz%26aufirst%3DB.%26aulast%3DLevisson%26aufirst%3DM.%26aulast%3DSmidt%26aufirst%3DH.%26aulast%3DOost%26aufirst%3DJ.%2BV.%26atitle%3DKinetic%2520and%2520Stoichiometric%2520Characterisation%2520of%2520Streptavidin-Binding%2520Aptamers%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26issue%3D6%26spage%3D829%26epage%3D836%26doi%3D10.1002%2Fcbic.201100774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bjerregaard, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreasen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, D. M.</span></span> <span> </span><span class="NLM_article-title">Expected and Unexpected Features of Protein-Binding RNA Aptamers</span>. <i>Wiley Interdisciplinary Reviews: RNA</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">744</span>– <span class="NLM_lpage">757</span>, <span class="refDoi"> DOI: 10.1002/wrna.1360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fwrna.1360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=27173731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Ogsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=744-757&issue=6&author=N.+Bjerregaardauthor=P.+A.+Andreasenauthor=D.+M.+Dupont&title=Expected+and+Unexpected+Features+of+Protein-Binding+RNA+Aptamers&doi=10.1002%2Fwrna.1360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Expected and unexpected features of protein-binding RNA aptamers</span></div><div class="casAuthors">Bjerregaard, Nils; Andreasen, Peter A.; Dupont, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">744-757</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">RNA mols. with high affinity to specific proteins can be isolated from libraries of up to 1016 different RNA sequences by systematic evolution of ligands by exponential enrichment (SELEX).  These so-called protein-binding RNA aptamers are often interesting, e.g., as modulators of protein function for therapeutic use, for probing the conformations of proteins, for studies of basic aspects of nucleic acid-protein interactions, etc.  Studies on the interactions between RNA aptamers and proteins display a no. of expected and unexpected features, including the chem. nature of the interacting RNA-protein surfaces, the conformation of protein-bound aptamer vs. free aptamer, the conformation of aptamer-bound protein vs. free protein, and the effects of aptamers on protein function.  Here, we review current insights into the details of RNA aptamer-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2S0mFCMa5T7Vg90H21EOLACvtfcHk0lgUkawPl-cO7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Ogsr7L&md5=77e55852cfefb34e39f0f8961380b240</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fwrna.1360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.1360%26sid%3Dliteratum%253Aachs%26aulast%3DBjerregaard%26aufirst%3DN.%26aulast%3DAndreasen%26aufirst%3DP.%2BA.%26aulast%3DDupont%26aufirst%3DD.%2BM.%26atitle%3DExpected%2520and%2520Unexpected%2520Features%2520of%2520Protein-Binding%2520RNA%2520Aptamers%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520RNA%26date%3D2016%26volume%3D7%26issue%3D6%26spage%3D744%26epage%3D757%26doi%3D10.1002%2Fwrna.1360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iwakiri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tateishi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patil, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenmochi, N.</span></span> <span> </span><span class="NLM_article-title">Dissecting the Protein–RNA Interface: The Role of Protein Surface Shapes and RNA Secondary Structures in Protein–RNA Recognition</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">3299</span>– <span class="NLM_lpage">3306</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr1225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkr1225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22199255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1yqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=3299-3306&issue=8&author=J.+Iwakiriauthor=H.+Tateishiauthor=A.+Chakrabortyauthor=P.+Patilauthor=N.+Kenmochi&title=Dissecting+the+Protein%E2%80%93RNA+Interface%3A+The+Role+of+Protein+Surface+Shapes+and+RNA+Secondary+Structures+in+Protein%E2%80%93RNA+Recognition&doi=10.1093%2Fnar%2Fgkr1225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting the protein-RNA interface: the role of protein surface shapes and RNA secondary structures in protein-RNA recognition</span></div><div class="casAuthors">Iwakiri, Junichi; Tateishi, Hiroki; Chakraborty, Anirban; Patil, Prakash; Kenmochi, Naoya</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3299-3306</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein-RNA interactions are essential for many biol. processes.  However, the structural mechanisms underlying these interactions are not fully understood.  Here, the authors analyzed the protein surface shape (dented, intermediate, or protruded) and the RNA base-pairing properties (paired or unpaired nucleotides) at the interfaces of 91 protein-RNA complexes derived from the Protein Data Bank.  It was found that dented protein surfaces preferred unpaired nucleotides to paired ones at the interface, and H-bonds frequently occurred between the protein backbone and RNA bases.  In contrast, protruded protein surfaces did not show such a preference; rather, electrostatic interactions initiated the formation of H-bonds between pos. charged amino acids and RNA phosphate groups.  Interestingly, in many protein-RNA complexes that interact via an RNA loop, an Asp residue was favored at the interface.  Moreover, in most of these complexes, nucleotide bases in the RNA loop were flipped out and formed H-bonds with the protein, which suggested that the Asp residue is important for RNA loop recognition through a base-flipping process.  This study provided fundamental insights into the role of the shape of the protein surface and RNA secondary structures in mediating protein-RNA interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZUXDPF80n87Vg90H21EOLACvtfcHk0lgUkawPl-cO7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1yqtbw%253D&md5=dfd4c632db6f9a33db949be61534f588</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1225%26sid%3Dliteratum%253Aachs%26aulast%3DIwakiri%26aufirst%3DJ.%26aulast%3DTateishi%26aufirst%3DH.%26aulast%3DChakraborty%26aufirst%3DA.%26aulast%3DPatil%26aufirst%3DP.%26aulast%3DKenmochi%26aufirst%3DN.%26atitle%3DDissecting%2520the%2520Protein%25E2%2580%2593RNA%2520Interface%253A%2520The%2520Role%2520of%2520Protein%2520Surface%2520Shapes%2520and%2520RNA%2520Secondary%2520Structures%2520in%2520Protein%25E2%2580%2593RNA%2520Recognition%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3D8%26spage%3D3299%26epage%3D3306%26doi%3D10.1093%2Fnar%2Fgkr1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellington, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, R.</span></span> <span> </span><span class="NLM_article-title">Aptamers as Potential Nucleic Acid Pharmaceuticals</span>. <i>Biotechnol. Annu. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/S1387-2656(08)70052-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS1387-2656%2808%2970052-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=9704089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK28XhvFCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1995&pages=185-214&author=A.+D.+Ellingtonauthor=R.+Conrad&title=Aptamers+as+Potential+Nucleic+Acid+Pharmaceuticals&doi=10.1016%2FS1387-2656%2808%2970052-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamers as potential nucleic acid pharmaceuticals</span></div><div class="casAuthors">Ellington, Andrew D.; Conrad, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Annual Review</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">185-214</span>CODEN:
                <span class="NLM_cas:coden">BAREFD</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review, with refs.  In vitro selection is a technique for the isolation of nucleic acid ligands that can bind to proteins with high affinity and specificity, and has potential applications in the development of new pharmaceuticals.  This review summarizes the protein targets that have successfully elicited nucleic acid binding species (also known as "aptamers") and explores examples of how they might be developed for clin. use.  In particular, the use of aptamers for the alleviation of blood clotting and the treatment of AIDS are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS3yBPoaoQ5LVg90H21EOLACvtfcHk0lj-INMSAxnIBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvFCisrs%253D&md5=3a6c29c6dddbfafcfbe289f16018430b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1387-2656%2808%2970052-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1387-2656%252808%252970052-8%26sid%3Dliteratum%253Aachs%26aulast%3DEllington%26aufirst%3DA.%2BD.%26aulast%3DConrad%26aufirst%3DR.%26atitle%3DAptamers%2520as%2520Potential%2520Nucleic%2520Acid%2520Pharmaceuticals%26jtitle%3DBiotechnol.%2520Annu.%2520Rev.%26date%3D1995%26volume%3D1%26spage%3D185%26epage%3D214%26doi%3D10.1016%2FS1387-2656%2808%2970052-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulbachinskiy, A. V.</span></span> <span> </span><span class="NLM_article-title">Methods for Selection of Aptamers to Protein Targets</span>. <i>Biochemistry (Moscow)</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>72</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">1505</span>– <span class="NLM_lpage">1518</span>, <span class="refDoi"> DOI: 10.1134/S000629790713007X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1134%2FS000629790713007X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=18282139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitFWktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=1505-1518&issue=13&author=A.+V.+Kulbachinskiy&title=Methods+for+Selection+of+Aptamers+to+Protein+Targets&doi=10.1134%2FS000629790713007X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for selection of aptamers to protein targets</span></div><div class="casAuthors">Kulbachinskiy, A. V.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry (Moscow)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1505-1518</span>CODEN:
                <span class="NLM_cas:coden">BIORAK</span>;
        ISSN:<span class="NLM_cas:issn">0006-2979</span>.
    
            (<span class="NLM_cas:orgname">Pleiades Publishing, Ltd.</span>)
        </div><div class="casAbstract">A review.  Aptamers are synthetic single-stranded RNA or DNA mols. capable of specific binding to other target mols.  In this review, the main aptamer properties are considered and methods for selection of aptamers against various protein targets are described.  Special attention is given to the methods for directed selection of aptamers, which allow one to obtain ligands with specified properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfvwIbRfeCl7Vg90H21EOLACvtfcHk0lj-INMSAxnIBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitFWktL8%253D&md5=f953811b8f4276a6bf3af3d12fe199aa</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1134%2FS000629790713007X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1134%252FS000629790713007X%26sid%3Dliteratum%253Aachs%26aulast%3DKulbachinskiy%26aufirst%3DA.%2BV.%26atitle%3DMethods%2520for%2520Selection%2520of%2520Aptamers%2520to%2520Protein%2520Targets%26jtitle%3DBiochemistry%2520%2528Moscow%2529%26date%3D2007%26volume%3D72%26issue%3D13%26spage%3D1505%26epage%3D1518%26doi%3D10.1134%2FS000629790713007X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Washietl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrix, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinn, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellis, M.</span></span> <span> </span><span class="NLM_article-title">Computational Analysis of Noncoding RNAs</span>. <i>Wiley Interdisciplinary Reviews: RNA</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">759</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1002/wrna.1134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fwrna.1134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22991327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyqtLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=759-778&issue=6&author=S.+Washietlauthor=S.+Willauthor=D.+A.+Hendrixauthor=L.+A.+Goffauthor=J.+L.+Rinnauthor=B.+Bergerauthor=M.+Kellis&title=Computational+Analysis+of+Noncoding+RNAs&doi=10.1002%2Fwrna.1134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of noncoding RNAs</span></div><div class="casAuthors">Washietl, Stefan; Will, Sebastian; Hendrix, David A.; Goff, Loyal A.; Rinn, John L.; Berger, Bonnie; Kellis, Manolis</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">759-778</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Noncoding RNAs have emerged as important key players in the cell.  Understanding their surprisingly diverse range of functions is challenging for exptl. and computational biol.  Here, we review computational methods to analyze noncoding RNAs.  The topics covered include basic and advanced techniques to predict RNA structures, annotation of noncoding RNAs in genomic data, mining RNA-seq data for novel transcripts and prediction of transcript structures, computational aspects of microRNAs, and database resources.  © 2012 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd17N6l4q84bVg90H21EOLACvtfcHk0lhd6SFIYwbYUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyqtLrN&md5=3b099675393e9643fc9b4d070ddcbf92</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fwrna.1134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.1134%26sid%3Dliteratum%253Aachs%26aulast%3DWashietl%26aufirst%3DS.%26aulast%3DWill%26aufirst%3DS.%26aulast%3DHendrix%26aufirst%3DD.%2BA.%26aulast%3DGoff%26aufirst%3DL.%2BA.%26aulast%3DRinn%26aufirst%3DJ.%2BL.%26aulast%3DBerger%26aufirst%3DB.%26aulast%3DKellis%26aufirst%3DM.%26atitle%3DComputational%2520Analysis%2520of%2520Noncoding%2520RNAs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520RNA%26date%3D2012%26volume%3D3%26issue%3D6%26spage%3D759%26epage%3D778%26doi%3D10.1002%2Fwrna.1134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tinoco, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bustamante, C.</span></span> <span> </span><span class="NLM_article-title">How RNA Folds</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>293</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1999.3001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1006%2Fjmbi.1999.3001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10550208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK1MXms12rtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=1999&pages=271-281&issue=2&author=I.+Tinocoauthor=C.+Bustamante&title=How+RNA+Folds&doi=10.1006%2Fjmbi.1999.3001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">How RNA Folds</span></div><div class="casAuthors">Tinoco, Ignacio, Jr.; Bustamante, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review and discussion with 55 refs.  Here, the authors describe the RNA folding problem and contrast it with the much more difficult protein folding problem.  RNA has 4 similar monomer units, whereas proteins have 20 very different residues.  The folding of RNA is hierarchical in that secondary structure is much more stable than tertiary folding.  In RNA, the 2 levels of folding (secondary and tertiary) can be exptl. sepd. by the presence or absence of Mg2+.  Secondary structure can be predicted successfully from exptl. thermodn. data on secondary structure elements: helixes, loops, and bulges.  Tertiary interactions can then be added without much distortion of the secondary structure.  These observations suggest a folding algorithm to predict the structure of an RNA from its sequence.  However, to solve the RNA folding problem one needs thermodn. data on tertiary structure interactions, and identification and characterization of metal ion-binding sites.  These data, together with force vs. extension measurements on single RNA mols., should provide the information necessary to test and refine the proposed algorithm.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL0G6bLmsiibVg90H21EOLACvtfcHk0lhd6SFIYwbYUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXms12rtLw%253D&md5=3985e0bc7b665f7e6ebc83099685130a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.3001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.3001%26sid%3Dliteratum%253Aachs%26aulast%3DTinoco%26aufirst%3DI.%26aulast%3DBustamante%26aufirst%3DC.%26atitle%3DHow%2520RNA%2520Folds%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D293%26issue%3D2%26spage%3D271%26epage%3D281%26doi%3D10.1006%2Fjmbi.1999.3001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hermann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span> <span> </span><span class="NLM_article-title">RNA Bulges as Architectural and Recognition Motifs</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">R47</span>– <span class="NLM_lpage">R54</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(00)00110-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS0969-2126%2800%2900110-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10745015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVOqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2000&pages=R47-R54&issue=3&author=T.+Hermannauthor=D.+J.+Patel&title=RNA+Bulges+as+Architectural+and+Recognition+Motifs&doi=10.1016%2FS0969-2126%2800%2900110-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">RNA bulges as architectural and recognition motifs</span></div><div class="casAuthors">Hermann, Thomas; Patel, Dinshaw J.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">R47-R54</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 64 refs.  RNA bulges constitute versatile structural motifs in the assembly of RNA architectures.  Three-dimensional structures of RNA mols. and their complexes reveal the role of bulges in RNA architectures and illustrate the mol. mechanisms by which they confer intramol. interactions and intermol. recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo73kYSWXFK7bVg90H21EOLACvtfcHk0ljvDgoTgG8SjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVOqt7w%253D&md5=0ffdd4504fcd38e6cd46e8a9172a7cac</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2800%2900110-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252800%252900110-6%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26atitle%3DRNA%2520Bulges%2520as%2520Architectural%2520and%2520Recognition%2520Motifs%26jtitle%3DStructure%26date%3D2000%26volume%3D8%26issue%3D3%26spage%3DR47%26epage%3DR54%26doi%3D10.1016%2FS0969-2126%2800%2900110-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thapar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denmon, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikonowicz, E. P.</span></span> <span> </span><span class="NLM_article-title">Recognition Modes of RNA Tetraloops and Tetraloop-like Motifs by RNA-binding Proteins</span>. <i>Wiley Interdisciplinary Reviews: RNA</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1002/wrna.1196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fwrna.1196" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=24124096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKqu7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=49-67&issue=1&author=R.+Thaparauthor=A.+P.+Denmonauthor=E.+P.+Nikonowicz&title=Recognition+Modes+of+RNA+Tetraloops+and+Tetraloop-like+Motifs+by+RNA-binding+Proteins&doi=10.1002%2Fwrna.1196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition modes of RNA tetraloops and tetraloop-like motifs by RNA-binding proteins</span></div><div class="casAuthors">Thapar, Roopa; Denmon, Andria P.; Nikonowicz, Edward P.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-67</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  RNA hairpins are the most commonly occurring secondary structural elements in RNAs and serve as nucleation sites for RNA folding, RNA-RNA, and RNA-protein interactions.  RNA hairpins are frequently capped by tetraloops, and based on sequence similarity, three broad classes of RNA tetraloops have been defined: GNRA, UNCG, and CUYG.  Other classes such as the UYUN tetraloop in histone mRNAs, the UGAA in 16S rRNA, the AUUA tetraloop from the MS2 bacteriophage, and the AGNN tetraloop that binds RNase III have also been characterized.  The tetraloop structure is compact and is usually characterized by a paired interaction between the first and fourth nucleotides.  The two unpaired nucleotides in the loop are usually involved in base-stacking or base-phosphate hydrogen bonding interactions.  Several structures of RNA tetraloops, free and complexed to other RNAs or proteins, are now available and these studies have increased our understanding of the diverse mechanisms by which this motif is recognized.  RNA tetraloops can mediate RNA-RNA contacts via the tetraloop-receptor motif, kissing hairpin loops, A-minor interactions, and pseudoknots.  While these RNA-RNA interactions are fairly well understood, how RNA-binding proteins recognize RNA tetraloops and tetraloop-like motifs remains unclear.  In this review, we summarize the structures of RNA tetraloop-protein complexes and the general themes that have emerged on sequence- and structure-specific recognition of RNA tetraloops.  We highlight how proteins achieve mol. recognition of this nucleic acid motif, the structural adaptations obsd. in the tetraloop to accommodate the protein-binding partner, and the role of dynamics in recognition.  WIREs RNA 2014, 5:49-67. doi: 10.1002/wrna.1196 Conflict of interest: The authors declare no competing financial interest.  For further resources related to this article, please visit the WIREs website.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzkdk3dbO0jbVg90H21EOLACvtfcHk0lgCrGXOkE_9ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKqu7nN&md5=6bc4639c4b5077e00e38c2f91798948d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fwrna.1196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.1196%26sid%3Dliteratum%253Aachs%26aulast%3DThapar%26aufirst%3DR.%26aulast%3DDenmon%26aufirst%3DA.%2BP.%26aulast%3DNikonowicz%26aufirst%3DE.%2BP.%26atitle%3DRecognition%2520Modes%2520of%2520RNA%2520Tetraloops%2520and%2520Tetraloop-like%2520Motifs%2520by%2520RNA-binding%2520Proteins%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520RNA%26date%3D2014%26volume%3D5%26issue%3D1%26spage%3D49%26epage%3D67%26doi%3D10.1002%2Fwrna.1196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulffen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brockmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flicke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafenbradl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebl, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohse, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krasel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blind, M.</span></span> <span> </span><span class="NLM_article-title">An RNA Molecule that Specifically Inhibits G-protein-Coupled Receptor Kinase 2 In Vitro</span>. <i>RNA</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1261/rna.821908</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1261%2Frna.821908" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=18230760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1cXivFaqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=524-534&issue=3&author=G.+Mayerauthor=B.+Wulffenauthor=C.+Huberauthor=J.+Brockmannauthor=B.+Flickeauthor=L.+Neumannauthor=D.+Hafenbradlauthor=B.+M.+Kleblauthor=M.+J.+Lohseauthor=C.+Kraselauthor=M.+Blind&title=An+RNA+Molecule+that+Specifically+Inhibits+G-protein-Coupled+Receptor+Kinase+2+In+Vitro&doi=10.1261%2Frna.821908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA molecule that specifically inhibits G-protein-coupled receptor kinase 2 in vitro</span></div><div class="casAuthors">Mayer, Guenter; Wulffen, Bernhard; Huber, Christian; Brockmann, Joerg; Flicke, Birgit; Neumann, Lars; Hafenbradl, Doris; Klebl, Bert M.; Lohse, Martin J.; Krasel, Cornelius; Blind, Michael</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">524-534</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">G-protein-coupled receptors are desensitized by a two-step process.  In a first step, G-protein-coupled receptor kinases (GRKs) phosphorylate agonist-activated receptors that subsequently bind to a second class of proteins, the arrestins.  GRKs can be classified into three subfamilies, which have been implicated in various diseases.  The physiol. role(s) of GRKs have been difficult to study as selective inhibitors are not available.  We have used SELEX (systematic evolution of ligands by exponential enrichment) to develop RNA aptamers that potently and selectively inhibit GRK2.  This process has yielded an aptamer, C13, which bound to GRK2 with a high affinity and inhibited GRK2-catalyzed rhodopsin phosphorylation with an IC50 of 4.1 nM.  Phosphorylation of rhodopsin catalyzed by GRK5 was also inhibited, albeit with 20-fold lower potency (IC50 of 79 nM).  Furthermore, C13 reveals significant specificity, since almost no inhibitory activity was detectable testing it against a panel of 14 other kinases.  The aptamer is two orders of magnitude more potent than the best GRK2 inhibitors described previously and shows high selectivity for the GRK family of protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDj6_B4gax2rVg90H21EOLACvtfcHk0lgCrGXOkE_9ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXivFaqsLs%253D&md5=af9993133b975c237bc6424082a4a268</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1261%2Frna.821908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.821908%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DWulffen%26aufirst%3DB.%26aulast%3DHuber%26aufirst%3DC.%26aulast%3DBrockmann%26aufirst%3DJ.%26aulast%3DFlicke%26aufirst%3DB.%26aulast%3DNeumann%26aufirst%3DL.%26aulast%3DHafenbradl%26aufirst%3DD.%26aulast%3DKlebl%26aufirst%3DB.%2BM.%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26aulast%3DKrasel%26aufirst%3DC.%26aulast%3DBlind%26aufirst%3DM.%26atitle%3DAn%2520RNA%2520Molecule%2520that%2520Specifically%2520Inhibits%2520G-protein-Coupled%2520Receptor%2520Kinase%25202%2520In%2520Vitro%26jtitle%3DRNA%26date%3D2008%26volume%3D14%26issue%3D3%26spage%3D524%26epage%3D534%26doi%3D10.1261%2Frna.821908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennarz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanism for Inhibition of G Protein-Coupled Receptor Kinase 2 by a Selective RNA Aptamer</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1300</span>– <span class="NLM_lpage">1309</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.str.2012.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22727813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVKgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1300-1309&issue=8&author=V.+Tesmerauthor=S.+Lennarzauthor=G.+Mayerauthor=J.+Tesmer&title=Molecular+Mechanism+for+Inhibition+of+G+Protein-Coupled+Receptor+Kinase+2+by+a+Selective+RNA+Aptamer&doi=10.1016%2Fj.str.2012.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanism for Inhibition of G Protein-Coupled Receptor Kinase 2 by a Selective RNA Aptamer</span></div><div class="casAuthors">Tesmer, Valerie M.; Lennarz, Sabine; Mayer, Gunter; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1300-1309</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cardiovascular homeostasis is maintained in part by the rapid desensitization of activated heptahelical receptors that have been phosphorylated by G protein-coupled receptor kinase 2 (GRK2).  However, during chronic heart failure GRK2 is upregulated and believed to contribute to disease progression.  We have detd. crystallog. structures of GRK2 bound to an RNA aptamer that potently and selectively inhibits kinase activity.  Key to the mechanism of inhibition is the positioning of an adenine nucleotide into the ATP-binding pocket and interactions with the basic αF-αG loop region of the GRK2 kinase domain.  Constraints imposed on the RNA by the terminal stem of the aptamer also play a role.  These results highlight how a high-affinity aptamer can be used to selectively trap a novel conformational state of a protein kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNVpjLXpH2bVg90H21EOLACvtfcHk0lgCrGXOkE_9ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVKgsLw%253D&md5=26119d2c9cb193d6d9cffa552540a53c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DTesmer%26aufirst%3DV.%26aulast%3DLennarz%26aufirst%3DS.%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DTesmer%26aufirst%3DJ.%26atitle%3DMolecular%2520Mechanism%2520for%2520Inhibition%2520of%2520G%2520Protein-Coupled%2520Receptor%2520Kinase%25202%2520by%2520a%2520Selective%2520RNA%2520Aptamer%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26issue%3D8%26spage%3D1300%26epage%3D1309%26doi%3D10.1016%2Fj.str.2012.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coulson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poyner, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chantry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, S.</span></span> <span> </span><span class="NLM_article-title">A Nonantisense Sequence-Selective Effect of a Phosphorothioate Oligodeoxynucleotide Directed Against the Epidermal Growth Factor Receptor in A431 cells</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">325</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=8700139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK28XkvFCqtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1996&pages=314-325&issue=2&author=J.+M.+Coulsonauthor=D.+R.+Poynerauthor=A.+Chantryauthor=W.+J.+Irwinauthor=S.+Akhtar&title=A+Nonantisense+Sequence-Selective+Effect+of+a+Phosphorothioate+Oligodeoxynucleotide+Directed+Against+the+Epidermal+Growth+Factor+Receptor+in+A431+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">A nonantisense sequence-selective effect of a phosphorothioate oligodeoxynucleotide directed against the epidermal growth factor receptor in A431 cells</span></div><div class="casAuthors">Coulson, Judy M.; Poyner, David R.; Chantry, Andrew; Irwin, William J.; Akhtar, Saghir</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">314-325</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">The overexpression of epidermal growth factor receptor (EGFr) has been implicated as a causative factor and a poor prognostic marker in a no. of carcinomas.  Therefore, strategies that down-regulate EGFr expression may be therapeutically useful.  We designed antisense ODNs complementary to the initiation codon region of the EGFr mRNA and evaluated their efficacy in several tumor-derived cells, including the A431 cell line, that express amplified levels of EGFr.  A 15-mer phosphorothioate (PS) antisense ODN (erbB1AS15) induced a concn.-dependent redn. in proliferation that was accompanied by a change in the morphol. of A431 cells into more tightly clustered and discrete colonies.  A 15-mer sense (PS) control oligodeoxynucleotide (ODN) and a phosphodiester (PS) control oligodeoxynucleotide (ODN) and a phosphodiester (PO) version of erbB1AS15 had little or no effect on cell no. or morphol., and erbB1AS15 (PS) did not induce these effects in control cell lines expressing lower levels of EGFr.  The effects of erbB1AS15 (PS) on A431 cells were not mediated by a true antisense mechanism in that there was no redn. in the level of EGFr mRNA or protein over a 24-h period, as detd. by Northern and Western blotting, resp.  However, autophosphorylation of the receptor was significantly reduced by erbB1AS15 (PS) and not by control ODNs.  The results of further studies suggested that this effect was mediated by a direct, dose-dependent inhibition of the EGFr tyrosine kinase enzyme and was not due to impairment of either ligand-binding or receptor dimerization.  These data suggest that erbB1AS15 (PS) can inhibit proliferation and alter the morphol. of A431 cells by a sequence-selective, but nonantisense, mechanism affecting receptor tyrosine kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBwByQXrQfQLVg90H21EOLACvtfcHk0lh9MwuNnf0omw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XkvFCqtb4%253D&md5=969f668cf9b3fdee9a80d828cc9bc350</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCoulson%26aufirst%3DJ.%2BM.%26aulast%3DPoyner%26aufirst%3DD.%2BR.%26aulast%3DChantry%26aufirst%3DA.%26aulast%3DIrwin%26aufirst%3DW.%2BJ.%26aulast%3DAkhtar%26aufirst%3DS.%26atitle%3DA%2520Nonantisense%2520Sequence-Selective%2520Effect%2520of%2520a%2520Phosphorothioate%2520Oligodeoxynucleotide%2520Directed%2520Against%2520the%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520in%2520A431%2520cells%26jtitle%3DMol.%2520Pharmacol.%26date%3D1996%26volume%3D50%26issue%3D2%26spage%3D314%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunnion, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poyner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackroyd, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bibby, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Double, J.</span></span> <span> </span><span class="NLM_article-title">Sequence and Chemistry Requirements for a Novel Aptameric Oligonucleotide Inhibitor of EGF Receptor Tyrosine Kinase Activity</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2187</span>– <span class="NLM_lpage">2195</span>, <span class="refDoi"> DOI: 10.1016/S0006-2952(02)00985-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS0006-2952%2802%2900985-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=12110378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD38XltFSqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2002&pages=2187-2195&issue=12&author=S.+Akhtarauthor=D.+Dunnionauthor=D.+Poynerauthor=J.+Ackroydauthor=M.+Bibbyauthor=J.+Double&title=Sequence+and+Chemistry+Requirements+for+a+Novel+Aptameric+Oligonucleotide+Inhibitor+of+EGF+Receptor+Tyrosine+Kinase+Activity&doi=10.1016%2FS0006-2952%2802%2900985-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence and chemistry requirements for a novel aptameric oligonucleotide inhibitor of EGF receptor tyrosine kinase activity</span></div><div class="casAuthors">Akhtar, Saghir; Dunnion, Debbie; Poyner, David; Ackroyd, John; Bibby, Mike; Double, John</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2187-2195</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">We have previously identified a phosphorothioate oligonucleotide (PS-ODN) that inhibited epidermal growth factor receptor tyrosine kinase (TK) activity both in cell fractions and in intact A431 cells.  Since ODN-based TK inhibitors may have anti-cancer applications and may also help understand the non-antisense mediated effects of PS-ODNs, we have further studied the sequence and chem. requirements of the parent PS-ODN (sequence: 5'-GGA GGG TCG CAT CGC-3') as a sequence-dependent TK inhibitor.  Sequence deletion and substitution studies revealed that the 5'-terminal GGA GGG hexamer sequence in the parent compd. was essential for anti-TK activity in A431 cells.  Site-specific substitution of any G with a T in this 5'-terminal motif within the parent compd. caused a significant loss in anti-TK activity.  The fully PS-modified hexameric motif alone exhibited equipotent activity as the parent 15-mer whereas phosphodiester (PO) or 2'-O-methyl-modified versions of this motif had significantly reduced anti-TK activity.  Further, T substitutions within the two 5'-terminal G residues of the hexameric PS-ODN to produce a sequence, TTA GGG, representing the telomeric repeats in human chromosomes, also did not exhibit a significant anti-TK activity.  Multiple repeats of the active hexameric motif in PS-ODNs resulted in more potent inhibitors of TK activity than the parent ODN.  These results suggested that PS-ODNs, but not PO or 2'-O-Me modified ODNs, contg. the GGA GGG motif can exert potent anti-TK activity which may be desirable in some anti-tumor applications.  Addnl., the presence of this previously unidentified motif in antisense PS-ODN constructs may contribute to their biol. effects in vitro and in vivo and should be accounted for in the design of the PS-modified antisense ODNs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsp7iuJxKpw7Vg90H21EOLACvtfcHk0lh9MwuNnf0omw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltFSqsbk%253D&md5=bdebe9d814b245090f32ad1487e62b7c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS0006-2952%2802%2900985-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-2952%252802%252900985-1%26sid%3Dliteratum%253Aachs%26aulast%3DAkhtar%26aufirst%3DS.%26aulast%3DDunnion%26aufirst%3DD.%26aulast%3DPoyner%26aufirst%3DD.%26aulast%3DAckroyd%26aufirst%3DJ.%26aulast%3DBibby%26aufirst%3DM.%26aulast%3DDouble%26aufirst%3DJ.%26atitle%3DSequence%2520and%2520Chemistry%2520Requirements%2520for%2520a%2520Novel%2520Aptameric%2520Oligonucleotide%2520Inhibitor%2520of%2520EGF%2520Receptor%2520Tyrosine%2520Kinase%2520Activity%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2002%26volume%3D63%26issue%3D12%26spage%3D2187%26epage%3D2195%26doi%3D10.1016%2FS0006-2952%2802%2900985-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connell, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahmy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L.</span></span> <span> </span><span class="NLM_article-title">Aptameric Inhibition of p210bcr-abl Tyrosine Kinase Autophosphorylation by Oligodeoxynucleotides of Defined Sequence and Backbone Structure</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2150</span>– <span class="NLM_lpage">2154</span>, <span class="refDoi"> DOI: 10.1093/nar/22.11.2150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2F22.11.2150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=8029025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2cXlslelt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1994&pages=2150-2154&issue=11&author=R.+Berganauthor=Y.+Connellauthor=B.+Fahmyauthor=E.+Kyleauthor=L.+Neckers&title=Aptameric+Inhibition+of+p210bcr-abl+Tyrosine+Kinase+Autophosphorylation+by+Oligodeoxynucleotides+of+Defined+Sequence+and+Backbone+Structure&doi=10.1093%2Fnar%2F22.11.2150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Aptameric inhibition of p210bcr-abl tyrosine kinase autophosphorylation by oligodeoxynucleotides of defined sequence and backbone structure</span></div><div class="casAuthors">Bergan, Raymond; Connell, Yvette; Fahmy, Brigid; Kyle, Edward; Neckers, Len</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2150-4</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    </div><div class="casAbstract">Protein tyrosine kinases play key roles in cellular physiol.  Specific inhibitors of these enzymes are important lab. tools and may prove to be novel therapeutic agents.  In this report the authors describe a new class of tyrosine kinase inhibitor, synthetic oligodeoxynucleotides (ODNs).  An ODN is described which specifically inhibits p210bc4-abl tyrosine kinase autophosphorylation in vitro with a Ki of 0.5 μM.  Inhibition is non-competitive with respect to ATP.  The effects upon inhibitory activity of ODN structure modifications are described.  The inhibition described is not mediated by classical antisense mechanisms and represents an example of the recently recognized aptameric properties of ODNs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgN4XH8VxO1LVg90H21EOLACvtfcHk0lh9MwuNnf0omw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlslelt7w%253D&md5=5aaeb6bdce940448c7a9a348cc574ce3</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F22.11.2150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F22.11.2150%26sid%3Dliteratum%253Aachs%26aulast%3DBergan%26aufirst%3DR.%26aulast%3DConnell%26aufirst%3DY.%26aulast%3DFahmy%26aufirst%3DB.%26aulast%3DKyle%26aufirst%3DE.%26aulast%3DNeckers%26aufirst%3DL.%26atitle%3DAptameric%2520Inhibition%2520of%2520p210bcr-abl%2520Tyrosine%2520Kinase%2520Autophosphorylation%2520by%2520Oligodeoxynucleotides%2520of%2520Defined%2520Sequence%2520and%2520Backbone%2520Structure%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1994%26volume%3D22%26issue%3D11%26spage%3D2150%26epage%3D2154%26doi%3D10.1093%2Fnar%2F22.11.2150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fay, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, P.</span></span> <span> </span><span class="NLM_article-title">RNA G-Quadruplexes in Biology: Principles and Molecular Mechanisms</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">2127</span>– <span class="NLM_lpage">2147</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2017.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.jmb.2017.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=28554731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVKhtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2127-2147&issue=14&author=M.+M.+Fayauthor=S.+M.+Lyonsauthor=P.+Ivanov&title=RNA+G-Quadruplexes+in+Biology%3A+Principles+and+Molecular+Mechanisms&doi=10.1016%2Fj.jmb.2017.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">RNA G-quadruplexes in biology: Principles and molecular mechanisms</span></div><div class="casAuthors">Fay, Marta M.; Lyons, Shawn M.; Ivanov, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2127-2147</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes (G4s) are extremely stable DNA or RNA secondary structures formed by sequences rich in guanine.  These structures are implicated in many essential cellular processes, and the no. of biol. functions attributed to them continues to grow.  While DNA G4s are well understood on structural and, to some extent, functional levels, RNA G4s and their functions have received less attention.  The presence of bona fide RNA G4s in cells has long been a matter of debate.  The development of G4-specific antibodies and ligands hinted at their presence in vivo, but recent advances in RNA sequencing coupled with chem. footprinting suggested the opposite.  Here, we critically discuss the biol. of RNA G4s focusing on the mol. mechanisms underlying their proposed functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUMCO9XOrQVLVg90H21EOLACvtfcHk0lituZerPBfDxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVKhtLo%253D&md5=987b0cb1715cb621beba9808ef73d5e2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2017.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2017.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DFay%26aufirst%3DM.%2BM.%26aulast%3DLyons%26aufirst%3DS.%2BM.%26aulast%3DIvanov%26aufirst%3DP.%26atitle%3DRNA%2520G-Quadruplexes%2520in%2520Biology%253A%2520Principles%2520and%2520Molecular%2520Mechanisms%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26issue%3D14%26spage%3D2127%26epage%3D2147%26doi%3D10.1016%2Fj.jmb.2017.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woodson, S. A.</span>; <span class="NLM_string-name">Koculi, E.</span></span> <span> </span><span class="NLM_article-title">Analysis of RNA Folding by Native Polyacrylamide Gel Electrophoresis</span>. In  <i>Part B: Methods in Enzymology: Biophysical, Chemical, and Functional Probes of RNA Structure, Interactions and Folding</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span>, Ed.; <span class="NLM_publisher-name">Elsevier Inc. Publishing</span>, <span class="NLM_year">2009</span>; pp  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">208</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS0076-6879%2809%2969009-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=189-208&author=S.+A.+Woodson&author=E.+Koculiauthor=D.+Herschlag&title=Part+B%3A+Methods+in+Enzymology%3A+Biophysical%2C+Chemical%2C+and+Functional+Probes+of+RNA+Structure%2C+Interactions+and+Folding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2809%2969009-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252809%252969009-1%26sid%3Dliteratum%253Aachs%26aulast%3DWoodson%26aufirst%3DS.%2BA.%26atitle%3DAnalysis%2520of%2520RNA%2520Folding%2520by%2520Native%2520Polyacrylamide%2520Gel%2520Electrophoresis%26btitle%3DPart%2520B%253A%2520Methods%2520in%2520Enzymology%253A%2520Biophysical%252C%2520Chemical%252C%2520and%2520Functional%2520Probes%2520of%2520RNA%2520Structure%252C%2520Interactions%2520and%2520Folding%26aulast%3DHerschlag%26aufirst%3DD.%26pub%3DElsevier%2520Inc.%2520Publishing%26date%3D2009%26spage%3D189%26epage%3D208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, C. A.</span>; <span class="NLM_string-name">Fried, M. G.</span></span> <span> </span><span class="NLM_article-title">Analysis of Protein-DNA Equilibria by Native Gel Electrophoresis</span>. In  <i>Protein Interactions: Biophysical Approaches for the Study of Complex Reversible Systems</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, P.</span></span>, Ed.; <span class="NLM_publisher-name">Springer Science Publishing</span>, <span class="NLM_year">2007</span>; pp  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">446</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1007%2F978-0-387-35966-3_14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=417-446&author=C.+A.+Adams&author=M.+G.+Friedauthor=P.+Schuck&title=Protein+Interactions%3A+Biophysical+Approaches+for+the+Study+of+Complex+Reversible+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-35966-3_14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-35966-3_14%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DC.%2BA.%26atitle%3DAnalysis%2520of%2520Protein-DNA%2520Equilibria%2520by%2520Native%2520Gel%2520Electrophoresis%26btitle%3DProtein%2520Interactions%253A%2520Biophysical%2520Approaches%2520for%2520the%2520Study%2520of%2520Complex%2520Reversible%2520Systems%26aulast%3DSchuck%26aufirst%3DP.%26pub%3DSpringer%2520Science%2520Publishing%26date%3D2007%26spage%3D417%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yakhnin, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakhnin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babitzke, P.</span></span> <span> </span><span class="NLM_article-title">Gel Mobility Shift Assays to Detect Protein–RNA Interactions</span>. <i>Bacterial Regulatory RNA</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-949-5_12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1007%2F978-1-61779-949-5_12" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=201-211&author=A.+V.+Yakhninauthor=H.+Yakhninauthor=P.+Babitzke&title=Gel+Mobility+Shift+Assays+to+Detect+Protein%E2%80%93RNA+Interactions&doi=10.1007%2F978-1-61779-949-5_12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-949-5_12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-949-5_12%26sid%3Dliteratum%253Aachs%26aulast%3DYakhnin%26aufirst%3DA.%2BV.%26aulast%3DYakhnin%26aufirst%3DH.%26aulast%3DBabitzke%26aufirst%3DP.%26atitle%3DGel%2520Mobility%2520Shift%2520Assays%2520to%2520Detect%2520Protein%25E2%2580%2593RNA%2520Interactions%26jtitle%3DBacterial%2520Regulatory%2520RNA%26date%3D2012%26spage%3D201%26epage%3D211%26doi%3D10.1007%2F978-1-61779-949-5_12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alshaer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ababneh, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatmal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izmirli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choukeife, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shraim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu-Shiekah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odeh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Bawab, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span> <span> </span><span class="NLM_article-title">Selection and Targeting of EpCAM Protein by ssDNA Aptamer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>), <span class="NLM_elocation-id">e0194529</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0194529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1371%2Fjournal.pone.0194529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29538434" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&issue=3&author=W.+Alshaerauthor=N.+A.+Ababnehauthor=M.+Hatmalauthor=H.+Izmirliauthor=M.+Choukeifeauthor=A.+Shraimauthor=N.+Shararauthor=A.+Abu-Shiekahauthor=F.+Odehauthor=A.+Al+Bawabauthor=A.+Awidiauthor=S.+Ismail&title=Selection+and+Targeting+of+EpCAM+Protein+by+ssDNA+Aptamer&doi=10.1371%2Fjournal.pone.0194529"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0194529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0194529%26sid%3Dliteratum%253Aachs%26aulast%3DAlshaer%26aufirst%3DW.%26aulast%3DAbabneh%26aufirst%3DN.%2BA.%26aulast%3DHatmal%26aufirst%3DM.%26aulast%3DIzmirli%26aufirst%3DH.%26aulast%3DChoukeife%26aufirst%3DM.%26aulast%3DShraim%26aufirst%3DA.%26aulast%3DSharar%26aufirst%3DN.%26aulast%3DAbu-Shiekah%26aufirst%3DA.%26aulast%3DOdeh%26aufirst%3DF.%26aulast%3DAl%2BBawab%26aufirst%3DA.%26aulast%3DAwidi%26aufirst%3DA.%26aulast%3DIsmail%26aufirst%3DS.%26atitle%3DSelection%2520and%2520Targeting%2520of%2520EpCAM%2520Protein%2520by%2520ssDNA%2520Aptamer%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26issue%3D3%26doi%3D10.1371%2Fjournal.pone.0194529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recht, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, J. R.</span></span> <span> </span><span class="NLM_article-title">Quantitative Analysis of Protein-RNA Interactions by Gel Mobility Shift</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>488</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1007/978-1-60327-475-3_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1007%2F978-1-60327-475-3_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=18982286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1CmtLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=488&publication_year=2008&pages=99-115&author=S.+P.+Ryderauthor=M.+I.+Rechtauthor=J.+R.+Williamson&title=Quantitative+Analysis+of+Protein-RNA+Interactions+by+Gel+Mobility+Shift&doi=10.1007%2F978-1-60327-475-3_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative analysis of protein-RNA interactions by gel mobility shift</span></div><div class="casAuthors">Ryder, Sean P.; Recht, Michael I.; Williamson, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (Totowa, NJ, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">488</span>
        (<span class="NLM_cas:issue">RNA-Protein Interaction Protocols (2nd Edition)</span>),
    <span class="NLM_cas:pages">99-115</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">A review.  The gel mobility shift assay is routinely used to visualize protein-RNA interactions.  Its power resides in the ability to resolve free from bound RNA with high resoln. in a gel matrix.  We review the quant. application of this approach to elucidate thermodn. properties of protein-RNA complexes.  Assay designs for titrn., competition, and stoichiometry expts. are presented for two unrelated model complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy2Q7u9qbd4rVg90H21EOLACvtfcHk0lituZerPBfDxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1CmtLfI&md5=54da92d375332a9370de762077ee46cc</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2F978-1-60327-475-3_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60327-475-3_7%26sid%3Dliteratum%253Aachs%26aulast%3DRyder%26aufirst%3DS.%2BP.%26aulast%3DRecht%26aufirst%3DM.%2BI.%26aulast%3DWilliamson%26aufirst%3DJ.%2BR.%26atitle%3DQuantitative%2520Analysis%2520of%2520Protein-RNA%2520Interactions%2520by%2520Gel%2520Mobility%2520Shift%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2008%26volume%3D488%26spage%3D99%26epage%3D115%26doi%3D10.1007%2F978-1-60327-475-3_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navani, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span> <span> </span><span class="NLM_article-title">Isolation of DNA Aptamers for CDP-Ribitol Synthase, and Characterization of Their Inhibitory and Structural Properties</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">2052</span>– <span class="NLM_lpage">2057</span>, <span class="refDoi"> DOI: 10.1002/cbic.200700257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fcbic.200700257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=17929340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsValu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=2052-2057&issue=17&author=A.+Allali-Hassaniauthor=M.+P.+Pereiraauthor=N.+K.+Navaniauthor=E.+D.+Brownauthor=Y.+Li&title=Isolation+of+DNA+Aptamers+for+CDP-Ribitol+Synthase%2C+and+Characterization+of+Their+Inhibitory+and+Structural+Properties&doi=10.1002%2Fcbic.200700257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of DNA aptamers for CDP-ribitol synthase, and characterization of their inhibitory and structural properties</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Pereira, Mark P.; Navani, Naveen K.; Brown, Eric D.; Li, Yingfu</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2052-2057</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Isolation of DNA aptamers for CDP-ribitol synthase is described.  Using the SELEX (systematic evolution of ligands by exponential enrichment) strategy, a pool of 81 nucleotide long single stranded DNA (ssDNA) mols. were produced by automated DNA synthesis.  DNA aptamer pool was then searched to isolate the aptamer for the enzyme Bcs1 from H. influenzae and to test whether any of th aptamers could inhibit either one or both the enzyme activities of this bifunctional enzyme.  The results show that the aptamers were developed for a bifunctional enzyme, and that the SELEX strategy has expanded the no. of protein binding DNA aptamers and has generated new DNA sequences and structures for functional studies of nucleic acid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIjfUKa7pokLVg90H21EOLACvtfcHk0ljfKOQ19F0TFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsValu7zK&md5=5fa40316d63671efae8f06225971bd36</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200700257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200700257%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DPereira%26aufirst%3DM.%2BP.%26aulast%3DNavani%26aufirst%3DN.%2BK.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DIsolation%2520of%2520DNA%2520Aptamers%2520for%2520CDP-Ribitol%2520Synthase%252C%2520and%2520Characterization%2520of%2520Their%2520Inhibitory%2520and%2520Structural%2520Properties%26jtitle%3DChemBioChem%26date%3D2007%26volume%3D8%26issue%3D17%26spage%3D2052%26epage%3D2057%26doi%3D10.1002%2Fcbic.200700257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reinholt, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craighead, H. G.</span></span> <span> </span><span class="NLM_article-title">Highly Multiplexed RNA Aptamer Selection Using a Microplate-based Microcolumn Device</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">29771</span>, <span class="refDoi"> DOI: 10.1038/srep29771</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1038%2Fsrep29771" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=27432610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=29771&issue=1&author=S.+J.+Reinholtauthor=A.+Ozerauthor=J.+T.+Lisauthor=H.+G.+Craighead&title=Highly+Multiplexed+RNA+Aptamer+Selection+Using+a+Microplate-based+Microcolumn+Device&doi=10.1038%2Fsrep29771"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Multiplexed RNA Aptamer Selection using a Microplate-based Microcolumn Device</span></div><div class="casAuthors">Reinholt, Sarah J.; Ozer, Abdullah; Lis, John T.; Craighead, Harold G.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29771</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe a multiplexed RNA aptamer selection to 19 different targets simultaneously using a microcolumn-based device, MEDUSA (Microplate-based Enrichment Device Used for the Selection of Aptamers), as well as a modified selection process, that significantly reduce the time and reagents needed for selections.  We exploited MEDUSA's reconfigurable design between parallel and serially-connected microcolumns to enable the use of just 2 aliquots of starting library, and its 96-well microplate compatibility to enable the continued use of high-throughput techniques in downstream processes.  Our modified selection protocol allowed us to perform the equiv. of a 10-cycle selection in the time it takes for 4 traditional selection cycles.  Several aptamers were discovered with nanomolar dissocn. consts.  Furthermore, aptamers were identified that not only bound with high affinity, but also acted as inhibitors to significantly reduce the activity of their target protein, mouse decapping exoribonuclease (DXO).  The aptamers resisted DXO's exoribonuclease activity, and in studies monitoring DXO's degrdn. of a 30-nucleotide substrate, less than 1 μM of aptamer demonstrated significant inhibition of DXO activity.  This aptamer selection method using MEDUSA helps to overcome some of the major challenges with traditional aptamer selections, and provides a platform for high-throughput selections that lends itself to process automation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprm5z4eWUm47Vg90H21EOLACvtfcHk0ljfKOQ19F0TFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVOgur4%253D&md5=dc850b0fb48e7f340ee8d3d5fb25b57c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fsrep29771&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep29771%26sid%3Dliteratum%253Aachs%26aulast%3DReinholt%26aufirst%3DS.%2BJ.%26aulast%3DOzer%26aufirst%3DA.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26aulast%3DCraighead%26aufirst%3DH.%2BG.%26atitle%3DHighly%2520Multiplexed%2520RNA%2520Aptamer%2520Selection%2520Using%2520a%2520Microplate-based%2520Microcolumn%2520Device%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26issue%3D1%26spage%3D29771%26doi%3D10.1038%2Fsrep29771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jellinek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janjic, N.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Receptor Binding by High-Affinity RNA Ligands to Vascular Endothelial Growth Factor</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">34</span>),  <span class="NLM_fpage">10450</span>– <span class="NLM_lpage">10456</span>, <span class="refDoi"> DOI: 10.1021/bi00200a028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00200a028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2cXltlWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1994&pages=10450-10456&issue=34&author=D.+Jellinekauthor=L.+S.+Greenauthor=C.+Bellauthor=N.+Janjic&title=Inhibition+of+Receptor+Binding+by+High-Affinity+RNA+Ligands+to+Vascular+Endothelial+Growth+Factor&doi=10.1021%2Fbi00200a028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of receptor binding by high-affinity RNA ligands to vascular endothelial growth factor</span></div><div class="casAuthors">Jellinek, Derek; Green, Louis S.; Bell, Carol; Janjic, Nebojsa</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">10450-6</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">The proliferation of new blood vessels (angiogenesis) is a process that accompanies many pathol. conditions including rheumatoid arthritis and solid tumor growth.  Among angiogenic cytokines that have been identified to date, vascular endothelial growth factor (VEGF) is one of the most potent.  SELEX (systematic evolution of ligands by exponential enrichment) was used to identify RNA ligands that bind to VEGF in a specific manner with affinities in the low nanomolar range.  Ligands were selected from a starting pool of ∼1014 RNA mols. contg. 30 randomized positions.  Isolates from the affinity-enriched pool were grouped into 6 distinct families on the basis of primary and secondary structure similarities.  Minimal sequence information required for high-affinity binding to VEGF is contained in 29-36-nucleotide motifs.  Binding of truncated (minimal) high-affinity ligands to VEGF is competitive with that of other truncated ligands and heparin.  Furthermore, truncated ligands from the 6 ligand families inhibit binding of [125I]VEGF to its cell-surface receptors.  Oligonucleotide ligands described here represent an initial set of lead compds. in the ongoing effort toward the development of potent and specific VEGF antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBaPkVkfhWg7Vg90H21EOLACvtfcHk0ljfKOQ19F0TFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXltlWqt70%253D&md5=0ae5ff503a899515bc9ecc1ae3861054</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi00200a028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00200a028%26sid%3Dliteratum%253Aachs%26aulast%3DJellinek%26aufirst%3DD.%26aulast%3DGreen%26aufirst%3DL.%2BS.%26aulast%3DBell%26aufirst%3DC.%26aulast%3DJanjic%26aufirst%3DN.%26atitle%3DInhibition%2520of%2520Receptor%2520Binding%2520by%2520High-Affinity%2520RNA%2520Ligands%2520to%2520Vascular%2520Endothelial%2520Growth%2520Factor%26jtitle%3DBiochemistry%26date%3D1994%26volume%3D33%26issue%3D34%26spage%3D10450%26epage%3D10456%26doi%3D10.1021%2Fbi00200a028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. E.</span>; <span class="NLM_string-name">Williams, K. P.</span></span> <span> </span><span class="NLM_article-title">Validating Identity, Mass Purity and Enzymatic Purity of Enzyme Preparations</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, S.</span></span>, Ed.; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences Publishing</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">43</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=31-43&author=J.+E.+Scott&author=K.+P.+Williamsauthor=S.+Sittampalam&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DJ.%2BE.%26atitle%3DValidating%2520Identity%252C%2520Mass%2520Purity%2520and%2520Enzymatic%2520Purity%2520of%2520Enzyme%2520Preparations%26btitle%3DAssay%2520Guidance%2520Manual%26aulast%3DSittampalam%26aufirst%3DS.%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%2520Publishing%26date%3D2012%26spage%3D31%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theis, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knorre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellersch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moelleken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolanus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famulok, M.</span></span> <span> </span><span class="NLM_article-title">Discriminatory Aptamer Reveals Serum Response Element Transcription Regulated by Cytohesin-2</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">11221</span>– <span class="NLM_lpage">11226</span>, <span class="refDoi"> DOI: 10.1073/pnas.0402901101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1073%2Fpnas.0402901101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=15277685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVKgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=11221-11226&issue=31&author=M.+G.+Theisauthor=A.+Knorreauthor=B.+Kellerschauthor=J.+Moellekenauthor=F.+Wielandauthor=W.+Kolanusauthor=M.+Famulok&title=Discriminatory+Aptamer+Reveals+Serum+Response+Element+Transcription+Regulated+by+Cytohesin-2&doi=10.1073%2Fpnas.0402901101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discriminatory aptamer reveals serum response element transcription regulated by cytohesin-2</span></div><div class="casAuthors">Theis, Mirko G.; Knorre, Alexander; Kellersch, Bettina; Moelleken, Joerg; Wieland, Felix; Kolanus, Waldemar; Famulok, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">11221-11226</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cytohesins are a family of highly homologous guanine nucleotide exchange factors (GEFs) that act on ADP-ribosylation factors (ARFs).  The small ARF-GEFs are involved in integrin signaling, actin cytoskeleton remodeling, and vesicle transport.  Here, we selected and applied a specific inhibitor for ARF nucleotide-binding site opener (ARNO)/cytohesin-2, an RNA aptamer that clearly discriminates between cytohesin-1 and cytohesin-2.  This reagent bound to an N-terminal segment of cytohesin-2 and did not inhibit ARF-GEF function in vitro.  When transfected into HeLa cells, it persisted for at least 6 h without requiring stabilization.  Its effect in vivo was to down-regulate gene expression mediated through the serum-response element and knockdown mitogen-activated protein kinase activation, indicating that cytohesin-2 acts by means of mitogen-activated protein kinase signaling.  We conclude that the N-terminal coiled-coil and parts of the Sec7 domain of cytohesin-2 are required for serum-mediated transcriptional activation in nonimmune cells, whereas cytohesin-1 is not.  Our results indicate that intramer technol. can be used not only for assigning novel biol. functions to proteins or protein domains but also to prove nonredundancy of highly homologous proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2nObwn4ouJLVg90H21EOLACvtfcHk0lhRetHzAeGapg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVKgtrg%253D&md5=3641a7ee14d0040340a4b6ff2c57ba63</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0402901101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0402901101%26sid%3Dliteratum%253Aachs%26aulast%3DTheis%26aufirst%3DM.%2BG.%26aulast%3DKnorre%26aufirst%3DA.%26aulast%3DKellersch%26aufirst%3DB.%26aulast%3DMoelleken%26aufirst%3DJ.%26aulast%3DWieland%26aufirst%3DF.%26aulast%3DKolanus%26aufirst%3DW.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DDiscriminatory%2520Aptamer%2520Reveals%2520Serum%2520Response%2520Element%2520Transcription%2520Regulated%2520by%2520Cytohesin-2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26issue%3D31%26spage%3D11221%26epage%3D11226%26doi%3D10.1073%2Fpnas.0402901101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pothukuchy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Syrett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husain, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrisha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosaraju, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellington, A. D.</span></span> <span> </span><span class="NLM_article-title">Aptamers that Recognize Drug-Resistant HIV-1 Reverse Transcriptase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">5236</span>– <span class="NLM_lpage">5236</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp663</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkp663" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKntbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=5236-5236&issue=15&author=N.+Liauthor=Y.+Wangauthor=A.+Pothukuchyauthor=A.+Syrettauthor=N.+Husainauthor=S.+Gopalakrishaauthor=P.+Kosarajuauthor=A.+D.+Ellington&title=Aptamers+that+Recognize+Drug-Resistant+HIV-1+Reverse+Transcriptase&doi=10.1093%2Fnar%2Fgkp663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamers that recognize drug-resistant HIV-1 reverse transcriptase. [Erratum to document cited in CA150:159457]</span></div><div class="casAuthors">Li, Na; Wang, Yuxuan; Pothukuchy, Arti; Syrett, Angel; Husain, Naeem; Gopalakrisha, Siddharth; Kosaraju, Pradeepa; Ellington, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5236</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The authors note that although the did collaborate with the University of Texas, they wish their affiliations to be cor. to Accacia International, 2113 Wells branch Pkwy, Suite 6900, Austin, Texas 78728.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6j5-QczVWJLVg90H21EOLACvtfcHk0lhRetHzAeGapg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKntbvN&md5=2c29e4ad6a5b271087225358faca4579</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp663%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DPothukuchy%26aufirst%3DA.%26aulast%3DSyrett%26aufirst%3DA.%26aulast%3DHusain%26aufirst%3DN.%26aulast%3DGopalakrisha%26aufirst%3DS.%26aulast%3DKosaraju%26aufirst%3DP.%26aulast%3DEllington%26aufirst%3DA.%2BD.%26atitle%3DAptamers%2520that%2520Recognize%2520Drug-Resistant%2520HIV-1%2520Reverse%2520Transcriptase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3D15%26spage%3D5236%26epage%3D5236%26doi%3D10.1093%2Fnar%2Fgkp663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lennarz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heider, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blind, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span> <span> </span><span class="NLM_article-title">An Aptamer to the MAP Kinase Insert Region</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1021/cb5005756</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5005756" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVOntL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=320-327&issue=1&author=S.+Lennarzauthor=E.+Heiderauthor=M.+Blindauthor=G.+Mayer&title=An+Aptamer+to+the+MAP+Kinase+Insert+Region&doi=10.1021%2Fcb5005756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">An Aptamer to the MAP Kinase Insert Region</span></div><div class="casAuthors">Lennarz, Sabine; Heider, Elena; Blind, Michael; Mayer, Guenter</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">320-327</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting functional, non-catalytic domains of protein kinases or other proteins is an emerging field in chem. biol. research.  Non-ATP competitive kinase inhibitors allow for the investigation of active-site independent functions, e.g., the biol. role of protein-protein interactions.  These inhibitors also serve as a starting point for developing novel therapeutic strategies.  However, the identification of such inhibitors by means of conventional low mol. wt. compds. represents a great challenge in modern drug discovery.  Employing the mitogen-activated protein (MAP) kinase Erk2, we show that RNA aptamers have the capacity to be a novel, promising class of protein kinase inhibitors that can be applied to target individual subdomains and block domain specific functions without affecting kinase activity per se.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooDd55t1VYxbVg90H21EOLACvtfcHk0lhRetHzAeGapg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVOntL7J&md5=dd4c086945b6d450b367aab2d3090e4e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fcb5005756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5005756%26sid%3Dliteratum%253Aachs%26aulast%3DLennarz%26aufirst%3DS.%26aulast%3DHeider%26aufirst%3DE.%26aulast%3DBlind%26aufirst%3DM.%26aulast%3DMayer%26aufirst%3DG.%26atitle%3DAn%2520Aptamer%2520to%2520the%2520MAP%2520Kinase%2520Insert%2520Region%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D1%26spage%3D320%26epage%3D327%26doi%3D10.1021%2Fcb5005756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seiwert, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahreini, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aigner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhlenbeck, O. C.</span></span> <span> </span><span class="NLM_article-title">RNA Aptamers as Pathway-specific MAP Kinase Inhibitors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">833</span>– <span class="NLM_lpage">843</span>, <span class="refDoi"> DOI: 10.1016/S1074-5521(00)00032-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2FS1074-5521%2800%2900032-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=11094337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVWit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=833-843&issue=11&author=S.+D.+Seiwertauthor=T.+S.+Nahreiniauthor=S.+Aignerauthor=N.+G.+Ahnauthor=O.+C.+Uhlenbeck&title=RNA+Aptamers+as+Pathway-specific+MAP+Kinase+Inhibitors&doi=10.1016%2FS1074-5521%2800%2900032-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">RNA aptamers as pathway-specific MAP kinase inhibitors</span></div><div class="casAuthors">Seiwert, Scott D.; Nahreini, Theresa Stines; Aigner, Stefan; Ahn, Natalie G.; Uhlenbeck, Olke C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">833-843</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Background: In eukaryotic cells, many intracellular signaling pathways have closely related mitogen activated protein kinase (MAPK) paralogs as central components.  Although MAPKs are therefore obvious targets to control the cellular responses resulting from the activation of these signaling pathways, the development of inhibitors which target specific cell signaling pathways involving MAPKs has proven difficult.  Results: We used an RNA combinatorial approach to isolate RNAs that inhibit the in vitro phosphorylation activity of extracellular regulated kinase 2 (ERK2).  These inhibitors block phosphorylation by ERK1 and ERK2, but do not inhibit Jun N-terminal kinase of p38 MAPKs.  Kinetic anal. indicates these inhibitors function at high picomolar concns. through the steric exclusion of substrate and ATP binding.  In one case, we identified a compact RNA structural domain responsible for inhibition.  Conclusions: RNA reagents can selectively recognize and inhibit MAPKs involved in a single signal transduction pathway.  The methodol. described here is readily generalizable, and can be used to develop inhibitors of MAPKs involved in other signal transduction pathways.  Such reagents may be valuable tools to analyze and distinguish homologous effectors which regulate distinct signaling responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfZyTtw9i0q7Vg90H21EOLACvtfcHk0li9IjzhnNszeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVWit7c%253D&md5=b0d67a6da7702b6226846505310a8363</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2800%2900032-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252800%252900032-6%26sid%3Dliteratum%253Aachs%26aulast%3DSeiwert%26aufirst%3DS.%2BD.%26aulast%3DNahreini%26aufirst%3DT.%2BS.%26aulast%3DAigner%26aufirst%3DS.%26aulast%3DAhn%26aufirst%3DN.%2BG.%26aulast%3DUhlenbeck%26aufirst%3DO.%2BC.%26atitle%3DRNA%2520Aptamers%2520as%2520Pathway-specific%2520MAP%2520Kinase%2520Inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2000%26volume%3D7%26issue%3D11%26spage%3D833%26epage%3D843%26doi%3D10.1016%2FS1074-5521%2800%2900032-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padlan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malashkevich, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenowitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girvin, M. E.</span></span> <span> </span><span class="NLM_article-title">An RNA Aptamer Possessing a Novel Monovalent Cation-Mediated Fold Inhibits Lysozyme Catalysis by Inhibiting the Binding of Long Natural Substrates</span>. <i>RNA</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1261/rna.043034.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1261%2Frna.043034.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=24570482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFCmtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=447-461&issue=4&author=C.+S.+Padlanauthor=V.+N.+Malashkevichauthor=S.+C.+Almoauthor=M.+Levyauthor=M.+Brenowitzauthor=M.+E.+Girvin&title=An+RNA+Aptamer+Possessing+a+Novel+Monovalent+Cation-Mediated+Fold+Inhibits+Lysozyme+Catalysis+by+Inhibiting+the+Binding+of+Long+Natural+Substrates&doi=10.1261%2Frna.043034.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA aptamer possessing a novel monovalent cation-mediated fold inhibits lysozyme catalysis by inhibiting the binding of long natural substrates</span></div><div class="casAuthors">Padlan, Camille S.; Malashkevich, Vladimir N.; Almo, Steve C.; Levy, Matthew; Brenowitz, Michael; Girvin, Mark E.</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-461</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">RNA aptamers are being developed as inhibitors of macromol. and cellular function, diagnostic tools, and potential therapeutics.  Our understanding of the phys. nature of this emerging class of nucleic acid-protein complexes is limited; few at. resoln. structures have been reported for aptamers bound to their protein target.  Guided by chem. mapping, we systematically minimized an RNA aptamer (Lys1) selected against hen egg white lysozyme.  The resultant 59-nucleotide compact aptamer (Lys1.2minE) retains nanomolar binding affinity and the ability to inhibit lysozyme's catalytic activity.  Our 2.0-Å crystal structure of the aptamer-protein complex reveals a helical stem stabilizing two loops to form a protein binding platform that binds lysozyme distal to the catalytic cleft.  This structure along with complementary soln. analyses illuminate a novel protein-nucleic acid interface.  (1) Only 410 Å2 of solvent accessible surface are buried by aptamer binding.  (2) An unusually small fraction (∼18%) of the RNA-protein interaction is electrostatic, consistent with the limited protein phosphate backbone contacts obsd. in the structure.  (3) A single Na+ stabilizes the loops that constitute the protein-binding platform, and consistent with this observation, Lys1.2minE-lysozyme complex formation takes up rather than displaces cations at low ionic strength.  (4) Lys1.2minE inhibits catalysis of large cell wall substrates but not catalysis of small model substrates; and (5) the helical stem of Lys1.2minE can be shortened to four base pairs (Lys1.2minF) without compromising binding affinity, yielding a 45-nucleotide aptamer whose structure may be an adaptable protein binding platform.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6gQVK4xWrXrVg90H21EOLACvtfcHk0li9IjzhnNszeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFCmtb4%253D&md5=adbc4a9ed26e7e27427fd27c4e103b31</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1261%2Frna.043034.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.043034.113%26sid%3Dliteratum%253Aachs%26aulast%3DPadlan%26aufirst%3DC.%2BS.%26aulast%3DMalashkevich%26aufirst%3DV.%2BN.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DBrenowitz%26aufirst%3DM.%26aulast%3DGirvin%26aufirst%3DM.%2BE.%26atitle%3DAn%2520RNA%2520Aptamer%2520Possessing%2520a%2520Novel%2520Monovalent%2520Cation-Mediated%2520Fold%2520Inhibits%2520Lysozyme%2520Catalysis%2520by%2520Inhibiting%2520the%2520Binding%2520of%2520Long%2520Natural%2520Substrates%26jtitle%3DRNA%26date%3D2014%26volume%3D20%26issue%3D4%26spage%3D447%26epage%3D461%26doi%3D10.1261%2Frna.043034.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, T. D.</span></span> <span> </span><span class="NLM_article-title">A Guide to Simple and Informative Binding Assays</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">4061</span>– <span class="NLM_lpage">4067</span>, <span class="refDoi"> DOI: 10.1091/mbc.e10-08-0683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1091%2Fmbc.e10-08-0683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=21115850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1OmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=4061-4067&issue=23&author=T.+D.+Pollard&title=A+Guide+to+Simple+and+Informative+Binding+Assays&doi=10.1091%2Fmbc.e10-08-0683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to simple and informative binding assays</span></div><div class="casAuthors">Pollard, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4061-4067</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">A review.  The aim of binding assays is to measure interactions between two mols., such as a protein binding another protein, a small mol., or a nucleic acid.  Hard work is required to prep. reagents, but flaws in the design of many binding expts. limit the information obtained.  In particular many expts. fail to measure the affinity of the reactants for each other.  This essay describes simple methods to get the most out of valuable reagents in binding expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKpdSYWYPwSrVg90H21EOLACvtfcHk0li9IjzhnNszeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXis1OmtQ%253D%253D&md5=f837bb1ec71a5babcc5898080d121c27</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e10-08-0683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e10-08-0683%26sid%3Dliteratum%253Aachs%26aulast%3DPollard%26aufirst%3DT.%2BD.%26atitle%3DA%2520Guide%2520to%2520Simple%2520and%2520Informative%2520Binding%2520Assays%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2010%26volume%3D21%26issue%3D23%26spage%3D4061%26epage%3D4067%26doi%3D10.1091%2Fmbc.e10-08-0683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalloway, D.</span></span> <span> </span><span class="NLM_article-title">Density-Dependent Cooperative Non-Specific Binding in Solid-Phase SELEX Affinity Selection</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">7167</span>– <span class="NLM_lpage">7175</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkt477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23737446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1CktLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=7167-7175&issue=14&author=A.+Ozerauthor=B.+S.+Whiteauthor=J.+T.+Lisauthor=D.+Shalloway&title=Density-Dependent+Cooperative+Non-Specific+Binding+in+Solid-Phase+SELEX+Affinity+Selection&doi=10.1093%2Fnar%2Fgkt477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Density-dependent cooperative non-specific binding in solid-phase SELEX affinity selection</span></div><div class="casAuthors">Ozer, Abdullah; White, Brian S.; Lis, John T.; Shalloway, David</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7167-7175</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The non-specific binding of undesired ligands to a target is the primary factor limiting the enrichment of tight-binding ligands in affinity selection.  Soln.-phase non-specific affinity is detd. by the free-energy of ligand binding to a single target.  However, the solid-phase affinity might be higher if a ligand bound concurrently to multiple adjacent immobilized targets in a cooperative manner.  Cooperativity could emerge in this case as a simple consequence of the relationship between the free energy of binding, localization entropy and the spatial distribution of the immobilized targets.  We tested this hypothesis using a SELEX exptl. design and found that non-specific RNA aptamer ligands can concurrently bind up to four bead-immobilized peptide targets, and that this can increase their effective binding affinity by two orders-of-magnitude.  Binding curves were quant. explained by a new statistical mech. model of d.-dependent cooperative binding, which relates cooperative binding to both the target concn. and the target surface d. on the immobilizing substrate.  Target immobilization plays a key role in SELEX and other ligand enrichment methods, particularly in new multiplexed microfluidic purifn. devices, and these results have strong implications for optimizing their performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4PMA4LeNFLLVg90H21EOLACvtfcHk0ljADRqm7gJ1dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1CktLbF&md5=b731565dbd11b2d4c2b861dcd3974a81</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt477%26sid%3Dliteratum%253Aachs%26aulast%3DOzer%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DB.%2BS.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26aulast%3DShalloway%26aufirst%3DD.%26atitle%3DDensity-Dependent%2520Cooperative%2520Non-Specific%2520Binding%2520in%2520Solid-Phase%2520SELEX%2520Affinity%2520Selection%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26issue%3D14%26spage%3D7167%26epage%3D7175%26doi%3D10.1093%2Fnar%2Fgkt477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, J. F.</span></span> <span> </span><span class="NLM_article-title">Assay Development for Protein Kinase Enzymes</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, S.</span></span>, Ed.; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences Publishing</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">102</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=85-102&author=J.+F.+Glickmanauthor=S.+Sittampalam&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGlickman%26aufirst%3DJ.%2BF.%26atitle%3DAssay%2520Development%2520for%2520Protein%2520Kinase%2520Enzymes%26btitle%3DAssay%2520Guidance%2520Manual%26aulast%3DSittampalam%26aufirst%3DS.%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%2520Publishing%26date%3D2012%26spage%3D85%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zegzouti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goueli, S. A.</span></span> <span> </span><span class="NLM_article-title">A Simple and Robust Automated Kinase Profiling Platform Using Luminescent ADP Accumulation Technology</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.1089/adt.2009.0216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Fadt.2009.0216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=20059362" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFejtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=573-584&issue=6&author=B.+Larsonauthor=P.+Banksauthor=H.+Zegzoutiauthor=S.+A.+Goueli&title=A+Simple+and+Robust+Automated+Kinase+Profiling+Platform+Using+Luminescent+ADP+Accumulation+Technology&doi=10.1089%2Fadt.2009.0216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A simple and robust automated kinase profiling platform using luminescent ADP accumulation technology</span></div><div class="casAuthors">Larson, Brad; Banks, Peter; Zegzouti, Hicham; Goueli, Said A.</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">573-584</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Kinases continue to be one of the most important targets in today's drug discovery efforts.  Following the identification of lead compds. through screening efforts, it is important to profile these leads against other kinases within that family, as well as from other families, to ascertain potential off-target effects.  Because many kinase assays require the use of different substrates, optimization time and costs during profiling can be prohibitive.  Here, the versatility of a luminescent ADP accumulation assay, where one set of reagents can be used for a wide variety of kinases with differing Kmapp for ATP and substrates, is demonstrated.  Assay sensitivity allows for the use of low enzyme concns. and small percent ATP conversion levels while still maintaining high signal:background ratios.  A simple, inexpensive automated pipetting system was used to automate the entire process from enzyme optimization through generation of compd. IC50 values.  Agreement with literature values proves this combination of chem. and instrumentation provides a simple, yet robust soln. for automated kinase profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoypq9mhtWGL7Vg90H21EOLACvtfcHk0ljADRqm7gJ1dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFejtLw%253D&md5=0ad5fb475c052ebb86607a8bc74bd349</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1089%2Fadt.2009.0216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2009.0216%26sid%3Dliteratum%253Aachs%26aulast%3DLarson%26aufirst%3DB.%26aulast%3DBanks%26aufirst%3DP.%26aulast%3DZegzouti%26aufirst%3DH.%26aulast%3DGoueli%26aufirst%3DS.%2BA.%26atitle%3DA%2520Simple%2520and%2520Robust%2520Automated%2520Kinase%2520Profiling%2520Platform%2520Using%2520Luminescent%2520ADP%2520Accumulation%2520Technology%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2009%26volume%3D7%26issue%3D6%26spage%3D573%26epage%3D584%26doi%3D10.1089%2Fadt.2009.0216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNitto, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">English, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greig, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunney, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McTigue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quenzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmett, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demetri, G. D.</span></span> <span> </span><span class="NLM_article-title">KIT Kinase Mutants Show Unique Mechanisms of Drug Resistance to Imatinib and Sunitinib in Gastrointestinal Stromal Tumor Patients</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1542</span>– <span class="NLM_lpage">1547</span>, <span class="refDoi"> DOI: 10.1073/pnas.0812413106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1073%2Fpnas.0812413106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=19164557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVOms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1542-1547&issue=5&author=K.+S.+Gajiwalaauthor=J.+C.+Wuauthor=J.+Christensenauthor=G.+D.+Deshmukhauthor=W.+Diehlauthor=J.+P.+DiNittoauthor=J.+M.+Englishauthor=M.+J.+Greigauthor=Y.+Heauthor=S.+L.+Jacquesauthor=E.+A.+Lunneyauthor=M.+McTigueauthor=D.+Molinaauthor=T.+Quenzerauthor=P.+A.+Wellsauthor=X.+Yuauthor=Y.+Zhangauthor=A.+Zouauthor=M.+R.+Emmettauthor=A.+G.+Marshallauthor=H.+Zhangauthor=G.+D.+Demetri&title=KIT+Kinase+Mutants+Show+Unique+Mechanisms+of+Drug+Resistance+to+Imatinib+and+Sunitinib+in+Gastrointestinal+Stromal+Tumor+Patients&doi=10.1073%2Fpnas.0812413106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">KIT kinase mutants show unique mechanisms of drug resistance to imatinib and sunitinib in gastrointestinal stromal tumor patients</span></div><div class="casAuthors">Gajiwala, Ketan S.; Wu, Joe C.; Christensen, James; Deshmukh, Gayatri D.; Diehl, Wade; DiNitto, Jonathan P.; English, Jessie M.; Greig, Michael J.; He, You-Ai; Jacques, Suzanne L.; Lunney, Elizabeth A.; McTigue, Michele; Molina, Dave; Quenzer, Terri; Wells, Peter A.; Yu, Xiu; Zhang, Yan; Zou, Aihua; Emmett, Mark R.; Marshall, Alan G.; Zhang, Hui-Min; Demetri, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1547</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Most gastrointestinal stromal tumors (GISTs) exhibit aberrant activation of the receptor tyrosine kinase (RTK) KIT.  The efficacy of the inhibitors imatinib mesylate and sunitinib malate in GIST patients has been linked to their inhibition of these mutant KIT proteins.  However, patients on imatinib can acquire secondary KIT mutations that render the protein insensitive to the inhibitor.  Sunitinib has shown efficacy against certain imatinib-resistant mutants, although a subset that resides in the activation loop, including D816H/V, remains resistant.  Biochem. and structural studies were undertaken to det. the mol. basis of sunitinib resistance.  Our results show that sunitinib targets the autoinhibited conformation of WT KIT and that the D816H mutant undergoes a shift in conformational equil. toward the active state.  These findings provide a structural and enzymol. explanation for the resistance profile obsd. with the KIT inhibitors.  Prospectively, they have implications for understanding oncogenic kinase mutants and for circumventing drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkwFxa-p-iWbVg90H21EOLACvtfcHk0ljADRqm7gJ1dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVOms78%253D&md5=dd226c914c6f9a258426c0f4984a5258</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812413106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812413106%26sid%3Dliteratum%253Aachs%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DWu%26aufirst%3DJ.%2BC.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DDeshmukh%26aufirst%3DG.%2BD.%26aulast%3DDiehl%26aufirst%3DW.%26aulast%3DDiNitto%26aufirst%3DJ.%2BP.%26aulast%3DEnglish%26aufirst%3DJ.%2BM.%26aulast%3DGreig%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLunney%26aufirst%3DE.%2BA.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DMolina%26aufirst%3DD.%26aulast%3DQuenzer%26aufirst%3DT.%26aulast%3DWells%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DEmmett%26aufirst%3DM.%2BR.%26aulast%3DMarshall%26aufirst%3DA.%2BG.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DDemetri%26aufirst%3DG.%2BD.%26atitle%3DKIT%2520Kinase%2520Mutants%2520Show%2520Unique%2520Mechanisms%2520of%2520Drug%2520Resistance%2520to%2520Imatinib%2520and%2520Sunitinib%2520in%2520Gastrointestinal%2520Stromal%2520Tumor%2520Patients%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26issue%3D5%26spage%3D1542%26epage%3D1547%26doi%3D10.1073%2Fpnas.0812413106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <i>Enzymes: A Practical Introduction to Structure, Mechanism, and Data Analysis</i>; <span class="NLM_publisher-name">Wiley-VCH, Inc.</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2000</span>; pp  <span class="NLM_fpage">266</span>– <span class="NLM_lpage">303</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2F0471220639.ch8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&pages=266-303&author=R.+A.+Copeland&title=Enzymes%3A+A+Practical+Introduction+to+Structure%2C+Mechanism%2C+and+Data+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2F0471220639.ch8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471220639.ch8%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEnzymes%253A%2520A%2520Practical%2520Introduction%2520to%2520Structure%252C%2520Mechanism%252C%2520and%2520Data%2520Analysis%26pub%3DWiley-VCH%252C%2520Inc%26date%3D2000%26spage%3D266%26epage%3D303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Georgi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Montalvan, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span> <span> </span><span class="NLM_article-title">Binding Kinetics in Drug Discovery-A Current Perspective</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.2741/4470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.2741%2F4470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=27814600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1CktLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=21-47&issue=1&author=V.+Georgiauthor=D.+Andresauthor=A.+E.+Fernandez-Montalvanauthor=C.+M.+Stegmannauthor=A.+Beckerauthor=A.+Mueller-Fahrnow&title=Binding+Kinetics+in+Drug+Discovery-A+Current+Perspective&doi=10.2741%2F4470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Binding kinetics in drug discovery - a current perspective</span></div><div class="casAuthors">Georgi, Victoria; Andres, Dorothee; Fernandez-Montalvan, Amaury E.; Stegmann, Christian M.; Becker, Andreas; Mueller-Fahrnow, Anke</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-47</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">The impact of target binding kinetics (BK) on the clin. performance of therapeutic agents is presently a topic of intense debate in drug discovery.  While retrospective studies suggest that BK is a differentiating parameter in marketed medicines, it is yet unclear how this information could be used to prioritize drug candidates during lead optimization.  Motivated by the question whether BK can be understood and rationally optimized, we review the most relevant literature in the field, with special focus on selected examples from our organization.  First we discuss structure-kinetic relationships (SKR), and how they can be influenced by factors such as conformational changes, mol. flexibility, hydrogen bonds, hydrophobicity, water mols. and (reversible-) covalent bonds.  We then introduce the methodologies currently used for the investigation of BK parameters, briefly commenting on their strengths, weaknesses and future trends.  Finally, we present our current perspective on the integration of BK in the drug discovery process, aiming to stimulate further thoughts on this important subject.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3xgmlxpV2zbVg90H21EOLACvtfcHk0lg-VJIEGQvwyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1CktLvM&md5=1ca3deab8d1b2e331a4789bf2f6d2d10</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.2741%2F4470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F4470%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgi%26aufirst%3DV.%26aulast%3DAndres%26aufirst%3DD.%26aulast%3DFernandez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26atitle%3DBinding%2520Kinetics%2520in%2520Drug%2520Discovery-A%2520Current%2520Perspective%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2017%26volume%3D22%26issue%3D1%26spage%3D21%26epage%3D47%26doi%3D10.2741%2F4470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span> <span> </span><span class="NLM_article-title">Defining Balanced Conditions for Inhibitor Screening Assays That Target Bisubstrate Enzymes</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1177/1087057108328763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1177%2F1087057108328763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=19196704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFWksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=111-120&issue=2&author=J.+Yangauthor=R.+A.+Copelandauthor=Z.+Lai&title=Defining+Balanced+Conditions+for+Inhibitor+Screening+Assays+That+Target+Bisubstrate+Enzymes&doi=10.1177%2F1087057108328763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Defining balanced conditions for inhibitor screening assays that target bisubstrate enzymes</span></div><div class="casAuthors">Yang, Jingsong; Copeland, Robert A.; Lai, Zhihong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-120</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is a common mechanism for identifying lead compds. for drug discovery efforts.  Small mols. can inhibit enzymes by a variety of mechanisms, such as competitive, noncompetitive, and uncompetitive with respect to the substrate(s) of the catalytic reaction.  To optimize the chances of finding the broadest diversity of inhibitor modalities during screening, one must run assays under "balanced" conditions where the potency of inhibitors with various modes of action falls within a similar range.  When an enzyme reaction involves more than one substrate, the definition and assessment of the apparent potency of inhibitors (IC50), in relation to their true potency (Ki), can be nontrivial.  This article provides a theor. anal., on the basis of the Cheng-Prusoff derivation, of the IC50/Ki relationship of bisubstrate enzyme reactions following various sequential kinetic mechanisms, as well as the application and limitations of this information for defining optimal screening conditions for such enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM9O6ZD0r6l7Vg90H21EOLACvtfcHk0lg-VJIEGQvwyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFWksr4%253D&md5=076f5a6ad75c4fc0e01d59ca5ea53fe2</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1177%2F1087057108328763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108328763%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DLai%26aufirst%3DZ.%26atitle%3DDefining%2520Balanced%2520Conditions%2520for%2520Inhibitor%2520Screening%2520Assays%2520That%2520Target%2520Bisubstrate%2520Enzymes%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26issue%3D2%26spage%3D111%26epage%3D120%26doi%3D10.1177%2F1087057108328763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelow, J.</span>; <span class="NLM_string-name">Dewe, W.</span>; <span class="NLM_string-name">Iversen, P. W.</span>; <span class="NLM_string-name">Brooks, H. B.</span>; <span class="NLM_string-name">Radding, J. A.</span>; <span class="NLM_string-name">McGee, J.</span>; <span class="NLM_string-name">Weidner, J.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Action Assays for Enzymes</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, S.</span></span>, Ed.; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences Publishing</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">84</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=58-84&author=J.+Strelow&author=W.+Dewe&author=P.+W.+Iversen&author=H.+B.+Brooks&author=J.+A.+Radding&author=J.+McGee&author=J.+Weidnerauthor=S.+Sittampalam&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%26atitle%3DMechanism%2520of%2520Action%2520Assays%2520for%2520Enzymes%26btitle%3DAssay%2520Guidance%2520Manual%26aulast%3DSittampalam%26aufirst%3DS.%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%2520Publishing%26date%3D2012%26spage%3D58%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span> <span> </span><span class="NLM_article-title">New Directions in Targeting Protein Kinases: Focusing Upon True Allosteric and Bivalent Inhibitors</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">2936</span>– <span class="NLM_lpage">2945</span>, <span class="refDoi"> DOI: 10.2174/138161212800672813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.2174%2F138161212800672813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=22571662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=2936-2945&issue=20&author=V.+Lambaauthor=I.+Ghosh&title=New+Directions+in+Targeting+Protein+Kinases%3A+Focusing+Upon+True+Allosteric+and+Bivalent+Inhibitors&doi=10.2174%2F138161212800672813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">New directions in targeting protein kinases: focusing upon true allosteric and bivalent inhibitors</span></div><div class="casAuthors">Lamba, Vandana; Ghosh, Indraneel</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2936-2945</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, therapeutics that target subsets of the 518 human protein kinases have played a vital role in the fight against cancer.  Protein kinases are typically targeted at the ATP-binding cleft by type I and II inhibitors; however, the high sequence and structural homol. shared by protein kinases, esp. at the ATP-binding site, inherently leads to polypharmacol.  In order to discover or design truly selective protein kinase inhibitors as both pharmacol. reagents and safer therapeutic leads, new efforts are needed to target kinases outside the ATP cleft.  Recent advances include the serendipitous discovery of type III inhibitors that bind a site proximal to the ATP pocket as well as the truly allosteric type IV inhibitors that target protein kinases distal to the substrate-binding pocket.  These new classes of inhibitors are often selective, but usually display moderate affinities.  Here, the authors discuss the different classes of inhibitors with an emphasis on bisubstrate and bivalent inhibitors (type V) that combine different inhibitor classes.  These inhibitors have the potential to couple the high affinity and potency of traditional active site-targeted small-mol. inhibitors with the selectivity of inhibitors that target the protein kinase surface outside ATP cleft.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIIgeMxhCrrVg90H21EOLACvtfcHk0lgAhOuyyza-2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaltrk%253D&md5=36a44e36709507b5053bfdfe7d09e5e9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.2174%2F138161212800672813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212800672813%26sid%3Dliteratum%253Aachs%26aulast%3DLamba%26aufirst%3DV.%26aulast%3DGhosh%26aufirst%3DI.%26atitle%3DNew%2520Directions%2520in%2520Targeting%2520Protein%2520Kinases%253A%2520Focusing%2520Upon%2520True%2520Allosteric%2520and%2520Bivalent%2520Inhibitors%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26issue%3D20%26spage%3D2936%26epage%3D2945%26doi%3D10.2174%2F138161212800672813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavrin, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiah, E.</span></span> <span> </span><span class="NLM_article-title">Approaches to Discover Non-ATP Site Kinase Inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1039/C2MD20180A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1039%2FC2MD20180A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=41-51&issue=1&author=L.+K.+Gavrinauthor=E.+Saiah&title=Approaches+to+Discover+Non-ATP+Site+Kinase+Inhibitors&doi=10.1039%2FC2MD20180A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to discover non-ATP site kinase inhibitors</span></div><div class="casAuthors">Gavrin, Lori Krim; Saiah, Eddine</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-51</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The catalytic domain of kinases shows a high degree of sequence homol., esp. for kinases that belong to the same family.  They share a common ATP binding site with a conserved activation loop and similar three-dimensional structure.  Consequently, a major challenge in kinase research exists in achieving selectivity among the >500 family members, since they all process the same substrate.  In addn. to requiring selectivity against other kinases, ATP site inhibitors must also bind tightly to overcome the high physiol. concn. of ATP in the cell.  Furthermore, the development of novel ATP site inhibitors is becoming increasingly challenging, as many ATP competitive scaffolds have previously been disclosed.  In order to develop compds. with better selectivity among kinases, inhibitors that bind outside the ATP site show great promise and are currently being explored by many groups.  This review will highlight the most commonly used methods to discover small mol. Type III and IV kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xJcf4-RJ9rVg90H21EOLACvtfcHk0lgAhOuyyza-2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N&md5=97acf2247e5516c1649c1d964c867656</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2FC2MD20180A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20180A%26sid%3Dliteratum%253Aachs%26aulast%3DGavrin%26aufirst%3DL.%2BK.%26aulast%3DSaiah%26aufirst%3DE.%26atitle%3DApproaches%2520to%2520Discover%2520Non-ATP%2520Site%2520Kinase%2520Inhibitors%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26issue%3D1%26spage%3D41%26epage%3D51%26doi%3D10.1039%2FC2MD20180A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatachalam, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surana, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C.</span></span> <span> </span><span class="NLM_article-title">Conformational Landscape of the Epidermal Growth Factor Receptor Kinase Reveals a Mutant Specific Allosteric Pocket</span>. <i>Chemical Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">5212</span>– <span class="NLM_lpage">5222</span>, <span class="refDoi"> DOI: 10.1039/C8SC01262H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1039%2FC8SC01262H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29997876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsFCntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=5212-5222&issue=23&author=S.+Kannanauthor=G.+Venkatachalamauthor=H.+H.+Limauthor=U.+Suranaauthor=C.+Verma&title=Conformational+Landscape+of+the+Epidermal+Growth+Factor+Receptor+Kinase+Reveals+a+Mutant+Specific+Allosteric+Pocket&doi=10.1039%2FC8SC01262H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational landscape of the epidermal growth factor receptor kinase reveals a mutant specific allosteric pocket</span></div><div class="casAuthors">Kannan, Srinivasaraghavan; Venkatachalam, Gireedhar; Lim, Hong Hwa; Surana, Uttam; Verma, Chandra</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5212-5222</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Activating mutations within the epidermal growth factor receptor (EGFR) kinase domain give rise to several cancers including Non-Small Cell Lung Cancer (NSCLC).  Small mol. inhibitors targeted at these mutants have proven to be clin. successful drugs.  These mols. are ATP competitive and rapidly result in the emergence of resistance.  Recently Jia et al. [Nature, 2016, 534, 129-132] reported a small mol. inhibitor (called EAI045) that binds at an allosteric pocket, does not compete with ATP and displays high potency and selectivity towards certain activating mutants (L858R, T790M, L858R/T790M) of EGFR, with IC50 values ranging from 3 nM to 49 nM.  We present here a study combining extensive mol. dynamics simulations with binding assays to provide a structural basis underlying the mechanism of binding of this mol.  It appears that in mutants, conformational destabilization of the short helix (that carries Leu858 in the wildtype), is key to the exposure of the allosteric pocket which otherwise is occluded by a set of sidechains including L858.  We extend this hypothesis to show that a similar mechanism would enable the mol. to inhibit EGFRL861Q which is another oncogenic mutant and validate this with binding expts.  The screening of the human structural kinome revealed at least 12 other oncogenic kinases which carry at least one activating mutant in this disorder-prone region and hence would be amenable to allosteric inhibition by mols. such as EAI045.  Our study characterizes a druggable allosteric pocket which appears to be specific to certain oncogenic mutants of the EGFR and holds therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNJMPumMkZFbVg90H21EOLACvtfcHk0lglMFmpwhSEMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsFCntLo%253D&md5=57b221846f35482d03d19944ccc48919</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1039%2FC8SC01262H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC01262H%26sid%3Dliteratum%253Aachs%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DVenkatachalam%26aufirst%3DG.%26aulast%3DLim%26aufirst%3DH.%2BH.%26aulast%3DSurana%26aufirst%3DU.%26aulast%3DVerma%26aufirst%3DC.%26atitle%3DConformational%2520Landscape%2520of%2520the%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Kinase%2520Reveals%2520a%2520Mutant%2520Specific%2520Allosteric%2520Pocket%26jtitle%3DChemical%2520Science%26date%3D2018%26volume%3D9%26issue%3D23%26spage%3D5212%26epage%3D5222%26doi%3D10.1039%2FC8SC01262H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span> <span> </span><span class="NLM_article-title">Identification of RNA Aptamer Specific to Mutant KRAS Protein</span>. <i>Oligonucleotides</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1089/oli.2010.0231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Foli.2010.0231" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=155-161&issue=3&author=S.+Jeongauthor=S.+R.+Hanauthor=Y.+J.+Leeauthor=J.+H.+Kimauthor=S.+Lee&title=Identification+of+RNA+Aptamer+Specific+to+Mutant+KRAS+Protein&doi=10.1089%2Foli.2010.0231"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1089%2Foli.2010.0231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Foli.2010.0231%26sid%3Dliteratum%253Aachs%26aulast%3DJeong%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DS.%2BR.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520RNA%2520Aptamer%2520Specific%2520to%2520Mutant%2520KRAS%2520Protein%26jtitle%3DOligonucleotides%26date%3D2012%26volume%3D20%26issue%3D3%26spage%3D155%26epage%3D161%26doi%3D10.1089%2Foli.2010.0231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rashid, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awan, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, G.</span></span> <span> </span><span class="NLM_article-title">The Isolation of an RNA Aptamer Targeting to p53 Protein with Single Amino Acid Mutation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span> (<span class="NLM_issue">32</span>),  <span class="NLM_fpage">10002</span>– <span class="NLM_lpage">10007</span>, <span class="refDoi"> DOI: 10.1073/pnas.1502159112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1073%2Fpnas.1502159112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=26216949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Crt73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=10002-10007&issue=32&author=L.+Chenauthor=F.+Rashidauthor=A.+Shahauthor=H.+M.+Awanauthor=M.+Wuauthor=A.+Liuauthor=M.+Wuauthor=J.+Wangauthor=T.+Zhuauthor=Z.+Luoauthor=G.+Shan&title=The+Isolation+of+an+RNA+Aptamer+Targeting+to+p53+Protein+with+Single+Amino+Acid+Mutation&doi=10.1073%2Fpnas.1502159112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">The isolation of an RNA aptamer targeting to p53 protein with single amino acid mutation</span></div><div class="casAuthors">Chen, Liang; Rashid, Farooq; Shah, Abdullah; Awan, Hassaan M.; Wu, Mingming; Liu, An; Wang, Jun; Zhu, Tao; Luo, Zhaofeng; Shan, Ge</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">10002-10007</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">P53, known as a tumor suppressor, is a DNA binding protein that regulates cell cycle, activates DNA repair proteins, and triggers apoptosis in multicellular animals.  More than 50% of human cancers contain a mutation or deletion of the p53 gene, and p53R175 is one of the hot spots of p53 mutation.  Nucleic acid aptamers are short single-stranded oligonucleotides that are able to bind various targets, and they are typically isolated from an exptl. procedure called systematic evolution of ligand exponential enrichment (SELEX).  Using a previously unidentified strategy of contrast screening with SELEX, we have isolated an RNA aptamer targeting p53R175H.  This RNA aptamer (p53R175H-APT) has a significantly stronger affinity to p53R175H than to the wild-type p53 in both in vitro and in vivo assays.  P53R175H-APT decreased the growth rate, weakened the migration capability, and triggered apoptosis in human lung cancer cells harboring p53R175H.  Further anal. actually indicated that p53R175H-APT might partially rescue or correct the p53R175H to function more like the wild-type p53.  In situ injections of p53R175H-APT to the tumor xenografts confirmed the effects of this RNA aptamer on p53R175H mutation in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-EyKwnVM037Vg90H21EOLACvtfcHk0lglMFmpwhSEMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Crt73L&md5=f744dd150680cf325cdc83234e328b96</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1502159112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1502159112%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DRashid%26aufirst%3DF.%26aulast%3DShah%26aufirst%3DA.%26aulast%3DAwan%26aufirst%3DH.%2BM.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DShan%26aufirst%3DG.%26atitle%3DThe%2520Isolation%2520of%2520an%2520RNA%2520Aptamer%2520Targeting%2520to%2520p53%2520Protein%2520with%2520Single%2520Amino%2520Acid%2520Mutation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26issue%3D32%26spage%3D10002%26epage%3D10007%26doi%3D10.1073%2Fpnas.1502159112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kratschmer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span> <span> </span><span class="NLM_article-title">Effect of Chemical Modifications on Aptamer Stability in Serum</span>. <i>Nucleic Acid Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">335</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1089/nat.2017.0680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Fnat.2017.0680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=28945147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKqtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=335-344&issue=6&author=C.+Kratschmerauthor=M.+Levy&title=Effect+of+Chemical+Modifications+on+Aptamer+Stability+in+Serum&doi=10.1089%2Fnat.2017.0680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of Chemical Modifications on Aptamer Stability in Serum</span></div><div class="casAuthors">Kratschmer, Christina; Levy, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acid Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">335-344</span>CODEN:
                <span class="NLM_cas:coden">NATUCU</span>;
        ISSN:<span class="NLM_cas:issn">2159-3337</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">There is increasing interest in the use of aptamers for the development of therapeutics.  However, as oligonucleotides, aptamers are susceptible to nuclease degrdn.; poor serum stability is likely to neg. affect in vivo function.  Modified nucleotides have been used to thwart nuclease degrdn.  However, few studies report the serum stability of selected aptamers.  In this study, we examd. the effect of various chem. modifications (2'-deoxy, 2'-hydroxyl, 2'-fluoro, and 2'-O-methyl) on the stability of a control oligonucleotide sequence following incubation in frozen human, fresh mouse, and fresh human serum.  We also assessed the effect of the 3' inverted dT cap on stability.  Surprisingly, we found that fYrR (2'-fluoro RNA) is only roughly as stable as DNA (2'-deoxy).  Interestingly, the inclusion of a 3' inverted dT cap had only a modest effect on serum stability, if any.  In one instance, the addn. of a 3' inverted dT cap rendered a mol. composed of DNA more stable than its fYrR counterpart.  By far, fully modified oligonucleotides (100% 2-O-Me or 2'-O-Me A, C, and U in combination with 2'-fluoro G, termed fGmH) had the longest half-lives.  These compns. demonstrated little degrdn. in human serum even after prolonged incubation.  Together these results support the need for using fully modified aptamers for in vivo applications and should encourage those in the field to exploit newer polymerase variants capable of directly generating such polymers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbb3qGrP1G77Vg90H21EOLACvtfcHk0lglMFmpwhSEMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKqtrzI&md5=0e3c675745073ea2967e7dfedd0502dc</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1089%2Fnat.2017.0680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fnat.2017.0680%26sid%3Dliteratum%253Aachs%26aulast%3DKratschmer%26aufirst%3DC.%26aulast%3DLevy%26aufirst%3DM.%26atitle%3DEffect%2520of%2520Chemical%2520Modifications%2520on%2520Aptamer%2520Stability%2520in%2520Serum%26jtitle%3DNucleic%2520Acid%2520Ther.%26date%3D2017%26volume%3D27%26issue%3D6%26spage%3D335%26epage%3D344%26doi%3D10.1089%2Fnat.2017.0680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pryer, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherrington, J. M.</span></span> <span> </span><span class="NLM_article-title">SU11248 Inhibits KIT and Platelet-derived Growth Factor Receptor β in Preclinical Models of Human Small Cell Lung Cancer</span>. <i>Molecular Cancer Therapeutics</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">478</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=12748309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvVersbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=471-478&issue=5&author=T.+J.+Abramsauthor=L.+B.+Leeauthor=L.+J.+Murrayauthor=N.+J.+Pryerauthor=J.+M.+Cherrington&title=SU11248+Inhibits+KIT+and+Platelet-derived+Growth+Factor+Receptor+%CE%B2+in+Preclinical+Models+of+Human+Small+Cell+Lung+Cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">SU11248 inhibits KIT and platelet-derived growth factor receptor β in preclinical models of human small cell lung cancer</span></div><div class="casAuthors">Abrams, Tinya J.; Lee, Leslie B.; Murray, Lesley J.; Pryer, Nancy K.; Cherrington, Julie M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-478</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The purpose of this study was to evaluate the activity of the indolinone kinase inhibitor SU11248 against the receptor tyrosine kinase KIT in vitro and in vivo, examine the role of KIT in small cell lung cancer (SCLC), and anticipate clin. utility of SU11248 in SCLC.  SU11248 is an oral, multitargeted tyrosine kinase inhibitor with direct antitumor and antiangiogenic activity through targeting platelet-derived growth factor receptor (PDGFR), vascular endothelial growth factor receptor, KIT, and FLT3 receptors.  Treatment of the KIT-expressing SCLC-derived NCI-H526 cell line in vitro with SU11248 resulted in dose-dependent inhibition of stem cell factor-stimulated KIT phosphotyrosine levels and proliferation.  The biol. significance of KIT inhibition was evaluated in vivo by treating mice bearing s.c. NCI-H526 tumors with SU11248 or another structurally unrelated KIT inhibitor, STI571 (Gleevec), which is also known to inhibit Bcr-AbI and PDGFRβ.  SU11248 treatment resulted in significant tumor growth inhibition, whereas inhibition from STI571 treatment was less dramatic.  Both compds. reduced phospho-KIT levels in NCI-H526 tumors, with a greater redn. by SU11248, correlating with efficacy.  Likewise, phospho-PDGFRβ levels contributed by tumor stroma and with known involvement in angiogenesis were strongly inhibited by SU11248 and less so by STI571.  Because platinum-based chemotherapy is part of the std. of care for SCLC, SU11248 was combined with cisplatin, and significant tumor growth delay was measured compared with either agent alone.  These results expand the profile of SU11248 as a KIT signaling inhibitor and suggest that SU11248 may have clin. potential in the treatment of SCLC via direct antitumor activity mediated via KIT as well as tumor angiogenesis via vascular endothelial growth factor receptor FLK1/KDR and PDGFRβ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLMjnXAK7tXLVg90H21EOLACvtfcHk0lifxVOBEBtJ5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvVersbw%253D&md5=732c0ed4b392c2d7ead9b1be6f6a7927</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbrams%26aufirst%3DT.%2BJ.%26aulast%3DLee%26aufirst%3DL.%2BB.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DPryer%26aufirst%3DN.%2BJ.%26aulast%3DCherrington%26aufirst%3DJ.%2BM.%26atitle%3DSU11248%2520Inhibits%2520KIT%2520and%2520Platelet-derived%2520Growth%2520Factor%2520Receptor%2520%25CE%25B2%2520in%2520Preclinical%2520Models%2520of%2520Human%2520Small%2520Cell%2520Lung%2520Cancer%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26date%3D2003%26volume%3D2%26issue%3D5%26spage%3D471%26epage%3D478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krystal, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carison, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litz, J.</span></span> <span> </span><span class="NLM_article-title">Induction of Apoptosis and Inhibition of Small Cell Lung Cancer Growth by the Quinoxaline Tyrphostins</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2203</span>– <span class="NLM_lpage">2208</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=9187122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2sXjslGrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=2203-2208&author=G.+W.+Krystalauthor=P.+Carisonauthor=J.+Litz&title=Induction+of+Apoptosis+and+Inhibition+of+Small+Cell+Lung+Cancer+Growth+by+the+Quinoxaline+Tyrphostins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis and inhibition of small cell lung cancer growth by the quinoxaline tyrphostins</span></div><div class="casAuthors">Krystal, Geoffrey W.; Carlson, Patricia; Litz, Julie</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2203-2208</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Coexpression of the Kit receptor tyrosine kinase and its ligand, stem cell factor (SCF), occurs in a high proportion of small cell lung cancers (SCLCs) and drives an autocrine loop that enhances proliferation.  To det. whether the autocrine loop affects apoptosis, SCLC cells expressing only SCF or both SCF and Kit were deprived of growth factors for 72 h and the relative no. of cells undergoing apoptosis was assessed using nuclear DNA content and terminal deoxynucleotidyl transferase-mediated dUTP nick end labeling assays.  Coexpression of SCF and Kit inhibited apoptosis; apoptosis could, in turn, be enhanced by the addn. of the quinoxaline tyrosine kinase inhibitors (AG 1295 and AG 1296), which are specific antagonists of the platelet-derived growth factor receptor and Kit.  Treatment of the H526 cell line, which is growth-stimulated by sol. SCF, with AG 1296 resulted in a marked decrease in growth and an increase in apoptosis in a dose-dependent fashion.  Growth inhibition correlated well with the inhibition of Kit tyrosine phosphorylation.  The AG 1296 compd. at its max. sol. concn. inhibited the growth of 5 of 6 SCLC cell lines in complete medium by an av. of 50%.  These data suggest that optimized pharmacol. inhibitors of Kit activity may be a new class of compds. potentially useful in the treatment of SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHfg0XwkNAT7Vg90H21EOLACvtfcHk0lifxVOBEBtJ5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjslGrtrg%253D&md5=118e61bb7a80d453dd0b88cfdc572e2b</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrystal%26aufirst%3DG.%2BW.%26aulast%3DCarison%26aufirst%3DP.%26aulast%3DLitz%26aufirst%3DJ.%26atitle%3DInduction%2520of%2520Apoptosis%2520and%2520Inhibition%2520of%2520Small%2520Cell%2520Lung%2520Cancer%2520Growth%2520by%2520the%2520Quinoxaline%2520Tyrphostins%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D2203%26epage%3D2208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krystal, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honsawek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchdunger, E.</span></span> <span> </span><span class="NLM_article-title">The Selective Tyrosine Kinase Inhibitor STI571 Inhibits Small Cell Lung Cancer Growth</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">3319</span>– <span class="NLM_lpage">3326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10955819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsVOmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=3319-3326&author=G.+W.+Krystalauthor=S.+Honsawekauthor=J.+Litzauthor=E.+Buchdunger&title=The+Selective+Tyrosine+Kinase+Inhibitor+STI571+Inhibits+Small+Cell+Lung+Cancer+Growth"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">The selective tyrosine kinase inhibitor STI571 inhibits small cell lung cancer growth</span></div><div class="casAuthors">Krystal, Geoffrey W.; Honsawek, Sittisak; Litz, Julie; Buchdunger, Elisabeth</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3319-3326</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">At least 70% of small cell lung cancers express the Kit receptor Tyr kinase and its ligand, stem cell factor (SCF).  Numerous lines of evidence have demonstrated that this coexpression constitutes a functional autocrine loop, suggesting that inhibitors of Kit Tyr kinase activity could have therapeutic efficacy in this disease.  STI571, formerly known as CGP 57148B, is a p.o. bioavailable 2-phenylaminopyrimide deriv. that was designed as an Abl Tyr kinase inhibitor, but also has efficacy against the platelet-derived growth factor receptor and Kit in vitro.  Pretreatment of the H526 small cell lung cancer (SCLC) cell line with STI571 inhibited SCF-mediated Kit activation with an IC50 of 0.1 μM as measured by inhibition of receptor Tyr phosphorylation and 0.2 μM as measured by immune complex kinase assay.  Tis paralleled the inhibition of SCF-mediated growth by STI571, which had an IC50 of ∼0.3 μM.  Growth inhibition in SCF-contg. medium was accompanied by induction of apoptosis.  STI571 efficiently blocked SCF-mediated activation of mitogen-activated protein kinase and Akt, but did not affect insulin-like growth factor-1 or serum-mediated mitogen-activated protein kinase or Akt activation.  Growth of 5 of 6 SCLC cell lines in medium contg. 10% FCS was inhibited by STI571 with an IC50 of ∼5 μM.  Growth inhibition in serum-contg. medium appeared to be cytostatic in nature because no increase in apoptosis was obsd.  Despite this growth inhibition, STI571 failed to enhance the cytotoxicity of either carboplatinum or etoposide when coadministered.  However, taken together with the minimal toxicity that this compd. has shown in preclin. studies, these data suggest that STI571 could have a role in the treatment of SCLC, possibly to block or slow recurrence after chemotherapy-induced remissions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXlmbLvNhphbVg90H21EOLACvtfcHk0lifxVOBEBtJ5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsVOmtLo%253D&md5=25ad853eb069e2741c21a9edc8104df4</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrystal%26aufirst%3DG.%2BW.%26aulast%3DHonsawek%26aufirst%3DS.%26aulast%3DLitz%26aufirst%3DJ.%26aulast%3DBuchdunger%26aufirst%3DE.%26atitle%3DThe%2520Selective%2520Tyrosine%2520Kinase%2520Inhibitor%2520STI571%2520Inhibits%2520Small%2520Cell%2520Lung%2520Cancer%2520Growth%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2000%26volume%3D6%26spage%3D3319%26epage%3D3326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krystal, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulanke, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litz, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Phosphatidylinositol 3-Kinase-Akt Signaling Blocks Growth, Promotes Apoptosis, and Enhances Sensitivity of Small Cell Lung Cancer Cells to Chemotherapy</span>. <i>Molecular Cancer Therapeutics</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">922</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=12481412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFOnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=913-922&author=G.+W.+Krystalauthor=G.+Sulankeauthor=J.+Litz&title=Inhibition+of+Phosphatidylinositol+3-Kinase-Akt+Signaling+Blocks+Growth%2C+Promotes+Apoptosis%2C+and+Enhances+Sensitivity+of+Small+Cell+Lung+Cancer+Cells+to+Chemotherapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of phosphatidylinositol 3-kinase-Akt signaling blocks growth, promotes apoptosis, and enhances sensitivity of small cell lung cancer cells to chemotherapy</span></div><div class="casAuthors">Krystal, Geoffrey W.; Sulanke, Geoffrey; Litz, Julie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">913-922</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A promising therapeutic alternative to inhibition of growth factor receptors is the inhibition of downstream signal transduction pathways.  Such an approach may be esp. important in tumors that can use signals from multiple growth factor receptors for growth and survival.  Both stem cell factor (SCF) and insulin-like growth factor (IGF)-I, components of prominent small cell lung cancer (SCLC) autocrine loops, as well as FCS, can potently activate phosphatidylinositol 3-kinase (PI3K)-Akt signaling, albeit with different kinetics.  SCF-induced PI3K-Akt activation occurs rapidly but fades within 60 min; IGF-I and FCS-induced activation persists for at least 6 h.  SCF and IGF-I-mediated growth was potently inhibited by LY294002 in proportion to its ability to inhibit phosphatidylinositol 3-kinase (PI3K)-Akt signaling.  A panel of six SCLC cell lines grown in 10% FCS was also very sensitive to LY294002, with av. IC50 and LD50 of 5 and 25 μM, resp.  These drug concns. suppressed the growth of the MRC-5 pulmonary fibroblast cell line and primary bronchial epithelial cells but did not induce significant cell death.  Because LY294002 can also inhibit PI3K-related enzymes, we confirmed the role of the PI3K-Akt pathway in SCLC using doxycycline-regulated expression of a dominant-neg. (kinase dead) and a constitutively active (CA; myristolated) Akt allele.  Expression of dominant-neg. Akt, which could only be achieved at relatively low levels, completely inhibited growth in the absence of exogenous growth factors and inhibited SCF-mediated growth but had no effect on IGF-I-mediated growth at the expression levels attained.  Expression of CA Akt markedly augmented growth in the absence of exogenous growth factors but had minimal effect on growth in the presence of satg. concns. SCF or IGF-I.  Because PI3K-Akt signaling is known to promote survival under apoptotic stresses, we detd. the effect of this pathway on SCLC sensitivity to etoposide.  LY294002 potentiated the effect of low concns. of etoposide in inhibiting growth and inducing apoptosis.  The effect of low concns. of LY294002 could largely be reversed by expression of CA Akt, suggesting that it was mediated by inhibition of Akt signaling.  Expression of CA Akt by itself also induced resistance to etoposide-mediated apoptosis.  Taken together, these data demonstrate that PI3K-Akt signaling promotes SCLC growth, survival, and chemotherapy resistance.  Therefore, selective inhibitors of PI3K or Akt could potentially be useful as novel therapeutic agents in the treatment of SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7p1hRSeRxJrVg90H21EOLACvtfcHk0lifxVOBEBtJ5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFOnsw%253D%253D&md5=16a348dbb16be9354417f47c07087276</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrystal%26aufirst%3DG.%2BW.%26aulast%3DSulanke%26aufirst%3DG.%26aulast%3DLitz%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520Phosphatidylinositol%25203-Kinase-Akt%2520Signaling%2520Blocks%2520Growth%252C%2520Promotes%2520Apoptosis%252C%2520and%2520Enhances%2520Sensitivity%2520of%2520Small%2520Cell%2520Lung%2520Cancer%2520Cells%2520to%2520Chemotherapy%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26date%3D2002%26volume%3D1%26spage%3D913%26epage%3D922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maulik, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broderick, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kijima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salgia, R.</span></span> <span> </span><span class="NLM_article-title">Modulation of c-Kit/SCF Pathway Leads to Alterations in Topoisomerase-I Activity in Small Cell Lung Cancer</span>. <i>J. Environ. Pathol., Toxicol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">251</span>, <span class="refDoi"> DOI: 10.1615/JEnvPathToxOncol.v23.i4.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1615%2FJEnvPathToxOncol.v23.i4.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=15511212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGgurbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=237-251&issue=4&author=G.+Maulikauthor=A.+Bhartiauthor=E.+Khanauthor=R.+J.+Broderickauthor=T.+Kijimaauthor=R.+Salgia&title=Modulation+of+c-Kit%2FSCF+Pathway+Leads+to+Alterations+in+Topoisomerase-I+Activity+in+Small+Cell+Lung+Cancer&doi=10.1615%2FJEnvPathToxOncol.v23.i4.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of c-Kit/SCF pathway leads to alterations in topoisomerase-I activity in small cell lung cancer</span></div><div class="casAuthors">Maulik, Gautam; Bharti, Ajit; Khan, Ehsan; Broderick, Ryan J.; Kijima, Takashi; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Environmental Pathology, Toxicology and Oncology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">237-251</span>CODEN:
                <span class="NLM_cas:coden">JEPOEC</span>;
        ISSN:<span class="NLM_cas:issn">0731-8898</span>.
    
            (<span class="NLM_cas:orgname">Begell House, Inc.</span>)
        </div><div class="casAbstract">Small cell lung cancer (SCLC) is an aggressive type of lung cancer, for which cytotoxic chemotherapy appears to have reached its maximal efficacy.  This neoplasm is characterized by the overexpression of several receptor tyrosine kinases (RTKs), esp. c-Kit.  The ligand for c-Kit is stem cell factor (SCF).  In SCLC, SCF can influence c-Kit activation by autocrine or paracrine mechanisms.  We have recently shown that the c-Kit/SCF pathway is operational in SCLC and can be inhibited by Glivec (STI571).  Because the inhibition of topoisomerase-I (topo-I) is one approach used to treat SCLC, we detd. the effects of c-Kit/SCF signaling on topo-I activity.  A unique phosphorylation of c-Kit on amino acid 823 and amino acid 703 was identified with the SCF stimulation of H526 cells.  We demonstrate that with SCF stimulation over 16 h (dose response 0-100 ng/mL) in H526 SCLC cells (c-Kit pos., SCF responsive), a decrease in topo-1 activity was obsd., whereas in H82 SCLC cells (c-Kit neg., SCF unresponsive) there was no modulation of topo-I activity by SCF.  Using STI571 (5 μM, 16 h) to inhibit the c-Kit pathway following stimulation with SCF (100 ng/mL), an upregulation of topo-I activity was obsd. in H526 cells but not in H82 cells.  Performing viability assays, we show that STI571 in combination with topo-I inhibition by camptothecin or SN38, the active metabolite of irinotecan, can cooperatively inhibit H526 cell viability (but not H82 cell viability) for 72 h.  We also show that STI571 does not directly inhibit topo-I activity in SCLC.  The combination of STI571 with topo-I inhibition could provide a useful combination in the treatment of SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv2FEqUNhT17Vg90H21EOLACvtfcHk0lj641m0AXqbxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGgurbN&md5=73b075276d2e5b4b79a9b98c593b3b89</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1615%2FJEnvPathToxOncol.v23.i4.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1615%252FJEnvPathToxOncol.v23.i4.10%26sid%3Dliteratum%253Aachs%26aulast%3DMaulik%26aufirst%3DG.%26aulast%3DBharti%26aufirst%3DA.%26aulast%3DKhan%26aufirst%3DE.%26aulast%3DBroderick%26aufirst%3DR.%2BJ.%26aulast%3DKijima%26aufirst%3DT.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DModulation%2520of%2520c-Kit%252FSCF%2520Pathway%2520Leads%2520to%2520Alterations%2520in%2520Topoisomerase-I%2520Activity%2520in%2520Small%2520Cell%2520Lung%2520Cancer%26jtitle%3DJ.%2520Environ.%2520Pathol.%252C%2520Toxicol.%2520Oncol.%26date%3D2004%26volume%3D23%26issue%3D4%26spage%3D237%26epage%3D251%26doi%3D10.1615%2FJEnvPathToxOncol.v23.i4.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGraff, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, J. B.</span></span> <span> </span><span class="NLM_article-title">Evaluation of a Tetrazolium-based Semiautomated Colorimetric Assay: Assessment of Chemosensitivity Testing</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">936</span>– <span class="NLM_lpage">942</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=3802100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaL2sXhsVyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1987&pages=936-942&issue=4&author=J.+Carmichaelauthor=W.+J.+DeGraffauthor=A.+F.+Gazdarauthor=J.+D.+Minnaauthor=J.+B.+Mitchell&title=Evaluation+of+a+Tetrazolium-based+Semiautomated+Colorimetric+Assay%3A+Assessment+of+Chemosensitivity+Testing"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of a tetrazolium-based semiautomated colorimetric assay:  assessment of chemosensitivity testing</span></div><div class="casAuthors">Carmichael, James; DeGraff, William G.; Gazdar, Adi F.; Minna, John D.; Mitchell, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">936-42</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">Drug sensitivity assays were performed using a variation of a colorimetric thiazolyl blue (MTT)  [298-93-1] assay on V79, CHO-AuxB1, CHRC5, NCI-H460, and NCI-H249 cell lines following optimization of exptl. conditions for each cell line.  Results from this assay were compared with data assimilated simultaneously by clonogenic assay and by dye exclusion assay.  Good correlation was obsd. using the CHO-AuxB1 cell line and the pleiotropic drug-resistant mutant CHRC5, with similar degrees of relative resistance obsd. with both the MTT and clonogenic assays.  Good correlation was obsd. between the clonogenic and MTT assays for 1-h drug exposures, although the MTT assay was more sensitive to vinblastine.  In general, the clonogenic assay was more sensitive when continuous drug exposures were utilized, although this was primarily related to the increased drug exposure time.  While the use of the MTT assay in drug sensitivity testing of primary tumor samples is limited, since contaminating normal cells may also reduce the tetrazolium, the MTT assay can be semiautomated, and therefore it offers a valid, simple method of assessing chemosensitivity in established cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj92XstI7WtLVg90H21EOLACvtfcHk0lj641m0AXqbxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXhsVyrtb0%253D&md5=2414230269c9f4d5f2910764c65fc042</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDeGraff%26aufirst%3DW.%2BJ.%26aulast%3DGazdar%26aufirst%3DA.%2BF.%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%26atitle%3DEvaluation%2520of%2520a%2520Tetrazolium-based%2520Semiautomated%2520Colorimetric%2520Assay%253A%2520Assessment%2520of%2520Chemosensitivity%2520Testing%26jtitle%3DCancer%2520Res.%26date%3D1987%26volume%3D47%26issue%3D4%26spage%3D936%26epage%3D942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcaro, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Receptor Tyrosine Kinase Signalling in Small Cell Lung Cancer (SCLC): What have we learned so far?</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2007.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.ctrv.2007.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=17368733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsF2rtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2007&pages=391-406&issue=4&author=B.+Fischerauthor=M.+Marinovauthor=A.+Arcaro&title=Targeting+Receptor+Tyrosine+Kinase+Signalling+in+Small+Cell+Lung+Cancer+%28SCLC%29%3A+What+have+we+learned+so+far%3F&doi=10.1016%2Fj.ctrv.2007.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting receptor tyrosine kinase signalling in small cell lung cancer (SCLC): What have we learned so far?</span></div><div class="casAuthors">Fischer, Barbara; Marinov, Marin; Arcaro, Alexandre</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">391-406</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Small cell lung cancer (SCLC) is an aggressive form of lung cancer, which represents 13% of all cases and is strongly assocd. with cigarette smoking.  The survival of SCLC patients is dismal and has not greatly improved in the last 20 years, despite advances in chemotherapy regimens and a better understanding of SCLC biol.  The development of resistance to chemotherapy and metastasis are commonly recognized as important causes of poor clin. outcome in SCLC.  Targeting receptor tyrosine kinase (RTK) signalling represents an attractive approach to develop new drugs for SCLC, in view of the accumulating data demonstrating that polypeptide growth factors play a key role in driving SCLC cell proliferation, chemoresistance and metastasis.  The insulin-like growth factor-I receptor (IGF-IR), c-Kit, vascular endothelial growth factor receptor (VEGFR) and epidermal growth factor receptor (EGFR) have been identified as potential drug targets in SCLC.  Moreover, downstream signalling mediators of RTKs, such as phosphoinositide 3-kinase (PI3K)/Akt and the mammalian target of rapamycin (mTOR) may also represent attractive candidate mols. for anti-cancer therapies in SCLC.  Here we will review the available data concerning results with RTK inhibitors in SCLC and the clin. trials undertaken to investigate the potential of these compds. as anti-tumor agents in SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKsJGB0yAQzbVg90H21EOLACvtfcHk0lj641m0AXqbxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsF2rtrg%253D&md5=e3572a7e560515b1c608d0f3d8fb105c</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2007.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2007.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DB.%26aulast%3DMarinov%26aufirst%3DM.%26aulast%3DArcaro%26aufirst%3DA.%26atitle%3DTargeting%2520Receptor%2520Tyrosine%2520Kinase%2520Signalling%2520in%2520Small%2520Cell%2520Lung%2520Cancer%2520%2528SCLC%2529%253A%2520What%2520have%2520we%2520learned%2520so%2520far%253F%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2007%26volume%3D33%26issue%3D4%26spage%3D391%26epage%3D406%26doi%3D10.1016%2Fj.ctrv.2007.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sekido, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zsebo, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, T.</span></span> <span> </span><span class="NLM_article-title">Recombinant Human Stem Cell Factor Mediates Chemotaxis of Small-Cell Lung Cancer Cell Lines Aberrantly Expressing the c-Kit Protooncogene</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1709</span>– <span class="NLM_lpage">1714</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=7680956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK3sXisFansro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1993&pages=1709-1714&author=Y.+Sekidoauthor=T.+Takahashiauthor=R.+Uedaauthor=M.+Takahashiauthor=H.+Suzukiauthor=K.+Nishidaauthor=T.+Tsukamotoauthor=T.+Hidaauthor=K.+Shimokataauthor=K.+M.+Zseboauthor=T.+Takahashi&title=Recombinant+Human+Stem+Cell+Factor+Mediates+Chemotaxis+of+Small-Cell+Lung+Cancer+Cell+Lines+Aberrantly+Expressing+the+c-Kit+Protooncogene"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Recombinant human stem cell factor mediates chemotaxis of small-cell lung cancer cell lines aberrantly expressing the c-kit protooncogene</span></div><div class="casAuthors">Sekido, Yoshitaka; Takahashi, Takashi; Ueda, Ryuzo; Takahashi, Masahide; Suzuki, Hiroko; Nishida, Keiko; Tsukamoto, Tetsuya; Hida, Toyoaki; Shimokata, Kaoru; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1709-14</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">Accumulating evidence suggests that c-kit and its ligand, stem cell factor (SCF), play an important role in the regulation of growth in at least three lineages of stem cell and possibly in leukemogenesis, while only limited data are available that suggest possible involvement of c-kit/SCF in the development of human solid tumors such as lung cancer.  We have recently reported that c-kit is aberrantly expressed almost exclusively in small-cell lung cancer (SCLC) among various types of solid tumors.  The present study revealed that c-kit protein ectopically expressed in SCLC is indistinguishable from that in leukemia cell lines with megakaryocytic characteristics with respect to amt., mol. size, and autophosphorylation status in response to recombinant human SCF.  Furthermore, significant chemotactic response as well as moderate in vitro cell growth was induced in SCLC cell lines by the addn. of recombinant human SCF, suggesting that c-kit/SCF may play an important biol. role in the development of SCLC.  Our extensive search for activating mutations naturally occurring in the c-kit gene revealed an amino acid substitution in the transmembrane domain of an SCLC cell line, although the functional consequences of this variant allele are yet to be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9gtA-Ir-7LbVg90H21EOLACvtfcHk0ljeWHOW7DDWWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisFansro%253D&md5=44359e197f9907ee312d840a1a7e8d88</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSekido%26aufirst%3DY.%26aulast%3DTakahashi%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DSuzuki%26aufirst%3DH.%26aulast%3DNishida%26aufirst%3DK.%26aulast%3DTsukamoto%26aufirst%3DT.%26aulast%3DHida%26aufirst%3DT.%26aulast%3DShimokata%26aufirst%3DK.%26aulast%3DZsebo%26aufirst%3DK.%2BM.%26aulast%3DTakahashi%26aufirst%3DT.%26atitle%3DRecombinant%2520Human%2520Stem%2520Cell%2520Factor%2520Mediates%2520Chemotaxis%2520of%2520Small-Cell%2520Lung%2520Cancer%2520Cell%2520Lines%2520Aberrantly%2520Expressing%2520the%2520c-Kit%2520Protooncogene%26jtitle%3DCancer%2520Res.%26date%3D1993%26volume%3D53%26spage%3D1709%26epage%3D1714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Detailed Conformational Dynamics of Juxtamembrane Region and Activation Loop in c-Kit Kinase Activation Process</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>72</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1002/prot.21928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1002%2Fprot.21928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=18214972" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2008&pages=323-332&issue=1&author=J.+Zouauthor=Y.+Wangauthor=F.+Maauthor=M.+Xiangauthor=B.+Shiauthor=Y.+Weiauthor=S.+Yang&title=Detailed+Conformational+Dynamics+of+Juxtamembrane+Region+and+Activation+Loop+in+c-Kit+Kinase+Activation+Process&doi=10.1002%2Fprot.21928"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1002%2Fprot.21928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21928%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DF.%26aulast%3DXiang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DDetailed%2520Conformational%2520Dynamics%2520of%2520Juxtamembrane%2520Region%2520and%2520Activation%2520Loop%2520in%2520c-Kit%2520Kinase%2520Activation%2520Process%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2008%26volume%3D72%26issue%3D1%26spage%3D323%26epage%3D332%26doi%3D10.1002%2Fprot.21928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollok, B. A.</span></span> <span> </span><span class="NLM_article-title">Developing Assays for Kinase Drug Discovery-Where have the Advances Come From?</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1517/17460441.3.1.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1517%2F17460441.3.1.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23480143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVGntLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=115-129&issue=1&author=K.+W.+Vogelauthor=Z.+Zhongauthor=K.+Biauthor=B.+A.+Pollok&title=Developing+Assays+for+Kinase+Drug+Discovery-Where+have+the+Advances+Come+From%3F&doi=10.1517%2F17460441.3.1.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Developing assays for kinase drug discovery - where have the advances come from?</span></div><div class="casAuthors">Vogel, Kurt W.; Zhong, Zhong; Bi, Kun; Pollok, Brian A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115-129</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Over the past decade, a variety of technologies for the identification and characterization of protein kinase inhibitors have been implemented in the labs. of nearly every major pharmaceutical and biotechnol. company.  Although the majority of these assay technologies are highly robust, the ability of many assays to identify compds. that target the kinase of interest in a true biol. context remains questionable.  Because every in vitro assay represents a trade-off between biol. relevancy and factors such as cost, throughput and accuracy, it is important to acknowledge and balance these trade-offs when interrogating a kinase target in such an assay.  This review addresses some of the factors that should be considered when developing protein kinase assays, as well as strategies used to address those factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo21WVdpzY72rVg90H21EOLACvtfcHk0ljeWHOW7DDWWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVGntLw%253D&md5=f4036599a04f9523175983e132f6e2ee</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1517%2F17460441.3.1.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.3.1.115%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DK.%2BW.%26aulast%3DZhong%26aufirst%3DZ.%26aulast%3DBi%26aufirst%3DK.%26aulast%3DPollok%26aufirst%3DB.%2BA.%26atitle%3DDeveloping%2520Assays%2520for%2520Kinase%2520Drug%2520Discovery-Where%2520have%2520the%2520Advances%2520Come%2520From%253F%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2008%26volume%3D3%26issue%3D1%26spage%3D115%26epage%3D129%26doi%3D10.1517%2F17460441.3.1.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connell, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Protein-Tyrosine Kinase Activity in Intact Cells by the Aptameric Action of Oligodeoxynucleotides</span>. <i>Antisense Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1089/ard.1995.5.33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Fard.1995.5.33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=7542047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADyaK2MXlsV2it78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1995&pages=33-38&issue=1&author=R.+C.+Berganauthor=E.+Kyleauthor=Y.+Connellauthor=L.+Neckers&title=Inhibition+of+Protein-Tyrosine+Kinase+Activity+in+Intact+Cells+by+the+Aptameric+Action+of+Oligodeoxynucleotides&doi=10.1089%2Fard.1995.5.33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of protein-tyrosine kinase activity in intact cells by the aptameric action of oligodeoxynucleotides</span></div><div class="casAuthors">Bergan, Raymond C.; Kyle, Edward; Connell, Yvette; Neckers, Len</div><div class="citationInfo"><span class="NLM_cas:title">Antisense Research and Development</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8</span>CODEN:
                <span class="NLM_cas:coden">AREDEI</span>;
        ISSN:<span class="NLM_cas:issn">1050-5261</span>.
    
            (<span class="NLM_cas:orgname">Liebert</span>)
        </div><div class="casAbstract">Direct interaction of oligodeoxynucleotides (ODNs) with proteins represents one of the nonantisense-mediated effects of ODNs.  Phosphorothioate-capped ODNs have been shown to inhibit directly the in vitro kinase activity of the chronic myelogenous leukemia-assocd. protein kinase p210bcr-abl.  In this study we have detd. the efficacy of this aptameric ODN in a cellular system using the K562 chronic myelogenous leukemia-derived cell line.  Significant effects upon cellular phosphotyrosine content, as well as cellular growth in soft agar, are obsd.  These effects are sequence specific and are not mediated through changes in p210bcr-abl protein levels.  Addnl. ODNs are described that also reduce cellular phosphotyrosine levels and inhibit growth in soft agar but do not inhibit p210bcr-abl kinase activity in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrevsJE3XttKrVg90H21EOLACvtfcHk0ljeWHOW7DDWWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlsV2it78%253D&md5=b7e4d62f3e603a0459dbc2e664685c5e</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1089%2Fard.1995.5.33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fard.1995.5.33%26sid%3Dliteratum%253Aachs%26aulast%3DBergan%26aufirst%3DR.%2BC.%26aulast%3DKyle%26aufirst%3DE.%26aulast%3DConnell%26aufirst%3DY.%26aulast%3DNeckers%26aufirst%3DL.%26atitle%3DInhibition%2520of%2520Protein-Tyrosine%2520Kinase%2520Activity%2520in%2520Intact%2520Cells%2520by%2520the%2520Aptameric%2520Action%2520of%2520Oligodeoxynucleotides%26jtitle%3DAntisense%2520Res.%2520Dev.%26date%3D1995%26volume%3D5%26issue%3D1%26spage%3D33%26epage%3D38%26doi%3D10.1089%2Fard.1995.5.33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazi, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rönnstrand, L.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of the Activation Loop Tyrosine 823 in c-Kit Is Crucial for Cell Survival and Proliferation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">22460</span>– <span class="NLM_lpage">22468</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.474072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1074%2Fjbc.M113.474072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=23803604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Whtr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=22460-22468&issue=31&author=S.+Agarwalauthor=J.+U.+Kaziauthor=L.+R%C3%B6nnstrand&title=Phosphorylation+of+the+Activation+Loop+Tyrosine+823+in+c-Kit+Is+Crucial+for+Cell+Survival+and+Proliferation&doi=10.1074%2Fjbc.M113.474072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of the activation loop tyrosine 823 in c-Kit Is Crucial for Cell Survival and Proliferation</span></div><div class="casAuthors">Agarwal, Shruti; Kazi, Julhash U.; Roennstrand, Lars</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">22460-22468</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The receptor tyrosine kinase c-Kit, also known as the stem cell factor receptor, plays a key role in several developmental processes.  Activating mutations in c-Kit lead to alteration of these cellular processes and have been implicated in many human cancers such as gastrointestinal stromal tumors, acute myeloid leukemia, testicular seminomas and mastocytosis.  Regulation of the catalytic activity of several kinases is known to be governed by phosphorylation of tyrosine residues in the activation loop of the kinase domain.  However, in the case of c-Kit phosphorylation of Tyr-823 has been demonstrated to be a late event that is not required for kinase activation.  However, because phosphorylation of Tyr-823 is a ligand-activated event, we sought to investigate the functional consequences of Tyr-823 phosphorylation.  By using a tyrosine-to-phenylalanine mutant of tyrosine 823, we investigated the impact of Tyr-823 on c-Kit signaling.  We demonstrate here that Tyr-823 is crucial for cell survival and proliferation and that mutation of Tyr-823 to phenylalanine leads to decreased sustained phosphorylation and ubiquitination of c-Kit as compared with the wild-type receptor.  Furthermore, the mutated receptor was, upon ligand-stimulation, quickly internalized and degraded.  Phosphorylation of the E3 ubiquitin ligase Cbl was transient, followed by a substantial redn. in phosphorylation of downstream signaling mols. such as Akt, Erk, p38, Shc, and Gab2.  Thus, we propose that activation loop tyrosine 823 is crucial for activation of both the MAPK and PI3K pathways and that its disruption leads to a destabilization of the c-Kit receptor and decreased survival of cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodqwzOEtfdkrVg90H21EOLACvtfcHk0ljwd4yvXMabcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Whtr7L&md5=e8c59e575709ed8a4137edfcb989722d</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.474072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.474072%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwal%26aufirst%3DS.%26aulast%3DKazi%26aufirst%3DJ.%2BU.%26aulast%3DR%25C3%25B6nnstrand%26aufirst%3DL.%26atitle%3DPhosphorylation%2520of%2520the%2520Activation%2520Loop%2520Tyrosine%2520823%2520in%2520c-Kit%2520Is%2520Crucial%2520for%2520Cell%2520Survival%2520and%2520Proliferation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26issue%3D31%26spage%3D22460%26epage%3D22468%26doi%3D10.1074%2Fjbc.M113.474072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazi, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohlin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Påhlman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rönnstrand, L.</span></span> <span> </span><span class="NLM_article-title">The Activation Loop Tyrosine 823 is Essential for the Transforming Capacity of the c-Kit Oncogenic Mutant D816V</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">35</span>),  <span class="NLM_fpage">4581</span>– <span class="NLM_lpage">4590</span>, <span class="refDoi"> DOI: 10.1038/onc.2014.383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1038%2Fonc.2014.383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=25435369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVGlsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=4581-4590&issue=35&author=S.+Agarwalauthor=J.+U.+Kaziauthor=S.+Mohlinauthor=S.+P%C3%A5hlmanauthor=L.+R%C3%B6nnstrand&title=The+Activation+Loop+Tyrosine+823+is+Essential+for+the+Transforming+Capacity+of+the+c-Kit+Oncogenic+Mutant+D816V&doi=10.1038%2Fonc.2014.383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">The activation loop tyrosine 823 is essential for the transforming capacity of the c-Kit oncogenic mutant D816V</span></div><div class="casAuthors">Agarwal, S.; Kazi, J. U.; Mohlin, S.; Paahlman, S.; Roennstrand, L.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">4581-4590</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oncogenic c-Kit mutations have been shown to display ligand-independent receptor activation and cell proliferation.  A substitution of aspartate to valine at amino acid 816 (D816V) is one of the most commonly found oncogenic c-Kit mutations and is found in >90% of cases of mastocytosis and less commonly in germ-cell tumors, core-binding factor acute myeloid leukemia and mucosal melanomas.  The mechanisms by which this mutation leads to constitutive activation and transformation are not fully understood.  Previous studies have shown that the D816V mutation causes a structural change in the activation loop (A-loop), resulting in weaker binding of the A-loop to the juxtamembrane domain.  In this paper, the authors have investigated the role of Y823, the only tyrosine residue in the A-loop, and its role in oncogenic transformation by c-Kit/D816V by introducing the Y823F mutation.  Although dispensable for the kinase activity of c-Kit/D816V, the presence of Y823 was crucial for cell proliferation and survival.  Furthermore, mutation of Y823 selectively downregulates the Ras/Erk and Akt pathways as well as the phosphorylation of STAT5 and reduces the transforming capacity of the D816V/c-Kit in vitro.  The authors further show that mice injected with cells expressing c-Kit/D816V/Y823F display significantly reduced tumor size as well as tumor wt. compared with controls.  Finally, microarray anal., comparing Y823F/D816V cells with cells expressing c-Kit/D816V, demonstrate that mutation of Y823 causes upregulation of proapoptotic genes, whereas genes of survival pathways are downregulated.  Thus, phosphorylation of Y823 is not necessary for kinase activation, but essential for the transforming ability of the c-Kit/D816V mutant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqntC9Ge4dmtbVg90H21EOLACvtfcHk0ljwd4yvXMabcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVGlsL%252FO&md5=c7a7838ab528b094b441da7035bdeebc</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.383%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwal%26aufirst%3DS.%26aulast%3DKazi%26aufirst%3DJ.%2BU.%26aulast%3DMohlin%26aufirst%3DS.%26aulast%3DP%25C3%25A5hlman%26aufirst%3DS.%26aulast%3DR%25C3%25B6nnstrand%26aufirst%3DL.%26atitle%3DThe%2520Activation%2520Loop%2520Tyrosine%2520823%2520is%2520Essential%2520for%2520the%2520Transforming%2520Capacity%2520of%2520the%2520c-Kit%2520Oncogenic%2520Mutant%2520D816V%26jtitle%3DOncogene%26date%3D2015%26volume%3D34%26issue%3D35%26spage%3D4581%26epage%3D4590%26doi%3D10.1038%2Fonc.2014.383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartig, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Restle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grohmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krausslich, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famulok, M.</span></span> <span> </span><span class="NLM_article-title">Aptamer Displacement Identifies Alternative Small-Molecule Target Sites that Escape Viral Resistance</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">804</span>– <span class="NLM_lpage">812</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2007.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.chembiol.2007.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=17656317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotlSnu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=804-812&issue=7&author=S.+Yamazakiauthor=L.+Tanauthor=G.+Mayerauthor=J.+S.+Hartigauthor=J.-N.+Songauthor=S.+Reuterauthor=T.+Restleauthor=S.+D.+Lauferauthor=D.+Grohmannauthor=H.-G.+Krausslichauthor=J.+Bajorathauthor=M.+Famulok&title=Aptamer+Displacement+Identifies+Alternative+Small-Molecule+Target+Sites+that+Escape+Viral+Resistance&doi=10.1016%2Fj.chembiol.2007.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamer displacement identifies alternative small-molecule target sites that escape viral resistance</span></div><div class="casAuthors">Yamazaki, Satoko; Tan, Lu; Mayer, Guenter; Hartig, Joerg S.; Song, Jin-Na; Reuter, Sandra; Restle, Tobias; Laufer, Sandra D.; Grohmann, Dina; Kraeusslich, Hans-Georg; Bajorath, Juergen; Famulok, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">804-812</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Aptamers targeting reverse transcriptase (RT) from HIV-1 inhibit viral replication in vitro, presumably by competing with binding of the primer/template complex.  This site is not targeted by the currently available small-mol. anti-HIV-1 RT inhibitors.  We have identified SY-3E4, a small-mol. inhibitor of HIV-1 RT, by applying a screening assay that utilizes a reporter-ribozyme regulated by the anti-HIV-1 RT aptamer.  SY-3E4 displaces the aptamer from the protein, selectively inhibits DNA-dependent, but not RNA-dependent, polymerase activity, and inhibits the replication of both the wild-type virus and a multidrug-resistant strain.  Anal. of available structural data of HIV-1 and HIV-2 RTs rationalizes many of the obsd. characteristics of the inhibitory profiles of SY-3E4 and the aptamer and suggests a previously not considered region in these RTs as a target for antiviral therapy.  Our study reveals unexplored ways for rapidly identifying alternative small-mol. target sites in proteins and illustrates strategies for overcoming resistance-conferring mutations with small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeHcTBKxon8bVg90H21EOLACvtfcHk0ljldxmIoGH-ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotlSnu7w%253D&md5=2f60fcb348a9016c75f0488787f43968</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2007.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2007.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DHartig%26aufirst%3DJ.%2BS.%26aulast%3DSong%26aufirst%3DJ.-N.%26aulast%3DReuter%26aufirst%3DS.%26aulast%3DRestle%26aufirst%3DT.%26aulast%3DLaufer%26aufirst%3DS.%2BD.%26aulast%3DGrohmann%26aufirst%3DD.%26aulast%3DKrausslich%26aufirst%3DH.-G.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DAptamer%2520Displacement%2520Identifies%2520Alternative%2520Small-Molecule%2520Target%2520Sites%2520that%2520Escape%2520Viral%2520Resistance%26jtitle%3DChem.%2520Biol.%26date%3D2007%26volume%3D14%26issue%3D7%26spage%3D804%26epage%3D812%26doi%3D10.1016%2Fj.chembiol.2007.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ababneh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alshaer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allozi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahafzah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Khateeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillaireau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noiray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattal, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, S.</span></span> <span> </span><span class="NLM_article-title">In Vitro Selection of Modified RNA Aptamers Against CD44 Cancer Stem Cell Marker</span>. <i>Nucleic Acid Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1089/nat.2013.0423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1089%2Fnat.2013.0423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=24171482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFaksLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=401-407&issue=6&author=N.+Ababnehauthor=W.+Alshaerauthor=O.+Alloziauthor=A.+Mahafzahauthor=M.+El-Khateebauthor=H.+Hillaireauauthor=M.+Noirayauthor=E.+Fattalauthor=S.+Ismail&title=In+Vitro+Selection+of+Modified+RNA+Aptamers+Against+CD44+Cancer+Stem+Cell+Marker&doi=10.1089%2Fnat.2013.0423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro Selection of Modified RNA Aptamers Against CD44 Cancer Stem Cell Marker</span></div><div class="casAuthors">Ababneh, Nidaa; Alshaer, Walhan; Allozi, Omar; Mahafzah, Azmi; El-Khateeb, Mohammed; Hillaireau, Herve; Noiray, Magali; Fattal, Elias; Ismail, Said</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acid Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-407</span>CODEN:
                <span class="NLM_cas:coden">NATUCU</span>;
        ISSN:<span class="NLM_cas:issn">2159-3337</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Cancer stem cells (CSCs) are a subset of tumor cells that has the ability to self-renew and to generate the diverse cells that comprise the tumor mass.  The cell-surface glycoprotein CD44 is one of the most common surface markers used to identify CSCs.  Aptamers are synthetic oligonucleotides selected from pools of random sequences that can bind to a wide range of targets with high affinity and specificity.  In this study, the systematic evolution of ligands by exponential enrichment (SELEX) technol. was used to isolate RNA aptamers using human recombinant full-length CD44 protein and 2'-F-pyrimidine modified RNA library with a complexity of around 1014 different mols.  Following 11 iterative rounds of SELEX, the selected aptamers were cloned and sequenced.  Three different sequences were identified.  The binding specificities for one of these RNA aptamers was assessed using representative breast cancer cell lines expressing CD44; namely, MDA-MB-231, MCF7, and T47D.  The selected RNA aptamer (Apt1) was found to interact specifically with such cancer cells when analyzed by flow cytometry and fluorescent microscopy, with different intensities of fluorescence reflecting the level of CD44 expression on the surface of these cells.  It can be concluded that the selected aptamers can be used to target CD44 pos. cells, including cancer stem cells, for detection, sorting, and enrichment and for drug delivery purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTuM3lAiwtS7Vg90H21EOLACvtfcHk0ljldxmIoGH-ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFaksLbM&md5=c1861790a4352a86d5715ce1852d8e33</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1089%2Fnat.2013.0423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fnat.2013.0423%26sid%3Dliteratum%253Aachs%26aulast%3DAbabneh%26aufirst%3DN.%26aulast%3DAlshaer%26aufirst%3DW.%26aulast%3DAllozi%26aufirst%3DO.%26aulast%3DMahafzah%26aufirst%3DA.%26aulast%3DEl-Khateeb%26aufirst%3DM.%26aulast%3DHillaireau%26aufirst%3DH.%26aulast%3DNoiray%26aufirst%3DM.%26aulast%3DFattal%26aufirst%3DE.%26aulast%3DIsmail%26aufirst%3DS.%26atitle%3DIn%2520Vitro%2520Selection%2520of%2520Modified%2520RNA%2520Aptamers%2520Against%2520CD44%2520Cancer%2520Stem%2520Cell%2520Marker%26jtitle%3DNucleic%2520Acid%2520Ther.%26date%3D2013%26volume%3D23%26issue%3D6%26spage%3D401%26epage%3D407%26doi%3D10.1089%2Fnat.2013.0423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kikin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantonio, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagga, P. S.</span></span> <span> </span><span class="NLM_article-title">QGRS Mapper: A Web-based Server for Predicting G-quadruplexes in Nucleotide Sequences</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">W676</span>– <span class="NLM_lpage">W682</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1093%2Fnar%2Fgkl253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=16845096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yhsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=W676-W682&author=O.+Kikinauthor=L.+Dantonioauthor=P.+S.+Bagga&title=QGRS+Mapper%3A+A+Web-based+Server+for+Predicting+G-quadruplexes+in+Nucleotide+Sequences&doi=10.1093%2Fnar%2Fgkl253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences</span></div><div class="casAuthors">Kikin, Oleg; D'Antonio, Lawrence; Bagga, Paramjeet S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W676-W682</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The quadruplex structures formed by guanine-rich nucleic acid sequences have received significant attention recently because of growing evidence for their role in important biol. processes and as therapeutic targets.  G-quadruplex DNA has been suggested to regulate DNA replication and may control cellular proliferation.  Sequences capable of forming G-quadruplexes in the RNA have been shown to play significant roles in regulation of polyadenylation and splicing events in mammalian transcripts.  Whether quadruplex structure directly plays a role in regulating RNA processing requires investigation.  Computational approaches to study G-quadruplexes allow detailed anal. of mammalian genomes.  There are no known easily accessible user-friendly tools that can compute G-quadruplexes in the nucleotide sequences.  We have developed a web-based server, QGRS Mapper, that predicts quadruplex forming G-rich sequences (QGRS) in nucleotide sequences.  It is a user-friendly application that provides many options for defining and studying G-quadruplexes.  It performs anal. of the user provided genomic sequences, e.g. promoter and telomeric regions, as well as RNA sequences.  It is also useful for predicting G-quadruplex structures in oligonucleotides.  The program provides options to search and retrieve desired gene/nucleotide sequence entries from NCBI databases for mapping G-quadruplexes in the context of RNA processing sites.  This feature is very useful for investigating the functional relevance of G-quadruplex structure, in particular its role in regulating the gene expression by alternative processing.  In addn. to providing data on compn. and locations of QGRS relative to the processing sites in the pre-mRNA sequence, QGRS Mapper features interactive graphic representation of the data.  The user can also use the graphics module to visualize QGRS distribution patterns among all the alternative RNA products of a gene simultaneously on a single screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_yCz4ebPV-bVg90H21EOLACvtfcHk0ljldxmIoGH-ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1yhsLY%253D&md5=e8f69845c3df9217df7f346a06c3db44</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl253%26sid%3Dliteratum%253Aachs%26aulast%3DKikin%26aufirst%3DO.%26aulast%3DDantonio%26aufirst%3DL.%26aulast%3DBagga%26aufirst%3DP.%2BS.%26atitle%3DQGRS%2520Mapper%253A%2520A%2520Web-based%2520Server%2520for%2520Predicting%2520G-quadruplexes%2520in%2520Nucleotide%2520Sequences%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW676%26epage%3DW682%26doi%3D10.1093%2Fnar%2Fgkl253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quang, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestourie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cibiel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducongé, F.</span></span> <span> </span><span class="NLM_article-title">How to Measure the Affinity of Aptamers for Membrane Proteins Expressed on the Surface of Living Adherent Cells</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2015.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.ymeth.2015.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=26589188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyku7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2016&pages=35-43&author=N.+N.+Quangauthor=C.+Pestourieauthor=A.+Cibielauthor=F.+Ducong%C3%A9&title=How+to+Measure+the+Affinity+of+Aptamers+for+Membrane+Proteins+Expressed+on+the+Surface+of+Living+Adherent+Cells&doi=10.1016%2Fj.ymeth.2015.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">How to measure the affinity of aptamers for membrane proteins expressed on the surface of living adherent cells</span></div><div class="casAuthors">Quang, Nam Nguyen; Pestourie, Carine; Cibiel, Agnes; Duconge, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-43</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recently, an increasing no. of aptamers have been selected against biomarkers that are expressed at the surface of cells.  This class of targets, mostly membrane proteins, is in close contact with the intra- and extra-cellular matrixes and their three-dimensional structures are inextricably linked to their inclusion in lipid bilayers.  Therefore, although binding studies can be performed on the isolated form of these proteins, it remains crucial to measure the affinity of these aptamers in a more physiol. environment, i.e., directly on living cells.  Here, we describe a procedure for radioactive binding assays that can be adapted for measuring the affinity of aptamers against different cell lines.  This method has been semi-automated using a liq. handling robot in order to reproducibly measure the apparent dissocn. const. Kd and the apparent no. of targets per cell.  Relevant issues are discussed including the labeling of aptamers, the cells prepn., the incubation, the washings, the use of non-specific competitors, the data anal. and finally the reporting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoSnro5u8pFbVg90H21EOLACvtfcHk0ljldxmIoGH-ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyku7jF&md5=8e4c1f2acc96289f4705e46bfad19bbf</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2015.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2015.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DQuang%26aufirst%3DN.%2BN.%26aulast%3DPestourie%26aufirst%3DC.%26aulast%3DCibiel%26aufirst%3DA.%26aulast%3DDucong%25C3%25A9%26aufirst%3DF.%26atitle%3DHow%2520to%2520Measure%2520the%2520Affinity%2520of%2520Aptamers%2520for%2520Membrane%2520Proteins%2520Expressed%2520on%2520the%2520Surface%2520of%2520Living%2520Adherent%2520Cells%26jtitle%3DMethods%26date%3D2016%26volume%3D97%26spage%3D35%26epage%3D43%26doi%3D10.1016%2Fj.ymeth.2015.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanlikilicer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozpolat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aslan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayraktar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbuz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Aguayo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayraktar, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denizli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Villasana, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokesh, G. L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catuogno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haemmerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorenstein, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volk, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franciscis, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Targeting of AXL Receptor Tyrosine Kinase Inhibits Tumor Growth and Intraperitoneal Metastasis in Ovarian Cancer Models</span>. <i>Mol. Ther.--Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1016/j.omtn.2017.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=10.1016%2Fj.omtn.2017.06.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=29246304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOnsLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=251-262&author=P.+Kanlikilicerauthor=B.+Ozpolatauthor=B.+Aslanauthor=R.+Bayraktarauthor=N.+Gurbuzauthor=C.+Rodriguez-Aguayoauthor=E.+Bayraktarauthor=M.+Denizliauthor=V.+Gonzalez-Villasanaauthor=C.+Ivanauthor=G.+L.+R.+Lokeshauthor=P.+Ameroauthor=S.+Catuognoauthor=M.+Haemmerleauthor=S.+Wuauthor=R.+Mitraauthor=D.+G.+Gorensteinauthor=D.+E.+Volkauthor=V.+D.+Franciscisauthor=A.+K.+Soodauthor=G.+Lopez-Berestein&title=Therapeutic+Targeting+of+AXL+Receptor+Tyrosine+Kinase+Inhibits+Tumor+Growth+and+Intraperitoneal+Metastasis+in+Ovarian+Cancer+Models&doi=10.1016%2Fj.omtn.2017.06.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Targeting of AXL Receptor Tyrosine Kinase Inhibits Tumor Growth and Intraperitoneal Metastasis in Ovarian Cancer Models</span></div><div class="casAuthors">Kanlikilicer, Pinar; Ozpolat, Bulent; Aslan, Burcu; Bayraktar, Recep; Gurbuz, Nilgun; Rodriguez-Aguayo, Cristian; Bayraktar, Emine; Denizli, Merve; Gonzalez-Villasana, Vianey; Ivan, Cristina; Lokesh, Ganesh L. R.; Amero, Paola; Catuogno, Silvia; Haemmerle, Monika; Wu, Sherry Yen-Yao; Mitra, Rahul; Gorenstein, David G.; Volk, David E.; de Franciscis, Vittorio; Sood, Anil K.; Lopez-Berestein, Gabriel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy--Nucleic Acids</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">251-262</span>CODEN:
                <span class="NLM_cas:coden">MTAOC5</span>;
        ISSN:<span class="NLM_cas:issn">2162-2531</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Despite substantial improvements in the treatment strategies, ovarian cancer is still the most lethal gynecol. malignancy.  Identification of drug treatable therapeutic targets and their safe and effective targeting is crit. to improve patient survival in ovarian cancer.  AXL receptor tyrosine kinase (RTK) has been proposed to be an important therapeutic target for metastatic and advanced-stage human ovarian cancer.  We found that AXL-RTK expression is assocd. with significantly shorter patient survival based on the The Cancer Genome Atlas patient database.  To target AXL-RTK, we developed a chem. modified serum nuclease-stable AXL aptamer (AXL-APTAMER), and we evaluated its in vitro and in vivo antitumor activity using in vitro assays as well as two i.p. animal models.  AXL-aptamer treatment inhibited the phosphorylation and the activity of AXL, impaired the migration and invasion ability of ovarian cancer cells, and led to the inhibition of tumor growth and no. of i.p. metastatic nodules, which was assocd. with the inhibition of AXL activity and angiogenesis in tumors.  When combined with paclitaxel, in vivo systemic (i.v. [i.v.]) administration of AXL-aptamer treatment markedly enhanced the antitumor efficacy of paclitaxel in mice.  Taken together, our data indicate that AXL-aptamers successfully target in vivo AXL-RTK and inhibit its AXL activity and tumor growth and progression, representing a promising strategy for the treatment of ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKE58FM_8OrbVg90H21EOLACvtfcHk0lgoE6YNDFxJMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOnsLvL&md5=40c844d3d267d23f198a94658b33babe</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.omtn.2017.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.omtn.2017.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DKanlikilicer%26aufirst%3DP.%26aulast%3DOzpolat%26aufirst%3DB.%26aulast%3DAslan%26aufirst%3DB.%26aulast%3DBayraktar%26aufirst%3DR.%26aulast%3DGurbuz%26aufirst%3DN.%26aulast%3DRodriguez-Aguayo%26aufirst%3DC.%26aulast%3DBayraktar%26aufirst%3DE.%26aulast%3DDenizli%26aufirst%3DM.%26aulast%3DGonzalez-Villasana%26aufirst%3DV.%26aulast%3DIvan%26aufirst%3DC.%26aulast%3DLokesh%26aufirst%3DG.%2BL.%2BR.%26aulast%3DAmero%26aufirst%3DP.%26aulast%3DCatuogno%26aufirst%3DS.%26aulast%3DHaemmerle%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DMitra%26aufirst%3DR.%26aulast%3DGorenstein%26aufirst%3DD.%2BG.%26aulast%3DVolk%26aufirst%3DD.%2BE.%26aulast%3DFranciscis%26aufirst%3DV.%2BD.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26atitle%3DTherapeutic%2520Targeting%2520of%2520AXL%2520Receptor%2520Tyrosine%2520Kinase%2520Inhibits%2520Tumor%2520Growth%2520and%2520Intraperitoneal%2520Metastasis%2520in%2520Ovarian%2520Cancer%2520Models%26jtitle%3DMol.%2520Ther.--Nucleic%2520Acids%26date%3D2017%26volume%3D9%26spage%3D251%26epage%3D262%26doi%3D10.1016%2Fj.omtn.2017.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PKG','PDB','1PKG'); return false;">PDB: 1PKG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1T45" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1T45','PDB','1T45'); return false;">PDB: 1T45</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i36"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00868">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_87502"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00868">10.1021/acs.jmedchem.9b00868</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplemental tables and figures: nucleotide sequences of all selected anti c-KIT<sup>WT</sup> aptamers, nucleotide sequences of all selected anti c-KIT<sup>D816H</sup> aptamers, nucleotide sequences of all selected anti c-KIT<sup>D816V</sup> aptamers, apparent <i>K</i><sub>m</sub> value for peptide substrate, apparent <i>K</i><sub>m</sub> values for ATP substrate, and antagonist inhibition curves of H30 aptamer (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00868/suppl_file/jm9b00868_si_001.pdf">jm9b00868_si_001.pdf (613.36 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00868&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-5%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00868" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00868" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b26137d40d1d7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
